<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Latent%20space%20simulators/">
      
      
        <link rel="next" href="../PINNs/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Parametrizing using ML
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Time-series%20forecasting/" class="md-tabs__link">
        
  
    
  
  Time-series forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Symbolic%20regression/" class="md-tabs__link">
        
  
    
  
  Symbolic regression

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Neural%20ODEs/" class="md-tabs__link">
        
  
    
  
  Neural ODEs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Physics-based%20GNNs/" class="md-tabs__link">
        
  
    
  
  Physics-based GNNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Latent%20space%20simulators/" class="md-tabs__link">
        
  
    
  
  Latent space simulators

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Parametrizing using ML

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../PINNs/" class="md-tabs__link">
        
  
    
  
  PINNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Koopman%20operator/" class="md-tabs__link">
        
  
    
  
  Koopman operator

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Time-series%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Time-series forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Symbolic%20regression/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Symbolic regression
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Neural%20ODEs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Neural ODEs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Physics-based%20GNNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Physics-based GNNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Latent%20space%20simulators/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Latent space simulators
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Parametrizing using ML
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../PINNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    PINNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Koopman%20operator/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Koopman operator
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Parametrizing using ML</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-05-27 07:04:19 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Parametrizing using ML</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Parametrizing using ML</a><br>
      <a href="#recommended_articles">3. Recommended articles on Parametrizing using ML</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Parametrizing using ML</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Parametrizing using ML</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e837965494c4edbec4d30832d31ba5639996da8" target='_blank'>
                Deep learning prediction of patient response time course from early data via neural-pharmacokinetic/pharmacodynamic modelling
                </a>
              </td>
          <td>
            James Lu, B. Bender, Jin Y. Jin, Y. Guan
          </td>
          <td>2020-10-22</td>
          <td>Nature Machine Intelligence</td>
          <td>31</td>
          <td>37</td>

            <td><a href='../recommendations/4e837965494c4edbec4d30832d31ba5639996da8' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Many real-world systems such as social networks and moving planets are dynamic in nature, where a set of coupled objects are connected via the interaction graph and exhibit complex behavior along the time. For example, the COVID-19 pandemic can be considered as a dynamical system, where objects represent geographical locations (e.g., states) whose daily confirmed cases of infection evolve over time. Outbreak at one location may influence another location as people travel between these locations, forming a graph. Thus, how to model and predict the complex dynamics for these systems becomes a critical research problem. Existing work on modeling graph-structured data mostly assumes a static setting. How to handle dynamic graphs remains to be further explored. On one hand, features of objects change over time, influenced by the linked objects in the interaction graph. On the other hand, the graph itself can also evolve, where new interactions (links) may form and existing links may drop, which may in turn be affected by the dynamic features of objects. In this paper, we propose coupled graph ODE: a novel latent ordinary differential equation (ODE) generative model that learns the coupled dynamics of nodes and edges with a graph neural network (GNN) based ODE in a continuous manner. Our model consists of two coupled ODE functions for modeling the dynamics of edges and nodes based on their latent representations respectively. It employs a novel encoder parameterized by a GNN for inferring the initial states from historical data, which serves as the starting point of the predicted latent trajectories. Experiment results on the COVID-19 dataset and the simulated social network dataset demonstrate the effectiveness of our proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf2145f9998f304513c0c9b530ee9f7750c6f55" target='_blank'>
                Coupled Graph ODE for Learning Interacting System Dynamics
                </a>
              </td>
          <td>
            Zijie Huang, Yizhou Sun, Wei Wang
          </td>
          <td>2021-08-14</td>
          <td>Knowledge Discovery and Data Mining, Proceedings of the 27th ACM SIGKDD Conference on Knowledge Discovery & Data Mining</td>
          <td>47</td>
          <td>57</td>

            <td><a href='../recommendations/aaf2145f9998f304513c0c9b530ee9f7750c6f55' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd28606fbae3449f831248804264c9885e992f9" target='_blank'>
                CellBox: Interpretable Machine Learning for Perturbation Biology with Application to the Design of Cancer Combination Therapy.
                </a>
              </td>
          <td>
            Bo Yuan, Ciyue Shen, Augustin Luna, Anil Korkut, D. Marks, John Ingraham, C. Sander
          </td>
          <td>2020-12-21</td>
          <td>Cell systems</td>
          <td>68</td>
          <td>152</td>

            <td><a href='../recommendations/6bd28606fbae3449f831248804264c9885e992f9' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="We introduce a flexible, scalable Bayesian inference framework for nonlinear dynamical systems characterised by distinct and hierarchical variability at the individual, group, and population levels. Our model class is a generalisation of nonlinear mixed-effects (NLME) dynamical systems, the statistical workhorse for many experimental sciences. We cast parameter inference as stochastic optimisation of an end-to-end differentiable, block-conditional variational autoencoder. We specify the dynamics of the data-generating process as an ordinary differential equation (ODE) such that both the ODE and its solver are fully differentiable. This model class is highly flexible: the ODE right-hand sides can be a mixture of user-prescribed or “white-box” sub-components and neural network or “black-box” sub-components. Using stochastic optimisation, our amortised inference algorithm could seamlessly scale up to massive data collection pipelines (common in labs with robotic automation). Finally, our framework supports interpretability with respect to the underlying dynamics, as well as predictive generalization to unseen combinations of group components (also called “zero-shot” learning). We empirically validate our method by predicting the dynamic behaviour of bacteria that were genetically engineered to function as biosensors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309c5ae93a4cabfd37747abd130866240e265b2d" target='_blank'>
                Efficient Amortised Bayesian Inference for Hierarchical and Nonlinear Dynamical Systems
                </a>
              </td>
          <td>
            Geoffrey Roeder, Paul K. Grant, Andrew Phillips, Neil Dalchau, Edward Meeds
          </td>
          <td>2019-05-24</td>
          <td>International Conference on Machine Learning, ArXiv</td>
          <td>21</td>
          <td>28</td>

            <td><a href='../recommendations/309c5ae93a4cabfd37747abd130866240e265b2d' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Parametrizing using ML'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Parametrizing using ML</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="A patient's digital twin is a computational model that describes the evolution of their health over time. Digital twins have the potential to revolutionize medicine by enabling individual-level computer simulations of human health, which can be used to conduct more efficient clinical trials or to recommend personalized treatment options. Due to the overwhelming complexity of human biology, machine learning approaches that leverage large datasets of historical patients' longitudinal health records to generate patients' digital twins are more tractable than potential mechanistic models. In this manuscript, we describe a neural network architecture that can learn conditional generative models of clinical trajectories, which we call Digital Twin Generators (DTGs), that can create digital twins of individual patients. We show that the same neural network architecture can be trained to generate accurate digital twins for patients across 13 different indications simply by changing the training set and tuning hyperparameters. By introducing a general purpose architecture, we aim to unlock the ability to scale machine learning approaches to larger datasets and across more indications so that a digital twin could be created for any patient in the world.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd216824a3b00951db6ff54f723148cfa3706e5" target='_blank'>
              Digital Twin Generators for Disease Modeling
              </a>
            </td>
          <td>
            Nameyeh Alam, Jake Basilico, Daniele Bertolini, S. C. Chetty, Heather D'Angelo, Ryan Douglas, Charles K. Fisher, Franklin Fuller, Melissa Gomes, Rishabh Gupta, Alex Lang, A. Loukianov, Rachel Mak-McCully, Cary Murray, Hanalei Pham, Susanna Qiao, Elena S. Ryapolova-Webb, Aaron M. Smith, Dimitri Theoharatos, Anil Tolwani, Eric W. Tramel, Anna A. Vidovszky, J. Viduya, Jonathan R. Walsh
          </td>
          <td>2024-05-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pharmacometric models are pivotal across drug discovery and development, playing a decisive role in determining the progression of candidate molecules. However, the derivation of mathematical equations governing the system is a labor-intensive trial-and-error process, often constrained by tight timelines. In this study, we introduce PKINNs, a novel purely data-driven pharmacokinetic-informed neural network model. PKINNs efficiently discovers and models intrinsic multi-compartment-based pharmacometric structures, reliably forecasting their derivatives. The resulting models are both interpretable and explainable through Symbolic Regression methods. Our computational framework demonstrates the potential for closed-form model discovery in pharmacometric applications, addressing the labor-intensive nature of traditional model derivation. With the increasing availability of large datasets, this framework holds the potential to significantly enhance model-informed drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecec2c77f9d85a4e75c8a8de808c4d916ca0015" target='_blank'>
              Discovering intrinsic multi-compartment pharmacometric models using Physics Informed Neural Networks
              </a>
            </td>
          <td>
            I. Nasim, Adam Nasim
          </td>
          <td>2024-04-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The deep-learning models like variational autoencoder have enabled low dimensional cellular embedding representation for large-scale single-cell transcriptomes and shown great flexibility in downstream tasks. However, biologically meaningful latent space is usually missing if no specific structure is designed. Here, we engineered a novel interpretable generative transcriptional program (iGTP) framework that could model the importance of TP space and protein-protein interactions (PPIs) between different biological states. We demonstrate the performance of iGTP in a diverse biological context using Gene Ontology, canonical pathway, and different PPI curation. iGTP not only elucidated the ground truth of cellular responses but also surpassed other deep learning models and traditional bioinformatics methods in functional enrichment tasks. By integrating the latent layer with a graph neural network (GNN) framework, iGTP effectively inferred cellular responses to perturbations. We anticipate that iGTP offers insights at both PPI and TP levels, and holds promise for predicting responses to novel perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8b53c25d331df61bf601a33636784d04ce05d68" target='_blank'>
              Learning interpretable cellular embedding for inferring biological mechanisms underlying single-cell transcriptomics
              </a>
            </td>
          <td>
            k.-l. Hsieh, Y. Chu, P. G. Pilie, K. Zhang
          </td>
          <td>2024-04-01</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c53a64b37da477c06468da0f1b359b77e2e273e9" target='_blank'>
              Biologically informed NeuralODEs for genome-wide regulatory dynamics
              </a>
            </td>
          <td>
            Intekhab Hossain, Viola Fanfani, Jonas Fischer, John Quackenbush, R. Burkholz
          </td>
          <td>2024-05-21</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Personalized cancer treatment requires a thorough understanding of complex interactions between drugs and cancer cell lines in varying genetic and molecular contexts. To address this, high-throughput screening has been used to generate large-scale drug response data, facilitating data-driven computational models. Such models can capture complex drug-cell line interactions across various contexts in a fully data-driven manner. However, accurately prioritizing the most effective drugs for each cell line still remains a significant challenge. To address this, we developed multiple neural ranking approaches that leverage large-scale drug response data across multiple cell lines from diverse cancer types. Unlike existing approaches that primarily utilize regression and classification techniques for drug response prediction, we formulated the objective of drug selection and prioritization as a drug ranking problem. In this work, we proposed multiple pairwise and listwise neural ranking methods that learn latent representations of drugs and cell lines and then use those representations to score drugs in each cell line via a learnable scoring function. Specifically, we developed neural pairwise and listwise ranking methods, Pair-PushC and List-One on top of the existing methods, pLETORg and ListNet, respectively. Additionally, we proposed a novel listwise ranking method, List-All, that focuses on all the effective drugs instead of the top effective drug, unlike List-One. We also provide an exhaustive empirical evaluation with state-of-the-art regression and ranking baselines on large-scale data sets across multiple experimental settings. Our results demonstrate that our proposed ranking methods mostly outperform the best baselines with significant improvements of as much as 25.6% in terms of selecting truly effective drugs within the top 20 predicted drugs (i.e., hit@20) across 50% test cell lines. Furthermore, our analyses suggest that the learned latent spaces from our proposed methods demonstrate informative clustering structures and capture relevant underlying biological features. Moreover, our comprehensive evaluation provides a thorough and objective comparison of the performance of different methods (including our proposed ones).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c0f5fb3d038f7182389a924d6a9c7f0a1db1816" target='_blank'>
              Improving Anticancer Drug Selection and Prioritization via Neural Learning to Rank.
              </a>
            </td>
          <td>
            Vishal Dey, Xia Ning
          </td>
          <td>2024-05-13</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Artificial intelligence (AI) technology has demonstrated remarkable potential in drug dis-covery, where pharmacokinetics plays a crucial role in determining the dosage, safety, and efficacy of new drugs. A major challenge for AI-driven drug discovery (AIDD) is the scarcity of high-quality data, which often requires extensive wet-lab work. A typical example of this is pharmacokinetic experiments. In this work, we develop a physical formula enhanced mul-ti-task learning (PEMAL) method that predicts four key parameters of pharmacokinetics simultaneously. By incorporating physical formulas into the multi-task framework, PEMAL facilitates effective knowledge sharing and target alignment among the pharmacokinetic parameters, thereby enhancing the accuracy of prediction. Our experiments reveal that PEMAL significantly lowers the data demand, compared to typical Graph Neural Networks. Moreover, we demonstrate that PEMAL enhances the robustness to noise, an advantage that conventional Neural Networks do not possess. Another advantage of PEMAL is its high flexibility, which can be potentially applied to other multi-task machine learning scenarios. Overall, our work illustrates the benefits and potential of using PEMAL in AIDD and other scenarios with data scarcity and noise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8909ccf04a951931d7cedb8b1142f5a37787182" target='_blank'>
              Physical formula enhanced multi-task learning for pharmacokinetics prediction
              </a>
            </td>
          <td>
            Ruifeng Li, Dongzhan Zhou, Ancheng Shen, Ao Zhang, Mao Su, Mingqian Li, Hongyang Chen, Gang Chen, Yin Zhang, Shufei Zhang, Yuqiang Li, Wanli Ouyang
          </td>
          <td>2024-04-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite the remarkable advances in cancer diagnosis, treatment, and management over the past decade, malignant tumors remain a major public health problem. Further progress in combating cancer may be enabled by personalizing the delivery of therapies according to the predicted response for each individual patient. The design of personalized therapies requires the integration of patient-specific information with an appropriate mathematical model of tumor response. A fundamental barrier to realizing this paradigm is the current lack of a rigorous yet practical mathematical theory of tumor initiation, development, invasion, and response to therapy. We begin this review with an overview of different approaches to modeling tumor growth and treatment, including mechanistic as well as data-driven models based on big data and artificial intelligence. We then present illustrative examples of mathematical models manifesting their utility and discuss the limitations of stand-alone mechanistic and data-driven models. We then discuss the potential of mechanistic models for not only predicting but also optimizing response to therapy on a patient-specific basis. We describe current efforts and future possibilities to integrate mechanistic and data-driven models. We conclude by proposing five fundamental challenges that must be addressed to fully realize personalized care for cancer patients driven by computational models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f694d2c55d26c981a3270eb504e2688631b8a8e" target='_blank'>
              Patient-Specific, Mechanistic Models of Tumor Growth Incorporating Artificial Intelligence and Big Data.
              </a>
            </td>
          <td>
            G. Lorenzo, Syed Rakin Ahmed, David A. Hormuth II, Brenna Vaughn, Jayashree Kalpathy-Cramer, Luis Solorio, T. Yankeelov, Hector Gomez
          </td>
          <td>2024-04-09</td>
          <td>Annual review of biomedical engineering</td>
          <td>2</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ae9f9ed096ad62f103bcd45fe4cf89ae91d36b" target='_blank'>
              MFSynDCP: multi-source feature collaborative interactive learning for drug combination synergy prediction
              </a>
            </td>
          <td>
            Yunyun Dong, Yunqing Chang, Yuxiang Wang, Qixuan Han, Xiaoyuan Wen, Ziting Yang, Yan Zhang, Yan Qiang, Kun Wu, Xiaole Fan, Xiaoqiang Ren
          </td>
          <td>2024-04-01</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Predicting cancer dynamics under treatment is challenging due to high inter-patient heterogeneity, lack of predictive biomarkers, and sparse and noisy longitudinal data. Mathematical models can summarize cancer dynamics by a few interpretable parameters per patient. Machine learning methods can then be trained to predict the model parameters from baseline covariates, but do not account for uncertainty in the parameter estimates. Instead, hierarchical Bayesian modeling can model the relationship between baseline covariates to longitudinal measurements via mechanistic parameters while accounting for uncertainty in every part of the model. The mapping from baseline covariates to model parameters can be modeled in several ways. A linear mapping simplifies inference but fails to capture nonlinear covariate effects and scale poorly for interaction modeling when the number of covariates is large. In contrast, Bayesian neural networks can potentially discover interactions between covariates automatically, but at a substantial cost in computational complexity. In this work, we develop a hierarchical Bayesian model of subpopulation dynamics that uses baseline covariate information to predict cancer dynamics under treatment, inspired by cancer dynamics in multiple myeloma (MM), where serum M protein is a well-known proxy of tumor burden. As a working example, we apply the model to a simulated dataset and compare its ability to predict M protein trajectories to a model with linear covariate effects. Our results show that the Bayesian neural network covariate effect model predicts cancer dynamics more accurately than a linear covariate effect model when covariate interactions are present. The framework can also be applied to other types of cancer or other time series prediction problems that can be described with a parametric model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ecd81405c7df09ef0fe2550029e319785f4a7b" target='_blank'>
              Prediction of cancer dynamics under treatment using Bayesian neural networks: A simulated study
              </a>
            </td>
          <td>
            E. M. Myklebust, A. Frigessi, Fredrik Schjesvold, J. Foo, K. Leder, Alvaro Kohn-Luque
          </td>
          <td>2024-05-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Latent variable models such as the Variational Auto-Encoders (VAEs) have shown impressive performance for inferring expression patterns for cell subtyping and biomarker identification from transcriptomics data. However, the limited interpretability of their latent variables obscures deriving meaningful biological understanding of cellular responses to different external and internal perturbations. We here propose a novel deep learning framework, EXPORT (EXPlainable VAE for ORdinally perturbed Transcriptomics data), for analyzing ordinally perturbed transcriptomics data that can incorporate any biological pathway knowledge in the VAE latent space. With the corresponding pathway-informed decoder, the learned latent expression patterns can be explained as pathway-level responses to perturbations, offering direct interpretability with biological understanding. More importantly, we explicitly model the ordinal nature of many real-world perturbations into the EXPORT framework by training an auxiliary ordinal regressor neural network to capture corresponding expression changes in the VAE latent representations, for example under different dosage levels of radiation exposure. By incorporating ordinal constraints during the training of our proposed framework, we further enhance the model interpretability by guiding the VAE latent space to organize perturbation responses in a hierarchical manner. We demonstrate the utility of the inferred guided latent space for downstream tasks, such as identifying key regulatory pathways associated with specific perturbation changes by analyzing transcriptomics datasets on both bulk and single-cell data. Overall, we envision that our proposed approach can unravel unprecedented biological intricacies in cellular responses to various perturbations while bringing an additional layer of interpretability to biology-inspired deep learning models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/385ecf17e4b8589c0b54575cc27c3b02323fabbb" target='_blank'>
              Biologically Interpretable VAE with Supervision for Transcriptomics Data Under Ordinal Perturbations
              </a>
            </td>
          <td>
            Seyednami Niyakan, Byung-Jun Yoon, Xiaoning Qian, Xihaier Luo
          </td>
          <td>2024-03-29</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a46186cfa337e51d671a8ac45c149e5f0bf01b" target='_blank'>
              Data-driven modeling of interrelated dynamical systems
              </a>
            </td>
          <td>
            Yonatan Elul, Eyal Rozenberg, Amit Boyarski, Yael Yaniv, Assaf Schuster, A. M. Bronstein
          </td>
          <td>2024-05-01</td>
          <td>Communications Physics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="This work proposes a novel framework for analyzing disease progression using time-aware neural ordinary differential equations (NODE). We introduce a"time-aware head"in a framework trained through self-supervised learning (SSL) to leverage temporal information in latent space for data augmentation. This approach effectively integrates NODEs with SSL, offering significant performance improvements compared to traditional methods that lack explicit temporal integration. We demonstrate the effectiveness of our strategy for diabetic retinopathy progression prediction using the OPHDIAT database. Compared to the baseline, all NODE architectures achieve statistically significant improvements in area under the ROC curve (AUC) and Kappa metrics, highlighting the efficacy of pre-training with SSL-inspired approaches. Additionally, our framework promotes stable training for NODEs, a commonly encountered challenge in time-aware modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba9952dbd34de0b508ed4f1290756368e714c81" target='_blank'>
              LaTiM: Longitudinal representation learning in continuous-time models to predict disease progression
              </a>
            </td>
          <td>
            Rachid Zeghlache, Pierre-Henri Conze, Mostafa EL HABIB DAHO, Yi-Hsuan Li, Hugo Le Boit'e, R. Tadayoni, Pascale Massin, B. Cochener, Alireza Rezaei, Ikram Brahim, G. Quellec, M. Lamard
          </td>
          <td>2024-04-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Motivation Successfully predicting the development of biological systems can lead to advances in various research fields, such as cellular biology and epidemiology. While machine learning has proven its capabilities in generalizing the underlying non-linear dynamics of such systems, unlocking its predictive power is often restrained by the limited availability of large, curated datasets. To supplement real-world data, informing machine learning by transfer learning with data simulated from ordinary differential equations has emerged as a promising solution. However, the success of this approach highly depends on the designed characteristics of the synthetic data. Results We optimize dataset characteristics such as size, diversity, and noise of ordinary differential equation-based synthetic time series datasets in three relevant and representative biological systems. To achieve this, we here, for the first time, present a framework to systematically evaluate the influence of such design choices on transfer learning performance in one place. We achieve a performance improvement of up to 92% in mean absolute error for our optimized simulation-based transfer learning compared to non-informed deep learning. We find a strong interdependency between dataset size and diversity effects. The optimal transfer learning setting heavily relies on real-world data characteristics as well as its coherence with the synthetic data’s dynamics, emphasizing the relevance of such a framework. Availability and Implementation The code is available at https://github.com/DILiS-lab/opt-synthdata-4tl.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1c7d62233a0ff735ce7ae98aa19a9ca10c7ff1" target='_blank'>
              Optimizing ODE-derived Synthetic Data for Transfer Learning in Dynamical Biological Systems
              </a>
            </td>
          <td>
            Julian Zabbarov, Simon Witzke, Maximilian Kleissl, Pascal Iversen, Bernhard Y. Renard, Katharina Baum
          </td>
          <td>2024-03-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Analyzing the drug response at the cellular level is crucial for identifying biomarkers and understanding the mechanisms of resistance. Although studies on the drug response of individual cells can provide novel insights into tumor heterogeneity, pharmacogenomic data related to single-cell (SC) RNA sequencing is often limited. Transfer learning provides a promising approach to translate the knowledge of drug response from bulk cell lines to SC analysis, potentially providing an effective solution to this challenge. Previous studies often use data from single drug-cell lines to pre-train specific models and adapt the models on SC datasets, which lack pharmacogenomic information from other drugs and hinder model generalization. In this work, we introduce MetaSCDrug as a unified meta pre-training framework that integrates molecular information with transcriptomic data to simultaneously modeling cellular heterogeneity in response to multiple pre-trained drugs and generalize to unseen drugs. Our model requires only one pre-training session, followed by fine-tuning on multiple single-cell datasets by few-shot learning, achieving an average of 4.58% accuracy increase in drug response prediction compared to the baselines. Furthermore, our meta pre-training strategy effectively captures transcriptome heterogeneity in the generalization of unseen drugs, achieving a 20% improvement over the model without meta pre-training. Case studies of our framework highlight its capability to identify critical genes for resistance, providing a method for exploring drug action pathways and understanding resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde20ee4a0135620d4f36e68f1c9aef919565e77" target='_blank'>
              MetaSCDrug: Meta-Transfer Learning for Single-Cell-Level Drug Response Prediction from Transcriptome and Molecular Representations
              </a>
            </td>
          <td>
            Shuang Ge, Shuqing Sun, Yiming Ren, Huan Xu, Zhixiang Ren
          </td>
          <td>2024-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The MRI-derived brain network serves as a pivotal instrument in elucidating both the structural and functional aspects of the brain, encompassing the ramifications of diseases and developmental processes. However, prevailing methodologies, often focusing on synchronous BOLD signals from functional MRI (fMRI), may not capture directional influences among brain regions and rarely tackle temporal functional dynamics. In this study, we first construct the brain-effective network via the dynamic causal model. Subsequently, we introduce an interpretable graph learning framework termed Spatio-Temporal Embedding ODE (STE-ODE). This framework incorporates specifically designed directed node embedding layers, aiming at capturing the dynamic interplay between structural and effective networks via an ordinary differential equation (ODE) model, which characterizes spatial-temporal brain dynamics. Our framework is validated on several clinical phenotype prediction tasks using two independent publicly available datasets (HCP and OASIS). The experimental results clearly demonstrate the advantages of our model compared to several state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/922be272d7098c93f25b6b7c27aad33f566c9494" target='_blank'>
              Interpretable Spatio-Temporal Embedding for Brain Structural-Effective Network with Ordinary Differential Equation
              </a>
            </td>
          <td>
            Haoteng Tang, Guodong Liu, Siyuan Dai, Kai Ye, Kun Zhao, Wenlu Wang, Carl Yang, Lifang He, Alex Leow, Paul M. Thompson, Heng Huang, Liang Zhan
          </td>
          <td>2024-05-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Predicting and enhancing inherent properties based on molecular structures is paramount to design tasks in medicine, materials science, and environmental management. Most of the current machine learning and deep learning approaches have become standard for predictions, but they face challenges when applied across different datasets due to reliance on correlations between molecular representation and target properties. These approaches typically depend on large datasets to capture the diversity within the chemical space, facilitating a more accurate approximation, interpolation, or extrapolation of the chemical behavior of molecules. In our research, we introduce an active learning approach that discerns underlying cause-effect relationships through strategic sampling with the use of a graph loss function. This method identifies the smallest subset of the dataset capable of encoding the most information representative of a much larger chemical space. The identified causal relations are then leveraged to conduct systematic interventions, optimizing the design task within a chemical space that the models have not encountered previously. While our implementation focused on the QM9 quantum-chemical dataset for a specific design task-finding molecules with a large dipole moment-our active causal learning approach, driven by intelligent sampling and interventions, holds potential for broader applications in molecular, materials design and discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8364c8b2f5ffe4dd218578294dd6ed9a6345ce87" target='_blank'>
              Active Causal Learning for Decoding Chemical Complexities with Targeted Interventions
              </a>
            </td>
          <td>
            Zachary R. Fox, Ayana Ghosh
          </td>
          <td>2024-04-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Motivation With the rapid advancement of single-cell sequencing technology, it becomes gradually possible to delve into the cellular responses to various external perturbations at the gene expression level. However, obtaining perturbed samples in certain scenarios may be considerably challenging, and the substantial costs associated with sequencing also curtail the feasibility of large-scale experimentation. A repertoire of methodologies has been employed for forecasting perturbative responses in single-cell gene expression. However, existing methods primarily focus on the average response of a specific cell type to perturbation, overlooking the single-cell specificity of perturbation responses and a more comprehensive prediction of the entire perturbation response distribution. Results Here, we present scPRAM, a method for predicting perturbation responses in single-cell gene expression based on attention mechanisms. Leveraging variational autoencoders and optimal transport, scPRAM aligns cell states before and after perturbation, followed by accurate prediction of gene expression responses to perturbations for unseen cell types through attention mechanisms. Experiments on multiple real perturbation datasets involving drug treatments and bacterial infections demonstrate that scPRAM attains heightened accuracy in perturbation prediction across cell types, species, and individuals, surpassing existing methodologies. Furthermore, scPRAM demonstrates outstanding capability in identifying differentially expressed genes under perturbation, capturing heterogeneity in perturbation responses across species, and maintaining stability in the presence of data noise and sample size variations. Availability and implementation https://github.com/jiang-q19/scPRAM and https://doi.org/10.5281/zenodo.10935038.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a7404da9e41fa69f1c46bb88fd5a87b6a5a437" target='_blank'>
              scPRAM accurately predicts single-cell gene expression perturbation response based on attention mechanism
              </a>
            </td>
          <td>
            Qun Jiang, Shengquan Chen, Xiaoyang Chen, Rui Jiang
          </td>
          <td>2024-04-15</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Locally interacting dynamical systems, such as epidemic spread, rumor propagation through crowd, and forest fire, exhibit complex global dynamics originated from local, relatively simple, and often stochastic interactions between dynamic elements. Their temporal evolution is often driven by transitions between a finite number of discrete states. Despite significant advancements in predictive modeling through deep learning, such interactions among many elements have rarely explored as a specific domain for predictive modeling. We present Attentive Recurrent Neural Cellular Automata (AR-NCA), to effectively discover unknown local state transition rules by associating the temporal information between neighboring cells in a permutation-invariant manner. AR-NCA exhibits the superior generalizability across various system configurations (i.e., spatial distribution of states), data efficiency and robustness in extremely data-limited scenarios even in the presence of stochastic interactions, and scalability through spatial dimension-independent prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1388550b7dbb3f7941311d9f11f9bec99880c0bf" target='_blank'>
              Learning Locally Interacting Discrete Dynamical Systems: Towards Data-Efficient and Scalable Prediction
              </a>
            </td>
          <td>
            Beomseok Kang, H. Kumar, Minah Lee, Biswadeep Chakraborty, Saibal Mukhopadhyay
          </td>
          <td>2024-04-09</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Predicting Remaining Useful Life (RUL) plays a crucial role in the prognostics and health management of industrial systems that involve a variety of interrelated sensors. Given a constant stream of time series sensory data from such systems, deep learning models have risen to prominence at identifying complex, nonlinear temporal dependencies in these data. In addition to the temporal dependencies of individual sensors, spatial dependencies emerge as important correlations among these sensors, which can be naturally modelled by a temporal graph that describes time-varying spatial relationships. However, the majority of existing studies have relied on capturing discrete snapshots of this temporal graph, a coarse-grained approach that leads to loss of temporal information. Moreover, given the variety of heterogeneous sensors, it becomes vital that such inherent heterogeneity is leveraged for RUL prediction in temporal sensor graphs. To capture the nuances of the temporal and spatial relationships and heterogeneous characteristics in an interconnected graph of sensors, we introduce a novel model named Temporal and Heterogeneous Graph Neural Networks (THGNN). Specifically, THGNN aggregates historical data from neighboring nodes to accurately capture the temporal dynamics and spatial correlations within the stream of sensor data in a fine-grained manner. Moreover, the model leverages Feature-wise Linear Modulation (FiLM) to address the diversity of sensor types, significantly improving the model's capacity to learn the heterogeneity in the data sources. Finally, we have validated the effectiveness of our approach through comprehensive experiments. Our empirical findings demonstrate significant advancements on the N-CMAPSS dataset, achieving improvements of up to 19.2% and 31.6% in terms of two different evaluation metrics over state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b4e301bda3ea480936b4a2e3038dcd0b960754d" target='_blank'>
              Temporal and Heterogeneous Graph Neural Network for Remaining Useful Life Prediction
              </a>
            </td>
          <td>
            Zhihao Wen, Yuan Fang, Pengcheng Wei, Fayao Liu, Zhenghua Chen, Min Wu
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="INTRODUCTION
Prediction of pharmacokinetic (PK) properties is crucial for drug discovery and development. Machine-learning (ML) models, which use statistical pattern recognition to learn correlations between input features (such as chemical structures) and target variables (such as PK parameters), are being increasingly used for this purpose. To embed ML models for PK prediction into workflows and to guide future development, a solid understanding of their applicability, advantages, limitations, and synergies with other approaches is necessary.


AREAS COVERED
This narrative review discusses the design and application of ML models to predict PK parameters of small molecules, especially in light of established approaches including in vitro-in vivo extrapolation (IVIVE) and physiologically based pharmacokinetic (PBPK) models. The authors illustrate scenarios in which the three approaches are used and emphasize how they enhance and complement each other. In particular, they highlight achievements, the state of the art and potentials of applying machine learning for PK prediction through a comphrehensive literature review.


EXPERT OPINION
ML models, when carefully crafted, regularly updated, and appropriately used, empower users to prioritize molecules with favorable PK properties. Informed practitioners can leverage these models to improve the efficiency of drug discovery and development process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4407f5485a20d2a2a26036c5e168a7a01ce78c14" target='_blank'>
              Another string to your bow: machine learning prediction of the pharmacokinetic properties of small molecules.
              </a>
            </td>
          <td>
            Davide Bassani, Neil John Parrott, Nenad Manevski, Jitao David Zhang
          </td>
          <td>2024-05-10</td>
          <td>Expert opinion on drug discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Reconstructing complex networks and predicting the dynamics are particularly challenging in real-world applications because the available information and data are incomplete. We develop a unified collaborative deep-learning framework consisting of three modules: network inference, state estimation, and dynamical learning. The complete network structure is first inferred and the states of the unobserved nodes are estimated, based on which the dynamical learning module is activated to determine the dynamical evolution rules. An alternating parameter updating strategy is deployed to improve the inference and prediction accuracy. Our framework outperforms baseline methods for synthetic and empirical networks hosting a variety of dynamical processes. A reciprocity emerges between network inference and dynamical prediction: better inference of network structure improves the accuracy of dynamical prediction, and vice versa. We demonstrate the superior performance of our framework on an influenza dataset consisting of 37 US States and a PM2.5 dataset covering 184 cities in China.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38794d512c02c8faee07849e126f9e6f2863fc85" target='_blank'>
              Deep-learning reconstruction of complex dynamical networks from incomplete data.
              </a>
            </td>
          <td>
            Xiao Ding, Ling-Wei Kong, Haifeng Zhang, Ying-Cheng Lai
          </td>
          <td>2024-04-01</td>
          <td>Chaos</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background The potential benefits of drug combination synergy in cancer medicine are significant, yet the risks must be carefully managed due to the possibility of increased toxicity. Although artificial intelligence applications have demonstrated notable success in predicting drug combination synergy, several key challenges persist: (1) Existing models often predict average synergy values across a restricted range of testing dosages, neglecting crucial dose amounts and the mechanisms of action of the drugs involved. (2) Many graph-based models rely on static protein-protein interactions, failing to adapt to dynamic and context-dependent networks. This limitation constrains the applicability of current methods. Results We introduced SAFER, a Sub-hypergraph Attention-based graph model, addressing these issues by incorporating complex relationships among biological knowledge networks and considering dosing effects on subject-specific networks. SAFER outperformed previous models on the benchmark and the independent test set. The analysis of subgraph attention weight for the lung cancer cell line highlighted JAK-STAT signaling pathway, PRDM12, ZNF781, and CDC5L that have been implicated in lung fibrosis. Conclusions SAFER presents an interpretable framework designed to identify drug-responsive signals. Tailored for comprehending dose effects on subject-specific molecular contexts, our model uniquely captures dose-level drug combination responses. This capability unlocks previously inaccessible avenues of investigation compared to earlier models. Finally, the SAFER framework can be leveraged by future inquiries to investigate molecular networks that uniquely characterize individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/500c4f42640ef697e84b8ae2a633d54a79da30c0" target='_blank'>
              SAFER: sub-hypergraph attention-based neural network for predicting effective responses to dose combinations
              </a>
            </td>
          <td>
            yiching tang, Rongbin Li, Jing Tang, W. J. Zheng, Xiaoqian Jiang
          </td>
          <td>2024-04-30</td>
          <td>Research Square</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MOTIVATION
Research is improving our understanding of how the microbiome interacts with the human body and its impact on human health. Existing machine learning methods have shown great potential in discriminating healthy from diseased microbiome states. However, Machine Learning based prediction using microbiome data has challenges such as, small sample size, imbalance between cases and controls and high cost of collecting large number of samples. To address these challenges, we propose a deep learning framework phylaGAN to augment the existing datasets with generated microbiome data using a combination of conditional generative adversarial network (C-GAN) and autoencoder. Conditional generative adversarial networks train two models against each other to compute larger simulated datasets that are representative of the original dataset. Autoencoder maps the original and the generated samples onto a common subspace to make the prediction more accurate.


RESULTS
Extensive evaluation and predictive analysis was conducted on two datasets, T2D study and Cirrhosis study showing an improvement in mean AUC using data augmentation by 11% and 5% respectively. External validation on a cohort classifying between obese and lean subjects, with a smaller sample size provided an improvement in mean AUC close to 32% when augmented through phylaGAN as compared to using the original cohort. Our findings not only indicate that the generative adversarial networks can create samples that mimic the original data across various diversity metrics, but also highlight the potential of enhancing disease prediction through machine learning models trained on synthetic data.


AVAILABILITY AND IMPLEMENTATION
https://github.com/divya031090/phylaGAN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d705c62a33eadeb75fe5c8e87e1d45ceb9effe" target='_blank'>
              phylaGAN: Data augmentation through conditional GANs and autoencoders for improving disease prediction accuracy using microbiome data.
              </a>
            </td>
          <td>
            Divya Sharma, Wendy Lou, Wei Xu
          </td>
          <td>2024-04-03</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In clinical research, translating findings from model organisms to human applications remains challenging due to biological differences between species as well as limitations of orthologous, and homologous, gene comparisons, which is fraugt with information loss as well as many-to-many mapping. To address these issues, we introduce a novel Universal Gene Embedding (UGE) model that leverages transformer-based few-shot learning for species-agnostic transfer learning with heterogeneous domain adaptation. The UGE model, trained on a dataset of gene expression from ten organs across rats and mice, establishes a unified biological latent space that effectively represents genes from any organ or species. By focusing on reactomes—comprehensive profiles of gene expression responses to drugs—the UGE model enables functional gene mapping across species based on the similarities of these profiles. Our contributions include a gene reactome vector prediction module, a robust framework for mapping drug-induced gene expression patterns across species, strategies for optimizing experimental design, and enhanced gene mapping precision. These advancements provide a new tool for genetic research and a new paradigm for cross-species insights, potentially revolutionizing our understanding of gene function, drug responses, and the translation of findings from model organisms to human clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ace13bfdfb4ef5fafaea4ed599da0866bb2fe13d" target='_blank'>
              Transfer Learning Of Gene Expression Using Reactome
              </a>
            </td>
          <td>
            Siham Belgadi, David Yu Zhang, Ashwin Gopinath
          </td>
          <td>2024-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Patients diagnosed with advanced metastatic colorectal cancer (mCRC) often exhibit heterogeneous disease progression and face poor survival prospects. In order to comprehensively analyze the varied treatment responses among individuals and the challenge of tumor recurrence resistant to drugs in advanced mRCR, we developed a novel quantitative cancer-immunity cycle (QCIC) model. The proposed model was meticulously crafted utilizing a blend of differential equations and randomized modeling techniques to quantitatively elucidate the intricate mechanisms governing the cancer-immunity cycle and forecast tumor dynamics under different treatment modalities. Furthermore, by integrating diverse clinical datasets and rigorous model analyses, we introduced two pivotal concepts: the treatment response index (TRI) and the death probability function (DPF). These concepts are crucial tools for translating model predictions into clinically relevant evaluation indexes. Using virtual patient technology, we extrapolated tumor predictive biomarkers from the model to predict survival outcomes for mCRC patients. Our findings underscore the significance of tumor-infiltrating CD8+CTL cell density as a key predictive biomarker for short-term treatment responses in advanced mCRC while emphasizing the potential predict value of the tumor-infiltrating CD4+Th1/Treg ratio in determining patient survival. This study presents a pioneering methodology bridging the divide between diverse clinical data sources and the generation of virtual patients, offering invaluable insights into understanding inter-individual treatment variations and forecasting survival outcomes in mCRC patients. Author summary This study introduces a sophisticated modeling approach to delineate the intricate dynamics of tumor-immune interactions within the cancer-immunity cycle. Utilizing a multi-compartmental, multi-scale, multi-dimensional differential equation model, we quantified the complex interplay between tumor cells, immune cells, cytokines, and chemokines. By integrating virtual patient technology, we have conducted in silico clinical trials that accurately predict disease progression across multiple treatment modalities for cancer patients, particularly in advanced mCRC. Through the combination of differential equations and randomized modeling, we effectively captured the diverse treatment responses and clinical manifestations of drug-resistant tumor recurrence. Furthermore, we explored the pivotal role of predictive biomarkers in determining patients’ survival prognosis, offering insights for tailored therapeutic strategies. Importantly, our computational framework holds promise for the extension to the investigation of patients with other tumor types, thus contributing to the personalized investigation of patients with other tumor types and cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a0274da7051ba17dd2effa0a322076737f25c2" target='_blank'>
              Quantitative cancer-immunity cycle modeling for predicting disease progression in advanced metastatic colorectal cancer
              </a>
            </td>
          <td>
            Chenghang Li, Yongchang Wei, J. Lei
          </td>
          <td>2024-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Deep generative models have been accelerating the inverse design process in material and drug design. Unlike their counterpart property predictors in typical molecular design frameworks, generative molecular design models have seen fewer efforts on uncertainty quantification (UQ) due to computational challenges in Bayesian inference posed by their large number of parameters. In this work, we focus on the junction-tree variational autoencoder (JT-VAE), a popular model for generative molecular design, and address this issue by leveraging the low dimensional active subspace to capture the uncertainty in the model parameters. Specifically, we approximate the posterior distribution over the active subspace parameters to estimate the epistemic model uncertainty in an extremely high dimensional parameter space. The proposed UQ scheme does not require alteration of the model architecture, making it readily applicable to any pre-trained model. Our experiments demonstrate the efficacy of the AS-based UQ and its potential impact on molecular optimization by exploring the model diversity under epistemic uncertainty.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a70f6922f6ea468e7916a2cd53f69787e147d2e3" target='_blank'>
              Leveraging Active Subspaces to Capture Epistemic Model Uncertainty in Deep Generative Models for Molecular Design
              </a>
            </td>
          <td>
            A. N. M. N. Abeer, Sanket R. Jantre, Nathan M. Urban, Byung-Jun Yoon
          </td>
          <td>2024-04-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Representation learning for tumor gene expression (GEx) data with deep neural networks is limited by the large gene feature space and the scarcity of available clinical and preclinical data. The translation of the learned representation between these data sources is further hindered by inherent molecular differences. To address these challenges, we propose GExMix (Gene Expression Mixup), a data augmentation method, which extends the Mixup concept to generate training samples accounting for the imbalance in both data classes and data sources. We leverage the GExMix-augmented training set in encoder-decoder models to learn a GEx latent representation. Subsequently, we combine the learned representation with drug chemical features in a dual-objective enhanced gene-centric drug response prediction, i.e., reconstruction of GEx latent embeddings and drug response classification. This dual-objective design strategically prioritizes gene-centric information to enhance the final drug response prediction. We demonstrate that augmenting training samples improves the GEx representation, benefiting the gene-centric drug response prediction model. Our findings underscore the effectiveness of our proposed GExMix in enriching GEx data for deep neural networks. Moreover, our proposed gene-centricity further improves drug response prediction when translating preclinical to clinical datasets. This highlights the untapped potential of the proposed framework for GEx data analysis, paving the way toward precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d8245b02f26c5698605a2d1bfdadb5793c2f155" target='_blank'>
              Enhancing Gene Expression Representation and Drug Response Prediction with Data Augmentation and Gene Emphasis
              </a>
            </td>
          <td>
            Diyuan Lu, Daksh P.S. Pamar, AJ Ohnmacht, G. Kutkaite, Michael P. Menden
          </td>
          <td>2024-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Interventions play a pivotal role in the study of complex biological systems. In drug discovery, genetic interventions (such as CRISPR base editing) have become central to both identifying potential therapeutic targets and understanding a drug's mechanism of action. With the advancement of CRISPR and the proliferation of genome-scale analyses such as transcriptomics, a new challenge is to navigate the vast combinatorial space of concurrent genetic interventions. Addressing this, our work concentrates on estimating the effects of pairwise genetic combinations on the cellular transcriptome. We introduce two novel contributions: Salt, a biologically-inspired baseline that posits the mostly additive nature of combination effects, and Peper, a deep learning model that extends Salt's additive assumption to achieve unprecedented accuracy. Our comprehensive comparison against existing state-of-the-art methods, grounded in diverse metrics, and our out-of-distribution analysis highlight the limitations of current models in realistic settings. This analysis underscores the necessity for improved modelling techniques and data acquisition strategies, paving the way for more effective exploration of genetic intervention effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e42acd4e6c209067c51c52b168ecb09fa85ef100" target='_blank'>
              Season combinatorial intervention predictions with Salt&Peper
              </a>
            </td>
          <td>
            Thomas Gaudelet, Alice Del Vecchio, Eli M. Carrami, Juliana Cudini, Chantriolnt-Andreas Kapourani, Caroline Uhler, Lindsay Edwards
          </td>
          <td>2024-04-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Data-driven modelling and scientific machine learning have been responsible for significant advances in determining suitable models to describe data. Within dynamical systems, neural ordinary differential equations (ODEs), where the system equations are set to be governed by a neural network, have become a popular tool for this challenge in recent years. However, less emphasis has been placed on systems that are only partially-observed. In this work, we employ a hybrid neural ODE structure, where the system equations are governed by a combination of a neural network and domain-specific knowledge, together with symbolic regression (SR), to learn governing equations of partially-observed dynamical systems. We test this approach on two case studies: A 3-dimensional model of the Lotka-Volterra system and a 5-dimensional model of the Lorenz system. We demonstrate that the method is capable of successfully learning the true underlying governing equations of unobserved states within these systems, with robustness to measurement noise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5f91324b2ca816ed10e0d9a1d6565fb12a4a1f" target='_blank'>
              Learning Governing Equations of Unobserved States in Dynamical Systems
              </a>
            </td>
          <td>
            Gevik Grigorian, Sandip V. George, S. Arridge
          </td>
          <td>2024-04-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Based on multi-omics data and drug information, predicting the response of cancer cell lines to drugs is a crucial area of research in modern oncology, as it can promote the development of personalized treatments. Despite the promising performance achieved by existing models, most of them overlook the variations among different omics and lack effective integration of multi-omics data. Moreover, the explicit modeling of cell line/drug attribute and cell line-drug association has not been thoroughly investigated in existing approaches. To address these issues, we propose RedCDR, a dual relation distillation model for cancer drug response (CDR) prediction. Specifically, a parallel dual-branch architecture is designed to enable both the independent learning and interactive fusion feasible for cell line/drug attribute and cell line-drug association information. To facilitate the adaptive interacting integration of multi-omics data, the proposed multi-omics encoder introduces the multiple similarity relations between cell lines and takes the importance of different omics data into account. To accomplish knowledge transfer from the two independent attribute and association branches to their fusion, a dual relation distillation mechanism consisting of representation distillation and prediction distillation is presented. Experiments conducted on the GDSC and CCLE datasets show that RedCDR outperforms previous state-of-the-art approaches in CDR prediction. The source code is available at https://github.com/mhxu1998/RedCDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1d774b32aa1d353908c13e835eb7b0507d698d" target='_blank'>
              RedCDR: Dual Relation Distillation for Cancer Drug Response Prediction.
              </a>
            </td>
          <td>
            Muhao Xu, Zhenfeng Zhu, Yawei Zhao, Kunlun He, Qinghua Huang, Yao Zhao
          </td>
          <td>2024-05-22</td>
          <td>IEEE/ACM transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Irregularly sampled time series with missing values are often observed in multiple real-world applications such as healthcare, climate and astronomy. They pose a significant challenge to standard deep learn- ing models that operate only on fully observed and regularly sampled time series. In order to capture the continuous dynamics of the irreg- ular time series, many models rely on solving an Ordinary Differential Equation (ODE) in the hidden state. These ODE-based models tend to perform slow and require large memory due to sequential operations and a complex ODE solver. As an alternative to complex ODE-based mod- els, we propose a family of models called Functional Latent Dynamics (FLD). Instead of solving the ODE, we use simple curves which exist at all time points to specify the continuous latent state in the model. The coefficients of these curves are learned only from the observed values in the time series ignoring the missing values. Through extensive experi- ments, we demonstrate that FLD achieves better performance compared to the best ODE-based model while reducing the runtime and memory overhead. Specifically, FLD requires an order of magnitude less time to infer the forecasts compared to the best performing forecasting model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2a5b8158db29854109275cb5c3fbcf47c080c1c" target='_blank'>
              Functional Latent Dynamics for Irregularly Sampled Time Series Forecasting
              </a>
            </td>
          <td>
            Christian Klotergens, Vijaya Krishna Yalavarthi, Maximilian Stubbemann, Lars Schmidt-Thieme
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The discovery of drug-target interactions (DTIs) plays a crucial role in pharmaceutical development. The deep learning model achieves more accurate results in DTI prediction due to its ability to extract robust and expressive features from drug and target chemical structures. However, existing deep learning methods typically generate drug features via aggregating molecular atom representations, ignoring the chemical properties carried by motifs, i.e., substructures of the molecular graph. The atom-drug double-level molecular representation learning can not fully exploit structure information and fails to interpret the DTI mechanism from the motif perspective. In addition, sequential model-based target feature extraction either fuses limited contextual information or requires expensive computational resources. To tackle the above issues, we propose a hierarchical graph representation learning-based DTI prediction method (HiGraphDTI). Specifically, HiGraphDTI learns hierarchical drug representations from triple-level molecular graphs to thoroughly exploit chemical information embedded in atoms, motifs, and molecules. Then, an attentional feature fusion module incorporates information from different receptive fields to extract expressive target features.Last, the hierarchical attention mechanism identifies crucial molecular segments, which offers complementary views for interpreting interaction mechanisms. The experiment results not only demonstrate the superiority of HiGraphDTI to the state-of-the-art methods, but also confirm the practical ability of our model in interaction interpretation and new DTI discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f199a680fe31a9cd2c0f4793abe2bc09da4f91f" target='_blank'>
              HiGraphDTI: Hierarchical Graph Representation Learning for Drug-Target Interaction Prediction
              </a>
            </td>
          <td>
            Bin Liu, Siqi Wu, Jin Wang, Xin Deng, Ao Zhou
          </td>
          <td>2024-04-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Neural Ordinary Differential Equations typically struggle to generalize to new dynamical behaviors created by parameter changes in the underlying system, even when the dynamics are close to previously seen behaviors. The issue gets worse when the changing parameters are unobserved, i.e., their value or influence is not directly measurable when collecting data. We introduce Neural Context Flow (NCF), a framework that encodes said unobserved parameters in a latent context vector as input to a vector field. NCFs leverage differentiability of the vector field with respect to the parameters, along with first-order Taylor expansion to allow any context vector to influence trajectories from other parameters. We validate our method and compare it to established Multi-Task and Meta-Learning alternatives, showing competitive performance in mean squared error for in-domain and out-of-distribution evaluation on the Lotka-Volterra, Glycolytic Oscillator, and Gray-Scott problems. This study holds practical implications for foundational models in science and related areas that benefit from conditional neural ODEs. Our code is openly available at https://github.com/ddrous/ncflow.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c738e0450649ed6c04ff7e82d993987c381e35a" target='_blank'>
              Neural Context Flows for Learning Generalizable Dynamical Systems
              </a>
            </td>
          <td>
            Roussel Desmond Nzoyem, David A.W. Barton, Tom Deakin
          </td>
          <td>2024-05-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract: An essential issue in computational personalised medicine is the prediction of drug responses. There have been several proposals for approaches to this problem that rely on machine learning, particularly deep learning. Nevertheless, these approaches often portray the medications as strings, an implausible representation of molecules. Furthermore, there has been a lack of comprehensive consideration of interpretation, such as whether mutations or copy number aberrations contribute to the medication response. Graph DRP, a new approach based on graph convolution networks, is suggested as a solution to the issue in this research. Cell lines were displayed as double vectors of genetic abnormalities in Graph DRP, whereas medications were shown as sub-atomic chartsthat straightforwardly caught the bonds among particles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b5611d40b7b831c904a8d7225c02e0bd1010dd" target='_blank'>
              Drug Response Prediction Using XGBOOST
              </a>
            </td>
          <td>
            P. Senthil
          </td>
          <td>2024-03-31</td>
          <td>International Journal for Research in Applied Science and Engineering Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Network inference is used to model transcriptional, signaling, and metabolic interactions among genes, proteins, and metabolites that identify biological pathways influencing disease pathogenesis. Advances in machine learning (ML)-based inference models exhibit the predictive capabilities of capturing latent patterns in genomic data. Such models are emerging as an alternative to the statistical models identifying causative factors driving complex diseases. We present CoVar, an ML-based framework that builds upon the properties of existing inference models, to find the central genes driving perturbed gene expression across biological states. Unlike differentially expressed genes (DEGs) that capture changes in individual gene expression across conditions, CoVar focuses on identifying variational genes that undergo changes in their expression network interaction profiles, providing insights into changes in the regulatory dynamics, such as in disease pathogenesis. Subsequently, it finds core genes from among the nearest neighbors of these variational genes, which are central to the variational activity and influence the coordinated regulatory processes underlying the observed changes in gene expression. Through the analysis of simulated as well as yeast expression data perturbed by the deletion of the mitochondrial genome, we show that CoVar captures the intrinsic variationality and modularity in the expression data, identifying key driver genes not found through existing differential analysis methodologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d91de570e66b22b6dcb9a4caf542415ea36c0206" target='_blank'>
              CoVar: A generalizable machine learning approach to identify the coordinated regulators driving variational gene expression.
              </a>
            </td>
          <td>
            Satyaki Roy, S. Sheikh, T. Furey
          </td>
          <td>2024-04-17</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Cancer, a leading cause of death globally, occurs due to genomic changes and manifests heterogeneously across patients. To advance research on personalized treatment strategies, the effectiveness of various drugs on cells derived from cancers (`cell lines') is experimentally determined in laboratory settings. Nevertheless, variations in the distribution of genomic data and drug responses between cell lines and humans arise due to biological and environmental differences. Moreover, while genomic profiles of many cancer patients are readily available, the scarcity of corresponding drug response data limits the ability to train machine learning models that can predict drug response in patients effectively. Recent cancer drug response prediction methods have largely followed the paradigm of unsupervised domain-invariant representation learning followed by a downstream drug response classification step. Introducing supervision in both stages is challenging due to heterogeneous patient response to drugs and limited drug response data. This paper addresses these challenges through a novel representation learning method in the first phase and weak supervision in the second. Experimental results on real patient data demonstrate the efficacy of our method (WISER) over state-of-the-art alternatives on predicting personalized drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28ee9b9ea8cc0a9f28f3c21fa707bc8f6007d90" target='_blank'>
              WISER: Weak supervISion and supErvised Representation learning to improve drug response prediction in cancer
              </a>
            </td>
          <td>
            Kumar Shubham, A. Jayagopal, Syed Mohammed Danish, AP Prathosh, V. Rajan
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d88d1ee244b5bbb95c70dc410ecf9f282eee608" target='_blank'>
              Prediction of anticancer drug sensitivity using an interpretable model guided by deep learning
              </a>
            </td>
          <td>
            Weixiong Pang, Ming Chen, Yufang Qin
          </td>
          <td>2024-05-09</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Motivation Understanding the intermolecular interactions of ligand–target pairs is key to guiding the optimization of drug research on cancers, which can greatly mitigate overburden workloads for wet labs. Several improved computational methods have been introduced and exhibit promising performance for these identification tasks, but some pitfalls restrict their practical applications: (i) first, existing methods do not sufficiently consider how multigranular molecule representations influence interaction patterns between proteins and compounds; and (ii) second, existing methods seldom explicitly model the binding sites when an interaction occurs to enable better prediction and interpretation, which may lead to unexpected obstacles to biological researchers. Results To address these issues, we here present DrugMGR, a deep multigranular drug representation model capable of predicting binding affinities and regions for each ligand–target pair. We conduct consistent experiments on three benchmark datasets using existing methods and introduce a new specific dataset to better validate the prediction of binding sites. For practical application, target-specific compound identification tasks are also carried out to validate the capability of real-world compound screen. Moreover, the visualization of some practical interaction scenarios provides interpretable insights from the results of the predictions. The proposed DrugMGR achieves excellent overall performance in these datasets, exhibiting its advantages and merits against state-of-the-art methods. Thus, the downstream task of DrugMGR can be fine-tuned for identifying the potential compounds that target proteins for clinical treatment. Availability and implementation https://github.com/lixiaokun2020/DrugMGR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e49f59678a2ffc18713aaf2dc4993017f21b1051" target='_blank'>
              DrugMGR: a deep bioactive molecule binding method to identify compounds targeting proteins
              </a>
            </td>
          <td>
            Xiaokun Li, Qiang Yang, Longpeng Xu, Weihe Dong, Gongning Luo, Wei Wang, Suyu Dong, Kuanquan Wang, Ping Xuan, Xianyu Zhang, Xin Gao
          </td>
          <td>2024-04-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="We present a study where predictive mechanistic modeling is used in combination with deep learning methods to predict individual patient survival probabilities under immune checkpoint inhibitor (ICI) therapy. This hybrid approach enables prediction based on both measures that are calculable from mechanistic models (but may not be directly measurable in the clinic) and easily measurable quantities or characteristics (that are not always readily incorporated into predictive mechanistic models). The mechanistic model we have applied here can predict tumor response from CT or MRI imaging based on key mechanisms underlying checkpoint inhibitor therapy, and in the present work, its parameters were combined with readily-available clinical measures from 93 patients into a hybrid training set for a deep learning time-to-event predictive model. Analysis revealed that training an artificial neural network with both mechanistic modeling-derived and clinical measures achieved higher per-patient predictive accuracy based on event-time concordance, Brier score, and negative binomial log-likelihood-based criteria than when only mechanistic model-derived values or only clinical data were used. Feature importance analysis revealed that both clinical and model-derived parameters play prominent roles in neural network decision making, and in increasing prediction accuracy, further supporting the advantage of our hybrid approach. We anticipate that many existing mechanistic models may be hybridized with deep learning methods in a similar manner to improve predictive accuracy through addition of additional data that may not be readily implemented in mechanistic descriptions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7b3da8173ee0e944bcc41b5f5c4c90ff6088d34" target='_blank'>
              Hybridizing mechanistic mathematical modeling with deep learning methods to predict individual cancer patient survival after immune checkpoint inhibitor therapy
              </a>
            </td>
          <td>
            J. D. Butner, P. Dogra, Caroline Chung, Eugene Koay, James Welsh, David Hong, Vittorio Cristini, Zhihui Wang
          </td>
          <td>2024-03-29</td>
          <td>Research Square</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Prediction of drug combination responses is a research question of growing importance for cancer and other complex diseases. Current machine learning approaches generally consider predicting either drug combination synergy summaries or single combination dose-response values, which fail to appropriately model the continuous nature of the underlying dose-response combination surface and can lead to inconsistencies when a synergy score or a dose-response matrix is reconstructed from separate predictions. We propose a structured prediction method, comboKR, that directly predicts the drug combination response surface for a drug combination. The method is based on a powerful input-output kernel regression technique and functional modeling of the response surface. As an important part of our approach, we develop a novel normalisation between response surfaces that standardizes the heterogeneous experimental designs used to measure the dose-responses, and thus allows training the method with data measured in different laboratories. Our experiments on two predictive scenarios highlight the suitability of the proposed approach especially in the traditionally challenging setting of predicting combination responses for new drugs not available in the training data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aa8dd703916af4caa0d6de87dc609764aa2db9f" target='_blank'>
              Predicting drug combination response surfaces
              </a>
            </td>
          <td>
            Riikka Huusari, Tianduanyi Wang, S. Szedmák, T. Aittokallio, Juho Rousu
          </td>
          <td>2024-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Modeling complex systems using standard neural ordinary differential equations (NODEs) often faces some essential challenges, including high computational costs and susceptibility to local optima. To address these challenges, we propose a simulation-free framework, called Fourier NODEs (FNODEs), that effectively trains NODEs by directly matching the target vector field based on Fourier analysis. Specifically, we employ the Fourier analysis to estimate temporal and potential high-order spatial gradients from noisy observational data. We then incorporate the estimated spatial gradients as additional inputs to a neural network. Furthermore, we utilize the estimated temporal gradient as the optimization objective for the output of the neural network. Later, the trained neural network generates more data points through an ODE solver without participating in the computational graph, facilitating more accurate estimations of gradients based on Fourier analysis. These two steps form a positive feedback loop, enabling accurate dynamics modeling in our framework. Consequently, our approach outperforms state-of-the-art methods in terms of training time, dynamics prediction, and robustness. Finally, we demonstrate the superior performance of our framework using a number of representative complex systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bedb3203006239c598b64a69777f9f9b9613ed" target='_blank'>
              From Fourier to Neural ODEs: Flow Matching for Modeling Complex Systems
              </a>
            </td>
          <td>
            Xin Li, Jingdong Zhang, Qunxi Zhu, Chengli Zhao, Xue Zhang, Xiaojun Duan, Wei Lin
          </td>
          <td>2024-05-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mechanistic models of genetic interactions are rarely feasible due to a lack of information and computational challenges. Alternatively, machine learning (ML) approaches may predict gene interactions if provided with enough data but they lack interpretability. Here, we propose an ML approach for interpretable genotype-fitness mapping, the Direct-Latent Interpretable Model (D-LIM). The neural network is built on a strong hypothesis: mutations in different genes cause independent effects in phenotypes, which then interact via non-linear relationships to determine fitness. D-LIM predicts genotype-fitness maps for combinations of mutations in multiple genes with state-of-the-art accuracy, showing the validity of the hypothesis in the case of a deep mutational scanning in a metabolic pathway. The hypothesisdriven structure of D-LIM offers interpretable features reminiscent of mechanistic models: the inference of phenotypes, fitness extrapolation outside of the data domain, and enhanced prediction in low-data regimes by the integration of prior knowledge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708c731d4cacd76d9401b0f0949c1422b368d303" target='_blank'>
              Hypothesis-driven interpretable neural network for interactions between genes
              </a>
            </td>
          <td>
            Shuhui Wang, Alexandre Allauzen, Philippe Nghe, Vaitea Opuu
          </td>
          <td>2024-04-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Standard-of-care treatment regimens have long been designed for maximal cell killing, yet these strategies often fail when applied to metastatic cancers due to the emergence of drug resistance. Adaptive treatment strategies have been developed as an alternative approach, dynamically adjusting treatment to suppress the growth of treatment-resistant populations and thereby delay, or even prevent, tumor progression. Promising clinical results in prostate cancer indicate the potential to optimize adaptive treatment protocols. Here, we applied deep reinforcement learning (DRL) to guide adaptive drug scheduling and demonstrated that these treatment schedules can outperform the current adaptive protocols in a mathematical model calibrated to prostate cancer dynamics, more than doubling the time to progression. The DRL strategies were robust to patient variability, including both tumor dynamics and clinical monitoring schedules. The DRL framework could produce interpretable, adaptive strategies based on a single tumor burden threshold, replicating and informing optimal treatment strategies. The DRL framework had no knowledge of the underlying mathematical tumor model, demonstrating the capability of DRL to help develop treatment strategies in novel or complex settings. Finally, a proposed five-step pathway, which combined mechanistic modeling with the DRL framework and integrated conventional tools to improve interpretability compared to traditional "black-box" DRL models, could allow translation of this approach to the clinic. Overall, the proposed framework generated personalized treatment schedules that consistently outperformed clinical standard-of-care protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6e1406ae7f5d35249e430eca7bc7c00ba8d4b40" target='_blank'>
              Mathematical Model-Driven Deep Learning Enables Personalized Adaptive Therapy.
              </a>
            </td>
          <td>
            Kit Gallagher, Maximillian Strobl, Derek S Park, Fabian C. Spoendlin, Robert A Gatenby, P. Maini, Alexander R. A. Anderson
          </td>
          <td>2024-04-03</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Motivation Mathematical modelling plays a crucial role in understanding of inter- and intracellular signalling processes. Currently, ordinary differential equations (ODEs) are the predominant approach utilized in systems biology for modelling such pathways. While such mechanistic ODE models offer interpretability, they also suffer from limitations, including the need to consider all relevant compounds, resulting in large models that are difficult to handle numerically and require extensive data. Results In our previous work, we introduced the retarded transient function (RTF) as an alternative method for modelling the temporal response of signalling pathways. In this work, we extend the RTF approach to also integrate concentration or dose-dependencies of dynamics to fully cover the application range of ordinary differential equation (ODE) models. This extension enables time- and dose-dependent predictions and offers an intuitive means to investigate and characterize signalling differences between biological conditions or cell types. To demonstrate the applicability of our extended RTF approach, we employ data from time- and dose-dependent inflammasome activation in primary murine bone marrow-derived dendritic cells (BMDCs) treated with tyrosine kinase inhibitors. Our results show the effectiveness of the extended RTF approach as a generic framework for modelling dose-dependent kinetic data as they typically occur in cellular signalling. The presented approach offers intuitive interpretability of signal dynamics and enables predictive modelling of time- and dose-dependencies even if only individual cellular components are quantified. Availability The presented approach is available within the MATLAB-based Data2Dynamics modelling toolbox at https://github.com/Data2Dynamics and as R code at https://github.com/kreutz-lab/RTF. Contact clemens.kreutz@uniklinik-freiburg.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acc3791d6241d168aa1cb932106f46769c05b26a" target='_blank'>
              Dynamic modelling of signalling pathways when ODEs are not feasible
              </a>
            </td>
          <td>
            Timo Rachel, Eva Brombacher, Svenja Wöhrle, Olaf Groß, Clemens Kreutz
          </td>
          <td>2024-04-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Motivation Mechanistic modeling based on ordinary differential equations has led to numerous findings in systems biology by integrating prior knowledge and experimental data. However, the manual curation of knowledge necessary when constructing models poses a bottleneck. As the speed of knowledge accumulation continues to grow, there is a demand for a scalable means of constructing executable models. Results We previously introduced BioMASS—an open-source, Python-based framework–to construct, simulate, and analyze mechanistic models of signaling networks. With one of its features, Text2Model, BioMASS allows users to define models in a natural language-like format, thereby facilitating the construction of large-scale models. We demonstrate that Text2Model can serve as a tool for integrating external knowledge for mathematical modeling by generating Text2Model files from a pathway database or through the use of a large language model, and simulating its dynamics through BioMASS. Our findings reveal the tool's capabilities to encourage exploration from prior knowledge and pave the way for a fully data-driven approach to constructing mathematical models. Availability and implementation The code and documentation for BioMASS are available at https://github.com/biomass-dev/biomass and https://biomass-core.readthedocs.io, respectively. The code used in this article are available at https://github.com/okadalabipr/text2model-from-knowledge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72c387266bbc39d831e5236216760fb26dc69509" target='_blank'>
              Extending BioMASS to construct mathematical models from external knowledge
              </a>
            </td>
          <td>
            Kiwamu Arakane, Hiroaki Imoto, Fabian Ormersbach, Mariko Okada
          </td>
          <td>2024-04-04</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="INTRODUCTION
This review explores the transformative impact of machine learning (ML) on carcinogenicity prediction within drug development. It discusses the historical context and recent advancements, emphasizing the significance of ML methodologies in overcoming challenges related to data interpretation, ethical considerations, and regulatory acceptance.


AREAS COVERED
The review comprehensively examines the integration of ML, deep learning, and diverse artificial intelligence (AI) approaches in various aspects of drug development safety assessments. It explores applications ranging from early-phase compound screening to clinical trial optimization, highlighting the versatility of ML in enhancing predictive accuracy and efficiency.


EXPERT OPINION
Through the analysis of traditional approaches such as in vivo rodent bioassays and in vitro assays, the review underscores the limitations and resource intensity associated with these methods. It provides expert insights into how ML offers innovative solutions to address these challenges, revolutionizing safety assessments in drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61374a9fb6566a155d65ae679b0e9a760d4938b3" target='_blank'>
              Recent progress in machine learning approaches for predicting carcinogenicity in drug development.
              </a>
            </td>
          <td>
            Nguyen Quoc Khanh Le, Thi-Xuan Tran, Phung-Anh Nguyen, Trang-Thi Ho, Van-Nui Nguyen
          </td>
          <td>2024-05-14</td>
          <td>Expert opinion on drug metabolism & toxicology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND
Heart failure poses a significant challenge in the global health domain, and accurate prediction of mortality is crucial for devising effective treatment plans. In this study, we employed a Seq2Seq model from deep learning, integrating 12 patient features. By finely modeling continuous medical records, we successfully enhanced the accuracy of mortality prediction.


OBJECTIVE
The objective of this research was to leverage the Seq2Seq model in conjunction with patient features for precise mortality prediction in heart failure cases, surpassing the performance of traditional machine learning methods.


METHODS
The study utilized a Seq2Seq model in deep learning, incorporating 12 patient features, to intricately model continuous medical records. The experimental design aimed to compare the performance of Seq2Seq with traditional machine learning methods in predicting mortality rates.


RESULTS
The experimental results demonstrated that the Seq2Seq model outperformed conventional machine learning methods in terms of predictive accuracy. Feature importance analysis provided critical patient risk factors, offering robust support for formulating personalized treatment plans.


CONCLUSIONS
This research sheds light on the significant applications of deep learning, specifically the Seq2Seq model, in enhancing the precision of mortality prediction in heart failure cases. The findings present a valuable direction for the application of deep learning in the medical field and provide crucial insights for future research and clinical practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76e2e69a438ba092dfabca2e8228bc14c6ca235" target='_blank'>
              Applications of deep learning models in precision prediction of survival rates for heart failure patients.
              </a>
            </td>
          <td>
            Qiaohui Zhang, Demin Xu
          </td>
          <td>2024-04-12</td>
          <td>Technology and health care : official journal of the European Society for Engineering and Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Causal representation learning promises to extend causal models to hidden causal variables from raw entangled measurements. However, most progress has focused on proving identifiability results in different settings, and we are not aware of any successful real-world application. At the same time, the field of dynamical systems benefited from deep learning and scaled to countless applications but does not allow parameter identification. In this paper, we draw a clear connection between the two and their key assumptions, allowing us to apply identifiable methods developed in causal representation learning to dynamical systems. At the same time, we can leverage scalable differentiable solvers developed for differential equations to build models that are both identifiable and practical. Overall, we learn explicitly controllable models that isolate the trajectory-specific parameters for further downstream tasks such as out-of-distribution classification or treatment effect estimation. We experiment with a wind simulator with partially known factors of variation. We also apply the resulting model to real-world climate data and successfully answer downstream causal questions in line with existing literature on climate change.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/012edc12bb8f81586eba3ee451de916124498e06" target='_blank'>
              Marrying Causal Representation Learning with Dynamical Systems for Science
              </a>
            </td>
          <td>
            Dingling Yao, Caroline Muller, Francesco Locatello
          </td>
          <td>2024-05-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="We consider a Graph Neural Network (GNN) non-Markovian modeling framework to identify coarse-grained dynamical systems on graphs. Our main idea is to systematically determine the GNN architecture by inspecting how the leading term of the Mori-Zwanzig memory term depends on the coarse-grained interaction coefficients that encode the graph topology. Based on this analysis, we found that the appropriate GNN architecture that will account for $K$-hop dynamical interactions has to employ a Message Passing (MP) mechanism with at least $2K$ steps. We also deduce that the memory length required for an accurate closure model decreases as a function of the interaction strength under the assumption that the interaction strength exhibits a power law that decays as a function of the hop distance. Supporting numerical demonstrations on two examples, a heterogeneous Kuramoto oscillator model and a power system, suggest that the proposed GNN architecture can predict the coarse-grained dynamics under fixed and time-varying graph topologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c24fe27eaf7f498fa7256c6c06dd99bcf8df096" target='_blank'>
              Learning Coarse-Grained Dynamics on Graph
              </a>
            </td>
          <td>
            Yin Yu, John Harlim, , Yan Li
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The spread of infectious diseases is a highly complex spatiotemporal process, difficult to understand, predict, and effectively respond to. Machine learning and artificial intelligence (AI) have achieved impressive results in other learning and prediction tasks; however, while many AI solutions are developed for disease prediction, only a few of them are adopted by decision-makers to support policy interventions. Among several issues preventing their uptake, AI methods are known to amplify the bias in the data they are trained on. This is especially problematic for infectious disease models that typically leverage large, open, and inherently biased spatiotemporal data. These biases may propagate through the modeling pipeline to decision-making, resulting in inequitable policy interventions. Therefore, there is a need to gain an understanding of how the AI disease modeling pipeline can mitigate biased input data, in-processing models, and biased outputs. Specifically, our vision is to develop a large-scale micro-simulation of individuals from which human mobility, population, and disease ground truth data can be obtained. From this complete dataset – which may not reflect the real world – we can sample and inject different types of bias. By using the sampled data in which bias is known (as it is given as the simulation parameter), we can explore how existing solutions for fairness in AI can mitigate and correct these biases and investigate novel AI fairness solutions. Achieving this vision would result in improved trust in such models for informing fair and equitable policy interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488b3d0489e2d6d305e64ce2ec54e6c86a1e8ccb" target='_blank'>
              Leveraging Simulation Data to Understand Bias in Predictive Models of Infectious Disease Spread
              </a>
            </td>
          <td>
            Andreas Züfle, Flora Salim, Taylor Anderson, M. Scotch, Li Xiong, Kacper Sokol, Hao Xue, Ruochen Kong, David Heslop, Hye-Young Paik, C. R. MacIntyre
          </td>
          <td>2024-04-25</td>
          <td>ACM Transactions on Spatial Algorithms and Systems</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Tumor growth inhibition (TGI) modeling attempts to describe the time course changes in tumor size for patients undergoing cancer therapy. TGI models present several advantages over traditional exposure-response models that are based explicitly on clinical end points and have become a popular tool in the pharmacometrics community. Unfortunately, the data required to fit TGI models, namely longitudinal tumor measurements, are sparse or often not available in literature or publicly accessible databases. On the contrary, common end points such as progression-free survival (PFS) and objective response rate (ORR) are directly derived from longitudinal tumor measurements and are routinely published. To this end, a Bayesian generative model relating underlying tumor dynamics to summary PFS and ORR data is introduced to learn TGI model parameters using only published summary data. The parameterized model can describe the tumor dynamics, quantify treatment effect, and account for differences in the study population. The utility of this model is shown by applying it to several published studies, and learned parameters are combined to simulate an in silico trial of a novel combination therapy in a novel setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5334e76ce88316ffc777a0522ed6c120271486bb" target='_blank'>
              A novel Bayesian generative approach for estimating tumor dynamics from published studies.
              </a>
            </td>
          <td>
            Arya Pourzanjani, Saurabh Modi, Jamie Connarn, Xinwen Zhang, V. Upreti, Chih-Wei Lin, Khamir Mehta
          </td>
          <td>2024-05-22</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Serum creatinine in neonates follows complex dynamics due to maturation processes, most pronounced in the first few weeks of life. The development of a mechanism-based model describing complex dynamics requires high expertise in pharmacometric (PMX) modeling and substantial model development time. A recently published machine learning (ML) approach of low-dimensional neural ordinary differential equations (NODEs) is capable of modeling such data from newborns automatically. However, this efficient data-driven approach in itself does not result in a clinically interpretable model. In this work, an approach to deriving an interpretable model with reasonable PMX-type functions is presented. This "translation" was applied to derive a PMX model for serum creatinine in neonates considering maturation processes and covariates. The developed model was compared to a previously published mechanism-based PMX model whereas both models had similar mechanistic structures. The developed model was then utilized to simulate serum creatinine concentrations in the first few weeks of life considering different covariate values for gestational age and birth weight. The reference serum creatinine values derived from these simulations are consistent with observed serum creatinine values and previously published reference values. Thus, the presented NODE-based ML approach to model complex serum creatinine dynamics in newborns and derive interpretable, mathematical-statistical components similar to those in a conventional PMX model demonstrates a novel, viable approach to facilitate the modeling of complex dynamics in clinical settings and pediatric drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fae58343bde99a83efb245281ab69b262f4cd7f" target='_blank'>
              Applying Neural ODEs to Derive a Mechanism-Based Model for Characterizing Maturation-Related Serum Creatinine Dynamics in Preterm Newborns.
              </a>
            </td>
          <td>
            Dominic Stefan Bräm, Gilbert Koch, Karel Allegaert, J. van den Anker, M. Pfister
          </td>
          <td>2024-05-16</td>
          <td>Journal of clinical pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The intersection of precision medicine and artificial intelligence (AI) holds profound implications for cancer treatment, with the potential to significantly advance our understanding of drug responses based on the intricate architecture of tumor cells. A recent study by Park and colleagues titled "A deep learning model of tumor cell architecture elucidates response and resistance to CDK4/6 inhibitors," epitomizes this intersection by leveraging an interpretable deep learning model grounded in a comprehensive map of multiprotein assemblies in cancer, known as Nested Systems in Tumors (NeST). This study not only elucidates mechanisms underlying the response to CDK4/6 inhibitors in breast cancer therapy but also highlights the critical role of model interpretability leading to new mechanistic insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6dbb19e42b7a66ce2daaa8d8f0f50ee5999758" target='_blank'>
              How Artificial Intelligence Unravels the Complex Web of Cancer Drug Response.
              </a>
            </td>
          <td>
            Olivier Elemento
          </td>
          <td>2024-04-08</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d751c25b5da56dee42debbf484a829190c3d74b" target='_blank'>
              PrCRS: a prediction model of severe CRS in CAR-T therapy based on transfer learning
              </a>
            </td>
          <td>
            Zhenyu Wei, Chengkui Zhao, Min Zhang, Jiayu Xu, Nan Xu, Shiwei Wu, Xiaohui Xin, Lei Yu, Weixing Feng
          </td>
          <td>2024-05-20</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background
Polygenic risk scores (PRS) are linear combinations of genetic markers weighted by effect size that are commonly used to predict disease risk. For complex heritable diseases such as late-onset Alzheimer's disease (LOAD), PRS models fail to capture much of the heritability. Additionally, PRS models are highly dependent on the population structure of the data on which effect sizes are assessed and have poor generalizability to new data.


Objective
The goal of this study is to construct a paragenic risk score that, in addition to single genetic marker data used in PRS, incorporates epistatic interaction features and machine learning methods to predict risk for LOAD.


Methods
We construct a new state-of-the-art genetic model for risk of Alzheimer's disease. Our approach innovates over PRS models in two ways: First, by directly incorporating epistatic interactions between SNP loci using an evolutionary algorithm guided by shared pathway information; and second, by estimating risk via an ensemble of non-linear machine learning models rather than a single linear model. We compare the paragenic model to several PRS models from the literature trained on the same dataset.


Results
The paragenic model is significantly more accurate than the PRS models under 10-fold cross-validation, obtaining an AUC of 83% and near-clinically significant matched sensitivity/specificity of 75%. It remains significantly more accurate when evaluated on an independent holdout dataset and maintains accuracy within APOE genotype strata.


Conclusions
Paragenic models show potential for improving disease risk prediction for complex heritable diseases such as LOAD over PRS models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48e4d2738533375cc9d9f9818d7c7df0ce79e61" target='_blank'>
              Epistatic Features and Machine Learning Improve Alzheimer's Disease Risk Prediction Over Polygenic Risk Scores.
              </a>
            </td>
          <td>
            Stephen Hermes, Janet Cady, Steven Armentrout, J. O'Connor, Sarah Carlson Holdaway, C. Cruchaga, Thomas Wingo, E. Greytak
          </td>
          <td>2024-05-23</td>
          <td>Journal of Alzheimer's disease : JAD</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="This research presents a novel deep learning model that effectively customizes treatment regimens for patients diagnosed with colorectal cancer, yielding exceptional results. The model's 97% success rate in predicting treatment reactions on the “The Cancer Genome Atlas (TCGA)” dataset provides evidence of its effectiveness. The model demonstrates its capacity to reduce false positives and negatives through its recall of 0.86 and accuracy of 0.91. The model exhibited exceptional discriminatory capability, as evidenced by its AUC-ROC value of 0.89. A balanced measure of accuracy and recall, the F1-score of 0.88, further suggests that the model successfully attained consistent performance across all treatment categories. The results of this research illustrate the immense potential of deep learning to transform the field of colorectal cancer treatment through the provision of tailored recommendations. This research makes a valuable contribution to the field of precision medicine by illuminating the transformative potential of data-driven models in enhancing patient outcomes and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cc7527037b5be657f492d5f7a9f2bda331c72a3" target='_blank'>
              Deep Learning for Personalized Treatment Strategies in Colorectal Tumor Patients
              </a>
            </td>
          <td>
            G. Vinudevi, S. P. Vijayaragavan, M. Anand, K. Subbulakshmi
          </td>
          <td>2024-04-18</td>
          <td>2024 International Conference on Advances in Data Engineering and Intelligent Computing Systems (ADICS)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="As the aging population grows, particularly for the baby boomer generation, the United States is witnessing a significant increase in the elderly population experiencing multifunctional disabilities. These disabilities, stemming from a variety of chronic diseases, injuries, and impairments, present a complex challenge due to their multidimensional nature, encompassing both physical and cognitive aspects. Traditional methods often use univariate regression-based methods to model and predict single degradation conditions and assume population homogeneity, which is inadequate to address the complexity and diversity of aging-related degradation. This study introduces a novel framework for multi-functional degradation modeling that captures the multidimensional (e.g., physical and cognitive) and heterogeneous nature of elderly disabilities. Utilizing deep learning, our approach predicts health degradation scores and uncovers latent heterogeneity from elderly health histories, offering both efficient estimation and explainable insights into the diverse effects and causes of aging-related degradation. A real-case study demonstrates the effectiveness and marks a pivotal contribution to accurately modeling the intricate dynamics of elderly degradation, and addresses the healthcare challenges in the aging population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fadd8122d115d7f50128bf90717b9216a98f47c" target='_blank'>
              Deep Representation Learning for Multi-functional Degradation Modeling of Community-dwelling Aging Population
              </a>
            </td>
          <td>
            Suiyao Chen, Xinyi Liu, Yulei Li, Jing Wu, Handong Yao
          </td>
          <td>2024-04-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Computational drug sensitivity models have the potential to improve therapeutic outcomes by identifying targeted drugs components that are tailored to the transcriptomic profile of a given primary tumor. The SMILES representation of molecules that is used by state-of-the-art drug-sensitivity models is not conducive for neural networks to generalize to new drugs, in part because the distance between atoms does not generally correspond to the distance between their representation in the SMILES strings. Graph-attention networks, on the other hand, are high-capacity models that require large training-data volumes which are not available for drug-sensitivity estimation. We develop a modular drug-sensitivity graph-attentional neural network. The modular architecture allows us to separately pre-train the graph encoder and graph-attentional pooling layer on related tasks for which more data are available. We observe that this model outperforms reference models for the use cases of precision oncology and drug discovery; in particular, it is better able to predict the specific interaction between drug and cell line that is not explained by the general cytotoxicity of the drug and the overall survivability of the cell line. The complete source code is available at https://zenodo.org/doi/10.5281/zenodo.8020945. All experiments are based on the publicly available GDSC data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1d95c29d808b802d81c2a052d2dbf5ee6f08b7" target='_blank'>
              Cancer drug sensitivity estimation using modular deep Graph Neural Networks
              </a>
            </td>
          <td>
            P. A. Campana, Paul Prasse, M. Lienhard, Kristina Thedinga, Ralf Herwig, Tobias Scheffer
          </td>
          <td>2024-04-04</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

The integration of artificial intelligence and machine learning holds great
promise for enhancing healthcare institutions and providing fresh perspectives on the origins and
advancement of long-term illnesses. In the healthcare sector, artificial intelligence and machine learning
are used to address supply and demand concerns, genomic applications, and new advancements
in drug development, cancer, and heart disease.



The article explores the ways that machine learning, AI, precision medicine, and genomics
are changing healthcare. The essay also discusses how AI's examination of various patient data could
enhance healthcare institutions, provide fresh insights into chronic conditions, and advance precision
medicine. The potential uses of machine learning for genome analysis are also examined in the paper,
particularly about genetic biomarker-based disease risk and symptom prediction.



The challenges posed by the phenotype-genotype relationship are examined, as well as
the significance of comprehending disease pathways in order to create tailored treatments. Moreover,
it offers a streamlined and modularized method that predicts how genotypes affect cell properties
using machine-learning models, enabling the development of personalized drugs. The collective feedback
highlights the rapid interdisciplinary growth of medical genomics following the completion of
the Human Genome Project. It also emphasizes how important genomic data is for improving
healthcare outcomes and facilitating personalized medicine.



The study's conclusions point to a revolutionary shift in healthcare: the application of
AI/ML to illness control. Even though these innovations have a lot of potential benefits, problems
like algorithm interpretability and ethical issues need to be worked out before they can be successfully
incorporated into routine medical practice. Using machine learning in medicine has enormous potential
benefits for the biotech industry. Further research, ongoing regulatory frameworks, and collaboration
between medical professionals and data analysts are necessary to fully utilize machine learning
as well as artificial intelligence in disease management.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/934c7ca8ae423804df2848baa556409ec7854a1b" target='_blank'>
              A State-of-the-art Review on Artificial Intelligence and Machine Learning Applications
              </a>
            </td>
          <td>
            Rishav Sharma, R. Malviya, Prerna Uniyal, Bhupendra Prajapati
          </td>
          <td>2024-04-24</td>
          <td>Current Drug Therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="In applications such as biomedical studies, epidemiology, and social sciences, recurrent events often co-occur with longitudinal measurements and a terminal event, such as death. Therefore, jointly modeling longitudinal measurements, recurrent events, and survival data while accounting for their dependencies is critical. While joint models for the three components exist in statistical literature, many of these approaches are limited by heavy parametric assumptions and scalability issues. Recently, incorporating deep learning techniques into joint modeling has shown promising results. However, current methods only address joint modeling of longitudinal measurements at regularly-spaced observation times and survival events, neglecting recurrent events. In this paper, we develop TransformerLSR, a flexible transformer-based deep modeling and inference framework to jointly model all three components simultaneously. TransformerLSR integrates deep temporal point processes into the joint modeling framework, treating recurrent and terminal events as two competing processes dependent on past longitudinal measurements and recurrent event times. Additionally, TransformerLSR introduces a novel trajectory representation and model architecture to potentially incorporate a priori knowledge of known latent structures among concurrent longitudinal variables. We demonstrate the effectiveness and necessity of TransformerLSR through simulation studies and analyzing a real-world medical dataset on patients after kidney transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be00caf52727fad3464be1a89970b50716795384" target='_blank'>
              TransformerLSR: Attentive Joint Model of Longitudinal Data, Survival, and Recurrent Events with Concurrent Latent Structure
              </a>
            </td>
          <td>
            Zhiyue Zhang, Yao Zhao, Yan Xu
          </td>
          <td>2024-04-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The inference of gene regulatory networks (GRNs) from gene expression profiles has been a key issue in systems biology, prompting many researchers to develop diverse computational methods. However, most of these methods do not reconstruct directed GRNs with regulatory types because of the lack of benchmark datasets or defects in the computational methods. Here, we collect benchmark datasets and propose a deep learning-based model, DeepFGRN, for reconstructing fine gene regulatory networks (FGRNs) with both regulation types and directions. In addition, the GRNs of real species are always large graphs with direction and high sparsity, which impede the advancement of GRN inference. Therefore, DeepFGRN builds a node bidirectional representation module to capture the directed graph embedding representation of the GRN. Specifically, the source and target generators are designed to learn the low-dimensional dense embedding of the source and target neighbors of a gene, respectively. An adversarial learning strategy is applied to iteratively learn the real neighbors of each gene. In addition, because the expression profiles of genes with regulatory associations are correlative, a correlation analysis module is designed. Specifically, this module not only fully extracts gene expression features, but also captures the correlation between regulators and target genes. Experimental results show that DeepFGRN has a competitive capability for both GRN and FGRN inference. Potential biomarkers and therapeutic drugs for breast cancer, liver cancer, lung cancer and coronavirus disease 2019 are identified based on the candidate FGRNs, providing a possible opportunity to advance our knowledge of disease treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85fac7e67a50e66dcd25346978f120d9c51b38f4" target='_blank'>
              DeepFGRN: inference of gene regulatory network with regulation type based on directed graph embedding
              </a>
            </td>
          <td>
            Zhen Gao, Yansen Su, Junfeng Xia, Ruifen Cao, Yun Ding, Chun-Hou Zheng, Pi-Jing Wei
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Traditional cancer treatment methods have become less effective due to the increasing diversity of cancer types. To address this, precision medicine has gained support within the medical community. This approach tailors treatment to individual patients based on their specific disease characteristics. However, a major challenge lies in accurately predicting how a patient will respond to a specialized drug. Numerous machine learning-based predictive systems have been developed to address this challenge. These systems utilize genomic signatures and the chemical structure of drugs to predict drug activity. In this paper, we introduce a Multi-Layer Perceptron (MLP) based system for predicting the response of anticancer drugs. Our system utilizes hybrid features derived from both genetic expression and the chemical structure of drugs. It is developed using the well-known GDSC dataset (Genomics of Drug Sensitivity in Cancer). Our system achieved a lower Root Mean Square Error (RMSE) value of 0.889, in contrast to the RMSE value of 0.983 obtained by the current state-of-the-art (SOTA) system, SwNet. This indicates superior predictive accuracy. The findings suggest that our proposed research holds promise for the development of targeted drugs for anticancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c82b5c941ccd82787222fb672dac393f13e9ea" target='_blank'>
              Developing MLP based prediction system for anticancer drug response using hybrid features of genomics and cheminformatics
              </a>
            </td>
          <td>
            Awais Raza Zaidi, Muhammad Bilal, Tuba Majid, Abdul Majid
          </td>
          <td>2024-04-24</td>
          <td>Lahore Garrison University Research Journal of Computer Science and Information Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Anti-cancer response of cell lines to drugs is in urgent need for individualized precision medical decision-making in the era of precision medicine. Measurements with wet-experiments is time-consuming and expensive and it is almost impossible for wide ranges of application. The design of computational models that can precisely predict the responses between drugs and cell lines could provide a credible reference for further research. Existing methods of response prediction based on matrix factorization or neural networks have revealed that both linear or nonlinear latent characteristics are applicable and effective for the precise prediction of drug responses. However, the majority of them consider only linear or nonlinear relationships for drug response prediction. Herein, we propose a Dual Branch Deep Neural Matrix Factorization (DBDNMF) method to address the above-mentioned issues. DBDNMF learns the latent representation of drugs and cell lines through flexible inputs and reconstructs the partially observed matrix through a series of hidden neural network layers. Experimental results on the datasets of Cancer Cell Line Encyclopedia (CCLE) and Genomics of Drug Sensitivity in Cancer (GDSC) show that the accuracy of drug prediction exceeds state-of-the-art drug response prediction algorithms, demonstrating its reliability and stability. The hierarchical clustering results show that drugs with similar response levels tend to target similar signaling pathway, and cell lines coming from the same tissue subtype tend to share the same pattern of response, which are consistent with previously published studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c139ad04d41a207bd4aaa54f2acb208eb45a04d" target='_blank'>
              DBDNMF: A Dual Branch Deep Neural Matrix Factorization method for drug response prediction
              </a>
            </td>
          <td>
            Hui Liu, Feng Wang, Jian Yu, Yong Pan, Chaoju Gong, Liang Zhang, Lin Zhang
          </td>
          <td>2024-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="High-throughput screening platforms for the profiling of drug sensitivity of hundreds of cancer cell lines (CCLs) have generated large datasets that hold the potential to unlock targeted, anti-tumor therapies. In this study, we leveraged these datasets to create predictive models of cancer cells drug sensitivity. To this aim we trained explainable machine learning algorithms by employing cell line transcriptomics to predict the growth inhibitory potential of drugs. We used large language models (LLMs) to expand descriptions of the mechanisms of action (MOA) for each drug starting from available annotations, which were matched to the semantically closest pathways from reference knowledge bases. By leveraging this AI-curated resource, and the interpretability of our model, we demonstrated that pathways enriched for genes crucial for prediction often matched known drug-MOAs and essential genes, suggesting that our models learned the molecular determinants of drug response. Furthermore, we demonstrated that by incorporating only LLM-curated genes associated with MOAs, we enhanced the predictive accuracy of our drug models. To enhance translatability to a clinical setting, we employed a pipeline to align bulk RNAseq from CCLs, used for training the models, to those from patient samples, used for inference. We proved the effectiveness of our approach on TCGA samples, where patients’ best scoring drugs matched those prescribed for their cancer type. We further showed its usefulness by predicting and experimentally validating effective drugs for the patients of two highly lethal solid tumors, i.e. pancreatic cancer and glioblastoma. In summary, our method facilitates the inference and interpretation of cancer cell line drug sensitivity and holds potential to effectively translate them into new cancer therapeutics. Highlights -Interpretable drug-response prediction models were trained on large scale pharmacogenomics data sets (i.e. GDSC and PRISM). -Large language models were used to enhance the curation of biological pathways associated to drugs MOA -Unbiased interpretation of the models demonstrated learning of drug MOAs and gene essentiality -Inference of TCGA cohort samples recovered mono- and combination cancer drug prescriptions and indicate potential repurposing candidates. -Drug candidates predicted from bulk RNAseq samples of pancreatic cancer and glioblastoma were experimentally validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/748f9da063fa715fefeaa0bfbccf264963bf4d60" target='_blank'>
              Learning and actioning general principles of cancer cell drug sensitivity
              </a>
            </td>
          <td>
            Francesco Carli, Pierluigi Di Chiaro, M. Morelli, Chakit Arora, Luisa Bisceglia, Natalia De Oliveira Rosa, Alice Cortesi, S. Franceschi, F. Lessi, A. D. Di Stefano, O. Santonocito, Francesco Pasqualetti, P. Aretini, Pasquale Miglionico, G. Diaferia, Fosca Giannotti, Pietro Liò, Miquel Duran-Frigola, C. M. Mazzanti, Gioacchino Natoli, Francesco Raimondi
          </td>
          <td>2024-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="A problem in nonlinear and complex dynamical systems with broad applications is forecasting the occurrence of a critical transition based solely on data without knowledge about the system equations. When such a transition leads to system collapse, as often is the case, all the available data are from the pre-critical regime where the system still functions normally, making the prediction problem challenging. In recent years, a machine-learning based approach tailored to solving this difficult prediction problem, adaptable reservoir computing, has been articulated. This Perspective introduces the basics of this machine-learning scheme and describes representative results. The general setting is that the system dynamics live on a normal attractor with oscillatory dynamics at the present time and, as a bifurcation parameter changes into the future, a critical transition can occur after which the system switches to a completely different attractor, signifying system collapse. To predict a critical transition, it is essential that the reservoir computer not only learns the dynamical "climate" of the system of interest at some specific parameter value but, more importantly, discovers how the system dynamics changes with the bifurcation parameter. It is demonstrated that this capability can be endowed into the machine through a training process with time series from a small number of distinct, pre-critical parameter values, thereby enabling accurate and reliable prediction of the catastrophic critical transition. Three applications are presented: predicting crisis, forecasting amplitude death, and creating digital twins of nonlinear dynamical systems. Limitations and future perspectives are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1749f284391bc6f9be0f2b8f564c797144a8e75" target='_blank'>
              Adaptable reservoir computing: A paradigm for model-free data-driven prediction of critical transitions in nonlinear dynamical systems.
              </a>
            </td>
          <td>
            Shirin Panahi, Ying-Cheng Lai
          </td>
          <td>2024-05-01</td>
          <td>Chaos</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Artificial intelligence and machine learning enable the construction of predictive models, which are currently used to assist in decision-making throughout the process of drug discovery and development. These computational models can be used to represent the heterogeneity of a disease, identify therapeutic targets, design and optimize drug candidates, and evaluate the efficacy of these drugs on virtual patients or digital twins. By combining detailed patient characteristics with the prediction of potential drug-candidate properties, artificial intelligence promotes the emergence of a "computational" precision medicine, allowing for more personalized treatments, better tailored to patient specificities with the aid of such predictive models. Based on such new capabilities, a mixed reality approach to the development of new drugs is being adopted by the pharmaceutical industry, which integrates the outputs of predictive virtual models with real-world empirical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57cba1eb82f6e6919ebfe3e6d794d361cc3420d0" target='_blank'>
              [The revolution of AI in drug development].
              </a>
            </td>
          <td>
            Philippe Moingeon, Christiane Garbay, Muriel Dahan, Irène Fermont, Ali Benmakhlouf, Alain Gouyette, Pierre Poitou, Alain Saint-Pierre
          </td>
          <td>2024-04-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Early identification of safe and efficacious disease targets is crucial to alleviating the tremendous cost of drug discovery projects. However, existing experimental methods for identifying new targets are generally labor-intensive and failure-prone. On the other hand, computational approaches, especially machine learning-based frameworks, have shown remarkable application potential in drug discovery. In this work, we propose Progeni, a novel machine learning-based framework for target identification. In addition to fully exploiting the known heterogeneous biological networks from various sources, Progeni integrates literature evidence about the relations between biological entities to construct a probabilistic knowledge graph. Graph neural networks are then employed in Progeni to learn the feature embeddings of biological entities to facilitate the identification of biologically relevant target candidates. A comprehensive evaluation of Progeni demonstrated its superior predictive power over the baseline methods on the target identification task. In addition, our extensive tests showed that Progeni exhibited high robustness to the negative effect of exposure bias, a common phenomenon in recommendation systems, and effectively identified new targets that can be strongly supported by the literature. Moreover, our wet lab experiments successfully validated the biological significance of the top target candidates predicted by Progeni for melanoma and colorectal cancer. All these results suggested that Progeni can identify biologically effective targets and thus provide a powerful and useful tool for advancing the drug discovery process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d2fa8d9cb5d3a8fbde95561c2616b16dcfc840" target='_blank'>
              A probabilistic knowledge graph for target identification
              </a>
            </td>
          <td>
            Chang Liu, Kaimin Xiao, Cuinan Yu, Yipin Lei, Kangbo Lyu, Tingzhong Tian, Dan Zhao, Fengfeng Zhou, Haidong Tang, Jianyang Zeng
          </td>
          <td>2024-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The integration of deep learning technology within the domain of drug target discovery represents a cutting-edge approach in the pharmaceutical sciences. This paper delves into the methods and prevailing issues associated with the application of deep learning in the identification of new drug targets. We initiate with an exposition of the fundamental concepts and methodologies underpinning deep learning, subsequently illuminating its utilization in the realm of drug target discovery. Specific attention is given to the deployment of architectures like convolutional neural networks (CNNs) and recurrent neural networks (RNNs) among others.
By interpreting experimental outcomes, it has been discerned that deep learning potentially enhances the precision of identifying drug targets. Nonetheless, this technological arena is not devoid of challenges. A significant hurdle lies in the interpretability of the models—understanding why and how these models arrive at their conclusions is often non-trivial. Moreover, the efficacy of deep learning is heavily reliant on the quality and volume of the datasets employed; insufficient or poor-quality data can severely impede the performance and reliability of predictive models.Through this inquiry, we aim to furnish novel insights into the utilization of deep learning for drug target discovery and underscore the challenges that must be addressed. The perspectives offered herein are vital to propel the progress of this burgeoning field, paving the way for more effective and efficient drug discovery processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8e3a7e952c0b4f80fe17c1f6d1f509a0eaf19a" target='_blank'>
              Application and difficulties of deep learning in drug target discovery
              </a>
            </td>
          <td>
            Xiaoqian Kuang
          </td>
          <td>2024-04-16</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clinical trials involving systemic neoadjuvant treatments in breast cancer aim to shrink tumors before surgery while simultaneously allowing for controlled evaluation of biomarkers, toxicity, and suppression of distant (occult) metastatic disease. Yet neoadjuvant clinical trials are rarely preceded by preclinical testing involving neoadjuvant treatment, surgery, and post-surgery monitoring of the disease. Here we used a mouse model of spontaneous metastasis occurring after surgical removal of orthotopically implanted primary tumors to develop a predictive mathematical model of neoadjuvant treatment response to sunitinib, a receptor tyrosine kinase inhibitor (RTKI). Treatment outcomes were used to validate a novel mathematical kinetics-pharmacodynamics model predictive of perioperative disease progression. Longitudinal measurements of presurgical primary tumor size and postsurgical metastatic burden were compiled using 128 mice receiving variable neoadjuvant treatment doses and schedules (released publicly at https://zenodo.org/records/10607753). A non-linear mixed-effects modeling approach quantified inter-animal variabilities in metastatic dynamics and survival, and machine-learning algorithms were applied to investigate the significance of several biomarkers at resection as predictors of individual kinetics. Biomarkers included circulating tumor- and immune-based cells (circulating tumor cells and myeloid-derived suppressor cells) as well as immunohistochemical tumor proteins (CD31 and Ki67). Our computational simulations show that neoadjuvant RTKI treatment inhibits primary tumor growth but has little efficacy in preventing (micro)-metastatic disease progression after surgery and treatment cessation. Machine learning algorithms that included support vector machines, random forests, and artificial neural networks, confirmed a lack of definitive biomarkers, which shows the value of preclinical modeling studies to identify potential failures that should be avoided clinically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c8ff9b498ddc0ddca957f0ddbc9e36ba56facbc" target='_blank'>
              Machine-learning and mechanistic modeling of metastatic breast cancer after neoadjuvant treatment
              </a>
            </td>
          <td>
            S. Benzekry, M. Mastri, C. Nicolò, J. Ebos
          </td>
          <td>2024-05-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: This research aims to improve glioblastoma survival prediction by integrating MR images, clinical and molecular-pathologic data in a transformer-based deep learning model, addressing data heterogeneity and performance generalizability. Method: We propose and evaluate a transformer-based non-linear and non-proportional survival prediction model. The model employs self-supervised learning techniques to effectively encode the high-dimensional MRI input for integration with non-imaging data using cross-attention. To demonstrate model generalizability, the model is assessed with the time-dependent concordance index (Cdt) in two training setups using three independent public test sets: UPenn-GBM, UCSF-PDGM, and RHUH-GBM, each comprising 378, 366, and 36 cases, respectively. Results: The proposed transformer model achieved promising performance for imaging as well as non-imaging data, effectively integrating both modalities for enhanced performance (UPenn-GBM test-set, imaging Cdt 0.645, multimodal Cdt 0.707) while outperforming state-of-the-art late-fusion 3D-CNN-based models. Consistent performance was observed across the three independent multicenter test sets with Cdt values of 0.707 (UPenn-GBM, internal test set), 0.672 (UCSF-PDGM, first external test set) and 0.618 (RHUH-GBM, second external test set). The model achieved significant discrimination between patients with favorable and unfavorable survival for all three datasets (logrank p 1.9\times{10}^{-8}, 9.7\times{10}^{-3}, and 1.2\times{10}^{-2}). Conclusions: The proposed transformer-based survival prediction model integrates complementary information from diverse input modalities, contributing to improved glioblastoma survival prediction compared to state-of-the-art methods. Consistent performance was observed across institutions supporting model generalizability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d85f3cd49ade2f18962f198150caa353b17d7f" target='_blank'>
              Comprehensive Multimodal Deep Learning Survival Prediction Enabled by a Transformer Architecture: A Multicenter Study in Glioblastoma
              </a>
            </td>
          <td>
            A. Gomaa, Yixing Huang, Amr Hagag, Charlotte Schmitter, Daniel Hofler, T. Weissmann, Katharina Breininger, Manuel Schmidt, Jenny Stritzelberger, D. Delev, Roland Coras, A. Dorfler, Oliver Schnell, Benjamin Frey, U. Gaipl, S. Semrau, Christoph Bert, R. Fietkau, F. Putz
          </td>
          <td>2024-05-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="This paper presents a comprehensive investigation into the applicability and performance of two prominent growth models, namely, the Verhulst model and the Montroll model, in the context of modeling tumor cell growth dynamics. Leveraging the power of Physics-Informed Neural Networks (PINNs), we aim to assess and compare the predictive capabilities of these models against experimental data obtained from the growth patterns of tumor cells. We employed a dataset comprising detailed measurements of tumor cell growth to train and evaluate the Verhulst and Montroll models. By integrating PINNs, we not only account for experimental noise but also embed physical insights into the learning process, enabling the models to capture the underlying mechanisms governing tumor cell growth. Our findings reveal the strengths and limitations of each growth model in accurately representing tumor cell proliferation dynamics. Furthermore, the study sheds light on the impact of incorporating physics-informed constraints on the model predictions. The insights gained from this comparative analysis contribute to advancing our understanding of growth models and their applications in predicting complex biological phenomena, particularly in the realm of tumor cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99a19673371a8d65fd7ea004ec6b44d862e2d9fb" target='_blank'>
              Using Physics-Informed Neural Networks (PINNs) for Tumor Cell Growth Modeling
              </a>
            </td>
          <td>
            J. A. Rodrigues
          </td>
          <td>2024-04-16</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Precisely predicting drug-protein interactions (DPIs) is pivotal for drug discovery and advancing precision medicine. A significant challenge in this domain is the high-dimensional and heterogeneous data characterizing drug and protein attributes, along with their intricate interactions. In our study, we introduce a novel deep learning architecture: the Multi-view Variational Auto-Encoder embedded within a cascade Deep Forest (MVAE-DFDPnet). This framework adeptly learns ultra-low-dimensional embedding for drugs and proteins. Notably, our t-SNE analysis reveals that two-dimensional embedding can clearly define clusters corresponding to diverse drug classes and protein families. These ultra-low-dimensional embedding likely contribute to the enhanced robustness and generalizability of our MVAE-DFDPnet. Impressively, our model surpasses current leading methods on benchmark datasets, functioning in significantly reduced dimensional spaces. The model’s resilience is further evidenced by its sustained accuracy in predicting interactions involving novel drugs, proteins, and drug classes. Additionally, we have corroborated several newly identified DPIs with experimental evidence from the scientific literature. The code used to generate and analyze these results can be accessed from https://github.com/Macau-LYXia/MVAE-DFDPnet-V2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/887353b3b42ec83c47f0d43699e712fe2a516477" target='_blank'>
              Redefining the Game: MVAE-DFDPnet’s Low-Dimensional Embeddings for Superior Drug-Protein Interaction Predictions
              </a>
            </td>
          <td>
            Liang-Yong Xia, Yu Wu, Longfei Zhao, Leying Chen, Shiyi Zhang, Mengdi Wang, Jie Luo
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The DrugMatrix Database contains systematically generated toxicogenomics data from short-term in vivo studies for over 600 chemicals. However, most of the potential endpoints in the database are missing due to a lack of experimental measurements. We present our study on leveraging matrix factorization and machine learning methods to predict the missing values in the DrugMatrix, which includes gene expression across eight tissues on two expression platforms along with paired clinical chemistry, hematology, and histopathology measurements. One major challenge we encounter is the skewed distribution of the available measured data, in terms of both tissue sources and values. We propose a method, ToxiCompl, that applies systematic hybrid sampling guided by Bayesian optimization in conjunction with low-rank matrix factorization to recover the missing values. ToxiCompl achieves good training and validation performance from a machine learning perspective. We further conduct an in-depth validation of the predicted data from biological and toxicological perspectives with a series of analyses. These include examining the connectivity pattern of predicted gene expression responses, characterizing molecular pathway-level responses from sets of differentially expressed genes, evaluating known transcriptional biomarkers of tissue toxicity, and characterizing pre-dicted apical endpoints. Our analysis shows that the predicted differential gene expression, broadly speaking, aligns with what would be anticipated. For example, in most instances, our predicted differentially expressed gene lists offer a connectivity level comparable to that of measured data in connectivity analysis. Using Havcr1, a known transcriptional biomarker of kidney injury, we identify treatments that, based on the predicted expression data, manifest kidney toxicity in a manner that is mechanistically plausible and supported by the literature. Characterization of the predicted clinical chemistry data suggests that strong effects are relatively reliably predicted, while more subtle effects pose a greater challenge. In the case of histopathological prediction, we find a significant overprediction due to positivity bias in the measured data. Developing methods to deal with this bias is one of the areas we plan to target for future improvement. The main advantage of the ToxiCompl approach is that, in the absence of additional experimental data, it drastically extends the toxicogenomic landscape into a number of data-poor tissues, thereby allowing researchers to formulate mechanistic hypotheses about effects in tissues that have been underrepresented in the literature. All measured and predicted DrugMatrix data (i.e., gene expression, clinical chemistry, hematology, and histopathology) are available to the public through an intuitive GUI interface that allows for data retrieval, gene set analysis and high dimensional visualization of gene expression similarity (https://rstudio.niehs.nih.gov/complete_drugmatrix/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d2ffbc53f79544dfb9e97a8e711c4353e3d6948" target='_blank'>
              Completion of the DrugMatrix Toxicogenomics Database using ToxCompl
              </a>
            </td>
          <td>
            Guojing Cong, Robert M. Patton, Frank Chao, Daniel L. Svoboda, Warren M. Casey, Charles P. Schmitt, Charles Murphy, Jeremy N. Erickson, Parker A. Combs, Scott Auerbach
          </td>
          <td>2024-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Detecting the unintended adverse reactions of drugs (ADRs) is a crucial concern in pharmacological research. The experimental validation of drug–ADR associations often entails expensive and time-consuming investigations. Thus, a computational model to predict ADRs from known associations is essential for enhanced efficiency and cost-effectiveness. Here, we propose BiMPADR, a novel model that integrates drug gene expression into adverse reaction features using a message passing neural network on a bipartite graph of drugs and adverse reactions, leveraging publicly available data. By combining the computed adverse reaction features with the structural fingerprints of drugs, we predict the association between drugs and adverse reactions. Our models obtained high AUC (area under the receiver operating characteristic curve) values ranging from 0.861 to 0.907 in an external drug validation dataset under differential experiment conditions. The case study on multiple BET inhibitors also demonstrated the high accuracy of our predictions, and our model’s exploration of potential adverse reactions for HWD-870 has contributed to its research and development for market approval. In summary, our method would provide a promising tool for ADR prediction and drug safety assessment in drug discovery and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/550c39d0ae4adb47f3ffa20c53a61d872d671478" target='_blank'>
              BiMPADR: A Deep Learning Framework for Predicting Adverse Drug Reactions in New Drugs
              </a>
            </td>
          <td>
            Shuang Li, Liuchao Zhang, Liuying Wang, Jianxin Ji, Jia He, Xiaohan Zheng, Lei Cao, Kang Li
          </td>
          <td>2024-04-01</td>
          <td>Molecules</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In constructing Gene Regulatory Networks (GRNs), it is crucial to consider cellular heterogeneity and differential gene regulatory modules. However, traditional methods have predominantly focused on cellular heterogeneity, approaching the subject from a relatively narrow scope. We present HyperG-VAE, a Bayesian deep generative model that utilizes a hypergraph to model single-cell RNA sequencing (scRNA-seq) data. HyperG-VAE employs a cell encoder with a Structural Equation Model to address cellular heterogeneity and build GRNs, alongside a gene encoder using hypergraph self-attention to identify gene modules. Encoders are synergistically optimized by a decoder, enabling HyperG-VAE to excel in GRN inference, single-cell clustering, and data visualization, evidenced by benchmarks. Additionally, HyperG-VAE effectively reveals gene regulation patterns and shows robustness in varied downstream analyses, demonstrated using B cell development data in bone marrow. The interplay of encoders by the overlapping genes between predicted GRNs and gene modules is further validated by gene set enrichment analysis, underscoring that the gene encoder boosts the GRN inference. HyperG-VAE proves efficient in scRNA-seq data analysis and GRN inference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/910e673103b27a58525cf647fe4c8fe79d5c8ef7" target='_blank'>
              Inferring gene regulatory networks by hypergraph variational autoencoder
              </a>
            </td>
          <td>
            Guangxin Su, Hanchen Wang, Ying Zhang, A. Coster, Marc R. Wilkins, Pablo F. Canete, Di Yu, Yang Yang, Wenjie Zhang
          </td>
          <td>2024-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Simple Summary The major impact from this work is two-fold, first in providing a list of reproducible candidate drugs nominated against multiple glioblastoma (GBM) datasets, and second in providing drug–gene biomarkers of interest for GBM as well as reproducible computational pipelines for identifying drug–biomarker leads that could be extended to other cancer types beyond GBM. We believe the research community will be interested and can utilize our results here for the further development of biomarker-specific drug therapies in both GBM as well as other disease types. Abstract Glioblastoma multiforme (GBM) is the deadliest, most heterogeneous, and most common brain cancer in adults. Not only is there an urgent need to identify efficacious therapeutics, but there is also a great need to pair these therapeutics with biomarkers that can help tailor treatment to the right patient populations. We built patient drug response models by integrating patient tumor transcriptome data with high-throughput cell line drug screening data as well as Bayesian networks to infer relationships between patient gene expression and drug response. Through these discovery pipelines, we identified agents of interest for GBM to be effective across five independent patient cohorts and in a mouse avatar model: among them are a number of MEK inhibitors (MEKis). We also predicted phosphoglycerate dehydrogenase enzyme (PHGDH) gene expression levels to be causally associated with MEKi efficacy, where knockdown of this gene increased tumor sensitivity to MEKi and overexpression led to MEKi resistance. Overall, our work demonstrated the power of integrating computational approaches. In doing so, we quickly nominated several drugs with varying known mechanisms of action that can efficaciously target GBM. By simultaneously identifying biomarkers with these drugs, we also provide tools to select the right patient populations for subsequent evaluation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b533ec6d5b79d0a8dc0aa09ce2fb660cc0c67f5f" target='_blank'>
              Integration of Computational Pipeline to Streamline Efficacious Drug Nomination and Biomarker Discovery in Glioblastoma
              </a>
            </td>
          <td>
            D. Maeser, Robert F. Gruener, R. Galvin, Adam Lee, Tomoyuki Koga, F. Grigore, Yuta Suzuki, F. Furnari, Clark Chen, R. S. Huang
          </td>
          <td>2024-04-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Developing new pharmaceuticals is a costly and time-consuming endeavor fraught with significant safety risks. A critical aspect of drug research and disease therapy is discerning the existence of interactions between drugs and proteins. The evolution of deep learning (DL) in computer science has been remarkably aided in this regard in recent years. Yet, two challenges remain: (i) balancing the extraction of profound, local cohesive characteristics while warding off gradient disappearance and (ii) globally representing and understanding the interactions between the drug and target local attributes, which is vital for delivering molecular level insights indispensable to drug development. In response to these challenges, we propose a DL network structure, MolLoG, primarily comprising two modules: local feature encoders (LFE) and global interactive learning (GIL). Within the LFE module, graph convolution networks and leap blocks capture the local features of drug and protein molecules, respectively. The GIL module enables the efficient amalgamation of feature information, facilitating the global learning of feature structural semantics and procuring multihead attention weights for abstract features stemming from two modalities, providing biologically pertinent explanations for black-box results. Finally, predictive outcomes are achieved by decoding the unified representation via a multilayer perceptron. Our experimental analysis reveals that MolLoG outperforms several cutting-edge baselines across four data sets, delivering superior overall performance and providing satisfactory results when elucidating various facets of drug-target interaction predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47d7134e78582b908d74226e67442dfea752621" target='_blank'>
              MolLoG: A Molecular Level Interpretability Model Bridging Local to Global for Predicting Drug Target Interactions.
              </a>
            </td>
          <td>
            Bao-Ming Feng, Yuanyuan Zhang, Xiao-Chen Zhou, Jin-Long Wang, Yinfei Feng
          </td>
          <td>2024-05-06</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Temporal link prediction is one of the most important tasks for predicting time-varying links by capturing dynamics within complex networks. However, it suffers from difficulties such as vulnerability to adversarial attacks and inadaptation to distinct evolutionary patterns. In this article, we propose a robust temporal link prediction architecture via stable gated models with reinforcement learning (SAGE-RL) consisting of a state encoding network (SEN) and a self-adaptive policy network (SPN). The former is utilized to capture network dynamics, while the latter helps the former adapt to distinct evolutionary patterns across various time periods. Within the SEN, a novel stable gate is introduced to ensure multiple spatiotemporal dependency paths and defend against adversarial attacks. An SPN is proposed to select different SEN instances by approximating the optimal action function, thereby adapting to various evolutionary patterns to learn the robust temporal and structural features from dynamic complex networks. It is proven that SAGE-LR with integral Lipschitz graph convolution is stable to relative perturbations in dynamic complex networks. With the aid of extensive experiments on five real-world graph benchmarks, SAGE-LR is shown to substantially outperform current state-of-the-art approaches in terms of precision and stability of temporal link prediction and ability to successfully defend against various attacks. We also implement the temporal link prediction in shipping transaction networks, which forecast effectively its potential transaction risks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc8eab5464296d20d63e8638a6cc53aff28a678d" target='_blank'>
              Robust Temporal Link Prediction in Dynamic Complex Networks via Stable Gated Models With Reinforcement Learning.
              </a>
            </td>
          <td>
            Liping Yang, Hongbo Liu, Daoqiang Sun, Seán F. McLoone, Kai Liu, C. L. P. Chen
          </td>
          <td>2024-05-14</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pathway engineering offers a promising avenue for sustainable chemical production. The design of efficient production systems requires understanding complex host-pathway interactions that shape the metabolic phenotype. While genome-scale metabolic models are widespread tools for studying static host-pathway interactions, it remains a challenge to predict dynamic effects such as metabolite accumulation or enzyme overexpression during the course of fermentation. Here, we propose a novel strategy to integrate kinetic pathway models with genome-scale metabolic models of the production host. Our method enables the simulation of the local nonlinear dynamics of pathway enzymes and metabolites, informed by the global metabolic state of the host as predicted by Flux Balance Analysis (FBA). To reduce computational costs, we make extensive use of surrogate machine learning models to replace FBA calculations, achieving simulation speed-ups of at least two orders of magnitude. Through case studies on two production pathways in Escherichia coli, we demonstrate the consistency of our simulations and the ability to predict metabolite dynamics under genetic perturbations and various carbon sources. We showcase the utility of our method for screening dynamic control circuits through large-scale parameter sampling and mixed-integer optimization. Our work links together genome-scale and kinetic models into a comprehensive framework for computational strain design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d2c6c72c5ef1dbac9514046b5174b0f4c3cf4d" target='_blank'>
              Modelling dynamic host-pathway interactions at the genome scale
              </a>
            </td>
          <td>
            C. Merzbacher, Oisin Mac Aodha, D. Oyarzún
          </td>
          <td>2024-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Investigation of important molecular features of glioma patients may reveal driving mechanisms of patient recurrence and treatment failures in glioma. Applying various deep learning techniques enables the integration and systematic comparison of multi-omics features, allowing robust marker prediction and patient stratification. We extracted mutation and copy number variation data from the Glioma Longitudinal Analysis (GLASS) Consortium, consisting of 628 primary or recurrent samples. To achieve the patient recurrence classification task, we designed a transformer-based model that learns the genomic profiles and predicts patient recurrence status. Our model outperforms traditional machine learning models, including support vector machine (SVM), logistic regression, and decision trees, for up to 34% and 40% improvement of AUROC and AUPRC, respectively. Including mutational features alone can achieve the best prediction performance, while copy number variations are less informative in predicting patient recurrence status. Our approach is an early attempt that employs deep learning for dissecting the contribution of diverse genomic profiles on glioma recurrence and might provide insights into developing novel therapies for treating glioma.
 Citation Format: Ko-Hong Lin, Kai Zhang, Dung-Fang Lee, Xiaoqian Jiang. Deep learning framework for patient recurrence and genetic marker prediction in diffuse glioma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2024; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2024 Apr 5-10; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2024;84(7_Suppl):Abstract nr LB246.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b61dafce86e85913504ff36d62a0700a1db4a2c4" target='_blank'>
              Abstract LB246: Deep learning framework for patient recurrence and genetic marker prediction in diffuse glioma
              </a>
            </td>
          <td>
            Ko-Hong Lin, Kai Zhang, Dung-Fang Lee, Xiaoqian Jiang
          </td>
          <td>2024-04-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Nanopore sequencing offers the ability for real-time analysis of long DNA sequences at a low cost, enabling new applications such as early detection of cancer. Due to the complex nature of nanopore measurements and the high cost of obtaining ground truth datasets, there is a need for nanopore simulators. Existing simulators rely on handcrafted rules and parameters and do not learn an internal representation that would allow for analysing underlying biological factors of interest. Instead, we propose VADA, a purely data-driven method for simulating nanopores based on an autoregressive latent variable model. We embed subsequences of DNA and introduce a conditional prior to address the challenge of a collapsing conditioning. We introduce an auxiliary regressor on the latent variable to encourage our model to learn an informative latent representation. We empirically demonstrate that our model achieves competitive simulation performance on experimental nanopore data. Moreover, we show we have learned an informative latent representation that is predictive of the DNA labels. We hypothesize that other biological factors of interest, beyond the DNA labels, can potentially be extracted from such a learned latent representation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7530ff6c1fe0f256833ec8dda30086a2061051" target='_blank'>
              VADA: a Data-Driven Simulator for Nanopore Sequencing
              </a>
            </td>
          <td>
            Jonas Niederle, Simon Koop, Marc Pagès-Gallego, V. Menkovski
          </td>
          <td>2024-04-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The development of models to predict sensitivity to anticancer drugs is an area of significant interest, given the diverse responses to treatment among patients and the considerable expense and time involved in anticancer drug development. Leveraging “omic” data and anticancer response information from the Cancer Cell Line Encyclopedia, we propose a novel approach utilizing multitask learning to enhance prediction accuracy and inference. We extended a multitask learning framework called the Data Shared Lasso to develop the Data Shared Elastic Net. This enabled the construction of tissue-specific models with information sharing while maintaining the attractive properties of Elastic Net regression. By employing this approach, we observed improvements in prediction accuracy compared to single-task Elastic Net models, particularly for cell lines displaying high sensitivity to treatment. Furthermore, the Data Shared Elastic Net facilitated the identification of predictors for anticancer drug sensitivity within specific tissue types, shedding light on cellular pathways targeted by these drugs across tissues. We also investigated the impact of data leakage on modeling outcomes from previous studies, which led to underestimating prediction error and erroneous inferences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83664bfa905ff011a2bcad47d103dd20fcdcd259" target='_blank'>
              Improving Drug Sensitivity Prediction and Inference by Multi-Task Learning
              </a>
            </td>
          <td>
            Jared Strauch, Amir Asiaee
          </td>
          <td>2024-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the trend of hybrid Artificial Intelligence (AI) techniques, Physic Informed Machine Learning has seen a growing interest. It operates mainly by imposing a data, learning or inductive bias with simulation data, Partial Differential Equations or equivariance and invariance properties. While these models have shown great success on tasks involving one physical domain such as fluid dynamics, existing methods still struggle on tasks with complex multi-physical and multi-domain phenomena. To address this challenge, we propose to leverage Bond Graphs, a multi-physics modeling approach together with Graph Neural Network. We thus propose Neural Bond Graph Encoder (NBgE), a model agnostic physical-informed encoder tailored for multi-physics systems. It provides an unified framework for any multi-physics informed AI with a graph encoder readable for any deep learning model. Our experiments on two challenging multi-domain physical systems - a Direct Current Motor and the Respiratory system - demonstrate the effectiveness of our approach on a multi-variate time series forecasting task.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5d7393bdb45ad9ec2527ea020c7d99f6d06d06" target='_blank'>
              Bond Graphs for multi-physics informed Neural Networks for multi-variate time series
              </a>
            </td>
          <td>
            Alexis-Raja Brachet, Pierre-Yves Richard, C'eline Hudelot
          </td>
          <td>2024-05-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Survival prediction is an important branch of cancer prognosis analysis. The model that predicts survival risk through TCGA genomics data can discover genes related to cancer and provide diagnosis and treatment recommendations based on patient characteristics. We found that deep learning models based on Cox proportional hazards often suffer from overfitting when dealing with high-throughput data. Moreover, we found that as the number of network layers increases, the experimental results will not get better, and network degradation will occur. Based on this problem, we propose a new framework based on Deep Residual Learning. Combine the ideas of Cox proportional hazards and Residual. And name it ResSurv. First, ResSurv is a feed-forward deep learning network stacked by multiple basic ResNet Blocks. In each ResNet Block, we add a Normalization Layer to prevent gradient disappearance and gradient explosion. Secondly, for the loss function of the neural network, we inherited the Cox proportional hazards methods, applied the semi-parametric of the CPH model to the neural network, combined with the partial likelihood model, established the loss function, and performed backpropagation and gradient update. Finally, we compared ResSurv networks of different depths and found that we can effectively extract high-dimensional features. Ablation experiments and comparative experiments prove that our model has reached SOTA(state of the art) in the field of deep learning, and our network can effectively extract deep information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2cbfe31657721f4bf6c2ca44aa455fb795d7549" target='_blank'>
              ResSurv: Cancer Survival Analysis Prediction Model Based on Residual Networks
              </a>
            </td>
          <td>
            Wankang Zhai
          </td>
          <td>2024-05-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This study aims to provide an overview of the current state-of-the-art applications of artificial intelligence (AI) and machine learning in the management of hepatocellular carcinoma (HCC), and to explore future directions for continued progress in this emerging field.  This study is a comprehensive literature review that synthesizes recent findings and advancements in the application of AI and machine learning techniques across various aspects of HCC care, including screening and early detection, diagnosis and staging, prognostic modeling, treatment planning, interventional guidance, and monitoring of treatment response. The review draws upon a wide range of published research studies, focusing on the integration of AI and machine learning with diverse data sources, such as medical imaging, clinical data, genomics, and other multimodal information.  The results demonstrate that AI-based systems have shown promise in improving the accuracy and efficiency of HCC screening, diagnosis, and tumor characterization compared to traditional methods. Machine learning models integrating clinical, imaging, and genomic data have outperformed conventional staging systems in predicting survival and recurrence risk. AI-based recommendation systems have the potential to optimize personalized therapy selection, while augmented reality techniques can guide interventional procedures in real-time. Moreover, longitudinal application of AI may enhance the assessment of treatment response and recurrence monitoring. Despite these promising findings, the review highlights the need for rigorous multicenter prospective validation studies, standardized multimodal datasets, and thoughtful consideration of ethical implications before widespread clinical implementation of AI technologies in HCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4171d77b0e3f46735aae116d624f4eba8b5bd5a5" target='_blank'>
              Realizing the Promise of Artificial Intelligence in Hepatocellular Carcinoma through Opportunities and Recommendations for Responsible Translation
              </a>
            </td>
          <td>
            T. Addissouky, Ibrahim El Tantawy El Sayed, Majeed M. A. Ali
          </td>
          <td>2024-04-23</td>
          <td>Jurnal Online Informatika</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Foundation models, such as large language models, have demonstrated success in addressing various language and image processing tasks. In this work, we introduce a multi-modal foundation model for scientific problems, named PROSE-PDE. Our model, designed for bi-modality to bi-modality learning, is a multi-operator learning approach which can predict future states of spatiotemporal systems while concurrently learning the underlying governing equations of the physical system. Specifically, we focus on multi-operator learning by training distinct one-dimensional time-dependent nonlinear constant coefficient partial differential equations, with potential applications to many physical applications including physics, geology, and biology. More importantly, we provide three extrapolation studies to demonstrate that PROSE-PDE can generalize physical features through the robust training of multiple operators and that the proposed model can extrapolate to predict PDE solutions whose models or data were unseen during the training. Furthermore, we show through systematic numerical experiments that the utilization of the symbolic modality in our model effectively resolves the well-posedness problems with training multiple operators and thus enhances our model's predictive capabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/819bc0d5e0ea2efadac1364064e40b76cf3a3a11" target='_blank'>
              Towards a Foundation Model for Partial Differential Equations: Multi-Operator Learning and Extrapolation
              </a>
            </td>
          <td>
            Jingmin Sun, Yuxuan Liu, Zecheng Zhang, Hayden Schaeffer
          </td>
          <td>2024-04-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e77a9f19eaa4ee2b782f8540e6f17c886c43437" target='_blank'>
              Fundamentals for predicting transcriptional regulations from DNA sequence patterns.
              </a>
            </td>
          <td>
            Masaru Koido, K. Tomizuka, Chikashi Terao
          </td>
          <td>2024-05-10</td>
          <td>Journal of human genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Identifying latent interactions within complex systems is key to unlocking deeper insights into their operational dynamics, including how their elements affect each other and contribute to the overall system behavior. For instance, in neuroscience, discovering neuron-to-neuron interactions is essential for understanding brain function; in ecology, recognizing the interactions among populations is key for understanding complex ecosystems. Such systems, often modeled as dynamical systems, typically exhibit noisy high-dimensional and non-stationary temporal behavior that renders their identification challenging. Existing dynamical system identification methods often yield operators that accurately capture short-term behavior but fail to predict long-term trends, suggesting an incomplete capture of the underlying process. Methods that consider extended forecasts (e.g., recurrent neural networks) lack explicit representations of element-wise interactions and require substantial training data, thereby failing to capture interpretable network operators. Here we introduce Lookahead-driven Inference of Networked Operators for Continuous Stability (LINOCS), a robust learning procedure for identifying hidden dynamical interactions in noisy time-series data. LINOCS integrates several multi-step predictions with adaptive weights during training to recover dynamical operators that can yield accurate long-term predictions. We demonstrate LINOCS' ability to recover the ground truth dynamical operators underlying synthetic time-series data for multiple dynamical systems models (including linear, piece-wise linear, time-changing linear systems' decomposition, and regularized linear time-varying systems) as well as its capability to produce meaningful operators with robust reconstructions through various real-world examples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f005c8b19ca1d9171afc191fc78b81f136ddfb8c" target='_blank'>
              LINOCS: Lookahead Inference of Networked Operators for Continuous Stability
              </a>
            </td>
          <td>
            Noga Mudrik, Eva Yezerets, Yenho Chen, Christopher Rozell, Adam Charles
          </td>
          <td>2024-04-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d63afa9ac9d835d94e1d88390467934811db53a5" target='_blank'>
              Task-oriented machine learning surrogates for tipping points of agent-based models
              </a>
            </td>
          <td>
            Gianluca Fabiani, N. Evangelou, Tianqi Cui, J. M. Bello-Rivas, Cristina P. Martin-Linares, Constantinos Siettos, I. Kevrekidis
          </td>
          <td>2024-05-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96bc93fec51b25d3ba75bcc36627b1513c999f0" target='_blank'>
              q-Diffusion leverages the full dimensionality of gene coexpression in single-cell transcriptomics.
              </a>
            </td>
          <td>
            Myrl G. Marmarelis, R. Littman, F. Battaglin, Donna Niedzwiecki, A. Venook, J. Ambite, A. Galstyan, H. Lenz, Greg Ver Steeg
          </td>
          <td>2024-04-02</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="BACKGROUND
In recent years, analyzing complex biological networks to predict future links in such networks has attracted the attention of many medical and computer science researchers. The discovery of new drugs is one of the application cases for predicting future connections in biological networks. The operation of drug-target interactions prediction (DTIP) can be considered a fundamental step in identifying potential interactions between drug and target to identify new drugs.


OBJECTIVE
The previous studies reveal that predictions are made based on known interactions using computational methods to solve the cost problem and avoid blind study of all interactions. But, there seem to be challenges such as the lack of confirmed negative samples and the low accuracy in some computational methods. Thus, we have proposed an efficient and hybrid approach called MKPUL-BLM to manage some of the aforementioned challenges for predicting drug-target interactions.


METHODS
The MKPUL-BLM combins multi-kernel and positive unlabeled learning (PUL) approaches. Our method uses more information to increase accuracy, in addition to minimizing small similarities using network information. Also, potential negative samples are produced using a PUL approach because of lacking negative laboratory samples. Finally, labels are expanded via a semi-supervised.


RESULTS
Our method improved to 0.98 and 0.94 in the old interactions set for the ROCAUC and AUPR criteria, respectively. Also, this method enhanced ROCAUC and AUPR criteria by 0.89 and 0.77 for the new interactions set.


CONCLUSION
The MKPUL-BLM can be considered an efficient alternative to achieve more reliable predictions in the field of DTIP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/517b4e428c381711c80b78a700350d74395f3bac" target='_blank'>
              An Enhanced Computational Approach Using Multi-kernel Positive Unlabeled Learning for Predicting Drug-target Interactions.
              </a>
            </td>
          <td>
            M. Keyvanpour, Soheila Mehrmolaei, Faraneh Haddadi
          </td>
          <td>2024-04-29</td>
          <td>Current computer-aided drug design</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Physiological and pathological processes are governed by a network of genes called gene regulatory networks (GRNs). By reconstructing GRNs, we can accurately model how cells behave in their natural state and predict how genetic changes will affect them. Transcriptomic data of single cells are now available for a wide range of cellular processes in multiple species. Thus, a method building predictive GRNs from single-cell RNA sequencing (scRNA-seq) data, without any additional prior knowledge, could have a great impact on our understanding of biological processes and the genes playing a key role in them. To this aim, we developed IGNITE (Inference of Gene Networks using Inverse kinetic Theory and Experiments), an unsupervised machine learning framework designed to infer directed, weighted, and signed GRNs directly from unperturbed single-cell RNA sequencing data. IGNITE uses the GRNs to generate gene expression data upon single and multiple genetic perturbations. IGNITE is based on the inverse problem for a kinetic Ising model, a model from statistical physics that has been successfully applied to biological networks. We tested IGNITE on murine pluripotent stem cells (PSCs) transitioning from the naïve to formative states. Using as input only scRNA-seq data of unperturbed PSCs, IGNITE simulated single and triple gene knockouts. Comparison with experimental data revealed high accuracy, up to 74%, outperforming currently available methods. In sum, IGNITE identifies predictive GRNs from scRNA-seq data without additional prior knowledge and faithfully simulates single and multiple gene perturbations. Applications of IGNITE range from studying cell differentiation to identifying genes specifically active under pathological conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f91d1ee7feee1c23aecfd8641c38451501491d9" target='_blank'>
              Unveiling Gene Perturbation Effects through Gene Regulatory Networks Inference from single-cell transcriptomic data
              </a>
            </td>
          <td>
            Clelia Corridori, Merrit Romeike, Giorgio Nicoletti, C. Buecker, S. Suweis, S. Azaele, Graziano Martello
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36fd625816fc82a989aba4767b781ec1f63c1523" target='_blank'>
              Strategies for improving the performance of prediction models for response to immune checkpoint blockade therapy in cancer
              </a>
            </td>
          <td>
            Tiantian Zeng, Jason Z. Zhang, Arnold Stromberg, Jin Chen, Chi Wang
          </td>
          <td>2024-04-09</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f319f5bedd470fa7ef18495bb61b8222606b89b6" target='_blank'>
              al-BERT: a semi-supervised denoising technique for disease prediction
              </a>
            </td>
          <td>
            Yun-Chien Tseng, Chuan-Wei Kuo, Wen-Chih Peng, Chih-Chieh Hung
          </td>
          <td>2024-05-16</td>
          <td>BMC Medical Informatics and Decision Making</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="In the era of big data, the sheer volume and complexity of datasets pose significant challenges in machine learning, particularly in image processing tasks. This paper introduces an innovative Autoencoder-based Dataset Condensation Model backed by Koopman operator theory that effectively packs large datasets into compact, information-rich representations. Inspired by the predictive coding mechanisms of the human brain, our model leverages a novel approach to encode and reconstruct data, maintaining essential features and label distributions. The condensation process utilizes an autoencoder neural network architecture, coupled with Optimal Transport theory and Wasserstein distance, to minimize the distributional discrepancies between the original and synthesized datasets. We present a two-stage implementation strategy: first, condensing the large dataset into a smaller synthesized subset; second, evaluating the synthesized data by training a classifier and comparing its performance with a classifier trained on an equivalent subset of the original data. Our experimental results demonstrate that the classifiers trained on condensed data exhibit comparable performance to those trained on the original datasets, thus affirming the efficacy of our condensation model. This work not only contributes to the reduction of computational resources but also paves the way for efficient data handling in constrained environments, marking a significant step forward in data-efficient machine learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04c1b90a5072e805987d8dd78ca85c2ab8e7018a" target='_blank'>
              Koopcon: A new approach towards smarter and less complex learning
              </a>
            </td>
          <td>
            Vahid Jebraeeli, , D. Cansever, Hamid Krim
          </td>
          <td>2024-05-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/985fb213001f71393d7581e0584e1c2910789136" target='_blank'>
              CVGAE: A Self-Supervised Generative Method for Gene Regulatory Network Inference Using Single-Cell RNA Sequencing Data.
              </a>
            </td>
          <td>
            Wei Liu, Zhijie Teng, Zejun Li, Jing Chen
          </td>
          <td>2024-05-23</td>
          <td>Interdisciplinary sciences, computational life sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The complexity, scale, and uncertainty in regulatory networks (e.g., gene regulatory networks and microbial networks) regularly pose a huge uncertainty in their models. These uncertainties often cannot be entirely reduced using limited and costly data acquired from the normal condition of systems. Meanwhile, regulatory networks often suffer from the non-identifiability issue, which refers to scenarios where the true underlying network model cannot be clearly distinguished from other possible models. Perturbation or excitation is a well-known process in systems biology for acquiring targeted data to reveal the complex underlying mechanisms of regulatory networks and overcome the non-identifiability issue. We consider a general class of Boolean network models for capturing the activation and inactivation of components and their complex interactions. Assuming partial available knowledge about the interactions between components of the networks, this paper formulates the inference process through the maximum aposteriori (MAP) criterion. We develop a Bayesian lookahead policy that systematically perturbs regulatory networks to maximize the performance of MAP inference under the perturbed data. This is achieved by optimally formulating the perturbation process in a reinforcement learning context and deriving a scalable deep reinforcement learning perturbation policy to compute near-optimal Bayesian policy. The proposed method learns the perturbation policy through planning without the need for any real data. The high performance of the proposed approach is demonstrated by comprehensive numerical experiments using the well-known mammalian cell cycle and gut microbial community networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13661311b83454b1f32c13b0d86a4299e417da66" target='_blank'>
              Bayesian Lookahead Perturbation Policy for Inference of Regulatory Networks.
              </a>
            </td>
          <td>
            Mohammad Alali, Mahdi Imani
          </td>
          <td>2024-05-17</td>
          <td>IEEE/ACM transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite the rapid and significant advancements in deep learning for Quantitative Structure-Activity Relationship (QSAR) models, the challenge of learning robust molecular representations that effectively generalize in real-world scenarios to novel compounds remains an elusive and unresolved task. This study examines how atom-level pretraining with quantum mechanics (QM) data can mitigate violations of assumptions regarding the distributional similarity between training and test data and therefore improve performance and generalization in downstream tasks. In the public dataset Therapeutics Data Commons (TDC), we show how pretraining on atom-level QM improves performance overall and makes the activation of the features distributes more Gaussian-like which results in a representation that is more robust to distribution shifts. To the best of our knowledge, this is the first time that hidden state molecular representations are analyzed to compare the effects of molecule-level and atom-level pretraining on QM data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e4a4119ef927e805dfce5f24ed3a700bfc54e00" target='_blank'>
              Analysis of Atom-level pretraining with QM data for Graph Neural Networks Molecular property models
              </a>
            </td>
          <td>
            Jose A. Arjona-Medina, Ramil I. Nugmanov
          </td>
          <td>2024-05-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The existence of large volumes of data has considerably alleviated concerns regarding the availability of sufficient data instances for machine learning experiments. Nevertheless, in certain contexts, addressing limited data availability may demand distinct strategies and efforts. Analyzing COVID-19 predictions at pandemic beginning emerged a question: how much data is needed to make reliable predictions? When does the volume of data provide a better understanding of the disease’s evolution and, in turn, offer reliable forecasts? Given these questions, the objective of this study is to analyze learning curves obtained from predicting the incidence of COVID-19 in Brazilian States using ARIMA models with limited available data. To fulfill the objective, a retrospective exploration of COVID-19 incidence across the Brazilian States was performed. After the data acquisition and modeling, the model errors were assessed by employing a learning curve analysis. The asymptotic exponential curve fitting enabled the evaluation of the errors in different points, reflecting the increased available data over time. For a comprehensive understanding of the results at distinct stages of the time evolution, the average derivative of the curves and the equilibrium points were calculated, aimed to identify the convergence of the ARIMA models to a stable pattern. We observed differences in average derivatives and equilibrium values among the multiple samples. While both metrics ultimately confirmed the convergence to stability, the equilibrium points were more sensitive to changes in the models’ accuracy and provided a better indication of the learning progress. The proposed method for constructing learning curves enabled consistent monitoring of prediction results, providing evidence-based understandings required for informed decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa718938177cf3d00e832bde3c94d3099493410" target='_blank'>
              Analysis of learning curves in predictive modeling using exponential curve fitting with an asymptotic approach
              </a>
            </td>
          <td>
            L. Vianna, Alexandre Leopoldo Gonçalves, João Artur Souza
          </td>
          <td>2024-04-18</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The open-world test dataset is often mixed with out-of-distribution (OOD) samples, where the deployed models will struggle to make accurate predictions. Traditional detection methods need to trade off OOD detection and in-distribution (ID) classification performance since they share the same representation learning model. In this work, we propose to detect OOD molecules by adopting an auxiliary diffusion model-based framework, which compares similarities between input molecules and reconstructed graphs. Due to the generative bias towards reconstructing ID training samples, the similarity scores of OOD molecules will be much lower to facilitate detection. Although it is conceptually simple, extending this vanilla framework to practical detection applications is still limited by two significant challenges. First, the popular similarity metrics based on Euclidian distance fail to consider the complex graph structure. Second, the generative model involving iterative denoising steps is time-consuming especially when it runs on the enormous pool of drugs. To address these challenges, our research pioneers an approach of Prototypical Graph Reconstruction for Molecular OOD Detection, dubbed as PGR-MOOD and hinges on three innovations: i) An effective metric to comprehensively quantify the matching degree of input and reconstructed molecules; ii) A creative graph generator to construct prototypical graphs that are in line with ID but away from OOD; iii) An efficient and scalable OOD detector to compare the similarity between test samples and pre-constructed prototypical graphs and omit the generative process on every new molecule. Extensive experiments on ten benchmark datasets and six baselines are conducted to demonstrate our superiority.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/954919a99b5a5a3a39861eae6c9d4b3e7325895a" target='_blank'>
              Optimizing OOD Detection in Molecular Graphs: A Novel Approach with Diffusion Models
              </a>
            </td>
          <td>
            Xu Shen, Yili Wang, Kaixiong Zhou, Shirui Pan, Xin Wang
          </td>
          <td>2024-04-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21038fc9ac697b06d9c2823da9c7d54fcaab783e" target='_blank'>
              Elucidating the semantics-topology trade-off for knowledge inference-based pharmacological discovery
              </a>
            </td>
          <td>
            Daniel N. Sosa, Georgiana Neculae, Julien Fauqueur, Russ B. Altman
          </td>
          <td>2024-05-01</td>
          <td>Journal of Biomedical Semantics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/620803fdb5f14b75e2d7602b6c40e8374eb91f5a" target='_blank'>
              Enhancing drug and cell line representations via contrastive learning for improved anti-cancer drug prioritization
              </a>
            </td>
          <td>
            Patrick J. Lawrence, Benjamin Burns, Xia Ning
          </td>
          <td>2024-05-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Since infectious pathogens start spreading into a susceptible population, mathematical models can provide policy makers with reliable forecasts and scenario analyses, which can be concretely implemented or solely consulted. In these complex epidemiological scenarios, machine learning architectures can play an important role, since they directly reconstruct data-driven models circumventing the specific modelling choices and the parameter calibration, typical of classical compartmental models. In this work, we discuss the efficacy of Kernel Operator Learning (KOL) to reconstruct population dynamics during epidemic outbreaks, where the transmission rate is ruled by an input strategy. In particular, we introduce two surrogate models, named KOL-m and KOL-$\partial$, which reconstruct in two different ways the evolution of the epidemics. Moreover, we evaluate the generalization performances of the two approaches with different kernels, including the Neural Tangent Kernels, and compare them with a classical neural network model learning method. Employing synthetic but semi-realistic data, we show how the two introduced approaches are suitable for realizing fast and robust forecasts and scenario analyses, and how these approaches are competitive for determining optimal intervention strategies with respect to specific performance measures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202709203f4dc2978114ea869baab2fd2e010a52" target='_blank'>
              Learning epidemic trajectories through Kernel Operator Learning: from modelling to optimal control
              </a>
            </td>
          <td>
            Giovanni Ziarelli, N. Parolini, M. Verani
          </td>
          <td>2024-04-17</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Manifold-learning is particularly useful to resolve the complex cellular state space from single-cell RNA sequences. While current manifold-learning methods provide insights into cell fate by inferring graph-based trajectory at cell level, challenges remain to retrieve interpretable biology underlying the diverse cellular states. Here, we described MGPfactXMBD, a model-based manifold-learning framework and capable to factorize complex development trajectories into independent bifurcation processes of gene sets, and thus enables trajectory inference based on relevant features. MGPfactXMBD offers more nuanced understanding of the biological processes underlying cellular trajectories with potential determinants. When bench-tested across 239 datasets, MGPfactXMBD showed advantages in major quantity-control metrics, such as branch division accuracy and trajectory topology, outperforming most established methods. In real datasets, MGPfactXMBD recovered the critical pathways and cell types in microglia development with experimentally valid regulons and markers. Furthermore, MGPfactXMBD discovered evolutionary trajectories of tumor-associated CD8+ T cells and yielded new subtypes of CD8+ T cells with gene expression signatures significantly predictive of the responses to immune checkpoint inhibitor in independent cohorts. In summary, MGPfactXMBD offers a manifold-learning framework in scRNA-seq data which enables feature selection for specific biological processes and contributing to advance our understanding of biological determination of cell fate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e3336a7a0a5aa174395d76ba4b2d0028d7d4bb4" target='_blank'>
              MGPfactXMBD: A Model-Based Factorization Method for scRNA Data Unveils Bifurcating Transcriptional Modules Underlying Cell Fate Determination
              </a>
            </td>
          <td>
            Jun Ren, Ying Zhou, Yudi Hu, Jing Yang, Hongkun Fang, Xuejing Lyu, Jintao Guo, Xiaodong Shi, Qiyuan Li
          </td>
          <td>2024-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Molecular Property Prediction (MPP) is vital for drug discovery, crop protection, and environmental science. Over the last decades, diverse computational techniques have been developed, from using simple physical and chemical properties and molecular fingerprints in statistical models and classical machine learning to advanced deep learning approaches. In this review, we aim to distill insights from current research on employing transformer models for MPP. We analyze the currently available models and explore key questions that arise when training and fine-tuning a transformer model for MPP. These questions encompass the choice and scale of the pre-training data, optimal architecture selections, and promising pre-training objectives. Our analysis highlights areas not yet covered in current research, inviting further exploration to enhance the field's understanding. Additionally, we address the challenges in comparing different models, emphasizing the need for standardized data splitting and robust statistical analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95085c326a7773552ac82629ffc405be371422b" target='_blank'>
              Transformers for molecular property prediction: Lessons learned from the past five years
              </a>
            </td>
          <td>
            Afnan Sultan, Jochen Sieg, M. Mathea, Andrea Volkamer
          </td>
          <td>2024-04-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="This paper explores the efficacy of diffusion-based generative models as neural operators for partial differential equations (PDEs). Neural operators are neural networks that learn a mapping from the parameter space to the solution space of PDEs from data, and they can also solve the inverse problem of estimating the parameter from the solution. Diffusion models excel in many domains, but their potential as neural operators has not been thoroughly explored. In this work, we show that diffusion-based generative models exhibit many properties favourable for neural operators, and they can effectively generate the solution of a PDE conditionally on the parameter or recover the unobserved parts of the system. We propose to train a single model adaptable to multiple tasks, by alternating between the tasks during training. In our experiments with multiple realistic dynamical systems, diffusion models outperform other neural operators. Furthermore, we demonstrate how the probabilistic diffusion model can elegantly deal with systems which are only partially identifiable, by producing samples corresponding to the different possible solutions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f7dd73bdb8cf078d6f19120987ab3c21100bc5" target='_blank'>
              Diffusion models as probabilistic neural operators for recovering unobserved states of dynamical systems
              </a>
            </td>
          <td>
            Katsiaryna Haitsiukevich, O. Poyraz, Pekka Marttinen, Alexander Ilin
          </td>
          <td>2024-05-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="A predictive model refers to a mathematical or computational model that is designed to make predictions or forecasts based on input data. These models are used in various fields such as finance, healthcare, marketing, and many others to analyze historical data and make predictions about future events or trends. This study focuses on developing a predictive model for well-differentiated thyroid cancer recurrence using machine learning algorithms. Leveraging a dataset with clinicopathologic features spanning a 15-year period, the study aims to contribute to the advancement of predictive modeling in thyroid cancer prognosis. The Various algorithms, including logistic regression, decision tree, random forest, support vector machine, k-nearest neighbors, and XGBoost, are evaluated for their performance in binary classification. The highest-performing model is then utilized to predict thyroid cancer recurrence in new data. Integrating this with frontend tools has resulted in an intuitive interface that streamlines the process of uploading datasets, executing the predictive model, and presenting the results. This integration has made the prediction process more accessible and user-friendly, ultimately benefiting clinicians and researchers in the field of thyroid cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/337a2b4bf2039263d5b293691654c2fd6748f0a0" target='_blank'>
              Predicting Recurrence in Differentiated Thyroid Cancer: A Machine Learning Approach
              </a>
            </td>
          <td>
            Bharath K, A. Sai Sabitha
          </td>
          <td>2024-04-18</td>
          <td>2024 International Conference on Advances in Data Engineering and Intelligent Computing Systems (ADICS)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pre-trained deep learning (DL) models are increasingly accessible in public repositories, i.e., model zoos. Given a new prediction task, finding the best model to fine-tune can be computationally intensive and costly, especially when the number of pre-trained models is large. Selecting the right pre-trained models is crucial, yet complicated by the diversity of models from various model families (like ResNet, Vit, Swin) and the hidden relationships between models and datasets. Existing methods, which utilize basic information from models and datasets to compute scores indicating model performance on target datasets, overlook the intrinsic relationships, limiting their effectiveness in model selection. In this study, we introduce TransferGraph, a novel framework that reformulates model selection as a graph learning problem. TransferGraph constructs a graph using extensive metadata extracted from models and datasets, while capturing their inherent relationships. Through comprehensive experiments across 16 real datasets, both images and texts, we demonstrate TransferGraph's effectiveness in capturing essential model-dataset relationships, yielding up to a 32% improvement in correlation between predicted performance and the actual fine-tuning results compared to the state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/544b82074bd539bbb55fc4123850eb991db9dc33" target='_blank'>
              Model Selection with Model Zoo via Graph Learning
              </a>
            </td>
          <td>
            Ziyu Li, Hilco van der Wilk, Danning Zhan, Megha Khosla, A. Bozzon, Rihan Hai
          </td>
          <td>2024-04-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/857fee206820f4063f7c9f0278f5accd39ea0ab1" target='_blank'>
              Pseudo datasets explain artificial neural networks
              </a>
            </td>
          <td>
            Yi-Chi Chu, Yi-Hau Chen, Chao-Yu Guo
          </td>
          <td>2024-04-10</td>
          <td>International Journal of Data Science and Analytics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Drug discovery is a complex and costly process, often requiring extensive testing and significant financial investment. However, the acquisition of sufficient data for effective machine learning models presents a significant challenge. In this study, it is to investigate the application of few-shot learning—a machine learning approach that can function with low data—to address this challenge. Specifically, to investigate various metric-based meta-learning techniques within the context of drug discovery. The proposed research analysis includes Prototypical, Matching, Siamese, and Relation Networks, two promising few-shot learning methods. Through rigorous experimentation and comparative analysis, to evaluate the efficacy of these approaches across diverse datasets, drawing insights into their suitability for drug discovery tasks with limited data. The resultant findings shed light on the potential of few-shot learning techniques to accelerate drug discovery processes and highlight the importance of selecting appropriate methodologies based on the nature of the available data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a831a5424d14e8ba489d2f13bd45b90be41fd96" target='_blank'>
              A Comparative Study of Metric-based Meta-Learning Methods for Improving Few-Shot Learning in Drug Discovery with Limited Data
              </a>
            </td>
          <td>
            Gilda George, S. Juliet
          </td>
          <td>2024-04-17</td>
          <td>2024 International Conference on Cognitive Robotics and Intelligent Systems (ICC - ROBINS)</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Gastric cancer (GC) is a prevalent malignancy worldwide, ranking as the fifth most common cancer with over 1 million new cases and 700 thousand deaths in 2020. Locally advanced gastric cancer (LAGC) accounts for approximately two-thirds of GC diagnoses, and neoadjuvant chemotherapy (NACT) has emerged as the standard treatment for LAGC. However, the effectiveness of NACT varies significantly among patients, with a considerable subset displaying treatment resistance. Ineffective NACT not only leads to adverse effects but also misses the optimal therapeutic window, resulting in lower survival rate. However, existing multimodal learning methods assume the availability of all modalities for each patient, which does not align with the reality of clinical practice. The limited availability of modalities for each patient would cause information loss, adversely affecting predictive accuracy. In this study, we propose an incomplete multimodal data integration framework for GC (iMD4GC) to address the challenges posed by incomplete multimodal data, enabling precise response prediction and survival analysis. Specifically, iMD4GC incorporates unimodal attention layers for each modality to capture intra-modal information. Subsequently, the cross-modal interaction layers explore potential inter-modal interactions and capture complementary information across modalities, thereby enabling information compensation for missing modalities. To evaluate iMD4GC, we collected three multimodal datasets for GC study: GastricRes (698 cases) for response prediction, GastricSur (801 cases) for survival analysis, and TCGA-STAD (400 cases) for survival analysis. The scale of our datasets is significantly larger than previous studies. The iMD4GC achieved impressive performance with an 80.2% AUC on GastricRes, 71.4% C-index on GastricSur, and 66.1% C-index on TCGA-STAD, significantly surpassing other compared methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688425385f6f52ec92780394b8a2ddc85ad21191" target='_blank'>
              iMD4GC: Incomplete Multimodal Data Integration to Advance Precise Treatment Response Prediction and Survival Analysis for Gastric Cancer
              </a>
            </td>
          <td>
            Fengtao Zhou, Ying Xu, Yanfen Cui, Shenyang Zhang, Yun Zhu, Weiyang He, Jiguang Wang, Xin Wang, Ronald Chan, Louis Ho Shing Lau, Chu Han, Dafu Zhang, Zhenhui Li, Hao Chen
          </td>
          <td>2024-04-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Computational chemistry is an indispensable tool for understanding molecules and predicting chemical properties. However, traditional computational methods face significant challenges due to the difficulty of solving the Schrödinger equations and the increasing computational cost with the size of the molecular system. In response, there has been a surge of interest in leveraging artificial intelligence (AI) and machine learning (ML) techniques to in silico experiments. Integrating AI and ML into computational chemistry increases the scalability and speed of the exploration of chemical space. However, challenges remain, particularly regarding the reproducibility and transferability of ML models. This review highlights the evolution of ML in learning from, complementing, or replacing traditional computational chemistry for energy and property predictions. Starting from models trained entirely on numerical data, a journey set forth toward the ideal model incorporating or learning the physical laws of quantum mechanics. This paper also reviews existing computational methods and ML models and their intertwining, outlines a roadmap for future research, and identifies areas for improvement and innovation. Ultimately, the goal is to develop AI architectures capable of predicting accurate and transferable solutions to the Schrödinger equation, thereby revolutionizing in silico experiments within chemistry and materials science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c965a92a4418b12137dd3227bb9be135171b111" target='_blank'>
              In Silico Chemical Experiments in the Age of AI: From Quantum Chemistry to Machine Learning and Back.
              </a>
            </td>
          <td>
            Abdulrahman Aldossary, Jorge Arturo Campos-Gonzalez-Angulo, Sergio Pablo-García, Shi Xuan Leong, Ella Miray Rajaonson, Luca Thiede, Gary Tom, Andrew Wang, Davide Avagliano, Alán Aspuru-Guzik
          </td>
          <td>2024-05-25</td>
          <td>Advanced materials</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Skin cancer mortality rates continue to rise, and survival analysis is increasingly needed to understand who is at risk and what interventions improve outcomes. However, current statistical methods are limited by inability to synthesize multiple data types, such as patient genetics, clinical history, demographics, and pathology and reveal significant multimodal relationships through predictive algorithms. Advances in computing power and data science enabled the rise of artificial intelligence (AI), which synthesizes vast amounts of data and applies algorithms that enable personalized diagnostic approaches. Here, we analyze AI methods used in skin cancer survival analysis, focusing on supervised learning, unsupervised learning, deep learning, and natural language processing. We illustrate strengths and weaknesses of these approaches with examples. Our PubMed search yielded 14 publications meeting inclusion criteria for this scoping review. Most publications focused on melanoma, particularly histopathologic interpretation with deep learning. Such concentration on a single type of skin cancer amid increasing focus on deep learning highlight growing areas for innovation; however, it also demonstrates opportunity for additional analysis that addresses other types of cutaneous malignancies and expands the scope of prognostication to combine both genetic, histopathologic, and clinical data. Moreover, researchers may leverage multiple AI methods for enhanced benefit in analyses. Expanding AI to this arena may enable improved survival analysis, targeted treatments, and outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c03be441acb79fa9e1b80312e9f34b546ca3baa4" target='_blank'>
              Current status of artificial intelligence methods for skin cancer survival analysis: a scoping review
              </a>
            </td>
          <td>
            Celine M. Schreidah, Emily R. Gordon, Oluwaseyi Adeuyan, Caroline Chen, Brigit A. Lapolla, Joshua A. Kent, George Bingham Reynolds, L. Fahmy, Chunhua Weng, Nicholas P. Tatonetti, Herbert S. Chase, I. Pe’er, L. J. Geskin
          </td>
          <td>2024-04-22</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Objective: Enhancing patient response to immune checkpoint inhibitors (ICIs) is crucial in cancer immunotherapy. This study aims to create a data-driven mathematical model of the tumor immune microenvironment (TIME) and utilize deep reinforcement learning (DRL) for optimizing patient-specific ICI therapy combined with chemotherapy (ICC). Methods: Using patients' genomic and transcriptomic data, we develop an ordinary differential equations (ODEs)-based TIME model to characterize interactions among chemotherapy, ICIs, immune cells, and cancer cells. A DRL algorithm is trained to determine the personalized optimal ICC therapy. Results: Numerical experiments with real-world data demonstrates that the proposed TIME model can predict ICI therapy response. The DRL-derived personalized ICC therapy outperforms predefined fixed schedules. For tumors with extremely low CD8+T cell infiltration ("extremely cold tumors"), DRL recommends high-dosage chemotherapy alone. For tumors with higher CD8+T cell infiltration ("cold" and "hot tumors"), an appropriate chemotherapy dosage induces CD8+T cell proliferation, enhancing ICI therapy outcomes. Specifically, for "hot tumors," chemotherapy and ICI are administered simultaneously, while for "cold tumors," a mid-dosage of chemotherapy makes the TIME "hotter" before ICI administration. However, a number of "cold tumors" with rapid resistant cancer cell growth, ICC eventually fails. Conclusion: This study highlights the potential of utilizing real-world clinical data and DRL to develop personalized optimal ICC by understanding the complex biological dynamics of a patient's TIME. Our ODE-based TIME model offers a theoretical framework for determining the best use of ICI, and the proposed DRL model may guide personalized ICC schedules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42e1a8b6d37982dcb30f20f3e82fd6f9629e7709" target='_blank'>
              Optimized patient-specific immune checkpoint inhibitors therapy for cancer treatment based on tumor immune microenvironment modeling
              </a>
            </td>
          <td>
            Y. Yao, F. Y. Chen, Q. Zhang
          </td>
          <td>2024-04-10</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The discovery of linear embedding is the key to the synthesis of linear control techniques for nonlinear systems. In recent years, while Koopman operator theory has become a prominent approach for learning these linear embeddings through data-driven methods, these algorithms often exhibit limitations in generalizability beyond the distribution captured by training data and are not robust to changes in the nominal system dynamics induced by intrinsic or environmental factors. To overcome these limitations, this study presents an adaptive Koopman architecture capable of responding to the changes in system dynamics online. The proposed framework initially employs an autoencoder-based neural network that utilizes input-output information from the nominal system to learn the corresponding Koopman embedding offline. Subsequently, we augment this nominal Koopman architecture with a feed-forward neural network that learns to modify the nominal dynamics in response to any deviation between the predicted and observed lifted states, leading to improved generalization and robustness to a wide range of uncertainties and disturbances compared to contemporary methods. Extensive tracking control simulations, which are undertaken by integrating the proposed scheme within a Model Predictive Control framework, are used to highlight its robustness against measurement noise, disturbances, and parametric variations in system dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302db61f58f8a2e62340fcfaacbceec2620e551a" target='_blank'>
              Adaptive Koopman Embedding for Robust Control of Complex Nonlinear Dynamical Systems
              </a>
            </td>
          <td>
            Rajpal Singh, Chandan Kumar Sah, J. Keshavan
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Individual level models are a class of mechanistic models that are widely used to infer infectious disease transmission dynamics. These models incorporate individual level covariate information accounting for population heterogeneity and are generally fitted in a Bayesian Markov chain Monte Carlo (MCMC) framework. However, Bayesian MCMC methods of inference are computationally expensive for large data sets. This issue becomes more severe when applied to infectious disease data collected from spatially heterogeneous populations, as the number of covariates increases. In addition, summary statistics over the global population may not capture the true spatio-temporal dynamics of disease transmission. In this study we propose to use ensemble learning methods to predict epidemic generating models instead of time consuming Bayesian MCMC method. We apply these methods to infer disease transmission dynamics over spatially clustered populations, considering the clusters as natural strata instead of a global population. We compare the performance of two tree-based ensemble learning techniques: random forest and gradient boosting. These methods are applied to the 2001 foot-and-mouth disease epidemic in the U.K. and evaluated using simulated data from a clustered population. It is shown that the spatially clustered data can help to predict epidemic generating models more accurately than the global data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/314e0223106aa4f6d83b8c73c3e419e3d7dfe818" target='_blank'>
              Ensemble learning methods of inference for spatially stratified infectious disease systems.
              </a>
            </td>
          <td>
            Jeffrey Peitsch, Gyanendra Pokharel, Shakhawat Hossain
          </td>
          <td>2024-04-10</td>
          <td>The international journal of biostatistics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="A joint modeling framework was developed using data from 75 patients of early amcenestrant phase I-II AMEERA-1-2 dose escalation and expansion cohorts. A semi-mechanistic tumor growth inhibition (TGI) model was developed. It accounts for the dynamics of sensitive and resistant tumor cells, an exposure-driven effect on tumor proliferation of sensitive cells, and a delay in the initiation of treatment effect to describe the time course of target lesion tumor size (TS) data. Individual treatment exposure overtime was introduced in the model using concentrations predicted by a population pharmacokinetic model of amcenestrant. This joint modeling framework integrated complex RECISTv1.1 criteria information, linked TS metrics to progression-free survival (PFS), and was externally evaluated using the randomized phase II trial AMEERA-3. We demonstrated that the instantaneous rate of change in TS (TS slope) was an important predictor of PFS and the developed joint model was able to predict well the PFS of amcenestrant phase II monotherapy trial using only early phase I-II data. This provides a good modeling and simulation tool to inform early development decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f272ea166df369dee08b9ff8cf1d206b8c10482e" target='_blank'>
              Joint modeling of tumor dynamics and progression-free survival in advanced breast cancer: Leveraging data from amcenestrant early phase I-II trials.
              </a>
            </td>
          <td>
            M. Cerou, Hoai-Thu Thai, Laure Deyme, Sophie Fliscounakis-Huynh, Emmanuelle Comets, Patrick Cohen, S. Cartot-Cotton, C. Veyrat-Follet
          </td>
          <td>2024-04-01</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65d94b705d85169b37491f103664b0080c773c3a" target='_blank'>
              Tribulations and future opportunities for artificial intelligence in precision medicine
              </a>
            </td>
          <td>
            Claudio Carini, A. Seyhan
          </td>
          <td>2024-04-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="RNA velocity estimation helps elucidate temporal changes in the single-cell transcriptome. However, current methodologies for inferring single-cell transcriptome dynamics ignore extrinsic factors, such as experimental conditions and neighboring cell. Here, we propose ExDyn—a deep generative model integrated with splicing kinetics for estimating cell state dynamics dependent on extrinsic factors. ExDyn enables the counterfactual inference of cell state dynamics under different conditions. Among the extrinsic factors, ExDyn can extract key features which have large effects on cell state dynamics. ExDyn correctly estimated the difference in dynamics between two conditions and showed better accuracy over existing RNA velocity methods. ExDyn were utilized for unveiling the effect of PERK-knockout on neurosphere differentiation, hematopoietic stem cell differentiation driven by chromatin activity and the dynamics of squamous cell carcinoma cells dependent on colocalized neighboring cells. These results demonstrated that ExDyn is useful for analyzing key features in the dynamic generation of heterogeneous cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61c8155d4adc30e01efe07ce513bce9f437059e" target='_blank'>
              Inferring extrinsic factor-dependent single-cell transcriptome dynamics using a deep generative model
              </a>
            </td>
          <td>
            Yasuhiro Kojima, Yuko Arioka, Haruka Hirose, Shuto Hayashi, Yusuke Mizuno, Keiki Nagaharu, Hiroki Okumura, Masato Ishikawa, Kohshi Ohishi, Yutaka Suzuki, Norio Ozaki, Teppei Shimamura
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Molecular property prediction is a key component of AI-driven drug discovery and molecular characterization learning. Despite recent advances, existing methods still face challenges such as limited ability to generalize, and inadequate representation of learning from unlabeled data, especially for tasks specific to molecular structures. To address these limitations, we introduce DIG-Mol, a novel self-supervised graph neural network framework for molecular property prediction. This architecture leverages the power of contrast learning with dual interaction mechanisms and unique molecular graph enhancement strategies. DIG-Mol integrates a momentum distillation network with two interconnected networks to efficiently improve molecular characterization. The framework's ability to extract key information about molecular structure and higher-order semantics is supported by minimizing loss of contrast. We have established DIG-Mol's state-of-the-art performance through extensive experimental evaluation in a variety of molecular property prediction tasks. In addition to demonstrating superior transferability in a small number of learning scenarios, our visualizations highlight DIG-Mol's enhanced interpretability and representation capabilities. These findings confirm the effectiveness of our approach in overcoming challenges faced by traditional methods and mark a significant advance in molecular property prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5644aee2f5b4bc0bb2a8738a0748e93641a85b6" target='_blank'>
              Contrastive Dual-Interaction Graph Neural Network for Molecular Property Prediction
              </a>
            </td>
          <td>
            Zexing Zhao, Guangsi Shi, Xiaopeng Wu, Ruohua Ren, Xiaojun Gao, Fuyi Li
          </td>
          <td>2024-05-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recent progress of deep generative models in the vision and language domain has stimulated significant interest in more structured data generation such as molecules. However, beyond generating new random molecules, efficient exploration and a comprehensive understanding of the vast chemical space are of great importance to molecular science and applications in drug design and materials discovery. In this paper, we propose a new framework, ChemFlow, to traverse chemical space through navigating the latent space learned by molecule generative models through flows. We introduce a dynamical system perspective that formulates the problem as learning a vector field that transports the mass of the molecular distribution to the region with desired molecular properties or structure diversity. Under this framework, we unify previous approaches on molecule latent space traversal and optimization and propose alternative competing methods incorporating different physical priors. We validate the efficacy of ChemFlow on molecule manipulation and single- and multi-objective molecule optimization tasks under both supervised and unsupervised molecular discovery settings. Codes and demos are publicly available on GitHub at https://github.com/garywei944/ChemFlow.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86e8d48c566221e91257c327ff1b729237cc177" target='_blank'>
              Navigating Chemical Space with Latent Flows
              </a>
            </td>
          <td>
            Guanghao Wei, Yining Huang, Chenru Duan, Yue Song, Yuanqi Du
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="When modeling the dynamics of infectious disease, the incorporation of contact network information allows for the capture of the non-randomness and heterogeneity of realistic contact patterns. Oftentimes, it is assumed that the underlying contact pattern is known with perfect certainty. However, in realistic settings, the observed data often serves as an imperfect proxy of the actual contact patterns in the population. Furthermore, the epidemic in the real world are often not fully observed; event times such as infection and recovery times may be missing. In order to conduct accurate inferences on parameters of contagion spread, it is crucial to incorporate these sources of uncertainty. In this paper, we propose the use of Mixture Density Network compressed ABC (MDN-ABC) to learn informative summary statistics for the available data. This method will allow for Bayesian inference on the epidemic parameters of a contagious process, while accounting for imperfect observations on the epidemic and the contact network. We will demonstrate the use of this method on simulated epidemics and networks, and extend this framework to analyze the spread of Tattoo Skin Disease (TSD) among bottlenose dolphins in Shark Bay, Australia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a33626cae7d2e848d107957fbee4daf5ada4a8f8" target='_blank'>
              Accounting for contact network uncertainty in epidemic inferences.
              </a>
            </td>
          <td>
            Maxwell H Wang, J. Onnela
          </td>
          <td>2024-04-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Transport phenomena (e.g., fluid flows) are governed by time-dependent partial differential equations (PDEs) describing mass, momentum, and energy conservation, and are ubiquitous in many engineering applications. However, deep learning architectures are fundamentally incompatible with the simulation of these PDEs. This paper clearly articulates and then solves this incompatibility. The local-dependency of generic transport PDEs implies that it only involves local information to predict the physical properties at a location in the next time step. However, the deep learning architecture will inevitably increase the scope of information to make such predictions as the number of layers increases, which can cause sluggish convergence and compromise generalizability. This paper aims to solve this problem by proposing a distributed data scoping method with linear time complexity to strictly limit the scope of information to predict the local properties. The numerical experiments over multiple physics show that our data scoping method significantly accelerates training convergence and improves the generalizability of benchmark models on large-scale engineering simulations. Specifically, over the geometries not included in the training data for heat transferring simulation, it can increase the accuracy of Convolutional Neural Networks (CNNs) by 21.7 \% and that of Fourier Neural Operators (FNOs) by 38.5 \% on average.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5355d8f1cafbe46f9c3a9a1f31193f495099644" target='_blank'>
              Data Scoping: Effectively Learning the Evolution of Generic Transport PDEs
              </a>
            </td>
          <td>
            Jiangce Chen, Wenzhuo Xu, Zeda Xu, Noelia Grande Guti'errez, S. Narra, Christopher McComb
          </td>
          <td>2024-05-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Lung cancer starts when lung cells grow uncontrollably, forming tumours that make breathing difficult. There are more than 100 types of human cancer, and in most cases, it is untreatable due to the unavailability of medico-infrastructure and facilities, even though the USFDA approved 57 anticancer drugs in 2020 alone. WHO reported more than 10 million cancer-related deaths yearly, and lung cancer alone accounts for more than 1.80 million deaths and a few studies suggest lung cancer incidence and deaths may surpass 3.8 million and 3.2 million by 2050, which demands rapid drug designing and repurposing and the role of artificial intelligence (AI) found to be the best solutions. AI in lung cancer therapeutics has emerged as a significant area of research in recent years. This state-of-the-art review aims to explore the various applications of AI in lung cancer treatment and its potential to revolutionise patient care, and predictive models can analyse large datasets, including clinical data, genetic information, and treatment outcomes, for novel drug design and to generate personalised treatment recommendations, having the potential to optimise therapeutic strategies, enhance treatment efficacy, and minimise adverse effects.Methods: A thorough and extensive literature review was conducted after reading relevant research papers and book chapters of the last decade, indexed in PubMed and Scopus to get high-quality articles to compile this article. Several engineering conference proceedings have also been included, as they meet our quality review standards.Results: Advanced algorithms accelerate the process and improve efficiency, with accuracy beyond 95% in many cases, validated with traditional computational drug designing and repurposing approaches such as Molecular Docking and Dynamic Simulations. We have also compiled the use of convolutional neural networks, recurrent neural networks, generative adversarial networks, variational autoencoders, reinforcement learning, and many more.Conclusion: The role of AI in lung cancer therapeutics holds excellent promise through accurate detection, personalised treatment planning, novel drug design, drug repurposing, and decision support. AI can potentially transform lung cancer therapeutics by providing a robust solution that is most accurate in the least time, which can save the time and effort of experimental biological scientists. Advanced AI algorithms such as Convolutional Neural Networks, Recurrent Neural Networks, Generative Adversarial Networks, Variational Autoencoders, and Reinforcement Learning have been used in various drug repurposing articles, and even the drugs and vaccines are in clinical trial stages in just years which earlier were taking decades to get a drug or vaccine in market, and the SARS CoV-2 vaccine is the result for the same. However, further research and collaboration are required to address the existing challenges and fully realise the potential of AI in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbf816f075fede93331e91515cad6569a930a4a" target='_blank'>
              An Extensive Review on Lung Cancer Therapeutics Using Machine Learning Techniques: State-of-the-art and Perspectives.
              </a>
            </td>
          <td>
            Shaban Ahmad, Khalid Raza
          </td>
          <td>2024-04-25</td>
          <td>Journal of drug targeting</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="We train prediction and survival models using multi-omics data for disease risk identification and stratification. Existing work on disease prediction focuses on risk analysis using datasets of individual data types (metabolomic, genomics, demographic), while our study creates an integrated model for disease risk assessment. We compare machine learning models such as Lasso Regression, Multi-Layer Perceptron, XG Boost, and ADA Boost to analyze multi-omics data, incorporating ROC-AUC score comparisons for various diseases and feature combinations. Additionally, we train Cox proportional hazard models for each disease to perform survival analysis. Although the integration of multi-omics data significantly improves risk prediction for 8 diseases, we find that the contribution of metabolomic data is marginal when compared to standard demographic, genetic, and biomarker features. Nonetheless, we see that metabolomics is a useful replacement for the standard biomarker panel when it is not readily available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d9967c5fe06a6563b4c5bd97156bf728eb4041d" target='_blank'>
              Integrative machine learning approaches for predicting disease risk using multi-omics data from the UK Biobank
              </a>
            </td>
          <td>
            Oscar Aguilar, Cheng Chang, Elsa Bismuth, Manuel A. Rivas
          </td>
          <td>2024-04-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cardiomyopathies, often caused by mutations in genes encoding muscle proteins, are traditionally treated by phenotyping hearts and addressing symptoms post irreversible damage. With advancements in genotyping, early diagnosis is now possible, potentially preventing such damage. However, the intricate structure of muscle and its myriad proteins make treatment predictions challenging. Here we approach the problem of estimating therapeutic targets for a mutation in mouse muscle using a spatially explicit half sarcomere muscle model. We selected 9 rate parameters in our model linked to both small molecules and cardiomyopathy-causing mutations. We then randomly varied these rate parameters and simulated an isometric twitch for each combination to generate a large training dataset. We used this dataset to train a Conditional Variational Autoencoder (CVAE), a technique used in Bayesian parameter estimation. Given simulated or experimental isometric twitches, this machine learning model is able to then predict the set of rate parameters which are most likely to yield that result. We then predict the set of rate parameters associated with both control and the cardiac Troponin C (cTnC) I61Q variant in mouse trabeculae and and model parameters that recover the abnormal 61Q cTnC twitches. Significance Statement Machine learning techniques have potential to accelerate discoveries in biologically complex systems. However, they require large data sets and can be challenging in high dimensional systems such as cardiac muscle. In this study, we combined experimental measures of cardiac muscle twitch forces with mechanistic simulations and a newly developed mixture of Bayesian inference with neural networks (in autoencoders) to solve the inverse problem of determining the underlying kinetics for observed force generation by cardiac muscle. The autoencoders are trained on millions of simulations spanning parameter spaces that correspond to the mechanochemistry of cardiac sarcomeres. We apply the trained model to experimental data in order to infer parameters that can explain a diseased twitch and ways to recover it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/898beeadaf2f7a68b5dd4751723ce9ccd2389b02" target='_blank'>
              Identifying mechanisms and therapeutic targets in muscle using Bayesian parameter estimation with conditional variational autoencoders
              </a>
            </td>
          <td>
            Travis Tune, Kristina B. Kooiker, Jennifer Davis, Thomas L. Daniel, F. Moussavi-Harami
          </td>
          <td>2024-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d580741ad8e3ade35f1de1352b4098caef4d52a" target='_blank'>
              PMF-GRN: a variational inference approach to single-cell gene regulatory network inference using probabilistic matrix factorization
              </a>
            </td>
          <td>
            Claudia Skok Gibbs, O. Mahmood, Richard Bonneau, Kyunghyun Cho
          </td>
          <td>2024-04-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc5edf31ff74929bdf67e64bc922c3d121ffeb8" target='_blank'>
              PERCEPTION predicts patient response and resistance to treatment using single-cell transcriptomics of their tumors.
              </a>
            </td>
          <td>
            Sanju Sinha, Rahulsimham Vegesna, Sumit Mukherjee, Ashwin V Kammula, S. R. Dhruba, Wei Wu, D. Kerr, N. Nair, Matthew G Jones, N. Yosef, Oleg V Stroganov, Ivan Grishagin, Kenneth D Aldape, Collin M Blakely, Peng Jiang, Craig J Thomas, C. Benes, T. Bivona, Alejandro A. Schäffer, E. Ruppin
          </td>
          <td>2024-04-18</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Abstract Motivation The crisis of antibiotic resistance, which causes antibiotics used to treat bacterial infections to become less effective, has emerged as one of the foremost challenges to public health. Identifying the properties of antibiotic resistance genes (ARGs) is an essential way to mitigate this issue. Although numerous methods have been proposed for this task, most of these approaches concentrate solely on predicting antibiotic class, disregarding other important properties of ARGs. In addition, existing methods for simultaneously predicting multiple properties of ARGs fail to account for the causal relationships among these properties, limiting the predictive performance. Results In this study, we propose a causality-guided framework for annotating properties of ARGs, in which causal inference is utilized for representation learning. More specifically, the hidden biological patterns determining the properties of ARGs are described by a Gaussian Mixture Model, and procedure of causal representation learning is used to derive the hidden features. In addition, a causal graph among different properties is constructed to capture the causal relationships among properties of ARGs, which is integrated into the task of annotating properties of ARGs. The experimental results on a real-world dataset demonstrate the effectiveness of the proposed framework on the task of annotating properties of ARGs. Availability and implementation The data and source codes are available in GitHub at https://github.com/David-WZhao/CausalARG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df7231709034f1f37f2b153d2161d1e6da9e2c7f" target='_blank'>
              Causal-ARG: a causality-guided framework for annotating properties of antibiotic resistance genes
              </a>
            </td>
          <td>
            Weizhong Zhao, Junze Wu, Xingpeng Jiang, Tingting He, Xiaohua Hu
          </td>
          <td>2024-04-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e40dc84348e865eac9c2cf362eecd90fa43f44" target='_blank'>
              Performance and robustness of small molecule retention time prediction with molecular graph neural networks in industrial drug discovery campaigns
              </a>
            </td>
          <td>
            Daniel Vik, David Pii, Chirag Mudaliar, Mads Nørregaard-Madsen, Aleksejs Kontijevskis
          </td>
          <td>2024-04-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Determining the structure of gene regulatory networks (GRNs) is a central problem in biology, with a variety of inference methods available for different types of data. However, for a prominent and intricate scenario with single-cell gene expression data collected post-intervention across multiple time points, where joint distributions remain unknown, there is only one known specifically developed method, which does not fully utilize the rich information contained in this data type. In response, we introduce an inference approach tailored to this challenging context: netWork infErence by covariaNce DYnamics, dubbed WENDY. The core idea of WENDY is to model the dynamics of the covariance matrix, and solve this dynamics as an optimization problem to determine the regulatory relationships. To assess its efficacy, we benchmark WENDY against alternative inference methods using synthetic data. Our findings underscore WENDY’s robust performance across diverse synthetic datasets. Moreover, we deploy WENDY to analyze three distinct experimental datasets, uncovering potential gene regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b4ba528526bb4328fc81d6fd71855cf3d45b87" target='_blank'>
              WENDY: Gene Regulatory Network Inference with Covariance Dynamics
              </a>
            </td>
          <td>
            Yue Wang, Peng Zheng, Yu-Chen Cheng, Zikun Wang, Aleksandr Y Aravkin
          </td>
          <td>2024-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Learning complex trajectories from demonstrations in robotic tasks has been effectively addressed through the utilization of Dynamical Systems (DS). State-of-the-art DS learning methods ensure stability of the generated trajectories; however, they have three shortcomings: a) the DS is assumed to have a single attractor, which limits the diversity of tasks it can achieve, b) state derivative information is assumed to be available in the learning process and c) the state of the DS is assumed to be measurable at inference time. We propose a class of provably stable latent DS with possibly multiple attractors, that inherit the training methods of Neural Ordinary Differential Equations, thus, dropping the dependency on state derivative information. A diffeomorphic mapping for the output and a loss that captures time-invariant trajectory similarity are proposed. We validate the efficacy of our approach through experiments conducted on a public dataset of handwritten shapes and within a simulated object manipulation task.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc227c83d593a317b47926de7d4a6905d2fc78a4" target='_blank'>
              Learning Deep Dynamical Systems using Stable Neural ODEs
              </a>
            </td>
          <td>
            Andreas Sochopoulos, M. Gienger, S. Vijayakumar
          </td>
          <td>2024-04-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Spatially-resolved datasets are revolutionizing knowledge in molecular biology, yet are under-utilized for questions in evolutionary biology. To gain insight from these large-scale datasets of spatial organization, we need mathematical representations and modeling techniques that can both capture their complexity, but also allow for mathematical tractability. Evolutionary graph theory utilizes the mathematical representation of networks as a proxy for heterogeneous population structure and has started to reshape our understanding of how spatial structure can direct evolutionary dynamics. However, previous results are derived for the case of a single new mutation appearing in the population and the role of network structure in shaping fitness landscape crossing is still poorly understood. Here we study how network structured populations cross fitness landscapes and show that even a simple extension to a two-mutational landscape can exhibit complex evolutionary dynamics that cannot be predicted using previous single-mutation results. We show how our results can be intuitively understood through the lens of how the two main evolutionary properties of a network, the amplification and acceleration factors, change the expected fate of the intermediate mutant in the population and further discuss how to link these models to spatially-resolved datasets of cellular organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/911293f0b38ae0b018a47894b1107474919de7b0" target='_blank'>
              Evolutionary graph theory beyond single mutation dynamics: on how network structured populations cross fitness landscapes.
              </a>
            </td>
          <td>
            Y. Kuo, Oana Carja
          </td>
          <td>2024-04-18</td>
          <td>Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="New technology for energy storage is necessary for the large-scale adoption of renewable energy sources like wind and solar. The ability to discover suitable catalysts is crucial for making energy storage more cost-effective and scalable. The Open Catalyst Project aims to apply advances in graph neural networks (GNNs) to accelerate progress in catalyst discovery, replacing Density Functional Theory-based (DFT) approaches that are computationally burdensome. Current approaches involve scaling GNNs to over 1 billion parameters, pushing the problem out of reach for a vast majority of machine learning practitioner around the world. This study aims to evaluate the performance and insights gained from using more lightweight approaches for this task that are more approachable for smaller teams to encourage participation from individuals from diverse backgrounds. By implementing robust design patterns like geometric and symmetric message passing, we were able to train a GNN model that reached a MAE of 0.0748 in predicting the per-atom forces of adsorbate-surface interactions, rivaling established model architectures like SchNet and DimeNet++ while using only a fraction of trainable parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1c2c8bf4df3944736f857ce3044d53ae1822ed" target='_blank'>
              Lightweight Geometric Deep Learning for Molecular Modelling in Catalyst Discovery
              </a>
            </td>
          <td>
            Patrick Geitner
          </td>
          <td>2024-04-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 The temporal resolution of environmental data sets plays a major role in the granularity of the information that can be derived from the data. In most cases, it is required that different data sets have a common temporal resolution to enable their consistent evaluations and applications in making informed decisions. This study leverages deep learning with long short-term memory (LSTM) neural networks and model inference to enhance the temporal resolution of climate datasets, specifically temperature, and precipitation, from daily to sub-daily scales. We trained our model to learn the relationship between daily and sub-daily data, subsequently applying this knowledge to increase the resolution of a separate dataset with a coarser (daily) temporal resolution. Our findings reveal a high degree of accuracy for temperature predictions, evidenced by a correlation of 0.99 and a mean absolute error of 0.21 °C, between the actual and predicted sub-daily values. In contrast, the approach was less effective for precipitation, achieving an explained variance of only 37%, compared to 98% for temperature. Further, besides the sub-daily interpolation of the climate data sets, we adapted our approach to increase the resolution of the Normalized difference vegetation index of Landsat (from 16-day to 5-day interval) using the LSTM model pre-trained from the Sentinel 2 Normalized difference vegetation index - that exists at a relatively higher temporal resolution. The explained variance between the predicted Landsat and Sentinel 1 data is 70% with a mean absolute error of 0.03. These results suggest that our method is particularly suitable for environmental datasets with less pronounced short-term variability, offering a promising tool for improving the resolution and utility of the data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b365a24bc7a55c6465814e7fae80c9eb78ae8b6" target='_blank'>
              Interpolation of Environmental Data Using Deep Learning and Model Inference
              </a>
            </td>
          <td>
            C. Ibebuchi, Itohan-Osa Abu
          </td>
          <td>2024-05-14</td>
          <td>Machine Learning: Science and Technology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Solubility is not only a significant physical property of molecules but also a vital factor in smallmolecule drug development. Determining drug solubility demands stringent equipment, controlled environments, and substantial human and material resources. The accurate prediction of drug solubility using computational methods has long been a goal for researchers. In this study, we introduce MSCSol, a solubility prediction model that integrates multidimensional molecular structure information. We incorporate a graph neural network with geometric vector perceptrons (GVP-GNN) to encode 3D molecular structures, representing spatial arrangement and orientation of atoms, as well as atomic sequences and interactions. We also employ Selective Kernel Convolution combined with Global and Local attention mechanisms to capture molecular features context at different scales. Additionally, various descriptors are calculated to enrich the molecular representation. For the 2D and 3D structural data of molecules, we design different data augmentation strategies to enhance generalization ability and prevent the model from learning irrelevant information. Extensive experiments on benchmark and independent datasets demonstrate MSCSol's superior performance. Ablation studies further confirm the effectiveness of different modules. Interpretability analysis highlights the importance of various atomic groups and substructures for solubility and verifies that our model effectively captures functional molecular structures and higher-order knowledge. The source code and datasets are freely available at https://github.com/ZiyuFanCSU/MSCSol.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aa873b8be5cb3b95dff75f43afd7b047937f102" target='_blank'>
              Enhancing Predictions of Drug Solubility through Multidimensional Structural Characterization Exploitation.
              </a>
            </td>
          <td>
            Ziyu Fan, Linying Chen, Xinyi Wu, Zhijian Huang, Lei Deng
          </td>
          <td>2024-05-07</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The ongoing expansion of human genomic datasets propels therapeutic target identification; however, extracting gene-disease associations from gene annotations remains challenging. Here, we introduce Mantis-ML 2.0, a framework integrating AstraZeneca’s Biological Insights Knowledge Graph and numerous tabular datasets, to assess gene-disease probabilities throughout the phenome. We use graph neural networks, capturing the graph’s holistic structure, and train them on hundreds of balanced datasets via a robust semi-supervised learning framework to provide gene-disease probabilities across the human exome. Mantis-ML 2.0 incorporates natural language processing to automate disease-relevant feature selection for thousands of diseases. The enhanced models demonstrate a 6.9% average classification power boost, achieving a median receiver operating characteristic (ROC) area under curve (AUC) score of 0.90 across 5220 diseases from Human Phenotype Ontology, OpenTargets, and Genomics England. Notably, Mantis-ML 2.0 prioritizes associations from an independent UK Biobank phenome-wide association study (PheWAS), providing a stronger form of triaging and mitigating against underpowered PheWAS associations. Results are exposed through an interactive web resource.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8a8dff231452d1dacf518826430fa5c77addce0" target='_blank'>
              Phenome-wide identification of therapeutic genetic targets, leveraging knowledge graphs, graph neural networks, and UK Biobank data
              </a>
            </td>
          <td>
            L. Middleton, Ioannis Melas, Chirag Vasavda, A. Raies, Benedek Rozemberczki, R. Dhindsa, Justin S. Dhindsa, Blake Weido, Quanli Wang, A. R. Harper, Gavin Edwards, S. Petrovski, D. Vitsios
          </td>
          <td>2024-05-08</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9e0572b203657323ee5ca7f8a580528219adc51" target='_blank'>
              Prospective de novo drug design with deep interactome learning
              </a>
            </td>
          <td>
            Kenneth Atz, Leandro Cotos, Clemens Isert, Maria Håkansson, Dorota Focht, Mattis Hilleke, David F. Nippa, Michael Iff, Jann Ledergerber, Carl C. G. Schiebroek, Valentina Romeo, J. Hiss, Daniel Merk, Petra Schneider, Bernd Kuhn, U. Grether, Gisbert Schneider
          </td>
          <td>2024-04-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="In many real-world applications where the system dynamics has an underlying interdependency among its variables (such as power grid, economics, neuroscience, omics networks, environmental ecosystems, and others), one is often interested in knowing whether the past values of one time series influences the future of another, known as Granger causality, and the associated underlying dynamics. This paper introduces a Koopman-inspired framework that leverages neural networks for data-driven learning of the Koopman bases, termed NeuroKoopman Dynamic Causal Discovery (NKDCD), for reliably inferring the Granger causality along with the underlying nonlinear dynamics. NKDCD employs an autoencoder architecture that lifts the nonlinear dynamics to a higher dimension using data-learned bases, where the lifted time series can be reliably modeled linearly. The lifting function, the linear Granger causality lag matrices, and the projection function (from lifted space to base space) are all represented as multilayer perceptrons and are all learned simultaneously in one go. NKDCD also utilizes sparsity-inducing penalties on the weights of the lag matrices, encouraging the model to select only the needed causal dependencies within the data. Through extensive testing on practically applicable datasets, it is shown that the NKDCD outperforms the existing nonlinear Granger causality discovery approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2063b00f6c1d0ea80dc4fa01e9b3b3bddc847ea9" target='_blank'>
              NeuroKoopman Dynamic Causal Discovery
              </a>
            </td>
          <td>
            Rahmat Adesunkanmi, Balaji Sesha Srikanth Pokuri, Ratnesh Kumar
          </td>
          <td>2024-04-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Exploring the missing values is an essential but challenging issue due to the complex latent spatio-temporal correlation and dynamic nature of time series. Owing to the outstanding performance in dealing with structure learning potentials, Graph Neural Networks (GNNs) and Recurrent Neural Networks (RNNs) are often used to capture such complex spatio-temporal features in multivariate time series. However, these data-driven models often fail to capture the essential spatio-temporal relationships when significant signal corruption occurs. Additionally, calculating the high-order neighbor nodes in these models is of high computational complexity. To address these problems, we propose a novel higher-order spatio-temporal physics-incorporated GNN (HSPGNN). Firstly, the dynamic Laplacian matrix can be obtained by the spatial attention mechanism. Then, the generic inhomogeneous partial differential equation (PDE) of physical dynamic systems is used to construct the dynamic higher-order spatio-temporal GNN to obtain the missing time series values. Moreover, we estimate the missing impact by Normalizing Flows (NF) to evaluate the importance of each node in the graph for better explainability. Experimental results on four benchmark datasets demonstrate the effectiveness of HSPGNN and the superior performance when combining various order neighbor nodes. Also, graph-like optical flow, dynamic graphs, and missing impact can be obtained naturally by HSPGNN, which provides better dynamic analysis and explanation than traditional data-driven models. Our code is available at https://github.com/gorgen2020/HSPGNN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99dd7e2f500bd8a4c1bfc703d640c7ba45e7f301" target='_blank'>
              Physics-incorporated Graph Neural Network for Multivariate Time Series Imputation
              </a>
            </td>
          <td>
            Guojun Liang, Prayag Tiwari, Slawomir Nowaczyk, S. Byttner
          </td>
          <td>2024-05-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract How complex phenotypes emerge from intricate gene expression patterns is a fundamental question in biology. Quantitative characterization of this relationship, however, is challenging due to the vast combinatorial possibilities and dynamic interplay between genotype and phenotype landscapes. Integrating high-content genotyping approaches such as single-cell RNA sequencing and advanced learning methods such as language models offers an opportunity for dissecting this complex relationship. Here, we present a computational integrated genetics framework designed to analyze and interpret the high-dimensional landscape of genotypes and their associated phenotypes simultaneously. We applied this approach to develop a multimodal foundation model to explore the genotype-phenotype relationship manifold for human transcriptomics at the cellular level. Analyzing this joint manifold showed a refined resolution of cellular heterogeneity, enhanced precision in phenotype annotating, and uncovered potential cross-tissue biomarkers that are undetectable through conventional gene expression analysis alone. Moreover, our results revealed that the gene networks are characterized by scale-free patterns and show context-dependent gene-gene interactions, both of which result in significant variations in the topology of the gene network, particularly evident during aging. Finally, utilizing contextualized embeddings, we investigated gene polyfunctionality which illustrates the multifaceted roles that genes play in different biological processes, and demonstrated that for VWF gene in endothelial cells. Overall, this study advances our understanding of the dynamic interplay between gene expression and phenotypic manifestation and demonstrates the potential of integrated genetics in uncovering new dimensions of cellular function and complexity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4827f60ad81afcd6b7c4af4bb58fc23e2809c1" target='_blank'>
              Multimodal Learning for Mapping the Genotype-Phenotype Dynamics
              </a>
            </td>
          <td>
            Farhan Khodaee, Rohola Zandie, Elazer Edelman
          </td>
          <td>2024-05-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Exploring the degree to which phenotypic variation, influenced by intrinsic nonlinear biological mechanisms, can be accurately captured using statistical methods is essential for advancing our comprehension of complex biological systems and predicting their functionality. Here, we examine this issue by combining a computational model of gene regulation networks with a linear additive prediction model, akin to polygenic scores utilized in genetic analyses. Inspired by the variational framework of quantitative genetics, we create a population of individual networks possessing identical topology yet showcasing diversity in regulatory strengths. By discerning which regulatory connections determine the prediction of phenotypes, we contextualize our findings within the framework of core and peripheral causal determinants, as proposed by the omnigenic model of complex traits. We establish connections between our results and concepts such as global sensitivity and local stability in dynamical systems, alongside the notion of sloppy parameters in biological models. Furthermore, we explore the implications of our investigation for the broader discourse surrounding the role of epistatic interactions in the prediction of complex phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9d655e857d6cdd755c46f5fbfbbbb85e241f74" target='_blank'>
              Polygenic prediction and gene regulation networks
              </a>
            </td>
          <td>
            J. F. Poyatos
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Patients with metastatic triple-negative breast cancer (TNBC) show variable responses to PD-1 inhibition. Efficient patient selection by predictive biomarkers would be desirable, but is hindered by the limited performance of existing biomarkers. Here, we leveraged in-silico patient cohorts generated using a quantitative systems pharmacology model of metastatic TNBC, informed by transcriptomic and clinical data, to explore potential ways to improve patient selection. We tested 90 biomarker candidates, including various cellular and molecular species, by a cutoff-based biomarker testing algorithm combined with machine learning-based feature selection. Combinations of pre-treatment biomarkers improved the specificity compared to single biomarkers at the cost of reduced sensitivity. On the other hand, early on-treatment biomarkers, such as the relative change in tumor diameter from baseline measured at two weeks after treatment initiation, achieved remarkably higher sensitivity and specificity. Further, blood-based biomarkers had a comparable ability to tumor- or lymph node-based biomarkers in identifying a subset of responders, potentially suggesting a less invasive way for patient selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79736289b9a8e4c9e35fea6f911f314586cacdbc" target='_blank'>
              Virtual patient analysis identifies strategies to improve the performance of predictive biomarkers for PD-1 blockade
              </a>
            </td>
          <td>
            Theinmozhi Arulraj, Hanwen Wang, Atul Deshpande, Ravi Varadhan, Leisha A Emens, Elizabeth M. Jaffee, E. Fertig, Cesar A. Santa-Maria, Aleksander S. Popel
          </td>
          <td>2024-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Drug-target interaction databases comprise millions of manually curated data points, yet there are missed opportunities for repurposing established interaction networks to infer novel drug-target interactions by interpolating on chemical neighborhoods. To address this gap, we collect drug-target interactions on 128 unique GPCRs across 187K molecules and establish an all-vs-all chemical space network, which can be efficiently calculated and parameterised as a graph with 32.4 billion potential edges. Beyond testing state-of-the-art machine learning approaches, we develop a chemical space neural network (CSNN) to infer drug activity classes with up to 98% accuracy, by leveraging the graph of chemical neighbourhoods. We combine this virtual library screen with a fast and cheap experimental platform to validate our predictions and to discover 14 novel drug-GPCR interactions. Altogether, our platform integrates virtual library screening and experimental validation to facilitate fast and efficient coverage of missing drug-target interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b5c062c6c74a88a48c33164dce0f2833a8fc1df" target='_blank'>
              Harnessing Chemical Space Neural Networks to Systematically Annotate GPCR ligands
              </a>
            </td>
          <td>
            Frederik G. Hansson, Niklas Gesmar Madsen, Lea Gram Hansen, Tadas Jakočiūnas, Bettina Lengger, Jay D. Keasling, Michael K Jensen, Carlos G. Acevedo-Rocha, Emil D. Jensen
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a3ce0a8aec6be70af6f6d05daafba40902c31f8" target='_blank'>
              MLASM: Machine learning based prediction of anticancer small molecules.
              </a>
            </td>
          <td>
            Priya Dharshini Balaji, Subathra Selvam, Honglae Sohn, T. Madhavan
          </td>
          <td>2024-03-30</td>
          <td>Molecular diversity</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Combination treatment–the mainstay of cancer therapy–often fails because treatment interactions evoke complex resistance mechanisms that are hard to predict. Designing combination therapy to prevent treatment resistance is especially challenging for rare cancers. Here, we introduce REsistance through COntext DRift (RECODR): a computational pipeline that tracks how genes change their transcriptome context across cancer development and drug treatment conditions. By applying RECODR to a genetically modified mouse model of choroid plexus carcinoma–a rare brain tumour of young children–we identified patterns of transcriptome evolution, cellular heterogeneity and treatment targets that emerged as tumours were initiated and resisted combination treatment. This enabled the prediction of treatment resistance mechanisms and the design of highly effective therapeutic protocols that avoided treatment failure. RECODR can describe complex and dynamic changes in normal and diseased tissues that could be applied to multimodal data from a variety of settings, mitigating treatment resistance across cancers and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920edce889e5d3cf5f44bc7f5d04ff4b697a9b88" target='_blank'>
              Natural language processing charts transcriptome evolution to design combination cancer therapies
              </a>
            </td>
          <td>
            Amir Jassim, Birgit V. Nimmervoll, Sabrina Terranova, Erica Nathan, Katherine E. Masih, Lisa Ruff, Matilde Duarte, Elizabeth Cooper, Linda P. Hu, Gunjan Katyal, Melika Akhbari, Reuben J. Gilbertson, Colton Terhune, Gabriel Balmus, Stephen P. Jackson, Mariella G. Filbin, Anthony Hill, Anarita Patrizi, Neil Dani, Aviv Regev, Maria K. Lehtinen, Richard J. Gilbertson
          </td>
          <td>2024-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Despite rapid advancements in treatments and technology, breast cancer remains the second leading cause of cancer related morbidity in women in the US. As a heterogeneous disease, the unique genetic and phenotypic profile of each tumor complicates the prediction of treatment responses and optimized treatment strategies. One primary manifestation of phenotypic plasticity and concomitant intra-tumoral heterogeneity is the Epithelial-Mesenchymal Transition (EMT). The acquisition of migratory and invasive properties through EMT facilitates tumor metastasis and results in worse patient outcomes. As a result, the EMT and its reverse process (MET) often influence treatment response. However, the different mechanisms through which these therapeutic drugs affect the EMT-related phenotypic switching rates as well as each phenotype’s replication rates are unclear, and further investigation is required for better understanding their precise effect for directing optimal treatment strategies and improved patient outcomes. Here, we compile an atlas of publicly available metastatic and non-metastatic RNA sequencing data of breast cancer cell lines treated with therapeutic and chemotherapeutic drugs and propose a dual data-driven theoretical framework for understanding the effects of each drug on breast cancer treatment response. We infer the cell fractions, EMT phenotypic transition rates and phenotypic replication rates of each case through an extension of our previously developed hybrid data-driven and theoretical model, COMET. We infer single-cell RNA sequencing-derived EMT-related trajectories and apply a continuous time Markov chain model to infer inter-state transition rates. By analyzing the atlas of RNA sequencing data and integration with our framework, we gain novel insights into the context-specific mechanisms by which therapeutic drugs affect the phenotypic transition rates. This comprehensive understanding will contribute to optimizing breast cancer therapy and ultimately improving patient outcomes.
 Citation Format: A. Najafi. Optimizing the Therapeutic Treatment of Breast Cancer Through a Dual Data-driven Theoretical Model [abstract]. In: Proceedings of the 2023 San Antonio Breast Cancer Symposium; 2023 Dec 5-9; San Antonio, TX. Philadelphia (PA): AACR; Cancer Res 2024;84(9 Suppl):Abstract nr PO5-24-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b06a9b9f939f3dbefa8f3e022b1bd86985d0877" target='_blank'>
              Abstract PO5-24-09: Optimizing the Therapeutic Treatment of Breast Cancer Through a Dual Data-driven Theoretical Model
              </a>
            </td>
          <td>
            A. Najafi
          </td>
          <td>2024-05-02</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73aa716b5c07b408cd6033f78ab02b1433429672" target='_blank'>
              Learning the mechanisms of network growth
              </a>
            </td>
          <td>
            Lourens Touwen, Doina Bucur, R. Hofstad, Alessandro Garavaglia, Nelly Litvak
          </td>
          <td>2024-03-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Networked dynamical systems are widely used as formal models of real-world cascading phenomena, such as the spread of diseases and information. Prior research has addressed the problem of learning the behavior of an unknown dynamical system when the underlying network has a single layer. In this work, we study the learnability of dynamical systems over multilayer networks, which are more realistic and challenging. First, we present an efficient PAC learning algorithm with provable guarantees to show that the learner only requires a small number of training examples to infer an unknown system. We further provide a tight analysis of the Natarajan dimension which measures the model complexity. Asymptotically, our bound on the Nararajan dimension is tight for almost all multilayer graphs. The techniques and insights from our work provide the theoretical foundations for future investigations of learning problems for multilayer dynamical systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4c0063e47ca85cf9c10b3ade6904eaa76704c7" target='_blank'>
              Efficient PAC Learnability of Dynamical Systems Over Multilayer Networks
              </a>
            </td>
          <td>
            Zirou Qiu, Abhijin Adiga, M. Marathe, S. S. Ravi, D. Rosenkrantz, R. Stearns, Anil Vullikanti
          </td>
          <td>2024-05-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/477bcc7b10d17b6d768d343469473e99b1a78282" target='_blank'>
              Integrated clinical and genomic models using machine-learning methods to predict the efficacy of paclitaxel-based chemotherapy in patients with advanced gastric cancer
              </a>
            </td>
          <td>
            Yonghwa Choi, Jangwoo Lee, Keewon Shin, Ji Won Lee, Ju Won Kim, Soohyeon Lee, Yoon Ji Choi, Kyong Hwa Park, Jwa Hoon Kim
          </td>
          <td>2024-04-20</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A new knowledge-based and machine learning hybrid modeling approach, called conditional Gaussian neural stochastic differential equation (CGNSDE), is developed to facilitate modeling complex dynamical systems and implementing analytic formulae of the associated data assimilation (DA). In contrast to the standard neural network predictive models, the CGNSDE is designed to effectively tackle both forward prediction tasks and inverse state estimation problems. The CGNSDE starts by exploiting a systematic causal inference via information theory to build a simple knowledge-based nonlinear model that nevertheless captures as much explainable physics as possible. Then, neural networks are supplemented to the knowledge-based model in a specific way, which not only characterizes the remaining features that are challenging to model with simple forms but also advances the use of analytic formulae to efficiently compute the nonlinear DA solution. These analytic formulae are used as an additional computationally affordable loss to train the neural networks that directly improve the DA accuracy. This DA loss function promotes the CGNSDE to capture the interactions between state variables and thus advances its modeling skills. With the DA loss, the CGNSDE is more capable of estimating extreme events and quantifying the associated uncertainty. Furthermore, crucial physical properties in many complex systems, such as the translate-invariant local dependence of state variables, can significantly simplify the neural network structures and facilitate the CGNSDE to be applied to high-dimensional systems. Numerical experiments based on chaotic systems with intermittency and strong non-Gaussian features indicate that the CGNSDE outperforms knowledge-based regression models, and the DA loss further enhances the modeling skills of the CGNSDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a639323a3ab8c39800f9e9f42ae3d95438cb1ec6" target='_blank'>
              CGNSDE: Conditional Gaussian Neural Stochastic Differential Equation for Modeling Complex Systems and Data Assimilation
              </a>
            </td>
          <td>
            Chuanqi Chen, Nan Chen, Jingbo Wu
          </td>
          <td>2024-04-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Various published COVID-19 models have been used in epidemiological studies and healthcare planning to model and predict the spread of the disease and appropriately realign health measures and priorities given the resource limitations in the field of healthcare. However, a significant issue arises when these models need help identifying the distribution of the constituent variants of COVID-19 infections. The emergence of such a challenge means that, given limited healthcare resources, health planning would be ineffective and cost lives. This work presents a universal neural network (NN) computational instrument for predicting the mainstream symptomatic infection rate of COVID-19 and models of the distribution of its associated variants. The NN is based on a mixture density network (MDN) with a Gaussian mixture model (GMM) object as a backbone. Twelve use cases were used to demonstrate the validity and reliability of the proposed MDN. The use cases included COVID-19 data for Canada and Saudi Arabia, two date ranges (300 and 500 days), two input data modes, and three activation functions, each with different implementations of the batch size and epoch value. This array of scenarios provided an opportunity to investigate the impacts of epistemic uncertainty (EU) and aleatoric uncertainty (AU) on the prediction model’s fitting. The model accuracy readings were in the high nineties based on a tolerance margin of 0.0125. The primary outcome of this work indicates that this easy-to-use universal MDN helps provide reliable predictions of COVID-19 variant distributions and the corresponding synthesized profile of the mainstream infection rate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bfb9fb92059e15480512d95ed1a467a9e84429c" target='_blank'>
              A Novel Computational Instrument Based on a Universal Mixture Density Network with a Gaussian Mixture Model as a Backbone for Predicting COVID-19 Variants’ Distributions
              </a>
            </td>
          <td>
            Y. Al-Hadeethi, Intesar F. El Ramley, Hiba Mohammed, Nada M. Bedaiwi, Abeer z. Barasheed
          </td>
          <td>2024-04-20</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ffa9bfa3c3984b355ec71361380dbb9ad8aca54" target='_blank'>
              An integrated framework for prognosis prediction and drug response modeling in colorectal liver metastasis drug discovery
              </a>
            </td>
          <td>
            Xiuman Zhou, Yuzhen Qian, Chen Ling, Zhuo-bin He, Peishang Shi, Yanfeng Gao, Xinghua Sui
          </td>
          <td>2024-03-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="This article presents an in-depth analysis and evaluation of artificial neural networks (ANNs) when applied to replicate trajectories in molecular dynamics (MD) simulations or other particle methods. This study focuses on several architectures—feedforward neural networks (FNNs), convolutional neural networks (CNNs), recurrent neural networks (RNNs), time convolutions (TCs), self-attention (SA), graph neural networks (GNNs), neural ordinary differential equation (ODENets), and an example of physics-informed machine learning (PIML) model—assessing their effectiveness and limitations in understanding and replicating the underlying physics of particle systems. Through this analysis, this paper introduces a comprehensive set of criteria designed to evaluate the capability of ANNs in this context. These criteria include the minimization of losses, the permutability of particle indices, the ability to predict trajectories recursively, the conservation of particles, the model’s handling of boundary conditions, and its scalability. Each network type is systematically examined to determine its strengths and weaknesses in adhering to these criteria. While, predictably, none of the networks fully meets all criteria, this study extends beyond the simple conclusion that only by integrating physics-based models into ANNs is it possible to fully replicate complex particle trajectories. Instead, it probes and delineates the extent to which various neural networks can “understand” and interpret aspects of the underlying physics, with each criterion targeting a distinct aspect of this understanding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d2cd2e98b8c0ad116f74af97b8583a17b803b5f" target='_blank'>
              Designing a set of criteria for evaluating artificial neural networks trained with physics-based data to replicate molecular dynamics and other particle method trajectories
              </a>
            </td>
          <td>
            Alessio Alexiadis
          </td>
          <td>2024-04-10</td>
          <td>Frontiers in Nanotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Machine learning models support computer-aided molecular design and compound optimization. However, the initial phases of drug discovery often face a scarcity of training data for these models. Meta-learning has emerged as a potentially promising strategy, harnessing the wealth of structure-activity data available for known targets to facilitate efficient few-shot model training for the specific target of interest. In this study, we assessed the effectiveness of two different meta-learning methods, namely model-agnostic meta-learning (MAML) and adaptive deep kernel fitting (ADKF), specifically in the regression setting. We investigated how factors such as dataset size and the similarity of training tasks impact predictability. The results indicate that ADKF significantly outperformed both MAML and a single-task baseline model on the inhibition data. However, the performance of ADKF varied across different test tasks. Our findings suggest that considerable enhancements in performance can be anticipated primarily when the task of interest is similar to the tasks incorporated in the meta-learning process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32a6e659b8193146d505ac34674aa54852ff102" target='_blank'>
              Task-similarity is a crucial factor for few-shot meta-learning of structure-activity relationships.
              </a>
            </td>
          <td>
            Alexander Kötter, Stephan Allenspach, Christoph Grebner, Hans Matter, J. Hiss, Gisbert Schneider, Gerhard Hessler
          </td>
          <td>2024-04-29</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="A network, whose nodes are genes and whose directed edges represent positive or negative influences of a regulatory gene and its targets, is often used as a representation of causality. To infer a network, researchers often develop a machine learning model and then evaluate the model based on its match with experimentally verified “gold standard” edges. The desired result of such a model is a network that may extend the gold standard edges. Since networks are a form of visual representation, one can compare their utility with architectural or machine blueprints. Blueprints are clearly useful because they provide precise guidance to builders in construction. If the primary role of gene regulatory networks is to characterize causality, then such networks should be good tools of prediction because prediction is the actionable benefit of knowing causality. But are they? In this paper, we compare prediction quality based on “gold standard” regulatory edges from previous experimental work with non-linear models inferred from time series data across four different species. We show that the same non-linear machine learning models have better predictive performance, with improvements from 5.3% to 25.3% in terms of the reduction in the root mean square error (RMSE) compared with the same models based on the gold standard edges. Having established that networks fail to characterize causality properly, we suggest that causality research should focus on four goals: (i) predictive accuracy; (ii) a parsimonious enumeration of predictive regulatory genes for each target gene g; (iii) the identification of disjoint sets of predictive regulatory genes for each target g of roughly equal accuracy; and (iv) the construction of a bipartite network (whose node types are genes and models) representation of causality. We provide algorithms for all goals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd7eda0fbce6aa023e877e8ea11a162f2aca8b4" target='_blank'>
              Bipartite networks represent causality better than simple networks: evidence, algorithms, and applications
              </a>
            </td>
          <td>
            Bingran Shen, Gloria Curozzi, Dennis Shasha
          </td>
          <td>2024-05-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Computational models are not just appealing because they can simulate and predict the development of biological phenomena across multiple spatial and temporal scales, but also because they can integrate information from well-established in vitro and in vivo models and test new hypotheses in cancer biomedicine. Agent-based models and simulations are especially interesting candidates among computational modeling procedures in cancer research due to the capability to, for instance, recapitulate the dynamics of neoplasia and tumor - host interactions. Yet, the absence of methods to validate the consistency of the results across scales can hinder adoption by turning fine-tuned models into black boxes. This review compiles relevant literature that explores strategies to leverage high-fidelity simulations of multi-scale, or multi-level, cancer models with a focus on verification approached as simulation calibration. We consolidate our review with an outline of modern approaches for agent-based models' validation and provide an ambitious outlook toward rigorous and reliable calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e0c3944c13aa863a5f0e7e95045776f19c4393" target='_blank'>
              Agent-based modeling in cancer biomedicine: applications and tools for calibration and validation.
              </a>
            </td>
          <td>
            Nicolò Cogno, Cristian Axenie, Roman Bauer, V. Vavourakis
          </td>
          <td>2024-04-28</td>
          <td>Cancer biology & therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="In the realm of predictive toxicology for small molecules, the applicability domain of QSAR models is often limited by the coverage of the chemical space in the training set. Consequently, classical models fail to provide reliable predictions for wide classes of molecules. However, the emergence of innovative data collection methods such as intensive hackathons have promise to quickly expand the available chemical space for model construction. Combined with algorithmic refinement methods, these tools can address the challenges of toxicity prediction, enhancing both the robustness and applicability of the corresponding models. This study aimed to investigate the roles of gradient boosting and strategic data aggregation in enhancing the predictivity ability of models for the toxicity of small organic molecules. We focused on evaluating the impact of incorporating fragment features and expanding the chemical space, facilitated by a comprehensive dataset procured in an open hackathon. We used gradient boosting techniques, accounting for critical features such as the structural fragments or functional groups often associated with manifestations of toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd79272eeef9dba73fe4181d2dcd91d255dffcc3" target='_blank'>
              Expanding Predictive Capacities in Toxicology: Insights from Hackathon-Enhanced Data and Model Aggregation
              </a>
            </td>
          <td>
            Dmitrii O. Shkil, Alina A. Muhamedzhanova, Philipp I. Petrov, Ekaterina V. Skorb, Timur A. Aliev, Ilya S Steshin, Alexander V. Tumanov, Alexander S. Kislinskiy, Maxim V. Fedorov
          </td>
          <td>2024-04-01</td>
          <td>Molecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39ff774114afdb5fa389a578e894de7644886e8b" target='_blank'>
              Optimized model architectures for deep learning on genomic data
              </a>
            </td>
          <td>
            Hüseyin Anil Gündüz, René Mreches, Julia Moosbauer, Gary Robertson, Xiao-Yin To, E. Franzosa, Curtis Huttenhower, Mina Rezaei, A. Mchardy, Bernd Bischl, Philipp C. Münch, Martin Binder
          </td>
          <td>2024-04-30</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Causal effect estimation aims at estimating the Average Treatment Effect as well as the Conditional Average Treatment Effect of a treatment to an outcome from the available data. This knowledge is important in many safety-critical domains, where it often needs to be extracted from observational data. In this work, we propose a new causal inference model, named C-XGBoost, for the prediction of potential outcomes. The motivation of our approach is to exploit the superiority of tree-based models for handling tabular data together with the notable property of causal inference neural network-based models to learn representations that are useful for estimating the outcome for both the treatment and non-treatment cases. The proposed model also inherits the considerable advantages of XGBoost model such as efficiently handling features with missing values requiring minimum preprocessing effort, as well as it is equipped with regularization techniques to avoid overfitting/bias. Furthermore, we propose a new loss function for efficiently training the proposed causal inference model. The experimental analysis, which is based on the performance profiles of Dolan and Mor{\'e} as well as on post-hoc and non-parametric statistical tests, provide strong evidence about the effectiveness of the proposed approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e484a192be7ca41717fc03c01d2a6f284d79b407" target='_blank'>
              C-XGBoost: A tree boosting model for causal effect estimation
              </a>
            </td>
          <td>
            Niki Kiriakidou, I. Livieris, Christos Diou
          </td>
          <td>2024-03-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892efafe4af541ea8214c1f23ba672dfd2b2b94c" target='_blank'>
              PheSeq, a Bayesian deep learning model to enhance and interpret the gene-disease association studies
              </a>
            </td>
          <td>
            Xinzhi Yao, Sizhuo Ouyang, Yulong Lian, Qianqian Peng, Xionghui Zhou, Feier Huang, Xuehai Hu, Feng Shi, Jingbo Xia
          </td>
          <td>2024-04-16</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Uncertainty in low dose ionizing radiation induced health risks stems from several factors. The complex biological pathways leading to diseases like cancer are not fully understood, making it difficult to distinguish the contribution of radiation, particularly at low doses which induce only small perturbations to background disease risks. Additionally, traditional dose response models, such as the Linear No Threshold formalism and competing threshold or hormesis models, impose rigid assumptions on dose response shapes, causing controversy and increasing model selection uncertainty. Furthermore, these modeling strategies operate on the level of correlations/associations, and are not designed to directly address the ultimate goal of radiation epidemiology assessing causal links between radiation and disease. A promising and rapidly developing approach for addressing some of these challenges is causal machine learning (CML), such as double/debiased machine learning (DML), which is designed to model causal effects in multi dimensional data sets. Our study employs DML to elucidate the causal impacts of radiation exposure on the incidence of leukemia, all solid tumors, and stomach tumors among Japanese atomic bomb survivors. Its goal was not to produce a definitive re analysis of these data sets, but to provide a useful example of implementing CML in radiation epidemiology, which can advance the field by supplementing traditional modeling approaches. The results revealed robust positive causal effects of radiation for all three tumor types, especially for leukemia and stomach tumors. The effect magnitudes, and uncertainties, were not dramatically different at low doses than at higher doses. The influences of age at exposure, attained age, sex and other covariates on the causal effects of radiation were assessed using Shapley Additive Explanations (SHAP) values. We believe that this analysis, based on a flexible machine learning framework with a causal inference motivation and without strict dose response assumptions, provides an important contribution to radiation epidemiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/792df0e8a3b11233cd61634953d8953b01530526" target='_blank'>
              Causal Machine Learning Analysis of Radiation-Induced Leukemia and Solid Tumor Incidence in Japanese Atomic Bomb Survivors
              </a>
            </td>
          <td>
            I. Shuryak, Zhenqiu Liu, Eric Wang, R. L. Ullrich, David J. Brenner
          </td>
          <td>2024-05-01</td>
          <td>None</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Randomized Clinical trials (RCT) suffer from a high failure rate which could be caused by heterogeneous responses to treatment. Despite many models being developed to estimate heterogeneous treatment effects (HTE), there remains a lack of interpretable methods to identify responsive subgroups. This work aims to develop a framework to identify subgroups based on treatment effects that prioritize model interpretability. The proposed framework leverages an ensemble uplift tree method to generate descriptive decision rules that separate samples given estimated responses to the treatment. Subsequently, we select a complementary set of these decision rules and rank them using a sparse linear model. To address the trial’s limited sample size problem, we proposed a data augmentation strategy by borrowing control patients from external studies and generating synthetic data. We apply the proposed framework to a failed randomized clinical trial for investigating an intracerebral hemorrhage therapy plan. The Qini-scores show that the proposed data augmentation strategy plan can boost the model’s performance and the framework achieves greater interpretability by selecting complementary descriptive rules without compromising estimation quality. Our model derives clinically meaningful subgroups. Specifically, we find those patients with Diastolic Blood Pressure≥70 mm hg and Systolic Blood Pressure<215 mm hg benefit more from intensive blood pressure reduction therapy. The proposed interpretable HTE analysis framework offers a promising potential for extracting meaningful insight from RCTs with neutral treatment effects. By identifying responsive subgroups, our framework can contribute to developing personalized treatment strategies for patients more efficiently.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9e9a9f508c58be4c6a2714873e926dcab3830e2" target='_blank'>
              An interpretable framework to identify responsive subgroups from clinical trials regarding treatment effects: Application to treatment of intracerebral hemorrhage
              </a>
            </td>
          <td>
            Yaobin Ling, Muhammad Bilal Tariq, Kaichen Tang, Jaroslaw Aronowski, Yang Fann, S. Savitz, Xiaoqian Jiang, Yejin Kim
          </td>
          <td>2024-05-01</td>
          <td>PLOS Digital Health</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer remains a leading cause of death worldwide, with its heterogeneous and patient-specific nature complicating treatment. Immunotherapy, especially immune checkpoint blockades (ICB), shows promising potential by reactivating the immune system to detect and destroy cancer cells. However, their effectiveness largely depends on the patient’s clinical and molecular characteristics, making them unsuitable for everyone. This highlights the importance of developing predictive and interpretable models that could distinguish potential responders from non-responders. This study aims to utilize gene expression data to predict patients’ responses to ICB and identify key genes whose expression is associated with treatment outcome using probabilistic Bayesian networks (BNs). To address the high-dimensional nature of gene expression data, we propose and compare the use of machine learning-based feature selection (FS) and clustering by weighted gene co-expression network analysis (WGCNA). To our knowledge, this is the first study to apply BNs along with FS and WGCNA dimensionality reduction techniques to tackle ICB response using transcriptomic data. We achieved a maximal classification accuracy of 75% and identified eight genes and two gene clusters directly influencing ICB response. These results may offer more biological insights and mechanisms underlying ICB response. The proposed pipeline will contribute to guiding treatment decisions, minimizing harmful side effects, and reducing the financial costs associated with ineffective treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbbc861e13246a2c515a393ebb44eb2b8dd4dc66" target='_blank'>
              AI In Healthcare: Using Bayesian Networks To Predict Response To Immune Checkpoint Blockades
              </a>
            </td>
          <td>
            Ahmed H. Saadawy, Aly F. Hissam, Eman Badr, Omar M. Fahmy
          </td>
          <td>2024-04-16</td>
          <td>2024 41st National Radio Science Conference (NRSC)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Identifying functionally important cell states and structure within a heterogeneous tumor remains a significant biological and computational challenge. Moreover, current clustering or trajectory-based computational models are ill-equipped to address the notion that cancer cells reside along a phenotypic continuum. To address this, we present Archetypal Analysis network (AAnet), a neural network that learns key archetypal cell states within a phenotypic continuum of cell states in single-cell data. Applied to single-cell RNA sequencing data from pre-clinical models and a cohort of 34 clinical breast cancers, AAnet identifies archetypes that resolve distinct biological cell states and processes, including cell proliferation, hypoxia, metabolism and immune interactions. Notably, archetypes identified in primary tumors are recapitulated in matched liver, lung and lymph node metastases, demonstrating that a significant component of intratumoral heterogeneity is driven by cell intrinsic properties. Using spatial transcriptomics as orthogonal validation, AAnet-derived archetypes show discrete spatial organization within tumors, supporting their distinct archetypal biology. We further reveal that ligand:receptor cross-talk between cancer and adjacent stromal cells contributes to intra-archetypal biological mimicry. Finally, we use AAnet archetype identifiers to validate GLUT3 as a critical mediator of a hypoxic cell archetype harboring a cancer stem cell population, which we validate in human triple-negative breast cancer specimens. AAnet is a powerful tool to reveal functional cell states within complex samples from multimodal single-cell data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32b3a7328824f9e25b9a694c5c5c104d8ffd3103" target='_blank'>
              AAnet resolves a continuum of spatially-localized cell states to unveil tumor complexity
              </a>
            </td>
          <td>
            Aarthi Venkat, Scott E. Youlten, B. P. S. Juan, Carley Purcell, Matthew Amodio, Daniel B. Burkhardt, Andrew Benz, Jeff Holst, Cerys McCool, A. Mollbrink, J. Lundeberg, David van Dijk, Leonard D Goldstein, Sarah Kummerfeld, Smita Krishnaswamy, Christine L. Chaffer
          </td>
          <td>2024-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="
 The capabilities of artificial intelligence (AI) and machine learning (ML) are pivotal for refining patient stratification and subtype discrimination in clinical trials. Conventional ML methods often rely on large data sets for meaningful discoveries. NetraAI is a novel ML approach designed and trained to work with smaller data sets. The challenge with smaller data sets is that they do not reflect the totality of the disease that they represent. NetraAI employs a novel approach termed “Sub-Insight Learning”, utilizing validated mathematical methods to analyze even small patient data sets. This allows the system to decompose the data sets into high and low confidence patient subpopulations, enhancing predictive model accuracy and reducing overfitting. Further, the system explains what variables are driving the etiology defining the subpopulations of patients. Using two non-small cell lung cancer (NSCLC) data sets (GSE18842 and GSE10245) consisting of only 104 samples from adenocarcinoma (ADC) and squamous cell carcinoma (SCC), NetraAI distinguished the two subtypes through unique genetic signatures. Notably, nine of the ten variables identified correlate with known NSCLC markers, with PIGX emerging as a novel target. Leveraging protein-protein interaction networks (PPI) revealed connections between PIGX and BACE1. BACE1 has been implicated as a driver of NSCLC brain metastasis. These findings shed light on the biology of membrane proteins and their post-translational modifications, a factor implicated in various diseases, prompting further exploration. NetraAI demonstrates a significant breakthrough in precision medicine for oncology, capable of generating meaningful insights from small data sets. The discovery of novel biomarkers and their implications in cancer and other diseases underline the potential of this AI-driven approach in advancing current research paradigms and patient-specific treatments.
 Citation Format: Bessi Qorri, Mike J. Tsay, Paul Leonchyk, Larry Alphs, Luca Pani, Joseph Geraci. The power of NetraAI: Precision medicine in oncology through sub-insight learning from small data sets [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2024; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2024 Apr 5-10; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2024;84(7_Suppl):Abstract nr LB396.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34efcb9bba7ff8ed0e70e0eb0e81c71163f53cce" target='_blank'>
              Abstract LB396: The power of NetraAI: Precision medicine in oncology through sub-insight learning from small data sets
              </a>
            </td>
          <td>
            Bessi Qorri, Mike J. Tsay, Paul Leonchyk, Larry Alphs, Luca Pani, Joseph Geraci
          </td>
          <td>2024-04-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The application of machine learning (ML) to -omics research is growing at an exponential rate owing to the increasing availability of large amounts of data for model training. Specifically, in metabolomics, ML has enabled the prediction of tandem mass spectrometry and retention time data. More recently, due to the advent of ion mobility, new ML models have been introduced for collision cross-section (CCS) prediction, but those have been trained with different and relatively small data sets covering a few thousands of small molecules, which hampers their systematic comparison. Here, we compared four existing ML-based CCS prediction models and their capacity to predict CCS values using the recently introduced METLIN-CCS data set. We also compared them with simple linear models and with ML models that used fingerprints as regressors. We analyzed the role of structural diversity of the data on which the ML models are trained with and explored the practical application of these models for metabolite annotation using CCS values. Results showed a limited capability of the existing models to achieve the necessary accuracy to be adopted for routine metabolomics analysis. We showed that for a particular molecule, this accuracy could only be improved when models were trained with a large number of structurally similar counterparts. Therefore, we suggest that current annotation capabilities will only be significantly altered with models trained with heterogeneous data sets composed of large homogeneous hubs of structurally similar molecules to those being predicted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eff2111bbdd8acbf556cdbd9aa48fde442fa07a" target='_blank'>
              Predicting the Predicted: A Comparison of Machine Learning-Based Collision Cross-Section Prediction Models for Small Molecules.
              </a>
            </td>
          <td>
            Sara M de Cripan, Trisha Arora, Adrià Olomí, N. Canela, Gary Siuzdak, Xavier Domingo-Almenara
          </td>
          <td>2024-05-24</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Multi-modal biological datasets provide rich information from diverse scales or facets of complex biological systems that can be analyzed to highlight the critical multi-scale interactions underlying specific biological phenomena. However, identifying the most vital associations among features and outputs can be beset by a high degree of spurious connections due to indirect effects of various immune features propagating through an unmapped biological network. Here, we applied a probabilistic graphical modeling approach, Markov Fields, to empirically dissect the most direct associations correlations between features from a public multi-modal dataset (antibody titers, antibody-dependent functions, cytokines, cytometry) from macaques undergoing intravenous BCG vaccination—a promising vaccine strategy against the major public health threat tuberculosis. This yielded a network influence model that interprets the assemblage of multi-scale paths by which vaccine effects propagate through the immune network to eventually protect against tuberculosis infection. Importantly, our modeling shows that the vast majority of apparent correlations between features arise from indirect effects relating distant immune features. For a test of predictive capability, we conducted experimental depletion of B cells during BCG IV vaccination in macaques--which did not reduce BCG IV-mediated protection against tuberculosis— and validated that our Markov Field model can predict systems-wide modulation of numerous features across the immune system in response to this perturbation. Finally, we applied our model to discern perturbations in the network that could have strong effects on IV-BCG efficacy. All together, we have demonstrated that probabilistic graphical modeling can increase interpretability and predictive value of multi-modal datasets for identifying new disease treatment targets. Summary Multi-modal datasets provide rich information that can help identify critical multi-scale interactions underlying biological systems. However, identifying associations between features and outputs can be beset by spurious connections due to indirect impacts propagating through an unmapped biological network. We applied a probabilistic graphical modeling approach, Markov Fields, to dissect correlations between immune features in a public multi-modal dataset (systems serology, cytokines, cytometry) of macaques undergoing intravenous BCG vaccination against tuberculosis. This yielded an interaction network that interprets network paths underlying vaccine efficacy, and shows how correlations between features often arise indirectly. We next conducted experimental depletion of B cells during vaccination in macaques—which did not reduce protection against tuberculosis—to validate our Markov Field model’s predictions of network-wide shifts post-depletion. Finally, we highlight immune changes predicted to strongly affect intravenous BCG vaccine efficacy, showing that probabilistic graphical models increase the interpretability of multi-modal datasets for identifying new disease targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad2f1e0ba10cbde79ba37f6d358ceb194ee91f48" target='_blank'>
              Markov Field network integration of multi-modal data predicts effects of immune system perturbations on intravenous BCG vaccination in macaques
              </a>
            </td>
          <td>
            Shu Wang, Amy J Myers, Edward B. Irvine, Chuangqi Wang, Pauline Maiello, Mark A. Rodgers, Jaime Tomko, Kara Kracinovsky, H. Borish, Michael C Chao, Douaa Mugahid, P. Darrah, Robert A. Seder, Mario Roederer, C. Scanga, P. Lin, Galit Alter, Sarah M Fortune, JoAnne L. Flynn, D. Lauffenburger
          </td>
          <td>2024-04-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7a0a8265ea7e318e65ccdf6ef53d19670ab7b3" target='_blank'>
              A novel non-negative Bayesian stacking modeling method for Cancer survival prediction using high-dimensional omics data
              </a>
            </td>
          <td>
            Junjie Shen, Shuo Wang, Hao Sun, Jie Huang, Lu Bai, Xichao Wang, Yongfei Dong, Zaixiang Tang
          </td>
          <td>2024-05-03</td>
          <td>BMC Medical Research Methodology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Optimizing techniques for discovering molecular structures with desired properties is crucial in artificial intelligence(AI)-based drug discovery. Combining deep generative models with reinforcement learning has emerged as an effective strategy for generating molecules with specific properties. Despite its potential, this approach is ineffective in exploring the vast chemical space and optimizing particular chemical properties. To overcome these limitations, we present Mol-AIR, a reinforcement learning-based framework using adaptive intrinsic rewards for effective goal-directed molecular generation. Mol-AIR leverages the strengths of both history-based and learning-based intrinsic rewards by exploiting random distillation network and counting-based strategies. In benchmark tests, Mol-AIR demonstrates superior performance over existing approaches in generating molecules with desired properties without any prior knowledge, including penalized LogP, QED, and celecoxib similarity. We believe that Mol-AIR represents a significant advancement in drug discovery, offering a more efficient path to discovering novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a1b39ded76621764814fe339cfbc69d9d7158b" target='_blank'>
              Mol-AIR: Molecular Reinforcement Learning with Adaptive Intrinsic Rewards for Goal-directed Molecular Generation
              </a>
            </td>
          <td>
            Jinyeong Park, Jaegyoon Ahn, Jonghwan Choi, Jibum Kim
          </td>
          <td>2024-03-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The acquisition of different data modalities can enhance our knowledge and understanding of various diseases, paving the way for a more personalized healthcare. Thus, medicine is progressively moving towards the generation of massive amounts of multi-modal data (\emph{e.g,} molecular, radiology, and histopathology). While this may seem like an ideal environment to capitalize data-centric machine learning approaches, most methods still focus on exploring a single or a pair of modalities due to a variety of reasons: i) lack of ready to use curated datasets; ii) difficulty in identifying the best multi-modal fusion strategy; and iii) missing modalities across patients. In this paper we introduce a real world multi-modal dataset called MMIST-CCRCC that comprises 2 radiology modalities (CT and MRI), histopathology, genomics, and clinical data from 618 patients with clear cell renal cell carcinoma (ccRCC). We provide single and multi-modal (early and late fusion) benchmarks in the task of 12-month survival prediction in the challenging scenario of one or more missing modalities for each patient, with missing rates that range from 26$\%$ for genomics data to more than 90$\%$ for MRI. We show that even with such severe missing rates the fusion of modalities leads to improvements in the survival forecasting. Additionally, incorporating a strategy to generate the latent representations of the missing modalities given the available ones further improves the performance, highlighting a potential complementarity across modalities. Our dataset and code are available here: https://multi-modal-ist.github.io/datasets/ccRCC">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2863bd595c7a0052e2f0b6b6b0f794b1dadd4695" target='_blank'>
              MMIST-ccRCC: A Real World Medical Dataset for the Development of Multi-Modal Systems
              </a>
            </td>
          <td>
            Tiago Mota, M. R. Verdelho, Diogo J. Ara´ujo, Alceu Bissoto, Carlos Santiago, Catarina Barata
          </td>
          <td>2024-05-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In the pursuit of advanced Predictive Health Management (PHM) for Proton Exchange Membrane Fuel Cells (PEMFCs), conventional data-driven models encounter considerable barriers due to data reconstruction resulting in poor data quality, and the complexity of models leading to insufficient interpretability. In addressing these challenges, this research introduces TabNet, a model aimed at augmenting predictive interpretability, and integrates it with an innovative data preprocessing technique to enhance the predictive performance of PEMFC health management. In traditional data processing approaches, reconstruction methods are employed on the original dataset, significantly reducing its size and consequently diminishing the accuracy of model predictions. To overcome this challenge, the Segmented Random Sampling Correction (SRSC) methodology proposed herein effectively eliminates noise from the original dataset whilst maintaining its effectiveness. Notably, as the majority of deep learning models operate as black boxes, it becomes challenging to identify the exact factors affecting the Remaining Useful Life (RUL) of PEMFCs, which is clearly disadvantageous for the health management of PEMFCs. Nonetheless, TabNet offers insights into the decision-making process for predicting the RUL of PEMFCs, for instance, identifying which experimental parameters significantly influence the prediction outcomes. Specifically, TabNet’s distinctive design employs sequential attention to choose features for reasoning at each decision-making step, not only enhancing the accuracy of RUL predictions in PEMFC but also offering interpretability of the results. Furthermore, this study utilized Gaussian augmentation techniques to boost the model’s generalization capability across varying operational conditions. Through pertinent case studies, the efficacy of this integrated framework, merging data processing with the TabNet architecture, was validated. This work not only evidences that the effective data processing and strategic deployment of TabNet can markedly elevate model performance but also, via a visual analysis of the parameters’ impact, provides crucial insights for the future health management of PEMFCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afebf444e22d23cfded78ce190b20d2a9cbdf4a1" target='_blank'>
              TabNet: Locally Interpretable Estimation and Prediction for Advanced Proton Exchange Membrane Fuel Cell Health Management
              </a>
            </td>
          <td>
            Benyuan Zhang, Xin Jin, Wenyu Liang, Xiaoyu Chen, Zhenhong Li, G. Panoutsos, Zepeng Liu, Zezhi Tang
          </td>
          <td>2024-04-03</td>
          <td>Electronics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93e99767340c76e0af7f776ed66f5a1299b84479" target='_blank'>
              Inferring gene regulatory networks from single-cell multiome data using atlas-scale external data.
              </a>
            </td>
          <td>
            Qiuyue Yuan, Zhana Duren
          </td>
          <td>2024-04-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 In order to develop strategies that can detect cancer earlier, we need to better understand the dynamics of cancer evolution during the human lifetime. Starting from gestation, this includes measuring both the timing of early somatic mutations that lead to clonal mosaicism as well as the growth rates of mutated clones that carry higher potential for malignant transformation. Continuous observation of this evolving clonal composition across a lifetime is not feasible, therefore mathematical models are needed to integrate data from longitudinal and cross-sectional samples and infer these dynamics. Stochastic models of clonal evolution offer such a framework and provide a means for predicting individual clone trajectories into the future. Furthermore, if measurements of evolution are to be utilized clinically for early detection and patient risk stratification, faster methods to apply these models to data are required. To this end, we derived methods using coalescent theory for single-cell DNA sequencing data that enable instantaneous estimation of the growth rate of a clone along with its confidence intervals. Our tool is available in an open source R package, cloneRate, and provides the first method for constructing valid confidence intervals of growth rates analytically without having to rely on Bayesian inference techniques and complex simulations. When applying our methods to recent datasets derived from normal and neoplastic human blood cells, we quantified increased fitness effects of multi-hit driver mutations and found that higher initial clone growth rates led to decreased time to cancer diagnosis in patients. We can also use these models to estimate other important biological parameters that are difficult to measure in vivo such as bounds for total number of hematopoietic stem cells and expansion rates in early development. Ultimately, the ability to easily and quickly parameterize clonal dynamics in individual patients will benefit future early detection efforts and screening strategies for many cancer types.
 Citation Format: Kit Curtius. Stochastic modeling of clonal evolution in carcinogenesis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2024; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2024 Apr 5-10; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2024;84(7_Suppl):Abstract nr SY02-03.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667bf584bf783399c32a44b6d6cc0431ed744efc" target='_blank'>
              Abstract SY02-03: Stochastic modeling of clonal evolution in carcinogenesis
              </a>
            </td>
          <td>
            Kit Curtius
          </td>
          <td>2024-04-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Revealing interactions in complex systems from observed collective dynamics constitutes a fundamental inverse problem in science. Some methods may reveal undirected network topology, e.g., using node-node correlation. Yet, the direction of the interaction, thus a causal inference, remains to be determined - especially in steady-state observations. We introduce a method to infer the directionality within this network only from a “snapshot” of the abundances of the relevant molecules. We examine the validity of the approach for different properties of the system and the data recorded, such as the molecule’s level variability, the effect of sampling and measurement errors. Simulations suggest that the given approach successfully infer the reaction rates in various cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4835c23912310f411c24ba509da9c73cfc8936cd" target='_blank'>
              Determining Interaction Directionality in Complex Biochemical Networks from Stationary Measurements
              </a>
            </td>
          <td>
            Nava Leibovich
          </td>
          <td>2024-04-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multi-modality spatio-temporal (MoST) data extends spatio-temporal (ST) data by incorporating multiple modalities, which is prevalent in monitoring systems, encompassing diverse traffic demands and air quality assessments. Despite significant strides in ST modeling in recent years, there remains a need to emphasize harnessing the potential of information from different modalities. Robust MoST forecasting is more challenging because it possesses (i) high-dimensional and complex internal structures and (ii) dynamic heterogeneity caused by temporal, spatial, and modality variations. In this study, we propose a novel MoST learning framework via Self-Supervised Learning, namely MoSSL, which aims to uncover latent patterns from temporal, spatial, and modality perspectives while quantifying dynamic heterogeneity. Experiment results on two real-world MoST datasets verify the superiority of our approach compared with the state-of-the-art baselines. Model implementation is available at https://github.com/beginner-sketch/MoSSL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c10d27669d165b26a1256f70ca1349c6cb5f00ea" target='_blank'>
              Multi-Modality Spatio-Temporal Forecasting via Self-Supervised Learning
              </a>
            </td>
          <td>
            Jiewen Deng, Renhe Jiang, Jiaqi Zhang, Xuan Song
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="We develop a framework for derivative Gaussian process latent variable models (DGP-LVM) that can handle multi-dimensional output data using modified derivative covariance functions. The modifications account for complexities in the underlying data generating process such as scaled derivatives, varying information across multiple output dimensions as well as interactions between outputs. Further, our framework provides uncertainty estimates for each latent variable samples using Bayesian inference. Through extensive simulations, we demonstrate that latent variable estimation accuracy can be drastically increased by including derivative information due to our proposed covariance function modifications. The developments are motivated by a concrete biological research problem involving the estimation of the unobserved cellular ordering from single-cell RNA (scRNA) sequencing data for gene expression and its corresponding derivative information known as RNA velocity. Since the RNA velocity is only an estimate of the exact derivative information, the derivative covariance functions need to account for potential scale differences. In a real-world case study, we illustrate the application of DGP-LVMs to such scRNA sequencing data. While motivated by this biological problem, our framework is generally applicable to all kinds of latent variable estimation problems involving derivative information irrespective of the field of study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1986daa831dd3d31707e327db8303c32d0687f05" target='_blank'>
              DGP-LVM: Derivative Gaussian process latent variable model
              </a>
            </td>
          <td>
            Soham Mukherjee, Manfred Claassen, Paul-Christian Burkner
          </td>
          <td>2024-04-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42c799ba35fd15bb0cf5c7b3a8a75540a86ba092" target='_blank'>
              Inverse problem solver for epidemiological geographic profiling
              </a>
            </td>
          <td>
            Y. Maeno
          </td>
          <td>2024-03-30</td>
          <td>Evolutionary and Institutional Economics Review</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The idea of the inheritance of biological processes, such as the developmental process or the life cycle of an organism, has been discussed in the biology literature, but formal mathematical descriptions and plausible data analysis frameworks are lacking. We introduce an extension of the nested Dirichlet Process (nDP) to a multiscale model to aid in understanding the mechanisms by which biological processes are inherited, remain stable, and are modified across generations. To address these issues, we introduce Nested Inheritance Dynamics Algorithm (NIDA). At its primary level, NIDA encompasses all processes unfolding within an individual organism's lifespan. The secondary level delineates the dynamics through which these processes evolve or remain stable over time. This framework allows for the specification of a physical system model at either scale, thus promoting seamless integration with established models of development and heredity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27878ba4e6c3fb15b252dec51e39b92e7cbcfa80" target='_blank'>
              Nested Inheritance Dynamics
              </a>
            </td>
          <td>
            Bahman Moraffah
          </td>
          <td>2024-04-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Early phase, personalized dose-finding trials for combination therapies seek to identify patient-specific optimal biological dose (OBD) combinations, which are defined as safe dose combinations which maximize therapeutic benefit for a specific covariate pattern. Given the small sample sizes which are typical of these trials, it is challenging for traditional parametric approaches to identify OBD combinations across multiple dosing agents and covariate patterns. To address these challenges, we propose a Bayesian optimization approach to dose-finding which formally incorporates toxicity information into both the initial data collection process and the sequential search strategy. Independent Gaussian processes are used to model the efficacy and toxicity surfaces, and an acquisition function is utilized to define the dose-finding strategy and an early stopping rule. This work is motivated by a personalized dose-finding trial which considers a dual-agent therapy for obstructive sleep apnea, where OBD combinations are tailored to obstructive sleep apnea severity. To compare the performance of the personalized approach to a standard approach where covariate information is ignored, a simulation study is performed. We conclude that personalized dose-finding is essential in the presence of heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90f080ef4e69d5073a927e58d4f4a03e75c9555" target='_blank'>
              Bayesian Optimization for Identification of Optimal Biological Dose Combinations in Personalized Dose-Finding Trials
              </a>
            </td>
          <td>
            James J. Willard, Shirin Golchi, Erica EM. Moodie
          </td>
          <td>2024-04-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In hematology, computational models offer significant potential to improve diagnostic accuracy, streamline workflows, and reduce the tedious work of analyzing single cells in peripheral blood or bone marrow smears. However, clinical adoption of computational models has been hampered by the lack of generalization due to large batch effects, small dataset sizes, and poor performance in transfer learning from natural images. To address these challenges, we introduce DinoBloom, the first foundation model for single cell images in hematology, utilizing a tailored DINOv2 pipeline. Our model is built upon an extensive collection of 13 diverse, publicly available datasets of peripheral blood and bone marrow smears, the most substantial open-source cohort in hematology so far, comprising over 380,000 white blood cell images. To assess its generalization capability, we evaluate it on an external dataset with a challenging domain shift. We show that our model outperforms existing medical and non-medical vision models in (i) linear probing and k-nearest neighbor evaluations for cell-type classification on blood and bone marrow smears and (ii) weakly supervised multiple instance learning for acute myeloid leukemia subtyping by a large margin. A family of four DinoBloom models (small, base, large, and giant) can be adapted for a wide range of downstream applications, be a strong baseline for classification problems, and facilitate the assessment of batch effects in new datasets. All models are available at github.com/marrlab/DinoBloom.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0beb7e4b15c3e24d18fff6db82c90f57bd57117b" target='_blank'>
              DinoBloom: A Foundation Model for Generalizable Cell Embeddings in Hematology
              </a>
            </td>
          <td>
            Valentin Koch, S. Wagner, Salome Kazeminia, Ece Sancar, Matthias Hehr, Julia A. Schnabel, Tingying Peng, Carsten Marr
          </td>
          <td>2024-04-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Increasing effort is put into the development of methods for learning mechanistic models from data. This task entails not only the accurate estimation of parameters, but also a suitable model structure. Recent work on the discovery of dynamical systems formulates this problem as a linear equation system. Here, we explore several simulation-based optimization approaches, which allow much greater freedom in the objective formulation and weaker conditions on the available data. We show that even for relatively small stochastic population models, simultaneous estimation of parameters and structure poses major challenges for optimization procedures. Particularly, we investigate the application of the local stochastic gradient descent method, commonly used for training machine learning models. We demonstrate accurate estimation of models but find that enforcing the inference of parsimonious, interpretable models drastically increases the difficulty. We give an outlook on how this challenge can be overcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f3306c5d346d91ce086b55f87d085d98c721dd" target='_blank'>
              Towards Learning Stochastic Population Models by Gradient Descent
              </a>
            </td>
          <td>
            J. N. Kreikemeyer, Philipp Andelfinger, A. Uhrmacher
          </td>
          <td>2024-04-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Recurrent Neural Networks excel at predicting and generating complex high-dimensional temporal patterns. Due to their inherent nonlinear dynamics and memory, they can learn unbounded temporal dependencies from data. In a Machine Learning setting, the network's parameters are adapted during a training phase to match the requirements of a given task/problem increasing its computational capabilities. After the training, the network parameters are kept fixed to exploit the learned computations. The static parameters thereby render the network unadaptive to changing conditions, such as external or internal perturbation. In this manuscript, we demonstrate how keeping parts of the network adaptive even after the training enhances its functionality and robustness. Here, we utilize the conceptor framework and conceptualize an adaptive control loop analyzing the network's behavior continuously and adjusting its time-varying internal representation to follow a desired target. We demonstrate how the added adaptivity of the network supports the computational functionality in three distinct tasks: interpolation of temporal patterns, stabilization against partial network degradation, and robustness against input distortion. Our results highlight the potential of adaptive networks in machine learning beyond training, enabling them to not only learn complex patterns but also dynamically adjust to changing environments, ultimately broadening their applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d77920a1f3057b33f9ab48db38a16dc2b0f292" target='_blank'>
              Adaptive control of recurrent neural networks using conceptors
              </a>
            </td>
          <td>
            Guillaume Pourcel, Mirko Goldmann, Ingo Fischer, Miguel C. Soriano
          </td>
          <td>2024-05-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In early phase drug development of combination therapy, the primary objective is to preliminarily assess whether there is additive activity when a novel agent combined with an established monotherapy. Due to potential feasibility issues with a large randomized study, uncontrolled single-arm trials have been the mainstream approach in cancer clinical trials. However, such trials often present significant challenges in deciding whether to proceed to the next phase of development. A hybrid design, leveraging data from a completed historical clinical study of the monotherapy, offers a valuable option to enhance study efficiency and improve informed decision-making. Compared to traditional single-arm designs, the hybrid design may significantly enhance power by borrowing external information, enabling a more robust assessment of activity. The primary challenge of hybrid design lies in handling information borrowing. We introduce a Bayesian dynamic power prior (DPP) framework with three components of controlling amount of dynamic borrowing. The framework offers flexible study design options with explicit interpretation of borrowing, allowing customization according to specific needs. Furthermore, the posterior distribution in the proposed framework has a closed form, offering significant advantages in computational efficiency. The proposed framework's utility is demonstrated through simulations and a case study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78e242e6116fa53c9b8b476f30d2afb4078de476" target='_blank'>
              A Bayesian Hybrid Design with Borrowing from Historical Study
              </a>
            </td>
          <td>
            Zhaohua Lu, John Toso, Girma Ayele, Philip He
          </td>
          <td>2024-04-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Widespread deployment of societal-scale machine learning systems necessitates a thorough understanding of the resulting long-term effects these systems have on their environment, including loss of trustworthiness, bias amplification, and violation of AI safety requirements. We introduce a repeated learning process to jointly describe several phenomena attributed to unintended hidden feedback loops, such as error amplification, induced concept drift, echo chambers and others. The process comprises the entire cycle of obtaining the data, training the predictive model, and delivering predictions to end-users within a single mathematical model. A distinctive feature of such repeated learning setting is that the state of the environment becomes causally dependent on the learner itself over time, thus violating the usual assumptions about the data distribution. We present a novel dynamical systems model of the repeated learning process and prove the limiting set of probability distributions for positive and negative feedback loop modes of the system operation. We conduct a series of computational experiments using an exemplary supervised learning problem on two synthetic data sets. The results of the experiments correspond to the theoretical predictions derived from the dynamical model. Our results demonstrate the feasibility of the proposed approach for studying the repeated learning processes in machine learning systems and open a range of opportunities for further research in the area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbdf526001c151c67507de963a0c683064b1f630" target='_blank'>
              A Mathematical Model of the Hidden Feedback Loop Effect in Machine Learning Systems
              </a>
            </td>
          <td>
            Andrey Veprikov, Alexander Afanasiev, Anton Khritankov
          </td>
          <td>2024-05-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Large language models (LLMs) have revolutionized information processing and are now being applied to complex problems in molecular biosciences. LLMs excel at learning patterns in biological sequences, enabling applications such as predicting the function of DNA regulatory elements, analysing chromatin modifications and predicting the effects of genetic variants. In drug discovery, LLMs are being used to optimize drug properties, design novel molecules and even automate compound synthesis. However, realizing the full potential of LLMs in the molecular biosciences requires addressing challenges such as designing effective learning objectives, generating high-quality, domain-specific datasets and building trust in AI-assisted decision-making. As these challenges are addressed, LLMs hold immense potential for advancing research and driving ground-breaking discoveries in molecular biosciences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9294fb20cdfbaebb4505ff551617846c24cfae9f" target='_blank'>
              The potential and pitfalls of large language models in molecular biosciences
              </a>
            </td>
          <td>
            Chengqi Xu, Olivier Elemento
          </td>
          <td>2024-05-16</td>
          <td>The Biochemist</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The role of surgery in metastatic breast cancer (MBC) is currently controversial. Several novel statistical and deep learning (DL) methods promise to infer the suitability of surgery at the individual level.The objective of this study was to identify the most applicable DL model for determining patients with MBC who could benefit from surgery and the type of surgery required.We introduced the deep survival regression with mixture effects (DSME), a semi‐parametric DL model integrating three causal inference methods. Six models were trained to make individualized treatment recommendations. Patients who received treatments in line with the DL models' recommendations were compared with those who underwent treatments divergent from the recommendations. Inverse probability weighting (IPW) was used to minimize bias. The effects of various features on surgery selection were visualized and quantified using multivariate linear regression and causal inference.In total, 5269 female patients with MBC were included. DSME was an independent protective factor, outperforming other models in recommending surgery (IPW‐adjusted hazard ratio [HR] = 0.39, 95% confidence interval [CI]: 0.19–0.78) and type of surgery (IPW‐adjusted HR = 0.66, 95% CI: 0.48–0.93). DSME was superior to other models and traditional guidelines, suggesting a higher proportion of patients benefiting from surgery, especially breast‐conserving surgery. The debiased effect of patient characteristics, including age, tumor size, metastatic sites, lymph node status, and breast cancer subtypes, on surgery decision was also quantified.Our findings suggested that DSME could effectively identify patients with MBC likely to benefit from surgery and the specific type of surgery needed. This method can facilitate the development of efficient, reliable treatment recommendation systems and provide quantifiable evidence for decision‐making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da62886f95973a7efe1b5f65259b9781cd201f80" target='_blank'>
              Personalized surgical recommendations and quantitative therapeutic insights for patients with metastatic breast cancer: Insights from deep learning
              </a>
            </td>
          <td>
            Enzhao Zhu, Linmei Zhang, Jiayi Wang, Chunyu Hu, Qi Jing, Weizhong Shi, Ziqin Xu, Pu Ai, Zhihao Dai, Dan Shan, Zisheng Ai
          </td>
          <td>2024-04-15</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In biological tasks, data is rarely plentiful as it is generated from hard-to-gather measurements. Therefore, pre-training foundation models on large quantities of available data and then transfer to low-data downstream tasks is a promising direction. However, how to design effective foundation models for molecular learning remains an open question, with existing approaches typically focusing on models with large parameter capacities. In this work, we propose $\texttt{MiniMol}$, a foundational model for molecular learning with 10 million parameters. $\texttt{MiniMol}$ is pre-trained on a mix of roughly 3300 sparsely defined graph- and node-level tasks of both quantum and biological nature. The pre-training dataset includes approximately 6 million molecules and 500 million labels. To demonstrate the generalizability of $\texttt{MiniMol}$ across tasks, we evaluate it on downstream tasks from the Therapeutic Data Commons (TDC) ADMET group showing significant improvements over the prior state-of-the-art foundation model across 17 tasks. $\texttt{MiniMol}$ will be a public and open-sourced model for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12358bf3f9e3a18c9170e8b81ec706937afc4cf0" target='_blank'>
              MiniMol: A Parameter-Efficient Foundation Model for Molecular Learning
              </a>
            </td>
          <td>
            Kerstin Kläser, Bla.zej Banaszewski, S. Maddrell-Mander, Callum McLean, Luis Müller, Ali Parviz, Shenyang Huang, Andrew Fitzgibbon
          </td>
          <td>2024-04-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="It is a challenging task to identify functional transcriptional regulators, which control expression of gene sets via regulatory elements and epigenomic signals, involving context-specific studies such as development and diseases. Integrating large-scale multi-omics epigenomic data enables the elucidation of the complex epigenomic control patterns of regulatory elements and regulators. Here, we propose TRAPT, a multi-modality deep learning framework that predicts functional transcriptional regulators from a queried gene set by integrating large-scale multi-omics epigenomic data, including histone modifications, ATAC-seq and TR-ChIP-seq. We design two-stage self-knowledge distillation model to learn nonlinear embedded representation of upstream and downstream regulatory element activity, and merge multi-modality epigenomic features from TR and the queried gene sets for inferring regulator activity. Experimental results on 1072 TR-related datasets demonstrate that TRAPT outperforms current state-of-the-art methods in predicting transcriptional regulators, especially in the prediction of transcription co-factors and chromatin regulators. Additionally, we have successfully identified key transcriptional regulators associated with the disease, genetic variation, cell fate decisions, and tissues. Our method provides an innovative perspective for integrating epigenomic data and has the potential to significantly assist researchers in deepening their understanding of gene expression regulation mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f37165b52d3abae6fd8c8c64ca21b2746c89a7e" target='_blank'>
              TRAPT: A multi-stage fused deep learning framework for transcriptional regulators prediction via integrating large-scale epigenomic data
              </a>
            </td>
          <td>
            Guorui Zhang, Chao Song, Mingxue Yin, Liyuan Liu, , Ye Li, Jianing Zhang, Maozu Guo, Chunquan Li
          </td>
          <td>2024-05-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Deep generative models, such as generative adversarial networks (GANs), have been employed for $de~novo$ molecular generation in drug discovery. Most prior studies have utilized reinforcement learning (RL) algorithms, particularly Monte Carlo tree search (MCTS), to handle the discrete nature of molecular representations in GANs. However, due to the inherent instability in training GANs and RL models, along with the high computational cost associated with MCTS sampling, MCTS RL-based GANs struggle to scale to large chemical databases. To tackle these challenges, this study introduces a novel GAN based on actor-critic RL with instant and global rewards, called InstGAN, to generate molecules at the token-level with multi-property optimization. Furthermore, maximized information entropy is leveraged to alleviate the mode collapse. The experimental results demonstrate that InstGAN outperforms other baselines, achieves comparable performance to state-of-the-art models, and efficiently generates molecules with multi-property optimization. The source code will be released upon acceptance of the paper.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7845d87a866850e61b73c648beaa1df4f83e7ce" target='_blank'>
              Molecular Generative Adversarial Network with Multi-Property Optimization
              </a>
            </td>
          <td>
            Huidong Tang, Chen Li, Sayaka Kamei, Yoshihiro Yamanishi, Yasuhiko Morimoto
          </td>
          <td>2024-03-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Complex diseases do not always follow gradual progressions. Instead, they may experience sudden shifts known as critical states or tipping points, where a marked qualitative change occurs. Detecting such a pivotal transition or pre-deterioration state holds paramount importance due to its association with severe disease deterioration. Nevertheless, the task of pinpointing the pre-deterioration state for complex diseases remains an obstacle, especially in scenarios involving high-dimensional data with limited samples, where conventional statistical methods frequently prove inadequate. In this study, we introduce an innovative quantitative approach termed sample-specific causality network entropy (SCNE), which infers a sample-specific causality network for each individual and effectively quantifies the dynamic alterations in causal relations among molecules, thereby capturing critical points or pre-deterioration states of complex diseases. We substantiated the accuracy and efficacy of our approach via numerical simulations and by examining various real-world datasets, including single-cell data of epithelial cell deterioration (EPCD) in colorectal cancer, influenza infection data, and three different tumor cases from The Cancer Genome Atlas (TCGA) repositories. Compared to other existing six single-sample methods, our proposed approach exhibits superior performance in identifying critical signals or pre-deterioration states. Additionally, the efficacy of computational findings is underscored by analyzing the functionality of signaling biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2066399908a66ea337f2aaefedc0236ac3edeb8" target='_blank'>
              Uncovering the Pre-Deterioration State during Disease Progression Based on Sample-Specific Causality Network Entropy (SCNE)
              </a>
            </td>
          <td>
            Jiayuan Zhong, Hui Tang, Ziyi Huang, Hua Chai, Fei Ling, Pei Chen, Rui Liu
          </td>
          <td>2024-04-08</td>
          <td>Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Today, machine learning methods are widely employed in drug discovery. However, the chronic lack of data continues to hamper their further development, validation, and application. Several modern strategies aim to mitigate the challenges associated with data scarcity by learning from data on related tasks. These knowledge-sharing approaches encompass transfer learning, multitask learning, and meta-learning. A key question remaining to be answered for these approaches is about the extent to which their performance can benefit from the relatedness of available source (training) tasks; in other words, how difficult ("hard") a test task is to a model, given the available source tasks. This study introduces a new method for quantifying and predicting the hardness of a bioactivity prediction task based on its relation to the available training tasks. The approach involves the generation of protein and chemical representations and the calculation of distances between the bioactivity prediction task and the available training tasks. In the example of meta-learning on the FS-Mol data set, we demonstrate that the proposed task hardness metric is inversely correlated with performance (Pearson's correlation coefficient r = -0.72). The metric will be useful in estimating the task-specific gain in performance that can be achieved through meta-learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcfa4dac8e8d06d6ca42757c4b9d06476021e771" target='_blank'>
              Quantifying the Hardness of Bioactivity Prediction Tasks for Transfer Learning.
              </a>
            </td>
          <td>
            Hosein Fooladi, Steffen Hirte, J. Kirchmair
          </td>
          <td>2024-05-13</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af19be9576fba29fc6ce10addc184ed0663d1348" target='_blank'>
              A Computational Predictor for Accurate Identification of Tumor Homing Peptides by Integrating Sequential and Deep BiLSTM Features.
              </a>
            </td>
          <td>
            Roha Arif, Sameera Kanwal, Saeed Ahmed, Muhammad Kabir
          </td>
          <td>2024-05-11</td>
          <td>Interdisciplinary sciences, computational life sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 In the real world, many dynamic behaviors can be explained by the propagation of perturbations, such as the transfer of chemical signals and the spread of infectious diseases. Previous researchers have achieved excellent results in approximating the global propagation time, revealing the mechanism of signal propagation through multiple paths. However, the known frameworks rely on the extension of physical concepts rather than mathematically rigorous derivations. As a result, they may not perfectly predict time or explain the underlying physical significance in certain specific cases. In this paper, we propose a novel method for decomposing network topology, focusing on two modules: the tree-like module and the path-module. Subsequently, we introduce a new framework for signal propagation analysis, which can be applied to estimate the propagation time for two fundamental global topology modules and provide a rigorous proof for the propagation time in global topology. Compared to previous work, our results are not only more concise, clearly defined, efficient, but also are more powerful in predicting propagation time which outperforms some known results in some cases, for example, biochemical dynamics. Additionally, the proposed framework is based on information transfer pathways, which can be also applied to other physical fields, such as network stability, network controlling and network resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5df6abe3831bf4c489162c2dc445d5e085e2f6f" target='_blank'>
              Information transfer pathways: signal propagation in complex global topologies
              </a>
            </td>
          <td>
            Qitong Hu, Xiao-Dong Zhang
          </td>
          <td>2024-05-20</td>
          <td>Physica Scripta</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A necessary component of understanding vector‐borne disease risk is accurate characterization of the distributions of their vectors. Species distribution models have been successfully applied to data‐rich species but may produce inaccurate results for sparsely documented vectors. In light of global change, vectors that are currently not well‐documented could become increasingly important, requiring tools to predict their distributions. One way to achieve this could be to leverage data on related species to inform the distribution of a sparsely documented vector based on the assumption that the environmental niches of related species are not independent. Relatedly, there is a natural dependence of the spatial distribution of a disease on the spatial dependence of its vector. Here, we propose to exploit these correlations by fitting a hierarchical model jointly to data on multiple vector species and their associated human diseases to improve distribution models of sparsely documented species. To demonstrate this approach, we evaluated the ability of twelve models – which differed in their pooling of data from multiple vector species and inclusion of disease data – to improve distribution estimates of sparsely documented vectors. We assessed our models on two simulated datasets, which allowed us to generalize our results and examine their mechanisms. We found that when the focal species is sparsely documented, incorporating data on related vector species reduces uncertainty and improves accuracy by reducing overfitting. When data on vector species are already incorporated, disease data only marginally improve model performance. However, when data on other vectors are not available, disease data can improve model accuracy and reduce overfitting and uncertainty. We then assessed the approach on empirical data on ticks and tick‐borne diseases in Florida and found that incorporating data on other vector species improved model performance. This study illustrates the value in exploiting correlated data via joint modeling to improve distribution models of data‐limited species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d0e58aa6e54eb2281159329031cdc3591c5f47" target='_blank'>
              Improving distribution models of sparsely documented disease vectors by incorporating information on related species via joint modeling
              </a>
            </td>
          <td>
            Stacy Mowry, Sean Moore, N. Achee, Benedicte Fustec, T. A. Perkins
          </td>
          <td>2024-05-03</td>
          <td>Ecography</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Deciphering the regulatory code of gene expression and interpreting the transcriptional effects of genome variation are critical challenges in human genetics. Modern experimental technologies have resulted in an abundance of data, enabling the development of sequence-based deep learning models that link patterns embedded in DNA to the biochemical and regulatory properties contributing to transcriptional regulation, including modeling epigenetic marks, 3D genome organization, and gene expression, with tissue and cell-type specificity. Such methods can predict the functional consequences of any noncoding variant in the human genome, even rare or never-before-observed variants, and systematically characterize their consequences beyond what is tractable from experiments or quantitative genetics studies alone. Recently, the development and application of interpretability approaches have led to the identification of key sequence patterns contributing to the predicted tasks, providing insights into the underlying biological mechanisms learned and revealing opportunities for improvement in future models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a7bf378adf27fa6309cf06df1f7c3b1f3e020e9" target='_blank'>
              Deep Learning Sequence Models for Transcriptional Regulation.
              </a>
            </td>
          <td>
            Ksenia Sokolova, Kathleen M. Chen, Yun Hao, Jian Zhou, O. Troyanskaya
          </td>
          <td>2024-04-09</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Predicting future events always comes with uncertainty, but traditional non-probabilistic methods cannot distinguish certain from uncertain predictions. In survival analysis, probabilistic methods applied to state-of-the-art solutions in the healthcare and biomedical field are still novel, and their implications have not been fully evaluated. In this paper, we study the benefits of modeling uncertainty in deep neural networks for survival analysis with a focus on prediction and calibration performance. For this, we present a Bayesian deep learning framework that consists of three probabilistic network architectures, which we train by optimizing the Cox partial likelihood and combining input-dependent aleatoric uncertainty together with epistemic uncertainty. This enables us to provide uncertainty estimates as credible intervals when predicting the survival curve or as a probability density function over the predicted median survival times. For our empirical analyses, we evaluated our proposed method on four benchmark datasets and found that our method demonstrates prediction performance comparable to the state-of-the-art based on the concordance index and outperforms all other Cox-based approaches in terms of the mean absolute error. Our work explicitly compares the extent to which different Bayesian approximation techniques differ from each other and improves the prediction over traditional non-probabilistic alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062022270313e044624255314c4ad044d0a6866b" target='_blank'>
              Probabilistic Survival Analysis by Approximate Bayesian Inference of Neural Networks
              </a>
            </td>
          <td>
            Christian Marius Lillelund, Martin Magris, Christian Fischer Pedersen
          </td>
          <td>2024-04-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gene expression programs that establish and maintain specific cellular states are orchestrated through a regulatory network composed of transcription factors, cofactors, and chromatin regulators. Dysregulation of this network can lead to a broad range of diseases by altering normal gene program. This article presents LaGrACE, a novel method designed to estimate dysregulation of gene programs utilizing omics data with clinical information. This approach facilitates grouping of samples exhibiting similar patterns of gene program dysregulation, thereby enhancing the discovery of underlying molecular mechanisms. We rigorously evaluated LaGrACE’s performance using synthetic data, breast cancer and chronic obstructive pulmonary disease (COPD) datasets, and single-cell RNA sequencing (scRNA-seq) datasets. Our findings demonstrate that LaGrACE is exceptionally robust in identifying biologically meaningful and prognostic subtypes. Additionally, it effectively discerns drug-response signals at a single-cell resolution. The COPD analysis revealed a new association between LEF1 and COPD molecular mechanisms and mortality. Collectively, these results underscore the utility of LaGrACE as a valuable tool for elucidating disease mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2d4a72bbe23e694e15804f601164485fcfac620" target='_blank'>
              LaGrACE: Estimating gene program dysregulation using latent gene regulatory network for biomedical discovery
              </a>
            </td>
          <td>
            Minxue Jia, Haiyi Mao, Mengli Zhou, Yu-Chih Chen, P. Benos
          </td>
          <td>2024-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e788d3b5198f57c0667d4a517aabe2709daae99" target='_blank'>
              Essentiality, protein–protein interactions and evolutionary properties are key predictors for identifying cancer-associated genes using machine learning
              </a>
            </td>
          <td>
            Amro Safadi, Simon C. Lovell, Andrew J. Doig
          </td>
          <td>2024-04-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78d12260785531902ba470dece593699953a398" target='_blank'>
              Image-based biomarkers for engineering neuroblastoma patient-specific computational models
              </a>
            </td>
          <td>
            S. Hervas-Raluy, D. Sainz-DeMena, M. Gómez-Benito, J. García-Aznar
          </td>
          <td>2024-04-10</td>
          <td>Engineering with Computers</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dabd071ab522df047a3703298ac2d13e4bb72edc" target='_blank'>
              Global ranking of the sensitivity of interaction potential contributions within classical molecular dynamics force fields
              </a>
            </td>
          <td>
            W. Edeling, M. Vassaux, Yiming Yang, S. Wan, Serge Guillas, Peter V. Coveney
          </td>
          <td>2024-05-03</td>
          <td>npj Computational Materials</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="Reduced order models based on the transport of a lower dimensional manifold representation of the thermochemical state, such as Principal Component (PC) transport and Machine Learning (ML) techniques, have been developed to reduce the computational cost associated with the Direct Numerical Simulations (DNS) of reactive flows. Both PC transport and ML normally require an abundance of data to exhibit sufficient predictive accuracy, which might not be available due to the prohibitive cost of DNS or experimental data acquisition. To alleviate such difficulties, similar data from an existing dataset or domain (source domain) can be used to train ML models, potentially resulting in adequate predictions in the domain of interest (target domain). This study presents a novel probabilistic transfer learning (TL) framework to enhance the trust in ML models in correctly predicting the thermochemical state in a lower dimensional manifold and a sparse data setting. The framework uses Bayesian neural networks, and autoencoders, to reduce the dimensionality of the state space and diffuse the knowledge from the source to the target domain. The new framework is applied to one-dimensional freely-propagating flame solutions under different data sparsity scenarios. The results reveal that there is an optimal amount of knowledge to be transferred, which depends on the amount of data available in the target domain and the similarity between the domains. TL can reduce the reconstruction error by one order of magnitude for cases with large sparsity. The new framework required 10 times less data for the target domain to reproduce the same error as in the abundant data scenario. Furthermore, comparisons with a state-of-the-art deterministic TL strategy show that the probabilistic method can require four times less data to achieve the same reconstruction error.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b40d43f32053fa66703ccc372c93df9eb12e60d" target='_blank'>
              Probabilistic transfer learning methodology to expedite high fidelity simulation of reactive flows
              </a>
            </td>
          <td>
            Bruno S. Soriano, Kisung Jung, T. Echekki, Jacqueline H. Chen, Mohammad Khalil
          </td>
          <td>2024-05-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The prognosis of lung adenocarcinoma (LUAD) is generally poor. Immunotherapy has emerged as a promising therapeutic modality, demonstrating remarkable potential for substantially prolonging the overall survival of individuals afflicted with LUAD. However, there is currently a lack of reliable signatures for identifying patients who would benefit from immunotherapy. We conducted a comparative analysis of two immunotherapy cohorts (OAK and POPLAR) and utilized single-factor COX regression to identify genes that significantly impact the prognosis of LUAD. Based on the TCGA-LUAD dataset, we employed a combination of 101 machine learning algorithms to construct a model and selected the optimal model. The model was validated on five GEO datasets and compared with 144 previously published signatures to assess its performance. Subsequently, we explored the underlying biological mechanisms through tumor mutation burden analysis, enrichment analysis, and immune infiltration analysis. An immunotherapy prognostic prediction signature (IPPS) was constructed based on 13 genes, showing robust performance in the TCGA-LUAD dataset. IPPS exhibited consistent predictive accuracy in the validation cohorts. Compared to 144 previously published signatures, IPPS consistently ranked among the top in terms of C-index values. Further exploration revealed differences between high and low-IPPS groups in terms of tumor mutation burden, pathway enrichment, and immune infiltration. IPPS demonstrates strong predictive capabilities for the prognosis of LUAD patients, offering the potential to identify suitable candidates for immunotherapy and contribute to precision treatment strategies for LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb9de6d5b40bb38175765c3f8212aeefaa91a6e" target='_blank'>
              Advancing predictive markers in lung adenocarcinoma: A machine learning-based immunotherapy prognostic prediction signature.
              </a>
            </td>
          <td>
            Zhongyan Li, Shengbin Pei, Yanjuan Wang, Ge Zhang, Haoran Lin, Shiyang Dong
          </td>
          <td>2024-04-09</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63fd7f95a4c30286acfdb94327aae7f75b5d36ef" target='_blank'>
              Challenges as catalysts: how Waymo’s Open Dataset Challenges shape AI development
              </a>
            </td>
          <td>
            Sam Hind, Fernando N. van der Vlist, Max Kanderske
          </td>
          <td>2024-04-17</td>
          <td>AI &amp; SOCIETY</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Molecular property prediction has gained substantial attention due to its potential for various bio-chemical applications. Numerous attempts have been made to enhance the performance by combining multiple molecular representations (1D, 2D, and 3D). However, most prior works only merged a limited number of representations or tried to embed multiple representations through a single network without using representation-specific networks. Furthermore, the heterogeneous characteristics of each representation made the fusion more challenging. Addressing these challenges, we introduce the Fusion Transformer for Multiple Molecular Representations (FTMMR) framework. Our strategy employs three distinct representation-specific networks and integrates information from each network using a fusion transformer architecture to generate fused representations. Additionally, we use self-supervised learning methods to align heterogeneous representations and to effectively utilize the limited chemical data available. In particular, we adopt a combinatorial loss function to leverage the contrastive loss for all three representations. We evaluate the performance of FTMMR using seven benchmark datasets, demonstrating that our framework outperforms existing fusion and self-supervised methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d7a915a21c085446afc3cd21611e1c4f111e16f" target='_blank'>
              FTMMR: Fusion Transformer for Integrating Multiple Molecular Representations.
              </a>
            </td>
          <td>
            Young-Han Son, Dong-Hee Shin, Tae-Eui Kam
          </td>
          <td>2024-03-29</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="This paper introduces a novel hybrid mathematical modeling approach that effectively couples Partial Differential Equations (PDEs) with Ordinary Differential Equations (ODEs), exemplified through the simulation of epidemiological processes. The hybrid model aims to integrate the spatially detailed representation of disease dynamics provided by PDEs with the computational efficiency of ODEs. In the presented epidemiological use-case, this integration allows for the rapid assessment of public health interventions and the potential impact of infectious diseases across large populations. We discuss the theoretical formulation of the hybrid PDE-ODE model, including the governing equations and boundary conditions. The model's capabilities are demonstrated through detailed simulations of disease spread in synthetic environments and real-world scenarios, specifically focusing on the regions of Lombardy, Italy, and Berlin, Germany. Results indicate that the hybrid model achieves a balance between computational speed and accuracy, making it a valuable tool for policymakers in real-time decision-making and scenario analysis in epidemiology and potentially in other fields requiring similar modeling approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c7722dca0ec03e3157013ae8cb1c2bc49ead53d" target='_blank'>
              Hybrid PDE-ODE Models for Efficient Simulation of Infection Spread in Epidemiology
              </a>
            </td>
          <td>
            Kristina Maier, Martin Weiser, Tim Conrad
          </td>
          <td>2024-05-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract WGCNA (weighted gene co-expression network analysis) is a very useful tool for identifying co-expressed gene modules and detecting their correlations to phenotypic traits. Here, we explored more possibilities about it and developed the R package CWGCNA (causal WGCNA), which works from the traditional WGCNA pipeline but mines more information. It couples a mediation model with WGCNA, so the causal relationships among WGCNA modules, module features, and phenotypes can be found, demonstrating whether the module change causes the phenotype change or vice versa. After that, when annotating the module gene set functions, it uses a novel network-based method, considering the modules' topological structures and capturing their influence on the gene set functions. In addition to conducting these biological explorations, CWGCNA also contains a machine learning section to perform clustering and classification on multi-omics data, given the increasing popularity of this data type. Some basic functions, such as differential feature identification, are also available in our package. Its effectiveness is proved by the performance on three single or multi-omics datasets, showing better performance than existing methods. CWGCNA is available at: https://github.com/yuabrahamliu/CWGCNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b244930586cbadefd75917d67c791912b6b3e315" target='_blank'>
              CWGCNA: an R package to perform causal inference from the WGCNA framework
              </a>
            </td>
          <td>
            Yu Liu
          </td>
          <td>2024-04-25</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96ce73317e29a3050795d9cfd776ded7c591953a" target='_blank'>
              Understanding and Quantifying Network Robustness to Stochastic Inputs
              </a>
            </td>
          <td>
            Hwai-Ray Tung, S. Lawley
          </td>
          <td>2024-04-12</td>
          <td>Bulletin of Mathematical Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Identifying partial differential equations (PDEs) from data is crucial for understanding the governing mechanisms of natural phenomena, yet it remains a challenging task. We present an extension to the ARGOS framework, ARGOS-RAL, which leverages sparse regression with the recurrent adaptive lasso to identify PDEs from limited prior knowledge automatically. Our method automates calculating partial derivatives, constructing a candidate library, and estimating a sparse model. We rigorously evaluate the performance of ARGOS-RAL in identifying canonical PDEs under various noise levels and sample sizes, demonstrating its robustness in handling noisy and non-uniformly distributed data. We also test the algorithm's performance on datasets consisting solely of random noise to simulate scenarios with severely compromised data quality. Our results show that ARGOS-RAL effectively and reliably identifies the underlying PDEs from data, outperforming the sequential threshold ridge regression method in most cases. We highlight the potential of combining statistical methods, machine learning, and dynamical systems theory to automatically discover governing equations from collected data, streamlining the scientific modeling process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6ba94461e38ca92209eaa1e802d6a39c777186" target='_blank'>
              Automating the Discovery of Partial Differential Equations in Dynamical Systems
              </a>
            </td>
          <td>
            Weizhen Li, Rui Carvalho
          </td>
          <td>2024-04-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Precisely predicting molecular properties is crucial in drug discovery, but the scarcity of labeled data poses a challenge for applying deep learning methods. While large-scale self-supervised pretraining has proven an effective solution, it often neglects domain-specific knowledge. To tackle this issue, we introduce Task-Oriented Multilevel Learning based on BERT (TOML-BERT), a dual-level pretraining framework that considers both structural patterns and domain knowledge of molecules. TOML-BERT achieved state-of-the-art prediction performance on 10 pharmaceutical datasets. It has the capability to mine contextual information within molecular structures and extract domain knowledge from massive pseudo-labeled data. The dual-level pretraining accomplished significant positive transfer, with its two components making complementary contributions. Interpretive analysis elucidated that the effectiveness of the dual-level pretraining lies in the prior learning of a task-related molecular representation. Overall, TOML-BERT demonstrates the potential of combining multiple pretraining tasks to extract task-oriented knowledge, advancing molecular property prediction in drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f91cd994a0476b1e1c720d21fe7f5ceed94b2fb" target='_blank'>
              Enhancing Molecular Property Prediction through Task-Oriented Transfer Learning: Integrating Universal Structural Insights and Domain-Specific Knowledge.
              </a>
            </td>
          <td>
            Yanjing Duan, Xixi Yang, Xiangxiang Zeng, Wenxuan Wang, Youchao Deng, Dongsheng Cao
          </td>
          <td>2024-05-15</td>
          <td>Journal of medicinal chemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="A comprehensive and reliable survival prediction model is of great importance to assist in the personalized management of Head and Neck Cancer (HNC) patient treated with curative Radiation Therapy (RT). In this work, we propose IMLSP, an Interpretable Multi-Label multi-modal deep Survival Prediction framework for predicting multiple HNC survival outcomes simultaneously and provide time-event specific visual explanation of the deep prediction process. We adopt Multi-Task Logistic Regression (MTLR) layers to convert survival prediction from a regression problem to a multi-time point classification task, and to enable predicting of multiple relevant survival outcomes at the same time. We also present Grad-Team, a Gradient-weighted Time-event activation mapping approach specifically developed for deep survival model visual explanation, to generate patient-specific time-to-event activation maps. We evaluate our method with the publicly available RADCURE HNC dataset, where it outperforms the corresponding single-modal models and single-label models on all survival outcomes. The generated activation maps show that the model focuses primarily on the tumor and nodal volumes when making the decision and the volume of interest varies for high- and low-risk patients. We demonstrate that the multi-label learning strategy can improve the learning efficiency and prognostic performance, while the interpretable survival prediction model is promising to help understand the decision-making process of AI and facilitate personalized treatment. The project website can be found at https://github.com/***.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c111a556e7fc68ec3b6b23ca5e10e1c255f8bd39" target='_blank'>
              Advancing Head and Neck Cancer Survival Prediction via Multi-Label Learning and Deep Model Interpretation
              </a>
            </td>
          <td>
            Meixu Chen, Kai Wang, Jing Wang
          </td>
          <td>2024-05-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="This news section offers Cancer readers timely information on events, public policy analysis, topical issues, and personalities. In this issue, a dynamic risk prediction model for multiple myeloma is discussed. Through deep learning, the model is able to adapt to future data, such as emerging treatment modalities and combinations. In addition, global phase 3 results show that bevacizumab with immunotherapy has activity in liver cancer, and D‐VRd is examined as a new standard of care in patients with multiple myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b64f34200af5443cad0d6f7912e702ed6cd6c03" target='_blank'>
              Dynamic risk prediction model for multiple myeloma: Through deep learning, the model is able to adapt to future data, such as emerging treatment modalities and combinations.
              </a>
            </td>
          <td>
            M. Nierengarten
          </td>
          <td>2024-04-11</td>
          <td>Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="To support mechanism online learning and facilitate digital twin development for biomanufacturing processes, this paper develops an efficient Bayesian inference approach for partially observed enzymatic stochastic reaction network (SRN), a fundamental building block of multi-scale bioprocess mechanistic model. To tackle the critical challenges brought by the nonlinear stochastic differential equations (SDEs)-based mechanistic model with partially observed state and having measurement error, an interpretable Bayesian updating linear noise approximation (LNA) metamodel, incorporating the structure information of the mechanistic model, is proposed to approximate the likelihood of observations. Then, an efficient posterior sampling approach is developed by utilizing the gradients of the derived likelihood to speed up the convergence of MCMC. The empirical study demonstrates that the proposed approach has a promising performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57d6d6543be531495a2c396519431852457ac4af" target='_blank'>
              Linear Noise Approximation Assisted Bayesian Inference on Mechanistic Model of Partially Observed Stochastic Reaction Network
              </a>
            </td>
          <td>
            Wandi Xu, Wei Xie
          </td>
          <td>2024-05-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Time-series experiments are crucial for understanding the transient and dynamic nature of biological phenomena. These experiments, leveraging advanced classification and clustering algorithms, allow for a deep dive into the cellular processes. However, while these approaches effectively identify patterns and trends within data, they often need to improve in elucidating the causal mechanisms behind these changes. Building on this foundation, our study introduces a novel algorithm for temporal causal signaling modeling, integrating established knowledge networks with sequential gene expression data to elucidate signal transduction pathways over time. Focusing on Escherichia coli’s (E. coli) aerobic to anaerobic transition (AAT), this research marks a significant leap in understanding the organism’s metabolic shifts. By applying our algorithm to a comprehensive E. coli regulatory network and a time-series microarray dataset, we constructed the cross-time point core signaling and regulatory processes of E. coli’s AAT. Through gene expression analysis, we validated the primary regulatory interactions governing this process. We identified a novel regulatory scheme wherein environmentally responsive genes, soxR and oxyR, activate fur, modulating the nitrogen metabolism regulators fnr and nac. This regulatory cascade controls the stress regulators ompR and lrhA, ultimately affecting the cell motility gene flhD, unveiling a novel regulatory axis that elucidates the complex regulatory dynamics during the AAT process. Our approach, merging empirical data with prior knowledge, represents a significant advance in modeling cellular signaling processes, offering a deeper understanding of microbial physiology and its applications in biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03f1ae2c4960a846a7510e092a37140e059d408" target='_blank'>
              A Causal Regulation Modeling Algorithm for Temporal Events with Application to Escherichia coli’s Aerobic to Anaerobic Transition
              </a>
            </td>
          <td>
            Yigang Chen, Runbo Mao, Jiatong Xu, Yixian Huang, , Shidong Cui, Zihao Zhu, , Shenghan Huang, , Hsi-Yuan Huang, Shih-Chung Yen, Yang-Chi-Dung Lin, Hsien-Da Huang
          </td>
          <td>2024-05-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Nonlinear differential equations are encountered as models of fluid flow, spiking neurons, and many other systems of interest in the real world. Common features of these systems are that their behaviors are difficult to describe exactly and invariably unmodeled dynamics present challenges in making precise predictions. In many cases the models exhibit extremely complicated behavior due to bifurcations and chaotic regimes. In this paper, we present a novel data-driven linear estimator that uses Koopman operator theory to extract finite-dimensional representations of complex nonlinear systems. The extracted model is used together with a deep reinforcement learning network that learns the optimal stepwise actions to predict future states of the original nonlinear system. Our estimator is also adaptive to a diffeomorphic transformation of the nonlinear system which enables transfer learning to compute state estimates of the transformed system without relearning from scratch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253489dec37ed05e844568d9dae4237b151b936f" target='_blank'>
              Koopman-based Deep Learning for Nonlinear System Estimation
              </a>
            </td>
          <td>
            Zexin Sun, Mingyu Chen, John Baillieul
          </td>
          <td>2024-05-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Techniques to autonomously drive research have been prominent in Computational Scientific Discovery, while Synthetic Biology is a field of science that focuses on designing and constructing new biological systems for useful purposes. Here we seek to apply logic-based machine learning techniques to facilitate cellular engineering and drive biological discovery. Comprehensive databases of metabolic processes called genome-scale metabolic network models (GEMs) are often used to evaluate cellular engineering strategies to optimise target compound production. However, predicted host behaviours are not always correctly described by GEMs, often due to errors in the models. The task of learning the intricate genetic interactions within GEMs presents computational and empirical challenges. To address these, we describe a novel approach called Boolean Matrix Logic Programming (BMLP) by leveraging boolean matrices to evaluate large logic programs. We introduce a new system, $BMLP_{active}$, which efficiently explores the genomic hypothesis space by guiding informative experimentation through active learning. In contrast to sub-symbolic methods, $BMLP_{active}$ encodes a state-of-the-art GEM of a widely accepted bacterial host in an interpretable and logical representation using datalog logic programs. Notably, $BMLP_{active}$ can successfully learn the interaction between a gene pair with fewer training examples than random experimentation, overcoming the increase in experimental design space. $BMLP_{active}$ enables rapid optimisation of metabolic models to reliably engineer biological systems for producing useful compounds. It offers a realistic approach to creating a self-driving lab for microbial engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4217fa5bc20619984d78d30c100e2b19faa0091" target='_blank'>
              Boolean matrix logic programming for active learning of gene functions in genome-scale metabolic network models
              </a>
            </td>
          <td>
            L. Ai, Stephen Muggleton, Shishun Liang, Geoff S. Baldwin
          </td>
          <td>2024-05-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5c0096fb17e04bd4be6cf810f7609eeca6aaca3" target='_blank'>
              MolPROP: Molecular Property prediction with multimodal language and graph fusion
              </a>
            </td>
          <td>
            Zachary A Rollins, Alan C Cheng, Essam Metwally
          </td>
          <td>2024-05-22</td>
          <td>Journal of Cheminformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Our paper deals with a collection of networks on a common set of nodes, where some of the networks are associated with responses. Assuming that the networks correspond to points on a one-dimensional manifold in a higher dimensional ambient space, we propose an algorithm to consistently predict the response at an unlabeled network. Our model involves a specific multiple random network model, namely the common subspace independent edge model, where the networks share a common invariant subspace, and the heterogeneity amongst the networks is captured by a set of low dimensional matrices. Our algorithm estimates these low dimensional matrices that capture the heterogeneity of the networks, learns the underlying manifold by isomap, and consistently predicts the response at an unlabeled network. We provide theoretical justifications for the use of our algorithm, validated by numerical simulations. Finally, we demonstrate the use of our algorithm on larval Drosophila connectome data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb592d6fdf3ce2b11fe06b667a08556edadacce2" target='_blank'>
              Consistent response prediction for multilayer networks on unknown manifolds
              </a>
            </td>
          <td>
            Aranyak Acharyya, Jes'us Arroyo Reli'on, Michael Clayton, Marta Zlatic, Youngser Park, Carey E. Priebe
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Deep learning, a crucial technique for achieving artificial intelligence (AI), has been successfully applied in many fields. The gradual application of the latest architectures of deep learning in the field of time series forecasting (TSF), such as Transformers, has shown excellent performance and results compared to traditional statistical methods. These applications are widely present in academia and in our daily lives, covering many areas including forecasting electricity consumption in power systems, meteorological rainfall, traffic flow, quantitative trading, risk control in finance, sales operations and price predictions for commercial companies, and pandemic prediction in the medical field. Deep learning-based TSF tasks stand out as one of the most valuable AI scenarios for research, playing an important role in explaining complex real-world phenomena. However, deep learning models still face challenges: they need to deal with the challenge of large-scale data in the information age, achieve longer forecasting ranges, reduce excessively high computational complexity, etc. Therefore, novel methods and more effective solutions are essential. In this paper, we review the latest developments in deep learning for TSF. We begin by introducing the recent development trends in the field of TSF and then propose a new taxonomy from the perspective of deep neural network models, comprehensively covering articles published over the past five years. We also organize commonly used experimental evaluation metrics and datasets. Finally, we point out current issues with the existing solutions and suggest promising future directions in the field of deep learning combined with TSF. This paper is the most comprehensive review related to TSF in recent years and will provide a detailed index for researchers in this field and those who are just starting out.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d93c9854b0ec15d7272319e91ee9c2b294327e49" target='_blank'>
              Deep Time Series Forecasting Models: A Comprehensive Survey
              </a>
            </td>
          <td>
            Xinhe Liu, Wenmin Wang
          </td>
          <td>2024-05-11</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Complex systems are prevalent in various disciplines encompassing the natural and social sciences, such as physics, biology, economics, and sociology. Leveraging data science techniques, particularly those rooted in artificial intelligence and machine learning, offers a promising avenue for comprehending the intricacies of complex systems without necessitating detailed knowledge of underlying dynamics. In this paper, we demonstrate that multiscale entropy (MSE) is pivotal in describing the steady state of complex systems. Introducing the multiscale entropy dynamics (MED) methodology, we provide a framework for dissecting system dynamics and uncovering the driving forces behind their evolution. Our investigation reveals that the MED methodology facilitates the expression of complex system dynamics through a Generalized Nonlinear Schrödinger Equation (GNSE) that thus demonstrates its potential applicability across diverse complex systems. By elucidating the entropic underpinnings of complexity, our study paves the way for a deeper understanding of dynamic phenomena. It offers insights into the behavior of complex systems across various domains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7dc6f13dbf0858892b1c74138d4646a809e932e" target='_blank'>
              Major Role of Multiscale Entropy Evolution in Complex Systems and Data Science
              </a>
            </td>
          <td>
            Shahid Nawaz, Muhammad Saleem, Fedor V. Kusmartsev, D. Anjum
          </td>
          <td>2024-04-01</td>
          <td>Entropy</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="The recently introduced class of architectures known as Neural Operators has emerged as highly versatile tools applicable to a wide range of tasks in the field of Scientific Machine Learning (SciML), including data representation and forecasting. In this study, we investigate the capabilities of Neural Implicit Flow (NIF), a recently developed mesh-agnostic neural operator, for representing the latent dynamics of canonical systems such as the Kuramoto-Sivashinsky (KS), forced Korteweg-de Vries (fKdV), and Sine-Gordon (SG) equations, as well as for extracting dynamically relevant information from them. Finally we assess the applicability of NIF as a dimensionality reduction algorithm and conduct a comparative analysis with another widely recognized family of neural operators, known as Deep Operator Networks (DeepONets).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2ea0dbc681d0e289d33f89b612d08a007f2fc6" target='_blank'>
              Using Neural Implicit Flow To Represent Latent Dynamics Of Canonical Systems
              </a>
            </td>
          <td>
            Imran Nasim, Joao Lucas de Sousa Almeida
          </td>
          <td>2024-04-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Contemporary radiation risk assessment predominantly depends on nonlinear parametric models, which typically include a baseline term, a dose-response term, and an effect modifier term. Despite their widespread application in estimating tumor risks, parametric models face a notable drawback: their rigid model structure can be overly restrictive, potentially introducing bias and inaccuracies into risk estimations. In this study, we analyze data on solid tumors and leukemia from the Life Span Study (LSS) to compare the performance of deep neural network (DNN) and nonlinear parametric (NLP) models in assessing ERRs. DNN presents novel perspectives for radiation risk assessment. Our findings indicate that DNN can perform better than the traditional parametric models. Even if DNN and NLP models exhibit similar performance in predicting tumor incidence, they diverge significantly in their estimated ERRs. Standard NLP models tend to underestimate ERRs directly linked to radiation dose, overestimate ERRs for individuals at younger attained ages and ages at exposure, and underestimate ERRs for those at older attained ages. Furthermore, DNN consistently identifies radiation dose as the primary and predominant risk factor for ERRs in leukemia and solid tumors, underscoring the critical role of radiation dose in risk assessment. The insights from DNN could enhance low-dose radiation risk assessment and improve parametric model development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c35286a4ae224457c64400796ef637896de940" target='_blank'>
              Novel Insights for Radiation Risk Assessment Unveiled by Deep Learning
              </a>
            </td>
          <td>
            Z. Liu, I. Shuryak, D. J. Brenner, R. L. Ullrich
          </td>
          <td>2024-04-30</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a974dfa919317837329608cfde8bcf1edf73233" target='_blank'>
              Edge-based relative entropy as a sensitive indicator of critical transitions in biological systems
              </a>
            </td>
          <td>
            Renhao Hong, Yuyan Tong, Huisheng Liu, Pei Chen, Rui Liu
          </td>
          <td>2024-04-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background. Individualized treatment decisions for patients with multiple myeloma (MM) requires accurate risk stratification that takes into account patient-specific consequences of genetic abnormalities and tumor microenvironment on disease outcome and therapy responsiveness. Methods. Previously, SYstems Genetic Network AnaLysis (SYGNAL) of multi-omics tumor profiles from 881 MM patients generated the mmSYGNAL network, which uncovered different causal and mechanistic drivers of genetic programs associated with disease progression across MM subtypes. Here, we have trained a machine learning (ML) algorithm on activities of mmSYGNAL programs within individual patient tumor samples to develop a risk classification scheme for MM that significantly outperformed cytogenetics, International Staging System, and multi-gene biomarker panels in predicting risk of PFS across four independent patient cohorts. Results. We demonstrate that, unlike other tests, mmSYGNAL can accurately predict disease progression risk at primary diagnosis, pre- and post-transplant and even after multiple relapses, making it useful for individualized dynamic risk assessment throughout the disease trajectory. Conclusion. mmSYGNAL provides improved individualized risk stratification that accounts for a patient's distinct set of genetic abnormalities and can monitor risk longitudinally as each patient's disease characteristics change.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cba3f9524bd02b07c52db6fe11f71cc70c07bfd" target='_blank'>
              Individualized dynamic risk assessment for multiple myeloma
              </a>
            </td>
          <td>
            C. Murie, S. Turkarslan, Anoop Patel, David G. Coffey, Pamela S. Becker, N. Baliga
          </td>
          <td>2024-04-03</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Lung cancer remains a significant health threat with high mortality rates. Using graph theory, we modeled the protein-protein interaction (PPI) network in lung cancer to explore its complex structure. This approach allows for the analysis of network properties and the identification of key proteins driving biological processes. Our analysis revealed RPS27A as a central proteinwithin the network, associated with diverse functions related to ribosome biogenesis, translation, cell growth, apoptosis, and cancer progression. This suggests that RPS27A may have multiple functions in cancer development and progression, including in MAPK signaling pathways. Importantly, our study uniquely identifies RPS27A as a central hub in lung cancer PPI networks, shedding light on its pivotal role in disease pathogenesis. Additionally, we identified a central network zone enriched with proteins involved in key signaling pathways, presenting novel insights into potential therapeutic targets for lung cancer treatment. Pathway enrichment analysis further highlighted functional specialization across network zones, providing a comprehensive understanding of the intricate interplay between biological pathways in lung cancer progression. This study underscores the multifaceted roles of central proteins like RPS27A within lung cancer’s PPI network and the network’s potential for pinpointing therapeutic targets, presenting a novel perspective on the intricate network of molecular interactions driving lung cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9bd1f3e57677944edc85a7db870bb3f72fa3cb5" target='_blank'>
              Exploring Lung Cancer Protein Network: Understanding Structure and Function through Metric Space Modeling
              </a>
            </td>
          <td>
            Emad Fadhal
          </td>
          <td>2024-04-30</td>
          <td>European Journal of Pure and Applied Mathematics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Explaining deep learning models operating on time series data is crucial in various applications of interest which require interpretable and transparent insights from time series signals. In this work, we investigate this problem from an information theoretic perspective and show that most existing measures of explainability may suffer from trivial solutions and distributional shift issues. To address these issues, we introduce a simple yet practical objective function for time series explainable learning. The design of the objective function builds upon the principle of information bottleneck (IB), and modifies the IB objective function to avoid trivial solutions and distributional shift issues. We further present TimeX++, a novel explanation framework that leverages a parametric network to produce explanation-embedded instances that are both in-distributed and label-preserving. We evaluate TimeX++ on both synthetic and real-world datasets comparing its performance against leading baselines, and validate its practical efficacy through case studies in a real-world environmental application. Quantitative and qualitative evaluations show that TimeX++ outperforms baselines across all datasets, demonstrating a substantial improvement in explanation quality for time series data. The source code is available at \url{https://github.com/zichuan-liu/TimeXplusplus}.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41331316e9b6cf30546793e0963a8a78d8691c86" target='_blank'>
              TimeX++: Learning Time-Series Explanations with Information Bottleneck
              </a>
            </td>
          <td>
            Zichuan Liu, Tianchun Wang, Jimeng Shi, Xu Zheng, Zhuomin Chen, Lei Song, Wenqian Dong, J. Obeysekera, Farhad Shirani, Dongsheng Luo
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Allostery is a fundamental mechanism driving biomolecular processes that holds significant therapeutic concern. Our study rigorously investigates how two distinct machine-learning algorithms uniquely classify two already close-to-active DFG-in states of TAK1, differing just by the presence or absence of its allosteric activator TAB1, from an ensemble mixture of conformations (obtained from 2.4 μs molecular dynamics (MD) simulations). The novelty, however, lies in understanding the deeper algorithmic potentials to systematically derive a diverse set of differential residue connectivity features that reconstruct the essential mechanistic architecture for TAK1-TAB1 allostery in such a close-to-active biochemical scenario. While the recursive, random forest-based workflow displays the potential of conducting discretized, hierarchical derivation of allosteric features, a multilayer perceptron-based approach gains considerable efficacy in revealing fluid connected patterns of features when hybridized with mutual information scoring. Interestingly, both pipelines benchmark similar directions of functional conformational changes for TAK1's activation. The findings significantly advance the depth of mechanistic understanding by highlighting crucial activation signatures along a directed C-lobe → activation loop → ATP pocket channel of information flow, including (1) the αF-αE biterminal alignments and (2) the "catalytic" drift of the activation loop toward kinase active site. Besides, some novel allosteric hotspots (K253, Y206, N189, etc.) are further recognized as TAB1 sensors, transducers, and responders, including a benchmark E70 mutation site, precisely mapping the important structural segments for sequential allosteric execution. Hence, our work demonstrates how to navigate through greater structural depths and dimensions of dynamic allosteric machineries just by leveraging standard ML methods in suitable streamlined workflows adaptive to the specific system and objectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e656737754f86ab9ba75a485691b12e7213422c" target='_blank'>
              Adaptive Workflows of Machine Learning Illuminate the Sequential Operation Mechanism of the TAK1's Allosteric Network.
              </a>
            </td>
          <td>
            Nibedita Ray Chaudhuri, Shubhra Ghosh Dastidar
          </td>
          <td>2024-05-14</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction While radiotherapy has long been recognized for its ability to directly ablate cancer cells through necrosis or apoptosis, radiotherapy-induced abscopal effect suggests that its impact extends beyond local tumor destruction thanks to immune response. Cellular proliferation and necrosis have been extensively studied using mathematical models that simulate tumor growth, such as Gompertz law, and the radiation effects, such as the linear-quadratic model. However, the effectiveness of radiotherapy-induced immune responses may vary among patients due to individual differences in radiation sensitivity and other factors. Methods We present a novel macroscopic approach designed to quantitatively analyze the intricate dynamics governing the interactions among the immune system, radiotherapy, and tumor progression. Building upon previous research demonstrating the synergistic effects of radiotherapy and immunotherapy in cancer treatment, we provide a comprehensive mathematical framework for understanding the underlying mechanisms driving these interactions. Results Our method leverages macroscopic observations and mathematical modeling to capture the overarching dynamics of this interplay, offering valuable insights for optimizing cancer treatment strategies. One shows that Gompertz law can describe therapy effects with two effective parameters. This result permits quantitative data analyses, which give useful indications for the disease progression and clinical decisions. Discussion Through validation against diverse data sets from the literature, we demonstrate the reliability and versatility of our approach in predicting the time evolution of the disease and assessing the potential efficacy of radiotherapy-immunotherapy combinations. This further supports the promising potential of the abscopal effect, suggesting that in select cases, depending on tumor size, it may confer full efficacy to radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f15cf29cf113e672b18b66c8e810f76f39836894" target='_blank'>
              Mathematical modeling of the synergistic interplay of radiotherapy and immunotherapy in anti-cancer treatments
              </a>
            </td>
          <td>
            Paolo Castorina, Filippo Castiglione, G. Ferini, Stefano Forte, E. Martorana, Dario Giuffrida
          </td>
          <td>2024-05-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Many complicated dynamical events may be broken down into simpler pieces and efficiently described by a system that shifts among a variety of conditionally dynamical modes. Building on switching linear dynamical systems, we develop a new model that extends the switching linear dynamical systems for better discovering these dynamical modes. In the proposed model, the linear dynamics of latent variables can be described by a higher-order vector autoregressive process, which makes it feasible to evaluate the higher-order dependency relationships in the dynamics. In addition, the transition of switching states is determined by a stick-breaking logistic regression, overcoming the limitation of a restricted geometric state duration and recovering the symmetric dependency between the switching states and the latent variables from asymmetric relationships. Furthermore, logistic regression evidence potentials can appear as conditionally Gaussian potentials by utilizing the Pólya-gamma augmentation strategy. Filtering and smoothing algorithms and Bayesian inference for parameter learning in the proposed model are presented. The utility and versatility of the proposed model are demonstrated on synthetic data and public functional magnetic resonance imaging data. Our model improves the current methods for learning the switching linear dynamical modes, which will facilitate the identification and assessment of the dynamics of complex systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f180f44aa0cab4e14882a44aef54b3e4ae757688" target='_blank'>
              Bayesian Inference of Recurrent Switching Linear Dynamical Systems with Higher-Order Dependence
              </a>
            </td>
          <td>
            Houxiang Wang, Jiaqing Chen
          </td>
          <td>2024-04-13</td>
          <td>Symmetry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Uncertainty in parameter estimates from fitting within-host models to empirical data limits the model’s ability to uncover mechanisms of infection, disease progression, and to guide pharmaceutical interventions. Understanding the effect of model structure and data availability on model predictions is important for informing model development and experimental design. To address sources of uncertainty in parameter estimation, we use four mathematical models of influenza A infection with increased degrees of biological realism. We test the ability of each model to reveal its parameters in the presence of unlimited data by performing structural identifiability analyses. We then refine the results by predicting practical identifiability of parameters under daily influenza A virus titers alone or together with daily adaptive immune cell data. Using these approaches, we present insight into the sources of uncertainty in parameter estimation and provide guidelines for the types of model assumptions, optimal experimental design, and biological information needed for improved predictions. Author summary Within-host models of virus infections fitted to data have improved our understanding of mechanisms of infection, disease progression, and allowed us to provide guidelines for pharmaceutical interventions. Given their predictive power, it is essential that we properly uncover and address uncertainty in model predictions and parameter estimation. Here, we focus on the effect of model structure and data availability on our ability to uncover unknown parameters. To address these questions, we use four mathematical models of influenza A infection with increased degrees of biological realism. We test the ability of each model to reveal its parameters in the presence of unlimited data by performing structural identifiability analysis. We then refine the results by predicting practical identifiability of parameters under daily influenza A virus titers alone or together with daily adaptive immune cell data. Using these approaches, we present insights into the sources of uncertainty in parameter estimations and provide guidelines for the types of model assumptions, optimal experimental design, and biological information needed for improved predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08312e1a270ca4ad6a36e75e2b5550f77112dd89" target='_blank'>
              Identifiability investigation of within-host models of acute virus infection
              </a>
            </td>
          <td>
            Yuganthi R. Liyanage, Nora Heitzman-Breen, N. Tuncer, Stanca M. Ciupe
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="We can use computational biology to analyze massive volumes of biological data and identify complicated patterns and relationships. Researchers can acquire insights into fundamental biological processes such as gene regulation, protein folding, and cellular signaling by using computer models and algorithms. This information is critical for furthering our understanding of biological systems. Computational biology is essential for speeding up the drug discovery and development process. Computational biologists can uncover promising drug candidates, forecast efficacy, and optimize their properties by modelling drug-target molecule interactions. This method decreases the time, money, and resources needed for standard experimental drug screening, allowing for the faster discovery of new medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/929d9866fe560c80e62fd7796043b1618824a13e" target='_blank'>
              Computational Biology: Research and Applications
              </a>
            </td>
          <td>
            Dr. Santhoshkumar M
          </td>
          <td>2024-04-06</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Polygenic risk scores (PRSs) are commonly used for predicting an individual’s genetic risk of complex diseases. Yet, their implication for disease pathogenesis remains largely limited. Here, we introduce scPRS, a geometric deep learning model that constructs single-cell-resolved PRS leveraging reference single-cell chromatin accessibility profiling data to enhance biological discovery as well as disease prediction. Real-world applications across multiple complex diseases, including type 2 diabetes (T2D), hypertrophic cardiomyopathy (HCM), and Alzheimer’s disease (AD), showcase the superior prediction power of scPRS compared to traditional PRS methods. Importantly, scPRS not only predicts disease risk but also uncovers disease-relevant cells, such as hormone-high alpha and beta cells for T2D, cardiomyocytes and pericytes for HCM, and astrocytes, microglia and oligodendrocyte progenitor cells for AD. Facilitated by a layered multi-omic analysis, scPRS further identifies cell-type-specific genetic underpinnings, linking disease-associated genetic variants to gene regulation within corresponding cell types. We substantiate the disease relevance of scPRS-prioritized HCM genes and demonstrate that the suppression of these genes in HCM cardiomyocytes is rescued by Mavacamten treatment. Additionally, we establish a novel microglia-specific regulatory relationship between the AD risk variant rs7922621 and its target genes ANXA11 and TSPAN14. We further illustrate the detrimental effects of suppressing these two genes on microglia phagocytosis. Our work provides a multi-tasking, interpretable framework for precise disease prediction and systematic investigation of the genetic, cellular, and molecular basis of complex diseases, laying the methodological foundation for single-cell genetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19087b72fb717b73e856f3e79f8291329571ac56" target='_blank'>
              Deconvolution of polygenic risk score in single cells unravels cellular and molecular heterogeneity of complex human diseases
              </a>
            </td>
          <td>
            Sai Zhang, Hantao Shu, Jingtian Zhou, Jasper Rubin-Sigler, Xiaoyu Yang, Yuxi Liu, J. Cooper-Knock, Emma Monte, Chenchen Zhu, Sharon Tu, Han Li, Mingming Tong, J. Ecker, Justin K. Ichida, Yin Shen, Jianyang Zeng, Philip S. Tsao, Michael P. Snyder
          </td>
          <td>2024-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Advancements in wearable sensor technologies and the digitization of medical records have contributed to the unprecedented ubiquity of biomedical time series data. Data-driven models have tremendous potential to assist clinical diagnosis and improve patient care by improving long-term monitoring capabilities, facilitating early disease detection and intervention, as well as promoting personalized healthcare delivery. However, accessing extensively labeled datasets to train data-hungry deep learning models encounters many barriers, such as long-tail distribution of rare diseases, cost of annotation, privacy and security concerns, data-sharing regulations, and ethical considerations. An emerging approach to overcome the scarcity of labeled data is to augment AI methods with human-like capabilities to leverage past experiences to learn new tasks with limited examples, called few-shot learning. This survey provides a comprehensive review and comparison of few-shot learning methods for biomedical time series applications. The clinical benefits and limitations of such methods are discussed in relation to traditional data-driven approaches. This paper aims to provide insights into the current landscape of few-shot learning for biomedical time series and its implications for future research and applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35712d135083d4d22b2d56f482c5d1d32a14eb41" target='_blank'>
              A Survey of Few-Shot Learning for Biomedical Time Series
              </a>
            </td>
          <td>
            Chenqi Li, Timothy Denison, Tingting Zhu
          </td>
          <td>2024-05-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Epidermal growth factor receptor (EGFR) T790M mutation is the standard predictive biomarker for third-generation epidermal growth factor receptor tyrosine kinase inhibitor (EGFR-TKI) treatment. While not all T790M-positive patients respond to third-generation EGFR-TKIs and have a good prognosis, it necessitates novel tools to supplement EGFR genotype detection for predicting efficacy and stratifying EGFR-mutant patients with various prognoses. Mixture-of-experts (MoE) is designed to disassemble a large model into many small models. Meanwhile, it is also a model ensembling method that can better capture multiple patterns of intrinsic subgroups of enrolled patients. Therefore, the combination of MoE and Cox algorithm has the potential to predict efficacy and stratify survival in non-small cell lung cancer (NSCLC) patients with EGFR mutations. Methods We utilized the electronic medical record (EMR) and pharmacokinetic parameters of 326 T790M-mutated NSCLC patients, including 283 patients treated with Abivertinib in phase I (n=177, for training) and II (n=106, for validation) clinical trials and an additional validation cohort 2 comprising 43 patients treated with BPI-7711. Furthermore, 18 patients underwent whole-exome sequencing for biological interpretation of CoxMoE. We evaluated the predictive performance for therapeutic response using the area under the curve (AUC) and the Concordance index (C-index) for progression-free survival (PFS). Results CoxMoE exhibited AUCs of 0.73–0.83 for predicting efficacy defined by best overall response (BoR) and achieved C-index values of 0.64–0.65 for PFS prediction in training and validating cohorts. The PFS of 198 patients with a low risk [median, 6.0 (range, 1.0–23.3) months in the abivertinib treated cohort; median 16.5 (range, 1.4–27.4) months in BPI-7711 treated cohort] of being non-responder increased by 43% [hazard ratio (HR), 0.56; 95% confidence interval (CI), 0.40–0.78; P=0.0013] and 50% (HR, 0; 95% CI, 0–0; P=0.01) compared to those at high-risk [median, 4.2 (range, 1.0–35) months in the abivertinib treated cohort; median, 11.0 (range, 1.4–25.1) months in BPI-7711 treated cohort]. Additionally, activated partial thromboplastin time (APTT), creatinine clearance (Ccr), monocyte, and steady-state plasma trough concentration utilited to construct model were found significantly associated with drug resistance and aggressive tumor pathways. A robust correlation was observed between APTT and Ccr with PFS (log-rank test; P<0.01) and treatment response (Wilcoxon test; P<0.05), respectively. Conclusions CoxMoE offers a valuable approach for patient selection by forecasting therapeutic response and PFS utilizing laboratory tests and pharmacokinetic parameters in the setting of early-phase clinical trials. Simultaneously, CoxMoE could predict the efficacy of third-generation EGFR-TKI non-invasively for T790M-positive NSCLC patients, thereby complementing existing EGFR genotype detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/971519ff9b0ca1f419778698dab825d127135d1c" target='_blank'>
              Development and validation of a deep learning-based model to predict response and survival of T790M mutant non-small cell lung cancer patients in early clinical phase trials using electronic medical record and pharmacokinetic data
              </a>
            </td>
          <td>
            N. Lou, X. Cui, Xinyuan Lin, Ruyun Gao, Chi Xu, N. Qiao, Ji Jiang, Lu Wang, Weicong Wang, Shanbo Wang, Wei Shen, Xin Zheng, Xiaohong Han
          </td>
          <td>2024-04-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Understanding the dynamics of spreading and diffusion on networks is of critical importance for a variety of processes in real life. However, predicting the temporal evolution of diffusion on networks remains challenging as the process is shaped by network topology, spreading non-linearities, and heterogeneous adaptation behavior. In this study, we propose the spindle vector, a new network topological feature, which shapes nodes according to the distance from the root node. The spindle vector captures the relative order of nodes in diffusion propagation, thus allowing us to approximate the spatiotemporal evolution of diffusion dynamics on networks. The approximation simplifies the detailed connections of node pairs by only focusing on the nodal count within individual layers and the interlayer connections, seeking a compromise between efficiency and complexity. Through experiments on various networks, we show that our method outperforms the state-of-the-art on BA networks with an average improvement of 38.6% on the Mean Absolute Error (MAE). Additionally, the predictive accuracy of our method exhibits a notable convergence with the Pairwise Approximation (PA) approach with the increasing presence of quadrangles and pentagons in WS networks. The new metric provides a general and computationally efficient approach to predict network diffusion problems and is of potential for a large range of network applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed48561e0457971d62b29015c23c30fcddfa8934" target='_blank'>
              The Spindle Approximation of Network Epidemiological Modeling
              </a>
            </td>
          <td>
            Jianhong Mou, Bitao Dai, Suoyi Tan, P. Holme, Sune Lehmann, Fredrik Liljeros, Xin Lu
          </td>
          <td>2024-04-18</td>
          <td>New Journal of Physics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background This study aims to identify precise biomarkers for breast cancer to improve patient outcomes, addressing the limitations of traditional staging in predicting treatment responses. Methods Our analysis encompassed data from over 7,000 breast cancer patients across 14 datasets, which included in-house clinical data and single-cell data from 8 patients (totaling 43,766 cells). We utilized an integrative approach, applying 10 machine learning algorithms in 54 unique combinations to analyze 100 existing breast cancer signatures. Immunohistochemistry assays were performed for empirical validation. The study also investigated potential immunotherapies and chemotherapies. Results Our research identified five consistent glutamine metabolic reprogramming (GMR)-related genes from multi-center cohorts, forming the foundation of a novel GMR-model. This model demonstrated superior accuracy in predicting recurrence and mortality risks compared to existing clinical and molecular features. Patients classified as high-risk by the model exhibited poorer outcomes. IHC validation in 30 patients reinforced these findings, suggesting the model’s broad applicability. Intriguingly, the model indicates a differential therapeutic response: low-risk patients may benefit more from immunotherapy, whereas high-risk patients showed sensitivity to specific chemotherapies like BI-2536 and ispinesib. Conclusions The GMR-model marks a significant leap forward in breast cancer prognosis and the personalization of treatment strategies, offering vital insights for the effective management of diverse breast cancer patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f45d05c5e2b2d87e7018b79a4ae4331c6d799edc" target='_blank'>
              Enhancing breast cancer outcomes with machine learning-driven glutamine metabolic reprogramming signature
              </a>
            </td>
          <td>
            Xukui Li, Xue Li, Bin Yang, Songyang Sun, Shu Wang, Fuxun Yu, Tao Wang
          </td>
          <td>2024-05-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The human body consists of microbiomes associated with the development and prevention of several diseases. These microbial organisms form several complex interactions that are informative to the scientific community for explaining disease progression and prevention. Contrary to the traditional view of the microbiome as a singular, assortative network, we introduce a novel statistical approach using a weighted stochastic infinite block model to analyze the complex community structures within microbial co-occurrence microbial interaction networks. Our model defines connections between microbial taxa using a novel semi-parametric rank-based correlation method on their transformed relative abundances within a fully connected network framework. Employing a Bayesian nonparametric approach, the proposed model effectively clusters taxa into distinct communities while estimating the number of communities. The posterior summary of the taxa community membership is obtained based on the posterior probability matrix, which could naturally solve the label switching problem. Through simulation studies and real-world application to microbiome data from postmenopausal patients with recurrent urinary tract infections, we demonstrate that our method has superior clustering accuracy over alternative approaches. This advancement provides a more nuanced understanding of microbiome organization, with significant implications for disease research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/516bb3541e92619cabaa19a8bb148c2f4d9c3ece" target='_blank'>
              Bayesian modeling of co-occurrence microbial interaction networks
              </a>
            </td>
          <td>
            Tejasv Bedi, Bencong Zhu, Michael L. Neugent, Kevin C. Lutz, N. D. Nisco, Qiwei Li
          </td>
          <td>2024-04-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In the fight against the COVID-19 pandemic, leveraging artificial intelligence to predict disease outcomes from chest radiographic images represents a significant scientific aim. The challenge, however, lies in the scarcity of large, labeled datasets with compatible tasks for training deep learning models without leading to overfitting. Addressing this issue, we introduce a novel multi-dataset multi-task training framework that predicts COVID-19 prognostic outcomes from chest X-rays (CXR) by integrating correlated datasets from disparate sources, distant from conventional multi-task learning approaches, which rely on datasets with multiple and correlated labeling schemes. Our framework hypothesizes that assessing severity scores enhances the model's ability to classify prognostic severity groups, thereby improving its robustness and predictive power. The proposed architecture comprises a deep convolutional network that receives inputs from two publicly available CXR datasets, AIforCOVID for severity prognostic prediction and BRIXIA for severity score assessment, and branches into task-specific fully connected output networks. Moreover, we propose a multi-task loss function, incorporating an indicator function, to exploit multi-dataset integration. The effectiveness and robustness of the proposed approach are demonstrated through significant performance improvements in prognosis classification tasks across 18 different convolutional neural network backbones in different evaluation strategies. This improvement is evident over single-task baselines and standard transfer learning strategies, supported by extensive statistical analysis, showing great application potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e750985ce21017eeaaa59a6118870a154d8bce46" target='_blank'>
              Multi-Dataset Multi-Task Learning for COVID-19 Prognosis
              </a>
            </td>
          <td>
            Filippo Ruffini, L. Tronchin, Zhuoru Wu, Wenting Chen, P. Soda, , V. Guarrasi
          </td>
          <td>2024-05-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The technological capability of artificial intelligence (AI) continues to advance with great strength. Recently, the release of large language models has taken the world by storm with concurrent excitement and concern. As a consequence of their impressive ability and versatility, their provide a potential opportunity for implementation in oncology. Areas of possible application include supporting clinical decision making, education, and contributing to cancer research. Despite the promises that these novel systems can offer, several limitations and barriers challenge their implementation. It is imperative that concerns, such as accountability, data inaccuracy, and data protection, are addressed prior to their integration in oncology. As the progression of artificial intelligence systems continues, new ethical and practical dilemmas will also be approached; thus, the evaluation of these limitations and concerns will be dynamic in nature. This review offers a comprehensive overview of the potential application of large language models in oncology, as well as concerns surrounding their implementation in cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dac62aef7416afb94c60bdc53e18b3ad4f573e6d" target='_blank'>
              Large Language Models in Oncology: Revolution or Cause for Concern?
              </a>
            </td>
          <td>
            Aydin Caglayan, Wojciech Slusarczyk, R. Rabbani, Aruni Ghose, Vasileios Papadopoulos, S. Boussios
          </td>
          <td>2024-03-29</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9345f2f030cb7979426cd2be18c7e1a1fbe44b" target='_blank'>
              Improved nonparametric survival prediction using CoxPH, Random Survival Forest & DeepHit Neural Network
              </a>
            </td>
          <td>
            Naseem Asghar, U. Khalil, Basheer Ahmad, H. Alshanbari, Muhammad Hamraz, Bakhtiyar Ahmad, Dost Muhammad Khan
          </td>
          <td>2024-05-07</td>
          <td>BMC medical informatics and decision making</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The intersection of chemistry and Artificial Intelligence (AI) is an active area of research focused on accelerating scientific discovery. While using large language models (LLMs) with scientific modalities has shown potential, there are significant challenges to address, such as improving training efficiency and dealing with the out-of-distribution problem. Focussing on the task of automated language-molecule translation, we are the first to use state-of-the art (SOTA) human-centric optimisation algorithms in the cross-modal setting, successfully aligning cross-language-molecule modals. We empirically show that we can augment the capabilities of scientific LLMs without the need for extensive data or large models. We conduct experiments using only 10% of the available data to mitigate memorisation effects associated with training large models on extensive datasets. We achieve significant performance gains, surpassing the best benchmark model trained on extensive in-distribution data by a large margin and reach new SOTA levels. Additionally we are the first to propose employing non-linear fusion for mixing cross-modal LLMs which further boosts performance gains without increasing training costs or data needs. Finally, we introduce a fine-grained, domain-agnostic evaluation method to assess hallucination in LLMs and promote responsible use.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e41b8f0b6ffa4742f4fe8a54fd5aa95e5792325" target='_blank'>
              Feedback-aligned Mixed LLMs for Machine Language-Molecule Translation
              </a>
            </td>
          <td>
            Dimitris Gkoumas, M. Liakata
          </td>
          <td>2024-05-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Machine learning (ML) offers promising new approaches to tackle complex problems and has been increasingly adopted in chemical and materials sciences. Broadly speaking, ML models employ generic mathematical functions and attempt to learn essential physics and chemistry from a large amount of data. Consequently, because of the lack of physical or chemical principles in the functional form, the reliability of the predictions is oftentimes not guaranteed, particularly for data far out of distribution. It is critical to quantify the uncertainty in model predictions and understand how the uncertainty propagates to downstream chemical and materials applications. Herein, we review and categorize existing uncertainty quantification (UQ) methods for atomistic ML under a united framework of probabilistic modeling with the aim of elucidating the similarities and differences between them. We also discuss performance metrics to evaluate the calibration, precision, accuracy, and efficiency of the UQ methods and techniques for model recalibration. In addition, we discuss uncertainty propagation (UP) in widely used simulation techniques in chemical and materials science, such as molecular dynamics and microkinetic modeling. We also provide remarks on the challenges and future opportunities of UQ and UP in atomistic ML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/362f043306942684a825aa8b606069d8f4c468b3" target='_blank'>
              Uncertainty Quantification and Propagation in Atomistic Machine Learning
              </a>
            </td>
          <td>
            Jin Dai, Santosh Adhikari, Mingjian Wen
          </td>
          <td>2024-05-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Deep neural networks give us a powerful method to model the training dataset’s relationship between input and output. We can regard that as a complex adaptive system consisting of many artificial neurons that work as an adaptive memory as a whole. The network’s behavior is training dynamics with a feedback loop from the evaluation of the loss function. We already know the training response can be constant or shows power law-like aging in some ideal situations. However, we still have gaps between those findings and other complex phenomena, like network fragility. To fill the gap, we introduce a very simple network and analyze it. We show the training response consists of some different factors based on training stages, activation functions, or training methods. In addition, we show feature space reduction as an effect of stochastic training dynamics, which can result in network fragility. Finally, we discuss some complex phenomena of deep networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe0a5a682bae32fc9da39ee442ef4b9d570ca111" target='_blank'>
              A simple theory for training response of deep neural networks
              </a>
            </td>
          <td>
            Kenichi Nakazato
          </td>
          <td>2024-05-07</td>
          <td>Physica Scripta</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Motivated by the pressing challenges in the digital twin development for biomanufacturing process, we introduce an adjoint sensitivity analysis (SA) approach to expedite the learning of mechanistic model parameters. In this paper, we consider enzymatic stochastic reaction networks representing a multi-scale bioprocess mechanistic model that allows us to integrate disparate data from diverse production processes and leverage the information from existing macro-kinetic and genome-scale models. To support forward prediction and backward reasoning, we develop a convergent adjoint SA algorithm studying how the perturbations of model parameters and inputs (e.g., initial state) propagate through enzymatic reaction networks and impact on output trajectory predictions. This SA can provide a sample efficient and interpretable way to assess the sensitivities between inputs and outputs accounting for their causal dependencies. Our empirical study underscores the resilience of these sensitivities and illuminates a deeper comprehension of the regulatory mechanisms behind bioprocess through sensitivities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa83caf636ea9a9577cbd13662fa8d1e926e34f" target='_blank'>
              Adjoint Sensitivity Analysis on Multi-Scale Bioprocess Stochastic Reaction Network
              </a>
            </td>
          <td>
            Keilung Choy, Wei Xie
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Functional data analysis, which models data as realizations of random functions over a continuum, has emerged as a useful tool for time series data. Often, the goal is to infer the dynamic connections (or time-varying conditional dependencies) among multiple functions or time series. For this task, we propose a dynamic and Bayesian functional graphical model. Our modeling approach prioritizes the careful definition of an appropriate graph to identify both time-invariant and time-varying connectivity patterns. We introduce a novel block-structured sparsity prior paired with a finite basis expansion, which together yield effective shrinkage and graph selection with efficient computations via a Gibbs sampling algorithm. Crucially, the model includes (one or more) graph changepoints, which are learned jointly with all model parameters and incorporate graph dynamics. Simulation studies demonstrate excellent graph selection capabilities, with significant improvements over competing methods. We apply the proposed approach to study of dynamic connectivity patterns of sea surface temperatures in the Pacific Ocean and discovers meaningful edges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5cc6623b051c0f56b325829f2faaeed65f2da4" target='_blank'>
              Bayesian Functional Graphical Models with Change-Point Detection
              </a>
            </td>
          <td>
            Chunshan Liu, Daniel R. Kowal, James Doss-Gollin, Marina Vannucci
          </td>
          <td>2024-05-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="INTRODUCTION
Well-calibrated models for personalized prognostication of patients with gastrointestinal neuroendocrine neoplasms (GINENs) are limited. This study aimed to develop and validate a machine-learning model to predict the survival of patients with GINENs.


METHODS
oblique random survival forest (ORSF) model, Cox proportional hazard risk model, Cox model with least absolute shrinkage and selection operator penalization, CoxBoost, Survival Gradient Boosting Machine, Extreme Gradient Boosting survival regression, DeepHit, DeepSurv, DNNSurv, Logistic-Hazard model, and PC-Hazard model were compared. We further tuned hyperparameters and selected variables for the best-performing ORSF. Then, the final ORSF model was validated.


RESULTS
43444 patients with GINENs were included. The median (interquartile range) survival time was 53 (19-102) months. The ORSF model performed best, in which age, histology, M stage, tumor size, primary tumor site, sex, tumor number, surgery, lymph nodes removed, N stage, race, and grade were ranked as important variables. However, chemotherapy and radiotherapy were not necessary for the ORSF model. The ORSF model had an overall C-index of 0.86 (95% confidence interval, 0.85-0.87). The area under the receiver operation curves at 1-, 3-, 5-, and 10-year were 0.91, 0.89, 0.87, and 0.80, respectively. The decision curve analysis showed superior clinical usefulness of the ORSF model than the American Joint Committee on Cancer Stage. A nomogram and an online tool were given.


CONCLUSION
The machine-learning ORSF model could precisely predict the survival of patients with GINENs, with the ability to identify patients at high risk for death and probably guide clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cf8f29fb5dcb8ce9dcb380c08e117a84d6d2132" target='_blank'>
              Development and Validation of a Novel Machine Learning Model to predict the Survival of Patients with Gastrointestinal Neuroendocrine Neoplasms.
              </a>
            </td>
          <td>
            Si Liu, Yun-Xiang Chen, Bing Dai, Li Chen
          </td>
          <td>2024-05-06</td>
          <td>Neuroendocrinology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In the realm of medicinal chemistry, the primary objective is to swiftly optimize a multitude of chemical properties of a set of compounds to yield a clinical candidate poised for clinical trials. In recent years, two computational techniques, machine learning (ML) and physics-based methods, have evolved substantially and are now frequently incorporated into the medicinal chemist’s toolbox to enhance the efficiency of both hit optimization and candidate design. Both computational methods come with their own set of limitations, and they are often used independently of each other. ML’s capability to screen extensive compound libraries expediently is tempered by its reliance on quality data, which can be scarce especially during early-stage optimization. Contrarily, physics-based approaches like free energy perturbation (FEP) are frequently constrained by low throughput and high cost by comparison; however, physics-based methods are capable of making highly accurate binding affinity predictions. In this study, we harnessed the strength of FEP to overcome data paucity in ML by generating virtual activity data sets which then inform the training of algorithms. Here, we show that ML algorithms trained with an FEP-augmented data set could achieve comparable predictive accuracy to data sets trained on experimental data from biological assays. Throughout the paper, we emphasize key mechanistic considerations that must be taken into account when aiming to augment data sets and lay the groundwork for successful implementation. Ultimately, the study advocates for the synergy of physics-based methods and ML to expedite the lead optimization process. We believe that the physics-based augmentation of ML will significantly benefit drug discovery, as these techniques continue to evolve.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c68098f1f72af3b2975f4a681ad88eea00c73c0e" target='_blank'>
              FEP Augmentation as a Means to Solve Data Paucity Problems for Machine Learning in Chemical Biology
              </a>
            </td>
          <td>
            Pieter B Burger, Xiaohu Hu, Ilya Balabin, Morné Muller, Megan Stanley, Fourie Joubert, Thomas M Kaiser
          </td>
          <td>2024-04-23</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The rewiring of molecular interactions in various conditions leads to distinct phenotypic outcomes. Differential Network Analysis (DNA) is dedicated to exploring these rewirings within gene and protein networks. Leveraging statistical learning and graph theory, DNA algorithms scrutinize alterations in interaction patterns derived from experimental data. Introducing a novel approach to differential network analysis, we incorporate differential gene expression based on sex and gender attributes. We hypothesize that gene expression can be accurately represented through non-Gaussian processes. Our methodology involves quantifying changes in non-parametric correlations among gene pairs and expression levels of individual genes. Applying our method to public expression datasets concerning diabetes mellitus and atherosclerosis in liver tissue, we identify gender-specific differential networks. Results underscore the biological relevance of our approach in uncovering meaningful molecular distinctions. Author summary This paper explores a novel technique for Differential Network Analysis (DNA) that considers sex-based variations. DNA compares biological networks under different conditions, like healthy vs. diseased states. Our method tackles the limitations of traditional DNA approaches, which often assume specific data distributions. We propose a non-parametric DNA methodology that integrates sex differences and identifies differential edges between networks. This approach utilizes data on gene expression levels and sex to construct a more accurate picture of the molecular mechanisms underlying diseases, particularly those exhibiting sex-dependent variations. Our method paves the way for a deeper understanding of how sex and age influence disease processes at the molecular level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/252dfc92a7acd248eea2cadc4e0c5dd93ed0f960" target='_blank'>
              Non Parametric Differential Network Analysis: A Tool for Unveiling Specific Molecular Signatures
              </a>
            </td>
          <td>
            P. Guzzi, Roy Arkaprava, Marianna Milano, Pierangelo Veltri
          </td>
          <td>2024-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The crisis of antibiotic resistance has become a significant global threat to human health. Understanding properties of antibiotic resistance genes (ARGs) is the first step to mitigate this issue. Although many methods have been proposed for predicting properties of ARGs, most of these methods focus only on predicting antibiotic classes, while ignoring other properties of ARGs, such as resistance mechanisms and transferability. However, acquiring all of these properties of ARGs can help researchers gain a more comprehensive understanding of the essence of antibiotic resistance, which will facilitate the development of antibiotics. In this paper, the task of predicting properties of ARGs is modeled as a multi-task learning problem, and an effective subtask-aware representation learning-based framework is proposed accordingly. More specifically, property-specific expert networks and shared expert networks are utilized respectively to learn subtask-specific features for each subtask and shared features among different subtasks. In addition, a gating-controlled mechanism is employed to dynamically allocate weights to subtask-specific semantics and shared semantics obtained respectively from property-specific expert networks and shared expert networks, thus adjusting distinctive contributions of subtask-specific features and shared features to achieve optimal performance for each subtask simultaneously. Extensive experiments are conducted on publicly available data, and experimental results demonstrate the effectiveness of the proposed framework on the task of ARGs properties prediction. The data and source codes are available in GitHub at https://github.com/David-WZhao/GCM-ARG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2fccf1f97337aace6821bace20d37e23b0fde0" target='_blank'>
              Subtask-aware Representation Learning for Predicting Antibiotic Resistance Gene Properties via Gating-controlled Mechanism.
              </a>
            </td>
          <td>
            Weizhong Zhao, Junze Wu, Shujie Luo, Xingpeng Jiang, Tingting He, Xiaohua Hu
          </td>
          <td>2024-04-19</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Characterizing and validating which mutations influence development of cancer is challenging. Machine learning has delivered significant advances in protein structure prediction, but its utility for identifying cancer drivers is less explored. We evaluated multiple computational methods for identifying cancer driver alterations. For identifying known drivers, methods incorporating protein structure or functional genomic data outperformed methods trained only on evolutionary data. We further validated VUSs annotated as pathogenic by testing their association with overall survival in two cohorts of patients with non-small cell lung cancer (N=7,965 and 977). “Pathogenic” VUSs in KEAP1 and SMARCA4 identified by several methods were associated with worse survival, unlike “benign” VUSs. “Pathogenic” VUSs exhibited mutual exclusivity with known oncogenic alterations at the pathway level, further suggesting biological validity. Despite training primarily on germline, rather than somatic, mutation data, computational predictions contribute to a more comprehensive understanding of tumor genetics as validated by real-world data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2da4aaea52580392b7866551af65631c9c292d4" target='_blank'>
              Machine learning predictions improve identification of real-world cancer driver mutations
              </a>
            </td>
          <td>
            T. Tran, Chris Fong, K. Pichotta, A. Luthra, Ronglai Shen, Yuan Chen, Michele Waters, Susie Kim, Michael F. Berger, Gregory Riely, M. Ladanyi, D. Chakravarty, N. Schultz, J. Jee
          </td>
          <td>2024-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>182</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f8e3ebfd71d10fc5d76b9ba030c82f14180485" target='_blank'>
              Demographic bias in misdiagnosis by computational pathology models.
              </a>
            </td>
          <td>
            Anurag Vaidya, Richard J. Chen, Drew F. K. Williamson, Andrew H. Song, Guillaume Jaume, Yuzhe Yang, Thomas Hartvigsen, Emma C Dyer, Ming Y. Lu, Jana Lipková, Muhammad Shaban, Tiffany Y. Chen, Faisal Mahmood
          </td>
          <td>2024-04-01</td>
          <td>Nature medicine</td>
          <td>2</td>
          <td>20</td>
        </tr>

        <tr id="Non-parametric models, such as Gaussian Processes (GP), show promising results in the analysis of complex data. Their applications in neuroscience data have recently gained traction. In this research, we introduce a novel neural decoder model built upon GP models. The core idea is that two GPs generate neural data and their associated labels using a set of low-dimensional latent variables. Under this modeling assumption, the latent variables represent the underlying manifold or essential features present in the neural data. When GPs are trained, the latent variable can be inferred from neural data to decode the labels with a high accuracy. We demonstrate an application of this decoder model in a verbal memory experiment dataset and show that the decoder accuracy in predicting stimulus significantly surpasses the state-of-the-art decoder models. The preceding performance of this model highlights the importance of utilizing non-parametric models in the analysis of neuroscience data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f436be1dbdb68d4e054f322cd858a90087509d" target='_blank'>
              Latent Variable Double Gaussian Process Model for Decoding Complex Neural Data
              </a>
            </td>
          <td>
            Navid Ziaei, Joshua J. Stim, Melanie D. Goodman-Keiser, Scott Sponheim, A. Widge, Sasoun Krikorian, Ali Yousefi
          </td>
          <td>2024-05-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Continuous adaptation to variable environments is crucial for the survival of living organisms. Here, we analyze how adaptation, forecasting, and resource mobilization towards a target state, termed actionability, interact to determine biological function. We develop a general theory and show that it is possible for organisms to continuously track their optimal state in a dynamic environment by adapting towards an actionable target that incorporates just current information on the optimal state and its rate of change. If the environmental information is precise and readily actionable, it is possible to implement perfect tracking without anticipatory mechanisms, irrespective of the adaptation rate. In contrast, predictive functions, like those of circadian rhythms, are beneficial if sensing the environment is slow or unreliable, as they allow better adaptation with fewer resources. To explore potential actionable forecasting mechanisms, we develop a general approach that implements the adaptation dynamics with forecasting through a dynamics-informed neural network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0501fbdeff40be925a2011d2799fdc10dde012be" target='_blank'>
              Actionable forecasting as a determinant of function in noisy biological systems
              </a>
            </td>
          <td>
            J. Vilar, L. Saiz
          </td>
          <td>2024-04-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Dynamical systems are often time-varying, whose modeling requires a function that evolves with respect to time. Recent studies such as the neural ordinary differential equation proposed a time-dependent neural network, which provides a neural network varying with respect to time. However, we claim that the architectural choice to build a time-dependent neural network significantly affects its time-awareness but still lacks sufficient validation in its current states. In this study, we conduct an in-depth analysis of the architecture of modern time-dependent neural networks. Here, we report a vulnerability of vanishing timestep embedding, which disables the time-awareness of a time-dependent neural network. Furthermore, we find that this vulnerability can also be observed in diffusion models because they employ a similar architecture that incorporates timestep embedding to discriminate between different timesteps during a diffusion process. Our analysis provides a detailed description of this phenomenon as well as several solutions to address the root cause. Through experiments on neural ordinary differential equations and diffusion models, we observed that ensuring alive time-awareness via proposed solutions boosted their performance, which implies that their current implementations lack sufficient time-dependency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7117f299b2e91fe74a8961413c4db654bc4b1f7" target='_blank'>
              The Disappearance of Timestep Embedding in Modern Time-Dependent Neural Networks
              </a>
            </td>
          <td>
            Bum Jun Kim, Yoshinobu Kawahara, Sang Woo Kim
          </td>
          <td>2024-05-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d15c9cee965609e63274b374f2806434839e18" target='_blank'>
              Reverse engineering morphogenesis through Bayesian optimization of physics-based models
              </a>
            </td>
          <td>
            Nilay Kumar, M. Mim, Alexander Dowling, Jeremiah J Zartman
          </td>
          <td>2024-05-07</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Physiologically-based kinetic (PBK) models are widely used in pharmacology and toxicology for predicting the internal disposition of substances upon exposure, voluntarily or not. Due to their complexity, a large number of model parameters need to be estimated, either through in silico tools, in vitro experiments or by fitting the model to in vivo data. In the latter case, fitting complex structural models on in vivo data can result in overparameterisation and produce unrealistic parameter estimates. To address these issues, we propose a novel parameter grouping approach, which reduces the parametric space by co-estimating groups of parameters across compartments. Grouping of parameters is performed using genetic algorithms and is fully automated, based on a novel goodness-of-fit metric. To illustrate the practical application of the proposed methodology, two case studies were conducted. The first case study demonstrates the development of a new PBK model, while the second focuses on model refinement. In the first case study, a PBK model was developed to elucidate the biodistribution of titanium dioxide (TiO2) nanoparticles in rats following intravenous injection. A variety of parameter estimation schemes were employed. Comparative analysis based on goodness-of-fit metrics demonstrated that the proposed methodology yields models that outperform standard estimation approaches, while utilising a reduced number of parameters. In the second case study, an existing PBK model for perfluorooctanoic acid (PFOA) in rats was extended to incorporate additional tissues, providing a more a comprehensive portrayal of PFOA biodistribution. Both models were validated through independent in vivo studies to ensure their reliability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4f018eed9b1fec78356090d7f793687019a810f" target='_blank'>
              Parameter grouping and Co-estimation in Physiologically-Based kinetic models using genetic algorithms.
              </a>
            </td>
          <td>
            P. Tsiros, Vasileios Minadakis, Dingsheng Li, H. Sarimveis
          </td>
          <td>2024-04-19</td>
          <td>Toxicological sciences : an official journal of the Society of Toxicology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Molecular simulations have assumed a paramount role in the fields of chemistry, biology, and material sciences, being able to capture the intricate dynamic properties of systems. Within this realm, coarse-grained (CG) techniques have emerged as invaluable tools to sample large-scale systems and reach extended timescales by simplifying system representation. However, CG approaches come with a trade-off: they sacrifice atomistic details that might hold significant relevance in deciphering the investigated process. Therefore, a recommended approach is to identify key CG conformations and process them using backmapping methods, which retrieve atomistic coordinates. Currently, rule-based methods yield subpar geometries and rely on energy relaxation, resulting in less-than-optimal outcomes. Conversely, machine learning techniques offer higher accuracy but are either limited in transferability between systems or tied to specific CG mappings. In this work, we introduce HEroBM, a dynamic and scalable method that employs deep equivariant graph neural networks and a hierarchical approach to achieve high-resolution backmapping. HEroBM handles any type of CG mapping, offering a versatile and efficient protocol for reconstructing atomistic structures with high accuracy. Focused on local principles, HEroBM spans the entire chemical space and is transferable to systems of varying sizes. We illustrate the versatility of our framework through diverse biological systems, including a complex real-case scenario. Here, our end-to-end backmapping approach accurately generates the atomistic coordinates of a G protein-coupled receptor bound to an organic small molecule within a cholesterol/phospholipid bilayer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94800faf04d01d15a154fe1bf79ec38f7c0406d2" target='_blank'>
              HEroBM: a deep equivariant graph neural network for universal backmapping from coarse-grained to all-atom representations
              </a>
            </td>
          <td>
            Daniele Angioletti, S. Raniolo, V. Limongelli
          </td>
          <td>2024-04-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6155f5d70357f51adefc8adbbf3af90792e5537c" target='_blank'>
              SEEI: spherical evolution with feedback mechanism for identifying epistatic interactions
              </a>
            </td>
          <td>
            Deyu Tang, Yi-jun Mao, Jie Zhao, Jin Yang, Shiyin Li, Fu-xiang Ren, Junxi Zheng
          </td>
          <td>2024-05-13</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Data-driven modelling techniques provide a method for deriving models of dynamical systems directly from complicated data streams. However, tracking and forecasting such data streams poses a significant challenge to most methods, as they assume the underlying process and model does not change over time. In this paper, we apply one such data-driven method, the Koopman autoencoder (KAE), to high-dimensional oscillatory data to generate a low-dimensional latent space and model, where the system's dynamics appear linear. This allows one to accurately track and forecast systems where the underlying model may change over time. States and the model in the reduced order latent space can then be efficiently updated as new data becomes available, using data assimilation techniques such as the ensemble Kalman filter (EnKF), in a technique we call the KAE EnKF. We demonstrate that this approach is able to effectively track and forecast time-varying, nonlinear dynamical systems in synthetic examples. We then apply the KAE EnKF to a video of a physical pendulum, and achieve a significant improvement over current state-of-the-art methods. By generating effective latent space reconstructions, we find that we are able to construct accurate short-term forecasts and efficient adaptations to externally forced changes to the pendulum's frequency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be5728bd13fd3650c966df73ef5772bf8b905261" target='_blank'>
              Tracking and forecasting oscillatory data streams using Koopman autoencoders and Kalman filtering
              </a>
            </td>
          <td>
            Stephen A Falconer, David J. B. Lloyd, N. Santitissadeekorn
          </td>
          <td>2024-05-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In this study, we propose a multi branched network approach to predict the dynamics of a physics attractor characterized by intricate and chaotic behavior. We introduce a unique neural network architecture comprised of Radial Basis Function (RBF) layers combined with an attention mechanism designed to effectively capture nonlinear inter-dependencies inherent in the attractor's temporal evolution. Our results demonstrate successful prediction of the attractor's trajectory across 100 predictions made using a real-world dataset of 36,700 time-series observations encompassing approximately 28 minutes of activity. To further illustrate the performance of our proposed technique, we provide comprehensive visualizations depicting the attractor's original and predicted behaviors alongside quantitative measures comparing observed versus estimated outcomes. Overall, this work showcases the potential of advanced machine learning algorithms in elucidating hidden structures in complex physical systems while offering practical applications in various domains requiring accurate short-term forecasting capabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/263a32e783722b09eefe90e0bbdf88a61c93c0c4" target='_blank'>
              A Multi-Branched Radial Basis Network Approach to Predicting Complex Chaotic Behaviours
              </a>
            </td>
          <td>
            Aarush Sinha
          </td>
          <td>2024-03-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2193c014996cdda5760beded2e7b3cdc0c050d" target='_blank'>
              Supervised latent factor modeling isolates cell-type-specific transcriptomic modules that underlie Alzheimer’s disease progression
              </a>
            </td>
          <td>
            Liam Hodgson, Yue Li, Y. Iturria-medina, Jo Anne Stratton, Guy Wolf, Smita Krishnaswamy, David A. Bennett, Danilo Bzdok
          </td>
          <td>2024-05-17</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Polygenic prediction of complex trait phenotypes has become important in human genetics, especially in the context of precision medicine. Recently, Morgante et al. introduced mr.mash, a flexible and computationally efficient method that models multiple phenotypes jointly and leverages sharing of effects across such phenotypes to improve prediction accuracy. However, a drawback of mr.mash is that it requires individual-level data, which are often not publicly available. In this work, we introduce mr.mash-rss, an extension of the mr.mash model that requires only summary statistics from Genome-Wide Association Studies (GWAS) and linkage disequilibrium (LD) estimates from a reference panel. By using summary data, we achieve the twin goal of increasing the applicability of the mr.mash model to data sets that are not publicly available and making it scalable to biobank-size data. Through simulations, we show that mr.mash-rss is competitive with, and often outperforms, current state-of-the-art methods for single- and multi-phenotype polygenic prediction in a variety of scenarios that differ in the pattern of effect sharing across phenotypes, the number of phenotypes, the number of causal variants, and the genomic heritability. We also present a real data analysis of 16 blood cell phenotypes in UK Biobank, showing that mr.mash-rss achieves higher prediction accuracy than competing methods for the majority of traits, especially when the data has smaller sample size. Author summary Polygenic prediction refers to the use of an individual’s genetic information (i.e., genotypes) to predict traits (i.e., phenotypes), which are often of medical relevance. It is known that some phenotypes are related and are affected by the same genotypes. When this is the case, it is possible to improve the accuracy of predictions by using methods that model multiple phenotypes jointly and account for shared effects. mr.mash is a recently developed multi-phenotype method that can learn which effects are shared and has been shown to improve prediction. However, mr.mash requires large data sets of genetic and phenotypic information collected at the individual level. Such data are often unavailable due to privacy concerns, or are difficult to work with due to the computational resources needed to analyze data of this size. Our work extends mr.mash to require only summary statistics from Genome-Wide Association Studies instead of individual-level data, which are usually publicly available. In addition, the computations using summary statistics do not depend on sample size, making the newly developed mr.mash-rss scalable to extremely large data sets. Using simulations and real data analysis, we show that our method is competitive with other methods for polygenic prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bedda1558ffa6d3f6b258dd1aafa7cf5aba328" target='_blank'>
              Improving polygenic prediction from summary data by learning patterns of effect sharing across multiple phenotypes
              </a>
            </td>
          <td>
            Deborah Kunkel, Peter Sørensen, Vijay Shankar, Fabio Morgante
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Longitudinal data from clinical trials are commonly analyzed using mixed models for repeated measures (MMRM) when the time variable is categorical or linear mixed-effects models (ie, random effects model) when the time variable is continuous. In these models, statistical inference is typically based on the absolute difference in the adjusted mean change (for categorical time) or the rate of change (for continuous time). Previously, we proposed a novel approach: modeling the percentage reduction in disease progression associated with the treatment relative to the placebo decline using proportional models. This concept of proportionality provides an innovative and flexible method for simultaneously modeling different cohorts, multivariate endpoints, and jointly modeling continuous and survival endpoints. Through simulated data, we demonstrate the implementation of these models using SAS procedures in both frequentist and Bayesian approaches. Additionally, we introduce a novel method for implementing MMRM models (ie, analysis of response profile) using the nlmixed procedure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78e1bc08bf7e8792786ada3ede4824f4e82403f5" target='_blank'>
              Novel non-linear models for clinical trial analysis with longitudinal data: A tutorial using SAS for both frequentist and Bayesian methods.
              </a>
            </td>
          <td>
            Guoqiao Wang, Whedy Wang, Brian Mangal, Yijie Liao, Lon S. Schneider, Yan Li, C. Xiong, E. McDade, Richard E. Kennedy, Randall J. Bateman, Gary Cutter
          </td>
          <td>2024-05-10</td>
          <td>Statistics in medicine</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0a9f61fdab177217fc947916c308a723a07e3c1" target='_blank'>
              Automatic design of gene regulatory mechanisms for spatial pattern formation
              </a>
            </td>
          <td>
            Reza Mousavi, Daniel Lobo
          </td>
          <td>2024-04-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="MOTIVATION
Systems biology aims to better understand living systems through mathematical modelling of experimental and clinical data. A pervasive challenge in quantitative dynamical modelling is the integration of time series measurements, which often have high variability and low sampling resolution. Approaches are required to utilise such information while consistently handling uncertainties.


RESULTS
We present BayModTS (Bayesian Modelling of Time Series data), a new FAIR (Findable, Accessible, Interoperable and Reusable) workflow for processing and analysing sparse and highly variable time series data. BayModTS consistently transfers uncertainties from data to model predictions, including process knowledge via parameterised models. Further, credible differences in the dynamics of different conditions can be identified by filtering noise. To demonstrate the power and versatility of BayModTS, we applied it to three hepatic datasets gathered from three different species and with different measurement techniques: (i) blood perfusion measurements by magnetic resonance imaging in rat livers after portal vein ligation, (ii) pharmacokinetic time series of different drugs in normal and steatotic mice, and (iii) CT-based volumetric assessment of human liver remnants after clinical liver resection.


AVAILABILITY AND IMPLEMENTATION
The BayModTS codebase is available on GitHub at https://github.com/Systems-Theory-in-Systems-Biology/BayModTS. The repository contains a Python script for the executable BayModTS workflow and a widely applicable SBML (Systems Biology Markup Language) model for retarded transient functions. In addition, all examples from the paper are included in the repository. Data and code of the application examples are stored on DaRUS https://doi.org/10.18419/darus-3876. The raw MRI ROI voxel data were uploaded to DaRUS https://doi.org/10.18419/darus-3878. The steatosis metabolite data are published on FairdomHub 10.15490/fairdomhub.1.study.1070.1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483f129e077df807f3f25aef2dde774505aa52b2" target='_blank'>
              Bayesian modelling of time series data (BayModTS) - a FAIR workflow to process sparse and highly variable data.
              </a>
            </td>
          <td>
            Sebastian Höpfl, Mohamed Albadry, U. Dahmen, Karl-Heinz Herrmann, E. Kindler, Matthias König, Jürgen Rainer Reichenbach, Hans-Michael Tautenhahn, Weiwei Wei, Wan-Ting Zhao, Nicole Erika Radde
          </td>
          <td>2024-05-13</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f090dcadd31a6e3dd8f370ea2d0fe1907c6a9a" target='_blank'>
              Stacked neural network for predicting polygenic risk score
              </a>
            </td>
          <td>
            Sun bin Kim, Joon Ho Kang, M. Cheon, Dong Jun Kim, Byung-Chul Lee
          </td>
          <td>2024-05-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction: Gene regulatory networks (GRNs) are characterized by their dynamism, meaning that the regulatory interactions which constitute these networks evolve with time. Identifying when changes in the GRN architecture occur can inform our understanding of fundamental biological processes, such as disease manifestation, development, and evolution. However, it is usually not possible to know a priori when a change in the network architecture will occur. Furthermore, an architectural shift may alter the underlying noise characteristics, such as the process noise covariance.Methods: We develop a fully Bayesian hierarchical model to address the following: a) sudden changes in the network architecture; b) unknown process noise covariance which may change along with the network structure; and c) unknown measurement noise covariance. We exploit the use of conjugate priors to develop an analytically tractable inference scheme using Bayesian sequential Monte Carlo (SMC) with a local Gibbs sampler.Results: Our Bayesian learning algorithm effectively estimates time-varying gene expression levels and architectural model indicators under varying noise conditions. It accurately captures sudden changes in network architecture and accounts for time-evolving process and measurement noise characteristics. Our algorithm performs well even under high noise conditions. By incorporating conjugate priors, we achieve analytical tractability, enabling robust inference despite the inherent complexities of the system. Furthermore, our method outperforms the standard particle filter in all test scenarios.Discussion: The results underscore our method’s efficacy in capturing architectural changes in GRNs. Its ability to adapt to a range of time-evolving noise conditions emphasizes its practical relevance for real-world biological data, where noise presents a significant challenge. Overall, our method provides a powerful tool for studying the dynamics of GRNs and has the potential to advance our understanding of fundamental biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8a243a140a8c2ad96013292fbe388c444c69974" target='_blank'>
              Bayesian learning of nonlinear gene regulatory networks with switching architectures
              </a>
            </td>
          <td>
            Nayely Vélez-Cruz, Antonia Papandreou-Suppappola
          </td>
          <td>2024-05-22</td>
          <td>Frontiers in Signal Processing</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A tremendous range of design tasks in materials, physics, and biology can be formulated as finding the optimum of an objective function depending on many parameters without knowing its closed-form expression or the derivative. Traditional derivative-free optimization techniques often rely on strong assumptions about objective functions, thereby failing at optimizing non-convex systems beyond 100 dimensions. Here, we present a tree search method for derivative-free optimization that enables accelerated optimal design of high-dimensional complex systems. Specifically, we introduce stochastic tree expansion, dynamic upper confidence bound, and short-range backpropagation mechanism to evade local optimum, iteratively approximating the global optimum using machine learning models. This development effectively confronts the dimensionally challenging problems, achieving convergence to global optima across various benchmark functions up to 2,000 dimensions, surpassing the existing methods by 10- to 20-fold. Our method demonstrates wide applicability to a wide range of real-world complex systems spanning materials, physics, and biology, considerably outperforming state-of-the-art algorithms. This enables efficient autonomous knowledge discovery and facilitates self-driving virtual laboratories. Although we focus on problems within the realm of natural science, the advancements in optimization techniques achieved herein are applicable to a broader spectrum of challenges across all quantitative disciplines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/383c030cc1d9a568808e59e7f90cf8e0ef70b283" target='_blank'>
              Derivative-free tree optimization for complex systems
              </a>
            </td>
          <td>
            Ye Wei, Bo Peng, Ruiwen Xie, Yangtao Chen, Yu Qin, Peng Wen, Stefan Bauer, Po-Yen Tung
          </td>
          <td>2024-04-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abc8de13051c73460314a650bbafead7ae47b002" target='_blank'>
              Determining individual suitability for neoadjuvant systemic therapy in breast cancer patients through deep learning.
              </a>
            </td>
          <td>
            Enzhao Zhu, Linmei Zhang, Yixian Liu, Tianyu Ji, Jianmeng Dai, Ruichen Tang, Jiayi Wang, Chunyu Hu, Kai Chen, Qianyi Yu, Qiuyi Lu, Zisheng Ai
          </td>
          <td>2024-04-28</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Large-scale prediction models (typically using tools from artificial intelligence, AI, or machine learning, ML) are increasingly ubiquitous across a variety of industries and scientific domains. Such methods are often paired with detailed data from sources such as electronic health records, wearable sensors, and omics data (high-throughput technology used to understand biology). Despite their utility, implementing AI and ML tools at the scale necessary to work with this data introduces two major challenges. First, it can cost tens of thousands of dollars to train a modern AI/ML model at scale. Second, once the model is trained, its predictions may become less relevant as patient and provider behavior change, and predictions made for one geographical area may be less accurate for another. These two challenges raise a fundamental question: how often should you refit the AI/ML model to optimally trade-off between cost and relevance? Our work provides a framework for making decisions about when to {\it refit} AI/ML models when the goal is to maintain valid statistical inference (e.g. estimating a treatment effect in a clinical trial). Drawing on portfolio optimization theory, we treat the decision of {\it recalibrating} versus {\it refitting} the model as a choice between ''investing'' in one of two ''assets.'' One asset, recalibrating the model based on another model, is quick and relatively inexpensive but bears uncertainty from sampling and the possibility that the other model is not relevant to current circumstances. The other asset, {\it refitting} the model, is costly but removes the irrelevance concern (though not the risk of sampling error). We explore the balancing act between these two potential investments in this paper.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a80d641f4493758596f70ff4f1b13f979d6d7a6" target='_blank'>
              Some models are useful, but for how long?: A decision theoretic approach to choosing when to refit large-scale prediction models
              </a>
            </td>
          <td>
            Kentaro Hoffman, Stephen Salerno, Jeff Leek, Tyler H. McCormick
          </td>
          <td>2024-05-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>