<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Latent%20space%20simulators/">
      
      
        <link rel="next" href="../PINNs/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Parametrizing using ML
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Time-series%20forecasting/" class="md-tabs__link">
        
  
    
  
  Time-series forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Symbolic%20regression/" class="md-tabs__link">
        
  
    
  
  Symbolic regression

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Neural%20ODEs/" class="md-tabs__link">
        
  
    
  
  Neural ODEs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Physics-based%20GNNs/" class="md-tabs__link">
        
  
    
  
  Physics-based GNNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Latent%20space%20simulators/" class="md-tabs__link">
        
  
    
  
  Latent space simulators

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Parametrizing using ML

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../PINNs/" class="md-tabs__link">
        
  
    
  
  PINNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Koopman%20operator/" class="md-tabs__link">
        
  
    
  
  Koopman operator

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Time-series%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Time-series forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Symbolic%20regression/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Symbolic regression
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Neural%20ODEs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Neural ODEs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Physics-based%20GNNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Physics-based GNNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Latent%20space%20simulators/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Latent space simulators
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Parametrizing using ML
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../PINNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    PINNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Koopman%20operator/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Koopman operator
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Parametrizing using ML</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-06-24 06:07:14 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Parametrizing using ML</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Parametrizing using ML</a><br>
      <a href="#recommended_articles">3. Recommended articles on Parametrizing using ML</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Parametrizing using ML</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Parametrizing using ML</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e837965494c4edbec4d30832d31ba5639996da8" target='_blank'>
                Deep learning prediction of patient response time course from early data via neural-pharmacokinetic/pharmacodynamic modelling
                </a>
              </td>
          <td>
            James Lu, B. Bender, Jin Y. Jin, Y. Guan
          </td>
          <td>2020-10-22</td>
          <td>Nature Machine Intelligence</td>
          <td>34</td>
          <td>38</td>

            <td><a href='../recommendations/4e837965494c4edbec4d30832d31ba5639996da8' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Many real-world systems such as social networks and moving planets are dynamic in nature, where a set of coupled objects are connected via the interaction graph and exhibit complex behavior along the time. For example, the COVID-19 pandemic can be considered as a dynamical system, where objects represent geographical locations (e.g., states) whose daily confirmed cases of infection evolve over time. Outbreak at one location may influence another location as people travel between these locations, forming a graph. Thus, how to model and predict the complex dynamics for these systems becomes a critical research problem. Existing work on modeling graph-structured data mostly assumes a static setting. How to handle dynamic graphs remains to be further explored. On one hand, features of objects change over time, influenced by the linked objects in the interaction graph. On the other hand, the graph itself can also evolve, where new interactions (links) may form and existing links may drop, which may in turn be affected by the dynamic features of objects. In this paper, we propose coupled graph ODE: a novel latent ordinary differential equation (ODE) generative model that learns the coupled dynamics of nodes and edges with a graph neural network (GNN) based ODE in a continuous manner. Our model consists of two coupled ODE functions for modeling the dynamics of edges and nodes based on their latent representations respectively. It employs a novel encoder parameterized by a GNN for inferring the initial states from historical data, which serves as the starting point of the predicted latent trajectories. Experiment results on the COVID-19 dataset and the simulated social network dataset demonstrate the effectiveness of our proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf2145f9998f304513c0c9b530ee9f7750c6f55" target='_blank'>
                Coupled Graph ODE for Learning Interacting System Dynamics
                </a>
              </td>
          <td>
            Zijie Huang, Yizhou Sun, Wei Wang
          </td>
          <td>2021-08-14</td>
          <td>Knowledge Discovery and Data Mining, Proceedings of the 27th ACM SIGKDD Conference on Knowledge Discovery & Data Mining</td>
          <td>49</td>
          <td>56</td>

            <td><a href='../recommendations/aaf2145f9998f304513c0c9b530ee9f7750c6f55' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd28606fbae3449f831248804264c9885e992f9" target='_blank'>
                CellBox: Interpretable Machine Learning for Perturbation Biology with Application to the Design of Cancer Combination Therapy.
                </a>
              </td>
          <td>
            Bo Yuan, Ciyue Shen, Augustin Luna, Anil Korkut, D. Marks, John Ingraham, C. Sander
          </td>
          <td>2020-12-21</td>
          <td>Cell systems</td>
          <td>69</td>
          <td>153</td>

            <td><a href='../recommendations/6bd28606fbae3449f831248804264c9885e992f9' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="We introduce a flexible, scalable Bayesian inference framework for nonlinear dynamical systems characterised by distinct and hierarchical variability at the individual, group, and population levels. Our model class is a generalisation of nonlinear mixed-effects (NLME) dynamical systems, the statistical workhorse for many experimental sciences. We cast parameter inference as stochastic optimisation of an end-to-end differentiable, block-conditional variational autoencoder. We specify the dynamics of the data-generating process as an ordinary differential equation (ODE) such that both the ODE and its solver are fully differentiable. This model class is highly flexible: the ODE right-hand sides can be a mixture of user-prescribed or “white-box” sub-components and neural network or “black-box” sub-components. Using stochastic optimisation, our amortised inference algorithm could seamlessly scale up to massive data collection pipelines (common in labs with robotic automation). Finally, our framework supports interpretability with respect to the underlying dynamics, as well as predictive generalization to unseen combinations of group components (also called “zero-shot” learning). We empirically validate our method by predicting the dynamic behaviour of bacteria that were genetically engineered to function as biosensors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309c5ae93a4cabfd37747abd130866240e265b2d" target='_blank'>
                Efficient Amortised Bayesian Inference for Hierarchical and Nonlinear Dynamical Systems
                </a>
              </td>
          <td>
            Geoffrey Roeder, Paul K. Grant, Andrew Phillips, Neil Dalchau, Edward Meeds
          </td>
          <td>2019-05-24</td>
          <td>International Conference on Machine Learning, ArXiv</td>
          <td>21</td>
          <td>28</td>

            <td><a href='../recommendations/309c5ae93a4cabfd37747abd130866240e265b2d' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Parametrizing using ML'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Parametrizing using ML</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="A variety of classical machine learning approaches have been developed over the past ten years with the aim to individualize drug dosages based on measured plasma concentrations. However, the interpretability of these models is challenging as they do not incorporate information on pharmacokinetic (PK) drug disposition. In this work we compare well-known population PK modelling with classical and a newly proposed scientific machine learning (SciML) framework, which combines knowledge on drug disposition with data-driven modelling. Our approach lets us estimate population PK parameters and their inter-individual variability (IIV) using multimodal covariate data of each patient. A dataset of 549 fluorouracil (5FU) plasma concentrations as example for an intravenously administered drug and a dataset of 308 sunitinib concentrations as example for an orally administered drug were used for analysis. Whereas classical machine learning models were not able to describe the data sufficiently, the proposed model allowed us to obtain highly accurate predictions even for new patients. Additionally, we demonstrated that our model could outperform traditional population PK models in terms of accuracy and greater flexibility when learning population parameters if given enough training data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01334b780295132c89212673bbf083ad5a36343e" target='_blank'>
              Scientific machine learning for predicting plasma concentrations in anti-cancer therapy
              </a>
            </td>
          <td>
            Diego Valderrama, O. Teplytska, Luca Marie, Koltermann, Elena Trunz, Eduard Schmulenson, Achim Fritsch, Ulrich Jaehde, Holger Fröhlich, Schloss Birlinghoven, Sankt Augustin, Germany, An der Immenburg, D. Bonn
          </td>
          <td>2024-05-08</td>
          <td>None</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="A patient's digital twin is a computational model that describes the evolution of their health over time. Digital twins have the potential to revolutionize medicine by enabling individual-level computer simulations of human health, which can be used to conduct more efficient clinical trials or to recommend personalized treatment options. Due to the overwhelming complexity of human biology, machine learning approaches that leverage large datasets of historical patients' longitudinal health records to generate patients' digital twins are more tractable than potential mechanistic models. In this manuscript, we describe a neural network architecture that can learn conditional generative models of clinical trajectories, which we call Digital Twin Generators (DTGs), that can create digital twins of individual patients. We show that the same neural network architecture can be trained to generate accurate digital twins for patients across 13 different indications simply by changing the training set and tuning hyperparameters. By introducing a general purpose architecture, we aim to unlock the ability to scale machine learning approaches to larger datasets and across more indications so that a digital twin could be created for any patient in the world.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd216824a3b00951db6ff54f723148cfa3706e5" target='_blank'>
              Digital Twin Generators for Disease Modeling
              </a>
            </td>
          <td>
            Nameyeh Alam, Jake Basilico, Daniele Bertolini, S. C. Chetty, Heather D'Angelo, Ryan Douglas, Charles K. Fisher, Franklin Fuller, Melissa Gomes, Rishabh Gupta, Alex Lang, A. Loukianov, Rachel Mak-McCully, Cary Murray, Hanalei Pham, Susanna Qiao, Elena S. Ryapolova-Webb, Aaron M. Smith, Dimitri Theoharatos, Anil Tolwani, Eric W. Tramel, Anna A. Vidovszky, J. Viduya, Jonathan R. Walsh
          </td>
          <td>2024-05-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de2e8855c3ac85fb287f6356a1f811bb41e6472" target='_blank'>
              Mechanism-based organization of neural networks to emulate systems biology and pharmacology models
              </a>
            </td>
          <td>
            John Mann, Hamed Meshkin, Joel Zirkle, Xiaomei Han, Bradlee Thrasher, Anik Chaturbedi, Ghazal Arabidarrehdor, Zhihua Li
          </td>
          <td>2024-05-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Precise and personalized drug application is crucial in the clinical treatment of complex diseases. Although neural networks offer a new approach to improving drug strategies, their internal structure is difficult to interpret. Here, we propose PBAC (Pathway‐Based Attention Convolution neural network), which integrates a deep learning framework and attention mechanism to address the complex biological pathway information, thereby provide a biology function‐based robust drug responsiveness prediction model. PBAC has four layers: gene‐pathway layer, attention layer, convolution layer and fully connected layer. PBAC improves the performance of predicting drug responsiveness by focusing on important pathways, helping us understand the mechanism of drug action in diseases. We validated the PBAC model using data from four chemotherapy drugs (Bortezomib, Cisplatin, Docetaxel and Paclitaxel) and 11 immunotherapy datasets. In the majority of datasets, PBAC exhibits superior performance compared to traditional machine learning methods and other research approaches (area under curve = 0.81, the area under the precision‐recall curve = 0.73). Using PBAC attention layer output, we identified some pathways as potential core cancer regulators, providing good interpretability for drug treatment prediction. In summary, we presented PBAC, a powerful tool to predict drug responsiveness based on the biology pathway information and explore the potential cancer‐driving pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe0a93616d2710fb68378b6a662b90c3486c1ca1" target='_blank'>
              PBAC: A pathway‐based attention convolution neural network for predicting clinical drug treatment responses
              </a>
            </td>
          <td>
            Dexun Deng, Xiaoqiang Xu, Ting Cui, Mingcong Xu, Kunpeng Luo, Han Zhang, Qiuyu Wang, Chao Song, Chao Li, Guohua Li, Desi Shang
          </td>
          <td>2024-04-29</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pharmacometric models are pivotal across drug discovery and development, playing a decisive role in determining the progression of candidate molecules. However, the derivation of mathematical equations governing the system is a labor-intensive trial-and-error process, often constrained by tight timelines. In this study, we introduce PKINNs, a novel purely data-driven pharmacokinetic-informed neural network model. PKINNs efficiently discovers and models intrinsic multi-compartment-based pharmacometric structures, reliably forecasting their derivatives. The resulting models are both interpretable and explainable through Symbolic Regression methods. Our computational framework demonstrates the potential for closed-form model discovery in pharmacometric applications, addressing the labor-intensive nature of traditional model derivation. With the increasing availability of large datasets, this framework holds the potential to significantly enhance model-informed drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecec2c77f9d85a4e75c8a8de808c4d916ca0015" target='_blank'>
              Discovering intrinsic multi-compartment pharmacometric models using Physics Informed Neural Networks
              </a>
            </td>
          <td>
            I. Nasim, Adam Nasim
          </td>
          <td>2024-04-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Multi-factor screenings are commonly used in diverse applications in medicine and bioengineering, including optimizing combination drug treatments and microbiome engineering. Despite the advances in high-throughput technologies, large-scale experiments typically remain prohibitively expensive. Here we introduce a machine learning platform, structure-augmented regression (SAR), that exploits the intrinsic structure of each biological system to learn a high-accuracy model with minimal data requirement. Under different environmental perturbations, each biological system exhibits a unique, structured phenotypic response. This structure can be learned based on limited data and once learned, can constrain subsequent quantitative predictions. We demonstrate that SAR requires significantly fewer data comparing to other existing machine-learning methods to achieve a high prediction accuracy, first on simulated data, then on experimental data of various systems and input dimensions. We then show how a learned structure can guide effective design of new experiments. Our approach has implications for predictive control of biological systems and an integration of machine learning prediction and experimental design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c960a00bc91e7ef861762c26cbaf5fb088878ab0" target='_blank'>
              Data-driven learning of structure augments quantitative prediction of biological responses.
              </a>
            </td>
          <td>
            Yuanchi Ha, Helena R. Ma, Feilun Wu, Andrea Weiss, Katherine Duncker, Helen Xu, Jia Lu, Max Golovsky, Daniel Reker, Lingchong You
          </td>
          <td>2024-06-03</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c53a64b37da477c06468da0f1b359b77e2e273e9" target='_blank'>
              Biologically informed NeuralODEs for genome-wide regulatory dynamics
              </a>
            </td>
          <td>
            Intekhab Hossain, Viola Fanfani, Jonas Fischer, John Quackenbush, R. Burkholz
          </td>
          <td>2024-05-21</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Introduction Prediction of pharmacokinetic (PK) properties is crucial for drug discovery and development. Machine-learning (ML) models, which use statistical pattern recognition to learn correlations between input features (such as chemical structures) and target variables (such as PK parameters), are being increasingly used for this purpose. To embed ML models for PK prediction into workflows and to guide future development, a solid understanding of their applicability, advantages, limitations, and synergies with other approaches is necessary. Areas covered This narrative review discusses the design and application of ML models to predict PK parameters of small molecules, especially in light of established approaches including in vitro-in vivo extrapolation (IVIVE) and physiologically based pharmacokinetic (PBPK) models. The authors illustrate scenarios in which the three approaches are used and emphasize how they enhance and complement each other. In particular, they highlight achievements, the state of the art and potentials of applying machine learning for PK prediction through a comphrehensive literature review. Expert opinion ML models, when carefully crafted, regularly updated, and appropriately used, empower users to prioritize molecules with favorable PK properties. Informed practitioners can leverage these models to improve the efficiency of drug discovery and development process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4407f5485a20d2a2a26036c5e168a7a01ce78c14" target='_blank'>
              Another string to your bow: machine learning prediction of the pharmacokinetic properties of small molecules
              </a>
            </td>
          <td>
            Davide Bassani, Neil John Parrott, Nenad Manevski, Jitao David Zhang
          </td>
          <td>2024-05-10</td>
          <td>Expert Opinion on Drug Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In oncology drug development, tumor dynamics modeling is widely applied to predict patients' overall survival (OS) via parametric models. However, the current modeling paradigm, which assumes a disease-specific link between tumor dynamics and survival, has its limitations. This is particularly evident in drug development scenarios where the clinical trial under consideration contains patients with tumor types for which there is little to no prior institutional data. In this work, we propose the use of a pan-indication solid tumor machine learning (ML) approach whereby all three tumor metrics (tumor shrinkage rate, tumor regrowth rate and time to tumor growth) are simultaneously used to predict patients' OS in a tumor type independent manner. We demonstrate the utility of this approach in a clinical trial of cancer patients treated with the tyrosine kinase inhibitor, pralsetinib. We compared the parametric and ML models and the results showed that the proposed ML approach is able to adequately predict patient OS across RET-altered solid tumors, including non-small cell lung cancer, medullary thyroid cancer as well as other solid tumors. While the findings of this study are promising, further research is needed for evaluating the generalizability of the ML model to other solid tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f96bc459032f6a187b68e4d09690f24f0e08e461" target='_blank'>
              Predicting overall survival from tumor dynamics metrics using parametric statistical and machine learning models: application to patients with RET-altered solid tumors
              </a>
            </td>
          <td>
            Erick Velasquez, Nastya Kassir, S. Cheeti, Denison Kuruvilla, Rucha Sane, Steve Dang, Dale Miles, James Lu
          </td>
          <td>2024-06-11</td>
          <td>Frontiers in Artificial Intelligence</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Accurate prediction of a new compound's pharmacokinetic (PK) profile is pivotal for the success of drug discovery programs. An initial assessment of PK in preclinical species and humans is typically performed through allometric scaling and mathematical modeling. These methods use parameters estimated from in vitro or in vivo experiments, which although helpful for an initial estimation, require extensive animal experiments. Furthermore, mathematical models are limited by the mechanistic underpinning of the drugs' absorption, distribution, metabolism, and elimination (ADME) which are largely unknown in the early stages of drug discovery. In this work, we propose a novel methodology in which concentration versus time profile of small molecules in rats is directly predicted by machine learning (ML) using structure‐driven molecular properties as input and thus mitigating the need for animal experimentation. The proposed framework initially predicts ADME properties based on molecular structure and then uses them as input to a ML model to predict the PK profile. For the compounds tested, our results demonstrate that PK profiles can be adequately predicted using the proposed algorithm, especially for compounds with Tanimoto score greater than 0.5, the average mean absolute percentage error between predicted PK profile and observed PK profile data was found to be less than 150%. The suggested framework aims to facilitate PK predictions and thus support molecular screening and design earlier in the drug discovery process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78aa04a2b81e41216bb87a34747852fb6128be98" target='_blank'>
              Machine learning framework to predict pharmacokinetic profile of small molecule drugs based on chemical structure
              </a>
            </td>
          <td>
            Nikhil Pillai, Alexandra Abos, Donato Teutonico, P. D. Mavroudis
          </td>
          <td>2024-05-01</td>
          <td>Clinical and Translational Science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Personalized cancer treatment requires a thorough understanding of complex interactions between drugs and cancer cell lines in varying genetic and molecular contexts. To address this, high-throughput screening has been used to generate large-scale drug response data, facilitating data-driven computational models. Such models can capture complex drug–cell line interactions across various contexts in a fully data-driven manner. However, accurately prioritizing the most effective drugs for each cell line still remains a significant challenge. To address this, we developed multiple neural ranking approaches that leverage large-scale drug response data across multiple cell lines from diverse cancer types. Unlike existing approaches that primarily utilize regression and classification techniques for drug response prediction, we formulated the objective of drug selection and prioritization as a drug ranking problem. In this work, we proposed multiple pairwise and listwise neural ranking methods that learn latent representations of drugs and cell lines and then use those representations to score drugs in each cell line via a learnable scoring function. Specifically, we developed neural pairwise and listwise ranking methods, Pair-PushC and List-One on top of the existing methods, pLETORg and ListNet, respectively. Additionally, we proposed a novel listwise ranking method, List-All, that focuses on all the effective drugs instead of the top effective drug, unlike List-One. We also provide an exhaustive empirical evaluation with state-of-the-art regression and ranking baselines on large-scale data sets across multiple experimental settings. Our results demonstrate that our proposed ranking methods mostly outperform the best baselines with significant improvements of as much as 25.6% in terms of selecting truly effective drugs within the top 20 predicted drugs (i.e., hit@20) across 50% test cell lines. Furthermore, our analyses suggest that the learned latent spaces from our proposed methods demonstrate informative clustering structures and capture relevant underlying biological features. Moreover, our comprehensive evaluation provides a thorough and objective comparison of the performance of different methods (including our proposed ones).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c0f5fb3d038f7182389a924d6a9c7f0a1db1816" target='_blank'>
              Improving Anticancer Drug Selection and Prioritization via Neural Learning to Rank
              </a>
            </td>
          <td>
            Vishal Dey, Xia Ning
          </td>
          <td>2024-05-13</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Predicting cancer dynamics under treatment is challenging due to high inter-patient heterogeneity, lack of predictive biomarkers, and sparse and noisy longitudinal data. Mathematical models can summarize cancer dynamics by a few interpretable parameters per patient. Machine learning methods can then be trained to predict the model parameters from baseline covariates, but do not account for uncertainty in the parameter estimates. Instead, hierarchical Bayesian modeling can model the relationship between baseline covariates to longitudinal measurements via mechanistic parameters while accounting for uncertainty in every part of the model. The mapping from baseline covariates to model parameters can be modeled in several ways. A linear mapping simplifies inference but fails to capture nonlinear covariate effects and scale poorly for interaction modeling when the number of covariates is large. In contrast, Bayesian neural networks can potentially discover interactions between covariates automatically, but at a substantial cost in computational complexity. In this work, we develop a hierarchical Bayesian model of subpopulation dynamics that uses baseline covariate information to predict cancer dynamics under treatment, inspired by cancer dynamics in multiple myeloma (MM), where serum M protein is a well-known proxy of tumor burden. As a working example, we apply the model to a simulated dataset and compare its ability to predict M protein trajectories to a model with linear covariate effects. Our results show that the Bayesian neural network covariate effect model predicts cancer dynamics more accurately than a linear covariate effect model when covariate interactions are present. The framework can also be applied to other types of cancer or other time series prediction problems that can be described with a parametric model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ecd81405c7df09ef0fe2550029e319785f4a7b" target='_blank'>
              Prediction of cancer dynamics under treatment using Bayesian neural networks: A simulated study
              </a>
            </td>
          <td>
            E. M. Myklebust, A. Frigessi, Fredrik Schjesvold, J. Foo, K. Leder, Alvaro Kohn-Luque
          </td>
          <td>2024-05-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a46186cfa337e51d671a8ac45c149e5f0bf01b" target='_blank'>
              Data-driven modeling of interrelated dynamical systems
              </a>
            </td>
          <td>
            Yonatan Elul, Eyal Rozenberg, Amit Boyarski, Yael Yaniv, Assaf Schuster, A. M. Bronstein
          </td>
          <td>2024-05-01</td>
          <td>Communications Physics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Modeling dynamical systems is a fundamental task in scientific and engineering fields, often accomplished by applying theory-based models with mathematical equations. Yet, in cases where these equations cannot be established or parameterized properly, theory-based models are not applicable. Instead, a viable alternative is to learn the system dynamics directly from data, for example with deep learning models. However, traditional deep learning models often produce physically inconsistent results and struggle to generalize to unseen data, especially when training data is limited. One solution to this shortcoming is knowledge-guided deep learning, leveraging prior knowledge about the expected behavior of a dynamical system. In this work, we identify and formalize permissible system states, a novel type of prior knowledge that is often available for systems in the context of temporal dynamics modeling. This prior knowledge describes dynamic states that the system is allowed to take during its operation. We propose a knowledge-guided multi-state constraint to encode this type of prior knowledge through a loss function, making it applicable to any deep learning model. This approach allows to create an accurate data-driven model with minimal effort and data requirements. We validate the effectiveness of our method by applying it to model the temporal behavior of a gas turbine in response to an input control signal. Our results indicate that the proposed method reduces the prediction error by up to 40%. In addition to reducing the dependency on extensive training data, our method mitigates training randomness and enhances the consistency of predictions with the expected behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba124a5ffc81be0320f4553459c69561edd30f5" target='_blank'>
              Knowledge-Guided Learning of Temporal Dynamics and its Application to Gas Turbines
              </a>
            </td>
          <td>
            Pawel Bielski, Aleksandr Eismont, Jakob Bach, Florian Leiser, D. Kottonau, Klemens Böhm
          </td>
          <td>2024-06-04</td>
          <td>Proceedings of the 15th ACM International Conference on Future and Sustainable Energy Systems</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Combination therapy is an important direction of continuous exploration in the field of medicine, with the core goals of improving treatment efficacy, reducing adverse reactions, and optimizing clinical outcomes. Machine learning technology holds great promise in improving the prediction of drug synergy combinations. However, most studies focus on single disease-oriented collaborative predictive models or involve excessive feature categories, making it challenging to predict the majority of new drugs. To address these challenges, the DrugSK comprehensive model was developed, which utilizes SMILES-BERT to extract structural information from 3492 drugs and trains on reactions from 48,756 drug combinations. DrugSK is an integrated learning model capable of predicting interactions among various drug categories. First, the primary learner is trained from the initial data set. Random forest, support vector machine, and XGboost model are selected as primary learners and logistic regression as secondary learners. A new data set is then "generated" to train level 2 learners, which can be thought of as a prediction for each model. Finally, the results are filtered using logistic regression. Furthermore, the combination of the new antibacterial drug Drafloxacin with other antibacterial agents was tested. The synergistic effect of Drafloxacin and Isavuconazonium in the fight against Candida albicans has been confirmed, providing enlightenment for the clinical treatment of skin infection. DrugSK's prediction is accurate in practical application and can also predict the probability of the outcome. In addition, the tendency of Drafloxacin and antifungal drugs to be synergistic was found. The development of DrugSK will provide a new blueprint for predicting drug combination synergies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055325b217f8174f457e2c50697ff15bdc4f66ea" target='_blank'>
              DrugSK: A Stacked Ensemble Learning Framework for Predicting Drug Combinations of Multiple Diseases.
              </a>
            </td>
          <td>
            Siqi Chen, Nana Gao, Chunzhi Li, Fei Zhai, Xiwei Jiang, Peng Zhang, Jibin Guan, Kefeng Li, Rongwu Xiang, Guixia Ling
          </td>
          <td>2024-06-20</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Analyzing the drug response at the cellular level is crucial for identifying biomarkers and understanding the mechanisms of resistance. Although studies on the drug response of individual cells can provide novel insights into tumor heterogeneity, pharmacogenomic data related to single-cell (SC) RNA sequencing is often limited. Transfer learning provides a promising approach to translate the knowledge of drug response from bulk cell lines to SC analysis, potentially providing an effective solution to this challenge. Previous studies often use data from single drug-cell lines to pre-train specific models and adapt the models on SC datasets, which lack pharmacogenomic information from other drugs and hinder model generalization. In this work, we introduce MetaSCDrug as a unified meta pre-training framework that integrates molecular information with transcriptomic data to simultaneously modeling cellular heterogeneity in response to multiple pre-trained drugs and generalize to unseen drugs. Our model requires only one pre-training session, followed by fine-tuning on multiple single-cell datasets by few-shot learning, achieving an average of 4.58% accuracy increase in drug response prediction compared to the baselines. Furthermore, our meta pre-training strategy effectively captures transcriptome heterogeneity in the generalization of unseen drugs, achieving a 20% improvement over the model without meta pre-training. Case studies of our framework highlight its capability to identify critical genes for resistance, providing a method for exploring drug action pathways and understanding resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde20ee4a0135620d4f36e68f1c9aef919565e77" target='_blank'>
              MetaSCDrug: Meta-Transfer Learning for Single-Cell-Level Drug Response Prediction from Transcriptome and Molecular Representations
              </a>
            </td>
          <td>
            Shuang Ge, Shuqing Sun, Yiming Ren, Huan Xu, Zhixiang Ren
          </td>
          <td>2024-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Physics informed neural networks have been gaining popularity due to their unique ability to incorporate physics laws into data-driven models, ensuring that the predictions are not only consistent with empirical data but also align with domain-specific knowledge in the form of physics equations. The integration of physics principles enables the method to require less data while maintaining the robustness of deep learning in modeling complex dynamical systems. However, current PINN frameworks are not sufficiently mature for real-world ODE systems, especially those with extreme multi-scale behavior such as mosquito population dynamical modelling. In this research, we propose a PINN framework with several improvements for forward and inverse problems for ODE systems with a case study application in modelling the dynamics of mosquito populations. The framework tackles the gradient imbalance and stiff problems posed by mosquito ordinary differential equations. The method offers a simple but effective way to resolve the time causality issue in PINNs by gradually expanding the training time domain until it covers entire domain of interest. As part of a robust evaluation, we conduct experiments using simulated data to evaluate the effectiveness of the approach. Preliminary results indicate that physics-informed machine learning holds significant potential for advancing the study of ecological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf1c97b75409ff1ee6e5c4298dd05295c9b772df" target='_blank'>
              Adapting Physics-Informed Neural Networks To Optimize ODEs in Mosquito Population Dynamics
              </a>
            </td>
          <td>
            D. V. Cuong, Branislava Lali'c, Mina Petri'c, Binh Nguyen, M. Roantree
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background The potential benefits of drug combination synergy in cancer medicine are significant, yet the risks must be carefully managed due to the possibility of increased toxicity. Although artificial intelligence applications have demonstrated notable success in predicting drug combination synergy, several key challenges persist: (1) Existing models often predict average synergy values across a restricted range of testing dosages, neglecting crucial dose amounts and the mechanisms of action of the drugs involved. (2) Many graph-based models rely on static protein-protein interactions, failing to adapt to dynamic and context-dependent networks. This limitation constrains the applicability of current methods. Results We introduced SAFER, a Sub-hypergraph Attention-based graph model, addressing these issues by incorporating complex relationships among biological knowledge networks and considering dosing effects on subject-specific networks. SAFER outperformed previous models on the benchmark and the independent test set. The analysis of subgraph attention weight for the lung cancer cell line highlighted JAK-STAT signaling pathway, PRDM12, ZNF781, and CDC5L that have been implicated in lung fibrosis. Conclusions SAFER presents an interpretable framework designed to identify drug-responsive signals. Tailored for comprehending dose effects on subject-specific molecular contexts, our model uniquely captures dose-level drug combination responses. This capability unlocks previously inaccessible avenues of investigation compared to earlier models. Finally, the SAFER framework can be leveraged by future inquiries to investigate molecular networks that uniquely characterize individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/500c4f42640ef697e84b8ae2a633d54a79da30c0" target='_blank'>
              SAFER: sub-hypergraph attention-based neural network for predicting effective responses to dose combinations
              </a>
            </td>
          <td>
            yiching tang, Rongbin Li, Jing Tang, W. J. Zheng, Xiaoqian Jiang
          </td>
          <td>2024-04-30</td>
          <td>Research Square</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Patients diagnosed with advanced metastatic colorectal cancer (mCRC) often exhibit heterogeneous disease progression and face poor survival prospects. In order to comprehensively analyze the varied treatment responses among individuals and the challenge of tumor recurrence resistant to drugs in advanced mRCR, we developed a novel quantitative cancer-immunity cycle (QCIC) model. The proposed model was meticulously crafted utilizing a blend of differential equations and randomized modeling techniques to quantitatively elucidate the intricate mechanisms governing the cancer-immunity cycle and forecast tumor dynamics under different treatment modalities. Furthermore, by integrating diverse clinical datasets and rigorous model analyses, we introduced two pivotal concepts: the treatment response index (TRI) and the death probability function (DPF). These concepts are crucial tools for translating model predictions into clinically relevant evaluation indexes. Using virtual patient technology, we extrapolated tumor predictive biomarkers from the model to predict survival outcomes for mCRC patients. Our findings underscore the significance of tumor-infiltrating CD8+CTL cell density as a key predictive biomarker for short-term treatment responses in advanced mCRC while emphasizing the potential predict value of the tumor-infiltrating CD4+Th1/Treg ratio in determining patient survival. This study presents a pioneering methodology bridging the divide between diverse clinical data sources and the generation of virtual patients, offering invaluable insights into understanding inter-individual treatment variations and forecasting survival outcomes in mCRC patients. Author summary This study introduces a sophisticated modeling approach to delineate the intricate dynamics of tumor-immune interactions within the cancer-immunity cycle. Utilizing a multi-compartmental, multi-scale, multi-dimensional differential equation model, we quantified the complex interplay between tumor cells, immune cells, cytokines, and chemokines. By integrating virtual patient technology, we have conducted in silico clinical trials that accurately predict disease progression across multiple treatment modalities for cancer patients, particularly in advanced mCRC. Through the combination of differential equations and randomized modeling, we effectively captured the diverse treatment responses and clinical manifestations of drug-resistant tumor recurrence. Furthermore, we explored the pivotal role of predictive biomarkers in determining patients’ survival prognosis, offering insights for tailored therapeutic strategies. Importantly, our computational framework holds promise for the extension to the investigation of patients with other tumor types, thus contributing to the personalized investigation of patients with other tumor types and cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a0274da7051ba17dd2effa0a322076737f25c2" target='_blank'>
              Quantitative cancer-immunity cycle modeling for predicting disease progression in advanced metastatic colorectal cancer
              </a>
            </td>
          <td>
            Chenghang Li, Yongchang Wei, J. Lei
          </td>
          <td>2024-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The proteins within the human epidermal growth factor receptor (EGFR) family, members of the tyrosine kinase receptor family, play a pivotal role in the molecular mechanisms driving the development of various tumors. Tyrosine kinase inhibitors, key compounds in targeted therapy, encounter challenges in cancer treatment due to emerging drug resistance mutations. Consequently, machine learning has undergone significant evolution to address the challenges of cancer drug discovery related to EGFR family proteins. However, the application of deep learning in this area is hindered by inherent difficulties associated with small-scale data, particularly the risk of overfitting. Moreover, the design of a model architecture that facilitates learning through multi-task and transfer learning, coupled with appropriate molecular representation, poses substantial challenges. In this study, we introduce GraphEGFR, a deep learning regression model designed to enhance molecular representation and model architecture for predicting the bioactivity of inhibitors against both wild-type and mutant EGFR family proteins. GraphEGFR integrates a graph attention mechanism for molecular graphs with deep and convolutional neural networks for molecular fingerprints. We observed that GraphEGFR models employing multi-task and transfer learning strategies generally achieve predictive performance comparable to existing competitive methods. The integration of molecular graphs and fingerprints adeptly captures relationships between atoms and enables both global and local pattern recognition. We further validated potential multi-targeted inhibitors for wild-type and mutant HER1 kinases, exploring key amino acid residues through molecular dynamics simulations to understand molecular interactions. This predictive model offers a robust strategy that could significantly contribute to overcoming the challenges of developing deep learning models for drug discovery with limited data and exploring new frontiers in multi-targeted kinase drug discovery for EGFR family proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a60c5741929cb3c480ef85e0d599beecfa969e6" target='_blank'>
              GraphEGFR: Multi-task and transfer learning based on molecular graph attention mechanism and fingerprints improving inhibitor bioactivity prediction for EGFR family proteins on data scarcity.
              </a>
            </td>
          <td>
            Bundit Boonyarit, Nattawin Yamprasert, Pawit Kaewnuratchadasorn, Jiramet Kinchakawat, Chanatkran Prommin, T. Rungrotmongkol, Sarana Nutanong
          </td>
          <td>2024-05-07</td>
          <td>Journal of computational chemistry</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Drug synergy therapy is a promising strategy for cancer treatment. However, the extensive variety of available drugs and the time-intensive process of determining effective drug combinations through clinical trials pose significant challenges. It requires a reliable method for the rapid and precise selection of drug synergies. In response, various computational strategies have been developed for predicting drug synergies, yet the exploitation of heterogeneous biological network features remains underexplored. In this study, we construct a heterogeneous graph that encompasses diverse biological entities and interactions, utilizing rich data sets from sources, such as DrugCombDB, PubChem, UniProt, and cancer cell line encyclopedia (CCLE). We initialize node feature representations and introduce a novel virtual node to enhance drug representation. Our proposed method, the heterogeneous graph attention network for drug-drug synergy prediction (HANSynergy), has been experimentally validated to demonstrate that the heterogeneous graph attention network can extract key node features, efficiently harness the diversity of information, and further enhance network functionality through the incorporation of a multihead attention mechanism. In the comparative experiment, the highest accuracy (Acc) and area under the curve (AUC) are 0.877 and 0.947, respectively, in DrugCombDB_early data set, demonstrating the superiority of HANSynergy over the competing methods. Moreover, protein-protein interactions are important in understanding the mechanism of action of drugs. The heterogeneous attention mechanism facilitates protein-protein interaction analysis. By analyzing the changes of attention weight before and after heterogeneous network training, we investigated proteins that may be associated with drug combinations. Additionally, case studies align our findings with existing research, underscoring the potential of HANSynergy in drug synergy prediction. This advancement not only contributes to the burgeoning field of drug synergy prediction but also holds the potential to provide valuable insights and uncover new drug synergies for combating cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef455b797d012d55321cf50d008df1f6c786323" target='_blank'>
              HANSynergy: Heterogeneous Graph Attention Network for Drug Synergy Prediction
              </a>
            </td>
          <td>
            Ning Cheng, Li Wang, Yiping Liu, Bosheng Song, Changsong Ding
          </td>
          <td>2024-05-06</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hybrid dynamical systems are prevalent in science and engineering to express complex systems with continuous and discrete states. To learn the laws of systems, all previous methods for equation discovery in hybrid systems follow a two-stage paradigm, i.e. they first group time series into small cluster fragments and then discover equations in each fragment separately through methods in non-hybrid systems. Although effective, these methods do not fully take advantage of the commonalities in the shared dynamics of multiple fragments that are driven by the same equations. Besides, the two-stage paradigm breaks the interdependence between categorizing and representing dynamics that jointly form hybrid systems. In this paper, we reformulate the problem and propose an end-to-end learning framework, i.e. Amortized Equation Discovery (AMORE), to jointly categorize modes and discover equations characterizing the dynamics of each mode by all segments of the mode. Experiments on four hybrid and six non-hybrid systems show that our method outperforms previous methods on equation discovery, segmentation, and forecasting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f40610f6f31195a8a68f3802b6cc12d6503f9bae" target='_blank'>
              Amortized Equation Discovery in Hybrid Dynamical Systems
              </a>
            </td>
          <td>
            Yongtuo Liu, Sara Magliacane, Miltiadis Kofinas, E. Gavves
          </td>
          <td>2024-06-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Systems biology tackles the challenge of understanding the high complexity in the internal regulation of homeostasis in the human body through mathematical modelling. These models can aid in the discovery of disease mechanisms and potential drug targets. However, on one hand the development and validation of knowledge-based mechanistic models is time-consuming and does not scale well with increasing features in medical data. On the other hand, more data-driven approaches such as machine learning models require large volumes of data to produce generalizable models. The integration of neural networks and mechanistic models, forming universal differential equation (UDE) models, enables the automated learning of unknown model terms with less data than the neural network alone. Nevertheless, estimating parameters for these hybrid models remains difficult with sparse data and limited sampling durations that are common in biological applications. In this work, we propose the use of physiology-informed regularization, penalizing biologically implausible model behavior to guide the UDE towards more physiologically plausible regions of the solution space. In a simulation study we show that physiology-informed regularization not only results in a more accurate forecasting of model behaviour, but also supports training with less data. We also applied this technique to learn a representation of the rate of glucose appearance in the glucose minimal model using meal response data measured in healthy people. In that case, the inclusion of regularization reduces variability between UDE-embedded neural networks that were trained from different initial parameter guesses. Author summary Systems biology concerns the modelling and analysis of biological processes, by viewing these as interconnected systems. Modelling is typically done either using mechanistic differential equations that are derived from experiments and known biology, or using machine learning on large biological datasets. While mathematical modelling from biological experiments can provide useful insights with limited data, building and validating these models takes a long time and often requires highly invasive measurements in humans. Efforts to combine this classical technique with machine learning have resulted in a framework termed universal differential equations, where the model equations contain a neural network to describe unknown biological interactions. While these methods have shown success in numerous fields, applications in biology are more challenging due to limited data-availability, high data sparsity. In this work, we have introduced physiology-informed regularization to overcome these instabilities and to constrain the model to biologically plausible behavior. Our results show that by using physiology-informed regularization, we can accurately predict future unseen observations in a simulated example, with much more limited data than a similar model without regularization. Additionally, we show an application of this technique on human data, applying a neural network to learn the appearance of glucose in the blood plasma after a meal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf0dda21d3a8fbe599f98eed4fe943aa342f41" target='_blank'>
              Physiology-informed regularization enables training of universal differential equation systems for biological applications
              </a>
            </td>
          <td>
            Max de Rooij, Balázs Erdős, N. V. van Riel, Shauna D. O’Donovan
          </td>
          <td>2024-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Representation learning for tumor gene expression (GEx) data with deep neural networks is limited by the large gene feature space and the scarcity of available clinical and preclinical data. The translation of the learned representation between these data sources is further hindered by inherent molecular differences. To address these challenges, we propose GExMix (Gene Expression Mixup), a data augmentation method, which extends the Mixup concept to generate training samples accounting for the imbalance in both data classes and data sources. We leverage the GExMix-augmented training set in encoder-decoder models to learn a GEx latent representation. Subsequently, we combine the learned representation with drug chemical features in a dual-objective enhanced gene-centric drug response prediction, i.e., reconstruction of GEx latent embeddings and drug response classification. This dual-objective design strategically prioritizes gene-centric information to enhance the final drug response prediction. We demonstrate that augmenting training samples improves the GEx representation, benefiting the gene-centric drug response prediction model. Our findings underscore the effectiveness of our proposed GExMix in enriching GEx data for deep neural networks. Moreover, our proposed gene-centricity further improves drug response prediction when translating preclinical to clinical datasets. This highlights the untapped potential of the proposed framework for GEx data analysis, paving the way toward precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d8245b02f26c5698605a2d1bfdadb5793c2f155" target='_blank'>
              Enhancing Gene Expression Representation and Drug Response Prediction with Data Augmentation and Gene Emphasis
              </a>
            </td>
          <td>
            Diyuan Lu, Daksh P.S. Pamar, AJ Ohnmacht, G. Kutkaite, Michael P. Menden
          </td>
          <td>2024-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Data-driven modelling and scientific machine learning have been responsible for significant advances in determining suitable models to describe data. Within dynamical systems, neural ordinary differential equations (ODEs), where the system equations are set to be governed by a neural network, have become a popular tool for this challenge in recent years. However, less emphasis has been placed on systems that are only partially-observed. In this work, we employ a hybrid neural ODE structure, where the system equations are governed by a combination of a neural network and domain-specific knowledge, together with symbolic regression (SR), to learn governing equations of partially-observed dynamical systems. We test this approach on two case studies: A 3-dimensional model of the Lotka-Volterra system and a 5-dimensional model of the Lorenz system. We demonstrate that the method is capable of successfully learning the true underlying governing equations of unobserved states within these systems, with robustness to measurement noise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5f91324b2ca816ed10e0d9a1d6565fb12a4a1f" target='_blank'>
              Learning Governing Equations of Unobserved States in Dynamical Systems
              </a>
            </td>
          <td>
            Gevik Grigorian, Sandip V. George, S. Arridge
          </td>
          <td>2024-04-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Based on multi-omics data and drug information, predicting the response of cancer cell lines to drugs is a crucial area of research in modern oncology, as it can promote the development of personalized treatments. Despite the promising performance achieved by existing models, most of them overlook the variations among different omics and lack effective integration of multi-omics data. Moreover, the explicit modeling of cell line/drug attribute and cell line-drug association has not been thoroughly investigated in existing approaches. To address these issues, we propose RedCDR, a dual relation distillation model for cancer drug response (CDR) prediction. Specifically, a parallel dual-branch architecture is designed to enable both the independent learning and interactive fusion feasible for cell line/drug attribute and cell line-drug association information. To facilitate the adaptive interacting integration of multi-omics data, the proposed multi-omics encoder introduces the multiple similarity relations between cell lines and takes the importance of different omics data into account. To accomplish knowledge transfer from the two independent attribute and association branches to their fusion, a dual relation distillation mechanism consisting of representation distillation and prediction distillation is presented. Experiments conducted on the GDSC and CCLE datasets show that RedCDR outperforms previous state-of-the-art approaches in CDR prediction. The source code is available at https://github.com/mhxu1998/RedCDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1d774b32aa1d353908c13e835eb7b0507d698d" target='_blank'>
              RedCDR: Dual Relation Distillation for Cancer Drug Response Prediction.
              </a>
            </td>
          <td>
            Muhao Xu, Zhenfeng Zhu, Yawei Zhao, Kunlun He, Qinghua Huang, Yao Zhao
          </td>
          <td>2024-05-22</td>
          <td>IEEE/ACM transactions on computational biology and bioinformatics</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Neural network models have become a popular machine-learning technique for the toxicity prediction of chemicals. However, due to their complex structure, it is difficult to understand predictions made by these models which limits confidence. Current techniques to tackle this problem such as SHAP or integrated gradients provide insights by attributing importance to the input features of individual compounds. While these methods have produced promising results in some cases, they do not shed light on how representations of compounds are transformed in hidden layers, which constitute how neural networks learn. We present a novel technique to interpret neural networks which identifies chemical substructures in training data found to be responsible for the activation of hidden neurons. For individual test compounds, the importance of hidden neurons is determined, and the associated substructures are leveraged to explain the model prediction. Using structural alerts for mutagenicity from the Derek Nexus expert system as ground truth, we demonstrate the validity of the approach and show that model explanations are competitive with and complementary to explanations obtained from an established feature attribution method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4942ed910678bf3396a6c139b299a1276cbf3ca8" target='_blank'>
              Interpreting Neural Network Models for Toxicity Prediction by Extracting Learned Chemical Features
              </a>
            </td>
          <td>
            Moritz Walter, Samuel J. Webb, Valerie J Gillet
          </td>
          <td>2024-04-30</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic cancer is often diagnosed at an advanced stage when treatment options are limited. Being one of the deadliest cancers that mandates longer medication and treatment phases, there is an inevitable need to have the knowledge of drug response of anti-pancreatic cancer drugs before it is recommended for a patient. AI-driven drug response prediction has proven potential to personalize treatment strategies, improve therapeutic outcomes, and reduce adverse effects and treatment costs for cancer patients. In this research work, we have accounted for the use of different drug descriptors and their core structures known as scaffolds along with three cell line features, chromatin profiling, reverse phase protein array, and metabolomics data to build a feature engineered dataset for drug response prediction tested on various computational learning models. The 53 unique drugs against 18 unique pancreatic cancer cell lines were taken as the raw dataset. The initial dataset having a large dimension was feature selected using an ensemble method derived from five different techniques. The dataset was evaluated on various computational methods and an accuracy of 89% was achieved using the TabNet architecture. Furthermore, the common scaffolds that were persistently found among the drugs that possess high IC50-valued drug clusters were also recorded.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f23eae37f7bd87a4aefe75268ff15ffb52910e9" target='_blank'>
              Tailoring therapies: a frontier approach to pancreatic cancer with AI-driven multiomics profiling
              </a>
            </td>
          <td>
            Janiel Jawahar, Paramasivan Selvi Rajendran
          </td>
          <td>2024-08-01</td>
          <td>Indonesian Journal of Electrical Engineering and Computer Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Clinical trial outcome prediction seeks to estimate the likelihood that a clinical trial will successfully reach its intended endpoint. This process predominantly involves the development of machine learning models that utilize a variety of data sources such as descriptions of the clinical trials, characteristics of the drug molecules, and specific disease conditions being targeted. Accurate predictions of trial outcomes are crucial for optimizing trial planning and prioritizing investments in a drug portfolio. While previous research has largely concentrated on small-molecule drugs, there is a growing need to focus on biologics-a rapidly expanding category of therapeutic agents that often lack the well-defined molecular properties associated with traditional drugs. Additionally, applying conventional methods like graph neural networks to biologics data proves challenging due to their complex nature. To address these challenges, we introduce the Language Interaction Network (LINT), a novel approach that predicts trial outcomes using only the free-text descriptions of the trials. We have rigorously tested the effectiveness of LINT across three phases of clinical trials, where it achieved ROC-AUC scores of 0.770, 0.740, and 0.748 for phases I, II, and III, respectively, specifically concerning trials involving biologic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd1760efcbfaf72e59a992776dc70c19c88d7e25" target='_blank'>
              Language Interaction Network for Clinical Trial Approval Estimation
              </a>
            </td>
          <td>
            Chufan Gao, Tianfan Fu, Jimeng Sun
          </td>
          <td>2024-04-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Drug‐drug interaction (DDI) event prediction is a challenging problem, and accurate prediction of DDI events is critical to patient health and new drug development. Recently, many machine learning‐based techniques have been proposed for predicting DDI events. However, most of the existing methods do not effectively integrate the multidimensional features of drugs and provide poor mitigation of noise to get effective feature information. To address these limitations, we propose a DDI‐Transform neural network framework for DDI event prediction. In DDI‐Transform, we design a drug structure information feature extraction module and a drug bind‐protein feature extraction module to obtain multidimensional feature information. A stack of DDI‐Transform layers in the DDI‐Transform network module are then used for adaptive learning, thus adaptively selecting the effective feature information for prediction. The results show that DDI‐Transform can accurately predict DDI events and outperform the state‐of‐the‐art models. Results on different scale datasets confirm the robustness of the method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b87e33fd2a4bef618fea9c12d74ad353d0d4cfe" target='_blank'>
              DDI‐Transform: A neural network for predicting drug‐drug interaction events
              </a>
            </td>
          <td>
            Jiaming Su, Ying Qian
          </td>
          <td>2024-04-29</td>
          <td>Quantitative Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Irregularly sampled time series with missing values are often observed in multiple real-world applications such as healthcare, climate and astronomy. They pose a significant challenge to standard deep learn- ing models that operate only on fully observed and regularly sampled time series. In order to capture the continuous dynamics of the irreg- ular time series, many models rely on solving an Ordinary Differential Equation (ODE) in the hidden state. These ODE-based models tend to perform slow and require large memory due to sequential operations and a complex ODE solver. As an alternative to complex ODE-based mod- els, we propose a family of models called Functional Latent Dynamics (FLD). Instead of solving the ODE, we use simple curves which exist at all time points to specify the continuous latent state in the model. The coefficients of these curves are learned only from the observed values in the time series ignoring the missing values. Through extensive experi- ments, we demonstrate that FLD achieves better performance compared to the best ODE-based model while reducing the runtime and memory overhead. Specifically, FLD requires an order of magnitude less time to infer the forecasts compared to the best performing forecasting model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2a5b8158db29854109275cb5c3fbcf47c080c1c" target='_blank'>
              Functional Latent Dynamics for Irregularly Sampled Time Series Forecasting
              </a>
            </td>
          <td>
            Christian Klötergens, Vijaya Krishna Yalavarthi, Maximilian Stubbemann, Lars Schmidt-Thieme
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The mammalian genome organizes into complex three-dimensional structures, where interactions among chromatin regulatory elements play a pivotal role in mediating biological functions, highlighting the significance of genomic region interactions in biological research. Traditional biological sequencing techniques like HiC and MicroC, commonly employed to estimate these interactions, are resource-intensive and time-consuming, especially given the vast array of cell lines and tissues involved. With the advent of advanced machine learning (ML) methodologies, there has been a push towards developing ML models to predict genomic interactions. However, while these models excel in predicting interactions for cell lines similar to their training data, they often fail to generalize across distantly related cell lines or accurately predict interactions specific to certain cell lines. Identifying the potential oversight of excluding example genomic region interaction information from model inputs as a fundamental limitation, this paper introduces GRACHIP, a model rooted in graph neural network technology aiming to address this issue by incorporating detailed interaction information as a hint. Through extensive testing across various cell lines, GRACHIP not only demonstrates exceptional accuracy in predicting chromatin interaction intensity but showcases remarkable generalizability to cell lines not encountered during training. Consequently, GRACHIP emerges as a potent research tool, offering a viable alternative to conventional sequencing methods for analyzing the interactions and three-dimensional organization of mammalian genomes, thus alleviating the dependency on expensive and time-consuming biological sequencing techniques. It also offers an alternative way for researchers to investigate 3D chromatin interactions and simulate their changes in model systems to test their hypotheses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92aaa802a4f13bf6fc0e8fce14687506f2b37a22" target='_blank'>
              Improving Cell-type-specific 3D Genome Architectures Prediction Leveraging Graph Neural Networks
              </a>
            </td>
          <td>
            Ruoyun Wang, Weicheng Ma, Aryan Soltani Mohammadi, Saba Shahsavari, Soroush Vosoughi, Xiaofeng Wang
          </td>
          <td>2024-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Motivation Anti-cancer drug response prediction (DRP) using cancer cell lines plays a vital role in stratified medicine and drug discovery. Recently there has been a surge of new deep learning (DL) models for DRP that show promising performance improvements. However, different models use different input data modalities and neural network architectures making it hard to find the source of these improvements. Results We consider three DL DRP models with reported state-of-the-art performance and use genomics or transcriptomics profiles of cell lines and chemical profiles of drugs as input data. We also construct null hypothesis benchmarks that simply leverage patterns in truth values and thus do not use omics or chemical profiles. By experimenting with these DL models and comparing them with our null hypothesis benchmarks we show that for multiple testing types, much or all of the current reported performance is a property of the training truth values. Furthermore, we show that for cancer blind testing (testing on unseen cell lines) no performance comes from drug features, instead, performance is due to the transcriptomics cell line profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c5705a239471a8f4b914d01d1c42bcefb5e083" target='_blank'>
              Understanding the Sources of Performance in Deep Learning Drug Response Prediction Models
              </a>
            </td>
          <td>
            Nikhil Branson, Pedro R. Cutillas, Conrad Besseant
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Neural Ordinary Differential Equations typically struggle to generalize to new dynamical behaviors created by parameter changes in the underlying system, even when the dynamics are close to previously seen behaviors. The issue gets worse when the changing parameters are unobserved, i.e., their value or influence is not directly measurable when collecting data. We introduce Neural Context Flow (NCF), a framework that encodes said unobserved parameters in a latent context vector as input to a vector field. NCFs leverage differentiability of the vector field with respect to the parameters, along with first-order Taylor expansion to allow any context vector to influence trajectories from other parameters. We validate our method and compare it to established Multi-Task and Meta-Learning alternatives, showing competitive performance in mean squared error for in-domain and out-of-distribution evaluation on the Lotka-Volterra, Glycolytic Oscillator, and Gray-Scott problems. This study holds practical implications for foundational models in science and related areas that benefit from conditional neural ODEs. Our code is openly available at https://github.com/ddrous/ncflow.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c738e0450649ed6c04ff7e82d993987c381e35a" target='_blank'>
              Neural Context Flows for Learning Generalizable Dynamical Systems
              </a>
            </td>
          <td>
            Roussel Desmond Nzoyem, David A.W. Barton, Tom Deakin
          </td>
          <td>2024-05-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The evaluation of drug-induced Torsades de pointes (TdP) risks is crucial in drug safety assessment. In this study, we discuss machine learning approaches in the prediction of drug-induced TdP risks using preclinical data. Specifically, a random forest model was trained on the dataset generated by the rabbit ventricular wedge assay. The model prediction performance was measured on 28 drugs from the Comprehensive In Vitro Proarrhythmia Assay initiative. Leave-one-drug-out cross-validation provided an unbiased estimation of model performance. Stratified bootstrap revealed the uncertainty in the asymptotic model prediction. Our study validated the utility of machine learning approaches in predicting drug-induced TdP risks from preclinical data. Our methods can be extended to other preclinical protocols and serve as a supplementary evaluation in drug safety assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aff64c32acf59cf216b66c0fc2cd3375c21ab4" target='_blank'>
              Drug safety assessment by machine learning models.
              </a>
            </td>
          <td>
            Nan Miles Xi, Dalong Patrick Huang
          </td>
          <td>2024-06-18</td>
          <td>Journal of biopharmaceutical statistics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer, a leading cause of death globally, occurs due to genomic changes and manifests heterogeneously across patients. To advance research on personalized treatment strategies, the effectiveness of various drugs on cells derived from cancers (`cell lines') is experimentally determined in laboratory settings. Nevertheless, variations in the distribution of genomic data and drug responses between cell lines and humans arise due to biological and environmental differences. Moreover, while genomic profiles of many cancer patients are readily available, the scarcity of corresponding drug response data limits the ability to train machine learning models that can predict drug response in patients effectively. Recent cancer drug response prediction methods have largely followed the paradigm of unsupervised domain-invariant representation learning followed by a downstream drug response classification step. Introducing supervision in both stages is challenging due to heterogeneous patient response to drugs and limited drug response data. This paper addresses these challenges through a novel representation learning method in the first phase and weak supervision in the second. Experimental results on real patient data demonstrate the efficacy of our method (WISER) over state-of-the-art alternatives on predicting personalized drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28ee9b9ea8cc0a9f28f3c21fa707bc8f6007d90" target='_blank'>
              WISER: Weak supervISion and supErvised Representation learning to improve drug response prediction in cancer
              </a>
            </td>
          <td>
            Kumar Shubham, A. Jayagopal, Syed Mohammed Danish, AP Prathosh, V. Rajan
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d88d1ee244b5bbb95c70dc410ecf9f282eee608" target='_blank'>
              Prediction of anticancer drug sensitivity using an interpretable model guided by deep learning
              </a>
            </td>
          <td>
            Weixiong Pang, Ming Chen, Yufang Qin
          </td>
          <td>2024-05-09</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Drug development is a lengthy process with a high failure rate. Increasingly, machine learning is utilized to facilitate the drug development processes. These models aim to enhance our understanding of drug characteristics, including their activity in biological contexts. However, a major challenge in drug response (DR) prediction is model interpretability as it aids in the validation of findings. This is important in biomedicine, where models need to be understandable in comparison with established knowledge of drug interactions with proteins. drGAT, a graph deep learning model, leverages a heterogeneous graph composed of relationships between proteins, cell lines, and drugs. drGAT is designed with two objectives: DR prediction as a binary sensitivity prediction and elucidation of drug mechanism from attention coefficients. drGAT has demonstrated superior performance over existing models, achieving 78% accuracy (and precision), and 76% F1 score for 269 DNA-damaging compounds of the NCI60 drug response dataset. To assess the model’s interpretability, we conducted a review of drug-gene co-occurrences in Pubmed abstracts in comparison to the top 5 genes with the highest attention coefficients for each drug. We also examined whether known relationships were retained in the model by inspecting the neighborhoods of topoisomerase-related drugs. For example, our model retained TOP1 as a highly weighted predictive feature for irinotecan and topotecan, in addition to other genes that could potentially be regulators of the drugs. Our method can be used to accurately predict sensitivity to drugs and may be useful in the identification of biomarkers relating to the treatment of cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23626c19de7aea3fe09bc589778f03d93292a717" target='_blank'>
              drGAT: Attention-Guided Gene Assessment of Drug Response Utilizing a Drug-Cell-Gene Heterogeneous Network
              </a>
            </td>
          <td>
            Yoshitaka Inoue, Hunmin Lee, Tianfan Fu, Augustin Luna
          </td>
          <td>2024-05-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Aim: Despite the advances in oncology and cancer treatment over the past decades, cancer remains one of the deadliest diseases. This study focuses on further understanding the complex nature of cancer by using mathematical tumor modeling to understand, capture as best as possible, and describe its complex dynamics under chemotherapy treatment. Materials and Methods: Focusing on autoregressive with exogenous inputs, i.e., ARX, and adaptive neuro-fuzzy inference system, i.e., ANFIS, models, this work investigates tumor growth dynamics under both single and combination anticancer agent chemotherapy treatments using chemotherapy treatment data on xenografted mice. Results: Four ARX and ANFIS models for tumor growth inhibition were developed, estimated, and evaluated, demonstrating a strong correlation with tumor weight data, with ANFIS models showing superior performance in handling the multi-agent tumor growth complexities. These findings suggest potential clinical applications of the ANFIS models through further testing. Both types of models were also tested for their prediction capabilities across different chemotherapy schedules, with accurate forecasting of tumor growth up to five days in advance. The use of adaptive prediction and sliding (moving) data window techniques allowed for continuous model updating, ensuring more robust predictive capabilities. However, long-term forecasting remains a challenge, with accuracy declining over longer prediction horizons. Conclusion: While ANFIS models showed greater reliability in predictions, the simplicity and rapid deployment of ARX models offer advantages in situations requiring immediate approximations. Future research with larger, more diverse datasets and by exploring varying model complexities is recommended to improve the models’ reliability and applicability in clinical decision-making, thereby aiding the development of personalized chemotherapy regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e4427fcccf3edcdd11ca0d2dc97f640c7df5d5" target='_blank'>
              A Comparative Analysis of ARX and ANFIS Models for Tumor Growth Prediction Under Single and Multi-agent Chemotherapy
              </a>
            </td>
          <td>
            S. Liliopoulos, G. Stavrakakis, Konstantinos S Dimas
          </td>
          <td>2024-05-31</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Introduction Mpox (formerly monkeypox) is an infectious disease that spreads mostly through direct contact with infected animals or people's blood, bodily fluids, or cutaneous or mucosal lesions. In light of the global outbreak that occurred in 2022–2023, in this paper, we analyzed global Mpox univariate time series data and provided a comprehensive analysis of disease outbreaks across the world, including the USA with Brazil and three continents: North America, South America, and Europe. The novelty of this study is that it delved into the Mpox time series data by implementing the data-driven methods and a mathematical model concurrently—an aspect not typically addressed in the existing literature. The study is also important because implementing these models concurrently improved our predictions' reliability for infectious diseases. Methods We proposed a traditional compartmental model and also implemented deep learning models (1D- convolutional neural network (CNN), long-short term memory (LSTM), bidirectional LSTM (BiLSTM), hybrid CNN-LSTM, and CNN-BiLSTM) as well as statistical time series models: autoregressive integrated moving average (ARIMA) and exponential smoothing on the Mpox data. We also employed the least squares method fitting to estimate the essential epidemiological parameters in the proposed deterministic model. Results The primary finding of the deterministic model is that vaccination rates can flatten the curve of infected dynamics and influence the basic reproduction number. Through the numerical simulations, we determined that increased vaccination among the susceptible human population is crucial to control disease transmission. Moreover, in case of an outbreak, our model showed the potential for epidemic control by adjusting the key epidemiological parameters, namely the baseline contact rate and the proportion of contacts within the human population. Next, we analyzed data-driven models that contribute to a comprehensive understanding of disease dynamics in different locations. Additionally, we trained models to provide short-term (eight-week) predictions across various geographical locations, and all eight models produced reliable results. Conclusion This study utilized a comprehensive framework to investigate univariate time series data to understand the dynamics of Mpox transmission. The prediction showed that Mpox is in its die-out situation as of July 29, 2023. Moreover, the deterministic model showed the importance of the Mpox vaccination in mitigating the Mpox transmission and highlighted the significance of effectively adjusting key epidemiological parameters during outbreaks, particularly the contact rate in high-risk groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302a4b8ee78aaa9f3ebf07b199bea13110d35c6e" target='_blank'>
              Exploring the dynamics of monkeypox transmission with data-driven methods and a deterministic model
              </a>
            </td>
          <td>
            Haridas K. Das
          </td>
          <td>2024-05-22</td>
          <td>Frontiers in Epidemiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Targeted monotherapies for cancer often fail due to inherent or acquired drug resistance. By aiming at multiple targets simultaneously, drug combinations can produce synergistic interactions that increase drug effectiveness and reduce resistance. Computational models based on the integration of omics data have been used to identify synergistic combinations, but predicting drug synergy remains a challenge. Here, we introduce DIPx, an algorithm for personalized prediction of drug synergy based on biologically motivated tumor- and drug-specific pathway activation scores (PASs). We trained and validated DIPx in the AstraZeneca-Sanger (AZS) DREAM Challenge dataset using two separate test sets: Test Set 1 comprised the combinations already present in the training set, while Test Set 2 contained combinations absent from the training set, thus indicating the model’s ability to handle novel combinations. The Spearman correlation coefficients between predicted and observed drug synergy were 0.50 (95% CI: 0.47–0.53) in Test Set 1 and 0.26 (95% CI: 0.22–0.30) in Test Set 2, compared to 0.38 (95% CI: 0.34–0.42) and 0.18 (95% CI: 0.16–0.20), respectively, for the best performing method in the Challenge. We show evidence that higher synergy is associated with higher functional interaction between the drug targets, and this functional interaction information is captured by PAS. We illustrate the use of PAS to provide a potential biological explanation in terms of activated pathways that mediate the synergistic effects of combined drugs. In summary, DIPx can be a useful tool for personalized prediction of drug synergy and exploration of activated pathways related to the effects of combined drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/967bc228620e27010cd037753de4c0665790671f" target='_blank'>
              Pathway activation model for personalized prediction of drug synergy
              </a>
            </td>
          <td>
            Quang Thinh Trac, Yue Huang, Tom Erkers, P. Östling, Anna Bohlin, A. Österroos, Mattias Vesterlund, R. Jafari, I. Siavelis, Helena Bäckvall, Santeri Kiviluoto, L. Orre, Mattias Rantalainen, Janne Lehtiö, Sören Lehmann, Olli-P. Kallioniemi, Y. Pawitan, T. Vu
          </td>
          <td>2024-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Analyzing the motion of multiple biological agents, be it cells or individual animals, is pivotal for the understanding of complex collective behaviors. With the advent of advanced microscopy, detailed images of complex tissue formations involving multiple cell types have become more accessible in recent years. However, deciphering the underlying rules that govern cell movements is far from trivial. Here, we present a novel deep learning framework to estimate the underlying equations of motion from observed trajectories, a pivotal step in decoding such complex dynamics. Our framework integrates graph neural networks with neural differential equations, enabling effective prediction of two-body interactions based on the states of the interacting entities. We demonstrate the efficacy of our approach through two numerical experiments. First, we used a simulated data from a toy model to tune the hyperparameters. Based on the obtained hyperparameters, we then applied this approach to a more complex model that describes interacting cells of cellular slime molds. Our results show that the proposed method can accurately estimate the function of two-body interactions, thereby precisely replicating both individual and collective behaviors within these systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bbd03a34ea3ecbe6332287f4b5f0d43638d30f4" target='_blank'>
              Integrating GNN and Neural ODEs for Estimating Two-Body Interactions in Mixed-Species Collective Motion
              </a>
            </td>
          <td>
            Masahito Uwamichi, S. Schnyder, Tetsuya J. Kobayashi, Satoshi Sawai
          </td>
          <td>2024-05-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Modeling complex systems using standard neural ordinary differential equations (NODEs) often faces some essential challenges, including high computational costs and susceptibility to local optima. To address these challenges, we propose a simulation-free framework, called Fourier NODEs (FNODEs), that effectively trains NODEs by directly matching the target vector field based on Fourier analysis. Specifically, we employ the Fourier analysis to estimate temporal and potential high-order spatial gradients from noisy observational data. We then incorporate the estimated spatial gradients as additional inputs to a neural network. Furthermore, we utilize the estimated temporal gradient as the optimization objective for the output of the neural network. Later, the trained neural network generates more data points through an ODE solver without participating in the computational graph, facilitating more accurate estimations of gradients based on Fourier analysis. These two steps form a positive feedback loop, enabling accurate dynamics modeling in our framework. Consequently, our approach outperforms state-of-the-art methods in terms of training time, dynamics prediction, and robustness. Finally, we demonstrate the superior performance of our framework using a number of representative complex systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bedb3203006239c598b64a69777f9f9b9613ed" target='_blank'>
              From Fourier to Neural ODEs: Flow Matching for Modeling Complex Systems
              </a>
            </td>
          <td>
            Xin Li, Jingdong Zhang, Qunxi Zhu, Chengli Zhao, Xue Zhang, Xiaojun Duan, Wei Lin
          </td>
          <td>2024-05-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Virtual clinical trials (VCTs) are growing in popularity as a tool for quantitatively predicting heterogeneous treatment responses across a population. In the context of a VCT, a plausible patient is an instance of a mathematical model with parameter (or attribute) values chosen to reflect features of the disease and response to treatment for that particular patient. A number of techniques have been introduced to determine the set of model parametrizations to include in a virtual patient cohort. These methodologies generally start with a prior distribution for each model parameter and utilize some criteria to determine whether a parameter set sampled from the priors should be included or excluded from the plausible population. No standard technique exists, however, for generating these prior distributions and choosing the inclusion/exclusion criteria. In this work, we rigorously quantify the impact that VCT design choices have on VCT predictions. Rather than use real data and a complex mathematical model, a spatial model of radiotherapy is used to generate simulated patient data and the mathematical model used to describe the patient data is a two-parameter ordinary differential equations model. This controlled setup allows us to isolate the impact of both the prior distribution and the inclusion/exclusion criteria on both the heterogeneity of plausible populations and on predicted treatment response. We find that the prior distribution, rather than the inclusion/exclusion criteria, has a larger impact on the heterogeneity of the plausible population. Yet, the percent of treatment responders in the plausible population was more sensitive to the inclusion/exclusion criteria utilized. This foundational understanding of the role of virtual clinical trial design should help inform the development of future VCTs that use more complex models and real data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231e7bc7a40e4003493fc06a15b04cc9b5fdabe2" target='_blank'>
              Assessing the Role of Patient Generation Techniques in Virtual Clinical Trial Outcomes
              </a>
            </td>
          <td>
            J. L. Gevertz, J. R. Wares
          </td>
          <td>2024-06-12</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Medicinal synergy prediction is a powerful tool in drug discovery and development that harnesses the principles of combination therapy to enhance therapeutic outcomes by improving efficacy, reducing toxicity, and preventing drug resistance. While a myriad of computational methods has emerged for predicting synergistic drug combinations, a large portion of them may overlook the intricate, yet critical relationships between various entities in drug interaction networks, such as drugs, cell lines, and diseases. These relationships are complex and multidimensional, requiring sophisticated modeling to capture nuanced interplay that can significantly influence therapeutic efficacy. We introduce a salient deep hypergraph learning method, namely, Heterogeneous Entity Representation for MEdicinal Synergy prediction (HERMES), to predict anti-cancer drug synergy. HERMES integrates heterogeneous data sources, encompassing drug, cell line, and disease information, to provide a comprehensive understanding of the interactions involved. By leveraging advanced hypergraph neural networks with gated residual mechanisms, HERMES can effectively learn complex relationships/interactions within the data. Our results show HERMES demonstrates state-of-the-art performance, particularly in forecasting new drug combinations, significantly surpassing previous methods. This advancement underscores the potential of HERMES to facilitate more effective and precise drug combination predictions, thereby enhancing the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9f935a6bee872e8d5f11ac3462761c64e3b2806" target='_blank'>
              Heterogeneous Entity Representation for Medicinal Synergy Prediction
              </a>
            </td>
          <td>
            Jiawei Wu, Jun Wen, Mingyuan Yan, Anqi Dong, Can Chen
          </td>
          <td>2024-06-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Predicting cancer drug response using both genomics and drug features has shown some success compared to using genomics features alone. However, there has been limited research done on how best to combine or fuse the two types of features. Using a visible neural network with two deep learning branches for genes and drug features as the base architecture, we experimented with different fusion functions and fusion points. Our experiments show that injecting multiplicative relationships between gene and drug latent features into the original concatenation-based architecture DrugCell significantly improved the overall predictive performance and outperformed other baseline models. We also show that different fusion methods respond differently to different fusion points, indicating that the relationship between drug features and different hierarchical biological level of gene features is optimally captured using different methods. Considering both predictive performance and runtime speed, tensor product partial is the best-performing fusion function to combine late-stage representations of drug and gene features to predict cancer drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1de5913ff2d9059af1cfcf445f809114c434905" target='_blank'>
              Optimal fusion of genotype and drug embeddings in predicting cancer drug response
              </a>
            </td>
          <td>
            Trang Nguyen, Anthony Campbell, Ankit Kumar, Edwin Amponsah, M. Fiterau, Leili Shahriyari
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumor growth inhibition (TGI) modeling attempts to describe the time course changes in tumor size for patients undergoing cancer therapy. TGI models present several advantages over traditional exposure-response models that are based explicitly on clinical end points and have become a popular tool in the pharmacometrics community. Unfortunately, the data required to fit TGI models, namely longitudinal tumor measurements, are sparse or often not available in literature or publicly accessible databases. On the contrary, common end points such as progression-free survival (PFS) and objective response rate (ORR) are directly derived from longitudinal tumor measurements and are routinely published. To this end, a Bayesian generative model relating underlying tumor dynamics to summary PFS and ORR data is introduced to learn TGI model parameters using only published summary data. The parameterized model can describe the tumor dynamics, quantify treatment effect, and account for differences in the study population. The utility of this model is shown by applying it to several published studies, and learned parameters are combined to simulate an in silico trial of a novel combination therapy in a novel setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5334e76ce88316ffc777a0522ed6c120271486bb" target='_blank'>
              A novel Bayesian generative approach for estimating tumor dynamics from published studies.
              </a>
            </td>
          <td>
            Arya Pourzanjani, Saurabh Modi, Jamie Connarn, Xinwen Zhang, V. Upreti, Chih-Wei Lin, Khamir Mehta
          </td>
          <td>2024-05-22</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Serum creatinine in neonates follows complex dynamics due to maturation processes, most pronounced in the first few weeks of life. The development of a mechanism-based model describing complex dynamics requires high expertise in pharmacometric (PMX) modeling and substantial model development time. A recently published machine learning (ML) approach of low-dimensional neural ordinary differential equations (NODEs) is capable of modeling such data from newborns automatically. However, this efficient data-driven approach in itself does not result in a clinically interpretable model. In this work, an approach to deriving an interpretable model with reasonable PMX-type functions is presented. This "translation" was applied to derive a PMX model for serum creatinine in neonates considering maturation processes and covariates. The developed model was compared to a previously published mechanism-based PMX model whereas both models had similar mechanistic structures. The developed model was then utilized to simulate serum creatinine concentrations in the first few weeks of life considering different covariate values for gestational age and birth weight. The reference serum creatinine values derived from these simulations are consistent with observed serum creatinine values and previously published reference values. Thus, the presented NODE-based ML approach to model complex serum creatinine dynamics in newborns and derive interpretable, mathematical-statistical components similar to those in a conventional PMX model demonstrates a novel, viable approach to facilitate the modeling of complex dynamics in clinical settings and pediatric drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fae58343bde99a83efb245281ab69b262f4cd7f" target='_blank'>
              Applying Neural ODEs to Derive a Mechanism-Based Model for Characterizing Maturation-Related Serum Creatinine Dynamics in Preterm Newborns.
              </a>
            </td>
          <td>
            Dominic Stefan Bräm, Gilbert Koch, Karel Allegaert, J. van den Anker, M. Pfister
          </td>
          <td>2024-05-16</td>
          <td>Journal of clinical pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Machine learning (ML) has been applied to predict the efficacy of biologic agents in ulcerative colitis (UC). ML can offer precision, personalization, efficiency, and automation. Moreover, it can improve decision support in predicting clinical outcomes. However, it faces challenges related to data quality and quantity, overfitting, generalization, and interpretability. This paper comments on two recent ML models that predict the efficacy of vedolizumab and ustekinumab in UC. Models that consider multiple pathways, multiple ethnicities, and combinations of real-world and clinical trial data are required for optimal shared decision-making and precision medicine. This paper also highlights the potential of combining ML with computational models to enhance clinical outcomes and personalized healthcare. Key Insights: (1) ML offers precision, personalization, efficiency, and decision support for predicting the efficacy of biologic agents in UC. (2) Challenging aspects in ML prediction include data quality, overfitting, and interpretability. (3) Multiple pathways, multiple ethnicities, and combinations of real-world and clinical trial data should be considered in predictive models for optimal decision-making. (4) Combining ML with computational models may improve clinical outcomes and personalized healthcare.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fd22b7c6f685cae80958a1bc2dd8ae70573c792" target='_blank'>
              Machine Learning for Predicting Biologic Agent Efficacy in Ulcerative Colitis: An Analysis for Generalizability and Combination with Computational Models
              </a>
            </td>
          <td>
            Philippe Pinton
          </td>
          <td>2024-06-21</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Digital twins require computationally-efficient reduced-order models (ROMs) that can accurately describe complex dynamics of physical assets. However, constructing ROMs from noisy high-dimensional data is challenging. In this work, we propose a data-driven, non-intrusive method that utilizes stochastic variational deep kernel learning (SVDKL) to discover low-dimensional latent spaces from data and a recurrent version of SVDKL for representing and predicting the evolution of latent dynamics. The proposed method is demonstrated with two challenging examples -- a double pendulum and a reaction-diffusion system. Results show that our framework is capable of (i) denoising and reconstructing measurements, (ii) learning compact representations of system states, (iii) predicting system evolution in low-dimensional latent spaces, and (iv) quantifying modeling uncertainties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e646edf723e20c982f81d29cf479c056b6d42cb" target='_blank'>
              Recurrent Deep Kernel Learning of Dynamical Systems
              </a>
            </td>
          <td>
            N. Botteghi, Paolo Motta, Andrea Manzoni, P. Zunino, Mengwu Guo
          </td>
          <td>2024-05-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Classical compartmental models in epidemiology often struggle to accurately capture real-world dynamics due to their inability to address the inherent heterogeneity of populations. In this paper, we introduce a novel approach that incorporates heterogeneity through a mobility variable, transforming the traditional ODE system into a system of integro-differential equations that describe the dynamics of population densities across different compartments. Our results show that, for the same basic reproduction number, our mobility-based model predicts a smaller final pandemic size compared to classic compartmental models, whose population densities are represented as Dirac delta functions in our density-based framework. This addresses the overestimation issue common in many classical models. Additionally, we demonstrate that the time series of the infected population is sufficient to uniquely identify the mobility distribution. We reconstruct this distribution using a machine-learning-based framework, providing both theoretical and algorithmic support to effectively constrain the mobility-based model with real-world data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c03a14a48a310a736f8149b03d7cfacc5775bf6" target='_blank'>
              Modeling, Inference, and Prediction in Mobility-Based Compartmental Models for Epidemiology
              </a>
            </td>
          <td>
            Ning Jiang, Weiqi Chu, Yao Li
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbc8395840f2621fd3a1b982cf278edaa1f8cb" target='_blank'>
              A review of machine learning methods for cancer characterization from microbiome data
              </a>
            </td>
          <td>
            Marco Teixeira, Francisco Silva, R. Ferreira, Tânia Pereira, Ceu Figueiredo, Hélder P. Oliveira
          </td>
          <td>2024-05-30</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The ever-expanding landscape of pharmaceutical compounds and medical treatments has underscored the critical need for an advanced system to predict drug-drug interactions (DDIs) using machine learning. This paper delves into the development of a novel predictive model that leverages diverse data sources, including chemical structures, pharmacological properties, and scientific literature, to comprehensively analyse and foresee potential DDIs. The current state of DDIs is rife with limitations in scope and accuracy, making the development of an advanced system not only a necessity for enhancing patient safety but also a boon to pharmaceutical research and regulatory oversight. The relationship between two medications in which one drug’s pharmacological actions are altered by another is known as a drug-drug interaction (DDI). Favourable drug-drug interactions (DDIs) generally contribute to improved therapeutic results for patients. On the contrary, adverse DDIs pose a significant risk, often resulting in undesirable drug reactions that can potentially lead to the withdrawal of a drug from the market and even patient fatalities. Consequently, the identification of DDIs is essential for advancing new drug development and effectively managing diseases. Keywords: Drug, Drug-drug interaction, Machine learning, Bioinformatics, Drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc31ce4df8b41262c53f949e913ee6c901766ec" target='_blank'>
              PREDICTING DRUG-DRUG INTERACTION USING MACHINE LEARNING
              </a>
            </td>
          <td>

          </td>
          <td>2024-05-29</td>
          <td>INTERANTIONAL JOURNAL OF SCIENTIFIC RESEARCH IN ENGINEERING AND MANAGEMENT</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A Marked Temporal Point Process (MTPP) is a stochastic process whose realization is a set of event-time data. MTPP is often used to understand complex dynamics of asynchronous temporal events such as money transaction, social media, healthcare, etc. Recent studies have utilized deep neural networks to capture complex temporal dependencies of events and generate embedding that aptly represent the observed events. While most previous studies focus on the inter-event dependencies and their representations, how individual events influence the overall dynamics over time has been under-explored. In this regime, we propose a Decoupled MTPP framework that disentangles characterization of a stochastic process into a set of evolving influences from different events. Our approach employs Neural Ordinary Differential Equations (Neural ODEs) to learn flexible continuous dynamics of these influences while simultaneously addressing multiple inference problems, such as density estimation and survival rate computation. We emphasize the significance of disentangling the influences by comparing our framework with state-of-the-art methods on real-life datasets, and provide analysis on the model behavior for potential applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01528db95c399e0822e827b108710fdf76fd5f21" target='_blank'>
              Decoupled Marked Temporal Point Process using Neural Ordinary Differential Equations
              </a>
            </td>
          <td>
            Yujee Song, Donghyun Lee, Rui Meng, Won Hwa Kim
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Computational drug sensitivity models have the potential to improve therapeutic outcomes by identifying targeted drugs components that are tailored to the transcriptomic profile of a given primary tumor. The SMILES representation of molecules that is used by state-of-the-art drug-sensitivity models is not conducive for neural networks to generalize to new drugs, in part because the distance between atoms does not generally correspond to the distance between their representation in the SMILES strings. Graph-attention networks, on the other hand, are high-capacity models that require large training-data volumes which are not available for drug-sensitivity estimation. We develop a modular drug-sensitivity graph-attentional neural network. The modular architecture allows us to separately pre-train the graph encoder and graph-attentional pooling layer on related tasks for which more data are available. We observe that this model outperforms reference models for the use cases of precision oncology and drug discovery; in particular, it is better able to predict the specific interaction between drug and cell line that is not explained by the general cytotoxicity of the drug and the overall survivability of the cell line. The complete source code is available at https://zenodo.org/doi/10.5281/zenodo.8020945. All experiments are based on the publicly available GDSC data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1d95c29d808b802d81c2a052d2dbf5ee6f08b7" target='_blank'>
              Cancer drug sensitivity estimation using modular deep Graph Neural Networks
              </a>
            </td>
          <td>
            P. A. Campana, Paul Prasse, M. Lienhard, Kristina Thedinga, Ralf Herwig, Tobias Scheffer
          </td>
          <td>2024-04-04</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In the complex landscape of hematologic samples such as peripheral blood or bone marrow derived from flow cytometry (FC) data, cell-level prediction presents profound challenges. This work explores injecting hierarchical prior knowledge into graph neural networks (GNNs) for single-cell multi-class classification of tabular cellular data. By representing the data as graphs and encoding hierarchical relationships between classes, we propose our hierarchical plug-in method to be applied to several GNN models, namely, FCHC-GNN, and effectively designed to capture neighborhood information crucial for single-cell FC domain. Extensive experiments on our cohort of 19 distinct patients, demonstrate that incorporating hierarchical biological constraints boosts performance significantly across multiple metrics compared to baseline GNNs without such priors. The proposed approach highlights the importance of structured inductive biases for gaining improved generalization in complex biological prediction tasks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e0fdaea8a065eda5545038bbdb830b51b0c573" target='_blank'>
              Injecting Hierarchical Biological Priors into Graph Neural Networks for Flow Cytometry Prediction
              </a>
            </td>
          <td>
            Fatemeh Nassajian Mojarrad, Lorenzo Bini, Thomas Matthes, Stéphane Marchand-Maillet
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mathematical models are indispensable to the system biology toolkit for studying the structure and behavior of intracellular signaling networks. A common approach to modeling is to develop a system of equations that encode the known biology using approximations and simplifying assumptions. As a result, the same signaling pathway can be represented by multiple models, each with its set of underlying assumptions, which opens up challenges for model selection and decreases certainty in model predictions. Here, we use Bayesian multimodel inference to develop a framework to increase certainty in systems biology models. Using models of the extracellular regulated kinase (ERK) pathway, we first show that multimodel inference increases predictive certainty and yields predictors that are robust to changes in the set of available models. We then show that predictions made with multimodel inference are robust to data uncertainties introduced by decreasing the measurement duration and reducing the sample size. Finally, we use multimodel inference to identify a new model to explain experimentally measured sub-cellular location-specific ERK activity dynamics. In summary, our framework highlights multimodel inference as a disciplined approach to increasing the certainty of intracellular signaling activity predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e6724b6c05e22b755836f73b25e0ce04843d4" target='_blank'>
              Increasing certainty in systems biology models using Bayesian multimodel inference
              </a>
            </td>
          <td>
            Nathaniel Linden-Santangeli, Jin Zhang, Boris Kramer, P. Rangamani
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Predicting drug efficacy and safety in vivo requires information on biological responses (e.g., cell morphology and gene expression) to small molecule perturbations. However, current molecular representation learning methods do not provide a comprehensive view of cell states under these perturbations and struggle to remove noise, hindering model generalization. We introduce the Information Alignment (InfoAlign) approach to learn molecular representations through the information bottleneck method in cells. We integrate molecules and cellular response data as nodes into a context graph, connecting them with weighted edges based on chemical, biological, and computational criteria. For each molecule in a training batch, InfoAlign optimizes the encoder's latent representation with a minimality objective to discard redundant structural information. A sufficiency objective decodes the representation to align with different feature spaces from the molecule's neighborhood in the context graph. We demonstrate that the proposed sufficiency objective for alignment is tighter than existing encoder-based contrastive methods. Empirically, we validate representations from InfoAlign in two downstream tasks: molecular property prediction against up to 19 baseline methods across four datasets, plus zero-shot molecule-morphology matching.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db33b94c598876b05b7af83901a3a69d60eb47d" target='_blank'>
              Learning Molecular Representation in a Cell
              </a>
            </td>
          <td>
            Gang Liu, Srijit Seal, John Arevalo, Zhenwen Liang, A. Carpenter, Meng Jiang, Shantanu Singh
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Parameter estimation is one of the central problems in computational modeling of biological systems. Typically, scientists must fully specify the mathematical structure of the model, often expressed as a system of ordinary differential equations, to estimate the parameters. This process poses significant challenges due to the necessity for a detailed understanding of the underlying biological mechanisms. In this paper, we present an approach for estimating model parameters and assessing their identifiability in situations where only partial knowledge of the system structure is available. The partially known model is extended into a system of Hybrid Neural Ordinary Differential Equations, which captures the unknown portions of the system using neural networks. Integrating neural networks into the model structure introduces two primary challenges for parameter estimation: the need to globally explore the search space while employing gradient-based optimization, and the assessment of parameter identifiability, which may be hindered by the expressive nature of neural networks. To overcome the first issue, we treat biological parameters as hyperparameters in the extended model, exploring the parameter search space during hyperparameter tuning. The second issue is then addressed by an a posteriori analysis of parameter identifiability, computed by introducing a variant of a well-established approach for mechanistic models. These two components are integrated into an end-to-end pipeline that is thoroughly described in the paper. We assess the effectiveness of the proposed workflow on test cases derived from three different benchmark models. These test cases have been designed to mimic real-world conditions, including the presence of noise in the training data and various levels of data availability for the system variables. Author summary Parameter estimation is a central challenge in modeling biological systems. Typically, scientists calibrate the parameters by aligning model predictions with measured data once the model structure is defined. Our paper introduces a workflow that leverages the integration between mechanistic modeling and machine learning to estimate model parameters when the model structure is not fully known. We focus mainly on analyzing the identifiability of the model parameters, which measures how confident we can be in the parameter estimates given the available experimental data and partial mechanistic understanding of the system. We assessed the effectiveness of our approach in various in silico scenarios. Our workflow represents a first step to adapting traditional methods used in fully mechanistic models to the scenario of hybrid modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a064c473effb10441bd8fe1c6bde24ed0162e1e3" target='_blank'>
              Robust parameter estimation and identifiability analysis with Hybrid Neural Ordinary Differential Equations in Computational Biology
              </a>
            </td>
          <td>
            Stefano Giampiccolo, Federico Reali, Anna Fochesato, Giovanni Iacca, Luca Marchetti
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Artificial intelligence (AI) is increasingly used in every stage of drug development. Continuing breakthroughs in AI-based methods for drug discovery require the creation, improvement, and refinement of drug discovery data. We posit a new data challenge that slows the advancement of drug discovery AI: datasets are often collected independently from each other, often with little overlap, creating data sparsity. Data sparsity makes data curation difficult for researchers looking to answer key research questions requiring values posed across multiple datasets. We propose a novel diffusion GNN model Syngand capable of generating ligand and pharmacokinetic data end-to-end. We show and provide a methodology for sampling pharmacokinetic data for existing ligands using our Syngand model. We show the initial promising results on the efficacy of the Syngand-generated synthetic target property data on downstream regression tasks with AqSolDB, LD50, and hERG central. Using our proposed model and methodology, researchers can easily generate synthetic ligand data to help them explore research questions that require data spanning multiple datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309b0837f1ce764a9066e2d9555ed9883956e6f0" target='_blank'>
              Synthetic Data from Diffusion Models Improve Drug Discovery Prediction
              </a>
            </td>
          <td>
            Bing Hu, Ashish Saragadam, Anita Layton, Helen Chen
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dynamic Positron Emission Tomography (dPET) imaging and Time-Activity Curve (TAC) analyses are essential for understanding and quantifying the biodistribution of radiopharmaceuticals over time and space. Traditional compartmental modeling, while foundational, commonly struggles to fully capture the complexities of biological systems, including non-linear dynamics and variability. This study introduces an innovative data-driven neural network-based framework, inspired by Reaction Diffusion systems, designed to address these limitations. Our approach, which adaptively fits TACs from dPET, enables the direct calibration of diffusion coefficients and reaction terms from observed data, offering significant improvements in predictive accuracy and robustness over traditional methods, especially in complex biological scenarios. By more accurately modeling the spatio-temporal dynamics of radiopharmaceuticals, our method advances modeling of pharmacokinetic and pharmacodynamic processes, enabling new possibilities in quantitative nuclear medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7e97f4a77c199435d67bf7917712e6bcfb5bc2" target='_blank'>
              Beyond Conventional Parametric Modeling: Data-Driven Framework for Estimation and Prediction of Time Activity Curves in Dynamic PET Imaging
              </a>
            </td>
          <td>
            N. Zakariaei, Arman Rahmim, Eldad Haber
          </td>
          <td>2024-05-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Motivation Gene regulatory networks (GRNs) encode gene regulation in living organisms, and have become a critical tool to understand complex biological processes. However, due to the dynamic and complex nature of gene regulation, inferring GRNs from scRNA-seq data is still a challenging task. Existing computational methods usually focus on the close connections between genes, and ignore the global structure and distal regulatory relationships. Results In this study, we develop a supervised deep learning framework, IGEGRNS, to infer GRNs from scRNA-seq data based on graph embedding. In the framework, contextual information of genes is captured by GraphSAGE, which aggregates gene features and neighborhood structures to generate low-dimensional embedding for genes. Then, the k most influential nodes in the whole graph are filtered through Top-k pooling. Finally, potential regulatory relationships between genes are predicted by stacking CNNs. Compared with nine competing supervised and unsupervised methods, our method achieves better performance on six time-series scRNA-seq datasets. Availability and implementation Our method IGEGRNS is implemented in Python using the Pytorch machine learning library, and it is freely available at https://github.com/DHUDBlab/IGEGRNS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3bff5e65efb7f880841724fadf659d594092ce" target='_blank'>
              Inferring gene regulatory networks from single-cell transcriptomics based on graph embedding
              </a>
            </td>
          <td>
            Yanglan Gan, Jiacheng Yu, Guangwei Xu, Cairong Yan, Guobing Zou
          </td>
          <td>2024-05-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d4bb229f62bcab0916b3d2b4498d9f9dcea2bea" target='_blank'>
              Beyond benchmarking and towards predictive models of dataset-specific single-cell RNA-seq pipeline performance
              </a>
            </td>
          <td>
            Cindy Fang, Alina Selega, Kieran R. Campbell
          </td>
          <td>2024-06-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Machine learning (ML) is increasingly being used to guide biological discovery in biomedicine such as prioritizing promising small molecules in drug discovery. In those applications, ML models are used to predict the properties of biological systems, and researchers use these predictions to prioritize candidates as new biological hypotheses for downstream experimental validations. However, when applied to unseen situations, these models can be overconfident and produce a large number of false positives. One solution to address this issue is to quantify the model’s prediction uncertainty and provide a set of hypotheses with a controlled false discovery rate (FDR) pre-specified by researchers. We propose CPEC, an ML framework for FDR-controlled biological discovery. We demonstrate its effectiveness using enzyme function annotation as a case study, simulating the discovery process of identifying the functions of less-characterized enzymes. CPEC integrates a deep learning model with a statistical tool known as conformal prediction, providing accurate and FDR-controlled function predictions for a given protein enzyme. Conformal prediction provides rigorous statistical guarantees to the predictive model and ensures that the expected FDR will not exceed a user-specified level with high probability. Evaluation experiments show that CPEC achieves reliable FDR control, better or comparable prediction performance at a lower FDR than existing methods, and accurate predictions for enzymes under-represented in the training data. We expect CPEC to be a useful tool for biological discovery applications where a high yield rate in validation experiments is desired but the experimental budget is limited.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec42a07fd6e6727051cdda1c2b2f1428f1f2ca97" target='_blank'>
              Leveraging conformal prediction to annotate enzyme function space with limited false positives
              </a>
            </td>
          <td>
            Kerr Ding, Jiaqi Luo, Yunan Luo
          </td>
          <td>2024-05-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Modeling the dynamics of flexible objects has become an emerging topic in the community as these objects become more present in many applications, e.g., soft robotics. Due to the properties of flexible materials, the movements of soft objects are often highly nonlinear and, thus, complex to predict. Data-driven approaches seem promising for modeling those complex dynamics but often neglect basic physical principles, which consequently makes them untrustworthy and limits generalization. To address this problem, we propose a physics-constrained learning method that combines powerful learning tools and reliable physical models. Our method leverages the data collected from observations by sending them into a Gaussian process that is physically constrained by a distributed Port-Hamiltonian model. Based on the Bayesian nature of the Gaussian process, we not only learn the dynamics of the system, but also enable uncertainty quantification. Furthermore, the proposed approach preserves the compositional nature of Port-Hamiltonian systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020e63b8b17cdec54c056d8a1edc98770c8fa7ab" target='_blank'>
              Physics-Constrained Learning for PDE Systems with Uncertainty Quantified Port-Hamiltonian Models
              </a>
            </td>
          <td>
            Kaiyuan Tan, Peilun Li, Thomas Beckers
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Clinical trials involving systemic neoadjuvant treatments in breast cancer aim to shrink tumors before surgery while simultaneously allowing for controlled evaluation of biomarkers, toxicity, and suppression of distant (occult) metastatic disease. Yet neoadjuvant clinical trials are rarely preceded by preclinical testing involving neoadjuvant treatment, surgery, and post-surgery monitoring of the disease. Here we used a mouse model of spontaneous metastasis occurring after surgical removal of orthotopically implanted primary tumors to develop a predictive mathematical model of neoadjuvant treatment response to sunitinib, a receptor tyrosine kinase inhibitor (RTKI). Treatment outcomes were used to validate a novel mathematical kinetics-pharmacodynamics model predictive of perioperative disease progression. Longitudinal measurements of presurgical primary tumor size and postsurgical metastatic burden were compiled using 128 mice receiving variable neoadjuvant treatment doses and schedules (released publicly at https://zenodo.org/records/10607753). A non-linear mixed-effects modeling approach quantified inter-animal variabilities in metastatic dynamics and survival, and machine-learning algorithms were applied to investigate the significance of several biomarkers at resection as predictors of individual kinetics. Biomarkers included circulating tumor- and immune-based cells (circulating tumor cells and myeloid-derived suppressor cells) as well as immunohistochemical tumor proteins (CD31 and Ki67). Our computational simulations show that neoadjuvant RTKI treatment inhibits primary tumor growth but has little efficacy in preventing (micro)-metastatic disease progression after surgery and treatment cessation. Machine learning algorithms that included support vector machines, random forests, and artificial neural networks, confirmed a lack of definitive biomarkers, which shows the value of preclinical modeling studies to identify potential failures that should be avoided clinically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c8ff9b498ddc0ddca957f0ddbc9e36ba56facbc" target='_blank'>
              Machine-learning and mechanistic modeling of metastatic breast cancer after neoadjuvant treatment
              </a>
            </td>
          <td>
            S. Benzekry, M. Mastri, C. Nicolò, J. Ebos
          </td>
          <td>2024-05-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Predicting adverse drug reactions (ADRs) early in the drug development process is crucial for ensuring drug safety and reducing costly late-stage failures. Traditional methods for toxicity assessment rely heavily on animal testing and empirical observations, which are often time-consuming, expensive, and ethically questionable. In recent years, deep learning techniques have emerged as powerful tools for predictive toxicology, offering the potential to accelerate the identification of potential ADRs while reducing reliance on animal models.This paper reviews the current state-of-the-art deep learning approaches for predictive toxicology assessment, focusing on their applications in the early detection of ADRs. [1],[2] We discuss various deep learning architectures, including convolutional neural networks (CNNs), recurrent neural networks (RNNs), and graph neural networks (GNNs), and their utilization in analyzing diverse data types such as chemical structures, omics data, and adverse event reports.Furthermore, we examine the challenges and limitations associated with deep learning-based predictive toxicology, including data availability, model interpretability, and regulatory acceptance. We also explore ongoing efforts to address these challenges, such as the development of standardized datasets, explainable AI techniques, and collaborations between academia, industry, and regulatory agencies.Overall, this paper highlights the potential of deep learning for early detection of ADRs in drug development and underscores the need for continued research and collaboration to realize the full benefits of these techniques in ensuring drug safety and improving public health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8818d43fe76e166c64a48971891b7b91a5fde4fe" target='_blank'>
              Deep Learning for Predictive Toxicology Assessment Early Detection of Adverse Drug Reactions.
              </a>
            </td>
          <td>
            Dr. B. Indira Priyadarshini, Dr. Deepak A. Vidhate, K. Sravani, Dr Jayasundar, Dr. Sandeep V. Binorkar, S. Karmode
          </td>
          <td>2024-04-26</td>
          <td>Power System Technology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Most approaches for assessing causality in complex dynamical systems fail when the interactions between variables are inherently non-linear and non-stationary. Here we introduce Temporal Autoencoders for Causal Inference (TACI), a methodology that combines a new surrogate data metric for assessing causal interactions with a novel two-headed machine learning architecture to identify and measure the direction and strength of time-varying causal interactions. Through tests on both synthetic and real-world datasets, we demonstrate TACI's ability to accurately quantify dynamic causal interactions across a variety of systems. Our findings display the method's effectiveness compared to existing approaches and also highlight our approach's potential to build a deeper understanding of the mechanisms that underlie time-varying interactions in physical and biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d22ad3492ed41cdaeed3d49e84841e11089e47" target='_blank'>
              Inferring the time-varying coupling of dynamical systems with temporal convolutional autoencoders
              </a>
            </td>
          <td>
            Josuan Calderon, Gordon J. Berman
          </td>
          <td>2024-06-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Underscoring the interwoven methodologies and shared objectives of Machine Learning (ML) and Statistics, this paper aims to explore the synergy between the two disciplines with the proliferation of large datasets and advanced computational power. The ability of observing accurate insights for complex datasets and addressing real world applications with sophisticated, hybrid approaches can enhance with the convergence of the ML and Statistics. Although the concepts of both disciplines started with distinct origins, two disciplines increasingly intersect, fostering methodological cross fertilization. To improve the generalization and interpretability of ML concepts, statistical techniques such as model selection and regularization can be used while ensemble methods and neural networks exemplify predictive modeling’s statistical applications. By integrating ML to address the challenges in statistics such as fairness, interpretability, robustness, and scalability, statistician can enhance the key feature of statistics more effectively. Overall, combined concepts of ML and statistics not only address the diverse analytical task but it pave the path for Artificial Intelligence and data science by highlighting the main role of their synergy in modern data exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dab5d71bc6a258240c71c3ca3fc10ee75a94d2c" target='_blank'>
              The Synergy between Machine Learning and Statistics: A Review
              </a>
            </td>
          <td>
            Smm Lakmali
          </td>
          <td>2024-06-07</td>
          <td>International Journal of Innovative Science and Research Technology (IJISRT)</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Spatiotemporal processes are a fundamental tool for modeling dynamics across various domains, from heat propagation in materials to oceanic and atmospheric flows. However, currently available neural network-based modeling approaches fall short when faced with data collected randomly over time and space, as is often the case with sensor networks in real-world applications like crowdsourced earthquake detection or pollution monitoring. In response, we developed a new spatiotemporal method that effectively handles such randomly sampled data. Our model integrates techniques from amortized variational inference, neural differential equations, neural point processes, and implicit neural representations to predict both the dynamics of the system and the probabilistic locations and timings of future observations. It outperforms existing methods on challenging spatiotemporal datasets by offering substantial improvements in predictive accuracy and computational efficiency, making it a useful tool for modeling and understanding complex dynamical systems observed under realistic, unconstrained conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3672c7dfec49ffe06ab53ce52945c37c38e785c8" target='_blank'>
              Modeling Randomly Observed Spatiotemporal Dynamical Systems
              </a>
            </td>
          <td>
            V. Iakovlev, Harri Lahdesmaki
          </td>
          <td>2024-06-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Autoimmune rheumatic diseases are often characterised by heterogeneity in presentation. The traditional approach to diseases guided by their phenotype may be suboptimal with the advent of precision medicine. Precision medicine is the integration and application of multiomics to predict the best-performing drug and its toxicity profile to derive optimal benefits. With novel drug discoveries and an expanding therapeutic armamentarium, it potentially aids in clinical and therapeutic decision-making, while saving time and averting adverse events. However, multiomics comes with ‘big data’, and owing to the costs, the sample size is usually small. Machine learning (ML) plays an important role in these scenarios where conventional statistics fall short. So, by integrating clinical data with the data from -omics, ML models can be built, which can accurately predict the clinical factors or even novel biomarkers that predict response. This approach has a potential for great benefit as valuable time or the ‘therapeutic window of opportunity’ would be saved, with fewer adverse events, eventually translating to lower damage accrual and better outcomes. Most of the evidence for the use of ML in precision rheumatology comes from rheumatoid arthritis and the factors predicting response to various drugs, including tumour necrosis factor inhibitors. This approach also has its limitations such as the lack of generalizability and the current scarcity of longitudinal data. These models must be tested in larger cohorts and population-based studies for validation, failing which there is a risk of apparent identification of multiple ‘novel’ biomarkers that may or may not be mechanistic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f607769c09b08a0d2145eee4ed39d8f624f5aad6" target='_blank'>
              Current Status of Machine Learning for Precision Rheumatology: Are We There Yet?
              </a>
            </td>
          <td>
            Prakashini Mruthyunjaya, Avarna Agarwal, Sakir Ahmed
          </td>
          <td>2024-06-16</td>
          <td>Indian Journal of Rheumatology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary The major impact from this work is two-fold, first in providing a list of reproducible candidate drugs nominated against multiple glioblastoma (GBM) datasets, and second in providing drug–gene biomarkers of interest for GBM as well as reproducible computational pipelines for identifying drug–biomarker leads that could be extended to other cancer types beyond GBM. We believe the research community will be interested and can utilize our results here for the further development of biomarker-specific drug therapies in both GBM as well as other disease types. Abstract Glioblastoma multiforme (GBM) is the deadliest, most heterogeneous, and most common brain cancer in adults. Not only is there an urgent need to identify efficacious therapeutics, but there is also a great need to pair these therapeutics with biomarkers that can help tailor treatment to the right patient populations. We built patient drug response models by integrating patient tumor transcriptome data with high-throughput cell line drug screening data as well as Bayesian networks to infer relationships between patient gene expression and drug response. Through these discovery pipelines, we identified agents of interest for GBM to be effective across five independent patient cohorts and in a mouse avatar model: among them are a number of MEK inhibitors (MEKis). We also predicted phosphoglycerate dehydrogenase enzyme (PHGDH) gene expression levels to be causally associated with MEKi efficacy, where knockdown of this gene increased tumor sensitivity to MEKi and overexpression led to MEKi resistance. Overall, our work demonstrated the power of integrating computational approaches. In doing so, we quickly nominated several drugs with varying known mechanisms of action that can efficaciously target GBM. By simultaneously identifying biomarkers with these drugs, we also provide tools to select the right patient populations for subsequent evaluation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b533ec6d5b79d0a8dc0aa09ce2fb660cc0c67f5f" target='_blank'>
              Integration of Computational Pipeline to Streamline Efficacious Drug Nomination and Biomarker Discovery in Glioblastoma
              </a>
            </td>
          <td>
            D. Maeser, Robert F. Gruener, R. Galvin, Adam Lee, Tomoyuki Koga, F. Grigore, Yuta Suzuki, F. Furnari, Clark Chen, R. S. Huang
          </td>
          <td>2024-04-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Developing new pharmaceuticals is a costly and time-consuming endeavor fraught with significant safety risks. A critical aspect of drug research and disease therapy is discerning the existence of interactions between drugs and proteins. The evolution of deep learning (DL) in computer science has been remarkably aided in this regard in recent years. Yet, two challenges remain: (i) balancing the extraction of profound, local cohesive characteristics while warding off gradient disappearance and (ii) globally representing and understanding the interactions between the drug and target local attributes, which is vital for delivering molecular level insights indispensable to drug development. In response to these challenges, we propose a DL network structure, MolLoG, primarily comprising two modules: local feature encoders (LFE) and global interactive learning (GIL). Within the LFE module, graph convolution networks and leap blocks capture the local features of drug and protein molecules, respectively. The GIL module enables the efficient amalgamation of feature information, facilitating the global learning of feature structural semantics and procuring multihead attention weights for abstract features stemming from two modalities, providing biologically pertinent explanations for black-box results. Finally, predictive outcomes are achieved by decoding the unified representation via a multilayer perceptron. Our experimental analysis reveals that MolLoG outperforms several cutting-edge baselines across four data sets, delivering superior overall performance and providing satisfactory results when elucidating various facets of drug-target interaction predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47d7134e78582b908d74226e67442dfea752621" target='_blank'>
              MolLoG: A Molecular Level Interpretability Model Bridging Local to Global for Predicting Drug Target Interactions
              </a>
            </td>
          <td>
            Bao-Ming Feng, Yuanyuan Zhang, Xiao-Chen Zhou, Jin-Long Wang, Yinfei Feng
          </td>
          <td>2024-05-06</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Drug-Drug Interactions is a critical health and safety concern that receives a lot of attention from both academia and business. Polypharmacy is often employed as a strategy to manage complex diseases such as cancer, diabetes, and age-related ailments. However, combining medications with other drugs can lead to unintended adverse reactions. Interactions between drugs may increase the chance of unanticipated negative effects and even unknown toxicity, putting patients at risk. Detecting and identifying Interactions not only helps clinicians avoid chronic but will also encourage the co-prescription of safe drugs for more effective therapies. It is expensive and time-consuming to identify drug-drug interactions and Adverse Reactions among several medication pairings, both in vivo and in vitro. Recent advancements in computer science, specifically in the field of Artificial Intelligence, have yielded techniques that enable researchers to identify drug-drug interactions. We present comprehensive approaches that enable in-depth analysis of potential interactions by taking into account various factors, including molecular structure, clinical data, network relationships, and existing literature. This paper offers an all-encompassing survey of research studies that utilize Machine Learning and Deep Learning algorithms for the prediction of Drug-Drug interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1998cb61f72b4cc4259a444d737c0696c8054df5" target='_blank'>
              A comprehensive survey explores Drug-Drug interaction prediction using Machine-Learning techniques
              </a>
            </td>
          <td>
            yasmin radwan, karam gouda, Ibrahim Abdelbaky, Mona M. Arafa
          </td>
          <td>2024-05-01</td>
          <td>Benha Journal of Applied Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="e13603 Background: Oncology therapeutic development has high failure rates with substantial costs. Advances in technology, including molecular characterisations of cancer and computational power, provide the opportunity to better model therapeutic response for existing and emerging therapies. Methods: Concr adapted Bayesian statistical principles used by astrophysicists to effectively integrate multi-dimensional data (genomic & transcriptomic) for accurate modelling of cancer biology. Our proprietary computational framework was applied to predict clinical response and survival through construction of digital twins and in silico simulations of clinical trials. Predicted log odds ratios (LOR) were generated for drug response and compared to published data from 8 suitable historical clinical trials. Three distinct models underpin this framework: Drug Efficacy, Treatment Response & Overall Survival (OS). Three publicly available datasets were used for training: Cancer Therapeutic Response Portal ( in vitro dose-response data), Cancer Cell Line Encyclopaedia and TCGA. Results: For all 8 clinical studies, the digital twin model accurately simulated both trial arms, compared drug efficacy across arms and predicted which treatment was most active. Unblinded evaluation: Our model accurately predicted gemcitabine would have greater clinical benefit than 5-FU (LOR -0.10, P <0.0001) in metastatic pancreatic cancer. In advanced breast cancer, the model predicted docetaxel had a higher response rate than doxorubicin, and carboplatin response rates were similar to docetaxel in BRCA wild-type patients. In platinum-sensitive recurrent ovarian cancer, the model predicted cisplatin-based combination had a higher response rate than paclitaxel. Blinded evaluation: Using data for paclitaxel, the model correctly predicted that nab-paclitaxel+gemcitabine response rates were higher than gemcitabine in metastatic pancreatic cancer (predicted LOR -0.090, p = <0.001). In all 3 early breast cancer studies, predictions mirrored clinical trial results, adjuvant ECF > CMF, adjuvant TC > AC and adjuvant tamoxifen was superior to capecitabine. The underlying pan-tumour Overall Survival model used a Random Survival Forest approach, trained on 23 cancers with predictive accuracy of 0.78 AUC ROC and C-index 0.71. To enable cohort enrichment for drug response, the model segmented patient cohorts into treatment responder and non-responder. Using this approach, the log odds increased for drug response rate for 16 of the 19 different cancers and 14 of the different 17 drugs. Conclusions: Our digital twin model represents transformative technology for clinical trial simulation with strong potential for accelerating and de-risking clinical drug development, and utility for synthetic control arms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a659d949c4ba645ce412729470f4959a538035" target='_blank'>
              Prediction of therapeutic response and cancer outcomes in solid tumours via in silico clinical trials.
              </a>
            </td>
          <td>
            Uzma Asghar, M. Griffiths, Amanzhol Kubeyev, Jordan Laurie, Andrea Giorni, Luiz A Zillmann da Silva, Prabu Sivasubramaniam, Matthew Foster, Andrew Biankin
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 A reliable means of hydrate plugging risk assessment in pipelines is critical to the modern practice of production in the hydrate management regime. Flow assurance engineers utilize computationally expensive multiphase flow simulations to characterize hydrate formation at desired conditions, however, there is no numerical method to assess the risk of a plug occurring from these results. Traditional machine learning models have shown reasonably accurate plugging risk classification and require just milliseconds to return an assessment. Despite this, there has been limited industry use due to concerns about the statistical nature of predictions and the sparsity of available training data.
 Deep neural networks (DNNs) are a purely data-driven machine learning model that require large quantities of labeled data to make accurate statistical predictions in their trained domain. Physics-informed neural networks (PINNs) are a variation of DNNs in which training additionally considers embedded domain physics, in the form of partial differential equations, to increase accuracy, lessen reliance on training data, and ground predictions. This work presents a PINN that has been trained to predict hydrate plugging risk. Training was directed by the mean squared error of the model's prediction against flowloop data and, critically, the residual of the hydrate intrinsic kinetics equation. The trained model showed improved accuracy over reference DNNs. A PINN of novel architecture embedded with the hydrate intrinsic kinetics equation was built in TensorFlow. Flowloop data from pilot-scale flowloops was used for the training and evaluation of the presented PINN. Performance was compared to two DNNs for plugging risk assessment. DNN1 was an earlier model presented at OTC 2019. DNN2 features identical architecture to the subject PINN but absent of the embedded physics. DNN1 was employed as a baseline for plugging risk assessment performance, whereas DNN2 was used to isolate the contribution of the embedded domain knowledge on inference accuracy. The PINN showed a plugging risk assessment accuracy of 98.7%, which is a meaningful improvement over the 95.0% accuracy offered by DNN1. Moreover, case studies show improved confidence in plug prediction. The effect of the embedded physics on model accuracy is quantified by a reduction in mean squared error of 13.3% in inference of hydrate volume fraction when compared to DNN2. These findings indicate that the increased accuracy is the result of the embedding of the hydrate intrinsic kinetics equation as well as the novel network architecture. Two additional PINNs were presented, further establish the superior behavior of PINNs in learning the solution to PDEs and under data-sparse conditions. This work provides a new approach for machine learning in hydrates by demonstrating a technique to accurately train neural networks through a combination of empirical data and domain knowledge. This line of research could ultimately lead to more informed quantification of hydrate plugging risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d4f41759d0a35ad1acec8879e15f205be843df5" target='_blank'>
              Physics-Informed Neural Networks for Gas Hydrate Plugging Risk Assessment Using Intrinsic Kinetics and Flowloop Data
              </a>
            </td>
          <td>
            Seth Dale, Doug Turner, S. Afra, A. Teixeira, L. Valim, C. Koh, Dinesh Mehta
          </td>
          <td>2024-04-29</td>
          <td>Day 4 Thu, May 09, 2024</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Paired single-cell sequencing technologies enable the simultaneous measurement of complementary modalities of molecular data at single-cell resolution. Along with the advances in these technologies, many methods based on variational autoencoders have been developed to integrate these data. However, these methods do not explicitly incorporate prior biological relationships between the data modalities, which could significantly enhance modeling and interpretation. We propose a novel probabilistic learning framework that explicitly incorporates conditional independence relationships between multi-modal data as a directed acyclic graph using a generalized hierarchical variational autoencoder. We demonstrate the versatility of our framework across various applications pertinent to single-cell multi-omics data integration. These include the isolation of common and distinct information from different modalities, modality-specific differential analysis, and integrated cell clustering. We anticipate that the proposed framework can facilitate the construction of highly flexible graphical models that can capture the complexities of biological hypotheses and unravel the connections between different biological data types, such as different modalities of paired single-cell multi-omics data. The implementation of the proposed framework can be found in the repository https://github.com/kuijjerlab/CAVACHON.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5edcd7da413e72fd7c1ca0c333150327939b18" target='_blank'>
              CAVACHON: a hierarchical variational autoencoder to integrate multi-modal single-cell data
              </a>
            </td>
          <td>
            Ping-Han Hsieh, Ru-Xiu Hsiao, Katalin Ferenc, Anthony Mathelier, R. Burkholz, Chien-Yu Chen, G. K. Sandve, T. Belova, M. Kuijjer
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract A major challenge of precision oncology is the identification and prioritization of suitable treatment options based on molecular biomarkers of the considered tumor. In pursuit of this goal, large cancer cell line panels have successfully been studied to elucidate the relationship between cellular features and treatment response. Due to the high dimensionality of these datasets, machine learning (ML) is commonly used for their analysis. However, choosing a suitable algorithm and set of input features can be challenging. We performed a comprehensive benchmarking of ML methods and dimension reduction (DR) techniques for predicting drug response metrics. Using the Genomics of Drug Sensitivity in Cancer cell line panel, we trained random forests, neural networks, boosting trees and elastic nets for 179 anti-cancer compounds with feature sets derived from nine DR approaches. We compare the results regarding statistical performance, runtime and interpretability. Additionally, we provide strategies for assessing model performance compared with a simple baseline model and measuring the trade-off between models of different complexity. Lastly, we show that complex ML models benefit from using an optimized DR strategy, and that standard models—even when using considerably fewer features—can still be superior in performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2eaacf9f8df2242ab429c53e183de03dc2d3222" target='_blank'>
              A comprehensive benchmarking of machine learning algorithms and dimensionality reduction methods for drug sensitivity prediction
              </a>
            </td>
          <td>
            Lea Eckhart, Kerstin Lenhof, Lisa-Marie Rolli, H-P. Lenhof
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The attrition rate of drugs in clinical trials is generally quite high, with estimates suggesting that approximately 90% of drugs fail to make it through the process. The identification of unexpected toxicity issues during preclinical stages is a significant factor contributing to this high rate of failure. These issues can have a major impact on the success of a drug and must be carefully considered throughout the development process. These late-stage rejections or withdrawals of drug candidates significantly increase the costs associated with drug development, particularly when toxicity is detected during clinical trials or after market release. Understanding drug-biological target interactions is essential for evaluating compound toxicity and safety, as well as predicting therapeutic effects and potential off-target effects that could lead to toxicity. This will enable scientists to predict and assess the safety profiles of drug candidates more accurately. Evaluation of toxicity and safety is a critical aspect of drug development, and biomolecules, particularly proteins, play vital roles in complex biological networks and often serve as targets for various chemicals. Therefore, a better understanding of these interactions is crucial for the advancement of drug development. The development of computational methods for evaluating protein–ligand interactions and predicting toxicity is emerging as a promising approach that adheres to the 3Rs principles (replace, reduce, and refine) and has garnered significant attention in recent years. In this review, we present a thorough examination of the latest breakthroughs in drug toxicity prediction, highlighting the significance of drug-target binding affinity in anticipating and mitigating possible adverse effects. In doing so, we aim to contribute to the development of more effective and secure drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1666882a08f5032830257ba6d7818b9c0952be11" target='_blank'>
              Advancing Drug Safety in Drug Development: Bridging Computational Predictions for Enhanced Toxicity Prediction
              </a>
            </td>
          <td>
            Ana M B Amorim, L. F. Piochi, Ana T Gaspar, A. J. Preto, N. Rosário-Ferreira, I. Moreira
          </td>
          <td>2024-05-17</td>
          <td>Chemical Research in Toxicology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract The inherent heterogeneity of cancer contributes to highly variable responses to any anticancer treatments. This underscores the need to first identify precise biomarkers through complex multi-omics datasets that are now available. Although much research has focused on this aspect, identifying biomarkers associated with distinct drug responders still remains a major challenge. Here, we develop MOMLIN, a multi-modal and -omics machine learning integration framework, to enhance drug-response prediction. MOMLIN jointly utilizes sparse correlation algorithms and class–specific feature selection algorithms, which identifies multi-modal and -omics–associated interpretable components. MOMLIN was applied to 147 patients’ breast cancer datasets (clinical, mutation, gene expression, tumor microenvironment cells and molecular pathways) to analyze drug-response class predictions for non-responders and variable responders. Notably, MOMLIN achieves an average AUC of 0.989, which is at least 10% greater when compared with current state-of-the-art (data integration analysis for biomarker discovery using latent components, multi-omics factor analysis, sparse canonical correlation analysis). Moreover, MOMLIN not only detects known individual biomarkers such as genes at mutation/expression level, most importantly, it correlates multi-modal and -omics network biomarkers for each response class. For example, an interaction between ER-negative-HMCN1-COL5A1 mutations-FBXO2-CSF3R expression-CD8 emerge as a multimodal biomarker for responders, potentially affecting antimicrobial peptides and FLT3 signaling pathways. In contrast, for resistance cases, a distinct combination of lymph node-TP53 mutation-PON3-ENSG00000261116 lncRNA expression-HLA-E-T-cell exclusions emerged as multimodal biomarkers, possibly impacting neurotransmitter release cycle pathway. MOMLIN, therefore, is expected advance precision medicine, such as to detect context–specific multi-omics network biomarkers and better predict drug-response classifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c9478bdc170073aa91d05881b2bed54efbf90f" target='_blank'>
              Advancing drug-response prediction using multi-modal and -omics machine learning integration (MOMLIN): a case study on breast cancer clinical data
              </a>
            </td>
          <td>
            Md Mamunur Rashid, Kumar Selvarajoo
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="One of the core concepts in science, and something that happens intuitively in every-day dynamic systems modeling, is the combination of models or methods. Especially in dynamical systems modeling, often two or more structures are combined to obtain a more powerful or efficient architecture regarding a specific application (area). Further, even physical simulations are combined with machine learning architectures, to increase prediction accuracy or optimize the computational performance. In this work, we shortly discuss, which types of models are usually combined and propose a model interface that is capable of expressing a width variety of mixed algebraic, discrete and differential equation based models. Further, we examine different established, as well as new ways of combining these models from a system theoretical point of view and highlight two challenges - algebraic loops and local event affect functions in discontinuous models - that require a special approach. Finally, we propose a new wildcard topology, that is capable of describing the generic connection between two combined models in an easy to interpret fashion that can be learned as part of a gradient based optimization procedure. The contributions of this paper are highlighted at a proof of concept: Different connection topologies between two models are learned, interpreted and compared applying the proposed methodology and software implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe81ab9d1a688154b1e40c6ca68ccf95c122d0c7" target='_blank'>
              Learnable&Interpretable Model Combination in Dynamic Systems Modeling
              </a>
            </td>
          <td>
            Tobias Thummerer, Lars Mikelsons
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In this work, we introduce a novel application of foundation models in the domain of nonlinear chemical process modeling. Given the challenges of obtaining accurate first-principles models for real-world chemical processes and the inefficiency of rebuilding and retraining models for new chemical processes, we pose a pivotal question: What if we could develop a single, universal neural network (i.e., foundation model) capable of rapidly adapting to modeling any new chemical process? To address this question, we propose a meta-learning-based approach using Reptile to construct the foundation model, followed by physics-informed adaptation to fine-tune it to new modeling tasks using only a few data samples. To assess the effectiveness of our methodology, we construct a foundation model for various chemical reactions in three classical generic reactors, including continuous stirred tank reactors (CSTRs), batch reactors (BRs), and plug flow reactors (PFRs). Our approach outperforms conventional methods such as data-driven learning, physics-informed learning, transfer learning, and pure meta-learning in a few-shot setting. Furthermore, our method achieves rapid adaptation to new CSTRs, BRs, and PFRs using only a few data samples from the designated tasks. Source code is available at https://github.com/killingbear999/chemical-process-foundation-model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4edf55b1063ea6f1fd059221cbe47191bb3da351" target='_blank'>
              Foundation Model for Chemical Process Modeling: Meta-Learning with Physics-Informed Adaptation
              </a>
            </td>
          <td>
            Zihao Wang, Zhen Wu
          </td>
          <td>2024-05-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In the current field of clinical medicine, traditional methods for analyzing recurrent events have limitations when dealing with complex time-dependent data. This study combines Long Short-Term Memory networks (LSTM) with the Cox model to enhance the model's performance in analyzing recurrent events with dynamic temporal information. Compared to classical models, the LSTM-Cox model significantly improves the accuracy of extracting clinical risk features and exhibits lower Akaike Information Criterion (AIC) values, while maintaining good performance on simulated datasets. In an empirical analysis of bladder cancer recurrence data, the model successfully reduced the mean squared error during the training phase and achieved a Concordance index of up to 0.90 on the test set. Furthermore, the model effectively distinguished between high and low-risk patient groups, and the identified recurrence risk features such as the number of tumor recurrences and maximum size were consistent with other research and clinical trial results. This study not only provides a straightforward and efficient method for analyzing recurrent data and extracting features but also offers a convenient pathway for integrating deep learning techniques into clinical risk prediction systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e83aa2b5db279cd8e060c5ac6ab858f591cf20e" target='_blank'>
              LSTM-COX Model: A Concise and Efficient Deep Learning Approach for Handling Recurrent Events
              </a>
            </td>
          <td>
            Runquan Zhang, Xiaoping Shi
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Drug safety relies on advanced methods for timely and accurate prediction of side effects. To tackle this requirement, this scoping review examines machine-learning approaches for predicting drug-related side effects with a particular focus on chemical, biological, and phenotypical features. Methods: This was a scoping review in which a comprehensive search was conducted in various databases from 1 January 2013 to 31 December 2023. Results: The results showed the widespread use of Random Forest, k-nearest neighbor, and support vector machine algorithms. Ensemble methods, particularly random forest, emphasized the significance of integrating chemical and biological features in predicting drug-related side effects. Conclusions: This review article emphasized the significance of considering a variety of features, datasets, and machine learning algorithms for predicting drug-related side effects. Ensemble methods and Random Forest showed the best performance and combining chemical and biological features improved prediction. The results suggested that machine learning techniques have some potential to improve drug development and trials. Future work should focus on specific feature types, selection techniques, and graph-based methods for even better prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b346d3270a77c7b4d467e797b36b5b738915e328" target='_blank'>
              Machine Learning Techniques for Predicting Drug-Related Side Effects: A Scoping Review
              </a>
            </td>
          <td>
            Esmaeel Toni, H. Ayatollahi, Reza Abbaszadeh, Alireza Fotuhi Siahpirani
          </td>
          <td>2024-06-17</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Fluid dynamics problems are characterized by being multidimensional and nonlinear, causing the experiments and numerical simulations being complex, time-consuming and monetarily expensive. In this sense, there is a need to find new ways to obtain data in a more economical manner. Thus, in this work we study the application of time series forecasting to fluid dynamics problems, where the aim is to predict the flow dynamics using only past information. We focus our study on models based on deep learning that do not require a high amount of data for training, as this is the problem we are trying to address. Specifically in this work we have tested three autoregressive models where two of them are fully based on deep learning and the other one is a hybrid model that combines modal decomposition with deep learning. We ask these models to generate $200$ time-ahead predictions of two datasets coming from a numerical simulation and experimental measurements, where the latter is characterized by being turbulent. We show how the hybrid model generates more reliable predictions in the experimental case, as it is physics-informed in the sense that the modal decomposition extracts the physics in a way that allows us to predict it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7b99cbcae24e51709d38ebd55a9ffa77001107f" target='_blank'>
              Exploring the efficacy of a hybrid approach with modal decomposition over fully deep learning models for flow dynamics forecasting
              </a>
            </td>
          <td>
            Rodrigo Abad'ia-Heredia, A. Corrochano, Manuel López Martín, S. L. Clainche
          </td>
          <td>2024-04-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains associated with a dismal prognosis despite standard therapies. While population-level survival statistics are established, generating individualized prognosis remains challenging. We aim to develop machine learning (ML) models that generate personalized survival predictions for GBM patients to enhance prognostication.



 Adult patients with histologically confirmed IDH-wildtype GBM from the National Cancer Database (NCDB) were analyzed. ML models were developed with TabPFN, TabNet, XGBoost, LightGBM, and Random Forest algorithms to predict mortality at 6, 12, 18, and 24 months post-diagnosis. SHapley Additive exPlanations (SHAP) were employed to enhance the interpretability of the models. Models were primarily evaluated using the area under the receiver operating characteristic (AUROC) values, and the top-performing models indicated by the highest AUROCs for each outcome were deployed in a web application that was created for individualized predictions.



 A total of 7,537 patients were retrieved from the NCDB. Performance evaluation revealed the top-performing models for each outcome were built using the TabPFN algorithm. The TabPFN models yielded mean AUROCs of 0.836, 0.78, 0.732, and 0.724 in predicting 6-, 12-, 18-, and 24-month mortality, respectively.



 This study establishes ML models tailored to individual patients to enhance GBM prognostication. Future work should focus on external validation and dynamic updating as new data emerge.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dc32348f572bb38d92dcc404255ab597d191058" target='_blank'>
              Advancing Precision Prognostication in Neuro-Oncology: Machine Learning Models for Data-Driven Personalized Survival Predictions in IDH-Wildtype Glioblastoma
              </a>
            </td>
          <td>
            Mert Karabacak, Pemla Jagtiani, L. Di, Ashish H Shah, Ricardo J Komotar, Konstantinos Margetis
          </td>
          <td>2024-06-11</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Predicting the Remaining Useful Life (RUL) is essential in Prognostic Health Management (PHM) for industrial systems. Although deep learning approaches have achieved considerable success in predicting RUL, challenges such as low prediction accuracy and interpretability pose significant challenges, hindering their practical implementation. In this work, we introduce a Spatio-temporal Attention-based Hidden Physics-informed Neural Network (STA-HPINN) for RUL prediction, which can utilize the associated physics of the system degradation. The spatio-temporal attention mechanism can extract important features from the input data. With the self-attention mechanism on both the sensor dimension and time step dimension, the proposed model can effectively extract degradation information. The hidden physics-informed neural network is utilized to capture the physics mechanisms that govern the evolution of RUL. With the constraint of physics, the model can achieve higher accuracy and reasonable predictions. The approach is validated on a benchmark dataset, demonstrating exceptional performance when compared to cutting-edge methods, especially in the case of complex conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74650fd4291c6e4873e508e2767994ab7f3520d4" target='_blank'>
              Spatio-temporal Attention-based Hidden Physics-informed Neural Network for Remaining Useful Life Prediction
              </a>
            </td>
          <td>
            Feilong Jiang, Xiaonan Hou, Min Xia
          </td>
          <td>2024-05-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The development of models to predict sensitivity to anticancer drugs is an area of significant interest, given the diverse responses to treatment among patients and the considerable expense and time involved in anticancer drug development. Leveraging “omic” data and anticancer response information from the Cancer Cell Line Encyclopedia, we propose a novel approach utilizing multitask learning to enhance prediction accuracy and inference. We extended a multitask learning framework called the Data Shared Lasso to develop the Data Shared Elastic Net. This enabled the construction of tissue-specific models with information sharing while maintaining the attractive properties of Elastic Net regression. By employing this approach, we observed improvements in prediction accuracy compared to single-task Elastic Net models, particularly for cell lines displaying high sensitivity to treatment. Furthermore, the Data Shared Elastic Net facilitated the identification of predictors for anticancer drug sensitivity within specific tissue types, shedding light on cellular pathways targeted by these drugs across tissues. We also investigated the impact of data leakage on modeling outcomes from previous studies, which led to underestimating prediction error and erroneous inferences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83664bfa905ff011a2bcad47d103dd20fcdcd259" target='_blank'>
              Improving Drug Sensitivity Prediction and Inference by Multi-Task Learning
              </a>
            </td>
          <td>
            Jared Strauch, Amir Asiaee
          </td>
          <td>2024-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="5583 Background: Tumors with homologous recombination repair deficiencies (HRD+) are susceptible to PARP inhibitors (PARPi) which exploit a synthetic lethality by targeting an essential base excision repair (BER) pathway. Current biomarkers designed to predict efficacy of PARPi (BRCA1/2mut, HRD+) are ineffective, particularly in advanced stages of therapy.There is an urgent unmet need for improved PARPi biomarkers that is particularly acute for patients who fail multiple lines of therapy. We sought to develop and validate a predictive signature for identifying olaparib-sensitive patients that uses real-world data (RWD) as input. Methods: We developed a machine learning (ML) model, called DrugBERT, that learns a relationship between 1) drug structure, 2) DNA alterations, 3) drug response and 4) binding affinity between a drug and its purported target. DrugBERT utilizes genetic information from clinical commercial NGS panels and a subset of clinical features, and outputs a predictive signature for a drug of interest, without need for additional training. Each prediction is accompanied by a Vulnerability Network (VN) which represents a predicted latent genetic sensitivity in tumors and provides biological interpretability to model outputs. Because our model uses RWD as input, we were able to retrospectively evaluate our signature in a RWD ovarian cancer cohort treated with the PARPi, olaparib. Samples were filtered to biopsies taken < 2 years prior to olaparib treatment (N= 48 samples). Drug combinations were analyzed by processing each drug separately and stratifying patients based on sensitivity predictions to all drugs in the regimen. Results: Zephyr's signature outperformed existing PARPi biomarkers in a complex real-world olaparib-treated ovarian cancer cohort. OS and PFS were not significantly different when stratifying patients by BRCA1/2 mutations or HRD+. For Zephyr’s signature, both OS (p-value < 5x10-3; HR = 3.37, 95% CI: 1.27-5.46) and PFS (p-value < 10-3; HR = 2.04, 95% CI: 1.35-3.10), were statistically and clinically significantly prolonged. Characterization of VNs enriched in olaparib-predicted sensitive patient tumors indicated enhanced sensitivity to perturbing HR and cell cycle pathways. Conversely, olaparib-predicted non-sensitive ovarian tumors were characterized by VNs indicating perturbation sensitivity to cell motility and stress response pathways and displayed higher expression of DNA repair, drug resistance, and cancer stem cell pathways. Conclusions: We present a novel and enhanced approach for stratifying patients for PARPi therapy, retrospectively validated in a complex real-world ovarian cancer cohort treated with olaparib. Our interpretable model not only offers a biological hypothesis for predicted responses but is also readily applicable in clinical settings and compatible with standard multi-gene commercial NGS panels for immediate use.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d277e5ee1c329dece0aff0db401470ab93c64da" target='_blank'>
              Evaluation of a novel signature for PARP inhibitor sensitivity prediction using real-world data.
              </a>
            </td>
          <td>
            Emily Vucic, Felicia Kuperwaser, Sunil Kumar, Dillon Tracy, Olena Marchenko, Zong Miao, Jacob Kaffey, Samantha Pindak, Nathaniel Tann, Jordan Wolinsky, Sean Klei, Andrey Chursov, Fahad Khan, Jean Michel Rouly, G. Komatsoulis, Jeff Sherman, Maayan Baron
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In the trend of hybrid Artificial Intelligence (AI) techniques, Physic Informed Machine Learning has seen a growing interest. It operates mainly by imposing a data, learning or inductive bias with simulation data, Partial Differential Equations or equivariance and invariance properties. While these models have shown great success on tasks involving one physical domain such as fluid dynamics, existing methods still struggle on tasks with complex multi-physical and multi-domain phenomena. To address this challenge, we propose to leverage Bond Graphs, a multi-physics modeling approach together with Graph Neural Network. We thus propose Neural Bond Graph Encoder (NBgE), a model agnostic physical-informed encoder tailored for multi-physics systems. It provides an unified framework for any multi-physics informed AI with a graph encoder readable for any deep learning model. Our experiments on two challenging multi-domain physical systems - a Direct Current Motor and the Respiratory system - demonstrate the effectiveness of our approach on a multi-variate time series forecasting task.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5d7393bdb45ad9ec2527ea020c7d99f6d06d06" target='_blank'>
              Bond Graphs for multi-physics informed Neural Networks for multi-variate time series
              </a>
            </td>
          <td>
            Alexis-Raja Brachet, Pierre-Yves Richard, C'eline Hudelot
          </td>
          <td>2024-05-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Survival prediction is an important branch of cancer prognosis analysis. The model that predicts survival risk through TCGA genomics data can discover genes related to cancer and provide diagnosis and treatment recommendations based on patient characteristics. We found that deep learning models based on Cox proportional hazards often suffer from overfitting when dealing with high-throughput data. Moreover, we found that as the number of network layers increases, the experimental results will not get better, and network degradation will occur. Based on this problem, we propose a new framework based on Deep Residual Learning. Combine the ideas of Cox proportional hazards and Residual. And name it ResSurv. First, ResSurv is a feed-forward deep learning network stacked by multiple basic ResNet Blocks. In each ResNet Block, we add a Normalization Layer to prevent gradient disappearance and gradient explosion. Secondly, for the loss function of the neural network, we inherited the Cox proportional hazards methods, applied the semi-parametric of the CPH model to the neural network, combined with the partial likelihood model, established the loss function, and performed backpropagation and gradient update. Finally, we compared ResSurv networks of different depths and found that we can effectively extract high-dimensional features. Ablation experiments and comparative experiments prove that our model has reached SOTA(state of the art) in the field of deep learning, and our network can effectively extract deep information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2cbfe31657721f4bf6c2ca44aa455fb795d7549" target='_blank'>
              ResSurv: Cancer Survival Analysis Prediction Model Based on Residual Networks
              </a>
            </td>
          <td>
            Wankang Zhai
          </td>
          <td>2024-05-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e77a9f19eaa4ee2b782f8540e6f17c886c43437" target='_blank'>
              Fundamentals for predicting transcriptional regulations from DNA sequence patterns.
              </a>
            </td>
          <td>
            Masaru Koido, K. Tomizuka, Chikashi Terao
          </td>
          <td>2024-05-10</td>
          <td>Journal of human genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Drug resistance in cancer treatment presents a significant challenge, necessitating innovative approaches to improve therapeutic efficacy. Integrating machine learning (ML) in cancer research is promising as ML algorithms outrival in analysing complex datasets, identifying patterns, and predicting treatment outcomes. Leveraging diverse data sources such as genomic profiles, clinical records, and drug response assays, ML uncovers molecular mechanisms of drug resistance, enabling personalised treatment, maximising efficacy and minimising adverse effects. Various ML algorithms contribute to the drug discovery process- Random Forest and Decision Trees predict drug-target interactions and aid in virtual screening, and SVM classify leads on bioactivity data. Neural Networks model QSAR to optimise lead compounds and K-means clustering group compounds with similar chemical properties aiding compound selection. Gaussian Processes predict drug responses, Bayesian Networks infer causal relationships, Autoencoders generate novel compounds, and Genetic Algorithms optimise molecular structures. These algorithms collectively enhance efficiency and success rates in drug design endeavours, from lead identification to optimisation and are cost-effective, empowering clinicians with real-time treatment monitoring and improving patient outcomes. This review highlights the immense potential of ML in revolutionising cancer care through effective drug design to reduce drug resistance, and we have also discussed various limitations and research gaps to understand better.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e16966bea454fb58dc07ad8a5bb340ce722c1069" target='_blank'>
              Harnessing Machine Learning Potential for Personalised Drug Design and Overcoming Drug Resistance.
              </a>
            </td>
          <td>
            Mohammed AgeeliHakami
          </td>
          <td>2024-06-06</td>
          <td>Journal of drug targeting</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Ovarian cancer (OC) is the most lethal gynecological cancer in the United States. Among the different types of OC, serous ovarian cancer (SOC) stands out as the most prevalent. Transcriptomics techniques generate extensive gene expression data, yet only a few of these genes are relevant to clinical diagnosis. Methods: Methods for feature selection (FS) address the challenges of high dimensionality in extensive datasets. This study proposes a computational framework that applies FS techniques to identify genes highly associated with platinum-based chemotherapy response on SOC patients. Using SOC datasets from the Gene Expression Omnibus (GEO) database, LASSO and varSelRF FS methods were employed. Machine learning classification algorithms such as random forest (RF) and support vector machine (SVM) were also used to evaluate the performance of the models. Results: The proposed framework has identified biomarkers panels with 9 and 10 genes that are highly correlated with platinum–paclitaxel and platinum-only response in SOC patients, respectively. The predictive models have been trained using the identified gene signatures and accuracy of above 90% was achieved. Conclusions: In this study, we propose that applying multiple feature selection methods not only effectively reduces the number of identified biomarkers, enhancing their biological relevance, but also corroborates the efficacy of drug response prediction models in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3840852461c3aaf6b8f84cbd6005d49de7fd0573" target='_blank'>
              Evaluating Ovarian Cancer Chemotherapy Response Using Gene Expression Data and Machine Learning
              </a>
            </td>
          <td>
            Soukaina Amniouel, Keertana Yalamanchili, Sreenidhi Sankararaman, M. S. Jafri
          </td>
          <td>2024-05-22</td>
          <td>BioMedInformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Identifying latent interactions within complex systems is key to unlocking deeper insights into their operational dynamics, including how their elements affect each other and contribute to the overall system behavior. For instance, in neuroscience, discovering neuron-to-neuron interactions is essential for understanding brain function; in ecology, recognizing the interactions among populations is key for understanding complex ecosystems. Such systems, often modeled as dynamical systems, typically exhibit noisy high-dimensional and non-stationary temporal behavior that renders their identification challenging. Existing dynamical system identification methods often yield operators that accurately capture short-term behavior but fail to predict long-term trends, suggesting an incomplete capture of the underlying process. Methods that consider extended forecasts (e.g., recurrent neural networks) lack explicit representations of element-wise interactions and require substantial training data, thereby failing to capture interpretable network operators. Here we introduce Lookahead-driven Inference of Networked Operators for Continuous Stability (LINOCS), a robust learning procedure for identifying hidden dynamical interactions in noisy time-series data. LINOCS integrates several multi-step predictions with adaptive weights during training to recover dynamical operators that can yield accurate long-term predictions. We demonstrate LINOCS' ability to recover the ground truth dynamical operators underlying synthetic time-series data for multiple dynamical systems models (including linear, piece-wise linear, time-changing linear systems' decomposition, and regularized linear time-varying systems) as well as its capability to produce meaningful operators with robust reconstructions through various real-world examples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f005c8b19ca1d9171afc191fc78b81f136ddfb8c" target='_blank'>
              LINOCS: Lookahead Inference of Networked Operators for Continuous Stability
              </a>
            </td>
          <td>
            Noga Mudrik, Eva Yezerets, Yenho Chen, Christopher Rozell, Adam Charles
          </td>
          <td>2024-04-28</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
In recent years, analyzing complex biological networks to predict future links in such networks has attracted the attention of many medical and computer science researchers. The discovery of new drugs is one of the application cases for predicting future connections in biological networks. The operation of drug-target interactions prediction (DTIP) can be considered a fundamental step in identifying potential interactions between drug and target to identify new drugs.


OBJECTIVE
The previous studies reveal that predictions are made based on known interactions using computational methods to solve the cost problem and avoid blind study of all interactions. But, there seem to be challenges such as the lack of confirmed negative samples and the low accuracy in some computational methods. Thus, we have proposed an efficient and hybrid approach called MKPUL-BLM to manage some of the aforementioned challenges for predicting drug-target interactions.


METHODS
The MKPUL-BLM combins multi-kernel and positive unlabeled learning (PUL) approaches. Our method uses more information to increase accuracy, in addition to minimizing small similarities using network information. Also, potential negative samples are produced using a PUL approach because of lacking negative laboratory samples. Finally, labels are expanded via a semi-supervised.


RESULTS
Our method improved to 0.98 and 0.94 in the old interactions set for the ROCAUC and AUPR criteria, respectively. Also, this method enhanced ROCAUC and AUPR criteria by 0.89 and 0.77 for the new interactions set.


CONCLUSION
The MKPUL-BLM can be considered an efficient alternative to achieve more reliable predictions in the field of DTIP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/517b4e428c381711c80b78a700350d74395f3bac" target='_blank'>
              An Enhanced Computational Approach Using Multi-kernel Positive Unlabeled Learning for Predicting Drug-target Interactions.
              </a>
            </td>
          <td>
            M. Keyvanpour, Soheila Mehrmolaei, Faraneh Haddadi
          </td>
          <td>2024-04-29</td>
          <td>Current computer-aided drug design</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Physiological and pathological processes are governed by a network of genes called gene regulatory networks (GRNs). By reconstructing GRNs, we can accurately model how cells behave in their natural state and predict how genetic changes will affect them. Transcriptomic data of single cells are now available for a wide range of cellular processes in multiple species. Thus, a method building predictive GRNs from single-cell RNA sequencing (scRNA-seq) data, without any additional prior knowledge, could have a great impact on our understanding of biological processes and the genes playing a key role in them. To this aim, we developed IGNITE (Inference of Gene Networks using Inverse kinetic Theory and Experiments), an unsupervised machine learning framework designed to infer directed, weighted, and signed GRNs directly from unperturbed single-cell RNA sequencing data. IGNITE uses the GRNs to generate gene expression data upon single and multiple genetic perturbations. IGNITE is based on the inverse problem for a kinetic Ising model, a model from statistical physics that has been successfully applied to biological networks. We tested IGNITE on murine pluripotent stem cells (PSCs) transitioning from the naïve to formative states. Using as input only scRNA-seq data of unperturbed PSCs, IGNITE simulated single and triple gene knockouts. Comparison with experimental data revealed high accuracy, up to 74%, outperforming currently available methods. In sum, IGNITE identifies predictive GRNs from scRNA-seq data without additional prior knowledge and faithfully simulates single and multiple gene perturbations. Applications of IGNITE range from studying cell differentiation to identifying genes specifically active under pathological conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f91d1ee7feee1c23aecfd8641c38451501491d9" target='_blank'>
              Unveiling Gene Perturbation Effects through Gene Regulatory Networks Inference from single-cell transcriptomic data
              </a>
            </td>
          <td>
            Clelia Corridori, Merrit Romeike, Giorgio Nicoletti, C. Buecker, S. Suweis, S. Azaele, Graziano Martello
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="e15064 Background: This real-world study underscores the transformative potential of multidimensional tumor profiling, including tissue and blood-based markers, in guiding personalized oncology treatments. We established a robust link between comprehensive diagnostics and improved clinical outcomes by correlating extensive profiling data with progression-free survival (PFS), the ratio of PFS2 to PFS1, and comparing these with treatment modalities. Methods: Over five years, 44 patients with advanced-stage solid tumors underwent comprehensive tumor profiling, including genetic markers from tissue and blood, mRNA expression, circulating tumor cells, pharmacogenomics, and immunohistochemistry, to tailor therapies. These multidimensional datasets were rigorously analyzed applying machine learning algorithms against responses to target, chemo, and combo therapies, using robust statistical frameworks to elucidate the impact of advanced diagnostics on therapy effectiveness. Results: Multidimensional tumor testing markedly extended progression-free survival, with median PFS1 and PFS2 at 22.5 and 30 weeks, respectively t(43) = 1.76, p = 0.043, and a significant median PFS2/1 ratio of 1.31. Targeted therapies achieved the highest median PFS2 at 61 weeks, surpassing chemo alone (10.5 weeks) and combination therapy (29.5 weeks) significantly. Patients treated based on gene expression, including mRNA profiling and immunohistochemistry/immunocytochemistry (n = 22), had a median PFS of 37.5 weeks, outperforming those treated based on NGS (n = 12) with 27.5 weeks significantly. Those not fully adhering to guideline treatments due to divergent findings had a median PFS2 of 16.5 weeks. Notably, 26 out of 44 patients with critical illness had a PFS2 over 6 months. Conclusions: A clear linkage was established between comprehensive profiling indicators, including genetic, molecular, and cellular markers, and the observed therapeutic outcomes. The study reaffirms the critical importance of multidimensional tumor profiling in the era of precision oncology. However, we proceed monitoring this growing group of patients to establish a clearer picture how to adapt complex diagnostic tools into modern oncological practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d850c6e4f18d0de72ca354916f7a112b7597673" target='_blank'>
              Multidimensional tumor profiling and its meaningful impact on personalized therapy outcomes: Insights from a real-world study.
              </a>
            </td>
          <td>
            Dörthe Schaffrin-Nabe, Timothy Crook, S. Limaye, S. Schuster, A. Josten-Nabe, D. Reismann, D. Patil, V. Datta, Adrian Heinze, M. Schaffrin, R. Voigtmann, R. Kurzrock
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6829641fd0ff9667a03286b7b3001f2f9ca670d1" target='_blank'>
              Developing a machine learning model for predicting venlafaxine active moiety concentration: a retrospective study using real-world evidence.
              </a>
            </td>
          <td>
            Luyao Chang, Xin Hao, Jing Yu, Jinyuan Zhang, Yimeng Liu, Xuxiao Ye, Ze Yu, Fei Gao, Xiaolu Pang, Chunhua Zhou
          </td>
          <td>2024-05-16</td>
          <td>International journal of clinical pharmacy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Reconstructing the topology of gene regulatory network from gene expression data has been extensively studied. With the abundance functional transcriptomic data available, it is now feasible to systematically decipher regulatory interaction dynamics in a logic form such as a Boolean network (BN) framework, which qualitatively indicates how multiple regulators aggregated to affect a common target gene. However, inferring both the network topology and gene interaction dynamics simultaneously is still a challenging problem since gene expression data are typically noisy and data discretization is prone to information loss. We propose a new method for BN inference from time-series transcriptional profiles, called LogicGep. LogicGep formulates the identification of Boolean functions as a symbolic regression problem that learns the Boolean function expression and solve it efficiently through multi-objective optimization using an improved gene expression programming algorithm. To avoid overly emphasizing dynamic characteristics at the expense of topology structure ones, as traditional methods often do, a set of promising Boolean formulas for each target gene is evolved firstly, and a feed-forward neural network trained with continuous expression data is subsequently employed to pick out the final solution. We validated the efficacy of LogicGep using multiple datasets including both synthetic and real-world experimental data. The results elucidate that LogicGep adeptly infers accurate BN models, outperforming other representative BN inference algorithms in both network topology reconstruction and the identification of Boolean functions. Moreover, the execution of LogicGep is hundreds of times faster than other methods, especially in the case of large network inference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81189774cd4d14cf52c8efb5374bdb7e2ca817d4" target='_blank'>
              LogicGep: Boolean networks inference using symbolic regression from time-series transcriptomic profiling data
              </a>
            </td>
          <td>
            Dezhen Zhang, Shuhua Gao, Zhi-Ping Liu, Rui Gao
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="10061 Background: Genomics precision medicine, deployed via tumor panel sequencing, now assists in deploying targeted therapies to cancer patients. Numerous clinical trials have investigated the utility and benefit of genomics precision medicine in multiple cancer indications. Current large-scale studies report actionability rates from ~35% to ~60%, although clinical benefit rates have been shown to be closer to 10%. While this has positively impacted patients in need, the gap between actionability and benefit remains a clinical challenge attributed to multiple factors including the complex, multi-factorial relationship between molecular status and response to therapy. These differences go beyond simple disease states and may be reflective of multiple clinically relevant features including age, sex, and race/ethnicity. Methods: We implemented a functional precision medicine (FPM) program where patients with advanced pediatric cancers were prospectively profiled via high-throughput drug sensitivity testing (DST) of FDA-approved agents on patient-derived tumor cells as well as genomics testing. The objective was to investigate the clinical utility and benefit of FPM guidance in the treatment of pediatric cancer and elucidate the relationship between molecular status of patients’ diseases and treatment responses. We generated DST data (n = 21 patients) and genomic profiling data (n = 20 patients) on pediatric cancer patients in Miami, FL, as well as post-hoc whole exome and transcriptome sequencing data (n = 13 patients) and investigated three specific relationships. Results: We analyzed the relationship between racial/ethnic background and functional response to anti-cancer agents, determining potential differences in response to therapeutic classes. Next, we examined relationships between functional response and cancer type, identifying an unanticipated lack of clustering between disease indications in patients with advanced pediatric cancers. Finally, we applied an explainable machine learning (xML) framework to the functional genomic dataset to develop multi-omics biomarker hypotheses for the chemotherapy agent idarubicin, pinpointing a potential multi-cancer relationship between response to idarubicin and known disease mechanisms in acute myeloid leukemia (AML), the sole indication where idarubicin is approved. We further present additional proof-of-concept studies generating biomarker hypotheses via xML, demonstrating a framework for development of multi-omics biomarkers. Conclusions: We are now expanding our pan-pediatric cancer functional genomics dataset through an NIMHD-funded expansion cohort (NCT05857969, n = 65 patients) to further investigate multi-omics relationships between functional and molecular characteristics and understand the role of race/ethnicity in the complex relationship.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/786847ce4af81d26fccb1bc7ad4a99896c1c5825" target='_blank'>
              Biomarker development from functional precision medicine datasets via explainable machine learning.
              </a>
            </td>
          <td>
            Noah Berlow, Arlet M Acanda De La Rocha, Maggie Fader, E. Coats, Cima Saghira, P. Espinal, Jeanette Galano, Z. Khatib, Haneen Y. Abdella, O. Maher, Cristina M. Andrade-Feraud, Baylee Holcomb, Yasmin Ghurani, Lilliam Rimblas, Tom á s R. Guilarte, Nan Hu, Daria Salyakina, Diana J Azzam
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/985fb213001f71393d7581e0584e1c2910789136" target='_blank'>
              CVGAE: A Self-Supervised Generative Method for Gene Regulatory Network Inference Using Single-Cell RNA Sequencing Data.
              </a>
            </td>
          <td>
            Wei Liu, Zhijie Teng, Zejun Li, Jing Chen
          </td>
          <td>2024-05-23</td>
          <td>Interdisciplinary sciences, computational life sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cardiovascular disease (CVD) is a leading cause of death globally, necessitating precise forecasting models for monitoring vital signs like heart rate, blood pressure, and ECG. Traditional models, such as ARIMA and Prophet, are limited by their need for manual parameter tuning and challenges in handling noisy, sparse, and highly variable medical data. This study investigates advanced deep learning models, including LSTM, and transformer-based architectures, for predicting heart rate time series from the MIT-BIH Database. Results demonstrate that deep learning models, particularly PatchTST, significantly outperform traditional models across multiple metrics, capturing complex patterns and dependencies more effectively. This research underscores the potential of deep learning to enhance patient monitoring and CVD management, suggesting substantial clinical benefits. Future work should extend these findings to larger, more diverse datasets and real-world clinical applications to further validate and optimize model performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26742ab4aed967a7f387e4d0213f9543aa72739c" target='_blank'>
              Time Series Modeling for Heart Rate Prediction: From ARIMA to Transformers
              </a>
            </td>
          <td>
            Haowei Ni, Shuchen Meng, Xieming Geng, Panfeng Li, Zhuoying Li, Xupeng Chen, Xiaotong Wang, Shiyao Zhang
          </td>
          <td>2024-06-18</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The complexity, scale, and uncertainty in regulatory networks (e.g., gene regulatory networks and microbial networks) regularly pose a huge uncertainty in their models. These uncertainties often cannot be entirely reduced using limited and costly data acquired from the normal condition of systems. Meanwhile, regulatory networks often suffer from the non-identifiability issue, which refers to scenarios where the true underlying network model cannot be clearly distinguished from other possible models. Perturbation or excitation is a well-known process in systems biology for acquiring targeted data to reveal the complex underlying mechanisms of regulatory networks and overcome the non-identifiability issue. We consider a general class of Boolean network models for capturing the activation and inactivation of components and their complex interactions. Assuming partial available knowledge about the interactions between components of the networks, this paper formulates the inference process through the maximum aposteriori (MAP) criterion. We develop a Bayesian lookahead policy that systematically perturbs regulatory networks to maximize the performance of MAP inference under the perturbed data. This is achieved by optimally formulating the perturbation process in a reinforcement learning context and deriving a scalable deep reinforcement learning perturbation policy to compute near-optimal Bayesian policy. The proposed method learns the perturbation policy through planning without the need for any real data. The high performance of the proposed approach is demonstrated by comprehensive numerical experiments using the well-known mammalian cell cycle and gut microbial community networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13661311b83454b1f32c13b0d86a4299e417da66" target='_blank'>
              Bayesian Lookahead Perturbation Policy for Inference of Regulatory Networks.
              </a>
            </td>
          <td>
            Mohammad Alali, Mahdi Imani
          </td>
          <td>2024-05-17</td>
          <td>IEEE/ACM transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9165fe7b8af9c461e14f5fa3a2cd250b938e0d" target='_blank'>
              Optimizing clinico-genomic disease prediction across ancestries: a machine learning strategy with Pareto improvement
              </a>
            </td>
          <td>
            Yan Gao, Yan Cui
          </td>
          <td>2024-06-04</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/620803fdb5f14b75e2d7602b6c40e8374eb91f5a" target='_blank'>
              Enhancing drug and cell line representations via contrastive learning for improved anti-cancer drug prioritization
              </a>
            </td>
          <td>
            Patrick J. Lawrence, Benjamin Burns, Xia Ning
          </td>
          <td>2024-05-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="With the increasing availability of diverse data types, particularly images and time series data from medical experiments, there is a growing demand for techniques designed to combine various modalities of data effectively. Our motivation comes from the important areas of predicting mortality and phenotyping where using different modalities of data could significantly improve our ability to predict. To tackle this challenge, we introduce a new method that uses two separate encoders, one for each type of data, allowing the model to understand complex patterns in both visual and time-based information. Apart from the technical challenges, our goal is to make the predictive model more robust in noisy conditions and perform better than current methods. We also deal with imbalanced datasets and use an uncertainty loss function, yielding improved results while simultaneously providing a principled means of modeling uncertainty. Additionally, we include attention mechanisms to fuse different modalities, allowing the model to focus on what's important for each task. We tested our approach using the comprehensive multimodal MIMIC dataset, combining MIMIC-IV and MIMIC-CXR datasets. Our experiments show that our method is effective in improving multimodal deep learning for clinical applications. The code will be made available online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a573d07f667d0e1529b994290cffba524892276" target='_blank'>
              Towards Precision Healthcare: Robust Fusion of Time Series and Image Data
              </a>
            </td>
          <td>
            Ali Rasekh, Reza Heidari, Amir Hosein Haji Mohammad Rezaie, Parsa Sharifi Sedeh, Zahra Ahmadi, Prasenjit Mitra, Wolfgang Nejdl
          </td>
          <td>2024-05-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We consider the problem of learning the dynamics in the topology of time-evolving point clouds, the prevalent spatiotemporal model for systems exhibiting collective behavior, such as swarms of insects and birds or particles in physics. In such systems, patterns emerge from (local) interactions among self-propelled entities. While several well-understood governing equations for motion and interaction exist, they are difficult to fit to data due to the often large number of entities and missing correspondences between the observation times, which may also not be equidistant. To evade such confounding factors, we investigate collective behavior from a \textit{topological perspective}, but instead of summarizing entire observation sequences (as in prior work), we propose learning a latent dynamical model from topological features \textit{per time point}. The latter is then used to formulate a downstream regression task to predict the parametrization of some a priori specified governing equation. We implement this idea based on a latent ODE learned from vectorized (static) persistence diagrams and show that this modeling choice is justified by a combination of recent stability results for persistent homology. Various (ablation) experiments not only demonstrate the relevance of each individual model component, but provide compelling empirical evidence that our proposed model -- \textit{neural persistence dynamics} -- substantially outperforms the state-of-the-art across a diverse set of parameter regression tasks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49c1f339720cec5e7fa7b827d7664d093032838" target='_blank'>
              Neural Persistence Dynamics
              </a>
            </td>
          <td>
            Sebastian Zeng, Florian Graf, M. Uray, Stefan Huber, R. Kwitt
          </td>
          <td>2024-05-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Heterogeneous, interconnected, systems-level, molecular data have become increasingly available and key in precision medicine. We need to utilize them to better stratify patients into risk groups, discover new biomarkers and targets, repurpose known and discover new drugs to personalize medical treatment. Existing methodologies are limited and a paradigm shift is needed to achieve quantitative and qualitative breakthroughs. In this perspective paper, we survey the literature and argue for the development of a comprehensive, general framework for embedding of multi-scale molecular network data that would enable their explainable exploitation in precision medicine in linear time. Network embedding methods map nodes to points in low-dimensional space, so that proximity in the learned space reflects the network's topology-function relationships. They have recently achieved unprecedented performance on hard problems of utilizing few omic data in various biomedical applications. However, research thus far has been limited to special variants of the problems and data, with the performance depending on the underlying topology-function network biology hypotheses, the biomedical applications and evaluation metrics. The availability of multi-omic data, modern graph embedding paradigms and compute power call for a creation and training of efficient, explainable and controllable models, having no potentially dangerous, unexpected behaviour, that make a qualitative breakthrough. We propose to develop a general, comprehensive embedding framework for multi-omic network data, from models to efficient and scalable software implementation, and to apply it to biomedical informatics. It will lead to a paradigm shift in computational and biomedical understanding of data and diseases that will open up ways to solving some of the major bottlenecks in precision medicine and other domains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2728c79f21b67623d3fdbf31a0f92bf8fe73b0aa" target='_blank'>
              Simplicity within biological complexity
              </a>
            </td>
          <td>
            Nataša Pržulj, N. Malod-Dognin
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Optimal control of multiple heating, ventilation, and air-conditioning units in an open-plan space demands fast and accurate thermodynamic modeling. While computational fluid dynamics is precise, it is computationally impractical for real-time control. Lumped thermal models divide the room into cells and explicitly model inter-cell air-mixing. However, solving the coupled differential equations is still computationally challenging. Data-driven methods like long-short term memory may handle non-linear thermal dynamics but face limitations in observability, adherence to physical constraints, and generalization. Physics-informed neural networks (PINN) impose physical constraints even while learning from data. However, PINNs still lack the scalability required for effective control in large open-plan offices, primarily due to air-mixing interactions Our approach, PhyGICS, combines graph neural networks with physics-informed learning (PI-GNN) to overcome these challenges. Specifically, we model the thermodynamic interactions as edges between nodes that represent cells. Further, the modeling approach allows explicit modeling of wall and window surface temperatures that are commonly ignored. PhyGICS, utilizing PI-GNN as a state-estimator, employs a receding-horizon approach for optimal HVAC control. We adapt PI-GNNs for building HVAC control by incorporating a time-resetting strategy to handle time-dependent ambient conditions and therefore set-points. Evaluations in a simulated environment with real-world data show PhyGICS outperforms a regular PINN model and other baseline control strategies on: 1) thermal model accuracy; 2) computation time; 3) energy consumption; and 4) user comfort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b9dc12e24efd410665c6606c578c428451e250" target='_blank'>
              PhyGICS – A Physics-informed Graph Neural Network-based Intelligent HVAC Controller for Open-plan Spaces
              </a>
            </td>
          <td>
            S. Nagarathinam, Arunchandar Vasan
          </td>
          <td>2024-06-04</td>
          <td>Proceedings of the 15th ACM International Conference on Future and Sustainable Energy Systems</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The intricacies of the human genome, manifested as a complex network of genes, transcend conventional representations in text or numerical matrices. The intricate gene-to-gene relationships inherent in this complexity find a more suitable depiction in graph structures. In the pursuit of predicting gene expression, an endeavor shared by predecessors like the L1000 and Enformer methods, we introduce a novel spatial graph-neural network (GNN) approach. This innovative strategy incorporates graph features, encompassing both regulatory and structural elements. The regulatory elements include pair-wise gene correlation, biological pathways, protein-protein interaction networks, and transcription factor regulation. The spatial structural elements include chromosomal distance, histone modification and Hi-C inferred 3D genomic features. Principal Node Aggregation models, validated independently, emerge as frontrunners, demonstrating superior performance compared to traditional regression and other deep learning models. By embracing the spatial GNN paradigm, our method significantly advances the description of the intricate network of gene interactions, surpassing the performance, predictable scope, and initial requirements set by previous methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f51120aeb183a8a8caaf3a4cca0f7e47b5212e" target='_blank'>
              Reinventing gene expression connectivity through regulatory and spatial structural empowerment via principal node aggregation graph neural network.
              </a>
            </td>
          <td>
            Fengyao Yan, Limin Jiang, Danqian Chen, M. Ceccarelli, Yan Guo
          </td>
          <td>2024-06-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We introduce RatchetEHR, a novel transformer-based framework designed for the predictive analysis of electronic health records (EHR) data in intensive care unit (ICU) settings, with a specific focus on bloodstream infection (BSI) prediction. Leveraging the MIMIC-IV dataset, RatchetEHR demonstrates superior predictive performance compared to other methods, including RNN, LSTM, and XGBoost, particularly due to its advanced handling of sequential and temporal EHR data. A key innovation in RatchetEHR is the integration of the Graph Convolutional Transformer (GCT) component, which significantly enhances the ability to identify hidden structural relationships within EHR data, resulting in more accurate clinical predictions. Through SHAP value analysis, we provide insights into influential features for BSI prediction. RatchetEHR integrates multiple advancements in deep learning which together provide accurate predictions even with a relatively small sample size and highly imbalanced dataset. This study contributes to medical informatics by showcasing the application of advanced AI techniques in healthcare and sets a foundation for further research to optimize these capabilities in EHR data analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cb1928ddf8dab68298fe44be14f053d2e18eea3" target='_blank'>
              ICU Bloodstream Infection Prediction: A Transformer-Based Approach for EHR Analysis
              </a>
            </td>
          <td>
            Ortal Hirszowicz, Dvir Aran
          </td>
          <td>2024-05-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 With the rising prevalence of chronic kidney disease (CKD), particularly among the elderly, the need for effective renal replacement therapy has become increasingly crucial. Peritoneal dialysis (PD), recognized as a gentler alternative to haemodialysis, offers advantages in cognitive preservation and overall quality of life. Patient survival on PD, a tangible outcome measure, gains particular importance, especially in an aging demographic confronting end-stage kidney disease. Predicting survival thus becomes vital in facilitating individualized treatment planning and has implications on resource allocation and patient counseling.



 Our study analysed data from the UK Renal Registry and included 22 711 incident dialysis patients choosing PD between 2007 and 2022. Our objective was to employ Artificial Intelligence (AI) algorithms in developing an advanced risk stratification indicator with high predictive capabilities, specifically tailored for use in the United Kingdom's transplant selection procedure. We conducted experiments using three distinct machine learning models and evaluated their performance in terms of calibration and discrimination, utilizing metrics such as the integrated Brier score (IBS) and Harrell's concordance index. We assessed the potential clinical utility using decision curve analysis.



 The XGBoost model demonstrated excellent predictive efficacy for patient survival, with a concordance of 0.78. Further scrutiny revealed AUC values at 1 year (0.81), 3 years (0.78), and 5 years (0.77). The integrated Brier score, a comprehensive measure of predictive accuracy, was 0.09. There was no statistical difference between the actual and predicted survival probabilities (P = 0.72). Decision curve analysis accentuated the model's clinical applicability.



 This study illustrates the potential of machine learning in prognosticating patient survival in PD, offering a pathway towards individualized patient-centric management in nephrology. The findings advocate for the integration of data-driven methodologies in clinical decision-making, paving the way for enhanced personalized care in CKD management and can serve as a guide for future research.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef1a12ec970328433d33de7e7fbc1bd4f47387c" target='_blank'>
              #426 “Predict-PD”: a machine learning tool to predict patient survival in peritoneal dialysis patients
              </a>
            </td>
          <td>
            Hatem Kaies Ibrahim Elsayed Ali, Jyoti Baharani, Rizwan Hamer
          </td>
          <td>2024-05-01</td>
          <td>Nephrology Dialysis Transplantation</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Influenced by breakthroughs in LLMs, single-cell foundation models are emerging. While these models show successful performance in cell type clustering, phenotype classification, and gene perturbation response prediction, it remains to be seen if a simpler model could achieve comparable or better results, especially with limited data. This is important, as the quantity and quality of single-cell data typically fall short of the standards in textual data used for training LLMs. Single-cell sequencing often suffers from technical artifacts, dropout events, and batch effects. These challenges are compounded in a weakly supervised setting, where the labels of cell states can be noisy, further complicating the analysis. To tackle these challenges, we present sc-OTGM, streamlined with less than 500K parameters, making it approximately 100x more compact than the foundation models, offering an efficient alternative. sc-OTGM is an unsupervised model grounded in the inductive bias that the scRNAseq data can be generated from a combination of the finite multivariate Gaussian distributions. The core function of sc-OTGM is to create a probabilistic latent space utilizing a GMM as its prior distribution and distinguish between distinct cell populations by learning their respective marginal PDFs. It uses a Hit-and-Run Markov chain sampler to determine the OT plan across these PDFs within the GMM framework. We evaluated our model against a CRISPR-mediated perturbation dataset, called CROP-seq, consisting of 57 one-gene perturbations. Our results demonstrate that sc-OTGM is effective in cell state classification, aids in the analysis of differential gene expression, and ranks genes for target identification through a recommender system. It also predicts the effects of single-gene perturbations on downstream gene regulation and generates synthetic scRNA-seq data conditioned on specific cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17340a9e8433044198b5184a2bd57e4cd4f8ad2b" target='_blank'>
              sc-OTGM: Single-Cell Perturbation Modeling by Solving Optimal Mass Transport on the Manifold of Gaussian Mixtures
              </a>
            </td>
          <td>
            Andac Demir, E. Solovyeva, James Boylan, Mei Xiao, Fabrizio Serluca, S. Hoersch, Jeremy L Jenkins, Murthy S. Devarakonda, B. Kiziltan
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Drug response prediction (DRP) is a crucial phase in drug discovery, and the most important metric for its evaluation is the IC50 score. DRP results are heavily dependent on the quality of the generated molecules. Existing molecule generation methods typically employ classifier-based guidance, enabling sampling within the IC50 classification range. However, these methods fail to ensure the sampling space range's effectiveness, generating numerous ineffective molecules. Through experimental and theoretical study, we hypothesize that conditional generation based on the target IC50 score can obtain a more effective sampling space. As a result, we introduce regressor-free guidance molecule generation to ensure sampling within a more effective space and support DRP. Regressor-free guidance combines a diffusion model's score estimation with a regression controller model's gradient based on number labels. To effectively map regression labels between drugs and cell lines, we design a common-sense numerical knowledge graph that constrains the order of text representations. Experimental results on the real-world dataset for the DRP task demonstrate our method's effectiveness in drug discovery. The code is available at:https://anonymous.4open.science/r/RMCD-DBD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc3590e9b6a96f27c063cbbf748a161eabfdbb7f" target='_blank'>
              Regressor-free Molecule Generation to Support Drug Response Prediction
              </a>
            </td>
          <td>
            Kun Li, Xiuwen Gong, Shirui Pan, Jia Wu, Bo Du, Wenbin Hu
          </td>
          <td>2024-05-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MOTIVATION
Accurately predicting the driver genes of cancer is of great significance for carcinogenesis progress research and cancer treatment. In recent years, more and more deep-learning-based methods have been used for predicting cancer driver genes. However, deep learning algorithms often have black box properties and cannot interpret the output results. Here, we propose a novel cancer driver gene mining method based on heterogeneous network metapaths (MCDHGN), which uses meta-path aggregation to enhance the interpretability of predictions.


RESULTS
MCDHGN constructs a heterogeneous network by using several types of multi-omics data that are biologically linked to genes. And the differential probabilities of SNV, DNA methylation, and gene expression data between cancerous tissues and normal tissues are extracted as initial features of genes. Nine meta-paths are manually selected, and the representation vectors obtained by aggregating information within and across metapath nodes are used as new features for subsequent classification and prediction tasks. By comparing with eight homogeneous and heterogeneous network models on two pan-cancer datasets, MCDHGN has better performance on AUC and AUPR values. Additionally, MCDHGN provides interpretability of predicted cancer driver genes through the varying weights of biologically meaningful meta-paths.


AVAILABILITY
https://github.com/1160300611/MCDHGN.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b32a7b3611c1e300542c76e55e6a0bb0480476b" target='_blank'>
              MCDHGN: Heterogeneous network-based cancer driver gene prediction and interpretability analysis.
              </a>
            </td>
          <td>
            Lexiang Wang, Jingli Zhou, Xuan Wang, Yadong Wang, Junyi Li
          </td>
          <td>2024-06-12</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="G-protein coupled receptors (GPCRs) are transmembrane proteins that transmit signals from the extracellular environment to the inside of the cells. Their ability to adopt various conformational states, which influence their function, makes them crucial in pharmacoproteomic studies. While many drugs target specific GPCR states to exert their effects—thereby regulating the protein’s activity—unraveling the activation pathway remains challenging due to the multitude of intermediate transformations occurring throughout this process, and intrinsically influencing the dynamics of the receptors. In this context, computational modeling, particularly molecular dynamics (MD) simulations, may offer valuable insights into the dynamics and energetics of GPCR transformations, especially when combined with machine learning (ML) methods and techniques for achieving model interpretability for knowledge generation. The current study builds upon previous work in which the layer relevance propagation (LRP) technique was employed to interpret the predictions in a multi-class classification problem concerning the conformational states of the β2-adrenergic (β2AR) receptor from MD simulations. Here, we address the challenges posed by class imbalance and extend previous analyses by evaluating the robustness and stability of deep learning (DL)-based predictions under different imbalance mitigation techniques. By meticulously evaluating explainability and imbalance strategies, we aim to produce reliable and robust insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa30cc0ae2a6e6381d995fe687640fc3f469403" target='_blank'>
              A Study on the Robustness and Stability of Explainable Deep Learning in an Imbalanced Setting: The Exploration of the Conformational Space of G Protein-Coupled Receptors
              </a>
            </td>
          <td>
            M. A. Gutierrez-Mondragon, Alfredo Vellido, Caroline König
          </td>
          <td>2024-06-14</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65d94b705d85169b37491f103664b0080c773c3a" target='_blank'>
              Tribulations and future opportunities for artificial intelligence in precision medicine
              </a>
            </td>
          <td>
            Claudio Carini, A. Seyhan
          </td>
          <td>2024-04-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In the context of transcriptional dynamics modeled by ordinary differential equations (ODEs), the RNA level in a single cell is controlled by specific RNA kinetics parameters, which include transcription rate, splicing rate, and degradation rate. Investigating these single-cell RNA kinetics rates is pivotal for understanding RNA metabolism and the heterogeneity of complex tissues. Although metabolic labeling is an effective method to estimate these kinetics rates experimentally, it is not suitable for current large-scale conventional single-cell RNA sequencing (scRNA-seq) data. Moreover, existing methods for scRNA-seq often either neglect certain specific kinetics parameters or use inappropriate ways to fit the parameters. To address these issues, we introduce scRNAkinetics, a parallelized method that fits the kinetics parameters of the ODE for each cell using pseudo-time derived from various biological priors (e.g. cell lineage tree and differentiation potential). This approach allows for the estimation of the relative kinetics of each cell and gene in a scRNA-seq dataset. Validated on simulated datasets, scRNAkinetics can accurately infer the kinetics rates of transcription boosting, multi-branch, and time-dependent RNA degradation systems. Nevertheless, the inferred kinetics trends are concordant with previous studies on metabolic labeling and conventional scRNA-seq datasets. Furthermore, we show that scRNAkinetics can provide valuable insights into different regulatory schemes and validate the coupling between transcription and splicing in RNA metabolism. The open-source implementation of scRNAkinetics is available at https://github.com/poseidonchan/scRNAkinetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94f71a1f0958ca99a7ef6222f0f0c7006b2e729" target='_blank'>
              Inferring Single-Cell RNA Kinetics from Various Biological Priors
              </a>
            </td>
          <td>
            Yanshuo Chen, Zhengmian Hu, Ke Ni, Site Feng, Wei Chen, Heng Huang
          </td>
          <td>2024-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Machine learning (ML) algorithms are necessary to efficiently identify potent drug combinations within a large candidate space to combat drug resistance. However, existing ML approaches cannot be applied to emerging and under-studied pathogens with limited training data. To address this, we developed a transfer learning and crowdsourcing framework (TACTIC) to train ML models on data from multiple bacteria. TACTIC was built using 2,965 drug interactions from 12 bacterial strains and outperformed traditional ML models in predicting drug interaction outcomes for species that lack training data. Top TACTIC model features revealed genetic and metabolic factors that influence cross- species and species-specific drug interaction outcomes. Upon analyzing ∼600,000 predicted drug interactions across 9 metabolic environments and 18 bacterial strains, we identified a small set of drug interactions that are selectively synergistic against Gram- negative (e.g., A. baumannii) and non-tuberculous mycobacteria (NTM) pathogens. We experimentally validated synergistic drug combinations containing clarithromycin, ampicillin, and mecillinam against M. abscessus, an emerging pathogen with growing levels of antibiotic resistance. Lastly, we leveraged TACTIC to propose selectively synergistic drug combinations to treat bacterial eye infections (endophthalmitis).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5017bc92dbb38d49a788006dff57d7610ff49b95" target='_blank'>
              Transfer learning predicts species-specific drug interactions in emerging pathogens
              </a>
            </td>
          <td>
            Carolina H. Chung, David C. Chang, Nicole M. Rhoads, Madeline R. Shay, Karthik Srinivasan, M. Okezue, Ashlee D. Brunaugh, Sriram Chandrasekaran
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="This position paper takes a broad look at Physics-Enhanced Machine Learning (PEML) -- also known as Scientific Machine Learning -- with particular focus to those PEML strategies developed to tackle dynamical systems' challenges. The need to go beyond Machine Learning (ML) strategies is driven by: (i) limited volume of informative data, (ii) avoiding accurate-but-wrong predictions; (iii) dealing with uncertainties; (iv) providing Explainable and Interpretable inferences. A general definition of PEML is provided by considering four physics and domain knowledge biases, and three broad groups of PEML approaches are discussed: physics-guided, physics-encoded and physics-informed. The advantages and challenges in developing PEML strategies for guiding high-consequence decision making in engineering applications involving complex dynamical systems, are presented.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8781213f7f1857ff688376f170953bcc9fee085" target='_blank'>
              Physics-Enhanced Machine Learning: a position paper for dynamical systems investigations
              </a>
            </td>
          <td>
            Alice Cicirello
          </td>
          <td>2024-05-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Determining the aqueous solubility of the chemical compound is of great importance in-silico drug discovery. However, correctly and rapidly predicting the aqueous solubility remains a challenging task. This paper explores and evaluates the predictability of multiple machine learning models in the aqueous solubility of compounds. Specifically, we apply a series of machine learning algorithms, including Random Forest, XG-Boost, LightGBM, and CatBoost, on a well-established aqueous solubility dataset (i. e., the Huuskonen dataset) of over 1200 compounds. Experimental results show that even traditional machine learning algorithms can achieve satisfactory performance with high accuracy. In addition, our investigation goes beyond mere prediction accuracy, delving into the interpretability of models to identify key features and understand the molecular properties that influence the predicted outcomes. This study sheds light on the ability to use machine learning approaches to predict compound solubility, significantly shortening the time that researchers spend on new drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edea23b8b0e1d8e76000d6da3d598ce6e99db3ee" target='_blank'>
              Evaluation of Machine Learning Models for Aqueous Solubility Prediction in Drug Discovery
              </a>
            </td>
          <td>
            Nian Xue, Yuzhu Zhang, Sensen Liu
          </td>
          <td>2024-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The study of effective connectivity (EC) is essential in understanding how the brain integrates and responds to various sensory inputs. Model-driven estimation of EC is a powerful approach that requires estimating global and local parameters of a generative model of neural activity. Insights gathered through this process can be used in various applications, such as studying neurodevelopmental disorders. However, accurately determining EC through generative models remains a significant challenge due to the complexity of brain dynamics and the inherent noise in neural recordings, e.g., in electroencephalography (EEG). Current model-driven methods to study EC are computationally complex and cannot scale to all brain regions as required by whole-brain analyses. To facilitate EC assessment, an inference algorithm must exhibit reliable prediction of parameters in the presence of noise. Further, the relationship between the model parameters and the neural recordings must be learnable. To progress toward these objectives, we benchmarked the performance of a Bi-LSTM model for parameter inference from the Jansen-Rit neural mass model (JR-NMM) simulated EEG under various noise conditions. Additionally, our study explores how the JR-NMM reacts to changes in key biological parameters (i.e., sensitivity analysis) like synaptic gains and time constants, a crucial step in understanding the connection between neural mechanisms and observed brain activity. Our results indicate that we can predict the local JR-NMM parameters from EEG, supporting the feasibility of our deep-learning-based inference approach. In future work, we plan to extend this framework to estimate local and global parameters from real EEG in clinically relevant applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd046ab8f0278c72f414f856472d80db3160b67" target='_blank'>
              Benchmarking Deep Jansen-Rit Parameter Inference: An in Silico Study
              </a>
            </td>
          <td>
            Deepa Tilwani, Christian O'Reilly
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mathematical modeling of disease and drug action is becoming an indispensable component of drug development, underscored by recent examples of models predicting trial results. To be able to rely on such approaches, decision-makers need to be able to verify those results independently with in silico confirmatory studies. Artificial Intelligence (AI) offers a valuable avenue for improving the reproducibility of complex models, enabling their swift and software-agnostic deployment. This paper highlights AI’s impact through a case study on an Ordinary Differential Equation (ODE) model of Retinitis Pigmentosa (RP). We use a version of Chat GPT 4, a sophisticated large language model (LLM) developed by OpenAI, as customized by Mathpix company with additional capabilities. This setup facilitated the extraction of equations from the PDF and converted into a human-readable, text-based definition language called Antimony, which is part of the Python package tellurium. Subsequently, the model was converted into Systems Biology Markup Language (SBML) using tellurium and uploaded onto the jinkō platform for simulation. The RP model was efficiently and accurately implemented using AI techniques. Furthermore, we were able to reproduce the model behavior presented in the literature. Our findings advocate for the broader application of AI in mathematical model re-implementations to ensure reliability and reproducibility of the results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165890411df84ae67a8b57ed45b2f2074ac10afb" target='_blank'>
              Leveraging AI for improved reproducibility of mathematical disease models: insights from a retinitis pigmentosa case study
              </a>
            </td>
          <td>
            E. Greugny, Nicolas Ratto, Loïc Etheve, Jean-Baptise Gourlet, Frédéric Cogny, C. Monteiro
          </td>
          <td>2024-04-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell time series data frequently display considerable variability across a cell population. The current gold standard for inferring parameter distributions across cell populations is the Global Two Stage (GTS) approach for nonlinear mixed-effects (NLME) models. However, this method is computationally intensive, as it makes repeated use of non-convex optimization that in turn requires numerical integration of the underlying system. Here, we propose the Gradient Matching GTS (GMGTS) method as an efficient alternative to GTS. Gradient matching offers an integration-free approach to parameter estimation that is particularly powerful for dynamical systems that are linear in the unknown parameters, such as biochemical networks modeled by mass action kinetics. Here, we harness the power of gradient matching by integrating it into the GTS framework. To this end, we significantly expand the capabilities of gradient matching via uncertainty propagation calculations and the development of an iterative estimation scheme for partially observed systems. Through comparisons of GMGTS with GTS in different inference setups, we demonstrate that our method provides a significant computational advantage, thereby facilitating the use of complex NLME models in systems biology applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dced58f5c5737541e67c9ad6ec12731d8224877" target='_blank'>
              Gradient matching accelerates mixed-effects inference for biochemical networks
              </a>
            </td>
          <td>
            Yulan B van Oppen, Andreas Milias-Argeitis
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cardiomyopathies, often caused by mutations in genes encoding muscle proteins, are traditionally treated by phenotyping hearts and addressing symptoms post irreversible damage. With advancements in genotyping, early diagnosis is now possible, potentially preventing such damage. However, the intricate structure of muscle and its myriad proteins make treatment predictions challenging. Here we approach the problem of estimating therapeutic targets for a mutation in mouse muscle using a spatially explicit half sarcomere muscle model. We selected 9 rate parameters in our model linked to both small molecules and cardiomyopathy-causing mutations. We then randomly varied these rate parameters and simulated an isometric twitch for each combination to generate a large training dataset. We used this dataset to train a Conditional Variational Autoencoder (CVAE), a technique used in Bayesian parameter estimation. Given simulated or experimental isometric twitches, this machine learning model is able to then predict the set of rate parameters which are most likely to yield that result. We then predict the set of rate parameters associated with both control and the cardiac Troponin C (cTnC) I61Q variant in mouse trabeculae and model parameters that recover the abnormal I61Q cTnC twitches. SIGNIFICANCE Machine learning techniques have potential to accelerate discoveries in biologically complex systems. However, they require large data sets and can be challenging in high dimensional systems such as cardiac muscle. In this study, we combined experimental measures of cardiac muscle twitch forces with mechanistic simulations and a newly developed mixture of Bayesian inference with neural networks (in autoencoders) to solve the inverse problem of determining the underlying kinetics for observed force generation by cardiac muscle. The autoencoders are trained on millions of simulations spanning parameter spaces that correspond to the mechanochemistry of cardiac sarcomeres. We apply the trained model to experimental data in order to infer parameters that can explain a diseased twitch and ways to recover it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/898beeadaf2f7a68b5dd4751723ce9ccd2389b02" target='_blank'>
              Identifying mechanisms and therapeutic targets in muscle using Bayesian parameter estimation with conditional variational autoencoders
              </a>
            </td>
          <td>
            Travis Tune, Kristina B. Kooiker, Jennifer Davis, Thomas L. Daniel, F. Moussavi-Harami
          </td>
          <td>2024-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Despite recent biotechnological breakthroughs, cancer risk prediction remains a formidable computational and experimental challenge. Addressing it is critical in order to improve prevention, early detection and survival rates. Here, I briefly summarize some key emerging theoretical and computational challenges as well as recent computational advances that promise to help realize the goals of cancer-risk prediction. The focus is on computational strategies based on single-cell data, in particular on bottom-up network modeling approaches that aim to estimate cancer stemness and dedifferentiation at single-cell resolution from a systems-biological perspective. I will describe two promising methods, a tissue and cell-lineage independent one based on the concept of diffusion network entropy, and a tissue and cell-lineage specific one that uses transcription factor regulons. Application of these tools to single-cell and single-nucleus RNA-seq data from stages prior to invasive cancer reveal that they can successfully delineate the heterogeneous inter-cellular cancer-risk landscape, identifying those cells that are more likely to turn cancerous. Bottom-up systems biological modeling of single-cell omic data is a novel computational analysis paradigm that promises to facilitate the development of preventive, early detection and cancer-risk prediction strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d7a13e6b2f47623a33253ae61889093d174b00d" target='_blank'>
              Computational single-cell methods for predicting cancer risk.
              </a>
            </td>
          <td>
            Andrew E Teschendorff
          </td>
          <td>2024-06-10</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Reconstructing gene regulatory networks (GRNs) using single-cell RNA sequencing (scRNA-seq) data holds great promise for unraveling cellular fate development and heterogeneity. While numerous machine-learning methods have been proposed to infer GRNs from scRNA-seq gene expression data, many of them operate solely in a statistical or black box manner, limiting their capacity for making causal inferences between genes. In this study, we introduce GRN inference with Accuracy and Causal Explanation (GRACE), a novel graph-based causal autoencoder framework that combines a structural causal model (SCM) with graph neural networks (GNNs) to enable GRN inference and gene causal reasoning from scRNA-seq data. By explicitly modeling causal relationships between genes, GRACE facilitates the learning of regulatory context and gene embeddings. With the learned gene signals, our model successfully decoding the causal structures and alleviates the accurate determination of multiple attributes of gene regulation that is important to determine the regulatory levels. Through extensive evaluations on seven benchmarks, we demonstrate that GRACE outperforms 14 state-of-the-art GRN inference methods, with the incorporation of causal mechanisms significantly enhancing the accuracy of GRN and gene causality inference. Furthermore, the application to human peripheral blood mononuclear cell (PBMC) samples reveals cell type-specific regulators in monocyte phagocytosis and immune regulation, validated through network analysis and functional enrichment analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78acc67c6cfb1020b9bf2b2c2f841815e9417f9d" target='_blank'>
              GRACE: Unveiling Gene Regulatory Networks With Causal Mechanistic Graph Neural Networks in Single-Cell RNA-Sequencing Data.
              </a>
            </td>
          <td>
            Jiacheng Wang, Yao-Jia Chen, Quan Zou
          </td>
          <td>2024-06-19</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This work aims to improve generalization and interpretability of dynamical systems by recovering the underlying lower-dimensional latent states and their time evolutions. Previous work on disentangled representation learning within the realm of dynamical systems focused on the latent states, possibly with linear transition approximations. As such, they cannot identify nonlinear transition dynamics, and hence fail to reliably predict complex future behavior. Inspired by the advances in nonlinear ICA, we propose a state-space modeling framework in which we can identify not just the latent states but also the unknown transition function that maps the past states to the present. We introduce a practical algorithm based on variational auto-encoders and empirically demonstrate in realistic synthetic settings that we can (i) recover latent state dynamics with high accuracy, (ii) correspondingly achieve high future prediction accuracy, and (iii) adapt fast to new environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91569481a5aad3715608ea42c4bee05aa9187d5" target='_blank'>
              Identifying latent state transition in non-linear dynamical systems
              </a>
            </td>
          <td>
            cCauglar Hizli, cCaugatay Yildiz, Matthias Bethge, ST John, Pekka Marttinen
          </td>
          <td>2024-06-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The obscure drug response continues to be a limiting factor for accurate cures for cancer. Next generation sequencing technologies have propelled the pharmacogenomic studies with characterized large panels of cancer cell line at multi‐omics level. However, the sufficient integration of the multi‐omics data and the efficient prediction for drug response and synergy still remain a challenge. To address these problems, ECFD is designed, an ensemble cascade forest‐based framework that predicts drug response and synergy using five types of omics data. Experimental results show the significant advantages of the ECFD model over existing models. The best integration of feature extraction is determined and the superiorities of robust stability in the face of new and small samples are highlighted. In addition, the methodological framework highlights the explainability of the model, the mechanisms of drug resistance and drug combination treatment strategies based on explainable analyses and biological networks. In sum, ECFD may facilitate the evaluation of drug response and speculation of potential synergy therapies in personalized and precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202be07a703e58d34b9d528b6a979074db9400b4" target='_blank'>
              An Ensemble Cascade Forest‐Based Framework for Multi‐Omics Drug Response and Synergy Prediction
              </a>
            </td>
          <td>
            Ruijiang Li, Binsheng Sui, Dongjing Leng, Song He, Kunhong Liu, Xiaochen Bo
          </td>
          <td>2024-06-09</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Measurement and analysis of cardio-respiratory function in animal models is essential for our understanding of functional and disease mechanisms that pave the way for human therapeutics and treatments. However, physiological phenotyping in conscious animals is often an onerous process requiring significant observer time, attention, and manual effort while being susceptible to bias and variability that ultimately limits the kinds of questions that can be asked and answered. Once data is collected, comprehensive and accurate analysis is challenging as it often requires either the use of expensive commercial software(s) with limited flexibility to meet the investigator’s needs or lab developed scripts that are often arduous and require specialized skills to utilize, resulting in incremental insights. Here we present an overview of our high throughput physiology assessment and analysis pipeline for interrogating disease mechanisms and enabling drug discovery in congenital cardio-respiratory disorders such as the lethal sudden infant death syndrome (SIDS). We first developed a closed loop robotic platform for automated neonate cardio-respiratory experiments (Looper) that enables automated interventional studies (i.e. hypoxic induction or drug application) based on real-time physiological feedback during the experiment. We show that deployment of multiple Looper platforms in parallel enables an order of magnitude increase in data collection for a single observer over a single workday and is highly scalable to achieve yet greater outputs. When combined with well designed genetically engineered or other disease mouse models, large scale genetic, molecular, and cellular mechanistic insights and drug discovery become feasible through physiological screening paradigms. Next, we developed an open-sourced cardio-respiratory assessment software tool (Breathe Easy) that enables facile deep interrogation of terabytes of data in a high throughput fashion while offering extensive customization capabilities to allow for unique case by case analysis by more skilled users. Breathe Easy output offers near publication ready graphical outputs with significance markings and statistical tables for rapid presentation of results through manuscripts and web-publishing. Lastly, we will present our early work leveraging our robotic and software pipeline to build an analytical machine learning framework that can predict cardio-respiratory failure (death) with high temporal specificity (more than 10 min out) in our SIDS like assay and our efforts to utilize the trained algorithms for 1) real-time prediction, feedback, and intervention application in our SIDS like assays on the Looper platform to screen small molecules for novel therapeutics; 2) elucidating unique physiological mechanisms through comparative training in distinct SIDS models; and 3) determining the potential for these machine learning approaches to be used in human polysomnographic recordings to identify at risk infants. R01HL161142, R01HL130249. This is the full abstract presented at the American Physiology Summit 2024 meeting and is only available in HTML format. There are no additional versions or additional content available for this abstract. Physiology was not involved in the peer review process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f7ffb0dc8e72492d9dea6f400fcce5d67053d84" target='_blank'>
              Physio-omics? Robotic development, software engineering, and machine learning for high throughput, whole animal physiology assays as a path to disease mechanism and drug discovery
              </a>
            </td>
          <td>
            Christopher Ward, Savannah J. Lusk, Andersen Chang, Dipak Patel, N. Memos, Mahmoud Al-Madi, Russell Ray
          </td>
          <td>2024-05-01</td>
          <td>Physiology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Swarm behavior is characterized by coordinated movement among agents and has vital applications in various domains. Predicting swarm motion is critical for understanding the underlying mechanisms and dynamics of swarm behaviors. This paper focuses on the problem of swarm motion prediction and establishes a simulation dataset based on the Boids mathematical model. The conventional motion prediction problem is converted into a density map prediction task. A deep learning model, named DeepSwarm, is proposed to tackle this challenge. DeepSwarm leverages the 3D Swin Transformer U-Net architecture to capture spatio-temporal correlations at different granularities. To address the sparsity issue of the swarm density data, the Sparsity Enhanced Module (SEM) is introduced by generating foreground masks. The model is optimized using a combination of multiple loss functions which considers masks and values simultaneously. Extensive experiments show the effectiveness of the proposed DeepSwarm in predicting swarm density maps. Further ablation studies demonstrate the significance of model components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2324d9de202ac07f9e4d0cfd72f2285254da9c8" target='_blank'>
              DeepSwarm: A Deep Model for Swarm Motion Prediction Using Swin Transformer U-Net
              </a>
            </td>
          <td>
            Yuchen Ji, Xiuxian Li, Min Meng
          </td>
          <td>2024-05-07</td>
          <td>2024 7th International Symposium on Autonomous Systems (ISAS)</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6910c46af5fbea0e4058e065fec3641ac657ad" target='_blank'>
              Leveraging machine learning for predicting acute graft-versus-host disease grades in allogeneic hematopoietic cell transplantation for T-cell prolymphocytic leukaemia
              </a>
            </td>
          <td>
            G. Chandra, Junfeng Wang, Pekka Siirtola, J. Röning
          </td>
          <td>2024-05-11</td>
          <td>BMC Medical Research Methodology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="e13641 Background: Patient eligibility is a routine concept in clinical research, comprised of simple or complex set of inclusion or exclusion criteria applied in a specific sequence. Our methodology introduces the concept of ‘availability’ as a narrower term that is closer to the actual number that will be considered for a specific study and who may decide to participate. This specific research develops Machine Learning (ML) model that dynamically predicts likelihood of Multiple Myeloma progression and departure from targeted outcomes to sustain therapy and to indicate potential availability for clinical trials – to reduce screening efforts and improve screening precision of clinical research organizations. Methods: The cohort for training these models was drawn from the ConcertAI Oncology Research database, further enriched by key variables derived from unstructured data sources. Eligible patients for the analysis had a diagnosis of Multiple Myeloma made after Jan 1, 2020. Progressions were labelled using International Myeloma Working Group Criteria. Patients that did not satisfy the criteria were censored. Labels were assigned to patients after initial diagnosis of MM, and the duration between diagnosis to label was computed to build time to event (TTE) models. Over 2000 features were created to build this model, e.g., laboratory values, vitals, biomarkers, medications, ECOG, stage, etc. Variable creation preserved temporality of lab variables and vitals and noted presence of other variables. Several ML methods were employed to build the TTE models, with gradient boosting resulting in the highest c-index (0.8) based on 12 clinically relevant features. A temporal validation of the model was performed on a simulated index date. Results: Cohort used for training the model comprised of N = 4315 patients with average age at diagnosis of 72.3 years, and ~11% of the patients labelled as progressing. Temporal validation was performed on N = 1874 patients, with ~9% progression rate. The area under the curve (AUC) and weighted F1 score for this cohort was 87% and 89% respectively. Conclusions: Patient availability assessments are complex, not supported effectively by existing solutions and often require 100’s of hours from highly trained clinical personnel. An AI model predicting Multiple Myeloma progression built using a large-scale real-world dataset demonstrated 2-4x improvement in forecasting patient availability over other approaches. Integrating these solutions into clinical workflows can improve the effectiveness of research teams; enable more patients to have clinical trials as viable options at the right time; and allow care teams to have forward visibility to choices of standard-of-care and clinical trial alternatives in the best interest of patient outcomes. [Table: see text]">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71f96261be2e68d4c685fffbbf7d6bf43b30dfae" target='_blank'>
              Dynamic machine learning model to forecast patient availability for clinical trials.
              </a>
            </td>
          <td>
            Rajeev Kulkarni, Vivek Prabhakar Vaidya, Dhaval Parmar, Abhishek Tibrewal, R. B. Parikh
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Exploring the degree to which phenotypic variation, influenced by intrinsic nonlinear biological mechanisms, can be accurately captured using statistical methods is essential for advancing our comprehension of complex biological systems and predicting their functionality. Here, we examine this issue by combining a computational model of gene regulation networks with a linear additive prediction model, akin to polygenic scores utilized in genetic analyses. Inspired by the variational framework of quantitative genetics, we create a population of individual networks possessing identical topology yet showcasing diversity in regulatory strengths. By discerning which regulatory connections determine the prediction of phenotypes, we contextualize our findings within the framework of core and peripheral causal determinants, as proposed by the omnigenic model of complex traits. We establish connections between our results and concepts such as global sensitivity and local stability in dynamical systems, alongside the notion of sloppy parameters in biological models. Furthermore, we explore the implications of our investigation for the broader discourse surrounding the role of epistatic interactions in the prediction of complex phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9d655e857d6cdd755c46f5fbfbbbb85e241f74" target='_blank'>
              Polygenic prediction and gene regulation networks
              </a>
            </td>
          <td>
            J. F. Poyatos
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction With the increasing prevalence of type 2 diabetes mellitus (T2DM), there is an urgent need to discover effective therapeutic targets for this complex condition. Coding and non-coding RNAs, with traditional biochemical parameters, have shown promise as viable targets for therapy. Machine learning (ML) techniques have emerged as powerful tools for predicting drug responses. Method In this study, we developed an ML-based model to identify the most influential features for drug response in the treatment of type 2 diabetes using three medicinal plant-based drugs (Rosavin, Caffeic acid, and Isorhamnetin), and a probiotics drug (Z-biotic), at different doses. A hundred rats were randomly assigned to ten groups, including a normal group, a streptozotocin-induced diabetic group, and eight treated groups. Serum samples were collected for biochemical analysis, while liver tissues (L) and adipose tissues (A) underwent histopathological examination and molecular biomarker extraction using quantitative PCR. Utilizing five machine learning algorithms, we integrated 32 molecular features and 12 biochemical features to select the most predictive targets for each model and the combined model. Results and discussion Our results indicated that high doses of the selected drugs effectively mitigated liver inflammation, reduced insulin resistance, and improved lipid profiles and renal function biomarkers. The machine learning model identified 13 molecular features, 10 biochemical features, and 20 combined features with an accuracy of 80% and AUC (0.894, 0.93, and 0.896), respectively. This study presents an ML model that accurately identifies effective therapeutic targets implicated in the molecular pathways associated with T2DM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884985e25d267f735c330d0f1ae8ec579a0bd05e" target='_blank'>
              Comprehensive machine learning models for predicting therapeutic targets in type 2 diabetes utilizing molecular and biochemical features in rats
              </a>
            </td>
          <td>
            M. Matboli, Hiba Al-Amodi, Abdelrahman Khaled, Radwa Khaled, M. Roushdy, Marwa Ali, Gouda Ibrahim Diab, Mahmoud Fawzy Elnagar, R. A. Elmansy, Hagir H. TAhmed, Enshrah M. E. Ahmed, Doaa M Elzoghby, Hala F. M.Kamel, Mohamed F. Farag, Hind A. ELsawi, Laila M. Farid, M. B. Abouelkhair, Eman K. Habib, H. Fikry, Lobna A Saleh, Ibrahim H. Aboughaleb
          </td>
          <td>2024-05-24</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The intricate nature of the blood-brain barrier (BBB) poses a significant challenge in predicting drug permeability, which is crucial for assessing central nervous system (CNS) drug efficacy and safety. This research utilizes an innovative approach, the classification read-across structure-activity relationship (c-RASAR) framework, that leverages machine learning (ML) to enhance the accuracy of BBB permeability predictions. The c-RASAR framework seamlessly integrates principles from both read-across and QSAR methodologies, underscoring the need to consider similarity-related aspects during the development of the c-RASAR model. It is crucial to note that the primary goal of this research is not to introduce yet another model for predicting BBB permeability but rather to showcase the refinement in predicting the BBB permeability of organic compounds through the introduction of a c-RASAR approach. This groundbreaking methodology aims to elevate the accuracy of assessing neuropharmacological implications and streamline the process of drug development. In this study, an ML-based c-RASAR linear discriminant analysis (LDA) model was developed using a dataset of 7807 compounds, encompassing both BBB-permeable and -nonpermeable substances sourced from the B3DB database (freely accessible from https://github.com/theochem/B3DB), for predicting BBB permeability in lead discovery for CNS drugs. The model's predictive capability was then validated using three external sets: one containing 276,518 natural products (NPs) from the LOTUS database (accessible from https://lotus.naturalproducts.net/download) for data gap filling, another comprising 13,002 drug-like/drug compounds from the DrugBank database (available from https://go.drugbank.com/), and a third set of 56 FDA-approved drugs to assess the model's reliability. Further diversifying the predictive arsenal, various other ML-based c-RASAR models were also developed for comparison purposes. The proposed c-RASAR framework emerged as a powerful tool for predicting BBB permeability. This research not only advances the understanding of molecular determinants influencing CNS drug permeability but also provides a versatile computational platform for the rapid assessment of diverse compounds, facilitating informed decision-making in drug development and design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8768e60d9dd1ef5d7da123c9e0549e3b8c26799" target='_blank'>
              Breaking the Barriers: Machine-Learning-Based c-RASAR Approach for Accurate Blood-Brain Barrier Permeability Prediction
              </a>
            </td>
          <td>
            Vinay Kumar, Arkaprava Banerjee, Kunal Roy
          </td>
          <td>2024-05-03</td>
          <td>Journal of chemical information and modeling</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Deep learning algorithms provide a new paradigm to study high-dimensional dynamical behaviors, such as those in fusion plasma systems. Development of novel model reduction methods, coupled with detection of abnormal modes with plasma physics, opens a unique opportunity for building efficient models to identify plasma instabilities for real-time control. Our Fusion Transfer Learning (FTL) model demonstrates success in reconstructing nonlinear kink mode structures by learning from a limited amount of nonlinear simulation data. The knowledge transfer process leverages a pre-trained neural encoder-decoder network, initially trained on linear simulations, to effectively capture nonlinear dynamics. The low-dimensional embeddings extract the coherent structures of interest, while preserving the inherent dynamics of the complex system. Experimental results highlight FTL's capacity to capture transitional behaviors and dynamical features in plasma dynamics -- a task often challenging for conventional methods. The model developed in this study is generalizable and can be extended broadly through transfer learning to address various magnetohydrodynamics (MHD) modes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c07715bd86e219597bf8c42b1d974b4316c9ebd8" target='_blank'>
              FTL: Transfer Learning Nonlinear Plasma Dynamic Transitions in Low Dimensional Embeddings via Deep Neural Networks
              </a>
            </td>
          <td>
            Zhe Bai, Xishuo Wei, William Tang, L. Oliker, Zhihong Lin, Samuel Williams
          </td>
          <td>2024-04-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Interpretable machine learning has emerged as central in leveraging artificial intelligence within high-stakes domains such as healthcare, where understanding the rationale behind model predictions is as critical as achieving high predictive accuracy. In this context, feature selection assumes a pivotal role in enhancing model interpretability by identifying the most important input features in black-box models. While random forests are frequently used in biomedicine for their remarkable performance on tabular datasets, the accuracy gained from aggregating decision trees comes at the expense of interpretability. Consequently, feature selection for enhancing interpretability in random forests has been extensively explored in supervised settings. However, its investigation in the unsupervised regime remains notably limited. To address this gap, the study introduces novel methods to construct feature graphs from unsupervised random forests and feature selection strategies to derive effective feature combinations from these graphs. Feature graphs are constructed for the entire dataset as well as individual clusters leveraging the parent-child node splits within the trees, such that feature centrality captures their relevance to the clustering task, while edge weights reflect the discriminating power of feature pairs. Graph-based feature selection methods are extensively evaluated on synthetic and benchmark datasets both in terms of their ability to reduce dimensionality while improving clustering performance, as well as to enhance model interpretability. An application on omics data for disease subtyping identifies the top features for each cluster, showcasing the potential of the proposed approach to enhance interpretability in clustering analyses and its utility in a real-world biomedical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d42749e8c9bec29373302ada10a2192d891624" target='_blank'>
              Feature graphs for interpretable unsupervised tree ensembles: centrality, interaction, and application in disease subtyping
              </a>
            </td>
          <td>
            Christel Sirocchi, Martin Urschler, Bastian Pfeifer
          </td>
          <td>2024-04-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Accurate prediction of survival days (SD) is vital for planning treatments in glioma patients, as type‐IV tumors typically have a poor prognosis and meager survival rates. SD prediction is challenging and heavily dependent on the extracted feature sets. Additionally, comprehending the behavior of complex machine learning models is a vital yet challenging aspect, particularly to integrate such models into the medical domain responsibly. Therefore, this study develops a robust feature set and an ensemble‐based regressor model to predict patients' SD accurately. We aim to understand how these features behave and contribute to predicting SD. To accomplish this, we employed post‐hoc interpretable techniques, precisely Shapley Additive exPlanations (SHAP), Partial Dependence Plots (PDP), and Accumulated Local Effects (ALE) plots. Furthermore, we introduced an investigation to establish a direct connection between radiomic features and their biological significance to enhance the interpretability of radiomic features. The best SD scores on the BraTS2020 training set are 0.504 for accuracy, 59927.38 mean squared error (MSE), 20101.86 median squared error (medianSE), and 0.725 Spearman ranking coefficient (SRC). The validation set's accuracy is 0.586, MSE is 76529.43, medianSE is 41402.78, and SRC is 0.52. The proposed predictor model exhibited superior performance compared with leading contemporary approaches across multiple performance metrics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1edb9099de878a8fcb0074941c298fe58b3c579d" target='_blank'>
              Deriving and interpreting robust features for survival prediction of brain tumor patients
              </a>
            </td>
          <td>
            Snehal Rajput, Rupal A. Kapdi, Mehul S Raval, Mohendra Roy, Jayendra M Bhalodiya
          </td>
          <td>2024-05-01</td>
          <td>International Journal of Imaging Systems and Technology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bac631713bf07524bc09bc575e775e0210d5bab9" target='_blank'>
              Exploring combinations of dimensionality reduction, transfer learning, and regularization methods for predicting binary phenotypes with transcriptomic data
              </a>
            </td>
          <td>
            S. R. Oshternian, S. Loipfinger, A. Bhattacharya, Rudolf S. N. Fehrmann
          </td>
          <td>2024-04-26</td>
          <td>BMC Bioinformatics</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Drug resistance is a major barrier to effectively treating HIV/AIDS, necessitating the exploration of novel drugs and an understanding of resistance mechanisms. Genotypic and phenotypic tests, while common, are costly and time-consuming. To address these challenges, machine learning models that effectively generalize available data have been leveraged. This study aims to create multi-drug artificial neural network (MD-ANN) models based on drug-isolate-fold change (DIF) to predict the drug resistance profiles of HIV-1 reverse transcriptase inhibitors (RTIs) using the Stanford HIV Drug Resistance Database. Unlike existing isolate-fold change (IF) models, the DIF-based models can test novel RTIs against a given mutant strain. It has been shown that the DIF-based models have competitive predictive capabilities in single-drug benchmarks compared to IF-based models while taking advantage of molecular learning to test novel RTIs. The DIF-based models can rank ten FDA-approved inhibitor pairs according to their drug resistance scores. We validate our observations with an external test set consisting of drug resistance scores for novel RTIs in the CheMBL database. By combining the Stanford dataset and the external dataset, our DIF model achieves impressive classification and regression results with an accuracy of 0.774, AUC of 0.859, and r of 0.701 on the test set. To demonstrate how our DIF models learn from drug molecules, we construct a null model that only takes into account isolate representations. This null model yields an accuracy of 0.641, AUC of 0.701, and r of 0.470. We highlight the significant contribution of drug information in improving the predictive accuracy and reliability of the DIF model. The DIF models have the potential to facilitate the testing of new inhibitors across various isolates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55a9343794b5cd8c36a6dd39d82403bfddeab848" target='_blank'>
              Multi-Drug Artificial Neural Network Models for Predicting HIV-1 RTI Resistance
              </a>
            </td>
          <td>
            Sumeyye Yilmaz, H. Tunc, M. Sari
          </td>
          <td>2024-05-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/439ad7dde58b258fd3ce0d16801a8e094dbdba2e" target='_blank'>
              Estimation of linezolid exposure in patients with hepatic impairment using machine learning based on a population pharmacokinetic model.
              </a>
            </td>
          <td>
            Ru Liao, Lihong Chen, Xiaoliang Cheng, Houli Li, Taotao Wang, Yalin Dong, Haiyan Dong
          </td>
          <td>2024-05-08</td>
          <td>European journal of clinical pharmacology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Metabolomics generates complex data necessitating advanced computational methods for generating biological insight. While machine learning (ML) is promising, the challenges of selecting the best algorithms and tuning hyperparameters, particularly for nonexperts, remain. Automated machine learning (AutoML) can streamline this process; however, the issue of interpretability could persist. This research introduces a unified pipeline that combines AutoML with explainable AI (XAI) techniques to optimize metabolomics analysis. We tested our approach on two data sets: renal cell carcinoma (RCC) urine metabolomics and ovarian cancer (OC) serum metabolomics. AutoML, using Auto-sklearn, surpassed standalone ML algorithms like SVM and k-Nearest Neighbors in differentiating between RCC and healthy controls, as well as OC patients and those with other gynecological cancers. The effectiveness of Auto-sklearn is highlighted by its AUC scores of 0.97 for RCC and 0.85 for OC, obtained from the unseen test sets. Importantly, on most of the metrics considered, Auto-sklearn demonstrated a better classification performance, leveraging a mix of algorithms and ensemble techniques. Shapley Additive Explanations (SHAP) provided a global ranking of feature importance, identifying dibutylamine and ganglioside GM(d34:1) as the top discriminative metabolites for RCC and OC, respectively. Waterfall plots offered local explanations by illustrating the influence of each metabolite on individual predictions. Dependence plots spotlighted metabolite interactions, such as the connection between hippuric acid and one of its derivatives in RCC, and between GM3(d34:1) and GM3(18:1_16:0) in OC, hinting at potential mechanistic relationships. Through decision plots, a detailed error analysis was conducted, contrasting feature importance for correctly versus incorrectly classified samples. In essence, our pipeline emphasizes the importance of harmonizing AutoML and XAI, facilitating both simplified ML application and improved interpretability in metabolomics data science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aee6d529e8db5d978657c3e38c7b266f8ff9f04" target='_blank'>
              Automated Machine Learning and Explainable AI (AutoML-XAI) for Metabolomics: Improving Cancer Diagnostics
              </a>
            </td>
          <td>
            O. Bifarin, Facundo M. Fernández
          </td>
          <td>2024-05-01</td>
          <td>Journal of the American Society for Mass Spectrometry</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Networked dynamical systems are widely used as formal models of real-world cascading phenomena, such as the spread of diseases and information. Prior research has addressed the problem of learning the behavior of an unknown dynamical system when the underlying network has a single layer. In this work, we study the learnability of dynamical systems over multilayer networks, which are more realistic and challenging. First, we present an efficient PAC learning algorithm with provable guarantees to show that the learner only requires a small number of training examples to infer an unknown system. We further provide a tight analysis of the Natarajan dimension which measures the model complexity. Asymptotically, our bound on the Nararajan dimension is tight for almost all multilayer graphs. The techniques and insights from our work provide the theoretical foundations for future investigations of learning problems for multilayer dynamical systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4c0063e47ca85cf9c10b3ade6904eaa76704c7" target='_blank'>
              Efficient PAC Learnability of Dynamical Systems Over Multilayer Networks
              </a>
            </td>
          <td>
            Zirou Qiu, Abhijin Adiga, M. Marathe, S. S. Ravi, D. Rosenkrantz, R. Stearns, Anil Vullikanti
          </td>
          <td>2024-05-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Drug discovery is a complex process that involves sequentially screening and examining a vast array of molecules to identify those with the target properties. This process, also referred to as sequential experimentation, faces challenges due to the vast search space, the rarity of target molecules, and constraints imposed by limited data and experimental budgets. To address these challenges, we introduce a human-in-the-loop framework for sequential experiments in drug discovery. This collaborative approach combines human expert knowledge with deep learning algorithms, enhancing the discovery of target molecules within a specified experimental budget. The proposed algorithm processes experimental data to recommend both promising molecules and those that could improve its performance to human experts. Human experts retain the final decision-making authority based on these recommendations and their domain expertise, including the ability to override algorithmic recommendations. We applied our method to drug discovery tasks using real-world data and found that it consistently outperforms all baseline methods, including those which rely solely on human or algorithmic input. This demonstrates the complementarity between human experts and the algorithm. Our results provide key insights into the levels of humans' domain knowledge, the importance of meta-knowledge, and effective work delegation strategies. Our findings suggest that such a framework can significantly accelerate the development of new vaccines and drugs by leveraging the best of both human and artificial intelligence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb16349d53232ad4fb92ecb11572a1778c99da9" target='_blank'>
              Collaborative Intelligence in Sequential Experiments: A Human-in-the-Loop Framework for Drug Discovery
              </a>
            </td>
          <td>
            Jinghai He, Cheng Hua, Yingfei Wang, Zeyu Zheng
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Modeling and forecasting the dynamics of complex systems, such as moderate pressure capacitively coupled plasma (CCP) systems, remains a challenge due to the interactions of physical and chemical processes across multiple scales. Historically, optimization for a given application would be accomplished via a design of experiment (DOE) study across the various external control parameters. Machine learning (ML) techniques show the potential to “forecast” process conditions not tested in a traditional DOE study and thereby allow better optimization and control of a plasma tool. In this article, we have used standard DOE as well as ML predictions to analyze I-V data in a moderate-pressure CCP system. We have demonstrated that supervised regression ML techniques can be a useful tool for extrapolating data even when a plasma system is undergoing a transition in the heating mode, in this case from the alpha to gamma mode. Classification analysis of control parameters is another possible application of ML techniques that can be deployed for system control. Here, we show that given a large set of measured data, the models can identify the gas ratio in the feed gas as well as correctly identify the operating pressure and electrode gap in almost all the cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5d56bdf2c513202454ebc1099849bb24098757b" target='_blank'>
              Case study in machine learning for predicting moderate pressure plasma behavior
              </a>
            </td>
          <td>
            S. Hussain, David J. Lary, K. Hara, K. Bera, Shahid Rauf, M. Goeckner
          </td>
          <td>2024-05-14</td>
          <td>Journal of Vacuum Science &amp; Technology A</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The integration of Graph Neural Networks (GNNs) and Neural Ordinary and Partial Differential Equations has been extensively studied in recent years. GNN architectures powered by neural differential equations allow us to reason about their behavior, and develop GNNs with desired properties such as controlled smoothing or energy conservation. In this paper we take inspiration from Turing instabilities in a Reaction Diffusion (RD) system of partial differential equations, and propose a novel family of GNNs based on neural RD systems. We \textcolor{black}{demonstrate} that our RDGNN is powerful for the modeling of various data types, from homophilic, to heterophilic, and spatio-temporal datasets. We discuss the theoretical properties of our RDGNN, its implementation, and show that it improves or offers competitive performance to state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57254c8083a6fe4f39ea4b61b190c97bb047ab60" target='_blank'>
              Graph Neural Reaction Diffusion Models
              </a>
            </td>
          <td>
            Moshe Eliasof, Eldad Haber, Eran Treister
          </td>
          <td>2024-06-16</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Modeling tools aim to predict potential drug side effects, although they suffer from imperfect performance. Specifically, protein-protein interaction models predict drug effects from proteins surrounding drug targets, but they tend to overpredict drug phenotypes and require well-defined pathway phenotypes. In this study, we used PathFX, a protein-protein interaction tool, to predict side effects for active ingredient-side effect pairs extracted from drug labels. We observed limited performance and defined new pathway phenotypes using pathway engineering strategies. We defined new pathway phenotypes using a network-based and gene expression-based approach. Overall, we discovered a trade-off between sensitivity and specificity values and demonstrated a way to limit overprediction for side effects with sufficient true positive examples. We compared our predictions to animal models and demonstrated similar performance metrics, suggesting that protein-protein interaction models do not need perfect evaluation metrics to be useful. Pathway engineering, through the inclusion of true positive examples and omics measurements, emerges as a promising approach to enhance the utility of protein interaction network models for drug effect prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87aba35205fd5415fbfb9a8e7326f3f6458e293a" target='_blank'>
              Preclinical side effect prediction through pathway engineering of protein interaction network models.
              </a>
            </td>
          <td>
            Mohammadali Alidoost, Jennifer L Wilson
          </td>
          <td>2024-05-12</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Machine learning is increasingly employed in engineering, with one of its primary applications being the construction of surrogate models to replace computationally expensive physical models for analysis and design, especially control co-design which integrates physical and control system design. In cases where generating labels for training through physical models is computationally intensive, label-free machine learning offers a viable alternative. However, surrogate models built from label-free machine learning typically have prediction errors, which can be characterized and quantified through epistemic uncertainty, representing model-form uncertainty. Moreover, when these surrogate models are used in optimization design for real-world applications, inherent random variables introduce aleatory uncertainty. This study introduces a robust design optimization method that addresses the intertwined epistemic and aleatory uncertainty. By optimizing both the average product performance and reducing uncertainty stemming from the coupled uncertainty, the method achieves improved robustness. A four-bar linkage mechanism design serves as a demonstration of this approach. The surrogate model for the design is constructed using label-free neural network, accommodating a system of physical equations. The error of the surrogate model is assessed through Gaussian Process regression, using existing training points and derivatives of the physical equations at these points. The design of the four-bar linkage aims to minimize both its average motion error and the variability of the error attributed to coupled uncertainty.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef8bdfc8b842dd50054b97d50b7f9695cf4df0d9" target='_blank'>
              Optimization Under Uncertainty Using Physics-Based Label-Free Machine Learning
              </a>
            </td>
          <td>
            Xiaoping Du
          </td>
          <td>2024-05-23</td>
          <td>2024 International Congress on Human-Computer Interaction, Optimization and Robotic Applications (HORA)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Understanding disease progression is crucial for detecting critical transitions and finding trigger molecules, facilitating early diagnosis interventions. However, the high dimensionality of data and the lack of aligned samples across disease stages have posed challenges in addressing these tasks. We present a novel framework, Gaussian Graphical Optimal Transport (GGOT), for analyzing disease progressions. The proposed GGOT uses Gaussian graphical models, incorporating protein interaction networks, to characterize the data distributions at different disease stages. Then we use population-level optimal transport to calculate the Wasserstein distances and transport maps between stages, enabling us to detect critical transitions. By analyzing the per-molecule transport distance, we quantify the importance of each molecule and identify trigger molecules. Moreover, GGOT predicts the occurrence of critical transitions in unseen samples and visualizes the disease progression process. We apply GGOT to the simulation dataset and six disease datasets with varying disease progression rates, to show its effectiveness for detecting critical transitions and identifying trigger molecules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c630a7db872ff99b8ab29b3e481270d83c0d71f" target='_blank'>
              Uncovering critical transitions and molecule mechanisms in disease progressions using Gaussian graphical optimal transport
              </a>
            </td>
          <td>
            Wenbo Hua, Ruixia Cui, Heran Yang, Jingyao Zhang, Chang Liu, Jian Sun
          </td>
          <td>2024-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Intratumour phenotypic heterogeneity is nowadays understood to play a critical role in disease progression and treatment failure. Accordingly, there has been increasing interest in the development of mathematical models capable of capturing its role in cancer cell adaptation. This can be systematically achieved by means of models comprising phenotype-structured nonlocal partial differential equations, tracking the evolution of the phenotypic density distribution of the cell population, which may be compared to gene and protein expression distributions obtained experimentally. Nevertheless, given the high analytical and computational cost of solving these models, much is to be gained from reducing them to systems of ordinary differential equations for the moments of the distribution. We propose a generalised method of model-reduction, relying on the use of a moment generating function, Taylor series expansion and truncation closure, to reduce a nonlocal reaction-advection-diffusion equation, with general phenotypic drift and proliferation rate functions, to a system of moment equations up to arbitrary order. Our method extends previous results in the literature, which we address via two examples, by removing any \textit{a priori} assumption on the shape of the distribution, and provides a flexible framework for mathematical modellers to account for the role of phenotypic heterogeneity in cancer adaptive dynamics, in a simpler mathematical framework.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d951ebfbeb9789f55de6ae13ba4fa4d0ddb0abc" target='_blank'>
              Reducing phenotype-structured PDE models of cancer evolution to systems of ODEs: a generalised moment dynamics approach
              </a>
            </td>
          <td>
            Chiara Villa, P. K. Maini, Alexander P Browning, A. Jenner, Sara Hamis, Tyler Cassidy
          </td>
          <td>2024-06-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In the present article, we propose a paradigm shift on evolving Artificial Neural Networks (ANNs) towards a new bio-inspired design that is grounded on the structural properties, interactions, and dynamics of protein networks (PNs): the Artificial Protein Network (APN). This introduces several advantages previously unrealized by state-of-the-art approaches in NE: (1) We can draw inspiration from how nature, thanks to millions of years of evolution, efficiently encodes protein interactions in the DNA to translate our APN to silicon DNA. This helps bridge the gap between syntax and semantics observed in current NE approaches. (2) We can learn from how nature builds networks in our genes, allowing us to design new and smarter networks through EA evolution. (3) We can perform EA crossover/mutation operations and evolution steps, replicating the operations observed in nature directly on the genotype of networks, thus exploring and exploiting the phenotypic space, such that we avoid getting trapped in sub-optimal solutions. (4) Our novel definition of APN opens new ways to leverage our knowledge about different living things and processes from biology. (5) Using biologically inspired encodings, we can model more complex demographic and ecological relationships (e.g., virus-host or predator-prey interactions), allowing us to optimise for multiple, often conflicting objectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdbe8cd6d06b80d544472cbb9efc805782d0782" target='_blank'>
              Protein pathways as a catalyst to directed evolution of the topology of artificial neural networks
              </a>
            </td>
          <td>
            Oscar Lao, Konstantinos Zacharopoulos, A. Fournaris, Rossano Schifanella, Ioannis Arapakis
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Study Design
A retrospective study.


Purpose
This study aimed to develop machine-learning algorithms for predicting survival in patients who underwent surgery for spinal metastasis.


Overview of Literature
This study develops machine-learning models to predict postoperative survival in spinal metastasis patients, filling the gaps of traditional prognostic systems. Utilizing data from 389 patients, the study highlights XGBoost and CatBoost algorithms̓ effectiveness for 90, 180, and 365-day survival predictions, with preoperative serum albumin as a key predictor. These models offer a promising approach for enhancing clinical decision-making and personalized patient care.


Methods
A registry of patients who underwent surgery (instrumentation, decompression, or fusion) for spinal metastases between 2004 and 2018 was used. The outcome measure was survival at postoperative days 90, 180, and 365. Preoperative variables were used to develop machine-learning algorithms to predict survival chance in each period. The performance of the algorithms was measured using the area under the receiver operating characteristic curve (AUC).


Results
A total of 389 patients were identified, with 90-, 180-, and 365-day mortality rates of 18%, 41%, and 45% postoperatively, respectively. The XGBoost algorithm showed the best performance for predicting 180-day and 365-day survival (AUCs of 0.744 and 0.693, respectively). The CatBoost algorithm demonstrated the best performance for predicting 90-day survival (AUC of 0.758). Serum albumin had the highest positive correlation with survival after surgery.


Conclusions
These machine-learning algorithms showed promising results in predicting survival in patients who underwent spinal palliative surgery for spinal metastasis, which may assist surgeons in choosing appropriate treatment and increasing awareness of mortality-related factors before surgery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da8b1a1123e75672c6b71b03b76367c6634251aa" target='_blank'>
              Development and internal validation of machine-learning models for predicting survival in patients who underwent surgery for spinal metastases.
              </a>
            </td>
          <td>
            Borriwat Santipas, Kanyakorn Veerakanjana, Piyalitt Ittichaiwong, Piya Chavalparit, Sirichai Wilartratsami, Panya Luksanapruksa
          </td>
          <td>2024-05-20</td>
          <td>Asian spine journal</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Precision dosing strategies require accounting for between-patient variability in pharmacokinetics (PK), affecting drug exposure, and in pharmacodynamics (PD), affecting response achieved at the same drug concentration at the site of action. Although liquid biopsy for assessing different levels of molecular drug targets has yet to be established, individual characterization of drug elimination pathways using liquid biopsy has recently been demonstrated. The feasibility of applying this approach in conjunction with modeling tools to guide individual dosing remains unexplored. In this study, we aimed to individualize physiologically-based pharmacokinetic (PBPK) models based on liquid biopsy measurements in plasma from 25 donors with different grades of renal function who were previously administered oral midazolam as part of a microdose cocktail. Virtual twin models were constructed based on demographics, renal function, and hepatic expression of relevant pharmacokinetic pathways projected from liquid biopsy output. Simulated exposure (AUC) to midazolam was in agreement with observed data (AFE = 1.38, AAFE = 1.78). Simulated AUC variability with three dosing approaches indicated higher variability with uniform dosing (14-fold) and stratified dosing (13-fold) compared with individualized dosing informed by liquid biopsy (fivefold). Further, exosome screening revealed mRNA expression of 532 targets relevant to drug metabolism and disposition (169 enzymes and 361 transporters). Data related to these targets can be used to further individualize PBPK models for pathways relevant to PK of other drugs. This study provides additional verification of liquid biopsy-informed PBPK modeling approaches, necessary to advance strategies that seek to achieve precise dosing from the start of treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2a6ba426b266ed03053a84f750828d9bb2885e2" target='_blank'>
              Dealing With Variable Drug Exposure Due to Variable Hepatic Metabolism: A Proof-of-Concept Application of Liquid Biopsy in Renal Impairment.
              </a>
            </td>
          <td>
            A. Rostami-Hodjegan, Zubida M. Al-Majdoub, Y. von Grabowiecki, K. Yee, Sudhakar Sahoo, Wolfgang Breitwieser, Aleksandra Galetin, Christopher Gibson, Brahim Achour
          </td>
          <td>2024-05-13</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Co-folding models are the newest innovation in deep-learning-based protein-ligand structure prediction. The recent publications of RoseTTAFold All-Atom and AlphaFold 3 have shown high-quality results on predicting the structures of proteins interacting with small-molecules, other proteins and nucleic-acids. Despite these advanced capabilities and broad potential, the current study presents critical findings that question these models’ adherence to fundamental physical principles and its susceptibility to overfitting specific protein families. Through adversarial examples based on established physical, chemical, and biological principles, we demonstrate notable discrepancies in protein-ligand structural predictions when subjected to biologically plausible perturbations. These discrepancies reveal a significant divergence from expected physical behaviors, indicating potential overfitting to particular data subsets within its training corpus. Our findings underscore the models’ limitations in generalizing effectively across diverse biological structures and highlight the necessity of integrating robust physical and chemical priors in the development of such predictive tools. The results advocate a measured reliance on deep-learning-based models for critical applications in drug discovery and protein engineering, where a deep understanding of the underlying physical and chemical properties is crucial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246574ae80d8bdbec4e6397732952f7f393a2387" target='_blank'>
              Do Deep Learning Models for Co-Folding Learn the Physics of Protein-Ligand Interactions?
              </a>
            </td>
          <td>
            Matthew Masters, Amr H. Mahmoud, Markus A. Lill
          </td>
          <td>2024-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Machine learning models support computer-aided molecular design and compound optimization. However, the initial phases of drug discovery often face a scarcity of training data for these models. Meta-learning has emerged as a potentially promising strategy, harnessing the wealth of structure-activity data available for known targets to facilitate efficient few-shot model training for the specific target of interest. In this study, we assessed the effectiveness of two different meta-learning methods, namely model-agnostic meta-learning (MAML) and adaptive deep kernel fitting (ADKF), specifically in the regression setting. We investigated how factors such as dataset size and the similarity of training tasks impact predictability. The results indicate that ADKF significantly outperformed both MAML and a single-task baseline model on the inhibition data. However, the performance of ADKF varied across different test tasks. Our findings suggest that considerable enhancements in performance can be anticipated primarily when the task of interest is similar to the tasks incorporated in the meta-learning process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32a6e659b8193146d505ac34674aa54852ff102" target='_blank'>
              Task-similarity is a crucial factor for few-shot meta-learning of structure-activity relationships.
              </a>
            </td>
          <td>
            Alexander Kötter, Stephan Allenspach, Christoph Grebner, Hans Matter, J. Hiss, Gisbert Schneider, Gerhard Hessler
          </td>
          <td>2024-04-29</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="A network, whose nodes are genes and whose directed edges represent positive or negative influences of a regulatory gene and its targets, is often used as a representation of causality. To infer a network, researchers often develop a machine learning model and then evaluate the model based on its match with experimentally verified “gold standard” edges. The desired result of such a model is a network that may extend the gold standard edges. Since networks are a form of visual representation, one can compare their utility with architectural or machine blueprints. Blueprints are clearly useful because they provide precise guidance to builders in construction. If the primary role of gene regulatory networks is to characterize causality, then such networks should be good tools of prediction because prediction is the actionable benefit of knowing causality. But are they? In this paper, we compare prediction quality based on “gold standard” regulatory edges from previous experimental work with non-linear models inferred from time series data across four different species. We show that the same non-linear machine learning models have better predictive performance, with improvements from 5.3% to 25.3% in terms of the reduction in the root mean square error (RMSE) compared with the same models based on the gold standard edges. Having established that networks fail to characterize causality properly, we suggest that causality research should focus on four goals: (i) predictive accuracy; (ii) a parsimonious enumeration of predictive regulatory genes for each target gene g; (iii) the identification of disjoint sets of predictive regulatory genes for each target g of roughly equal accuracy; and (iv) the construction of a bipartite network (whose node types are genes and models) representation of causality. We provide algorithms for all goals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd7eda0fbce6aa023e877e8ea11a162f2aca8b4" target='_blank'>
              Bipartite networks represent causality better than simple networks: evidence, algorithms, and applications
              </a>
            </td>
          <td>
            Bingran Shen, Gloria Curozzi, Dennis Shasha
          </td>
          <td>2024-05-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Computational models are not just appealing because they can simulate and predict the development of biological phenomena across multiple spatial and temporal scales, but also because they can integrate information from well-established in vitro and in vivo models and test new hypotheses in cancer biomedicine. Agent-based models and simulations are especially interesting candidates among computational modeling procedures in cancer research due to the capability to, for instance, recapitulate the dynamics of neoplasia and tumor - host interactions. Yet, the absence of methods to validate the consistency of the results across scales can hinder adoption by turning fine-tuned models into black boxes. This review compiles relevant literature that explores strategies to leverage high-fidelity simulations of multi-scale, or multi-level, cancer models with a focus on verification approached as simulation calibration. We consolidate our review with an outline of modern approaches for agent-based models' validation and provide an ambitious outlook toward rigorous and reliable calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e0c3944c13aa863a5f0e7e95045776f19c4393" target='_blank'>
              Agent-based modeling in cancer biomedicine: applications and tools for calibration and validation.
              </a>
            </td>
          <td>
            Nicolò Cogno, Cristian Axenie, Roman Bauer, V. Vavourakis
          </td>
          <td>2024-04-28</td>
          <td>Cancer biology & therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39ff774114afdb5fa389a578e894de7644886e8b" target='_blank'>
              Optimized model architectures for deep learning on genomic data
              </a>
            </td>
          <td>
            Hüseyin Anil Gündüz, René Mreches, Julia Moosbauer, Gary Robertson, Xiao-Yin To, E. Franzosa, Curtis Huttenhower, Mina Rezaei, A. Mchardy, Bernd Bischl, Philipp C. Münch, Martin Binder
          </td>
          <td>2024-04-30</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Time series data in real-world scenarios contain a substantial amount of nonlinear information, which significantly interferes with the training process of models, leading to decreased prediction performance. Therefore, during the time series forecasting process, extracting the local and global time series patterns and understanding the potential nonlinear features among different time observations are highly significant. To address this challenge, we introduce multi-resolution convolution and deformable convolution operations. By enlarging the receptive field using convolution kernels with different dilation factors to capture temporal correlation information at different resolutions, and adaptively adjusting the sampling positions through additional offset vectors, we enhance the network's ability to capture potential nonlinear features among time observations. Building upon this, we propose ACNet, an adaptive convolutional network designed to effectively model the local and global temporal dependencies and the nonlinear features between observations in multivariate time series. Specifically, by extracting and fusing time series features at different resolutions, we capture both local contextual information and global patterns in the time series. The designed nonlinear feature adaptive extraction module captures the nonlinear features among different time observations in the time series. We evaluated the performance of ACNet across twelve real-world datasets. The results indicate that ACNet consistently achieves state-of-the-art performance in both short-term and long-term forecasting tasks with favorable runtime efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f06b12265651df47706f7f621ed71191314915f" target='_blank'>
              Adaptive Convolutional Forecasting Network Based on Time Series Feature-Driven
              </a>
            </td>
          <td>
            Dandan Zhang, Zhiqiang Zhang, Nanguang Chen, Yun Wang
          </td>
          <td>2024-05-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Subcutaneous injections are an increasingly prevalent route of administration for delivering biological therapies including monoclonal antibodies (mAbs). Compared with intravenous delivery, subcutaneous injections reduce administration costs, shorten the administration time, and are strongly preferred from a patient experience point of view. An understanding of the absorption process of a mAb from the injection site to the systemic circulation is critical to the process of subcutaneous mAb formulation development. In this study, we built a model to predict the absorption rate constant (ka), which denotes how fast a mAb is absorbed from the site of administration. Once trained, our model (enabled by the XGBoost algorithm in machine learning) can predict the ka of a mAb following a subcutaneous injection using in silico molecular properties alone (generated from the primary sequence). Our model does not need clinically observed plasma concentration-time data; this is a novel capability not previously achieved in predictive pharmacokinetic models. The model also showed improved performance when benchmarked against a recently reported mechanistic model that relied on clinical data to predict subcutaneous absorption of mAbs. We further interpreted the model to understand which molecular properties affect the absorption rate and showed that our findings are consistent with previous studies evaluating subcutaneous absorption through direct experimentation. Taken altogether, this study reports the development, validation, benchmarking, and interpretation of a model that can predict the clinical ka of a mAb using its primary sequence as the only input.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b3adc9ed762fabcaf1f9d5c986090c3bbbc9e47" target='_blank'>
              Predicting the clinical subcutaneous absorption rate constant of monoclonal antibodies using only the primary sequence: a machine learning approach
              </a>
            </td>
          <td>
            Ronghua Bei, Justin Thomas, Shiven Kapur, Mahlet A Woldeyes, Adam Rauk, Jason Robarge, Jiangyan Feng, Kaoutar Abbou Oucherif
          </td>
          <td>2024-05-14</td>
          <td>mAbs</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="This study assesses the predictive performance of six machine learning models and a 1D Convolutional Neural Network (CNN) in forecasting tumor dynamics within three months following Gamma Knife radiosurgery (GKRS) in 77 brain metastasis (BM) patients. The analysis meticulously evaluates each model before and after hyperparameter tuning, utilizing accuracy, AUC, and other metrics derived from confusion matrices. The CNN model showcased notable performance with an accuracy of 98% and an AUC of 0.97, effectively complementing the broader model analysis. Initial findings highlighted that XGBoost significantly outperformed other models with an accuracy of 0.95 and an AUC of 0.95 before tuning. Post-tuning, the Support Vector Machine (SVM) demonstrated the most substantial improvement, achieving an accuracy of 0.98 and an AUC of 0.98. Conversely, XGBoost showed a decline in performance after tuning, indicating potential overfitting. The study also explores feature importance across models, noting that features like “control at one year”, “age of the patient”, and “beam-on time for volume V1 treated” were consistently influential across various models, albeit their impacts were interpreted differently depending on the model’s underlying mechanics. This comprehensive evaluation not only underscores the importance of model selection and hyperparameter tuning but also highlights the practical implications in medical diagnostic scenarios, where the accuracy of positive predictions can be crucial. Our research explores the effects of staged Gamma Knife radiosurgery (GKRS) on larger tumors, revealing no significant outcome differences across protocols. It uniquely considers the impact of beam-on time and fraction intervals on treatment efficacy. However, the investigation is limited by a small patient cohort and data from a single institution, suggesting the need for future multicenter research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1145edccd36b1fcabce5d3713aae781c5134f5c4" target='_blank'>
              Predicting Tumor Dynamics Post-Staged GKRS: Machine Learning Models in Brain Metastases Prognosis
              </a>
            </td>
          <td>
            A. Trofin, C. Buzea, Razvan-Cristian Buga, M. Agop, L. Ochiuz, D. Iancu, Lucian Eva
          </td>
          <td>2024-06-15</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Networked datasets are often enriched by different types of information about individual nodes or edges. However, most existing methods for analyzing such datasets struggle to handle the complexity of heterogeneous data, often requiring substantial model-specific analysis. In this paper, we develop a probabilistic generative model to perform inference in multilayer networks with arbitrary types of information. Our approach employs a Bayesian framework combined with the Laplace matching technique to ease interpretation of inferred parameters. Furthermore, the algorithmic implementation relies on automatic differentiation, avoiding the need for explicit derivations. This makes our model scalable and flexible to adapt to any combination of input data. We demonstrate the effectiveness of our method in detecting overlapping community structures and performing various prediction tasks on heterogeneous multilayer data, where nodes and edges have different types of attributes. Additionally, we showcase its ability to unveil a variety of patterns in a social support network among villagers in rural India by effectively utilizing all input information in a meaningful way.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60564e4fb5bc52e893efd75a3b624decc564a7a8" target='_blank'>
              Flexible inference in heterogeneous and attributed multilayer networks
              </a>
            </td>
          <td>
            Martina Contisciani, Marius Hobbhahn, Eleanor A. Power, Philipp Hennig, C. D. Bacco
          </td>
          <td>2024-05-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract In precision medicine, both predicting the disease susceptibility of an individual and forecasting its disease-free survival are areas of key research. Besides the classical epidemiological predictor variables, data from multiple (omic) platforms are increasingly available. To integrate this wealth of information, we propose new methodology to combine both cooperative learning, a recent approach to leverage the predictive power of several datasets, and polygenic hazard score models. Polygenic hazard score models provide a practitioner with a more differentiated view of the predicted disease-free survival than the one given by merely a point estimate, for instance computed with a polygenic risk score. Our aim is to leverage the advantages of cooperative learning for the computation of polygenic hazard score models via Cox’s proportional hazard model, thereby improving the prediction of the disease-free survival. In our experimental study, we apply our methodology to forecast the disease-free survival for Alzheimer’s disease (AD) using three layers of data. One layer contains epidemiological variables such as sex, APOE (apolipoprotein E, a genetic risk factor for AD) status and 10 leading principal components. Another layer contains selected genomic loci, and the last layer contains methylation data for selected CpG sites. We demonstrate that the survival curves computed via cooperative learning yield an AUC of around \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $0.7$\end{document}, above the state-of-the-art performance of its competitors. Importantly, the proposed methodology returns (1) a linear score that can be easily interpreted (in contrast to machine learning approaches), and (2) a weighting of the predictive power of the involved data layers, allowing for an assessment of the importance of each omic (or other) platform. Similarly to polygenic hazard score models, our methodology also allows one to compute individual survival curves for each patient.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9168bf7c51756564400a8c3a96e0936a62287e5" target='_blank'>
              Prediction of disease-free survival for precision medicine using cooperative learning on multi-omic data
              </a>
            </td>
          <td>
            Georg Hahn, D. Prokopenko, J. Hecker, S. M. Lutz, K. Mullin, Leinal Sejour, Winston Hide, Ioannis Vlachos, Stacia DeSantis, Rudolph E. Tanzi, Christoph Lange
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ebf5cc0cdb31adc80effde28168f8f01dc75c8" target='_blank'>
              Technologies for design-build-test-learn automation and computational modelling across the synthetic biology workflow: a review
              </a>
            </td>
          <td>
            R. Matzko, Savas Konur
          </td>
          <td>2024-05-03</td>
          <td>Netw. Model. Anal. Health Informatics Bioinform.</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Uncertainty in low dose ionizing radiation induced health risks stems from several factors. The complex biological pathways leading to diseases like cancer are not fully understood, making it difficult to distinguish the contribution of radiation, particularly at low doses which induce only small perturbations to background disease risks. Additionally, traditional dose response models, such as the Linear No Threshold formalism and competing threshold or hormesis models, impose rigid assumptions on dose response shapes, causing controversy and increasing model selection uncertainty. Furthermore, these modeling strategies operate on the level of correlations/associations, and are not designed to directly address the ultimate goal of radiation epidemiology assessing causal links between radiation and disease. A promising and rapidly developing approach for addressing some of these challenges is causal machine learning (CML), such as double/debiased machine learning (DML), which is designed to model causal effects in multi dimensional data sets. Our study employs DML to elucidate the causal impacts of radiation exposure on the incidence of leukemia, all solid tumors, and stomach tumors among Japanese atomic bomb survivors. Its goal was not to produce a definitive re analysis of these data sets, but to provide a useful example of implementing CML in radiation epidemiology, which can advance the field by supplementing traditional modeling approaches. The results revealed robust positive causal effects of radiation for all three tumor types, especially for leukemia and stomach tumors. The effect magnitudes, and uncertainties, were not dramatically different at low doses than at higher doses. The influences of age at exposure, attained age, sex and other covariates on the causal effects of radiation were assessed using Shapley Additive Explanations (SHAP) values. We believe that this analysis, based on a flexible machine learning framework with a causal inference motivation and without strict dose response assumptions, provides an important contribution to radiation epidemiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/792df0e8a3b11233cd61634953d8953b01530526" target='_blank'>
              Causal Machine Learning Analysis of Radiation-Induced Leukemia and Solid Tumor Incidence in Japanese Atomic Bomb Survivors
              </a>
            </td>
          <td>
            I. Shuryak, Zhenqiu Liu, Eric Wang, R. L. Ullrich, David J. Brenner
          </td>
          <td>2024-05-01</td>
          <td>None</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Large language models (LLMs) have demonstrated potential in the innovation of many disciplines. However, how they can best be developed for oncology remains underdeveloped. State-of-the-art OpenAI models were fine-tuned on a clinical dataset and clinical guidelines text corpus for two important cancer treatment factors, adjuvant radiation therapy and chemotherapy, using a novel Langchain prompt engineering pipeline. A high accuracy (0.85+) was achieved in the classification of adjuvant radiation therapy and chemotherapy for breast cancer patients. Furthermore, a confidence interval was formed from observational data on the quality of treatment from human oncologists to estimate the proportion of scenarios in which the model must outperform the original oncologist in its treatment prediction to be a better solution overall as 8.2% to 13.3%. Due to indeterminacy in the outcomes of cancer treatment decisions, future investigation, potentially a clinical trial, would be required to determine if this threshold was met by the models. Nevertheless, with 85% of U.S. cancer patients receiving treatment at local community facilities, these kinds of models could play an important part in expanding access to quality care with outcomes that lie, at minimum, close to a human oncologist.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcfef9abf40f3af1d41523a3a4bd2288447f71d1" target='_blank'>
              A Large Language Model Pipeline for Breast Cancer Oncology
              </a>
            </td>
          <td>
            Tristen Pool, Dennis Trujillo
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Behaviour change lies at the heart of many observable collective phenomena such as the transmission and control of infectious diseases, adoption of public health policies, and migration of animals to new habitats. Representing the process of individual behaviour change in computer simulations of these phenomena remains an open challenge. Often, computational models use phenomenological implementations with limited support from behavioural data. Without a strong connection to observable quantities, such models have limited utility for simulating observed and counterfactual scenarios of emergent phenomena because they cannot be validated or calibrated. Here, we present a simple stochastic individual-based model of reversal learning that captures fundamental properties of individual behaviour change, namely, the capacity to learn based on accumulated reward signals, and the transient persistence of learned behaviour after rewards are removed or altered. The model has only two parameters, and we use approximate Bayesian computation to demonstrate that they are fully identifiable from empirical reversal learning time series data. Finally, we demonstrate how the model can be extended to account for the increased complexity of behavioural dynamics over longer time scales involving fluctuating stimuli. This work is a step towards the development and evaluation of fully identifiable individual-level behaviour change models that can function as validated submodels for complex simulations of collective behaviour change.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f644e0376aeabf0dada3dac47c58b26bee7e1acc" target='_blank'>
              An agent-based model of behaviour change calibrated to reversal learning data
              </a>
            </td>
          <td>
            R. D. Reyes, Hugo Lyons Keenan, Cameron Zachreson
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The purpose of this article is to infer patient level outcomes from population level randomized control trials (RCTs). In this pursuit, we utilize the recently proposed synthetic nearest neighbors (SNN) estimator. At its core, SNN leverages information across patients to impute missing data associated with each patient of interest. We focus on two types of missing data: (i) unrecorded outcomes from discontinuing the assigned treatments and (ii) unobserved outcomes associated with unassigned treatments. Data imputation in the former powers and de-biases RCTs, while data imputation in the latter simulates"synthetic RCTs"to predict the outcomes for each patient under every treatment. The SNN estimator is interpretable, transparent, and causally justified under a broad class of missing data scenarios. Relative to several standard methods, we empirically find that SNN performs well for the above two applications using Phase 3 clinical trial data on patients with Alzheimer's Disease. Our findings directly suggest that SNN can tackle a current pain point within the clinical trial workflow on patient dropouts and serve as a new tool towards the development of precision medicine. Building on our insights, we discuss how SNN can further generalize to real-world applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcdeec4a0325846707e0b5cd5fb19c9ec671951f" target='_blank'>
              Personalized Predictions from Population Level Experiments: A Study on Alzheimer's Disease
              </a>
            </td>
          <td>
            Dennis Shen, Anish Agarwal, Vishal Misra, Bjoern Schelter, Devavrat Shah, Helen Shiells, C. Wischik
          </td>
          <td>2024-05-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Abstract Motivation Systems biology aims to better understand living systems through mathematical modelling of experimental and clinical data. A pervasive challenge in quantitative dynamical modelling is the integration of time series measurements, which often have high variability and low sampling resolution. Approaches are required to utilize such information while consistently handling uncertainties. Results We present BayModTS (Bayesian modelling of time series data), a new FAIR (findable, accessible, interoperable, and reusable) workflow for processing and analysing sparse and highly variable time series data. BayModTS consistently transfers uncertainties from data to model predictions, including process knowledge via parameterized models. Further, credible differences in the dynamics of different conditions can be identified by filtering noise. To demonstrate the power and versatility of BayModTS, we applied it to three hepatic datasets gathered from three different species and with different measurement techniques: (i) blood perfusion measurements by magnetic resonance imaging in rat livers after portal vein ligation, (ii) pharmacokinetic time series of different drugs in normal and steatotic mice, and (iii) CT-based volumetric assessment of human liver remnants after clinical liver resection. Availability and implementation The BayModTS codebase is available on GitHub at https://github.com/Systems-Theory-in-Systems-Biology/BayModTS. The repository contains a Python script for the executable BayModTS workflow and a widely applicable SBML (systems biology markup language) model for retarded transient functions. In addition, all examples from the paper are included in the repository. Data and code of the application examples are stored on DaRUS: https://doi.org/10.18419/darus-3876. The raw MRI ROI voxel data were uploaded to DaRUS: https://doi.org/10.18419/darus-3878. The steatosis metabolite data are published on FairdomHub: 10.15490/fairdomhub.1.study.1070.1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483f129e077df807f3f25aef2dde774505aa52b2" target='_blank'>
              Bayesian modelling of time series data (BayModTS)—a FAIR workflow to process sparse and highly variable data
              </a>
            </td>
          <td>
            Sebastian Höpfl, Mohamed Albadry, U. Dahmen, Karl-Heinz Herrmann, E. Kindler, Matthias König, Jürgen R. Reichenbach, Hans-Michael Tautenhahn, Weiwei Wei, Wan-Ting Zhao, Nicole Erika Radde
          </td>
          <td>2024-05-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cancer is a genetic disease where gene mutations are pivotal in disease initiation and pathophysiology. The gene expression profile follows a specific pattern exclusive to each cancer which can be utilized for early and accurate diagnosis. Microarray techniques have emerged as powerful tools capable of simultaneously capturing the expression profiles of thousands of genes. However, because of the high dimensionality of the produced transcriptome data, analysis of the resulting datasets is challenging. Recent advancements in Artificial Intelligence (AI) techniques like Machine Learning (ML) and Deep Learning can be instrumental in efficiently processing these high-dimensional datasets. LASSO-regression is a ML technique that can help to rank the features which could help in feature selection leading to dimensionality reduction. Deep Learning is one of the most sophisticated ML techniques that can process high-dimensional data owing to the presence of more number of hidden layers in its neural network. We designed a Deep Neural Network (DNN) classifier model fused with a LASSO-based significant feature extractor for classifying the gene expression dataset containing a total of 51 samples of which 24 samples are of lung cancer patients and the remaining 27 samples are of normal individuals. A LASSO regression model was implemented to identify the genes that played a significant role in the classification. These significant gene expressions were then fed into a convergent Deep Neural Architecture. The classifier was trained with 70% data and the rest 30% was used for validation. The proposed classifier proved to provide better classification as compared to LASSO regression and DNN used individually. The two classes were classified with an average accuracy of 96.25%, average precision of 99.67%, average specificity of 99.45% and average sensitivity of 91.73% measured over thirty independent assessments. In some cases, the model was able to obtain a classification accuracy of 100%. This could open the path to early and better diagnosis of cancers from transcriptome data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a04d7a7531045d5a23d94088bf4e437783d0a21e" target='_blank'>
              Feature Selection Using Lasso Regression Enhances Deep Learning Model Performance For Diagnosis Of Lung Cancer from Transcriptomic Data
              </a>
            </td>
          <td>
            Souvik Guha
          </td>
          <td>2024-05-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7a0a8265ea7e318e65ccdf6ef53d19670ab7b3" target='_blank'>
              A novel non-negative Bayesian stacking modeling method for Cancer survival prediction using high-dimensional omics data
              </a>
            </td>
          <td>
            Junjie Shen, Shuo Wang, Hao Sun, Jie Huang, Lu Bai, Xichao Wang, Yongfei Dong, Zaixiang Tang
          </td>
          <td>2024-05-03</td>
          <td>BMC Medical Research Methodology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Neural networks, whice have had a profound effect on how researchers study complex phenomena, do so through a complex, nonlinear mathematical structure which can be difficult for human researchers to interpret. This obstacle can be especially salient when researchers want to better understand the emergence of particular model behaviors such as bias, overfitting, overparametrization, and more. In Neuroimaging, the understanding of how such phenomena emerge is fundamental to preventing and informing users of the potential risks involved in practice. In this work, we present a novel introspection framework for Deep Learning on Neuroimaging data, which exploits the natural structure of gradient computations via the singular value decomposition of gradient components during reverse-mode auto-differentiation. Unlike post-hoc introspection techniques, which require fully-trained models for evaluation, our method allows for the study of training dynamics on the fly, and even more interestingly, allow for the decomposition of gradients based on which samples belong to particular groups of interest. We demonstrate how the gradient spectra for several common deep learning models differ between schizophrenia and control participants from the COBRE study, and illustrate how these trajectories may reveal specific training dynamics helpful for further analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3b7b7d7596c45733b4e736f2a2d839214005fc" target='_blank'>
              Spectral Introspection Identifies Group Training Dynamics in Deep Neural Networks for Neuroimaging
              </a>
            </td>
          <td>
            Bradley T. Baker, Vince D. Calhoun, S. Plis
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Causal models provide rich descriptions of complex systems as sets of mechanisms by which each variable is influenced by its direct causes. They support reasoning about manipulating parts of the system and thus hold promise for addressing some of the open challenges of artificial intelligence (AI), such as planning, transferring knowledge in changing environments, or robustness to distribution shifts. However, a key obstacle to more widespread use of causal models in AI is the requirement that the relevant variables be specified a priori, which is typically not the case for the high-dimensional, unstructured data processed by modern AI systems. At the same time, machine learning (ML) has proven quite successful at automatically extracting useful and compact representations of such complex data. Causal representation learning (CRL) aims to combine the core strengths of ML and causality by learning representations in the form of latent variables endowed with causal model semantics. In this thesis, we study and present new results for different CRL settings. A central theme is the question of identifiability: Given infinite data, when are representations satisfying the same learning objective guaranteed to be equivalent? This is an important prerequisite for CRL, as it formally characterises if and when a learning task is, at least in principle, feasible. Since learning causal models, even without a representation learning component, is notoriously difficult, we require additional assumptions on the model class or rich data beyond the classical i.i.d. setting. By partially characterising identifiability for different settings, this thesis investigates what is possible for CRL without direct supervision, and thus contributes to its theoretical foundations. Ideally, the developed insights can help inform data collection practices or inspire the design of new practical estimation methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e04aac37f37d12a0fb6d08faf3b9b67cbb5ea13" target='_blank'>
              Identifiable Causal Representation Learning: Unsupervised, Multi-View, and Multi-Environment
              </a>
            </td>
          <td>
            Julius von Kugelgen
          </td>
          <td>2024-06-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor heterogeneity is a primary factor that contributes to treatment failure. Predictive tools, capable of classifying cancer cells based on their functions, may substantially enhance therapy and extend patient life span. The connection between cell biomechanics and cancer cell functions is used here to classify cells through mechanical measurements, via particle uptake. Machine learning (ML) was used to classify cells based on single-cell patterns of uptake of particles with diverse sizes. Three pairs of human cancer cell subpopulations, varied in their level of drug resistance or malignancy, were studied. Cells were allowed to interact with fluorescently labeled polystyrene particles ranging in size from 0.04 to 3.36 μm and analyzed for their uptake patterns using flow cytometry. ML algorithms accurately classified cancer cell subtypes with accuracy rates exceeding 95%. The uptake data were especially advantageous for morphologically similar cell subpopulations. Moreover, the uptake data were found to serve as a form of "normalization" that could reduce variation in repeated experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f66cc5838fae9c10d30fe2b5543c3e9eb1e0ab75" target='_blank'>
              Particle uptake in cancer cells can predict malignancy and drug resistance using machine learning.
              </a>
            </td>
          <td>
            Yoel Goldstein, Ora T. Cohen, Ori Wald, Danny Bavli, Tommy Kaplan, Ofra Benny
          </td>
          <td>2024-05-31</td>
          <td>Science advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The estimation of directed couplings between the nodes of a network from indirect measurements is a central methodological challenge in scientific fields such as neuroscience, systems biology and economics. Unfortunately, the problem is generally ill-posed due to the possible presence of unknown delays in the measurements. In this paper, we offer a solution of this problem by using a variational Bayes framework, where the uncertainty over the delays is marginalized in order to obtain conservative coupling estimates. To overcome the well-known overconfidence of classical variational methods, we use a hybrid-VI scheme where the (possibly flat or multimodal) posterior over the measurement parameters is estimated using a forward KL loss while the (nearly convex) conditional posterior over the couplings is estimated using the highly scalable gradient-based VI. In our ground-truth experiments, we show that the network provides reliable and conservative estimates of the couplings, greatly outperforming similar methods such as regression DCM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a02bb50bd2145ddb88212879e50602c28462b00e" target='_blank'>
              Robust and highly scalable estimation of directional couplings from time-shifted signals
              </a>
            </td>
          <td>
            Luca Ambrogioni, Louis Rouillard, Demian Wassermann
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Stepwise covariate modeling (SCM) has a high computational burden and can select the wrong covariates. Machine learning (ML) has been proposed as a screening tool to improve the efficiency of covariate selection, but little is known about how to apply ML on actual clinical data. First, we simulated datasets based on clinical data to compare the performance of various ML and traditional pharmacometrics (PMX) techniques with and without accounting for highly-correlated covariates. This simulation step identified the ML algorithm and the number of top covariates to select when using the actual clinical data. A previously developed desipramine population-pharmacokinetic model was used to simulate virtual subjects. Fifteen covariates were considered with four having an effect included. Based on the F1 score (an accuracy measure), ridge regression was the most accurate ML technique on 200 simulated datasets (F1 score = 0.475 ± 0.231), a performance which almost doubled when highly-correlated covariates were accounted for (F1 score = 0.860 ± 0.158). These performances were better than forwards selection with SCM (F1 score = 0.251 ± 0.274 and 0.499 ± 0.381 without/with correlations respectively). In terms of computational cost, ridge regression (0.42 ± 0.07 seconds/simulated dataset, 1 thread) was ~20,000 times faster than SCM (2.30 ± 2.29 hours, 15 threads). On the clinical dataset, prescreening with the selected ML algorithm reduced SCM runtime by 42.86% (from 1.75 to 1.00 days) and produced the same final model as SCM only. In conclusion, we have demonstrated that accounting for highly-correlated covariates improves ML prescreening accuracy. The choice of ML method and the proportion of important covariates (unknown a priori) can be guided by simulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6e659fbccaa48361b3511d11da2b5a70822817" target='_blank'>
              Machine-Learning Assisted Screening of Correlated Covariates: Application to Clinical Data of Desipramine.
              </a>
            </td>
          <td>
            Innocent Gerald Asiimwe, Bonginkosi S'fiso Ndzamba, Samer Mouksassi, G. Pillai, Aurelie Lombard, Jennifer Lang
          </td>
          <td>2024-05-30</td>
          <td>The AAPS journal</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Decision-making in healthcare relies on the ability to understand patients' past and current health state to predict, and ultimately change, their future course. Artificial intelligence (AI) methods promise to aid this task by learning patterns of disease progression from large corpora of health records to predict detailed outcomes for an individual. However, the potential of AI has not yet been fully investigated at scale yet. Here we modify the GPT (generative pretrained transformer) architecture to model the temporal progression and competing nature of human diseases in a population scale cohort. We train this model, termed Delphi-2M, on data from 0.4 million participants of the UK Biobank and validate it using external data from 1.9 million Danish individuals with no change in parameters. Delphi-2M predicts the rates of more than 1,000 different ICD-10 coded diseases and death, conditional on each individual's past disease history, age, sex and baseline lifestyle information, and with accuracy comparable to existing single-disease models. Delphi-2M's generative nature also enables sampling future health trajectories at any point within an individual's life course with outcomes across the entire disease spectrum. Sampled health trajectories provide meaningful estimates of future disease burden for up to 20 years and enable training AI models which have never seen actual data. Explainable AI methods provide insights into Delphi-2M's predictions, revealing temporal clusters of co-morbidities within and across different disease chapters and their time-dependent consequences on the future health course. These analyses, however, also reveal that biases underlying the available training data, which in the case of the UK Biobank stem from distinct healthcare sources, are learned and highlighted. In summary GPT-based models appear well suited for predictive and generative health related tasks, are applicable to population scale health data sets and provide insights into the temporal dependencies of past events that shape future health impacting our ability to obtain an instantaneous view of personalised health state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d480376f7b315aba50e4b214b4bd545b6d63fefb" target='_blank'>
              Learning the natural history of human disease with generative transformers
              </a>
            </td>
          <td>
            A. Shmatko, A. W. Jung, K. Gaurav, S. Brunak, L. Mortensen, E. Birney, T. Fitzgerald, M. Gerstung
          </td>
          <td>2024-06-07</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Modern recordings of neural activity provide diverse observations of neurons across brain areas, behavioral conditions, and subjects -- thus presenting an exciting opportunity to reveal the fundamentals of brain-wide dynamics underlying cognitive function. Current methods, however, often fail to fully harness the richness of such data as they either provide an uninterpretable representation (e.g., via"black box"deep networks) or over-simplify the model (e.g., assume stationary dynamics or analyze each session independently). Here, instead of regarding asynchronous recordings that lack alignment in neural identity or brain areas as a limitation, we exploit these diverse views of the same brain system to learn a unified model of brain dynamics. We assume that brain observations stem from the joint activity of a set of functional neural ensembles (groups of co-active neurons) that are similar in functionality across recordings, and propose to discover the ensemble and their non-stationary dynamical interactions in a new model we term CrEIMBO (Cross-Ensemble Interactions in Multi-view Brain Observations). CrEIMBO identifies the composition of the per-session neural ensembles through graph-driven dictionary learning and models the ensemble dynamics as a latent sparse time-varying decomposition of global sub-circuits, thereby capturing non-stationary dynamics. CrEIMBO identifies multiple co-active sub-circuits while maintaining representation interpretability due to sharing sub-circuits across sessions. CrEIMBO distinguishes session-specific from global (session-invariant) computations by exploring when distinct sub-circuits are active. We demonstrate CrEIMBO's ability to recover ground truth components in synthetic data and uncover meaningful brain dynamics, capturing cross-subject and inter- and intra-area variability, in high-density electrode recordings of humans performing a memory task.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d7b62d73540f0ee3170538475eecbf533fc4b56" target='_blank'>
              CrEIMBO: Cross Ensemble Interactions in Multi-view Brain Observations
              </a>
            </td>
          <td>
            Noga Mudrik, Ryan Ly, Oliver Ruebel, Adam S. Charles
          </td>
          <td>2024-05-27</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Motivation: Advances in sequencing technologies have enabled the early detection of genetic diseases and the development of personalized medicine. However, the variance explained by genetic variations is typically small compared to the heritability estimates. Consequently, there is a pressing need to develop enhanced polygenic risk score (PRS) prediction models. We seek an approach that transcends the limitations of the routinely used additive model for PRS. Results: Here we present DROP-DEEP, a novel method for calculating PRS that enhances the explanation of the heritability variance of complex traits by incorporating high-dimensional genetic interactions. The first stage of DROP-DEEP employs an unsupervised approach to reduce dimensionality, while the second stage involves training a prediction model using a supervised machine-learning algorithm. Notably, the first stage of training is phenotype-agnostic. Thus, while it is computationally intensive, it is performed only once. Its output can serve as input for predicting any chosen trait or disease. We evaluated the efficacy of the DROP-DEEP dimensionality reduction models using principal component analysis (PCA) and deep neural networks (DNN). All models were trained using the UK Biobank (UKB) dataset with over 340,000 subjects and a set of approximately 460,000 single nucleotide variants (SNVs) across the genome. The results of DROP-DEEP, which was established for patients diagnosed with hypertension, outperformed other approaches. We extended the analysis to include an additional five binary and continuous phenotypes, each repeated five times for reproducibility assessment. For each phenotype, DROP-DEEP results were compared to commonly used PRS methodologies, and the performance of all models was discussed. Conclusion: Our approach overcomes the need for variable selection while maintaining computational feasibility. We conclude that the DROP-DEEP approach exhibits significant advantages compared to commonly used PRS methods and can be used efficiently for hundreds of genetic traits. Availability and Implementation: All the codes and the trained dimensionality reduction models are available at: https://github.com/HadasaK1/DROP-DEEP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/922113f50d291038110bd62e393d1a106cb49f30" target='_blank'>
              DROP-DEEP: Dimensionality Reduction for Polygenic Risk Score Using Deep Learning Approach
              </a>
            </td>
          <td>
            Hadasa Kaufman, Yarden Hochenberg, M. Linial, Nadav Rappoport, M. L. N. Rappoport
          </td>
          <td>2024-05-02</td>
          <td>None</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Interactions between drugs can lead to adverse side effects for patients taking combination therapies to treat complex diseases such as cancer. Knowledge of drug-action towards a receptor would allow these drug-drug interactions to be predicted, and in this study, we trained a total of 5 different machine learning models to classify whether a given drug was an agonist (activator), antagonist (blocker), or a decoy (non-binder) to each of the androgen, estrogen, glucocorticoid, and progesterone receptors. The classification performance and efficiency, measured in training time, of the decision tree, naïve Bayes, neural network, random forest, and support vector machine models for each receptor were then compared. The results showed that the decision tree and naïve Bayes models were best suited for drug-action prediction across all receptors while only requiring minutes of training time at most. Future work will focus on increasing the prediction accuracy of antagonist drugs, integrating experimental data during training, and using other targets outside of nuclear type I receptors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3f1fd185e5d36f4f38477342cd4d936ce7439" target='_blank'>
              Application of Supervised Machine Learning Models for Drug-Action Prediction Towards Nuclear Type I Receptors
              </a>
            </td>
          <td>
            Rajeev Jaundoo, J. Tuszynski, T. Craddock
          </td>
          <td>2024-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Drug-induced liver injury (DILI) poses a significant challenge for the pharmaceutical industry and regulatory bodies. Despite extensive toxicological research aimed at mitigating DILI risk, the effectiveness of these techniques in predicting DILI in humans remains limited. Consequently, researchers have explored novel approaches and procedures to enhance the accuracy of DILI risk prediction for drug candidates under development. In this study, we leveraged a large human dataset to develop machine learning models for assessing DILI risk. The performance of these prediction models was rigorously evaluated using a 10-fold cross-validation approach and an external test set. Notably, the random forest (RF) and multilayer perceptron (MLP) models emerged as the most effective in predicting DILI. During cross-validation, RF achieved an average prediction accuracy of 0.631, while MLP achieved the highest Matthews Correlation Coefficient (MCC) of 0.245. To validate the models externally, we applied them to a set of drug candidates that had failed in clinical development due to hepatotoxicity. Both RF and MLP accurately predicted the toxic drug candidates in this external validation. Our findings suggest that in silico machine learning approaches hold promise for identifying DILI liabilities associated with drug candidates during development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d63ff6235a3810b439a842acc3a51e5d9b36667d" target='_blank'>
              Machine Learning to Predict Drug-Induced Liver Injury and Its Validation on Failed Drug Candidates in Development
              </a>
            </td>
          <td>
            Fahad Mostafa, Victoria Howle, Minjun Chen
          </td>
          <td>2024-05-24</td>
          <td>Toxics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Real-world networks are sparse. As we show in this article, even when a large number of interactions is observed most node pairs remain disconnected. We demonstrate that classical multi-edge network models, such as the $G(N,p)$, configuration models, and stochastic block models, fail to accurately capture this phenomenon. To mitigate this issue, zero-inflation must be integrated into these traditional models. Through zero-inflation, we incorporate a mechanism that accounts for the excess number of zeroes (disconnected pairs) observed in empirical data. By performing an analysis on all the datasets from the Sociopatterns repository, we illustrate how zero-inflated models more accurately reflect the sparsity and heavy-tailed edge count distributions observed in empirical data. Our findings underscore that failing to account for these ubiquitous properties in real-world networks inadvertently leads to biased models which do not accurately represent complex systems and their dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f81d421fba2326c997d828fe267b1ec0356ef368" target='_blank'>
              Empirical Networks are Sparse: Enhancing Multi-Edge Models with Zero-Inflation
              </a>
            </td>
          <td>
            Giona Casiraghi, George Andres
          </td>
          <td>2024-06-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In the domain of cyber-physical systems, there is an increasing relevance of data-driven approaches for the learning of hybrid system dynamics. In particular, accurate models have been successfully abstracted from continuous (real-valued) traces and applied for various goals. However, industrial applications involving online modeling or rapid prototyping have two additional requirements: 1) runtime efficiency and 2) the interpretability of the approach and results. This work adopts a common break down of this learning problem into four steps: 1) trace segmentation, 2) segment clustering, 3) characterization of the dynamics for each cluster (mode) and 4) learning of the overall model of mode transitions. Correspondingly, the bottlenecks in the state-of-the-art approaches are identified and discussed. Then, in a heuristic manner, interpretable and time-efficient algorithms for each of the steps are proposed giving a novel approach named FaMoS. The accuracy and runtime efficiency of the approach are evaluated for several system examples. FaMoS shows very short learning time, while the model’s predictions of system dynamics are close to the ground truth behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3187616ebf58e393e91671d47472a1fac56b7b62" target='_blank'>
              FaMoS– Fast Model Learning for Hybrid Cyber-Physical Systems using Decision Trees
              </a>
            </td>
          <td>
            Swantje Plambeck, Aaron Bracht, Nemanja Hranisavljevic, Görschwin Fey
          </td>
          <td>2024-05-14</td>
          <td>Proceedings of the 27th ACM International Conference on Hybrid Systems: Computation and Control</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="It is now possible to conduct large scale perturbation screens with complex readout modalities, such as different molecular profiles or high content cell images. While these open the way for systematic dissection of causal cell circuits, integrated such data across screens to maximize our ability to predict circuits poses substantial computational challenges, which have not been addressed. Here, we extend two Gromov-Wasserstein Optimal Transport methods to incorporate the perturbation label for cross-modality alignment. The obtained alignment is then employed to train a predictive model that estimates cellular responses to perturbations observed with only one measurement modality. We validate our method for the tasks of cross-modality alignment and cross-modality prediction in a recent multi-modal single-cell perturbation dataset. Our approach opens the way to unified causal models of cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7774e110cad7701d2a98f2cd1f4dd9bc6b0fa0c3" target='_blank'>
              Cross-modality Matching and Prediction of Perturbation Responses with Labeled Gromov-Wasserstein Optimal Transport
              </a>
            </td>
          <td>
            Jayoung Ryu, Romain Lopez, Charlotte Bunne, Aviv Regev
          </td>
          <td>2024-05-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="A plant growth simulation can be characterized as a reconstructed visual representation of a plant or plant system. The phenotypic characteristics and plant structures are controlled by the scene environment and other contextual attributes. Considering the temporal dependencies and compounding effects of various factors on growth trajectories, we formulate a probabilistic approach to the simulation task by solving a frame synthesis and pattern recognition problem. We introduce a sequence-informed plant growth simulation framework (SI-PGS) that employs a conditional generative model to implicitly learn a distribution of possible plant representations within a dynamic scene from a fusion of low dimensional temporal sensor and context data. Methods such as controlled latent sampling and recurrent output connections are used to improve coherence in the plant structures between frames of predictions. In this work, we demonstrate that SI-PGS is able to capture temporal dependencies and continuously generate realistic frames of plant growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b66ec18bfe7664cef77de1499f9c53579af68c" target='_blank'>
              Generative Plant Growth Simulation from Sequence-Informed Environmental Conditions
              </a>
            </td>
          <td>
            Mohamed Debbagh, Yixue Liu, Zhouzhou Zheng, Xintong Jiang, Shangpeng Sun, M. Lefsrud
          </td>
          <td>2024-05-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The disease known as breast cancer continues to be one of the most common and potentially fatal diseases that affect women all over the world. In order to provide effective therapy and more favorable outcomes for patients, early detection and correct diagnosis are absolutely necessary. This research explores the integration of operational research (OR) methodologies and advanced statistical methods within machine learning frameworks to enhance the prediction accuracy of breast cancer outcomes. By harnessing the power of nonlinear optimization and statistical techniques, including regression analysis and probability distribution models, we aim to refine the predictive capabilities of existing algorithms. The research employs a comprehensive dataset derived from clinical trials and patient records, analyzed through a series of machine learning models that incorporate elements of combinatorial optimization, decision analysis, and stochastic modeling. Key performance metrics, such as accuracy, sensitivity, and specificity, are evaluated against standard benchmarks to determine the efficacy of the integrated approaches. Preliminary results indicate that incorporating OR and statistical methods significantly improves model robustness and predictive accuracy. The study not only demonstrates the potential of applied mathematics in medical diagnostics but also provides a framework for future research in enhancing machine learning models for health outcomes prediction through mathematical innovations. This investigation contributes to the field of mathematical oncology by demonstrating how applied nonlinear analysis can bridge the gap between theoretical mathematical approaches and practical clinical applications, offering new pathways for early and more accurate detection of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91e2ad7ffea30ec7bd6b8cdf76f263552d6be77" target='_blank'>
              Integration of Complex Mathematical Approaches and Statistical Methodologies in Machine Learning for Design of Robust Prediction Models
              </a>
            </td>
          <td>
            Sarita Silaich, Dr Rajesh yadav
          </td>
          <td>2024-05-17</td>
          <td>Communications on Applied Nonlinear Analysis</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Predicting drug toxicity is a critical aspect of ensuring patient safety during the drug design process. Although conventional machine learning techniques have shown some success in this field, the scarcity of annotated toxicity data poses a significant challenge in enhancing models’ performance. In this study, we explore the potential of leveraging large unlabeled datasets using semi-supervised learning to improve predictive performance for cardiotoxicity across three targets: the voltage-gated potassium channel (hERG), the voltage-gated calcium channel (Cav1.2), and the voltage-gated sodium channel (Nav1.5). We extensively mined the ChEMBL database, comprising approximately 2 million small molecules, then employed semi-supervised learning to construct robust classification models for this purpose. We achieved a performance boost on highly diverse (i.e. structurally dissimilar) test datasets across all three targets. Using our built models, we screened the whole ChEMBL database and a large set of FDA-approved drugs, identifying several compounds with potential cardiac channel activity. To ensure broad accessibility and usability for both technical and non-technical users, we developed a cross-platform graphical user interface that allows users to make predictions and gain insights into the cardiotoxicity of drugs and other small molecules. The software is made available as open source under the permissive MIT license at https://github.com/issararab/CToxPred2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f0e09b146e54215719071ab7b9f648e458c75b" target='_blank'>
              Semi-Supervised Learning to Boost Cardiotoxicity Prediction by Mining a Large Unlabeled Small Molecule Dataset
              </a>
            </td>
          <td>
            Issar Arab, Kris Laukens, Wout Bittremieux
          </td>
          <td>2024-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Widespread deployment of societal-scale machine learning systems necessitates a thorough understanding of the resulting long-term effects these systems have on their environment, including loss of trustworthiness, bias amplification, and violation of AI safety requirements. We introduce a repeated learning process to jointly describe several phenomena attributed to unintended hidden feedback loops, such as error amplification, induced concept drift, echo chambers and others. The process comprises the entire cycle of obtaining the data, training the predictive model, and delivering predictions to end-users within a single mathematical model. A distinctive feature of such repeated learning setting is that the state of the environment becomes causally dependent on the learner itself over time, thus violating the usual assumptions about the data distribution. We present a novel dynamical systems model of the repeated learning process and prove the limiting set of probability distributions for positive and negative feedback loop modes of the system operation. We conduct a series of computational experiments using an exemplary supervised learning problem on two synthetic data sets. The results of the experiments correspond to the theoretical predictions derived from the dynamical model. Our results demonstrate the feasibility of the proposed approach for studying the repeated learning processes in machine learning systems and open a range of opportunities for further research in the area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbdf526001c151c67507de963a0c683064b1f630" target='_blank'>
              A Mathematical Model of the Hidden Feedback Loop Effect in Machine Learning Systems
              </a>
            </td>
          <td>
            Andrey Veprikov, Alexander Afanasiev, Anton Khritankov
          </td>
          <td>2024-05-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The field of machine learning has been evolving and applied in medical applications. We utilised a public dataset, MIMIC-III, to develop compact models that can accurately predict the outcome of mechanically ventilated patients in the first 24 h of first-time hospital admission.67 predictive features, grouped into 6 categories, were selected for the classification and prediction task. 4 tree-based algorithms (Decision Tree, Bagging, eXtreme Gradient Boosting and Random Forest), and 5 non-tree-based algorithms (Logistic Regression, K-Nearest Neighbour, Linear Discriminant Analysis, Support Vector Machine and Naïve Bayes), were employed to predict the outcome of 18,883 mechanically ventilated patients. 5 scenarios were crafted to mirror the target population as per existing literature. S1.1 reflected an imbalanced situation, with significantly fewer mortality cases than survival ones, and both the training and test sets played similar target class distributions. S1.2 and S2.2 featured balanced classes; however, instances from the majority class were removed from the test set and/or the training set. S1.3 and S 2.3 generated additional instances of the minority class via the Synthetic Minority Over-sampling Technique. Standard evaluation metrics were used to determine the best-performing models for each scenario. With the best performers, Autofeat, an automated feature engineering library, was used to eliminate less important features per scenario.Tree-based models generally outperformed the non-tree-based ones. Moreover, XGB consistently yielded the highest AUC score (between 0.91 and 0.97), while exhibiting relatively high Sensitivity (between 0.58 and 0.88) on 4 scenarios (1.2, 2.2, 1.3, and 2.3). After reducing a significant number of predictors, the selected calibrated ML models were still able to achieve similar AUC and MCC scores across those scenarios. The calibration curves of the XGB and BG models, both prior to and post dimension reduction in Scenario 2.2, showed better alignment to the perfect calibration line than curves produced from other algorithms.This study demonstrated that dimension-reduced models can perform well and are able to retain the important features for the classification tasks. Deploying a compact machine learning model into production helps reduce costs in terms of computational resources and monitoring changes in input data over time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/287a2fc8b7b19b61518b06105a0998ead22cc3db" target='_blank'>
              Compact machine learning model for the accurate prediction of first 24-hour survival of mechanically ventilated patients
              </a>
            </td>
          <td>
            Quynh T. Nguyen, Mai P. Tran, Vishnu Prabhakaran, Andrew Liu, Ghi H Nguyen
          </td>
          <td>2024-06-20</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The temporal resolution of environmental data sets plays a major role in the granularity of the information that can be derived from the data. In most cases, it is required that different data sets have a common temporal resolution to enable their consistent evaluations and applications in making informed decisions. This study leverages deep learning with long short-term memory (LSTM) neural networks and model inference to enhance the temporal resolution of climate datasets, specifically temperature, and precipitation, from daily to sub-daily scales. We trained our model to learn the relationship between daily and sub-daily data, subsequently applying this knowledge to increase the resolution of a separate dataset with a coarser (daily) temporal resolution. Our findings reveal a high degree of accuracy for temperature predictions, evidenced by a correlation of 0.99 and a mean absolute error of 0.21 °C, between the actual and predicted sub-daily values. In contrast, the approach was less effective for precipitation, achieving an explained variance of only 37%, compared to 98% for temperature. Further, besides the sub-daily interpolation of the climate data sets, we adapted our approach to increase the resolution of the Normalized difference vegetation index of Landsat (from 16 d to 5 d interval) using the LSTM model pre-trained from the Sentinel 2 Normalized difference vegetation index—that exists at a relatively higher temporal resolution. The explained variance between the predicted Landsat and Sentinel 2 data is 70% with a mean absolute error of 0.03. These results suggest that our method is particularly suitable for environmental datasets with less pronounced short-term variability, offering a promising tool for improving the resolution and utility of the data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b365a24bc7a55c6465814e7fae80c9eb78ae8b6" target='_blank'>
              Interpolation of environmental data using deep learning and model inference
              </a>
            </td>
          <td>
            C. Ibebuchi, Itohan-Osa Abu
          </td>
          <td>2024-05-14</td>
          <td>Machine Learning: Science and Technology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Interpretable mathematical expressions defining discrete-time dynamical systems (iterated maps) can model many phenomena of scientific interest, enabling a deeper understanding of system behaviors. Since formulating governing expressions from first principles can be difficult, it is of particular interest to identify expressions for iterated maps given only their data streams. In this work, we consider a modified Symbolic Artificial Neural Network-Trained Expressions (SymANNTEx) architecture for this task, an architecture more expressive than others in the literature. We make a modification to the model pipeline to optimize the regression, then characterize the behavior of the adjusted model in identifying several classical chaotic maps. With the goal of parsimony, sparsity-inducing weight regularization and information theory-informed simplification are implemented. We show that our modified SymANNTEx model properly identifies single-state maps and achieves moderate success in approximating a dual-state attractor. These performances offer significant promise for data-driven scientific discovery and interpretation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3502a1e8d5f9351f334ca3982d09f0f0f0433666" target='_blank'>
              Expressive Symbolic Regression for Interpretable Models of Discrete-Time Dynamical Systems
              </a>
            </td>
          <td>
            Adarsh Iyer, N. Boddupalli, Jeff Moehlis
          </td>
          <td>2024-06-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Universal differential equations (UDEs) leverage the respective advantages of mechanistic models and artificial neural networks and combine them into one dynamic model. However, these hybrid models can suffer from unrealistic solutions, such as negative values for biochemical quantities. We present non-negative UDE (nUDEs), a constrained UDE variant that guarantees non-negative values. Furthermore, we explore regularisation techniques to improve generalisation and interpretability of UDEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44abab70f51501595f3b1ac64da11aea862e8c1e" target='_blank'>
              Non-Negative Universal Differential Equations With Applications in Systems Biology
              </a>
            </td>
          <td>
            Maren Philipps, Antonia Korner, Jakob Vanhoefer, Dilan Pathirana, Jan Hasenauer
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c1cd074b744953a14cf72dff23f50b500e57211" target='_blank'>
              APF2: an improved ensemble method for pharmacogenomic variant effect prediction
              </a>
            </td>
          <td>
            Yitian Zhou, Sebastian Pirmann, V. Lauschke
          </td>
          <td>2024-05-27</td>
          <td>The Pharmacogenomics Journal</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983a84ba84d4e4978cb8f3fe20f03553417b6b7e" target='_blank'>
              DeepARV: ensemble deep learning to predict drug-drug interaction of clinical relevance with antiretroviral therapy
              </a>
            </td>
          <td>
            Thao Pham, Mohamed Ghafoor, Sandra Grañana-Castillo, C. Marzolini, Sara Gibbons, Saye Khoo, J. Chiong, Dennis Wang, Marco Siccardi
          </td>
          <td>2024-05-06</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Symbolic Regression (SR) is a machine learning paradigm that focuses on the automatic discovery of mathematical equations that best describe the relationship between input and output features in experimental datasets. Such an approach leads to interpretable models that make it easy to incorporate the knowledge available in the system. This article addresses the application of SR as an interpretable modelling approach for industrial applications involving nonlinear dynamical systems. Interpretability in model identification is crucial for understanding system processes. While SR models are inherently interpretable, the inclusion of explainability techniques such as SHAP is being explored to improve model validation while facilitating model interpretation by process technologists. In this paper, the Narendra-Li system, which is commonly used as a testbed for dynamical system identification, is considered. The performance of the proposed solutions is evaluated using appropriate interpretability metrics that provide insight into their efficiency in capturing the underlying dynamics of the system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b23544b02be725692cbfb6ad66b208d105ff368" target='_blank'>
              Interpretability analysis of Symbolic Regression models for dynamical systems
              </a>
            </td>
          <td>
            Marco Calapristi, Luca Patané, F. Sapuppo, M. Xibilia
          </td>
          <td>2024-05-15</td>
          <td>2024 International Conference on Control, Automation and Diagnosis (ICCAD)</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Spinal metastasis is exceedingly common in patients with cancer and its prevalence is expected to increase. Surgical management of symptomatic spinal metastasis is indicated for pain relief, preservation or restoration of neurologic function, and mechanical stability. The overall prognosis is a major driver of treatment decisions; however, clinicians’ ability to accurately predict survival is limited. In this narrative review, we first discuss the NOMS decision framework used to guide decision making in the treatment of patients with spinal metastasis. Given that decision making hinges on prognosis, multiple scoring systems have been developed over the last three decades to predict survival in patients with spinal metastasis; these systems have largely been developed using expert opinions or regression modeling. Although these tools have provided significant advances in our ability to predict prognosis, their utility is limited by the relative lack of patient-specific survival probability. Machine learning models have been developed in recent years to close this gap. Employing a greater number of features compared to models developed with conventional statistics, machine learning algorithms have been reported to predict 30-day, 6-week, 90-day, and 1-year mortality in spinal metastatic disease with excellent discrimination. These models are well calibrated and have been externally validated with domestic and international independent cohorts. Despite hypothesized and realized limitations, the role of machine learning methodology in predicting outcomes in spinal metastatic disease is likely to grow.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76ef7d5f5e3bb535fac5a3ea87a5f9ca5d7f86a" target='_blank'>
              Predictive Modeling for Spinal Metastatic Disease
              </a>
            </td>
          <td>
            Akash A. Shah, Joseph H. Schwab
          </td>
          <td>2024-05-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cells utilize bioelectricity to form networks as well as regulate and control a variety of processes such as apoptosis, tumor suppression, and voltage-gated ion channels. In-silico modeling of bioelectrical networks can be performed using BETSE, an application that models gap junctions and ion channel activity of networked cells, but its usage of matrix-based differential equations to estimate these properties limits simulations based on the amount of computational resources available. To alleviate this issue, we trained a total of 8 machine learning models to replace three core functions of BETSE, that is, 1) predicting the average transmembrane potential (Vmem) of an entire cellular network, 2) predicting the Vmem of each individual cell within the network, and finally, 3) predicting the average ion concentrations of sodium, potassium, chloride, and calcium within the cell network. For objective 1, the random forest model was shown to be most performant over all 4 scoring metrics, in objective 2 both the decision tree and k-nearest neighbors models scored best in half of all metrics, and for objective 3 the super learner, a meta-learner comprised of multiple base learners, scored best among all scoring metrics. Overall, these models provide a more resource efficient method of predicting properties of bioelectric cellular networks, and future work will include further properties such as temperature and pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de2b41c77b0e5ab34cf613efc079a6b8d8b1a9a4" target='_blank'>
              Supervised Machine Learning for Bioelectrical Cellular Networks
              </a>
            </td>
          <td>
            Rajeev Jaundoo, T. Craddock, J. Tuszynski
          </td>
          <td>2024-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Our interest is in replicated network data with multiple networks observed across the same set of nodes. Examples include brain connection networks, in which nodes corresponds to brain regions and replicates to different individuals, and ecological networks, in which nodes correspond to species and replicates to samples collected at different locations and/or times. Our goal is to infer a hierarchical structure of the nodes at a population level, while performing multi-resolution clustering of the individual replicates. In brain connectomics, the focus is on inferring common relationships among the brain regions, while characterizing inter-individual variability in an easily interpretable manner. To accomplish this, we propose a Bayesian hierarchical model, while providing theoretical support in terms of identifiability and posterior consistency, and design efficient methods for posterior computation. We provide novel technical tools for proving model identifiability, which are of independent interest. Our simulations and application to brain connectome data provide support for the proposed methodology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a92bd9ad8c8ee3e22f5fcf7a819cb2d7dd4b471" target='_blank'>
              Bayesian Deep Generative Models for Replicated Networks with Multiscale Overlapping Clusters
              </a>
            </td>
          <td>
            Yuren Zhou, Yuqi Gu, D. Dunson
          </td>
          <td>2024-05-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="One of the central goals of neuroscience is to gain a mechanistic understanding of how the dynamics of neural circuits give rise to their observed function. A popular approach towards this end is to train recurrent neural networks (RNNs) to reproduce experimental recordings of neural activity. These trained RNNs are then treated as surrogate models of biological neural circuits, whose properties can be dissected via dynamical systems analysis. How reliable are the mechanistic insights derived from this procedure? While recent advances in population-level recording technologies have allowed simultaneous recording of up to tens of thousands of neurons, this represents only a tiny fraction of most cortical circuits. Here we show that observing only a subset of neurons in a circuit can create mechanistic mismatches between a simulated teacher network and a data-constrained student, even when the two networks have matching single-unit dynamics. In particular, partial observation of models of low-dimensional cortical dynamics based on functionally feedforward or low-rank connectivity can lead to surrogate models with spurious attractor structure. Our results illustrate the challenges inherent in accurately uncovering neural mechanisms from single-trial data, and suggest the need for new methods of validating data-constrained models for neural dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ff706d2507265f316f84e0d354f6a7ce5c6f8a" target='_blank'>
              Partial observation can induce mechanistic mismatches in data-constrained models of neural dynamics
              </a>
            </td>
          <td>
            William Qian, Jacob A. Zavatone-Veth, Benjamin S. Ruben, C. Pehlevan
          </td>
          <td>2024-05-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="MOTIVATION
Complex diseases are often caused and characterized by misregulation of multiple biological pathways. Differential network analysis aims to detect significant rewiring of biological network structures under different conditions and has become an important tool for understanding the molecular etiology of disease progression and therapeutic response. With few exceptions, most existing differential network analysis tools perform differential tests on separately learned network structures that are computationally expensive and prone to collapse when grouped samples are limited or less consistent.


RESULTS
We previously developed an accurate differential network analysis method-Differential Dependency Networks (DDN), that enables joint learning of common and rewired network structures under different conditions. We now introduce the DDN3.0 tool that improves this framework with three new and highly efficient algorithms, namely, unbiased model estimation with a weighted error measure applicable to imbalance sample groups, multiple acceleration strategies to improve learning efficiency, and data-driven determination of proper hyperparameters. The comparative experimental results obtained from both realistic simulations and case studies show that DDN3.0 can help biologists more accurately identify, in a study-specific and often unknown conserved regulatory circuitry, a network of significantly rewired molecular players potentially responsible for phenotypic transitions.


AVAILABILITY
The Python package of DDN3.0 is freely available at https://github.com/cbil-vt/DDN3. A user's guide and a vignette are provided at https://ddn-30.readthedocs.io/.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f7f53646bc589a6aedd4807e452f6e2c56e13c4" target='_blank'>
              DDN3.0: Determining significant rewiring of biological network structure with differential dependency networks.
              </a>
            </td>
          <td>
            Yi Fu, Yingzhou Lu, Yizhi Wang, Bai Zhang, Zhen Zhang, Guoqiang Yu, Chunyu Liu, Robert Clarke, David M. Herrington, Yue Wang
          </td>
          <td>2024-06-20</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Despite innumerable efforts to characterize C. elegans neural activity and behavior, the roles of each individual neuron and the compositional mechanisms of all their interactions are still not completely known. In particular, there is evidence that C. elegans neurons perform changing roles over time, in different behavioral contexts, etc.; however, existing stationary anatomical and functional connectivity can average across time and obfuscate the nonstationary nature of the neural dynamics. We contribute to these efforts by leveraging recent advances in decomposed linear dynamical systems (dLDS) models. dLDS models neural dynamics in a latent space with a set of linear operators that can be recombined and reused over time, enabling the discovery of multiple parallel neural processes on different timescales. We leverage the ability to identify reused patterns of dynamical neural interactions, which we call “dynamical connectivity,” to 1) identify contextually dependent roles of neurons; 2) discover the underlying variability of neural representations even under discrete behaviors; 3) quantify differences between anatomical, functional, and dynamical connectivity; and 4) learn a single aligned latent space underlying multiple individual worms’ activity. These results highlight the importance of defining a neuron’s functions not solely by its internal activity or place in a time-averaged network, but by its evolving interactions across context-dependent circuits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165b84af20920579578d673faaf369e271200b1a" target='_blank'>
              Decomposed Linear Dynamical Systems (dLDS) models reveal context-dependent dynamic connectivity in C. elegans
              </a>
            </td>
          <td>
            Eva Yezerets, Noga Mudrik, Adam S. Charles
          </td>
          <td>2024-06-01</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="In the dynamic realm of time series analysis and forecasting, the pursuit of more precise and efficient models persists as a fundamental objective. This research contributes by presenting a comprehensive comparison involving our recent Hybrid Multivariate model (GRU-1DCNN). Our proposed model capitalizes on intricate architecture, drawing inspiration from a fusion of 1D Convolutional Neural Networks (1DCNN) and Gated Recurrent Unit (GRU) memory cells, with the primary goal of surpassing the performance achieved by previously developed custom models in this domain. Evaluation of the predictive capabilities of the models is conducted using key metrics such as Root Mean Squared Error (RMSE), R-square, Mean Absolute Error (MAE), Mean Squared Logarithmic Error (MSLE), and E-Variance. Notably, the study focusses on addressing the challenges posed by real-world data, exemplified by the volatility inherent in Market data, representing a worst-case scenario. The proposed hybrid model remains modular and exhibits applicability to diverse time series datasets, allowing for the capture of nuanced temporal patterns. This research contributes to the ongoing pursuit of advancing time series modeling techniques, offering insights into the efficacy of our hybrid model and its potential to outperform existing approaches across various domains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/448b03fb69c6feb23215507f68049002c5124ddf" target='_blank'>
              A Contribution to Time Series Analysis and Forecasting Using Deep Learning Approaches
              </a>
            </td>
          <td>
            Abdellah El zaar, Amine Mansouri, Nabil Benaya, Abderrahim El Allati, Toufik Bakir
          </td>
          <td>2024-05-15</td>
          <td>2024 International Conference on Control, Automation and Diagnosis (ICCAD)</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE
The RECIST guidelines provide a standardized approach for evaluating the response of cancer to treatment, allowing for consistent comparison of treatment efficacy across different therapies and patients. However, collecting such information from electronic health records manually can be extremely labor-intensive and time-consuming because of the complexity and volume of clinical notes. The aim of this study is to apply natural language processing (NLP) techniques to automate this process, minimizing manual data collection efforts, and improving the consistency and reliability of the results.


METHODS
We proposed a complex, hybrid NLP system that automates the process of extracting, linking, and summarizing anticancer therapy and associated RECIST-like responses from narrative clinical text. The system consists of multiple machine learning-/deep learning-based and rule-based modules for diverse NLP tasks such as named entity recognition, assertion classification, relation extraction, and text normalization, to address different challenges associated with anticancer therapy and response information extraction. We then evaluated the system performances on two independent test sets from different institutions to demonstrate its effectiveness and generalizability.


RESULTS
The system used domain-specific language models, BioBERT and BioClinicalBERT, for high-performance therapy mentions identification and RECIST responses extraction and categorization. The best-performing model achieved a 0.66 score in linking therapy and RECIST response mentions, with end-to-end performance peaking at 0.74 after relation normalization, indicating substantial efficacy with room for improvement.


CONCLUSION
We developed, implemented, and tested an information extraction system from clinical notes for cancer treatment and efficacy assessment information. We expect this system will support future cancer research, particularly oncologic studies that focus on efficiently assessing the effectiveness and reliability of cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271d99d9a41766cd31875aa69e2912853b6312fe" target='_blank'>
              Extracting Systemic Anticancer Therapy and Response Information From Clinical Notes Following the RECIST Definition.
              </a>
            </td>
          <td>
            X. Zuo, Ashok Kumar, Shuhan Shen, Jianfu Li, Grace Cong, Edward Jin, Qingxia Chen, Jeremy L. Warner, Ping Yang, Hua Xu
          </td>
          <td>2024-06-01</td>
          <td>JCO clinical cancer informatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Predicting the physico-chemical properties of pure substances and mixtures is a central task in thermodynamics. Established prediction methods range from fully physics-based ab-initio calculations, which are only feasible for very simple systems, over descriptor-based methods that use some information on the molecules to be modeled together with fitted model parameters (e.g., quantitative-structure-property relationship methods or classical group contribution methods), to representation-learning methods, which may, in extreme cases, completely ignore molecular descriptors and extrapolate only from existing data on the property to be modeled (e.g., matrix completion methods). In this work, we propose a general method for combining molecular descriptors with representation learning using the so-called expectation maximization algorithm from the probabilistic machine learning literature, which uses uncertainty estimates to trade off between the two approaches. The proposed hybrid model exploits chemical structure information using graph neural networks, but it automatically detects cases where structure-based predictions are unreliable, in which case it corrects them by representation-learning based predictions that can better specialize to unusual cases. The effectiveness of the proposed method is demonstrated using the prediction of activity coefficients in binary mixtures as an example. The results are compelling, as the method significantly improves predictive accuracy over the current state of the art, showcasing its potential to advance the prediction of physico-chemical properties in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ef678f0a68b0a367fe2e0a484ff38a9207367fc" target='_blank'>
              Balancing Molecular Information and Empirical Data in the Prediction of Physico-Chemical Properties
              </a>
            </td>
          <td>
            Johannes Zenn, Dominik Gond, F. Jirasek, Robert Bamler
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Levothyroxine is one of the most prescribed drugs in the western world. Dosing is challenging due to high-interindividual differences in effective dosage and the narrow therapeutic window. Model-informed precision dosing (MIPD) using machine learning could assist general practitioners (GPs), but no such models exist for primary care. Furthermore, introduction of decision-support algorithms in healthcare is limited due to the substantial gap between developers and clinicians' perspectives. We report the development, validation, and a clinical simulation trial of the first MIPD application for primary care. Stable maintenance dosage of levothyroxine was the model target. The multiclass model generates predictions for individual patients, for different dosing classes. Random forest was trained and tested on a national primary care database (n = 19,004) with a final weighted AUC across dosing options of 0.71, even in subclinical hypothyroidism. TSH, fT4, weight, and age were most predictive. To assess the safety, feasibility, and clinical impact of MIPD for levothyroxine, we performed clinical simulation studies in GPs and compared MIPD to traditional prescription. Fifty-one GPs selected starting dosages for 20 primary hypothyroidism cases without and then with MIPD 2 weeks later. Overdosage and underdosage were defined as higher and lower than 12.5 μg relative to stable maintenance dosage. MIPD decreased overdosage in number (30.5 to 23.9%, P < 0.01) and magnitude (median 50 to 37.5 μg, P < 0.01) and increased optimal starting dosages (18.3 to 30.2%, P < 0.01). GPs considered lab results more often with MIPD and most would use the model frequently. This study demonstrates the clinical relevance, safety, and effectiveness of MIPD for levothyroxine in primary care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bb62b88b01331a3ef7595f9fd5ac6627f03ef92" target='_blank'>
              Model-Informed Precision Dosing Using Machine Learning for Levothyroxine in General Practice: Development, Validation and Clinical Simulation Trial.
              </a>
            </td>
          <td>
            Jules M Janssen Daalen, Djoeke Doesburg, Liesbeth Hunik, Rogier Kessel, Thomas Herngreen, Dennis Knol, Thony Ruys, B. J. F. van den Bemt, Henk J. Schers
          </td>
          <td>2024-05-06</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8112c18b6d184d24cba044589140338bfc4b2d5" target='_blank'>
              Advanced methods for gene network identification and noise decomposition from single-cell data
              </a>
            </td>
          <td>
            Zhou Fang, Ankit Gupta, Sant Kumar, Mustafa Khammash
          </td>
          <td>2024-06-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Many clinical trials generate both longitudinal biomarker and time-to-event data. We might be interested in their relationship, as in the case of tumor size and overall survival in oncology drug development. Many well-established methods exist for analyzing such data either sequentially (two-stage models) or simultaneously (joint models). Two-stage modeling (2stgM) has been challenged (i) for not acknowledging that biomarkers are endogenous covariable to the survival submodel and (ii) for not propagating the uncertainty of the longitudinal biomarker submodel to the survival submodel. On the other hand, joint modeling (JM), which properly circumvents both problems, has been criticized for being time-consuming, and difficult to use in practice. In this paper, we explore a third approach, referred to as a novel two-stage modeling (N2stgM). This strategy reduces the model complexity without compromising the parameter estimate accuracy. The three approaches (2stgM, JM, and N2stgM) are formulated, and a Bayesian framework is considered for their implementation. Both real and simulated data were used to analyze the performance of such approaches. In all scenarios, our proposal estimated the parameters approximately as JM but without being computationally expensive, while 2stgM produced biased results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1431f8841ec1dea194d5b4d31717f328d35e5b2" target='_blank'>
              Bridging the gap between two-stage and joint models: The case of tumor growth inhibition and overall survival models.
              </a>
            </td>
          <td>
            Danilo Alvares, F. Mercier
          </td>
          <td>2024-06-03</td>
          <td>Statistics in medicine</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Gene expression programs that establish and maintain specific cellular states are orchestrated through a regulatory network composed of transcription factors, cofactors, and chromatin regulators. Dysregulation of this network can lead to a broad range of diseases by altering gene programs. This article presents LaGrACE, a novel method designed to estimate dysregulation of gene programs utilizing omics data with clinical information. This approach facilitates grouping of samples exhibiting similar patterns of gene program dysregulation, thereby enhancing the discovery of underlying molecular mechanisms. We rigorously evaluated LaGrACE’s performance using synthetic data, bulk RNA-seq clinical datasets (breast cancer, chronic obstructive pulmonary disease (COPD)), and single-cell RNA-seq drug perturbation datasets. Our findings demonstrate that LaGrACE is exceptionally robust in identifying biologically meaningful and prognostic molecular subtypes. Additionally, it effectively discerns drug-response signals at a single-cell resolution. Moreover, the COPD analysis revealed a new association between LEF1 and COPD molecular mechanisms and mortality. Collectively, these results underscore the utility of LaGrACE as a valuable tool for elucidating underlying mechanisms of diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2d4a72bbe23e694e15804f601164485fcfac620" target='_blank'>
              LaGrACE: Estimating gene program dysregulation using latent gene regulatory network for biomedical discovery
              </a>
            </td>
          <td>
            Minxue Jia, Haiyi Mao, Mengli Zhou, Yu-Chih Chen, P. Benos
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af19be9576fba29fc6ce10addc184ed0663d1348" target='_blank'>
              A Computational Predictor for Accurate Identification of Tumor Homing Peptides by Integrating Sequential and Deep BiLSTM Features.
              </a>
            </td>
          <td>
            Roha Arif, Sameera Kanwal, Saeed Ahmed, Muhammad Kabir
          </td>
          <td>2024-05-11</td>
          <td>Interdisciplinary sciences, computational life sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="In the real world, many dynamic behaviors can be explained by the propagation of perturbations, such as the transfer of chemical signals and the spread of infectious diseases. Previous researchers have achieved excellent results in approximating the global propagation time, revealing the mechanism of signal propagation through multiple paths. However, the known frameworks rely on the extension of physical concepts rather than mathematically rigorous derivations. As a result, they may not perfectly predict time or explain the underlying physical significance in certain specific cases. In this paper, we propose a novel method for decomposing network topology, focusing on two modules: the tree-like module and the path-module. Subsequently, we introduce a new framework for signal propagation analysis, which can be applied to estimate the propagation time for two fundamental global topology modules and provide a rigorous proof for the propagation time in global topology. Compared to previous work, our results are not only more concise, clearly defined, efficient, but also are more powerful in predicting propagation time which outperforms some known results in some cases, for example, biochemical dynamics.Additionally, the proposed framework is based on information transfer pathways, which can be also applied to other physical fields, such as network stability, network controlling and network resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5df6abe3831bf4c489162c2dc445d5e085e2f6f" target='_blank'>
              Information transfer pathways: signal propagation in complex global topologies
              </a>
            </td>
          <td>
            Qitong Hu, Xiao-Dong Zhang
          </td>
          <td>2024-05-20</td>
          <td>Physica Scripta</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="1614 Background: Despite the unique ability of machine learning (ML) techniques to uncover complex interactive effects in datasets, studies exploring predictors of clinical trial enrollment have largely used standard statistical methods to discern associations. Furthermore, there have been limited applications of ML to close existing disparities in clinical trial enrollment of underrepresented patients in neuro-oncology. This study aims to use supervised machine learning to define and validate primary contributors of therapeutic clinical trial enrollment for a) all patients, b) women and c) NIH-designated minority patients with low- and high-grade glioma. Methods: Adult glioma patients who received care from the UCSF Brain Tumor Center between 1997- 2017 were identified in a prospective registry. Bootstrap forest (BF) and Recursive Partitioning (RP) models were created by randomly dividing patients in a 70/30 split into development (DEV) and validation (VAL) cohorts among all patients, women, and NIH-designated minority patients, separately. Model performance was assessed using the area under the curve (AUC) in the DEV and VAL cohorts. Results: Among 1042 patients, 350 patients (33.6%) enrolled in a therapeutic clinical trial. There were 445 women (42.7%), of which 144 (32.4%) enrolled, and 141 minority patients (13.5%), of which 39 (27.7%) enrolled. For all patients, in order of decreasing influence, the BF model selected median neighborhood household income, age, neighborhood poverty level, distance from hospital, tumor location, tumor volume, treatment with chemotherapy, occupation, KPS, insurance status, employment status, seizure at presentation, and volumetric extent of resection (EOR) to predict trial enrollment (DEV AUC: 0.984; VAL AUC: 0.746). For women, the BF model selected distance from hospital, age, household income, neighborhood poverty level, tumor location, treatment with chemotherapy, and KPS for trial prediction (DEV AUC: 0.972; VAL AUC: 0.746), whereas for minority patients, the BF model selected neighborhood poverty level, age, occupation, distance from hospital, tumor volume, insurance status, employment status, and EOR to predict trial enrollment (DEV AUC: 0.9830; VAL AUC: 0.769). The RP model for women designated those with non-White race as less likely to enroll (DEV AUC: 0.735; VAL AUC: 0.665) while the RP model for minority patients characterized those who preferred a non-English language or who were divorced or widowed as less likely to enroll (DEV AUC: 0.775; VAL AUC: 0.601). Conclusions: Supervised machine learning models achieved similar predictive performance among patient subgroups, while uncovering interactions between gender identity, minority status, and sociodemographic variables. These findings can guide targeted recruiting efforts to advance equitable trial enrollment for women and minorities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f1876cfabc7bccab3e267b2ca9706dc1d0f3bf" target='_blank'>
              Predicting therapeutic clinical trial enrollment for underrepresented patients in neuro-oncology using supervised machine learning.
              </a>
            </td>
          <td>
            Mulki Mehari, G. Warrier, Abraham Dada, Aymen Kabir, U. Chukwueke, Alyx B Porter, Shawn L. Hervey-Jumper
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Data science is playing an increasingly important role in the design and analysis of engineered biology. This has been fueled by the development of high-throughput methods like massively parallel reporter assays, data-rich microscopy techniques, computational protein structure prediction and design, and the development of whole-cell models able to generate huge volumes of data. Although the ability to apply data-centric analyses in these contexts is appealing and increasingly simple to do, it comes with potential risks. For example, how might biases in the underlying data affect the validity of a result and what might the environmental impact of large-scale data analyses be? Here, we present a community-developed framework for assessing data hazards to help address these concerns and demonstrate its application to two synthetic biology case studies. We show the diversity of considerations that arise in common types of bioengineering projects and provide some guidelines and mitigating steps. Understanding potential issues and dangers when working with data and proactively addressing them will be essential for ensuring the appropriate use of emerging data-intensive AI methods and help increase the trustworthiness of their applications in synthetic biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/993a0859db6bdf23a69bbfc26332b2731030205e" target='_blank'>
              Data hazards in synthetic biology
              </a>
            </td>
          <td>
            Natalie R Zelenka, Nina Di Cara, Kieren Sharma, Seeralan Sarvaharman, Jasdeep S. Ghataora, Fabio Parmeggiani, Jeff Nivala, Z. Abdallah, Lucia Marucci, T. Gorochowski
          </td>
          <td>2024-06-21</td>
          <td>Synthetic Biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Nowcasting leverages real-time atmospheric conditions to forecast weather over short periods. State-of-the-art models, including PySTEPS, encounter difficulties in accurately forecasting extreme weather events because of their unpredictable distribution patterns. In this study, we design a physics-informed neural network to perform precipitation nowcasting using the precipitation and meteorological data from the Royal Netherlands Meteorological Institute (KNMI). This model draws inspiration from the novel Physics-Informed Discriminator GAN (PID-GAN) formulation, directly integrating physics-based supervision within the adversarial learning framework. The proposed model adopts a GAN structure, featuring a Vector Quantization Generative Adversarial Network (VQ-GAN) and a Transformer as the generator, with a temporal discriminator serving as the discriminator. Our findings demonstrate that the PID-GAN model outperforms numerical and SOTA deep generative models in terms of precipitation nowcasting downstream metrics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c60f51fd0bc597ca89a337855e1ba9488b62c377" target='_blank'>
              Precipitation Nowcasting Using Physics Informed Discriminator Generative Models
              </a>
            </td>
          <td>
            Junzhe Yin, Cristian Meo, Ankush Roy, Zeineh Bou Cher, Yanbo Wang, R. Imhoff, R. Uijlenhoet, Justin Dauwels
          </td>
          <td>2024-06-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Proteins are complex molecules responsible for different functions in nature. Enhancing the functionality of proteins and cellular fitness can significantly impact various industries. However, protein optimization using computational methods remains challenging, especially when starting from low-fitness sequences. We propose LatProtRL, an optimization method to efficiently traverse a latent space learned by an encoder-decoder leveraging a large protein language model. To escape local optima, our optimization is modeled as a Markov decision process using reinforcement learning acting directly in latent space. We evaluate our approach on two important fitness optimization tasks, demonstrating its ability to achieve comparable or superior fitness over baseline methods. Our findings and in vitro evaluation show that the generated sequences can reach high-fitness regions, suggesting a substantial potential of LatProtRL in lab-in-the-loop scenarios.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9589c7a5062134012d56bf3ea6aedd1d76bfb1d" target='_blank'>
              Robust Optimization in Protein Fitness Landscapes Using Reinforcement Learning in Latent Space
              </a>
            </td>
          <td>
            Minji Lee, L. Vecchietti, Hyunkyu Jung, Hyun Joo Ro, Meeyoung Cha, Ho Min Kim
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="To make accurate predictions, understand mechanisms, and design interventions in systems of many variables, we wish to learn causal graphs from large scale data. Unfortunately the space of all possible causal graphs is enormous so scalably and accurately searching for the best fit to the data is a challenge. In principle we could substantially decrease the search space, or learn the graph entirely, by testing the conditional independence of variables. However, deciding if two variables are adjacent in a causal graph may require an exponential number of tests. Here we build a scalable and flexible method to evaluate if two variables are adjacent in a causal graph, the Differentiable Adjacency Test (DAT). DAT replaces an exponential number of tests with a provably equivalent relaxed problem. It then solves this problem by training two neural networks. We build a graph learning method based on DAT, DAT-Graph, that can also learn from data with interventions. DAT-Graph can learn graphs of 1000 variables with state of the art accuracy. Using the graph learned by DAT-Graph, we also build models that make much more accurate predictions of the effects of interventions on large scale RNA sequencing data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aba9e483fc80b2246e0043768867a60815d1c888" target='_blank'>
              Scalable and Flexible Causal Discovery with an Efficient Test for Adjacency
              </a>
            </td>
          <td>
            Alan Nawzad Amin, Andrew Gordon Wilson
          </td>
          <td>2024-06-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adefdc4e200af3697cccf8de7ce2fdd6f957f293" target='_blank'>
              LORIS robustly predicts patient outcomes with immune checkpoint blockade therapy using common clinical, pathologic and genomic features.
              </a>
            </td>
          <td>
            Tiangen Chang, Yingying Cao, Hannah J Sfreddo, S. R. Dhruba, Se-Hoon Lee, Cristina Valero, Seong-Keun Yoo, D. Chowell, Luc G T Morris, E. Ruppin
          </td>
          <td>2024-06-03</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The aging process is a complex and multifaceted phenomenon affecting all living organisms. It involves a gradual deterioration of tissue and cellular function, leading to a higher risk of developing various age-related diseases (ARDs), including cancer, neurodegenerative, and cardiovascular diseases. The gene regulatory networks (GRNs) and their respective niches are crucial in determining the aging rate. Unveiling these GRNs holds promise for developing novel therapies and diagnostic tools to enhance healthspan and longevity. This review examines GRN modeling approaches in aging, encompassing differential equations, Boolean/fuzzy logic decision trees, Bayesian networks, mutual information, and regression clustering. These approaches provide nuanced insights into the intricate gene-protein interactions in aging, unveiling potential therapeutic targets and ARD biomarkers. Nevertheless, outstanding challenges persist, demanding more comprehensive datasets and advanced algorithms to comprehend and predict GRN behavior accurately. Despite these hurdles, identifying GRNs associated with aging bears immense potential and is poised to transform our comprehension of human health and aging. This review aspires to stimulate further research in aging, fostering the innovation of computational approaches for promoting healthspan and longevity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126e6aff53e70f3203e5ef0b41bbc26dab6f16fc" target='_blank'>
              Computational modeling of aging-related gene networks: a review
              </a>
            </td>
          <td>
            José Américo Nabuco Leva Ferreira Freitas, Oliver Bischof
          </td>
          <td>2024-05-28</td>
          <td>Frontiers in Applied Mathematics and Statistics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="With the development of high-throughput technologies, genomics datasets rapidly grow in size, including functional genomics data. This has allowed the training of large Deep Learning (DL) models to predict epigenetic readouts, such as protein binding or histone modifications, from genome sequences. However, large dataset sizes come at a price of data consistency, often aggregating results from a large number of studies, conducted under varying experimental conditions. While data from large-scale consortia are useful as they allow studying the effects of different biological conditions, they can also contain unwanted biases from confounding experimental factors. Here, we introduce Metadata-guided Feature Disentanglement (MFD) - an approach that allows disentangling biologically relevant features from potential technical biases. MFD incorporates target metadata into model training, by conditioning weights of the model output layer on different experimental factors. It then separates the factors into disjoint groups and enforces independence of the corresponding feature subspaces with an adversarially learned penalty. We show that the metadata-driven disentanglement approach allows for better model introspection, by connecting latent features to experimental factors, without compromising, or even improving performance in downstream tasks, such as enhancer prediction, or genetic variant discovery. The code for our implemementation is available at https://github.com/HealthML/MFD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32dd14e31f771ca226153c9210adaac58da256a4" target='_blank'>
              Metadata-guided Feature Disentanglement for Functional Genomics
              </a>
            </td>
          <td>
            Alexander Rakowski, Remo Monti, Viktoriia Huryn, Marta Lemanczyk, Uwe Ohler, Christoph Lippert
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="To support mechanism online learning and facilitate digital twin development for biomanufacturing processes, this paper develops an efficient Bayesian inference approach for partially observed enzymatic stochastic reaction network (SRN), a fundamental building block of multi-scale bioprocess mechanistic model. To tackle the critical challenges brought by the nonlinear stochastic differential equations (SDEs)-based mechanistic model with partially observed state and having measurement error, an interpretable Bayesian updating linear noise approximation (LNA) metamodel, incorporating the structure information of the mechanistic model, is proposed to approximate the likelihood of observations. Then, an efficient posterior sampling approach is developed by utilizing the gradients of the derived likelihood to speed up the convergence of MCMC. The empirical study demonstrates that the proposed approach has a promising performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57d6d6543be531495a2c396519431852457ac4af" target='_blank'>
              Linear Noise Approximation Assisted Bayesian Inference on Mechanistic Model of Partially Observed Stochastic Reaction Network
              </a>
            </td>
          <td>
            Wandi Xu, Wei Xie
          </td>
          <td>2024-05-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Techniques to autonomously drive research have been prominent in Computational Scientific Discovery, while Synthetic Biology is a field of science that focuses on designing and constructing new biological systems for useful purposes. Here we seek to apply logic-based machine learning techniques to facilitate cellular engineering and drive biological discovery. Comprehensive databases of metabolic processes called genome-scale metabolic network models (GEMs) are often used to evaluate cellular engineering strategies to optimise target compound production. However, predicted host behaviours are not always correctly described by GEMs, often due to errors in the models. The task of learning the intricate genetic interactions within GEMs presents computational and empirical challenges. To address these, we describe a novel approach called Boolean Matrix Logic Programming (BMLP) by leveraging boolean matrices to evaluate large logic programs. We introduce a new system, $BMLP_{active}$, which efficiently explores the genomic hypothesis space by guiding informative experimentation through active learning. In contrast to sub-symbolic methods, $BMLP_{active}$ encodes a state-of-the-art GEM of a widely accepted bacterial host in an interpretable and logical representation using datalog logic programs. Notably, $BMLP_{active}$ can successfully learn the interaction between a gene pair with fewer training examples than random experimentation, overcoming the increase in experimental design space. $BMLP_{active}$ enables rapid optimisation of metabolic models to reliably engineer biological systems for producing useful compounds. It offers a realistic approach to creating a self-driving lab for microbial engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4217fa5bc20619984d78d30c100e2b19faa0091" target='_blank'>
              Boolean matrix logic programming for active learning of gene functions in genome-scale metabolic network models
              </a>
            </td>
          <td>
            L. Ai, Stephen Muggleton, Shishun Liang, Geoff S. Baldwin
          </td>
          <td>2024-05-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genotype-to-phenotype mapping is an essential problem in the current genomic era. While qualitative case-control predictions have received significant attention, less emphasis has been placed on predicting quantitative phenotypes. This emerging field holds great promise in revealing intricate connections between microbial communities and host health. However, the presence of heterogeneity in microbiome datasets poses a substantial challenge to the accuracy of predictions and undermines the reproducibility of models. To tackle this challenge, we investigated 22 normalization methods that aimed at removing heterogeneity across multiple datasets, conducted a comprehensive review of them, and evaluated their effectiveness in predicting quantitative phenotypes in three simulation scenarios and 31 real datasets. The results indicate that none of these methods demonstrate significant superiority in predicting quantitative phenotypes or attain a noteworthy reduction in Root Mean Squared Error (RMSE) of the predictions. Given the frequent occurrence of batch effects and the satisfactory performance of batch correction methods in predicting datasets affected by these effects, we strongly recommend utilizing batch correction methods as the initial step in predicting quantitative phenotypes. In summary, the performance of normalization methods in predicting metagenomic data remains a dynamic and ongoing research area. Our study contributes to this field by undertaking a comprehensive evaluation of diverse methods and offering valuable insights into their effectiveness in predicting quantitative phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a680651a3cf41ba9343ce9f9c1efc81ca6d5163" target='_blank'>
              Evaluation of normalization methods for predicting quantitative phenotypes in metagenomic data analysis
              </a>
            </td>
          <td>
            Beibei Wang, Yihui Luan
          </td>
          <td>2024-06-05</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Inferring causal protein signalling networks from human immune system cell data is a promising approach to unravel the underlying tissue signalling biology and dysfunction in diseased cells, which has attracted considerable attention within the bioinformatics field. Recently, Bayesian network (BN) techniques have gained significant popularity in inferring causal protein signalling networks from multiparameter single‐cell data. However, current BN methods may exhibit high computational complexity and ignore interactions among protein signalling molecules from different single cells. A novel BN method is presented for learning causal protein signalling networks based on parallel discrete artificial bee colony (PDABC), named PDABC. Specifically, PDABC is a score‐based BN method that utilises the parallel artificial bee colony to search for the global optimal causal protein signalling networks with the highest discrete K2 metric. The experimental results on several simulated datasets, as well as a previously published multi‐parameter fluorescence‐activated cell sorter dataset, indicate that PDABC surpasses the existing state‐of‐the‐art methods in terms of performance and computational efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45fc9744b37a980e5e05a5eeb12a6c928df4c0fc" target='_blank'>
              Inferring causal protein signalling networks from single‐cell data based on parallel discrete artificial bee colony algorithm
              </a>
            </td>
          <td>
            Jinduo Liu, Jihao Zhai, Junzhong Ji
          </td>
          <td>2024-05-11</td>
          <td>CAAI Transactions on Intelligence Technology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Our paper deals with a collection of networks on a common set of nodes, where some of the networks are associated with responses. Assuming that the networks correspond to points on a one-dimensional manifold in a higher dimensional ambient space, we propose an algorithm to consistently predict the response at an unlabeled network. Our model involves a specific multiple random network model, namely the common subspace independent edge model, where the networks share a common invariant subspace, and the heterogeneity amongst the networks is captured by a set of low dimensional matrices. Our algorithm estimates these low dimensional matrices that capture the heterogeneity of the networks, learns the underlying manifold by isomap, and consistently predicts the response at an unlabeled network. We provide theoretical justifications for the use of our algorithm, validated by numerical simulations. Finally, we demonstrate the use of our algorithm on larval Drosophila connectome data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb592d6fdf3ce2b11fe06b667a08556edadacce2" target='_blank'>
              Consistent response prediction for multilayer networks on unknown manifolds
              </a>
            </td>
          <td>
            Aranyak Acharyya, Jes'us Arroyo Reli'on, Michael Clayton, Marta Zlatic, Youngser Park, Carey E. Priebe
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="3064 Background: In recent years, the use of tumor molecular profiling in clinical settings has enhanced cancer diagnostics, as well as the delivery of precision oncology. Recently, several methods for predicting gene expression directly from Haematoxylin-and-Eosin-stained (H&E) histology images have offered a new way to leverage the easily obtainable and cost-effective H&E images for multiple precision oncology applications. We previously introduced such a method – DeepPT – and demonstrated how we can leverage its imputed gene expression for successful prediction of drug response, through our ENLIGHT-DeepPT platform. In our previous publication and in an independent study comparing six different methods, DeepPT exhibited the best overall gene expression prediction accuracy. Methods: Here we present a new version, DeepPT 2.0, with improved architecture, including multi-task learning and a feature space based on a self-supervised pre-trained deep network. We tested both versions of DeepPT and the leading competing methods on patient data from 17 cancer subtypes included in TCGA. In addition, we obtained a new dataset collected at the Medical University of Bialystok (UMB) consisting of matched slide images and mRNA expression from 151 cases representing 7 cancer subtypes. These serve as an external validation to demonstrate that DeepPT generalizes well. Results: Our findings indicate that both versions of DeepPT show statistically significant improvement compared with the other methods in terms of median Pearson correlation of top predicted genes - the only metric available for all methods. Moreover, DeepPT 2.0 significantly improves upon version 1.0, demonstrating up to a 3-fold increase in the number of well-predicted genes (defined as genes with Pearson 𝜌 > 0.4 between actual and predicted mRNA expressions) for 14 of the 17 cancer subtypes tested. On the external validation data from UMB, DeepPT 2.0 improves gene expression prediction in 6 of 7 of the tested cancer subtypes, with up to a 7-fold increase in the number of well-predicted genes, thus mitigating the concern of overfitting on the training set. Immune genes are particularly well predicted, as we previously observed: using a set of 826 hallmark immune genes, DeepPT 2.0 exhibits up to 3.5-fold and 2-fold increase in the percentage of well-predicted genes in the TCGA and UMB data, respectively, compared to all other genes. Conclusions: DeepPT 2.0 significantly improves upon competing methods for predicting mRNA expression from H&E slides across multiple metrics and diverse cancer types. Furthermore, it demonstrates robust generalization to slides from sources not previously seen. The method’s good ability to predict the expression of immune genes suggests a potential benefit in predicting response to immunotherapy using the ENLIGHT-DeepPT platform, as indeed demonstrated elsewhere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c75aa806813e51a7f41b199e9391b3e01188e73" target='_blank'>
              Development, comparative study, and external validation of a new deep learning model for predicting genome-wide gene expression from histopathology slides.
              </a>
            </td>
          <td>
            Leon Gugel, O. Tirosh, Yaron Kinar, Anna Michalska-Falkowska, Joanna Reszeć-Giełażyn, Witold Bauer, Magdalena Niemira, Boris Temkin, T. Beker, R. Aharonov, Gal Dinstag
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Contemporary radiation risk assessment predominantly depends on nonlinear parametric models, which typically include a baseline term, a dose-response term, and an effect modifier term. Despite their widespread application in estimating tumor risks, parametric models face a notable drawback: their rigid model structure can be overly restrictive, potentially introducing bias and inaccuracies into risk estimations. In this study, we analyze data on solid tumors and leukemia from the Life Span Study (LSS) to compare the performance of deep neural network (DNN) and nonlinear parametric (NLP) models in assessing ERRs. DNN presents novel perspectives for radiation risk assessment. Our findings indicate that DNN can perform better than the traditional parametric models. Even if DNN and NLP models exhibit similar performance in predicting tumor incidence, they diverge significantly in their estimated ERRs. Standard NLP models tend to underestimate ERRs directly linked to radiation dose, overestimate ERRs for individuals at younger attained ages and ages at exposure, and underestimate ERRs for those at older attained ages. Furthermore, DNN consistently identifies radiation dose as the primary and predominant risk factor for ERRs in leukemia and solid tumors, underscoring the critical role of radiation dose in risk assessment. The insights from DNN could enhance low-dose radiation risk assessment and improve parametric model development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c35286a4ae224457c64400796ef637896de940" target='_blank'>
              Novel Insights for Radiation Risk Assessment Unveiled by Deep Learning
              </a>
            </td>
          <td>
            Z. Liu, I. Shuryak, D. J. Brenner, R. L. Ullrich
          </td>
          <td>2024-04-30</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Computational pathology, utilizing whole slide image (WSI) for pathological diagnosis, has advanced the development of intelligent healthcare. However, the scarcity of annotated data and histological differences hinder the general application of existing methods. Extensive histopathological data and the robustness of self-supervised models in small-scale data demonstrate promising prospects for developing foundation pathology models. In this work, we propose the BEPH (BEiT-based model Pre-training on Histopathological image), a general method that leverages self-supervised learning to learn meaningful representations from 11 million unlabeled histopathological images. These representations are then efficiently adapted to various tasks, including patch-level cancer recognition, WSI-level cancer classification, and survival prediction for multiple cancer subtypes. Experimental results demonstrate that our model consistently outperforms several comparative models, even with limited training data reduced to 50%. Especially when the downstream structure is the same, the model can improve ResNet and DINO by up to a maximum increase of 8.8% and 7.2% (WSI level classification), and 6.44% and 3.28% on average (survival prediction), respectively. Therefore, BEPH offers a universal solution to enhance model performance, reduce the burden of expert annotations, and enable widespread clinical applications of artificial intelligence. The code and models can be obtained at https://github.com/Zhcyoung/BEPH. And currently, online fine-tuning of WSI classification tasks is available for use on http://yulab-sjtu.natapp1.cc/BEPH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6572398e4b60a8f1eaab9108d27cf5629267e013" target='_blank'>
              A foundation model for generalizable cancer diagnosis and survival prediction from histopathological images
              </a>
            </td>
          <td>
            Zhaochang Yang, Ting Wei, Ying Liang, Xin Yuan, Ruitian Gao, Yujia Xia, Jie Zhou, Yue Zhang, Zhangsheng Yu
          </td>
          <td>2024-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In dynamical systems reconstruction (DSR) we seek to infer from time series measurements a generative model of the underlying dynamical process. This is a prime objective in any scientific discipline, where we are particularly interested in parsimonious models with a low parameter load. A common strategy here is parameter pruning, removing all parameters with small weights. However, here we find this strategy does not work for DSR, where even low magnitude parameters can contribute considerably to the system dynamics. On the other hand, it is well known that many natural systems which generate complex dynamics, like the brain or ecological networks, have a sparse topology with comparatively few links. Inspired by this, we show that geometric pruning, where in contrast to magnitude-based pruning weights with a low contribution to an attractor's geometrical structure are removed, indeed manages to reduce parameter load substantially without significantly hampering DSR quality. We further find that the networks resulting from geometric pruning have a specific type of topology, and that this topology, and not the magnitude of weights, is what is most crucial to performance. We provide an algorithm that automatically generates such topologies which can be used as priors for generative modeling of dynamical systems by RNNs, and compare it to other well studied topologies like small-world or scale-free networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7262300a7dc61b5b9020753dc9d2bbd63a06cc2" target='_blank'>
              Optimal Recurrent Network Topologies for Dynamical Systems Reconstruction
              </a>
            </td>
          <td>
            Christoph Jurgen Hemmer, Manuel Brenner, Florian Hess, Daniel Durstewitz
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Among many types of cancers, to date, lung cancer remains one of the deadliest cancers around the world. Many researchers, scientists, doctors, and people from other fields continuously contribute to this subject regarding early prediction and diagnosis. One of the significant problems in prediction is the black-box nature of machine learning models. Though the detection rate is comparatively satisfactory, people have yet to learn how a model came to that decision, causing trust issues among patients and healthcare workers. This work uses multiple machine learning models on a numerical dataset of lung cancer-relevant parameters and compares performance and accuracy. After comparison, each model has been explained using different methods. The main contribution of this research is to give logical explanations of why the model reached a particular decision to achieve trust. This research has also been compared with a previous study that worked with a similar dataset and took expert opinions regarding their proposed model. We also showed that our research achieved better results than their proposed model and specialist opinion using hyperparameter tuning, having an improved accuracy of almost 100% in all four models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ab3266b664a06cc9fa514bfddfc0915e16092bd" target='_blank'>
              The efficacy of machine learning models in lung cancer risk prediction with explainability
              </a>
            </td>
          <td>
            Refat Khan Pathan, Israt Jahan Shorna, Md Sayem Hossain, Mayeen Uddin Khandaker, H. Almohammed, Zuhal Y. Hamd
          </td>
          <td>2024-06-13</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The growth of bacterial gene expression datasets has offered unprecedented coverage of achievable transcriptomes, reflecting diverse activity states of the transcription regulatory network. Machine learning methods like Independent Component Analysis (ICA) can decompose gene expression datasets into regulatory modules and condition-specific regulator activities. Here, we present a workflow to utilize inferred regulator activities to construct quantitative models of promoter regulation in E. coli. Resulting models are validated by predicting condition-specific TF effector concentrations and binding site motif strength based on differential gene expression data alone. We show how reconstructed promoter models can capture multi-scale regulation and disentangle regulator interactions, including resolving the apparent paradox where argR expression is positively correlated with its regulon despite being a repressor. We applied the workflow for all regulator-linked components extracted by ICA, demonstrating the scalability of the workflow to capture the E. coli TRN. This work suggests a path toward systematic, quantitative reconstruction of transcription regulatory networks driven by the large-scale databases that are now available for many organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54beacc49acede7e1ab1316132bfe8fb330701f" target='_blank'>
              Data-driven modeling of E. coli transcriptional regulation
              </a>
            </td>
          <td>
            Christopher Dalldorf, Griffith Hughes, Gaoyuan Li, Bernhard O. Palsson, Daniel C. Zielinski
          </td>
          <td>2024-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Identifying predictive biomarkers, which forecast individual treatment effectiveness, is crucial for personalized medicine and informs decision-making across diverse disciplines. These biomarkers are extracted from pre-treatment data, often within randomized controlled trials, and have to be distinguished from prognostic biomarkers, which are independent of treatment assignment. Our study focuses on the discovery of predictive imaging biomarkers, aiming to leverage pre-treatment images to unveil new causal relationships. Previous approaches relied on labor-intensive handcrafted or manually derived features, which may introduce biases. In response, we present a new task of discovering predictive imaging biomarkers directly from the pre-treatment images to learn relevant image features. We propose an evaluation protocol for this task to assess a model's ability to identify predictive imaging biomarkers and differentiate them from prognostic ones. It employs statistical testing and a comprehensive analysis of image feature attribution. We explore the suitability of deep learning models originally designed for estimating the conditional average treatment effect (CATE) for this task, which previously have been primarily assessed for the precision of CATE estimation, overlooking the evaluation of imaging biomarker discovery. Our proof-of-concept analysis demonstrates promising results in discovering and validating predictive imaging biomarkers from synthetic outcomes and real-world image datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1812c80b6fcbeea9ef27bb1b213713647799840b" target='_blank'>
              Enhancing predictive imaging biomarker discovery through treatment effect analysis
              </a>
            </td>
          <td>
            Shuhan Xiao, Lukas Klein, Jens Petersen, Philipp Vollmuth, P. Jaeger, Klaus H Maier-Hein
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Explaining deep learning models operating on time series data is crucial in various applications of interest which require interpretable and transparent insights from time series signals. In this work, we investigate this problem from an information theoretic perspective and show that most existing measures of explainability may suffer from trivial solutions and distributional shift issues. To address these issues, we introduce a simple yet practical objective function for time series explainable learning. The design of the objective function builds upon the principle of information bottleneck (IB), and modifies the IB objective function to avoid trivial solutions and distributional shift issues. We further present TimeX++, a novel explanation framework that leverages a parametric network to produce explanation-embedded instances that are both in-distributed and label-preserving. We evaluate TimeX++ on both synthetic and real-world datasets comparing its performance against leading baselines, and validate its practical efficacy through case studies in a real-world environmental application. Quantitative and qualitative evaluations show that TimeX++ outperforms baselines across all datasets, demonstrating a substantial improvement in explanation quality for time series data. The source code is available at \url{https://github.com/zichuan-liu/TimeXplusplus}.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41331316e9b6cf30546793e0963a8a78d8691c86" target='_blank'>
              TimeX++: Learning Time-Series Explanations with Information Bottleneck
              </a>
            </td>
          <td>
            Zichuan Liu, Tianchun Wang, Jimeng Shi, Xu Zheng, Zhuomin Chen, Lei Song, Wenqian Dong, J. Obeysekera, Farhad Shirani, Dongsheng Luo
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Uncertainty in parameter estimates from fitting within-host models to empirical data limits the model’s ability to uncover mechanisms of infection, disease progression, and to guide pharmaceutical interventions. Understanding the effect of model structure and data availability on model predictions is important for informing model development and experimental design. To address sources of uncertainty in parameter estimation, we use four mathematical models of influenza A infection with increased degrees of biological realism. We test the ability of each model to reveal its parameters in the presence of unlimited data by performing structural identifiability analyses. We then refine the results by predicting practical identifiability of parameters under daily influenza A virus titers alone or together with daily adaptive immune cell data. Using these approaches, we present insight into the sources of uncertainty in parameter estimation and provide guidelines for the types of model assumptions, optimal experimental design, and biological information needed for improved predictions. Author summary Within-host models of virus infections fitted to data have improved our understanding of mechanisms of infection, disease progression, and allowed us to provide guidelines for pharmaceutical interventions. Given their predictive power, it is essential that we properly uncover and address uncertainty in model predictions and parameter estimation. Here, we focus on the effect of model structure and data availability on our ability to uncover unknown parameters. To address these questions, we use four mathematical models of influenza A infection with increased degrees of biological realism. We test the ability of each model to reveal its parameters in the presence of unlimited data by performing structural identifiability analysis. We then refine the results by predicting practical identifiability of parameters under daily influenza A virus titers alone or together with daily adaptive immune cell data. Using these approaches, we present insights into the sources of uncertainty in parameter estimations and provide guidelines for the types of model assumptions, optimal experimental design, and biological information needed for improved predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08312e1a270ca4ad6a36e75e2b5550f77112dd89" target='_blank'>
              Identifiability investigation of within-host models of acute virus infection
              </a>
            </td>
          <td>
            Yuganthi R. Liyanage, Nora Heitzman-Breen, N. Tuncer, Stanca M. Ciupe
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Purpose: To evaluate the efficacy of prominent machine learning algorithms in predicting normal tissue complication probability utilizing clinical data obtained from two distinct disease sites, and to create a software tool that facilitates the automatic determination of the optimal algorithm to model any given labeled dataset. Methods and Materials: We obtained 3 sets of radiation toxicity data (478 patients) from our clinic, gastrointestinal toxicity (GIT), radiation pneumonitis (RP), and radiation esophagitis (RE). These data comprised clinicopathological and dosimetric information for patients diagnosed with non-small cell lung cancer and anal squamous cell carcinoma. Each dataset was modeled using ten commonly employed machine learning algorithms (elastic net, LASSO, random forest, regression forest, support vector machine, XGBoost, k-nearest-neighbors, neural network, Bayesian-LASSO, and Bayesian neural network) by randomly dividing the dataset into a training and test set. The training set was used to create and tune the model, and the test set served to assess it by calculating performance metrics. This process was repeated 100 times by each algorithm for each dataset. Figures were generated to visually compare the performance of the algorithms. A graphical user interface was developed to automate this whole process. Results: LASSO achieved the highest area under the precision-recall curve (AUPRC) (0.807{+/-}0.067) for RE, random forest for GIT (0.726{+/-}0.096), and the neural network for RP (0.878{+/-}0.060). Area-under-the-curve was 0.754{+/-}0.069, 0.889{+/-}0.043, and 0.905{+/-}0.045, respectively. The graphical user interface was used to compare all algorithms for each dataset automatically. When averaging AUPRC across all toxicities, Bayesian-LASSO was the best model. Conclusion: Our results show that there is no best algorithm for all datasets. Therefore, it is important to compare multiple algorithms when training an outcome prediction model on a new dataset. The graphical user interface created for this study automatically compares the performance of these ten algorithms for any dataset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5e39f5ff1a961de94f4b53bfa977bf8735ef11" target='_blank'>
              Performance comparison of ten state-of-the-art machine learning algorithms for outcome prediction modeling of radiation-induced toxicity
              </a>
            </td>
          <td>
            PhD Ramon M. Salazar, MSc Saurabh S. Nair, Mbs Alexandra O. Leone, PhD Ting Xu, BS Raymond P. Mumme, BA Jack D. Duryea, MD Brian De, MD Kelsey L. Corrigan, MD Michael K. Rooney, MD Matthew S. Ning, MD Prajnan Das, MD Emma B. Holliday, MD Zhongxing Liao, PhD Laurence E. Court, PhD Joshua S. Niedzielski
          </td>
          <td>2024-05-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Allostery is a fundamental mechanism driving biomolecular processes that holds significant therapeutic concern. Our study rigorously investigates how two distinct machine-learning algorithms uniquely classify two already close-to-active DFG-in states of TAK1, differing just by the presence or absence of its allosteric activator TAB1, from an ensemble mixture of conformations (obtained from 2.4 μs molecular dynamics (MD) simulations). The novelty, however, lies in understanding the deeper algorithmic potentials to systematically derive a diverse set of differential residue connectivity features that reconstruct the essential mechanistic architecture for TAK1-TAB1 allostery in such a close-to-active biochemical scenario. While the recursive, random forest-based workflow displays the potential of conducting discretized, hierarchical derivation of allosteric features, a multilayer perceptron-based approach gains considerable efficacy in revealing fluid connected patterns of features when hybridized with mutual information scoring. Interestingly, both pipelines benchmark similar directions of functional conformational changes for TAK1's activation. The findings significantly advance the depth of mechanistic understanding by highlighting crucial activation signatures along a directed C-lobe → activation loop → ATP pocket channel of information flow, including (1) the αF-αE biterminal alignments and (2) the "catalytic" drift of the activation loop toward kinase active site. Besides, some novel allosteric hotspots (K253, Y206, N189, etc.) are further recognized as TAB1 sensors, transducers, and responders, including a benchmark E70 mutation site, precisely mapping the important structural segments for sequential allosteric execution. Hence, our work demonstrates how to navigate through greater structural depths and dimensions of dynamic allosteric machineries just by leveraging standard ML methods in suitable streamlined workflows adaptive to the specific system and objectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e656737754f86ab9ba75a485691b12e7213422c" target='_blank'>
              Adaptive Workflows of Machine Learning Illuminate the Sequential Operation Mechanism of the TAK1's Allosteric Network.
              </a>
            </td>
          <td>
            Nibedita Ray Chaudhuri, Shubhra Ghosh Dastidar
          </td>
          <td>2024-05-14</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Drug allergy is responsible for a huge burden on public healthcare systems, representing in some instances a threat for patient's life. Diagnosis is complex due to the heterogeneity of clinical phenotypes and mechanisms involved, the limitations of in vitro tests, and the associated risk to in vivo tests. Predictive models, including those using recent advances in artificial intelligence, may circumvent these drawbacks, leading to an appropriate classification of patients and improving their management in clinical settings.


RECENT FINDINGS
Scores and predictive models to assess drug allergy development, including patient risk stratification, are scarce and usually apply logistic regression analysis. Over recent years, different methods encompassed under the general umbrella of artificial intelligence, including machine and deep learning, and artificial neural networks, are emerging as powerful tools to provide reliable and optimal models for clinical diagnosis, prediction, and precision medicine in different types of drug allergy.


SUMMARY
This review provides general concepts and current evidence supporting the potential utility of predictive models and artificial intelligence branches in drug allergy diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8035591f56be14f579fd4dad147b5d984378cff" target='_blank'>
              Predictive models and applicability of artificial intelligence-based approaches in drug allergy.
              </a>
            </td>
          <td>
            Rafael Núñez, Inmaculada Doña, J. Cornejo‐García
          </td>
          <td>2024-05-23</td>
          <td>Current opinion in allergy and clinical immunology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Kolmogorov-Arnold Networks (KAN) is a groundbreaking model recently proposed by the MIT team, representing a revolutionary approach with the potential to be a game-changer in the field. This innovative concept has rapidly garnered worldwide interest within the AI community. Inspired by the Kolmogorov-Arnold representation theorem, KAN utilizes spline-parametrized univariate functions in place of traditional linear weights, enabling them to dynamically learn activation patterns and significantly enhancing interpretability. In this paper, we explore the application of KAN to time series forecasting and propose two variants: T-KAN and MT-KAN. T-KAN is designed to detect concept drift within time series and can explain the nonlinear relationships between predictions and previous time steps through symbolic regression, making it highly interpretable in dynamically changing environments. MT-KAN, on the other hand, improves predictive performance by effectively uncovering and leveraging the complex relationships among variables in multivariate time series. Experiments validate the effectiveness of these approaches, demonstrating that T-KAN and MT-KAN significantly outperform traditional methods in time series forecasting tasks, not only enhancing predictive accuracy but also improving model interpretability. This research opens new avenues for adaptive forecasting models, highlighting the potential of KAN as a powerful and interpretable tool in predictive analytics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10145b2238569436754c4d9be3f9c7db501cc65c" target='_blank'>
              Kolmogorov-Arnold Networks for Time Series: Bridging Predictive Power and Interpretability
              </a>
            </td>
          <td>
            Kunpeng Xu, Lifei Chen, Shengrui Wang
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a838384985bce96bffd76cccdd013e7c0961848" target='_blank'>
              Generative deep learning for data generation in natural hazard analysis: motivations, advances, challenges, and opportunities
              </a>
            </td>
          <td>
            Zhengjing Ma, Gang Mei, N. Xu
          </td>
          <td>2024-05-30</td>
          <td>Artif. Intell. Rev.</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Microbial interactions are of fundamental importance for the functioning and the maintenance of microbial communities. Deciphering these interactions from observational data or controlled lab experiments remains a formidable challenge due to their context-dependent nature, i.e., their dependence on (a)biotic factors, host characteristics, and overall community composition. Here, we present a statistical regression framework for microbial data that allows the inclusion and parsimonious estimation of species interaction effects for an outcome of interest. We adapt the penalized quadratic interaction model to accommodate common microbial data types as predictors, including microbial presence-absence data, relative (or compositional) abundance data from microbiome surveys, and quantitative (absolute abundance) microbiome data. We study the effect of including hierarchical interaction constraints and stability-based model selection on model performance and propose novel interaction model formulations for compositional data. To illustrate our framework’s versatility, we consider prediction tasks across a wide range of microbial datasets and ecosystems, including metabolite production in model communities in designed experiments and environmental covariate prediction from marine microbiome data. While we generally observe superior predictive performance of our interaction models, we also assess limits of these models in presence of extreme data sparsity and with respect to data type. On a large-scale gut microbiome cohort data, we identify sparse family-level interaction models that accurately predict the abundance of antimicrobial resistance genes, enabling the formulation of novel biological hypotheses about microbial community interactions and antimicrobial resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f749ddea04d09c20b29e9c7150b3154d6bae9bb" target='_blank'>
              Predictive modeling of microbial data with interaction effects
              </a>
            </td>
          <td>
            Mara Stadler, Jacob Bien, Christian L. Müller
          </td>
          <td>2024-04-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="e15170 Background: A key goal in precision oncology is to develop predictive biomarkers that can identify patients likely to benefit from a given treatment. The main hurdle for generating such biomarkers is the availability of datasets that match predictive features for each patient (clinical, omics etc.) with treatment and individual outcome data. Such datasets are hard to obtain for approved drugs, and nonexistent for drugs in development. In contrast, there is ample data matching predictive features with survival data, but methods that use such data to develop and validate drug-specific predictive biomarkers are lacking. Here, we introduce a method termed the Surrogate Clinical Outcome Test (SCOT), which assesses, using non-interventional data, whether a drug-specific biomarker will be beneficial in clinical settings. Together with our previously published ENLIGHT platform, this enables early development and validation of predictive biomarkers. Methods: The SCOT test assumes that a patient population that exhibits low expression of the drug target(s) can serve as a surrogate for patients receiving the investigated drug, in the sense that survival dependency on the biomarker score would be similar to that of patients with the same features who actually receive the drug. Hence, a good predictive marker will be characterized by a negative hazard ratio in this population. To ensure that the marker is drug-specific and not merely prognostic, cases for which the same is true also in the population with high target expression should not pass the test. We studied 6 targeted drugs for which interventional clinical data is available. We used SCOT to classify two known prognostic biomarkers, as well as drug-specific transcriptomic biomarkers developed by our ENLIGHT algorithm, and compared the classifications to marker predictivities on the interventional clinical datasets. Results: Of the 6 drug-specific biomarkers developed by ENLIGHT, SCOT classified 4 as predictive of response to the respective drug, one as prognostic, and one as neither. The classification of 4 of these 6 biomarkers was confirmed by the interventional data (67% accuracy). The two prognostic markers - KI67 and Proliferation Index - were identified by SCOT as such. That is, SCOT found an association between these two markers and survival, regardless of target expression, for targets of the 6 drugs tested. Conclusions: Our work presents a novel method to test predictive markers for virtually any targeted cancer drug using big non-interventional clinical data. A key feature of this method is that it offers an independent validation of predictive markers for drugs, based on human clinical data, even before the drug enters clinical development. Combined with our ENLIGHT platform for generating predictive biomarkers in the absence of direct response data, this creates a powerful pipeline for early biomarker development and validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d395b86d0c7df368a3e80fa5afc246f75f1eec27" target='_blank'>
              A method for assessing the predictivity of drug-specific biomarkers before interventional clinical data is available.
              </a>
            </td>
          <td>
            Yaron Kinar, Rachel E. Bell, Doreen S. Ben-Zvi, Efrat Elis, Eden Maimon, T. Beker, R. Aharonov, Gal Dinstag
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Integrable partial differential equation (PDE) systems are of great interest in natural science, but are exceedingly rare and difficult to discover. To solve this, we introduce OptPDE, a first-of-its-kind machine learning approach that Optimizes PDEs' coefficients to maximize their number of conserved quantities, $n_{\rm CQ}$, and thus discover new integrable systems. We discover four families of integrable PDEs, one of which was previously known, and three of which have at least one conserved quantity but are new to the literature to the best of our knowledge. We investigate more deeply the properties of one of these novel PDE families, $u_t = (u_x+a^2u_{xxx})^3$. Our paper offers a promising schema of AI-human collaboration for integrable system discovery: machine learning generates interpretable hypotheses for possible integrable systems, which human scientists can verify and analyze, to truly close the discovery loop.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6cc89b3ade1d5ca2f3035e1d7a3f55890ce38c3" target='_blank'>
              OptPDE: Discovering Novel Integrable Systems via AI-Human Collaboration
              </a>
            </td>
          <td>
            Subhash Kantamneni, Ziming Liu, Max Tegmark
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Mathematical modeling involves representing real-world phenomena, systems, or problems using mathematical expressions and equations to analyze, understand, and predict their behavior. Given that this process typically requires experienced experts, there is an interest in exploring whether Large Language Models (LLMs) can undertake mathematical modeling to potentially decrease human labor. To evaluate of LLMs in mathematical modeling, we introduce a new benchmark, Mamo, that transcends traditional result-oriented assessments. Unlike conventional methods that primarily assess LLMs based on the accuracy of solutions to mathematical problems, our approach offers deeper insight into the modeling process itself. By focusing on the processes LLMs undertake rather than the correctness of their final solutions, Mamo pioneers a novel evaluation paradigm. This shift underscores the importance of understanding the inherent modeling capabilities of LLMs, paving the way for a more nuanced and comprehensive analysis of their problem-solving strategies. Our work marks a significant advancement in the field, suggesting a new direction for future research by emphasizing the evaluation of LLMs' modeling processes over the mere correctness of answers. This benchmark not only facilitates a better understanding of LLMs' mathematical modeling capabilities but also sets a new standard for evaluating their performance in complex problem-solving scenarios.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b2f9a406ad1488ac05f12d0857c12acc5e757d9" target='_blank'>
              Mamo: a Mathematical Modeling Benchmark with Solvers
              </a>
            </td>
          <td>
            Xuhan Huang, Qingning Shen, Yan Hu, Anningzhe Gao, Benyou Wang
          </td>
          <td>2024-05-21</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Single-cell datasets often lack individual cell labels, making it challenging to identify cells associated with disease. To address this, we introduce Mixture Modeling for Multiple Instance Learning (MMIL), an expectation maximization method that enables the training and calibration of cell-level classifiers using patient-level labels. Our approach can be used to train e.g. lasso logistic regression models, gradient boosted trees, and neural networks. When applied to clinically-annotated, primary patient samples in Acute Myeloid Leukemia (AML) and Acute Lymphoblastic Leukemia (ALL), our method accurately identifies cancer cells, generalizes across tissues and treatment timepoints, and selects biologically relevant features. In addition, MMIL is capable of incorporating cell labels into model training when they are known, providing a powerful framework for leveraging both labeled and unlabeled data simultaneously. Mixture Modeling for MIL offers a novel approach for cell classification, with significant potential to advance disease understanding and management, especially in scenarios with unknown gold-standard labels and high dimensionality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf614cea92ffef23dcd05fbea55a21a48d81f7c" target='_blank'>
              MMIL: A novel algorithm for disease associated cell type discovery
              </a>
            </td>
          <td>
            Erin Craig, Timothy Keyes, J. Sarno, Maxim E. Zaslavsky, Garry Nolan, Kara Davis, Trevor Hastie, R. Tibshirani
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>152</td>
        </tr>

        <tr id="Introduction & Objective: High-risk single-nucleotide polymorphisms (SNPs) account for only ~20% of type 2 diabetes heritability, implying islet genes greatly influence one another to alter diabetes risk. Correlative analysis does not reveal conditional dependencies between key players regulating islet function whereas machine learning (ML) may do so.
 Methods: Using data from 374 genetically diverse mice, we derived models predicting islet function from protein abundance with gradient-boosted decision tree algorithms. Some mice (70%) were used to train the models and the rest were used to validate them. We also applied our models to similar data from other mouse studies and to data from humans to see if the models’ accuracies replicated. We then determined if the models included any known islet regulators, proteins with human orthologues possessing glycemia-related SNPs, and enriched for islet-relevant functional pathways. Finally, using the proteins’ predictive influences as meta-traits, we ran quantitative trait locus (QTL) scans to identify genomic regions altering the influence of proteins on islet function.
 Results: Our analysis revealed < 150 of the ~ 5000 detected proteins sufficiently modeled any one functional trait, with > 90% of a model’s prediction influenced by < 50 proteins. Some models predicted as well or better in the other mouse sets (R > 0.6) and reasonably well in human data (R > 0.5). ML analysis correctly identified the direction of effect for candidates (e.g. DPP8) where prior studies using correlation alone failed. Models for islet traits enriched for pathways potentially relevant for islet function and several unstudied proteins (e.g. CRELD2) present in multiple models have SNPs for glycemia-related traits. Finally, QTL scans revealed genomic regions that may alter the influence of key proteins on islet function and are far from those proteins’ genomic positions.
 Conclusions: ML can be used to identify conditional dependencies regulating islet function.


 C. Emfinger: None. E. Freiberger: Employee; AbbVie Inc. M. Rabaglia: None. J. Kolic: None. S. Simonett: Employee; Exact Sciences. M. Shortreed: None. L. Clark: None. D. Stapleton: None. K. Schueler: None. K. Mitok: None. T.R. Price: None. J.J. Coon: Research Support; Agilent, Eli Lilly and Company, Merck & Co., Inc. L. Smith: None. M.J. Merrins: None. J.D. Johnson: None. M. Keller: None. A.D. Attie: None.



 American Diabetes Association (7-21-PDF-157); National Institutes of Health (R01-DK101573-06); National Institutes of Health (GM070683); National Institutes of Health (P41-GM108538); National Institutes of Health (R35GM126914); National Institutes of Health (R35-GM118110); National Institutes of Health (T32-HL007936); National Institutes of Health (R01-DK113103); National Institutes of Health (R01DK127637); Canadian Institutes for Health Research (CIHR) operating grant 168857 CIHR Team Grant (ASD-179092/5-SRA-2021-1149-S-B); CIHR-JDRF Team (ASD-173663/5-SRA-2020-1059-S-B); CIHR Banting fellowship United States Department of Veterans Affairs Biomedical Laboratory Research and Development Service (I01BX005113)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae9b0cfbbd675841b7923b5ba1ab6fcfd03c60f2" target='_blank'>
              1483-P: Identifying Conditional Dependence of Proteins Regulating Islet Biology with Machine Learning
              </a>
            </td>
          <td>
            Christopher H Emfinger, Elyse C. Freiberger, Mary R. Rabaglia, Jelena Kolic, Shane P. Simonett, Michael Shortreed, Lauren E. Clark, Donnie Stapleton, Kathryn Schueler, Kelly A. Mitok, Tara R. Price, Joshua J. Coon, Lloyd Smith, M. Merrins, James D. Johnson, Mark Keller, A. Attie
          </td>
          <td>2024-06-14</td>
          <td>Diabetes</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/893c7aed1d984e0b366eef711ea7794ffd56e69b" target='_blank'>
              Predicting disease recurrence in breast cancer patients using machine learning models with clinical and radiomic characteristics: a retrospective study.
              </a>
            </td>
          <td>
            Saadia Azeroual, Fatima-Ezzahraa Ben-Bouazza, Amine Naqi, Rajaa Sebihi
          </td>
          <td>2024-06-10</td>
          <td>Journal of the Egyptian National Cancer Institute</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/480567bb55e580af5f22e324481c693a5881a9b8" target='_blank'>
              Unveiling gene regulatory networks during cellular state transitions without linkage across time points
              </a>
            </td>
          <td>
            Ruosi Wan, Yuhao Zhang, Yongli Peng, Feng Tian, Ge Gao, Fuchou Tang, Jinzhu Jia, Hao Ge
          </td>
          <td>2024-05-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
Machine learning (ML) has been harnessed as a promising modelling tool for materials research. However, small data, or data scarcity, is a bottleneck when incorporating ML in studies involving experimentation. Current experiment planning methods show several disadvantages: one‐factor‐at‐a‐time (OFAT) experimentation became impractical due to limited laboratory resources; conventional design of experiments (DoE) failed to incorporate high‐dimensional features in ML; Surrogate‐based or Bayesian optimization (BO) shifted the goal to optimize material properties rather than guiding training data accumulation. The present research proposes leveraging active learning (AL) to strategically select critical data for experimentation. Two AL strategies, query‐by‐Committee (QBC) algorithm and Greedy method, are benchmarked against random query baseline on various materials datasets. AL is shown to efficiently reduce model prediction errors with minimal additional experiment data. Investigation of hyperparameters revealed benefits of applying AL at an early stage of experimental dataset construction. Moreover, AL is implemented and validated for an in‐house materials development task ‐ electrospray modelling. AL exploration as a paradigm is highlighted to guide experiment design for efficient data accumulation purposes, and its potential for further ML modelling. In doing so, the power of ML is expected to be fully unleashed to experimental researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41751ad3a09025a836bdcc2a49f5c274ea5e9855" target='_blank'>
              Tackling Data Scarcity Challenge through Active Learning in Materials Processing with Electrospray
              </a>
            </td>
          <td>
            Fanjin Wang, A. Harker, Mohan Edirisinghe, M. Parhizkar
          </td>
          <td>2024-05-23</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Modelling gene expression is a central problem in systems biology. Single-cell technologies have revolutionised the field by enabling sequencing at the resolution of individual cells. This results in a much richer data compared to what is obtained by bulk technologies, offering new possibilities and challenges for gene regulatory network inference. Here we introduce GRIT — a method to fit a differential equation model and to infer gene regulatory networks from single-cell data using the theory of optimal transport. The idea consists in tracking the evolution of the cell distribution over time and finding the system that minimises the transport cost between consecutive time points.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0edd4d732cd966d7162a3543e7c8bfa54031593f" target='_blank'>
              Gene regulatory network inference from single-cell data using optimal transport
              </a>
            </td>
          <td>
            François Lamoline, Isabel Haasler, Johan Karlsson, Jorge Gonçalves, A. Aalto
          </td>
          <td>2024-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The differentiable programming paradigm is a cornerstone of modern scientific computing. It refers to numerical methods for computing the gradient of a numerical model's output. Many scientific models are based on differential equations, where differentiable programming plays a crucial role in calculating model sensitivities, inverting model parameters, and training hybrid models that combine differential equations with data-driven approaches. Furthermore, recognizing the strong synergies between inverse methods and machine learning offers the opportunity to establish a coherent framework applicable to both fields. Differentiating functions based on the numerical solution of differential equations is non-trivial. Numerous methods based on a wide variety of paradigms have been proposed in the literature, each with pros and cons specific to the type of problem investigated. Here, we provide a comprehensive review of existing techniques to compute derivatives of numerical solutions of differential equations. We first discuss the importance of gradients of solutions of differential equations in a variety of scientific domains. Second, we lay out the mathematical foundations of the various approaches and compare them with each other. Third, we cover the computational considerations and explore the solutions available in modern scientific software. Last but not least, we provide best-practices and recommendations for practitioners. We hope that this work accelerates the fusion of scientific models and data, and fosters a modern approach to scientific modelling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef6261d72acb3b8f584e24594af1af686836c8" target='_blank'>
              Differentiable Programming for Differential Equations: A Review
              </a>
            </td>
          <td>
            Facundo Sapienza, Jordi Bolibar, Frank Schafer, Brian Groenke, Avik Pal, Victor Boussange, Patrick Heimbach, Giles Hooker, Fernando P'erez, Per-Olof Persson, Christopher Rackauckas
          </td>
          <td>2024-06-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Diagnosis and prognosis are especially difficult areas of medical research related to cancer due to the high incidence of breast cancer, which has surpassed all other cancers in terms of female mortality. Another factor that has a substantial influence on the quality of life of cancer patients is the fear that they may experience a relapse of their disease. The objective of the study is to give medical practitioners a more effective strategy for using ensemble learning techniques to forecast when breast cancer may recur. This research aimed to investigate the usage of deep neural networks (DNNs) and artificial neural networks (ANNs) in addition to machine learning (ML) based approaches, including bagging, averaging, and voting, to enhance the efficacy of breast cancer relapse diagnosis on two breast cancer relapse datasets. Results from the empirical study demonstrate that the proposed ensemble learning-enabled approach improves accuracies by 96.31% and 95.81%, precisions by 96.70% and 96.15%, sensitivities by 98.88% and 98.68%, specificities by 84.62% in both, F1-scores by 97.78% and 97.40%, and area under the curve (AUCs) of 0.987 and 0.978, with University Medical Centre, Institute of Oncology (UMCIO) and Wisconsin prognostic breast cancer (WPBC) datasets respectively. Consequently, these improved disease outcomes may encourage physicians to use this model to make better treatment choices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cda0380ec84db5b7b900c42f0f27833d1ab99f3" target='_blank'>
              Breast cancer relapse disease prediction improvements with ensemble learning approaches
              </a>
            </td>
          <td>
            Ghanashyam Sahoo, Ajit Kumar Nayak, Pradyumna Kumar Tripathy, Abhilash Pati, Amrutanshu Panigrahi, Adyasha Rath, Bhimasen Moharana
          </td>
          <td>2024-07-01</td>
          <td>Indonesian Journal of Electrical Engineering and Computer Science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Certain cancer types, namely pancreatic cancer is difficult to detect at an early stage; sparking the importance of discovering the causal relationship between biomarkers and cancer to identify cancer efficiently. By allowing for the detection and monitoring of specific biomarkers through a non-invasive method, liquid biopsies enhance the precision and efficacy of medical interventions, advocating the move towards personalized healthcare. Several machine learning algorithms such as Random Forest, SVM are utilized for classification, yet causing inefficiency due to the need for conducting hyperparameter tuning. We leverage a meta-trained Hyperfast model for classifying cancer, accomplishing the highest AUC of 0.9929 and simultaneously achieving robustness especially on highly imbalanced datasets compared to other ML algorithms in several binary classification tasks (e.g. breast invasive carcinoma; BRCA vs. non-BRCA). We also propose a novel ensemble model combining pre-trained Hyperfast model, XGBoost, and LightGBM for multi-class classification tasks, achieving an incremental increase in accuracy (0.9464) while merely using 500 PCA features; distinguishable from previous studies where they used more than 2,000 features for similar results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c9894bd9c35a06d4073a4397d4705474cc1e65b" target='_blank'>
              Biomarker based Cancer Classification using an Ensemble with Pre-trained Models
              </a>
            </td>
          <td>
            Chongmin Lee, Jihie Kim
          </td>
          <td>2024-06-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="We investigate trends in the data-error scaling behavior of machine learning (ML) models trained on discrete combinatorial spaces that are prone-to-mutation, such as proteins or organic small molecules. We trained and evaluated kernel ridge regression machines using variable amounts of computationally generated training data. Our synthetic datasets comprise i) two na\"ive functions based on many-body theory; ii) binding energy estimates between a protein and a mutagenised peptide; and iii) solvation energies of two 6-heavy atom structural graphs. In contrast to typical data-error scaling, our results showed discontinuous monotonic phase transitions during learning, observed as rapid drops in the test error at particular thresholds of training data. We observed two learning regimes, which we call saturated and asymptotic decay, and found that they are conditioned by the level of complexity (i.e. number of mutations) enclosed in the training set. We show that during training on this class of problems, the predictions were clustered by the ML models employed in the calibration plots. Furthermore, we present an alternative strategy to normalize learning curves (LCs) and the concept of mutant based shuffling. This work has implications for machine learning on mutagenisable discrete spaces such as chemical properties or protein phenotype prediction, and improves basic understanding of concepts in statistical learning theory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ffa62d0de0dd69c6cbdb95f7d32cb464245d35" target='_blank'>
              Data-Error Scaling in Machine Learning on Natural Discrete Combinatorial Mutation-prone Sets: Case Studies on Peptides and Small Molecules
              </a>
            </td>
          <td>
            Vanni Doffini, O. V. Lilienfeld, Michael A. Nash
          </td>
          <td>2024-05-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="In various biological systems, analyzing how cell behaviors are coordinated over time would enable a deeper understanding of tissue-scale response to physiologic or superphysiologic stimuli. Such data is necessary for establishing both normal tissue function and the sequence of events after injury that lead to chronic disease. However, collecting and analyzing these large datasets presents a challenge—such systems are time-consuming to process, and the overwhelming scale of data makes it difficult to parse overall behaviors. This problem calls for an analysis technique that can quickly provide an overview of the groups present in the entire system and also produce meaningful categorization of cell behaviors. Here, we demonstrate the application of an unsupervised method—the Variational Autoencoder (VAE)—to learn the features of cells in cartilage tissue after impact-induced injury and identify meaningful clusters of chondrocyte behavior. This technique quickly generated new insights into the spatial distribution of specific cell behavior phenotypes and connected specific peracute calcium signaling timeseries with long term cellular outcomes, demonstrating the value of the VAE technique.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b92339847fe62e1bb5e9e8b25a050f9e1f85707" target='_blank'>
              Application of a variational autoencoder for clustering and analyzing in situ articular cartilage cellular response to mechanical stimuli
              </a>
            </td>
          <td>
            Jingyang Zheng, Han Kheng Teoh, M. Delco, L. Bonassar, I. Cohen
          </td>
          <td>2024-05-20</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Active learning optimizes the exploration of large parameter spaces by strategically selecting which experiments or simulations to conduct, thus reducing resource consumption and potentially accelerating scientific discovery. A key component of this approach is a probabilistic surrogate model, typically a Gaussian Process (GP), which approximates an unknown functional relationship between control parameters and a target property. However, conventional GPs often struggle when applied to systems with discontinuities and non-stationarities, prompting the exploration of alternative models. This limitation becomes particularly relevant in physical science problems, which are often characterized by abrupt transitions between different system states and rapid changes in physical property behavior. Fully Bayesian Neural Networks (FBNNs) serve as a promising substitute, treating all neural network weights probabilistically and leveraging advanced Markov Chain Monte Carlo techniques for direct sampling from the posterior distribution. This approach enables FBNNs to provide reliable predictive distributions, crucial for making informed decisions under uncertainty in the active learning setting. Although traditionally considered too computationally expensive for 'big data' applications, many physical sciences problems involve small amounts of data in relatively low-dimensional parameter spaces. Here, we assess the suitability and performance of FBNNs with the No-U-Turn Sampler for active learning tasks in the 'small data' regime, highlighting their potential to enhance predictive accuracy and reliability on test functions relevant to problems in physical sciences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebe4e8caad7fd908989b7e37a05fb880b373a0e4" target='_blank'>
              Active Learning with Fully Bayesian Neural Networks for Discontinuous and Nonstationary Data
              </a>
            </td>
          <td>
            Maxim Ziatdinov
          </td>
          <td>2024-05-16</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Preventing, diagnosing, and treating diseases requires accurate clinical biomarkers, which remains challenging. Recently, advanced computational approaches have accelerated the discovery of promising biomarkers from high-dimensional multimodal data. Although machine-learning methods have greatly contributed to the research fields, handling data sparseness, which is not unusual in research settings, is still an issue as it leads to limited interpretability and performance in the presence of missing information. Here, we propose a novel pipeline integrating joint non-negative matrix factorization (JNMF), identifying key features within sparse high-dimensional heterogeneous data, and a biological pathway analysis, interpreting the functionality of features by detecting activated signaling pathways. By applying our pipeline to large-scale public cancer datasets, we identified sets of genomic features relevant to specific cancer types as common pattern modules (CPMs) of JNMF. We further detected COPS5 as a potential upstream regulator of pathways associated with diffuse large B-cell lymphoma (DLBCL). COPS5 exhibited co-overexpression with MYC, TP53, and BCL2, known DLBCL marker genes, and its high expression was correlated with a lower survival probability of DLBCL patients. Using the CRISPR-Cas9 system, we confirmed the tumor growth effect of COPS5, which suggests it as a novel prognostic biomarker for DLBCL. Our results highlight that integrating multiple high-dimensional data and effectively decomposing them to interpretable dimensions unravels hidden biological importance, which enhances the discovery of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ab81c7bf9d4d54c741aeecc346cc29faef5638f" target='_blank'>
              Integrative analysis of cancer multimodality data identifying COPS5 as a novel biomarker of diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Yutong Dai, Jingmei Li, Keita Yamamoto, Susumu Goyama, Martin Loza, , Kenta Nakai
          </td>
          <td>2024-06-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The field of computer-aided synthesis planning (CASP) has witnessed significant growth in recent years. Still, many CASP programs rely on large data sets to train neural networks, resulting in limitations due to the data quality and prior knowledge from chemists. In response, we propose Retrosynthesis Zero (ReSynZ), a reaction template-based method that combines Monte Carlo Tree Search with reinforcement learning inspired by AlphaGo Zero. Unlike other single-step reaction template-based CASP methods, ReSynZ takes complete synthesis paths for complex molecules, determined by reaction rules, as input for training the neural network. ReSynZ enables neural networks trained with relatively small reaction data sets (tens of thousands of data) to generate multiple synthesis pathways for a target molecule and suggest possible reaction conditions. On multiple data sets of molecular retrosynthesis, ReSynZ demonstrates excellent predictive performance compared to existing algorithms. The advantages, such as self-improving model features, flexible reward settings, the potential to surpass human limitations in chemical synthesis route planning, and others, make ReSynZ a valuable tool in chemical synthesis design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/057818f87aa2d7015f09a417ae24db26de33cce9" target='_blank'>
              Retrosynthesis Zero: Self-Improving Global Synthesis Planning Using Reinforcement Learning.
              </a>
            </td>
          <td>
            Jiasheng Guo, Chenning Yu, Kenan Li, Yijian Zhang, Guoqiang Wang, Shuhua Li, Hao Dong
          </td>
          <td>2024-05-15</td>
          <td>Journal of chemical theory and computation</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34c5808381cdbd9cb872a993495afdf00d6e6a33" target='_blank'>
              Note on importance of correct stoichiometric assumptions for modeling of monoclonal antibodies.
              </a>
            </td>
          <td>
            L. Gibiansky, E. Gibiansky
          </td>
          <td>2024-05-03</td>
          <td>Journal of pharmacokinetics and pharmacodynamics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Motivated by the pressing challenges in the digital twin development for biomanufacturing process, we introduce an adjoint sensitivity analysis (SA) approach to expedite the learning of mechanistic model parameters. In this paper, we consider enzymatic stochastic reaction networks representing a multi-scale bioprocess mechanistic model that allows us to integrate disparate data from diverse production processes and leverage the information from existing macro-kinetic and genome-scale models. To support forward prediction and backward reasoning, we develop a convergent adjoint SA algorithm studying how the perturbations of model parameters and inputs (e.g., initial state) propagate through enzymatic reaction networks and impact on output trajectory predictions. This SA can provide a sample efficient and interpretable way to assess the sensitivities between inputs and outputs accounting for their causal dependencies. Our empirical study underscores the resilience of these sensitivities and illuminates a deeper comprehension of the regulatory mechanisms behind bioprocess through sensitivities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa83caf636ea9a9577cbd13662fa8d1e926e34f" target='_blank'>
              Adjoint Sensitivity Analysis on Multi-Scale Bioprocess Stochastic Reaction Network
              </a>
            </td>
          <td>
            Keilung Choy, Wei Xie
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="10043 Background: Prediction, understanding, and management, of neuroblastoma (NBL) outcomes, from spontaneous regression to relapse and death, remain limited, and rely mostly on the International Neuroblastoma Staging System (INSS) stage, age, and the one-gene test for MYCN amplification. The entire multi-ome affects every aspect of the disease. But typical artificial intelligence and machine learning (AI/ML) cannot discover effective biomarkers from real-life, small-cohort, multi-omic, and noisy data. Methods: We use our AI/ML to identify two whole-genome biomarkers from open-source pre-treatment NBL patient-matched primary tumor and whole blood DNA copy numbers, and primary tumor mRNA expression. Our data-agnostic, unsupervised algorithms extend the mathematics that underlies quantum mechanics to overcome the limitations of typical AI/ML [1,2]. Our platform- and reference genome-agnostic biomarkers in astrocytoma, including glioblastoma, and lung, ovarian, and uterine adenocarcinomas, were mathematically discovered, and computationally and experimentally validated, repeatedly, in federated, imbalanced datasets from as few as 50–100 patients [3]. Results: We discover the biomarkers in “skinny” datasets of minimally preprocessed ≈3M-bin whole-genome sequences from tumors and blood of a set of 101 patients. Blindly, i.e., label-free, we separate both biomarkers from the normal X chromosome-number variation, demonstrating sex-agnostic learning. Combined, the biomarkers are statistically independent of all standard-of-care indicators, with the univariate Cox hazard ratio of 4.0 (log-rank P-value=2.3×10–5) within the 95% confidence intervals of the bivariate ratios. The risk that the tumor’s whole genome confers upon outcome, as is reflected by the bivariate ratios, is greater than that conferred by all standard-of-care indicators, including DNA ploidy, the Children’s Oncology Group (COG) risk, and the mitosis-karyorrhexis index (MKI), except for histopathology. We validate the biomarkers in ≈10K-bin target-capture sequences of a mutually-exclusive set of 419 patients, demonstrating generalizability as well as site-, platform-, and protocol-agnostic transfer learning. At 73–80% concordance, in both the tumor and blood profiles of both the discovery and validation sets, the biomarkers combined are more accurate than MYCN. We show that the biomarkers identify known and previously unrecognized disease mechanisms and druggable gene alterations, including co-amplification of MYCN with genes encoding for extra-embryonic transcripts, as well as the hijacking of embryonic development toward aneuploidy, which can spontaneously regress via embryonic self-correction. Conclusions: Our AI/ML is uniquely suited for personalized medicine. 1. doi: 10.1063/1.5099268 2. doi: 10.1073/pnas.0530258100 3. doi: 10.1063/1.5142559.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0234e4be40726e2547947696f74629005c40508" target='_blank'>
              AI/ML-derived mechanistically interpretable whole-genome biomarkers of patient survival in pre-treatment primary neuroblastoma tumors and whole blood.
              </a>
            </td>
          <td>
            Orly Alter, Elizabeth Newman, Sri Priya Ponnapalli, Jessica W. Tsai
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Functional data analysis, which models data as realizations of random functions over a continuum, has emerged as a useful tool for time series data. Often, the goal is to infer the dynamic connections (or time-varying conditional dependencies) among multiple functions or time series. For this task, we propose a dynamic and Bayesian functional graphical model. Our modeling approach prioritizes the careful definition of an appropriate graph to identify both time-invariant and time-varying connectivity patterns. We introduce a novel block-structured sparsity prior paired with a finite basis expansion, which together yield effective shrinkage and graph selection with efficient computations via a Gibbs sampling algorithm. Crucially, the model includes (one or more) graph changepoints, which are learned jointly with all model parameters and incorporate graph dynamics. Simulation studies demonstrate excellent graph selection capabilities, with significant improvements over competing methods. We apply the proposed approach to study of dynamic connectivity patterns of sea surface temperatures in the Pacific Ocean and discovers meaningful edges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5cc6623b051c0f56b325829f2faaeed65f2da4" target='_blank'>
              Bayesian Functional Graphical Models with Change-Point Detection
              </a>
            </td>
          <td>
            Chunshan Liu, Daniel R. Kowal, James Doss-Gollin, Marina Vannucci
          </td>
          <td>2024-05-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="INTRODUCTION
Well-calibrated models for personalized prognostication of patients with gastrointestinal neuroendocrine neoplasms (GINENs) are limited. This study aimed to develop and validate a machine-learning model to predict the survival of patients with GINENs.


METHODS
oblique random survival forest (ORSF) model, Cox proportional hazard risk model, Cox model with least absolute shrinkage and selection operator penalization, CoxBoost, Survival Gradient Boosting Machine, Extreme Gradient Boosting survival regression, DeepHit, DeepSurv, DNNSurv, Logistic-Hazard model, and PC-Hazard model were compared. We further tuned hyperparameters and selected variables for the best-performing ORSF. Then, the final ORSF model was validated.


RESULTS
43444 patients with GINENs were included. The median (interquartile range) survival time was 53 (19-102) months. The ORSF model performed best, in which age, histology, M stage, tumor size, primary tumor site, sex, tumor number, surgery, lymph nodes removed, N stage, race, and grade were ranked as important variables. However, chemotherapy and radiotherapy were not necessary for the ORSF model. The ORSF model had an overall C-index of 0.86 (95% confidence interval, 0.85-0.87). The area under the receiver operation curves at 1-, 3-, 5-, and 10-year were 0.91, 0.89, 0.87, and 0.80, respectively. The decision curve analysis showed superior clinical usefulness of the ORSF model than the American Joint Committee on Cancer Stage. A nomogram and an online tool were given.


CONCLUSION
The machine-learning ORSF model could precisely predict the survival of patients with GINENs, with the ability to identify patients at high risk for death and probably guide clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cf8f29fb5dcb8ce9dcb380c08e117a84d6d2132" target='_blank'>
              Development and Validation of a Novel Machine Learning Model to predict the Survival of Patients with Gastrointestinal Neuroendocrine Neoplasms.
              </a>
            </td>
          <td>
            Si Liu, Yun-Xiang Chen, Bing Dai, Li Chen
          </td>
          <td>2024-05-06</td>
          <td>Neuroendocrinology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Dynamic processes in complex networks are crucial for better understanding collective behavior in human societies, biological systems, and the internet. In this paper, we first focus on the continuous Markov-based modeling of evolving networks with the birth-death of individuals. A new individual arrives at the group by the Poisson process, while new links are established in the network through either uniform connection or preferential attachment. Moreover, an existing individual has a limited lifespan before leaving the network. We determine stationary topological properties of these networks, including their size and mean degree. To address the effect of the birth-death evolution, we further study the information dynamics in the proposed network model from the random drift and natural selection perspective, based on assumptions of total-stochastic and fitness-driven evolution, respectively. In simulations, we analyze the fixation probability of individual information and find that means of new connections affect the random drift process but do not affect the natural selection process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cc97d3b6a2007d5cad5a44d3da12a6a5c9c0d40" target='_blank'>
              Information Dynamics in Evolving Networks Based on the Birth-Death Process: Random Drift and Natural Selection Perspective
              </a>
            </td>
          <td>
            Minyu Feng, Ziyan Zeng, Qin Li, M. Perc, Jürgen Kurths
          </td>
          <td>2024-05-29</td>
          <td>IEEE Transactions on Systems, Man, and Cybernetics: Systems</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="5065 Background: Overall survival (OS) remains the gold standard endpoint for clinical trials in metastatic prostate cancer (metPC) yet requires extensive follow-up. We sought to develop a model utilizing short term (≤ 4 months) prostate specific antigen (PSA) kinetic data to predict the OS readout from phase 3 clinical trials, with the goal of accelerating readout time of trials for patients (pts) with metPC. Methods: Clinical and PSA data were obtained from 7 metPC trials: 6 randomized, double-blind, phase 3 trials (TITAN, COU-AA-301, COU-AA-302, LATITUDE, ACIS, and MAGNITUDE), and one multicenter, phase II trial (GALAHAD). We developed 18 PSA kinetic variables from the first 4 months of enrollment including: 50% decline in PSA (PSA50), 90% decline in PSA, PSA=0.1, and PSA=0.2 within 1, 2, 3, and 4 months; and slope and base of the exponential fit of PSA over the first 4 months on study). To balance pt and disease characteristics, TITAN, COU-AA-301, and ACIS were used to train the intermediate endpoint, with remaining studies held out as an external testing cohort. A random 60/40% split of the pooled pt-level data was performed as a sensitivity analysis. Adaptive least absolute shrinkage and selection operator (aLASSO)-based Cox proportional hazards models were applied to select previously identified prognostic baseline clinical variables with and without PSA kinetic variables most predictive of OS on five-fold cross validation. Performance with and without PSA kinetics was assessed using C-index, integrated Brier score (IBS), and time-dependent area under the receiver operating characteristic curve (tAUC) at 12, 24, 36, and 48 months. Results: The study included 6,451 pts with median follow-up of 22 months and 85,785 PSA values. aLASSO selected Eastern Cooperative Oncology Group performance status, disease extent, lactate dehydrogenase, albumin, hemoglobin, PSA, and alkaline phosphatase as baseline variables for the prognostic comparator model. In the model including PSA kinetics, aLASSO identified PSA50 at 4 months, PSA0.1 at 1 month, and PSA slope as kinetics providing additional intra-treatment information to improve prediction of OS. The model with PSA kinetics showed improved performance on the test trials: C-index (0.72 vs. 0.66), IBS (0.158 vs. 0.173), and tAUC (0.84, 0.78, 0.78, and 0.76 vs. 0.78, 0.71, 0.69, and 0.65 at times 12, 24, 36, and 48 months, respectively; p < 0.05.) Sensitivity analysis using a random split of the pooled pt-level data supported the importance of PSA50 at 4 months and PSA slope. Conclusions: Data from 6,451 mPC pts enrolled on prospective clinical trials were used to develop a machine learning model which included kinetic PSA data and predicted the long-term OS readout in Phase 3 trials with just the first four months of data. This model will be validated in independent data sets comprised of other completed phase 3 trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dac296b786d21d2a4c231f4e0d52d3f32e79335" target='_blank'>
              Development of a machine learning model to predict overall survival results from randomized clinical trials of patients with metastatic prostate cancer.
              </a>
            </td>
          <td>
            A. Sabbagh, David A Quigley, Nicholas Lillis, Li Zhang, I. Friesner, A. Bailey, Rahul R Aggarwal, Meera Reddy Chappidi, Hari Singhal, Ke Zhang, Joel Greshock, Jinhui Li, Margaret K. Yu, Eric J. Small, Julian C. Hong
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="We propose a Bayesian framework for fine-tuning large diffusion models with a novel network structure called Bayesian Power Steering (BPS). We clarify the meaning behind adaptation from a \textit{large probability space} to a \textit{small probability space} and explore the task of fine-tuning pre-trained models using learnable modules from a Bayesian perspective. BPS extracts task-specific knowledge from a pre-trained model's learned prior distribution. It efficiently leverages large diffusion models, differentially intervening different hidden features with a head-heavy and foot-light configuration. Experiments highlight the superiority of BPS over contemporary methods across a range of tasks even with limited amount of data. Notably, BPS attains an FID score of 10.49 under the sketch condition on the COCO17 dataset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74b8bbcef89c39d55694067d089765b8f6efd30" target='_blank'>
              Bayesian Power Steering: An Effective Approach for Domain Adaptation of Diffusion Models
              </a>
            </td>
          <td>
            Ding Huang, Ting Li, Jian Huang
          </td>
          <td>2024-06-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Developing and optimizing models for complex systems poses challenges due to the inherent complexity introduced by multiple types of input information and sources of uncertainty. In this study, we utilize Bayesian formalism to analytically examine the propagation of probability in the modeling process and propose quantitative assessments for it. Upon which, we develop a method for optimizing models for complex systems by (i) minimizing model uncertainty; (ii) maximizing model consistency; and (iii) minimizing model complexity, following the Bayesian Occam’s razor rationale. We showcase the benefits of this method by optimizing the modeling of the dynamic system of glucose-stimulated insulin secretion in pancreatic β-cells, leading to an optimized model that demonstrates better alignment with experimental observations compared to the non-optimized one. We anticipate that this method will facilitate the construction of accurate, precise, and sufficiently simple models for diverse complex systems. It is implemented in our open-source software Integrative Modeling Platform (IMP), ensuring its broad applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2852aa1c64c573614d3968195a3e4de98331119b" target='_blank'>
              Bayesian Occam’s Razor to Optimize Models for Complex Systems
              </a>
            </td>
          <td>
            Chenxi Wang, Jihui Zhao, Jingjing Zheng, Barak Raveh, Xuming He, Liping Sun
          </td>
          <td>2024-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="A deep latent variable model is a powerful method for capturing complex distributions. These models assume that underlying structures, but unobserved, are present within the data. In this dissertation, we explore high-dimensional problems related to physiological monitoring using latent variable models. First, we present a novel deep state-space model to generate electrical waveforms of the heart using optically obtained signals as inputs. This can bring about clinical diagnoses of heart disease via simple assessment through wearable devices. Second, we present a brain signal modeling scheme that combines the strengths of probabilistic graphical models and deep adversarial learning. The structured representations can provide interpretability and encode inductive biases to reduce the data complexity of neural oscillations. The efficacy of the learned representations is further studied in epilepsy seizure detection formulated as an unsupervised learning problem. Third, we propose a framework for the joint modeling of physiological measures and behavior. Existing methods to combine multiple sources of brain data provided are limited. Direct analysis of the relationship between different types of physiological measures usually does not involve behavioral data. Our method can identify the unique and shared contributions of brain regions to behavior and can be used to discover new functions of brain regions. The success of these innovative computational methods would allow the translation of biomarker findings across species and provide insight into neurocognitive analysis in numerous biological studies and clinical diagnoses, as well as emerging consumer applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf07061c3b5a4943a43f226d9cab1e727639eec0" target='_blank'>
              Deep Latent Variable Modeling of Physiological Signals
              </a>
            </td>
          <td>
            Khuong Vo
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Antimicrobial peptides (AMPs) are promising candidates for new antibiotics due to their broad-spectrum activity against pathogens and reduced susceptibility to resistance development. Deep-learning techniques, such as deep generative models, offer a promising avenue to expedite the discovery and optimization of AMPs. A remarkable example is the Feedback Generative Adversarial Network (FBGAN), a deep generative model that incorporates a classifier during its training phase. Our study aims to explore the impact of enhanced classifiers on the generative capabilities of FBGAN. To this end, we introduce two alternative classifiers for the FBGAN framework, both surpassing the accuracy of the original classifier. The first classifier utilizes the k-mers technique, while the second applies transfer learning from the large protein language model Evolutionary Scale Modeling 2 (ESM2). Integrating these classifiers into FBGAN not only yields notable performance enhancements compared to the original FBGAN but also enables the proposed generative models to achieve comparable or even superior performance to established methods such as AMPGAN and HydrAMP. This achievement underscores the effectiveness of leveraging advanced classifiers within the FBGAN framework, enhancing its computational robustness for AMP de novo design and making it comparable to existing literature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49abe4fa28324be639267113afb2f291680a13fb" target='_blank'>
              De Novo Antimicrobial Peptide Design with Feedback Generative Adversarial Networks
              </a>
            </td>
          <td>
            M. Zervou, E. Doutsi, Yannis Pantazis, P. Tsakalides
          </td>
          <td>2024-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d15c9cee965609e63274b374f2806434839e18" target='_blank'>
              Reverse engineering morphogenesis through Bayesian optimization of physics-based models
              </a>
            </td>
          <td>
            Nilay Kumar, M. Mim, Alexander Dowling, Jeremiah J Zartman
          </td>
          <td>2024-05-07</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Antibodies play a crucial role in adaptive immune responses by determining B cell specificity to antigens and focusing immune function on target pathogens. Accurate prediction of antibody-antigen specificity directly from antibody sequencing data would be a great aid in understanding immune responses, guiding vaccine design, and developing antibody-based therapeutics. In this study, we present a method of supervised fine-tuning for antibody language models, which improves on previous results in binding specificity prediction to SARS-CoV-2 spike protein and influenza hemagglutinin. We perform supervised fine-tuning on four pre-trained antibody language models to predict specificity to these antigens and demonstrate that fine-tuned language model classifiers exhibit enhanced predictive accuracy compared to classifiers trained on pretrained model embeddings. The change of model attention activations after supervised fine-tuning suggested that this performance was driven by an increased model focus on the complementarity determining regions (CDRs). Application of the supervised fine-tuned models to BCR repertoire data demonstrated that these models could recognize the specific responses elicited by influenza and SARS-CoV-2 vaccination. Overall, our study highlights the benefits of supervised fine-tuning on pre-trained antibody language models as a mechanism to improve antigen specificity prediction. Author Summary Antibodies are vigilant sentinels of our adaptive immune system that recognize and bind to targets on foreign pathogens, known as antigens. This interaction between antibody and antigen is highly specific, akin to a fitting lock and key mechanism, to ensure each antibody precisely targets its intended antigen. Recent advancements in language modeling have led to the development of antibody language model to decode specificity information in the sequences of antibodies. We introduce a method based on supervised fine-tuning, which enhances the accuracy of antibody language models in predicting antibody-antigen interactions. By training these models on large datasets of antibody sequences, we can better predict which antibodies will bind to important antigens such as those found on the surface of viruses like SARS-CoV-2 and influenza. Moreover, our study demonstrates the potential of the models to “read” B cell repertoire data and predict ongoing responses, offering new insights into how our bodies respond to vaccination. These findings have significant implications for vaccine design, as accurate prediction of antibody specificity can guide the development of more effective vaccines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ddef4b9150873a66bf594c429f74b69d5535877" target='_blank'>
              Supervised fine-tuning of pre-trained antibody language models improves antigen specificity prediction
              </a>
            </td>
          <td>
            Meng Wang, J. Patsenker, Henry Li, Y. Kluger, S. Kleinstein
          </td>
          <td>2024-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Diffusion models, a specific type of generative model, have achieved unprecedented performance in recent years and consistently produce high-quality synthetic samples. A critical prerequisite for their notable success lies in the presence of a substantial number of training samples, which can be impractical in real-world applications due to high collection costs or associated risks. Consequently, various finetuning and regularization approaches have been proposed to transfer knowledge from existing pre-trained models to specific target domains with limited data. This paper introduces the Transfer Guided Diffusion Process (TGDP), a novel approach distinct from conventional finetuning and regularization methods. We prove that the optimal diffusion model for the target domain integrates pre-trained diffusion models on the source domain with additional guidance from a domain classifier. We further extend TGDP to a conditional version for modeling the joint distribution of data and its corresponding labels, together with two additional regularization terms to enhance the model performance. We validate the effectiveness of TGDP on Gaussian mixture simulations and on real electrocardiogram (ECG) datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eda00750b12cb38d2204b15aec8ce9fecfb3fe1" target='_blank'>
              Transfer Learning for Diffusion Models
              </a>
            </td>
          <td>
            Yidong Ouyang, Liyan Xie, Hongyuan Zha, Guang Cheng
          </td>
          <td>2024-05-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6155f5d70357f51adefc8adbbf3af90792e5537c" target='_blank'>
              SEEI: spherical evolution with feedback mechanism for identifying epistatic interactions
              </a>
            </td>
          <td>
            Deyu Tang, Yi-jun Mao, Jie Zhao, Jin Yang, Shiyin Li, Fu-xiang Ren, Junxi Zheng
          </td>
          <td>2024-05-13</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Large Language Models (LLMs) have achieved significant progress across various fields and have exhibited strong potential in evolutionary computation, such as generating new solutions and automating algorithm design. Surrogate-assisted selection is a core step in evolutionary algorithms to solve expensive optimization problems by reducing the number of real evaluations. Traditionally, this has relied on conventional machine learning methods, leveraging historical evaluated evaluations to predict the performance of new solutions. In this work, we propose a novel surrogate model based purely on LLM inference capabilities, eliminating the need for training. Specifically, we formulate model-assisted selection as a classification and regression problem, utilizing LLMs to directly evaluate the quality of new solutions based on historical data. This involves predicting whether a solution is good or bad, or approximating its value. This approach is then integrated into evolutionary algorithms, termed LLM-assisted EA (LAEA). Detailed experiments compared the visualization results of 2D data from 9 mainstream LLMs, as well as their performance on optimization problems. The experimental results demonstrate that LLMs have significant potential as surrogate models in evolutionary computation, achieving performance comparable to traditional surrogate models only using inference. This work offers new insights into the application of LLMs in evolutionary computation. Code is available at: https://github.com/hhyqhh/LAEA.git">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48979beaa83197e5e9c83fe403afff8c2e917941" target='_blank'>
              Large Language Models as Surrogate Models in Evolutionary Algorithms: A Preliminary Study
              </a>
            </td>
          <td>
            Hao Hao, Xiaoqun Zhang, Aimin Zhou
          </td>
          <td>2024-06-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Causality is essential for understanding complex systems, such as the economy, the brain, and the climate. Constructing causal graphs often relies on either data-driven or expert-driven approaches, both fraught with challenges. The former methods, like the celebrated PC algorithm, face issues with data requirements and assumptions of causal sufficiency, while the latter demand substantial time and domain knowledge. This work explores the capabilities of Large Language Models (LLMs) as an alternative to domain experts for causal graph generation. We frame conditional independence queries as prompts to LLMs and employ the PC algorithm with the answers. The performance of the LLM-based conditional independence oracle on systems with known causal graphs shows a high degree of variability. We improve the performance through a proposed statistical-inspired voting schema that allows some control over false-positive and false-negative rates. Inspecting the chain-of-thought argumentation, we find causal reasoning to justify its answer to a probabilistic query. We show evidence that knowledge-based CIT could eventually become a complementary tool for data-driven causal discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9dadfb199bb68abb40b3e9d38abab16fd52056f" target='_blank'>
              Large Language Models for Constrained-Based Causal Discovery
              </a>
            </td>
          <td>
            Kai-Hendrik Cohrs, Gherardo Varando, Emiliano Díaz, Vasileios Sitokonstantinou, Gustau Camps-Valls
          </td>
          <td>2024-06-11</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>10</td>
        </tr>

        <tr id="What Was the Problem or Question? 

With the availability of advanced genomic profiling methods, basket trials are more commonly used for the clinical assessment of treatments targeting multiple cancer types with a common biomarker. Basket trials study a primary intervention across a range of different patient subgroups (e.g., with specific cancer types) that share similar features (e.g., mutations or biomarkers), in which the effect of the primary intervention may be expected to differ. Heterogeneity among cancer types, limited sample sizes, a lack of comparators, and the use of surrogate end points pose challenges when considering the application to economic evaluations. Bayesian hierarchical models (BHMs) are well suited to account for the heterogeneity among cancer types while borrowing information across cancer types in basket trials. There is a need for clearer guidance about how these methods can be used in health technology assessment (HTA). 

What Did We Do? 

To better understand the role of BHMs in economic evaluations, an overview of basket trials and the current understanding of their use in economic evaluations was conducted, an illustration of how BHMs can inform economic evaluations was provided with a detailed example, and considerations for interpretating and appraising this information were listed. 

What Did We Find? 

BHMs offer an approach to use information from basket trials to inform economic evaluations; however, there are limitations to be aware of when using this information in economic evaluations and also when interpreting the results of the economic information. This document describes these limitations, and contains an appraisal list that was developed to highlight the implications. These are aspects that CADTH considers when assessing analyses based on BHMs. The field of BHMs is evolving, and advice in this document will need to be revisited as methods develop. 

What Does This Mean? 

As CADTH anticipates the increased need for appraisal of tumour-agnostic therapies, this document provides clarity around an important and useful set of methods for their evaluation, while also highlighting necessary areas for further research. 
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd6b67061276c597a1b79d1df746918ef256a56" target='_blank'>
              Bayesian Hierarchical Models of Basket Trials in the Context of Economic Evaluation
              </a>
            </td>
          <td>
            PhD Alimu Dayimu, PhD Nikos Demiris, MA Karen Lee, PhD Ian Cromwell, MMath Anna Heath, PhD Petros Pechlivanoglou
          </td>
          <td>2024-05-31</td>
          <td>Canadian Journal of Health Technologies</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d59ff8e53b3c079d7c33a6288e138a814261486" target='_blank'>
              Flexible modeling of regulatory networks improves transcription factor activity estimation
              </a>
            </td>
          <td>
            Chen Chen, M. Padi
          </td>
          <td>2024-05-28</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Functional brain networks can change rapidly as a function of stimuli or cognitive shifts. Tracking dynamic functional connectivity is particularly challenging as it requires estimating the structure of the network at each moment as well as how it is shifting through time. In this paper, we describe a general modeling framework and a set of specific models that provides substantially increased statistical power for estimating rhythmic dynamic networks, based on the assumption that for a particular experiment or task, the network state at any moment is chosen from a discrete set of possible network modes. Each model is comprised of three components: (1) a set of latent switching states that represent transitions between the expression of each network mode; (2) a set of latent oscillators, each characterized by an estimated mean oscillation frequency and an instantaneous phase and amplitude at each time point; and (3) an observation model that relates the observed activity at each electrode to a linear combination of the latent oscillators. We develop an expectation-maximization procedure to estimate the network structure for each switching state and the probability of each state being expressed at each moment. We conduct a set of simulation studies to illustrate the application of these models and quantify their statistical power, even in the face of model misspecification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03731ee95e6e7485de9b4889cdb01803afb2804" target='_blank'>
              Switching Models of Oscillatory Networks Greatly Improve Inference of Dynamic Functional Connectivity
              </a>
            </td>
          <td>
            Wan-Chi Hsin, Uri T. Eden, E. Stephen
          </td>
          <td>2024-04-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Causal dynamics models (CDMs) have demonstrated significant potential in addressing various challenges in reinforcement learning. To learn CDMs, recent studies have performed causal discovery to capture the causal dependencies among environmental variables. However, the learning of CDMs is still confined to small-scale environments due to computational complexity and sample efficiency constraints. This paper aims to extend CDMs to large-scale object-oriented environments, which consist of a multitude of objects classified into different categories. We introduce the Object-Oriented CDM (OOCDM) that shares causalities and parameters among objects belonging to the same class. Furthermore, we propose a learning method for OOCDM that enables it to adapt to a varying number of objects. Experiments on large-scale tasks indicate that OOCDM outperforms existing CDMs in terms of causal discovery, prediction accuracy, generalization, and computational efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c292e2b38be98da95880bdab26b3e273ed4bd9fa" target='_blank'>
              Learning Causal Dynamics Models in Object-Oriented Environments
              </a>
            </td>
          <td>
            Zhongwei Yu, Jingqing Ruan, Dengpeng Xing
          </td>
          <td>2024-05-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND
Synthetic patient data (SPD) generation for survival analysis in oncology trials holds significant potential for accelerating clinical development. Various machine learning methods, including classification and regression trees (CART), random forest (RF), Bayesian network (BN), and CTGAN, have been employed for this purpose, but their performance in reflecting actual patient survival data remains under investigation.


OBJECTIVE
The aim of this study was to determine the most suitable SPD generation method for oncology trials, specifically focusing on both progression free survival (PFS) and overall survival (OS), which are the primary evaluation endpoints in oncology trials. To achieve this goal, we conducted a comparative simulation of 4 generation methods: CART, RF, BN, and the CTGAN, and the performance of each method was evaluated.


METHODS
Using multiple clinical trial datasets, 1000 datasets were generated by using each method for each clinical trial dataset and evaluated as follows: 1) mean survival time (MST) of PFS and OS, 2) hazard ratio distance (HRD), which indicates the similarity between the actual survival function and a synthetic survival function, and 3) visual analysis of Kaplan‒Meier (KM) plots. Each method's ability to mimic the statistical properties of real patient data was evaluated from these multiple angles.


RESULTS
In most simulation cases, CART demonstrated the high percentages of MSTSs falling within the range of 95% confidence interval (CI) of the MSTA. These percentages ranged from 88.8% to 98.0% for PFS and from 60.8% to 96.1% for OS. In the evaluation of HRD, CART demonstrated that HRD values were concentrated at approximately 0.9. Conversely, for the other methods, no consistent trend was observed for either PFS or OS. The reason why CART demonstrated better similarity than RF was that CART caused overfitting and RF, which is a kind of ensemble learning, prevented it. In SPD generation, the statistical properties close to the actual data should be the focus, not a well-generalized prediction model. Both the BN and CTGAN methods cannot accurately reflect the statistical properties of the actual data because small datasets are not suitable.


CONCLUSIONS
As a method for generating SPD for survival data from small datasets, such as clinical trial data, CART demonstrated to be the most effective method compared to RF, BN, and CTGAN. Additionally, it is possible to improve CART-based generation methods by incorporating feature engineering and other methods in future work.


CLINICALTRIAL">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fab7085faab1ee98cbf5d8f17cdd2902588dbcd2" target='_blank'>
              A Comparison of Synthetic Data Generation Techniques for Control Group Survival Data in Oncology Clinical Trials: Simulation Study.
              </a>
            </td>
          <td>
            Ippei Akiya, Takuma Ishihara, Keiichi Yamamoto
          </td>
          <td>2023-12-03</td>
          <td>JMIR medical informatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Accurate prediction of complex traits is an important task in quantitative genetics that has become increasingly relevant for personalized medicine. Genotypes have traditionally been used for trait prediction using a variety of methods such as mixed models, Bayesian methods, penalized regressions, dimension reductions, and machine learning methods. Recent studies have shown that gene expression levels can produce higher prediction accuracy than genotypes. However, only a few prediction methods were used in these studies. Thus, a comprehensive assessment of methods is needed to fully evaluate the potential of gene expression as a predictor of complex trait phenotypes. Here, we used data from the Drosophila Genetic Reference Panel (DGRP) to compare the ability of several existing statistical learning methods to predict starvation resistance from gene expression in the two sexes separately. The methods considered differ in assumptions about the distribution of gene effect sizes – ranging from models that assume that every gene affects the trait to more sparse models – and their ability to capture gene-gene interactions. We also used functional annotation (i.e., Gene Ontology (GO)) as an external source of biological information to inform prediction models. The results show that differences in prediction accuracy between methods exist, although they are generally not large. Methods performing variable selection gave higher accuracy in females while methods assuming a more polygenic architecture performed better in males. Incorporating GO annotations further improved prediction accuracy for a few GO terms of biological significance. Biological significance extended to the genes underlying highly predictive GO terms with different genes emerging between sexes. Notably, the Insulin-like Receptor (InR) was prevalent across methods and sexes. Our results confirmed the potential of transcriptomic prediction and highlighted the importance of selecting appropriate methods and strategies in order to achieve accurate predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d73933f0662a38a7e71a31b72482d73e233bf93b" target='_blank'>
              Comparing statistical learning methods for complex trait prediction from gene expression
              </a>
            </td>
          <td>
            Noah Klimkowski Arango, Fabio Morgante
          </td>
          <td>2024-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Shariat et al previously investigated the possibility of predicting, from preoperative biomarkers and clinical data, which of any pair of patients would suffer recurrence of prostate cancer first. We wished to establish the extent to which predictions of time of relapse from such a model could be improved upon using Bayesian methodology. The same dataset was reanalysed using a Bayesian skew-Student mixture model. Predictions were made of which of any pair of patients would relapse first and of the time of relapse. The benefit of using these biomarkers relative to predictions made without them, was measured by the apparent Shannon information, using as prior a simple exponential attrition model of relapse time independent of input variables. Using half the dataset for training and the other half for testing, predictions of relapse time from the strict Cox model gave $-\infty$ nepers of apparent Shannon information, (it predicts that relapse can only occur at times when patients in the training set relapsed). Deliberately smoothed predictions from the Cox model gave -0.001 (-0.131 to +0.120) nepers, while the Bayesian model gave +0.109 (+0.021 to +0.192) nepers (mean, 2.5 to 97.5 centiles), being positive with posterior probability 0.993 and beating the blurred Cox model with posterior probability 0.927. These predictions from the Bayesian model thus outperform those of the Cox model, but the overall yield of predictive information leaves scope for improvement of the range of biomarkers in use. The Bayesian model presented here is the first such model for prostate cancer to consider the variation of relapse hazard with biomarker concentrations to be smooth, as is intuitive. It is also the first model to be shown to provide more apparent Shannon information than the Cox model and the first to be shown to provide positive apparent information relative to an exponential prior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cc6b31cfe40f0eb0337a1dbad085f46b2695b53" target='_blank'>
              Bayesian analysis of biomarker levels can predict time of recurrence of prostate cancer with strictly positive apparent Shannon information against an exponential attrition prior
              </a>
            </td>
          <td>
            Roger Sewell, Elisabeth Crowe, S. Shariat
          </td>
          <td>2024-04-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Introduction: Gene regulatory networks (GRNs) are characterized by their dynamism, meaning that the regulatory interactions which constitute these networks evolve with time. Identifying when changes in the GRN architecture occur can inform our understanding of fundamental biological processes, such as disease manifestation, development, and evolution. However, it is usually not possible to know a priori when a change in the network architecture will occur. Furthermore, an architectural shift may alter the underlying noise characteristics, such as the process noise covariance.Methods: We develop a fully Bayesian hierarchical model to address the following: a) sudden changes in the network architecture; b) unknown process noise covariance which may change along with the network structure; and c) unknown measurement noise covariance. We exploit the use of conjugate priors to develop an analytically tractable inference scheme using Bayesian sequential Monte Carlo (SMC) with a local Gibbs sampler.Results: Our Bayesian learning algorithm effectively estimates time-varying gene expression levels and architectural model indicators under varying noise conditions. It accurately captures sudden changes in network architecture and accounts for time-evolving process and measurement noise characteristics. Our algorithm performs well even under high noise conditions. By incorporating conjugate priors, we achieve analytical tractability, enabling robust inference despite the inherent complexities of the system. Furthermore, our method outperforms the standard particle filter in all test scenarios.Discussion: The results underscore our method’s efficacy in capturing architectural changes in GRNs. Its ability to adapt to a range of time-evolving noise conditions emphasizes its practical relevance for real-world biological data, where noise presents a significant challenge. Overall, our method provides a powerful tool for studying the dynamics of GRNs and has the potential to advance our understanding of fundamental biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8a243a140a8c2ad96013292fbe388c444c69974" target='_blank'>
              Bayesian learning of nonlinear gene regulatory networks with switching architectures
              </a>
            </td>
          <td>
            Nayely Vélez-Cruz, Antonia Papandreou-Suppappola
          </td>
          <td>2024-05-22</td>
          <td>Frontiers in Signal Processing</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 21],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>