<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Latent%20space%20simulators/">
      
      
        <link rel="next" href="../PINNs/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Parametrizing using ML
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Time-series%20forecasting/" class="md-tabs__link">
        
  
    
  
  Time-series forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Symbolic%20regression/" class="md-tabs__link">
        
  
    
  
  Symbolic regression

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Neural%20ODEs/" class="md-tabs__link">
        
  
    
  
  Neural ODEs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Physics-based%20GNNs/" class="md-tabs__link">
        
  
    
  
  Physics-based GNNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Latent%20space%20simulators/" class="md-tabs__link">
        
  
    
  
  Latent space simulators

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Parametrizing using ML

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../PINNs/" class="md-tabs__link">
        
  
    
  
  PINNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Koopman%20operator/" class="md-tabs__link">
        
  
    
  
  Koopman operator

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Time-series%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Time-series forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Symbolic%20regression/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Symbolic regression
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Neural%20ODEs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Neural ODEs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Physics-based%20GNNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Physics-based GNNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Latent%20space%20simulators/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Latent space simulators
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Parametrizing using ML
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../PINNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    PINNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Koopman%20operator/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Koopman operator
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Parametrizing using ML</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-05-21 05:06:17 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Parametrizing using ML</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Parametrizing using ML</a><br>
      <a href="#recommended_articles">3. Recommended articles on Parametrizing using ML</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Parametrizing using ML</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Parametrizing using ML</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e837965494c4edbec4d30832d31ba5639996da8" target='_blank'>
                Deep learning prediction of patient response time course from early data via neural-pharmacokinetic/pharmacodynamic modelling
                </a>
              </td>
          <td>
            James Lu, B. Bender, Jin Y. Jin, Y. Guan
          </td>
          <td>2020-10-22</td>
          <td>Nature Machine Intelligence</td>
          <td>31</td>
          <td>37</td>

            <td><a href='../recommendations/4e837965494c4edbec4d30832d31ba5639996da8' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Many real-world systems such as social networks and moving planets are dynamic in nature, where a set of coupled objects are connected via the interaction graph and exhibit complex behavior along the time. For example, the COVID-19 pandemic can be considered as a dynamical system, where objects represent geographical locations (e.g., states) whose daily confirmed cases of infection evolve over time. Outbreak at one location may influence another location as people travel between these locations, forming a graph. Thus, how to model and predict the complex dynamics for these systems becomes a critical research problem. Existing work on modeling graph-structured data mostly assumes a static setting. How to handle dynamic graphs remains to be further explored. On one hand, features of objects change over time, influenced by the linked objects in the interaction graph. On the other hand, the graph itself can also evolve, where new interactions (links) may form and existing links may drop, which may in turn be affected by the dynamic features of objects. In this paper, we propose coupled graph ODE: a novel latent ordinary differential equation (ODE) generative model that learns the coupled dynamics of nodes and edges with a graph neural network (GNN) based ODE in a continuous manner. Our model consists of two coupled ODE functions for modeling the dynamics of edges and nodes based on their latent representations respectively. It employs a novel encoder parameterized by a GNN for inferring the initial states from historical data, which serves as the starting point of the predicted latent trajectories. Experiment results on the COVID-19 dataset and the simulated social network dataset demonstrate the effectiveness of our proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf2145f9998f304513c0c9b530ee9f7750c6f55" target='_blank'>
                Coupled Graph ODE for Learning Interacting System Dynamics
                </a>
              </td>
          <td>
            Zijie Huang, Yizhou Sun, Wei Wang
          </td>
          <td>2021-08-14</td>
          <td>Proceedings of the 27th ACM SIGKDD Conference on Knowledge Discovery & Data Mining, Knowledge Discovery and Data Mining</td>
          <td>47</td>
          <td>57</td>

            <td><a href='../recommendations/aaf2145f9998f304513c0c9b530ee9f7750c6f55' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd28606fbae3449f831248804264c9885e992f9" target='_blank'>
                CellBox: Interpretable Machine Learning for Perturbation Biology with Application to the Design of Cancer Combination Therapy.
                </a>
              </td>
          <td>
            Bo Yuan, Ciyue Shen, Augustin Luna, Anil Korkut, D. Marks, John Ingraham, C. Sander
          </td>
          <td>2020-12-21</td>
          <td>Cell systems</td>
          <td>68</td>
          <td>152</td>

            <td><a href='../recommendations/6bd28606fbae3449f831248804264c9885e992f9' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="We introduce a flexible, scalable Bayesian inference framework for nonlinear dynamical systems characterised by distinct and hierarchical variability at the individual, group, and population levels. Our model class is a generalisation of nonlinear mixed-effects (NLME) dynamical systems, the statistical workhorse for many experimental sciences. We cast parameter inference as stochastic optimisation of an end-to-end differentiable, block-conditional variational autoencoder. We specify the dynamics of the data-generating process as an ordinary differential equation (ODE) such that both the ODE and its solver are fully differentiable. This model class is highly flexible: the ODE right-hand sides can be a mixture of user-prescribed or “white-box” sub-components and neural network or “black-box” sub-components. Using stochastic optimisation, our amortised inference algorithm could seamlessly scale up to massive data collection pipelines (common in labs with robotic automation). Finally, our framework supports interpretability with respect to the underlying dynamics, as well as predictive generalization to unseen combinations of group components (also called “zero-shot” learning). We empirically validate our method by predicting the dynamic behaviour of bacteria that were genetically engineered to function as biosensors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309c5ae93a4cabfd37747abd130866240e265b2d" target='_blank'>
                Efficient Amortised Bayesian Inference for Hierarchical and Nonlinear Dynamical Systems
                </a>
              </td>
          <td>
            Geoffrey Roeder, Paul K. Grant, Andrew Phillips, Neil Dalchau, Edward Meeds
          </td>
          <td>2019-05-24</td>
          <td>ArXiv, International Conference on Machine Learning</td>
          <td>21</td>
          <td>28</td>

            <td><a href='../recommendations/309c5ae93a4cabfd37747abd130866240e265b2d' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Parametrizing using ML'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Parametrizing using ML</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="A variety of classical machine learning approaches have been developed over the past ten years with the aim to individualize drug dosages based on measured plasma concentrations. However, the interpretability of these models is challenging as they do not incorporate information on pharmacokinetic (PK) drug disposition. In this work we compare well-known population PK modelling with classical and a newly proposed scientific machine learning (SciML) framework, which combines knowledge on drug disposition with data-driven modelling. Our approach lets us estimate population PK parameters and their inter-individual variability (IIV) using multimodal covariate data of each patient. A dataset of 549 fluorouracil (5FU) plasma concentrations as example for an intravenously administered drug and a dataset of 308 sunitinib concentrations as example for an orally administered drug were used for analysis. Whereas classical machine learning models were not able to describe the data sufficiently, the proposed model allowed us to obtain highly accurate predictions even for new patients. Additionally, we demonstrated that our model could outperform traditional population PK models in terms of accuracy and greater flexibility when learning population parameters if given enough training data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01334b780295132c89212673bbf083ad5a36343e" target='_blank'>
              Scientific machine learning for predicting plasma concentrations in anti-cancer therapy
              </a>
            </td>
          <td>
            Diego Valderrama, O. Teplytska, Luca Marie, Koltermann, Elena Trunz, Eduard Schmulenson, Achim Fritsch, Ulrich Jaehde, Holger Fröhlich, Schloss Birlinghoven, Sankt Augustin, Germany, An der Immenburg, D. Bonn
          </td>
          <td>2024-05-08</td>
          <td>None</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="A patient's digital twin is a computational model that describes the evolution of their health over time. Digital twins have the potential to revolutionize medicine by enabling individual-level computer simulations of human health, which can be used to conduct more efficient clinical trials or to recommend personalized treatment options. Due to the overwhelming complexity of human biology, machine learning approaches that leverage large datasets of historical patients' longitudinal health records to generate patients' digital twins are more tractable than potential mechanistic models. In this manuscript, we describe a neural network architecture that can learn conditional generative models of clinical trajectories, which we call Digital Twin Generators (DTGs), that can create digital twins of individual patients. We show that the same neural network architecture can be trained to generate accurate digital twins for patients across 13 different indications simply by changing the training set and tuning hyperparameters. By introducing a general purpose architecture, we aim to unlock the ability to scale machine learning approaches to larger datasets and across more indications so that a digital twin could be created for any patient in the world.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd216824a3b00951db6ff54f723148cfa3706e5" target='_blank'>
              Digital Twin Generators for Disease Modeling
              </a>
            </td>
          <td>
            Nameyeh Alam, Jake Basilico, Daniele Bertolini, S. C. Chetty, Heather D'Angelo, Ryan Douglas, Charles K. Fisher, Franklin Fuller, Melissa Gomes, Rishabh Gupta, Alex Lang, A. Loukianov, Rachel Mak-McCully, Cary Murray, Hanalei Pham, Susanna Qiao, Elena S. Ryapolova-Webb, Aaron M. Smith, Dimitri Theoharatos, Anil Tolwani, Eric W. Tramel, Anna A. Vidovszky, J. Viduya, Jonathan R. Walsh
          </td>
          <td>2024-05-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="As a vital component in autonomous driving, accurate trajectory prediction effectively prevents traffic accidents and improves driving efficiency. To capture complex spatial-temporal dynamics and social interactions, recent studies developed models based on advanced deep-learning methods. On the other hand, recent studies have explored the use of deep generative models to further account for trajectory uncertainties. However, the current approaches demonstrating indeterminacy involve inefficient and time-consuming practices such as sampling from trained models. To fill this gap, we proposed a novel model named Graph Recurrent Attentive Neural Process (GRANP) for vehicle trajectory prediction while efficiently quantifying prediction uncertainty. In particular, GRANP contains an encoder with deterministic and latent paths, and a decoder for prediction. The encoder, including stacked Graph Attention Networks, LSTM and 1D convolutional layers, is employed to extract spatial-temporal relationships. The decoder is used to learn a latent distribution and thus quantify prediction uncertainty. To reveal the effectiveness of our model, we evaluate the performance of GRANP on the highD dataset. Extensive experiments show that GRANP achieves state-of-the-art results and can efficiently quantify uncertainties. Additionally, we undertake an intuitive case study that showcases the interpretability of the proposed approach. The code is available at https://github.com/joy-driven/GRANP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aeabd23a4da81ec6061d050b0004bf8b34090fa4" target='_blank'>
              GRANP: A Graph Recurrent Attentive Neural Process Model for Vehicle Trajectory Prediction
              </a>
            </td>
          <td>
            Yuhao Luo, Kehua Chen, Meixin Zhu
          </td>
          <td>2024-04-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Predicting the drug response of cancer cell lines is crucial for advancing personalized cancer treatment, yet remains challenging due to tumor heterogeneity and individual diversity. In this study, we present a deep learning-based framework named Deep neural network Integrating Prior Knowledge (DIPK) (DIPK), which adopts self-supervised techniques to integrate multiple valuable information, including gene interaction relationships, gene expression profiles and molecular topologies, to enhance prediction accuracy and robustness. We demonstrated the superior performance of DIPK compared to existing methods on both known and novel cells and drugs, underscoring the importance of gene interaction relationships in drug response prediction. In addition, DIPK extends its applicability to single-cell RNA sequencing data, showcasing its capability for single-cell-level response prediction and cell identification. Further, we assess the applicability of DIPK on clinical data. DIPK accurately predicted a higher response to paclitaxel in the pathological complete response (pCR) group compared to the residual disease group, affirming the better response of the pCR group to the chemotherapy compound. We believe that the integration of DIPK into clinical decision-making processes has the potential to enhance individualized treatment strategies for cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61de3d8efdb488ba8fbc8b55f3dba2a03c6703d7" target='_blank'>
              Improving drug response prediction via integrating gene relationships with deep learning
              </a>
            </td>
          <td>
            Pengyong Li, Zhengxiang Jiang, Tianxiao Liu, Xinyu Liu, Hui Qiao, Xiaojun Yao
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Pharmacometric models are pivotal across drug discovery and development, playing a decisive role in determining the progression of candidate molecules. However, the derivation of mathematical equations governing the system is a labor-intensive trial-and-error process, often constrained by tight timelines. In this study, we introduce PKINNs, a novel purely data-driven pharmacokinetic-informed neural network model. PKINNs efficiently discovers and models intrinsic multi-compartment-based pharmacometric structures, reliably forecasting their derivatives. The resulting models are both interpretable and explainable through Symbolic Regression methods. Our computational framework demonstrates the potential for closed-form model discovery in pharmacometric applications, addressing the labor-intensive nature of traditional model derivation. With the increasing availability of large datasets, this framework holds the potential to significantly enhance model-informed drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecec2c77f9d85a4e75c8a8de808c4d916ca0015" target='_blank'>
              Discovering intrinsic multi-compartment pharmacometric models using Physics Informed Neural Networks
              </a>
            </td>
          <td>
            I. Nasim, Adam Nasim
          </td>
          <td>2024-04-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The deep-learning models like variational autoencoder have enabled low dimensional cellular embedding representation for large-scale single-cell transcriptomes and shown great flexibility in downstream tasks. However, biologically meaningful latent space is usually missing if no specific structure is designed. Here, we engineered a novel interpretable generative transcriptional program (iGTP) framework that could model the importance of TP space and protein-protein interactions (PPIs) between different biological states. We demonstrate the performance of iGTP in a diverse biological context using Gene Ontology, canonical pathway, and different PPI curation. iGTP not only elucidated the ground truth of cellular responses but also surpassed other deep learning models and traditional bioinformatics methods in functional enrichment tasks. By integrating the latent layer with a graph neural network (GNN) framework, iGTP effectively inferred cellular responses to perturbations. We anticipate that iGTP offers insights at both PPI and TP levels, and holds promise for predicting responses to novel perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8b53c25d331df61bf601a33636784d04ce05d68" target='_blank'>
              Learning interpretable cellular embedding for inferring biological mechanisms underlying single-cell transcriptomics
              </a>
            </td>
          <td>
            k.-l. Hsieh, Y. Chu, P. G. Pilie, K. Zhang
          </td>
          <td>2024-04-01</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af03bba04f9d8a9c6f275eb6c2df9c1a05800cfc" target='_blank'>
              On inductive biases for the robust and interpretable prediction of drug concentrations using deep compartment models.
              </a>
            </td>
          <td>
            A. Janssen, Frank C Bennis, M. Cnossen, R. Mathôt
          </td>
          <td>2024-03-26</td>
          <td>Journal of pharmacokinetics and pharmacodynamics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Forecasting human trajectories in traffic scenes is critical for safety within mixed or fully autonomous systems. Human future trajectories are driven by two major stimuli, social interactions, and stochastic goals. Thus, reliable forecasting needs to capture these two stimuli. Edge-based relation modeling represents social interactions using pairwise correlations from precise individual states. Nevertheless, edge-based relations can be vulnerable under perturbations. To alleviate these issues, we propose a region-based relation learning paradigm that models social interactions via region-wise dynamics of joint states, i.e., the changes in the density of crowds. In particular, region-wise agent joint information is encoded within convolutional feature grids. Social relations are modeled by relating the temporal changes of local joint information from a global perspective. We show that region-based relations are less susceptible to perturbations. In order to account for the stochastic individual goals, we exploit a conditional variational autoencoder to realize multi-goal estimation and diverse future prediction. Specifically, we perform variational inference via the latent distribution, which is conditioned on the correlation between input states and associated target goals. Sampling from the latent distribution enables the framework to reliably capture the stochastic behavior in test data. We integrate multi-goal estimation and region-based relation learning to model the two stimuli, social interactions, and stochastic goals, in a prediction framework. We evaluate our framework on the ETH-UCY dataset and Stanford Drone Dataset (SDD). We show that the diverse prediction better fits the ground truth when incorporating the relation module. Our framework outperforms the state-of-the-art models on SDD by $27.61\%$/$18.20\%$ of ADE/FDE metrics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b02b0c05987b0686dafb277246ac09b4efef7f91" target='_blank'>
              TrajPRed: Trajectory Prediction with Region-based Relation Learning
              </a>
            </td>
          <td>
            Chen Zhou, Ghassan AlRegib, Armin Parchami, Kunjan Singh
          </td>
          <td>2024-04-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Accurate prediction of a new compound's pharmacokinetic (PK) profile is pivotal for the success of drug discovery programs. An initial assessment of PK in preclinical species and humans is typically performed through allometric scaling and mathematical modeling. These methods use parameters estimated from in vitro or in vivo experiments, which although helpful for an initial estimation, require extensive animal experiments. Furthermore, mathematical models are limited by the mechanistic underpinning of the drugs' absorption, distribution, metabolism, and elimination (ADME) which are largely unknown in the early stages of drug discovery. In this work, we propose a novel methodology in which concentration versus time profile of small molecules in rats is directly predicted by machine learning (ML) using structure‐driven molecular properties as input and thus mitigating the need for animal experimentation. The proposed framework initially predicts ADME properties based on molecular structure and then uses them as input to a ML model to predict the PK profile. For the compounds tested, our results demonstrate that PK profiles can be adequately predicted using the proposed algorithm, especially for compounds with Tanimoto score greater than 0.5, the average mean absolute percentage error between predicted PK profile and observed PK profile data was found to be less than 150%. The suggested framework aims to facilitate PK predictions and thus support molecular screening and design earlier in the drug discovery process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78aa04a2b81e41216bb87a34747852fb6128be98" target='_blank'>
              Machine learning framework to predict pharmacokinetic profile of small molecule drugs based on chemical structure
              </a>
            </td>
          <td>
            Nikhil Pillai, Alexandra Abos, Donato Teutonico, P. D. Mavroudis
          </td>
          <td>2024-05-01</td>
          <td>Clinical and Translational Science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Personalized cancer treatment requires a thorough understanding of complex interactions between drugs and cancer cell lines in varying genetic and molecular contexts. To address this, high-throughput screening has been used to generate large-scale drug response data, facilitating data-driven computational models. Such models can capture complex drug-cell line interactions across various contexts in a fully data-driven manner. However, accurately prioritizing the most effective drugs for each cell line still remains a significant challenge. To address this, we developed multiple neural ranking approaches that leverage large-scale drug response data across multiple cell lines from diverse cancer types. Unlike existing approaches that primarily utilize regression and classification techniques for drug response prediction, we formulated the objective of drug selection and prioritization as a drug ranking problem. In this work, we proposed multiple pairwise and listwise neural ranking methods that learn latent representations of drugs and cell lines and then use those representations to score drugs in each cell line via a learnable scoring function. Specifically, we developed neural pairwise and listwise ranking methods, Pair-PushC and List-One on top of the existing methods, pLETORg and ListNet, respectively. Additionally, we proposed a novel listwise ranking method, List-All, that focuses on all the effective drugs instead of the top effective drug, unlike List-One. We also provide an exhaustive empirical evaluation with state-of-the-art regression and ranking baselines on large-scale data sets across multiple experimental settings. Our results demonstrate that our proposed ranking methods mostly outperform the best baselines with significant improvements of as much as 25.6% in terms of selecting truly effective drugs within the top 20 predicted drugs (i.e., hit@20) across 50% test cell lines. Furthermore, our analyses suggest that the learned latent spaces from our proposed methods demonstrate informative clustering structures and capture relevant underlying biological features. Moreover, our comprehensive evaluation provides a thorough and objective comparison of the performance of different methods (including our proposed ones).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c0f5fb3d038f7182389a924d6a9c7f0a1db1816" target='_blank'>
              Improving Anticancer Drug Selection and Prioritization via Neural Learning to Rank.
              </a>
            </td>
          <td>
            Vishal Dey, Xia Ning
          </td>
          <td>2024-05-13</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Artificial intelligence (AI) technology has demonstrated remarkable potential in drug dis-covery, where pharmacokinetics plays a crucial role in determining the dosage, safety, and efficacy of new drugs. A major challenge for AI-driven drug discovery (AIDD) is the scarcity of high-quality data, which often requires extensive wet-lab work. A typical example of this is pharmacokinetic experiments. In this work, we develop a physical formula enhanced mul-ti-task learning (PEMAL) method that predicts four key parameters of pharmacokinetics simultaneously. By incorporating physical formulas into the multi-task framework, PEMAL facilitates effective knowledge sharing and target alignment among the pharmacokinetic parameters, thereby enhancing the accuracy of prediction. Our experiments reveal that PEMAL significantly lowers the data demand, compared to typical Graph Neural Networks. Moreover, we demonstrate that PEMAL enhances the robustness to noise, an advantage that conventional Neural Networks do not possess. Another advantage of PEMAL is its high flexibility, which can be potentially applied to other multi-task machine learning scenarios. Overall, our work illustrates the benefits and potential of using PEMAL in AIDD and other scenarios with data scarcity and noise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8909ccf04a951931d7cedb8b1142f5a37787182" target='_blank'>
              Physical formula enhanced multi-task learning for pharmacokinetics prediction
              </a>
            </td>
          <td>
            Ruifeng Li, Dongzhan Zhou, Ancheng Shen, Ao Zhang, Mao Su, Mingqian Li, Hongyang Chen, Gang Chen, Yin Zhang, Shufei Zhang, Yuqiang Li, Wanli Ouyang
          </td>
          <td>2024-04-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite the remarkable advances in cancer diagnosis, treatment, and management over the past decade, malignant tumors remain a major public health problem. Further progress in combating cancer may be enabled by personalizing the delivery of therapies according to the predicted response for each individual patient. The design of personalized therapies requires the integration of patient-specific information with an appropriate mathematical model of tumor response. A fundamental barrier to realizing this paradigm is the current lack of a rigorous yet practical mathematical theory of tumor initiation, development, invasion, and response to therapy. We begin this review with an overview of different approaches to modeling tumor growth and treatment, including mechanistic as well as data-driven models based on big data and artificial intelligence. We then present illustrative examples of mathematical models manifesting their utility and discuss the limitations of stand-alone mechanistic and data-driven models. We then discuss the potential of mechanistic models for not only predicting but also optimizing response to therapy on a patient-specific basis. We describe current efforts and future possibilities to integrate mechanistic and data-driven models. We conclude by proposing five fundamental challenges that must be addressed to fully realize personalized care for cancer patients driven by computational models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f694d2c55d26c981a3270eb504e2688631b8a8e" target='_blank'>
              Patient-Specific, Mechanistic Models of Tumor Growth Incorporating Artificial Intelligence and Big Data.
              </a>
            </td>
          <td>
            G. Lorenzo, Syed Rakin Ahmed, David A. Hormuth II, Brenna Vaughn, Jayashree Kalpathy-Cramer, Luis Solorio, T. Yankeelov, Hector Gomez
          </td>
          <td>2024-04-09</td>
          <td>Annual review of biomedical engineering</td>
          <td>2</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ae9f9ed096ad62f103bcd45fe4cf89ae91d36b" target='_blank'>
              MFSynDCP: multi-source feature collaborative interactive learning for drug combination synergy prediction
              </a>
            </td>
          <td>
            Yunyun Dong, Yunqing Chang, Yuxiang Wang, Qixuan Han, Xiaoyuan Wen, Ziting Yang, Yan Zhang, Yan Qiang, Kun Wu, Xiaole Fan, Xiaoqiang Ren
          </td>
          <td>2024-04-01</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Latent variable models such as the Variational Auto-Encoders (VAEs) have shown impressive performance for inferring expression patterns for cell subtyping and biomarker identification from transcriptomics data. However, the limited interpretability of their latent variables obscures deriving meaningful biological understanding of cellular responses to different external and internal perturbations. We here propose a novel deep learning framework, EXPORT (EXPlainable VAE for ORdinally perturbed Transcriptomics data), for analyzing ordinally perturbed transcriptomics data that can incorporate any biological pathway knowledge in the VAE latent space. With the corresponding pathway-informed decoder, the learned latent expression patterns can be explained as pathway-level responses to perturbations, offering direct interpretability with biological understanding. More importantly, we explicitly model the ordinal nature of many real-world perturbations into the EXPORT framework by training an auxiliary ordinal regressor neural network to capture corresponding expression changes in the VAE latent representations, for example under different dosage levels of radiation exposure. By incorporating ordinal constraints during the training of our proposed framework, we further enhance the model interpretability by guiding the VAE latent space to organize perturbation responses in a hierarchical manner. We demonstrate the utility of the inferred guided latent space for downstream tasks, such as identifying key regulatory pathways associated with specific perturbation changes by analyzing transcriptomics datasets on both bulk and single-cell data. Overall, we envision that our proposed approach can unravel unprecedented biological intricacies in cellular responses to various perturbations while bringing an additional layer of interpretability to biology-inspired deep learning models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/385ecf17e4b8589c0b54575cc27c3b02323fabbb" target='_blank'>
              Biologically Interpretable VAE with Supervision for Transcriptomics Data Under Ordinal Perturbations
              </a>
            </td>
          <td>
            Seyednami Niyakan, Byung-Jun Yoon, Xiaoning Qian, Xihaier Luo
          </td>
          <td>2024-03-29</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7edb6831fed7889f78e4a04d11e72176410264" target='_blank'>
              A review of machine learning-based methods for predicting drug–target interactions
              </a>
            </td>
          <td>
            Wen Shi, Hong Yang, Linhai Xie, X. Yin, Yanchun Zhang
          </td>
          <td>2024-04-12</td>
          <td>Health Information Science and Systems</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a46186cfa337e51d671a8ac45c149e5f0bf01b" target='_blank'>
              Data-driven modeling of interrelated dynamical systems
              </a>
            </td>
          <td>
            Yonatan Elul, Eyal Rozenberg, Amit Boyarski, Yael Yaniv, Assaf Schuster, A. M. Bronstein
          </td>
          <td>2024-05-01</td>
          <td>Communications Physics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Balancing accuracy and efficiency is a common problem in molecular simulation. This tradeoff is evident in coarse-grained molecular dynamics simulation, which prioritizes efficiency, and all-atom molecular simulation, which prioritizes accuracy. Despite continuous efforts, creating a coarse-grained model that accurately captures both the system's structure and dynamics remains elusive. In this article, we present a data-driven approach for constructing coarse-grained models that aim to describe both the structure and dynamics of the system equally well. While the development of machine learning models is well-received in the scientific community, the significance of dataset creation for these models is often overlooked. However, data-driven approaches cannot progress without a robust dataset. To address this, we construct a database of synthetic coarse-grained potentials generated from unphysical all-atom models. A neural network is trained with the generated database to predict the coarse-grained potentials of real liquids. We evaluate their quality by calculating the combined loss of structural and dynamical accuracy upon coarse-graining. When we compare our machine learning-based coarse-grained potential with the one from iterative Boltzmann inversion, the machine learning prediction turns out better for all eight hydrocarbon liquids we studied. As all-atom surfaces turn more nonspherical, both ways of coarse-graining degrade. Still, the neural network outperforms iterative Boltzmann inversion in constructing good quality coarse-grained models for such cases. The synthetic database and the developed machine learning models are freely available to the community, and we believe that our approach will generate interest in efficiently deriving accurate coarse-grained models for liquids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f150b0dd190b041bbf23edab970f507000eac410" target='_blank'>
              Synthetic Force-Field Database for Training Machine Learning Models to Predict Mobility-Preserving Coarse-Grained Molecular-Simulation Potentials.
              </a>
            </td>
          <td>
            Saientan Bag, Melissa K. Meinel, F. Müller-Plathe
          </td>
          <td>2024-04-09</td>
          <td>Journal of chemical theory and computation</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="This work proposes a novel framework for analyzing disease progression using time-aware neural ordinary differential equations (NODE). We introduce a"time-aware head"in a framework trained through self-supervised learning (SSL) to leverage temporal information in latent space for data augmentation. This approach effectively integrates NODEs with SSL, offering significant performance improvements compared to traditional methods that lack explicit temporal integration. We demonstrate the effectiveness of our strategy for diabetic retinopathy progression prediction using the OPHDIAT database. Compared to the baseline, all NODE architectures achieve statistically significant improvements in area under the ROC curve (AUC) and Kappa metrics, highlighting the efficacy of pre-training with SSL-inspired approaches. Additionally, our framework promotes stable training for NODEs, a commonly encountered challenge in time-aware modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba9952dbd34de0b508ed4f1290756368e714c81" target='_blank'>
              LaTiM: Longitudinal representation learning in continuous-time models to predict disease progression
              </a>
            </td>
          <td>
            Rachid Zeghlache, Pierre-Henri Conze, Mostafa EL HABIB DAHO, Yi-Hsuan Li, Hugo Le Boit'e, R. Tadayoni, Pascale Massin, B. Cochener, Alireza Rezaei, Ikram Brahim, G. Quellec, M. Lamard
          </td>
          <td>2024-04-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background Accurate and robust estimation of the synergistic drug combination is important for precision medicine. Although some computational methods have been developed, some predictions are still unreliable especially for the cross-dataset predictions, due to the complex mechanism of drug combinations and heterogeneity of cancer samples. Methods We have proposed JointSyn that utilizes dual-view jointly learning to predict sample-specific effects of drug combination from drug and cell features. JointSyn capture the drug synergy related features from two views. One view is the embedding of drug combination on cancer cell lines, and the other view is the combination of two drugs’ embeddings on cancer cell lines. Finally, the prediction net uses the features learned from the two views to predict the drug synergy of the drug combination on the cell line. In addition, we used the fine-tuning method to improve the JointSyn’s performance on the unseen subset within a dataset or cross dataset. Results JointSyn outperforms existing state-of-the-art methods in predictive accuracy and robustness across various benchmarks. Each view of JointSyn captures drug synergy-related characteristics and make complementary contributes to the final accurate prediction of drug combination. Moreover, JointSyn with fine-tuning improves its generalization ability to predict a novel drug combination or cancer sample only using a small number of experimental measurements. We also used JointSyn to generate an estimated atlas of drug synergy for pan-cancer and explored the differential pattern among cancers. Conclusions These results demonstrate the potential of JointSyn to predict drug synergy, supporting the development of personalized combinatorial therapies. The source code is available on GitHub at https://github.com/LiHongCSBLab/JointSyn.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9488eec35424227f278c505eef1285f2fcb3bb3" target='_blank'>
              Dual-view jointly learning improves personalized drug synergy prediction
              </a>
            </td>
          <td>
            Xueliang Li, Bihan Shen, Fangyoumin Feng, Kun Li, Hong Li
          </td>
          <td>2024-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Human-designed algorithms have long been fundamental in solving a variety of scientific and engineering challenges. Recently, data-driven deep learning methods have also risen to prominence, offering innovative solutions across numerous scientific fields. While traditional algorithms excel in capturing the core aspects of specific problems, they often lack the flexibility needed for varying problem conditions due to the absence of specific data. Conversely, while data-driven approaches utilize vast datasets, they frequently fall short in domain-specific knowledge. To bridge these gaps, we introduce \textbf{FMint} (Foundation Model based on Initialization), a generative pre-trained model that synergizes the precision of human-designed algorithms with the adaptability of data-driven methods. This model is specifically engineered for high-accuracy simulation of dynamical systems. Starting from initial trajectories provided by conventional methods, FMint quickly delivers highly accurate solutions. It incorporates in-context learning and has been pre-trained on a diverse corpus of 500,000 dynamical systems, showcasing exceptional generalization across a broad spectrum of real-world applications. By effectively combining algorithmic rigor with data-driven flexibility, FMint sets the stage for the next generation of scientific foundation models, tackling complex problems with both efficiency and high accuracy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdb103a4bfafda077b36aa592536425c2695eef3" target='_blank'>
              FMint: Bridging Human Designed and Data Pretrained Models for Differential Equation Foundation Model
              </a>
            </td>
          <td>
            Zezheng Song, Jiaxin Yuan, Haizhao Yang
          </td>
          <td>2024-04-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Motivation Successfully predicting the development of biological systems can lead to advances in various research fields, such as cellular biology and epidemiology. While machine learning has proven its capabilities in generalizing the underlying non-linear dynamics of such systems, unlocking its predictive power is often restrained by the limited availability of large, curated datasets. To supplement real-world data, informing machine learning by transfer learning with data simulated from ordinary differential equations has emerged as a promising solution. However, the success of this approach highly depends on the designed characteristics of the synthetic data. Results We optimize dataset characteristics such as size, diversity, and noise of ordinary differential equation-based synthetic time series datasets in three relevant and representative biological systems. To achieve this, we here, for the first time, present a framework to systematically evaluate the influence of such design choices on transfer learning performance in one place. We achieve a performance improvement of up to 92% in mean absolute error for our optimized simulation-based transfer learning compared to non-informed deep learning. We find a strong interdependency between dataset size and diversity effects. The optimal transfer learning setting heavily relies on real-world data characteristics as well as its coherence with the synthetic data’s dynamics, emphasizing the relevance of such a framework. Availability and Implementation The code is available at https://github.com/DILiS-lab/opt-synthdata-4tl.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1c7d62233a0ff735ce7ae98aa19a9ca10c7ff1" target='_blank'>
              Optimizing ODE-derived Synthetic Data for Transfer Learning in Dynamical Biological Systems
              </a>
            </td>
          <td>
            Julian Zabbarov, Simon Witzke, Maximilian Kleissl, Pascal Iversen, Bernhard Y. Renard, Katharina Baum
          </td>
          <td>2024-03-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Analyzing the drug response at the cellular level is crucial for identifying biomarkers and understanding the mechanisms of resistance. Although studies on the drug response of individual cells can provide novel insights into tumor heterogeneity, pharmacogenomic data related to single-cell (SC) RNA sequencing is often limited. Transfer learning provides a promising approach to translate the knowledge of drug response from bulk cell lines to SC analysis, potentially providing an effective solution to this challenge. Previous studies often use data from single drug-cell lines to pre-train specific models and adapt the models on SC datasets, which lack pharmacogenomic information from other drugs and hinder model generalization. In this work, we introduce MetaSCDrug as a unified meta pre-training framework that integrates molecular information with transcriptomic data to simultaneously modeling cellular heterogeneity in response to multiple pre-trained drugs and generalize to unseen drugs. Our model requires only one pre-training session, followed by fine-tuning on multiple single-cell datasets by few-shot learning, achieving an average of 4.58% accuracy increase in drug response prediction compared to the baselines. Furthermore, our meta pre-training strategy effectively captures transcriptome heterogeneity in the generalization of unseen drugs, achieving a 20% improvement over the model without meta pre-training. Case studies of our framework highlight its capability to identify critical genes for resistance, providing a method for exploring drug action pathways and understanding resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde20ee4a0135620d4f36e68f1c9aef919565e77" target='_blank'>
              MetaSCDrug: Meta-Transfer Learning for Single-Cell-Level Drug Response Prediction from Transcriptome and Molecular Representations
              </a>
            </td>
          <td>
            Shuang Ge, Shuqing Sun, Yiming Ren, Huan Xu, Zhixiang Ren
          </td>
          <td>2024-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Predicting and enhancing inherent properties based on molecular structures is paramount to design tasks in medicine, materials science, and environmental management. Most of the current machine learning and deep learning approaches have become standard for predictions, but they face challenges when applied across different datasets due to reliance on correlations between molecular representation and target properties. These approaches typically depend on large datasets to capture the diversity within the chemical space, facilitating a more accurate approximation, interpolation, or extrapolation of the chemical behavior of molecules. In our research, we introduce an active learning approach that discerns underlying cause-effect relationships through strategic sampling with the use of a graph loss function. This method identifies the smallest subset of the dataset capable of encoding the most information representative of a much larger chemical space. The identified causal relations are then leveraged to conduct systematic interventions, optimizing the design task within a chemical space that the models have not encountered previously. While our implementation focused on the QM9 quantum-chemical dataset for a specific design task-finding molecules with a large dipole moment-our active causal learning approach, driven by intelligent sampling and interventions, holds potential for broader applications in molecular, materials design and discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8364c8b2f5ffe4dd218578294dd6ed9a6345ce87" target='_blank'>
              Active Causal Learning for Decoding Chemical Complexities with Targeted Interventions
              </a>
            </td>
          <td>
            Zachary R. Fox, Ayana Ghosh
          </td>
          <td>2024-04-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) is widely used to assess drug effects and toxicity before the drug goes to clinical trial. However, significant manual distillation of the literature is needed in order to construct a QSP model. Parameters may need to be fit, and simplifying assumptions of the model need to be made. In this work, we apply Universal Physics-Informed Neural Networks (UPINNs) to learn unknown components of various differential equations that model chemotherapy pharmacodynamics. We learn three commonly employed chemotherapeutic drug actions (log-kill, Norton-Simon, and E_max) from synthetic data. Then, we use the UPINN method to fit the parameters for several synthetic datasets simultaneously. Finally, we learn the net proliferation rate in a model of doxorubicin (a chemotherapeutic) pharmacodynamics. As these are only toy examples, we highlight the usefulness of UPINNs in learning unknown terms in pharmacodynamic and pharmacokinetic models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b4e22d35a48c931e327882bbf735dcb7cc3fbc" target='_blank'>
              Learning Chemotherapy Drug Action via Universal Physics-Informed Neural Networks
              </a>
            </td>
          <td>
            Lena Podina, Ali Ghodsi, Mohammad Kohandel
          </td>
          <td>2024-04-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Motivation With the rapid advancement of single-cell sequencing technology, it becomes gradually possible to delve into the cellular responses to various external perturbations at the gene expression level. However, obtaining perturbed samples in certain scenarios may be considerably challenging, and the substantial costs associated with sequencing also curtail the feasibility of large-scale experimentation. A repertoire of methodologies has been employed for forecasting perturbative responses in single-cell gene expression. However, existing methods primarily focus on the average response of a specific cell type to perturbation, overlooking the single-cell specificity of perturbation responses and a more comprehensive prediction of the entire perturbation response distribution. Results Here, we present scPRAM, a method for predicting perturbation responses in single-cell gene expression based on attention mechanisms. Leveraging variational autoencoders and optimal transport, scPRAM aligns cell states before and after perturbation, followed by accurate prediction of gene expression responses to perturbations for unseen cell types through attention mechanisms. Experiments on multiple real perturbation datasets involving drug treatments and bacterial infections demonstrate that scPRAM attains heightened accuracy in perturbation prediction across cell types, species, and individuals, surpassing existing methodologies. Furthermore, scPRAM demonstrates outstanding capability in identifying differentially expressed genes under perturbation, capturing heterogeneity in perturbation responses across species, and maintaining stability in the presence of data noise and sample size variations. Availability and implementation https://github.com/jiang-q19/scPRAM and https://doi.org/10.5281/zenodo.10935038.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a7404da9e41fa69f1c46bb88fd5a87b6a5a437" target='_blank'>
              scPRAM accurately predicts single-cell gene expression perturbation response based on attention mechanism
              </a>
            </td>
          <td>
            Qun Jiang, Shengquan Chen, Xiaoyang Chen, Rui Jiang
          </td>
          <td>2024-04-15</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Motivation Personalized cancer treatments require accurate drug response predictions. Existing deep learning methods show promise but higher accuracy is needed to serve the purpose of precision medicine. The prediction accuracy can be improved with not only topology but geometrical information of drugs. Results A novel deep learning methodology for drug response prediction is presented, named Hi-GeoMVP. It synthesizes hierarchical drug representation with multi-omics data, leveraging graph neural networks and variational autoencoders for detailed drug and cell line representations. Multi-task learning is employed to make better prediction, while both 2D and 3D molecular representations capture comprehensive drug information. Testing on the GDSC dataset confirms Hi-GeoMVP’s enhanced performance, surpassing prior state-of-the-art methods by improving the Pearson correlation coefficient from 0.934 to 0.941 and decreasing the root mean square error from 0.969 to 0.931. In the case of blind test, Hi-GeoMVP demonstrated robustness, outperforming the best previous models with a superior Pearson correlation coefficient in the drug-blind test. These results underscore Hi-GeoMVP’s capabilities in drug response prediction, implying its potential for precision medicine. Availability and implementation The source code is available at https://github.com/matcyr/Hi-GeoMVP">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f37ad472d96f6de22b0dcbf5a9d25e3bec92b10" target='_blank'>
              Hi-GeoMVP: a hierarchical geometry-enhanced deep learning model for drug response prediction
              </a>
            </td>
          <td>
            Yurui Chen, Louxin Zhang
          </td>
          <td>2024-04-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Locally interacting dynamical systems, such as epidemic spread, rumor propagation through crowd, and forest fire, exhibit complex global dynamics originated from local, relatively simple, and often stochastic interactions between dynamic elements. Their temporal evolution is often driven by transitions between a finite number of discrete states. Despite significant advancements in predictive modeling through deep learning, such interactions among many elements have rarely explored as a specific domain for predictive modeling. We present Attentive Recurrent Neural Cellular Automata (AR-NCA), to effectively discover unknown local state transition rules by associating the temporal information between neighboring cells in a permutation-invariant manner. AR-NCA exhibits the superior generalizability across various system configurations (i.e., spatial distribution of states), data efficiency and robustness in extremely data-limited scenarios even in the presence of stochastic interactions, and scalability through spatial dimension-independent prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1388550b7dbb3f7941311d9f11f9bec99880c0bf" target='_blank'>
              Learning Locally Interacting Discrete Dynamical Systems: Towards Data-Efficient and Scalable Prediction
              </a>
            </td>
          <td>
            Beomseok Kang, H. Kumar, Minah Lee, Biswadeep Chakraborty, Saibal Mukhopadhyay
          </td>
          <td>2024-04-09</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Trajectory prediction is a cornerstone in autonomous driving (AD), playing a critical role in enabling vehicles to navigate safely and efficiently in dynamic environments. To address this task, this paper presents a novel trajectory prediction model tailored for accuracy in the face of heterogeneous and uncertain traffic scenarios. At the heart of this model lies the Characterized Diffusion Module, an innovative module designed to simulate traffic scenarios with inherent uncertainty. This module enriches the predictive process by infusing it with detailed semantic information, thereby enhancing trajectory prediction accuracy. Complementing this, our Spatio-Temporal (ST) Interaction Module captures the nuanced effects of traffic scenarios on vehicle dynamics across both spatial and temporal dimensions with remarkable effectiveness. Demonstrated through exhaustive evaluations, our model sets a new standard in trajectory prediction, achieving state-of-the-art (SOTA) results on the Next Generation Simulation (NGSIM), Highway Drone (HighD), and Macao Connected Autonomous Driving (MoCAD) datasets across both short and extended temporal spans. This performance underscores the model's unparalleled adaptability and efficacy in navigating complex traffic scenarios, including highways, urban streets, and intersections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a44b7dcd3d33f09e15acfcfc6237521273059346" target='_blank'>
              Characterized Diffusion and Spatial-Temporal Interaction Network for Trajectory Prediction in Autonomous Driving
              </a>
            </td>
          <td>
            Haicheng Liao, Xuelin Li, Yongkang Li, Hanlin Kong, Chengyue Wang, Bonan Wang, Yanchen Guan, Kahou Tam, Zhenning Li, Chengzhong Xu
          </td>
          <td>2024-05-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Predicting Remaining Useful Life (RUL) plays a crucial role in the prognostics and health management of industrial systems that involve a variety of interrelated sensors. Given a constant stream of time series sensory data from such systems, deep learning models have risen to prominence at identifying complex, nonlinear temporal dependencies in these data. In addition to the temporal dependencies of individual sensors, spatial dependencies emerge as important correlations among these sensors, which can be naturally modelled by a temporal graph that describes time-varying spatial relationships. However, the majority of existing studies have relied on capturing discrete snapshots of this temporal graph, a coarse-grained approach that leads to loss of temporal information. Moreover, given the variety of heterogeneous sensors, it becomes vital that such inherent heterogeneity is leveraged for RUL prediction in temporal sensor graphs. To capture the nuances of the temporal and spatial relationships and heterogeneous characteristics in an interconnected graph of sensors, we introduce a novel model named Temporal and Heterogeneous Graph Neural Networks (THGNN). Specifically, THGNN aggregates historical data from neighboring nodes to accurately capture the temporal dynamics and spatial correlations within the stream of sensor data in a fine-grained manner. Moreover, the model leverages Feature-wise Linear Modulation (FiLM) to address the diversity of sensor types, significantly improving the model's capacity to learn the heterogeneity in the data sources. Finally, we have validated the effectiveness of our approach through comprehensive experiments. Our empirical findings demonstrate significant advancements on the N-CMAPSS dataset, achieving improvements of up to 19.2% and 31.6% in terms of two different evaluation metrics over state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b4e301bda3ea480936b4a2e3038dcd0b960754d" target='_blank'>
              Temporal and Heterogeneous Graph Neural Network for Remaining Useful Life Prediction
              </a>
            </td>
          <td>
            Zhihao Wen, Yuan Fang, Pengcheng Wei, Fayao Liu, Zhenghua Chen, Min Wu
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="INTRODUCTION
Prediction of pharmacokinetic (PK) properties is crucial for drug discovery and development. Machine-learning (ML) models, which use statistical pattern recognition to learn correlations between input features (such as chemical structures) and target variables (such as PK parameters), are being increasingly used for this purpose. To embed ML models for PK prediction into workflows and to guide future development, a solid understanding of their applicability, advantages, limitations, and synergies with other approaches is necessary.


AREAS COVERED
This narrative review discusses the design and application of ML models to predict PK parameters of small molecules, especially in light of established approaches including in vitro-in vivo extrapolation (IVIVE) and physiologically based pharmacokinetic (PBPK) models. The authors illustrate scenarios in which the three approaches are used and emphasize how they enhance and complement each other. In particular, they highlight achievements, the state of the art and potentials of applying machine learning for PK prediction through a comphrehensive literature review.


EXPERT OPINION
ML models, when carefully crafted, regularly updated, and appropriately used, empower users to prioritize molecules with favorable PK properties. Informed practitioners can leverage these models to improve the efficiency of drug discovery and development process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4407f5485a20d2a2a26036c5e168a7a01ce78c14" target='_blank'>
              Another string to your bow: machine learning prediction of the pharmacokinetic properties of small molecules.
              </a>
            </td>
          <td>
            Davide Bassani, Neil John Parrott, Nenad Manevski, Jitao David Zhang
          </td>
          <td>2024-05-10</td>
          <td>Expert opinion on drug discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Reconstructing complex networks and predicting the dynamics are particularly challenging in real-world applications because the available information and data are incomplete. We develop a unified collaborative deep-learning framework consisting of three modules: network inference, state estimation, and dynamical learning. The complete network structure is first inferred and the states of the unobserved nodes are estimated, based on which the dynamical learning module is activated to determine the dynamical evolution rules. An alternating parameter updating strategy is deployed to improve the inference and prediction accuracy. Our framework outperforms baseline methods for synthetic and empirical networks hosting a variety of dynamical processes. A reciprocity emerges between network inference and dynamical prediction: better inference of network structure improves the accuracy of dynamical prediction, and vice versa. We demonstrate the superior performance of our framework on an influenza dataset consisting of 37 US States and a PM2.5 dataset covering 184 cities in China.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38794d512c02c8faee07849e126f9e6f2863fc85" target='_blank'>
              Deep-learning reconstruction of complex dynamical networks from incomplete data.
              </a>
            </td>
          <td>
            Xiao Ding, Ling-Wei Kong, Haifeng Zhang, Ying-Cheng Lai
          </td>
          <td>2024-04-01</td>
          <td>Chaos</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Molecular representation learning (MRL) is a specialized field in which deep-learning models condense essential molecular information into a vectorized form. Whereas recent research has predominantly emphasized drug discovery and bioactivity applications, MRL holds significant potential for diverse chemical properties beyond these contexts. The recently published study by King-Smith introduces a novel application of molecular representation training and compellingly demonstrates its value in predicting molecular properties (E. King-Smith, Chem. Sci., 2024, https://doi.org/10.1039/D3SC04928K). In this focus article, we will briefly delve into MRL in chemistry and the significance of King-Smith's work within the dynamic landscape of this evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790831b1859645ef6b64a5f8ecf77fdd1e97378d" target='_blank'>
              A focus on molecular representation learning for the prediction of chemical properties
              </a>
            </td>
          <td>
            Yonatan Harnik, Anat Milo
          </td>
          <td>2024-03-25</td>
          <td>Chemical Science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Modern graph representation learning works mostly under the assumption of dealing with regularly sampled temporal graph snapshots, which is far from realistic, e.g., social networks and physical systems are characterized by continuous dynamics and sporadic observations. To address this limitation, we introduce the Temporal Graph Ordinary Differential Equation (TG-ODE) framework, which learns both the temporal and spatial dynamics from graph streams where the intervals between observations are not regularly spaced. We empirically validate the proposed approach on several graph benchmarks, showing that TG-ODE can achieve state-of-the-art performance in irregular graph stream tasks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95463223b2faba0ccf928714ba4373a07dee1860" target='_blank'>
              Temporal Graph ODEs for Irregularly-Sampled Time Series
              </a>
            </td>
          <td>
            Alessio Gravina, Daniele Zambon, Davide Bacciu, C. Alippi
          </td>
          <td>2024-04-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Background The potential benefits of drug combination synergy in cancer medicine are significant, yet the risks must be carefully managed due to the possibility of increased toxicity. Although artificial intelligence applications have demonstrated notable success in predicting drug combination synergy, several key challenges persist: (1) Existing models often predict average synergy values across a restricted range of testing dosages, neglecting crucial dose amounts and the mechanisms of action of the drugs involved. (2) Many graph-based models rely on static protein-protein interactions, failing to adapt to dynamic and context-dependent networks. This limitation constrains the applicability of current methods. Results We introduced SAFER, a Sub-hypergraph Attention-based graph model, addressing these issues by incorporating complex relationships among biological knowledge networks and considering dosing effects on subject-specific networks. SAFER outperformed previous models on the benchmark and the independent test set. The analysis of subgraph attention weight for the lung cancer cell line highlighted JAK-STAT signaling pathway, PRDM12, ZNF781, and CDC5L that have been implicated in lung fibrosis. Conclusions SAFER presents an interpretable framework designed to identify drug-responsive signals. Tailored for comprehending dose effects on subject-specific molecular contexts, our model uniquely captures dose-level drug combination responses. This capability unlocks previously inaccessible avenues of investigation compared to earlier models. Finally, the SAFER framework can be leveraged by future inquiries to investigate molecular networks that uniquely characterize individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/500c4f42640ef697e84b8ae2a633d54a79da30c0" target='_blank'>
              SAFER: sub-hypergraph attention-based neural network for predicting effective responses to dose combinations
              </a>
            </td>
          <td>
            yiching tang, Rongbin Li, Jing Tang, W. J. Zheng, Xiaoqian Jiang
          </td>
          <td>2024-04-30</td>
          <td>Research Square</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Physics-informed machine learning emerges as a transformative approach, bridging the gap between the high fidelity of mechanistic models and the adaptive, data-driven insights afforded by artificial intelligence and machine learning. In the realm of chemical reaction network modeling, this synergy is particularly valuable. It offers a solution to the pro-hibitive computational costs associated with detailed mechanistic models, while also capitalizing on the predictive power and flexibility of machine learning algorithms. This study exemplifies this innovative fusion by applying it to the critical biomedical challenge of Aβ fibril aggregation, shedding light on the mechanisms underlying Alzheimer’s disease. A corner-stone of this research is the introduction of an automatic reaction order model reduction framework, tailored to optimize the scale of reduced order kinetic models. This framework is not merely a technical enhancement; it represents a paradigm shift in how models are constructed and refined. By automatically determining the most appropriate level of detail for modeling reaction networks, our proposed approach significantly enhances the efficiency and accuracy of simulations. This is particularly crucial for systems like Aβ aggregation, where the precise characterization of nucleation and growth kinetics can provide insights into potential therapeutic targets. The potential generalizability of this automatic model reduction technique to other network models is a key highlight of this study. The methodology developed here has far-reaching implications, offering a scalable and adaptable tool for a wide range of applications beyond biomedical research. The ability to dynamically adjust model complexity in response to the specific demands of the system under study is a powerful asset. This flexibility ensures that the models remain both computationally feasible and scientifically relevant, capable of accommodating new data and evolving understandings of complex phenomena.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c94bd71338acf8a4f759c45587c85b0b03c79f0e" target='_blank'>
              Physics-informed machine learning for automatic model reduction in chemical reaction networks
              </a>
            </td>
          <td>
            J. Pateras, Colin Zhang, Shriya Majumdar, Ayush Pal, Preetam Ghosh
          </td>
          <td>2024-03-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Patients diagnosed with advanced metastatic colorectal cancer (mCRC) often exhibit heterogeneous disease progression and face poor survival prospects. In order to comprehensively analyze the varied treatment responses among individuals and the challenge of tumor recurrence resistant to drugs in advanced mRCR, we developed a novel quantitative cancer-immunity cycle (QCIC) model. The proposed model was meticulously crafted utilizing a blend of differential equations and randomized modeling techniques to quantitatively elucidate the intricate mechanisms governing the cancer-immunity cycle and forecast tumor dynamics under different treatment modalities. Furthermore, by integrating diverse clinical datasets and rigorous model analyses, we introduced two pivotal concepts: the treatment response index (TRI) and the death probability function (DPF). These concepts are crucial tools for translating model predictions into clinically relevant evaluation indexes. Using virtual patient technology, we extrapolated tumor predictive biomarkers from the model to predict survival outcomes for mCRC patients. Our findings underscore the significance of tumor-infiltrating CD8+CTL cell density as a key predictive biomarker for short-term treatment responses in advanced mCRC while emphasizing the potential predict value of the tumor-infiltrating CD4+Th1/Treg ratio in determining patient survival. This study presents a pioneering methodology bridging the divide between diverse clinical data sources and the generation of virtual patients, offering invaluable insights into understanding inter-individual treatment variations and forecasting survival outcomes in mCRC patients. Author summary This study introduces a sophisticated modeling approach to delineate the intricate dynamics of tumor-immune interactions within the cancer-immunity cycle. Utilizing a multi-compartmental, multi-scale, multi-dimensional differential equation model, we quantified the complex interplay between tumor cells, immune cells, cytokines, and chemokines. By integrating virtual patient technology, we have conducted in silico clinical trials that accurately predict disease progression across multiple treatment modalities for cancer patients, particularly in advanced mCRC. Through the combination of differential equations and randomized modeling, we effectively captured the diverse treatment responses and clinical manifestations of drug-resistant tumor recurrence. Furthermore, we explored the pivotal role of predictive biomarkers in determining patients’ survival prognosis, offering insights for tailored therapeutic strategies. Importantly, our computational framework holds promise for the extension to the investigation of patients with other tumor types, thus contributing to the personalized investigation of patients with other tumor types and cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a0274da7051ba17dd2effa0a322076737f25c2" target='_blank'>
              Quantitative cancer-immunity cycle modeling for predicting disease progression in advanced metastatic colorectal cancer
              </a>
            </td>
          <td>
            Chenghang Li, Yongchang Wei, J. Lei
          </td>
          <td>2024-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Recent advancements in deep learning (DL) have played a crucial role in aiding experts to develop personalized healthcare services, particularly in drug response prediction (DRP) for cancer patients. The DL’s techniques contribution to this field is significant, and they have proven indispensable in the medical field. This review aims to analyze the diverse effectiveness of various DL models in making these predictions, drawing on research published from 2017 to 2023. We utilized the VOS-Viewer 1.6.18 software to create a word cloud from the titles and abstracts of the selected studies. This study offers insights into the focus areas within DL models used for drug response. The word cloud revealed a strong link between certain keywords and grouped themes, highlighting terms such as deep learning, machine learning, precision medicine, precision oncology, drug response prediction, and personalized medicine. In order to achieve an advance in DRP using DL, the researchers need to work on enhancing the models’ generalizability and interoperability. It is also crucial to develop models that not only accurately represent various architectures but also simplify these architectures, balancing the complexity with the predictive capabilities. In the future, researchers should try to combine methods that make DL models easier to understand; this will make DRP reviews more open and help doctors trust the decisions made by DL models in cancer DRP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbf7aa2fca70809d02c658e412ce888f4fc4ad94" target='_blank'>
              Optimized models and deep learning methods for drug response prediction in cancer treatments: a review
              </a>
            </td>
          <td>
            W. I. Hajim, Suhaila Zainudin, K. M. Daud, Khattab Alheeti
          </td>
          <td>2024-03-25</td>
          <td>PeerJ Computer Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The proteins within the human epidermal growth factor receptor (EGFR) family, members of the tyrosine kinase receptor family, play a pivotal role in the molecular mechanisms driving the development of various tumors. Tyrosine kinase inhibitors, key compounds in targeted therapy, encounter challenges in cancer treatment due to emerging drug resistance mutations. Consequently, machine learning has undergone significant evolution to address the challenges of cancer drug discovery related to EGFR family proteins. However, the application of deep learning in this area is hindered by inherent difficulties associated with small-scale data, particularly the risk of overfitting. Moreover, the design of a model architecture that facilitates learning through multi-task and transfer learning, coupled with appropriate molecular representation, poses substantial challenges. In this study, we introduce GraphEGFR, a deep learning regression model designed to enhance molecular representation and model architecture for predicting the bioactivity of inhibitors against both wild-type and mutant EGFR family proteins. GraphEGFR integrates a graph attention mechanism for molecular graphs with deep and convolutional neural networks for molecular fingerprints. We observed that GraphEGFR models employing multi-task and transfer learning strategies generally achieve predictive performance comparable to existing competitive methods. The integration of molecular graphs and fingerprints adeptly captures relationships between atoms and enables both global and local pattern recognition. We further validated potential multi-targeted inhibitors for wild-type and mutant HER1 kinases, exploring key amino acid residues through molecular dynamics simulations to understand molecular interactions. This predictive model offers a robust strategy that could significantly contribute to overcoming the challenges of developing deep learning models for drug discovery with limited data and exploring new frontiers in multi-targeted kinase drug discovery for EGFR family proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a60c5741929cb3c480ef85e0d599beecfa969e6" target='_blank'>
              GraphEGFR: Multi-task and transfer learning based on molecular graph attention mechanism and fingerprints improving inhibitor bioactivity prediction for EGFR family proteins on data scarcity.
              </a>
            </td>
          <td>
            Bundit Boonyarit, Nattawin Yamprasert, Pawit Kaewnuratchadasorn, Jiramet Kinchakawat, Chanatkran Prommin, T. Rungrotmongkol, Sarana Nutanong
          </td>
          <td>2024-05-07</td>
          <td>Journal of computational chemistry</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Deep generative models have been accelerating the inverse design process in material and drug design. Unlike their counterpart property predictors in typical molecular design frameworks, generative molecular design models have seen fewer efforts on uncertainty quantification (UQ) due to computational challenges in Bayesian inference posed by their large number of parameters. In this work, we focus on the junction-tree variational autoencoder (JT-VAE), a popular model for generative molecular design, and address this issue by leveraging the low dimensional active subspace to capture the uncertainty in the model parameters. Specifically, we approximate the posterior distribution over the active subspace parameters to estimate the epistemic model uncertainty in an extremely high dimensional parameter space. The proposed UQ scheme does not require alteration of the model architecture, making it readily applicable to any pre-trained model. Our experiments demonstrate the efficacy of the AS-based UQ and its potential impact on molecular optimization by exploring the model diversity under epistemic uncertainty.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a70f6922f6ea468e7916a2cd53f69787e147d2e3" target='_blank'>
              Leveraging Active Subspaces to Capture Epistemic Model Uncertainty in Deep Generative Models for Molecular Design
              </a>
            </td>
          <td>
            A. N. M. N. Abeer, Sanket R. Jantre, Nathan M. Urban, Byung-Jun Yoon
          </td>
          <td>2024-04-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background  Pharmacogenomics is an emerging field that explores the relationship between a person’s genetic makeup and their response to drugs. 
Objectives: Computer applications have been instrumental in advancing pharmacogenomics research and facilitating personalized medicine. In this abstract, we review some of the key computer applications used in pharmacogenomics research, including bioinformatics tools for analyzing genetic data, machine learning algorithms for predicting drug response, and clinical decision support systems for guiding drug selection and dosing. We also highlight the challenges and opportunities in the field of pharmacogenomics, including the need for large-scale, multi-omics datasets, and the importance of developing robust and interpretable machine learning models. Overall, computer applications have played a critical role in enabling pharmacogenomics research and have the potential to revolutionize the practice of medicine by facilitating the development of more personalized and effective drug therapies. 
Bangladesh Journal of Medical Science Vol. 23 No. 02 April’24 Page : 300-308">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1572e7449105e139d30e7eb460955a688d5510fb" target='_blank'>
              Application of computer on Pharmacogenomics
              </a>
            </td>
          <td>
            M. Amanat, Randhir Singh
          </td>
          <td>2024-03-27</td>
          <td>Bangladesh Journal of Medical Science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The COVID-19 pandemic has reshaped global health and societal norms, necessitating effective tracking and forecasting methods. We propose a novel machine learning model that integrates diverse datasets, including epidemiological, demographic, and environmental variables. By employing advanced techniques such as deep learning and time-series analysis, our model accurately predicts the pandemic's trajectory, considering factors like non-pharmaceutical interventions and variant emergence. Through comprehensive data integration and feature engineering, our approach provides actionable insights for policymakers, aiding in proactive response strategies and adaptive tracking of COVID-19 propagation dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5f1aabce2f232887a9d95b94a3e38a296e64465" target='_blank'>
              Computational Tracking and Estimating the Covid-19 Dynamic Broadcast Based on Machine Learning Model
              </a>
            </td>
          <td>
            Dr.Manimozhi I
          </td>
          <td>2024-05-08</td>
          <td>International Journal of Innovative Research in Information Security</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Interventions play a pivotal role in the study of complex biological systems. In drug discovery, genetic interventions (such as CRISPR base editing) have become central to both identifying potential therapeutic targets and understanding a drug's mechanism of action. With the advancement of CRISPR and the proliferation of genome-scale analyses such as transcriptomics, a new challenge is to navigate the vast combinatorial space of concurrent genetic interventions. Addressing this, our work concentrates on estimating the effects of pairwise genetic combinations on the cellular transcriptome. We introduce two novel contributions: Salt, a biologically-inspired baseline that posits the mostly additive nature of combination effects, and Peper, a deep learning model that extends Salt's additive assumption to achieve unprecedented accuracy. Our comprehensive comparison against existing state-of-the-art methods, grounded in diverse metrics, and our out-of-distribution analysis highlight the limitations of current models in realistic settings. This analysis underscores the necessity for improved modelling techniques and data acquisition strategies, paving the way for more effective exploration of genetic intervention effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e42acd4e6c209067c51c52b168ecb09fa85ef100" target='_blank'>
              Season combinatorial intervention predictions with Salt&Peper
              </a>
            </td>
          <td>
            Thomas Gaudelet, Alice Del Vecchio, Eli M. Carrami, Juliana Cudini, Chantriolnt-Andreas Kapourani, Caroline Uhler, Lindsay Edwards
          </td>
          <td>2024-04-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Data-driven modelling and scientific machine learning have been responsible for significant advances in determining suitable models to describe data. Within dynamical systems, neural ordinary differential equations (ODEs), where the system equations are set to be governed by a neural network, have become a popular tool for this challenge in recent years. However, less emphasis has been placed on systems that are only partially-observed. In this work, we employ a hybrid neural ODE structure, where the system equations are governed by a combination of a neural network and domain-specific knowledge, together with symbolic regression (SR), to learn governing equations of partially-observed dynamical systems. We test this approach on two case studies: A 3-dimensional model of the Lotka-Volterra system and a 5-dimensional model of the Lorenz system. We demonstrate that the method is capable of successfully learning the true underlying governing equations of unobserved states within these systems, with robustness to measurement noise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5f91324b2ca816ed10e0d9a1d6565fb12a4a1f" target='_blank'>
              Learning Governing Equations of Unobserved States in Dynamical Systems
              </a>
            </td>
          <td>
            Gevik Grigorian, Sandip V. George, S. Arridge
          </td>
          <td>2024-04-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Neural network models have become a popular machine-learning technique for the toxicity prediction of chemicals. However, due to their complex structure, it is difficult to understand predictions made by these models which limits confidence. Current techniques to tackle this problem such as SHAP or integrated gradients provide insights by attributing importance to the input features of individual compounds. While these methods have produced promising results in some cases, they do not shed light on how representations of compounds are transformed in hidden layers, which constitute how neural networks learn. We present a novel technique to interpret neural networks which identifies chemical substructures in training data found to be responsible for the activation of hidden neurons. For individual test compounds, the importance of hidden neurons is determined, and the associated substructures are leveraged to explain the model prediction. Using structural alerts for mutagenicity from the Derek Nexus expert system as ground truth, we demonstrate the validity of the approach and show that model explanations are competitive with and complementary to explanations obtained from an established feature attribution method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4942ed910678bf3396a6c139b299a1276cbf3ca8" target='_blank'>
              Interpreting Neural Network Models for Toxicity Prediction by Extracting Learned Chemical Features
              </a>
            </td>
          <td>
            Moritz Walter, Samuel J. Webb, Valerie J Gillet
          </td>
          <td>2024-04-30</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Irregularly sampled time series with missing values are often observed in multiple real-world applications such as healthcare, climate and astronomy. They pose a significant challenge to standard deep learn- ing models that operate only on fully observed and regularly sampled time series. In order to capture the continuous dynamics of the irreg- ular time series, many models rely on solving an Ordinary Differential Equation (ODE) in the hidden state. These ODE-based models tend to perform slow and require large memory due to sequential operations and a complex ODE solver. As an alternative to complex ODE-based mod- els, we propose a family of models called Functional Latent Dynamics (FLD). Instead of solving the ODE, we use simple curves which exist at all time points to specify the continuous latent state in the model. The coefficients of these curves are learned only from the observed values in the time series ignoring the missing values. Through extensive experi- ments, we demonstrate that FLD achieves better performance compared to the best ODE-based model while reducing the runtime and memory overhead. Specifically, FLD requires an order of magnitude less time to infer the forecasts compared to the best performing forecasting model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2a5b8158db29854109275cb5c3fbcf47c080c1c" target='_blank'>
              Functional Latent Dynamics for Irregularly Sampled Time Series Forecasting
              </a>
            </td>
          <td>
            Christian Klotergens, Vijaya Krishna Yalavarthi, Maximilian Stubbemann, Lars Schmidt-Thieme
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The rapid integration of machine learning methodologies in healthcare has ignited innovative strategies for disease prediction, particularly with the vast repositories of Electronic Health Records (EHR) data. This article delves into the realm of multi-disease prediction, presenting a comprehensive study that introduces a pioneering ensemble feature selection model. This model, designed to optimize learning systems, combines statistical, deep, and optimally selected features through the innovative Stabilized Energy Valley Optimization with Enhanced Bounds (SEV-EB) algorithm. The objective is to achieve unparalleled accuracy and stability in predicting various disorders. This work proposes an advanced ensemble model that synergistically integrates statistical, deep, and optimally selected features. This combination aims to enhance the predictive power of the model by capturing diverse aspects of the health data. At the heart of the proposed model lies the SEV-EB algorithm, a novel approach to optimal feature selection. The algorithm introduces enhanced bounds and stabilization techniques, contributing to the robustness and accuracy of the overall prediction model. To further elevate the predictive capabilities, an HSC-AttentionNet is introduced. This network architecture combines deep temporal convolution capabilities with LSTM, allowing the model to capture both short-term patterns and long-term dependencies in health data. Rigorous evaluations showcase the remarkable performance of the proposed model. Achieving a 95% accuracy and 94% F1-score in predicting various disorders, the model surpasses traditional methods, signifying a significant advancement in disease prediction accuracy. The implications of this research extend beyond the confines of academia. By harnessing the wealth of information embedded in EHR data, the proposed model presents a paradigm shift in healthcare interventions. The optimized diagnosis and treatment pathways facilitated by this approach hold promise for more accurate and personalized healthcare, potentially revolutionizing patient outcomes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4af998a709f38b6d0ae294ebde5bf6f9e316068" target='_blank'>
              Optimizing Disease Prediction with Artificial Intelligence Driven Feature Selection and Attention Networks
              </a>
            </td>
          <td>

          </td>
          <td>2024-04-04</td>
          <td>Journal of Electrical Systems</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This paper introduces a trajectory prediction model tailored for autonomous driving, focusing on capturing complex interactions in dynamic traffic scenarios without reliance on high-definition maps. The model, termed MFTraj, harnesses historical trajectory data combined with a novel dynamic geometric graph-based behavior-aware module. At its core, an adaptive structure-aware interactive graph convolutional network captures both positional and behavioral features of road users, preserving spatial-temporal intricacies. Enhanced by a linear attention mechanism, the model achieves computational efficiency and reduced parameter overhead. Evaluations on the Argoverse, NGSIM, HighD, and MoCAD datasets underscore MFTraj's robustness and adaptability, outperforming numerous benchmarks even in data-challenged scenarios without the need for additional information such as HD maps or vectorized maps. Importantly, it maintains competitive performance even in scenarios with substantial missing data, on par with most existing state-of-the-art models. The results and methodology suggest a significant advancement in autonomous driving trajectory prediction, paving the way for safer and more efficient autonomous systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1239e1b7c2b543f557179d824a304cbf2cbf59d1" target='_blank'>
              MFTraj: Map-Free, Behavior-Driven Trajectory Prediction for Autonomous Driving
              </a>
            </td>
          <td>
            Haicheng Liao, Zhenning Li, Chengyue Wang, Huanming Shen, Bonan Wang, Dongping Liao, Guofa Li, Chengzhong Xu
          </td>
          <td>2024-05-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Dynamics of many complex systems, from weather and climate to spread of infectious diseases, can be described by partial differential equations (PDEs). Such PDEs involve unknown function(s), partial derivatives, and typically multiple independent variables. The traditional numerical methods for solving PDEs assume that the data are observed on a regular grid. However, in many applications, for example, weather and air pollution monitoring delivered by the arbitrary located weather stations of the National Weather Services, data records are irregularly spaced. Furthermore, in problems involving prediction analytics such as forecasting wildfire smoke plumes, the primary focus may be on a set of irregular locations associated with urban development. In recent years, deep learning (DL) methods and, in particular, graph neural networks (GNNs) have emerged as a new promising tool that can complement traditional PDE solvers in scenarios of the irregular spaced data, contributing to the newest research trend of physics informed machine learning (PIML). However, most existing PIML methods tend to be limited in their ability to describe higher dimensional structural properties exhibited by real world phenomena, especially, ones that live on manifolds. To address this fundamental challenge, we bring the elements of the Hodge theory and, in particular, simplicial convolution defined on the Hodge Laplacian to the emerging nexus of DL and PDEs. In contrast to conventional Laplacian and the associated convolution operation, the simplicial convolution allows us to rigorously describe diffusion across higher order structures and to better approximate the complex underlying topology and geometry of the data. The new approach, Simplicial Neural Networks for Partial Differential Equations (SNN PDE) offers a computationally efficient yet effective solution for time dependent PDEs. Our studies of a broad range of synthetic data and wildfire processes demonstrate that SNN PDE improves upon state of the art baselines in handling unstructured grids and irregular time intervals of complex physical systems and offers competitive forecasting capabilities for weather and air quality forecasting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2928c9c56daf45d01b8df307cb7baf81ffbc7bc" target='_blank'>
              SNN-PDE: Learning Dynamic PDEs from Data with Simplicial Neural Networks
              </a>
            </td>
          <td>
            J. Choi, Yuzhou Chen, Huikyo Lee, Hyun Kim, Yulia R. Gel
          </td>
          <td>2024-03-24</td>
          <td>DBLP</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bb5d08a605e13f3aee7409ac89075d346310599" target='_blank'>
              MISATO: machine learning dataset of protein-ligand complexes for structure-based drug discovery.
              </a>
            </td>
          <td>
            Till Siebenmorgen, Filipe Menezes, S. Benassou, Erinc Merdivan, Kieran Didi, André Santos Dias Mourão, Radosław Kitel, Pietro Lio, Stefan Kesselheim, M. Piraud, F. Theis, Michael Sattler, G. Popowicz
          </td>
          <td>2024-05-10</td>
          <td>Nature computational science</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="The discovery of drug-target interactions (DTIs) plays a crucial role in pharmaceutical development. The deep learning model achieves more accurate results in DTI prediction due to its ability to extract robust and expressive features from drug and target chemical structures. However, existing deep learning methods typically generate drug features via aggregating molecular atom representations, ignoring the chemical properties carried by motifs, i.e., substructures of the molecular graph. The atom-drug double-level molecular representation learning can not fully exploit structure information and fails to interpret the DTI mechanism from the motif perspective. In addition, sequential model-based target feature extraction either fuses limited contextual information or requires expensive computational resources. To tackle the above issues, we propose a hierarchical graph representation learning-based DTI prediction method (HiGraphDTI). Specifically, HiGraphDTI learns hierarchical drug representations from triple-level molecular graphs to thoroughly exploit chemical information embedded in atoms, motifs, and molecules. Then, an attentional feature fusion module incorporates information from different receptive fields to extract expressive target features.Last, the hierarchical attention mechanism identifies crucial molecular segments, which offers complementary views for interpreting interaction mechanisms. The experiment results not only demonstrate the superiority of HiGraphDTI to the state-of-the-art methods, but also confirm the practical ability of our model in interaction interpretation and new DTI discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f199a680fe31a9cd2c0f4793abe2bc09da4f91f" target='_blank'>
              HiGraphDTI: Hierarchical Graph Representation Learning for Drug-Target Interaction Prediction
              </a>
            </td>
          <td>
            Bin Liu, Siqi Wu, Jin Wang, Xin Deng, Ao Zhou
          </td>
          <td>2024-04-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Neural Ordinary Differential Equations typically struggle to generalize to new dynamical behaviors created by parameter changes in the underlying system, even when the dynamics are close to previously seen behaviors. The issue gets worse when the changing parameters are unobserved, i.e., their value or influence is not directly measurable when collecting data. We introduce Neural Context Flow (NCF), a framework that encodes said unobserved parameters in a latent context vector as input to a vector field. NCFs leverage differentiability of the vector field with respect to the parameters, along with first-order Taylor expansion to allow any context vector to influence trajectories from other parameters. We validate our method and compare it to established Multi-Task and Meta-Learning alternatives, showing competitive performance in mean squared error for in-domain and out-of-distribution evaluation on the Lotka-Volterra, Glycolytic Oscillator, and Gray-Scott problems. This study holds practical implications for foundational models in science and related areas that benefit from conditional neural ODEs. Our code is openly available at https://github.com/ddrous/ncflow.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c738e0450649ed6c04ff7e82d993987c381e35a" target='_blank'>
              Neural Context Flows for Learning Generalizable Dynamical Systems
              </a>
            </td>
          <td>
            Roussel Desmond Nzoyem, David A.W. Barton, Tom Deakin
          </td>
          <td>2024-05-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract: An essential issue in computational personalised medicine is the prediction of drug responses. There have been several proposals for approaches to this problem that rely on machine learning, particularly deep learning. Nevertheless, these approaches often portray the medications as strings, an implausible representation of molecules. Furthermore, there has been a lack of comprehensive consideration of interpretation, such as whether mutations or copy number aberrations contribute to the medication response. Graph DRP, a new approach based on graph convolution networks, is suggested as a solution to the issue in this research. Cell lines were displayed as double vectors of genetic abnormalities in Graph DRP, whereas medications were shown as sub-atomic chartsthat straightforwardly caught the bonds among particles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b5611d40b7b831c904a8d7225c02e0bd1010dd" target='_blank'>
              Drug Response Prediction Using XGBOOST
              </a>
            </td>
          <td>
            P. Senthil
          </td>
          <td>2024-03-31</td>
          <td>International Journal for Research in Applied Science and Engineering Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Network inference is used to model transcriptional, signaling, and metabolic interactions among genes, proteins, and metabolites that identify biological pathways influencing disease pathogenesis. Advances in machine learning (ML)-based inference models exhibit the predictive capabilities of capturing latent patterns in genomic data. Such models are emerging as an alternative to the statistical models identifying causative factors driving complex diseases. We present CoVar, an ML-based framework that builds upon the properties of existing inference models, to find the central genes driving perturbed gene expression across biological states. Unlike differentially expressed genes (DEGs) that capture changes in individual gene expression across conditions, CoVar focuses on identifying variational genes that undergo changes in their expression network interaction profiles, providing insights into changes in the regulatory dynamics, such as in disease pathogenesis. Subsequently, it finds core genes from among the nearest neighbors of these variational genes, which are central to the variational activity and influence the coordinated regulatory processes underlying the observed changes in gene expression. Through the analysis of simulated as well as yeast expression data perturbed by the deletion of the mitochondrial genome, we show that CoVar captures the intrinsic variationality and modularity in the expression data, identifying key driver genes not found through existing differential analysis methodologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d91de570e66b22b6dcb9a4caf542415ea36c0206" target='_blank'>
              CoVar: A generalizable machine learning approach to identify the coordinated regulators driving variational gene expression.
              </a>
            </td>
          <td>
            Satyaki Roy, S. Sheikh, T. Furey
          </td>
          <td>2024-04-17</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Advanced machine learning methods capable of capturing complex and nonlinear relationships can be used in biomedical research to accurately predict time-to-event outcomes. However, these methods have been criticized as "black boxes" that are not interpretable and thus are difficult to trust in making important clinical decisions. Explainable machine learning proposes the use of model-agnostic explainers that can be applied to predictions from any complex model. These explainers describe how a patient's characteristics are contributing to their prediction, and thus provide insight into how the model is arriving at that prediction. The specific application of these explainers to survival prediction models can be used to obtain explanations for (i) survival predictions at particular follow-up times, and (ii) a patient's overall predicted survival curve. Here, we present a model-agnostic approach for obtaining these explanations from any survival prediction model. We extend the local interpretable model-agnostic explainer framework for classification outcomes to survival prediction models. Using simulated data, we assess the performance of the proposed approaches under various settings. We illustrate application of the new methodology using prostate cancer data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9b902c873bb51f0b7991aa1779eff3ba3d6183c" target='_blank'>
              Model-agnostic explanations for survival prediction models.
              </a>
            </td>
          <td>
            Krithika Suresh, Carsten Görg, Debashis Ghosh
          </td>
          <td>2024-03-26</td>
          <td>Statistics in medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer, a leading cause of death globally, occurs due to genomic changes and manifests heterogeneously across patients. To advance research on personalized treatment strategies, the effectiveness of various drugs on cells derived from cancers (`cell lines') is experimentally determined in laboratory settings. Nevertheless, variations in the distribution of genomic data and drug responses between cell lines and humans arise due to biological and environmental differences. Moreover, while genomic profiles of many cancer patients are readily available, the scarcity of corresponding drug response data limits the ability to train machine learning models that can predict drug response in patients effectively. Recent cancer drug response prediction methods have largely followed the paradigm of unsupervised domain-invariant representation learning followed by a downstream drug response classification step. Introducing supervision in both stages is challenging due to heterogeneous patient response to drugs and limited drug response data. This paper addresses these challenges through a novel representation learning method in the first phase and weak supervision in the second. Experimental results on real patient data demonstrate the efficacy of our method (WISER) over state-of-the-art alternatives on predicting personalized drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28ee9b9ea8cc0a9f28f3c21fa707bc8f6007d90" target='_blank'>
              WISER: Weak supervISion and supErvised Representation learning to improve drug response prediction in cancer
              </a>
            </td>
          <td>
            Kumar Shubham, A. Jayagopal, Syed Mohammed Danish, AP Prathosh, V. Rajan
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d88d1ee244b5bbb95c70dc410ecf9f282eee608" target='_blank'>
              Prediction of anticancer drug sensitivity using an interpretable model guided by deep learning
              </a>
            </td>
          <td>
            Weixiong Pang, Ming Chen, Yufang Qin
          </td>
          <td>2024-05-09</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Information diffusion prediction plays a crucial role in understanding the propagation of information in social networks, encompassing both macroscopic and microscopic prediction tasks. Macroscopic prediction estimates the overall impact of information diffusion, while microscopic prediction focuses on identifying the next user to be influenced. While prior research often concentrates on one of these aspects, a few tackle both concurrently. These two tasks provide complementary insights into the diffusion process at different levels, revealing common traits and unique attributes. The exploration of leveraging common features across these tasks to enhance information prediction remains an underexplored avenue. In this paper, we propose an intuitive and effective model that addresses both macroscopic and microscopic prediction tasks. Our approach considers the interactions and dynamics among cascades at the macro level and incorporates the social homophily of users in social networks at the micro level. Additionally, we introduce adversarial training and orthogonality constraints to ensure the integrity of shared features. Experimental results on four datasets demonstrate that our model significantly outperforms state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1e23cc5b8d054d01a30a67dc7c33cd0fd49df5b" target='_blank'>
              Enhancing Multi-Scale Diffusion Prediction via Sequential Hypergraphs and Adversarial Learning
              </a>
            </td>
          <td>
            Pengfei Jiao, Hongqian Chen, Qing Bao, Wang Zhang, Huamin Wu
          </td>
          <td>2024-03-24</td>
          <td>DBLP</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="We present a study where predictive mechanistic modeling is used in combination with deep learning methods to predict individual patient survival probabilities under immune checkpoint inhibitor (ICI) therapy. This hybrid approach enables prediction based on both measures that are calculable from mechanistic models (but may not be directly measurable in the clinic) and easily measurable quantities or characteristics (that are not always readily incorporated into predictive mechanistic models). The mechanistic model we have applied here can predict tumor response from CT or MRI imaging based on key mechanisms underlying checkpoint inhibitor therapy, and in the present work, its parameters were combined with readily-available clinical measures from 93 patients into a hybrid training set for a deep learning time-to-event predictive model. Analysis revealed that training an artificial neural network with both mechanistic modeling-derived and clinical measures achieved higher per-patient predictive accuracy based on event-time concordance, Brier score, and negative binomial log-likelihood-based criteria than when only mechanistic model-derived values or only clinical data were used. Feature importance analysis revealed that both clinical and model-derived parameters play prominent roles in neural network decision making, and in increasing prediction accuracy, further supporting the advantage of our hybrid approach. We anticipate that many existing mechanistic models may be hybridized with deep learning methods in a similar manner to improve predictive accuracy through addition of additional data that may not be readily implemented in mechanistic descriptions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7b3da8173ee0e944bcc41b5f5c4c90ff6088d34" target='_blank'>
              Hybridizing mechanistic mathematical modeling with deep learning methods to predict individual cancer patient survival after immune checkpoint inhibitor therapy
              </a>
            </td>
          <td>
            J. D. Butner, P. Dogra, Caroline Chung, Eugene Koay, James Welsh, David Hong, Vittorio Cristini, Zhihui Wang
          </td>
          <td>2024-03-29</td>
          <td>Research Square</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Drug repurposing offers a viable strategy for discovering new drugs and therapeutic targets through the analysis of drug–gene interactions. However, traditional experimental methods are plagued by their costliness and inefficiency. Despite graph convolutional network (GCN)-based models’ state-of-the-art performance in prediction, their reliance on supervised learning makes them vulnerable to data sparsity, a common challenge in drug discovery, further complicating model development. In this study, we propose SGCLDGA, a novel computational model leveraging graph neural networks and contrastive learning to predict unknown drug–gene associations. SGCLDGA employs GCNs to extract vector representations of drugs and genes from the original bipartite graph. Subsequently, singular value decomposition (SVD) is employed to enhance the graph and generate multiple views. The model performs contrastive learning across these views, optimizing vector representations through a contrastive loss function to better distinguish positive and negative samples. The final step involves utilizing inner product calculations to determine association scores between drugs and genes. Experimental results on the DGIdb4.0 dataset demonstrate SGCLDGA’s superior performance compared with six state-of-the-art methods. Ablation studies and case analyses validate the significance of contrastive learning and SVD, highlighting SGCLDGA’s potential in discovering new drug–gene associations. The code and dataset for SGCLDGA are freely available at https://github.com/one-melon/SGCLDGA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2c6f148229f4c500374f889c9a38505eb3a904d" target='_blank'>
              SGCLDGA: unveiling drug–gene associations through simple graph contrastive learning
              </a>
            </td>
          <td>
            Yanhao Fan, Che Zhang, Xiaowen Hu, Zhijian Huang, , Lei Deng
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The clinical trial process, also known as drug development, is an indispensable step toward the development of new treatments. The major objective of interventional clinical trials is to assess the safety and effectiveness of drug-based treatment in treating certain diseases in the human body. However, clinical trials are lengthy, labor-intensive, and costly. The duration of a clinical trial is a crucial factor that influences overall expenses. Therefore, effective management of the timeline of a clinical trial is essential for controlling the budget and maximizing the economic viability of the research. To address this issue, We propose TrialDura, a machine learning-based method that estimates the duration of clinical trials using multimodal data, including disease names, drug molecules, trial phases, and eligibility criteria. Then, we encode them into Bio-BERT embeddings specifically tuned for biomedical contexts to provide a deeper and more relevant semantic understanding of clinical trial data. Finally, the model's hierarchical attention mechanism connects all of the embeddings to capture their interactions and predict clinical trial duration. Our proposed model demonstrated superior performance with a mean absolute error (MAE) of 1.04 years and a root mean square error (RMSE) of 1.39 years compared to the other models, indicating more accurate clinical trial duration prediction. Publicly available code can be found at https://anonymous.4open.science/r/TrialDura-F196">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9f42d73801bd30b24b52747f85702ef0036027" target='_blank'>
              TrialDura: Hierarchical Attention Transformer for Interpretable Clinical Trial Duration Prediction
              </a>
            </td>
          <td>
            Ling Yue, Jonathan Li, Md Zabirul Islam, Bolun Xia, Tianfan Fu, Jintai Chen
          </td>
          <td>2024-04-20</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05865f65c46fc557199a535bb5addff911c4cc60" target='_blank'>
              Causal machine learning for predicting treatment outcomes.
              </a>
            </td>
          <td>
            Stefan Feuerriegel, Dennis Frauen, Valentyn Melnychuk, J. Schweisthal, Konstantin Hess, Alicia Curth, Stefan Bauer, Niki Kilbertus, Isaac S. Kohane, Mihaela van der Schaar
          </td>
          <td>2024-04-01</td>
          <td>Nature medicine</td>
          <td>3</td>
          <td>10</td>
        </tr>

        <tr id="Prediction of drug combination responses is a research question of growing importance for cancer and other complex diseases. Current machine learning approaches generally consider predicting either drug combination synergy summaries or single combination dose-response values, which fail to appropriately model the continuous nature of the underlying dose-response combination surface and can lead to inconsistencies when a synergy score or a dose-response matrix is reconstructed from separate predictions. We propose a structured prediction method, comboKR, that directly predicts the drug combination response surface for a drug combination. The method is based on a powerful input-output kernel regression technique and functional modeling of the response surface. As an important part of our approach, we develop a novel normalisation between response surfaces that standardizes the heterogeneous experimental designs used to measure the dose-responses, and thus allows training the method with data measured in different laboratories. Our experiments on two predictive scenarios highlight the suitability of the proposed approach especially in the traditionally challenging setting of predicting combination responses for new drugs not available in the training data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aa8dd703916af4caa0d6de87dc609764aa2db9f" target='_blank'>
              Predicting drug combination response surfaces
              </a>
            </td>
          <td>
            Riikka Huusari, Tianduanyi Wang, S. Szedmák, T. Aittokallio, Juho Rousu
          </td>
          <td>2024-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Mechanistic models of genetic interactions are rarely feasible due to a lack of information and computational challenges. Alternatively, machine learning (ML) approaches may predict gene interactions if provided with enough data but they lack interpretability. Here, we propose an ML approach for interpretable genotype-fitness mapping, the Direct-Latent Interpretable Model (D-LIM). The neural network is built on a strong hypothesis: mutations in different genes cause independent effects in phenotypes, which then interact via non-linear relationships to determine fitness. D-LIM predicts genotype-fitness maps for combinations of mutations in multiple genes with state-of-the-art accuracy, showing the validity of the hypothesis in the case of a deep mutational scanning in a metabolic pathway. The hypothesisdriven structure of D-LIM offers interpretable features reminiscent of mechanistic models: the inference of phenotypes, fitness extrapolation outside of the data domain, and enhanced prediction in low-data regimes by the integration of prior knowledge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708c731d4cacd76d9401b0f0949c1422b368d303" target='_blank'>
              Hypothesis-driven interpretable neural network for interactions between genes
              </a>
            </td>
          <td>
            Shuhui Wang, Alexandre Allauzen, Philippe Nghe, Vaitea Opuu
          </td>
          <td>2024-04-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Standard-of-care treatment regimens have long been designed for maximal cell killing, yet these strategies often fail when applied to metastatic cancers due to the emergence of drug resistance. Adaptive treatment strategies have been developed as an alternative approach, dynamically adjusting treatment to suppress the growth of treatment-resistant populations and thereby delay, or even prevent, tumor progression. Promising clinical results in prostate cancer indicate the potential to optimize adaptive treatment protocols. Here, we applied deep reinforcement learning (DRL) to guide adaptive drug scheduling and demonstrated that these treatment schedules can outperform the current adaptive protocols in a mathematical model calibrated to prostate cancer dynamics, more than doubling the time to progression. The DRL strategies were robust to patient variability, including both tumor dynamics and clinical monitoring schedules. The DRL framework could produce interpretable, adaptive strategies based on a single tumor burden threshold, replicating and informing optimal treatment strategies. The DRL framework had no knowledge of the underlying mathematical tumor model, demonstrating the capability of DRL to help develop treatment strategies in novel or complex settings. Finally, a proposed five-step pathway, which combined mechanistic modeling with the DRL framework and integrated conventional tools to improve interpretability compared to traditional "black-box" DRL models, could allow translation of this approach to the clinic. Overall, the proposed framework generated personalized treatment schedules that consistently outperformed clinical standard-of-care protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6e1406ae7f5d35249e430eca7bc7c00ba8d4b40" target='_blank'>
              Mathematical Model-Driven Deep Learning Enables Personalized Adaptive Therapy.
              </a>
            </td>
          <td>
            Kit Gallagher, Maximillian Strobl, Derek S Park, Fabian C. Spoendlin, Robert A Gatenby, P. Maini, Alexander R. A. Anderson
          </td>
          <td>2024-04-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Motivation Mathematical modelling plays a crucial role in understanding of inter- and intracellular signalling processes. Currently, ordinary differential equations (ODEs) are the predominant approach utilized in systems biology for modelling such pathways. While such mechanistic ODE models offer interpretability, they also suffer from limitations, including the need to consider all relevant compounds, resulting in large models that are difficult to handle numerically and require extensive data. Results In our previous work, we introduced the retarded transient function (RTF) as an alternative method for modelling the temporal response of signalling pathways. In this work, we extend the RTF approach to also integrate concentration or dose-dependencies of dynamics to fully cover the application range of ordinary differential equation (ODE) models. This extension enables time- and dose-dependent predictions and offers an intuitive means to investigate and characterize signalling differences between biological conditions or cell types. To demonstrate the applicability of our extended RTF approach, we employ data from time- and dose-dependent inflammasome activation in primary murine bone marrow-derived dendritic cells (BMDCs) treated with tyrosine kinase inhibitors. Our results show the effectiveness of the extended RTF approach as a generic framework for modelling dose-dependent kinetic data as they typically occur in cellular signalling. The presented approach offers intuitive interpretability of signal dynamics and enables predictive modelling of time- and dose-dependencies even if only individual cellular components are quantified. Availability The presented approach is available within the MATLAB-based Data2Dynamics modelling toolbox at https://github.com/Data2Dynamics and as R code at https://github.com/kreutz-lab/RTF. Contact clemens.kreutz@uniklinik-freiburg.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acc3791d6241d168aa1cb932106f46769c05b26a" target='_blank'>
              Dynamic modelling of signalling pathways when ODEs are not feasible
              </a>
            </td>
          <td>
            Timo Rachel, Eva Brombacher, Svenja Wöhrle, Olaf Groß, Clemens Kreutz
          </td>
          <td>2024-04-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Motivation Mechanistic modeling based on ordinary differential equations has led to numerous findings in systems biology by integrating prior knowledge and experimental data. However, the manual curation of knowledge necessary when constructing models poses a bottleneck. As the speed of knowledge accumulation continues to grow, there is a demand for a scalable means of constructing executable models. Results We previously introduced BioMASS—an open-source, Python-based framework–to construct, simulate, and analyze mechanistic models of signaling networks. With one of its features, Text2Model, BioMASS allows users to define models in a natural language-like format, thereby facilitating the construction of large-scale models. We demonstrate that Text2Model can serve as a tool for integrating external knowledge for mathematical modeling by generating Text2Model files from a pathway database or through the use of a large language model, and simulating its dynamics through BioMASS. Our findings reveal the tool's capabilities to encourage exploration from prior knowledge and pave the way for a fully data-driven approach to constructing mathematical models. Availability and implementation The code and documentation for BioMASS are available at https://github.com/biomass-dev/biomass and https://biomass-core.readthedocs.io, respectively. The code used in this article are available at https://github.com/okadalabipr/text2model-from-knowledge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72c387266bbc39d831e5236216760fb26dc69509" target='_blank'>
              Extending BioMASS to construct mathematical models from external knowledge
              </a>
            </td>
          <td>
            Kiwamu Arakane, Hiroaki Imoto, Fabian Ormersbach, Mariko Okada
          </td>
          <td>2024-04-04</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Heart failure poses a significant challenge in the global health domain, and accurate prediction of mortality is crucial for devising effective treatment plans. In this study, we employed a Seq2Seq model from deep learning, integrating 12 patient features. By finely modeling continuous medical records, we successfully enhanced the accuracy of mortality prediction.


OBJECTIVE
The objective of this research was to leverage the Seq2Seq model in conjunction with patient features for precise mortality prediction in heart failure cases, surpassing the performance of traditional machine learning methods.


METHODS
The study utilized a Seq2Seq model in deep learning, incorporating 12 patient features, to intricately model continuous medical records. The experimental design aimed to compare the performance of Seq2Seq with traditional machine learning methods in predicting mortality rates.


RESULTS
The experimental results demonstrated that the Seq2Seq model outperformed conventional machine learning methods in terms of predictive accuracy. Feature importance analysis provided critical patient risk factors, offering robust support for formulating personalized treatment plans.


CONCLUSIONS
This research sheds light on the significant applications of deep learning, specifically the Seq2Seq model, in enhancing the precision of mortality prediction in heart failure cases. The findings present a valuable direction for the application of deep learning in the medical field and provide crucial insights for future research and clinical practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76e2e69a438ba092dfabca2e8228bc14c6ca235" target='_blank'>
              Applications of deep learning models in precision prediction of survival rates for heart failure patients.
              </a>
            </td>
          <td>
            Qiaohui Zhang, Demin Xu
          </td>
          <td>2024-04-12</td>
          <td>Technology and health care : official journal of the European Society for Engineering and Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="We consider a Graph Neural Network (GNN) non-Markovian modeling framework to identify coarse-grained dynamical systems on graphs. Our main idea is to systematically determine the GNN architecture by inspecting how the leading term of the Mori-Zwanzig memory term depends on the coarse-grained interaction coefficients that encode the graph topology. Based on this analysis, we found that the appropriate GNN architecture that will account for $K$-hop dynamical interactions has to employ a Message Passing (MP) mechanism with at least $2K$ steps. We also deduce that the memory length required for an accurate closure model decreases as a function of the interaction strength under the assumption that the interaction strength exhibits a power law that decays as a function of the hop distance. Supporting numerical demonstrations on two examples, a heterogeneous Kuramoto oscillator model and a power system, suggest that the proposed GNN architecture can predict the coarse-grained dynamics under fixed and time-varying graph topologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c24fe27eaf7f498fa7256c6c06dd99bcf8df096" target='_blank'>
              Learning Coarse-Grained Dynamics on Graph
              </a>
            </td>
          <td>
            Yin Yu, John Harlim, , Yan Li
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gastric cancer and Colon adenocarcinoma represent widespread and challenging malignancies with high mortality rates and complex treatment landscapes. In response to the critical need for accurate prognosis in cancer patients, the medical community has embraced the 5-year survival rate as a vital metric for estimating patient outcomes. This study introduces a pioneering approach to enhance survival prediction models for gastric and Colon adenocarcinoma patients. Leveraging advanced image analysis techniques, we sliced whole slide images (WSI) of these cancers, extracting comprehensive features to capture nuanced tumor characteristics. Subsequently, we constructed patient-level graphs, encapsulating intricate spatial relationships within tumor tissues. These graphs served as inputs for a sophisticated 4-layer graph convolutional neural network (GCN), designed to exploit the inherent connectivity of the data for comprehensive analysis and prediction. By integrating patients' total survival time and survival status, we computed C-index values for gastric cancer and Colon adenocarcinoma, yielding 0.57 and 0.64, respectively. Significantly surpassing previous convolutional neural network models, these results underscore the efficacy of our approach in accurately predicting patient survival outcomes. This research holds profound implications for both the medical and AI communities, offering insights into cancer biology and progression while advancing personalized treatment strategies. Ultimately, our study represents a significant stride in leveraging AI-driven methodologies to revolutionize cancer prognosis and improve patient outcomes on a global scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba258c711b31b547b66ebc9937fce2daf39e214e" target='_blank'>
              Survival Prediction Across Diverse Cancer Types Using Neural Networks
              </a>
            </td>
          <td>
            Xu Yan, Weimin Wang, Mingxuan Xiao, Yufeng Li, Min Gao
          </td>
          <td>2024-04-11</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>3</td>
        </tr>

        <tr id="The spread of infectious diseases is a highly complex spatiotemporal process, difficult to understand, predict, and effectively respond to. Machine learning and artificial intelligence (AI) have achieved impressive results in other learning and prediction tasks; however, while many AI solutions are developed for disease prediction, only a few of them are adopted by decision-makers to support policy interventions. Among several issues preventing their uptake, AI methods are known to amplify the bias in the data they are trained on. This is especially problematic for infectious disease models that typically leverage large, open, and inherently biased spatiotemporal data. These biases may propagate through the modeling pipeline to decision-making, resulting in inequitable policy interventions. Therefore, there is a need to gain an understanding of how the AI disease modeling pipeline can mitigate biased input data, in-processing models, and biased outputs. Specifically, our vision is to develop a large-scale micro-simulation of individuals from which human mobility, population, and disease ground truth data can be obtained. From this complete dataset – which may not reflect the real world – we can sample and inject different types of bias. By using the sampled data in which bias is known (as it is given as the simulation parameter), we can explore how existing solutions for fairness in AI can mitigate and correct these biases and investigate novel AI fairness solutions. Achieving this vision would result in improved trust in such models for informing fair and equitable policy interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488b3d0489e2d6d305e64ce2ec54e6c86a1e8ccb" target='_blank'>
              Leveraging Simulation Data to Understand Bias in Predictive Models of Infectious Disease Spread
              </a>
            </td>
          <td>
            Andreas Züfle, Flora Salim, Taylor Anderson, M. Scotch, Li Xiong, Kacper Sokol, Hao Xue, Ruochen Kong, David Heslop, Hye-Young Paik, C. R. MacIntyre
          </td>
          <td>2024-04-25</td>
          <td>ACM Transactions on Spatial Algorithms and Systems</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Drug synergy prediction is a challenging and important task in the treatment of complex diseases including cancer. In this manuscript, we present a novel Foundation Model, known as BAITSAO, for tasks related to drug synergy prediction with a unified pipeline to handle different datasets. We construct the training datasets for BAITSAO based on the context-enriched embeddings from Large Language Models for the initial representation of drugs and cell lines. After demonstrating the relevance of these embeddings, we pre-train BAITSAO with a large-scale drug synergy database under a multi-task learning framework with rigorous selections of tasks. We demonstrate the superiority of the model architecture and the pre-trained strategies of BAITSAO over other methods through comprehensive benchmark analysis. Moreover, we investigate the sensitivity of BAITSAO and illustrate its unique functions including new drug discoveries, drug combinations-gene interaction, and multi-drug synergy predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/204e81f59339fc28e63d20df64778fc49d3edc01" target='_blank'>
              BAITSAO: Building A Foundation Model for Drug Synergy Analysis Powered by Language Models
              </a>
            </td>
          <td>
            Tianyu Liu, Tinyi Chu, Xiao Luo, Hongyu Zhao
          </td>
          <td>2024-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The intersection of precision medicine and artificial intelligence (AI) holds profound implications for cancer treatment, with the potential to significantly advance our understanding of drug responses based on the intricate architecture of tumor cells. A recent study by Park and colleagues titled "A deep learning model of tumor cell architecture elucidates response and resistance to CDK4/6 inhibitors," epitomizes this intersection by leveraging an interpretable deep learning model grounded in a comprehensive map of multiprotein assemblies in cancer, known as Nested Systems in Tumors (NeST). This study not only elucidates mechanisms underlying the response to CDK4/6 inhibitors in breast cancer therapy but also highlights the critical role of model interpretability leading to new mechanistic insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6dbb19e42b7a66ce2daaa8d8f0f50ee5999758" target='_blank'>
              How Artificial Intelligence Unravels the Complex Web of Cancer Drug Response.
              </a>
            </td>
          <td>
            Olivier Elemento
          </td>
          <td>2024-04-08</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose: Optimizing brain bioavailability is highly relevant for the development of drugs targeting the central nervous system. Several pharmacokinetic parameters have been used for measuring drug bioavailability in the brain. The most biorelevant among them is possibly the unbound brain-to-plasma partition coefficient, Kpuu,brain,ss, which relates unbound brain and plasma drug concentrations under steady-state conditions. In this study, we developed new in silico models to predict Kpuu,brain,ss.Methods: A manually curated 157-compound dataset was compiled from literature and split into training and test sets using a clustering approach. Additional models were trained with a refined dataset generated by removing known P-gp and/or Breast Cancer Resistance Protein substrates from the original dataset. Different supervised machine learning algorithms have been tested, including Support Vector Machine, Gradient Boosting Machine, k-nearest neighbors, classificatory Partial Least Squares, Random Forest, Extreme Gradient Boosting, Deep Learning and Linear Discriminant Analysis. Good practices of predictive Quantitative Structure-Activity Relationships modeling were followed for the development of the models.Results: The best performance in the complete dataset was achieved by extreme gradient boosting, with an accuracy in the test set of 85.1%. A similar estimation of accuracy was observed in a prospective validation experiment, using a small sample of compounds and comparing predicted unbound brain bioavailability with observed experimental data.Conclusion: New in silico models were developed to predict the Kpuu,brain,ss of drug candidates. The dataset used in this study is publicly disclosed, so that the models may be reproduced, refined, or expanded, as a useful tool to assist drug discovery processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52afc251825ef12585cbed611131c124d727750f" target='_blank'>
              Application of machine learning to predict unbound drug bioavailability in the brain
              </a>
            </td>
          <td>
            J. F. Morales, M. E. Ruiz, Robert E. Stratford, Alan Talevi
          </td>
          <td>2024-04-04</td>
          <td>Frontiers in Drug Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Computational drug sensitivity models have the potential to improve therapeutic outcomes by identifying targeted drugs components that are tailored to the transcriptomic profile of a given primary tumor. The SMILES representation of molecules that is used by state-of-the-art drug-sensitivity models is not conducive for neural networks to generalize to new drugs, in part because the distance between atoms does not generally correspond to the distance between their representation in the SMILES strings. Graph-attention networks, on the other hand, are high-capacity models that require large training-data volumes which are not available for drug-sensitivity estimation. We develop a modular drug-sensitivity graph-attentional neural network. The modular architecture allows us to separately pre-train the graph encoder and graph-attentional pooling layer on related tasks for which more data are available. We observe that this model outperforms reference models for the use cases of precision oncology and drug discovery; in particular, it is better able to predict the specific interaction between drug and cell line that is not explained by the general cytotoxicity of the drug and the overall survivability of the cell line. The complete source code is available at https://zenodo.org/doi/10.5281/zenodo.8020945. All experiments are based on the publicly available GDSC data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1d95c29d808b802d81c2a052d2dbf5ee6f08b7" target='_blank'>
              Cancer drug sensitivity estimation using modular deep Graph Neural Networks
              </a>
            </td>
          <td>
            P. A. Campana, Paul Prasse, M. Lienhard, Kristina Thedinga, Ralf Herwig, Tobias Scheffer
          </td>
          <td>2024-04-04</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Predicting chemical reaction yields is pivotal for efficient chemical synthesis, an area that focuses on the creation of novel compounds for diverse uses. 
Yield prediction demands accurate representations of reactions for forecasting practical transformation rates. Yet, the uncertainty issues broadcasting in real-world situations prohibit current models to excel in this task owing to the high sensitivity of yield activities and the uncertainty in yield measurements. Existing models often utilize single-modal feature representations, such as molecular fingerprints, SMILES sequences, or molecular graphs, which is not sufficient to capture the complex interactions and dynamic behavior of molecules in reactions. In this paper, we present an advanced Uncertainty-Aware Multimodal model (UAM) to tackle these challenges. Our approach seamlessly integrates data sources from multiple modalities by encompassing sequence representations, molecular graphs, and expert-defined chemical reaction features for a comprehensive representation of reactions. Additionally, we address both the model and data-based uncertainty, refining the model's predictive capability. Extensive experiments on three datasets, including two high throughput experiment (HTE) datasets and one chemist-constructed Amide coupling reaction dataset, demonstrate that UAM outperforms the state-of-the-art methods. The code and used datasets are available at https://github.com/jychen229/Multimodal-reaction-yield-prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fedccf6c70a7d075753e68deeb4d03f936c7e824" target='_blank'>
              Uncertainty-Aware Yield Prediction with Multimodal Molecular Features
              </a>
            </td>
          <td>
            Jiayuan Chen, Kehan Guo, Zhen Liu, O. Isayev, Xiangliang Zhang
          </td>
          <td>2024-03-24</td>
          <td>DBLP</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="

The integration of artificial intelligence and machine learning holds great
promise for enhancing healthcare institutions and providing fresh perspectives on the origins and
advancement of long-term illnesses. In the healthcare sector, artificial intelligence and machine learning
are used to address supply and demand concerns, genomic applications, and new advancements
in drug development, cancer, and heart disease.



The article explores the ways that machine learning, AI, precision medicine, and genomics
are changing healthcare. The essay also discusses how AI's examination of various patient data could
enhance healthcare institutions, provide fresh insights into chronic conditions, and advance precision
medicine. The potential uses of machine learning for genome analysis are also examined in the paper,
particularly about genetic biomarker-based disease risk and symptom prediction.



The challenges posed by the phenotype-genotype relationship are examined, as well as
the significance of comprehending disease pathways in order to create tailored treatments. Moreover,
it offers a streamlined and modularized method that predicts how genotypes affect cell properties
using machine-learning models, enabling the development of personalized drugs. The collective feedback
highlights the rapid interdisciplinary growth of medical genomics following the completion of
the Human Genome Project. It also emphasizes how important genomic data is for improving
healthcare outcomes and facilitating personalized medicine.



The study's conclusions point to a revolutionary shift in healthcare: the application of
AI/ML to illness control. Even though these innovations have a lot of potential benefits, problems
like algorithm interpretability and ethical issues need to be worked out before they can be successfully
incorporated into routine medical practice. Using machine learning in medicine has enormous potential
benefits for the biotech industry. Further research, ongoing regulatory frameworks, and collaboration
between medical professionals and data analysts are necessary to fully utilize machine learning
as well as artificial intelligence in disease management.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/934c7ca8ae423804df2848baa556409ec7854a1b" target='_blank'>
              A State-of-the-art Review on Artificial Intelligence and Machine Learning Applications
              </a>
            </td>
          <td>
            Rishav Sharma, R. Malviya, Prerna Uniyal, Bhupendra Prajapati
          </td>
          <td>2024-04-24</td>
          <td>Current Drug Therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="In applications such as biomedical studies, epidemiology, and social sciences, recurrent events often co-occur with longitudinal measurements and a terminal event, such as death. Therefore, jointly modeling longitudinal measurements, recurrent events, and survival data while accounting for their dependencies is critical. While joint models for the three components exist in statistical literature, many of these approaches are limited by heavy parametric assumptions and scalability issues. Recently, incorporating deep learning techniques into joint modeling has shown promising results. However, current methods only address joint modeling of longitudinal measurements at regularly-spaced observation times and survival events, neglecting recurrent events. In this paper, we develop TransformerLSR, a flexible transformer-based deep modeling and inference framework to jointly model all three components simultaneously. TransformerLSR integrates deep temporal point processes into the joint modeling framework, treating recurrent and terminal events as two competing processes dependent on past longitudinal measurements and recurrent event times. Additionally, TransformerLSR introduces a novel trajectory representation and model architecture to potentially incorporate a priori knowledge of known latent structures among concurrent longitudinal variables. We demonstrate the effectiveness and necessity of TransformerLSR through simulation studies and analyzing a real-world medical dataset on patients after kidney transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be00caf52727fad3464be1a89970b50716795384" target='_blank'>
              TransformerLSR: Attentive Joint Model of Longitudinal Data, Survival, and Recurrent Events with Concurrent Latent Structure
              </a>
            </td>
          <td>
            Zhiyue Zhang, Yao Zhao, Yan Xu
          </td>
          <td>2024-04-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Traditional cancer treatment methods have become less effective due to the increasing diversity of cancer types. To address this, precision medicine has gained support within the medical community. This approach tailors treatment to individual patients based on their specific disease characteristics. However, a major challenge lies in accurately predicting how a patient will respond to a specialized drug. Numerous machine learning-based predictive systems have been developed to address this challenge. These systems utilize genomic signatures and the chemical structure of drugs to predict drug activity. In this paper, we introduce a Multi-Layer Perceptron (MLP) based system for predicting the response of anticancer drugs. Our system utilizes hybrid features derived from both genetic expression and the chemical structure of drugs. It is developed using the well-known GDSC dataset (Genomics of Drug Sensitivity in Cancer). Our system achieved a lower Root Mean Square Error (RMSE) value of 0.889, in contrast to the RMSE value of 0.983 obtained by the current state-of-the-art (SOTA) system, SwNet. This indicates superior predictive accuracy. The findings suggest that our proposed research holds promise for the development of targeted drugs for anticancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c82b5c941ccd82787222fb672dac393f13e9ea" target='_blank'>
              Developing MLP based prediction system for anticancer drug response using hybrid features of genomics and cheminformatics
              </a>
            </td>
          <td>
            Awais Raza Zaidi, Muhammad Bilal, Tuba Majid, Abdul Majid
          </td>
          <td>2024-04-24</td>
          <td>Lahore Garrison University Research Journal of Computer Science and Information Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Anti-cancer response of cell lines to drugs is in urgent need for individualized precision medical decision-making in the era of precision medicine. Measurements with wet-experiments is time-consuming and expensive and it is almost impossible for wide ranges of application. The design of computational models that can precisely predict the responses between drugs and cell lines could provide a credible reference for further research. Existing methods of response prediction based on matrix factorization or neural networks have revealed that both linear or nonlinear latent characteristics are applicable and effective for the precise prediction of drug responses. However, the majority of them consider only linear or nonlinear relationships for drug response prediction. Herein, we propose a Dual Branch Deep Neural Matrix Factorization (DBDNMF) method to address the above-mentioned issues. DBDNMF learns the latent representation of drugs and cell lines through flexible inputs and reconstructs the partially observed matrix through a series of hidden neural network layers. Experimental results on the datasets of Cancer Cell Line Encyclopedia (CCLE) and Genomics of Drug Sensitivity in Cancer (GDSC) show that the accuracy of drug prediction exceeds state-of-the-art drug response prediction algorithms, demonstrating its reliability and stability. The hierarchical clustering results show that drugs with similar response levels tend to target similar signaling pathway, and cell lines coming from the same tissue subtype tend to share the same pattern of response, which are consistent with previously published studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c139ad04d41a207bd4aaa54f2acb208eb45a04d" target='_blank'>
              DBDNMF: A Dual Branch Deep Neural Matrix Factorization method for drug response prediction
              </a>
            </td>
          <td>
            Hui Liu, Feng Wang, Jian Yu, Yong Pan, Chaoju Gong, Liang Zhang, Lin Zhang
          </td>
          <td>2024-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Prediction of drug-target interactions (DTIs) is a crucial step in drug discovery, and deep learning methods have shown great promise on various DTI datasets. However, existing approaches still face several challenges, including limited labeled data, hidden bias issue, and a lack of generalization ability to out-of-domain data. These challenges hinder the model's capacity to learn truly informative interaction features, leading to shortcut learning and inferior predictive performance on novel drug-target pairs. To address these issues, we propose MlanDTI, a semi-supervised domain adaptive multilevel attention network (Mlan) for DTI prediction. We utilize two pre-trained BERT models to acquire bidirectional representations enriched with information from unlabeled data. Then, we introduce a multilevel attention mechanism, enabling the model to learn domain-invariant DTIs at different hierarchical levels. Moreover, we present a simple yet effective semi-supervised pseudo-labeling method to further enhance our model's predictive ability in cross-domain scenarios. Experiments on four datasets show that MlanDTI achieves state-of-the-art performances over other methods under intra-domain settings and outperforms all other approaches under cross-domain settings. The source code is available at https://github.com/CMACH508/MlanDTI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da523e004d5375958f0c7bb4b2559aa7e43dd84c" target='_blank'>
              Multilevel Attention Network with Semi-supervised Domain Adaptation for Drug-Target Prediction
              </a>
            </td>
          <td>
            Zhousan Xie, Shikui Tu, Lei Xu
          </td>
          <td>2024-03-24</td>
          <td>DBLP</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="High-throughput screening platforms for the profiling of drug sensitivity of hundreds of cancer cell lines (CCLs) have generated large datasets that hold the potential to unlock targeted, anti-tumor therapies. In this study, we leveraged these datasets to create predictive models of cancer cells drug sensitivity. To this aim we trained explainable machine learning algorithms by employing cell line transcriptomics to predict the growth inhibitory potential of drugs. We used large language models (LLMs) to expand descriptions of the mechanisms of action (MOA) for each drug starting from available annotations, which were matched to the semantically closest pathways from reference knowledge bases. By leveraging this AI-curated resource, and the interpretability of our model, we demonstrated that pathways enriched for genes crucial for prediction often matched known drug-MOAs and essential genes, suggesting that our models learned the molecular determinants of drug response. Furthermore, we demonstrated that by incorporating only LLM-curated genes associated with MOAs, we enhanced the predictive accuracy of our drug models. To enhance translatability to a clinical setting, we employed a pipeline to align bulk RNAseq from CCLs, used for training the models, to those from patient samples, used for inference. We proved the effectiveness of our approach on TCGA samples, where patients’ best scoring drugs matched those prescribed for their cancer type. We further showed its usefulness by predicting and experimentally validating effective drugs for the patients of two highly lethal solid tumors, i.e. pancreatic cancer and glioblastoma. In summary, our method facilitates the inference and interpretation of cancer cell line drug sensitivity and holds potential to effectively translate them into new cancer therapeutics. Highlights -Interpretable drug-response prediction models were trained on large scale pharmacogenomics data sets (i.e. GDSC and PRISM). -Large language models were used to enhance the curation of biological pathways associated to drugs MOA -Unbiased interpretation of the models demonstrated learning of drug MOAs and gene essentiality -Inference of TCGA cohort samples recovered mono- and combination cancer drug prescriptions and indicate potential repurposing candidates. -Drug candidates predicted from bulk RNAseq samples of pancreatic cancer and glioblastoma were experimentally validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/748f9da063fa715fefeaa0bfbccf264963bf4d60" target='_blank'>
              Learning and actioning general principles of cancer cell drug sensitivity
              </a>
            </td>
          <td>
            Francesco Carli, Pierluigi Di Chiaro, M. Morelli, Chakit Arora, Luisa Bisceglia, Natalia De Oliveira Rosa, Alice Cortesi, S. Franceschi, F. Lessi, A. D. Di Stefano, O. Santonocito, Francesco Pasqualetti, P. Aretini, Pasquale Miglionico, G. Diaferia, Fosca Giannotti, Pietro Liò, Miquel Duran-Frigola, C. M. Mazzanti, Gioacchino Natoli, Francesco Raimondi
          </td>
          <td>2024-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="A problem in nonlinear and complex dynamical systems with broad applications is forecasting the occurrence of a critical transition based solely on data without knowledge about the system equations. When such a transition leads to system collapse, as often is the case, all the available data are from the pre-critical regime where the system still functions normally, making the prediction problem challenging. In recent years, a machine-learning based approach tailored to solving this difficult prediction problem, adaptable reservoir computing, has been articulated. This Perspective introduces the basics of this machine-learning scheme and describes representative results. The general setting is that the system dynamics live on a normal attractor with oscillatory dynamics at the present time and, as a bifurcation parameter changes into the future, a critical transition can occur after which the system switches to a completely different attractor, signifying system collapse. To predict a critical transition, it is essential that the reservoir computer not only learns the dynamical "climate" of the system of interest at some specific parameter value but, more importantly, discovers how the system dynamics changes with the bifurcation parameter. It is demonstrated that this capability can be endowed into the machine through a training process with time series from a small number of distinct, pre-critical parameter values, thereby enabling accurate and reliable prediction of the catastrophic critical transition. Three applications are presented: predicting crisis, forecasting amplitude death, and creating digital twins of nonlinear dynamical systems. Limitations and future perspectives are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1749f284391bc6f9be0f2b8f564c797144a8e75" target='_blank'>
              Adaptable reservoir computing: A paradigm for model-free data-driven prediction of critical transitions in nonlinear dynamical systems.
              </a>
            </td>
          <td>
            Shirin Panahi, Ying-Cheng Lai
          </td>
          <td>2024-05-01</td>
          <td>Chaos</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Artificial intelligence and machine learning enable the construction of predictive models, which are currently used to assist in decision-making throughout the process of drug discovery and development. These computational models can be used to represent the heterogeneity of a disease, identify therapeutic targets, design and optimize drug candidates, and evaluate the efficacy of these drugs on virtual patients or digital twins. By combining detailed patient characteristics with the prediction of potential drug-candidate properties, artificial intelligence promotes the emergence of a "computational" precision medicine, allowing for more personalized treatments, better tailored to patient specificities with the aid of such predictive models. Based on such new capabilities, a mixed reality approach to the development of new drugs is being adopted by the pharmaceutical industry, which integrates the outputs of predictive virtual models with real-world empirical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57cba1eb82f6e6919ebfe3e6d794d361cc3420d0" target='_blank'>
              [The revolution of AI in drug development].
              </a>
            </td>
          <td>
            Philippe Moingeon, Christiane Garbay, Muriel Dahan, Irène Fermont, Ali Benmakhlouf, Alain Gouyette, Pierre Poitou, Alain Saint-Pierre
          </td>
          <td>2024-04-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug discovery is a crucial element of biomedical research, with the goal of finding and creating new medical treatments for a variety of illnesses. Yet, the conventional process of finding new drugs is frequently impeded by its intrinsic difficulties, such as expensive expenses, long durations, and poor success rates in trials with patients. Recently, the incorporation of machine learning (ML) algorithms has become a revolutionary method to streamline and improve different phases of drug discovery. This summary offers a glimpse into the rapidly growing area of drug discovery using machine learning algorithms, emphasizing its potential to transform the process of developing treatments.
The usual process of discovering drugs involves various stages such as identifying the target, finding lead compounds, conducting preclinical tests, undergoing clinical trials, and obtaining regulatory approval. All these phases require a lot of labor, time, and resources, leading to high attrition rates and limited success in turning potential compounds into approved therapies. Nevertheless, researchers can enhance and speed up crucial parts of the drug discovery process by using ML algorithms.
ML algorithms use data to aid in drug discovery by utilizing computational models to examine large quantities of biological, chemical, and clinical data. These algorithms can learn from various types of data, such as genomic data, chemical structures, protein interactions, and clinical outcomes, to discover hidden patterns, find new targets for drugs, and forecast the effectiveness and safety of potential treatments. Moreover, machine learning algorithms allow for the investigation of intricate connections between molecular structures and biological effects, making it easier to create improved drug candidates with better effectiveness and specificity.
Important uses of machine learning in pharmaceutical research involve finding and confirming targets, screening compounds and improving leads, repurposing drugs, and tailoring treatments for individuals. Commonly used for classification and regression tasks, supervised learning algorithms like support vector machines and random forests predict compound activity, toxicity, and pharmacokinetic properties. Clustering and dimensionality reduction techniques utilized in unsupervised learning algorithms help analyze vast datasets and discover new drug-target interactions. Advanced abilities for analyzing molecular structures, virtual screening, and designing new drugs are provided by deep learning models like convolutional neural networks and recurrent neural networks.
Multiple case studies demonstrate how ML algorithms can significantly impact drug discovery. Collaboration among academia, industry, and research institutions has resulted in the creation of new ML-based methods for drug development, identifying targets, and categorizing patients. Nevertheless, there are challenges accompanying the widespread use of ML in drug discovery. In healthcare, it is crucial to address ethical considerations, regulatory hurdles, and data privacy concerns to ensure the responsible and ethical use of ML algorithms.
The potential for transforming drug discovery and therapeutic development is immense with the incorporation of machine learning algorithms. Through utilizing data-driven methods, researchers can speed up the discovery of new potential drugs, enhance the effectiveness of treatments, and ultimately enhance the results for patients. Ongoing innovation, teamwork, and cross-disciplinary studies are crucial to fully leverage the potential of ML in revolutionizing drug discovery and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39298bed391759133bbc6e09e80ea4f002528fdc" target='_blank'>
              Revolutionizing Drug Discovery: Harnessing Machine Learning Algorithms
              </a>
            </td>
          <td>
            Tushar Khinvasara
          </td>
          <td>2024-04-11</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Early identification of safe and efficacious disease targets is crucial to alleviating the tremendous cost of drug discovery projects. However, existing experimental methods for identifying new targets are generally labor-intensive and failure-prone. On the other hand, computational approaches, especially machine learning-based frameworks, have shown remarkable application potential in drug discovery. In this work, we propose Progeni, a novel machine learning-based framework for target identification. In addition to fully exploiting the known heterogeneous biological networks from various sources, Progeni integrates literature evidence about the relations between biological entities to construct a probabilistic knowledge graph. Graph neural networks are then employed in Progeni to learn the feature embeddings of biological entities to facilitate the identification of biologically relevant target candidates. A comprehensive evaluation of Progeni demonstrated its superior predictive power over the baseline methods on the target identification task. In addition, our extensive tests showed that Progeni exhibited high robustness to the negative effect of exposure bias, a common phenomenon in recommendation systems, and effectively identified new targets that can be strongly supported by the literature. Moreover, our wet lab experiments successfully validated the biological significance of the top target candidates predicted by Progeni for melanoma and colorectal cancer. All these results suggested that Progeni can identify biologically effective targets and thus provide a powerful and useful tool for advancing the drug discovery process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d2fa8d9cb5d3a8fbde95561c2616b16dcfc840" target='_blank'>
              A probabilistic knowledge graph for target identification
              </a>
            </td>
          <td>
            Chang Liu, Kaimin Xiao, Cuinan Yu, Yipin Lei, Kangbo Lyu, Tingzhong Tian, Dan Zhao, Fengfeng Zhou, Haidong Tang, Jianyang Zeng
          </td>
          <td>2024-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The integration of deep learning technology within the domain of drug target discovery represents a cutting-edge approach in the pharmaceutical sciences. This paper delves into the methods and prevailing issues associated with the application of deep learning in the identification of new drug targets. We initiate with an exposition of the fundamental concepts and methodologies underpinning deep learning, subsequently illuminating its utilization in the realm of drug target discovery. Specific attention is given to the deployment of architectures like convolutional neural networks (CNNs) and recurrent neural networks (RNNs) among others.
By interpreting experimental outcomes, it has been discerned that deep learning potentially enhances the precision of identifying drug targets. Nonetheless, this technological arena is not devoid of challenges. A significant hurdle lies in the interpretability of the models—understanding why and how these models arrive at their conclusions is often non-trivial. Moreover, the efficacy of deep learning is heavily reliant on the quality and volume of the datasets employed; insufficient or poor-quality data can severely impede the performance and reliability of predictive models.Through this inquiry, we aim to furnish novel insights into the utilization of deep learning for drug target discovery and underscore the challenges that must be addressed. The perspectives offered herein are vital to propel the progress of this burgeoning field, paving the way for more effective and efficient drug discovery processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8e3a7e952c0b4f80fe17c1f6d1f509a0eaf19a" target='_blank'>
              Application and difficulties of deep learning in drug target discovery
              </a>
            </td>
          <td>
            Xiaoqian Kuang
          </td>
          <td>2024-04-16</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Continuous graph neural models based on differential equations have expanded the architecture of graph neural networks (GNNs). Due to the connection between graph diffusion and message passing, diffusion-based models have been widely studied. However, diffusion naturally drives the system towards an equilibrium state, leading to issues like over-smoothing. To this end, we propose GRADE inspired by graph aggregation-diffusion equations, which includes the delicate balance between nonlinear diffusion and aggregation induced by interaction potentials. The node representations obtained through aggregation-diffusion equations exhibit metastability, indicating that features can aggregate into multiple clusters. In addition, the dynamics within these clusters can persist for long time periods, offering the potential to alleviate over-smoothing effects. This nonlinear diffusion in our model generalizes existing diffusion-based models and establishes a connection with classical GNNs. We prove that GRADE achieves competitive performance across various benchmarks and alleviates the over-smoothing issue in GNNs evidenced by the enhanced Dirichlet energy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02a90af710f12a5eb7ce7a59f3a59b57ec44046c" target='_blank'>
              Graph Neural Aggregation-diffusion with Metastability
              </a>
            </td>
          <td>
            Kaiyuan Cui, Xinyan Wang, Zicheng Zhang, Weichen Zhao
          </td>
          <td>2024-03-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Developing models that can answer questions of the form"How would $x$ change if $y$ had been $z$?"is fundamental for advancing medical image analysis. Training causal generative models that address such counterfactual questions, though, currently requires that all relevant variables have been observed and that corresponding labels are available in training data. However, clinical data may not have complete records for all patients and state of the art causal generative models are unable to take full advantage of this. We thus develop, for the first time, a semi-supervised deep causal generative model that exploits the causal relationships between variables to maximise the use of all available data. We explore this in the setting where each sample is either fully labelled or fully unlabelled, as well as the more clinically realistic case of having different labels missing for each sample. We leverage techniques from causal inference to infer missing values and subsequently generate realistic counterfactuals, even for samples with incomplete labels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36ebc43e3829fe62086b844e9599c73df1889410" target='_blank'>
              Semi-Supervised Learning for Deep Causal Generative Models
              </a>
            </td>
          <td>
            Yasin Ibrahim, Hermione Warr, Konstantinos Kamnitsas
          </td>
          <td>2024-03-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clinical trials involving systemic neoadjuvant treatments in breast cancer aim to shrink tumors before surgery while simultaneously allowing for controlled evaluation of biomarkers, toxicity, and suppression of distant (occult) metastatic disease. Yet neoadjuvant clinical trials are rarely preceded by preclinical testing involving neoadjuvant treatment, surgery, and post-surgery monitoring of the disease. Here we used a mouse model of spontaneous metastasis occurring after surgical removal of orthotopically implanted primary tumors to develop a predictive mathematical model of neoadjuvant treatment response to sunitinib, a receptor tyrosine kinase inhibitor (RTKI). Treatment outcomes were used to validate a novel mathematical kinetics-pharmacodynamics model predictive of perioperative disease progression. Longitudinal measurements of presurgical primary tumor size and postsurgical metastatic burden were compiled using 128 mice receiving variable neoadjuvant treatment doses and schedules (released publicly at https://zenodo.org/records/10607753). A non-linear mixed-effects modeling approach quantified inter-animal variabilities in metastatic dynamics and survival, and machine-learning algorithms were applied to investigate the significance of several biomarkers at resection as predictors of individual kinetics. Biomarkers included circulating tumor- and immune-based cells (circulating tumor cells and myeloid-derived suppressor cells) as well as immunohistochemical tumor proteins (CD31 and Ki67). Our computational simulations show that neoadjuvant RTKI treatment inhibits primary tumor growth but has little efficacy in preventing (micro)-metastatic disease progression after surgery and treatment cessation. Machine learning algorithms that included support vector machines, random forests, and artificial neural networks, confirmed a lack of definitive biomarkers, which shows the value of preclinical modeling studies to identify potential failures that should be avoided clinically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c8ff9b498ddc0ddca957f0ddbc9e36ba56facbc" target='_blank'>
              Machine-learning and mechanistic modeling of metastatic breast cancer after neoadjuvant treatment
              </a>
            </td>
          <td>
            S. Benzekry, M. Mastri, C. Nicolò, J. Ebos
          </td>
          <td>2024-05-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="This paper presents a comprehensive investigation into the applicability and performance of two prominent growth models, namely, the Verhulst model and the Montroll model, in the context of modeling tumor cell growth dynamics. Leveraging the power of Physics-Informed Neural Networks (PINNs), we aim to assess and compare the predictive capabilities of these models against experimental data obtained from the growth patterns of tumor cells. We employed a dataset comprising detailed measurements of tumor cell growth to train and evaluate the Verhulst and Montroll models. By integrating PINNs, we not only account for experimental noise but also embed physical insights into the learning process, enabling the models to capture the underlying mechanisms governing tumor cell growth. Our findings reveal the strengths and limitations of each growth model in accurately representing tumor cell proliferation dynamics. Furthermore, the study sheds light on the impact of incorporating physics-informed constraints on the model predictions. The insights gained from this comparative analysis contribute to advancing our understanding of growth models and their applications in predicting complex biological phenomena, particularly in the realm of tumor cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99a19673371a8d65fd7ea004ec6b44d862e2d9fb" target='_blank'>
              Using Physics-Informed Neural Networks (PINNs) for Tumor Cell Growth Modeling
              </a>
            </td>
          <td>
            J. A. Rodrigues
          </td>
          <td>2024-04-16</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Precisely predicting drug-protein interactions (DPIs) is pivotal for drug discovery and advancing precision medicine. A significant challenge in this domain is the high-dimensional and heterogeneous data characterizing drug and protein attributes, along with their intricate interactions. In our study, we introduce a novel deep learning architecture: the Multi-view Variational Auto-Encoder embedded within a cascade Deep Forest (MVAE-DFDPnet). This framework adeptly learns ultra-low-dimensional embedding for drugs and proteins. Notably, our t-SNE analysis reveals that two-dimensional embedding can clearly define clusters corresponding to diverse drug classes and protein families. These ultra-low-dimensional embedding likely contribute to the enhanced robustness and generalizability of our MVAE-DFDPnet. Impressively, our model surpasses current leading methods on benchmark datasets, functioning in significantly reduced dimensional spaces. The model’s resilience is further evidenced by its sustained accuracy in predicting interactions involving novel drugs, proteins, and drug classes. Additionally, we have corroborated several newly identified DPIs with experimental evidence from the scientific literature. The code used to generate and analyze these results can be accessed from https://github.com/Macau-LYXia/MVAE-DFDPnet-V2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/887353b3b42ec83c47f0d43699e712fe2a516477" target='_blank'>
              Redefining the Game: MVAE-DFDPnet’s Low-Dimensional Embeddings for Superior Drug-Protein Interaction Predictions
              </a>
            </td>
          <td>
            Liang-Yong Xia, Yu Wu, Longfei Zhao, Leying Chen, Shiyi Zhang, Mengdi Wang, Jie Luo
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The DrugMatrix Database contains systematically generated toxicogenomics data from short-term in vivo studies for over 600 chemicals. However, most of the potential endpoints in the database are missing due to a lack of experimental measurements. We present our study on leveraging matrix factorization and machine learning methods to predict the missing values in the DrugMatrix, which includes gene expression across eight tissues on two expression platforms along with paired clinical chemistry, hematology, and histopathology measurements. One major challenge we encounter is the skewed distribution of the available measured data, in terms of both tissue sources and values. We propose a method, ToxiCompl, that applies systematic hybrid sampling guided by Bayesian optimization in conjunction with low-rank matrix factorization to recover the missing values. ToxiCompl achieves good training and validation performance from a machine learning perspective. We further conduct an in-depth validation of the predicted data from biological and toxicological perspectives with a series of analyses. These include examining the connectivity pattern of predicted gene expression responses, characterizing molecular pathway-level responses from sets of differentially expressed genes, evaluating known transcriptional biomarkers of tissue toxicity, and characterizing pre-dicted apical endpoints. Our analysis shows that the predicted differential gene expression, broadly speaking, aligns with what would be anticipated. For example, in most instances, our predicted differentially expressed gene lists offer a connectivity level comparable to that of measured data in connectivity analysis. Using Havcr1, a known transcriptional biomarker of kidney injury, we identify treatments that, based on the predicted expression data, manifest kidney toxicity in a manner that is mechanistically plausible and supported by the literature. Characterization of the predicted clinical chemistry data suggests that strong effects are relatively reliably predicted, while more subtle effects pose a greater challenge. In the case of histopathological prediction, we find a significant overprediction due to positivity bias in the measured data. Developing methods to deal with this bias is one of the areas we plan to target for future improvement. The main advantage of the ToxiCompl approach is that, in the absence of additional experimental data, it drastically extends the toxicogenomic landscape into a number of data-poor tissues, thereby allowing researchers to formulate mechanistic hypotheses about effects in tissues that have been underrepresented in the literature. All measured and predicted DrugMatrix data (i.e., gene expression, clinical chemistry, hematology, and histopathology) are available to the public through an intuitive GUI interface that allows for data retrieval, gene set analysis and high dimensional visualization of gene expression similarity (https://rstudio.niehs.nih.gov/complete_drugmatrix/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d2ffbc53f79544dfb9e97a8e711c4353e3d6948" target='_blank'>
              Completion of the DrugMatrix Toxicogenomics Database using ToxCompl
              </a>
            </td>
          <td>
            Guojing Cong, Robert M. Patton, Frank Chao, Daniel L. Svoboda, Warren M. Casey, Charles P. Schmitt, Charles Murphy, Jeremy N. Erickson, Parker A. Combs, Scott Auerbach
          </td>
          <td>2024-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract The interrelation and complementary nature of multi-omics data can provide valuable insights into the intricate molecular mechanisms underlying diseases. However, challenges such as limited sample size, high data dimensionality and differences in omics modalities pose significant obstacles to fully harnessing the potential of these data. The prior knowledge such as gene regulatory network and pathway information harbors useful gene–gene interaction and gene functional module information. To effectively integrate multi-omics data and make full use of the prior knowledge, here, we propose a Multilevel-graph neural network (GNN): a hierarchically designed deep learning algorithm that sequentially leverages multi-omics data, gene regulatory networks and pathway information to extract features and enhance accuracy in predicting survival risk. Our method achieved better accuracy compared with existing methods. Furthermore, key factors nonlinearly associated with the tumor pathogenesis are prioritized by employing two interpretation algorithms (i.e. GNN-Explainer and IGscore) for neural networks, at gene and pathway level, respectively. The top genes and pathways exhibit strong associations with disease in survival analyses, many of which such as SEC61G and CYP27B1 are previously reported in the literature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015f9781ff14975162703fe9c3ed52bddbaf0e81" target='_blank'>
              Prior knowledge-guided multilevel graph neural network for tumor risk prediction and interpretation via multi-omics data integration
              </a>
            </td>
          <td>
            Hongxi Yan, Dawei Weng, Dongguo Li, Yu Gu, Wenji Ma, Qingjie Liu
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In constructing Gene Regulatory Networks (GRNs), it is crucial to consider cellular heterogeneity and differential gene regulatory modules. However, traditional methods have predominantly focused on cellular heterogeneity, approaching the subject from a relatively narrow scope. We present HyperG-VAE, a Bayesian deep generative model that utilizes a hypergraph to model single-cell RNA sequencing (scRNA-seq) data. HyperG-VAE employs a cell encoder with a Structural Equation Model to address cellular heterogeneity and build GRNs, alongside a gene encoder using hypergraph self-attention to identify gene modules. Encoders are synergistically optimized by a decoder, enabling HyperG-VAE to excel in GRN inference, single-cell clustering, and data visualization, evidenced by benchmarks. Additionally, HyperG-VAE effectively reveals gene regulation patterns and shows robustness in varied downstream analyses, demonstrated using B cell development data in bone marrow. The interplay of encoders by the overlapping genes between predicted GRNs and gene modules is further validated by gene set enrichment analysis, underscoring that the gene encoder boosts the GRN inference. HyperG-VAE proves efficient in scRNA-seq data analysis and GRN inference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/910e673103b27a58525cf647fe4c8fe79d5c8ef7" target='_blank'>
              Inferring gene regulatory networks by hypergraph variational autoencoder
              </a>
            </td>
          <td>
            Guangxin Su, Hanchen Wang, Ying Zhang, A. Coster, Marc R. Wilkins, Pablo F. Canete, Di Yu, Yang Yang, Wenjie Zhang
          </td>
          <td>2024-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Simple Summary The major impact from this work is two-fold, first in providing a list of reproducible candidate drugs nominated against multiple glioblastoma (GBM) datasets, and second in providing drug–gene biomarkers of interest for GBM as well as reproducible computational pipelines for identifying drug–biomarker leads that could be extended to other cancer types beyond GBM. We believe the research community will be interested and can utilize our results here for the further development of biomarker-specific drug therapies in both GBM as well as other disease types. Abstract Glioblastoma multiforme (GBM) is the deadliest, most heterogeneous, and most common brain cancer in adults. Not only is there an urgent need to identify efficacious therapeutics, but there is also a great need to pair these therapeutics with biomarkers that can help tailor treatment to the right patient populations. We built patient drug response models by integrating patient tumor transcriptome data with high-throughput cell line drug screening data as well as Bayesian networks to infer relationships between patient gene expression and drug response. Through these discovery pipelines, we identified agents of interest for GBM to be effective across five independent patient cohorts and in a mouse avatar model: among them are a number of MEK inhibitors (MEKis). We also predicted phosphoglycerate dehydrogenase enzyme (PHGDH) gene expression levels to be causally associated with MEKi efficacy, where knockdown of this gene increased tumor sensitivity to MEKi and overexpression led to MEKi resistance. Overall, our work demonstrated the power of integrating computational approaches. In doing so, we quickly nominated several drugs with varying known mechanisms of action that can efficaciously target GBM. By simultaneously identifying biomarkers with these drugs, we also provide tools to select the right patient populations for subsequent evaluation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b533ec6d5b79d0a8dc0aa09ce2fb660cc0c67f5f" target='_blank'>
              Integration of Computational Pipeline to Streamline Efficacious Drug Nomination and Biomarker Discovery in Glioblastoma
              </a>
            </td>
          <td>
            D. Maeser, Robert F. Gruener, R. Galvin, Adam Lee, Tomoyuki Koga, F. Grigore, Yuta Suzuki, F. Furnari, Clark Chen, R. S. Huang
          </td>
          <td>2024-04-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Developing new pharmaceuticals is a costly and time-consuming endeavor fraught with significant safety risks. A critical aspect of drug research and disease therapy is discerning the existence of interactions between drugs and proteins. The evolution of deep learning (DL) in computer science has been remarkably aided in this regard in recent years. Yet, two challenges remain: (i) balancing the extraction of profound, local cohesive characteristics while warding off gradient disappearance and (ii) globally representing and understanding the interactions between the drug and target local attributes, which is vital for delivering molecular level insights indispensable to drug development. In response to these challenges, we propose a DL network structure, MolLoG, primarily comprising two modules: local feature encoders (LFE) and global interactive learning (GIL). Within the LFE module, graph convolution networks and leap blocks capture the local features of drug and protein molecules, respectively. The GIL module enables the efficient amalgamation of feature information, facilitating the global learning of feature structural semantics and procuring multihead attention weights for abstract features stemming from two modalities, providing biologically pertinent explanations for black-box results. Finally, predictive outcomes are achieved by decoding the unified representation via a multilayer perceptron. Our experimental analysis reveals that MolLoG outperforms several cutting-edge baselines across four data sets, delivering superior overall performance and providing satisfactory results when elucidating various facets of drug-target interaction predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47d7134e78582b908d74226e67442dfea752621" target='_blank'>
              MolLoG: A Molecular Level Interpretability Model Bridging Local to Global for Predicting Drug Target Interactions.
              </a>
            </td>
          <td>
            Bao-Ming Feng, Yuanyuan Zhang, Xiao-Chen Zhou, Jin-Long Wang, Yinfei Feng
          </td>
          <td>2024-05-06</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Temporal link prediction is one of the most important tasks for predicting time-varying links by capturing dynamics within complex networks. However, it suffers from difficulties such as vulnerability to adversarial attacks and inadaptation to distinct evolutionary patterns. In this article, we propose a robust temporal link prediction architecture via stable gated models with reinforcement learning (SAGE-RL) consisting of a state encoding network (SEN) and a self-adaptive policy network (SPN). The former is utilized to capture network dynamics, while the latter helps the former adapt to distinct evolutionary patterns across various time periods. Within the SEN, a novel stable gate is introduced to ensure multiple spatiotemporal dependency paths and defend against adversarial attacks. An SPN is proposed to select different SEN instances by approximating the optimal action function, thereby adapting to various evolutionary patterns to learn the robust temporal and structural features from dynamic complex networks. It is proven that SAGE-LR with integral Lipschitz graph convolution is stable to relative perturbations in dynamic complex networks. With the aid of extensive experiments on five real-world graph benchmarks, SAGE-LR is shown to substantially outperform current state-of-the-art approaches in terms of precision and stability of temporal link prediction and ability to successfully defend against various attacks. We also implement the temporal link prediction in shipping transaction networks, which forecast effectively its potential transaction risks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc8eab5464296d20d63e8638a6cc53aff28a678d" target='_blank'>
              Robust Temporal Link Prediction in Dynamic Complex Networks via Stable Gated Models With Reinforcement Learning.
              </a>
            </td>
          <td>
            Liping Yang, Hongbo Liu, Daoqiang Sun, Seán F. McLoone, Kai Liu, C. L. P. Chen
          </td>
          <td>2024-05-14</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Establishing appropriate mathematical models for complex systems in natural phenomena not only helps deepen our understanding of nature but can also be used for state estimation and prediction. However, the extreme complexity of natural phenomena makes it extremely challenging to develop full-order models (FOMs) and apply them to studying many quantities of interest. In contrast, appropriate reduced-order models (ROMs) are favored due to their high computational efficiency and ability to describe the key dynamics and statistical characteristics of natural phenomena. Taking the viscous Burgers equation as an example, this paper constructs a Convolutional Autoencoder-Reservoir Computing-Normalizing Flow algorithm framework, where the Convolutional Autoencoder is used to construct latent space representations, and the Reservoir Computing-Normalizing Flow framework is used to characterize the evolution of latent state variables. In this way, a data-driven stochastic parameter reduced-order model is constructed to describe the complex system and its dynamic behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f199be4cede75b9048f7e55d590a2a44646f487b" target='_blank'>
              Stochastic parameter reduced-order model based on hybrid machine learning approaches
              </a>
            </td>
          <td>
            Cheng Fang, Jinqiao Duan
          </td>
          <td>2024-03-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Contemporary machine learning algorithms have largely succeeded in automating the development of mathematical models from data. Although this is a striking accomplishment, it leaves unaddressed the multitude of scenarios, especially across the chemical sciences and engineering, where deductive, rather than inductive, reasoning is required and still depends on manual intervention by an expert. This review describes the characteristics of deductive reasoning that are helpful for understanding the role played by expert intervention in problem-solving and explains why such interventions are often relatively resistant to disruption by typical machine learning strategies. The article then discusses the factors that contribute to creating a deductive bottleneck, how deductive bottlenecks are currently addressed in several application areas, and how machine learning models capable of deduction can be designed. The review concludes with a tutorial case study that illustrates the challenges of deduction problems and a notebook for readers to experiment with on their own.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165715a4f113ec52e0dce6a488ddfb51435944ed" target='_blank'>
              Deductive Machine Learning Challenges and Opportunities in Chemical Applications.
              </a>
            </td>
          <td>
            Tianfan Jin, B. Savoie
          </td>
          <td>2024-04-10</td>
          <td>Annual review of chemical and biomolecular engineering</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/620803fdb5f14b75e2d7602b6c40e8374eb91f5a" target='_blank'>
              Enhancing drug and cell line representations via contrastive learning for improved anti-cancer drug prioritization.
              </a>
            </td>
          <td>
            Patrick J. Lawrence, Benjamin Burns, Xia Ning
          </td>
          <td>2024-05-18</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Investigation of important molecular features of glioma patients may reveal driving mechanisms of patient recurrence and treatment failures in glioma. Applying various deep learning techniques enables the integration and systematic comparison of multi-omics features, allowing robust marker prediction and patient stratification. We extracted mutation and copy number variation data from the Glioma Longitudinal Analysis (GLASS) Consortium, consisting of 628 primary or recurrent samples. To achieve the patient recurrence classification task, we designed a transformer-based model that learns the genomic profiles and predicts patient recurrence status. Our model outperforms traditional machine learning models, including support vector machine (SVM), logistic regression, and decision trees, for up to 34% and 40% improvement of AUROC and AUPRC, respectively. Including mutational features alone can achieve the best prediction performance, while copy number variations are less informative in predicting patient recurrence status. Our approach is an early attempt that employs deep learning for dissecting the contribution of diverse genomic profiles on glioma recurrence and might provide insights into developing novel therapies for treating glioma.
 Citation Format: Ko-Hong Lin, Kai Zhang, Dung-Fang Lee, Xiaoqian Jiang. Deep learning framework for patient recurrence and genetic marker prediction in diffuse glioma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2024; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2024 Apr 5-10; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2024;84(7_Suppl):Abstract nr LB246.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b61dafce86e85913504ff36d62a0700a1db4a2c4" target='_blank'>
              Abstract LB246: Deep learning framework for patient recurrence and genetic marker prediction in diffuse glioma
              </a>
            </td>
          <td>
            Ko-Hong Lin, Kai Zhang, Dung-Fang Lee, Xiaoqian Jiang
          </td>
          <td>2024-04-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Fluid dynamics problems are characterized by being multidimensional and nonlinear, causing the experiments and numerical simulations being complex, time-consuming and monetarily expensive. In this sense, there is a need to find new ways to obtain data in a more economical manner. Thus, in this work we study the application of time series forecasting to fluid dynamics problems, where the aim is to predict the flow dynamics using only past information. We focus our study on models based on deep learning that do not require a high amount of data for training, as this is the problem we are trying to address. Specifically in this work we have tested three autoregressive models where two of them are fully based on deep learning and the other one is a hybrid model that combines modal decomposition with deep learning. We ask these models to generate $200$ time-ahead predictions of two datasets coming from a numerical simulation and experimental measurements, where the latter is characterized by being turbulent. We show how the hybrid model generates more reliable predictions in the experimental case, as it is physics-informed in the sense that the modal decomposition extracts the physics in a way that allows us to predict it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7b99cbcae24e51709d38ebd55a9ffa77001107f" target='_blank'>
              Exploring the efficacy of a hybrid approach with modal decomposition over fully deep learning models for flow dynamics forecasting
              </a>
            </td>
          <td>
            Rodrigo Abad'ia-Heredia, A. Corrochano, Manuel López-Martín, S. L. Clainche
          </td>
          <td>2024-04-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The development of models to predict sensitivity to anticancer drugs is an area of significant interest, given the diverse responses to treatment among patients and the considerable expense and time involved in anticancer drug development. Leveraging “omic” data and anticancer response information from the Cancer Cell Line Encyclopedia, we propose a novel approach utilizing multitask learning to enhance prediction accuracy and inference. We extended a multitask learning framework called the Data Shared Lasso to develop the Data Shared Elastic Net. This enabled the construction of tissue-specific models with information sharing while maintaining the attractive properties of Elastic Net regression. By employing this approach, we observed improvements in prediction accuracy compared to single-task Elastic Net models, particularly for cell lines displaying high sensitivity to treatment. Furthermore, the Data Shared Elastic Net facilitated the identification of predictors for anticancer drug sensitivity within specific tissue types, shedding light on cellular pathways targeted by these drugs across tissues. We also investigated the impact of data leakage on modeling outcomes from previous studies, which led to underestimating prediction error and erroneous inferences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83664bfa905ff011a2bcad47d103dd20fcdcd259" target='_blank'>
              Improving Drug Sensitivity Prediction and Inference by Multi-Task Learning
              </a>
            </td>
          <td>
            Jared Strauch, Amir Asiaee
          </td>
          <td>2024-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Amidst the rapid advancements in generative language models, the investigation of how training data shapes the performance of GPT models is still emerging. This paper presents GPTfluence, a novel approach that leverages a featurized simulation to assess the impact of training examples on the training dynamics of GPT models. Our approach not only traces the influence of individual training instances on performance trajectories, such as loss and other key metrics, on targeted test points but also enables a comprehensive comparison with existing methods across various training scenarios in GPT models, ranging from 14 million to 2.8 billion parameters, across a range of downstream tasks. Contrary to earlier methods that struggle with generalization to new data, GPTfluence introduces a parameterized simulation of training dynamics, demonstrating robust generalization capabilities to unseen training data. This adaptability is evident across both fine-tuning and instruction-tuning scenarios, spanning tasks in natural language understanding and generation. We will make our code and data publicly available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02ffeb8cb7a1d1336e0e135c7c1000610ae7b942" target='_blank'>
              On Training Data Influence of GPT Models
              </a>
            </td>
          <td>
            Qingyi Liu, Yekun Chai, Shuohuan Wang, Yu Sun, Keze Wang, Hua Wu
          </td>
          <td>2024-04-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Survival prediction is an important branch of cancer prognosis analysis. The model that predicts survival risk through TCGA genomics data can discover genes related to cancer and provide diagnosis and treatment recommendations based on patient characteristics. We found that deep learning models based on Cox proportional hazards often suffer from overfitting when dealing with high-throughput data. Moreover, we found that as the number of network layers increases, the experimental results will not get better, and network degradation will occur. Based on this problem, we propose a new framework based on Deep Residual Learning. Combine the ideas of Cox proportional hazards and Residual. And name it ResSurv. First, ResSurv is a feed-forward deep learning network stacked by multiple basic ResNet Blocks. In each ResNet Block, we add a Normalization Layer to prevent gradient disappearance and gradient explosion. Secondly, for the loss function of the neural network, we inherited the Cox proportional hazards methods, applied the semi-parametric of the CPH model to the neural network, combined with the partial likelihood model, established the loss function, and performed backpropagation and gradient update. Finally, we compared ResSurv networks of different depths and found that we can effectively extract high-dimensional features. Ablation experiments and comparative experiments prove that our model has reached SOTA(state of the art) in the field of deep learning, and our network can effectively extract deep information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2cbfe31657721f4bf6c2ca44aa455fb795d7549" target='_blank'>
              ResSurv: Cancer Survival Analysis Prediction Model Based on Residual Networks
              </a>
            </td>
          <td>
            Wankang Zhai
          </td>
          <td>2024-05-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89a40750265888d7f9c1c557675e1656108cf7e" target='_blank'>
              Machine learning heralding a new development phase in molecular dynamics simulations
              </a>
            </td>
          <td>
            Eva Prasnikar, M. Ljubic, Andrej Perdih, J. Borišek
          </td>
          <td>2024-03-29</td>
          <td>Artif. Intell. Rev.</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="This study aims to provide an overview of the current state-of-the-art applications of artificial intelligence (AI) and machine learning in the management of hepatocellular carcinoma (HCC), and to explore future directions for continued progress in this emerging field.  This study is a comprehensive literature review that synthesizes recent findings and advancements in the application of AI and machine learning techniques across various aspects of HCC care, including screening and early detection, diagnosis and staging, prognostic modeling, treatment planning, interventional guidance, and monitoring of treatment response. The review draws upon a wide range of published research studies, focusing on the integration of AI and machine learning with diverse data sources, such as medical imaging, clinical data, genomics, and other multimodal information.  The results demonstrate that AI-based systems have shown promise in improving the accuracy and efficiency of HCC screening, diagnosis, and tumor characterization compared to traditional methods. Machine learning models integrating clinical, imaging, and genomic data have outperformed conventional staging systems in predicting survival and recurrence risk. AI-based recommendation systems have the potential to optimize personalized therapy selection, while augmented reality techniques can guide interventional procedures in real-time. Moreover, longitudinal application of AI may enhance the assessment of treatment response and recurrence monitoring. Despite these promising findings, the review highlights the need for rigorous multicenter prospective validation studies, standardized multimodal datasets, and thoughtful consideration of ethical implications before widespread clinical implementation of AI technologies in HCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4171d77b0e3f46735aae116d624f4eba8b5bd5a5" target='_blank'>
              Realizing the Promise of Artificial Intelligence in Hepatocellular Carcinoma through Opportunities and Recommendations for Responsible Translation
              </a>
            </td>
          <td>
            T. Addissouky, Ibrahim El Tantawy El Sayed, Majeed M. A. Ali
          </td>
          <td>2024-04-23</td>
          <td>Jurnal Online Informatika</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Foundation models, such as large language models, have demonstrated success in addressing various language and image processing tasks. In this work, we introduce a multi-modal foundation model for scientific problems, named PROSE-PDE. Our model, designed for bi-modality to bi-modality learning, is a multi-operator learning approach which can predict future states of spatiotemporal systems while concurrently learning the underlying governing equations of the physical system. Specifically, we focus on multi-operator learning by training distinct one-dimensional time-dependent nonlinear constant coefficient partial differential equations, with potential applications to many physical applications including physics, geology, and biology. More importantly, we provide three extrapolation studies to demonstrate that PROSE-PDE can generalize physical features through the robust training of multiple operators and that the proposed model can extrapolate to predict PDE solutions whose models or data were unseen during the training. Furthermore, we show through systematic numerical experiments that the utilization of the symbolic modality in our model effectively resolves the well-posedness problems with training multiple operators and thus enhances our model's predictive capabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/819bc0d5e0ea2efadac1364064e40b76cf3a3a11" target='_blank'>
              Towards a Foundation Model for Partial Differential Equations: Multi-Operator Learning and Extrapolation
              </a>
            </td>
          <td>
            Jingmin Sun, Yuxuan Liu, Zecheng Zhang, Hayden Schaeffer
          </td>
          <td>2024-04-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The field of theranostics is rapidly advancing, driven by the goals of enhancing patient care. Recent breakthroughs in artificial intelligence (AI) and its innovative theranostic applications have marked a critical step forward in nuclear medicine, leading to a significant paradigm shift in precision oncology. For instance, AI-assisted tumor characterization, including automated image interpretation, tumor segmentation, feature identification, and prediction of high-risk lesions, improves diagnostic processes, offering a precise and detailed evaluation. With a comprehensive assessment tailored to an individual's unique clinical profile, AI algorithms promise to enhance patient risk classification, thereby benefiting the alignment of patient needs with the most appropriate treatment plans. By uncovering potential factors unseeable to the human eye, such as intrinsic variations in tumor radiosensitivity or molecular profile, AI software has the potential to revolutionize the prediction of response heterogeneity. For accurate and efficient dosimetry calculations, AI technology offers significant advantages by providing customized phantoms and streamlining complex mathematical algorithms, making personalized dosimetry feasible and accessible in busy clinical settings. AI tools have the potential to be leveraged to predict and mitigate treatment-related adverse events, allowing early interventions. Additionally, generative AI can be utilized to find new targets for developing novel radiopharmaceuticals and facilitate drug discovery. However, while there is immense potential and notable interest in the role of AI in theranostics, these technologies do not lack limitations and challenges. There remains still much to be explored and understood. In this study, we investigate the current applications of AI in theranostics and seek to broaden the horizons for future research and innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79833fab01c356ecb4d9c20ae6757ff3c5103859" target='_blank'>
              Theranostics and artificial intelligence: new frontiers in personalized medicine
              </a>
            </td>
          <td>
            Gokce Belge Bilgin, Cem Bilgin, Brian Burkett, Jacob J Orme, Daniel S Childs, Matthew P. Thorpe, T. Halfdanarson, Geoffrey B Johnson, A. Kendi, Oliver Sartor
          </td>
          <td>2024-03-25</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Identifying latent interactions within complex systems is key to unlocking deeper insights into their operational dynamics, including how their elements affect each other and contribute to the overall system behavior. For instance, in neuroscience, discovering neuron-to-neuron interactions is essential for understanding brain function; in ecology, recognizing the interactions among populations is key for understanding complex ecosystems. Such systems, often modeled as dynamical systems, typically exhibit noisy high-dimensional and non-stationary temporal behavior that renders their identification challenging. Existing dynamical system identification methods often yield operators that accurately capture short-term behavior but fail to predict long-term trends, suggesting an incomplete capture of the underlying process. Methods that consider extended forecasts (e.g., recurrent neural networks) lack explicit representations of element-wise interactions and require substantial training data, thereby failing to capture interpretable network operators. Here we introduce Lookahead-driven Inference of Networked Operators for Continuous Stability (LINOCS), a robust learning procedure for identifying hidden dynamical interactions in noisy time-series data. LINOCS integrates several multi-step predictions with adaptive weights during training to recover dynamical operators that can yield accurate long-term predictions. We demonstrate LINOCS' ability to recover the ground truth dynamical operators underlying synthetic time-series data for multiple dynamical systems models (including linear, piece-wise linear, time-changing linear systems' decomposition, and regularized linear time-varying systems) as well as its capability to produce meaningful operators with robust reconstructions through various real-world examples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f005c8b19ca1d9171afc191fc78b81f136ddfb8c" target='_blank'>
              LINOCS: Lookahead Inference of Networked Operators for Continuous Stability
              </a>
            </td>
          <td>
            Noga Mudrik, Eva Yezerets, Yenho Chen, Christopher Rozell, Adam Charles
          </td>
          <td>2024-04-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d63afa9ac9d835d94e1d88390467934811db53a5" target='_blank'>
              Task-oriented machine learning surrogates for tipping points of agent-based models
              </a>
            </td>
          <td>
            Gianluca Fabiani, N. Evangelou, Tianqi Cui, J. M. Bello-Rivas, Cristina P. Martin-Linares, Constantinos Siettos, I. Kevrekidis
          </td>
          <td>2024-05-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96bc93fec51b25d3ba75bcc36627b1513c999f0" target='_blank'>
              q-Diffusion leverages the full dimensionality of gene coexpression in single-cell transcriptomics.
              </a>
            </td>
          <td>
            Myrl G. Marmarelis, R. Littman, F. Battaglin, Donna Niedzwiecki, A. Venook, J. Ambite, A. Galstyan, H. Lenz, Greg Ver Steeg
          </td>
          <td>2024-04-02</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="BACKGROUND
In recent years, analyzing complex biological networks to predict future links in such networks has attracted the attention of many medical and computer science researchers. The discovery of new drugs is one of the application cases for predicting future connections in biological networks. The operation of drug-target interactions prediction (DTIP) can be considered a fundamental step in identifying potential interactions between drug and target to identify new drugs.


OBJECTIVE
The previous studies reveal that predictions are made based on known interactions using computational methods to solve the cost problem and avoid blind study of all interactions. But, there seem to be challenges such as the lack of confirmed negative samples and the low accuracy in some computational methods. Thus, we have proposed an efficient and hybrid approach called MKPUL-BLM to manage some of the aforementioned challenges for predicting drug-target interactions.


METHODS
The MKPUL-BLM combins multi-kernel and positive unlabeled learning (PUL) approaches. Our method uses more information to increase accuracy, in addition to minimizing small similarities using network information. Also, potential negative samples are produced using a PUL approach because of lacking negative laboratory samples. Finally, labels are expanded via a semi-supervised.


RESULTS
Our method improved to 0.98 and 0.94 in the old interactions set for the ROCAUC and AUPR criteria, respectively. Also, this method enhanced ROCAUC and AUPR criteria by 0.89 and 0.77 for the new interactions set.


CONCLUSION
The MKPUL-BLM can be considered an efficient alternative to achieve more reliable predictions in the field of DTIP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/517b4e428c381711c80b78a700350d74395f3bac" target='_blank'>
              An Enhanced Computational Approach Using Multi-kernel Positive Unlabeled Learning for Predicting Drug-target Interactions.
              </a>
            </td>
          <td>
            M. Keyvanpour, Soheila Mehrmolaei, Faraneh Haddadi
          </td>
          <td>2024-04-29</td>
          <td>Current computer-aided drug design</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Physiological and pathological processes are governed by a network of genes called gene regulatory networks (GRNs). By reconstructing GRNs, we can accurately model how cells behave in their natural state and predict how genetic changes will affect them. Transcriptomic data of single cells are now available for a wide range of cellular processes in multiple species. Thus, a method building predictive GRNs from single-cell RNA sequencing (scRNA-seq) data, without any additional prior knowledge, could have a great impact on our understanding of biological processes and the genes playing a key role in them. To this aim, we developed IGNITE (Inference of Gene Networks using Inverse kinetic Theory and Experiments), an unsupervised machine learning framework designed to infer directed, weighted, and signed GRNs directly from unperturbed single-cell RNA sequencing data. IGNITE uses the GRNs to generate gene expression data upon single and multiple genetic perturbations. IGNITE is based on the inverse problem for a kinetic Ising model, a model from statistical physics that has been successfully applied to biological networks. We tested IGNITE on murine pluripotent stem cells (PSCs) transitioning from the naïve to formative states. Using as input only scRNA-seq data of unperturbed PSCs, IGNITE simulated single and triple gene knockouts. Comparison with experimental data revealed high accuracy, up to 74%, outperforming currently available methods. In sum, IGNITE identifies predictive GRNs from scRNA-seq data without additional prior knowledge and faithfully simulates single and multiple gene perturbations. Applications of IGNITE range from studying cell differentiation to identifying genes specifically active under pathological conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f91d1ee7feee1c23aecfd8641c38451501491d9" target='_blank'>
              Unveiling Gene Perturbation Effects through Gene Regulatory Networks Inference from single-cell transcriptomic data
              </a>
            </td>
          <td>
            Clelia Corridori, Merrit Romeike, Giorgio Nicoletti, C. Buecker, S. Suweis, S. Azaele, Graziano Martello
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36fd625816fc82a989aba4767b781ec1f63c1523" target='_blank'>
              Strategies for improving the performance of prediction models for response to immune checkpoint blockade therapy in cancer
              </a>
            </td>
          <td>
            Tiantian Zeng, Jason Z. Zhang, Arnold Stromberg, Jin Chen, Chi Wang
          </td>
          <td>2024-04-09</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Distribution shifts, where statistical properties differ between training and test datasets, present a significant challenge in real-world machine learning applications where they directly impact model generalization and robustness. In this study, we explore model adaptation and generalization by utilizing synthetic data to systematically address distributional disparities. Our investigation aims to identify the prerequisites for successful model adaptation across diverse data distributions, while quantifying the associated uncertainties. Specifically, we generate synthetic data using the Van der Waals equation for gases and employ quantitative measures such as Kullback-Leibler divergence, Jensen-Shannon distance, and Mahalanobis distance to assess data similarity. These metrics en able us to evaluate both model accuracy and quantify the associated uncertainty in predictions arising from data distribution shifts. Our findings suggest that utilizing statistical measures, such as the Mahalanobis distance, to determine whether model predictions fall within the low-error"interpolation regime"or the high-error"extrapolation regime"provides a complementary method for assessing distribution shift and model uncertainty. These insights hold significant value for enhancing model robustness and generalization, essential for the successful deployment of machine learning applications in real-world scenarios.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83cbfb76db5c8020ddf128edf66e89d86bbb8fed" target='_blank'>
              Quantifying Distribution Shifts and Uncertainties for Enhanced Model Robustness in Machine Learning Applications
              </a>
            </td>
          <td>
            Vegard Flovik
          </td>
          <td>2024-05-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Integrating intelligent systems, such as robots, into dynamic group settings poses challenges due to the mutual influence of human behaviors and internal states. A robust representation of social interaction dynamics is essential for effective human-robot collaboration. Existing approaches often narrow their focus to facial expressions or speech, overlooking the broader context. We propose employing an adapted Temporal Graph Networks to comprehensively represent social interaction dynamics while enabling its practical implementation. Our method incorporates temporal multi-modal behavioral data including gaze interaction, voice activity and environmental context. This representation of social interaction dynamics is trained as a link prediction problem using annotated gaze interaction data. The F1-score outperformed the baseline model by 37.0%. This improvement is consistent for a secondary task of next speaker prediction which achieves an improvement of 29.0%. Our contributions are two-fold, including a model to representing social interaction dynamics which can be used for many downstream human-robot interaction tasks like human state inference and next speaker prediction. More importantly, this is achieved using a more concise yet efficient message passing method, significantly reducing it from 768 to 14 elements, while outperforming the baseline model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3fa60c7363a51de6b381c99522a2337c5ffa78" target='_blank'>
              Modeling social interaction dynamics using temporal graph networks
              </a>
            </td>
          <td>
            J. Kim, Archit Naik, Isuru Jayarathne, Sehoon Ha, Jouh Yeong Chew
          </td>
          <td>2024-04-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f319f5bedd470fa7ef18495bb61b8222606b89b6" target='_blank'>
              al-BERT: a semi-supervised denoising technique for disease prediction
              </a>
            </td>
          <td>
            Yun-Chien Tseng, Chuan-Wei Kuo, Wen-Chih Peng, Chih-Chieh Hung
          </td>
          <td>2024-05-16</td>
          <td>BMC Medical Informatics and Decision Making</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The complexity, scale, and uncertainty in regulatory networks (e.g., gene regulatory networks and microbial networks) regularly pose a huge uncertainty in their models. These uncertainties often cannot be entirely reduced using limited and costly data acquired from the normal condition of systems. Meanwhile, regulatory networks often suffer from the non-identifiability issue, which refers to scenarios where the true underlying network model cannot be clearly distinguished from other possible models. Perturbation or excitation is a well-known process in systems biology for acquiring targeted data to reveal the complex underlying mechanisms of regulatory networks and overcome the non-identifiability issue. We consider a general class of Boolean network models for capturing the activation and inactivation of components and their complex interactions. Assuming partial available knowledge about the interactions between components of the networks, this paper formulates the inference process through the maximum aposteriori (MAP) criterion. We develop a Bayesian lookahead policy that systematically perturbs regulatory networks to maximize the performance of MAP inference under the perturbed data. This is achieved by optimally formulating the perturbation process in a reinforcement learning context and deriving a scalable deep reinforcement learning perturbation policy to compute near-optimal Bayesian policy. The proposed method learns the perturbation policy through planning without the need for any real data. The high performance of the proposed approach is demonstrated by comprehensive numerical experiments using the well-known mammalian cell cycle and gut microbial community networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13661311b83454b1f32c13b0d86a4299e417da66" target='_blank'>
              Bayesian Lookahead Perturbation Policy for Inference of Regulatory Networks.
              </a>
            </td>
          <td>
            Mohammad Alali, Mahdi Imani
          </td>
          <td>2024-05-17</td>
          <td>IEEE/ACM transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Increasing evidence has shown that noncoding RNAs (ncRNAs) can affect drug efficiency by modulating drug sensitivity genes. Exploring the association between ncRNAs and drug sensitivity is essential for drug discovery and disease prevention. However, traditional biological experiments for identifying ncRNA-drug sensitivity associations are time-consuming and laborious. In this study, we develop a novel graph contrastive learning approach named NDSGCL to predict ncRNA-drug sensitivity. NDSGCL uses graph convolutional networks to learn feature representations of ncRNAs and drugs in ncRNA-drug bipartite graphs. It integrates local structural neighbours and global semantic neighbours to learn a more comprehensive representation by contrastive learning. Specifically, the local structural neighbours aim to capture the higher-order relationship in the ncRNA-drug graph, while the global semantic neighbours are defined based on semantic clusters of the graph that can alleviate the impact of data sparsity. The experimental results show that NDSGCL outperforms basic graph convolutional network methods, existing contrastive learning methods, and state-of-the-art prediction methods. Visualization experiments show that the contrastive objectives of local structural neighbours and global semantic neighbours play a significant role in contrastive learning. Case studies on two drugs show that NDSGCL is an effective tool for predicting ncRNA-drug sensitivity associations. Source code and datasets can be available on https://github.com/altriavin/NDSGCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57cdeafce14039ec06c096ded31a290c26bd3408" target='_blank'>
              Exploring ncRNA-Drug Sensitivity Associations Via Graph Contrastive Learning.
              </a>
            </td>
          <td>
            Xiaowen Hu, Ying Jiang, Lei Deng
          </td>
          <td>2024-04-05</td>
          <td>IEEE/ACM transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The growing availability and importance of time series data across various domains, including environmental science, epidemiology, and economics, has led to an increasing need for time-series causal discovery methods that can identify the intricate relationships in the non-stationary, non-linear, and often noisy real world data. However, the majority of current time series causal discovery methods assume stationarity and linear relations in data, making them infeasible for the task. Further, the recent deep learning-based methods rely on the traditional causal structure learning approaches making them computationally expensive. In this paper, we propose a Time-Series Causal Neural Network (TS-CausalNN) - a deep learning technique to discover contemporaneous and lagged causal relations simultaneously. Our proposed architecture comprises (i) convolutional blocks comprising parallel custom causal layers, (ii) acyclicity constraint, and (iii) optimization techniques using the augmented Lagrangian approach. In addition to the simple parallel design, an advantage of the proposed model is that it naturally handles the non-stationarity and non-linearity of the data. Through experiments on multiple synthetic and real world datasets, we demonstrate the empirical proficiency of our proposed approach as compared to several state-of-the-art methods. The inferred graphs for the real world dataset are in good agreement with the domain understanding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8072f6e96371a2006e81a6cc3b2f05e744a24241" target='_blank'>
              TS-CausalNN: Learning Temporal Causal Relations from Non-linear Non-stationary Time Series Data
              </a>
            </td>
          <td>
            Omar Faruque, Sahara Ali, Xue Zheng, Jianwu Wang
          </td>
          <td>2024-04-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The existence of large volumes of data has considerably alleviated concerns regarding the availability of sufficient data instances for machine learning experiments. Nevertheless, in certain contexts, addressing limited data availability may demand distinct strategies and efforts. Analyzing COVID-19 predictions at pandemic beginning emerged a question: how much data is needed to make reliable predictions? When does the volume of data provide a better understanding of the disease’s evolution and, in turn, offer reliable forecasts? Given these questions, the objective of this study is to analyze learning curves obtained from predicting the incidence of COVID-19 in Brazilian States using ARIMA models with limited available data. To fulfill the objective, a retrospective exploration of COVID-19 incidence across the Brazilian States was performed. After the data acquisition and modeling, the model errors were assessed by employing a learning curve analysis. The asymptotic exponential curve fitting enabled the evaluation of the errors in different points, reflecting the increased available data over time. For a comprehensive understanding of the results at distinct stages of the time evolution, the average derivative of the curves and the equilibrium points were calculated, aimed to identify the convergence of the ARIMA models to a stable pattern. We observed differences in average derivatives and equilibrium values among the multiple samples. While both metrics ultimately confirmed the convergence to stability, the equilibrium points were more sensitive to changes in the models’ accuracy and provided a better indication of the learning progress. The proposed method for constructing learning curves enabled consistent monitoring of prediction results, providing evidence-based understandings required for informed decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa718938177cf3d00e832bde3c94d3099493410" target='_blank'>
              Analysis of learning curves in predictive modeling using exponential curve fitting with an asymptotic approach
              </a>
            </td>
          <td>
            L. Vianna, Alexandre Leopoldo Gonçalves, João Artur Souza
          </td>
          <td>2024-04-18</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Inferring potential drug indications plays a vital role in the drug discovery process. It can be time-consuming and costly to discover novel drug indications through biological experiments. Recently, graph learning-based methods have gained popularity for this task. These methods typically treat the prediction task as a binary classification problem, focusing on modeling associations between drugs and diseases within a graph. However, labeled data for drug indication prediction is often limited and expensive to acquire. Contrastive learning addresses this challenge by aligning similar drug-disease pairs and separating dissimilar pairs in the embedding space. Thus, we developed a model called DrIGCL for drug indication prediction, which utilizes graph convolutional networks and contrastive learning. DrIGCL incorporates drug structure, disease comorbidities, and known drug indications to extract representations of drugs and diseases. By combining contrastive and classification losses, DrIGCL predicts drug indications effectively. In multiple runs of hold-out validation experiments, DrIGCL consistently outperformed existing computational methods for drug indication prediction, particularly in terms of top-k. Furthermore, our ablation study has demonstrated a significant improvement in the predictive capabilities of our model when utilizing contrastive learning. Finally, we validated the practical usefulness of DrIGCL by examining the predicted novel indications of Aspirin. The prediction model's code is available at https://github.com/yuxunluo9/DrIGCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49694d45aeac19bdafc8996c5994d56753bc6156" target='_blank'>
              A computational framework for predicting novel drug indications using graph convolutional network with contrastive learning.
              </a>
            </td>
          <td>
            Yuxun Luo, Wenyu Shan, Li Peng, Lingyun Luo, Pingjian Ding, Wei Liang
          </td>
          <td>2024-04-12</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Translational research in toxicology is essential for understanding how molecular alterations manifest across various biological systems to, for example, decrease reliance on animal models and extrapolation from animals to humans. Toxicogenomics (TGx) significantly contributes to assessing chemical and drug toxicity by providing insights into underlying toxicity mechanisms and developing gene expression-based biomarkers for toxicant classification. Despite the recognized need for a multi-organ approach in evaluating organism-level toxicity, most TGx research has been focused on a limited number of organs, primarily the liver, due to resource-intensive experiments. This paper is the first effort to utilize Generative Adversarial Network (GAN) for bidirectional translation of transcriptomic profiles between organs under chemical treatment. In this study, we developed a novel GAN model, TransTox, to bridge transcriptomic data between the liver and kidney. This model demonstrated robust performance in various evaluations, including external validation on independent datasets from both the training set’s source labs and a different lab. The study investigated the concordance between the real data and synthetic data generated by TransTox in elucidating toxicity mechanisms with respect to differential expressed genes (DEGs) and enriched pathways analyses. It showed comparable results in comparison to that obtained from real experimental settings. Moreover, TransTox proved valuable in biomarker applications, where synthetic data could be used to develop valid biomarkers or serve as “digital twins” for diagnostic applications. TransTox holds the potential to extend insights into toxicological effects in other organs by leveraging historical liver-centric TGx experiments, thereby opening avenues for reducing reliance on animal testing in organ toxicity research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126365a7e1f03e51528ccab96ab069e0d26d5990" target='_blank'>
              Bidirectional Translation of Transcriptomic Profiles between Liver and Kidney under Drug Treatment Using Generative Adversarial Networks (GANs)
              </a>
            </td>
          <td>
            Ting Li, Xi Chen, Weida Tong
          </td>
          <td>2024-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21038fc9ac697b06d9c2823da9c7d54fcaab783e" target='_blank'>
              Elucidating the semantics-topology trade-off for knowledge inference-based pharmacological discovery
              </a>
            </td>
          <td>
            Daniel N. Sosa, Georgiana Neculae, Julien Fauqueur, Russ B. Altman
          </td>
          <td>2024-05-01</td>
          <td>Journal of Biomedical Semantics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Since infectious pathogens start spreading into a susceptible population, mathematical models can provide policy makers with reliable forecasts and scenario analyses, which can be concretely implemented or solely consulted. In these complex epidemiological scenarios, machine learning architectures can play an important role, since they directly reconstruct data-driven models circumventing the specific modelling choices and the parameter calibration, typical of classical compartmental models. In this work, we discuss the efficacy of Kernel Operator Learning (KOL) to reconstruct population dynamics during epidemic outbreaks, where the transmission rate is ruled by an input strategy. In particular, we introduce two surrogate models, named KOL-m and KOL-$\partial$, which reconstruct in two different ways the evolution of the epidemics. Moreover, we evaluate the generalization performances of the two approaches with different kernels, including the Neural Tangent Kernels, and compare them with a classical neural network model learning method. Employing synthetic but semi-realistic data, we show how the two introduced approaches are suitable for realizing fast and robust forecasts and scenario analyses, and how these approaches are competitive for determining optimal intervention strategies with respect to specific performance measures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202709203f4dc2978114ea869baab2fd2e010a52" target='_blank'>
              Learning epidemic trajectories through Kernel Operator Learning: from modelling to optimal control
              </a>
            </td>
          <td>
            Giovanni Ziarelli, N. Parolini, M. Verani
          </td>
          <td>2024-04-17</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="In recent years, there has been a growing interest in using machine learning techniques for the estimation of treatment effects. Most of the best-performing methods rely on representation learning strategies that encourage shared behavior among potential outcomes to increase the precision of treatment effect estimates. In this paper we discuss and classify these models in terms of their algorithmic inductive biases and present a new model, NN-CGC, that considers additional information from the causal graph. NN-CGC tackles bias resulting from spurious variable interactions by implementing novel constraints on models, and it can be integrated with other representation learning methods. We test the effectiveness of our method using three different base models on common benchmarks. Our results indicate that our model constraints lead to significant improvements, achieving new state-of-the-art results in treatment effects estimation. We also show that our method is robust to imperfect causal graphs and that using partial causal information is preferable to ignoring it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ad7ed455fc11c02f75f46966ed60a3ebdf8b821" target='_blank'>
              Neural Networks with Causal Graph Constraints: A New Approach for Treatment Effects Estimation
              </a>
            </td>
          <td>
            Roger Pros, Jordi Vitria
          </td>
          <td>2024-04-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Molecular Property Prediction (MPP) is vital for drug discovery, crop protection, and environmental science. Over the last decades, diverse computational techniques have been developed, from using simple physical and chemical properties and molecular fingerprints in statistical models and classical machine learning to advanced deep learning approaches. In this review, we aim to distill insights from current research on employing transformer models for MPP. We analyze the currently available models and explore key questions that arise when training and fine-tuning a transformer model for MPP. These questions encompass the choice and scale of the pre-training data, optimal architecture selections, and promising pre-training objectives. Our analysis highlights areas not yet covered in current research, inviting further exploration to enhance the field's understanding. Additionally, we address the challenges in comparing different models, emphasizing the need for standardized data splitting and robust statistical analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95085c326a7773552ac82629ffc405be371422b" target='_blank'>
              Transformers for molecular property prediction: Lessons learned from the past five years
              </a>
            </td>
          <td>
            Afnan Sultan, Jochen Sieg, M. Mathea, Andrea Volkamer
          </td>
          <td>2024-04-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="This paper explores the efficacy of diffusion-based generative models as neural operators for partial differential equations (PDEs). Neural operators are neural networks that learn a mapping from the parameter space to the solution space of PDEs from data, and they can also solve the inverse problem of estimating the parameter from the solution. Diffusion models excel in many domains, but their potential as neural operators has not been thoroughly explored. In this work, we show that diffusion-based generative models exhibit many properties favourable for neural operators, and they can effectively generate the solution of a PDE conditionally on the parameter or recover the unobserved parts of the system. We propose to train a single model adaptable to multiple tasks, by alternating between the tasks during training. In our experiments with multiple realistic dynamical systems, diffusion models outperform other neural operators. Furthermore, we demonstrate how the probabilistic diffusion model can elegantly deal with systems which are only partially identifiable, by producing samples corresponding to the different possible solutions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f7dd73bdb8cf078d6f19120987ab3c21100bc5" target='_blank'>
              Diffusion models as probabilistic neural operators for recovering unobserved states of dynamical systems
              </a>
            </td>
          <td>
            Katsiaryna Haitsiukevich, O. Poyraz, Pekka Marttinen, Alexander Ilin
          </td>
          <td>2024-05-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="This review critically examines the integration of Machine Learning (ML) in drug discovery, highlighting its applications across target identification, hit discovery, lead optimization, and predictive toxicology. Despite ML's potential to revolutionize drug discovery through enhanced efficiency, predictive accuracy, and novel insights, significant challenges persist. These include issues related to data quality, model interpretability, integration into existing workflows, and regulatory and ethical considerations. The review advocates for advancements in algorithmic approaches, interdisciplinary collaboration, improved data-sharing practices, and evolving regulatory frameworks as potential solutions to these challenges. By addressing these hurdles and leveraging the capabilities of ML, the drug discovery process can be significantly accelerated, paving the way for the development of new therapeutics. This review calls for continued research, collaboration, and dialogue among stakeholders to realize the transformative potential of ML in drug discovery fully. 
Keywords: Machine Learning, Drug Discovery, Predictive Toxicology, Data Quality, Interdisciplinary Collaboration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4063e1611a5c48529290411ce341b6fc7e82815a" target='_blank'>
              MACHINE LEARNING IN DRUG DISCOVERY: A CRITICAL REVIEW OF APPLICATIONS AND CHALLENGES
              </a>
            </td>
          <td>
            Francisca Chibugo Udegbe, Ogochukwu Roseline Ebulue, Charles Chukwudalu Ebulue, Chukwunonso Sylvester Ekesiobi
          </td>
          <td>2024-04-17</td>
          <td>Computer Science &amp; IT Research Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Deep neural networks (DNNs) have achieved significant advancements in click-through rate (CTR) prediction by demonstrating strong generalization on training data. However, in real-world scenarios, the assumption of independent and identically distributed (i.i.d.) conditions, which is fundamental to this problem, is often violated due to temporal distribution shifts. This violation can lead to suboptimal model performance when optimizing empirical risk without access to future data, resulting in overfitting on the training data and convergence to a single sharp minimum. To address this challenge, we propose a novel model updating framework called Slow and Fast Trajectory Learning (SFTL) network. SFTL aims to mitigate the discrepancy between past and future domains while quickly adapting to recent changes in small temporal drifts. This mechanism entails two interactions among three complementary learners: (i) the Working Learner, which updates model parameters using modern optimizers (e.g., Adam, Adagrad) and serves as the primary learner in the recommendation system, (ii) the Slow Learner, which is updated in each temporal domain by directly assigning the model weights of the working learner, and (iii) the Fast Learner, which is updated in each iteration by assigning exponentially moving average weights of the working learner. Additionally, we propose a novel rank-based trajectory loss to facilitate interaction between the working learner and trajectory learner, aiming to adapt to temporal drift and enhance performance in the current domain compared to the past. We provide theoretical understanding and conduct extensive experiments on real-world CTR prediction datasets to validate the effectiveness and efficiency of SFTL in terms of both convergence speed and model performance. The results demonstrate the superiority of SFTL over existing approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30170172a98512badd6b1e6ad2ccb3e7d97e116e" target='_blank'>
              Generalize for Future: Slow and Fast Trajectory Learning for CTR Prediction
              </a>
            </td>
          <td>
            Jian Zhu, Congcong Liu, Xue Jiang, Changping Peng, Zhangang Lin, Jingping Shao
          </td>
          <td>2024-03-24</td>
          <td>DBLP</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Drug resistance to chemotherapeutic agents remains a formidable challenge in cancer treatment, significantly impacting treatment efficacy. Extensive research has exposed the intimate involvement of noncoding RNAs (ncRNAs) in conferring resistance to cancer drugs. Understanding the intricate associations between ncRNAs and drug resistance is of pivotal importance in advancing clinical interventions and expediting drug development. However, traditional biological experimental methods are hampered by limitations, such as labor intensiveness, time consumption, and constraints in scalability. Addressing these challenges necessitates the development of efficient computational methods for the accurate prediction of potential ncRNA-drug resistance associations (NDRA). However, most existing predictive models primarily focus on known ncRNA-drug resistance associations, often neglecting the critical aspect of similarity information between ncRNAs and drug resistance. This oversight may hinder the accuracy of characterizing these associations. To overcome the limitations of existing computational models, we proposed B-NDRA, a computational framework designed for the discovery of drug resistance-related ncRNA. Initially, we constructed a heterogeneous graph that integrates ncRNA-drug resistance pairs, leveraging both known associations and similarity fusion information between ncRNAs and drug resistance. Subsequently, we employed an attention mechanism to aggregate local features of graph nodes following a dimensionality reduction of node features. Further, a graph neural network (GNN) facilitated the learning of global node embeddings. Notably, the integration of dual adaptive deep adjustment architectures, encompassing intrablock and interblock methodologies, enabled efficient extraction of global features while balancing local and global features. Finally, B-NDRA employed a multilayer perceptron to predict associations between ncRNAs and drug resistance. Through rigorous 5-fold cross-validation, B-NDRA achieved average AUC, AUPR, Accuracy, Precision, Recall, and F1-score values of 92.2%, 91.9%, 84.88%, 86.9%, 82.37%, and 84.44%, respectively. Furthermore, comparative evaluations were conducted on established models, namely, GAEMDA, GRPAMDA, and LRGCPND. The results, obtained through three distinct 5-fold cross-validation strategies, demonstrated a notable performance improvement across almost all metrics for our B-NDRA. Specific case studies targeting Doxorubicin and Imatinib further validated the practicality of our B-NDRA in discovering potential NDRA. These results confirm the potential of our B-NDRA as a valuable tool in advancing cancer research and therapeutic development. The source code and data set of B-NDRA can be found at https://github.com/XuanLi1145/B-NDRA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f90daae5dfa441b98e8280b638f1290aaf9c8bcd" target='_blank'>
              Empowering Graph Neural Networks with Block-Based Dual Adaptive Deep Adjustment for Drug Resistance-Related NcRNA Discovery
              </a>
            </td>
          <td>
            Yi Zhang, Xuanzhao Li
          </td>
          <td>2024-03-24</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Stochastic reaction networks are widely used in the modeling of stochastic systems across diverse domains such as biology, chemistry, physics, and ecology. However, the comprehension of the dynamic behaviors inherent in stochastic reaction networks is a formidable undertaking, primarily due to the exponential growth in the number of possible states or trajectories as the state space dimension increases. In this study, we introduce a knowledge distillation method based on reinforcement learning principles, aimed at compressing the dynamical knowledge encoded in stochastic reaction networks into a singular neural network construct. The trained neural network possesses the capability to accurately predict the state conditional joint probability distribution that corresponds to the given query contexts, when prompted with rate parameters, initial conditions, and time values. This obviates the need to track the dynamical process, enabling the direct estimation of normalized state and trajectory probabilities, without necessitating the integration over the complete state space. By applying our method to representative examples, we have observed a high degree of accuracy in both multimodal and high-dimensional systems. Additionally, the trained neural network can serve as a foundational model for developing efficient algorithms for parameter inference and trajectory ensemble generation. These results collectively underscore the efficacy of our approach as a universal means of distilling knowledge from stochastic reaction networks. Importantly, our methodology also spotlights the potential utility in harnessing a singular, pretrained, large-scale model to encapsulate the solution space underpinning a wide spectrum of stochastic dynamical systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ceacaafe31a2cf6dd9a5da06470e4a7369110d8" target='_blank'>
              Distilling dynamical knowledge from stochastic reaction networks.
              </a>
            </td>
          <td>
            Chuanbo Liu, Jin Wang
          </td>
          <td>2024-03-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59062dea4ff554dc8afa542056cfdf212280e698" target='_blank'>
              SurvConvMixer: robust and interpretable cancer survival prediction based on ConvMixer using pathway-level gene expression images
              </a>
            </td>
          <td>
            Shuo Wang, Yuanning Liu, Hao Zhang, Zhen Liu
          </td>
          <td>2024-03-27</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/857fee206820f4063f7c9f0278f5accd39ea0ab1" target='_blank'>
              Pseudo datasets explain artificial neural networks
              </a>
            </td>
          <td>
            Yi-Chi Chu, Yi-Hau Chen, Chao-Yu Guo
          </td>
          <td>2024-04-10</td>
          <td>International Journal of Data Science and Analytics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Computationally predicting the performance of catalysts under reaction conditions is a challenging task due to the complexity of catalytic surfaces and their evolution in situ, different reaction paths, and the presence of solid-liquid interfaces in the case of electrochemistry. We demonstrate here how relatively simple machine learning models can be found that enable prediction of experimentally observed onset potentials. Inputs to our model are comprised of data from the oxygen reduction reaction on non-precious transition-metal antimony oxide nanoparticulate catalysts with a combination of experimental conditions and computationally affordable bulk atomic and electronic structural descriptors from density functional theory simulations. From human-interpretable genetic programming models, we identify key experimental descriptors and key supplemental bulk electronic and atomic structural descriptors that govern trends in onset potentials for these oxides and deduce how these descriptors should be tuned to increase onset potentials. We finally validate these machine learning predictions by experimentally confirming that scandium as a dopant in nickel antimony oxide leads to a desired onset potential increase. Macroscopic experimental factors are found to be crucially important descriptors to be considered for models of catalytic performance, highlighting the important role machine learning can play here even in the presence of small datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b431a2fefdae9f651abc6517db280557c0655021" target='_blank'>
              Interpretable Machine Learning Models for Practical Antimonate Electrocatalyst Performance.
              </a>
            </td>
          <td>
            Shyam Deo, Melissa E. Kreider, G. Kamat, McKenzie Hubert, J. Z. Zamora Zeledon, Ling Wei, Jesse Matthews, Nathaniel Keyes, Ishaan Singh, Thomas F. Jaramillo, F. Abild-Pedersen, Michaela Burke Stevens, Kirsten Winther, Johannes Voss
          </td>
          <td>2024-03-28</td>
          <td>Chemphyschem : a European journal of chemical physics and physical chemistry</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="The discovery of linear embedding is the key to the synthesis of linear control techniques for nonlinear systems. In recent years, while Koopman operator theory has become a prominent approach for learning these linear embeddings through data-driven methods, these algorithms often exhibit limitations in generalizability beyond the distribution captured by training data and are not robust to changes in the nominal system dynamics induced by intrinsic or environmental factors. To overcome these limitations, this study presents an adaptive Koopman architecture capable of responding to the changes in system dynamics online. The proposed framework initially employs an autoencoder-based neural network that utilizes input-output information from the nominal system to learn the corresponding Koopman embedding offline. Subsequently, we augment this nominal Koopman architecture with a feed-forward neural network that learns to modify the nominal dynamics in response to any deviation between the predicted and observed lifted states, leading to improved generalization and robustness to a wide range of uncertainties and disturbances compared to contemporary methods. Extensive tracking control simulations, which are undertaken by integrating the proposed scheme within a Model Predictive Control framework, are used to highlight its robustness against measurement noise, disturbances, and parametric variations in system dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302db61f58f8a2e62340fcfaacbceec2620e551a" target='_blank'>
              Adaptive Koopman Embedding for Robust Control of Complex Dynamical Systems
              </a>
            </td>
          <td>
            Rajpal Singh, Chandan Kumar Sah, J. Keshavan
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Gene essentiality, the necessity of a specific gene for the survival of an organism, is crucial to our understanding of cellular processes and identifying drug targets. Experimental determination of gene essentiality requires large growth screens that are time-consuming and expensive, motivating the development of in-silico approaches. Existing methods predominantly utilise flux balance analysis (FBA), a constraint-based optimisation algorithm; however, they are fundamentally limited by the necessity of a predefined cellular objective function. This requirement introduces an element of observer bias, as the objective function often reflects the researcher's assumptions rather than the cell's biological goals. Here, we present FluxGAT, a graph neural network (GNN) model capable of predicting gene essentiality directly from graphical representations of flux sampling data. Flux sampling removes the need for objective functions, thereby eliminating observer bias. FluxGAT leverages the unique strengths of GNNs in learning representations of complex relationships within metabolic reaction networks. The success of our approach in predicting experimentally determined gene essentiality, with almost double the sensitivity of FBA, explores the possibility of predicting cellular phenotypes in cases when objectives are less understood. Thus, we demonstrate a method for more general gene essentiality predictions across a broader spectrum of biological systems and environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de3e7a8e0e2b64c5846a28712b94cb18ecbeacc6" target='_blank'>
              FluxGAT: Integrating Flux Sampling with Graph Neural Networks for Unbiased Gene Essentiality Classification
              </a>
            </td>
          <td>
            Kieren Sharma, Lucia Marucci, Z. Abdallah
          </td>
          <td>2024-03-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Gastric cancer (GC) is a prevalent malignancy worldwide, ranking as the fifth most common cancer with over 1 million new cases and 700 thousand deaths in 2020. Locally advanced gastric cancer (LAGC) accounts for approximately two-thirds of GC diagnoses, and neoadjuvant chemotherapy (NACT) has emerged as the standard treatment for LAGC. However, the effectiveness of NACT varies significantly among patients, with a considerable subset displaying treatment resistance. Ineffective NACT not only leads to adverse effects but also misses the optimal therapeutic window, resulting in lower survival rate. However, existing multimodal learning methods assume the availability of all modalities for each patient, which does not align with the reality of clinical practice. The limited availability of modalities for each patient would cause information loss, adversely affecting predictive accuracy. In this study, we propose an incomplete multimodal data integration framework for GC (iMD4GC) to address the challenges posed by incomplete multimodal data, enabling precise response prediction and survival analysis. Specifically, iMD4GC incorporates unimodal attention layers for each modality to capture intra-modal information. Subsequently, the cross-modal interaction layers explore potential inter-modal interactions and capture complementary information across modalities, thereby enabling information compensation for missing modalities. To evaluate iMD4GC, we collected three multimodal datasets for GC study: GastricRes (698 cases) for response prediction, GastricSur (801 cases) for survival analysis, and TCGA-STAD (400 cases) for survival analysis. The scale of our datasets is significantly larger than previous studies. The iMD4GC achieved impressive performance with an 80.2% AUC on GastricRes, 71.4% C-index on GastricSur, and 66.1% C-index on TCGA-STAD, significantly surpassing other compared methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688425385f6f52ec92780394b8a2ddc85ad21191" target='_blank'>
              iMD4GC: Incomplete Multimodal Data Integration to Advance Precise Treatment Response Prediction and Survival Analysis for Gastric Cancer
              </a>
            </td>
          <td>
            Fengtao Zhou, Ying Xu, Yanfen Cui, Shenyang Zhang, Yun Zhu, Weiyang He, Jiguang Wang, Xin Wang, Ronald Chan, Louis Ho Shing Lau, Chu Han, Dafu Zhang, Zhenhui Li, Hao Chen
          </td>
          <td>2024-04-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Skin cancer mortality rates continue to rise, and survival analysis is increasingly needed to understand who is at risk and what interventions improve outcomes. However, current statistical methods are limited by inability to synthesize multiple data types, such as patient genetics, clinical history, demographics, and pathology and reveal significant multimodal relationships through predictive algorithms. Advances in computing power and data science enabled the rise of artificial intelligence (AI), which synthesizes vast amounts of data and applies algorithms that enable personalized diagnostic approaches. Here, we analyze AI methods used in skin cancer survival analysis, focusing on supervised learning, unsupervised learning, deep learning, and natural language processing. We illustrate strengths and weaknesses of these approaches with examples. Our PubMed search yielded 14 publications meeting inclusion criteria for this scoping review. Most publications focused on melanoma, particularly histopathologic interpretation with deep learning. Such concentration on a single type of skin cancer amid increasing focus on deep learning highlight growing areas for innovation; however, it also demonstrates opportunity for additional analysis that addresses other types of cutaneous malignancies and expands the scope of prognostication to combine both genetic, histopathologic, and clinical data. Moreover, researchers may leverage multiple AI methods for enhanced benefit in analyses. Expanding AI to this arena may enable improved survival analysis, targeted treatments, and outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c03be441acb79fa9e1b80312e9f34b546ca3baa4" target='_blank'>
              Current status of artificial intelligence methods for skin cancer survival analysis: a scoping review
              </a>
            </td>
          <td>
            Celine M. Schreidah, Emily R. Gordon, Oluwaseyi Adeuyan, Caroline Chen, Brigit A. Lapolla, Joshua A. Kent, George Bingham Reynolds, L. Fahmy, Chunhua Weng, Nicholas P. Tatonetti, Herbert S. Chase, I. Pe’er, L. J. Geskin
          </td>
          <td>2024-04-22</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Objective: Enhancing patient response to immune checkpoint inhibitors (ICIs) is crucial in cancer immunotherapy. This study aims to create a data-driven mathematical model of the tumor immune microenvironment (TIME) and utilize deep reinforcement learning (DRL) for optimizing patient-specific ICI therapy combined with chemotherapy (ICC). Methods: Using patients' genomic and transcriptomic data, we develop an ordinary differential equations (ODEs)-based TIME model to characterize interactions among chemotherapy, ICIs, immune cells, and cancer cells. A DRL algorithm is trained to determine the personalized optimal ICC therapy. Results: Numerical experiments with real-world data demonstrates that the proposed TIME model can predict ICI therapy response. The DRL-derived personalized ICC therapy outperforms predefined fixed schedules. For tumors with extremely low CD8+T cell infiltration ("extremely cold tumors"), DRL recommends high-dosage chemotherapy alone. For tumors with higher CD8+T cell infiltration ("cold" and "hot tumors"), an appropriate chemotherapy dosage induces CD8+T cell proliferation, enhancing ICI therapy outcomes. Specifically, for "hot tumors," chemotherapy and ICI are administered simultaneously, while for "cold tumors," a mid-dosage of chemotherapy makes the TIME "hotter" before ICI administration. However, a number of "cold tumors" with rapid resistant cancer cell growth, ICC eventually fails. Conclusion: This study highlights the potential of utilizing real-world clinical data and DRL to develop personalized optimal ICC by understanding the complex biological dynamics of a patient's TIME. Our ODE-based TIME model offers a theoretical framework for determining the best use of ICI, and the proposed DRL model may guide personalized ICC schedules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42e1a8b6d37982dcb30f20f3e82fd6f9629e7709" target='_blank'>
              Optimized patient-specific immune checkpoint inhibitors therapy for cancer treatment based on tumor immune microenvironment modeling
              </a>
            </td>
          <td>
            Y. Yao, F. Y. Chen, Q. Zhang
          </td>
          <td>2024-04-10</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65d94b705d85169b37491f103664b0080c773c3a" target='_blank'>
              Tribulations and future opportunities for artificial intelligence in precision medicine
              </a>
            </td>
          <td>
            Claudio Carini, A. Seyhan
          </td>
          <td>2024-04-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="This position paper takes a broad look at Physics-Enhanced Machine Learning (PEML) -- also known as Scientific Machine Learning -- with particular focus to those PEML strategies developed to tackle dynamical systems' challenges. The need to go beyond Machine Learning (ML) strategies is driven by: (i) limited volume of informative data, (ii) avoiding accurate-but-wrong predictions; (iii) dealing with uncertainties; (iv) providing Explainable and Interpretable inferences. A general definition of PEML is provided by considering four physics and domain knowledge biases, and three broad groups of PEML approaches are discussed: physics-guided, physics-encoded and physics-informed. The advantages and challenges in developing PEML strategies for guiding high-consequence decision making in engineering applications involving complex dynamical systems, are presented.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8781213f7f1857ff688376f170953bcc9fee085" target='_blank'>
              Physics-Enhanced Machine Learning: a position paper for dynamical systems investigations
              </a>
            </td>
          <td>
            Alice Cicirello
          </td>
          <td>2024-05-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug combination therapies are well-established strategies for the treatment of cancer with low toxicity and fewer adverse effects. Computational drug synergy prediction approaches can accelerate the discovery of novel combination therapies, but the existing methods do not explicitly consider the key role of important substructures in producing synergistic effects. To this end, we propose a significant substructure-aware anticancer drug synergy prediction method, named SDDSynergy, to adaptively identify critical functional groups in drug synergy. SDDSynergy splits the task of predicting drug synergy into predicting the effect of individual substructures on cancer cell lines and highlights the impact of important substructures through a novel drug-cell line attention mechanism. And a substructure pair attention mechanism is incorporated to capture the information on internal substructure pairs interaction in drug combinations, which aids in predicting synergy. The substructures of different sizes and shapes are directly obtained from the molecular graph of the drugs by multilayer substructure information passing networks. Extensive experiments on three real-world data sets demonstrate that SDDSynergy outperforms other state-of-the-art methods. We also verify that many of the novel drug combinations predicted by SDDSynergy are supported by previous studies or clinical trials through an in-depth literature survey.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dedd963b7e4662076a7f800ab29413ba330ec4" target='_blank'>
              SDDSynergy: Learning Important Molecular Substructures for Explainable Anticancer Drug Synergy Prediction.
              </a>
            </td>
          <td>
            Yunjiong Liu, Peiliang Zhang, Chao Che, Ziqi Wei
          </td>
          <td>2024-04-30</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="A network, whose nodes are genes and whose directed edges represent positive or negative influences of a regulatory gene and its targets, is often used as a representation of causality. To infer a network, researchers often develop a machine learning model and then evaluate the model based on its match with experimentally verified “gold standard” edges. The desired result of such a model is a network that may extend the gold standard edges. Since networks are a form of visual representation, one can compare their utility with architectural or machine blueprints. Blueprints are clearly useful because they provide precise guidance to builders in construction. If the primary role of gene regulatory networks is to characterize causality, then such networks should be good tools of prediction because prediction is the actionable benefit of knowing causality. But are they? In this paper, we compare prediction quality based on “gold standard” regulatory edges from previous experimental work with non-linear models inferred from time series data across four different species. We show that the same non-linear machine learning models have better predictive performance, with improvements from 5.3% to 25.3% in terms of the reduction in the root mean square error (RMSE) compared with the same models based on the gold standard edges. Having established that networks fail to characterize causality properly, we suggest that causality research should focus on four goals: (i) predictive accuracy; (ii) a parsimonious enumeration of predictive regulatory genes for each target gene g; (iii) the identification of disjoint sets of predictive regulatory genes for each target g of roughly equal accuracy; and (iv) the construction of a bipartite network (whose node types are genes and models) representation of causality. We provide algorithms for all goals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd7eda0fbce6aa023e877e8ea11a162f2aca8b4" target='_blank'>
              Bipartite networks represent causality better than simple networks: evidence, algorithms, and applications
              </a>
            </td>
          <td>
            Bingran Shen, Gloria Curozzi, Dennis Shasha
          </td>
          <td>2024-05-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Purpose: Combination therapies are a promising approach for improving cancer treatment, but it is challenging to predict their resulting adverse events in a real-world setting. Experimental Design: We provide here a proof-of-concept study using 15 million patient records from the FDA Adverse Event Reporting System (FAERS). Complex adverse event frequencies of drugs or their combinations were visualized as heat maps onto a two-dimensional grid. Adverse event frequencies were shown as colors to assess the ratio between individual and combined drug effects. To capture these patterns, we trained a convolutional neural network (CNN) autoencoder using 7,300 single-drug heat maps. In addition, statistical synergy analyses were performed on the basis of BLISS independence or χ2 testing. Results: The trained CNN model was able to decode patterns, showing that adverse events occur in global rather than isolated and unique patterns. Patterns were not likely to be attributed to disease symptoms given their relatively limited contribution to drug-associated adverse events. Pattern recognition was validated using trial data from ClinicalTrials.gov and drug combination data. We examined the adverse event interactions of 140 drug combinations known to be avoided in the clinic and found that near all of them showed additive rather than synergistic interactions, also when assessed statistically. Conclusions: Our study provides a framework for analyzing adverse events and suggests that adverse drug interactions commonly result in additive effects with a high level of overlap of adverse event patterns. These real-world insights may advance the implementation of new combination therapies in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5bf4a6e2efa78ea71250a101d8fda8d86724c34" target='_blank'>
              A Real-world Toxicity Atlas Shows that Adverse Events of Combination Therapies Commonly Result in Additive Interactions
              </a>
            </td>
          <td>
            Asli Küçükosmanoglu, Silvia Scoarta, M. Houweling, N. Spînu, Thomas Wijnands, Niek Geerdink, Carolien Meskers, Georgi K. Kanev, Bert Kiewiet, M. Kouwenhoven, David Noske, Tom Wurdinger, Marianne Pouwer, Mark Wolff, Bart A Westerman
          </td>
          <td>2024-04-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Molecular property prediction is a key component of AI-driven drug discovery and molecular characterization learning. Despite recent advances, existing methods still face challenges such as limited ability to generalize, and inadequate representation of learning from unlabeled data, especially for tasks specific to molecular structures. To address these limitations, we introduce DIG-Mol, a novel self-supervised graph neural network framework for molecular property prediction. This architecture leverages the power of contrast learning with dual interaction mechanisms and unique molecular graph enhancement strategies. DIG-Mol integrates a momentum distillation network with two interconnected networks to efficiently improve molecular characterization. The framework's ability to extract key information about molecular structure and higher-order semantics is supported by minimizing loss of contrast. We have established DIG-Mol's state-of-the-art performance through extensive experimental evaluation in a variety of molecular property prediction tasks. In addition to demonstrating superior transferability in a small number of learning scenarios, our visualizations highlight DIG-Mol's enhanced interpretability and representation capabilities. These findings confirm the effectiveness of our approach in overcoming challenges faced by traditional methods and mark a significant advance in molecular property prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5644aee2f5b4bc0bb2a8738a0748e93641a85b6" target='_blank'>
              Contrastive Dual-Interaction Graph Neural Network for Molecular Property Prediction
              </a>
            </td>
          <td>
            Zexing Zhao, Guangsi Shi, Xiaopeng Wu, Ruohua Ren, Xiaojun Gao, Fuyi Li
          </td>
          <td>2024-05-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recent progress of deep generative models in the vision and language domain has stimulated significant interest in more structured data generation such as molecules. However, beyond generating new random molecules, efficient exploration and a comprehensive understanding of the vast chemical space are of great importance to molecular science and applications in drug design and materials discovery. In this paper, we propose a new framework, ChemFlow, to traverse chemical space through navigating the latent space learned by molecule generative models through flows. We introduce a dynamical system perspective that formulates the problem as learning a vector field that transports the mass of the molecular distribution to the region with desired molecular properties or structure diversity. Under this framework, we unify previous approaches on molecule latent space traversal and optimization and propose alternative competing methods incorporating different physical priors. We validate the efficacy of ChemFlow on molecule manipulation and single- and multi-objective molecule optimization tasks under both supervised and unsupervised molecular discovery settings. Codes and demos are publicly available on GitHub at https://github.com/garywei944/ChemFlow.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86e8d48c566221e91257c327ff1b729237cc177" target='_blank'>
              Navigating Chemical Space with Latent Flows
              </a>
            </td>
          <td>
            Guanghao Wei, Yining Huang, Chenru Duan, Yue Song, Yuanqi Du
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="When modeling the dynamics of infectious disease, the incorporation of contact network information allows for the capture of the non-randomness and heterogeneity of realistic contact patterns. Oftentimes, it is assumed that the underlying contact pattern is known with perfect certainty. However, in realistic settings, the observed data often serves as an imperfect proxy of the actual contact patterns in the population. Furthermore, the epidemic in the real world are often not fully observed; event times such as infection and recovery times may be missing. In order to conduct accurate inferences on parameters of contagion spread, it is crucial to incorporate these sources of uncertainty. In this paper, we propose the use of Mixture Density Network compressed ABC (MDN-ABC) to learn informative summary statistics for the available data. This method will allow for Bayesian inference on the epidemic parameters of a contagious process, while accounting for imperfect observations on the epidemic and the contact network. We will demonstrate the use of this method on simulated epidemics and networks, and extend this framework to analyze the spread of Tattoo Skin Disease (TSD) among bottlenose dolphins in Shark Bay, Australia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a33626cae7d2e848d107957fbee4daf5ada4a8f8" target='_blank'>
              Accounting for contact network uncertainty in epidemic inferences.
              </a>
            </td>
          <td>
            Maxwell H Wang, J. Onnela
          </td>
          <td>2024-04-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Transport phenomena (e.g., fluid flows) are governed by time-dependent partial differential equations (PDEs) describing mass, momentum, and energy conservation, and are ubiquitous in many engineering applications. However, deep learning architectures are fundamentally incompatible with the simulation of these PDEs. This paper clearly articulates and then solves this incompatibility. The local-dependency of generic transport PDEs implies that it only involves local information to predict the physical properties at a location in the next time step. However, the deep learning architecture will inevitably increase the scope of information to make such predictions as the number of layers increases, which can cause sluggish convergence and compromise generalizability. This paper aims to solve this problem by proposing a distributed data scoping method with linear time complexity to strictly limit the scope of information to predict the local properties. The numerical experiments over multiple physics show that our data scoping method significantly accelerates training convergence and improves the generalizability of benchmark models on large-scale engineering simulations. Specifically, over the geometries not included in the training data for heat transferring simulation, it can increase the accuracy of Convolutional Neural Networks (CNNs) by 21.7 \% and that of Fourier Neural Operators (FNOs) by 38.5 \% on average.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5355d8f1cafbe46f9c3a9a1f31193f495099644" target='_blank'>
              Data Scoping: Effectively Learning the Evolution of Generic Transport PDEs
              </a>
            </td>
          <td>
            Jiangce Chen, Wenzhuo Xu, Zeda Xu, Noelia Grande Guti'errez, S. Narra, Christopher McComb
          </td>
          <td>2024-05-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Motivation Signal transduction networks regulate a multitude of essential biological processes and are frequently aberrated in diseases such as cancer. Developing a mechanistic understanding of such networks is essential to understand disease or cell population specific signaling and to design effective treatment strategies. Typically, such networks are computationally reconstructed based on systematic perturbation experiments, followed by quantification of signaling protein activity. Recent technological advances now allow for the quantification of the activity of many (signaling) proteins simultaneously in single cells. This makes it feasible to reconstruct signaling networks from single cell data. Results Here we introduce single cell Comparative Network Reconstruction (scCNR) to derive signal transduction networks by exploiting the heterogeneity of single cell (phospho)protein measurements. scCNR treats stochastic variation in total protein abundances as natural perturbation experiments, whose effects propagate through the network. scCNR reconstructs cell population specific networks of the same underlying topology for cells from diverse populations. We extensively validated scCNR on simulated single cell data, and we applied it to a dataset of EGFR-inhibitor treated keratinocytes to recover signaling differences downstream of EGFR and in protein interactions associated with proliferation. scCNR will help to unravel the mechanistic signaling differences between cell populations by making use of single-cell data, and will subsequently guide the development of well-informed treatment strategies. Availability and implementation scCNR is available as a python module at https://github.com/ibivu/scmra. Additionally, code to reproduce all figures is available at https://github.com/tstohn/scmra_analysis. Supplementary information Supplementary information and data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/516c13660aa9eadff6cefbce4ac0292281fdc217" target='_blank'>
              Reconstructing and comparing signal transduction networks from single cell protein quantification data
              </a>
            </td>
          <td>
            Tim Stohn, Roderick van Eijl, Klaas W. Mulder, L. Wessels, Evert Bosdriesz
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Lung cancer starts when lung cells grow uncontrollably, forming tumours that make breathing difficult. There are more than 100 types of human cancer, and in most cases, it is untreatable due to the unavailability of medico-infrastructure and facilities, even though the USFDA approved 57 anticancer drugs in 2020 alone. WHO reported more than 10 million cancer-related deaths yearly, and lung cancer alone accounts for more than 1.80 million deaths and a few studies suggest lung cancer incidence and deaths may surpass 3.8 million and 3.2 million by 2050, which demands rapid drug designing and repurposing and the role of artificial intelligence (AI) found to be the best solutions. AI in lung cancer therapeutics has emerged as a significant area of research in recent years. This state-of-the-art review aims to explore the various applications of AI in lung cancer treatment and its potential to revolutionise patient care, and predictive models can analyse large datasets, including clinical data, genetic information, and treatment outcomes, for novel drug design and to generate personalised treatment recommendations, having the potential to optimise therapeutic strategies, enhance treatment efficacy, and minimise adverse effects.Methods: A thorough and extensive literature review was conducted after reading relevant research papers and book chapters of the last decade, indexed in PubMed and Scopus to get high-quality articles to compile this article. Several engineering conference proceedings have also been included, as they meet our quality review standards.Results: Advanced algorithms accelerate the process and improve efficiency, with accuracy beyond 95% in many cases, validated with traditional computational drug designing and repurposing approaches such as Molecular Docking and Dynamic Simulations. We have also compiled the use of convolutional neural networks, recurrent neural networks, generative adversarial networks, variational autoencoders, reinforcement learning, and many more.Conclusion: The role of AI in lung cancer therapeutics holds excellent promise through accurate detection, personalised treatment planning, novel drug design, drug repurposing, and decision support. AI can potentially transform lung cancer therapeutics by providing a robust solution that is most accurate in the least time, which can save the time and effort of experimental biological scientists. Advanced AI algorithms such as Convolutional Neural Networks, Recurrent Neural Networks, Generative Adversarial Networks, Variational Autoencoders, and Reinforcement Learning have been used in various drug repurposing articles, and even the drugs and vaccines are in clinical trial stages in just years which earlier were taking decades to get a drug or vaccine in market, and the SARS CoV-2 vaccine is the result for the same. However, further research and collaboration are required to address the existing challenges and fully realise the potential of AI in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbf816f075fede93331e91515cad6569a930a4a" target='_blank'>
              An Extensive Review on Lung Cancer Therapeutics Using Machine Learning Techniques: State-of-the-art and Perspectives.
              </a>
            </td>
          <td>
            Shaban Ahmad, Khalid Raza
          </td>
          <td>2024-04-25</td>
          <td>Journal of drug targeting</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="We train prediction and survival models using multi-omics data for disease risk identification and stratification. Existing work on disease prediction focuses on risk analysis using datasets of individual data types (metabolomic, genomics, demographic), while our study creates an integrated model for disease risk assessment. We compare machine learning models such as Lasso Regression, Multi-Layer Perceptron, XG Boost, and ADA Boost to analyze multi-omics data, incorporating ROC-AUC score comparisons for various diseases and feature combinations. Additionally, we train Cox proportional hazard models for each disease to perform survival analysis. Although the integration of multi-omics data significantly improves risk prediction for 8 diseases, we find that the contribution of metabolomic data is marginal when compared to standard demographic, genetic, and biomarker features. Nonetheless, we see that metabolomics is a useful replacement for the standard biomarker panel when it is not readily available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d9967c5fe06a6563b4c5bd97156bf728eb4041d" target='_blank'>
              Integrative machine learning approaches for predicting disease risk using multi-omics data from the UK Biobank
              </a>
            </td>
          <td>
            Oscar Aguilar, Cheng Chang, Elsa Bismuth, Manuel A. Rivas
          </td>
          <td>2024-04-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d580741ad8e3ade35f1de1352b4098caef4d52a" target='_blank'>
              PMF-GRN: a variational inference approach to single-cell gene regulatory network inference using probabilistic matrix factorization
              </a>
            </td>
          <td>
            Claudia Skok Gibbs, O. Mahmood, Richard Bonneau, Kyunghyun Cho
          </td>
          <td>2024-04-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc5edf31ff74929bdf67e64bc922c3d121ffeb8" target='_blank'>
              PERCEPTION predicts patient response and resistance to treatment using single-cell transcriptomics of their tumors.
              </a>
            </td>
          <td>
            Sanju Sinha, Rahulsimham Vegesna, Sumit Mukherjee, Ashwin V Kammula, S. R. Dhruba, Wei Wu, D. Kerr, N. Nair, Matthew G Jones, N. Yosef, Oleg V Stroganov, Ivan Grishagin, Kenneth D Aldape, Collin M Blakely, Peng Jiang, Craig J Thomas, C. Benes, T. Bivona, Alejandro A. Schäffer, E. Ruppin
          </td>
          <td>2024-04-18</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Abstract Motivation The crisis of antibiotic resistance, which causes antibiotics used to treat bacterial infections to become less effective, has emerged as one of the foremost challenges to public health. Identifying the properties of antibiotic resistance genes (ARGs) is an essential way to mitigate this issue. Although numerous methods have been proposed for this task, most of these approaches concentrate solely on predicting antibiotic class, disregarding other important properties of ARGs. In addition, existing methods for simultaneously predicting multiple properties of ARGs fail to account for the causal relationships among these properties, limiting the predictive performance. Results In this study, we propose a causality-guided framework for annotating properties of ARGs, in which causal inference is utilized for representation learning. More specifically, the hidden biological patterns determining the properties of ARGs are described by a Gaussian Mixture Model, and procedure of causal representation learning is used to derive the hidden features. In addition, a causal graph among different properties is constructed to capture the causal relationships among properties of ARGs, which is integrated into the task of annotating properties of ARGs. The experimental results on a real-world dataset demonstrate the effectiveness of the proposed framework on the task of annotating properties of ARGs. Availability and implementation The data and source codes are available in GitHub at https://github.com/David-WZhao/CausalARG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df7231709034f1f37f2b153d2161d1e6da9e2c7f" target='_blank'>
              Causal-ARG: a causality-guided framework for annotating properties of antibiotic resistance genes
              </a>
            </td>
          <td>
            Weizhong Zhao, Junze Wu, Xingpeng Jiang, Tingting He, Xiaohua Hu
          </td>
          <td>2024-04-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Neural networks in fluid mechanics offer an efficient approach for exploring complex flows, including multiphase and free surface flows. The recurrent neural network, particularly the Long Short-Term Memory (LSTM) model, proves attractive for learning mappings from transient inputs to dynamic outputs. This study applies LSTM to predict transient and static outputs for fluid flows under surface tension effects. Specifically, we explore two distinct droplet dynamic scenarios: droplets with diverse initial shapes impacting with solid surfaces, as well as the coalescence of two droplets following collision. Using only dimensionless numbers and geometric time series data from numerical simulations, LSTM predicts the energy budget. The marker-and-cell front-tracking methodology combined with a marker-and-cell finite-difference strategy is adopted for simulating the droplet dynamics. Using a recurrent neural network (RNN) architecture fed with time series data derived from geometrical parameters, as for example droplet diameter variation, our study shows the accuracy of our approach in predicting energy budgets, as for instance the kinetic, dissipation, and surface energy trends, across a range of Reynolds and Weber numbers in droplet dynamic problems. Finally, a two-phase sequential neural network using only geometric data, which is readily available in experimental settings, is employed to predict the energies and then use them to estimate static parameters, such as the Reynolds and Weber numbers. While our methodology has been primarily validated with simulation data, its adaptability to experimental datasets is a promising avenue for future exploration. We hope that our strategy can be useful for diverse applications, spanning from inkjet printing to combustion engines, where the prediction of energy budgets or dissipation energies is crucial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3e51c446c3a8a6b0a0b8275259b4269b089e2ed" target='_blank'>
              Predicting Energy Budgets in Droplet Dynamics: A Recurrent Neural Network Approach
              </a>
            </td>
          <td>
            Diego A. de Aguiar, Hugo L. França, C. Oishi
          </td>
          <td>2024-03-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e40dc84348e865eac9c2cf362eecd90fa43f44" target='_blank'>
              Performance and robustness of small molecule retention time prediction with molecular graph neural networks in industrial drug discovery campaigns
              </a>
            </td>
          <td>
            Daniel Vik, David Pii, Chirag Mudaliar, Mads Nørregaard-Madsen, Aleksejs Kontijevskis
          </td>
          <td>2024-04-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Determining the structure of gene regulatory networks (GRNs) is a central problem in biology, with a variety of inference methods available for different types of data. However, for a prominent and intricate scenario with single-cell gene expression data collected post-intervention across multiple time points, where joint distributions remain unknown, there is only one known specifically developed method, which does not fully utilize the rich information contained in this data type. In response, we introduce an inference approach tailored to this challenging context: netWork infErence by covariaNce DYnamics, dubbed WENDY. The core idea of WENDY is to model the dynamics of the covariance matrix, and solve this dynamics as an optimization problem to determine the regulatory relationships. To assess its efficacy, we benchmark WENDY against alternative inference methods using synthetic data. Our findings underscore WENDY’s robust performance across diverse synthetic datasets. Moreover, we deploy WENDY to analyze three distinct experimental datasets, uncovering potential gene regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b4ba528526bb4328fc81d6fd71855cf3d45b87" target='_blank'>
              WENDY: Gene Regulatory Network Inference with Covariance Dynamics
              </a>
            </td>
          <td>
            Yue Wang, Peng Zheng, Yu-Chen Cheng, Zikun Wang, Aleksandr Y Aravkin
          </td>
          <td>2024-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Learning complex trajectories from demonstrations in robotic tasks has been effectively addressed through the utilization of Dynamical Systems (DS). State-of-the-art DS learning methods ensure stability of the generated trajectories; however, they have three shortcomings: a) the DS is assumed to have a single attractor, which limits the diversity of tasks it can achieve, b) state derivative information is assumed to be available in the learning process and c) the state of the DS is assumed to be measurable at inference time. We propose a class of provably stable latent DS with possibly multiple attractors, that inherit the training methods of Neural Ordinary Differential Equations, thus, dropping the dependency on state derivative information. A diffeomorphic mapping for the output and a loss that captures time-invariant trajectory similarity are proposed. We validate the efficacy of our approach through experiments conducted on a public dataset of handwritten shapes and within a simulated object manipulation task.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc227c83d593a317b47926de7d4a6905d2fc78a4" target='_blank'>
              Learning Deep Dynamical Systems using Stable Neural ODEs
              </a>
            </td>
          <td>
            Andreas Sochopoulos, M. Gienger, S. Vijayakumar
          </td>
          <td>2024-04-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Continuous time recurrent neural networks (CTRNNs) are systems of coupled ordinary differential equations (ODEs) inspired by the structure of neural networks in the brain. CTRNNs are known to be universal dynamical approximators: given a large enough system, the parameters of a CTRNN can be tuned to produce output that is arbitrarily close to that of any other dynamical system. However, in practice, both designing systems of CTRNN to have a certain output, and the reverse-understanding the dynamics of a given system of CTRNN-can be nontrivial. In this article, we describe a method for embedding any specified Turing machine in its entirety into a CTRNN. As such, we describe in detail a continuous time dynamical system that performs arbitrary discrete-state computations. We suggest that in acting as both a continuous time dynamical system and as a computer, the study of such systems can help refine and advance the debate concerning the Computational Hypothesis that cognition is a form of computation and the Dynamical Hypothesis that cognitive systems are dynamical systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4c18d67633c17f70ffc53a4f2853ff51348378c" target='_blank'>
              A Continuous Time Dynamical Turing Machine.
              </a>
            </td>
          <td>
            C. Postlethwaite, Peter Ashwin, Matthew Egbert
          </td>
          <td>2024-05-16</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="New technology for energy storage is necessary for the large-scale adoption of renewable energy sources like wind and solar. The ability to discover suitable catalysts is crucial for making energy storage more cost-effective and scalable. The Open Catalyst Project aims to apply advances in graph neural networks (GNNs) to accelerate progress in catalyst discovery, replacing Density Functional Theory-based (DFT) approaches that are computationally burdensome. Current approaches involve scaling GNNs to over 1 billion parameters, pushing the problem out of reach for a vast majority of machine learning practitioner around the world. This study aims to evaluate the performance and insights gained from using more lightweight approaches for this task that are more approachable for smaller teams to encourage participation from individuals from diverse backgrounds. By implementing robust design patterns like geometric and symmetric message passing, we were able to train a GNN model that reached a MAE of 0.0748 in predicting the per-atom forces of adsorbate-surface interactions, rivaling established model architectures like SchNet and DimeNet++ while using only a fraction of trainable parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1c2c8bf4df3944736f857ce3044d53ae1822ed" target='_blank'>
              Lightweight Geometric Deep Learning for Molecular Modelling in Catalyst Discovery
              </a>
            </td>
          <td>
            Patrick Geitner
          </td>
          <td>2024-04-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 The temporal resolution of environmental data sets plays a major role in the granularity of the information that can be derived from the data. In most cases, it is required that different data sets have a common temporal resolution to enable their consistent evaluations and applications in making informed decisions. This study leverages deep learning with long short-term memory (LSTM) neural networks and model inference to enhance the temporal resolution of climate datasets, specifically temperature, and precipitation, from daily to sub-daily scales. We trained our model to learn the relationship between daily and sub-daily data, subsequently applying this knowledge to increase the resolution of a separate dataset with a coarser (daily) temporal resolution. Our findings reveal a high degree of accuracy for temperature predictions, evidenced by a correlation of 0.99 and a mean absolute error of 0.21 °C, between the actual and predicted sub-daily values. In contrast, the approach was less effective for precipitation, achieving an explained variance of only 37%, compared to 98% for temperature. Further, besides the sub-daily interpolation of the climate data sets, we adapted our approach to increase the resolution of the Normalized difference vegetation index of Landsat (from 16-day to 5-day interval) using the LSTM model pre-trained from the Sentinel 2 Normalized difference vegetation index - that exists at a relatively higher temporal resolution. The explained variance between the predicted Landsat and Sentinel 1 data is 70% with a mean absolute error of 0.03. These results suggest that our method is particularly suitable for environmental datasets with less pronounced short-term variability, offering a promising tool for improving the resolution and utility of the data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b365a24bc7a55c6465814e7fae80c9eb78ae8b6" target='_blank'>
              Interpolation of Environmental Data Using Deep Learning and Model Inference
              </a>
            </td>
          <td>
            C. Ibebuchi, Itohan-Osa Abu
          </td>
          <td>2024-05-14</td>
          <td>Machine Learning: Science and Technology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Attaining the equilibrium geometry of a catalyst-adsorbate system is key to fundamentally assessing its effective properties, such as adsorption energy. While machine learning methods with advanced representation or supervision strategies have been applied to boost and guide the relaxation processes of catalysis systems, existing methods that produce linearly aggregated geometry predictions are susceptible to edge representations ambiguity, and are therefore vulnerable to graph variations. In this paper, we present a novel graph neural network (GNN) supervision and prediction strategy DR-Label. Our approach mitigates the multiplicity of solutions in edge representation and encourages model predictions that are independent of graph structural variations. DR-Label first Deconstructs finer-grained equilibrium state information to the model by projecting the node-level supervision signal to each edge. Reversely, the model Reconstructs a more robust equilibrium state prediction by converting edge-level predictions to node-level via a sphere-fitting algorithm. When applied to three fundamentally different models, DR-Label consistently enhanced performance. Leveraging the graph structure invariance of the DR-Label strategy, we further propose DRFormer, which applied explicit intermediate positional update and achieves a new state-of-the-art performance on the Open Catalyst 2020 (OC20) dataset and the Cu-based single-atom alloys CO adsorption (SAA) dataset. We expect our work to highlight vital principles for advancing geometric GNN models for catalysis systems and beyond. Our code is available at https://github.com/bowenwang77/DR-Label">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a6ea01655322f614d7ac84d7a94ab11c171c315" target='_blank'>
              DR-Label: Label Deconstruction and Reconstruction of GNN Models for Catalysis Systems
              </a>
            </td>
          <td>
            Bo-Lan Wang, Chen Liang, Jiaze Wang, Jiezhong Qiu, Furui Liu, Shaogang Hao, Dong Li, Guangyong Chen, Xiaolong Zou, P. Heng
          </td>
          <td>2024-03-24</td>
          <td>DBLP</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Predicting human displacements is crucial for addressing various societal challenges, including urban design, traffic congestion, epidemic management, and migration dynamics. While predictive models like deep learning and Markov models offer insights into individual mobility, they often struggle with out-of-routine behaviours. Our study introduces an approach that dynamically integrates individual and collective mobility behaviours, leveraging collective intelligence to enhance prediction accuracy. Evaluating the model on millions of privacy-preserving trajectories across three US cities, we demonstrate its superior performance in predicting out-of-routine mobility, surpassing even advanced deep learning methods. Spatial analysis highlights the model's effectiveness near urban areas with a high density of points of interest, where collective behaviours strongly influence mobility. During disruptive events like the COVID-19 pandemic, our model retains predictive capabilities, unlike individual-based models. By bridging the gap between individual and collective behaviours, our approach offers transparent and accurate predictions, crucial for addressing contemporary mobility challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ee655b343c17af6ef466dab1afa210ee1ccd584" target='_blank'>
              Mixing Individual and Collective Behaviours to Predict Out-of-Routine Mobility
              </a>
            </td>
          <td>
            Sebastiano Bontorin, Simone Centellegher, Riccardo Gallotti, Luca Pappalardo, Bruno Lepri, Massimiliano Luca
          </td>
          <td>2024-04-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Solubility is not only a significant physical property of molecules but also a vital factor in smallmolecule drug development. Determining drug solubility demands stringent equipment, controlled environments, and substantial human and material resources. The accurate prediction of drug solubility using computational methods has long been a goal for researchers. In this study, we introduce MSCSol, a solubility prediction model that integrates multidimensional molecular structure information. We incorporate a graph neural network with geometric vector perceptrons (GVP-GNN) to encode 3D molecular structures, representing spatial arrangement and orientation of atoms, as well as atomic sequences and interactions. We also employ Selective Kernel Convolution combined with Global and Local attention mechanisms to capture molecular features context at different scales. Additionally, various descriptors are calculated to enrich the molecular representation. For the 2D and 3D structural data of molecules, we design different data augmentation strategies to enhance generalization ability and prevent the model from learning irrelevant information. Extensive experiments on benchmark and independent datasets demonstrate MSCSol's superior performance. Ablation studies further confirm the effectiveness of different modules. Interpretability analysis highlights the importance of various atomic groups and substructures for solubility and verifies that our model effectively captures functional molecular structures and higher-order knowledge. The source code and datasets are freely available at https://github.com/ZiyuFanCSU/MSCSol.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aa873b8be5cb3b95dff75f43afd7b047937f102" target='_blank'>
              Enhancing Predictions of Drug Solubility through Multidimensional Structural Characterization Exploitation.
              </a>
            </td>
          <td>
            Ziyu Fan, Linying Chen, Xinyi Wu, Zhijian Huang, Lei Deng
          </td>
          <td>2024-05-07</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The ongoing expansion of human genomic datasets propels therapeutic target identification; however, extracting gene-disease associations from gene annotations remains challenging. Here, we introduce Mantis-ML 2.0, a framework integrating AstraZeneca’s Biological Insights Knowledge Graph and numerous tabular datasets, to assess gene-disease probabilities throughout the phenome. We use graph neural networks, capturing the graph’s holistic structure, and train them on hundreds of balanced datasets via a robust semi-supervised learning framework to provide gene-disease probabilities across the human exome. Mantis-ML 2.0 incorporates natural language processing to automate disease-relevant feature selection for thousands of diseases. The enhanced models demonstrate a 6.9% average classification power boost, achieving a median receiver operating characteristic (ROC) area under curve (AUC) score of 0.90 across 5220 diseases from Human Phenotype Ontology, OpenTargets, and Genomics England. Notably, Mantis-ML 2.0 prioritizes associations from an independent UK Biobank phenome-wide association study (PheWAS), providing a stronger form of triaging and mitigating against underpowered PheWAS associations. Results are exposed through an interactive web resource.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8a8dff231452d1dacf518826430fa5c77addce0" target='_blank'>
              Phenome-wide identification of therapeutic genetic targets, leveraging knowledge graphs, graph neural networks, and UK Biobank data
              </a>
            </td>
          <td>
            L. Middleton, Ioannis Melas, Chirag Vasavda, A. Raies, Benedek Rozemberczki, R. Dhindsa, Justin S. Dhindsa, Blake Weido, Quanli Wang, A. R. Harper, Gavin Edwards, S. Petrovski, D. Vitsios
          </td>
          <td>2024-05-08</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9e0572b203657323ee5ca7f8a580528219adc51" target='_blank'>
              Prospective de novo drug design with deep interactome learning
              </a>
            </td>
          <td>
            Kenneth Atz, Leandro Cotos, Clemens Isert, Maria Håkansson, Dorota Focht, Mattis Hilleke, David F. Nippa, Michael Iff, Jann Ledergerber, Carl C. G. Schiebroek, Valentina Romeo, J. Hiss, Daniel Merk, Petra Schneider, Bernd Kuhn, U. Grether, Gisbert Schneider
          </td>
          <td>2024-04-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Butyrylcholinesterase (BChE) is a target of interest in late-stage Alzheimer's Disease (AD) where selective BChE inhibitors (BIs) may offer symptomatic treatment without the harsh side effects of acetylcholinesterase (AChE) inhibitors. In this study, we explore multiple machine learning strategies to identify BIs in silico, optimizing for precision over all other metrics. We compare state-of-the-art supervised contrastive learning (CL) with deep learning (DL) and Random Forest (RF) machine learning, across single and sequential modeling configurations, to identify the best models for BChE selectivity. We used these models to virtually screen a vendor library of 5 million compounds for BIs and tested 20 of these compounds in vitro. Seven of the 20 compounds displayed selectivity for BChE over AChE, reflecting a hit rate of 35% for our model predictions, suggesting a highly efficient strategy for modeling selective inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1736a5d4242d00cc373cdab5f8375cb7ba6af57d" target='_blank'>
              Sequential Contrastive and Deep Learning Models to Identify Selective Butyrylcholinesterase Inhibitors
              </a>
            </td>
          <td>
            Mustafa Kemal Ozalp, Patricia A Vignaux, A. C. Puhl, Thomas R. Lane, Fabio Urbina, Sean Ekins
          </td>
          <td>2024-03-26</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In many real-world applications where the system dynamics has an underlying interdependency among its variables (such as power grid, economics, neuroscience, omics networks, environmental ecosystems, and others), one is often interested in knowing whether the past values of one time series influences the future of another, known as Granger causality, and the associated underlying dynamics. This paper introduces a Koopman-inspired framework that leverages neural networks for data-driven learning of the Koopman bases, termed NeuroKoopman Dynamic Causal Discovery (NKDCD), for reliably inferring the Granger causality along with the underlying nonlinear dynamics. NKDCD employs an autoencoder architecture that lifts the nonlinear dynamics to a higher dimension using data-learned bases, where the lifted time series can be reliably modeled linearly. The lifting function, the linear Granger causality lag matrices, and the projection function (from lifted space to base space) are all represented as multilayer perceptrons and are all learned simultaneously in one go. NKDCD also utilizes sparsity-inducing penalties on the weights of the lag matrices, encouraging the model to select only the needed causal dependencies within the data. Through extensive testing on practically applicable datasets, it is shown that the NKDCD outperforms the existing nonlinear Granger causality discovery approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2063b00f6c1d0ea80dc4fa01e9b3b3bddc847ea9" target='_blank'>
              NeuroKoopman Dynamic Causal Discovery
              </a>
            </td>
          <td>
            Rahmat Adesunkanmi, Balaji Sesha Srikanth Pokuri, Ratnesh Kumar
          </td>
          <td>2024-04-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Artificial intelligence holds promise to fundamentally enhance healthcare. Developing an integrated many-to-many framework leveraging multimodal data for multiple tasks is essential to unifying modern medicine. We introduce M3H, an explainable Multimodal Multitask Machine Learning for Healthcare framework that consolidates learning from tabular, time-series, language, and vision data for supervised binary/multiclass classification, regression, and unsupervised clustering. M3H encompasses an unprecedented range of medical tasks and problem domains and consistently outperforms traditional single-task models by on average 11.6% across 40 disease diagnoses from 16 medical departments, three hospital operation forecasts, and one patient phenotyping task. It features a novel attention mechanism balancing self-exploitation (learning source-task), and cross-exploration (learning cross-tasks), and offers explainability through a proposed TIM score, shedding light on the dynamics of task learning interdependencies. Its adaptable architecture supports easy customization and integration of new data modalities and tasks, establishing it as a robust, scalable solution for advancing AI-driven healthcare systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dcbec29a8491be77a1d7a2d2ac4c946562112b4" target='_blank'>
              M3H: Multimodal Multitask Machine Learning for Healthcare
              </a>
            </td>
          <td>
            Dimitris Bertsimas, Yu Ma
          </td>
          <td>2024-04-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Exploring the degree to which phenotypic variation, influenced by intrinsic nonlinear biological mechanisms, can be accurately captured using statistical methods is essential for advancing our comprehension of complex biological systems and predicting their functionality. Here, we examine this issue by combining a computational model of gene regulation networks with a linear additive prediction model, akin to polygenic scores utilized in genetic analyses. Inspired by the variational framework of quantitative genetics, we create a population of individual networks possessing identical topology yet showcasing diversity in regulatory strengths. By discerning which regulatory connections determine the prediction of phenotypes, we contextualize our findings within the framework of core and peripheral causal determinants, as proposed by the omnigenic model of complex traits. We establish connections between our results and concepts such as global sensitivity and local stability in dynamical systems, alongside the notion of sloppy parameters in biological models. Furthermore, we explore the implications of our investigation for the broader discourse surrounding the role of epistatic interactions in the prediction of complex phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9d655e857d6cdd755c46f5fbfbbbb85e241f74" target='_blank'>
              Polygenic prediction and gene regulation networks
              </a>
            </td>
          <td>
            J. F. Poyatos
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Drug-target interaction databases comprise millions of manually curated data points, yet there are missed opportunities for repurposing established interaction networks to infer novel drug-target interactions by interpolating on chemical neighborhoods. To address this gap, we collect drug-target interactions on 128 unique GPCRs across 187K molecules and establish an all-vs-all chemical space network, which can be efficiently calculated and parameterised as a graph with 32.4 billion potential edges. Beyond testing state-of-the-art machine learning approaches, we develop a chemical space neural network (CSNN) to infer drug activity classes with up to 98% accuracy, by leveraging the graph of chemical neighbourhoods. We combine this virtual library screen with a fast and cheap experimental platform to validate our predictions and to discover 14 novel drug-GPCR interactions. Altogether, our platform integrates virtual library screening and experimental validation to facilitate fast and efficient coverage of missing drug-target interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b5c062c6c74a88a48c33164dce0f2833a8fc1df" target='_blank'>
              Harnessing Chemical Space Neural Networks to Systematically Annotate GPCR ligands
              </a>
            </td>
          <td>
            Frederik G. Hansson, Niklas Gesmar Madsen, Lea Gram Hansen, Tadas Jakočiūnas, Bettina Lengger, Jay D. Keasling, Michael K Jensen, Carlos G. Acevedo-Rocha, Emil D. Jensen
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Motion prediction is a challenging problem in autonomous driving as it demands the system to comprehend stochastic dynamics and the multi-modal nature of real-world agent interactions. Diffusion models have recently risen to prominence, and have proven particularly effective in pedestrian motion prediction tasks. However, the significant time consumption and sensitivity to noise have limited the real-time predictive capability of diffusion models. In response to these impediments, we propose a novel diffusion-based, acceleratable framework that adeptly predicts future trajectories of agents with enhanced resistance to noise. The core idea of our model is to learn a coarse-grained prior distribution of trajectory, which can skip a large number of denoise steps. This advancement not only boosts sampling efficiency but also maintains the fidelity of prediction accuracy. Our method meets the rigorous real-time operational standards essential for autonomous vehicles, enabling prompt trajectory generation that is vital for secure and efficient navigation. Through extensive experiments, our method speeds up the inference time to 136ms compared to standard diffusion model, and achieves significant improvement in multi-agent motion prediction on the Argoverse 1 motion forecasting dataset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb88ca9828b8d82e6d7a523089d3bd17e7d419f" target='_blank'>
              ADM: Accelerated Diffusion Model via Estimated Priors for Robust Motion Prediction under Uncertainties
              </a>
            </td>
          <td>
            Jiahui Li, Tianle Shen, Zekai Gu, Jiawei Sun, Chengran Yuan, Yuhang Han, Shuo Sun, Marcelo H. Ang
          </td>
          <td>2024-05-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a3ce0a8aec6be70af6f6d05daafba40902c31f8" target='_blank'>
              MLASM: Machine learning based prediction of anticancer small molecules.
              </a>
            </td>
          <td>
            Priya Dharshini Balaji, Subathra Selvam, Honglae Sohn, T. Madhavan
          </td>
          <td>2024-03-30</td>
          <td>Molecular diversity</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Although phase separation is a ubiquitous phenomenon, the interactions between multiple components make it difficult to accurately model and predict. In recent years, machine learning has been widely used in physics simulations. Here, we present a physical information-enhanced graph neural network (PIENet) to simulate and predict the evolution of phase separation. The accuracy of our model in predicting particle positions is improved by 40.3% and 51.77% compared with CNN and SVM respectively. Moreover, we design an order parameter based on local density to measure the evolution of phase separation and analyze the systematic changes with different repulsion coefficients and different Schmidt numbers. The results demonstrate that our model can achieve long-term accurate predictions of order parameters without requiring complex handcrafted features. These results prove that graph neural networks can become new tools and methods for predicting the structure and properties of complex physical systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/000ae9d974cb78e5ef8f6f051d61197096a536b9" target='_blank'>
              Physical information-enhanced graph neural network for predicting phase separation
              </a>
            </td>
          <td>
            Yaqiang Zhang, Xuwen Wang, Yanan Wang, Wen Zheng
          </td>
          <td>2024-04-25</td>
          <td>Chinese Physics B</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Combination treatment–the mainstay of cancer therapy–often fails because treatment interactions evoke complex resistance mechanisms that are hard to predict. Designing combination therapy to prevent treatment resistance is especially challenging for rare cancers. Here, we introduce REsistance through COntext DRift (RECODR): a computational pipeline that tracks how genes change their transcriptome context across cancer development and drug treatment conditions. By applying RECODR to a genetically modified mouse model of choroid plexus carcinoma–a rare brain tumour of young children–we identified patterns of transcriptome evolution, cellular heterogeneity and treatment targets that emerged as tumours were initiated and resisted combination treatment. This enabled the prediction of treatment resistance mechanisms and the design of highly effective therapeutic protocols that avoided treatment failure. RECODR can describe complex and dynamic changes in normal and diseased tissues that could be applied to multimodal data from a variety of settings, mitigating treatment resistance across cancers and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920edce889e5d3cf5f44bc7f5d04ff4b697a9b88" target='_blank'>
              Natural language processing charts transcriptome evolution to design combination cancer therapies
              </a>
            </td>
          <td>
            Amir Jassim, Birgit V. Nimmervoll, Sabrina Terranova, Erica Nathan, Katherine E. Masih, Lisa Ruff, Matilde Duarte, Elizabeth Cooper, Linda P. Hu, Gunjan Katyal, Melika Akhbari, Reuben J. Gilbertson, Colton Terhune, Gabriel Balmus, Stephen P. Jackson, Mariella G. Filbin, Anthony Hill, Anarita Patrizi, Neil Dani, Aviv Regev, Maria K. Lehtinen, Richard J. Gilbertson
          </td>
          <td>2024-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Complex survey designs are commonly employed in many medical cohorts. In such scenarios, developing case-specific predictive risk score models that reflect the unique characteristics of the study design is essential. This approach is key to minimizing potential selective biases in results. The objectives of this paper are: (i) To propose a general predictive framework for regression and classification using neural network (NN) modeling, which incorporates survey weights into the estimation process; (ii) To introduce an uncertainty quantification algorithm for model prediction, tailored for data from complex survey designs; (iii) To apply this method in developing robust risk score models to assess the risk of Diabetes Mellitus in the US population, utilizing data from the NHANES 2011-2014 cohort. The theoretical properties of our estimators are designed to ensure minimal bias and the statistical consistency, thereby ensuring that our models yield reliable predictions and contribute novel scientific insights in diabetes research. While focused on diabetes, this NN predictive framework is adaptable to create clinical models for a diverse range of diseases and medical cohorts. The software and the data used in this paper is publicly available on GitHub.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f46d80473f8c0ce0f22ced90ed65d3132d374089" target='_blank'>
              Deep Learning Framework with Uncertainty Quantification for Survey Data: Assessing and Predicting Diabetes Mellitus Risk in the American Population
              </a>
            </td>
          <td>
            Marcos Matabuena, J. Vidal, Rahul Ghosal, J. Onnela
          </td>
          <td>2024-03-28</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09cd6579411f667e3dd47f78a07657ee3d86cb89" target='_blank'>
              Machine Learning Exploration of the Relationship Between Drugs and the Blood-Brain Barrier: Guiding Molecular Modification.
              </a>
            </td>
          <td>
            Qi Yang, Lili Fan, Erwei Hao, Xiaotao Hou, Jiagang Deng, Zhongshang Xia, Zhengcai Du
          </td>
          <td>2024-04-11</td>
          <td>Pharmaceutical research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mean-field games (MFGs) are developed to model the decision-making processes of a large number of interacting agents in multi-agent systems. This paper studies mean-field games on graphs (G-MFGs). The equilibria of G-MFGs, namely, mean-field equilibria (MFE), are challenging to solve for their high-dimensional action space because each agent has to make decisions when they are at junction nodes or on edges. Furthermore, when the initial population state varies on graphs, we have to recompute MFE, which could be computationally challenging and memory-demanding. To improve the scalability and avoid repeatedly solving G-MFGs every time their initial state changes, this paper proposes physics-informed graph neural operators (PIGNO). The PIGNO utilizes a graph neural operator to generate population dynamics, given initial population distributions. To better train the neural operator, it leverages physics knowledge to propagate population state transitions on graphs. A learning algorithm is developed, and its performance is evaluated on autonomous driving games on road networks. Our results demonstrate that the PIGNO is scalable and generalizable when tested under unseen initial conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2666d3e3bdd62dca760db009b5aa32ca5ff16ae" target='_blank'>
              Physics-Informed Graph Neural Operator for Mean Field Games on Graph: A Scalable Learning Approach
              </a>
            </td>
          <td>
            Xu Chen, Shuo Liu, Xuan Di
          </td>
          <td>2024-03-30</td>
          <td>Games</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Causal effect estimation of individual heterogeneity is a core issue in the field of causal inference, and its application in medicine poses an active and challenging problem. In high-risk decision-making domain such as healthcare, inappropriate treatments can have serious negative impacts on patients. Recently, machine learning-based methods have been proposed to improve the accuracy of causal effect estimation results. However, many of these methods concentrate on estimating causal effects of continuous outcome variables under binary intervention conditions, and give less consideration to multivariate intervention conditions or discrete outcome variables, thus limiting their scope of application. To tackle this issue, we combine the double machine learning framework with Light Gradient Boosting Machine (LightGBM) and propose a double LightGBM model. This model can estimate binary causal effects more accurately and in less time. Two cyclic structures were added to the model. Data correction method was introduced and improved to transform discrete outcome variables into continuous outcome variables. Multivariate Cyclic Double LightGBM model (MCD-LightGBM) was proposed to intelligently estimate multivariate treatment effects. A visual human-computer interaction system for heterogeneous causal effect estimation was designed, which can be applied to different types of data. This paper reports that the system improved the Logarithm of the Minimum Angle of Resolution (LogMAR) of visual acuity change after Vascular Endothelial Growth Factor (anti-VEGF) treatment in patients with diabetic macular degeneration. The improvement was observed in two clinical problems, from 0.05 to 0.33, and the readmission rate of diabetic patients after cure was reduced from 48.4% to 10.5%. The results above demonstrate the potential of the proposed system in predicting heterogeneous clinical drug treatment effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ccbe7cc618aab059e138694e686970d18635593" target='_blank'>
              MCD-LightGBM System for Intelligent Analyzing Heterogeneous Clinical Drug Therapeutic Effects.
              </a>
            </td>
          <td>
            Xiao-Hui Yang, Hao-Jie Liao, Pei- Yu Sun, Jing Ma, Bing Wang, Yan He, Liu-Gen Xue, Li-Min Su, Bin-Jie Wang
          </td>
          <td>2024-03-27</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Despite rapid advancements in treatments and technology, breast cancer remains the second leading cause of cancer related morbidity in women in the US. As a heterogeneous disease, the unique genetic and phenotypic profile of each tumor complicates the prediction of treatment responses and optimized treatment strategies. One primary manifestation of phenotypic plasticity and concomitant intra-tumoral heterogeneity is the Epithelial-Mesenchymal Transition (EMT). The acquisition of migratory and invasive properties through EMT facilitates tumor metastasis and results in worse patient outcomes. As a result, the EMT and its reverse process (MET) often influence treatment response. However, the different mechanisms through which these therapeutic drugs affect the EMT-related phenotypic switching rates as well as each phenotype’s replication rates are unclear, and further investigation is required for better understanding their precise effect for directing optimal treatment strategies and improved patient outcomes. Here, we compile an atlas of publicly available metastatic and non-metastatic RNA sequencing data of breast cancer cell lines treated with therapeutic and chemotherapeutic drugs and propose a dual data-driven theoretical framework for understanding the effects of each drug on breast cancer treatment response. We infer the cell fractions, EMT phenotypic transition rates and phenotypic replication rates of each case through an extension of our previously developed hybrid data-driven and theoretical model, COMET. We infer single-cell RNA sequencing-derived EMT-related trajectories and apply a continuous time Markov chain model to infer inter-state transition rates. By analyzing the atlas of RNA sequencing data and integration with our framework, we gain novel insights into the context-specific mechanisms by which therapeutic drugs affect the phenotypic transition rates. This comprehensive understanding will contribute to optimizing breast cancer therapy and ultimately improving patient outcomes.
 Citation Format: A. Najafi. Optimizing the Therapeutic Treatment of Breast Cancer Through a Dual Data-driven Theoretical Model [abstract]. In: Proceedings of the 2023 San Antonio Breast Cancer Symposium; 2023 Dec 5-9; San Antonio, TX. Philadelphia (PA): AACR; Cancer Res 2024;84(9 Suppl):Abstract nr PO5-24-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b06a9b9f939f3dbefa8f3e022b1bd86985d0877" target='_blank'>
              Abstract PO5-24-09: Optimizing the Therapeutic Treatment of Breast Cancer Through a Dual Data-driven Theoretical Model
              </a>
            </td>
          <td>
            A. Najafi
          </td>
          <td>2024-05-02</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Networked dynamical systems are widely used as formal models of real-world cascading phenomena, such as the spread of diseases and information. Prior research has addressed the problem of learning the behavior of an unknown dynamical system when the underlying network has a single layer. In this work, we study the learnability of dynamical systems over multilayer networks, which are more realistic and challenging. First, we present an efficient PAC learning algorithm with provable guarantees to show that the learner only requires a small number of training examples to infer an unknown system. We further provide a tight analysis of the Natarajan dimension which measures the model complexity. Asymptotically, our bound on the Nararajan dimension is tight for almost all multilayer graphs. The techniques and insights from our work provide the theoretical foundations for future investigations of learning problems for multilayer dynamical systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4c0063e47ca85cf9c10b3ade6904eaa76704c7" target='_blank'>
              Efficient PAC Learnability of Dynamical Systems Over Multilayer Networks
              </a>
            </td>
          <td>
            Zirou Qiu, Abhijin Adiga, M. Marathe, S. S. Ravi, D. Rosenkrantz, R. Stearns, Anil Vullikanti
          </td>
          <td>2024-05-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/477bcc7b10d17b6d768d343469473e99b1a78282" target='_blank'>
              Integrated clinical and genomic models using machine-learning methods to predict the efficacy of paclitaxel-based chemotherapy in patients with advanced gastric cancer
              </a>
            </td>
          <td>
            Yonghwa Choi, Jangwoo Lee, Keewon Shin, Ji Won Lee, Ju Won Kim, Soohyeon Lee, Yoon Ji Choi, Kyong Hwa Park, Jwa Hoon Kim
          </td>
          <td>2024-04-20</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A novel approach to the utilization of nuclear magnetic resonance (NMR) spectroscopy data in the prediction of logD through machine learning algorithms is shown. In the analysis, a data set of 754 chemical compounds, organized into 30 clusters, was evaluated using advanced machine learning models, such as Support Vector Regression (SVR), Gradient Boosting, and AdaBoost, and comprehensive validation and testing methods were employed, including 10-fold cross-validation, bootstrapping, and leave-one-out. The study revealed the superior performance of the Bucket Integration method for dimensionality reduction, consistently yielding the lowest root mean square error (RMSE) across all data sets and normalization schemes. The SVR prediction models demonstrated remarkable computational efficiency and low cost, with the best RMSE value reaching 0.66. Our best model outperformed existing tools like JChem Suite's logD Predictor (0.91) and CplogD (1.27), and a comparison with traditional molecular representations yielded a comparable RMSE (0.50), emphasizing the robustness of our NMR data integration. The widespread availability of NMR data in pharmaceutical and industrial research presents an untapped resource for predictive modeling, highlighting the need for accessible methodologies like ours that complement the analytical toolbox beyond conventional 2D approaches. Our approach, designed to leverage the rich spatial data from NMR spectroscopy, provides additional insights and enriches drug discovery and computational chemistry with a freely accessible tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a67eb4df62428dc27efedda2a8bebee0d3527d1" target='_blank'>
              From NMR to AI: Designing a Novel Chemical Representation to Enhance Machine Learning Predictions of Physicochemical Properties
              </a>
            </td>
          <td>
            Arkadiusz Leniak, Wojciech Pietruś, Rafał Kurczab
          </td>
          <td>2024-03-26</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="A new knowledge-based and machine learning hybrid modeling approach, called conditional Gaussian neural stochastic differential equation (CGNSDE), is developed to facilitate modeling complex dynamical systems and implementing analytic formulae of the associated data assimilation (DA). In contrast to the standard neural network predictive models, the CGNSDE is designed to effectively tackle both forward prediction tasks and inverse state estimation problems. The CGNSDE starts by exploiting a systematic causal inference via information theory to build a simple knowledge-based nonlinear model that nevertheless captures as much explainable physics as possible. Then, neural networks are supplemented to the knowledge-based model in a specific way, which not only characterizes the remaining features that are challenging to model with simple forms but also advances the use of analytic formulae to efficiently compute the nonlinear DA solution. These analytic formulae are used as an additional computationally affordable loss to train the neural networks that directly improve the DA accuracy. This DA loss function promotes the CGNSDE to capture the interactions between state variables and thus advances its modeling skills. With the DA loss, the CGNSDE is more capable of estimating extreme events and quantifying the associated uncertainty. Furthermore, crucial physical properties in many complex systems, such as the translate-invariant local dependence of state variables, can significantly simplify the neural network structures and facilitate the CGNSDE to be applied to high-dimensional systems. Numerical experiments based on chaotic systems with intermittency and strong non-Gaussian features indicate that the CGNSDE outperforms knowledge-based regression models, and the DA loss further enhances the modeling skills of the CGNSDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a639323a3ab8c39800f9e9f42ae3d95438cb1ec6" target='_blank'>
              CGNSDE: Conditional Gaussian Neural Stochastic Differential Equation for Modeling Complex Systems and Data Assimilation
              </a>
            </td>
          <td>
            Chuanqi Chen, Nan Chen, Jingbo Wu
          </td>
          <td>2024-04-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Multi-modal learning that combines pathological images with genomic data has significantly enhanced the accuracy of survival prediction. Nevertheless, existing methods have not fully utilized the inherent hierarchical structure within both whole slide images (WSIs) and transcriptomic data, from which better intra-modal representations and inter-modal integration could be derived. Moreover, many existing studies attempt to improve multi-modal representations through attention mechanisms, which inevitably lead to high complexity when processing high-dimensional WSIs and transcriptomic data. Recently, a structured state space model named Mamba emerged as a promising approach for its superior performance in modeling long sequences with low complexity. In this study, we propose Mamba with multi-grained multi-modal interaction (SurvMamba) for survival prediction. SurvMamba is implemented with a Hierarchical Interaction Mamba (HIM) module that facilitates efficient intra-modal interactions at different granularities, thereby capturing more detailed local features as well as rich global representations. In addition, an Interaction Fusion Mamba (IFM) module is used for cascaded inter-modal interactive fusion, yielding more comprehensive features for survival prediction. Comprehensive evaluations on five TCGA datasets demonstrate that SurvMamba outperforms other existing methods in terms of performance and computational cost.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b11b6a0edf07bb1d14ee8ed721d7fbce29b62c08" target='_blank'>
              SurvMamba: State Space Model with Multi-grained Multi-modal Interaction for Survival Prediction
              </a>
            </td>
          <td>
            Ying Chen, Jiajing Xie, Yuxiang Lin, Yuhang Song, Wenxian Yang, Rongshan Yu
          </td>
          <td>2024-04-11</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>11</td>
        </tr>

        <tr id="This study investigates how missing data samples in continuous blood glucose data affect the prediction of postprandial hypoglycemia, which is crucial for diabetes management. We analyzed the impact of missing samples at different times before meals using two datasets: virtual patient data and real patient data. The study uses six commonly used machine learning models under varying conditions of missing samples, including custom and random patterns reflective of device failures and arbitrary data loss, with different levels of data removal before mealtimes. Additionally, the study explored different interpolation techniques to counter the effects of missing data samples. The research shows that missing samples generally reduce the model performance, but random forest is more robust to missing samples. The study concludes that the adverse effects of missing samples can be mitigated by leveraging complementary and informative non-point features. Consequently, our research highlights the importance of strategically handling missing data, selecting appropriate machine learning models, and considering feature types to enhance the performance of postprandial hypoglycemia predictions, thereby improving diabetes management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef13bbaba355c0d3942d70075410d6ec5760a4a6" target='_blank'>
              The Impact of Missing Continuous Blood Glucose Samples on Machine Learning Models for Predicting Postprandial Hypoglycemia: An Experimental Analysis
              </a>
            </td>
          <td>
            Najib Ur Rehman, Iván Contreras, Aleix Beneyto, J. Vehi
          </td>
          <td>2024-05-17</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ffa9bfa3c3984b355ec71361380dbb9ad8aca54" target='_blank'>
              An integrated framework for prognosis prediction and drug response modeling in colorectal liver metastasis drug discovery
              </a>
            </td>
          <td>
            Xiuman Zhou, Yuzhen Qian, Chen Ling, Zhuo-bin He, Peishang Shi, Yanfeng Gao, Xinghua Sui
          </td>
          <td>2024-03-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="To accelerate the process of materials design, materials science has increasingly used data driven techniques to extract information from collected data. Specially, machine learning (ML) algorithms, which span the ML discipline, have demonstrated ability to predict various properties of materials with the level of accuracy similar to explicit calculation of quantum mechanical theories, but with significantly reduced run time and computational resources. Within ML, graph neural networks have emerged as an important algorithm within the field of machine learning, since they are capable of predicting accurately a wide range of important physical, chemical and electronic properties due to their higher learning ability based on the graph representation of material and molecular descriptors through the aggregation of information embedded within the graph. In parallel with the development of state of the art classical machine learning applications, the fusion of quantum computing and machine learning have created a new paradigm where classical machine learning model can be augmented with quantum layers which are able to encode high dimensional data more efficiently. Leveraging the structure of existing algorithms, we developed a unique and novel gradient free hybrid quantum classical convoluted graph neural network (HyQCGNN) to predict formation energies of perovskite materials. The performance of our hybrid statistical model is competitive with the results obtained purely from a classical convoluted graph neural network, and other classical machine learning algorithms, such as XGBoost. Consequently, our study suggests a new pathway to explore how quantum feature encoding and parametric quantum circuits can yield drastic improvements of complex ML algorithm like graph neural network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b365db803a1590eed133a254bcdb9c88e01783cc" target='_blank'>
              Hybrid Quantum Graph Neural Network for Molecular Property Prediction
              </a>
            </td>
          <td>
            Michael Vitz, Hamed Mohammadbagherpoor, Samarth Sandeep, Andrew Vlasic, Richard Padbury, Anh Pham
          </td>
          <td>2024-05-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This article presents an in-depth analysis and evaluation of artificial neural networks (ANNs) when applied to replicate trajectories in molecular dynamics (MD) simulations or other particle methods. This study focuses on several architectures—feedforward neural networks (FNNs), convolutional neural networks (CNNs), recurrent neural networks (RNNs), time convolutions (TCs), self-attention (SA), graph neural networks (GNNs), neural ordinary differential equation (ODENets), and an example of physics-informed machine learning (PIML) model—assessing their effectiveness and limitations in understanding and replicating the underlying physics of particle systems. Through this analysis, this paper introduces a comprehensive set of criteria designed to evaluate the capability of ANNs in this context. These criteria include the minimization of losses, the permutability of particle indices, the ability to predict trajectories recursively, the conservation of particles, the model’s handling of boundary conditions, and its scalability. Each network type is systematically examined to determine its strengths and weaknesses in adhering to these criteria. While, predictably, none of the networks fully meets all criteria, this study extends beyond the simple conclusion that only by integrating physics-based models into ANNs is it possible to fully replicate complex particle trajectories. Instead, it probes and delineates the extent to which various neural networks can “understand” and interpret aspects of the underlying physics, with each criterion targeting a distinct aspect of this understanding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d2cd2e98b8c0ad116f74af97b8583a17b803b5f" target='_blank'>
              Designing a set of criteria for evaluating artificial neural networks trained with physics-based data to replicate molecular dynamics and other particle method trajectories
              </a>
            </td>
          <td>
            Alessio Alexiadis
          </td>
          <td>2024-04-10</td>
          <td>Frontiers in Nanotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Machine learning models support computer-aided molecular design and compound optimization. However, the initial phases of drug discovery often face a scarcity of training data for these models. Meta-learning has emerged as a potentially promising strategy, harnessing the wealth of structure-activity data available for known targets to facilitate efficient few-shot model training for the specific target of interest. In this study, we assessed the effectiveness of two different meta-learning methods, namely model-agnostic meta-learning (MAML) and adaptive deep kernel fitting (ADKF), specifically in the regression setting. We investigated how factors such as dataset size and the similarity of training tasks impact predictability. The results indicate that ADKF significantly outperformed both MAML and a single-task baseline model on the inhibition data. However, the performance of ADKF varied across different test tasks. Our findings suggest that considerable enhancements in performance can be anticipated primarily when the task of interest is similar to the tasks incorporated in the meta-learning process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32a6e659b8193146d505ac34674aa54852ff102" target='_blank'>
              Task-similarity is a crucial factor for few-shot meta-learning of structure-activity relationships.
              </a>
            </td>
          <td>
            Alexander Kötter, Stephan Allenspach, Christoph Grebner, Hans Matter, J. Hiss, Gisbert Schneider, Gerhard Hessler
          </td>
          <td>2024-04-29</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Computational models are not just appealing because they can simulate and predict the development of biological phenomena across multiple spatial and temporal scales, but also because they can integrate information from well-established in vitro and in vivo models and test new hypotheses in cancer biomedicine. Agent-based models and simulations are especially interesting candidates among computational modeling procedures in cancer research due to the capability to, for instance, recapitulate the dynamics of neoplasia and tumor - host interactions. Yet, the absence of methods to validate the consistency of the results across scales can hinder adoption by turning fine-tuned models into black boxes. This review compiles relevant literature that explores strategies to leverage high-fidelity simulations of multi-scale, or multi-level, cancer models with a focus on verification approached as simulation calibration. We consolidate our review with an outline of modern approaches for agent-based models' validation and provide an ambitious outlook toward rigorous and reliable calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e0c3944c13aa863a5f0e7e95045776f19c4393" target='_blank'>
              Agent-based modeling in cancer biomedicine: applications and tools for calibration and validation.
              </a>
            </td>
          <td>
            Nicolò Cogno, Cristian Axenie, Roman Bauer, V. Vavourakis
          </td>
          <td>2024-04-28</td>
          <td>Cancer biology & therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="In the realm of predictive toxicology for small molecules, the applicability domain of QSAR models is often limited by the coverage of the chemical space in the training set. Consequently, classical models fail to provide reliable predictions for wide classes of molecules. However, the emergence of innovative data collection methods such as intensive hackathons have promise to quickly expand the available chemical space for model construction. Combined with algorithmic refinement methods, these tools can address the challenges of toxicity prediction, enhancing both the robustness and applicability of the corresponding models. This study aimed to investigate the roles of gradient boosting and strategic data aggregation in enhancing the predictivity ability of models for the toxicity of small organic molecules. We focused on evaluating the impact of incorporating fragment features and expanding the chemical space, facilitated by a comprehensive dataset procured in an open hackathon. We used gradient boosting techniques, accounting for critical features such as the structural fragments or functional groups often associated with manifestations of toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd79272eeef9dba73fe4181d2dcd91d255dffcc3" target='_blank'>
              Expanding Predictive Capacities in Toxicology: Insights from Hackathon-Enhanced Data and Model Aggregation
              </a>
            </td>
          <td>
            Dmitrii O. Shkil, Alina A. Muhamedzhanova, Philipp I. Petrov, Ekaterina V. Skorb, Timur A. Aliev, Ilya S Steshin, Alexander V. Tumanov, Alexander S. Kislinskiy, Maxim V. Fedorov
          </td>
          <td>2024-04-01</td>
          <td>Molecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Causal effect estimation aims at estimating the Average Treatment Effect as well as the Conditional Average Treatment Effect of a treatment to an outcome from the available data. This knowledge is important in many safety-critical domains, where it often needs to be extracted from observational data. In this work, we propose a new causal inference model, named C-XGBoost, for the prediction of potential outcomes. The motivation of our approach is to exploit the superiority of tree-based models for handling tabular data together with the notable property of causal inference neural network-based models to learn representations that are useful for estimating the outcome for both the treatment and non-treatment cases. The proposed model also inherits the considerable advantages of XGBoost model such as efficiently handling features with missing values requiring minimum preprocessing effort, as well as it is equipped with regularization techniques to avoid overfitting/bias. Furthermore, we propose a new loss function for efficiently training the proposed causal inference model. The experimental analysis, which is based on the performance profiles of Dolan and Mor{\'e} as well as on post-hoc and non-parametric statistical tests, provide strong evidence about the effectiveness of the proposed approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e484a192be7ca41717fc03c01d2a6f284d79b407" target='_blank'>
              C-XGBoost: A tree boosting model for causal effect estimation
              </a>
            </td>
          <td>
            Niki Kiriakidou, I. Livieris, Christos Diou
          </td>
          <td>2024-03-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Recently, we have witnessed impressive achievements in cancer survival analysis by integrating multimodal data, e.g., pathology images and genomic profiles. However, the heterogeneity and high dimensionality of these modalities pose significant challenges for extracting discriminative representations while maintaining good generalization. In this paper, we propose a Cohort-individual Cooperative Learning (CCL) framework to advance cancer survival analysis by collaborating knowledge decomposition and cohort guidance. Specifically, first, we propose a Multimodal Knowledge Decomposition (MKD) module to explicitly decompose multimodal knowledge into four distinct components: redundancy, synergy and uniqueness of the two modalities. Such a comprehensive decomposition can enlighten the models to perceive easily overlooked yet important information, facilitating an effective multimodal fusion. Second, we propose a Cohort Guidance Modeling (CGM) to mitigate the risk of overfitting task-irrelevant information. It can promote a more comprehensive and robust understanding of the underlying multimodal data, while avoiding the pitfalls of overfitting and enhancing the generalization ability of the model. By cooperating the knowledge decomposition and cohort guidance methods, we develop a robust multimodal survival analysis model with enhanced discrimination and generalization abilities. Extensive experimental results on five cancer datasets demonstrate the effectiveness of our model in integrating multimodal data for survival analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a04715aa7a48ad08f6906507f9c578bbde0bcb" target='_blank'>
              Cohort-Individual Cooperative Learning for Multimodal Cancer Survival Analysis
              </a>
            </td>
          <td>
            Huajun Zhou, Fengtao Zhou, Hao Chen
          </td>
          <td>2024-04-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer remains a leading cause of death worldwide, with its heterogeneous and patient-specific nature complicating treatment. Immunotherapy, especially immune checkpoint blockades (ICB), shows promising potential by reactivating the immune system to detect and destroy cancer cells. However, their effectiveness largely depends on the patient’s clinical and molecular characteristics, making them unsuitable for everyone. This highlights the importance of developing predictive and interpretable models that could distinguish potential responders from non-responders. This study aims to utilize gene expression data to predict patients’ responses to ICB and identify key genes whose expression is associated with treatment outcome using probabilistic Bayesian networks (BNs). To address the high-dimensional nature of gene expression data, we propose and compare the use of machine learning-based feature selection (FS) and clustering by weighted gene co-expression network analysis (WGCNA). To our knowledge, this is the first study to apply BNs along with FS and WGCNA dimensionality reduction techniques to tackle ICB response using transcriptomic data. We achieved a maximal classification accuracy of 75% and identified eight genes and two gene clusters directly influencing ICB response. These results may offer more biological insights and mechanisms underlying ICB response. The proposed pipeline will contribute to guiding treatment decisions, minimizing harmful side effects, and reducing the financial costs associated with ineffective treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbbc861e13246a2c515a393ebb44eb2b8dd4dc66" target='_blank'>
              AI In Healthcare: Using Bayesian Networks To Predict Response To Immune Checkpoint Blockades
              </a>
            </td>
          <td>
            Ahmed H. Saadawy, Aly F. Hissam, Eman Badr, Omar M. Fahmy
          </td>
          <td>2024-04-16</td>
          <td>2024 41st National Radio Science Conference (NRSC)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="While radiotherapy has long been recognized for its ability to directly ablate cancer cells through necrosis or apoptosis, radiotherapy-induced abscopal effect suggests that its impact extends beyond local tumor destruction thanks to immune response. Cellular proliferation and necrosis have been extensively studied using mathematical models that simulate tumor growth, such as Gompertz law, and the radiation effects, such as the linear-quadratic model. However, the effectiveness of radiotherapy-induced immune responses may vary among patients due to individual differences in radiation sensitivity and other factors.We present a novel macroscopic approach designed to quantitatively analyze the intricate dynamics governing the interactions among the immune system, radiotherapy, and tumor progression. Building upon previous research demonstrating the synergistic effects of radiotherapy and immunotherapy in cancer treatment, we provide a comprehensive mathematical framework for understanding the underlying mechanisms driving these interactions.Our method leverages macroscopic observations and mathematical modeling to capture the overarching dynamics of this interplay, offering valuable insights for optimizing cancer treatment strategies. One shows that Gompertz law can describe therapy effects with two effective parameters. This result permits quantitative data analyses, which give useful indications for the disease progression and clinical decisions.Through validation against diverse data sets from the literature, we demonstrate the reliability and versatility of our approach in predicting the time evolution of the disease and assessing the potential efficacy of radiotherapy-immunotherapy combinations. This further supports the promising potential of the abscopal effect, suggesting that in select cases, depending on tumor size, it may confer full efficacy to radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f15cf29cf113e672b18b66c8e810f76f39836894" target='_blank'>
              Mathematical modeling of the synergistic interplay of radiotherapy and immunotherapy in anti-cancer treatments
              </a>
            </td>
          <td>
            Paolo Castorina, Filippo Castiglione, G. Ferini, Stefano Forte, E. Martorana, Dario Giuffrida
          </td>
          <td>2024-05-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Identifying functionally important cell states and structure within a heterogeneous tumor remains a significant biological and computational challenge. Moreover, current clustering or trajectory-based computational models are ill-equipped to address the notion that cancer cells reside along a phenotypic continuum. To address this, we present Archetypal Analysis network (AAnet), a neural network that learns key archetypal cell states within a phenotypic continuum of cell states in single-cell data. Applied to single-cell RNA sequencing data from pre-clinical models and a cohort of 34 clinical breast cancers, AAnet identifies archetypes that resolve distinct biological cell states and processes, including cell proliferation, hypoxia, metabolism and immune interactions. Notably, archetypes identified in primary tumors are recapitulated in matched liver, lung and lymph node metastases, demonstrating that a significant component of intratumoral heterogeneity is driven by cell intrinsic properties. Using spatial transcriptomics as orthogonal validation, AAnet-derived archetypes show discrete spatial organization within tumors, supporting their distinct archetypal biology. We further reveal that ligand:receptor cross-talk between cancer and adjacent stromal cells contributes to intra-archetypal biological mimicry. Finally, we use AAnet archetype identifiers to validate GLUT3 as a critical mediator of a hypoxic cell archetype harboring a cancer stem cell population, which we validate in human triple-negative breast cancer specimens. AAnet is a powerful tool to reveal functional cell states within complex samples from multimodal single-cell data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32b3a7328824f9e25b9a694c5c5c104d8ffd3103" target='_blank'>
              AAnet resolves a continuum of spatially-localized cell states to unveil tumor complexity
              </a>
            </td>
          <td>
            Aarthi Venkat, Scott E. Youlten, B. P. S. Juan, Carley Purcell, Matthew Amodio, Daniel B. Burkhardt, Andrew Benz, Jeff Holst, Cerys McCool, A. Mollbrink, J. Lundeberg, David van Dijk, Leonard D Goldstein, Sarah Kummerfeld, Smita Krishnaswamy, Christine L. Chaffer
          </td>
          <td>2024-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="The task of chemical reaction predictions (CRPs) plays a pivotal role in advancing drug discovery and material science. However, its effectiveness is constrained by the vast and uncertain chemical reaction space and challenges in capturing reaction selectivity, particularly due to existing methods' limitations in exploiting the data's inherent knowledge. To address these challenges, we introduce a data-curated self-feedback knowledge elicitation approach. This method starts from iterative optimization of molecular representations and facilitates the extraction of knowledge on chemical reaction types (RTs). Then, we employ adaptive prompt learning to infuse the prior knowledge into the large language model (LLM). As a result, we achieve significant enhancements: a 14.2% increase in retrosynthesis prediction accuracy, a 74.2% rise in reagent prediction accuracy, and an expansion in the model's capability for handling multi-task chemical reactions. This research offers a novel paradigm for knowledge elicitation in scientific research and showcases the untapped potential of LLMs in CRPs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2040402938c8542f9c5ef89f2b2c8774332de8a1" target='_blank'>
              A Self-feedback Knowledge Elicitation Approach for Chemical Reaction Predictions
              </a>
            </td>
          <td>
            Pengfei Liu, Jun Tao, Zhixiang Ren
          </td>
          <td>2024-04-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Half-life is a significant pharmacokinetic parameter included in the excretion phase of absorption, distribution, metabolism, and excretion. It is one of the key factors for the successful marketing of drug candidates. Therefore, predicting half-life is of great significance in drug design. In this study, we employed eXtreme Gradient Boosting (XGboost), randomForest (RF), gradient boosting machine (GBM), and supporting vector machine (SVM) to build quantitative structure-activity relationship (QSAR) models on 3512 compounds and evaluated model performance by using root-mean-square error (RMSE), R2, and mean absolute error (MAE) metrics and interpreted features by SHapley Additive exPlanation (SHAP). Furthermore, we developed consensus models through integrating four individual models and validated their performance using a Y-randomization test and applicability domain analysis. Finally, matched molecular pair analysis was used to extract the transformation rules. Our results revealed that XGboost outperformed other individual models (RMSE = 0.176, R2 = 0.845, MAE = 0.141). The consensus model integrating all four models continued to enhance prediction performance (RMSE = 0.172, R2 = 0.856, MAE = 0.138). We evaluated the reliability, robustness, and generalization ability via Y-randomization test and applicability domain analysis. Meanwhile, we utilized SHAP to interpret features and employed matched molecular pair analysis to extract chemical transformation rules that provide suggestions for optimizing drug structure. In conclusion, we believe that the consensus model developed in this study serve as a reliable tool to evaluate half-life in drug discovery, and the chemical transformation rules concluded in this study could provide valuable suggestions in drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31c93018584a4e6ad56bdbe5afca31bdfc9b13a" target='_blank'>
              Predicting Elimination of Small-Molecule Drug Half-Life in Pharmacokinetics Using Ensemble and Consensus Machine Learning Methods
              </a>
            </td>
          <td>
            Jianing Fan, Shaohua Shi, Hong Xiang, Li Fu, Yanjing Duan, Dongsheng Cao, Hongwei Lu
          </td>
          <td>2024-04-02</td>
          <td>Journal of chemical information and modeling</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="The chemistry of an astrophysical environment is closely coupled to its dynamics, the latter often found to be complex. Hence, to properly model these environments a 3D context is necessary. However, solving chemical kinetics within a 3D hydro simulation is computationally infeasible for a even a modest parameter study. In order to develop a feasible 3D hydro-chemical simulation, the classical chemical approach needs to be replaced by a faster alternative. We present mace, a Machine learning Approach to Chemistry Emulation, as a proof-of-concept work on emulating chemistry in a dynamical environment. Using the context of AGB outflows, we have developed an architecture that combines the use of an autoencoder (to reduce the dimensionality of the chemical network) and a set of latent ordinary differential equations (that are solved to perform the temporal evolution of the reduced features). Training this architecture with an integrated scheme makes it possible to successfully reproduce a full chemical pathway in a dynamical environment. mace outperforms its classical analogue on average by a factor 26. Furthermore, its efficient implementation in PyTorch results in a sub-linear scaling with respect to the number of hydrodynamical simulation particles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a68a92bb31d2d2c457cfda8c158e7d23a370619" target='_blank'>
              MACE: A Machine learning Approach to Chemistry Emulation
              </a>
            </td>
          <td>
            S. Maes, F. D. Ceuster, M. Sande, L. Decin
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multi-modal biological datasets provide rich information from diverse scales or facets of complex biological systems that can be analyzed to highlight the critical multi-scale interactions underlying specific biological phenomena. However, identifying the most vital associations among features and outputs can be beset by a high degree of spurious connections due to indirect effects of various immune features propagating through an unmapped biological network. Here, we applied a probabilistic graphical modeling approach, Markov Fields, to empirically dissect the most direct associations correlations between features from a public multi-modal dataset (antibody titers, antibody-dependent functions, cytokines, cytometry) from macaques undergoing intravenous BCG vaccination—a promising vaccine strategy against the major public health threat tuberculosis. This yielded a network influence model that interprets the assemblage of multi-scale paths by which vaccine effects propagate through the immune network to eventually protect against tuberculosis infection. Importantly, our modeling shows that the vast majority of apparent correlations between features arise from indirect effects relating distant immune features. For a test of predictive capability, we conducted experimental depletion of B cells during BCG IV vaccination in macaques--which did not reduce BCG IV-mediated protection against tuberculosis— and validated that our Markov Field model can predict systems-wide modulation of numerous features across the immune system in response to this perturbation. Finally, we applied our model to discern perturbations in the network that could have strong effects on IV-BCG efficacy. All together, we have demonstrated that probabilistic graphical modeling can increase interpretability and predictive value of multi-modal datasets for identifying new disease treatment targets. Summary Multi-modal datasets provide rich information that can help identify critical multi-scale interactions underlying biological systems. However, identifying associations between features and outputs can be beset by spurious connections due to indirect impacts propagating through an unmapped biological network. We applied a probabilistic graphical modeling approach, Markov Fields, to dissect correlations between immune features in a public multi-modal dataset (systems serology, cytokines, cytometry) of macaques undergoing intravenous BCG vaccination against tuberculosis. This yielded an interaction network that interprets network paths underlying vaccine efficacy, and shows how correlations between features often arise indirectly. We next conducted experimental depletion of B cells during vaccination in macaques—which did not reduce protection against tuberculosis—to validate our Markov Field model’s predictions of network-wide shifts post-depletion. Finally, we highlight immune changes predicted to strongly affect intravenous BCG vaccine efficacy, showing that probabilistic graphical models increase the interpretability of multi-modal datasets for identifying new disease targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad2f1e0ba10cbde79ba37f6d358ceb194ee91f48" target='_blank'>
              Markov Field network integration of multi-modal data predicts effects of immune system perturbations on intravenous BCG vaccination in macaques
              </a>
            </td>
          <td>
            Shu Wang, Amy J Myers, Edward B. Irvine, Chuangqi Wang, Pauline Maiello, Mark A. Rodgers, Jaime Tomko, Kara Kracinovsky, H. Borish, Michael C Chao, Douaa Mugahid, P. Darrah, Robert A. Seder, Mario Roederer, C. Scanga, P. Lin, Galit Alter, Sarah M Fortune, JoAnne L. Flynn, D. Lauffenburger
          </td>
          <td>2024-04-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7a0a8265ea7e318e65ccdf6ef53d19670ab7b3" target='_blank'>
              A novel non-negative Bayesian stacking modeling method for Cancer survival prediction using high-dimensional omics data
              </a>
            </td>
          <td>
            Junjie Shen, Shuo Wang, Hao Sun, Jie Huang, Lu Bai, Xichao Wang, Yongfei Dong, Zaixiang Tang
          </td>
          <td>2024-05-03</td>
          <td>BMC Medical Research Methodology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Optimizing techniques for discovering molecular structures with desired properties is crucial in artificial intelligence(AI)-based drug discovery. Combining deep generative models with reinforcement learning has emerged as an effective strategy for generating molecules with specific properties. Despite its potential, this approach is ineffective in exploring the vast chemical space and optimizing particular chemical properties. To overcome these limitations, we present Mol-AIR, a reinforcement learning-based framework using adaptive intrinsic rewards for effective goal-directed molecular generation. Mol-AIR leverages the strengths of both history-based and learning-based intrinsic rewards by exploiting random distillation network and counting-based strategies. In benchmark tests, Mol-AIR demonstrates superior performance over existing approaches in generating molecules with desired properties without any prior knowledge, including penalized LogP, QED, and celecoxib similarity. We believe that Mol-AIR represents a significant advancement in drug discovery, offering a more efficient path to discovering novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a1b39ded76621764814fe339cfbc69d9d7158b" target='_blank'>
              Mol-AIR: Molecular Reinforcement Learning with Adaptive Intrinsic Rewards for Goal-directed Molecular Generation
              </a>
            </td>
          <td>
            Jinyeong Park, Jaegyoon Ahn, Jonghwan Choi, Jibum Kim
          </td>
          <td>2024-03-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract In this review article, we explore the transformative impact of deep learning (DL) on structural bioinformatics, emphasizing its pivotal role in a scientific revolution driven by extensive data, accessible toolkits and robust computing resources. As big data continue to advance, DL is poised to become an integral component in healthcare and biology, revolutionizing analytical processes. Our comprehensive review provides detailed insights into DL, featuring specific demonstrations of its notable applications in bioinformatics. We address challenges tailored for DL, spotlight recent successes in structural bioinformatics and present a clear exposition of DL—from basic shallow neural networks to advanced models such as convolution, recurrent, artificial and transformer neural networks. This paper discusses the emerging use of DL for understanding biomolecular structures, anticipating ongoing developments and applications in the realm of structural bioinformatics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83f5f9d82fd33de9988d4a1a8b2c0a9b27b82bc3" target='_blank'>
              Deep learning in structural bioinformatics: current applications and future perspectives
              </a>
            </td>
          <td>
            Niranjan Kumar, Rakesh Srivastava
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The acquisition of different data modalities can enhance our knowledge and understanding of various diseases, paving the way for a more personalized healthcare. Thus, medicine is progressively moving towards the generation of massive amounts of multi-modal data (\emph{e.g,} molecular, radiology, and histopathology). While this may seem like an ideal environment to capitalize data-centric machine learning approaches, most methods still focus on exploring a single or a pair of modalities due to a variety of reasons: i) lack of ready to use curated datasets; ii) difficulty in identifying the best multi-modal fusion strategy; and iii) missing modalities across patients. In this paper we introduce a real world multi-modal dataset called MMIST-CCRCC that comprises 2 radiology modalities (CT and MRI), histopathology, genomics, and clinical data from 618 patients with clear cell renal cell carcinoma (ccRCC). We provide single and multi-modal (early and late fusion) benchmarks in the task of 12-month survival prediction in the challenging scenario of one or more missing modalities for each patient, with missing rates that range from 26$\%$ for genomics data to more than 90$\%$ for MRI. We show that even with such severe missing rates the fusion of modalities leads to improvements in the survival forecasting. Additionally, incorporating a strategy to generate the latent representations of the missing modalities given the available ones further improves the performance, highlighting a potential complementarity across modalities. Our dataset and code are available here: https://multi-modal-ist.github.io/datasets/ccRCC">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2863bd595c7a0052e2f0b6b6b0f794b1dadd4695" target='_blank'>
              MMIST-ccRCC: A Real World Medical Dataset for the Development of Multi-Modal Systems
              </a>
            </td>
          <td>
            Tiago Mota, M. R. Verdelho, Diogo J. Ara´ujo, Alceu Bissoto, Carlos Santiago, Catarina Barata
          </td>
          <td>2024-05-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is a group of diseases which share one common feature: the growth of abnormal cells, thus ranking as the second leading cause of death globally, after cardiovascular diseases in WHO's report. Evaluating gene expression is based on the fact that it is the genesis of early cancer detection, the concurrence of the molecular and genetic processes. By using DNA microarray techniques and RNA-sequencing approaches, researchers in computational genomics can give quantitative measures of gene expression levels providing very accurate input data for computational evaluation. The current paper is about machine learning technology, which identifies cancer subtypes according to patterns of gene expression. It embraces the two distinct methodologies which are traditional plus deep learning with high proficiency focused on the cancer-related gene. The outline includes the most popular deep neural network designs such as MLPs, CNN, RNN, GNN, and the recently emerged Transformer networks. The review describes common data collection methods used in this field and some of the essential datasets for supervised machine learning. In addition, the specific techniques developed to cover the complicated horizontal spread of gene expression data are also presented. The article explores theoretical possibilities for the promotion of machine learning-based gene expression analysis in cancer classification towards the end.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9b359e6581f350c23c2c118eb02c4bdf9fce64a" target='_blank'>
              Utilizing Machine Learning Techniques for Categorizing Cancer Based on Gene Expression Data: A Review
              </a>
            </td>
          <td>
            Begum S., Sandipan Dey, Chakraborty D., Hembrom T., Hazra S., Barman D.
          </td>
          <td>2024-04-04</td>
          <td>Journal of Electrical Systems</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Deep learning (DL) has revolutionized machine learning, especially in the era of big data, by autonomously learning hierarchical representations from vast datasets. This paper offers a comprehensive overview of DL, covering foundational concepts, recent advances, practical applications, challenges, and future directions. It traces DLs historical roots from the perceptron model to contemporary deep neural networks, highlighting its resurgence in the early 21st century due to computational advancements and large-scale datasets. The paper discusses DLs objectives, including methodologies overview, challenges analysis, and future trends exploration. Foundational DL concepts are explained, including artificial neurons, activation functions, and network architectures like feedforward, convolutional, and recurrent neural networks. Recent advancements such as transfer learning, reinforcement learning, meta-learning, and self-supervised learning are explored, enhancing model performance across domains like computer vision, natural language processing, and robotics. Applications of DL in healthcare, finance, automotive, and more are detailed with real-world examples. The paper also analyzes challenges like data scarcity, computational complexity, and ethical considerations, proposing strategies for mitigation. Future directions in DL, including explainable AI and ethical considerations, are discussed, emphasizing DLs transformative impact and its role in driving innovation in big data analytics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6e39c7230e0996aa47ffeb5e4b4f07f37263a0" target='_blank'>
              DEEP LEARNING IN THE ERA OF BIG DATA: FOUNDATIONS, ADVANCES, APPLICATIONS, CHALLENGES, AND FUTURE DIRECTIONS
              </a>
            </td>
          <td>
            A. Alkhudhayr
          </td>
          <td>2024-04-30</td>
          <td>International Journal of Advanced Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Revealing interactions in complex systems from observed collective dynamics constitutes a fundamental inverse problem in science. Some methods may reveal undirected network topology, e.g., using node-node correlation. Yet, the direction of the interaction, thus a causal inference, remains to be determined - especially in steady-state observations. We introduce a method to infer the directionality within this network only from a “snapshot” of the abundances of the relevant molecules. We examine the validity of the approach for different properties of the system and the data recorded, such as the molecule’s level variability, the effect of sampling and measurement errors. Simulations suggest that the given approach successfully infer the reaction rates in various cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4835c23912310f411c24ba509da9c73cfc8936cd" target='_blank'>
              Determining Interaction Directionality in Complex Biochemical Networks from Stationary Measurements
              </a>
            </td>
          <td>
            Nava Leibovich
          </td>
          <td>2024-04-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multi-modality spatio-temporal (MoST) data extends spatio-temporal (ST) data by incorporating multiple modalities, which is prevalent in monitoring systems, encompassing diverse traffic demands and air quality assessments. Despite significant strides in ST modeling in recent years, there remains a need to emphasize harnessing the potential of information from different modalities. Robust MoST forecasting is more challenging because it possesses (i) high-dimensional and complex internal structures and (ii) dynamic heterogeneity caused by temporal, spatial, and modality variations. In this study, we propose a novel MoST learning framework via Self-Supervised Learning, namely MoSSL, which aims to uncover latent patterns from temporal, spatial, and modality perspectives while quantifying dynamic heterogeneity. Experiment results on two real-world MoST datasets verify the superiority of our approach compared with the state-of-the-art baselines. Model implementation is available at https://github.com/beginner-sketch/MoSSL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c10d27669d165b26a1256f70ca1349c6cb5f00ea" target='_blank'>
              Multi-Modality Spatio-Temporal Forecasting via Self-Supervised Learning
              </a>
            </td>
          <td>
            Jiewen Deng, Renhe Jiang, Jiaqi Zhang, Xuan Song
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Interactions between drugs can lead to adverse side effects for patients taking combination therapies to treat complex diseases such as cancer. Knowledge of drug-action towards a receptor would allow these drug-drug interactions to be predicted, and in this study, we trained a total of 5 different machine learning models to classify whether a given drug was an agonist (activator), antagonist (blocker), or a decoy (non-binder) to each of the androgen, estrogen, glucocorticoid, and progesterone receptors. The classification performance and efficiency, measured in training time, of the decision tree, naïve Bayes, neural network, random forest, and support vector machine models for each receptor were then compared. The results showed that the decision tree and naïve Bayes models were best suited for drug-action prediction across all receptors while only requiring minutes of training time at most. Future work will focus on increasing the prediction accuracy of antagonist drugs, integrating experimental data during training, and using other targets outside of nuclear type I receptors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3f1fd185e5d36f4f38477342cd4d936ce7439" target='_blank'>
              Application of Supervised Machine Learning Models for Drug-Action Prediction Towards Nuclear Type I Receptors
              </a>
            </td>
          <td>
            Rajeev Jaundoo, J. Tuszynski, T. Craddock
          </td>
          <td>2024-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Spatiotemporal predictive learning is a paradigm that empowers models to learn spatial and temporal patterns by predicting future frames from past frames in an unsupervised manner. This method typically uses recurrent units to capture long-term dependencies, but these units often come with high computational costs and limited performance in real-world scenes. This paper presents an innovative Wavelet-based SpatioTemporal (WaST) framework, which extracts and adaptively controls both low and high-frequency components at image and feature levels via 3D discrete wavelet transform for faster processing while maintaining high-quality predictions. We propose a Time-Frequency Aware Translator uniquely crafted to efficiently learn short- and long-range spatiotemporal information by individually modeling spatial frequency and temporal variations. Meanwhile, we design a wavelet-domain High-Frequency Focal Loss that effectively supervises high-frequency variations. Extensive experiments across various real-world scenarios, such as driving scene prediction, traffic flow prediction, human motion capture, and weather forecasting, demonstrate that our proposed WaST achieves state-of-the-art performance over various spatiotemporal prediction methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d2527c78839dcb903bbf1b178df2c1fbcbc30d" target='_blank'>
              Wavelet-Driven Spatiotemporal Predictive Learning: Bridging Frequency and Time Variations
              </a>
            </td>
          <td>
            Xuesong Nie, Yunfeng Yan, Siyuan Li, Cheng Tan, Xi Chen, Haoyuan Jin, Zhihang Zhu, Stan Z. Li, Donglian Qi
          </td>
          <td>2024-03-24</td>
          <td>DBLP</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Early phase, personalized dose-finding trials for combination therapies seek to identify patient-specific optimal biological dose (OBD) combinations, which are defined as safe dose combinations which maximize therapeutic benefit for a specific covariate pattern. Given the small sample sizes which are typical of these trials, it is challenging for traditional parametric approaches to identify OBD combinations across multiple dosing agents and covariate patterns. To address these challenges, we propose a Bayesian optimization approach to dose-finding which formally incorporates toxicity information into both the initial data collection process and the sequential search strategy. Independent Gaussian processes are used to model the efficacy and toxicity surfaces, and an acquisition function is utilized to define the dose-finding strategy and an early stopping rule. This work is motivated by a personalized dose-finding trial which considers a dual-agent therapy for obstructive sleep apnea, where OBD combinations are tailored to obstructive sleep apnea severity. To compare the performance of the personalized approach to a standard approach where covariate information is ignored, a simulation study is performed. We conclude that personalized dose-finding is essential in the presence of heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90f080ef4e69d5073a927e58d4f4a03e75c9555" target='_blank'>
              Bayesian Optimization for Identification of Optimal Biological Dose Combinations in Personalized Dose-Finding Trials
              </a>
            </td>
          <td>
            James J. Willard, Shirin Golchi, Erica EM. Moodie
          </td>
          <td>2024-04-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Increasing effort is put into the development of methods for learning mechanistic models from data. This task entails not only the accurate estimation of parameters, but also a suitable model structure. Recent work on the discovery of dynamical systems formulates this problem as a linear equation system. Here, we explore several simulation-based optimization approaches, which allow much greater freedom in the objective formulation and weaker conditions on the available data. We show that even for relatively small stochastic population models, simultaneous estimation of parameters and structure poses major challenges for optimization procedures. Particularly, we investigate the application of the local stochastic gradient descent method, commonly used for training machine learning models. We demonstrate accurate estimation of models but find that enforcing the inference of parsimonious, interpretable models drastically increases the difficulty. We give an outlook on how this challenge can be overcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f3306c5d346d91ce086b55f87d085d98c721dd" target='_blank'>
              Towards Learning Stochastic Population Models by Gradient Descent
              </a>
            </td>
          <td>
            J. N. Kreikemeyer, Philipp Andelfinger, A. Uhrmacher
          </td>
          <td>2024-04-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Recurrent Neural Networks excel at predicting and generating complex high-dimensional temporal patterns. Due to their inherent nonlinear dynamics and memory, they can learn unbounded temporal dependencies from data. In a Machine Learning setting, the network's parameters are adapted during a training phase to match the requirements of a given task/problem increasing its computational capabilities. After the training, the network parameters are kept fixed to exploit the learned computations. The static parameters thereby render the network unadaptive to changing conditions, such as external or internal perturbation. In this manuscript, we demonstrate how keeping parts of the network adaptive even after the training enhances its functionality and robustness. Here, we utilize the conceptor framework and conceptualize an adaptive control loop analyzing the network's behavior continuously and adjusting its time-varying internal representation to follow a desired target. We demonstrate how the added adaptivity of the network supports the computational functionality in three distinct tasks: interpolation of temporal patterns, stabilization against partial network degradation, and robustness against input distortion. Our results highlight the potential of adaptive networks in machine learning beyond training, enabling them to not only learn complex patterns but also dynamically adjust to changing environments, ultimately broadening their applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d77920a1f3057b33f9ab48db38a16dc2b0f292" target='_blank'>
              Adaptive control of recurrent neural networks using conceptors
              </a>
            </td>
          <td>
            Guillaume Pourcel, Mirko Goldmann, Ingo Fischer, Miguel C. Soriano
          </td>
          <td>2024-05-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1446b534858fc3fc6d9406920450b5c3ca87ee13" target='_blank'>
              Predicting equilibrium distributions for molecular systems with deep learning
              </a>
            </td>
          <td>
            Shuxin Zheng, Jiyan He, Chang Liu, Yu Shi, Ziheng Lu, Weitao Feng, Fusong Ju, Jiaxi Wang, Jianwei Zhu, Yaosen Min, He Zhang, Shidi Tang, Hongxia Hao, Peiran Jin, Chi Chen, Frank Noé, Haiguang Liu, Tie-Yan Liu
          </td>
          <td>2024-05-08</td>
          <td>Nature Machine Intelligence</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Widespread deployment of societal-scale machine learning systems necessitates a thorough understanding of the resulting long-term effects these systems have on their environment, including loss of trustworthiness, bias amplification, and violation of AI safety requirements. We introduce a repeated learning process to jointly describe several phenomena attributed to unintended hidden feedback loops, such as error amplification, induced concept drift, echo chambers and others. The process comprises the entire cycle of obtaining the data, training the predictive model, and delivering predictions to end-users within a single mathematical model. A distinctive feature of such repeated learning setting is that the state of the environment becomes causally dependent on the learner itself over time, thus violating the usual assumptions about the data distribution. We present a novel dynamical systems model of the repeated learning process and prove the limiting set of probability distributions for positive and negative feedback loop modes of the system operation. We conduct a series of computational experiments using an exemplary supervised learning problem on two synthetic data sets. The results of the experiments correspond to the theoretical predictions derived from the dynamical model. Our results demonstrate the feasibility of the proposed approach for studying the repeated learning processes in machine learning systems and open a range of opportunities for further research in the area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbdf526001c151c67507de963a0c683064b1f630" target='_blank'>
              A Mathematical Model of the Hidden Feedback Loop Effect in Machine Learning Systems
              </a>
            </td>
          <td>
            Andrey Veprikov, Alexander Afanasiev, Anton Khritankov
          </td>
          <td>2024-05-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="An accurate description of information is relevant for a range of problems in atomistic modeling, such as sampling methods, detecting rare events, analyzing datasets, or performing uncertainty quantification (UQ) in machine learning (ML)-driven simulations. Although individual methods have been proposed for each of these tasks, they lack a common theoretical background integrating their solutions. Here, we introduce an information theoretical framework that unifies predictions of phase transformations, kinetic events, dataset optimality, and model-free UQ from atomistic simulations, thus bridging materials modeling, ML, and statistical mechanics. We first demonstrate that, for a proposed representation, the information entropy of a distribution of atom-centered environments is a surrogate value for thermodynamic entropy. Using molecular dynamics (MD) simulations, we show that information entropy differences from trajectories can be used to build phase diagrams, identify rare events, and recover classical theories of nucleation. Building on these results, we use this general concept of entropy to quantify information in datasets for ML interatomic potentials (IPs), informing compression, explaining trends in testing errors, and evaluating the efficiency of active learning strategies. Finally, we propose a model-free UQ method for MLIPs using information entropy, showing it reliably detects extrapolation regimes, scales to millions of atoms, and goes beyond model errors. This method is made available as the package QUESTS: Quick Uncertainty and Entropy via STructural Similarity, providing a new unifying theory for data-driven atomistic modeling and combining efforts in ML, first-principles thermodynamics, and simulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e99b6feff6791ccfee904ee2f02b239d0ced776" target='_blank'>
              Information theory unifies atomistic machine learning, uncertainty quantification, and materials thermodynamics
              </a>
            </td>
          <td>
            Daniel Schwalbe-Koda, Sebastien Hamel, Babak Sadigh, Fei Zhou, Vincenzo Lordi
          </td>
          <td>2024-04-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983a84ba84d4e4978cb8f3fe20f03553417b6b7e" target='_blank'>
              DeepARV: ensemble deep learning to predict drug-drug interaction of clinical relevance with antiretroviral therapy
              </a>
            </td>
          <td>
            Thao Pham, Mohamed Ghafoor, Sandra Grañana-Castillo, C. Marzolini, Sara Gibbons, Saye Khoo, J. Chiong, Dennis Wang, Marco Siccardi
          </td>
          <td>2024-05-06</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="In clinical settings, we often face the challenge of building prediction models based on small observational data sets. For example, such a data set might be from a medical center in a multi-center study. Differences between centers might be large, thus requiring specific models based on the data set from the target center. Still, we want to borrow information from the external centers, to deal with small sample sizes. There are approaches that either assign weights to each external data set or each external observation. To incorporate information on differences between data sets and observations, we propose an approach that combines both into weights that can be incorporated into a likelihood for fitting regression models. Specifically, we suggest weights at the data set level that incorporate information on how well the models that provide the observation weights distinguish between data sets. Technically, this takes the form of inverse probability weighting. We explore different scenarios where covariates and outcomes differ among data sets, informing our simulation design for method evaluation. The concept of effective sample size is used for understanding the effectiveness of our subgroup modeling approach. We demonstrate our approach through a clinical application, predicting applied radiotherapy doses for cancer patients. Generally, the proposed approach provides improved prediction performance when external data sets are similar. We thus provide a method for quantifying similarity of external data sets to the target data set and use this similarity to include external observations for improving performance in a target data set prediction modeling task with small data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/806bd6d29b956df4eb6efcad0b248b9f647ca1ca" target='_blank'>
              Improving prediction models by incorporating external data with weights based on similarity
              </a>
            </td>
          <td>
            Max Behrens, Maryam Farhadizadeh, Angelika Rohde, Alexander Ruhle, N. H. Nicolay, Harald Binder, Daniela Zoller
          </td>
          <td>2024-05-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The idea of using deep-learning-based molecular generation to accelerate discovery of drug candidates has attracted extraordinary attention, and many deep generative models have been developed for automated drug design, termed molecular generation. In general, molecular generation encompasses two main strategies: de novo design, which generates novel molecular structures from scratch, and lead optimization, which refines existing molecules into drug candidates. Among them, lead optimization plays an important role in real-world drug design. For example, it can enable the development of me-better drugs that are chemically distinct yet more effective than the original drugs. It can also facilitate fragment-based drug design, transforming virtual-screened small ligands with low affinity into first-in-class medicines. Despite its importance, automated lead optimization remains underexplored compared to the well-established de novo generative models, due to its reliance on complex biological and chemical knowledge. To bridge this gap, we conduct a systematic review of traditional computational methods for lead optimization, organizing these strategies into four principal sub-tasks with defined inputs and outputs. This review delves into the basic concepts, goals, conventional CADD techniques, and recent advancements in AIDD. Additionally, we introduce a unified perspective based on constrained subgraph generation to harmonize the methodologies of de novo design and lead optimization. Through this lens, de novo design can incorporate strategies from lead optimization to address the challenge of generating hard-to-synthesize molecules; inversely, lead optimization can benefit from the innovations in de novo design by approaching it as a task of generating molecules conditioned on certain substructures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7334967da38a0e9d6f0719d0a0e1ffa3403140b3" target='_blank'>
              Deep Lead Optimization: Leveraging Generative AI for Structural Modification
              </a>
            </td>
          <td>
            Odin Zhang, Haitao Lin, Hui Zhang, Huifeng Zhao, Yufei Huang, Yuansheng Huang, Dejun Jiang, Chang-Yu Hsieh, P. Pan, Tingjun Hou
          </td>
          <td>2024-04-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The role of surgery in metastatic breast cancer (MBC) is currently controversial. Several novel statistical and deep learning (DL) methods promise to infer the suitability of surgery at the individual level.The objective of this study was to identify the most applicable DL model for determining patients with MBC who could benefit from surgery and the type of surgery required.We introduced the deep survival regression with mixture effects (DSME), a semi‐parametric DL model integrating three causal inference methods. Six models were trained to make individualized treatment recommendations. Patients who received treatments in line with the DL models' recommendations were compared with those who underwent treatments divergent from the recommendations. Inverse probability weighting (IPW) was used to minimize bias. The effects of various features on surgery selection were visualized and quantified using multivariate linear regression and causal inference.In total, 5269 female patients with MBC were included. DSME was an independent protective factor, outperforming other models in recommending surgery (IPW‐adjusted hazard ratio [HR] = 0.39, 95% confidence interval [CI]: 0.19–0.78) and type of surgery (IPW‐adjusted HR = 0.66, 95% CI: 0.48–0.93). DSME was superior to other models and traditional guidelines, suggesting a higher proportion of patients benefiting from surgery, especially breast‐conserving surgery. The debiased effect of patient characteristics, including age, tumor size, metastatic sites, lymph node status, and breast cancer subtypes, on surgery decision was also quantified.Our findings suggested that DSME could effectively identify patients with MBC likely to benefit from surgery and the specific type of surgery needed. This method can facilitate the development of efficient, reliable treatment recommendation systems and provide quantifiable evidence for decision‐making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da62886f95973a7efe1b5f65259b9781cd201f80" target='_blank'>
              Personalized surgical recommendations and quantitative therapeutic insights for patients with metastatic breast cancer: Insights from deep learning
              </a>
            </td>
          <td>
            Enzhao Zhu, Linmei Zhang, Jiayi Wang, Chunyu Hu, Qi Jing, Weizhong Shi, Ziqin Xu, Pu Ai, Zhihao Dai, Dan Shan, Zisheng Ai
          </td>
          <td>2024-04-15</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Plasma medicine has attracted tremendous interest in a variety of medical conditions, ranging from wound healing to antimicrobial applications, even in cancer treatment, through the interactions of cold atmospheric plasma (CAP) and various biological tissues directly or indirectly. The underlying mechanisms of CAP treatment are still poorly understood although the oxidative effects of CAP with amino acids, peptides, and proteins have been explored experimentally. In this study, machine learning (ML) technology is introduced to efficiently unveil the interaction mechanisms of amino acids and reactive oxygen species (ROS) in seconds based on the data obtained from the reactive molecular dynamics (MD) simulations, which are performed to probe the interaction of five types of amino acids with various ROS on the timescale of hundreds of picoseconds but with the huge computational load of several days. The oxidative reactions typically start with H‐abstraction, and the details of the breaking and formation of chemical bonds are revealed; the modification types, such as nitrosylation, hydroxylation, and carbonylation, can be observed. The dose effects of ROS are also investigated by varying the number of ROS in the simulation box, indicating agreement with the experimental observation. To overcome the limits of timescales and the size of molecular systems in reactive MD simulations, a deep neural network (DNN) with five hidden layers is constructed according to the reaction data and employed to predict the type of oxidative modification and the probability of occurrence only in seconds as the dose of ROS varies. The well‐trained DNN can effectively and accurately predict the oxidative processes and productions, which greatly improves the computational efficiency by almost ten orders of magnitude compared with the reactive MD simulation. This study shows the great potential of ML technology to efficiently unveil the underpinning mechanisms in plasma medicine based on the data from reactive MD simulations or experimental measurements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ade84d4b8341e195f41099db326bfe6fe40c358" target='_blank'>
              Unveiling the interaction mechanisms of cold atmospheric plasma and amino acids by machine learning
              </a>
            </td>
          <td>
            Zhao‐Nan Chai, Xucheng Wang, M. Yusupov, Yuantao Zhang
          </td>
          <td>2024-04-15</td>
          <td>Plasma Processes and Polymers</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Levothyroxine is one of the most prescribed drugs in the western world. Dosing is challenging due to high-interindividual differences in effective dosage and the narrow therapeutic window. Model-informed precision dosing (MIPD) using machine learning could assist general practitioners (GPs), but no such models exist for primary care. Furthermore, introduction of decision-support algorithms in healthcare is limited due to the substantial gap between developers and clinicians' perspectives. We report the development, validation, and a clinical simulation trial of the first MIPD application for primary care. Stable maintenance dosage of levothyroxine was the model target. The multiclass model generates predictions for individual patients, for different dosing classes. Random forest was trained and tested on a national primary care database (n = 19,004) with a final weighted AUC across dosing options of 0.71, even in subclinical hypothyroidism. TSH, fT4, weight, and age were most predictive. To assess the safety, feasibility, and clinical impact of MIPD for levothyroxine, we performed clinical simulation studies in GPs and compared MIPD to traditional prescription. Fifty-one GPs selected starting dosages for 20 primary hypothyroidism cases without and then with MIPD 2 weeks later. Overdosage and underdosage were defined as higher and lower than 12.5 μg relative to stable maintenance dosage. MIPD decreased overdosage in number (30.5 to 23.9%, P < 0.01) and magnitude (median 50 to 37.5 μg, P < 0.01) and increased optimal starting dosages (18.3 to 30.2%, P < 0.01). GPs considered lab results more often with MIPD and most would use the model frequently. This study demonstrates the clinical relevance, safety, and effectiveness of MIPD for levothyroxine in primary care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bb62b88b01331a3ef7595f9fd5ac6627f03ef92" target='_blank'>
              Model-Informed Precision Dosing Using Machine Learning for Levothyroxine in General Practice: Development, Validation and Clinical Simulation Trial.
              </a>
            </td>
          <td>
            Jules M Janssen Daalen, Djoeke Doesburg, Liesbeth Hunik, Rogier Kessel, Thomas Herngreen, Dennis Knol, Thony Ruys, B. J. F. van den Bemt, Henk J. Schers
          </td>
          <td>2024-05-06</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Complex diseases do not always follow gradual progressions. Instead, they may experience sudden shifts known as critical states or tipping points, where a marked qualitative change occurs. Detecting such a pivotal transition or pre-deterioration state holds paramount importance due to its association with severe disease deterioration. Nevertheless, the task of pinpointing the pre-deterioration state for complex diseases remains an obstacle, especially in scenarios involving high-dimensional data with limited samples, where conventional statistical methods frequently prove inadequate. In this study, we introduce an innovative quantitative approach termed sample-specific causality network entropy (SCNE), which infers a sample-specific causality network for each individual and effectively quantifies the dynamic alterations in causal relations among molecules, thereby capturing critical points or pre-deterioration states of complex diseases. We substantiated the accuracy and efficacy of our approach via numerical simulations and by examining various real-world datasets, including single-cell data of epithelial cell deterioration (EPCD) in colorectal cancer, influenza infection data, and three different tumor cases from The Cancer Genome Atlas (TCGA) repositories. Compared to other existing six single-sample methods, our proposed approach exhibits superior performance in identifying critical signals or pre-deterioration states. Additionally, the efficacy of computational findings is underscored by analyzing the functionality of signaling biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2066399908a66ea337f2aaefedc0236ac3edeb8" target='_blank'>
              Uncovering the Pre-Deterioration State during Disease Progression Based on Sample-Specific Causality Network Entropy (SCNE)
              </a>
            </td>
          <td>
            Jiayuan Zhong, Hui Tang, Ziyi Huang, Hua Chai, Fei Ling, Pei Chen, Rui Liu
          </td>
          <td>2024-04-08</td>
          <td>Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Today, machine learning methods are widely employed in drug discovery. However, the chronic lack of data continues to hamper their further development, validation, and application. Several modern strategies aim to mitigate the challenges associated with data scarcity by learning from data on related tasks. These knowledge-sharing approaches encompass transfer learning, multitask learning, and meta-learning. A key question remaining to be answered for these approaches is about the extent to which their performance can benefit from the relatedness of available source (training) tasks; in other words, how difficult ("hard") a test task is to a model, given the available source tasks. This study introduces a new method for quantifying and predicting the hardness of a bioactivity prediction task based on its relation to the available training tasks. The approach involves the generation of protein and chemical representations and the calculation of distances between the bioactivity prediction task and the available training tasks. In the example of meta-learning on the FS-Mol data set, we demonstrate that the proposed task hardness metric is inversely correlated with performance (Pearson's correlation coefficient r = -0.72). The metric will be useful in estimating the task-specific gain in performance that can be achieved through meta-learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcfa4dac8e8d06d6ca42757c4b9d06476021e771" target='_blank'>
              Quantifying the Hardness of Bioactivity Prediction Tasks for Transfer Learning.
              </a>
            </td>
          <td>
            Hosein Fooladi, Steffen Hirte, J. Kirchmair
          </td>
          <td>2024-05-13</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af19be9576fba29fc6ce10addc184ed0663d1348" target='_blank'>
              A Computational Predictor for Accurate Identification of Tumor Homing Peptides by Integrating Sequential and Deep BiLSTM Features.
              </a>
            </td>
          <td>
            Roha Arif, Sameera Kanwal, Saeed Ahmed, Muhammad Kabir
          </td>
          <td>2024-05-11</td>
          <td>Interdisciplinary sciences, computational life sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="In the evolving landscape of drug discovery, traditional methods face significant challenges due to low efficiency and high resource demands. This study introduces our GALILEO AI drug discovery platform and its foundational ChemPrint model, designed to improve the efficiency of drug discovery. Addressing the challenges of low hit rates and the difficulty in exploring novel chemical spaces, this platform adopts adaptive molecular embeddings and stringent model training environments to enhance predictive capabilities and navigate uncharted molecular territories. We validate this approach through a case study targeting the AXL and BRD4 oncology targets, where ChemPrint achieved a 45.5% in vitro hit rate and identified 20 novel acting compounds. These compounds exhibited substantial chemical novelty, with an average Tanimoto similarity score of 0.32 to their training set, significantly diverging from known compounds and demonstrating ChemPrint’s capability to extrapolate, surpassing traditional benchmarks and industry standards. This underscores the platform’s ability to bridge the gap between the potential of AI and its practical application in therapeutic discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7a47c3bf7bae09d7eec168709fd507b8b22100" target='_blank'>
              ChemPrint: An AI-Driven Framework for Enhanced Drug Discovery
              </a>
            </td>
          <td>
            Tyler Umansky, Navya Ramesh, Virgil Woods, Sean M. Russell, Davey Smith, Daniel Haders
          </td>
          <td>2024-03-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The rate at which the anticancer drug paclitaxel is cleared from the body markedly impacts its dosage and chemotherapy effectiveness. Importantly, paclitaxel clearance varies among individuals, primarily because of genetic polymorphisms. This metabolic variability arises from a nonlinear process that is influenced by multiple single nucleotide polymorphisms (SNPs). Conventional bioinformatics methods struggle to accurately analyze this complex process and, currently, there is no established efficient algorithm for investigating SNP interactions.We developed a novel machine‐learning approach called GEP‐CSIs data mining algorithm. This algorithm, an advanced version of GEP, uses linear algebra computations to handle discrete variables. The GEP‐CSI algorithm calculates a fitness function score based on paclitaxel clearance data and genetic polymorphisms in patients with nonsmall cell lung cancer. The data were divided into a primary set and a validation set for the analysis.We identified and validated 1184 three‐SNP combinations that had the highest fitness function values. Notably, SERPINA1, ATF3 and EGF were found to indirectly influence paclitaxel clearance by coordinating the activity of genes previously reported to be significant in paclitaxel clearance. Particularly intriguing was the discovery of a combination of three SNPs in genes FLT1, EGF and MUC16. These SNPs‐related proteins were confirmed to interact with each other in the protein–protein interaction network, which formed the basis for further exploration of their functional roles and mechanisms.We successfully developed an effective deep‐learning algorithm tailored for the nuanced mining of SNP interactions, leveraging data on paclitaxel clearance and individual genetic polymorphisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a0123f1a98ea05c383a12137d7f078326c6ee2a" target='_blank'>
              Novel progressive deep learning algorithm for uncovering multiple single nucleotide polymorphism interactions to predict paclitaxel clearance in patients with nonsmall cell lung cancer
              </a>
            </td>
          <td>
            Wei Chen, Haiyan Zhou, Mingyu Zhang, Yafei Shi, Taifeng Li, Di Qian, Jun Yang, Feng Yu, Guohui Li
          </td>
          <td>2024-05-12</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Adverse reactions caused by drug combinations are an increasingly common phenomenon, making their accurate prediction an important challenge in modern medicine. However, the polynomial nature of this problem renders lab-based identification of adverse reactions insufficient. Dozens of computational approaches have therefore been proposed for the task in recent years, with varying degrees of success. One group of methods that has seemingly been under-utilised in this area is tensor factorisation, despite their clear applicability to this type of data. In this work, we apply three such models to a benchmark dataset in order to compare them against established techniques. We find, in contrast to previous reports, that for this task tensor factorisation models are competitive with state-of-the-art graph neural network models and we recommend that future work in this field considers cheaper methods with linear complexity before running costly deep learning processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad7affcf7fd788c02f0036fbc1daa7bcc16d9ab1" target='_blank'>
              Tensor Factorisation for Polypharmacy Side Effect Prediction
              </a>
            </td>
          <td>
            Oliver Lloyd, Yi Liu, T. Gaunt
          </td>
          <td>2024-04-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Classical molecular dynamics (MD) simulations represent a very popular and powerful tool for materials modeling and design. The predictive power of MD hinges on the ability of the interatomic potential to capture the underlying physics and chemistry. There have been decades of seminal work on developing interatomic potentials, albeit with a focus predominantly on capturing the properties of bulk materials. Such physics-based models, while extensively deployed for predicting the dynamics and properties of nanoscale systems over the past two decades, tend to perform poorly in predicting nanoscale potential energy surfaces (PESs) when compared to high-fidelity first-principles calculations. These limitations stem from the lack of flexibility in such models, which rely on a predefined functional form. Machine learning (ML) models and approaches have emerged as a viable alternative to capture the diverse size-dependent cluster geometries, nanoscale dynamics, and the complex nanoscale PESs, without sacrificing the bulk properties. Here, we introduce an ML workflow that combines transfer and active learning strategies to develop high-dimensional neural networks (NNs) for capturing the cluster and bulk properties for several different transition metals with applications in catalysis, microelectronics, and energy storage, to name a few. Our NN first learns the bulk PES from the high-quality physics-based models in literature and subsequently augments this learning via retraining with a higher-fidelity first-principles training data set to concurrently capture both the nanoscale and bulk PES. Our workflow departs from status-quo in its ability to learn from a sparsely sampled data set that nonetheless covers a diverse range of cluster configurations from near-equilibrium to highly nonequilibrium as well as learning strategies that iteratively improve the fingerprinting depending on model fidelity. All the developed models are rigorously tested against an extensive first-principles data set of energies and forces of cluster configurations as well as several properties of bulk configurations for 10 different transition metals. Our approach is material agnostic and provides a methodology to transfer and build upon the learnings from decades of seminal work in molecular simulations on to a new generation of ML-trained potentials to accelerate materials discovery and design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76b4d6e760d1b287b310104f11d2c8aea330394" target='_blank'>
              Active and Transfer Learning of High-Dimensional Neural Network Potentials for Transition Metals.
              </a>
            </td>
          <td>
            Bilvin Varughese, Sukriti Manna, T. Loeffler, Rohit Batra, M. Cherukara, SubramanianK.R.S. Sankaranarayanan
          </td>
          <td>2024-04-09</td>
          <td>ACS applied materials & interfaces</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Predicting future events always comes with uncertainty, but traditional non-probabilistic methods cannot distinguish certain from uncertain predictions. In survival analysis, probabilistic methods applied to state-of-the-art solutions in the healthcare and biomedical field are still novel, and their implications have not been fully evaluated. In this paper, we study the benefits of modeling uncertainty in deep neural networks for survival analysis with a focus on prediction and calibration performance. For this, we present a Bayesian deep learning framework that consists of three probabilistic network architectures, which we train by optimizing the Cox partial likelihood and combining input-dependent aleatoric uncertainty together with epistemic uncertainty. This enables us to provide uncertainty estimates as credible intervals when predicting the survival curve or as a probability density function over the predicted median survival times. For our empirical analyses, we evaluated our proposed method on four benchmark datasets and found that our method demonstrates prediction performance comparable to the state-of-the-art based on the concordance index and outperforms all other Cox-based approaches in terms of the mean absolute error. Our work explicitly compares the extent to which different Bayesian approximation techniques differ from each other and improves the prediction over traditional non-probabilistic alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062022270313e044624255314c4ad044d0a6866b" target='_blank'>
              Probabilistic Survival Analysis by Approximate Bayesian Inference of Neural Networks
              </a>
            </td>
          <td>
            Christian Marius Lillelund, Martin Magris, Christian Fischer Pedersen
          </td>
          <td>2024-04-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Predictive uncertainty quantification is crucial for reliable decision-making in various applied domains. Bayesian neural networks offer a powerful framework for this task. However, defining meaningful priors and ensuring computational efficiency remain significant challenges, especially for complex real-world applications. This paper addresses these challenges by proposing a novel neural adaptive empirical Bayes (NA-EB) framework. NA-EB leverages a class of implicit generative priors derived from low-dimensional distributions. This allows for efficient handling of complex data structures and effective capture of underlying relationships in real-world datasets. The proposed NA-EB framework combines variational inference with a gradient ascent algorithm. This enables simultaneous hyperparameter selection and approximation of the posterior distribution, leading to improved computational efficiency. We establish the theoretical foundation of the framework through posterior and classification consistency. We demonstrate the practical applications of our framework through extensive evaluations on a variety of tasks, including the two-spiral problem, regression, 10 UCI datasets, and image classification tasks on both MNIST and CIFAR-10 datasets. The results of our experiments highlight the superiority of our proposed framework over existing methods, such as sparse variational Bayesian and generative models, in terms of prediction accuracy and uncertainty quantification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6575c13ef806ffa498b6a7d3b66ef253aee49bd" target='_blank'>
              Implicit Generative Prior for Bayesian Neural Networks
              </a>
            </td>
          <td>
            Yijia Liu, Xiao Wang
          </td>
          <td>2024-04-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78d12260785531902ba470dece593699953a398" target='_blank'>
              Image-based biomarkers for engineering neuroblastoma patient-specific computational models
              </a>
            </td>
          <td>
            S. Hervas-Raluy, D. Sainz-DeMena, M. Gómez-Benito, J. García-Aznar
          </td>
          <td>2024-04-10</td>
          <td>Engineering with Computers</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dabd071ab522df047a3703298ac2d13e4bb72edc" target='_blank'>
              Global ranking of the sensitivity of interaction potential contributions within classical molecular dynamics force fields
              </a>
            </td>
          <td>
            W. Edeling, M. Vassaux, Yiming Yang, S. Wan, Serge Guillas, Peter V. Coveney
          </td>
          <td>2024-05-03</td>
          <td>npj Computational Materials</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="A crucial component of the treatment of genetic disorders is identifying and characterising the genes and gene modules that drive disease processes. Recent advances in Next-Generation Sequencing improve the prospects for achieving this goal. However, many machine learning techniques are not explainable and fail to account for gene correlations. In this work, we develop a comprehensive set of explainable machine learning techniques to perform patient stratification for inflammatory bowel disease. We focus on Crohn's disease (CD) and its subtypes: CD with deep ulcer, CD without deep ulcer and IBD-controls. We produce an interpretable probabilistic model over disease subtypes using Gaussian Mixture Modelling. We then apply class-contrastive and feature-attribution techniques to identify potential target genes and modules. We modify the widely used kernelSHAP (Shapley Additive Explanations) algorithm to account for gene correlations. We obtain relevant gene modules for each disease subtype. We develop a class-contrastive technique to visually explain why a particular patient is predicted to have a particular subtype of the disease. We show that our results are relevant to the disease through Gene Ontology enrichment analysis and a review of the literature. We also uncover some novel findings, including currently uncharacterised genes. These approaches maybe beneficial, in personalised medicine, to inform decision-making regarding the diagnosis and treatment of genetic disorders. Our approach is model-agnostic and can potentially be applied to other diseases and domains where explainability and feature correlations are important.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b436a9056cfaa95c2cfd6d981ba44b816f12a585" target='_blank'>
              ENHANCING PATIENT STRATIFICATION AND INTERPRETABILITY THROUGH CLASS-CONTRASTIVE AND FEATURE ATTRIBUTION TECHNIQUES
              </a>
            </td>
          <td>
            S. Olowu, N. Lawrence, S. Banerjee
          </td>
          <td>2024-03-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Reduced order models based on the transport of a lower dimensional manifold representation of the thermochemical state, such as Principal Component (PC) transport and Machine Learning (ML) techniques, have been developed to reduce the computational cost associated with the Direct Numerical Simulations (DNS) of reactive flows. Both PC transport and ML normally require an abundance of data to exhibit sufficient predictive accuracy, which might not be available due to the prohibitive cost of DNS or experimental data acquisition. To alleviate such difficulties, similar data from an existing dataset or domain (source domain) can be used to train ML models, potentially resulting in adequate predictions in the domain of interest (target domain). This study presents a novel probabilistic transfer learning (TL) framework to enhance the trust in ML models in correctly predicting the thermochemical state in a lower dimensional manifold and a sparse data setting. The framework uses Bayesian neural networks, and autoencoders, to reduce the dimensionality of the state space and diffuse the knowledge from the source to the target domain. The new framework is applied to one-dimensional freely-propagating flame solutions under different data sparsity scenarios. The results reveal that there is an optimal amount of knowledge to be transferred, which depends on the amount of data available in the target domain and the similarity between the domains. TL can reduce the reconstruction error by one order of magnitude for cases with large sparsity. The new framework required 10 times less data for the target domain to reproduce the same error as in the abundant data scenario. Furthermore, comparisons with a state-of-the-art deterministic TL strategy show that the probabilistic method can require four times less data to achieve the same reconstruction error.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b40d43f32053fa66703ccc372c93df9eb12e60d" target='_blank'>
              Probabilistic transfer learning methodology to expedite high fidelity simulation of reactive flows
              </a>
            </td>
          <td>
            Bruno S. Soriano, , T. Echekki, Jacqueline H. Chen, Mohammad Khalil
          </td>
          <td>2024-05-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The proposed system introduces a novel Deep Metric Algorithm as a cornerstone for enhancing the feasibility and optimality of network traffic prediction within the dynamic landscape of cloud technologies. The Deep Metric Algorithm is designed to operate as a physically interpretable probabilistic model that captures network-wide traffic patterns. Unlike traditional methods that rely solely on current traffic states observed at specific intervals, this algorithm leverages an initially expensive set of measurements to construct a network-specific traffic model.The algorithm's innovation lies in its ability to utilize this network-specific model alongside readily available, less expensive measurements, thereby significantly improving the accuracy of traffic predictions. By employing the Deep Metric Algorithm, the system can predict traffic fluctuations not only over observed links but also extend its predictions to unobserved links. This is a crucial advancement as it enables the system to offer more comprehensive insights into network behavior, even where direct measurements might be limited.One notable strength of the proposed Deep Metric Algorithm is its demonstrated applicability across various traffic periods within the same network. This adaptability ensures that the learned model remains effective over time, showcasing its potential utility in optimizing network performance as it evolves. The algorithm's predictive capabilities, coupled with its ability to adapt to changing network conditions, position it as a valuable tool for load-aware resource management and predictive control, thereby contributing to the efficient management of multi-provider networks in the evolving cloud landscape. KEYWORDS: Quality of Service (QoS), network traffic, accurate predictions, network-specific traffic model, Deep Metric Algorithm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f2047cbd476efb4ec5f3007e231edafe9e596a5" target='_blank'>
              ROBUST LEARNING AND ACCURATE PREDICTION OF WIRELESS TRAFFIC USING IMPROVED NEURAL NETWORK
              </a>
            </td>
          <td>
            P. S
          </td>
          <td>2024-05-07</td>
          <td>INTERANTIONAL JOURNAL OF SCIENTIFIC RESEARCH IN ENGINEERING AND MANAGEMENT</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a9879b34cb5e60fcdaf34ab6ca25479599cc08" target='_blank'>
              Explainable and visualizable machine learning models to predict biochemical recurrence of prostate cancer.
              </a>
            </td>
          <td>
            Wenhao Lu, Lin Zhao, Shenfan Wang, Huiyong Zhang, Kangxian Jiang, Jin Ji, Shaohua Chen, Chengbang Wang, Chunmeng Wei, Rongbin Zhou, Zuheng Wang, Xiao Li, Fubo Wang, Xuedong Wei, Wenlei Hou
          </td>
          <td>2024-04-11</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="We consider using deep neural networks to solve time-dependent partial differential equations (PDEs), where multi-scale processing is crucial for modeling complex, time-evolving dynamics. While the U-Net architecture with skip connections is commonly used by prior studies to enable multi-scale processing, our analysis shows that the need for features to evolve across layers results in temporally misaligned features in skip connections, which limits the model's performance. To address this limitation, we propose SineNet, consisting of multiple sequentially connected U-shaped network blocks, referred to as waves. In SineNet, high-resolution features are evolved progressively through multiple stages, thereby reducing the amount of misalignment within each stage. We furthermore analyze the role of skip connections in enabling both parallel and sequential processing of multi-scale information. Our method is rigorously tested on multiple PDE datasets, including the Navier-Stokes equations and shallow water equations, showcasing the advantages of our proposed approach over conventional U-Nets with a comparable parameter budget. We further demonstrate that increasing the number of waves in SineNet while maintaining the same number of parameters leads to a monotonically improved performance. The results highlight the effectiveness of SineNet and the potential of our approach in advancing the state-of-the-art in neural PDE solver design. Our code is available as part of AIRS (https://github.com/divelab/AIRS).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc950089c44fe403b89b1c7866ac826557e08284" target='_blank'>
              SineNet: Learning Temporal Dynamics in Time-Dependent Partial Differential Equations
              </a>
            </td>
          <td>
            Xuan Zhang, Jacob Helwig, Yu-Ching Lin, Yaochen Xie, Cong Fu, Stephan Wojtowytsch, Shuiwang Ji
          </td>
          <td>2024-03-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Molecular property prediction has gained substantial attention due to its potential for various bio-chemical applications. Numerous attempts have been made to enhance the performance by combining multiple molecular representations (1D, 2D, and 3D). However, most prior works only merged a limited number of representations or tried to embed multiple representations through a single network without using representation-specific networks. Furthermore, the heterogeneous characteristics of each representation made the fusion more challenging. Addressing these challenges, we introduce the Fusion Transformer for Multiple Molecular Representations (FTMMR) framework. Our strategy employs three distinct representation-specific networks and integrates information from each network using a fusion transformer architecture to generate fused representations. Additionally, we use self-supervised learning methods to align heterogeneous representations and to effectively utilize the limited chemical data available. In particular, we adopt a combinatorial loss function to leverage the contrastive loss for all three representations. We evaluate the performance of FTMMR using seven benchmark datasets, demonstrating that our framework outperforms existing fusion and self-supervised methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d7a915a21c085446afc3cd21611e1c4f111e16f" target='_blank'>
              FTMMR: Fusion Transformer for Integrating Multiple Molecular Representations.
              </a>
            </td>
          <td>
            Young-Han Son, Dong-Hee Shin, Tae-Eui Kam
          </td>
          <td>2024-03-29</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96ce73317e29a3050795d9cfd776ded7c591953a" target='_blank'>
              Understanding and Quantifying Network Robustness to Stochastic Inputs
              </a>
            </td>
          <td>
            Hwai-Ray Tung, S. Lawley
          </td>
          <td>2024-04-12</td>
          <td>Bulletin of Mathematical Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Identifying partial differential equations (PDEs) from data is crucial for understanding the governing mechanisms of natural phenomena, yet it remains a challenging task. We present an extension to the ARGOS framework, ARGOS-RAL, which leverages sparse regression with the recurrent adaptive lasso to identify PDEs from limited prior knowledge automatically. Our method automates calculating partial derivatives, constructing a candidate library, and estimating a sparse model. We rigorously evaluate the performance of ARGOS-RAL in identifying canonical PDEs under various noise levels and sample sizes, demonstrating its robustness in handling noisy and non-uniformly distributed data. We also test the algorithm's performance on datasets consisting solely of random noise to simulate scenarios with severely compromised data quality. Our results show that ARGOS-RAL effectively and reliably identifies the underlying PDEs from data, outperforming the sequential threshold ridge regression method in most cases. We highlight the potential of combining statistical methods, machine learning, and dynamical systems theory to automatically discover governing equations from collected data, streamlining the scientific modeling process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6ba94461e38ca92209eaa1e802d6a39c777186" target='_blank'>
              Automating the Discovery of Partial Differential Equations in Dynamical Systems
              </a>
            </td>
          <td>
            Weizhen Li, Rui Carvalho
          </td>
          <td>2024-04-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Precisely predicting molecular properties is crucial in drug discovery, but the scarcity of labeled data poses a challenge for applying deep learning methods. While large-scale self-supervised pretraining has proven an effective solution, it often neglects domain-specific knowledge. To tackle this issue, we introduce Task-Oriented Multilevel Learning based on BERT (TOML-BERT), a dual-level pretraining framework that considers both structural patterns and domain knowledge of molecules. TOML-BERT achieved state-of-the-art prediction performance on 10 pharmaceutical datasets. It has the capability to mine contextual information within molecular structures and extract domain knowledge from massive pseudo-labeled data. The dual-level pretraining accomplished significant positive transfer, with its two components making complementary contributions. Interpretive analysis elucidated that the effectiveness of the dual-level pretraining lies in the prior learning of a task-related molecular representation. Overall, TOML-BERT demonstrates the potential of combining multiple pretraining tasks to extract task-oriented knowledge, advancing molecular property prediction in drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f91cd994a0476b1e1c720d21fe7f5ceed94b2fb" target='_blank'>
              Enhancing Molecular Property Prediction through Task-Oriented Transfer Learning: Integrating Universal Structural Insights and Domain-Specific Knowledge.
              </a>
            </td>
          <td>
            , Xixi Yang, Xiangxiang Zeng, Wenxuan Wang, Youchao Deng, Dongsheng Cao
          </td>
          <td>2024-05-15</td>
          <td>Journal of medicinal chemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="This news section offers Cancer readers timely information on events, public policy analysis, topical issues, and personalities. In this issue, a dynamic risk prediction model for multiple myeloma is discussed. Through deep learning, the model is able to adapt to future data, such as emerging treatment modalities and combinations. In addition, global phase 3 results show that bevacizumab with immunotherapy has activity in liver cancer, and D‐VRd is examined as a new standard of care in patients with multiple myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b64f34200af5443cad0d6f7912e702ed6cd6c03" target='_blank'>
              Dynamic risk prediction model for multiple myeloma: Through deep learning, the model is able to adapt to future data, such as emerging treatment modalities and combinations.
              </a>
            </td>
          <td>
            M. Nierengarten
          </td>
          <td>2024-04-11</td>
          <td>Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="To support mechanism online learning and facilitate digital twin development for biomanufacturing processes, this paper develops an efficient Bayesian inference approach for partially observed enzymatic stochastic reaction network (SRN), a fundamental building block of multi-scale bioprocess mechanistic model. To tackle the critical challenges brought by the nonlinear stochastic differential equations (SDEs)-based mechanistic model with partially observed state and having measurement error, an interpretable Bayesian updating linear noise approximation (LNA) metamodel, incorporating the structure information of the mechanistic model, is proposed to approximate the likelihood of observations. Then, an efficient posterior sampling approach is developed by utilizing the gradients of the derived likelihood to speed up the convergence of MCMC. The empirical study demonstrates that the proposed approach has a promising performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57d6d6543be531495a2c396519431852457ac4af" target='_blank'>
              Linear Noise Approximation Assisted Bayesian Inference on Mechanistic Model of Partially Observed Stochastic Reaction Network
              </a>
            </td>
          <td>
            Wandi Xu, Wei Xie
          </td>
          <td>2024-05-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A comprehensive and reliable survival prediction model is of great importance to assist in the personalized management of Head and Neck Cancer (HNC) patients treated with curative Radiation Therapy (RT). In this work, we propose IMLSP, an Interpretable Multi-Label multi-modal deep Survival Prediction framework for predicting multiple HNC survival outcomes simultaneously and provide time-event specific visual explanation of the deep prediction process. We adopt Multi-Task Logistic Regression (MTLR) layers to convert survival prediction from a regression problem to a multi-time point classification task, and to enable predicting of multiple relevant survival outcomes at the same time. We also present Grad-TEAM, a Gradient-weighted Time-Event Activation Mapping approach specifically developed for deep survival model visual explanation, to generate patient-specific time-to-event activation maps. We evaluate our method with the publicly available RADCURE HNC dataset, where it outperforms the corresponding single-modal models and single-label models on all survival outcomes. The generated activation maps show that the model focuses primarily on the tumor and nodal volumes when making the decision and the volume of interest varies for high- and low-risk patients. We demonstrate that the multi-label learning strategy can improve the learning efficiency and prognostic performance, while the interpretable survival prediction model is promising to help understand the decision-making process of AI and facilitate personalized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c111a556e7fc68ec3b6b23ca5e10e1c255f8bd39" target='_blank'>
              Advancing Head and Neck Cancer Survival Prediction via Multi-Label Learning and Deep Model Interpretation
              </a>
            </td>
          <td>
            Meixu Chen, Kai Wang, Jing Wang
          </td>
          <td>2024-05-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Nonlinear differential equations are encountered as models of fluid flow, spiking neurons, and many other systems of interest in the real world. Common features of these systems are that their behaviors are difficult to describe exactly and invariably unmodeled dynamics present challenges in making precise predictions. In many cases the models exhibit extremely complicated behavior due to bifurcations and chaotic regimes. In this paper, we present a novel data-driven linear estimator that uses Koopman operator theory to extract finite-dimensional representations of complex nonlinear systems. The extracted model is used together with a deep reinforcement learning network that learns the optimal stepwise actions to predict future states of the original nonlinear system. Our estimator is also adaptive to a diffeomorphic transformation of the nonlinear system which enables transfer learning to compute state estimates of the transformed system without relearning from scratch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253489dec37ed05e844568d9dae4237b151b936f" target='_blank'>
              Koopman-based Deep Learning for Nonlinear System Estimation
              </a>
            </td>
          <td>
            Zexin Sun, Mingyu Chen, John Baillieul
          </td>
          <td>2024-05-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Techniques to autonomously drive research have been prominent in Computational Scientific Discovery, while Synthetic Biology is a field of science that focuses on designing and constructing new biological systems for useful purposes. Here we seek to apply logic-based machine learning techniques to facilitate cellular engineering and drive biological discovery. Comprehensive databases of metabolic processes called genome-scale metabolic network models (GEMs) are often used to evaluate cellular engineering strategies to optimise target compound production. However, predicted host behaviours are not always correctly described by GEMs, often due to errors in the models. The task of learning the intricate genetic interactions within GEMs presents computational and empirical challenges. To address these, we describe a novel approach called Boolean Matrix Logic Programming (BMLP) by leveraging boolean matrices to evaluate large logic programs. We introduce a new system, $BMLP_{active}$, which efficiently explores the genomic hypothesis space by guiding informative experimentation through active learning. In contrast to sub-symbolic methods, $BMLP_{active}$ encodes a state-of-the-art GEM of a widely accepted bacterial host in an interpretable and logical representation using datalog logic programs. Notably, $BMLP_{active}$ can successfully learn the interaction between a gene pair with fewer training examples than random experimentation, overcoming the increase in experimental design space. $BMLP_{active}$ enables rapid optimisation of metabolic models to reliably engineer biological systems for producing useful compounds. It offers a realistic approach to creating a self-driving lab for microbial engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4217fa5bc20619984d78d30c100e2b19faa0091" target='_blank'>
              Boolean matrix logic programming for active learning of gene functions in genome-scale metabolic network models
              </a>
            </td>
          <td>
            L. Ai, Stephen Muggleton, Shishun Liang, Geoff S. Baldwin
          </td>
          <td>2024-05-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Investigating reconstructive phase transitions in large-sized systems requires a highly efficient computational framework with computational cost proportional to the system size. Traditionally, widely used frameworks such as density functional theory (DFT) have been prohibitively expensive for extensive simulations on large systems that require long-time scales. To address this challenge, this study employed well-trained machine learning potential to simulate phase transitions in a large-size system. This work integrates the metadynamics simulation approach with machine learning potential, specifically deep potential, to enhance computational efficiency and accelerate the study of phase transition and consequent development of grains and dislocation defects in a system. The new method is demonstrated using the phase transitions of bulk silicon under high pressure. This approach has revealed the transition path and formation of polycrystalline silicon systems under specific stress conditions, demonstrating the effectiveness of deep potential-driven metadynamics simulations in gaining insights into complex material behaviors in large-sized systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182e0e700457a67c0ed1191b6833eaba4ec2a3fb" target='_blank'>
              Phase Transition in Silicon from Machine Learning Informed Metadynamics.
              </a>
            </td>
          <td>
            Mangladeep Bhullar, Zihao Bai, Akinwumi Akinpelu, Yansun Yao
          </td>
          <td>2024-04-22</td>
          <td>Chemphyschem : a European journal of chemical physics and physical chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Our paper deals with a collection of networks on a common set of nodes, where some of the networks are associated with responses. Assuming that the networks correspond to points on a one-dimensional manifold in a higher dimensional ambient space, we propose an algorithm to consistently predict the response at an unlabeled network. Our model involves a specific multiple random network model, namely the common subspace independent edge model, where the networks share a common invariant subspace, and the heterogeneity amongst the networks is captured by a set of low dimensional matrices. Our algorithm estimates these low dimensional matrices that capture the heterogeneity of the networks, learns the underlying manifold by isomap, and consistently predicts the response at an unlabeled network. We provide theoretical justifications for the use of our algorithm, validated by numerical simulations. Finally, we demonstrate the use of our algorithm on larval Drosophila connectome data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb592d6fdf3ce2b11fe06b667a08556edadacce2" target='_blank'>
              Consistent response prediction for multilayer networks on unknown manifolds
              </a>
            </td>
          <td>
            Aranyak Acharyya, Jes'us Arroyo Reli'on, Michael Clayton, Marta Zlatic, Youngser Park, Carey E. Priebe
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Deep learning, a crucial technique for achieving artificial intelligence (AI), has been successfully applied in many fields. The gradual application of the latest architectures of deep learning in the field of time series forecasting (TSF), such as Transformers, has shown excellent performance and results compared to traditional statistical methods. These applications are widely present in academia and in our daily lives, covering many areas including forecasting electricity consumption in power systems, meteorological rainfall, traffic flow, quantitative trading, risk control in finance, sales operations and price predictions for commercial companies, and pandemic prediction in the medical field. Deep learning-based TSF tasks stand out as one of the most valuable AI scenarios for research, playing an important role in explaining complex real-world phenomena. However, deep learning models still face challenges: they need to deal with the challenge of large-scale data in the information age, achieve longer forecasting ranges, reduce excessively high computational complexity, etc. Therefore, novel methods and more effective solutions are essential. In this paper, we review the latest developments in deep learning for TSF. We begin by introducing the recent development trends in the field of TSF and then propose a new taxonomy from the perspective of deep neural network models, comprehensively covering articles published over the past five years. We also organize commonly used experimental evaluation metrics and datasets. Finally, we point out current issues with the existing solutions and suggest promising future directions in the field of deep learning combined with TSF. This paper is the most comprehensive review related to TSF in recent years and will provide a detailed index for researchers in this field and those who are just starting out.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d93c9854b0ec15d7272319e91ee9c2b294327e49" target='_blank'>
              Deep Time Series Forecasting Models: A Comprehensive Survey
              </a>
            </td>
          <td>
            Xinhe Liu, Wenmin Wang
          </td>
          <td>2024-05-11</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Complex systems are prevalent in various disciplines encompassing the natural and social sciences, such as physics, biology, economics, and sociology. Leveraging data science techniques, particularly those rooted in artificial intelligence and machine learning, offers a promising avenue for comprehending the intricacies of complex systems without necessitating detailed knowledge of underlying dynamics. In this paper, we demonstrate that multiscale entropy (MSE) is pivotal in describing the steady state of complex systems. Introducing the multiscale entropy dynamics (MED) methodology, we provide a framework for dissecting system dynamics and uncovering the driving forces behind their evolution. Our investigation reveals that the MED methodology facilitates the expression of complex system dynamics through a Generalized Nonlinear Schrödinger Equation (GNSE) that thus demonstrates its potential applicability across diverse complex systems. By elucidating the entropic underpinnings of complexity, our study paves the way for a deeper understanding of dynamic phenomena. It offers insights into the behavior of complex systems across various domains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7dc6f13dbf0858892b1c74138d4646a809e932e" target='_blank'>
              Major Role of Multiscale Entropy Evolution in Complex Systems and Data Science
              </a>
            </td>
          <td>
            Shahid Nawaz, Muhammad Saleem, Fedor V. Kusmartsev, D. Anjum
          </td>
          <td>2024-04-01</td>
          <td>Entropy</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Patch-seq yields multi-modal data (e.g., transcriptomic and electrophysiological data) from single cells. However, currently used analytical methods are based on a few global electrophysiological features predefined from chronological potential changes and overlook the importance of time-frequency domain. In this study, we present LincSpectr, a deep neural network model that integrates transcriptomic and electrophysiological features with the latent variables of various variational autoencoders. This model combines the two modalities according to the connection between the latent variables of different modalities calculated by attention-like mechanisms and achieves cross-modal predictions and an inverse analysis. We discovered that the predicted electrophysiological features changed continuously along with their transcriptional profiles and that the neighborhood relationships between the latent states of the transcriptional profiles were consistent with those of the electrophysiological features. Inverse analysis of our model enabled the extraction of gene sets affecting specific time-frequency domains; some genes were likely to be involved in neural activity. Our approach represents a potential avenue to facilitate the discovery of molecular mechanisms underlying time-frequency activities in various cell types, enhancing our understanding of their roles in neural function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33e2497b1e15298450dc20300cf805fcf6d4e545" target='_blank'>
              A deep generative model integrating single-cell time-frequency characteristics transformed from electrophysiological data with transcriptomic features
              </a>
            </td>
          <td>
            Kazuki Furumichi, Yasuhiro Kojima, Satoshi Nomura, Teppei Shimamura
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The recently introduced class of architectures known as Neural Operators has emerged as highly versatile tools applicable to a wide range of tasks in the field of Scientific Machine Learning (SciML), including data representation and forecasting. In this study, we investigate the capabilities of Neural Implicit Flow (NIF), a recently developed mesh-agnostic neural operator, for representing the latent dynamics of canonical systems such as the Kuramoto-Sivashinsky (KS), forced Korteweg-de Vries (fKdV), and Sine-Gordon (SG) equations, as well as for extracting dynamically relevant information from them. Finally we assess the applicability of NIF as a dimensionality reduction algorithm and conduct a comparative analysis with another widely recognized family of neural operators, known as Deep Operator Networks (DeepONets).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2ea0dbc681d0e289d33f89b612d08a007f2fc6" target='_blank'>
              Using Neural Implicit Flow To Represent Latent Dynamics Of Canonical Systems
              </a>
            </td>
          <td>
            Imran Nasim, Joao Lucas de Sousa Almeida
          </td>
          <td>2024-04-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Toxicity is a roadblock that prevents an inordinate number of drugs from being used in potentially life-saving applications. Deep learning provides a promising solution to finding ideal drug candidates; however, the vastness of chemical space coupled with the underlying $\mathcal{O}(n^3)$ matrix multiplication means these efforts quickly become computationally demanding. To remedy this, we present a hybrid quantum-classical neural network for predicting drug toxicity, utilizing a quantum circuit design that mimics classical neural behavior by explicitly calculating matrix products with complexity $\mathcal{O}(n^2)$. Leveraging the Hadamard test for efficient inner product estimation rather than the conventionally used swap test, we reduce the number qubits by half and remove the need for quantum phase estimation. Directly computing matrix products quantum mechanically allows for learnable weights to be transferred from a quantum to a classical device for further training. We apply our framework to the Tox21 dataset and show that it achieves commensurate predictive accuracy to the model's fully classical $\mathcal{O}(n^3)$ analog. Additionally, we demonstrate the model continues to learn, without disruption, once transferred to a fully classical architecture. We believe combining the quantum advantage of reduced complexity and the classical advantage of noise-free calculation will pave the way to more scalable machine learning models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e28fe708f0180a7608fc6e4b387363cc0b6b7d1" target='_blank'>
              Quantum to Classical Neural Network Transfer Learning Applied to Drug Toxicity Prediction
              </a>
            </td>
          <td>
            Anthony M. Smaldone, Victor S. Batista
          </td>
          <td>2024-03-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/004f7e93481418400ef26a11ce5ab16395e458f1" target='_blank'>
              Machine learning designs new GCGR/GLP-1R dual agonists with enhanced biological potency.
              </a>
            </td>
          <td>
            Anna M Puszkarska, Bruck Taddese, Jefferson Revell, Graeme Davies, Joss Field, David C Hornigold, Andrew Buchanan, Tristan J Vaughan, Lucy J Colwell
          </td>
          <td>2024-05-16</td>
          <td>Nature chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9345f2f030cb7979426cd2be18c7e1a1fbe44b" target='_blank'>
              Improved nonparametric survival prediction using CoxPH, Random Survival Forest & DeepHit Neural Network
              </a>
            </td>
          <td>
            Naseem Asghar, U. Khalil, Basheer Ahmad, H. Alshanbari, Muhammad Hamraz, Bakhtiyar Ahmad, Dost Muhammad Khan
          </td>
          <td>2024-05-07</td>
          <td>BMC medical informatics and decision making</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Contemporary radiation risk assessment predominantly depends on nonlinear parametric models, which typically include a baseline term, a dose-response term, and an effect modifier term. Despite their widespread application in estimating tumor risks, parametric models face a notable drawback: their rigid model structure can be overly restrictive, potentially introducing bias and inaccuracies into risk estimations. In this study, we analyze data on solid tumors and leukemia from the Life Span Study (LSS) to compare the performance of deep neural network (DNN) and nonlinear parametric (NLP) models in assessing ERRs. DNN presents novel perspectives for radiation risk assessment. Our findings indicate that DNN can perform better than the traditional parametric models. Even if DNN and NLP models exhibit similar performance in predicting tumor incidence, they diverge significantly in their estimated ERRs. Standard NLP models tend to underestimate ERRs directly linked to radiation dose, overestimate ERRs for individuals at younger attained ages and ages at exposure, and underestimate ERRs for those at older attained ages. Furthermore, DNN consistently identifies radiation dose as the primary and predominant risk factor for ERRs in leukemia and solid tumors, underscoring the critical role of radiation dose in risk assessment. The insights from DNN could enhance low-dose radiation risk assessment and improve parametric model development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c35286a4ae224457c64400796ef637896de940" target='_blank'>
              Novel Insights for Radiation Risk Assessment Unveiled by Deep Learning
              </a>
            </td>
          <td>
            Z. Liu, I. Shuryak, D. J. Brenner, R. L. Ullrich
          </td>
          <td>2024-04-30</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Environmental toxicants overwhelmingly occur together as mixtures. The variety of possible chemical interactions makes it difficult to predict the danger of the mixture. In this work, we propose the novel Reflected Generalized Concentration Addition (RGCA), a piece-wise, geometric technique for sigmoidal dose-responsed inverse functions that extends the use of generalized concentration addition (GCA) for 3+ parameter models. Since experimental tests of all relevant mixtures is costly and intractable, we rely only on the individual chemical dose responses. Additionally, RGCA enhances the classical two-step model for the cumulative effects of mixtures, which assumes a combination of GCA and independent action (IA). We explore how various clustering methods can dramatically improve predictions. We compare our technique to the IA, CA, and GCA models and show in a simulation study that the two-step approach performs well under a variety of true models. We then apply our method to a challenging data set of individual chemical and mixture responses where the target is an androgen receptor (Tox21 AR-luc). Our results show significantly improved predictions for larger mixtures. Our work complements ongoing efforts to predict environmental exposure to various chemicals and offers a starting point for combining different exposure predictions to quantify a total risk to health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fad5ac9245e8b973315ac894a90e08107c6462f" target='_blank'>
              Reflected generalized concentration addition and Bayesian hierarchical models to improve chemical mixture prediction
              </a>
            </td>
          <td>
            Daniel Zilber, Kyle Messier
          </td>
          <td>2024-03-28</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Understanding drug residence times in target proteins is key to improving drug efficacy and understanding target recognition in biochemistry. While drug residence time is just as important as binding affinity, atomiclevel understanding of drug residence times through molecular dynamics (MD) simulations has been difficult primarily due to the extremely long timescales. Recent advances in rare event sampling have allowed us to reach these timescales, yet predicting protein-ligand residence times remains a significant challenge. Here we present a semi-automated protocol to calculate the ligand residence times across 12 orders of magnitudes of timescales. In our proposed framework, we integrate a deep learning-based method, the state predictive information bottleneck (SPIB), to learn an approximate reaction coordinate (RC) and use it to guide the enhanced sampling method metadynamics. We demonstrate the performance of our algorithm by applying it to six different protein-ligand complexes with available benchmark residence times, including the dissociation of the widely studied anti-cancer drug Imatinib (Gleevec) from both wild-type Abl kinase and drug-resistant mutants. We show how our protocol can recover quantitatively accurate residence times, potentially opening avenues for deeper insights into drug development possibilities and ligand recognition mechanisms. TOC Graphic">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d9e0ffe0d154c41dd1902b028393745d33c781" target='_blank'>
              Calculating Protein-Ligand Residence Times Through State Predictive Information Bottleneck based Enhanced Sampling
              </a>
            </td>
          <td>
            Suemin Lee, Dedi Wang, Markus A. Seeliger, P. Tiwary
          </td>
          <td>2024-04-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background. Individualized treatment decisions for patients with multiple myeloma (MM) requires accurate risk stratification that takes into account patient-specific consequences of genetic abnormalities and tumor microenvironment on disease outcome and therapy responsiveness. Methods. Previously, SYstems Genetic Network AnaLysis (SYGNAL) of multi-omics tumor profiles from 881 MM patients generated the mmSYGNAL network, which uncovered different causal and mechanistic drivers of genetic programs associated with disease progression across MM subtypes. Here, we have trained a machine learning (ML) algorithm on activities of mmSYGNAL programs within individual patient tumor samples to develop a risk classification scheme for MM that significantly outperformed cytogenetics, International Staging System, and multi-gene biomarker panels in predicting risk of PFS across four independent patient cohorts. Results. We demonstrate that, unlike other tests, mmSYGNAL can accurately predict disease progression risk at primary diagnosis, pre- and post-transplant and even after multiple relapses, making it useful for individualized dynamic risk assessment throughout the disease trajectory. Conclusion. mmSYGNAL provides improved individualized risk stratification that accounts for a patient's distinct set of genetic abnormalities and can monitor risk longitudinally as each patient's disease characteristics change.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cba3f9524bd02b07c52db6fe11f71cc70c07bfd" target='_blank'>
              Individualized dynamic risk assessment for multiple myeloma
              </a>
            </td>
          <td>
            C. Murie, S. Turkarslan, Anoop Patel, David G. Coffey, Pamela S. Becker, N. Baliga
          </td>
          <td>2024-04-03</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Explaining deep learning models operating on time series data is crucial in various applications of interest which require interpretable and transparent insights from time series signals. In this work, we investigate this problem from an information theoretic perspective and show that most existing measures of explainability may suffer from trivial solutions and distributional shift issues. To address these issues, we introduce a simple yet practical objective function for time series explainable learning. The design of the objective function builds upon the principle of information bottleneck (IB), and modifies the IB objective function to avoid trivial solutions and distributional shift issues. We further present TimeX++, a novel explanation framework that leverages a parametric network to produce explanation-embedded instances that are both in-distributed and label-preserving. We evaluate TimeX++ on both synthetic and real-world datasets comparing its performance against leading baselines, and validate its practical efficacy through case studies in a real-world environmental application. Quantitative and qualitative evaluations show that TimeX++ outperforms baselines across all datasets, demonstrating a substantial improvement in explanation quality for time series data. The source code is available at \url{https://github.com/zichuan-liu/TimeXplusplus}.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41331316e9b6cf30546793e0963a8a78d8691c86" target='_blank'>
              TimeX++: Learning Time-Series Explanations with Information Bottleneck
              </a>
            </td>
          <td>
            Zichuan Liu, Tianchun Wang, Jimeng Shi, Xu Zheng, Zhuomin Chen, Lei Song, Wenqian Dong, J. Obeysekera, Farhad Shirani, Dongsheng Luo
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="The majority of the results on modeling recurrent neural networks (RNNs) are obtained using delayed differential equations, which imply continuous time representation. On the other hand, these models must be discrete in time, given their practical implementation in computer systems, requiring their versatile utilization across arbitrary time scales. Hence, the goal of this research is to model and investigate the architecture design of a delayed RNN using delayed differential equations on a time scale. Internal memory can be utilized to describe the calculation of the future states using discrete and distributed delays, which is a representation of the deep learning architecture for artificial RNNs. We focus on qualitative behavior and stability study of the system. Special attention is paid to taking into account the effect of the time-scale parameters on neural network dynamics. Here, we delve into the exploration of exponential stability in RNN models on a time scale that incorporates multiple discrete and distributed delays. Two approaches for constructing exponential estimates, including the Hilger and the usual exponential functions, are considered and compared. The Lyapunov-Krasovskii (L-K) functional method is employed to study stability on a time scale in both cases. The established stability criteria, resulting in an exponential-like estimate, utilizes a tuple of positive definite matrices, decay rate, and graininess of the time scale. The models of RNNs for the two-neuron network with four discrete and distributed delays, as well as the ring lattice delayed network of seven identical neurons, are numerically investigated. The results indicate how the time scale (graininess) and model characteristics (weights) influence the qualitative behavior, leading to a transition from stable focus to quasiperiodic limit cycles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27b4148511b5000c41b92473f35e7fc1cd669169" target='_blank'>
              On Model of Recurrent Neural Network on a Time Scale: Exponential Convergence and Stability Research.
              </a>
            </td>
          <td>
            V. Martsenyuk, Marcin Bernas, A. Kłos-Witkowska
          </td>
          <td>2024-03-26</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Over the past few years, machine learning potentials (MLPs) have been fully developed and can now be applied to a variety of large-scale atomic simulations. MLPs encompass a number of different machine learning algorithms, of which neural network potentials are the most dominant and representative, and have achieved great success. This review focuses on the second generation of neural network potentials, high-dimensional neural network potentials (HDNNPs). While HDNNP has accomplished noteworthy results in the implementation of ion diffusion, nuclear magnetic resonance parameters, defect formation, and thermal conductivity, it still encounters hurdles in the compilation of training datasets, transferability, accuracy constraints, and the management of multiple chemical species. In spite of this, HDNNP remains a highly promising methodology. Looking forward, we anticipate the development of more specialized software for HDNNP to augment research efficiency and lower the barrier to usage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38626e547c688d77146228989a84f95e74168f15" target='_blank'>
              Review of High Dimensional Neural Network Potentials
              </a>
            </td>
          <td>
            Yichen Zou
          </td>
          <td>2024-03-27</td>
          <td>Applied and Computational Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Allostery is a fundamental mechanism driving biomolecular processes that holds significant therapeutic concern. Our study rigorously investigates how two distinct machine-learning algorithms uniquely classify two already close-to-active DFG-in states of TAK1, differing just by the presence or absence of its allosteric activator TAB1, from an ensemble mixture of conformations (obtained from 2.4 μs molecular dynamics (MD) simulations). The novelty, however, lies in understanding the deeper algorithmic potentials to systematically derive a diverse set of differential residue connectivity features that reconstruct the essential mechanistic architecture for TAK1-TAB1 allostery in such a close-to-active biochemical scenario. While the recursive, random forest-based workflow displays the potential of conducting discretized, hierarchical derivation of allosteric features, a multilayer perceptron-based approach gains considerable efficacy in revealing fluid connected patterns of features when hybridized with mutual information scoring. Interestingly, both pipelines benchmark similar directions of functional conformational changes for TAK1's activation. The findings significantly advance the depth of mechanistic understanding by highlighting crucial activation signatures along a directed C-lobe → activation loop → ATP pocket channel of information flow, including (1) the αF-αE biterminal alignments and (2) the "catalytic" drift of the activation loop toward kinase active site. Besides, some novel allosteric hotspots (K253, Y206, N189, etc.) are further recognized as TAB1 sensors, transducers, and responders, including a benchmark E70 mutation site, precisely mapping the important structural segments for sequential allosteric execution. Hence, our work demonstrates how to navigate through greater structural depths and dimensions of dynamic allosteric machineries just by leveraging standard ML methods in suitable streamlined workflows adaptive to the specific system and objectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e656737754f86ab9ba75a485691b12e7213422c" target='_blank'>
              Adaptive Workflows of Machine Learning Illuminate the Sequential Operation Mechanism of the TAK1's Allosteric Network.
              </a>
            </td>
          <td>
            Nibedita Ray Chaudhuri, Shubhra Ghosh Dastidar
          </td>
          <td>2024-05-14</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Single-cell RNA sequencing (scRNA-seq) experiments have become instrumental in developmental and differentiation studies, enabling the profiling of cells at a single or multiple time-points to uncover subtle variations in expression profiles reflecting underlying biological processes. Benchmarking studies have compared many of the computational methods used to reconstruct cellular dynamics; however, researchers still encounter challenges in their analysis due to uncertainty with respect to selecting the most appropriate methods and parameters. Even among universal data processing steps used by trajectory inference methods such as feature selection and dimension reduction, trajectory methods’ performances are highly dataset-specific. To address these challenges, we developed Escort, a novel framework for evaluating a dataset’s suitability for trajectory inference and quantifying trajectory properties influenced by analysis decisions. Escort evaluates the suitability of trajectory analysis and the combined effects of processing choices using trajectory-specific metrics. Escort navigates single-cell trajectory analysis through these data-driven assessments, reducing uncertainty and much of the decision burden inherent to trajectory inference analyses. Escort is implemented in an accessible R package and R/Shiny application, providing researchers with the necessary tools to make informed decisions during trajectory analysis and enabling new insights into dynamic biological processes at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7ae2ec3bd17e2ad419e1f1cd13b5fd93a8fcb7" target='_blank'>
              Data-driven selection of analysis decisions in single-cell RNA-seq trajectory inference
              </a>
            </td>
          <td>
            Xiaoru Dong, Jack R. Leary, Chuanhao Yang, Maigan A. Brusko, Todd M. Brusko, Rhonda L. Bacher
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Computational models that predict pharmacokinetic properties are critical to deprioritize drug candidates that emerge as hits in high-throughput screening campaigns. We collected, curated, and integrated a database of compounds tested in 12 major end points comprising over 10,000 unique molecules. We then employed these data to build and validate binary quantitative structure-activity relationship (QSAR) models. All trained models achieved a correct classification rate above 0.60 and a positive predictive value above 0.50. To illustrate their utility in drug discovery, we used these models to predict the pharmacokinetic properties for drugs in the NCATS Inxight Drugs database. In addition, we employed the developed models to predict the pharmacokinetic properties of all compounds in the DrugBank. All models described in this paper have been integrated and made publicly available via the PhaKinPro Web-portal that can be accessed at https://phakinpro.mml.unc.edu/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34d352c950ec6cbe9bd9aea7f1b7f841220cb6af" target='_blank'>
              Pharmacokinetics Profiler (PhaKinPro): Model Development, Validation, and Implementation as a Web Tool for Triaging Compounds with Undesired Pharmacokinetics Profiles.
              </a>
            </td>
          <td>
            Marielle Rath, James Wellnitz, H. Martin, C. Melo-Filho, Joshua E. Hochuli, Guilherme Martins Silva, Jon-Michael Beasley, Maxfield Travis, Zoe L. Sessions, Konstantin I. Popov, Alexey V Zakharov, Artem Cherkasov, Vinicius Alves, E. Muratov, Alexander Tropsha
          </td>
          <td>2024-04-03</td>
          <td>Journal of medicinal chemistry</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Many complicated dynamical events may be broken down into simpler pieces and efficiently described by a system that shifts among a variety of conditionally dynamical modes. Building on switching linear dynamical systems, we develop a new model that extends the switching linear dynamical systems for better discovering these dynamical modes. In the proposed model, the linear dynamics of latent variables can be described by a higher-order vector autoregressive process, which makes it feasible to evaluate the higher-order dependency relationships in the dynamics. In addition, the transition of switching states is determined by a stick-breaking logistic regression, overcoming the limitation of a restricted geometric state duration and recovering the symmetric dependency between the switching states and the latent variables from asymmetric relationships. Furthermore, logistic regression evidence potentials can appear as conditionally Gaussian potentials by utilizing the Pólya-gamma augmentation strategy. Filtering and smoothing algorithms and Bayesian inference for parameter learning in the proposed model are presented. The utility and versatility of the proposed model are demonstrated on synthetic data and public functional magnetic resonance imaging data. Our model improves the current methods for learning the switching linear dynamical modes, which will facilitate the identification and assessment of the dynamics of complex systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f180f44aa0cab4e14882a44aef54b3e4ae757688" target='_blank'>
              Bayesian Inference of Recurrent Switching Linear Dynamical Systems with Higher-Order Dependence
              </a>
            </td>
          <td>
            Houxiang Wang, Jiaqing Chen
          </td>
          <td>2024-04-13</td>
          <td>Symmetry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Temporal network embedding (TNE) has promoted the research of knowledge discovery and reasoning on networks. It aims to embed vertices of temporal networks into a low-dimensional vector space while preserving network structures and temporal properties. However, most existing methods have limitations in capturing dynamics over long distances, which makes it difficult to explore multihop topological associations among vertices. To tackle this challenge, we propose LongTNE, which learns the long-range dynamics of vertices to endow TNE with the ability to capture high-order proximity (HP) of networks. In LongTNE, we employ graph self-supervised learning (Graph SSL) to optimize the establishment probability of deep links in each network snapshot. We also present an accumulated forward update (AFU) module to fathom global temporal evolution among multiple network snapshots. The empirical results on six temporal networks demonstrate that, in addition to achieving state-of-the-art performance on network mining tasks, LongTNE can be handily extended to existing TNE methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b599578f0b1fd437c10657cbaf6ef590fc7b067" target='_blank'>
              Temporal Network Embedding Enhanced With Long-Range Dynamics and Self-Supervised Learning.
              </a>
            </td>
          <td>
            Zhizheng Wang, Yuanyuan Sun, Zhihao Yang, Liang Yang, Hongfei Lin
          </td>
          <td>2024-04-15</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Machine learning has the potential to provide tremendous value to life sciences by providing models that aid in the discovery of new molecules and reduce the time for new products to come to market. Chemical reactions play a significant role in these fields, but there is a lack of high-quality open-source chemical reaction data sets for training machine learning models. Herein, we present ORDerly, an open-source Python package for the customizable and reproducible preparation of reaction data stored in accordance with the increasingly popular Open Reaction Database (ORD) schema. We use ORDerly to clean United States patent data stored in ORD and generate data sets for forward prediction, retrosynthesis, as well as the first benchmark for reaction condition prediction. We train neural networks on data sets generated with ORDerly for condition prediction and show that data sets missing key cleaning steps can lead to silently overinflated performance metrics. Additionally, we train transformers for forward and retrosynthesis prediction and demonstrate how non-patent data can be used to evaluate model generalization. By providing a customizable open-source solution for cleaning and preparing large chemical reaction data, ORDerly is poised to push forward the boundaries of machine learning applications in chemistry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c0ae3528ac06ca9bafc03b00c72f1cc95f14db" target='_blank'>
              ORDerly: Data Sets and Benchmarks for Chemical Reaction Data
              </a>
            </td>
          <td>
            Daniel S. Wigh, Joe Arrowsmith, Alexander Pomberger, Kobi C. Felton, A. Lapkin
          </td>
          <td>2024-04-22</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>2</td>
          <td>45</td>
        </tr>

        <tr id="PURPOSE
Patients diagnosed with advanced-stage Hodgkin lymphoma (aHL) have historically been risk-stratified using the International Prognostic Score (IPS). This study investigated if a machine learning (ML) approach could outperform existing models when it comes to predicting overall survival (OS) and progression-free survival (PFS).


PATIENTS AND METHODS
This study used patient data from the Danish National Lymphoma Register for model development (development cohort). The ML model was developed using stacking, which combines several predictive survival models (Cox proportional hazard, flexible parametric model, IPS, principal component, penalized regression) into a single model, and was compared with two versions of IPS (IPS-3 and IPS-7) and the newly developed aHL international prognostic index (A-HIPI). Internal model validation was performed using nested cross-validation, and external validation was performed using patient data from the Swedish Lymphoma Register and Cancer Registry of Norway (validation cohort).


RESULTS
In total, 707 and 760 patients with aHL were included in the development and validation cohorts, respectively. Examining model performance for OS in the development cohort, the concordance index (C-index) for the ML model, IPS-7, IPS-3, and A-HIPI was found to be 0.789, 0.608, 0.650, and 0.768, respectively. The corresponding estimates in the validation cohort were 0.749, 0.700, 0.663, and 0.741. For PFS, the ML model achieved the highest C-index in both cohorts (0.665 in the development cohort and 0.691 in the validation cohort). The time-varying AUCs for both the ML model and the A-HIPI were consistently higher in both cohorts compared with the IPS models within the first 5 years after diagnosis.


CONCLUSION
The new prognostic model for aHL on the basis of ML techniques demonstrated a substantial improvement compared with the IPS models, but yielded a limited improvement in predictive performance compared with the A-HIPI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4b607c53708459e2e40ab5a6b05c3254ba360a" target='_blank'>
              Machine Learning-Based Survival Prediction Models for Progression-Free and Overall Survival in Advanced-Stage Hodgkin Lymphoma.
              </a>
            </td>
          <td>
            R. Rask Kragh Jørgensen, Fanny Bergström, S. Eloranta, M. Tang Severinsen, K. Bjøro Smeland, Alexander Fosså, J. Haaber Christensen, Martin Hutchings, Rasmus Bo Dahl-Sørensen, P. Kamper, I. Glimelius, Karin E Smedby, Susan K Parsons, Angie Mae Rodday, Matthew J Maurer, Andrew M Evens, Tarec C El-Galaly, L. Hjort Jakobsen
          </td>
          <td>2024-04-01</td>
          <td>JCO clinical cancer informatics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Advancements in wearable sensor technologies and the digitization of medical records have contributed to the unprecedented ubiquity of biomedical time series data. Data-driven models have tremendous potential to assist clinical diagnosis and improve patient care by improving long-term monitoring capabilities, facilitating early disease detection and intervention, as well as promoting personalized healthcare delivery. However, accessing extensively labeled datasets to train data-hungry deep learning models encounters many barriers, such as long-tail distribution of rare diseases, cost of annotation, privacy and security concerns, data-sharing regulations, and ethical considerations. An emerging approach to overcome the scarcity of labeled data is to augment AI methods with human-like capabilities to leverage past experiences to learn new tasks with limited examples, called few-shot learning. This survey provides a comprehensive review and comparison of few-shot learning methods for biomedical time series applications. The clinical benefits and limitations of such methods are discussed in relation to traditional data-driven approaches. This paper aims to provide insights into the current landscape of few-shot learning for biomedical time series and its implications for future research and applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35712d135083d4d22b2d56f482c5d1d32a14eb41" target='_blank'>
              A Survey of Few-Shot Learning for Biomedical Time Series
              </a>
            </td>
          <td>
            Chenqi Li, Timothy Denison, Tingting Zhu
          </td>
          <td>2024-05-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Epidermal growth factor receptor (EGFR) T790M mutation is the standard predictive biomarker for third-generation epidermal growth factor receptor tyrosine kinase inhibitor (EGFR-TKI) treatment. While not all T790M-positive patients respond to third-generation EGFR-TKIs and have a good prognosis, it necessitates novel tools to supplement EGFR genotype detection for predicting efficacy and stratifying EGFR-mutant patients with various prognoses. Mixture-of-experts (MoE) is designed to disassemble a large model into many small models. Meanwhile, it is also a model ensembling method that can better capture multiple patterns of intrinsic subgroups of enrolled patients. Therefore, the combination of MoE and Cox algorithm has the potential to predict efficacy and stratify survival in non-small cell lung cancer (NSCLC) patients with EGFR mutations. Methods We utilized the electronic medical record (EMR) and pharmacokinetic parameters of 326 T790M-mutated NSCLC patients, including 283 patients treated with Abivertinib in phase I (n=177, for training) and II (n=106, for validation) clinical trials and an additional validation cohort 2 comprising 43 patients treated with BPI-7711. Furthermore, 18 patients underwent whole-exome sequencing for biological interpretation of CoxMoE. We evaluated the predictive performance for therapeutic response using the area under the curve (AUC) and the Concordance index (C-index) for progression-free survival (PFS). Results CoxMoE exhibited AUCs of 0.73–0.83 for predicting efficacy defined by best overall response (BoR) and achieved C-index values of 0.64–0.65 for PFS prediction in training and validating cohorts. The PFS of 198 patients with a low risk [median, 6.0 (range, 1.0–23.3) months in the abivertinib treated cohort; median 16.5 (range, 1.4–27.4) months in BPI-7711 treated cohort] of being non-responder increased by 43% [hazard ratio (HR), 0.56; 95% confidence interval (CI), 0.40–0.78; P=0.0013] and 50% (HR, 0; 95% CI, 0–0; P=0.01) compared to those at high-risk [median, 4.2 (range, 1.0–35) months in the abivertinib treated cohort; median, 11.0 (range, 1.4–25.1) months in BPI-7711 treated cohort]. Additionally, activated partial thromboplastin time (APTT), creatinine clearance (Ccr), monocyte, and steady-state plasma trough concentration utilited to construct model were found significantly associated with drug resistance and aggressive tumor pathways. A robust correlation was observed between APTT and Ccr with PFS (log-rank test; P<0.01) and treatment response (Wilcoxon test; P<0.05), respectively. Conclusions CoxMoE offers a valuable approach for patient selection by forecasting therapeutic response and PFS utilizing laboratory tests and pharmacokinetic parameters in the setting of early-phase clinical trials. Simultaneously, CoxMoE could predict the efficacy of third-generation EGFR-TKI non-invasively for T790M-positive NSCLC patients, thereby complementing existing EGFR genotype detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/971519ff9b0ca1f419778698dab825d127135d1c" target='_blank'>
              Development and validation of a deep learning-based model to predict response and survival of T790M mutant non-small cell lung cancer patients in early clinical phase trials using electronic medical record and pharmacokinetic data
              </a>
            </td>
          <td>
            N. Lou, X. Cui, Xinyuan Lin, Ruyun Gao, Chi Xu, N. Qiao, Ji Jiang, Lu Wang, Weicong Wang, Shanbo Wang, Wei Shen, Xin Zheng, Xiaohong Han
          </td>
          <td>2024-04-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background This study aims to identify precise biomarkers for breast cancer to improve patient outcomes, addressing the limitations of traditional staging in predicting treatment responses. Methods Our analysis encompassed data from over 7,000 breast cancer patients across 14 datasets, which included in-house clinical data and single-cell data from 8 patients (totaling 43,766 cells). We utilized an integrative approach, applying 10 machine learning algorithms in 54 unique combinations to analyze 100 existing breast cancer signatures. Immunohistochemistry assays were performed for empirical validation. The study also investigated potential immunotherapies and chemotherapies. Results Our research identified five consistent glutamine metabolic reprogramming (GMR)-related genes from multi-center cohorts, forming the foundation of a novel GMR-model. This model demonstrated superior accuracy in predicting recurrence and mortality risks compared to existing clinical and molecular features. Patients classified as high-risk by the model exhibited poorer outcomes. IHC validation in 30 patients reinforced these findings, suggesting the model’s broad applicability. Intriguingly, the model indicates a differential therapeutic response: low-risk patients may benefit more from immunotherapy, whereas high-risk patients showed sensitivity to specific chemotherapies like BI-2536 and ispinesib. Conclusions The GMR-model marks a significant leap forward in breast cancer prognosis and the personalization of treatment strategies, offering vital insights for the effective management of diverse breast cancer patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f45d05c5e2b2d87e7018b79a4ae4331c6d799edc" target='_blank'>
              Enhancing breast cancer outcomes with machine learning-driven glutamine metabolic reprogramming signature
              </a>
            </td>
          <td>
            Xukui Li, Xue Li, Bin Yang, Songyang Sun, Shu Wang, Fuxun Yu, Tao Wang
          </td>
          <td>2024-05-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [25],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>