<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Latent%20space%20simulators/">
      
      
        <link rel="next" href="../PINNs/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Parametrizing using ML
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Time-series%20forecasting/" class="md-tabs__link">
        
  
    
  
  Time-series forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Symbolic%20regression/" class="md-tabs__link">
        
  
    
  
  Symbolic regression

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Neural%20ODEs/" class="md-tabs__link">
        
  
    
  
  Neural ODEs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Physics-based%20GNNs/" class="md-tabs__link">
        
  
    
  
  Physics-based GNNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Latent%20space%20simulators/" class="md-tabs__link">
        
  
    
  
  Latent space simulators

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Parametrizing using ML

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../PINNs/" class="md-tabs__link">
        
  
    
  
  PINNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Koopman%20operator/" class="md-tabs__link">
        
  
    
  
  Koopman operator

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Time-series%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Time-series forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Symbolic%20regression/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Symbolic regression
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Neural%20ODEs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Neural ODEs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Physics-based%20GNNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Physics-based GNNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Latent%20space%20simulators/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Latent space simulators
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Parametrizing using ML
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../PINNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    PINNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Koopman%20operator/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Koopman operator
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Parametrizing using ML</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-07-08 06:07:24 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Parametrizing using ML</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Parametrizing using ML</a><br>
      <a href="#recommended_articles">3. Recommended articles on Parametrizing using ML</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Parametrizing using ML</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Parametrizing using ML</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e837965494c4edbec4d30832d31ba5639996da8" target='_blank'>
                Deep learning prediction of patient response time course from early data via neural-pharmacokinetic/pharmacodynamic modelling
                </a>
              </td>
          <td>
            James Lu, B. Bender, Jin Y. Jin, Y. Guan
          </td>
          <td>2020-10-22</td>
          <td>Nature Machine Intelligence</td>
          <td>34</td>
          <td>38</td>

            <td><a href='../recommendations/4e837965494c4edbec4d30832d31ba5639996da8' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Many real-world systems such as social networks and moving planets are dynamic in nature, where a set of coupled objects are connected via the interaction graph and exhibit complex behavior along the time. For example, the COVID-19 pandemic can be considered as a dynamical system, where objects represent geographical locations (e.g., states) whose daily confirmed cases of infection evolve over time. Outbreak at one location may influence another location as people travel between these locations, forming a graph. Thus, how to model and predict the complex dynamics for these systems becomes a critical research problem. Existing work on modeling graph-structured data mostly assumes a static setting. How to handle dynamic graphs remains to be further explored. On one hand, features of objects change over time, influenced by the linked objects in the interaction graph. On the other hand, the graph itself can also evolve, where new interactions (links) may form and existing links may drop, which may in turn be affected by the dynamic features of objects. In this paper, we propose coupled graph ODE: a novel latent ordinary differential equation (ODE) generative model that learns the coupled dynamics of nodes and edges with a graph neural network (GNN) based ODE in a continuous manner. Our model consists of two coupled ODE functions for modeling the dynamics of edges and nodes based on their latent representations respectively. It employs a novel encoder parameterized by a GNN for inferring the initial states from historical data, which serves as the starting point of the predicted latent trajectories. Experiment results on the COVID-19 dataset and the simulated social network dataset demonstrate the effectiveness of our proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf2145f9998f304513c0c9b530ee9f7750c6f55" target='_blank'>
                Coupled Graph ODE for Learning Interacting System Dynamics
                </a>
              </td>
          <td>
            Zijie Huang, Yizhou Sun, Wei Wang
          </td>
          <td>2021-08-14</td>
          <td>Knowledge Discovery and Data Mining, Proceedings of the 27th ACM SIGKDD Conference on Knowledge Discovery & Data Mining</td>
          <td>50</td>
          <td>57</td>

            <td><a href='../recommendations/aaf2145f9998f304513c0c9b530ee9f7750c6f55' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd28606fbae3449f831248804264c9885e992f9" target='_blank'>
                CellBox: Interpretable Machine Learning for Perturbation Biology with Application to the Design of Cancer Combination Therapy.
                </a>
              </td>
          <td>
            Bo Yuan, Ciyue Shen, Augustin Luna, Anil Korkut, D. Marks, John Ingraham, C. Sander
          </td>
          <td>2020-12-21</td>
          <td>Cell systems</td>
          <td>70</td>
          <td>153</td>

            <td><a href='../recommendations/6bd28606fbae3449f831248804264c9885e992f9' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="We introduce a flexible, scalable Bayesian inference framework for nonlinear dynamical systems characterised by distinct and hierarchical variability at the individual, group, and population levels. Our model class is a generalisation of nonlinear mixed-effects (NLME) dynamical systems, the statistical workhorse for many experimental sciences. We cast parameter inference as stochastic optimisation of an end-to-end differentiable, block-conditional variational autoencoder. We specify the dynamics of the data-generating process as an ordinary differential equation (ODE) such that both the ODE and its solver are fully differentiable. This model class is highly flexible: the ODE right-hand sides can be a mixture of user-prescribed or “white-box” sub-components and neural network or “black-box” sub-components. Using stochastic optimisation, our amortised inference algorithm could seamlessly scale up to massive data collection pipelines (common in labs with robotic automation). Finally, our framework supports interpretability with respect to the underlying dynamics, as well as predictive generalization to unseen combinations of group components (also called “zero-shot” learning). We empirically validate our method by predicting the dynamic behaviour of bacteria that were genetically engineered to function as biosensors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309c5ae93a4cabfd37747abd130866240e265b2d" target='_blank'>
                Efficient Amortised Bayesian Inference for Hierarchical and Nonlinear Dynamical Systems
                </a>
              </td>
          <td>
            Geoffrey Roeder, Paul K. Grant, Andrew Phillips, Neil Dalchau, Edward Meeds
          </td>
          <td>2019-05-24</td>
          <td>International Conference on Machine Learning, ArXiv</td>
          <td>21</td>
          <td>28</td>

            <td><a href='../recommendations/309c5ae93a4cabfd37747abd130866240e265b2d' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Parametrizing using ML'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Parametrizing using ML</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de2e8855c3ac85fb287f6356a1f811bb41e6472" target='_blank'>
              Mechanism-based organization of neural networks to emulate systems biology and pharmacology models
              </a>
            </td>
          <td>
            John Mann, Hamed Meshkin, Joel Zirkle, Xiaomei Han, Bradlee Thrasher, Anik Chaturbedi, Ghazal Arabidarrehdor, Zhihua Li
          </td>
          <td>2024-05-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="SUMMARY
Phenotype-based drug screening emerges as a powerful approach for identifying compounds that actively interact with cells. Transcriptional and proteomic profiling of cell lines and individual cells provide insights into the cellular state alterations that occur at the molecular level in response to external perturbations, such as drugs or genetic manipulations. In this paper, we propose cycleCDR, a novel deep learning framework to predict cellular response to external perturbations. We leverage the autoencoder to map the unperturbed cellular states to a latent space, in which we postulate the effects of drug perturbations on cellular states follow a linear additive model. Next, we introduce the cycle consistency constraints to ensure that unperturbed cellular state subjected to drug perturbation in the latent space would produces the perturbed cellular state through the decoder. Conversely, removal of perturbations from the perturbed cellular states can restore the unperturbed cellular state. The cycle consistency constraints and linear modeling in the latent space enable to learn transferable representations of external perturbations, so that our model can generalize well to unseen drugs during training stage. We validate our model on four different types of datasets, including bulk transcriptional responses, bulk proteomic responses, and single-cell transcriptional responses to drug/gene perturbations. The experimental results demonstrate that our model consistently outperforms existing state-of-the-art methods, indicating our method is highly versatile and applicable to a wide range of scenarios.


AVAILABILITY AND IMPLEMENTATION
The source code is available at: https://github.com/hliulab/cycleCDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1c2b23f3deb29435692d8627c8b4e0b406ec968" target='_blank'>
              Predicting single-cell cellular responses to perturbations using cycle consistency learning.
              </a>
            </td>
          <td>
            Wei Huang, Hui Liu
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multi-factor screenings are commonly used in diverse applications in medicine and bioengineering, including optimizing combination drug treatments and microbiome engineering. Despite the advances in high-throughput technologies, large-scale experiments typically remain prohibitively expensive. Here we introduce a machine learning platform, structure-augmented regression (SAR), that exploits the intrinsic structure of each biological system to learn a high-accuracy model with minimal data requirement. Under different environmental perturbations, each biological system exhibits a unique, structured phenotypic response. This structure can be learned based on limited data and once learned, can constrain subsequent quantitative predictions. We demonstrate that SAR requires significantly fewer data comparing to other existing machine-learning methods to achieve a high prediction accuracy, first on simulated data, then on experimental data of various systems and input dimensions. We then show how a learned structure can guide effective design of new experiments. Our approach has implications for predictive control of biological systems and an integration of machine learning prediction and experimental design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c960a00bc91e7ef861762c26cbaf5fb088878ab0" target='_blank'>
              Data-driven learning of structure augments quantitative prediction of biological responses.
              </a>
            </td>
          <td>
            Yuanchi Ha, Helena R. Ma, Feilun Wu, Andrea Weiss, Katherine Duncker, Helen Xu, Jia Lu, Max Golovsky, Daniel Reker, Lingchong You
          </td>
          <td>2024-06-03</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c53a64b37da477c06468da0f1b359b77e2e273e9" target='_blank'>
              Biologically informed NeuralODEs for genome-wide regulatory dynamics
              </a>
            </td>
          <td>
            Intekhab Hossain, Viola Fanfani, Jonas Fischer, John Quackenbush, R. Burkholz
          </td>
          <td>2024-05-21</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Synergistic drug combination prediction tasks based on the computational models have been widely studied and applied in the cancer field. However, most of models only consider the interactions between drug pairs and specific cell lines, without taking into account the multiple biological relationships of drug-drug and cell line-cell line that also largely affect synergistic mechanisms. To this end, here we propose a multi-modal deep learning framework, termed MDNNSyn, which adequately applies multi-source information and trains multi-modal features to infer potential synergistic drug combinations. MDNNSyn extracts topology modality features by implementing the multi-layer hypergraph neural network on drug synergy hypergraph and constructs semantic modality features through similarity strategy. A multi-modal fusion network layer with gated neural network is then employed for synergy score prediction. MDNNSyn is compared to five classic and state-of-the-art prediction methods on DrugCombDB and Oncology-Screen datasets. The model achieves area under the curve (AUC) scores of 0.8682 and 0.9013 on two datasets, an improvement of 3.70% and 2.71% over the second-best model. Case study indicates that MDNNSyn is capable of detecting potential synergistic drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc49f5c1e8bd773b6cd154ec4110f9a46b6b817" target='_blank'>
              MDNNSyn: A Multi-Modal Deep Learning Framework for Drug Synergy Prediction.
              </a>
            </td>
          <td>
            Lei Li, Haitao Li, Tseren-Onolt Ishdorj, Chunhou Zheng, Yansen Su
          </td>
          <td>2024-07-02</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Modeling complex physical dynamics is a fundamental task in science and engineering. Traditional physics-based models are first-principled, explainable, and sample-efficient. However, they often rely on strong modeling assumptions and expensive numerical integration, requiring significant computational resources and domain expertise. While deep learning (DL) provides efficient alternatives for modeling complex dynamics, they require a large amount of labeled training data. Furthermore, its predictions may disobey the governing physical laws and are difficult to interpret. Physics-guided DL aims to integrate first-principled physical knowledge into data-driven methods. It has the best of both worlds and is well equipped to better solve scientific problems. Recently, this field has gained great progress and has drawn considerable interest across discipline Here, we introduce the framework of physics-guided DL with a special emphasis on learning dynamical systems. We describe the learning pipeline and categorize state-of-the-art methods under this framework. We also offer our perspectives on the open challenges and emerging opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60d721e89c2f9c549241a4982b77f9c752b34460" target='_blank'>
              Learning dynamical systems from data: An introduction to physics-guided deep learning.
              </a>
            </td>
          <td>
            Rose Yu, Rui Wang
          </td>
          <td>2024-06-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Personalized cancer treatment requires a thorough understanding of complex interactions between drugs and cancer cell lines in varying genetic and molecular contexts. To address this, high-throughput screening has been used to generate large-scale drug response data, facilitating data-driven computational models. Such models can capture complex drug–cell line interactions across various contexts in a fully data-driven manner. However, accurately prioritizing the most effective drugs for each cell line still remains a significant challenge. To address this, we developed multiple neural ranking approaches that leverage large-scale drug response data across multiple cell lines from diverse cancer types. Unlike existing approaches that primarily utilize regression and classification techniques for drug response prediction, we formulated the objective of drug selection and prioritization as a drug ranking problem. In this work, we proposed multiple pairwise and listwise neural ranking methods that learn latent representations of drugs and cell lines and then use those representations to score drugs in each cell line via a learnable scoring function. Specifically, we developed neural pairwise and listwise ranking methods, Pair-PushC and List-One on top of the existing methods, pLETORg and ListNet, respectively. Additionally, we proposed a novel listwise ranking method, List-All, that focuses on all the effective drugs instead of the top effective drug, unlike List-One. We also provide an exhaustive empirical evaluation with state-of-the-art regression and ranking baselines on large-scale data sets across multiple experimental settings. Our results demonstrate that our proposed ranking methods mostly outperform the best baselines with significant improvements of as much as 25.6% in terms of selecting truly effective drugs within the top 20 predicted drugs (i.e., hit@20) across 50% test cell lines. Furthermore, our analyses suggest that the learned latent spaces from our proposed methods demonstrate informative clustering structures and capture relevant underlying biological features. Moreover, our comprehensive evaluation provides a thorough and objective comparison of the performance of different methods (including our proposed ones).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c0f5fb3d038f7182389a924d6a9c7f0a1db1816" target='_blank'>
              Improving Anticancer Drug Selection and Prioritization via Neural Learning to Rank
              </a>
            </td>
          <td>
            Vishal Dey, Xia Ning
          </td>
          <td>2024-05-13</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Machine learning (ML) has been applied to predict the efficacy of biologic agents in ulcerative colitis (UC). ML can offer precision, personalization, efficiency, and automation. Moreover, it can improve decision support in predicting clinical outcomes. However, it faces challenges related to data quality and quantity, overfitting, generalization, and interpretability. This paper comments on two recent ML models that predict the efficacy of vedolizumab and ustekinumab in UC. Models that consider multiple pathways, multiple ethnicities, and combinations of real-world and clinical trial data are required for optimal shared decision-making and precision medicine. This paper also highlights the potential of combining ML with computational models to enhance clinical outcomes and personalized healthcare. Key Insights: (1) ML offers precision, personalization, efficiency, and decision support for predicting the efficacy of biologic agents in UC. (2) Challenging aspects in ML prediction include data quality, overfitting, and interpretability. (3) Multiple pathways, multiple ethnicities, and combinations of real-world and clinical trial data should be considered in predictive models for optimal decision-making. (4) Combining ML with computational models may improve clinical outcomes and personalized healthcare.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fd22b7c6f685cae80958a1bc2dd8ae70573c792" target='_blank'>
              Machine Learning for Predicting Biologic Agent Efficacy in Ulcerative Colitis: An Analysis for Generalizability and Combination with Computational Models
              </a>
            </td>
          <td>
            Philippe Pinton
          </td>
          <td>2024-06-21</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Predicting cancer dynamics under treatment is challenging due to high inter-patient heterogeneity, lack of predictive biomarkers, and sparse and noisy longitudinal data. Mathematical models can summarize cancer dynamics by a few interpretable parameters per patient. Machine learning methods can then be trained to predict the model parameters from baseline covariates, but do not account for uncertainty in the parameter estimates. Instead, hierarchical Bayesian modeling can model the relationship between baseline covariates to longitudinal measurements via mechanistic parameters while accounting for uncertainty in every part of the model. The mapping from baseline covariates to model parameters can be modeled in several ways. A linear mapping simplifies inference but fails to capture nonlinear covariate effects and scale poorly for interaction modeling when the number of covariates is large. In contrast, Bayesian neural networks can potentially discover interactions between covariates automatically, but at a substantial cost in computational complexity. In this work, we develop a hierarchical Bayesian model of subpopulation dynamics that uses baseline covariate information to predict cancer dynamics under treatment, inspired by cancer dynamics in multiple myeloma (MM), where serum M protein is a well-known proxy of tumor burden. As a working example, we apply the model to a simulated dataset and compare its ability to predict M protein trajectories to a model with linear covariate effects. Our results show that the Bayesian neural network covariate effect model predicts cancer dynamics more accurately than a linear covariate effect model when covariate interactions are present. The framework can also be applied to other types of cancer or other time series prediction problems that can be described with a parametric model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ecd81405c7df09ef0fe2550029e319785f4a7b" target='_blank'>
              Prediction of cancer dynamics under treatment using Bayesian neural networks: A simulated study
              </a>
            </td>
          <td>
            E. M. Myklebust, A. Frigessi, Fredrik Schjesvold, J. Foo, K. Leder, Alvaro Kohn-Luque
          </td>
          <td>2024-05-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="SUMMARY
Drug response is conventionally measured at the cell level, often quantified by metrics like IC50. However, to gain a deeper understanding of drug response, cellular outcomes need to be understood in terms of pathway perturbation. This perspective leads us to recognize a challenge posed by the gap between two widely used large-scale databases, LINCS L1000 and GDSC, measuring drug response at different levels-L1000 captures information at the gene expression level, while GDSC operates at the cell line level. Our study aims to bridge this gap by integrating the two databases through transfer learning, focusing on condition-specific perturbations in gene interactions from L1000 to interpret drug response integrating both gene and cell levels in GDSC. This transfer learning strategy involves pretraining on the transcriptomic-level L1000 dataset, with parameter-frozen fine-tuning to cell line-level drug response. Our novel condition-specific gene-gene attention (CSG2A) mechanism dynamically learns gene interactions specific to input conditions, guided by both data and biological network priors. The CSG2A network, equipped with transfer learning strategy, achieves state-of-the-art performance in cell line-level drug response prediction. In two case studies, well-known mechanisms of drugs are well represented in both the learned gene-gene attention and the predicted transcriptomic profiles. This alignment supports the modeling power in terms of interpretability and biological relevance. Furthermore, our model's unique capacity to capture drug response in terms of both pathway perturbation and cell viability extends predictions to the patient level using TCGA data, demonstrating its expressive power obtained from both gene and cell levels.


AVAILABILITY AND IMPLEMENTATION
The source code for the CSG2A network is available at https://github.com/eugenebang/CSG2A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33a0e168c290638894183c35c3f97f05d25895b1" target='_blank'>
              Transfer learning of condition-specific perturbation in gene interactions improves drug response prediction.
              </a>
            </td>
          <td>
            D. Bang, Bonil Koo, Sun Kim
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In pharmacometrics, developing and selecting models is crucial for quantitatively assessing drug-biological interactions, treatment planning, and gaining insights into underlying processes. These validated models are essential for predictive analytics and strategic decision-making in drug development and clinical practice. Unlike traditional methods, machine learning (ML) offers a data-driven alternative to conventional, first-principle approaches. This paper presents an automatic method to derive unknown or uncertain (sub)models using longitudinal, heterogeneous data. Initially, we employ deep nonlinear mixed effect (DeepNLME) to train a neural network as a universal approximator, which then generates a parameterized representation of the underlying process. Subsequently, we apply symbolic regression to identify a set of potential models expressed as equations. Within the study parameters, the proposed method outperforms the baseline models and demonstrates the validity of both the DeepNLME approach and the symbolic regression approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e126fc4a7b791eeb4ca10543f627ff0e54a67c1" target='_blank'>
              Data-Driven Discovery of Feedback Mechanisms in Acute Myeloid Leukaemia: Alternatives to classical models using Deep Nonlinear Mixed Effect modeling and Symbolic Regression
              </a>
            </td>
          <td>
            Carl Julius Martensen, Niklas Korsbo, V. Ivaturi, Sebastian Sager
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Modeling dynamical systems is a fundamental task in scientific and engineering fields, often accomplished by applying theory-based models with mathematical equations. Yet, in cases where these equations cannot be established or parameterized properly, theory-based models are not applicable. Instead, a viable alternative is to learn the system dynamics directly from data, for example with deep learning models. However, traditional deep learning models often produce physically inconsistent results and struggle to generalize to unseen data, especially when training data is limited. One solution to this shortcoming is knowledge-guided deep learning, leveraging prior knowledge about the expected behavior of a dynamical system. In this work, we identify and formalize permissible system states, a novel type of prior knowledge that is often available for systems in the context of temporal dynamics modeling. This prior knowledge describes dynamic states that the system is allowed to take during its operation. We propose a knowledge-guided multi-state constraint to encode this type of prior knowledge through a loss function, making it applicable to any deep learning model. This approach allows to create an accurate data-driven model with minimal effort and data requirements. We validate the effectiveness of our method by applying it to model the temporal behavior of a gas turbine in response to an input control signal. Our results indicate that the proposed method reduces the prediction error by up to 40%. In addition to reducing the dependency on extensive training data, our method mitigates training randomness and enhances the consistency of predictions with the expected behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba124a5ffc81be0320f4553459c69561edd30f5" target='_blank'>
              Knowledge-Guided Learning of Temporal Dynamics and its Application to Gas Turbines
              </a>
            </td>
          <td>
            Pawel Bielski, Aleksandr Eismont, Jakob Bach, Florian Leiser, D. Kottonau, Klemens Böhm
          </td>
          <td>2024-06-04</td>
          <td>Proceedings of the 15th ACM International Conference on Future and Sustainable Energy Systems</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Combination therapy is an important direction of continuous exploration in the field of medicine, with the core goals of improving treatment efficacy, reducing adverse reactions, and optimizing clinical outcomes. Machine learning technology holds great promise in improving the prediction of drug synergy combinations. However, most studies focus on single disease-oriented collaborative predictive models or involve excessive feature categories, making it challenging to predict the majority of new drugs. To address these challenges, the DrugSK comprehensive model was developed, which utilizes SMILES-BERT to extract structural information from 3492 drugs and trains on reactions from 48,756 drug combinations. DrugSK is an integrated learning model capable of predicting interactions among various drug categories. First, the primary learner is trained from the initial data set. Random forest, support vector machine, and XGboost model are selected as primary learners and logistic regression as secondary learners. A new data set is then "generated" to train level 2 learners, which can be thought of as a prediction for each model. Finally, the results are filtered using logistic regression. Furthermore, the combination of the new antibacterial drug Drafloxacin with other antibacterial agents was tested. The synergistic effect of Drafloxacin and Isavuconazonium in the fight against Candida albicans has been confirmed, providing enlightenment for the clinical treatment of skin infection. DrugSK's prediction is accurate in practical application and can also predict the probability of the outcome. In addition, the tendency of Drafloxacin and antifungal drugs to be synergistic was found. The development of DrugSK will provide a new blueprint for predicting drug combination synergies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055325b217f8174f457e2c50697ff15bdc4f66ea" target='_blank'>
              DrugSK: A Stacked Ensemble Learning Framework for Predicting Drug Combinations of Multiple Diseases.
              </a>
            </td>
          <td>
            Siqi Chen, Nana Gao, Chunzhi Li, Fei Zhai, Xiwei Jiang, Peng Zhang, Jibin Guan, Kefeng Li, Rongwu Xiang, Guixia Ling
          </td>
          <td>2024-06-20</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Physics informed neural networks have been gaining popularity due to their unique ability to incorporate physics laws into data-driven models, ensuring that the predictions are not only consistent with empirical data but also align with domain-specific knowledge in the form of physics equations. The integration of physics principles enables the method to require less data while maintaining the robustness of deep learning in modeling complex dynamical systems. However, current PINN frameworks are not sufficiently mature for real-world ODE systems, especially those with extreme multi-scale behavior such as mosquito population dynamical modelling. In this research, we propose a PINN framework with several improvements for forward and inverse problems for ODE systems with a case study application in modelling the dynamics of mosquito populations. The framework tackles the gradient imbalance and stiff problems posed by mosquito ordinary differential equations. The method offers a simple but effective way to resolve the time causality issue in PINNs by gradually expanding the training time domain until it covers entire domain of interest. As part of a robust evaluation, we conduct experiments using simulated data to evaluate the effectiveness of the approach. Preliminary results indicate that physics-informed machine learning holds significant potential for advancing the study of ecological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf1c97b75409ff1ee6e5c4298dd05295c9b772df" target='_blank'>
              Adapting Physics-Informed Neural Networks To Optimize ODEs in Mosquito Population Dynamics
              </a>
            </td>
          <td>
            D. V. Cuong, Branislava Lali'c, Mina Petri'c, Binh Nguyen, M. Roantree
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In this article, a distributed neural network modeling framework including a novel neural hybrid system model is proposed for enhancing the scalability of neural network models in modeling dynamical systems. First, high-dimensional training data samples will be mapped to a low-dimensional feature space through the principal component analysis (PCA) featuring process. Following that, the feature space is bisected into multiple partitions based on the variation of the Shannon entropy under the maximum entropy (ME) bisecting process. The behavior of subsystems in the prespecified state space partitions will then be approximated using a group of shallow neural networks (SNNs) known as extreme learning machines (ELMs), and then it can further simplify the model by merging the redundant lattices based on their training error performance. The proposed modeling framework can handle high-dimensional dynamical system modeling problems with the advantages of reducing model complexity and improving model performance in training and verification. To demonstrate the effectiveness of the proposed modeling framework, examples of modeling the LASA dataset and an industrial robot are presented.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a2a2b97f2bbf0735400cc24119a8d6645252543" target='_blank'>
              A Distributed Neural Hybrid System Learning Framework in Modeling Complex Dynamical Systems.
              </a>
            </td>
          <td>
            Yejiang Yang, Tao Wang, Weiming Xiang
          </td>
          <td>2024-06-28</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Biosensors play a crucial role in detecting cancer signals by orchestrating a series of intricate biological and physical transduction processes. Among various cancers, breast cancer stands out due to its genetic underpinnings, which trigger uncontrolled cell proliferation, predominantly impacting women, and resulting in significant mortality rates. The utilization of biosensors in predicting survival time becomes paramount in formulating an optimal treatment strategy. However, conventional biosensors employing traditional machine learning methods encounter challenges in preprocessing features for the learning task. Despite the potential of deep learning techniques to automatically extract useful features, they often struggle to effectively leverage the intricate relationships between features and instances. To address this challenge, our study proposes a novel smart biosensor architecture that integrates a multi-view multi-way graph learning (MVMWGL) approach for predicting breast cancer survival time. This innovative approach enables the assimilation of insights from gene interactions and biosensor similarities. By leveraging real-world data, we conducted comprehensive evaluations, and our experimental results unequivocally demonstrate the superiority of the MVMWGL approach over existing methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db73a8e2e2652d0c023c22fd83e59042136d1b61" target='_blank'>
              Smart Biosensor for Breast Cancer Survival Prediction Based on Multi-View Multi-Way Graph Learning
              </a>
            </td>
          <td>
            Wenming Ma, Mingqi Li, Zihao Chu, Hao Chen
          </td>
          <td>2024-05-21</td>
          <td>Sensors (Basel, Switzerland)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Motivation In recent years, applying computational modeling to systems biology has caused a substantial surge in both discovery and practical applications and a significant shift in our understanding of the complexity inherent in biological systems. Results In this perspective article, we briefly overview computational modeling in biology, highlighting recent advancements such as multi-scale modeling due to the omics revolution, single-cell technology, and integration of artificial intelligence and machine learning approaches. We also discuss the primary challenges faced: integration, standardization, model complexity, scalability, and interdisciplinary collaboration. Lastly, we highlight the contribution made by the Computational Modeling of Biological Systems (SysMod) Community of Special Interest (COSI) associated with the International Society of Computational Biology (ISCB) in driving progress within this rapidly evolving field through community engagement (via both in person and virtual meetings, social media interactions), webinars, and conferences. Availability and implementation Additional information about SysMod is available at https://sysmod.info.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b174dea12ab92a7f7139cb9122517aefde98aa6" target='_blank'>
              Perspectives on computational modeling of biological systems and the significance of the SysMod community
              </a>
            </td>
          <td>
            B. L. Puniya, Meghna Verma, Chiara Damiani, Shaimaa Bakr, Andreas Dräger
          </td>
          <td>2024-06-26</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Recent advances in microfluidics and sequencing technologies allow researchers to explore cellular heterogeneity at single-cell resolution. In recent years, deep learning frameworks, such as generative models, have brought great changes to the analysis of transcriptomic data. Nevertheless, relying on the potential space of these generative models alone is insufficient to generate biological explanations. In addition, most of the previous work based on generative models is limited to shallow neural networks with one to three layers of latent variables, which may limit the capabilities of the models. Here, we propose a deep interpretable generative model called d-scIGM for single-cell data analysis. d-scIGM combines sawtooth connectivity techniques and residual networks, thereby constructing a deep generative framework. In addition, d-scIGM incorporates hierarchical prior knowledge of biological domains to enhance the interpretability of the model. We show that d-scIGM achieves excellent performance in a variety of fundamental tasks, including clustering, visualization, and pseudo-temporal inference. Through topic pathway studies, we found that d-scIGM-learned topics are better enriched for biologically meaningful pathways compared to the baseline models. Furthermore, the analysis of drug response data shows that d-scIGM can capture drug response patterns in large-scale experiments, which provides a promising way to elucidate the underlying biological mechanisms. Lastly, in the melanoma dataset, d-scIGM accurately identified different cell types and revealed multiple melanin-related driver genes and key pathways, which are critical for understanding disease mechanisms and drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39b1eecd23692d90bedf7211e73433834e2697ed" target='_blank'>
              Comprehensive single-cell RNA-seq analysis using deep interpretable generative modeling guided by biological hierarchy knowledge
              </a>
            </td>
          <td>
            Hegang Chen, Yuyin Lu, Zhiming Dai, Yuedong Yang, Qing Li, Yanghui Rao
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Machine learning has been proposed as an alternative to theoretical modeling when dealing with complex problems in biological physics. However, in this perspective, we argue that a more successful approach is a proper combination of these two methodologies. We discuss how ideas coming from physical modeling neuronal processing led to early formulations of computational neural networks, e.g., Hopfield networks. We then show how modern learning approaches like Potts models, Boltzmann machines, and the transformer architecture are related to each other, specifically, through a shared energy representation. We summarize recent efforts to establish these connections and provide examples on how each of these formulations integrating physical modeling and machine learning have been successful in tackling recent problems in biomolecular structure, dynamics, function, evolution, and design. Instances include protein structure prediction; improvement in computational complexity and accuracy of molecular dynamics simulations; better inference of the effects of mutations in proteins leading to improved evolutionary modeling and finally how machine learning is revolutionizing protein engineering and design. Going beyond naturally existing protein sequences, a connection to protein design is discussed where synthetic sequences are able to fold to naturally occurring motifs driven by a model rooted in physical principles. We show that this model is "learnable" and propose its future use in the generation of unique sequences that can fold into a target structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/857c85e6a44f8c49c7d41155c041eb3e9361ca60" target='_blank'>
              Machine learning in biological physics: From biomolecular prediction to design.
              </a>
            </td>
          <td>
            Jonathan Martin, Marcos Lequerica Mateos, J. Onuchic, Ivan Coluzza, F. Morcos
          </td>
          <td>2024-06-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="Mammalian cells are commonly used as hosts in cell culture for biologics production in the pharmaceutical industry. Structured mechanistic models of metabolism have been used to capture complex cellular mechanisms that contribute to varying metabolic shifts in different cell lines. However, little research has focused on the impact of temporal changes in enzyme abundance and activity on the modeling of cell metabolism. In this work, we present a framework for constructing mechanistic models of metabolism that integrate growth-signaling control of enzyme activity and transcript dynamics. The proposed approach is applied to build models for three Chinese hamster ovary (CHO) cell lines using fed-batch culture data and time-series transcript profiles. Leveraging information from the transcriptome data, we develop a parameter estimation approach based on multi-cell-line (MCL) learning, which combines data sets from different cell lines and trains the individual cell-line models jointly to improve model accuracy. The computational results demonstrate the important role of growth signaling and transcript variability in metabolic models as well as the virtue of the MCL approach for constructing cell-line models with a limited amount of data. The resulting models exhibit a high level of accuracy in predicting distinct metabolic behaviors in the different cell lines; these models can potentially be used to accelerate the process and cell-line development for the biomanufacturing of new protein therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e85b0ee10ff62351e38806eafe8cf7db9748e226" target='_blank'>
              Multi-cell-line learning for the data-driven construction of mechanistic metabolic models.
              </a>
            </td>
          <td>
            Yen-An Lu, Meghan G. McCann, Wei-Shou Hu, Qi Zhang
          </td>
          <td>2024-06-03</td>
          <td>Biotechnology and bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Introduction: Heterogeneity between patients and within tumors poses challenges to effective cancer treatment. Drug combinations can overcome heterogeneity by increasing the probability that a patient will have a response to one or more drugs. This approach is successful in numerous disease settings including multiple lymphomas, but technological advances have led to a new approach to addressing heterogeneity: precision approaches that stratify patients into subpopulations with more uniform, stronger responses to specific treatments. These strategies can be implemented together to improve patient outcomes. We have developed computational mechanistic models of cancer treatment in the context of both sources of heterogeneity which explain and predict clinical outcomes of treatment of Large B-cell lymphoma (LBCL) with combination chemoimmunotherapy and CAR T-cell therapy.
 Combination therapy: Our novel population tumor kinetics (pop-TK) model simulates clinical combination therapy outcomes by implementing multidrug dose response functions in heterogeneous populations of tumor cells, within heterogeneous cohorts of patients. Informed by the outcomes of novel drugs in phase 2 trials in relapsed/refractory (r/r) patients, the pop-TK model accurately predicted of outcomes of seven first-line phase 3 trials of novel combinations. Most notably, informed by treatment of r/r LBCL with polatuzumab-vedotin (pola), the pop-TK predicted the success of modifying the standard treatment regimen (RCHOP) with pola before the clinical trial results were published. The pola combination improves overall survival (OS) and PFS in Activated B-Cell (ABC) LBCL (hazard ratio (HR): OS 0.3, PFS 0.4), but reduces OS (HR: 1.6) and does not improve PFS (HR: 1.0) in Germinal Center B-Cell (GCB) LBCL. The subtype-specific difference in PFS was also predictable based on subtype results of r/r trials (predicted HR: ABC 0.3, GCB 0.9).
 CAR T-cell therapy: We used a population pharmacokinetic/pharmacodynamic (pop-PK/PD) model of CAR T-cell therapy to reproduce the PFS distribution and the distribution CAR T-cell abundance for CAR T therapy in r/r LBCL patients. The impact of tumor size on patient outcomes in our model and in clinical studies led us to computationally explore the impact of treating smaller tumors with CAR T-cell therapy on treatment outcomes. We found that using detectable circulating tumor DNA (ctDNA) as a biomarker for administering CAR T-cell therapy immediately after RCHOP increased the proportion of simulated patients without a second disease progression event by 30% (HR: 0.43 [0.34, 0.55]).
 Conclusions: The pop-TK model supports LBCL cell-of-origin subtype as a biomarker for pola performance, and the pop-PK/PD model demonstrates that detectable ctDNA is a potential biomarker for early CAR T-cell administration. These examples of model-driven precision trial design have the potential to improve outcomes using readily available biomarkers and existing therapies.
 Citation Format: Amy E. Pomeroy, Adam C. Palmer. Mechanistic models of cancer heterogeneity explain and predict clinical outcomes of Large B-Cell Lymphoma (LBCL) treatment [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PO-005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d895a967e89449a96dfc98b6bea1d188560e4b" target='_blank'>
              Abstract PO-005: Mechanistic models of cancer heterogeneity explain and predict clinical outcomes of Large B-Cell Lymphoma (LBCL) treatment
              </a>
            </td>
          <td>
            Amy E. Pomeroy, Adam C Palmer
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Systems biology tackles the challenge of understanding the high complexity in the internal regulation of homeostasis in the human body through mathematical modelling. These models can aid in the discovery of disease mechanisms and potential drug targets. However, on one hand the development and validation of knowledge-based mechanistic models is time-consuming and does not scale well with increasing features in medical data. On the other hand, more data-driven approaches such as machine learning models require large volumes of data to produce generalizable models. The integration of neural networks and mechanistic models, forming universal differential equation (UDE) models, enables the automated learning of unknown model terms with less data than the neural network alone. Nevertheless, estimating parameters for these hybrid models remains difficult with sparse data and limited sampling durations that are common in biological applications. In this work, we propose the use of physiology-informed regularization, penalizing biologically implausible model behavior to guide the UDE towards more physiologically plausible regions of the solution space. In a simulation study we show that physiology-informed regularization not only results in a more accurate forecasting of model behaviour, but also supports training with less data. We also applied this technique to learn a representation of the rate of glucose appearance in the glucose minimal model using meal response data measured in healthy people. In that case, the inclusion of regularization reduces variability between UDE-embedded neural networks that were trained from different initial parameter guesses. Author summary Systems biology concerns the modelling and analysis of biological processes, by viewing these as interconnected systems. Modelling is typically done either using mechanistic differential equations that are derived from experiments and known biology, or using machine learning on large biological datasets. While mathematical modelling from biological experiments can provide useful insights with limited data, building and validating these models takes a long time and often requires highly invasive measurements in humans. Efforts to combine this classical technique with machine learning have resulted in a framework termed universal differential equations, where the model equations contain a neural network to describe unknown biological interactions. While these methods have shown success in numerous fields, applications in biology are more challenging due to limited data-availability, high data sparsity. In this work, we have introduced physiology-informed regularization to overcome these instabilities and to constrain the model to biologically plausible behavior. Our results show that by using physiology-informed regularization, we can accurately predict future unseen observations in a simulated example, with much more limited data than a similar model without regularization. Additionally, we show an application of this technique on human data, applying a neural network to learn the appearance of glucose in the blood plasma after a meal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf0dda21d3a8fbe599f98eed4fe943aa342f41" target='_blank'>
              Physiology-informed regularization enables training of universal differential equation systems for biological applications
              </a>
            </td>
          <td>
            Max de Rooij, Balázs Erdős, N. V. van Riel, Shauna D. O’Donovan
          </td>
          <td>2024-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Addressing challenges in domain invariance within single-cell genomics necessitates innovative strategies to manage the heterogeneity of multi-source datasets while maintaining the integrity of biological signals. We introduce TarDis, a novel deep generative model designed to disentangle intricate covariate structures across diverse biological datasets, distinguishing technical artifacts from true biological variations. By employing tailored covariate-specific loss components and a self-supervised approach, TarDis effectively generates multiple latent space representations that capture each continuous and categorical target covariate separately, along with unexplained variation. Our extensive evaluations demonstrate that TarDis outperforms existing methods in data integration, covariate disentanglement, and robust out-of-distribution predictions. The model’s capacity to produce interpretable and structured latent spaces, including ordered latent representations for continuous covariates, enhances its utility in hypothesis-driven research. Consequently, TarDis offers a promising analytical platform for advancing scientific discovery, providing insights into cellular dynamics, and enabling targeted therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e80bfb7a8f97a589b30e8f62e51f13cbdd3f23" target='_blank'>
              TarDis: Achieving Robust and Structured Disentanglement of Multiple Covariates
              </a>
            </td>
          <td>
            Kemal Inecik, Aleyna Kara, Antony Rose, M. Haniffa, F. Theis
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Representation learning for tumor gene expression (GEx) data with deep neural networks is limited by the large gene feature space and the scarcity of available clinical and preclinical data. The translation of the learned representation between these data sources is further hindered by inherent molecular differences. To address these challenges, we propose GExMix (Gene Expression Mixup), a data augmentation method, which extends the Mixup concept to generate training samples accounting for the imbalance in both data classes and data sources. We leverage the GExMix-augmented training set in encoder-decoder models to learn a GEx latent representation. Subsequently, we combine the learned representation with drug chemical features in a dual-objective enhanced gene-centric drug response prediction, i.e., reconstruction of GEx latent embeddings and drug response classification. This dual-objective design strategically prioritizes gene-centric information to enhance the final drug response prediction. We demonstrate that augmenting training samples improves the GEx representation, benefiting the gene-centric drug response prediction model. Our findings underscore the effectiveness of our proposed GExMix in enriching GEx data for deep neural networks. Moreover, our proposed gene-centricity further improves drug response prediction when translating preclinical to clinical datasets. This highlights the untapped potential of the proposed framework for GEx data analysis, paving the way toward precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d8245b02f26c5698605a2d1bfdadb5793c2f155" target='_blank'>
              Enhancing Gene Expression Representation and Drug Response Prediction with Data Augmentation and Gene Emphasis
              </a>
            </td>
          <td>
            Diyuan Lu, Daksh P.S. Pamar, AJ Ohnmacht, G. Kutkaite, Michael P. Menden
          </td>
          <td>2024-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVES
Heart failure (HF) impacts millions of patients worldwide, yet the variability in treatment responses remains a major challenge for healthcare professionals. The current treatment strategies, largely derived from population based evidence, often fail to consider the unique characteristics of individual patients, resulting in suboptimal outcomes. This study aims to develop computational models that are patient-specific in predicting treatment outcomes, by utilizing a large Electronic Health Records (EHR) database. The goal is to improve drug response predictions by identifying specific HF patient subgroups that are likely to benefit from existing HF medications.


MATERIALS AND METHODS
A novel, graph-based model capable of predicting treatment responses, combining Graph Neural Network and Transformer was developed. This method differs from conventional approaches by transforming a patient's EHR data into a graph structure. By defining patient subgroups based on this representation via K-Means Clustering, we were able to enhance the performance of drug response predictions.


RESULTS
Leveraging EHR data from 11 627 Mayo Clinic HF patients, our model significantly outperformed traditional models in predicting drug response using NT-proBNP as a HF biomarker across five medication categories (best RMSE of 0.0043). Four distinct patient subgroups were identified with differential characteristics and outcomes, demonstrating superior predictive capabilities over existing HF subtypes (best mean RMSE of 0.0032).


DISCUSSION
These results highlight the power of graph-based modeling of EHR in improving HF treatment strategies. The stratification of patients sheds light on particular patient segments that could benefit more significantly from tailored response predictions.


CONCLUSIONS
Longitudinal EHR data have the potential to enhance personalized prognostic predictions through the application of graph-based AI techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa8cc7976d905e9ea5b2ad49bcac0dd24901aa3" target='_blank'>
              Stratifying heart failure patients with graph neural network and transformer using Electronic Health Records to optimize drug response prediction.
              </a>
            </td>
          <td>
            Shaika Chowdhury, Yongbin Chen, Pengyang Li, Sivaraman Rajaganapathy, Andrew Wen, Xiao Ma, Qiying Dai, Yue Yu, Sunyang Fu, Xiaoqian Jiang, Zhe He, Sunghwan Sohn, Xiaoke Liu, S. Bielinski, A. Chamberlain, James R Cerhan, Nansu Zong
          </td>
          <td>2024-06-26</td>
          <td>Journal of the American Medical Informatics Association : JAMIA</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Based on multi-omics data and drug information, predicting the response of cancer cell lines to drugs is a crucial area of research in modern oncology, as it can promote the development of personalized treatments. Despite the promising performance achieved by existing models, most of them overlook the variations among different omics and lack effective integration of multi-omics data. Moreover, the explicit modeling of cell line/drug attribute and cell line-drug association has not been thoroughly investigated in existing approaches. To address these issues, we propose RedCDR, a dual relation distillation model for cancer drug response (CDR) prediction. Specifically, a parallel dual-branch architecture is designed to enable both the independent learning and interactive fusion feasible for cell line/drug attribute and cell line-drug association information. To facilitate the adaptive interacting integration of multi-omics data, the proposed multi-omics encoder introduces the multiple similarity relations between cell lines and takes the importance of different omics data into account. To accomplish knowledge transfer from the two independent attribute and association branches to their fusion, a dual relation distillation mechanism consisting of representation distillation and prediction distillation is presented. Experiments conducted on the GDSC and CCLE datasets show that RedCDR outperforms previous state-of-the-art approaches in CDR prediction. The source code is available at https://github.com/mhxu1998/RedCDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1d774b32aa1d353908c13e835eb7b0507d698d" target='_blank'>
              RedCDR: Dual Relation Distillation for Cancer Drug Response Prediction.
              </a>
            </td>
          <td>
            Muhao Xu, Zhenfeng Zhu, Yawei Zhao, Kunlun He, Qinghua Huang, Yao Zhao
          </td>
          <td>2024-05-22</td>
          <td>IEEE/ACM transactions on computational biology and bioinformatics</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="Multi-omic characterization and integration remains a challenge due to data complexity and sparsity. Addressing this, our study introduces an unsupervised deep learning model, MOVE (Multi-Omic Variational Encoder), specifically designed to integrate and augment the Cancer Dependency Map (DepMap). Harnessing orthogonal multi-omic information, this model successfully generates molecular and phenotypic profiles, resulting in an increase of 32.7% in the number of multi-omic profiles and thereby generating a complete DepMap for 1,523 cancer cell lines. The synthetically enhanced data increases statistical power, uncovering less studied mechanisms associated with drug resistance, and refines the identification of genetic associations and clustering of cancer cell lines. By applying SHAP for model interpretation, MOVE reveals multi-omic features essential for cell clustering and biomarker identification related to drug and gene dependencies. This understanding is crucial for the development of much-needed, effective strategies in prioritizing cancer targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5f98740294adae572cbbfc27a5b1f5503d6c2b3" target='_blank'>
              Synthetic augmentation of cancer cell line multi-omic datasets using unsupervised deep learning
              </a>
            </td>
          <td>
            Zhaoxiang Cai, Sofia Apolinário, A. R. Baião, Clare Pacini, , Susana Vinga, R. Reddel, Phillip J Robinson, Mathew J. Garnett, Qing Zhong, Emanuel Gonçalves
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction: Synergistic medication, a crucial therapeutic strategy in cancer treatment, involves combining multiple drugs to enhance therapeutic effectiveness and mitigate side effects. Current research predominantly employs deep learning models for extracting features from cell line and cancer drug structure data. However, these methods often overlook the intricate nonlinear relationships within the data, neglecting the distribution characteristics and weighted probability densities of gene expression data in multi-dimensional space. It also fails to fully exploit the structural information of cancer drugs and the potential interactions between drug molecules. Methods: To overcome these challenges, we introduce an innovative end-to-end learning model specifically tailored for cancer drugs, named Dual Kernel Density and Positional Encoding (DKPE) for Graph Synergy Representation Network (DKPEGraphSYN). This model is engineered to refine the prediction of drug combination synergy effects in cancer. DKPE-GraphSYN utilizes Dual Kernel Density Estimation and Positional Encoding techniques to effectively capture the weighted probability density and spatial distribution information of gene expression, while exploring the interactions and potential relationships between cancer drug molecules via a graph neural network. Results: Experimental results show that our prediction model achieves significant performance enhancements in forecasting drug synergy effects on a comprehensive cancer drug and cell line synergy dataset, achieving an AUPR of 0.969 and an AUC of 0.976. Discussion: These results confirm our model’s superior accuracy in predicting cancer drug combinations, providing a supportive method for clinical medication strategy in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282777dc9478ed17a298154b1e985c4aec15b5e9" target='_blank'>
              DKPE-GraphSYN: a drug synergy prediction model based on joint dual kernel density estimation and positional encoding for graph representation
              </a>
            </td>
          <td>
            Yunyun Dong, Yujie Bai, Haitao Liu, Ziting Yang, Yunqing Chang, Jianguang Li, Qixuan Han, Xiufang Feng, Xiaole Fan, Xiaoqiang Ren
          </td>
          <td>2024-06-14</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In oncology drug development, tumor dynamics modeling is widely applied to predict patients' overall survival (OS) via parametric models. However, the current modeling paradigm, which assumes a disease-specific link between tumor dynamics and survival, has its limitations. This is particularly evident in drug development scenarios where the clinical trial under consideration contains patients with tumor types for which there is little to no prior institutional data. In this work, we propose the use of a pan-indication solid tumor machine learning (ML) approach whereby all three tumor metrics (tumor shrinkage rate, tumor regrowth rate and time to tumor growth) are simultaneously used to predict patients' OS in a tumor type independent manner. We demonstrate the utility of this approach in a clinical trial of cancer patients treated with the tyrosine kinase inhibitor, pralsetinib. We compared the parametric and ML models and the results showed that the proposed ML approach is able to adequately predict patient OS across RET-altered solid tumors, including non-small cell lung cancer, medullary thyroid cancer as well as other solid tumors. While the findings of this study are promising, further research is needed for evaluating the generalizability of the ML model to other solid tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f96bc459032f6a187b68e4d09690f24f0e08e461" target='_blank'>
              Predicting overall survival from tumor dynamics metrics using parametric statistical and machine learning models: application to patients with RET-altered solid tumors
              </a>
            </td>
          <td>
            Erick Velasquez, Nastya Kassir, S. Cheeti, Denison Kuruvilla, Rucha Sane, Steve Dang, Dale Miles, James Lu
          </td>
          <td>2024-06-11</td>
          <td>Frontiers in Artificial Intelligence</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The mammalian genome organizes into complex three-dimensional structures, where interactions among chromatin regulatory elements play a pivotal role in mediating biological functions, highlighting the significance of genomic region interactions in biological research. Traditional biological sequencing techniques like HiC and MicroC, commonly employed to estimate these interactions, are resource-intensive and time-consuming, especially given the vast array of cell lines and tissues involved. With the advent of advanced machine learning (ML) methodologies, there has been a push towards developing ML models to predict genomic interactions. However, while these models excel in predicting interactions for cell lines similar to their training data, they often fail to generalize across distantly related cell lines or accurately predict interactions specific to certain cell lines. Identifying the potential oversight of excluding example genomic region interaction information from model inputs as a fundamental limitation, this paper introduces GRACHIP, a model rooted in graph neural network technology aiming to address this issue by incorporating detailed interaction information as a hint. Through extensive testing across various cell lines, GRACHIP not only demonstrates exceptional accuracy in predicting chromatin interaction intensity but showcases remarkable generalizability to cell lines not encountered during training. Consequently, GRACHIP emerges as a potent research tool, offering a viable alternative to conventional sequencing methods for analyzing the interactions and three-dimensional organization of mammalian genomes, thus alleviating the dependency on expensive and time-consuming biological sequencing techniques. It also offers an alternative way for researchers to investigate 3D chromatin interactions and simulate their changes in model systems to test their hypotheses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92aaa802a4f13bf6fc0e8fce14687506f2b37a22" target='_blank'>
              Improving Cell-type-specific 3D Genome Architectures Prediction Leveraging Graph Neural Networks
              </a>
            </td>
          <td>
            Ruoyun Wang, Weicheng Ma, Aryan Soltani Mohammadi, Saba Shahsavari, Soroush Vosoughi, Xiaofeng Wang
          </td>
          <td>2024-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bb5d08a605e13f3aee7409ac89075d346310599" target='_blank'>
              MISATO: machine learning dataset of protein–ligand complexes for structure-based drug discovery
              </a>
            </td>
          <td>
            Till Siebenmorgen, Filipe Menezes, Sabrina Benassou, Erinc Merdivan, Kieran Didi, André Santos Dias Mourão, Radosław Kitel, Pietro Liò, Stefan Kesselheim, M. Piraud, F. Theis, Michael Sattler, G. Popowicz
          </td>
          <td>2024-05-10</td>
          <td>Nature Computational Science</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="Motivation Anti-cancer drug response prediction (DRP) using cancer cell lines plays a vital role in stratified medicine and drug discovery. Recently there has been a surge of new deep learning (DL) models for DRP that show promising performance improvements. However, different models use different input data modalities and neural network architectures making it hard to find the source of these improvements. Results We consider three DL DRP models with reported state-of-the-art performance and use genomics or transcriptomics profiles of cell lines and chemical profiles of drugs as input data. We also construct null hypothesis benchmarks that simply leverage patterns in truth values and thus do not use omics or chemical profiles. By experimenting with these DL models and comparing them with our null hypothesis benchmarks we show that for multiple testing types, much or all of the current reported performance is a property of the training truth values. Furthermore, we show that for cancer blind testing (testing on unseen cell lines) no performance comes from drug features, instead, performance is due to the transcriptomics cell line profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c5705a239471a8f4b914d01d1c42bcefb5e083" target='_blank'>
              Understanding the Sources of Performance in Deep Learning Drug Response Prediction Models
              </a>
            </td>
          <td>
            Nikhil Branson, Pedro R. Cutillas, Conrad Besseant
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The evaluation of drug-induced Torsades de pointes (TdP) risks is crucial in drug safety assessment. In this study, we discuss machine learning approaches in the prediction of drug-induced TdP risks using preclinical data. Specifically, a random forest model was trained on the dataset generated by the rabbit ventricular wedge assay. The model prediction performance was measured on 28 drugs from the Comprehensive In Vitro Proarrhythmia Assay initiative. Leave-one-drug-out cross-validation provided an unbiased estimation of model performance. Stratified bootstrap revealed the uncertainty in the asymptotic model prediction. Our study validated the utility of machine learning approaches in predicting drug-induced TdP risks from preclinical data. Our methods can be extended to other preclinical protocols and serve as a supplementary evaluation in drug safety assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aff64c32acf59cf216b66c0fc2cd3375c21ab4" target='_blank'>
              Drug safety assessment by machine learning models.
              </a>
            </td>
          <td>
            Nan Miles Xi, D. Huang
          </td>
          <td>2024-06-18</td>
          <td>Journal of biopharmaceutical statistics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Drug development is a lengthy process with a high failure rate. Increasingly, machine learning is utilized to facilitate the drug development processes. These models aim to enhance our understanding of drug characteristics, including their activity in biological contexts. However, a major challenge in drug response (DR) prediction is model interpretability as it aids in the validation of findings. This is important in biomedicine, where models need to be understandable in comparison with established knowledge of drug interactions with proteins. drGAT, a graph deep learning model, leverages a heterogeneous graph composed of relationships between proteins, cell lines, and drugs. drGAT is designed with two objectives: DR prediction as a binary sensitivity prediction and elucidation of drug mechanism from attention coefficients. drGAT has demonstrated superior performance over existing models, achieving 78% accuracy (and precision), and 76% F1 score for 269 DNA-damaging compounds of the NCI60 drug response dataset. To assess the model’s interpretability, we conducted a review of drug-gene co-occurrences in Pubmed abstracts in comparison to the top 5 genes with the highest attention coefficients for each drug. We also examined whether known relationships were retained in the model by inspecting the neighborhoods of topoisomerase-related drugs. For example, our model retained TOP1 as a highly weighted predictive feature for irinotecan and topotecan, in addition to other genes that could potentially be regulators of the drugs. Our method can be used to accurately predict sensitivity to drugs and may be useful in the identification of biomarkers relating to the treatment of cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23626c19de7aea3fe09bc589778f03d93292a717" target='_blank'>
              drGAT: Attention-Guided Gene Assessment of Drug Response Utilizing a Drug-Cell-Gene Heterogeneous Network
              </a>
            </td>
          <td>
            Yoshitaka Inoue, Hunmin Lee, Tianfan Fu, Augustin Luna
          </td>
          <td>2024-05-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Aim: Despite the advances in oncology and cancer treatment over the past decades, cancer remains one of the deadliest diseases. This study focuses on further understanding the complex nature of cancer by using mathematical tumor modeling to understand, capture as best as possible, and describe its complex dynamics under chemotherapy treatment. Materials and Methods: Focusing on autoregressive with exogenous inputs, i.e., ARX, and adaptive neuro-fuzzy inference system, i.e., ANFIS, models, this work investigates tumor growth dynamics under both single and combination anticancer agent chemotherapy treatments using chemotherapy treatment data on xenografted mice. Results: Four ARX and ANFIS models for tumor growth inhibition were developed, estimated, and evaluated, demonstrating a strong correlation with tumor weight data, with ANFIS models showing superior performance in handling the multi-agent tumor growth complexities. These findings suggest potential clinical applications of the ANFIS models through further testing. Both types of models were also tested for their prediction capabilities across different chemotherapy schedules, with accurate forecasting of tumor growth up to five days in advance. The use of adaptive prediction and sliding (moving) data window techniques allowed for continuous model updating, ensuring more robust predictive capabilities. However, long-term forecasting remains a challenge, with accuracy declining over longer prediction horizons. Conclusion: While ANFIS models showed greater reliability in predictions, the simplicity and rapid deployment of ARX models offer advantages in situations requiring immediate approximations. Future research with larger, more diverse datasets and by exploring varying model complexities is recommended to improve the models’ reliability and applicability in clinical decision-making, thereby aiding the development of personalized chemotherapy regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e4427fcccf3edcdd11ca0d2dc97f640c7df5d5" target='_blank'>
              A Comparative Analysis of ARX and ANFIS Models for Tumor Growth Prediction Under Single and Multi-agent Chemotherapy
              </a>
            </td>
          <td>
            S. Liliopoulos, G. Stavrakakis, Konstantinos S Dimas
          </td>
          <td>2024-05-31</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31621df67f25b8790adc049089632cf09d1f7ce7" target='_blank'>
              Pharmacokinetics-Pharmacodynamics Modeling for Evaluating Drug-Drug Interactions in Polypharmacy: Development and Challenges.
              </a>
            </td>
          <td>
            Di Zhao, Ping Huang, Li Yu, Yu He
          </td>
          <td>2024-06-18</td>
          <td>Clinical pharmacokinetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Mpox (formerly monkeypox) is an infectious disease that spreads mostly through direct contact with infected animals or people's blood, bodily fluids, or cutaneous or mucosal lesions. In light of the global outbreak that occurred in 2022–2023, in this paper, we analyzed global Mpox univariate time series data and provided a comprehensive analysis of disease outbreaks across the world, including the USA with Brazil and three continents: North America, South America, and Europe. The novelty of this study is that it delved into the Mpox time series data by implementing the data-driven methods and a mathematical model concurrently—an aspect not typically addressed in the existing literature. The study is also important because implementing these models concurrently improved our predictions' reliability for infectious diseases. Methods We proposed a traditional compartmental model and also implemented deep learning models (1D- convolutional neural network (CNN), long-short term memory (LSTM), bidirectional LSTM (BiLSTM), hybrid CNN-LSTM, and CNN-BiLSTM) as well as statistical time series models: autoregressive integrated moving average (ARIMA) and exponential smoothing on the Mpox data. We also employed the least squares method fitting to estimate the essential epidemiological parameters in the proposed deterministic model. Results The primary finding of the deterministic model is that vaccination rates can flatten the curve of infected dynamics and influence the basic reproduction number. Through the numerical simulations, we determined that increased vaccination among the susceptible human population is crucial to control disease transmission. Moreover, in case of an outbreak, our model showed the potential for epidemic control by adjusting the key epidemiological parameters, namely the baseline contact rate and the proportion of contacts within the human population. Next, we analyzed data-driven models that contribute to a comprehensive understanding of disease dynamics in different locations. Additionally, we trained models to provide short-term (eight-week) predictions across various geographical locations, and all eight models produced reliable results. Conclusion This study utilized a comprehensive framework to investigate univariate time series data to understand the dynamics of Mpox transmission. The prediction showed that Mpox is in its die-out situation as of July 29, 2023. Moreover, the deterministic model showed the importance of the Mpox vaccination in mitigating the Mpox transmission and highlighted the significance of effectively adjusting key epidemiological parameters during outbreaks, particularly the contact rate in high-risk groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302a4b8ee78aaa9f3ebf07b199bea13110d35c6e" target='_blank'>
              Exploring the dynamics of monkeypox transmission with data-driven methods and a deterministic model
              </a>
            </td>
          <td>
            Haridas K. Das
          </td>
          <td>2024-05-22</td>
          <td>Frontiers in Epidemiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dose-response prediction in cancer is an active application field in machine learning. Using large libraries of \textit{in-vitro} drug sensitivity screens, the goal is to develop accurate predictive models that can be used to guide experimental design or inform treatment decisions. Building on previous work that makes use of permutation invariant multi-output Gaussian Processes in the context of dose-response prediction for drug combinations, we develop a variational approximation to these models. The variational approximation enables a more scalable model that provides uncertainty quantification and naturally handles missing data. Furthermore, we propose using a deep generative model to encode the chemical space in a continuous manner, enabling prediction for new drugs and new combinations. We demonstrate the performance of our model in a simple setting using a high-throughput dataset and show that the model is able to efficiently borrow information across outputs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54716d9548a4c571e30130186ea85e7d8aec3712" target='_blank'>
              Permutation invariant multi-output Gaussian Processes for drug combination prediction in cancer
              </a>
            </td>
          <td>
            Leiv Rønneberg, V. Lalchand, Paul D. W. Kirk
          </td>
          <td>2024-06-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Analyzing the motion of multiple biological agents, be it cells or individual animals, is pivotal for the understanding of complex collective behaviors. With the advent of advanced microscopy, detailed images of complex tissue formations involving multiple cell types have become more accessible in recent years. However, deciphering the underlying rules that govern cell movements is far from trivial. Here, we present a novel deep learning framework to estimate the underlying equations of motion from observed trajectories, a pivotal step in decoding such complex dynamics. Our framework integrates graph neural networks with neural differential equations, enabling effective prediction of two-body interactions based on the states of the interacting entities. We demonstrate the efficacy of our approach through two numerical experiments. First, we used a simulated data from a toy model to tune the hyperparameters. Based on the obtained hyperparameters, we then applied this approach to a more complex model that describes interacting cells of cellular slime molds. Our results show that the proposed method can accurately estimate the function of two-body interactions, thereby precisely replicating both individual and collective behaviors within these systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bbd03a34ea3ecbe6332287f4b5f0d43638d30f4" target='_blank'>
              Integrating GNN and Neural ODEs for Estimating Two-Body Interactions in Mixed-Species Collective Motion
              </a>
            </td>
          <td>
            Masahito Uwamichi, S. Schnyder, Tetsuya J. Kobayashi, Satoshi Sawai
          </td>
          <td>2024-05-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Modeling complex systems using standard neural ordinary differential equations (NODEs) often faces some essential challenges, including high computational costs and susceptibility to local optima. To address these challenges, we propose a simulation-free framework, called Fourier NODEs (FNODEs), that effectively trains NODEs by directly matching the target vector field based on Fourier analysis. Specifically, we employ the Fourier analysis to estimate temporal and potential high-order spatial gradients from noisy observational data. We then incorporate the estimated spatial gradients as additional inputs to a neural network. Furthermore, we utilize the estimated temporal gradient as the optimization objective for the output of the neural network. Later, the trained neural network generates more data points through an ODE solver without participating in the computational graph, facilitating more accurate estimations of gradients based on Fourier analysis. These two steps form a positive feedback loop, enabling accurate dynamics modeling in our framework. Consequently, our approach outperforms state-of-the-art methods in terms of training time, dynamics prediction, and robustness. Finally, we demonstrate the superior performance of our framework using a number of representative complex systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bedb3203006239c598b64a69777f9f9b9613ed" target='_blank'>
              From Fourier to Neural ODEs: Flow Matching for Modeling Complex Systems
              </a>
            </td>
          <td>
            Xin Li, Jingdong Zhang, Qunxi Zhu, Chengli Zhao, Xue Zhang, Xiaojun Duan, Wei Lin
          </td>
          <td>2024-05-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Virtual clinical trials (VCTs) are growing in popularity as a tool for quantitatively predicting heterogeneous treatment responses across a population. In the context of a VCT, a plausible patient is an instance of a mathematical model with parameter (or attribute) values chosen to reflect features of the disease and response to treatment for that particular patient. A number of techniques have been introduced to determine the set of model parametrizations to include in a virtual patient cohort. These methodologies generally start with a prior distribution for each model parameter and utilize some criteria to determine whether a parameter set sampled from the priors should be included or excluded from the plausible population. No standard technique exists, however, for generating these prior distributions and choosing the inclusion/exclusion criteria. In this work, we rigorously quantify the impact that VCT design choices have on VCT predictions. Rather than use real data and a complex mathematical model, a spatial model of radiotherapy is used to generate simulated patient data and the mathematical model used to describe the patient data is a two-parameter ordinary differential equations model. This controlled setup allows us to isolate the impact of both the prior distribution and the inclusion/exclusion criteria on both the heterogeneity of plausible populations and on predicted treatment response. We find that the prior distribution, rather than the inclusion/exclusion criteria, has a larger impact on the heterogeneity of the plausible population. Yet, the percent of treatment responders in the plausible population was more sensitive to the inclusion/exclusion criteria utilized. This foundational understanding of the role of virtual clinical trial design should help inform the development of future VCTs that use more complex models and real data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231e7bc7a40e4003493fc06a15b04cc9b5fdabe2" target='_blank'>
              Assessing the Role of Patient Generation Techniques in Virtual Clinical Trial Outcomes
              </a>
            </td>
          <td>
            J. L. Gevertz, J. R. Wares
          </td>
          <td>2024-06-12</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Medicinal synergy prediction is a powerful tool in drug discovery and development that harnesses the principles of combination therapy to enhance therapeutic outcomes by improving efficacy, reducing toxicity, and preventing drug resistance. While a myriad of computational methods has emerged for predicting synergistic drug combinations, a large portion of them may overlook the intricate, yet critical relationships between various entities in drug interaction networks, such as drugs, cell lines, and diseases. These relationships are complex and multidimensional, requiring sophisticated modeling to capture nuanced interplay that can significantly influence therapeutic efficacy. We introduce a salient deep hypergraph learning method, namely, Heterogeneous Entity Representation for MEdicinal Synergy prediction (HERMES), to predict anti-cancer drug synergy. HERMES integrates heterogeneous data sources, encompassing drug, cell line, and disease information, to provide a comprehensive understanding of the interactions involved. By leveraging advanced hypergraph neural networks with gated residual mechanisms, HERMES can effectively learn complex relationships/interactions within the data. Our results show HERMES demonstrates state-of-the-art performance, particularly in forecasting new drug combinations, significantly surpassing previous methods. This advancement underscores the potential of HERMES to facilitate more effective and precise drug combination predictions, thereby enhancing the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9f935a6bee872e8d5f11ac3462761c64e3b2806" target='_blank'>
              Heterogeneous Entity Representation for Medicinal Synergy Prediction
              </a>
            </td>
          <td>
            Jiawei Wu, Jun Wen, Mingyuan Yan, Anqi Dong, Can Chen
          </td>
          <td>2024-06-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor growth inhibition (TGI) modeling attempts to describe the time course changes in tumor size for patients undergoing cancer therapy. TGI models present several advantages over traditional exposure-response models that are based explicitly on clinical end points and have become a popular tool in the pharmacometrics community. Unfortunately, the data required to fit TGI models, namely longitudinal tumor measurements, are sparse or often not available in literature or publicly accessible databases. On the contrary, common end points such as progression-free survival (PFS) and objective response rate (ORR) are directly derived from longitudinal tumor measurements and are routinely published. To this end, a Bayesian generative model relating underlying tumor dynamics to summary PFS and ORR data is introduced to learn TGI model parameters using only published summary data. The parameterized model can describe the tumor dynamics, quantify treatment effect, and account for differences in the study population. The utility of this model is shown by applying it to several published studies, and learned parameters are combined to simulate an in silico trial of a novel combination therapy in a novel setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5334e76ce88316ffc777a0522ed6c120271486bb" target='_blank'>
              A novel Bayesian generative approach for estimating tumor dynamics from published studies.
              </a>
            </td>
          <td>
            Arya Pourzanjani, Saurabh Modi, Jamie Connarn, Xinwen Zhang, V. Upreti, Chih-Wei Lin, Khamir Mehta
          </td>
          <td>2024-05-22</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Serum creatinine in neonates follows complex dynamics due to maturation processes, most pronounced in the first few weeks of life. The development of a mechanism-based model describing complex dynamics requires high expertise in pharmacometric (PMX) modeling and substantial model development time. A recently published machine learning (ML) approach of low-dimensional neural ordinary differential equations (NODEs) is capable of modeling such data from newborns automatically. However, this efficient data-driven approach in itself does not result in a clinically interpretable model. In this work, an approach to deriving an interpretable model with reasonable PMX-type functions is presented. This "translation" was applied to derive a PMX model for serum creatinine in neonates considering maturation processes and covariates. The developed model was compared to a previously published mechanism-based PMX model whereas both models had similar mechanistic structures. The developed model was then utilized to simulate serum creatinine concentrations in the first few weeks of life considering different covariate values for gestational age and birth weight. The reference serum creatinine values derived from these simulations are consistent with observed serum creatinine values and previously published reference values. Thus, the presented NODE-based ML approach to model complex serum creatinine dynamics in newborns and derive interpretable, mathematical-statistical components similar to those in a conventional PMX model demonstrates a novel, viable approach to facilitate the modeling of complex dynamics in clinical settings and pediatric drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fae58343bde99a83efb245281ab69b262f4cd7f" target='_blank'>
              Applying Neural ODEs to Derive a Mechanism-Based Model for Characterizing Maturation-Related Serum Creatinine Dynamics in Preterm Newborns.
              </a>
            </td>
          <td>
            Dominic Stefan Bräm, Gilbert Koch, Karel Allegaert, J. van den Anker, M. Pfister
          </td>
          <td>2024-05-16</td>
          <td>Journal of clinical pharmacology</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f0f239ff184cb06fa871df53cfa4741c1a7fab" target='_blank'>
              Deep Kernel learning for reaction outcome prediction and optimization
              </a>
            </td>
          <td>
            Sukriti Singh, José Miguel Hernández-Lobato
          </td>
          <td>2024-06-14</td>
          <td>Communications Chemistry</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Classical compartmental models in epidemiology often struggle to accurately capture real-world dynamics due to their inability to address the inherent heterogeneity of populations. In this paper, we introduce a novel approach that incorporates heterogeneity through a mobility variable, transforming the traditional ODE system into a system of integro-differential equations that describe the dynamics of population densities across different compartments. Our results show that, for the same basic reproduction number, our mobility-based model predicts a smaller final pandemic size compared to classic compartmental models, whose population densities are represented as Dirac delta functions in our density-based framework. This addresses the overestimation issue common in many classical models. Additionally, we demonstrate that the time series of the infected population is sufficient to uniquely identify the mobility distribution. We reconstruct this distribution using a machine-learning-based framework, providing both theoretical and algorithmic support to effectively constrain the mobility-based model with real-world data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c03a14a48a310a736f8149b03d7cfacc5775bf6" target='_blank'>
              Modeling, Inference, and Prediction in Mobility-Based Compartmental Models for Epidemiology
              </a>
            </td>
          <td>
            Ning Jiang, Weiqi Chu, Yao Li
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In recent years, the escalating environmental challenges have contributed to a rising incidence of cancer. The precise anticipation of cancer incidence and mortality rates has emerged as a pivotal focus in scientific inquiry, exerting a profound impact on the formulation of public health policies. This investigation adopts a pioneering machine learning framework to address this critical issue, utilizing a dataset encompassing 72,591 comprehensive records that include essential variables such as age, case count, population size, race, gender, site, and year of diagnosis. Diverse machine learning algorithms, including decision trees, random forests, logistic regression, support vector machines, and neural networks, were employed in this study. The ensuing analysis revealed testing accuracies of 62.17%, 61.92%, 54.53%, 55.72%, and 62.30% for the respective models. This state-of-the-art model not only enhances our understanding of cancer dynamics but also equips researchers and policymakers with the capability of making meticulous projections concerning forthcoming cancer incidence and mortality rates. Considering sustainability, the application of this advanced machine learning framework emphasizes the importance of judiciously utilizing extensive and intricate databases. By doing so, it facilitates a more sustainable approach to healthcare planning, allowing for informed decision-making that takes into account the long-term ecological and societal impacts of cancer-related policies. This integrative perspective underscores the broader commitment to sustainable practices in both health research and public policy formulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4560de26fc63a5fa7d9e33f55b5a31acaa915143" target='_blank'>
              Machine Learning Model Construction and Testing: Anticipating Cancer Incidence and Mortality
              </a>
            </td>
          <td>
            Yuanzhao Ding
          </td>
          <td>2024-06-30</td>
          <td>Diseases</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55d2566d48623ecfc68341b1562423b2d582647" target='_blank'>
              Deep learning-based approach for COVID-19 spread prediction
              </a>
            </td>
          <td>
            S. Cumbane, Gyözö Gidófalvi
          </td>
          <td>2024-06-10</td>
          <td>International Journal of Data Science and Analytics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The ever-expanding landscape of pharmaceutical compounds and medical treatments has underscored the critical need for an advanced system to predict drug-drug interactions (DDIs) using machine learning. This paper delves into the development of a novel predictive model that leverages diverse data sources, including chemical structures, pharmacological properties, and scientific literature, to comprehensively analyse and foresee potential DDIs. The current state of DDIs is rife with limitations in scope and accuracy, making the development of an advanced system not only a necessity for enhancing patient safety but also a boon to pharmaceutical research and regulatory oversight. The relationship between two medications in which one drug’s pharmacological actions are altered by another is known as a drug-drug interaction (DDI). Favourable drug-drug interactions (DDIs) generally contribute to improved therapeutic results for patients. On the contrary, adverse DDIs pose a significant risk, often resulting in undesirable drug reactions that can potentially lead to the withdrawal of a drug from the market and even patient fatalities. Consequently, the identification of DDIs is essential for advancing new drug development and effectively managing diseases. Keywords: Drug, Drug-drug interaction, Machine learning, Bioinformatics, Drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc31ce4df8b41262c53f949e913ee6c901766ec" target='_blank'>
              PREDICTING DRUG-DRUG INTERACTION USING MACHINE LEARNING
              </a>
            </td>
          <td>

          </td>
          <td>2024-05-29</td>
          <td>INTERANTIONAL JOURNAL OF SCIENTIFIC RESEARCH IN ENGINEERING AND MANAGEMENT</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mathematical models are indispensable to the system biology toolkit for studying the structure and behavior of intracellular signaling networks. A common approach to modeling is to develop a system of equations that encode the known biology using approximations and simplifying assumptions. As a result, the same signaling pathway can be represented by multiple models, each with its set of underlying assumptions, which opens up challenges for model selection and decreases certainty in model predictions. Here, we use Bayesian multimodel inference to develop a framework to increase certainty in systems biology models. Using models of the extracellular regulated kinase (ERK) pathway, we first show that multimodel inference increases predictive certainty and yields predictors that are robust to changes in the set of available models. We then show that predictions made with multi-model inference are robust to data uncertainties introduced by decreasing the measurement duration and reducing the sample size. Finally, we use multimodel inference to identify a new model to explain experimentally measured sub-cellular location-specific ERK activity dynamics. In summary, our framework highlights multimodel inference as a disciplined approach to increasing the certainty of intracellular signaling activity predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e6724b6c05e22b755836f73b25e0ce04843d4" target='_blank'>
              Increasing certainty in systems biology models using Bayesian multimodel inference
              </a>
            </td>
          <td>
            Nathaniel Linden-Santangeli, Jin Zhang, Boris Kramer, P. Rangamani
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="e21567 Background: Immune checkpoint inhibitor (ICI) related toxicity is common in melanoma patients, but identifying who will experience toxicity can be challenging. Precision medicine tools that accurately predict ICI toxicity could improve patient outcomes. This study aimed to use Machine Learning (ML) to predict ICI toxicity in patients with melanoma. Methods: 384 datasets were available for patients started on immunotherapy between 2014-2023 in a single, large regional cancer center in Kent, U.K. Raw data was preprocessed using a standard scaling function and one hot encoding, followed by SMOTE to balance the imbalanced classes. Six ML algorithms were developed using 80% of the training data, tested on 20% of validation data and used the Grid Search Cross-Validation technique for hyperparameter optimization. Shapley Additive Explanations (SHAP) explainable artificial intelligence was used to interpret the toxicity prediction model to improve the interpretation and transparency of the decision boundary. Results: Of the 384 patients, 112 patients (29%) had completely resected disease and then had adjuvant treatment while 272 (71%) were unresectable or had metastatic disease and had palliative treatment intent. In the metastatic setting, the commonest immunotherapy regimen was Pembrolizumab (155, 57%), followed by Ipilimumab/Nivolumab (85, 31%). The commonest regimen in the adjuvant setting was Pembrolizumab (86, 77%). 95 patients (25%) experienced ICI-related toxicity. The number of patients with Grades 1-4 toxicity was 2 ,4, 18 and 4 respectively, the remaining had unspecified toxicity grades. The commonest class of toxicity (CTCAE v5) was gastrointestinal (36, 38%), followed by endocrine (20, 21%) and skin (19, 20%). Linear Regression and Gaussian Naïve Bayes predicted toxicity with the most accuracy, 92% (Table 1). The model showed that increasing age, female gender and not receiving Nivolumab predicted toxicity. Regardless of the regimen or treatment intent, our ML model could also predict immunotherapy response with 90% accuracy. Conclusions: These ML algorithms used clinically available data to predict ICI toxicity accurately. Key predictors of toxicity included increasing age, female gender and not receiving Nivolumab in the setting of both adjuvant and palliative intent. Future work will aim to externally validate these models in other centers. We will also explore if models could be trained for use in patients treated with nivolumab in combination with new anti-LAG-3(relatlimab). [Table: see text]">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82af763afba24d382c5aea59de63ca29a799a576" target='_blank'>
              Immunotherapy toxicity prediction in patients with melanoma using machine learning algorithms.
              </a>
            </td>
          <td>
            Lakshya Sharma, Esha Mohan, V. Balaji, Jonathan Wong, Adwait Joshi, I. Mporas, Jiaqi Shi, Yi Zhao, Amy C. Stoll-D'Astice, V. Tailor, Georgie Powell, D. Alrifai, Riyaz N.H. Shah, Justin S. Waters, S. Adeleke
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Machine-learning (ML) and deep-learning (DL) approaches to predict the molecular properties of small molecules are increasingly deployed within the design-make-test-analyze (DMTA) drug design cycle to predict molecular properties of interest. Despite this uptake, there are only a few automated packages to aid their development and deployment that also support uncertainty estimation, model explainability, and other key aspects of model usage. This represents a key unmet need within the field, and the large number of molecular representations and algorithms (and associated parameters) means it is nontrivial to robustly optimize, evaluate, reproduce, and deploy models. Here, we present QSARtuna, a molecule property prediction modeling pipeline, written in Python and utilizing the Optuna, Scikit-learn, RDKit, and ChemProp packages, which enables the efficient and automated comparison between molecular representations and machine learning models. The platform was developed by considering the increasingly important aspect of model uncertainty quantification and explainability by design. We provide details for our framework and provide illustrative examples to demonstrate the capability of the software when applied to simple molecular property, reaction/reactivity prediction, and DNA encoded library enrichment classification. We hope that the release of QSARtuna will further spur innovation in automatic ML modeling and provide a platform for education of best practices in molecular property modeling. The code for the QSARtuna framework is made freely available via GitHub.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d79165e7710fcacddea5e1ab595a6e3520735bc2" target='_blank'>
              QSARtuna: An Automated QSAR Modeling Platform for Molecular Property Prediction in Drug Design.
              </a>
            </td>
          <td>
            Lewis Mervin, Alexey Voronov, Mikhail Kabeshov, O. Engkvist
          </td>
          <td>2024-07-01</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In the complex landscape of hematologic samples such as peripheral blood or bone marrow derived from flow cytometry (FC) data, cell-level prediction presents profound challenges. This work explores injecting hierarchical prior knowledge into graph neural networks (GNNs) for single-cell multi-class classification of tabular cellular data. By representing the data as graphs and encoding hierarchical relationships between classes, we propose our hierarchical plug-in method to be applied to several GNN models, namely, FCHC-GNN, and effectively designed to capture neighborhood information crucial for single-cell FC domain. Extensive experiments on our cohort of 19 distinct patients, demonstrate that incorporating hierarchical biological constraints boosts performance significantly across multiple metrics compared to baseline GNNs without such priors. The proposed approach highlights the importance of structured inductive biases for gaining improved generalization in complex biological prediction tasks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e0fdaea8a065eda5545038bbdb830b51b0c573" target='_blank'>
              Injecting Hierarchical Biological Priors into Graph Neural Networks for Flow Cytometry Prediction
              </a>
            </td>
          <td>
            Fatemeh Nassajian Mojarrad, Lorenzo Bini, Thomas Matthes, Stéphane Marchand-Maillet
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MOTIVATION
The rise of single-cell RNA sequencing (scRNA-seq) technology presents new opportunities for constructing detailed cell type-specific gene regulatory networks (GRNs) to study cell heterogeneity. However, challenges caused by noises, technical errors, and dropout phenomena in scRNA-seq data pose significant obstacles to GRN inference, making the design of accurate GRN inference algorithms still essential. The recent growth of both single-cell and spatial transcriptomic sequencing data enables the development of supervised deep learning methods to infer GRNs on these diverse single-cell datasets.


RESULTS
In this study, we introduce a novel deep learning framework based on shared factor neighborhood and integrated neural network (SFINN) for inferring potential interactions and causalities between transcription factors and target genes from single-cell and spatial transcriptomic data. SFINN utilizes shared factor neighborhood to construct cellular neighborhood network based on gene expression data and additionally integrates cellular network generated from spatial location information. Subsequently, the cell adjacency matrix and gene pair expression are fed into an integrated neural network framework consisting of a graph convolutional neural network and a fully-connected neural network to determine whether the genes interact. Performance evaluation in the tasks of gene interaction and causality prediction against the existing GRN reconstruction algorithms demonstrates the usability and competitiveness of SFINN across different kinds of data. SFINN can be applied to infer gene regulatory networks from conventional single-cell sequencing data and spatial transcriptomic data.


AVAILABILITY
SFINN can be accessed at GitHub: https://github.com/JGuan-lab/SFINN.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52724caebdd9eb86a3aab7e26584a02fd43a37c9" target='_blank'>
              SFINN: inferring gene regulatory network from single-cell and spatial transcriptomic data with shared factor neighborhood and integrated neural network.
              </a>
            </td>
          <td>
            Yongjie Wang, Fengfan Zhou, Jinting Guan
          </td>
          <td>2024-07-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Predicting drug efficacy and safety in vivo requires information on biological responses (e.g., cell morphology and gene expression) to small molecule perturbations. However, current molecular representation learning methods do not provide a comprehensive view of cell states under these perturbations and struggle to remove noise, hindering model generalization. We introduce the Information Alignment (InfoAlign) approach to learn molecular representations through the information bottleneck method in cells. We integrate molecules and cellular response data as nodes into a context graph, connecting them with weighted edges based on chemical, biological, and computational criteria. For each molecule in a training batch, InfoAlign optimizes the encoder’s latent representation with a minimality objective to discard redundant structural information. A sufficiency objective decodes the representation to align with different feature spaces from the molecule’s neighborhood in the context graph. We demonstrate that the proposed sufficiency objective for alignment is tighter than existing encoder-based contrastive methods. Empirically, we validate representations from InfoAlign in two downstream tasks: molecular property prediction against up to 19 baseline methods across four datasets, plus zero-shot molecule-morphology matching.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db33b94c598876b05b7af83901a3a69d60eb47d" target='_blank'>
              Learning Molecular Representation in a Cell
              </a>
            </td>
          <td>
            Gang Liu, Srijit Seal, John Arevalo, Zhenwen Liang, A. Carpenter, Meng Jiang, Shantanu Singh
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Parameter estimation is one of the central problems in computational modeling of biological systems. Typically, scientists must fully specify the mathematical structure of the model, often expressed as a system of ordinary differential equations, to estimate the parameters. This process poses significant challenges due to the necessity for a detailed understanding of the underlying biological mechanisms. In this paper, we present an approach for estimating model parameters and assessing their identifiability in situations where only partial knowledge of the system structure is available. The partially known model is extended into a system of Hybrid Neural Ordinary Differential Equations, which captures the unknown portions of the system using neural networks. Integrating neural networks into the model structure introduces two primary challenges for parameter estimation: the need to globally explore the search space while employing gradient-based optimization, and the assessment of parameter identifiability, which may be hindered by the expressive nature of neural networks. To overcome the first issue, we treat biological parameters as hyperparameters in the extended model, exploring the parameter search space during hyperparameter tuning. The second issue is then addressed by an a posteriori analysis of parameter identifiability, computed by introducing a variant of a well-established approach for mechanistic models. These two components are integrated into an end-to-end pipeline that is thoroughly described in the paper. We assess the effectiveness of the proposed workflow on test cases derived from three different benchmark models. These test cases have been designed to mimic real-world conditions, including the presence of noise in the training data and various levels of data availability for the system variables. Author summary Parameter estimation is a central challenge in modeling biological systems. Typically, scientists calibrate the parameters by aligning model predictions with measured data once the model structure is defined. Our paper introduces a workflow that leverages the integration between mechanistic modeling and machine learning to estimate model parameters when the model structure is not fully known. We focus mainly on analyzing the identifiability of the model parameters, which measures how confident we can be in the parameter estimates given the available experimental data and partial mechanistic understanding of the system. We assessed the effectiveness of our approach in various in silico scenarios. Our workflow represents a first step to adapting traditional methods used in fully mechanistic models to the scenario of hybrid modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a064c473effb10441bd8fe1c6bde24ed0162e1e3" target='_blank'>
              Robust parameter estimation and identifiability analysis with Hybrid Neural Ordinary Differential Equations in Computational Biology
              </a>
            </td>
          <td>
            Stefano Giampiccolo, Federico Reali, Anna Fochesato, Giovanni Iacca, Luca Marchetti
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Dynamic Positron Emission Tomography (dPET) imaging and Time-Activity Curve (TAC) analyses are essential for understanding and quantifying the biodistribution of radiopharmaceuticals over time and space. Traditional compartmental modeling, while foundational, commonly struggles to fully capture the complexities of biological systems, including non-linear dynamics and variability. This study introduces an innovative data-driven neural network-based framework, inspired by Reaction Diffusion systems, designed to address these limitations. Our approach, which adaptively fits TACs from dPET, enables the direct calibration of diffusion coefficients and reaction terms from observed data, offering significant improvements in predictive accuracy and robustness over traditional methods, especially in complex biological scenarios. By more accurately modeling the spatio-temporal dynamics of radiopharmaceuticals, our method advances modeling of pharmacokinetic and pharmacodynamic processes, enabling new possibilities in quantitative nuclear medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7e97f4a77c199435d67bf7917712e6bcfb5bc2" target='_blank'>
              Beyond Conventional Parametric Modeling: Data-Driven Framework for Estimation and Prediction of Time Activity Curves in Dynamic PET Imaging
              </a>
            </td>
          <td>
            N. Zakariaei, Arman Rahmim, Eldad Haber
          </td>
          <td>2024-05-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Advances in molecular technologies underlie an enormous growth in the size of data sets pertaining to biology and biomedicine. These advances parallel those in the deep learning subfield of machine learning. Components in the differentiable programming toolbox that makes deep learning possible are allowing computer scientists to address an increasingly large array of problems with flexible and effective tools. However many of these tools have not fully proliferated into the computational biology and bioinformatics fields. In this perspective we survey some of these advances and highlight exemplary examples of their utilization in the biosciences, with the goal of increasing awareness among practitioners of emerging opportunities to blend expert knowledge with newly emerging deep learning architectural tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dab377c45c143c62caa8200f42e162d634ff1e2" target='_blank'>
              Opportunities in deep learning methods development for computational biology
              </a>
            </td>
          <td>
            Alex J. Lee, Reza Abbasi-Asl
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1f54608476ba4ebd37c1c1c7ed98f64403bee2f" target='_blank'>
              A comprehensive review of computational cell cycle models in guiding cancer treatment strategies
              </a>
            </td>
          <td>
            Chenhui Ma, Evren Gurkan-Cavusoglu
          </td>
          <td>2024-07-05</td>
          <td>npj Systems Biology and Applications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Modeling the dynamics of flexible objects has become an emerging topic in the community as these objects become more present in many applications, e.g., soft robotics. Due to the properties of flexible materials, the movements of soft objects are often highly nonlinear and, thus, complex to predict. Data-driven approaches seem promising for modeling those complex dynamics but often neglect basic physical principles, which consequently makes them untrustworthy and limits generalization. To address this problem, we propose a physics-constrained learning method that combines powerful learning tools and reliable physical models. Our method leverages the data collected from observations by sending them into a Gaussian process that is physically constrained by a distributed Port-Hamiltonian model. Based on the Bayesian nature of the Gaussian process, we not only learn the dynamics of the system, but also enable uncertainty quantification. Furthermore, the proposed approach preserves the compositional nature of Port-Hamiltonian systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020e63b8b17cdec54c056d8a1edc98770c8fa7ab" target='_blank'>
              Physics-constrained learning for PDE systems with uncertainty quantified port-Hamiltonian models
              </a>
            </td>
          <td>
            Kaiyuan Tan, Peilun Li, Thomas Beckers
          </td>
          <td>2024-06-17</td>
          <td>DBLP, ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In a landscape where scientific discovery is increasingly driven by data, the integration of machine learning (ML) with traditional scientific methodologies has emerged as a transformative approach. This paper introduces a novel, data-driven framework that synergizes physics-based priors with advanced ML techniques to address the computational and practical limitations inherent in first-principle-based methods and brute-force machine learning methods. Our framework showcases four algorithms, each embedding a specific physics-based prior tailored to a particular class of nonlinear systems, including separable and nonseparable Hamiltonian systems, hyperbolic partial differential equations, and incompressible fluid dynamics. The intrinsic incorporation of physical laws preserves the system's intrinsic symmetries and conservation laws, ensuring solutions are physically plausible and computationally efficient. The integration of these priors also enhances the expressive power of neural networks, enabling them to capture complex patterns typical in physical phenomena that conventional methods often miss. As a result, our models outperform existing data-driven techniques in terms of prediction accuracy, robustness, and predictive capability, particularly in recognizing features absent from the training set, despite relying on small datasets, short training periods, and small sample sizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba424643899decf4b48792c380cc0c66abe63c7" target='_blank'>
              Data-Driven Computing Methods for Nonlinear Physics Systems with Geometric Constraints
              </a>
            </td>
          <td>
            Yunjin Tong
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Detecting dynamic patterns of task-specific responses shared across heterogeneous datasets is an essential and challenging problem in many scientific applications in medical science and neuroscience. In our motivating example of rodent electrophysiological data, identifying the dynamical patterns in neuronal activity associated with ongoing cognitive demands and behavior is key to uncovering the neural mechanisms of memory. One of the greatest challenges in investigating a cross-subject biological process is that the systematic heterogeneity across individuals could significantly undermine the power of existing machine learning methods to identify the underlying biological dynamics. In addition, many technically challenging neurobiological experiments are conducted on only a handful of subjects where rich longitudinal data are available for each subject. The low sample sizes of such experiments could further reduce the power to detect common dynamic patterns among subjects. In this paper, we propose a novel heterogeneous data integration framework based on optimal transport to extract shared patterns in complex biological processes. The key advantages of the proposed method are that it can increase discriminating power in identifying common patterns by reducing heterogeneity unrelated to the signal by aligning the extracted latent spatiotemporal information across subjects. Our approach is effective even with a small number of subjects, and does not require auxiliary matching information for the alignment. In particular, our method can align longitudinal data across heterogeneous subjects in a common latent space to capture the dynamics of shared patterns while utilizing temporal dependency within subjects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ec1c357c0fb04d4fe9b61bfbd42d9d7f0cc948" target='_blank'>
              Optimal Transport for Latent Integration with An Application to Heterogeneous Neuronal Activity Data
              </a>
            </td>
          <td>
            Yubai Yuan, B. Shahbaba, Norbert Fortin, Keiland Cooper, Qing Nie, Annie Qu
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Various computational models have been developed to understand the physiological effects of drug-drug interactions, which can contribute to more effective drug treatments. However, they mostly focus on interactions of only two drugs, and do not consider the patient information. To address this challenge, we use publicly available electronic health record (EHR), MIMIC-IV, to develop machine learning models that predict the physiological effects of two or more drugs. This study involves extensive preprocessing of laboratory measurement data, prescription data and patient data. The resulting machine learning models predict potential abnormalities across 20 selected measurement items (e.g., concentrations of metabolites and blood cells) in the form of a sentence. The models showed an average AUROC of 0.75, and age, specific active pharmaceutical ingredients, and gender appeared to be the most influential features. The model development process showcased in this study can be extended to other measurement items for a target EHR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eedc9ef3564438d393987221af43b370a584e0b0" target='_blank'>
              Predicting the physiological effects of multiple drugs using electronic health record
              </a>
            </td>
          <td>
            Junhyeok Jeon, Eujin Hong, Jong-Yeup Kim, Suehyun Lee, Hyun Uk Kim
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Most approaches for assessing causality in complex dynamical systems fail when the interactions between variables are inherently non-linear and non-stationary. Here we introduce Temporal Autoencoders for Causal Inference (TACI), a methodology that combines a new surrogate data metric for assessing causal interactions with a novel two-headed machine learning architecture to identify and measure the direction and strength of time-varying causal interactions. Through tests on both synthetic and real-world datasets, we demonstrate TACI's ability to accurately quantify dynamic causal interactions across a variety of systems. Our findings display the method's effectiveness compared to existing approaches and also highlight our approach's potential to build a deeper understanding of the mechanisms that underlie time-varying interactions in physical and biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d22ad3492ed41cdaeed3d49e84841e11089e47" target='_blank'>
              Inferring the time-varying coupling of dynamical systems with temporal convolutional autoencoders
              </a>
            </td>
          <td>
            Josuan Calderon, Gordon J. Berman
          </td>
          <td>2024-06-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose: Artificial Intelligence (AI), which mimics the human brain structure and operation, simulates intelligence. The aim of Machine Learning (ML), which is a branch of artificial intelligence, is to create models by analyzing data. Another type of artificial intelligence, Deep Learning (DL), depicts geometric changes using several layers of model representations. Since DL broke the computational analysis record, AI has advanced in many areas. 
Materials and Methods: Contrary to the widespread use of conventional ML methodologies, there is still a need to promote the use and popularity of DL for pharmaceutical research and development. Drug discovery and design have been enhanced by ML and DL in major research projects. To fully realize its potential, drug design must overcome many challenges and issues. Various aspects of medication design must be considered to successfully address these concerns and challenges. This review article explains DL's significance both in technological breakthroughs and in effective medications. 
Results: There are numerous barriers and substantial challenges associated with drug design associated with DL architectures and key application domains. The article discusses several elements of medication development that have been influenced by existing research. Two widely used and efficient Neural Network (NN) designs are discussed in this article: Convolutional Neural Networks (CNNs) and Recurrent Neural Networks (RNNs). 
Conclusion: It is described how these tools can be utilized to design and discover small molecules for drug discovery. They are also given an overview of the history of DL approaches, as well as a discussion of some of their drawbacks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0fdca353e21ebc2e1d16a9b8425165c55ad670" target='_blank'>
              Deep Learning in Drug Design—Progress, Methods, and Challenges
              </a>
            </td>
          <td>
            Seyyed Saeid Masoomkhah, Khosro Rezaee, Mojtaba Ansari, Hossein Eslami
          </td>
          <td>2024-06-24</td>
          <td>Frontiers in Biomedical Technologies</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The rapid evolution of single-cell sequencing technologies has facilitated precise transcriptomics profiling at the single-cell level, shedding light on the intricate heterogeneity within cellular populations. Despite these advances, the inherent diversity of cells and data challenges such as noise, batch effects, and sparsity, underscores the pressing need for a unified model to learn and represent cellular states effectively. Single-cell Large Language Models (LLMs) have been crafted to bridge this gap yet exhibit limited performance on human cells. This short-fall may stem from the confounding effects of training data from diverse species, partly because of limited cells for the single species. Here, we have compiled a dataset of approximately 100 million human cells sequenced by multiple technolo-gies from human single-cell datasets with various file types deposited in public databases and websites. Leveraging these extensive data cohorts, we developed CellFM, a robust single-cell foundation model with an impressive 800 million parameters, marking an eight-fold increase over the current largest single-species model. To ensure the training of CellFM on the MindSpore AI framework from Huawei, we have integrated RetNet, a Transformer architecture variant with lin-ear complexity for a balance between efficiency and performance, serving as the backbone of our model. Our comprehensive experiments have shown that CellFM outperforms existing models across diverse applications, such as cell annotation, perturbation prediction, and gene function prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd45aed312f5393a6aaf2e612c46c292cadd436e" target='_blank'>
              CellFM: a large-scale foundation model pre-trained on transcriptomics of 100 million human cells
              </a>
            </td>
          <td>
            Yuansong Zeng, Jiancong Xie, Zhuoyi Wei, Yun Su, Ningyuan Shangguan, Shuangyu Yang, Chengyang Zhang, Wenbing Li, Jinbo Zhang, Nan Fang, Hongyu Zhang, Huiying Zhao, Yutong Lu, Jue Fan, Weijiang Yu, Yuedong Yang
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Autoimmune rheumatic diseases are often characterised by heterogeneity in presentation. The traditional approach to diseases guided by their phenotype may be suboptimal with the advent of precision medicine. Precision medicine is the integration and application of multiomics to predict the best-performing drug and its toxicity profile to derive optimal benefits. With novel drug discoveries and an expanding therapeutic armamentarium, it potentially aids in clinical and therapeutic decision-making, while saving time and averting adverse events. However, multiomics comes with ‘big data’, and owing to the costs, the sample size is usually small. Machine learning (ML) plays an important role in these scenarios where conventional statistics fall short. So, by integrating clinical data with the data from -omics, ML models can be built, which can accurately predict the clinical factors or even novel biomarkers that predict response. This approach has a potential for great benefit as valuable time or the ‘therapeutic window of opportunity’ would be saved, with fewer adverse events, eventually translating to lower damage accrual and better outcomes. Most of the evidence for the use of ML in precision rheumatology comes from rheumatoid arthritis and the factors predicting response to various drugs, including tumour necrosis factor inhibitors. This approach also has its limitations such as the lack of generalizability and the current scarcity of longitudinal data. These models must be tested in larger cohorts and population-based studies for validation, failing which there is a risk of apparent identification of multiple ‘novel’ biomarkers that may or may not be mechanistic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f607769c09b08a0d2145eee4ed39d8f624f5aad6" target='_blank'>
              Current Status of Machine Learning for Precision Rheumatology: Are We There Yet?
              </a>
            </td>
          <td>
            Prakashini Mruthyunjaya, Avarna Agarwal, Sakir Ahmed
          </td>
          <td>2024-06-16</td>
          <td>Indian Journal of Rheumatology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Deep learning approaches have demonstrated remarkable potential in predicting cancer drug responses (CDRs), using cell line and drug features. However, existing methods predominantly rely on single-omics data of cell lines, potentially overlooking the complex biological mechanisms governing cell line responses. This paper introduces DeepFusionCDR, a novel approach employing unsupervised contrastive learning to amalgamate multi-omics features, including mutation, transcriptome, methylome, and copy number variation data, from cell lines. Furthermore, we incorporate molecular SMILES-specific transformers to derive drug features from their chemical structures. The unified multi-omics and drug signatures are combined, and a multi-layer perceptron (MLP) is applied to predict IC50 values for cell line-drug pairs. Moreover, this MLP can discern whether a cell line is resistant or sensitive to a particular drug. We assessed DeepFusionCDR's performance on the GDSC dataset and juxtaposed it against cutting-edge methods, demonstrating its superior performance in regression and classification tasks. We also conducted ablation studies and case analyses to exhibit the effectiveness and versatility of our proposed approach. Our results underscore the potential of DeepFusionCDR to enhance CDR predictions by harnessing the power of multi-omics fusion and molecular-specific transformers. The prediction of DeepFusionCDR on TCGA patient data and case study highlight the practical application scenarios of DeepFusionCDR in real-world environments. Source code and datasets can be available on https://github.com/altriavin/DeepFusionCDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55237e128fd3db51c88289d302cb3a3283ce281e" target='_blank'>
              DeepFusionCDR: Employing Multi-Omics Integration and Molecule-Specific Transformers for Enhanced Prediction of Cancer Drug Responses.
              </a>
            </td>
          <td>
            Xiaowen Hu, Pan Zhang, Jiaxuan Zhang, Lei Deng
          </td>
          <td>2024-06-27</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="e13603 Background: Oncology therapeutic development has high failure rates with substantial costs. Advances in technology, including molecular characterisations of cancer and computational power, provide the opportunity to better model therapeutic response for existing and emerging therapies. Methods: Concr adapted Bayesian statistical principles used by astrophysicists to effectively integrate multi-dimensional data (genomic & transcriptomic) for accurate modelling of cancer biology. Our proprietary computational framework was applied to predict clinical response and survival through construction of digital twins and in silico simulations of clinical trials. Predicted log odds ratios (LOR) were generated for drug response and compared to published data from 8 suitable historical clinical trials. Three distinct models underpin this framework: Drug Efficacy, Treatment Response & Overall Survival (OS). Three publicly available datasets were used for training: Cancer Therapeutic Response Portal ( in vitro dose-response data), Cancer Cell Line Encyclopaedia and TCGA. Results: For all 8 clinical studies, the digital twin model accurately simulated both trial arms, compared drug efficacy across arms and predicted which treatment was most active. Unblinded evaluation: Our model accurately predicted gemcitabine would have greater clinical benefit than 5-FU (LOR -0.10, P <0.0001) in metastatic pancreatic cancer. In advanced breast cancer, the model predicted docetaxel had a higher response rate than doxorubicin, and carboplatin response rates were similar to docetaxel in BRCA wild-type patients. In platinum-sensitive recurrent ovarian cancer, the model predicted cisplatin-based combination had a higher response rate than paclitaxel. Blinded evaluation: Using data for paclitaxel, the model correctly predicted that nab-paclitaxel+gemcitabine response rates were higher than gemcitabine in metastatic pancreatic cancer (predicted LOR -0.090, p = <0.001). In all 3 early breast cancer studies, predictions mirrored clinical trial results, adjuvant ECF > CMF, adjuvant TC > AC and adjuvant tamoxifen was superior to capecitabine. The underlying pan-tumour Overall Survival model used a Random Survival Forest approach, trained on 23 cancers with predictive accuracy of 0.78 AUC ROC and C-index 0.71. To enable cohort enrichment for drug response, the model segmented patient cohorts into treatment responder and non-responder. Using this approach, the log odds increased for drug response rate for 16 of the 19 different cancers and 14 of the different 17 drugs. Conclusions: Our digital twin model represents transformative technology for clinical trial simulation with strong potential for accelerating and de-risking clinical drug development, and utility for synthetic control arms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a659d949c4ba645ce412729470f4959a538035" target='_blank'>
              Prediction of therapeutic response and cancer outcomes in solid tumours via in silico clinical trials.
              </a>
            </td>
          <td>
            Uzma Asghar, M. Griffiths, Amanzhol Kubeyev, Jordan Laurie, Andrea Giorni, Luiz A Zillmann da Silva, Prabu Sivasubramaniam, Matthew Foster, Andrew Biankin
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Circulating tumor DNA assays are promising tools for the prediction of cancer treatment response. Here, we build a framework for the design of ctDNA biomarkers of therapy response that incorporate variations in ctDNA dynamics driven by specific treatment mechanisms. We develop mathematical models of ctDNA kinetics driven by tumor response to several therapy classes, and utilize them to simulate randomized virtual patient cohorts to test candidate biomarkers. Using this approach, we propose specific biomarkers, based on ctDNA longitudinal features, for targeted therapy, chemotherapy and radiation therapy. We evaluate and demonstrate the efficacy of these biomarkers in predicting treatment response within a randomized virtual patient cohort dataset. These biomarkers are based on novel proposals for ctDNA sampling protocols, consisting of frequent sampling within a compact time window surrounding therapy initiation – which we hypothesize to hold valuable prognostic information on longer-term treatment response. This study highlights a need for tailoring ctDNA sampling protocols and interpretation methodology to specific biological mechanisms of therapy response, and it provides a novel modeling and simulation framework for doing so. In addition, it highlights the potential of ctDNA assays for making early, rapid predictions of treatment response within the first days or weeks of treatment, and generates hypotheses for further clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2935b3131b17e4bcb531bdaac6cb25d84901bf3f" target='_blank'>
              Early ctDNA kinetics as a dynamic biomarker of cancer treatment response
              </a>
            </td>
          <td>
            Aaron Li, Emil Lou, K. Leder, Jasmine Foo
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Paired single-cell sequencing technologies enable the simultaneous measurement of complementary modalities of molecular data at single-cell resolution. Along with the advances in these technologies, many methods based on variational autoencoders have been developed to integrate these data. However, these methods do not explicitly incorporate prior biological relationships between the data modalities, which could significantly enhance modeling and interpretation. We propose a novel probabilistic learning framework that explicitly incorporates conditional independence relationships between multi-modal data as a directed acyclic graph using a generalized hierarchical variational autoencoder. We demonstrate the versatility of our framework across various applications pertinent to single-cell multi-omics data integration. These include the isolation of common and distinct information from different modalities, modality-specific differential analysis, and integrated cell clustering. We anticipate that the proposed framework can facilitate the construction of highly flexible graphical models that can capture the complexities of biological hypotheses and unravel the connections between different biological data types, such as different modalities of paired single-cell multi-omics data. The implementation of the proposed framework can be found in the repository https://github.com/kuijjerlab/CAVACHON.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5edcd7da413e72fd7c1ca0c333150327939b18" target='_blank'>
              CAVACHON: a hierarchical variational autoencoder to integrate multi-modal single-cell data
              </a>
            </td>
          <td>
            Ping-Han Hsieh, Ru-Xiu Hsiao, Katalin Ferenc, Anthony Mathelier, R. Burkholz, Chien-Yu Chen, G. K. Sandve, T. Belova, M. Kuijjer
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Deep generative models are becoming a tool of choice for exploring the molecular space. One important application area of deep generative models is the reverse design of drug compounds for given attributes (solubility, ease of synthesis, etc.). Although there are many generative models, these models cannot generate specific intervals of attributes. This paper proposes a AC-ModNet model that effectively combines VAE with AC-GAN to generate molecular structures in specific attribute intervals. The AC-ModNet is trained and evaluated using the open 250K ZINC dataset. In comparison with related models, our method performs best in the FCD and Frag model evaluation indicators. Moreover, we prove the AC-ModNet created molecules have potential application value in drug design by comparing and analyzing them with medical records in the PubChem database. The results of this paper will provide a new method for machine learning drug reverse design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2a0c9044c8aa0c08c17758aa4b9b3d3d82c4c7b" target='_blank'>
              AC-ModNet: Molecular Reverse Design Network Based on Attribute Classification
              </a>
            </td>
          <td>
            Wei Wei, Jun Fang, Ning Yang, Qi Li, Lin Hu, Lanbo Zhao, Jie Han
          </td>
          <td>2024-06-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract A major challenge of precision oncology is the identification and prioritization of suitable treatment options based on molecular biomarkers of the considered tumor. In pursuit of this goal, large cancer cell line panels have successfully been studied to elucidate the relationship between cellular features and treatment response. Due to the high dimensionality of these datasets, machine learning (ML) is commonly used for their analysis. However, choosing a suitable algorithm and set of input features can be challenging. We performed a comprehensive benchmarking of ML methods and dimension reduction (DR) techniques for predicting drug response metrics. Using the Genomics of Drug Sensitivity in Cancer cell line panel, we trained random forests, neural networks, boosting trees and elastic nets for 179 anti-cancer compounds with feature sets derived from nine DR approaches. We compare the results regarding statistical performance, runtime and interpretability. Additionally, we provide strategies for assessing model performance compared with a simple baseline model and measuring the trade-off between models of different complexity. Lastly, we show that complex ML models benefit from using an optimized DR strategy, and that standard models—even when using considerably fewer features—can still be superior in performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2eaacf9f8df2242ab429c53e183de03dc2d3222" target='_blank'>
              A comprehensive benchmarking of machine learning algorithms and dimensionality reduction methods for drug sensitivity prediction
              </a>
            </td>
          <td>
            Lea Eckhart, Kerstin Lenhof, Lisa-Marie Rolli, H-P. Lenhof
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1491d337e12daab70edf38ee62d8f8ee586b8e98" target='_blank'>
              Learning nonlinear operators in latent spaces for real-time predictions of complex dynamics in physical systems
              </a>
            </td>
          <td>
            Katiana Kontolati, S. Goswami, G. Em Karniadakis, Michael D Shields
          </td>
          <td>2024-06-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84a5d0b1c26573809df096903d64fc1da14e7531" target='_blank'>
              Advancing cancer drug development with mechanistic mathematical modeling: bridging the gap between theory and practice.
              </a>
            </td>
          <td>
            Alexander Kulesza, C. Couty, Paul Lemarre, Craig J Thalhauser, Yanguang Cao
          </td>
          <td>2024-06-21</td>
          <td>Journal of pharmacokinetics and pharmacodynamics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="After the coronavirus disease 2019 (COVID-19) outbreak erupted, it swiftly spread globally and triggered a severe public health crisis in 2019. To contain the virus’s spread, several countries implemented various lockdown measures. As the governments faced this unprecedented challenge, understanding the impact of lockdown policies became paramount. The goal of addressing the pandemic crisis is to devise prudent policies that strike a balance between safeguarding lives and maintaining economic stability. Traditional mathematical and statistical models for studying virus transmission only offer macro-level predictions of epidemic development and often overlook individual variations’ impact, therefore failing to reflect the role of government decisions. To address this challenge, we propose an integrated framework that combines agent-based modeling (ABM) and deep Q-network (DQN) techniques. This framework enables a more comprehensive analysis and optimization of epidemic control strategies while considering real human behavior. We construct a pandemic simulator based on the ABM method, accurately simulating agents’ daily activities, interactions, and the dynamic spread of the virus. Additionally, we employ a data-driven approach and adjust the model through real statistical data to enhance its effectiveness. Subsequently, we integrated ABM into a decision-making framework using reinforcement learning techniques to explore the most effective strategies. In experiments, we validated the model’s effectiveness by simulating virus transmission across different countries globally. In this model, we obtained decision outcomes when governments focused on various factors. Our research findings indicate that our model serves as a valuable tool for decision-makers, enabling them to formulate prudent and rational policies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce211bae4071f08f917efe2eb6cd613c89e2263" target='_blank'>
              A Data-Driven Pandemic Simulator with Reinforcement Learning
              </a>
            </td>
          <td>
            Yuting Zhang, Biyang Ma, Langcai Cao, Yanyu Liu
          </td>
          <td>2024-06-27</td>
          <td>Electronics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract The inherent heterogeneity of cancer contributes to highly variable responses to any anticancer treatments. This underscores the need to first identify precise biomarkers through complex multi-omics datasets that are now available. Although much research has focused on this aspect, identifying biomarkers associated with distinct drug responders still remains a major challenge. Here, we develop MOMLIN, a multi-modal and -omics machine learning integration framework, to enhance drug-response prediction. MOMLIN jointly utilizes sparse correlation algorithms and class–specific feature selection algorithms, which identifies multi-modal and -omics–associated interpretable components. MOMLIN was applied to 147 patients’ breast cancer datasets (clinical, mutation, gene expression, tumor microenvironment cells and molecular pathways) to analyze drug-response class predictions for non-responders and variable responders. Notably, MOMLIN achieves an average AUC of 0.989, which is at least 10% greater when compared with current state-of-the-art (data integration analysis for biomarker discovery using latent components, multi-omics factor analysis, sparse canonical correlation analysis). Moreover, MOMLIN not only detects known individual biomarkers such as genes at mutation/expression level, most importantly, it correlates multi-modal and -omics network biomarkers for each response class. For example, an interaction between ER-negative-HMCN1-COL5A1 mutations-FBXO2-CSF3R expression-CD8 emerge as a multimodal biomarker for responders, potentially affecting antimicrobial peptides and FLT3 signaling pathways. In contrast, for resistance cases, a distinct combination of lymph node-TP53 mutation-PON3-ENSG00000261116 lncRNA expression-HLA-E-T-cell exclusions emerged as multimodal biomarkers, possibly impacting neurotransmitter release cycle pathway. MOMLIN, therefore, is expected advance precision medicine, such as to detect context–specific multi-omics network biomarkers and better predict drug-response classifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c9478bdc170073aa91d05881b2bed54efbf90f" target='_blank'>
              Advancing drug-response prediction using multi-modal and -omics machine learning integration (MOMLIN): a case study on breast cancer clinical data
              </a>
            </td>
          <td>
            Md Mamunur Rashid, Kumar Selvarajoo
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVE
The aim of this project was to create time-aware, individual-level risk score models for adverse drug events related to multiple sclerosis disease-modifying therapy and to provide interpretable explanations for model prediction behavior.


MATERIALS AND METHODS
We used temporal sequences of observational medical outcomes partnership common data model (OMOP CDM) concepts derived from an electronic health record as model features. Each concept was assigned an embedding representation that was learned from a graph convolution network trained on a knowledge graph (KG) of OMOP concept relationships. Concept embeddings were fed into long short-term memory networks for 1-year adverse event prediction following drug exposure. Finally, we implemented a novel extension of the local interpretable model agnostic explanation (LIME) method, knowledge graph LIME (KG-LIME) to leverage the KG and explain individual predictions of each model.


RESULTS
For a set of 4859 patients, we found that our model was effective at predicting 32 out of 56 adverse event types (P < .05) when compared to demographics and past diagnosis as variables. We also assessed discrimination in the form of area under the curve (AUC = 0.77 ± 0.15) and area under the precision-recall curve (AUC-PR = 0.31 ± 0.27) and assessed calibration in the form of Brier score (BS = 0.04 ± 0.04). Additionally, KG-LIME generated interpretable literature-validated lists of relevant medical concepts used for prediction.


DISCUSSION AND CONCLUSION
Many of our risk models demonstrated high calibration and discrimination for adverse event prediction. Furthermore, our novel KG-LIME method was able to utilize the knowledge graph to highlight concepts that were important to prediction. Future work will be required to further explore the temporal window of adverse event occurrence beyond the generic 1-year window used here, particularly for short-term inpatient adverse events and long-term severe adverse events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1966744286d710728ee88bd61e935e315ec74e5" target='_blank'>
              KG-LIME: predicting individualized risk of adverse drug events for multiple sclerosis disease-modifying therapy.
              </a>
            </td>
          <td>
            Jason Patterson, Nicholas Tatonetti
          </td>
          <td>2024-07-04</td>
          <td>Journal of the American Medical Informatics Association : JAMIA</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="EpiLearn is a Python toolkit developed for modeling, simulating, and analyzing epidemic data. Although there exist several packages that also deal with epidemic modeling, they are often restricted to mechanistic models or traditional statistical tools. As machine learning continues to shape the world, the gap between these packages and the latest models has become larger. To bridge the gap and inspire innovative research in epidemic modeling, EpiLearn not only provides support for evaluating epidemic models based on machine learning, but also incorporates comprehensive tools for analyzing epidemic data, such as simulation, visualization, transformations, etc. For the convenience of both epidemiologists and data scientists, we provide a unified framework for training and evaluation of epidemic models on two tasks: Forecasting and Source Detection. To facilitate the development of new models, EpiLearn follows a modular design, making it flexible and easy to use. In addition, an interactive web application is also developed to visualize the real-world or simulated epidemic data. Our package is available at https://github.com/Emory-Melody/EpiLearn.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e0a4bf71b4cf7906c2ed42050b45a3e4b7e895" target='_blank'>
              EpiLearn: A Python Library for Machine Learning in Epidemic Modeling
              </a>
            </td>
          <td>
            Zewen Liu, Yunxiao Li, Mingyang Wei, Guancheng Wan, Max S. Y. Lau, Wei Jin
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In this work, we introduce a novel application of foundation models in the domain of nonlinear chemical process modeling. Given the challenges of obtaining accurate first-principles models for real-world chemical processes and the inefficiency of rebuilding and retraining models for new chemical processes, we pose a pivotal question: What if we could develop a single, universal neural network (i.e., foundation model) capable of rapidly adapting to modeling any new chemical process? To address this question, we propose a meta-learning-based approach using Reptile to construct the foundation model, followed by physics-informed adaptation to fine-tune it to new modeling tasks using only a few data samples. To assess the effectiveness of our methodology, we construct a foundation model for various chemical reactions in three classical generic reactors, including continuous stirred tank reactors (CSTRs), batch reactors (BRs), and plug flow reactors (PFRs). Our approach outperforms conventional methods such as data-driven learning, physics-informed learning, transfer learning, and pure meta-learning in a few-shot setting. Furthermore, our method achieves rapid adaptation to new CSTRs, BRs, and PFRs using only a few data samples from the designated tasks. Source code is available at https://github.com/killingbear999/chemical-process-foundation-model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4edf55b1063ea6f1fd059221cbe47191bb3da351" target='_blank'>
              Foundation Model for Chemical Process Modeling: Meta-Learning with Physics-Informed Adaptation
              </a>
            </td>
          <td>
            Zihao Wang, Zhen Wu
          </td>
          <td>2024-05-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In the current field of clinical medicine, traditional methods for analyzing recurrent events have limitations when dealing with complex time-dependent data. This study combines Long Short-Term Memory networks (LSTM) with the Cox model to enhance the model's performance in analyzing recurrent events with dynamic temporal information. Compared to classical models, the LSTM-Cox model significantly improves the accuracy of extracting clinical risk features and exhibits lower Akaike Information Criterion (AIC) values, while maintaining good performance on simulated datasets. In an empirical analysis of bladder cancer recurrence data, the model successfully reduced the mean squared error during the training phase and achieved a Concordance index of up to 0.90 on the test set. Furthermore, the model effectively distinguished between high and low-risk patient groups, and the identified recurrence risk features such as the number of tumor recurrences and maximum size were consistent with other research and clinical trial results. This study not only provides a straightforward and efficient method for analyzing recurrent data and extracting features but also offers a convenient pathway for integrating deep learning techniques into clinical risk prediction systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e83aa2b5db279cd8e060c5ac6ab858f591cf20e" target='_blank'>
              LSTM-COX Model: A Concise and Efficient Deep Learning Approach for Handling Recurrent Events
              </a>
            </td>
          <td>
            Runquan Huang, Xiaoping Shi
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains associated with a dismal prognosis despite standard therapies. While population-level survival statistics are established, generating individualized prognosis remains challenging. We aim to develop machine learning (ML) models that generate personalized survival predictions for GBM patients to enhance prognostication.



 Adult patients with histologically confirmed IDH-wildtype GBM from the National Cancer Database (NCDB) were analyzed. ML models were developed with TabPFN, TabNet, XGBoost, LightGBM, and Random Forest algorithms to predict mortality at 6, 12, 18, and 24 months post-diagnosis. SHapley Additive exPlanations (SHAP) were employed to enhance the interpretability of the models. Models were primarily evaluated using the area under the receiver operating characteristic (AUROC) values, and the top-performing models indicated by the highest AUROCs for each outcome were deployed in a web application that was created for individualized predictions.



 A total of 7,537 patients were retrieved from the NCDB. Performance evaluation revealed the top-performing models for each outcome were built using the TabPFN algorithm. The TabPFN models yielded mean AUROCs of 0.836, 0.78, 0.732, and 0.724 in predicting 6-, 12-, 18-, and 24-month mortality, respectively.



 This study establishes ML models tailored to individual patients to enhance GBM prognostication. Future work should focus on external validation and dynamic updating as new data emerge.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dc32348f572bb38d92dcc404255ab597d191058" target='_blank'>
              Advancing Precision Prognostication in Neuro-Oncology: Machine Learning Models for Data-Driven Personalized Survival Predictions in IDH-Wildtype Glioblastoma
              </a>
            </td>
          <td>
            Mert Karabacak, Pemla Jagtiani, L. Di, Ashish H Shah, Ricardo J Komotar, Konstantinos Margetis
          </td>
          <td>2024-06-11</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The development of models to predict sensitivity to anticancer drugs is an area of significant interest, given the diverse responses to treatment among patients and the considerable expense and time involved in anticancer drug development. Leveraging “omic” data and anticancer response information from the Cancer Cell Line Encyclopedia, we propose a novel approach utilizing multitask learning to enhance prediction accuracy and inference. We extended a multitask learning framework called the Data Shared Lasso to develop the Data Shared Elastic Net. This enabled the construction of tissue-specific models with information sharing while maintaining the attractive properties of Elastic Net regression. By employing this approach, we observed improvements in prediction accuracy compared to single-task Elastic Net models, particularly for cell lines displaying high sensitivity to treatment. Furthermore, the Data Shared Elastic Net facilitated the identification of predictors for anticancer drug sensitivity within specific tissue types, shedding light on cellular pathways targeted by these drugs across tissues. We also investigated the impact of data leakage on modeling outcomes from previous studies, which led to underestimating prediction error and erroneous inferences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83664bfa905ff011a2bcad47d103dd20fcdcd259" target='_blank'>
              Improving Drug Sensitivity Prediction and Inference by Multi-Task Learning
              </a>
            </td>
          <td>
            Jared Strauch, Amir Asiaee
          </td>
          <td>2024-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Therapeutic interventions are designed to perturb the function of a biological system. However, there are many types of proteins that cannot be targeted with conventional small molecule drugs. Accordingly, many identified gene-regulatory drivers and downstream effectors are currently undruggable. Drivers and effectors are often connected by druggable signaling and regulatory intermediates. Methods to identify druggable intermediates therefore have general value in expanding the set of targets available for hypothesis-driven validation. Here we identify and prioritize potential druggable intermediates by developing a network perturbation theory, termed NetPert, for response functions of biological networks. Dynamics are defined by a network structure in which vertices represent genes and proteins, and edges represent gene-regulatory interactions and protein-protein interactions. Perturbation theory for network dynamics prioritizes targets that interfere with signaling from driver to response genes. Applications to organoid models for metastatic breast cancer demonstrate the ability of this mathematical framework to identify and prioritize druggable intermediates. While the short-time limit of the perturbation theory resembles betweenness centrality, NetPert is superior in generating target rankings that correlate with previous wet-lab assays and are more robust to incomplete or noisy network data. NetPert also performs better than a related graph diffusion approach. Wet-lab assays demonstrate that drugs for targets identified by NetPert, including targets that are not themselves differentially expressed, are active in suppressing additional metastatic phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e0d901129a82304d94972066bb691c9c95e075f" target='_blank'>
              Prioritizing drug targets by perturbing biological network response functions.
              </a>
            </td>
          <td>
            Matthew C Perrone, Michael G Lerner, Matthew Dunworth, Andrew J. Ewald, Joel S Bader
          </td>
          <td>2024-06-27</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In the trend of hybrid Artificial Intelligence (AI) techniques, Physic Informed Machine Learning has seen a growing interest. It operates mainly by imposing a data, learning or inductive bias with simulation data, Partial Differential Equations or equivariance and invariance properties. While these models have shown great success on tasks involving one physical domain such as fluid dynamics, existing methods still struggle on tasks with complex multi-physical and multi-domain phenomena. To address this challenge, we propose to leverage Bond Graphs, a multi-physics modeling approach together with Graph Neural Network. We thus propose Neural Bond Graph Encoder (NBgE), a model agnostic physical-informed encoder tailored for multi-physics systems. It provides an unified framework for any multi-physics informed AI with a graph encoder readable for any deep learning model. Our experiments on two challenging multi-domain physical systems - a Direct Current Motor and the Respiratory system - demonstrate the effectiveness of our approach on a multi-variate time series forecasting task.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5d7393bdb45ad9ec2527ea020c7d99f6d06d06" target='_blank'>
              Bond Graphs for multi-physics informed Neural Networks for multi-variate time series
              </a>
            </td>
          <td>
            Alexis-Raja Brachet, Pierre-Yves Richard, C'eline Hudelot
          </td>
          <td>2024-05-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Survival prediction is an important branch of cancer prognosis analysis. The model that predicts survival risk through TCGA genomics data can discover genes related to cancer and provide diagnosis and treatment recommendations based on patient characteristics. We found that deep learning models based on Cox proportional hazards often suffer from overfitting when dealing with high-throughput data. Moreover, we found that as the number of network layers increases, the experimental results will not get better, and network degradation will occur. Based on this problem, we propose a new framework based on Deep Residual Learning. Combine the ideas of Cox proportional hazards and Residual. And name it ResSurv. First, ResSurv is a feed-forward deep learning network stacked by multiple basic ResNet Blocks. In each ResNet Block, we add a Normalization Layer to prevent gradient disappearance and gradient explosion. Secondly, for the loss function of the neural network, we inherited the Cox proportional hazards methods, applied the semi-parametric of the CPH model to the neural network, combined with the partial likelihood model, established the loss function, and performed backpropagation and gradient update. Finally, we compared ResSurv networks of different depths and found that we can effectively extract high-dimensional features. Ablation experiments and comparative experiments prove that our model has reached SOTA(state of the art) in the field of deep learning, and our network can effectively extract deep information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2cbfe31657721f4bf6c2ca44aa455fb795d7549" target='_blank'>
              ResSurv: Cancer Survival Analysis Prediction Model Based on Residual Networks
              </a>
            </td>
          <td>
            Wankang Zhai
          </td>
          <td>2024-05-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Physiological and pathological processes are governed by a network of genes called gene regulatory networks (GRNs). By reconstructing GRNs, we can accurately model how cells behave in their natural state and predict how genetic changes will affect them. Transcriptomic data of single cells are now available for a wide range of cellular processes in multiple species. Thus, a method building predictive GRNs from single-cell RNA sequencing (scRNA-seq) data, without any additional prior knowledge, could have a great impact on our understanding of biological processes and the genes playing a key role in them. To this aim, we developed IGNITE (Inference of Gene Networks using Inverse kinetic Theory and Experiments), an unsupervised machine learning framework designed to infer directed, weighted, and signed GRNs directly from unperturbed single-cell RNA sequencing data. IGNITE uses the GRNs to generate gene expression data upon single and multiple genetic perturbations. IGNITE is based on the inverse problem for a kinetic Ising model, a model from statistical physics that has been successfully applied to biological networks. We tested IGNITE on murine pluripotent stem cells (PSCs) transitioning from the naïve to formative states. Using as input only scRNA-seq data of unperturbed PSCs, IGNITE simulated single and triple gene knockouts. Comparison with experimental data revealed high accuracy, up to 74%, outperforming currently available methods. In sum, IGNITE identifies predictive GRNs from scRNA-seq data without additional prior knowledge and faithfully simulates single and multiple gene perturbations. Applications of IGNITE range from studying cell differentiation to identifying genes specifically active under pathological conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f91d1ee7feee1c23aecfd8641c38451501491d9" target='_blank'>
              Unveiling Gene Perturbation Effects through Gene Regulatory Networks Inference from single-cell transcriptomic data
              </a>
            </td>
          <td>
            Clelia Corridori, Merrit Romeike, Giorgio Nicoletti, C. Buecker, S. Suweis, S. Azaele, Graziano Martello
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancer remains one of the foremost challenges in medical research, necessitating diverse and sophisticated models to understand its complexity and develop effective treatments. This review explores the evolution and utility of experimental cancer models, highlighting their pivotal role in bridging the gap between basic research and clinical application. From the traditional use of xenografts, which provide a direct avenue for studying tumor growth and drug response in a living organism, to the innovative approaches of genetically engineered mouse models (GEMMs) that replicate human cancer's genetic and phenotypic traits, each model offers unique insights into cancer biology. Recent advances have introduced organoid models, offering a three-dimensional perspective that closely mimics the tumor's microenvironment, and computational models, which leverage patient-specific data to predict disease progression and treatment outcomes. These models enhance our understanding of cancer's molecular drivers, facilitate the development of targeted therapies, and underscore the importance of personalized medicine in oncology. Despite the diversity and potential of these experimental models, challenges remain, including the replication of the tumor's complexity and the integration of immune system interactions. Future research is directed toward refining these models, improving their predictive accuracy, and combining their strengths to offer a holistic view of cancer biology and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/757ed0c76cfcd427eee0ddc9e657a3423f46a092" target='_blank'>
              Traditional Approaches and Innovative Strategies in Laboratory Animal Models for Cancer Research: A Comprehensive Review
              </a>
            </td>
          <td>
            M. Erdoğan
          </td>
          <td>2024-05-13</td>
          <td>Düzce Tıp Fakültesi Dergisi</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="By using omics, we can now examine all components of biological systems simultaneously. Deep learning-based drug prediction methods have shown promise by integrating cancer-related multi-omics data. However, the complex interaction between genes poses challenges in accurately projecting multi-omics data. In this research, we present a predictive model for drug response that incorporates diverse types of omics data, comprising genetic mutation, copy number variation, methylation, and gene expression data. This study proposes latent alignment for information mismatch in integration, which is achieved through an attention module capturing interactions among diverse types of omics data. The latent alignment and attention modules significantly improve predictions, outperforming the baseline model, with MSE = 1.1333, F1-score = 0.5342, and AUROC = 0.5776. High accuracy was achieved in predicting drug responses for piplartine and tenovin-6, while the accuracy was comparatively lower for mitomycin-C and obatoclax. The latent alignment module exclusively outperforms the baseline model, enhancing the MSE by 0.2375, the F1-score by 4.84%, and the AUROC by 6.1%. Similarly, the attention module only improves these metrics by 0.1899, 2.88%, and 2.84%, respectively. In the interpretability case study, panobinostat exhibited the most effective predicted response, with a value of −4.895. We provide reliable insights for drug selection in personalized medicine by identifying crucial genetic factors influencing drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f40ba043b018c0ffb309d5df62d65dac2ff38ebf" target='_blank'>
              An Innovative Multi-Omics Model Integrating Latent Alignment and Attention Mechanism for Drug Response Prediction
              </a>
            </td>
          <td>
            Hui-O Chen, Yuan-Chi Cui, Peng-Chan Lin, Jung-Hsien Chiang
          </td>
          <td>2024-06-27</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Introduction Prostate cancer (PC) is the most common malignancy and accounts for a significant proportion of cancer deaths among men. Although initial therapy success can often be observed in patients diagnosed with localized PC, many patients eventually develop disease recurrence and metastasis. Without effective treatments, patients with aggressive PC display very poor survival. To curb the current high mortality rate, many investigations have been carried out to identify efficacious therapeutics. Compared to de novo drug designs, computational methods have been widely employed to offer actionable drug predictions in a fast and cost-efficient way. Particularly, powered by an increasing availability of next-generation sequencing molecular profiles from PC patients, computer-aided approaches can be tailored to screen for candidate drugs. Areas covered Herein, the authors review the recent advances in computational methods for drug discovery utilizing molecular profiles from PC patients. Given the uniqueness in PC therapeutic needs, they discuss in detail the drug discovery goals of these studies, highlighting their translational values for clinically impactful drug nomination. Expert opinion Evolving molecular profiling techniques may enable new perspectives for computer-aided approaches to offer drug candidates for different tumor microenvironments. With ongoing efforts to incorporate new compounds into large-scale high-throughput screens, the authors envision continued expansion of drug candidate pools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ab81ced264dcb3b837ac31f1cf1f804ed3d305d" target='_blank'>
              Computer-aided drug discovery strategies for novel therapeutics for prostate cancer leveraging next-generating sequencing data
              </a>
            </td>
          <td>
            Weijie Zhang, R. S. Huang
          </td>
          <td>2024-06-11</td>
          <td>Expert Opinion on Drug Discovery</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Cardiotoxicity is a major cause of drug withdrawal. The hERG channel, regulating ion flow, is pivotal for heart and nervous system function. Its blockade is a concern in drug development. Predicting hERG blockade is essential for identifying cardiac safety issues. Various QSAR models exist, but their performance varies. Ongoing improvements show promise, necessitating continued efforts to enhance accuracy using emerging deep learning algorithms in predicting potential hERG blockade.


STUDY DESIGN AND METHOD
Using a large training dataset, six individual QSAR models were developed. Additionally, three ensemble models were constructed. All models were evaluated using 10-fold cross-validations and two external datasets.


RESULTS
The 10-fold cross-validations resulted in Mathews correlation coefficient (MCC) values from 0.682 to 0.730, surpassing the best-reported model on the same dataset (0.689). External validations yielded MCC values from 0.520 to 0.715 for the first dataset, exceeding those of previously reported models (0 - 0.599). For the second dataset, MCC values fell between 0.025 and 0.215, aligning with those of reported models (0.112 - 0.220).


CONCLUSIONS
The developed models can assist the pharmaceutical industry and regulatory agencies in predicting hERG blockage activity, thereby enhancing safety assessments and reducing the risk of adverse cardiac events associated with new drug candidates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38a44a5f72bc076ae255decea155cfb06c06a618" target='_blank'>
              Machine learning and deep learning approaches for enhanced prediction of hERG blockade: a comprehensive QSAR modeling study.
              </a>
            </td>
          <td>
            Jie Liu, Md Kamrul Hasan Khan, Wenjing Guo, Fan Dong, W. Ge, Chaoyang Zhang, Ping Gong, T. Patterson, Huixiao Hong
          </td>
          <td>2024-07-05</td>
          <td>Expert opinion on drug metabolism & toxicology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cellular metabolism is a highly interconnected network with thousands of reactions that convert nutrients into the molecular building blocks of life. Metabolic connectivity varies greatly with cellular context and environmental conditions, and it remains a challenge to compare genome-scale metabolism across cell types because of the high dimensionality of the reaction flux space. Here, we employ self-supervised learning and genome-scale metabolic models to compress the flux space into low-dimensional representations that preserve structure across cell types. We trained variational autoencoders (VAEs) on large fluxomic data (N = 800, 000) sampled from patient-derived models for various cancer cell types. The VAE embeddings have an improved ability to distinguish cell types than the uncompressed fluxomic data, and sufficient predictive power to classify cell types with high accuracy. We tested the ability of these classifiers to assign cell type identities to unlabelled patient-derived metabolic models not employed during VAE training. We further employed the pre-trained VAE to embed another 38 cell types and trained multilabel classifiers that display promising generalization performance. Our approach distils the metabolic space into a semantically rich vector that can be used as a foundation for predictive modelling, clustering or comparing metabolic capabilities across organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a633f3c5ab10ca81df0bae18c68dd5b91e2f278" target='_blank'>
              Low-dimensional representations of genome-scale metabolism
              </a>
            </td>
          <td>
            Samuel Cain, C. Merzbacher, D. Oyarzún
          </td>
          <td>2024-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Reconstructing the topology of gene regulatory network from gene expression data has been extensively studied. With the abundance functional transcriptomic data available, it is now feasible to systematically decipher regulatory interaction dynamics in a logic form such as a Boolean network (BN) framework, which qualitatively indicates how multiple regulators aggregated to affect a common target gene. However, inferring both the network topology and gene interaction dynamics simultaneously is still a challenging problem since gene expression data are typically noisy and data discretization is prone to information loss. We propose a new method for BN inference from time-series transcriptional profiles, called LogicGep. LogicGep formulates the identification of Boolean functions as a symbolic regression problem that learns the Boolean function expression and solve it efficiently through multi-objective optimization using an improved gene expression programming algorithm. To avoid overly emphasizing dynamic characteristics at the expense of topology structure ones, as traditional methods often do, a set of promising Boolean formulas for each target gene is evolved firstly, and a feed-forward neural network trained with continuous expression data is subsequently employed to pick out the final solution. We validated the efficacy of LogicGep using multiple datasets including both synthetic and real-world experimental data. The results elucidate that LogicGep adeptly infers accurate BN models, outperforming other representative BN inference algorithms in both network topology reconstruction and the identification of Boolean functions. Moreover, the execution of LogicGep is hundreds of times faster than other methods, especially in the case of large network inference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81189774cd4d14cf52c8efb5374bdb7e2ca817d4" target='_blank'>
              LogicGep: Boolean networks inference using symbolic regression from time-series transcriptomic profiling data
              </a>
            </td>
          <td>
            Dezhen Zhang, Shuhua Gao, Zhiping Liu, Rui Gao
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/985fb213001f71393d7581e0584e1c2910789136" target='_blank'>
              CVGAE: A Self-Supervised Generative Method for Gene Regulatory Network Inference Using Single-Cell RNA Sequencing Data.
              </a>
            </td>
          <td>
            Wei Liu, Zhijie Teng, Zejun Li, Jing Chen
          </td>
          <td>2024-05-23</td>
          <td>Interdisciplinary sciences, computational life sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The complexity, scale, and uncertainty in regulatory networks (e.g., gene regulatory networks and microbial networks) regularly pose a huge uncertainty in their models. These uncertainties often cannot be entirely reduced using limited and costly data acquired from the normal condition of systems. Meanwhile, regulatory networks often suffer from the non-identifiability issue, which refers to scenarios where the true underlying network model cannot be clearly distinguished from other possible models. Perturbation or excitation is a well-known process in systems biology for acquiring targeted data to reveal the complex underlying mechanisms of regulatory networks and overcome the non-identifiability issue. We consider a general class of Boolean network models for capturing the activation and inactivation of components and their complex interactions. Assuming partial available knowledge about the interactions between components of the networks, this paper formulates the inference process through the maximum aposteriori (MAP) criterion. We develop a Bayesian lookahead policy that systematically perturbs regulatory networks to maximize the performance of MAP inference under the perturbed data. This is achieved by optimally formulating the perturbation process in a reinforcement learning context and deriving a scalable deep reinforcement learning perturbation policy to compute near-optimal Bayesian policy. The proposed method learns the perturbation policy through planning without the need for any real data. The high performance of the proposed approach is demonstrated by comprehensive numerical experiments using the well-known mammalian cell cycle and gut microbial community networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13661311b83454b1f32c13b0d86a4299e417da66" target='_blank'>
              Bayesian Lookahead Perturbation Policy for Inference of Regulatory Networks.
              </a>
            </td>
          <td>
            Mohammad Alali, Mahdi Imani
          </td>
          <td>2024-05-17</td>
          <td>IEEE/ACM transactions on computational biology and bioinformatics</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="In recent decades, the utilization of machine learning (ML) and artificial intelligence (AI) approaches have been explored for process modelling applications. However, different types of ML models may have contrasting advantages and disadvantages, which become critical during the optimal selection of a specific data‐driven model for a particular application as well as estimation of parameters during model training. This paper compares and contrasts two different types of data‐driven modelling approaches, namely the series/parallel all‐nonlinear static‐dynamic neural network models and models from a Bayesian ML approach. Both types of AI modelling approaches considered in this work have shown to significantly outperform several state‐of‐the‐art steady‐state and dynamic data‐driven modelling techniques for various performance measures, specifically, model sparsity, predictive capabilities, and computational expense. The performances of the proposed model structures and algorithms have been evaluated for two nonlinear dynamic chemical engineering systems—a plug‐flow reactor for vapour phase cracking of acetone for production of acetic anhydride and a pilot‐plant for post‐combustion CO2 capture using monoethanolamine as the solvent. For the validation data from the CO2 capture pilot plant, root mean squared error (RMSE) for flue gas outlet temperature, flowrate and CO2 concentration is 0.05%, 1.07%, and 5.0%, respectively, for the all‐nonlinear static‐dynamic neural networks and 0.1%, 1.75%, and 14.14%, respectively, for the Bayesian ML models. For the plug flow reactor data, the Bayesian ML models yield superior RMSE compared to the all‐nonlinear static‐dynamic neural networks when the measurement data are corrupted with Gaussian, auto‐correlated, or cross‐correlated noise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/056ceb214531414b85db491a54767e185fb5c38a" target='_blank'>
              All‐nonlinear static‐dynamic neural networks versus Bayesian machine learning for data‐driven modelling of chemical processes
              </a>
            </td>
          <td>
            Angan Mukherjee, Samuel Adeyemo, D. Bhattacharyya
          </td>
          <td>2024-06-24</td>
          <td>The Canadian Journal of Chemical Engineering</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9165fe7b8af9c461e14f5fa3a2cd250b938e0d" target='_blank'>
              Optimizing clinico-genomic disease prediction across ancestries: a machine learning strategy with Pareto improvement
              </a>
            </td>
          <td>
            Yan Gao, Yan Cui
          </td>
          <td>2024-06-04</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/620803fdb5f14b75e2d7602b6c40e8374eb91f5a" target='_blank'>
              Enhancing drug and cell line representations via contrastive learning for improved anti-cancer drug prioritization
              </a>
            </td>
          <td>
            Patrick J. Lawrence, Benjamin Burns, Xia Ning
          </td>
          <td>2024-05-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Computational modelling of cell state transitions has been a great interest of many in the field of developmental biology, cancer biology and cell fate engineering because it enables performing perturbation experiments in silico more rapidly and cheaply than could be achieved in a wet lab. Recent advancements in single-cell RNA sequencing (scRNA-seq) allow the capture of high- resolution snapshots of cell states as they transition along temporal trajectories. Using these high-throughput datasets, we can train computational models to generate in silico ‘synthetic’ cells that faithfully mimic the temporal trajectories. Here we present OneSC, a platform that can simulate synthetic cells across developmental trajectories using systems of stochastic differential equations govern by a core transcription factors (TFs) regulatory network. Different from the current network inference methods, OneSC prioritizes on generating Boolean network that produces faithful cell state transitions and steady cell states that mimic real biological systems. Applying OneSC to real data, we inferred a core TF network using a mouse myeloid progenitor scRNA-seq dataset and showed that the dynamical simulations of that network generate synthetic single-cell expression profiles that faithfully recapitulate the four myeloid differentiation trajectories going into differentiated cell states (erythrocytes, megakaryocytes, granulocytes and monocytes). Finally, through the in-silico perturbations of the mouse myeloid progenitor core network, we showed that OneSC can accurately predict cell fate decision biases of TF perturbations that closely match with previous experimental observations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323a3d5011fce70df45683664c3935ee296ca7b1" target='_blank'>
              OneSC: A computational platform for recapitulating cell state transitions
              </a>
            </td>
          <td>
            Dan Peng, Patrick Cahan
          </td>
          <td>2024-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Physics-informed deep learning has been developed as a novel paradigm for learning physical dynamics recently. While general physics-informed deep learning methods have shown early promise in learning fluid dynamics, they are difficult to generalize in arbitrary time instants in real-world scenario, where the fluid motion can be considered as a time-variant trajectory involved large-scale particles. Inspired by the advantage of diffusion model in learning the distribution of data, we first propose Pi-fusion, a physics-informed diffusion model for predicting the temporal evolution of velocity and pressure field in fluid dynamics. Physics-informed guidance sampling is proposed in the inference procedure of Pi-fusion to improve the accuracy and interpretability of learning fluid dynamics. Furthermore, we introduce a training strategy based on reciprocal learning to learn the quasiperiodical pattern of fluid motion and thus improve the generalizability of the model. The proposed approach are then evaluated on both synthetic and real-world dataset, by comparing it with state-of-the-art physics-informed deep learning methods. Experimental results show that the proposed approach significantly outperforms existing methods for predicting temporal evolution of velocity and pressure field, confirming its strong generalization by drawing probabilistic inference of forward process and physics-informed guidance sampling. The proposed Pi-fusion can also be generalized in learning other physical dynamics governed by partial differential equations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d533bf917f7e4f8a81a52e26c206fadbb3f08b" target='_blank'>
              Pi-fusion: Physics-informed diffusion model for learning fluid dynamics
              </a>
            </td>
          <td>
            Jing Qiu, Jiancheng Huang, Xiangdong Zhang, Zeng Lin, Minglei Pan, Zengding Liu, F. Miao
          </td>
          <td>2024-06-06</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="The headway and process steps are in innovation and appropriate strategies are set out a freedom for progress in numerous logical fields and different businesses. AI has become significant apparatus for drug plans and disclosure with the accessibility of spot information from enormous data sets. This article discuss about dissect ML and DL procedures it helps Pharmacy company’s in major related phases in medication disclosure they incorporates approval, future clinical events, and medical preliminaries. This application can be developed using machine learning techniques and using some important methods. This approach is very helpful in medicine industry. In present COVID situation vaccine development plays a major role to find the best medicine for that we have to test the medicine multiple times to test the results, if they did manual process they will take lot of time to update the details for that we are introducing this method. KEYWORDS: pharmaceutical synthesis, machine learning ML, Reinforcement learning RL, Medicine industry, supervised Machine Learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f1641a36b7144edabbb4e9793feb87a3610604" target='_blank'>
              Pharmaceutical Synthesis and Discloser Approach Using Naïve Bayesian Method in Machine Learning a.
              </a>
            </td>
          <td>

          </td>
          <td>2024-06-13</td>
          <td>INTERANTIONAL JOURNAL OF SCIENTIFIC RESEARCH IN ENGINEERING AND MANAGEMENT</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Heterogeneous, interconnected, systems-level, molecular data have become increasingly available and key in precision medicine. We need to utilize them to better stratify patients into risk groups, discover new biomarkers and targets, repurpose known and discover new drugs to personalize medical treatment. Existing methodologies are limited and a paradigm shift is needed to achieve quantitative and qualitative breakthroughs. In this perspective paper, we survey the literature and argue for the development of a comprehensive, general framework for embedding of multi-scale molecular network data that would enable their explainable exploitation in precision medicine in linear time. Network embedding methods map nodes to points in low-dimensional space, so that proximity in the learned space reflects the network's topology-function relationships. They have recently achieved unprecedented performance on hard problems of utilizing few omic data in various biomedical applications. However, research thus far has been limited to special variants of the problems and data, with the performance depending on the underlying topology-function network biology hypotheses, the biomedical applications and evaluation metrics. The availability of multi-omic data, modern graph embedding paradigms and compute power call for a creation and training of efficient, explainable and controllable models, having no potentially dangerous, unexpected behaviour, that make a qualitative breakthrough. We propose to develop a general, comprehensive embedding framework for multi-omic network data, from models to efficient and scalable software implementation, and to apply it to biomedical informatics. It will lead to a paradigm shift in computational and biomedical understanding of data and diseases that will open up ways to solving some of the major bottlenecks in precision medicine and other domains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2728c79f21b67623d3fdbf31a0f92bf8fe73b0aa" target='_blank'>
              Simplicity within biological complexity
              </a>
            </td>
          <td>
            Nataša Pržulj, N. Malod-Dognin
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Optimal control of multiple heating, ventilation, and air-conditioning units in an open-plan space demands fast and accurate thermodynamic modeling. While computational fluid dynamics is precise, it is computationally impractical for real-time control. Lumped thermal models divide the room into cells and explicitly model inter-cell air-mixing. However, solving the coupled differential equations is still computationally challenging. Data-driven methods like long-short term memory may handle non-linear thermal dynamics but face limitations in observability, adherence to physical constraints, and generalization. Physics-informed neural networks (PINN) impose physical constraints even while learning from data. However, PINNs still lack the scalability required for effective control in large open-plan offices, primarily due to air-mixing interactions Our approach, PhyGICS, combines graph neural networks with physics-informed learning (PI-GNN) to overcome these challenges. Specifically, we model the thermodynamic interactions as edges between nodes that represent cells. Further, the modeling approach allows explicit modeling of wall and window surface temperatures that are commonly ignored. PhyGICS, utilizing PI-GNN as a state-estimator, employs a receding-horizon approach for optimal HVAC control. We adapt PI-GNNs for building HVAC control by incorporating a time-resetting strategy to handle time-dependent ambient conditions and therefore set-points. Evaluations in a simulated environment with real-world data show PhyGICS outperforms a regular PINN model and other baseline control strategies on: 1) thermal model accuracy; 2) computation time; 3) energy consumption; and 4) user comfort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b9dc12e24efd410665c6606c578c428451e250" target='_blank'>
              PhyGICS – A Physics-informed Graph Neural Network-based Intelligent HVAC Controller for Open-plan Spaces
              </a>
            </td>
          <td>
            S. Nagarathinam, Arunchandar Vasan
          </td>
          <td>2024-06-04</td>
          <td>Proceedings of the 15th ACM International Conference on Future and Sustainable Energy Systems</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The intricacies of the human genome, manifested as a complex network of genes, transcend conventional representations in text or numerical matrices. The intricate gene-to-gene relationships inherent in this complexity find a more suitable depiction in graph structures. In the pursuit of predicting gene expression, an endeavor shared by predecessors like the L1000 and Enformer methods, we introduce a novel spatial graph-neural network (GNN) approach. This innovative strategy incorporates graph features, encompassing both regulatory and structural elements. The regulatory elements include pair-wise gene correlation, biological pathways, protein-protein interaction networks, and transcription factor regulation. The spatial structural elements include chromosomal distance, histone modification and Hi-C inferred 3D genomic features. Principal Node Aggregation models, validated independently, emerge as frontrunners, demonstrating superior performance compared to traditional regression and other deep learning models. By embracing the spatial GNN paradigm, our method significantly advances the description of the intricate network of gene interactions, surpassing the performance, predictable scope, and initial requirements set by previous methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f51120aeb183a8a8caaf3a4cca0f7e47b5212e" target='_blank'>
              Reinventing gene expression connectivity through regulatory and spatial structural empowerment via principal node aggregation graph neural network.
              </a>
            </td>
          <td>
            Fengyao Yan, Limin Jiang, Danqian Chen, M. Ceccarelli, Yan Guo
          </td>
          <td>2024-06-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Molecular analytics increasingly utilize machine learning (ML) for predictive modeling based on data acquired through molecular profiling technologies. However, developing robust models that accurately capture physiological phenotypes is challenged by a multitude of factors. These include the dynamics inherent to biological systems, variability stemming from analytical procedures, and the resource-intensive nature of obtaining sufficiently representative datasets. Here, we propose and evaluate a new method: Contextual Out-of-Distribution Integration (CODI). Based on experimental observations, CODI generates synthetic data that integrate unrepresented sources of variation encountered in real-world applications into a given molecular fingerprint dataset. By augmenting a dataset with out-of-distribution variance, CODI enables an ML model to better generalize to samples beyond the initial training data. Using three independent longitudinal clinical studies and a case-control study, we demonstrate CODI’s application to several classification scenarios involving vibrational spectroscopy of human blood. We showcase our approach’s ability to enable personalized fingerprinting for multi-year longitudinal molecular monitoring and enhance the robustness of trained ML models for improved disease detection. Our comparative analyses revealed that incorporating CODI into the classification workflow consistently led to significantly improved classification accuracy while minimizing the requirement of collecting extensive experimental observations. SIGNIFICANCE STATEMENT Analyzing molecular fingerprint data is challenging due to multiple sources of biological and analytical variability. This variability hinders the capacity to collect sufficiently large and representative datasets that encompass realistic data distributions. Consequently, the development of machine learning models that generalize to unseen, independently collected samples is often compromised. Here, we introduce CODI, a versatile framework that enhances traditional classifier training methodologies. CODI is a general framework that incorporates information about possible out-of-distribution variations into a given training dataset, augmenting it with simulated samples that better capture the true distribution of the data. This allows the classification to achieve improved predictive performance on samples beyond the original distribution of the training data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f5a765a1c0f589cfa9eccf1b65dd12320bb5114" target='_blank'>
              CODI: Enhancing machine learning-based molecular profiling through contextual out-of-distribution integration
              </a>
            </td>
          <td>
            Tarek Eissa, Marinus Huber, Barbara Obermayer-Pietsch, Birgit Linkohr, A. Peters, Frank Fleischmann, M. Žigman
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Recent advancements in computational pathology have introduced deep learning methods to predict genomic, transcriptomic and molecular biomarkers from routine histology whole slide images (WSIs) for cancer diagnosis, prognosis, and treatment. However, existing methods often overlook the critical role of co-dependencies among biomarker statuses during training and inference. We hypothesize that this oversight results in models that predict the combined effect of multiple interdependent biomarkers rather than individual statuses independently, akin to attributing the quality of an orchestral symphony to a single instrument, highlighting limitations of current predictors. Methods Using large datasets (n = 8,221 patients), we conducted statistical co-dependence testing to demonstrate significant interdependencies among biomarker statuses in training datasets. Following standard protocols, we trained two machine learning models to predict biomarkers from WSIs achieving or matching state-of-the-art predictive performance. We then employed permutation testing and stratification analysis to evaluate their predictive quality based on the principle of conditional independence, i.e., if a model accurately captures the phenotypic influence of a specific biomarker independent of other biomarkers, its performance should remain consistent across subgroups of patients stratified by other biomarkers, aligning with its overall performance on the entire dataset. Findings Our statistical analysis reveals significant interdependencies among biomarkers, reflecting expected co-occurrence and mutual exclusivity patterns influenced by pathological and biological processes that are consistent across datasets, as well as sampling artefacts that can be different across datasets. Our results indicate that the predictive quality of an image-based predictor for a biomarker is contingent on the status of other biomarkers, revealing that models capture aggregated influences rather than predicting individual statuses independently. For example, mutation predictions are confounded by the overall tumour mutation burden. We also show that, due to the presence of such correlations, deep learning models may not offer significant advantages in predicting certain biomarkers in comparison to simply using pathologist-assigned grades for their prediction. Interpretation We show that current deep learning models in computational pathology fall short in isolating individual biomarker effects, leading to confounded and less precise predictions. Our findings suggest revisiting model training protocols to recognize and adjust for biomarker interdependencies at all development stages—from problem definition to usage guidelines. This involves selecting diverse datasets to reflect clinical heterogeneity, defining prediction variables or grouping patients based on co-dependencies, designing models to disentangle complex relationships, and stringent stratification testing. Clinically, failure to account for interdependencies may lead to suboptimal decisions, necessitating appropriate usage guidelines for predictive models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a318afdaf3385141a6d7bb9fce1bc08ba68f39c4" target='_blank'>
              Buyer Beware: confounding factors and biases abound when predicting omics-based biomarkers from histological images
              </a>
            </td>
          <td>
            Muhammad Dawood, Kim Branson, Sabine Tejpar, N. Rajpoot, F. Minhas
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="This study proposes a new method to improve the efficiency of drug discovery by repurposing existing drugs, aiming to reduce the time and costs associated with traditional drug development processes, which can span 10 to 15 years and cost billions of dollars. Current approaches focus on leveraging heterogeneous data, such as drug-protein and disease-protein interactions, to construct complex networks that link drugs, proteins, and diseases. However, a significant challenge is the imbalance in data, where numerous unconfirmed potential drug-disease interactions (the majority class) outnumber approved drugs (the minority class), severely impacting the predictive performance of machine learning models. Previous attempts to address this issue have shown limited success. This study introduces a novel approach that integrates meta-paths in heterogeneous information networks with data balancing techniques to tackle this imbalance. Experimental results demonstrate that the proposed method enhances model performance and reliability in identifying new relationships between drugs and diseases. This research represents a promising advancement by leveraging network-based strategies and data balancing techniques to facilitate the rediscovery of drug applications, thereby potentially revolutionizing the pharmaceutical industry’s approach to drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25438f7cc83aaed0d4842e6721d7f9054712298b" target='_blank'>
              ENHANCING DRUG DISCOVERY THROUGH A META-PATH BASED OVERSAMPLING APPROACH FOR IMBALANCED DATA
              </a>
            </td>
          <td>
            Manh Hung Le
          </td>
          <td>2024-06-30</td>
          <td>Journal of Science and Technique</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Capturing the time evolution and predicting future kinetic states of physicochemical systems present significant challenges due to the precision and computational effort required. In this study, we demonstrate that the transformer, a machine learning model renowned for machine translation and natural language processing, can be effectively adapted to predict the dynamical state-to-state transition kinetics of biologically relevant physicochemical systems. Specifically, by using sequences of time-discretized states from Molecular Dynamics (MD) simulation trajectories as input, we show that a transformer can learn the complex syntactic and semantic relationships within the trajectory. This enables this generative pre-trained transformer (GPT) to predict kinetically accurate sequences of future states for a diverse set of models and biomolecules of varying complexity. Remarkably, the GPT can predict future states much faster than traditional MD simulations. We show that it is particularly adept at forecasting the time evolution of an out-of-equilibrium active system that do not maintain detailed balance. An analysis of self-attention mechanism inherent in transformers is found to hold crucial role for capturing the long-range correlations necessary for accurate state-to-state transition predictions. Together, our results highlight the ability of transformer based machine learning model in generating future states of physicochemical systems with statistical precision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f68df42b37752017ad05438942ecb38a7e24c6e" target='_blank'>
              Predicting Future Kinetic States of Physicochemical Systems Using Generative Pre-trained Transformer
              </a>
            </td>
          <td>
            Palash Bera, Jagannath Mondal
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="We address the problem of model misspecification in population pharmacokinetics (PopPK), by modeling residual unexplained variability (RUV) by machine learning (ML) methods in a postprocessing step after conventional model building. The practical purpose of the method is the generation of realistic virtual patient profiles and the quantification of the extent of model misspecification, by introducing an appropriate metric, to be used as an additional diagnostic of model quality. The proposed methodology consists of the following steps: After developing a PopPK model, the individual residual errors IRES = DV-IPRED, are computed, where DV are the observations and IPRED the individual predictions and are modeled by ML to obtain IRESML. Correction of the IPREDs can then be carried out as DVML = IPRED + IRESML. The methodology was tested in a PK study of ropinirole, for which a PopPK model was developed while a second deliberately misspecified model was also considered. Various supervised ML algorithms were tested, among which Random Forest gave the best results. The ML model was able to correct individual predictions as inspected in diagnostic plots and most importantly it simulated realistic profiles that resembled the real data and canceled out the artifacts introduced by the elevated RUV, even in the case of the heavily misspecified model. Furthermore, a metric to quantify the extent of model misspecification was introduced based on the R2 between IRES and IRESML, following the rationale that the greater the extent of variability explained by the ML model, the higher the degree of model misspecification present in the original model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69296a29a3089d2ff48e2bc6b1e62f03d5894045" target='_blank'>
              Simulating realistic patient profiles from pharmacokinetic models by a machine learning postprocessing correction of residual variability.
              </a>
            </td>
          <td>
            Christos Kaikousidis, Robert R Bies, A. Dokoumetzidis
          </td>
          <td>2024-06-14</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55aa55ce9bae4e173f97d3be1d9cc988e4eeeb71" target='_blank'>
              Property-guided generation of complex polymer topologies using variational autoencoders
              </a>
            </td>
          <td>
            Shengli Jiang, Adji B. Dieng, Michael A. Webb
          </td>
          <td>2024-06-29</td>
          <td>npj Computational Materials</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="When studying ecosystems, hierarchical trees are often used to organize entities based on proximity criteria, such as the taxonomy in microbiology, social classes in geography, or product types in retail businesses, offering valuable insights into entity relationships. Despite their significance, current count-data models do not leverage this structured information. In particular, the widely used Poisson log-normal (PLN) model, known for its ability to model interactions between entities from count data, lacks the possibility to incorporate such hierarchical tree structures, limiting its applicability in domains characterized by such complexities. To address this matter, we introduce the PLN-Tree model as an extension of the PLN model, specifically designed for modeling hierarchical count data. By integrating structured variational inference techniques, we propose an adapted training procedure and establish identifiability results, enhancisng both theoretical foundations and practical interpretability. Additionally, we extend our framework to classification tasks as a preprocessing pipeline, showcasing its versatility. Experimental evaluations on synthetic datasets as well as real-world microbiome data demonstrate the superior performance of the PLN-Tree model in capturing hierarchical dependencies and providing valuable insights into complex data structures, showing the practical interest of knowledge graphs like the taxonomy in ecosystems modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86dd1a5a64cacf81f6895c4656147dbabe2e7e27" target='_blank'>
              Tree-based variational inference for Poisson log-normal models
              </a>
            </td>
          <td>
            Alexandre Chaussard, Anna Bonnet, Elisabeth Gassiat, S. Corff
          </td>
          <td>2024-06-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Computational chemistry is an indispensable tool for understanding molecules and predicting chemical properties. However, traditional computational methods face significant challenges due to the difficulty of solving the Schrödinger equations and the increasing computational cost with the size of the molecular system. In response, there has been a surge of interest in leveraging artificial intelligence (AI) and machine learning (ML) techniques to in silico experiments. Integrating AI and ML into computational chemistry increases the scalability and speed of the exploration of chemical space. However, challenges remain, particularly regarding the reproducibility and transferability of ML models. This review highlights the evolution of ML in learning from, complementing, or replacing traditional computational chemistry for energy and property predictions. Starting from models trained entirely on numerical data, a journey set forth toward the ideal model incorporating or learning the physical laws of quantum mechanics. This paper also reviews existing computational methods and ML models and their intertwining, outlines a roadmap for future research, and identifies areas for improvement and innovation. Ultimately, the goal is to develop AI architectures capable of predicting accurate and transferable solutions to the Schrödinger equation, thereby revolutionizing in silico experiments within chemistry and materials science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c965a92a4418b12137dd3227bb9be135171b111" target='_blank'>
              In Silico Chemical Experiments in the Age of AI: From Quantum Chemistry to Machine Learning and Back.
              </a>
            </td>
          <td>
            Abdulrahman Aldossary, J. A. Campos-Gonzalez-Angulo, Sergio Pablo-García, Shi Xuan Leong, Ella Miray Rajaonson, Luca Thiede, Gary Tom, Andrew Wang, Davide Avagliano, Alán Aspuru-Guzik
          </td>
          <td>2024-05-25</td>
          <td>Advanced materials</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The ability to accurately predict disease progression is paramount for optimizing multiple myeloma patient care. This study introduces a hybrid neural network architecture, combining Long Short-Term Memory networks with a Conditional Restricted Boltzmann Machine, to predict future blood work of affected patients from a series of historical laboratory results. We demonstrate that our model can replicate the statistical moments of the time series ($0.95~\pm~0.01~\geq~R^2~\geq~0.83~\pm~0.03$) and forecast future blood work features with high correlation to actual patient data ($0.92\pm0.02~\geq~r~\geq~0.52~\pm~0.09$). Subsequently, a second Long Short-Term Memory network is employed to detect and annotate disease progression events within the forecasted blood work time series. We show that these annotations enable the prediction of progression events with significant reliability (AUROC$~=~0.88~\pm~0.01$), up to 12 months in advance (AUROC($t+12~$mos)$~=0.65~\pm~0.01$). Our system is designed in a modular fashion, featuring separate entities for forecasting and progression event annotation. This structure not only enhances interpretability but also facilitates the integration of additional modules to perform subsequent operations on the generated outputs. Our approach utilizes a minimal set of routine blood work measurements, which avoids the need for expensive or resource-intensive tests and ensures accessibility of the system in clinical routine. This capability allows for individualized risk assessment and making informed treatment decisions tailored to a patient's unique disease kinetics. The represented approach contributes to the development of a scalable and cost-effective virtual human twin system for optimized healthcare resource utilization and improved patient outcomes in multiple myeloma care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720eddebaebf55df59d7ccef7e7c2748b0a79603" target='_blank'>
              Predicting Progression Events in Multiple Myeloma from Routine Blood Work
              </a>
            </td>
          <td>
            Maximilian Ferle, N. Grieb, Markus Kreuz, U. Platzbecker, Thomas Neumuth, Kristin Reiche, Alexander Oeser, M. Merz
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Drug response prediction (DRP) is a crucial phase in drug discovery, and the most important metric for its evaluation is the IC50 score. DRP results are heavily dependent on the quality of the generated molecules. Existing molecule generation methods typically employ classifier-based guidance, enabling sampling within the IC50 classification range. However, these methods fail to ensure the sampling space range's effectiveness, generating numerous ineffective molecules. Through experimental and theoretical study, we hypothesize that conditional generation based on the target IC50 score can obtain a more effective sampling space. As a result, we introduce regressor-free guidance molecule generation to ensure sampling within a more effective space and support DRP. Regressor-free guidance combines a diffusion model's score estimation with a regression controller model's gradient based on number labels. To effectively map regression labels between drugs and cell lines, we design a common-sense numerical knowledge graph that constrains the order of text representations. Experimental results on the real-world dataset for the DRP task demonstrate our method's effectiveness in drug discovery. The code is available at:https://anonymous.4open.science/r/RMCD-DBD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc3590e9b6a96f27c063cbbf748a161eabfdbb7f" target='_blank'>
              Regressor-free Molecule Generation to Support Drug Response Prediction
              </a>
            </td>
          <td>
            Kun Li, Xiuwen Gong, Shirui Pan, Jia Wu, Bo Du, Wenbin Hu
          </td>
          <td>2024-05-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Models have always been central to inferring molecular evolution and to reconstructing phylogenetic trees. Their use typically involves the development of a mechanistic framework reflecting our understanding of the underlying biological processes, such as nucleotide substitu- tions, and the estimation of model parameters by maximum likelihood or Bayesian inference. However, deriving and optimizing the likelihood of the data is not always possible under complex evolutionary scenarios or even tractable for large datasets, often leading to unrealistic simplifying assumptions in the fitted models. To overcome this issue, we coupled stochastic simulations of genome evolution with a new supervised deep learning model to infer key parameters of molecular evolution. Our model is designed to directly analyze multiple sequence alignments and estimate per-site evolutionary rates and divergence, without requiring a known phylogenetic tree. The accuracy of our predictions matched that of likelihood-based phylogenetic inference, when rate heterogeneity followed a simple gamma distribution, but it strongly exceeded it under more complex patterns of rate variation, such as codon models. Our approach is highly scalable and can be efficiently applied to genomic data, as we showed on a dataset of 26 million nucleotides from the clownfish clade. Our simulations also showed that the integration of per-site rates obtained by deep learning within a Bayesian framework led to significantly more accu- rate phylogenetic inference, particularly with respect to the estimated branch lengths. We thus propose that future advancements in phylogenetic analysis will benefit from a semi-supervised learning approach that combines deep-learning estimation of substitution rates, which allows for more flexible models of rate variation, and probabilistic inference of the phylogenetic tree, which guarantees interpretability and a rigorous assessment of statistical support.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e5726c0265f212865a58fafea78c3fde46d235" target='_blank'>
              Toward a semi-supervised learning approach to phylogenetic estimation.
              </a>
            </td>
          <td>
            Daniele Silvestro, Thibault Latrille, N. Salamin
          </td>
          <td>2024-06-25</td>
          <td>Systematic biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Obtaining accurate estimates of uncertainty in climate scenarios often requires generating large ensembles of high-resolution climate simulations, a computationally expensive and memory intensive process. To address this challenge, we train a novel generative deep learning approach on extensive sets of climate simulations. The model consists of two components: a variational autoencoder for dimensionality reduction and a denoising diffusion probabilistic model that generates multiple ensemble members. We validate our model on the Max Planck Institute Grand Ensemble and show that it achieves good agreement with the original ensemble in terms of variability. By leveraging the latent space representation, our model can rapidly generate large ensembles on-the-fly with minimal memory requirements, which can significantly improve the efficiency of uncertainty quantification in climate simulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bc0b1801a363ae98c2f59c86a03c296f284d553" target='_blank'>
              Latent Diffusion Model for Generating Ensembles of Climate Simulations
              </a>
            </td>
          <td>
            Johannes Meuer, Maximilian Witte, C. Timmreck, Thomas Ludwig, Christopher Kadow
          </td>
          <td>2024-07-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="MOTIVATION
Accurately predicting the driver genes of cancer is of great significance for carcinogenesis progress research and cancer treatment. In recent years, more and more deep-learning-based methods have been used for predicting cancer driver genes. However, deep learning algorithms often have black box properties and cannot interpret the output results. Here, we propose a novel cancer driver gene mining method based on heterogeneous network metapaths (MCDHGN), which uses meta-path aggregation to enhance the interpretability of predictions.


RESULTS
MCDHGN constructs a heterogeneous network by using several types of multi-omics data that are biologically linked to genes. And the differential probabilities of SNV, DNA methylation, and gene expression data between cancerous tissues and normal tissues are extracted as initial features of genes. Nine meta-paths are manually selected, and the representation vectors obtained by aggregating information within and across metapath nodes are used as new features for subsequent classification and prediction tasks. By comparing with eight homogeneous and heterogeneous network models on two pan-cancer datasets, MCDHGN has better performance on AUC and AUPR values. Additionally, MCDHGN provides interpretability of predicted cancer driver genes through the varying weights of biologically meaningful meta-paths.


AVAILABILITY
https://github.com/1160300611/MCDHGN.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b32a7b3611c1e300542c76e55e6a0bb0480476b" target='_blank'>
              MCDHGN: Heterogeneous network-based cancer driver gene prediction and interpretability analysis.
              </a>
            </td>
          <td>
            Lexiang Wang, Jingli Zhou, Xuan Wang, Yadong Wang, Junyi Li
          </td>
          <td>2024-06-12</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Scientific Machine Learning is a new class of approaches that integrate physical knowledge and mechanistic models with data-driven techniques for uncovering governing equations of complex processes. Among the available approaches, Universal Differential Equations (UDEs) are used to combine prior knowledge in the form of mechanistic formulations with universal function approximators, like neural networks. Integral to the efficacy of UDEs is the joint estimation of parameters within mechanistic formulations and the universal function approximators using empirical data. The robustness and applicability of resultant models, however, hinge upon the rigorous quantification of uncertainties associated with these parameters, as well as the predictive capabilities of the overall model or its constituent components. With this work, we provide a formalisation of uncertainty quantification (UQ) for UDEs and investigate important frequentist and Bayesian methods. By analysing three synthetic examples of varying complexity, we evaluate the validity and efficiency of ensembles, variational inference and Markov chain Monte Carlo sampling as epistemic UQ methods for UDEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320dbbdecad0c6895c5ce6a4b03811b5cb0b85c8" target='_blank'>
              Assessment of Uncertainty Quantification in Universal Differential Equations
              </a>
            </td>
          <td>
            Nina Schmid, David Fernandes del Pozo, Willem Waegeman, Jan Hasenauer
          </td>
          <td>2024-06-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the context of transcriptional dynamics modeled by ordinary differential equations (ODEs), the RNA level in a single cell is controlled by specific RNA kinetics parameters, which include transcription rate, splicing rate, and degradation rate. Investigating these single-cell RNA kinetics rates is pivotal for understanding RNA metabolism and the heterogeneity of complex tissues. Although metabolic labeling is an effective method to estimate these kinetics rates experimentally, it is not suitable for current large-scale conventional single-cell RNA sequencing (scRNA-seq) data. Moreover, existing methods for scRNA-seq often either neglect certain specific kinetics parameters or use inappropriate ways to fit the parameters. To address these issues, we introduce scRNAkinetics, a parallelized method that fits the kinetics parameters of the ODE for each cell using pseudo-time derived from various biological priors (e.g. cell lineage tree and differentiation potential). This approach allows for the estimation of the relative kinetics of each cell and gene in a scRNA-seq dataset. Validated on simulated datasets, scRNAkinetics can accurately infer the kinetics rates of transcription boosting, multi-branch, and time-dependent RNA degradation systems. Nevertheless, the inferred kinetics trends are concordant with previous studies on metabolic labeling and conventional scRNA-seq datasets. Furthermore, we show that scRNAkinetics can provide valuable insights into different regulatory schemes and validate the coupling between transcription and splicing in RNA metabolism. The open-source implementation of scRNAkinetics is available at https://github.com/poseidonchan/scRNAkinetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94f71a1f0958ca99a7ef6222f0f0c7006b2e729" target='_blank'>
              Inferring Single-Cell RNA Kinetics from Various Biological Priors
              </a>
            </td>
          <td>
            Yanshuo Chen, Zhengmian Hu, Ke Ni, Site Feng, Wei Chen, Heng Huang
          </td>
          <td>2024-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Machine learning (ML) algorithms are necessary to efficiently identify potent drug combinations within a large candidate space to combat drug resistance. However, existing ML approaches cannot be applied to emerging and under-studied pathogens with limited training data. To address this, we developed a transfer learning and crowdsourcing framework (TACTIC) to train ML models on data from multiple bacteria. TACTIC was built using 2,965 drug interactions from 12 bacterial strains and outperformed traditional ML models in predicting drug interaction outcomes for species that lack training data. Top TACTIC model features revealed genetic and metabolic factors that influence cross- species and species-specific drug interaction outcomes. Upon analyzing ∼600,000 predicted drug interactions across 9 metabolic environments and 18 bacterial strains, we identified a small set of drug interactions that are selectively synergistic against Gram- negative (e.g., A. baumannii) and non-tuberculous mycobacteria (NTM) pathogens. We experimentally validated synergistic drug combinations containing clarithromycin, ampicillin, and mecillinam against M. abscessus, an emerging pathogen with growing levels of antibiotic resistance. Lastly, we leveraged TACTIC to propose selectively synergistic drug combinations to treat bacterial eye infections (endophthalmitis).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5017bc92dbb38d49a788006dff57d7610ff49b95" target='_blank'>
              Transfer learning predicts species-specific drug interactions in emerging pathogens
              </a>
            </td>
          <td>
            Carolina H. Chung, David C. Chang, Nicole M. Rhoads, Madeline R. Shay, Karthik Srinivasan, M. Okezue, Ashlee D. Brunaugh, Sriram Chandrasekaran
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="PURPOSE
Development of a computational biomarker to predict, prior to treatment, the response to CDK4/6 inhibition (CDK4/6i) in combination with endocrine therapy in patients with breast cancer.


EXPERIMENTAL DESIGN
A mechanistic mathematical model that accounts for protein signaling and drug mechanisms of action was developed and trained on extensive, publicly available data from breast cancer cell lines. The model was built to provide a patient-specific response score based on the expression of six genes (CCND1, CCNE1, ESR1, RB1, MYC and CDKN1A). The model was validated in five independent cohorts of 148 patients in total with early-stage or advanced breast cancer treated with endocrine therapy and CDK4/6i. Response was measured either by evaluating Ki67 levels and PAM50 risk of relapse (ROR) after neoadjuvant treatment or by evaluating progression-free survival (PFS).


RESULTS
The model showed significant association with patient´s outcomes in all five cohorts. The model predicted high Ki67 (area under the curve; AUC (95% confidence interval) of 0.80 (0.64 - 0.92), 0.81 (0.60 - 1.00) and 0.80 (0.65 - 0.93)) and high PAM50 ROR (AUC of 0.78 (0.64 - 0.89)). This observation was not obtained in patients treated with chemotherapy. In the other cohorts, patient stratification based on the model prediction was significantly associated with PFS (hazard ratio=2.92 (95% CI 1.08 - 7.86), p=0.034 and HR=2.16 (1.02 4.55), p=0.043).


CONCLUSION
A mathematical modeling approach accurately predicts patient outcome following CDK4/6i plus endocrine therapy, which marks a step towards more personalized treatments in patients with Luminal B breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff7482f93dce7e3bccbe71f152dd92800690c632" target='_blank'>
              Computational model predicts patient outcomes in Luminal B breast cancer treated with endocrine therapy and CDK4/6 inhibition.
              </a>
            </td>
          <td>
            Leonard Schmiester, F. Brasó-Maristany, B. González-Farré, T. Pascual, J. Gavilá, Xavier Tekpli, Jürgen Geisler, Vessela N Kristensen, A. Frigessi, A. Prat, A. Köhn-Luque
          </td>
          <td>2024-06-26</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The study of effective connectivity (EC) is essential in understanding how the brain integrates and responds to various sensory inputs. Model-driven estimation of EC is a powerful approach that requires estimating global and local parameters of a generative model of neural activity. Insights gathered through this process can be used in various applications, such as studying neurodevelopmental disorders. However, accurately determining EC through generative models remains a significant challenge due to the complexity of brain dynamics and the inherent noise in neural recordings, e.g., in electroencephalography (EEG). Current model-driven methods to study EC are computationally complex and cannot scale to all brain regions as required by whole-brain analyses. To facilitate EC assessment, an inference algorithm must exhibit reliable prediction of parameters in the presence of noise. Further, the relationship between the model parameters and the neural recordings must be learnable. To progress toward these objectives, we benchmarked the performance of a Bi-LSTM model for parameter inference from the Jansen-Rit neural mass model (JR-NMM) simulated EEG under various noise conditions. Additionally, our study explores how the JR-NMM reacts to changes in key biological parameters (i.e., sensitivity analysis) like synaptic gains and time constants, a crucial step in understanding the connection between neural mechanisms and observed brain activity. Our results indicate that we can predict the local JR-NMM parameters from EEG, supporting the feasibility of our deep-learning-based inference approach. In future work, we plan to extend this framework to estimate local and global parameters from real EEG in clinically relevant applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd046ab8f0278c72f414f856472d80db3160b67" target='_blank'>
              Benchmarking Deep Jansen-Rit Parameter Inference: An in Silico Study
              </a>
            </td>
          <td>
            Deepa Tilwani, Christian O'Reilly
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell time series data frequently display considerable variability across a cell population. The current gold standard for inferring parameter distributions across cell populations is the Global Two Stage (GTS) approach for nonlinear mixed-effects (NLME) models. However, this method is computationally intensive, as it makes repeated use of non-convex optimization that in turn requires numerical integration of the underlying system. Here, we propose the Gradient Matching GTS (GMGTS) method as an efficient alternative to GTS. Gradient matching offers an integration-free approach to parameter estimation that is particularly powerful for dynamical systems that are linear in the unknown parameters, such as biochemical networks modeled by mass action kinetics. Here, we harness the power of gradient matching by integrating it into the GTS framework. To this end, we significantly expand the capabilities of gradient matching via uncertainty propagation calculations and the development of an iterative estimation scheme for partially observed systems. Through comparisons of GMGTS with GTS in different inference setups, we demonstrate that our method provides a significant computational advantage, thereby facilitating the use of complex NLME models in systems biology applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dced58f5c5737541e67c9ad6ec12731d8224877" target='_blank'>
              Gradient matching accelerates mixed-effects inference for biochemical networks
              </a>
            </td>
          <td>
            Yulan B van Oppen, Andreas Milias-Argeitis
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Drug safety relies on advanced methods for timely and accurate prediction of side effects. To tackle this requirement, this scoping review examines machine-learning approaches for predicting drug-related side effects with a particular focus on chemical, biological, and phenotypical features. Methods: This was a scoping review in which a comprehensive search was conducted in various databases from 1 January 2013 to 31 December 2023. Results: The results showed the widespread use of Random Forest, k-nearest neighbor, and support vector machine algorithms. Ensemble methods, particularly random forest, emphasized the significance of integrating chemical and biological features in predicting drug-related side effects. Conclusions: This review article emphasized the significance of considering a variety of features, datasets, and machine learning algorithms for predicting drug-related side effects. Ensemble methods and Random Forest showed the best performance and combining chemical and biological features improved prediction. The results suggested that machine learning techniques have some potential to improve drug development and trials. Future work should focus on specific feature types, selection techniques, and graph-based methods for even better prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b346d3270a77c7b4d467e797b36b5b738915e328" target='_blank'>
              Machine Learning Techniques for Predicting Drug-Related Side Effects: A Scoping Review
              </a>
            </td>
          <td>
            Esmaeel Toni, H. Ayatollahi, Reza Abbaszadeh, Alireza Fotuhi Siahpirani
          </td>
          <td>2024-06-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cardiomyopathies, often caused by mutations in genes encoding muscle proteins, are traditionally treated by phenotyping hearts and addressing symptoms post irreversible damage. With advancements in genotyping, early diagnosis is now possible, potentially preventing such damage. However, the intricate structure of muscle and its myriad proteins make treatment predictions challenging. Here we approach the problem of estimating therapeutic targets for a mutation in mouse muscle using a spatially explicit half sarcomere muscle model. We selected 9 rate parameters in our model linked to both small molecules and cardiomyopathy-causing mutations. We then randomly varied these rate parameters and simulated an isometric twitch for each combination to generate a large training dataset. We used this dataset to train a Conditional Variational Autoencoder (CVAE), a technique used in Bayesian parameter estimation. Given simulated or experimental isometric twitches, this machine learning model is able to then predict the set of rate parameters which are most likely to yield that result. We then predict the set of rate parameters associated with both control and the cardiac Troponin C (cTnC) I61Q variant in mouse trabeculae and model parameters that recover the abnormal I61Q cTnC twitches. SIGNIFICANCE Machine learning techniques have potential to accelerate discoveries in biologically complex systems. However, they require large data sets and can be challenging in high dimensional systems such as cardiac muscle. In this study, we combined experimental measures of cardiac muscle twitch forces with mechanistic simulations and a newly developed mixture of Bayesian inference with neural networks (in autoencoders) to solve the inverse problem of determining the underlying kinetics for observed force generation by cardiac muscle. The autoencoders are trained on millions of simulations spanning parameter spaces that correspond to the mechanochemistry of cardiac sarcomeres. We apply the trained model to experimental data in order to infer parameters that can explain a diseased twitch and ways to recover it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/898beeadaf2f7a68b5dd4751723ce9ccd2389b02" target='_blank'>
              Identifying mechanisms and therapeutic targets in muscle using Bayesian parameter estimation with conditional variational autoencoders
              </a>
            </td>
          <td>
            Travis Tune, Kristina B. Kooiker, Jennifer Davis, Thomas L. Daniel, F. Moussavi-Harami
          </td>
          <td>2024-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Despite recent biotechnological breakthroughs, cancer risk prediction remains a formidable computational and experimental challenge. Addressing it is critical in order to improve prevention, early detection and survival rates. Here, I briefly summarize some key emerging theoretical and computational challenges as well as recent computational advances that promise to help realize the goals of cancer-risk prediction. The focus is on computational strategies based on single-cell data, in particular on bottom-up network modeling approaches that aim to estimate cancer stemness and dedifferentiation at single-cell resolution from a systems-biological perspective. I will describe two promising methods, a tissue and cell-lineage independent one based on the concept of diffusion network entropy, and a tissue and cell-lineage specific one that uses transcription factor regulons. Application of these tools to single-cell and single-nucleus RNA-seq data from stages prior to invasive cancer reveal that they can successfully delineate the heterogeneous inter-cellular cancer-risk landscape, identifying those cells that are more likely to turn cancerous. Bottom-up systems biological modeling of single-cell omic data is a novel computational analysis paradigm that promises to facilitate the development of preventive, early detection and cancer-risk prediction strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d7a13e6b2f47623a33253ae61889093d174b00d" target='_blank'>
              Computational single-cell methods for predicting cancer risk.
              </a>
            </td>
          <td>
            Andrew E Teschendorff
          </td>
          <td>2024-06-10</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute Myocardial Infarction (AMI), a common disease that can have serious consequences, occurs when myocardial blood flow stops due to occlusion of the coronary artery. Early and accurate prediction of AMI is critical for rapid prognosis and improved patient outcomes. Metabolomics, the study of small molecules within biological systems, is an effective tool used to discover biomarkers associated with many diseases. This study intended to construct a predictive model for AMI utilizing metabolomics data and an explainable machine learning approach called Explainable Boosting Machines (EBM). The EBM model was trained on a dataset of 102 prognostic metabolites gathered from 99 individuals, including 34 healthy controls and 65 AMI patients. After a comprehensive data preprocessing, 21 metabolites were determined as the candidate predictors to predict AMI. The EBM model displayed satisfactory performance in predicting AMI, with various classification performance metrics. The model’s predictions were based on the combined effects of individual metabolites and their interactions. In this context, the results obtained in two different EBM modeling, including both only individual metabolite features and their interaction effects, were discussed. The most important predictors included creatinine, nicotinamide, and isocitrate. These metabolites are involved in different biological activities, such as energy metabolism, DNA repair, and cellular signaling. The results demonstrate the potential of the combination of metabolomics and the EBM model in constructing reliable and interpretable prediction outputs for AMI. The discussed metabolite biomarkers may assist in early diagnosis, risk assessment, and personalized treatment methods for AMI patients. This study successfully developed a pipeline incorporating extensive data preprocessing and the EBM model to identify potential metabolite biomarkers for predicting AMI. The EBM model, with its ability to incorporate interaction terms, demonstrated satisfactory classification performance and revealed significant metabolite interactions that could be valuable in assessing AMI risk. However, the results obtained from this study should be validated with studies to be carried out in larger and well-defined samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2eefc7a5a6dd6ee4a7ea1bd2688c9f1437cfb7" target='_blank'>
              Combining the Strengths of the Explainable Boosting Machine and Metabolomics Approaches for Biomarker Discovery in Acute Myocardial Infarction
              </a>
            </td>
          <td>
            A. Arslan, F. Yağın, Abdulmohsen Algarni, F. Al-Hashem, L. P. Ardigò
          </td>
          <td>2024-06-26</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Reconstructing gene regulatory networks (GRNs) using single-cell RNA sequencing (scRNA-seq) data holds great promise for unraveling cellular fate development and heterogeneity. While numerous machine-learning methods have been proposed to infer GRNs from scRNA-seq gene expression data, many of them operate solely in a statistical or black box manner, limiting their capacity for making causal inferences between genes. In this study, we introduce GRN inference with Accuracy and Causal Explanation (GRACE), a novel graph-based causal autoencoder framework that combines a structural causal model (SCM) with graph neural networks (GNNs) to enable GRN inference and gene causal reasoning from scRNA-seq data. By explicitly modeling causal relationships between genes, GRACE facilitates the learning of regulatory context and gene embeddings. With the learned gene signals, our model successfully decoding the causal structures and alleviates the accurate determination of multiple attributes of gene regulation that is important to determine the regulatory levels. Through extensive evaluations on seven benchmarks, we demonstrate that GRACE outperforms 14 state-of-the-art GRN inference methods, with the incorporation of causal mechanisms significantly enhancing the accuracy of GRN and gene causality inference. Furthermore, the application to human peripheral blood mononuclear cell (PBMC) samples reveals cell type-specific regulators in monocyte phagocytosis and immune regulation, validated through network analysis and functional enrichment analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78acc67c6cfb1020b9bf2b2c2f841815e9417f9d" target='_blank'>
              GRACE: Unveiling Gene Regulatory Networks With Causal Mechanistic Graph Neural Networks in Single-Cell RNA-Sequencing Data.
              </a>
            </td>
          <td>
            Jiacheng Wang, Yao-Jia Chen, Quan Zou
          </td>
          <td>2024-06-19</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/157e8eac3c1eeee1b8294dac26a9c49a1297cc1d" target='_blank'>
              NNICE: a deep quantile neural network algorithm for expression deconvolution
              </a>
            </td>
          <td>
            Y. Jin, Pingzhao Hu, Qian Liu
          </td>
          <td>2024-06-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The high-dimensional genomic data presents significant challenges, and traditional analytical methods often struggle to capture the complex, non-linear relationships within these datasets. This study elaborates into the application of machine learning methods for dimensionality reduction and predictive modeling of binary phenotypes using gene expression data. Various dimensionality reduction techniques are explored, including t-distributed stochastic neighbor embedding (t-SNE), Non-negative matrix factorization (NMF), Principal component analysis (PCA), and manifold learning methods. Additionally, various algorithms such as logistic regression, random forests, support vector machines (SVMs)cand naive Bayes models are evaluated for predicting phenotypes. The study employs rigorous cross-validation, permutation testing, and evaluation metrics like the Matthews Correlation Coefficient (MCC) to assess model performance. The study rigorously assesses current genomics strategies, pinpointing their drawbacks and suggesting areas for future investigation, while delving into the potential of machine learning to overcome these hurdles and offer valuable insights in genomics. Keywords- Gene Expression , Dimensionality Reduction, Biomarkers, Cancer Prediction, Epigenetics, Machine Learning, Feature extraction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08bb904a8b352ae02dc78261d9490b90e729cf74" target='_blank'>
              Machine Learning Techniques and Hybrid Feature Selection Methods for Efficient Prediction of Cancer
              </a>
            </td>
          <td>
            Remyamol Km
          </td>
          <td>2024-05-28</td>
          <td>INTERANTIONAL JOURNAL OF SCIENTIFIC RESEARCH IN ENGINEERING AND MANAGEMENT</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="We present a mathematical model of the evolutionary dynamics of a metastatic tumour under chemotherapy, comprising non-local partial differential equations for the phenotype-structured cell populations in the primary tumour and its metastasis. These equations are coupled with a physiologically-based pharmacokinetic model of drug delivery, implementing a realistic delivery schedule. The model is carefully calibrated from the literature, focusing on BRAF-mutated melanoma treated with Dabrafenib as a case study. By means of long-time asymptotic analysis, global sensitivity analysis and numerical simulations, we explore the impact of cell migration from the primary to the metastatic site, physiological aspects of the tumour sites and drug dose on the development of drug resistance and treatment efficacy. Our findings provide a possible explanation for empirical evidence indicating that chemotherapy may foster metastatic spread and that metastatic sites may be less impacted by chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75caa1850e37b71799c05e2fad947723dcd9cdaf" target='_blank'>
              The development of drug resistance in metastatic tumours under chemotherapy: an evolutionary perspective
              </a>
            </td>
          <td>
            Federica Padovano, Chiara Villa
          </td>
          <td>2024-05-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Liquid biopsy demonstrates excellent potential in patient management by providing a minimally invasive and cost-effective approach to detecting and monitoring cancer, even at its early stages. Due to the complexity of liquid biopsy data, machine-learning techniques are increasingly gaining attention in sample analysis, especially for multidimensional data such as RNA expression profiles. Yet, there is no agreement in the community on which methods are the most effective or how to process the data. To circumvent this, we performed a large-scale study using various machine-learning techniques. First, we took a closer look at existing datasets and filtered out some patients to assert data collection quality. The final data collection included platelet RNA samples acquired from 1397 cancer patients (17 types of cancer) and 354 asymptomatic, presumed healthy, donors. Then, we assessed an array of different machine-learning models and techniques (e.g., feature selection of RNA transcripts) in pan-cancer detection and multiclass classification. Our results show that simple logistic regression performs the best, reaching a 68% cancer detection rate at a 99% specificity level, and multiclass classification accuracy of 79.38% when distinguishing between five cancer types. In summary, by revisiting classical machine-learning models, we have exceeded the previously used method by 5% and 9.65% in cancer detection and multiclass classification, respectively. To ease further research, we open-source our code and data processing pipelines (https://gitlab.com/jopekmaksym/improving-platelet-rna-based-diagnostics), which we hope will serve the community as a strong baseline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/479013c1f2cb89512c4830381ffb51b56ae291e2" target='_blank'>
              Improving platelet-RNA-based diagnostics: a comparative analysis of machine learning models for cancer detection and multiclass classification.
              </a>
            </td>
          <td>
            M. Jopek, Krzysztof Pastuszak, Michal Sieczczynski, Sebastian Cygert, A. Żaczek, Matthew T Rondina, A. Supernat
          </td>
          <td>2024-06-17</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Recently, Conditional Neural Fields (NeFs) have emerged as a powerful modelling paradigm for PDEs, by learning solutions as flows in the latent space of the Conditional NeF. Although benefiting from favourable properties of NeFs such as grid-agnosticity and space-time-continuous dynamics modelling, this approach limits the ability to impose known constraints of the PDE on the solutions -- e.g. symmetries or boundary conditions -- in favour of modelling flexibility. Instead, we propose a space-time continuous NeF-based solving framework that - by preserving geometric information in the latent space - respects known symmetries of the PDE. We show that modelling solutions as flows of pointclouds over the group of interest $G$ improves generalization and data-efficiency. We validated that our framework readily generalizes to unseen spatial and temporal locations, as well as geometric transformations of the initial conditions - where other NeF-based PDE forecasting methods fail - and improve over baselines in a number of challenging geometries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8295fdfb6de078027cabc7d49b319df8abb0ee" target='_blank'>
              Space-Time Continuous PDE Forecasting using Equivariant Neural Fields
              </a>
            </td>
          <td>
            David M. Knigge, David R. Wessels, Riccardo Valperga, Samuele Papa, J. Sonke, E. Gavves, E. J. Bekkers
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>56</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a complex ecosystem of diverse cell types whose interactions govern tumor growth and clinical outcome. While the TME’s impact on immunotherapy has been extensively studied, its role in chemotherapy response remains less explored. To address this, we developed DECODEM (DEcoupling Cell-type-specific Outcomes using DEconvolution and Machine learning), a generic computational framework leveraging cellular deconvolution of bulk transcriptomics to associate the gene expression of individual cell types in the TME with clinical response. Employing DECODEM to analyze the gene expression of breast cancer (BC) patients treated with neoadjuvant chemotherapy, we find that the gene expression of specific immune cells (myeloid, plasmablasts, B-cells) and stromal cells (endothelial, normal epithelial, CAFs) are highly predictive of chemotherapy response, going beyond that of the malignant cells. These findings are further tested and validated in a single-cell cohort of triple negative breast cancer. To investigate the possible role of immune cell-cell interactions (CCIs) in mediating chemotherapy response, we extended DECODEM to DECODEMi to identify such CCIs, validated in single-cell data. Our findings highlight the importance of active pre-treatment immune infiltration for chemotherapy success. The tools developed here are made publicly available and are applicable for studying the role of the TME in mediating response from readily available bulk tumor expression in a wide range of cancer treatments and indications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec38ea4f715fa72ae5e3c095b32d63b6ede6629" target='_blank'>
              The expression patterns of different cell types and their interactions in the tumor microenvironment are predictive of breast cancer patient response to neoadjuvant chemotherapy
              </a>
            </td>
          <td>
            S. R. Dhruba, S. Sahni, Binbin Wang, Di Wu, P. Rajagopal, Yael Schmidt, E. Shulman, Sanju Sinha, Stephen-John Sammut, Carlos Caldas, Kun Wang, E. Ruppin
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The obscure drug response continues to be a limiting factor for accurate cures for cancer. Next generation sequencing technologies have propelled the pharmacogenomic studies with characterized large panels of cancer cell line at multi‐omics level. However, the sufficient integration of the multi‐omics data and the efficient prediction for drug response and synergy still remain a challenge. To address these problems, ECFD is designed, an ensemble cascade forest‐based framework that predicts drug response and synergy using five types of omics data. Experimental results show the significant advantages of the ECFD model over existing models. The best integration of feature extraction is determined and the superiorities of robust stability in the face of new and small samples are highlighted. In addition, the methodological framework highlights the explainability of the model, the mechanisms of drug resistance and drug combination treatment strategies based on explainable analyses and biological networks. In sum, ECFD may facilitate the evaluation of drug response and speculation of potential synergy therapies in personalized and precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202be07a703e58d34b9d528b6a979074db9400b4" target='_blank'>
              An Ensemble Cascade Forest‐Based Framework for Multi‐Omics Drug Response and Synergy Prediction
              </a>
            </td>
          <td>
            Ruijiang Li, Binsheng Sui, Dongjing Leng, Song He, Kunhong Liu, Xiaochen Bo
          </td>
          <td>2024-06-09</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="MOTIVATION
Predicting cancer drug response requires a comprehensive assessment of many mutations present across a tumor genome. While current drug response models generally use a binary mutated/unmutated indicator for each gene, not all mutations in a gene are equivalent.


RESULTS
Here, we construct and evaluate a series of predictive models based on leading methods for quantitative mutation scoring. Such methods include VEST4 and CADD, which score the impact of a mutation on gene function, and CHASMplus, which scores the likelihood a mutation drives cancer. The resulting predictive models capture cellular responses to dabrafenib, which targets BRAF-V600 mutations, whereas models based on binary mutation status do not. Performance improvements generalize to other drugs, extending genetic indications for PIK3CA, ERBB2, EGFR, PARP1, and ABL1 inhibitors. Introducing quantitative mutation features in drug response models increases performance and mechanistic understanding.


AVAILABILITY AND IMPLEMENTATION
Code and example datasets are available at https://github.com/pgwall/qms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b78691ef04081b6ccf1c8699cd4747a45bf1d4f" target='_blank'>
              Representing mutations for predicting cancer drug response.
              </a>
            </td>
          <td>
            Patrick Wall, T. Ideker
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="e13641 Background: Patient eligibility is a routine concept in clinical research, comprised of simple or complex set of inclusion or exclusion criteria applied in a specific sequence. Our methodology introduces the concept of ‘availability’ as a narrower term that is closer to the actual number that will be considered for a specific study and who may decide to participate. This specific research develops Machine Learning (ML) model that dynamically predicts likelihood of Multiple Myeloma progression and departure from targeted outcomes to sustain therapy and to indicate potential availability for clinical trials – to reduce screening efforts and improve screening precision of clinical research organizations. Methods: The cohort for training these models was drawn from the ConcertAI Oncology Research database, further enriched by key variables derived from unstructured data sources. Eligible patients for the analysis had a diagnosis of Multiple Myeloma made after Jan 1, 2020. Progressions were labelled using International Myeloma Working Group Criteria. Patients that did not satisfy the criteria were censored. Labels were assigned to patients after initial diagnosis of MM, and the duration between diagnosis to label was computed to build time to event (TTE) models. Over 2000 features were created to build this model, e.g., laboratory values, vitals, biomarkers, medications, ECOG, stage, etc. Variable creation preserved temporality of lab variables and vitals and noted presence of other variables. Several ML methods were employed to build the TTE models, with gradient boosting resulting in the highest c-index (0.8) based on 12 clinically relevant features. A temporal validation of the model was performed on a simulated index date. Results: Cohort used for training the model comprised of N = 4315 patients with average age at diagnosis of 72.3 years, and ~11% of the patients labelled as progressing. Temporal validation was performed on N = 1874 patients, with ~9% progression rate. The area under the curve (AUC) and weighted F1 score for this cohort was 87% and 89% respectively. Conclusions: Patient availability assessments are complex, not supported effectively by existing solutions and often require 100’s of hours from highly trained clinical personnel. An AI model predicting Multiple Myeloma progression built using a large-scale real-world dataset demonstrated 2-4x improvement in forecasting patient availability over other approaches. Integrating these solutions into clinical workflows can improve the effectiveness of research teams; enable more patients to have clinical trials as viable options at the right time; and allow care teams to have forward visibility to choices of standard-of-care and clinical trial alternatives in the best interest of patient outcomes. [Table: see text]">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71f96261be2e68d4c685fffbbf7d6bf43b30dfae" target='_blank'>
              Dynamic machine learning model to forecast patient availability for clinical trials.
              </a>
            </td>
          <td>
            Rajeev Kulkarni, Vivek Prabhakar Vaidya, Dhaval Parmar, Abhishek Tibrewal, R. B. Parikh
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Exploring the degree to which phenotypic variation, influenced by intrinsic nonlinear biological mechanisms, can be accurately captured using statistical methods is essential for advancing our comprehension of complex biological systems and predicting their functionality. Here, we examine this issue by combining a computational model of gene regulation networks with a linear additive prediction model, akin to polygenic scores utilized in genetic analyses. Inspired by the variational framework of quantitative genetics, we create a population of individual networks possessing identical topology yet showcasing diversity in regulatory strengths. By discerning which regulatory connections determine the prediction of phenotypes, we contextualize our findings within the framework of core and peripheral causal determinants, as proposed by the omnigenic model of complex traits. We establish connections between our results and concepts such as global sensitivity and local stability in dynamical systems, alongside the notion of sloppy parameters in biological models. Furthermore, we explore the implications of our investigation for the broader discourse surrounding the role of epistatic interactions in the prediction of complex phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9d655e857d6cdd755c46f5fbfbbbb85e241f74" target='_blank'>
              Polygenic prediction and gene regulation networks
              </a>
            </td>
          <td>
            J. F. Poyatos
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="How complex phenotypes emerge from intricate gene expression patterns is a fundamental question in biology. Quantitative characterization of this relationship, however, is challenging due to the vast combinatorial possibilities and dynamic interplay between genotype and phenotype landscapes. Integrating high-content genotyping approaches such as single-cell RNA sequencing and advanced learning methods such as language models offers an opportunity for dissecting this complex relationship. Here, we present a computational integrated genetics framework designed to analyze and interpret the high-dimensional landscape of genotypes and their associated phenotypes simultaneously. We applied this approach to develop a multimodal foundation model to explore the genotype-phenotype relationship manifold for human transcriptomics at the cellular level. Analyzing this joint manifold showed a refined resolution of cellular heterogeneity, enhanced precision in phenotype annotating, and uncovered potential cross-tissue biomarkers that are undetectable through conventional gene expression analysis alone. Moreover, our results revealed that the gene networks are characterized by scale-free patterns and show context-dependent gene-gene interactions, both of which result in significant variations in the topology of the gene network, particularly evident during aging. Finally, utilizing contextualized embeddings, we investigated gene polyfunctionality which illustrates the multifaceted roles that genes play in different biological processes, and demonstrated that for VWF gene in endothelial cells. Overall, this study advances our understanding of the dynamic interplay between gene expression and phenotypic manifestation and demonstrates the potential of integrated genetics in uncovering new dimensions of cellular function and complexity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4827f60ad81afcd6b7c4af4bb58fc23e2809c1" target='_blank'>
              Multimodal Learning for Mapping the Genotype-Phenotype Dynamics
              </a>
            </td>
          <td>
            Farhan Khodaee, Rohola Zandie, E. Edelman
          </td>
          <td>2024-05-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Abstract Motivation In drug discovery, it is crucial to assess the drug–target binding affinity (DTA). Although molecular docking is widely used, computational efficiency limits its application in large-scale virtual screening. Deep learning-based methods learn virtual scoring functions from labeled datasets and can quickly predict affinity. However, there are three limitations. First, existing methods only consider the atom-bond graph or one-dimensional sequence representations of compounds, ignoring the information about functional groups (pharmacophores) with specific biological activities. Second, relying on limited labeled datasets fails to learn comprehensive embedding representations of compounds and proteins, resulting in poor generalization performance in complex scenarios. Third, existing feature fusion methods cannot adequately capture contextual interaction information. Results Therefore, we propose a novel DTA prediction method named HeteroDTA. Specifically, a multi-view compound feature extraction module is constructed to model the atom–bond graph and pharmacophore graph. The residue concat graph and protein sequence are also utilized to model protein structure and function. Moreover, to enhance the generalization capability and reduce the dependence on task-specific labeled data, pre-trained models are utilized to initialize the atomic features of the compounds and the embedding representations of the protein sequence. A context-aware nonlinear feature fusion method is also proposed to learn interaction patterns between compounds and proteins. Experimental results on public benchmark datasets show that HeteroDTA significantly outperforms existing methods. In addition, HeteroDTA shows excellent generalization performance in cold-start experiments and superiority in the representation learning ability of drug–target pairs. Finally, the effectiveness of HeteroDTA is demonstrated in a real-world drug discovery study. Availability and implementation The source code and data are available at https://github.com/daydayupzzl/HeteroDTA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38cae834827755fe52cdc2576ba9cfde2dc9d11b" target='_blank'>
              Enhancing generalizability and performance in drug–target interaction identification by integrating pharmacophore and pre-trained models
              </a>
            </td>
          <td>
            Zuolong Zhang, Xin He, Dazhi Long, Gang Luo, Shengbo Chen
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4042b81dac3cbb2b1c6eb651f000e5a8614aac9d" target='_blank'>
              Machine learning identifies prognostic subtypes of the tumor microenvironment of NSCLC
              </a>
            </td>
          <td>
            Duo Yu, M. Kane, Eugene J. Koay, I. Wistuba, Brian,P. Hobbs
          </td>
          <td>2024-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11e1c2a5d86f70ab1b9f878c3469d6237c1ad691" target='_blank'>
              Deep learning models for predicting the survival of patients with medulloblastoma based on a surveillance, epidemiology, and end results analysis
              </a>
            </td>
          <td>
            Meng Sun, Jikui Sun, M. Li
          </td>
          <td>2024-06-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Accumulation models, where a system progressively acquires binary features over time, are common in the study of cancer progression, evolutionary biology, and other fields. Many approaches have been developed to infer the accumulation pathways by which features (for example, mutations) are acquired over time. However, most of these approaches do not support reversibility: the loss of a feature once it has been acquired (for example, the clearing of a mutation from a tumour or population). Here, we demonstrate how the well-established Mk model from evolutionary biology, embedded on a hypercubic transition graph, can be used to infer the dynamics of accumulation processes, including the possibility of reversible transitions, from data which may be uncertain and cross-sectional, longitudinal, or phylogenetically / phylogenomically embedded. Positive and negative interactions between arbitrary sets of features (not limited to pairwise interactions) are supported. We demonstrate this approach with synthetic datasets and real data on bacterial drug resistance and cancer progression. While this implementation is limited in the number of features that can be considered, we discuss how this limitation may be relaxed to deal with larger systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/096bc6a723931c9c16f263760f1c2f885bca8b3c" target='_blank'>
              A hypercubic Mk model framework for capturing reversibility in disease, cancer, and evolutionary accumulation modelling
              </a>
            </td>
          <td>
            Iain G. Johnston, Ramon Diaz-Uriarte
          </td>
          <td>2024-07-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Black box deep learning models trained on genomic sequences excel at predicting the outcomes of different gene regulatory mechanisms. Therefore, interpreting these models may provide novel insights into the underlying biology, supporting downstream biomedical applications. Due to their complexity, interpretable surrogate models can only be built for local explanations (e.g., a single instance). However, accomplishing this requires generating a dataset in the neighborhood of the input, which must maintain syntactic similarity to the original data while introducing semantic variability in the model's predictions. This task is challenging due to the complex sequence-to-function relationship of DNA. We propose using Genetic Programming to generate datasets by evolving perturbations in sequences that contribute to their semantic diversity. Our custom, domain-guided individual representation effectively constrains syntactic similarity, and we provide two alternative fitness functions that promote diversity with no computational effort. Applied to the RNA splicing domain, our approach quickly achieves good diversity and significantly outperforms a random baseline in exploring the search space, as shown by our proof-of-concept, short RNA sequence. Furthermore, we assess its generalizability and demonstrate scalability to larger sequences, resulting in a $\approx$30\% improvement over the baseline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef43bb7883ab3d6b68f8e2d282f90a2d9dd3b4ff" target='_blank'>
              Semantically Rich Local Dataset Generation for Explainable AI in Genomics
              </a>
            </td>
          <td>
            Pedro Barbosa, Rosina Savisaar, Alcides Fonseca
          </td>
          <td>2024-07-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Numerous Pre-trained cell foundation models (CFM) have been crafted to encapsulate the comprehensive gene-gene interaction network within cells, leveraging extensive single-cell sequencing data. These models have shown promise in various cell biology applications, including cell type annotation, perturbation inference, and cell state embedding, etc. However, their clinical utility, particularly in cancer diagnosis and prognosis, remains an open question. We introduce the GeneBag model, a novel CFM that represents a cell as “a bag of unordered genes” with continuous expression values and a full-length gene list. Pre-trained on single-cell data and fine-tuned on bulk RNA-seq datasets, GeneBag achieves superior performance across cancer diagnosis and prognosis scenarios. In a zero-shot learning setting, GeneBag can classify cancer and non-cancer tissues with approximately 96.2% accuracy. With fine-tuning, it can annotate 40 different types of cancers and corresponding normal biopsies with an overall accuracy of ∼97.2%. It notably excels in classifying challenging cancers such as bladder (93%) and stomach (90%). Furthermore, GeneBag is capable of cancer staging with 68.5% accuracy and 5-year survival prediction with an AUC of ∼80.4%. This study marks the first to demonstrate the potential of CFMs in RNA-based cancer diagnostics and prognostics, indicating a promising avenue for AI-assisted molecular diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb775112abc591a1a520cde17c597775a658b0b" target='_blank'>
              GeneBag: training a cell foundation model for broad-spectrum cancer diagnosis and prognosis with bulk RNA-seq data
              </a>
            </td>
          <td>
            Yu-ping Liang, Dan Li, Aguix Guohua Xu, Yan Shao, Kun Tang
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The integration of Graph Neural Networks (GNNs) and Neural Ordinary and Partial Differential Equations has been extensively studied in recent years. GNN architectures powered by neural differential equations allow us to reason about their behavior, and develop GNNs with desired properties such as controlled smoothing or energy conservation. In this paper we take inspiration from Turing instabilities in a Reaction Diffusion (RD) system of partial differential equations, and propose a novel family of GNNs based on neural RD systems. We \textcolor{black}{demonstrate} that our RDGNN is powerful for the modeling of various data types, from homophilic, to heterophilic, and spatio-temporal datasets. We discuss the theoretical properties of our RDGNN, its implementation, and show that it improves or offers competitive performance to state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57254c8083a6fe4f39ea4b61b190c97bb047ab60" target='_blank'>
              Graph Neural Reaction Diffusion Models
              </a>
            </td>
          <td>
            Moshe Eliasof, Eldad Haber, Eran Treister
          </td>
          <td>2024-06-16</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Supervised machine learning models rely on training datasets with positive (target class) and negative examples. Therefore, the composition of the training dataset has a direct influence on model performance. Specifically, negative sample selection bias, concerning samples not representing the target class, presents challenges across a range of domains such as text classification and protein-protein interaction prediction. Machine-learning-based immunotherapeutics design is an increasingly important area of research, focusing on designing antibodies or T-cell receptors (TCRs) that can bind to their target molecules with high specificity and affinity. Given the biomedical importance of immunotherapeutics, there is a need to address the unresolved question of how negative training set composition impacts model generalization and biological rule discovery to enable rational and safe drug design. We set out to study this question in the context of the antibody-antigen binding prediction problem by varying the negative class, encompassing a binding affinity gradient. We based our investigation on large synthetic datasets that provide ground truth structure-based antibody-antigen binding data, allowing access to residue-wise binding energy on the binding interface. We found that both out-of-distribution generalization and binding rule discovery depended on the type of negative dataset used. Importantly, we discovered that a model’s capacity to learn the binding rules of the positive dataset is not a trivial correlate of its classification accuracy. We confirmed our findings with real-world relevant experimental data. Our work highlights the importance of considering training dataset composition for achieving optimal out-of-distribution performance and rule learning in machine-learning-based research. Significance Statement The effectiveness of supervised machine learning models hinges on the composition of their training datasets, particularly the inclusion of negative examples. This bias in negative sample selection can greatly impact model performance. As the development of immunotherapeutic agents using machine learning is becoming increasingly crucial in biomedicine, understanding the impact of negative training set composition is imperative. Our study, focused on the antibody-antigen binding prediction problem, reveals that the choice of negative dataset significantly affects both out-of-distribution generalization and binding rule discovery across synthetic and experimental data. These findings underscore the necessity of carefully considering training dataset composition in machine-learning-driven biomedical research for optimal performance, robustness and meaningful rule acquisition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23c2f1758056f3fe1173a00d561a41e8889e3482" target='_blank'>
              Training data composition determines machine learning generalization and biological rule discovery
              </a>
            </td>
          <td>
            E. Ursu, Aygul Minnegalieva, Puneet Rawat, Maria Chernigovskaya, Robi Tacutu, G. K. Sandve, Philippe A. Robert, Victor Greiff
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cancers are commonly characterised by a complex pathology encompassing genetic, microscopic and macroscopic features, which can be probed individually using imaging and omics technologies. Integrating this data to obtain a full understanding of pathology remains challenging. We introduce a new method called Deep Latent Variable Path Modelling (DLVPM), which combines the representational power of deep learning with the capacity of path modelling to identify relationships between interacting elements in a complex system. To evaluate the capabilities of DLVPM, we initially trained a foundational model to map dependencies between SNV, Methylation, miRNA-Seq, RNA-Seq and Histological data using Breast Cancer data from The Cancer Genome Atlas (TCGA). This method exhibited superior performance in mapping associations between data types compared to classical path modelling. We additionally performed successful applications of the model to: stratify single-cell data, identify synthetic lethal interactions using CRISPR-Cas9 screens derived from cell-lines, and detect histologic-transcriptional associations using spatial transcriptomic data. Results from each of these data types can then be understood with reference to the same holistic model of illness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36a969a03ce741afe8ca32855f97d3ec725e814d" target='_blank'>
              Integrating Multi-Modal Cancer Data Using Deep Latent Variable Path Modelling
              </a>
            </td>
          <td>
            Alex Ing, Alvaro Andrades, M. R. Cosenza, J. Korbel
          </td>
          <td>2024-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="8614 Background: Advanced lung adenocarcinoma (aLUAD) is divided into multiple molecularly defined subsets, each with specific biological characteristics and responses to targeted therapy. Consequently, conducting practice-changing and cost-effective randomized trials is challenging. In silico clinical trials, utilizing mechanistic computational models, offer a potential solution, yet their ability to predict clinical trial results has not been proven. Methods: We updated a previously published version of the computational mechanistic model of EGFR-mutated aLUAD, notably by incorporating metastases. Disease model was integrated with physiologically based pharmacokinetic (PBPK) models of cisplatin, pemetrexed, and osimertinib, offering a more comprehensive understanding of the disease's dynamics. These models are founded on causal connections between biological entities and phenomena, establishing a link between the specific biological behaviors of EGFR-mutated aLUAD and the evolution of tumor size and progression over time. We have independently and prospectively simulated the outcomes of the FLAURA2 and MARIPOSA randomized trials dedicated to EGFR-mutated aLUAD. This was achieved by replicating the experimental protocols and generating cohorts of virtual twins of patients using publicly available data. Results: In silico results were released before the actual outcomes of the FLAURA2 and MARIPOSA trials were publicly available. The simulations yielded predictions with overlapping confidence intervals, similar hazard ratios, median survivals, and shapes of curves compared to the actual trial data. Bootstrapped weighted log-rank tests were used to compare the simulated and observed Kaplan-Meier curves for the two arms of FLAURA2 and for the osimertinib arm of MARIPOSA. Between 94 and 98% of the tests were statistically non-significant (α=0.05), confirming the accuracy of the predictions. Conclusions: This first-of-its-kind prediction of clinical trial outcomes demonstrates that in silico trials, when based on robust mechanistic models, can be a reliable tool for enhancing the design of actual trials, particularly for EGFR-mutated aLUAD. They offer a promising avenue for overcoming the challenges of comparator arms in single-arm trials and may serve as a new standard for formulating statistical hypotheses in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76086ed48f1470fe5129435b0503e60cd9f7700" target='_blank'>
              Prospective validation of in silico clinical trials prediction using an EGFR-mutated NSCLC mechanistic computational disease model.
              </a>
            </td>
          <td>
            Michaël Duruisseaux, P. Masson, Arnaud Nativel, Nicolas Girard, J. Cadranel, A. Swalduz, Matthieu Coudron, Guillaume Bouchard, R. Kahoul, Evgueni Jacob, J. Bosley, J. Palgen, A. L’Hostis, C. Monteiro
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="We present the first use of machine learning tools to predict multistationarity of reaction networks. Chemical Reaction Networks (CRNs) are the mathematical formulation of how the quantities associated to a set of species (molecules, proteins, cells, or animals) vary as time passes with respect to their interactions with each other. Their mathematics does not describe just chemical reactions but many other areas of the life sciences such as ecology, epidemiology, and population dynamics. We say a CRN is at a steady state when the concentration (or number) of species do not vary anymore. Some CRNs do not attain a steady state while some others may have more than one possible steady state. The CRNs in the later group are called multistationary. Multistationarity is an important property, e.g. switch-like behaviour in cells needs multistationarity to occur. Existing algorithms to detect whether a CRN is multistationary or not are either extremely expensive or restricted in the type of CRNs they can be used on, motivating a new machine learning approach. We address the problem of representing variable-length CRN data to machine learning models by developing a new graph representation of CRNs for use with graph learning algorithms. We contribute a large dataset of labelled fully open CRNs whose production necessitated the development of new CRN theory. Then we present experimental results on the training and testing of a graph attention network model on this dataset, showing excellent levels of performance. We finish by testing the model predictions on validation data produced independently, demonstrating generalisability of the model to different types of CRN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd097d9cd785834b1edb4051c13330afba0eaa4" target='_blank'>
              Designing Machine Learning Tools to Characterize Multistationarity of Fully Open Reaction Networks
              </a>
            </td>
          <td>
            Shenghao Yao, AmirHosein Sadeghimanesh, Matthew England
          </td>
          <td>2024-07-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Many causal systems such as biological processes in cells can only be observed indirectly via measurements, such as gene expression. Causal representation learning -- the task of correctly mapping low-level observations to latent causal variables -- could advance scientific understanding by enabling inference of latent variables such as pathway activation. In this paper, we develop methods for inferring latent variables from multiple related datasets (environments) and tasks. As a running example, we consider the task of predicting a phenotype from gene expression, where we often collect data from multiple cell types or organisms that are related in known ways. The key insight is that the mapping from latent variables driven by gene expression to the phenotype of interest changes sparsely across closely related environments. To model sparse changes, we introduce Tree-Based Regularization (TBR), an objective that minimizes both prediction error and regularizes closely related environments to learn similar predictors. We prove that under assumptions about the degree of sparse changes, TBR identifies the true latent variables up to some simple transformations. We evaluate the theory empirically with both simulations and ground-truth gene expression data. We find that TBR recovers the latent causal variables better than related methods across these settings, even under settings that violate some assumptions of the theory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/828daf97066cdcac9612888fea4854de9c05ee25" target='_blank'>
              Sparsity regularization via tree-structured environments for disentangled representations
              </a>
            </td>
          <td>
            Elliot Layne, Dhanya Sridhar, Jason S. Hartford, M. Blanchette
          </td>
          <td>2024-05-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The rising incidence of cancer underscores the imperative for innovative diagnostic and prognostic methodologies. This study delves into the potential of RNA-Seq gene expression data to enhance cancer classification accuracy. Introducing a pioneering approach, we model gene expression data as point clouds, capitalizing on the data’s intrinsic properties to bolster classification performance. Utilizing PointNet, a typical technique for processing point cloud data, as our framework’s cornerstone, we incorporate inductive biases pertinent to gene expression and pathways. This integration markedly elevates model efficacy, culminating in developing an end-to-end deep learning classifier with an accuracy rate surpassing 99%. Our findings not only illuminate the capabilities of AI-driven models in the realm of oncology but also highlight the criticality of acknowledging biological dataset nuances in model design. This research provides insights into application of deep learning in medical science, setting the stage for further innovation in cancer classification through sophisticated biological data analysis. The source code for our study is accessible at: https://github.com/cialab/GPNet.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c85ae4150a56cebe269a38a88f677eeec386e5b" target='_blank'>
              Gene PointNet for Tumor Classification
              </a>
            </td>
          <td>
            Hao Lu, Mostafa Rezapour, Haseebullah Baha, Muhammad Khalid Khan Niazi, Aarthi Narayanan, Metin Nafi Gurcan
          </td>
          <td>2024-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Current medical limitations in predicting cancer survival status and time necessitate advancements beyond traditional methods and physical indicators. This research introduces a novel two-stage prognostic framework for renal cell carcinoma, addressing the inadequacies of existing diagnostic approaches. In the first stage, the framework accurately predicts the survival status (alive or deceased) with metrics Accuracy, Precision, Recall, and F1 score to evaluate the effects of the classification results, while the second stage focuses on forecasting the future survival time of deceased patients with Root Mean Square Error and Mean Absolute Error to evaluate the regression results. Leveraging popular machine learning models, such as Adaptive Boosting, Extra Trees, Gradient Boosting, Random Forest, and Extreme Gradient Boosting, along with fusion models like Voting, Stacking, and Blending, our approach significantly improves prognostic accuracy as shown in our experiments. The novelty of our research lies in the integration of a logistic regression meta-model for interpreting the blending model’s predictions, enhancing transparency. By the SHapley Additive exPlanations’ interpretability, we provide insights into variable contributions, aiding understanding at both global and local levels. Through modal segmentation and multimodal fusion applied to raw data from the Surveillance, Epidemiology, and End Results program, we enhance the precision of renal cell carcinoma prognosis. Our proposed model provides an interpretable analysis of model predictions, highlighting key variables influencing classification and regression decisions in the two-stage renal cell carcinoma prognosis framework. By addressing the black-box problem inherent in machine learning, our proposed model helps healthcare practitioners with a more reliable and transparent basis for applying machine learning in cancer prognostication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c5bf32f78b871e516aeb2c6d33ef41469bb2a1a" target='_blank'>
              Multimodal Machine Learning for Prognosis and Survival Prediction in Renal Cell Carcinoma Patients: A Two-Stage Framework with Model Fusion and Interpretability Analysis
              </a>
            </td>
          <td>
            Keyue Yan, Simon Fong, Tengyue Li, Qun Song
          </td>
          <td>2024-06-29</td>
          <td>Applied Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Machine learning is increasingly employed in engineering, with one of its primary applications being the construction of surrogate models to replace computationally expensive physical models for analysis and design, especially control co-design which integrates physical and control system design. In cases where generating labels for training through physical models is computationally intensive, label-free machine learning offers a viable alternative. However, surrogate models built from label-free machine learning typically have prediction errors, which can be characterized and quantified through epistemic uncertainty, representing model-form uncertainty. Moreover, when these surrogate models are used in optimization design for real-world applications, inherent random variables introduce aleatory uncertainty. This study introduces a robust design optimization method that addresses the intertwined epistemic and aleatory uncertainty. By optimizing both the average product performance and reducing uncertainty stemming from the coupled uncertainty, the method achieves improved robustness. A four-bar linkage mechanism design serves as a demonstration of this approach. The surrogate model for the design is constructed using label-free neural network, accommodating a system of physical equations. The error of the surrogate model is assessed through Gaussian Process regression, using existing training points and derivatives of the physical equations at these points. The design of the four-bar linkage aims to minimize both its average motion error and the variability of the error attributed to coupled uncertainty.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef8bdfc8b842dd50054b97d50b7f9695cf4df0d9" target='_blank'>
              Optimization Under Uncertainty Using Physics-Based Label-Free Machine Learning
              </a>
            </td>
          <td>
            Xiaoping Du
          </td>
          <td>2024-05-23</td>
          <td>2024 International Congress on Human-Computer Interaction, Optimization and Robotic Applications (HORA)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intratumour phenotypic heterogeneity is nowadays understood to play a critical role in disease progression and treatment failure. Accordingly, there has been increasing interest in the development of mathematical models capable of capturing its role in cancer cell adaptation. This can be systematically achieved by means of models comprising phenotype-structured nonlocal partial differential equations, tracking the evolution of the phenotypic density distribution of the cell population, which may be compared to gene and protein expression distributions obtained experimentally. Nevertheless, given the high analytical and computational cost of solving these models, much is to be gained from reducing them to systems of ordinary differential equations for the moments of the distribution. We propose a generalised method of model-reduction, relying on the use of a moment generating function, Taylor series expansion and truncation closure, to reduce a nonlocal reaction-advection-diffusion equation, with general phenotypic drift and proliferation rate functions, to a system of moment equations up to arbitrary order. Our method extends previous results in the literature, which we address via two examples, by removing any \textit{a priori} assumption on the shape of the distribution, and provides a flexible framework for mathematical modellers to account for the role of phenotypic heterogeneity in cancer adaptive dynamics, in a simpler mathematical framework.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d951ebfbeb9789f55de6ae13ba4fa4d0ddb0abc" target='_blank'>
              Reducing phenotype-structured PDE models of cancer evolution to systems of ODEs: a generalised moment dynamics approach
              </a>
            </td>
          <td>
            Chiara Villa, P. K. Maini, Alexander P Browning, A. Jenner, Sara Hamis, Tyler Cassidy
          </td>
          <td>2024-06-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In the present article, we propose a paradigm shift on evolving Artificial Neural Networks (ANNs) towards a new bio-inspired design that is grounded on the structural properties, interactions, and dynamics of protein networks (PNs): the Artificial Protein Network (APN). This introduces several advantages previously unrealized by state-of-the-art approaches in NE: (1) We can draw inspiration from how nature, thanks to millions of years of evolution, efficiently encodes protein interactions in the DNA to translate our APN to silicon DNA. This helps bridge the gap between syntax and semantics observed in current NE approaches. (2) We can learn from how nature builds networks in our genes, allowing us to design new and smarter networks through EA evolution. (3) We can perform EA crossover/mutation operations and evolution steps, replicating the operations observed in nature directly on the genotype of networks, thus exploring and exploiting the phenotypic space, such that we avoid getting trapped in sub-optimal solutions. (4) Our novel definition of APN opens new ways to leverage our knowledge about different living things and processes from biology. (5) Using biologically inspired encodings, we can model more complex demographic and ecological relationships (e.g., virus-host or predator-prey interactions), allowing us to optimise for multiple, often conflicting objectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdbe8cd6d06b80d544472cbb9efc805782d0782" target='_blank'>
              Protein pathways as a catalyst to directed evolution of the topology of artificial neural networks
              </a>
            </td>
          <td>
            Oscar Lao, Konstantinos Zacharopoulos, A. Fournaris, Rossano Schifanella, Ioannis Arapakis
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Co-folding models are the newest innovation in deep-learning-based protein-ligand structure prediction. The recent publications of RoseTTAFold All-Atom and AlphaFold 3 have shown high-quality results on predicting the structures of proteins interacting with small-molecules, other proteins and nucleic-acids. Despite these advanced capabilities and broad potential, the current study presents critical findings that question these models’ adherence to fundamental physical principles and its susceptibility to overfitting specific protein families. Through adversarial examples based on established physical, chemical, and biological principles, we demonstrate notable discrepancies in protein-ligand structural predictions when subjected to biologically plausible perturbations. These discrepancies reveal a significant divergence from expected physical behaviors, indicating potential overfitting to particular data subsets within its training corpus. Our findings underscore the models’ limitations in generalizing effectively across diverse biological structures and highlight the necessity of integrating robust physical and chemical priors in the development of such predictive tools. The results advocate a measured reliance on deep-learning-based models for critical applications in drug discovery and protein engineering, where a deep understanding of the underlying physical and chemical properties is crucial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246574ae80d8bdbec4e6397732952f7f393a2387" target='_blank'>
              Do Deep Learning Models for Co-Folding Learn the Physics of Protein-Ligand Interactions?
              </a>
            </td>
          <td>
            Matthew Masters, Amr H. Mahmoud, Markus A. Lill
          </td>
          <td>2024-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In the field of drug discovery, a proliferation of pre-trained models has surfaced, exhibiting exceptional performance across a variety of tasks. However, the extensive size of these models, coupled with the limited interpretative capabilities of current fine-tuning methods, impedes the integration of pre-trained models into the drug discovery process. This paper pushes the boundaries of pre-trained models in drug discovery by designing a novel fine-tuning paradigm known as the Head Feature Parallel Adapter (HFPA), which is highly interpretable, high-performing, and has fewer parameters than other widely used methods. Specifically, this approach enables the model to consider diverse information across representation subspaces concurrently by strategically using Adapters, which can operate directly within the model's feature space. Our tactic freezes the backbone model and forces various small-size Adapters' corresponding subspaces to focus on exploring different atomic and chemical bond knowledge, thus maintaining a small number of trainable parameters and enhancing the interpretability of the model. Moreover, we furnish a comprehensive interpretability analysis, imparting valuable insights into the chemical area. HFPA outperforms over seven physiology and toxicity tasks and achieves state-of-the-art results in three physical chemistry tasks. We also test ten additional molecular datasets, demonstrating the robustness and broad applicability of HFPA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89c47edeb51c005371552573aa512d2ca9791cf9" target='_blank'>
              Advancing the Boundary of Pre-trained Models for Drug Discovery: Interpretable Fine-Tuning Empowered by Molecular Physicochemical Properties.
              </a>
            </td>
          <td>
            Xiaoqing Lian, Jie Zhu, Tianxu Lv, Xiaoyan Hong, Longzhen Ding, Wei Chu, Jianming Ni, Xiang Pan
          </td>
          <td>2024-06-18</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract In precision medicine, both predicting the disease susceptibility of an individual and forecasting its disease-free survival are areas of key research. Besides the classical epidemiological predictor variables, data from multiple (omic) platforms are increasingly available. To integrate this wealth of information, we propose new methodology to combine both cooperative learning, a recent approach to leverage the predictive power of several datasets, and polygenic hazard score models. Polygenic hazard score models provide a practitioner with a more differentiated view of the predicted disease-free survival than the one given by merely a point estimate, for instance computed with a polygenic risk score. Our aim is to leverage the advantages of cooperative learning for the computation of polygenic hazard score models via Cox’s proportional hazard model, thereby improving the prediction of the disease-free survival. In our experimental study, we apply our methodology to forecast the disease-free survival for Alzheimer’s disease (AD) using three layers of data. One layer contains epidemiological variables such as sex, APOE (apolipoprotein E, a genetic risk factor for AD) status and 10 leading principal components. Another layer contains selected genomic loci, and the last layer contains methylation data for selected CpG sites. We demonstrate that the survival curves computed via cooperative learning yield an AUC of around \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $0.7$\end{document}, above the state-of-the-art performance of its competitors. Importantly, the proposed methodology returns (1) a linear score that can be easily interpreted (in contrast to machine learning approaches), and (2) a weighting of the predictive power of the involved data layers, allowing for an assessment of the importance of each omic (or other) platform. Similarly to polygenic hazard score models, our methodology also allows one to compute individual survival curves for each patient.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9168bf7c51756564400a8c3a96e0936a62287e5" target='_blank'>
              Prediction of disease-free survival for precision medicine using cooperative learning on multi-omic data
              </a>
            </td>
          <td>
            Georg Hahn, D. Prokopenko, J. Hecker, S. M. Lutz, K. Mullin, Leinal Sejour, Winston Hide, Ioannis Vlachos, Stacia DeSantis, Rudolph E. Tanzi, Christoph Lange
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Deep learning has emerged as a transformative technology in data science, revolutionizing various domains through its powerful capabilities. This paper explores the theoretical foundations, practical applications, and comparative analysis of deep learning models. The theoretical foundations section discusses key neural network architectures such as Convolutional Neural Networks (CNNs), Recurrent Neural Networks (RNNs), and Transformers, highlighting their unique capabilities in processing different types of data. Optimization algorithms crucial for effective training, including Stochastic Gradient Descent (SGD) and Adam, are examined. Regularization techniques for preventing overfitting and enhancing generalization are also addressed. Practical applications in healthcare, finance, and retail showcase the real-world impact of deep learning. A comparative analysis of performance metrics demonstrates the superiority of deep learning models over traditional methods. Despite their advantages, deep learning models face limitations and challenges, including data dependency and interpretability issues. The paper concludes by emphasizing the ongoing research efforts to mitigate these challenges and ensure the continued advancement of deep learning in data science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b019728ec1ec6b00f0a46d426f7a25c881bbf25" target='_blank'>
              Deep learning in data science: Theoretical foundations, practical applications, and comparative analysis
              </a>
            </td>
          <td>
            Yingxuan Chai, Liangning Jin
          </td>
          <td>2024-06-21</td>
          <td>Applied and Computational Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia, a pervasive form of the carcinogenic disease, is a lethal ailment subjecting numerous pediatric patients globally to terminal conditions. is a rapidly progressive condition, that exposes patients to conditions including Tumor Lysis Syndrome which often occurs early after the induction chemotherapy, contemporary research focuses primarily on the development of techniques for the early diagnosis of Acute Lymphoblastic Leukemia (ALL), leaving a gap within the literature. This study examines the application of machine learning techniques for the prognosis the mutation rate of cancer cells in pediatric patients with Acute Lymphoblastic Leukemia using clinical data from patients with ALL, who have undergone tests using Next Generation Sequencing (NGS) technology. An overview of the clinical data utilized is provided in this study, with a comprehensive workflow encompassing, data analysis, dimensionality reduction, classification and regression tree algorithm (CART), and neural networks. Results here demonstrate the efficiency with which these methods are able to target and decipher cancer cell proliferation in pediatric patients suffering from acute lymphoblastic leukemia. Valuable insights into relationships between key factors and conversion rates were also derived through data mining. However, tree classification and regression algorithms and neural networks used herein indicate the flexibility and the power of machine learning models in predicting the recurrence of cancer cells accurately. This study’s results affirm previous findings thus giving clinical proof for mutational drivers among pediatric patients having Acute Lymphoblastic Leukemia. This adds value to results by providing an applicable utility in medical practice. Principally, this study denotes a substantial advancement in leveraging machine learning workflows for mutation rate analysis of cancer cells. By appraising clinical corroboration, emphasizing the explain ability and interpretability, and building upon these findings, future research can contribute to improving patient care and results in the field of Leukaemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c3003ce399e57bfbeed0fdbd2dbe8c5d3c42c03" target='_blank'>
              Stochastic Machine Learning Models for Mutation Rate Analysis of Malignant Cancer Cells in Patients with Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Martsenyuk Vasyl, Abubakar Umar Sadiq, Sverstiuk Andriy, Dimitrov Georgi, Gancarczyk Tomasz
          </td>
          <td>2024-07-05</td>
          <td>International Journal of Circuits, Systems and Signal Processing</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Identifying functionally important cell states and structure within a heterogeneous tumor remains a significant biological and computational challenge. Moreover, current clustering or trajectory-based computational models are ill-equipped to address the notion that cancer cells reside along a phenotypic continuum. To address this, we present Archetypal Analysis network (AAnet), a neural network that learns key archetypal cell states within a phenotypic continuum of cell states in single-cell data. Applied to single-cell RNA sequencing data from pre-clinical models and a cohort of 34 clinical breast cancers, AAnet identifies archetypes that resolve distinct biological cell states and processes, including cell proliferation, hypoxia, metabolism and immune interactions. Notably, archetypes identified in primary tumors are recapitulated in matched liver, lung and lymph node metastases, demonstrating that a significant component of intratumoral heterogeneity is driven by cell intrinsic properties. Using spatial transcriptomics as orthogonal validation, AAnet-derived archetypes show discrete spatial organization within tumors, supporting their distinct archetypal biology. We further reveal that ligand:receptor cross-talk between cancer and adjacent stromal cells contributes to intra-archetypal biological mimicry. Finally, we use AAnet archetype identifiers to validate GLUT3 as a critical mediator of a hypoxic cell archetype harboring a cancer stem cell population, which we validate in human triple-negative breast cancer specimens. AAnet is a powerful tool to reveal functional cell states within complex samples from multimodal single-cell data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32b3a7328824f9e25b9a694c5c5c104d8ffd3103" target='_blank'>
              AAnet resolves a continuum of spatially-localized cell states to unveil tumor complexity
              </a>
            </td>
          <td>
            Aarthi Venkat, Scott E. Youlten, B. P. S. Juan, Carley Purcell, Matthew Amodio, Daniel B. Burkhardt, Andrew Benz, Jeff Holst, Cerys McCool, A. Mollbrink, J. Lundeberg, David van Dijk, Leonard D Goldstein, Sarah Kummerfeld, Smita Krishnaswamy, C. Chaffer
          </td>
          <td>2024-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Large language models (LLMs) have demonstrated potential in the innovation of many disciplines. However, how they can best be developed for oncology remains underdeveloped. State-of-the-art OpenAI models were fine-tuned on a clinical dataset and clinical guidelines text corpus for two important cancer treatment factors, adjuvant radiation therapy and chemotherapy, using a novel Langchain prompt engineering pipeline. A high accuracy (0.85+) was achieved in the classification of adjuvant radiation therapy and chemotherapy for breast cancer patients. Furthermore, a confidence interval was formed from observational data on the quality of treatment from human oncologists to estimate the proportion of scenarios in which the model must outperform the original oncologist in its treatment prediction to be a better solution overall as 8.2% to 13.3%. Due to indeterminacy in the outcomes of cancer treatment decisions, future investigation, potentially a clinical trial, would be required to determine if this threshold was met by the models. Nevertheless, with 85% of U.S. cancer patients receiving treatment at local community facilities, these kinds of models could play an important part in expanding access to quality care with outcomes that lie, at minimum, close to a human oncologist.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcfef9abf40f3af1d41523a3a4bd2288447f71d1" target='_blank'>
              A Large Language Model Pipeline for Breast Cancer Oncology
              </a>
            </td>
          <td>
            Tristen Pool, Dennis Trujillo
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Emerging developments in artificial intelligence have opened infinite possibilities for material simulation. Depending on the powerful fitting of machine learning algorithms to first-principles data, machine learning interatomic potentials (MLIPs) can effectively balance the accuracy and efficiency problems in molecular dynamics (MD) simulations, serving as powerful tools in various complex physicochemical systems. Consequently, this brings unprecedented enthusiasm for researchers to apply such novel technology in multiple fields to revisit the major scientific problems that have remained controversial owing to the limitations of previous computational methods. Herein, we introduce the evolution of MLIPs, provide valuable application examples for solid-liquid interfaces, and present current challenges. Driven by solving multitudinous difficulties in terms of the accuracy, efficiency, and versatility of MLIPs, this booming technique, combined with molecular simulation methods, will provide an underlying and valuable understanding of interdisciplinary scientific challenges, including materials, physics, and chemistry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e9798b714816b66e1577ce3f8a8e63b5b7b470e" target='_blank'>
              Improving Molecular Dynamics Simulations for Solid-Liquid Interface with Machine Learning Interatomic Potentials.
              </a>
            </td>
          <td>
            Pengfei Hou, Yumiao Tian, Xing Meng
          </td>
          <td>2024-06-14</td>
          <td>Chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Biological neural networks can perform complex computations to predict their environment, far above the limited predictive capabilities of individual neurons. While conventional approaches to understanding these computations often focus on isolating the contributions of single neurons, here we argue that a deeper understanding requires considering emergent dynamics - dynamics that make the whole system"more than the sum of its parts". Specifically, we examine the relationship between prediction performance and emergence by leveraging recent quantitative metrics of emergence, derived from Partial Information Decomposition, and by modelling the prediction of environmental dynamics in a bio-inspired computational framework known as reservoir computing. Notably, we reveal a bidirectional coupling between prediction performance and emergence, which generalises across task environments and reservoir network topologies, and is recapitulated by three key results: 1) Optimising hyperparameters for performance enhances emergent dynamics, and vice versa; 2) Emergent dynamics represent a near sufficient criterion for prediction success in all task environments, and an almost necessary criterion in most environments; 3) Training reservoir computers on larger datasets results in stronger emergent dynamics, which contain task-relevant information crucial for performance. Overall, our study points to a pivotal role of emergence in facilitating environmental predictions in a bio-inspired computational architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2572feb06e7ea060ecb305043a5d4b589af5886d" target='_blank'>
              Evolving reservoir computers reveals bidirectional coupling between predictive power and emergent dynamics
              </a>
            </td>
          <td>
            Hanna M. Tolle, A. Luppi, Anil K. Seth, P. Mediano
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="

 In the rapidly evolving pharmaceutical industry, drug efficacy and safety stand as critical concerns. The vast accumulation of data, including customer feedback, drug popularity, and usage details, offers a rich resource for improving healthcare outcomes.



 The primary aim of this study is to harness machine learning and Natural Language Processing (NLP) techniques to sift through extensive pharmaceutical data, identifying the most effective drugs for various conditions and uncovering patterns that could guide better decision-making in drug efficacy and safety.



 This research seeks to construct a sophisticated model capable of analyzing diverse data points to pinpoint the most efficacious drugs for specific health conditions, thereby providing pharmaceutical companies with data-driven insights to optimize drug safety and effectiveness.



 Employing a blend of Natural Language Processing (NLP) and machine learning strategies, the study analyzes a comprehensive dataset featuring customer reviews, drug popularity metrics, usage information, and other relevant data collected over an extended period. This methodological approach aims to reveal latent trends and patterns that are crucial for assessing drug performance.



 The developed model adeptly identifies leading medications for various conditions, elucidating efficacy and safety profiles derived from patient reviews and drug utilization trends. These findings furnish pharmaceutical companies with actionable intelligence for enhancing drug development and patient care strategies.



 The integration of machine learning and NLP for the analysis of vast drug-related datasets presents a powerful method for advancing drug efficacy and safety. The insights yielded by the proposed model significantly empower the decision-making processes of the pharmaceutical industry, ultimately fostering improved health outcomes for patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa74cbe78ea6c68e73d842096a74cc272cfa464" target='_blank'>
              Leveraging Machine Learning and Patient Reviews for Developing a Drug Recommendation System to Reduce Medical Errors
              </a>
            </td>
          <td>
            K. P. Swain, S.K. Mohapatra, Vinayakumar Ravi, S. R. Nayak, Tahani Jaser Alahmadi, Prabhishek Singh, M. Diwakar
          </td>
          <td>2024-06-25</td>
          <td>The Open Bioinformatics Journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="“AI for science” is widely recognized as a future trend in the development of scientific research. Currently, although machine learning algorithms have played a crucial role in scientific research with numerous successful cases, relatively few instances exist where AI assists researchers in uncovering the underlying physical mechanisms behind a certain phenomenon and subsequently using that mechanism to improve machine learning algorithms' efficiency. This article uses the investigation into the relationship between extreme Poisson's ratio values and the structure of amorphous networks as a case study to illustrate how machine learning methods can assist in revealing underlying physical mechanisms. Upon recognizing that the Poisson's ratio relies on the low-frequency vibrational modes of the dynamical matrix, we can then employ a convolutional neural network, trained on the dynamical matrix instead of traditional image recognition, to predict the Poisson's ratio of amorphous networks with a much higher efficiency. Through this example, we aim to showcase the role that artificial intelligence can play in revealing fundamental physical mechanisms, which subsequently improves the machine learning algorithms significantly.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9eeafc65f4bc1ffaea8975a25fa66765acac9a2" target='_blank'>
              A cyclical route linking fundamental mechanism and AI algorithm: An example from tuning Poisson's ratio in amorphous networks
              </a>
            </td>
          <td>
            Changliang Zhu, Chenchao Fang, Zhipeng Jin, Baowen Li, Xiangying Shen, Lei Xu
          </td>
          <td>2024-07-02</td>
          <td>Applied Physics Reviews</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Biological sequences encode fundamental instructions for the building blocks of life, in the form of DNA, RNA, and proteins. Modeling these sequences is key to understand disease mechanisms and is an active research area in computational biology. Recently, Large Language Models have shown great promise in solving certain biological tasks but current approaches are limited to a single sequence modality (DNA, RNA, or protein). Key problems in genomics intrinsically involve multiple modalities, but it remains unclear how to adapt general-purpose sequence models to those cases. In this work we propose a multi-modal model that connects DNA, RNA, and proteins by leveraging information from different pre-trained modality-specific encoders. We demonstrate its capabilities by applying it to the largely unsolved problem of predicting how multiple RNA transcript isoforms originate from the same gene (i.e. same DNA sequence) and map to different transcription expression levels across various human tissues. We show that our model, dubbed IsoFormer, is able to accurately predict differential transcript expression, outperforming existing methods and leveraging the use of multiple modalities. Our framework also achieves efficient transfer knowledge from the encoders pre-training as well as in between modalities. We open-source our model, paving the way for new multi-modal gene expression approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c94cf63c86cbeefe668b6cb6b118506e8e144b7a" target='_blank'>
              Multi-modal Transfer Learning between Biological Foundation Models
              </a>
            </td>
          <td>
            Juan Jose Garau-Luis, Patrick Bordes, Liam Gonzalez, Masa Roller, Bernardo P. de Almeida, Lorenz Hexemer, Christopher Blum, Stefan Laurent, Jan Grzegorzewski, Maren Lang, Thomas Pierrot, Guillaume Richard
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Predicting future physical behavior through the limited theoretical simulation data available is an emerging research paradigm resulted by the integration of artificial intelligence technology and quantum physics. In this work, the charge-transport(CT) behavior was forecasted over a long time by a deep learning model, the long short-term memory (LSTM) network with error threshold training method in the photosynthesis II reaction center (PSII-RC). The theoretical simulation data within 8 fs was fed to the modified LSTM network for training, which brings out a distinct prediction with difference of $10^{-4}$ orders of magnitude over a long time period compared to the collection time for training sets. The results indicate the potential of employing LSTM to reveal the physics governing CT in addition to quantum physical methods. The implications of this work warrant further investigation to fully elucidate the scope and efficacy of LSTM for advancing our understanding of photosynthesis at the molecular scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b129f3020221fea7ebbd2fb39bca0b4125c3ef" target='_blank'>
              Charge-transport forecasted via deep learning in the photosystem II reaction center
              </a>
            </td>
          <td>
            Zi-Ran Zhao, Shun-Cai Zhao, Yi-Meng Huang
          </td>
          <td>2024-05-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Wildland fires pose terrifying natural hazards, underscoring the urgent need to develop data-driven and physics-informed digital twins for wildfire prevention, monitoring, intervention, and response. In this direction of research, this work introduces a physics-informed neural network (PiNN) to learn the unknown parameters of an interpretable wildfire spreading model. The considered wildfire spreading model integrates fundamental physical laws articulated by key model parameters, essential for capturing the complex behavior of wildfires. The proposed machine learning approach leverages the theory of artificial neural networks with the physical constraints governing wildfire dynamics, such as the first principles of mass and energy conservation. Training of the PiNN for physics-informed parameter identification is realized using data of the temporal evolution of one- and two-dimensional (plane surface) fire fronts that have been obtained from a high-fidelity simulator of the wildfire spreading model under consideration. The parameter learning results demonstrate the remarkable predictive ability of the proposed PiNN in uncovering the unknown coefficients in both the one- and two-dimensional fire spreading scenarios. Additionally, this methodology exhibits robustness by identifying the same parameters in the presence of noisy data. The proposed framework is envisioned to be incorporated in a physics-informed digital twin for intelligent wildfire management and risk assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62a95332da195a72205da26346da59e96afa55ce" target='_blank'>
              Physics-informed neural networks for parameter learning of wildfire spreading
              </a>
            </td>
          <td>
            K. Vogiatzoglou, C. Papadimitriou, V. Bontozoglou, Konstantinos Ampountolas
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The dynamics of biological systems, from proteins to cells to organisms, is complex and stochastic. To decipher their physical laws, we need to bridge between experimental observations and theoretical modeling. Thanks to progress in microscopy and tracking, there is today an abundance of experimental trajectories reflecting these dynamical laws. Inferring physical models from noisy and imperfect experimental data, however, is challenging. Because there are no inference methods that are robust and efficient, model reconstruction from experimental trajectories is a bottleneck to data-driven biophysics. In this Thesis, I present a set of tools developed to bridge this gap and permit robust and universal inference of stochastic dynamical models from experimental trajectories. These methods are rooted in an information-theoretical framework that quantifies how much can be inferred from trajectories that are short, partial and noisy. They permit the efficient inference of dynamical models for overdamped and underdamped Langevin systems, as well as the inference of entropy production rates. I finally present early applications of these techniques, as well as future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5a9b10d628b238c485c3e97f0ad456101dedb" target='_blank'>
              Learning dynamical models from stochastic trajectories
              </a>
            </td>
          <td>
            Pierre Ronceray
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Motivation Complex diseases are often caused and characterized by misregulation of multiple biological pathways. Differential network analysis aims to detect significant rewiring of biological network structures under different conditions and has become an important tool for understanding the molecular etiology of disease progression and therapeutic response. With few exceptions, most existing differential network analysis tools perform differential tests on separately learned network structures that are computationally expensive and prone to collapse when grouped samples are limited or less consistent. Results We previously developed an accurate differential network analysis method—differential dependency networks (DDN), that enables joint learning of common and rewired network structures under different conditions. We now introduce the DDN3.0 tool that improves this framework with three new and highly efficient algorithms, namely, unbiased model estimation with a weighted error measure applicable to imbalance sample groups, multiple acceleration strategies to improve learning efficiency, and data-driven determination of proper hyperparameters. The comparative experimental results obtained from both realistic simulations and case studies show that DDN3.0 can help biologists more accurately identify, in a study-specific and often unknown conserved regulatory circuitry, a network of significantly rewired molecular players potentially responsible for phenotypic transitions. Availability and implementation The Python package of DDN3.0 is freely available at https://github.com/cbil-vt/DDN3. A user’s guide and a vignette are provided at https://ddn-30.readthedocs.io/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f7f53646bc589a6aedd4807e452f6e2c56e13c4" target='_blank'>
              DDN3.0: determining significant rewiring of biological network structure with differential dependency networks
              </a>
            </td>
          <td>
            Yi Fu, Yingzhou Lu, Yizhi Wang, Bai Zhang, Zhen Zhang, Guoqiang Yu, Chunyu Liu, Robert Clarke, David M. Herrington, Yue Wang
          </td>
          <td>2024-06-01</td>
          <td>Bioinformatics</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="Chronic Kidney Disease (CKD) is a global health issue impacting over 800 million people, characterized by a gradual loss of kidney function leading to severe complications. Traditional diagnostic methods, relying on laboratory tests and clinical assessments, have limitations in sensitivity and are prone to human error, particularly in the early stages of CKD. Recent advances in machine learning (ML) offer promising tools for disease diagnosis, but a lack of interpretability often hinders their adoption in clinical practice. Gaussian Processes (GP) provide a flexible ML model capable of delivering predictions and uncertainty estimates, essential for high-stakes medical applications. However, the integration of GP with interpretable methods remains underexplored. We developed an interpretable CKD classification model to address this knowledge gap by combining GP with Shapley Additive Explanations (SHAP). We assessed the model's performance using three GP kernels (Radial Basis Function, Matern, and Rational Quadratic). The results show that the Rational Quadratic kernel outperforms the other kernels, achieving an accuracy of 98.75%, precision of 100%, sensitivity of 97.87%, specificity of 100%, and an F1-score of 98.51%. SHAP values indicate that haemoglobin and specific gravity are the most influential features. The results demonstrate that the Rational Quadratic kernel enhances predictive accuracy and provides robust uncertainty estimates and interpretable explanations. This combination of accuracy and interpretability supports clinicians in making informed decisions and improving patient management and outcomes in CKD. Our study connects advanced ML techniques with practical medical applications, leading to more effective and reliable ML-driven healthcare solutions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adf6d004ccf334eefb041c57041421906b537c8e" target='_blank'>
              Interpretable Machine Learning for Chronic Kidney Disease Diagnosis: A Gaussian Processes Approach
              </a>
            </td>
          <td>
            T. R. Noviandy, Ghifari Maulana Idroes, M. Syukri, R. Idroes
          </td>
          <td>2024-06-29</td>
          <td>Indonesian Journal of Case Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Tumor heterogeneity is a primary factor that contributes to treatment failure. Predictive tools, capable of classifying cancer cells based on their functions, may substantially enhance therapy and extend patient life span. The connection between cell biomechanics and cancer cell functions is used here to classify cells through mechanical measurements, via particle uptake. Machine learning (ML) was used to classify cells based on single-cell patterns of uptake of particles with diverse sizes. Three pairs of human cancer cell subpopulations, varied in their level of drug resistance or malignancy, were studied. Cells were allowed to interact with fluorescently labeled polystyrene particles ranging in size from 0.04 to 3.36 μm and analyzed for their uptake patterns using flow cytometry. ML algorithms accurately classified cancer cell subtypes with accuracy rates exceeding 95%. The uptake data were especially advantageous for morphologically similar cell subpopulations. Moreover, the uptake data were found to serve as a form of "normalization" that could reduce variation in repeated experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f66cc5838fae9c10d30fe2b5543c3e9eb1e0ab75" target='_blank'>
              Particle uptake in cancer cells can predict malignancy and drug resistance using machine learning.
              </a>
            </td>
          <td>
            Yoel Goldstein, Ora T. Cohen, Ori Wald, Danny Bavli, Tommy Kaplan, Ofra Benny
          </td>
          <td>2024-05-31</td>
          <td>Science advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The estimation of directed couplings between the nodes of a network from indirect measurements is a central methodological challenge in scientific fields such as neuroscience, systems biology and economics. Unfortunately, the problem is generally ill-posed due to the possible presence of unknown delays in the measurements. In this paper, we offer a solution of this problem by using a variational Bayes framework, where the uncertainty over the delays is marginalized in order to obtain conservative coupling estimates. To overcome the well-known overconfidence of classical variational methods, we use a hybrid-VI scheme where the (possibly flat or multimodal) posterior over the measurement parameters is estimated using a forward KL loss while the (nearly convex) conditional posterior over the couplings is estimated using the highly scalable gradient-based VI. In our ground-truth experiments, we show that the network provides reliable and conservative estimates of the couplings, greatly outperforming similar methods such as regression DCM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a02bb50bd2145ddb88212879e50602c28462b00e" target='_blank'>
              Robust and highly scalable estimation of directional couplings from time-shifted signals
              </a>
            </td>
          <td>
            Luca Ambrogioni, Louis Rouillard, Demian Wassermann
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Large-scale single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) have transformed biomedical research into a data-driven field, enabling the creation of comprehensive data atlases. These methodologies facilitate detailed understanding of biology and pathophysiology, aiding in the discovery of new therapeutic targets. However, the complexity and sheer volume of data from these technologies present analytical challenges, particularly in robust cell typing, integration and understanding complex spatial relationships of cells. To address these challenges, we developed CELLama (Cell Embedding Leverage Language Model Abilities), a framework that leverage language model to transform cell data into ’sentences’ that encapsulate gene expressions and metadata, enabling universal cellular data embedding for various analysis. CELLama, serving as a foundation model, supports flexible applications ranging from cell typing to the analysis of spatial contexts, independently of manual reference data selection or intricate dataset-specific analytical workflows. Our results demonstrate that CELLama has significant potential to transform cellular analysis in various contexts, from determining cell types across multi-tissue atlases and their interactions to unraveling intricate tissue dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c4d45afa0f09312ed61002da36b33744fbf684" target='_blank'>
              CELLama: Foundation Model for Single Cell and Spatial Transcriptomics by Cell Embedding Leveraging Language Model Abilities
              </a>
            </td>
          <td>
            Hongyoon Choi, Jeongbin Park, Sumin Kim, Jiwon Kim, Dongjoo Lee, Sungwoo Bae, Haenara Shin, D. Lee
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d03a9ff2ae1d1c085223172f82f70126105827e" target='_blank'>
              Discrete latent embedding of single-cell chromatin accessibility sequencing data for uncovering cell heterogeneity
              </a>
            </td>
          <td>
            Xuejian Cui, Xiaoyang Chen, Zhen Li, Zijing Gao, Shengquan Chen, Rui Jiang
          </td>
          <td>2024-05-10</td>
          <td>Nature computational science</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d28f61744b20fc249683d660aaa11e7efc57768" target='_blank'>
              Explainable chemical artificial intelligence from accurate machine learning of real-space chemical descriptors.
              </a>
            </td>
          <td>
            Miguel Gallegos, Valentin Vassilev-Galindo, I. Poltavsky, Á. Martín Pendás, Alexandre Tkatchenko
          </td>
          <td>2024-05-21</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="We present an application of Physics-Informed Neural Networks to handle MultiPhase-Field simulations of microstructure evolution. It has been showcased that a combination of optimization techniques extended and adapted from the PINNs literature, and the introduction of specific techniques inspired by the MPF Method background, is required. The numerical resolution is realized through a multi-variable time-series problem by using fully discrete resolution. Within each interval, space, time, and phases are treated separately, constituting discrete subdomains. An extended multi-networking concept is implemented to subdivide the simulation domain into multiple batches, with each batch associated with an independent Neural Network trained to predict the solution. To ensure efficient interaction across different phasesand in the spatio-temporal-phasic subdomain, a Master NN handles efficient interaction among the multiple networks, as well as the transfer of learning in different directions. A set of systematic simulations with increasing complexity was performed, that benchmarks various critical aspects of MPF simulations, including different geometries, types of interface dynamics and the evolution of an interfacial triple junction. A comprehensive approach is adopted to specifically focus the attention on the interfacial regions through an automatic and dynamic meshing process, significantly simplifying the tuning of hyper-parameters and serving as a fundamental key for addressing MPF problems using Machine Learning. The pyramidal training approach is proposed to the PINN community as a dual-impact method: it facilitates the initialization of training and allows an extended transfer of learning. The proposed PINNs-MPF framework successfully reproduces benchmark tests with high fidelity and Mean Squared Error loss values ranging from 10$^{-4}$ to 10$^{-6}$ compared to ground truth solutions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f31d909eac4f45aba2c5d7eda80b9159e9f0772f" target='_blank'>
              PINNs-MPF: A Physics-Informed Neural Network Framework for Multi-Phase-Field Simulation of Interface Dynamics
              </a>
            </td>
          <td>
            Seifallah Elfetni, R. D. Kamachali
          </td>
          <td>2024-07-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In this study, we develop and implement the Extended Kalman Filter (EKF) to forecast the rate of change in tumor cells, healthy host cells, and effector immune cells within the Itik-Banks model. This novel application of EKF in cancer dynamics modeling aims to provide precise real-time estimations of cellular interactions, especially in constructing a new state space representation from the Itik-Banks model. We use a first-order Taylor series to linearize the model. The numerical simulations were performed to analyze the accuracy of this new state space with data from William Gilpin’s GitHub repository. The results show that the EKF predictions strongly align with actual data, i.e., in the prior and posterior steps for tumor and healthy host cells, there is a strong agreement between the predictions and the actual data. The EKF captures the oscillatory nature of the tumor and healthy host cell population well. The peaks and troughs of the predictions align closely with the actual data, indicating the EKF’s effectiveness in modeling the dynamic behavior of the tumor and healthy host cells. However, for effector immune cells, the oscillatory nature of the data in these cells gives rise to slight deviations. This represents a significant challenge in the future for updating the state space representations. Despite minor discrepancies, the EKF demonstrates a strong performance in both the training and testing data, with the posterior step estimates significantly improving the prior step accuracy. This study emphasizes the importance of data availability for accurate predictions, noting a symmetric Mean Absolute Percentage Error (sMAPE) of 35.92% when data is unavailable. Prompt correction with new data is essential to maintain accuracy. This research underscores the EKF’s potential for real-time monitoring and prediction in complex biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c47c21057eed2151b03ae81e7e3e1609e8f172d" target='_blank'>
              Prediction of the Change Rate of Tumor Cells, Healthy Host Cells, and Effector Immune Cells in a Three-Dimensional Cancer Model using Extended Kalman Filter
              </a>
            </td>
          <td>
            Nina Fitriyati, Salma Abidah Faizah, Taufik Edy Sutanto
          </td>
          <td>2024-06-27</td>
          <td>Jambura Journal of Biomathematics (JJBM)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Real-world networks are sparse. As we show in this article, even when a large number of interactions is observed most node pairs remain disconnected. We demonstrate that classical multi-edge network models, such as the $G(N,p)$, configuration models, and stochastic block models, fail to accurately capture this phenomenon. To mitigate this issue, zero-inflation must be integrated into these traditional models. Through zero-inflation, we incorporate a mechanism that accounts for the excess number of zeroes (disconnected pairs) observed in empirical data. By performing an analysis on all the datasets from the Sociopatterns repository, we illustrate how zero-inflated models more accurately reflect the sparsity and heavy-tailed edge count distributions observed in empirical data. Our findings underscore that failing to account for these ubiquitous properties in real-world networks inadvertently leads to biased models which do not accurately represent complex systems and their dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f81d421fba2326c997d828fe267b1ec0356ef368" target='_blank'>
              Empirical Networks are Sparse: Enhancing Multi-Edge Models with Zero-Inflation
              </a>
            </td>
          <td>
            Giona Casiraghi, George Andres
          </td>
          <td>2024-06-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology provides a means for studying biology from a cellular perspective. The fundamental goal of scRNA-seq data analysis is to discriminate single-cell types using unsupervised clustering. Few single-cell clustering algorithms have taken into account both deep and surface information, despite the recent slew of suggestions. Consequently, this article constructs a fusion learning framework based on deep learning, namely scGASI. For learning a clustering similarity matrix, scGASI integrates data affinity recovery and deep feature embedding in a unified scheme based on various top feature sets. Next, scGASI learns the low-dimensional latent representation underlying the data using a graph autoencoder to mine the hidden information residing in the data. To efficiently merge the surface information from raw area and the deeper potential information from underlying area, we then construct a fusion learning model based on self-expression. scGASI uses this fusion learning model to learn the similarity matrix of an individual feature set as well as the clustering similarity matrix of all feature sets. Lastly, gene marker identification, visualization, and clustering are accomplished using the clustering similarity matrix. Extensive verification on actual data sets demonstrates that scGASI outperforms many widely used clustering techniques in terms of clustering accuracy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e5026ad18d2a22bd7b53216320a6ffec3330ef" target='_blank'>
              A Fusion Learning Model Based on Deep Learning for Single-Cell RNA Sequencing Data Clustering.
              </a>
            </td>
          <td>
            Tian-Jing Qiao, Feng Li, Sha-Sha Yuan, Ling-Yun Dai, Juan Wang
          </td>
          <td>2024-05-20</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="We describe version 2 of the SPICE dataset, a collection of quantum chemistry calculations for training machine learning potentials. It expands on the original dataset by adding much more sampling of chemical space and more data on non-covalent interactions. We train a set of potential energy functions called Nutmeg on it. They use a novel mechanism to improve performance on charged and polar molecules, injecting precomputed partial charges into the model to provide a reference for the large scale charge distribution. Evaluation of the new models shows they do an excellent job of reproducing energy differences between conformations, even on highly charged molecules or ones that are significantly larger than the molecules in the training set. They also produce stable molecular dynamics trajectories, and are fast enough to be useful for routine simulation of small molecules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/173c7c4074c9adca31fa419f912637ed59b884a7" target='_blank'>
              Nutmeg and SPICE: Models and Data for Biomolecular Machine Learning
              </a>
            </td>
          <td>
            Peter K. Eastman, Benjamin P. Pritchard, J. Chodera, T. Markland
          </td>
          <td>2024-06-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Deterministic Neural Networks (NNs) have been widely adopted to represent system behaviors in various engineering problems, which have been criticized for providing overconfident predictions. This challenge, i.e., overfitting, refers to a model that captures data patterns and random noise during the learning process, leading to an inferior predictive capability in the unobserved domains. As known to all, Bayesian inference offers an effective way for uncertainty quantification (UQ) to help enhance model predictions and offer informed decision-making. Integrating deterministic NN models with Bayesian inference provides a way to fast simulate and quantify both data and model uncertainty simultaneously. Thereby, Bayesian Neural Networks (BNNs), a combination of standard NNs with Bayesian philosophy, introduce probability to the network parameters, i.e., biases and weights. In BNNs, predictive uncertainty reflects how confident model predictions are regarding the corresponding credible intervals, which helps identify overfitting and interpreting abnormal predictions. Another advantage is that BNNs naturally implement regularization by integrating over the weights such that BNNs can reach good generalization when data availability is limited. BNNs rely heavily on observations as a purely data-driven approach. Besides, standard BNNs have relatively complex network structures, which makes it hard to provide detailed interpretability on model predictions. Researchers share concerns about their predictions that may violate real-world phenomena. The combination of physics knowledge with the optimization objective can improve the adherence of predictions with the known physical principles. Physics laws are usually embedded as constraint functions in the process of optimization. With the help of constrained optimization, model predictions keep consistent with the known physical principles.
 The authors refine BNNs with physics-guided information, i.e., Physics-Guided BNNs (PG-BNNs), to reduce dependence on data, improve model interpretability, and offer reliable model predictions. PG-BNNs interpret the known physics knowledge as constraint functions to better guide optimization processes, especially for case studies with sparse and limited data sets. A case study of an ordinary differential equation (ODE) problem is carried out to compare the performance of PG-BNN and unconstrained BNN models and demonstrate the effectiveness of PG-BNN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd95612093b011b8ce714901de80f0888b466c28" target='_blank'>
              Physics-Guided Bayesian Neural Networks and Their Application in ODE Problems
              </a>
            </td>
          <td>
            Xinyue Xu, Suman Paneru, Sez A. Russcher, Julian Wang
          </td>
          <td>2024-05-15</td>
          <td>ASME 2024 Verification, Validation, and Uncertainty Quantification Symposium</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spaceflight studies present novel insights into biological processes through exposure to stressors outside the evolutionary path of terrestrial organisms. Despite limited access to space environments, numerous transcriptomic datasets from spaceflight experiments are now available through NASA’s GeneLab data repository, which allows public access to these datasets, encouraging further analysis. While various computational pipelines and methods have been used to process these transcriptomic datasets, learning-model-driven analyses have yet to be applied to a broad array of such spaceflight-related datasets. In this study, we propose an open-source framework, GLARE: GeneLAb Representation learning pipelinE, which consists of training different representation learning approaches from manifold learning to self-supervised learning that enhances the performance of downstream analytical tasks such as pattern recognition. We illustrate the utility of GLARE by applying it to gene-level transcriptional values from the results of the CARA spaceflight experiment, an Arabidopsis root tip transcriptome dataset that spanned light, dark, and microgravity treatments. We show that GLARE not only substantiated the findings of the original study concerning cell wall remodeling but also revealed additional patterns of gene expression affected by the treatments, including evidence of hypoxia. This work suggests there is great potential to supplement the insights drawn from initial studies on spaceflight omics-level data through further machine-learning-enabled analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c46ff5b99e48dd2b0f16b61252b7124d1818f6" target='_blank'>
              GLARE: Discovering hidden patterns in spaceflight transcriptome using representation learning
              </a>
            </td>
          <td>
            DongHyeon Seo, Hunter F Strickland, Mingqi Zhou, R. Barker, R. Ferl, A. Paul, Simon Gilroy
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The disease known as breast cancer continues to be one of the most common and potentially fatal diseases that affect women all over the world. In order to provide effective therapy and more favorable outcomes for patients, early detection and correct diagnosis are absolutely necessary. This research explores the integration of operational research (OR) methodologies and advanced statistical methods within machine learning frameworks to enhance the prediction accuracy of breast cancer outcomes. By harnessing the power of nonlinear optimization and statistical techniques, including regression analysis and probability distribution models, we aim to refine the predictive capabilities of existing algorithms. The research employs a comprehensive dataset derived from clinical trials and patient records, analyzed through a series of machine learning models that incorporate elements of combinatorial optimization, decision analysis, and stochastic modeling. Key performance metrics, such as accuracy, sensitivity, and specificity, are evaluated against standard benchmarks to determine the efficacy of the integrated approaches. Preliminary results indicate that incorporating OR and statistical methods significantly improves model robustness and predictive accuracy. The study not only demonstrates the potential of applied mathematics in medical diagnostics but also provides a framework for future research in enhancing machine learning models for health outcomes prediction through mathematical innovations. This investigation contributes to the field of mathematical oncology by demonstrating how applied nonlinear analysis can bridge the gap between theoretical mathematical approaches and practical clinical applications, offering new pathways for early and more accurate detection of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91e2ad7ffea30ec7bd6b8cdf76f263552d6be77" target='_blank'>
              Integration of Complex Mathematical Approaches and Statistical Methodologies in Machine Learning for Design of Robust Prediction Models
              </a>
            </td>
          <td>
            Sarita Silaich, Dr Rajesh yadav
          </td>
          <td>2024-05-17</td>
          <td>Communications on Applied Nonlinear Analysis</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Summary The retarded transient function (RTF) approach has been introduced as a complementary approach to employing ordinary differential equations (ODEs) for modelling dynamics typically observed for cellular signalling processes. Here, we introduce the R package to the RTF approach, which has originally been implemented within the MATLAB-based Data2Dynamics modelling framework. The package enables modelling of time and dose dependencies and includes model reduction to minimize overfitting. It can be applied not only to experimental data but also to trajectories of ODE models, to characterize the dynamics or to identify major targets of experimental perturbations in the low-dimensional representations calculated by the package. Availability and Implementation The R package RTF is available at https://github.com/kreutz-lab/RTF. Contact clemens.kreutz@uniklinik-freiburg.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0620e9d50945b256e9edbd53674e984230196e5b" target='_blank'>
              RTF: An R package for modelling time course data
              </a>
            </td>
          <td>
            Eva Brombacher, Clemens Kreutz
          </td>
          <td>2024-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) is a major global health concern. Detecting BC at an early stage gives more treatment options and can help avoid more aggressive treatments. The use of machine learning (ML) in BC prediction offers significant potential for improving the accuracy and speed of diagnosis, personalizing treatment, and identifying high-risk patients.  However, there are significant challenges associated with the use of ML, including the need for high-quality data and more flexible models with optimal parameters to achieve high efficiency. In this paper, we propose an optimized framework based on multi-stage data exploration. This framework is designed to provide a comprehensive approach to data exploration, ensuring that the data is well-prepared for ML. In addition, the framework includes dynamic ensemble-based classifiers, which combine multiple independent classifiers to improve accuracy and mitigate the risk of overfitting in conjunction with the cross-validation techniques. These classifiers are optimized using Bayesian hyperparameter tuning, which involves selecting the optimal values for the various hyperparameters of the model. This approach can significantly improve the prediction accuracy of the resulting model. The study evaluates the framework using the publicly available Wisconsin Diagnostic Breast Cancer (WDBC) dataset and compares our results with other state-of-the-art models. The experimental results show that the best result is 100% for accuracy and recall with hyperparameters of (Ensemble Method = AdaBoost, Number of learners = 322, learning rate = 0.9350, and the Maximum number of splits = 1). The highest performance has been achieved with the proposed framework compared with the other models in terms of accuracy (mean = 99.35%, best = 100%, worst = 98.7%, and Standard Deviation = 0.325). The framework can potentially improve the accuracy and efficiency of BC prediction, ultimately leading to better outcomes for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6713f5a51450ea1b2ff5aad362a36d27ed56ef0" target='_blank'>
              An Optimized Framework Based on Data Exploration and Dynamic Ensemble-Based Models for Breast Cancer Prediction
              </a>
            </td>
          <td>
            Ayman Alsabry, Hamzah Ali Abdulrahman Qasem, Malek Algabri, Amin Mohamed Ahsan, Mogeeb A. A. Mosleh, F. E. Hanash
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Computing</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Drug-side effect prediction has become an essential area of research in the field of pharmacology. As the use of medications continues to rise, so does the importance of understanding and mitigating the potential risks associated with them. At present, researchers have turned to data-driven methods to predict drug-side effects. Drug-side effect prediction is a link prediction problem, and the related data can be described from various perspectives. To process these kinds of data, a multi-view method, called Multiple Kronecker RLS fusion-based link propagation (MKronRLSF-LP), is proposed. MKronRLSF-LP extends the Kron-RLS by finding the consensus partitions and multiple graph Laplacian constraints in the multi-view setting. Both of these multi-view settings contribute to a higher quality result. Extensive experiments have been conducted on drug-side effect datasets, and our empirical results provide evidence that our approach is effective and robust.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63527bfc5bdbb286aecbc16d31c2e4de057109d" target='_blank'>
              Multiple Kronecker RLS fusion-based link propagation for drug-side effect prediction
              </a>
            </td>
          <td>
            Yuqing Qian, Ziyu Zheng, Prayag Tiwari, Yijie Ding, Quan Zou
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="A growing number of mathematical models have been developed, providing a powerful means to forecast tumor growth. These models hold great promise as tools in the advancement of personalized medical care. Yet, their clinical relevance often remains restricted. Adequate parameter estimation and integration of existing clinical data from the patient is crucial for individualized predictions. Since CT scans form an integral part of radiotherapy, a widely used cancer treatment, we propose the use of CT-derived radiomic features to attempt to predict the course of tumor growth curves. Therefore, in this exploratory study, we examine the association between radiomic features and the three parameters of the Gompertz tumor growth model. As a proof of concept, murine tumor growth (CT26) is monitored and the Gompertz model is fitted to each animal's data to obtain the parameters. Radiomic features of the tumors were then obtained via CT, which were subsequently analyzed using exploratory factor analysis. Finally, correlations between the two were computed. We were able to discover latent factors within the radiomic features that revealed significant correlations with the parameters of the Gompertz model. Additionally, distinct features exhibiting strong correlations with the factors were identified. These may be instrumental in predicting future tumor growth. These findings offer encouraging preliminary evidence for supporting the notion that CT scans contain information that can estimate the future course of tumor growth curves. This approach opens up promising possibilities for personalized medicine in oncology, by facilitating the development of treatment plans tailored to individual patients' tumor growth patterns, right from the time of diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c33415d539a913b222861f5873ec836c14602c7" target='_blank'>
              Exploring the Potential of Radiomic Features in Predictive Murine Tumor Growth Modeling
              </a>
            </td>
          <td>
            Assia Hamitou, A. Rifi, I. Dufait, Camille Raets, M. De Ridder, Kurt Barbé
          </td>
          <td>2024-05-20</td>
          <td>2024 IEEE International Instrumentation and Measurement Technology Conference (I2MTC)</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Predicting drug toxicity is a critical aspect of ensuring patient safety during the drug design process. Although conventional machine learning techniques have shown some success in this field, the scarcity of annotated toxicity data poses a significant challenge in enhancing models’ performance. In this study, we explore the potential of leveraging large unlabeled datasets using semi-supervised learning to improve predictive performance for cardiotoxicity across three targets: the voltage-gated potassium channel (hERG), the voltage-gated calcium channel (Cav1.2), and the voltage-gated sodium channel (Nav1.5). We extensively mined the ChEMBL database, comprising approximately 2 million small molecules, then employed semi-supervised learning to construct robust classification models for this purpose. We achieved a performance boost on highly diverse (i.e. structurally dissimilar) test datasets across all three targets. Using our built models, we screened the whole ChEMBL database and a large set of FDA-approved drugs, identifying several compounds with potential cardiac channel activity. To ensure broad accessibility and usability for both technical and non-technical users, we developed a cross-platform graphical user interface that allows users to make predictions and gain insights into the cardiotoxicity of drugs and other small molecules. The software is made available as open source under the permissive MIT license at https://github.com/issararab/CToxPred2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f0e09b146e54215719071ab7b9f648e458c75b" target='_blank'>
              Semi-Supervised Learning to Boost Cardiotoxicity Prediction by Mining a Large Unlabeled Small Molecule Dataset
              </a>
            </td>
          <td>
            Issar Arab, Kris Laukens, Wout Bittremieux
          </td>
          <td>2024-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The joint analysis of two datasets X and Y that describe the same phenomena (e.g. the cellular state), but measure disjoint sets of variables (e.g. mRNA vs. protein levels) is currently challenging. Traditional methods typically analyze single interaction patterns such as variance or covariance. However, problem-tailored external knowledge may contain multiple different information about the interaction between the measured variables. We introduce MIASA, a holistic framework for the joint analysis of multiple different variables. It consists of assembling multiple different information such as similarity vs. association, expressed in terms of interaction-scores or distances, for subsequent clustering/classification. In addition, our framework includes a novel qualitative Euclidean embedding method (qEE-Transition) which enables using Euclidean-distance/vector-based clustering/classification methods on datasets that have a non-Euclidean-based interaction structure. As an alternative to conventional optimization-based multidimensional scaling methods which are prone to uncertainties, our qEE-Transition generates a new vector representation for each element of the dataset union X∪Y in a common Euclidean space while strictly preserving the original ordering of the assembled interaction-distances. To demonstrate our work, we applied the framework to three types of simulated datasets: samples from families of distributions, samples from correlated random variables, and time-courses of statistical moments for three different types of stochastic two-gene interaction models. We then compared different clustering methods with vs. without the qEE-Transition. For all examples, we found that the qEE-Transition followed by Ward clustering had superior performance compared to non-agglomerative clustering methods but had a varied performance against ultrametric-based agglomerative methods. We also tested the qEE-Transition followed by supervised and unsupervised machine learning methods and found promising results, however, more work is needed for optimal parametrization of these methods. As a future perspective, our framework points to the importance of more developments and validation of distance-distribution models aiming to capture multiple-complex interactions between different variables.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36132a43a8480f80c7e771e1d24ff804f82ae2b" target='_blank'>
              Multi-Input data ASsembly for joint Analysis (MIASA): A framework for the joint analysis of disjoint sets of variables
              </a>
            </td>
          <td>
            N. A. Raharinirina, V. Sunkara, Max von Kleist, K. Fackeldey, M. Weber
          </td>
          <td>2024-05-10</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Single-cell multi-omics technologies have empowered increasingly refined characterisation of the heterogeneity of cell populations. Automated cell type annotation methods have been developed to transfer cell type labels from well-annotated reference datasets to emerging query datasets. However, these methods suffer from some common caveats, including the failure to characterise transitional and novel cell states, sensitivity to batch effects and under-utilisation of phenotypic information other than cell types (e.g. sample source and disease conditions). We developed Φ-Space, a computational framework for the continuous phenotyping of single-cell multi-omics data. In Φ-Space we adopt a highly versatile modelling strategy to continuously characterise query cell identity in a low-dimensional phenotype space, defined by reference phenotypes. The phenotype space embedding enables various downstream analyses, including insightful visualisations, clustering and cell type labelling. We demonstrate through three case studies that Φ-Space (i) characterises developing and out-of-reference cell states; (ii) is robust against batch effects in both reference and query; (iii) adapts to annotation tasks involving multiple omics types; (iv) overcomes technical differences between reference and query. The versatility of Φ-Space makes it applicable to a wide range analytical tasks beyond cell type transfer, and its ability to model complex phenotypic variation will facilitate biological discoveries from different omics types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f8f58c02f14e009289058adb00560711a02d08b" target='_blank'>
              Φ-Space: Continuous phenotyping of single-cell multi-omics data
              </a>
            </td>
          <td>
            Jiadong Mao, Yidi Deng, K. Lê Cao
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="In chemical engineering, process data is often expensive to acquire, and complex phenomena are difficult to model rigorously, rendering both entirely data-driven and purely mechanistic modeling approaches impractical. We explore using physics-informed neural networks (PINNs) for modeling dynamic processes governed by differential-algebraic equation systems when process data is scarce and complete mechanistic knowledge is missing. In particular, we focus on estimating states for which neither direct observational data nor constitutive equations are available. For demonstration purposes, we study a continuously stirred tank reactor and a liquid-liquid separator. We find that PINNs can infer unmeasured states with reasonable accuracy, and they generalize better in low-data scenarios than purely data-driven models. We thus show that PINNs, similar to hybrid mechanistic/data-driven models, are capable of modeling processes when relatively few experimental data and only partially known mechanistic descriptions are available, and conclude that they constitute a promising avenue that warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4e0161b8d6befa3c8a0687b96bbfc9b32e40f7" target='_blank'>
              Physics-Informed Neural Networks for Dynamic Process Operations with Limited Physical Knowledge and Data
              </a>
            </td>
          <td>
            M. Velioglu, Song Zhai, Sophia Rupprecht, Alexander Mitsos, Andreas Jupke, M. Dahmen
          </td>
          <td>2024-06-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Bayesian graphical models are powerful tools to infer complex relationships in high dimension, yet are often fraught with computational and statistical challenges. If exploited in a principled way, the increasing information collected alongside the data of primary interest constitutes an opportunity to mitigate these difficulties by guiding the detection of dependence structures. For instance, gene network inference may be informed by the use of publicly available summary statistics on the regulation of genes by genetic variants. Here we present a novel Gaussian graphical modeling framework to identify and leverage information on the centrality of nodes in conditional independence graphs. Specifically, we consider a fully joint hierarchical model to simultaneously infer (i) sparse precision matrices and (ii) the relevance of node-level information for uncovering the sought-after network structure. We encode such information as candidate auxiliary variables using a spike-and-slab submodel on the propensity of nodes to be hubs, which allows hypothesis-free selection and interpretation of a sparse subset of relevant variables. As efficient exploration of large posterior spaces is needed for real-world applications, we develop a variational expectation conditional maximization algorithm that scales inference to hundreds of samples, nodes and auxiliary variables. We illustrate and exploit the advantages of our approach in simulations and in a gene network study which identifies hub genes involved in biological pathways relevant to immune-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfbfe802cb5f3c9c2e7e580929c13fe21fe00c25" target='_blank'>
              A modeling framework for detecting and leveraging node-level information in Bayesian network inference.
              </a>
            </td>
          <td>
            Xiaoyue Xi, H. Ruffieux
          </td>
          <td>2024-06-25</td>
          <td>Biostatistics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Pharmacogenomics, the study of how genetic variations influence drug response, has emerged as a cornerstone of personalized medicine, revolutionizing clinical practice across various medical specialties. This comprehensive review explores the applications, challenges, and prospects of pharmacogenomics in healthcare, with a particular focus on oncology. The integration of pharmacogenomic data into clinical decision-making processes enables healthcare providers to tailor drug therapy to individual patients based on their genetic makeup, thereby optimizing treatment outcomes while minimizing adverse effects. In oncology, pharmacogenomic testing plays a pivotal role in individualizing cancer therapy, predicting chemotherapy response, and selecting targeted therapies based on patients' genetic profiles.
However, the widespread implementation of pharmacogenomics faces several challenges, including the need for robust evidence supporting its clinical utility, standardization of testing methodologies, and integration of genetic data into electronic health records. Despite these challenges, ongoing research efforts continue to advance our understanding of the genetic determinants of drug response, paving the way for personalized medicine to become an integral part of routine clinical practice. By addressing these challenges and leveraging technological advancements, pharmacogenomics holds the promise of enhancing patient care, improving treatment outcomes, and ultimately transforming the delivery of healthcare.

Key words: Pharmacogenomics, personalized medicine, drug response, oncology, genetic variations, clinical practice">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2831472a14960f1801563eb3b3c25dc1ab5bad5e" target='_blank'>
              Pharmacogenomics in Healthcare: Applications, Challenges, and Future Directions with a Focus on Oncology
              </a>
            </td>
          <td>
            Karra Geetha, T. Chandana, R. Sakshi, C. S. Chandu, T. Ramarao
          </td>
          <td>2024-06-08</td>
          <td>International Journal of Health Sciences and Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="e19029 Background: Chronic lymphocytic leukemia (CLL) exhibits a broad spectrum of clinical behaviors, from indolent courses requiring minimal intervention to aggressive forms demanding immediate treatment. The underlying heterogeneity of CLL poses significant challenges in predicting disease progression and tailoring patient-specific therapeutic strategies. Leveraging advanced and novel artificial intelligence (AI) and machine learning (ML) methodologies, our study aims to dissect the complexity of CLL, focusing on the identification and characterization of biomarkers that differentiate aggressive and indolent disease forms. Methods: The NetraAI is a priorietaryAI/ML system that is also being utilized in a research collaboration at a leading American oncology institute (not disclosed) that integrates machine intelligence, augmented intelligence, and an agent that interacts with AI-derived patient representations. This system was designed to learn from datasets with a small number of samples and many independent variables. This system operates by generating hypotheses in the form of interactive representations of patient populations that reveal heterogeneity along with statistically significant driving factors. Finally, an automated agent called AutoPlay interacts with these representations and generates an accounting of all variables, subpopulations of samples, and it also generates a human readable perspective on what was discovered by the data set. Results: We applied the NetraAI to a CLL dataset and used it to refine the analysis and interpretation of the corresponding transcriptomic data (GSE39411). Significantly lower levels of FADS3, GSDME, LPL, IMMT, NMB, and AEBP1 and higher expressions of COBLL1, P2RY1R, PDE8A, SYNE2, and FCRL3 were identified as hallmarks of indolent CLL, suggesting these markers could inform less aggressive disease management strategies. Furthermore, within a total of 104 CLL samples, we uncovered two distinct subpopulations within aggressive CLL, delineated by unique genetic markers. Group A, encompassing 31 aggressive CLL samples is predominantly characterized by the expression of lipoprotein lipase (LPL), while Group B, comprising 22/23 aggressive samples, is characterized by ZBTB20 and SYNE2. Our poster will also provide glimpses into a more refined heterogeneity that emerged from the analysis. Our results not only underscore the heterogeneity within aggressive CLL, but also highlights the potential for these markers to guide prognostic assessments and therapeutic decisions. Conclusions: This study not only advances our understanding of CLL heterogeneity, but also sets the stage for the development of more personalized and effective treatment modalities. By identifying drivers of disease aggression and indolence, we pave the way for targeted therapies that could significantly improve patient outcomes in CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9669c518d434fca01e039d62dafe67ad664fe0a8" target='_blank'>
              Revealing heterogeneity in chronic lymphocytic leukemia: AI-driven insights into aggressive and indolent disease subtypes.
              </a>
            </td>
          <td>
            Bessi Qorri, Joseph Geraci, Mike J. Tsay, Christian Cumbaa, Larry Alphs, Luca Pani
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Interpretable mathematical expressions defining discrete-time dynamical systems (iterated maps) can model many phenomena of scientific interest, enabling a deeper understanding of system behaviors. Since formulating governing expressions from first principles can be difficult, it is of particular interest to identify expressions for iterated maps given only their data streams. In this work, we consider a modified Symbolic Artificial Neural Network-Trained Expressions (SymANNTEx) architecture for this task, an architecture more expressive than others in the literature. We make a modification to the model pipeline to optimize the regression, then characterize the behavior of the adjusted model in identifying several classical chaotic maps. With the goal of parsimony, sparsity-inducing weight regularization and information theory-informed simplification are implemented. We show that our modified SymANNTEx model properly identifies single-state maps and achieves moderate success in approximating a dual-state attractor. These performances offer significant promise for data-driven scientific discovery and interpretation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3502a1e8d5f9351f334ca3982d09f0f0f0433666" target='_blank'>
              Expressive Symbolic Regression for Interpretable Models of Discrete-Time Dynamical Systems
              </a>
            </td>
          <td>
            Adarsh Iyer, N. Boddupalli, Jeff Moehlis
          </td>
          <td>2024-06-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32686276485d13ba0f45ab92508b824a86c9f0fd" target='_blank'>
              Mixed effect estimation in deep compartment models: Variational methods outperform first-order approximations.
              </a>
            </td>
          <td>
            A. Janssen, Frank C Bennis, M. Cnossen, R. Mathôt
          </td>
          <td>2024-07-04</td>
          <td>Journal of pharmacokinetics and pharmacodynamics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Optimizing complex and high-dimensional black-box functions is ubiquitous in science and engineering fields. Unfortunately, the online evaluation of these functions is restricted due to time and safety constraints in most cases. In offline model-based optimization (MBO), we aim to find a design that maximizes the target function using only a pre-existing offline dataset. While prior methods consider forward or inverse approaches to address the problem, these approaches are limited by conservatism and the difficulty of learning highly multi-modal mappings. Recently, there has been an emerging paradigm of learning to improve solutions with synthetic trajectories constructed from the offline dataset. In this paper, we introduce a novel conditional generative modeling approach to produce trajectories toward high-scoring regions. First, we construct synthetic trajectories toward high-scoring regions using the dataset while injecting locality bias for consistent improvement directions. Then, we train a conditional diffusion model to generate trajectories conditioned on their scores. Lastly, we sample multiple trajectories from the trained model with guidance to explore high-scoring regions beyond the dataset and select high-fidelity designs among generated trajectories with the proxy function. Extensive experiment results demonstrate that our method outperforms competitive baselines on Design-Bench and its practical variants. The code is publicly available in \texttt{https://github.com/dbsxodud-11/GTG}.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fa109cf8ef9888e5f0dbfe9a6a6d7918594856e" target='_blank'>
              Guided Trajectory Generation with Diffusion Models for Offline Model-based Optimization
              </a>
            </td>
          <td>
            Taeyoung Yun, Sujin Yun, Jaewoo Lee, Jinkyoo Park
          </td>
          <td>2024-06-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c1cd074b744953a14cf72dff23f50b500e57211" target='_blank'>
              APF2: an improved ensemble method for pharmacogenomic variant effect prediction
              </a>
            </td>
          <td>
            Yitian Zhou, Sebastian Pirmann, V. Lauschke
          </td>
          <td>2024-05-27</td>
          <td>The Pharmacogenomics Journal</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4725662c7bf90014714ca908ee159702b1c16d9" target='_blank'>
              Decoding temporal heterogeneity in NSCLC through machine learning and prognostic model construction
              </a>
            </td>
          <td>
            Junpeng Cheng, Meizhu Xiao, Qingkang Meng, Min Zhang, Denan Zhang, Lei Liu, Qing Jin, Zhijin Fu, Yanjiao Li, Xiujie Chen, Hongbo Xie
          </td>
          <td>2024-06-13</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In clinical settings, we often face the challenge of building prediction models based on small observational data sets. For example, such a data set might be from a medical center in a multi-center study. Differences between centers might be large, thus requiring specific models based on the data set from the target center. Still, we want to borrow information from the external centers, to deal with small sample sizes. There are approaches that either assign weights to each external data set or each external observation. To incorporate information on differences between data sets and observations, we propose an approach that combines both into weights that can be incorporated into a likelihood for fitting regression models. Specifically, we suggest weights at the data set level that incorporate information on how well the models that provide the observation weights distinguish between data sets. Technically, this takes the form of inverse probability weighting. We explore different scenarios where covariates and outcomes differ among data sets, informing our simulation design for method evaluation. The concept of effective sample size is used for understanding the effectiveness of our subgroup modeling approach. We demonstrate our approach through a clinical application, predicting applied radiotherapy doses for cancer patients. Generally, the proposed approach provides improved prediction performance when external data sets are similar. We thus provide a method for quantifying similarity of external data sets to the target data set and use this similarity to include external observations for improving performance in a target data set prediction modeling task with small data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/806bd6d29b956df4eb6efcad0b248b9f647ca1ca" target='_blank'>
              Improving prediction models by incorporating external data with weights based on similarity
              </a>
            </td>
          <td>
            Max Behrens, Maryam Farhadizadeh, Angelika Rohde, Alexander Ruhle, N. H. Nicolay, Harald Binder, Daniela Zoller
          </td>
          <td>2024-05-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In the past two decades, machine learning potentials (MLPs) have driven significant developments in chemical, biological, and material sciences. The construction and training of MLPs enable fast and accurate simulations and analysis of thermodynamic and kinetic properties. This review focuses on the application of MLPs to reaction systems with consideration of bond breaking and formation. We review the development of MLP models, primarily with neural network and kernel-based algorithms, and recent applications of reactive MLPs (RMLPs) to systems at different scales. We show how RMLPs are constructed, how they speed up the calculation of reactive dynamics, and how they facilitate the study of reaction trajectories, reaction rates, free energy calculations, and many other calculations. Different data sampling strategies applied in building RMLPs are also discussed with a focus on how to collect structures for rare events and how to further improve their performance with active learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe08859388b658711cbf4ac096f214e5d2eca580" target='_blank'>
              Machine Learning of Reactive Potentials.
              </a>
            </td>
          <td>
            Yinuo Yang, Shuhao Zhang, Kavindri D. Ranasinghe, O. Isayev, A. Roitberg
          </td>
          <td>2024-06-01</td>
          <td>Annual review of physical chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Our interest is in replicated network data with multiple networks observed across the same set of nodes. Examples include brain connection networks, in which nodes corresponds to brain regions and replicates to different individuals, and ecological networks, in which nodes correspond to species and replicates to samples collected at different locations and/or times. Our goal is to infer a hierarchical structure of the nodes at a population level, while performing multi-resolution clustering of the individual replicates. In brain connectomics, the focus is on inferring common relationships among the brain regions, while characterizing inter-individual variability in an easily interpretable manner. To accomplish this, we propose a Bayesian hierarchical model, while providing theoretical support in terms of identifiability and posterior consistency, and design efficient methods for posterior computation. We provide novel technical tools for proving model identifiability, which are of independent interest. Our simulations and application to brain connectome data provide support for the proposed methodology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a92bd9ad8c8ee3e22f5fcf7a819cb2d7dd4b471" target='_blank'>
              Bayesian Deep Generative Models for Replicated Networks with Multiscale Overlapping Clusters
              </a>
            </td>
          <td>
            Yuren Zhou, Yuqi Gu, D. Dunson
          </td>
          <td>2024-05-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Predicting the physico-chemical properties of pure substances and mixtures is a central task in thermodynamics. Established prediction methods range from fully physics-based ab-initio calculations, which are only feasible for very simple systems, over descriptor-based methods that use some information on the molecules to be modeled together with fitted model parameters (e.g., quantitative-structure-property relationship methods or classical group contribution methods), to representation-learning methods, which may, in extreme cases, completely ignore molecular descriptors and extrapolate only from existing data on the property to be modeled (e.g., matrix completion methods). In this work, we propose a general method for combining molecular descriptors with representation learning using the so-called expectation maximization algorithm from the probabilistic machine learning literature, which uses uncertainty estimates to trade off between the two approaches. The proposed hybrid model exploits chemical structure information using graph neural networks, but it automatically detects cases where structure-based predictions are unreliable, in which case it corrects them by representation-learning based predictions that can better specialize to unusual cases. The effectiveness of the proposed method is demonstrated using the prediction of activity coefficients in binary mixtures as an example. The results are compelling, as the method significantly improves predictive accuracy over the current state of the art, showcasing its potential to advance the prediction of physico-chemical properties in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ef678f0a68b0a367fe2e0a484ff38a9207367fc" target='_blank'>
              Balancing Molecular Information and Empirical Data in the Prediction of Physico-Chemical Properties
              </a>
            </td>
          <td>
            Johannes Zenn, Dominik Gond, F. Jirasek, Robert Bamler
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Neuropeptides are biomolecules with crucial physiological functions. Accurate identification of neuropeptides is essential for understanding nervous system regulatory mechanisms. However, traditional analysis methods are expensive and laborious, and the development of effective machine learning models continues to be a subject of current research. Hence, in this research, we constructed an SVM-based machine learning neuropeptide predictor, iNP_ESM, by integrating protein language models Evolutionary Scale Modeling (ESM) and Unified Representation (UniRep) for the first time. Our model utilized feature fusion and feature selection strategies to improve prediction accuracy during optimization. In addition, we validated the effectiveness of the optimization strategy with UMAP (Uniform Manifold Approximation and Projection) visualization. iNP_ESM outperforms existing models on a variety of machine learning evaluation metrics, with an accuracy of up to 0.937 in cross-validation and 0.928 in independent testing, demonstrating optimal neuropeptide recognition capabilities. We anticipate improved neuropeptide data in the future, and we believe that the iNP_ESM model will have broader applications in the research and clinical treatment of neurological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8e92215fee91abba9a480dfc2b4401a6c9ecf2" target='_blank'>
              iNP_ESM: Neuropeptide Identification Based on Evolutionary Scale Modeling and Unified Representation Embedding Features
              </a>
            </td>
          <td>
            Honghao Li, Liangzhen Jiang, Kaixiang Yang, Shulin Shang, Mingxin Li, Zhibin Lv
          </td>
          <td>2024-06-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tissue engineering predominantly relies on trial and error in vitro and ex vivo experiments to develop protocols and bioreactors to generate functional tissues. As an alternative, in silico methods have the potential to significantly reduce the timelines and costs of experimental programs for tissue engineering. In this paper, we propose a methodology to formulate, select, calibrate, and test mathematical models to predict cell population growth as a function of the biochemical environment and to design optimal experimental protocols for model inference of in silico model parameters. We systematically combine methods from the experimental design, mathematical statistics, and optimization literature to develop unique and explainable mathematical models for cell population dynamics. The proposed methodology is applied to the development of this first published model for a population of the airway-relevant bronchio-alveolar epithelial (BEAS-2B) cell line as a function of the concentration of metabolic-related biochemical substrates. The resulting model is a system of ordinary differential equations that predict the temporal dynamics of BEAS-2B cell populations as a function of the initial seeded cell population and the glucose, oxygen, and lactate concentrations in the growth media, using seven parameters rigorously inferred from optimally designed in vitro experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525c18992f42c1b22450139ca64a3aaff597dd9d" target='_blank'>
              In silico model development and optimization of in vitro lung cell population growth
              </a>
            </td>
          <td>
            Amirmahdi Mostofinejad, David A. Romero, Dana Brinson, Alba E. Marin-Araujo, A. Bazylak, Thomas K. Waddell, S. Haykal, G. Karoubi, Cristina H. Amon
          </td>
          <td>2024-05-15</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="This study assesses the predictive performance of six machine learning models and a 1D Convolutional Neural Network (CNN) in forecasting tumor dynamics within three months following Gamma Knife radiosurgery (GKRS) in 77 brain metastasis (BM) patients. The analysis meticulously evaluates each model before and after hyperparameter tuning, utilizing accuracy, AUC, and other metrics derived from confusion matrices. The CNN model showcased notable performance with an accuracy of 98% and an AUC of 0.97, effectively complementing the broader model analysis. Initial findings highlighted that XGBoost significantly outperformed other models with an accuracy of 0.95 and an AUC of 0.95 before tuning. Post-tuning, the Support Vector Machine (SVM) demonstrated the most substantial improvement, achieving an accuracy of 0.98 and an AUC of 0.98. Conversely, XGBoost showed a decline in performance after tuning, indicating potential overfitting. The study also explores feature importance across models, noting that features like “control at one year”, “age of the patient”, and “beam-on time for volume V1 treated” were consistently influential across various models, albeit their impacts were interpreted differently depending on the model’s underlying mechanics. This comprehensive evaluation not only underscores the importance of model selection and hyperparameter tuning but also highlights the practical implications in medical diagnostic scenarios, where the accuracy of positive predictions can be crucial. Our research explores the effects of staged Gamma Knife radiosurgery (GKRS) on larger tumors, revealing no significant outcome differences across protocols. It uniquely considers the impact of beam-on time and fraction intervals on treatment efficacy. However, the investigation is limited by a small patient cohort and data from a single institution, suggesting the need for future multicenter research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1145edccd36b1fcabce5d3713aae781c5134f5c4" target='_blank'>
              Predicting Tumor Dynamics Post-Staged GKRS: Machine Learning Models in Brain Metastases Prognosis
              </a>
            </td>
          <td>
            A. Trofin, C. Buzea, Razvan-Cristian Buga, M. Agop, L. Ochiuz, D. Iancu, Lucian Eva
          </td>
          <td>2024-06-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d69703a20ce4710f71900130dd935761bbf01fa" target='_blank'>
              Promising directions of machine learning for partial differential equations.
              </a>
            </td>
          <td>
            Steve Brunton, J. Kutz
          </td>
          <td>2024-06-28</td>
          <td>Nature computational science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8112c18b6d184d24cba044589140338bfc4b2d5" target='_blank'>
              Advanced methods for gene network identification and noise decomposition from single-cell data
              </a>
            </td>
          <td>
            Zhou Fang, Ankit Gupta, Sant Kumar, Mustafa Khammash
          </td>
          <td>2024-06-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Many clinical trials generate both longitudinal biomarker and time-to-event data. We might be interested in their relationship, as in the case of tumor size and overall survival in oncology drug development. Many well-established methods exist for analyzing such data either sequentially (two-stage models) or simultaneously (joint models). Two-stage modeling (2stgM) has been challenged (i) for not acknowledging that biomarkers are endogenous covariable to the survival submodel and (ii) for not propagating the uncertainty of the longitudinal biomarker submodel to the survival submodel. On the other hand, joint modeling (JM), which properly circumvents both problems, has been criticized for being time-consuming, and difficult to use in practice. In this paper, we explore a third approach, referred to as a novel two-stage modeling (N2stgM). This strategy reduces the model complexity without compromising the parameter estimate accuracy. The three approaches (2stgM, JM, and N2stgM) are formulated, and a Bayesian framework is considered for their implementation. Both real and simulated data were used to analyze the performance of such approaches. In all scenarios, our proposal estimated the parameters approximately as JM but without being computationally expensive, while 2stgM produced biased results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1431f8841ec1dea194d5b4d31717f328d35e5b2" target='_blank'>
              Bridging the gap between two-stage and joint models: The case of tumor growth inhibition and overall survival models.
              </a>
            </td>
          <td>
            Danilo Alvares, F. Mercier
          </td>
          <td>2024-06-03</td>
          <td>Statistics in medicine</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Alzheimer's Disease (AD) significantly aggravates human dignity and quality of life. While newly approved amyloid immunotherapy has been reported, effective AD drugs remain to be identified. Here, we propose a novel AI-driven drug-repurposing method, DeepDrug, to identify a lead combination of approved drugs to treat AD patients. DeepDrug advances drug-repurposing methodology in four aspects. Firstly, it incorporates expert knowledge to extend candidate targets to include long genes, immunological and aging pathways, and somatic mutation markers that are associated with AD. Secondly, it incorporates a signed directed heterogeneous biomedical graph encompassing a rich set of nodes and edges, and node/edge weighting to capture crucial pathways associated with AD. Thirdly, it encodes the weighted biomedical graph through a Graph Neural Network into a new embedding space to capture the granular relationships across different nodes. Fourthly, it systematically selects the high-order drug combinations via diminishing return-based thresholds. A five-drug lead combination, consisting of Tofacitinib, Niraparib, Baricitinib, Empagliflozin, and Doxercalciferol, has been selected from the top drug candidates based on DeepDrug scores to achieve the maximum synergistic effect. These five drugs target neuroinflammation, mitochondrial dysfunction, and glucose metabolism, which are all related to AD pathology. DeepDrug offers a novel AI-and-big-data, expert-guided mechanism for new drug combination discovery and drug-repurposing across AD and other neuro-degenerative diseases, with immediate clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1482ea7ac4a2a14a60d729ce7c241246b189614" target='_blank'>
              DeepDrug: An Expert-led Domain-specific AI-Driven Drug-Repurposing Mechanism for Selecting the Lead Combination of Drugs for Alzheimer's Disease
              </a>
            </td>
          <td>
            V. Li, Y. Han, T. Kaistha, Q. Zhang, J. Downey, I. Gozes, J. C. K. Lam
          </td>
          <td>2024-07-07</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A central aim in computational neuroscience is to relate the activity of large populations of neurons to an underlying dynamical system. Models of these neural dynamics should ideally be both interpretable and fit the observed data well. Low-rank recurrent neural networks (RNNs) exhibit such interpretability by having tractable dynamics. However, it is unclear how to best fit low-rank RNNs to data consisting of noisy observations of an underlying stochastic system. Here, we propose to fit stochastic low-rank RNNs with variational sequential Monte Carlo methods. We validate our method on several datasets consisting of both continuous and spiking neural data, where we obtain lower dimensional latent dynamics than current state of the art methods. Additionally, for low-rank models with piecewise linear nonlinearities, we show how to efficiently identify all fixed points in polynomial rather than exponential cost in the number of units, making analysis of the inferred dynamics tractable for large RNNs. Our method both elucidates the dynamical systems underlying experimental recordings and provides a generative model whose trajectories match observed trial-to-trial variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579f56813a03517163b86fa89044dc78505bf2cb" target='_blank'>
              Inferring stochastic low-rank recurrent neural networks from neural data
              </a>
            </td>
          <td>
            Matthijs Pals, A. E. Saugtekin, Felix Pei, Manuel Gloeckler, J. H. Macke
          </td>
          <td>2024-06-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af19be9576fba29fc6ce10addc184ed0663d1348" target='_blank'>
              A Computational Predictor for Accurate Identification of Tumor Homing Peptides by Integrating Sequential and Deep BiLSTM Features.
              </a>
            </td>
          <td>
            Roha Arif, Sameera Kanwal, Saeed Ahmed, Muhammad Kabir
          </td>
          <td>2024-05-11</td>
          <td>Interdisciplinary sciences, computational life sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="In this paper, we present a guide to the foundations of learning Dynamic Bayesian Networks (DBNs) from data in the form of multiple samples of trajectories for some length of time. We present the formalism for a generic as well as a set of common types of DBNs for particular variable distributions. We present the analytical form of the models, with a comprehensive discussion on the interdependence between structure and weights in a DBN model and their implications for learning. Next, we give a broad overview of learning methods and describe and categorize them based on the most important statistical features, and how they treat the interplay between learning structure and weights. We give the analytical form of the likelihood and Bayesian score functions, emphasizing the distinction from the static case. We discuss functions used in optimization to enforce structural requirements. We briefly discuss more complex extensions and representations. Finally we present a set of comparisons in different settings for various distinct but representative algorithms across the variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c93e8d8300c924f0796c2d1e335ce010a7267e" target='_blank'>
              Learning Dynamic Bayesian Networks from Data: Foundations, First Principles and Numerical Comparisons
              </a>
            </td>
          <td>
            Vyacheslav Kungurtsev, Petr Rysavy, Fadwa Idlahcen, Pavel Rytir, Ales Wodecki
          </td>
          <td>2024-06-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In the real world, many dynamic behaviors can be explained by the propagation of perturbations, such as the transfer of chemical signals and the spread of infectious diseases. Previous researchers have achieved excellent results in approximating the global propagation time, revealing the mechanism of signal propagation through multiple paths. However, the known frameworks rely on the extension of physical concepts rather than mathematically rigorous derivations. As a result, they may not perfectly predict time or explain the underlying physical significance in certain specific cases. In this paper, we propose a novel method for decomposing network topology, focusing on two modules: the tree-like module and the path-module. Subsequently, we introduce a new framework for signal propagation analysis, which can be applied to estimate the propagation time for two fundamental global topology modules and provide a rigorous proof for the propagation time in global topology. Compared to previous work, our results are not only more concise, clearly defined, efficient, but also are more powerful in predicting propagation time which outperforms some known results in some cases, for example, biochemical dynamics.Additionally, the proposed framework is based on information transfer pathways, which can be also applied to other physical fields, such as network stability, network controlling and network resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5df6abe3831bf4c489162c2dc445d5e085e2f6f" target='_blank'>
              Information transfer pathways: signal propagation in complex global topologies
              </a>
            </td>
          <td>
            Qitong Hu, Xiao-Dong Zhang
          </td>
          <td>2024-05-20</td>
          <td>Physica Scripta</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="1614 Background: Despite the unique ability of machine learning (ML) techniques to uncover complex interactive effects in datasets, studies exploring predictors of clinical trial enrollment have largely used standard statistical methods to discern associations. Furthermore, there have been limited applications of ML to close existing disparities in clinical trial enrollment of underrepresented patients in neuro-oncology. This study aims to use supervised machine learning to define and validate primary contributors of therapeutic clinical trial enrollment for a) all patients, b) women and c) NIH-designated minority patients with low- and high-grade glioma. Methods: Adult glioma patients who received care from the UCSF Brain Tumor Center between 1997- 2017 were identified in a prospective registry. Bootstrap forest (BF) and Recursive Partitioning (RP) models were created by randomly dividing patients in a 70/30 split into development (DEV) and validation (VAL) cohorts among all patients, women, and NIH-designated minority patients, separately. Model performance was assessed using the area under the curve (AUC) in the DEV and VAL cohorts. Results: Among 1042 patients, 350 patients (33.6%) enrolled in a therapeutic clinical trial. There were 445 women (42.7%), of which 144 (32.4%) enrolled, and 141 minority patients (13.5%), of which 39 (27.7%) enrolled. For all patients, in order of decreasing influence, the BF model selected median neighborhood household income, age, neighborhood poverty level, distance from hospital, tumor location, tumor volume, treatment with chemotherapy, occupation, KPS, insurance status, employment status, seizure at presentation, and volumetric extent of resection (EOR) to predict trial enrollment (DEV AUC: 0.984; VAL AUC: 0.746). For women, the BF model selected distance from hospital, age, household income, neighborhood poverty level, tumor location, treatment with chemotherapy, and KPS for trial prediction (DEV AUC: 0.972; VAL AUC: 0.746), whereas for minority patients, the BF model selected neighborhood poverty level, age, occupation, distance from hospital, tumor volume, insurance status, employment status, and EOR to predict trial enrollment (DEV AUC: 0.9830; VAL AUC: 0.769). The RP model for women designated those with non-White race as less likely to enroll (DEV AUC: 0.735; VAL AUC: 0.665) while the RP model for minority patients characterized those who preferred a non-English language or who were divorced or widowed as less likely to enroll (DEV AUC: 0.775; VAL AUC: 0.601). Conclusions: Supervised machine learning models achieved similar predictive performance among patient subgroups, while uncovering interactions between gender identity, minority status, and sociodemographic variables. These findings can guide targeted recruiting efforts to advance equitable trial enrollment for women and minorities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f1876cfabc7bccab3e267b2ca9706dc1d0f3bf" target='_blank'>
              Predicting therapeutic clinical trial enrollment for underrepresented patients in neuro-oncology using supervised machine learning.
              </a>
            </td>
          <td>
            Mulki Mehari, G. Warrier, Abraham Dada, Aymen Kabir, U. Chukwueke, Alyx B Porter, Shawn L. Hervey-Jumper
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Data science is playing an increasingly important role in the design and analysis of engineered biology. This has been fueled by the development of high-throughput methods like massively parallel reporter assays, data-rich microscopy techniques, computational protein structure prediction and design, and the development of whole-cell models able to generate huge volumes of data. Although the ability to apply data-centric analyses in these contexts is appealing and increasingly simple to do, it comes with potential risks. For example, how might biases in the underlying data affect the validity of a result and what might the environmental impact of large-scale data analyses be? Here, we present a community-developed framework for assessing data hazards to help address these concerns and demonstrate its application to two synthetic biology case studies. We show the diversity of considerations that arise in common types of bioengineering projects and provide some guidelines and mitigating steps. Understanding potential issues and dangers when working with data and proactively addressing them will be essential for ensuring the appropriate use of emerging data-intensive AI methods and help increase the trustworthiness of their applications in synthetic biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/993a0859db6bdf23a69bbfc26332b2731030205e" target='_blank'>
              Data hazards in synthetic biology
              </a>
            </td>
          <td>
            Natalie R Zelenka, Nina Di Cara, Kieren Sharma, Seeralan Sarvaharman, Jasdeep S. Ghataora, Fabio Parmeggiani, Jeff Nivala, Z. Abdallah, Lucia Marucci, T. Gorochowski
          </td>
          <td>2024-06-21</td>
          <td>Synthetic Biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Proteins are complex molecules responsible for different functions in nature. Enhancing the functionality of proteins and cellular fitness can significantly impact various industries. However, protein optimization using computational methods remains challenging, especially when starting from low-fitness sequences. We propose LatProtRL, an optimization method to efficiently traverse a latent space learned by an encoder-decoder leveraging a large protein language model. To escape local optima, our optimization is modeled as a Markov decision process using reinforcement learning acting directly in latent space. We evaluate our approach on two important fitness optimization tasks, demonstrating its ability to achieve comparable or superior fitness over baseline methods. Our findings and in vitro evaluation show that the generated sequences can reach high-fitness regions, suggesting a substantial potential of LatProtRL in lab-in-the-loop scenarios.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9589c7a5062134012d56bf3ea6aedd1d76bfb1d" target='_blank'>
              Robust Optimization in Protein Fitness Landscapes Using Reinforcement Learning in Latent Space
              </a>
            </td>
          <td>
            Minji Lee, L. Vecchietti, Hyunkyu Jung, Hyun Joo Ro, Meeyoung Cha, Ho Min Kim
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="To make accurate predictions, understand mechanisms, and design interventions in systems of many variables, we wish to learn causal graphs from large scale data. Unfortunately the space of all possible causal graphs is enormous so scalably and accurately searching for the best fit to the data is a challenge. In principle we could substantially decrease the search space, or learn the graph entirely, by testing the conditional independence of variables. However, deciding if two variables are adjacent in a causal graph may require an exponential number of tests. Here we build a scalable and flexible method to evaluate if two variables are adjacent in a causal graph, the Differentiable Adjacency Test (DAT). DAT replaces an exponential number of tests with a provably equivalent relaxed problem. It then solves this problem by training two neural networks. We build a graph learning method based on DAT, DAT-Graph, that can also learn from data with interventions. DAT-Graph can learn graphs of 1000 variables with state of the art accuracy. Using the graph learned by DAT-Graph, we also build models that make much more accurate predictions of the effects of interventions on large scale RNA sequencing data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aba9e483fc80b2246e0043768867a60815d1c888" target='_blank'>
              Scalable and Flexible Causal Discovery with an Efficient Test for Adjacency
              </a>
            </td>
          <td>
            Alan Nawzad Amin, Andrew Gordon Wilson
          </td>
          <td>2024-06-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The key goals in learning Bayesian networks (BNs) from data are to identify significant statistical relationships between variables and to build a Directed Acyclic Graph (DAG) that represents these relationships through Joint Probability Distributions. Most research relies on score-based or conditional test methods for model selection. However, when using real-world data, it can be challenging to identify whether the learned DAG represents the underlying relations inherent in the limited datasets, particularly when evaluating data obtained from multiple independent sources. This study presents a methodology to assess the credible interval for both the existence and direction of each edge within Bayesian networks derived from data. Furthermore, it explores the fusion of models acquired from distinct and independent datasets. This approach enables the Bayesian learning of Bayesian Networks (BNs) from data by treating the uncertainty associated with the existence and orientation of each edge as a random variable. By evaluating the probability of the orientation of each edge, it is possible to suggest the existence of a potential latent variable within the dataset. If an edge exhibits equiprobable directions and is verified to exist, it becomes a plausible hypothesis for a latent variable. The Fast Causal Algorithm, originally introduced by [1], is the foundation of this approach. Finally, by employing a maximum a posteriori estimation, the most prominent edges and their respective orientations are identified and employed to create the leading DAG. We present our findings in simulated datasets with different length sizes. By comparing the structure of the learned DAGs with existing structures and evaluating the inference capabilities of the final BN, we establish that our approach achieves results comparable to the most recent studies in the field, while offering insights into the model’s reliability and improving the use of the data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e44ff83d27d27148c5c9f26a576e56ad85ceed40" target='_blank'>
              Quantifying uncertainty in Bayesian Networks structural learning
              </a>
            </td>
          <td>
            Vitor O. Barth, H. O. Caetano, C. Maciel, Marco Aiello
          </td>
          <td>2024-05-23</td>
          <td>2024 IEEE International Conference on Evolving and Adaptive Intelligent Systems (EAIS)</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Fully capturing cellular state requires examining genomic, epigenomic, transcriptomic, proteomic, and other assays for a biological sample and comprehensive computational modeling to reason with the complex and sometimes conflicting measurements. Modeling these so-called multi-omic data is especially beneficial in disease analysis, where observations across omic data types may reveal unexpected patient groupings and inform clinical outcomes and treatments. We present Multi-omic Pathway Analysis of Cancer (MPAC), a computational framework that interprets multi-omic data through prior knowledge from biological pathways. MPAC uses network relationships encoded in pathways using a factor graph to infer consensus activity levels for proteins and associated pathway entities from multi-omic data, runs permutation testing to eliminate spurious activity predictions, and groups biological samples by pathway activities to prioritize proteins with potential clinical relevance. Using DNA copy number alteration and RNA-seq data from head and neck squamous cell carcinoma patients from The Cancer Genome Atlas as an example, we demonstrate that MPAC predicts a patient subgroup related to immune responses not identified by analysis with either input omic data type alone. Key proteins identified via this subgroup have pathway activities related to clinical outcome as well as immune cell compositions. Our MPAC R package, available at https://bioconductor.org/packages/MPAC, enables similar multi-omic analyses on new datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45acaadfd1836282319ab14b566bdee48920988" target='_blank'>
              MPAC: a computational framework for inferring cancer pathway activities from multi-omic data
              </a>
            </td>
          <td>
            Peng Liu, David Page, Paul Ahlquist, Irene M. Ong, Anthony Gitter
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Predicting cancer-associated clinical events is challenging in oncology. In Multiple Myeloma (MM), a cancer of plasma cells, disease progression is determined by changes in biomarkers, such as serum concentration of the paraprotein secreted by plasma cells (M-protein). Therefore, the time-dependent behaviour of M-protein and the transition across lines of therapy (LoT) that may be a consequence of disease progression should be accounted for in statistical models to predict relevant clinical outcomes. Furthermore, it is important to understand the contribution of the patterns of longitudinal biomarkers, upon each LoT initiation, to time-to-death or time-to-next-LoT. Motivated by these challenges, we propose a Bayesian joint model for trajectories of multiple longitudinal biomarkers, such as M-protein, and the competing risks of death and transition to next LoT. Additionally, we explore two estimation approaches for our joint model: simultaneous estimation of all parameters (joint estimation) and sequential estimation of parameters using a corrected two-stage strategy aiming to reduce computational time. Our proposed model and estimation methods are applied to a retrospective cohort study from a real-world database of patients diagnosed with MM in the US from January 2015 to February 2022. We split the data into training and test sets in order to validate the joint model using both estimation approaches and make dynamic predictions of times until clinical events of interest, informed by longitudinally measured biomarkers and baseline variables available up to the time of prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55580b4051352431cf661d36a73d6403047a57db" target='_blank'>
              A Bayesian joint model of multiple nonlinear longitudinal and competing risks outcomes for dynamic prediction in multiple myeloma: joint estimation and corrected two-stage approaches
              </a>
            </td>
          <td>
            Danilo Alvares, Jessica K. Barrett, Franccois Mercier, S. Roumpanis, Sean Yiu, Felipe Castro, Jochen Schulze, Yajing Zhu
          </td>
          <td>2024-05-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cell lines are frequently used in biological and translational research to study cellular mechanisms and explore treatment options. However, cancer cell lines may display mutational profiles divergent from native cancers or may be misidentified or contaminated. We explored how similar cancer cell lines are to native cancers to find the most suitable representations for the corresponding diseases by utilising large collections of copy number variation (CNV) profiles and applied machine learning (ML) algorithms to predict cell line classifications. Our results confirm that cancer cell lines indeed accumulate more mutations compared to native cancers but retain similar CNV profiles. We demonstrate that many relevant oncogenes and tumor suppressor genes are altered by CNV events in both cancers and their corresponding cell lines. Based on the similarities between the two groups and the predictions of the ML model, we provide some recommendations about cell lines with good potential to represent selected cancer types in in vitro studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a40be46120d1a70ea751af03bf49fa1749336d2d" target='_blank'>
              In Search of the Perfect Model: How Cancer Cell Lines Relate to Native Cancers
              </a>
            </td>
          <td>
            Rahel Paloots, Ziying Yang, M. Baudis
          </td>
          <td>2024-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="High-content image-based phenotypic profiling combines automated microscopy and analysis to identify phenotypic alterations in cell morphology and provide insight into the cell’s physiological state. Classical representations of the phenotypic profile can not capture the full underlying complexity in cell organization, while recent weakly machine-learning based representation-learning methods are hard to biologically interpret. We used the abundance of control wells to learn the in-distribution of control experiments and use it to formulate a self-supervised reconstruction anomaly-based representation that encodes the intricate morphological inter-feature dependencies while preserving the representation interpretability. The performance of our anomaly-based representations was evaluated for downstream tasks with respect to two classical representations across four public Cell Painting datasets. Anomaly-based representations improved reproducibility, Mechanism of Action classification, and complemented classical representations. Unsupervised explainability of autoencoder-based anomalies identified specific inter-feature dependencies causing anomalies. The general concept of anomaly-based representations can be adapted to other applications in cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c25e02edd59fc80ca4e5c42466769051e0cd90b2" target='_blank'>
              Anomaly detection for high-content image-based phenotypic cell profiling
              </a>
            </td>
          <td>
            Alon Shpigler, Naor Kolet, Shahar Golan, Erin Weisbart, A. Zaritsky
          </td>
          <td>2024-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5916a879490d67fb48ad872b52d27539ca4a54" target='_blank'>
              Deep representation learning of chemical-induced transcriptional profile for phenotype-based drug discovery
              </a>
            </td>
          <td>
            X. Tong, Ning Qu, Xiangtai Kong, Shengkun Ni, Jingyi Zhou, Kun Wang, Lehan Zhang, Yiming Wen, Jiangshan Shi, Sulin Zhang, Xutong Li, Mingyue Zheng
          </td>
          <td>2024-06-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="With the development of high-throughput technologies, genomics datasets rapidly grow in size, including functional genomics data. This has allowed the training of large Deep Learning (DL) models to predict epigenetic readouts, such as protein binding or histone modifications, from genome sequences. However, large dataset sizes come at a price of data consistency, often aggregating results from a large number of studies, conducted under varying experimental conditions. While data from large-scale consortia are useful as they allow studying the effects of different biological conditions, they can also contain unwanted biases from confounding experimental factors. Here, we introduce Metadata-guided Feature Disentanglement (MFD) - an approach that allows disentangling biologically relevant features from potential technical biases. MFD incorporates target metadata into model training, by conditioning weights of the model output layer on different experimental factors. It then separates the factors into disjoint groups and enforces independence of the corresponding feature subspaces with an adversarially learned penalty. We show that the metadata-driven disentanglement approach allows for better model introspection, by connecting latent features to experimental factors, without compromising, or even improving performance in downstream tasks, such as enhancer prediction, or genetic variant discovery. The code for our implemementation is available at https://github.com/HealthML/MFD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32dd14e31f771ca226153c9210adaac58da256a4" target='_blank'>
              Metadata-guided Feature Disentanglement for Functional Genomics
              </a>
            </td>
          <td>
            Alexander Rakowski, Remo Monti, Viktoriia Huryn, Marta Lemanczyk, Uwe Ohler, Christoph Lippert
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Recent advancements in single-cell genomics necessitate precision in gene panel selection to interpret complex biological data effectively. Those methods aim to streamline the analysis of scRNA-seq data by focusing on the most informative genes that contribute significantly to the specific analysis task. Traditional selection methods, which often rely on expert domain knowledge, embedded machine learning models, or heuristic-based iterative optimization, are prone to biases and inefficiencies that may obscure critical genomic signals. Recognizing the limitations of traditional methods, we aim to transcend these constraints with a refined strategy. In this study, we introduce an iterative gene panel selection strategy that is applicable to clustering tasks in single-cell genomics. Our method uniquely integrates results from other gene selection algorithms, providing valuable preliminary boundaries or prior knowledge as initial guides in the search space to enhance the efficiency of our framework. Furthermore, we incorporate the stochastic nature of the exploration process in reinforcement learning (RL) and its capability for continuous optimization through reward-based feedback. This combination mitigates the biases inherent in the initial boundaries and harnesses RL's adaptability to refine and target gene panel selection dynamically. To illustrate the effectiveness of our method, we conducted detailed comparative experiments, case studies, and visualization analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e89dc4802b5da2d3584b95250446f7b5148b7c" target='_blank'>
              Enhanced Gene Selection in Single-Cell Genomics: Pre-Filtering Synergy and Reinforced Optimization
              </a>
            </td>
          <td>
            Weiliang Zhang, Zhen Meng, Dongjie Wang, Min Wu, Kunpeng Liu, Yuanchun Zhou, Meng Xiao
          </td>
          <td>2024-06-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Techniques to autonomously drive research have been prominent in Computational Scientific Discovery, while Synthetic Biology is a field of science that focuses on designing and constructing new biological systems for useful purposes. Here we seek to apply logic-based machine learning techniques to facilitate cellular engineering and drive biological discovery. Comprehensive databases of metabolic processes called genome-scale metabolic network models (GEMs) are often used to evaluate cellular engineering strategies to optimise target compound production. However, predicted host behaviours are not always correctly described by GEMs, often due to errors in the models. The task of learning the intricate genetic interactions within GEMs presents computational and empirical challenges. To address these, we describe a novel approach called Boolean Matrix Logic Programming (BMLP) by leveraging boolean matrices to evaluate large logic programs. We introduce a new system, $BMLP_{active}$, which efficiently explores the genomic hypothesis space by guiding informative experimentation through active learning. In contrast to sub-symbolic methods, $BMLP_{active}$ encodes a state-of-the-art GEM of a widely accepted bacterial host in an interpretable and logical representation using datalog logic programs. Notably, $BMLP_{active}$ can successfully learn the interaction between a gene pair with fewer training examples than random experimentation, overcoming the increase in experimental design space. $BMLP_{active}$ enables rapid optimisation of metabolic models to reliably engineer biological systems for producing useful compounds. It offers a realistic approach to creating a self-driving lab for microbial engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4217fa5bc20619984d78d30c100e2b19faa0091" target='_blank'>
              Boolean matrix logic programming for active learning of gene functions in genome-scale metabolic network models
              </a>
            </td>
          <td>
            L. Ai, Stephen Muggleton, Shishun Liang, Geoff S. Baldwin
          </td>
          <td>2024-05-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genotype-to-phenotype mapping is an essential problem in the current genomic era. While qualitative case-control predictions have received significant attention, less emphasis has been placed on predicting quantitative phenotypes. This emerging field holds great promise in revealing intricate connections between microbial communities and host health. However, the presence of heterogeneity in microbiome datasets poses a substantial challenge to the accuracy of predictions and undermines the reproducibility of models. To tackle this challenge, we investigated 22 normalization methods that aimed at removing heterogeneity across multiple datasets, conducted a comprehensive review of them, and evaluated their effectiveness in predicting quantitative phenotypes in three simulation scenarios and 31 real datasets. The results indicate that none of these methods demonstrate significant superiority in predicting quantitative phenotypes or attain a noteworthy reduction in Root Mean Squared Error (RMSE) of the predictions. Given the frequent occurrence of batch effects and the satisfactory performance of batch correction methods in predicting datasets affected by these effects, we strongly recommend utilizing batch correction methods as the initial step in predicting quantitative phenotypes. In summary, the performance of normalization methods in predicting metagenomic data remains a dynamic and ongoing research area. Our study contributes to this field by undertaking a comprehensive evaluation of diverse methods and offering valuable insights into their effectiveness in predicting quantitative phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a680651a3cf41ba9343ce9f9c1efc81ca6d5163" target='_blank'>
              Evaluation of normalization methods for predicting quantitative phenotypes in metagenomic data analysis
              </a>
            </td>
          <td>
            Beibei Wang, Yihui Luan
          </td>
          <td>2024-06-05</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Inferring causal protein signalling networks from human immune system cell data is a promising approach to unravel the underlying tissue signalling biology and dysfunction in diseased cells, which has attracted considerable attention within the bioinformatics field. Recently, Bayesian network (BN) techniques have gained significant popularity in inferring causal protein signalling networks from multiparameter single‐cell data. However, current BN methods may exhibit high computational complexity and ignore interactions among protein signalling molecules from different single cells. A novel BN method is presented for learning causal protein signalling networks based on parallel discrete artificial bee colony (PDABC), named PDABC. Specifically, PDABC is a score‐based BN method that utilises the parallel artificial bee colony to search for the global optimal causal protein signalling networks with the highest discrete K2 metric. The experimental results on several simulated datasets, as well as a previously published multi‐parameter fluorescence‐activated cell sorter dataset, indicate that PDABC surpasses the existing state‐of‐the‐art methods in terms of performance and computational efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45fc9744b37a980e5e05a5eeb12a6c928df4c0fc" target='_blank'>
              Inferring causal protein signalling networks from single‐cell data based on parallel discrete artificial bee colony algorithm
              </a>
            </td>
          <td>
            Jinduo Liu, Jihao Zhai, Junzhong Ji
          </td>
          <td>2024-05-11</td>
          <td>CAAI Transactions on Intelligence Technology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Motivation Despite the development of several computational methods to predict DNA modifications, two main limitations persist in the current methodologies: 1) All existing models are confined to binary predictor which merely determine the presence or absence of DNA modifications, constraining comprehensive analyses of the interrelations among varied modification types. While multi-class classification models for RNA modifications have been developed, a comparable approach for DNA remains a critical need. 2) The majority of previous studies lack adequate explanations of how models make decisions, relying on the extraction and visualization of attention matrices which identified few motifs, and do not provide sufficient insight into the model decision making process. Result In this study, we introduce iResNetDM, a deep learning model that integrates ResNet and self-attention mechanisms. To the best of our knowledge, iResNetDM is the first model capable of distinguishing between four types of DNA modifications. It not only demonstrates high performance across various DNA modifications but also unveils the potential capabilities of CNN and ResNet in this domain. To augment the interpretability of our model, we implemented the integrated gradients technique, which was pivotal in demystifying the model’s decision-making framework, allowing for the successful identification of multiple motifs. Importantly, our model exhibits remarkable robustness, successfully identifying unique motifs across different modifications. Furthermore, we compared the motifs discovered in various modifications, revealing that some motifs share significant sequence similarities which suggests that these motifs may be subjected to different types of modifications, underscoring their potential importance in gene regulation. Contact zeruiyang2-c@my.cityu.edu.hk">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dc84776c4ffe19ea1a049861b9a2aa12621e736" target='_blank'>
              iResNetDM: interpretable and comprehensive deep learning model for 4 types of DNA modifications prediction
              </a>
            </td>
          <td>
            Zerui Yang, Wei Shao, Yudai Matsuda, Linqi Song
          </td>
          <td>2024-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The most advanced diffusion models have recently adopted increasingly deep stacked networks (e.g., U-Net or Transformer) to promote the generative emergence capabilities of vision generation models similar to large language models (LLMs). However, progressively deeper stacked networks will intuitively cause numerical propagation errors and reduce noisy prediction capabilities on generative data, which hinders massively deep scalable training of vision generation models. In this paper, we first uncover the nature that neural networks being able to effectively perform generative denoising lies in the fact that the intrinsic residual unit has consistent dynamic property with the input signal's reverse diffusion process, thus supporting excellent generative abilities. Afterwards, we stand on the shoulders of two common types of deep stacked networks to propose a unified and massively scalable Neural Residual Diffusion Models framework (Neural-RDM for short), which is a simple yet meaningful change to the common architecture of deep generative networks by introducing a series of learnable gated residual parameters that conform to the generative dynamics. Experimental results on various generative tasks show that the proposed neural residual models obtain state-of-the-art scores on image's and video's generative benchmarks. Rigorous theoretical proofs and extensive experiments also demonstrate the advantages of this simple gated residual mechanism consistent with dynamic modeling in improving the fidelity and consistency of generated content and supporting large-scale scalable training. Code is available at https://github.com/Anonymous/Neural-RDM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6615dc6131db626329d21e88997e7cbca98de06" target='_blank'>
              Neural Residual Diffusion Models for Deep Scalable Vision Generation
              </a>
            </td>
          <td>
            Zhiyuan Ma, Liangliang Zhao, Biqing Qi, Bowen Zhou
          </td>
          <td>2024-06-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Deep learning has been increasingly popular in omics data analysis. Recent works incorporating variable selection into deep learning have greatly enhanced the model’s interpretability. However, because deep learning desires a large sample size, the existing methods may result in uncertain findings when the dataset has a small sample size, commonly seen in omics data analysis. With the explosion and availability of omics data from multiple populations/studies, the existing methods naively pool them into one dataset to enhance the sample size while ignoring that variable structures can differ across datasets, which might lead to inaccurate variable selection results. We propose a penalized integrative deep neural network (PIN) to simultaneously select important variables from multiple datasets. PIN directly aggregates multiple datasets as input and considers both homogeneity and heterogeneity situations among multiple datasets in an integrative analysis framework. Results from extensive simulation studies and applications of PIN to gene expression datasets from elders with different cognitive statuses or ovarian cancer patients at different stages demonstrate that PIN outperforms existing methods with considerably improved performance among multiple datasets. The source code is freely available on Github (rucliyang/PINFunc). We speculate that the proposed PIN method will promote the identification of disease‐related important variables based on multiple studies/datasets from diverse origins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1667454b3841428b4c868841dbc397f6a3e10d6" target='_blank'>
              A penalized integrative deep neural network for variable selection among multiple omics datasets
              </a>
            </td>
          <td>
            Yang Li, Xiaonan Ren, Haochen Yu, Tao Sun, Shuangge Ma
          </td>
          <td>2024-06-07</td>
          <td>Quantitative Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129758877c0cd26445a33434e51b6c889c6028cd" target='_blank'>
              Exploration and verification a 13-gene diagnostic framework for ulcerative colitis across multiple platforms via machine learning algorithms
              </a>
            </td>
          <td>
            Jing Wang, Lin Li, Pingbo Chen, C. He, Xiaoping Niu
          </td>
          <td>2024-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="3064 Background: In recent years, the use of tumor molecular profiling in clinical settings has enhanced cancer diagnostics, as well as the delivery of precision oncology. Recently, several methods for predicting gene expression directly from Haematoxylin-and-Eosin-stained (H&E) histology images have offered a new way to leverage the easily obtainable and cost-effective H&E images for multiple precision oncology applications. We previously introduced such a method – DeepPT – and demonstrated how we can leverage its imputed gene expression for successful prediction of drug response, through our ENLIGHT-DeepPT platform. In our previous publication and in an independent study comparing six different methods, DeepPT exhibited the best overall gene expression prediction accuracy. Methods: Here we present a new version, DeepPT 2.0, with improved architecture, including multi-task learning and a feature space based on a self-supervised pre-trained deep network. We tested both versions of DeepPT and the leading competing methods on patient data from 17 cancer subtypes included in TCGA. In addition, we obtained a new dataset collected at the Medical University of Bialystok (UMB) consisting of matched slide images and mRNA expression from 151 cases representing 7 cancer subtypes. These serve as an external validation to demonstrate that DeepPT generalizes well. Results: Our findings indicate that both versions of DeepPT show statistically significant improvement compared with the other methods in terms of median Pearson correlation of top predicted genes - the only metric available for all methods. Moreover, DeepPT 2.0 significantly improves upon version 1.0, demonstrating up to a 3-fold increase in the number of well-predicted genes (defined as genes with Pearson 𝜌 > 0.4 between actual and predicted mRNA expressions) for 14 of the 17 cancer subtypes tested. On the external validation data from UMB, DeepPT 2.0 improves gene expression prediction in 6 of 7 of the tested cancer subtypes, with up to a 7-fold increase in the number of well-predicted genes, thus mitigating the concern of overfitting on the training set. Immune genes are particularly well predicted, as we previously observed: using a set of 826 hallmark immune genes, DeepPT 2.0 exhibits up to 3.5-fold and 2-fold increase in the percentage of well-predicted genes in the TCGA and UMB data, respectively, compared to all other genes. Conclusions: DeepPT 2.0 significantly improves upon competing methods for predicting mRNA expression from H&E slides across multiple metrics and diverse cancer types. Furthermore, it demonstrates robust generalization to slides from sources not previously seen. The method’s good ability to predict the expression of immune genes suggests a potential benefit in predicting response to immunotherapy using the ENLIGHT-DeepPT platform, as indeed demonstrated elsewhere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c75aa806813e51a7f41b199e9391b3e01188e73" target='_blank'>
              Development, comparative study, and external validation of a new deep learning model for predicting genome-wide gene expression from histopathology slides.
              </a>
            </td>
          <td>
            Leon Gugel, O. Tirosh, Yaron Kinar, Anna Michalska-Falkowska, Joanna Reszeć-Giełażyn, Witold Bauer, Magdalena Niemira, Boris Temkin, T. Beker, R. Aharonov, Gal Dinstag
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
Fusing known physics into data-driven learners allows modelling practitioners to combine the expressive power of traditional machine learning with known mechanistic laws, where the objective is to enhance predictive performance, interpretability, and model generalisation. A core consideration that must be made when implementing a physics-informed learning architecture is how relevant knowledge will be embedded into the model structure, which, generally, is informed by the type of physics that is available. Frequently this knowledge may not be complete, with only a partial understanding of the governing physics available. 

In this work, possible paths for deriving Gaussian process kernels that are representative of partial knowledge will be considered. How the type of knowledge that is possessed influences the derivation will be explored, particularly when there is the potential for some aspect of misspecified physics. An example of deriving partially structured kernels will be investigated for modelling the decoupled response of a GARTEUR laboratory aircraft structure, where the derived kernels are used to decompose the dynamics of the aircraft into modal contributions. 

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da9bbf07470b8cc6d6606c4b51a040b8e45ce3d6" target='_blank'>
              Gaussian process kernels for partial physical insight
              </a>
            </td>
          <td>
            Matthew R. Jones, D. J. Pitchforth, E. J. Cross
          </td>
          <td>2024-07-01</td>
          <td>e-Journal of Nondestructive Testing</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Single-cell RNA sequencing has revealed cellular heterogeneity in complex tissues, notably benefiting research on diseases such as cancer. However, the integration of single-cell data from small samples with extensive clinical features in bulk data remains underexplored. In this study, we introduce PIPET, an algorithmic method for predicting relevant subpopulations in single-cell data based on multivariate phenotypic information from bulk data. PIPET generates feature vectors for each phenotype from differentially expressed genes in bulk data and then identifies relevant cellular subpopulations by assessing the similarity between single-cell data and these vectors. Subsequently, phenotype-related cell states can be analyzed based on these subpopulations. In simulated datasets, PIPET showed robust performance in predicting multiclassification cellular subpopulations. Application of PIPET to lung adenocarcinoma single-cell RNA sequencing data revealed cellular subpopulations with poor survival and associations with TP53 mutations. Similarly, in breast cancer single-cell data, PIPET identified cellular subpopulations associated with the PAM50 clinical subtypes and triple-negative breast cancer subtypes. Overall, PIPET effectively identified relevant cellular subpopulations in single-cell data, guided by phenotypic information from bulk data. This approach comprehensively delineates the molecular characteristics of each cellular subpopulation, offering insights into disease-related subpopulations and guiding personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c27106028332ba4232d07206280311318212e1f" target='_blank'>
              PIPET: predicting relevant subpopulations in single-cell data using phenotypic information from bulk data
              </a>
            </td>
          <td>
            Xinjia Ruan, Yu Cheng, Yuqing Ye, Yuhang Wang, Xinyi Chen, Yuqing Yang, Tiantian Liu, Fangrong Yan
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In dynamical systems reconstruction (DSR) we seek to infer from time series measurements a generative model of the underlying dynamical process. This is a prime objective in any scientific discipline, where we are particularly interested in parsimonious models with a low parameter load. A common strategy here is parameter pruning, removing all parameters with small weights. However, here we find this strategy does not work for DSR, where even low magnitude parameters can contribute considerably to the system dynamics. On the other hand, it is well known that many natural systems which generate complex dynamics, like the brain or ecological networks, have a sparse topology with comparatively few links. Inspired by this, we show that geometric pruning, where in contrast to magnitude-based pruning weights with a low contribution to an attractor's geometrical structure are removed, indeed manages to reduce parameter load substantially without significantly hampering DSR quality. We further find that the networks resulting from geometric pruning have a specific type of topology, and that this topology, and not the magnitude of weights, is what is most crucial to performance. We provide an algorithm that automatically generates such topologies which can be used as priors for generative modeling of dynamical systems by RNNs, and compare it to other well studied topologies like small-world or scale-free networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7262300a7dc61b5b9020753dc9d2bbd63a06cc2" target='_blank'>
              Optimal Recurrent Network Topologies for Dynamical Systems Reconstruction
              </a>
            </td>
          <td>
            Christoph Jurgen Hemmer, Manuel Brenner, Florian Hess, Daniel Durstewitz
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Radiobiological data, whether obtained at the clinical, biological or molecular level has significantly contributed to a better description and prediction of the individual dose-response to ionizing radiation and a better estimation of the radiation-induced risks. Particularly, over the last seventy years, the amount of radiobiological data has considerably increased, and permitted the mathematical formulas describing dose-response to become less empirical. A better understanding of the basic radiobiological mechanisms has also contributed to establish quantitative inter-correlations between clinical, biological and molecular biomarkers, refining again the mathematical models of description. Today, big data approaches and, more recently, artificial intelligence may finally complete and secure this long process of thinking from the multi-scale description of radiation-induced events to their prediction. Here, we reviewed the major dose-response models applied in radiobiology for quantifying molecular and cellular radiosensitivity and aimed to explain their evolution: Specifically, we highlighted the advances concerning the target theory with the cell survival models and the progressive introduction of the DNA repair process in the mathematical models. Furthermore, we described how the technological advances have changed the description of DNA double-strand break (DSB) repair kinetics by introducing the important notion of DSB recognition, independent of that of DSB repair. Initially developed separately, target theory on one hand and, DSB recognition and repair, on the other hand may be now fused into a unified model involving the cascade of phosphorylations mediated by the ATM kinase in response to any genotoxic stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cdbdf7288e7e261d821e90454903709c5fbfc0e" target='_blank'>
              Seventy Years of Dose-response Models: From the Target Theory to the Use of Big Databases Involving Cell Survival and DNA Repair.
              </a>
            </td>
          <td>
            L. Bodgi, Laurent Pujo-Menjouet, Audrey Bouchet, Michel Bourguignon, N. Foray
          </td>
          <td>2024-05-28</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The growth of bacterial gene expression datasets has offered unprecedented coverage of achievable transcriptomes, reflecting diverse activity states of the transcription regulatory network. Machine learning methods like Independent Component Analysis (ICA) can decompose gene expression datasets into regulatory modules and condition-specific regulator activities. Here, we present a workflow to utilize inferred regulator activities to construct quantitative models of promoter regulation in E. coli. Resulting models are validated by predicting condition-specific TF effector concentrations and binding site motif strength based on differential gene expression data alone. We show how reconstructed promoter models can capture multi-scale regulation and disentangle regulator interactions, including resolving the apparent paradox where argR expression is positively correlated with its regulon despite being a repressor. We applied the workflow for all regulator-linked components extracted by ICA, demonstrating the scalability of the workflow to capture the E. coli TRN. This work suggests a path toward systematic, quantitative reconstruction of transcription regulatory networks driven by the large-scale databases that are now available for many organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54beacc49acede7e1ab1316132bfe8fb330701f" target='_blank'>
              Data-driven modeling of E. coli transcriptional regulation
              </a>
            </td>
          <td>
            Christopher Dalldorf, Griffith Hughes, Gaoyuan Li, Bernhard O. Palsson, Daniel C. Zielinski
          </td>
          <td>2024-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Uncertainty in parameter estimates from fitting within-host models to empirical data limits the model’s ability to uncover mechanisms of infection, disease progression, and to guide pharmaceutical interventions. Understanding the effect of model structure and data availability on model predictions is important for informing model development and experimental design. To address sources of uncertainty in parameter estimation, we use four mathematical models of influenza A infection with increased degrees of biological realism. We test the ability of each model to reveal its parameters in the presence of unlimited data by performing structural identifiability analyses. We then refine the results by predicting practical identifiability of parameters under daily influenza A virus titers alone or together with daily adaptive immune cell data. Using these approaches, we present insight into the sources of uncertainty in parameter estimation and provide guidelines for the types of model assumptions, optimal experimental design, and biological information needed for improved predictions. Author summary Within-host models of virus infections fitted to data have improved our understanding of mechanisms of infection, disease progression, and allowed us to provide guidelines for pharmaceutical interventions. Given their predictive power, it is essential that we properly uncover and address uncertainty in model predictions and parameter estimation. Here, we focus on the effect of model structure and data availability on our ability to uncover unknown parameters. To address these questions, we use four mathematical models of influenza A infection with increased degrees of biological realism. We test the ability of each model to reveal its parameters in the presence of unlimited data by performing structural identifiability analysis. We then refine the results by predicting practical identifiability of parameters under daily influenza A virus titers alone or together with daily adaptive immune cell data. Using these approaches, we present insights into the sources of uncertainty in parameter estimations and provide guidelines for the types of model assumptions, optimal experimental design, and biological information needed for improved predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08312e1a270ca4ad6a36e75e2b5550f77112dd89" target='_blank'>
              Identifiability investigation of within-host models of acute virus infection
              </a>
            </td>
          <td>
            Yuganthi R. Liyanage, Nora Heitzman-Breen, N. Tuncer, Stanca M. Ciupe
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Modeling dynamical systems, e.g. in climate and engineering sciences, often necessitates solving partial differential equations. Neural operators are deep neural networks designed to learn nontrivial solution operators of such differential equations from data. As for all statistical models, the predictions of these models are imperfect and exhibit errors. Such errors are particularly difficult to spot in the complex nonlinear behaviour of dynamical systems. We introduce a new framework for approximate Bayesian uncertainty quantification in neural operators using function-valued Gaussian processes. Our approach can be interpreted as a probabilistic analogue of the concept of currying from functional programming and provides a practical yet theoretically sound way to apply the linearized Laplace approximation to neural operators. In a case study on Fourier neural operators, we show that, even for a discretized input, our method yields a Gaussian closure--a structured Gaussian process posterior capturing the uncertainty in the output function of the neural operator, which can be evaluated at an arbitrary set of points. The method adds minimal prediction overhead, can be applied post-hoc without retraining the neural operator, and scales to large models and datasets. We showcase the efficacy of our approach through applications to different types of partial differential equations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb48eb416a7b362eb444cb5d9c111a6939db5fe" target='_blank'>
              Linearization Turns Neural Operators into Function-Valued Gaussian Processes
              </a>
            </td>
          <td>
            Emilia Magnani, Marvin Pfortner, Tobias Weber, Philipp Hennig
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Allostery is a fundamental mechanism driving biomolecular processes that holds significant therapeutic concern. Our study rigorously investigates how two distinct machine-learning algorithms uniquely classify two already close-to-active DFG-in states of TAK1, differing just by the presence or absence of its allosteric activator TAB1, from an ensemble mixture of conformations (obtained from 2.4 μs molecular dynamics (MD) simulations). The novelty, however, lies in understanding the deeper algorithmic potentials to systematically derive a diverse set of differential residue connectivity features that reconstruct the essential mechanistic architecture for TAK1-TAB1 allostery in such a close-to-active biochemical scenario. While the recursive, random forest-based workflow displays the potential of conducting discretized, hierarchical derivation of allosteric features, a multilayer perceptron-based approach gains considerable efficacy in revealing fluid connected patterns of features when hybridized with mutual information scoring. Interestingly, both pipelines benchmark similar directions of functional conformational changes for TAK1's activation. The findings significantly advance the depth of mechanistic understanding by highlighting crucial activation signatures along a directed C-lobe → activation loop → ATP pocket channel of information flow, including (1) the αF-αE biterminal alignments and (2) the "catalytic" drift of the activation loop toward kinase active site. Besides, some novel allosteric hotspots (K253, Y206, N189, etc.) are further recognized as TAB1 sensors, transducers, and responders, including a benchmark E70 mutation site, precisely mapping the important structural segments for sequential allosteric execution. Hence, our work demonstrates how to navigate through greater structural depths and dimensions of dynamic allosteric machineries just by leveraging standard ML methods in suitable streamlined workflows adaptive to the specific system and objectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e656737754f86ab9ba75a485691b12e7213422c" target='_blank'>
              Adaptive Workflows of Machine Learning Illuminate the Sequential Operation Mechanism of the TAK1's Allosteric Network.
              </a>
            </td>
          <td>
            Nibedita Ray Chaudhuri, Shubhra Ghosh Dastidar
          </td>
          <td>2024-05-14</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Ab‐initio molecular dynamics (AIMD) is a key method for realistic simulation of complex atomistic systems and processes in nanoscale. In AIMD, finite‐temperature dynamical trajectories are generated by using forces computed from electronic structure calculations. In systems with high numbers of components a typical AIMD run is computationally demanding. On the other hand, machine learning (ML) is a subfield of the artificial intelligence that consist in a set of algorithms that show learning by experience with the use of input and output data where algorithms are capable of analysing and predicting the future. At present, the main application of ML techniques in atomic simulations is the development of new interatomic potentials to correctly describe the potential energy surfaces (PES). This technique is in constant progress since its inception around 30 years ago. The ML potentials combine the advantages of classical and Ab‐initio methods, that is, the efficiency of a simple functional form and the accuracy of first principles calculations. In this article we review the evolution of four generations of machine learning potentials and some of their most notable applications. This review focuses on MLPs based on neural networks. Also, we present a state of art of this topic and future trends. Finally, we report the results of a scientometric study (covering the period 1995–2023) about the impact of ML techniques applied to atomistic simulations, distribution of publications by geographical regions and hot topics investigated in the literature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207474cf491b03e0ccde7e380d0d8f516f28fb14" target='_blank'>
              An overview about neural networks potentials in molecular dynamics simulation
              </a>
            </td>
          <td>
            R. Martin‐Barrios, E. Navas‐Conyedo, Xuyi Zhang, Yunwei Chen, J. Gulín‐González
          </td>
          <td>2024-05-21</td>
          <td>International Journal of Quantum Chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Radiation exposure poses a significant threat to human health. Emerging research indicates that even low-dose radiation once believed to be safe, may have harmful effects. This perception has spurred a growing interest in investigating the potential risks associated with low-dose radiation exposure across various scenarios. To comprehensively explore the health consequences of low-dose radiation, our study employs a robust statistical framework that examines whether specific groups of genes, belonging to known pathways, exhibit coordinated expression patterns that align with the radiation levels. Notably, our findings reveal the existence of intricate yet consistent signatures that reflect the molecular response to radiation exposure, distinguishing between low-dose and high-dose radiation. Moreover, we leverage a pathway-constrained variational autoencoder to capture the nonlinear interactions within gene expression data. By comparing these two analytical approaches, our study aims to gain valuable insights into the impact of low-dose radiation on gene expression patterns, identify pathways that are differentially affected, and harness the potential of machine learning to uncover hidden activity within biological networks. This comparative analysis contributes to a deeper understanding of the molecular consequences of low-dose radiation exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a09859ef65434cf10f4a79a9ac8f8014d32be9" target='_blank'>
              Pathway-based analyses of gene expression profiles at low doses of ionizing radiation
              </a>
            </td>
          <td>
            Xihaier Luo, Seyednami Niyakan, Patrick R. Johnstone, Sean McCorkle, Gilchan Park, Vanessa López-Marrero, Shinjae Yoo, Edward R. Dougherty, Xiaoning Qian, Francis J. Alexander, S. Jha, Byung-Jun Yoon
          </td>
          <td>2024-05-14</td>
          <td>Frontiers in Bioinformatics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="By accelerating time-consuming processes with high efficiency, computing has become an essential part of many modern chemical pipelines. Machine learning is a class of computing methods that can discover patterns within chemical data and utilize this knowledge for a wide variety of downstream tasks, such as property prediction or substance generation. The complex and diverse chemical space requires complex machine learning architectures with great learning power. Recently, learning models based on transformer architectures have revolutionized multiple domains of machine learning, including natural language processing and computer vision. Naturally, there have been ongoing endeavors in adopting these techniques to the chemical domain, resulting in a surge of publications within a short period. The diversity of chemical structures, use cases, and learning models necessitate a comprehensive summarization of existing works. In this paper, we review recent innovations in adapting transformers to solve learning problems in chemistry. Because chemical data is diverse and complex, we structure our discussion based on chemical representations. Specifically, we highlight the strengths and weaknesses of each representation, the current progress of adapting transformer architectures, and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b87d12995df53c3c70af414ab54a764866506152" target='_blank'>
              Application of Transformers in Cheminformatics
              </a>
            </td>
          <td>
            Kha-Dinh Luong, Ambuj Singh
          </td>
          <td>2024-05-30</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-cell datasets often lack individual cell labels, making it challenging to identify cells associated with disease. To address this, we introduce Mixture Modeling for Multiple Instance Learning (MMIL), an expectation maximization method that enables the training and calibration of cell-level classifiers using patient-level labels. Our approach can be used to train e.g. lasso logistic regression models, gradient boosted trees, and neural networks. When applied to clinically-annotated, primary patient samples in Acute Myeloid Leukemia (AML) and Acute Lymphoblastic Leukemia (ALL), our method accurately identifies cancer cells, generalizes across tissues and treatment timepoints, and selects biologically relevant features. In addition, MMIL is capable of incorporating cell labels into model training when they are known, providing a powerful framework for leveraging both labeled and unlabeled data simultaneously. Mixture Modeling for MIL offers a novel approach for cell classification, with significant potential to advance disease understanding and management, especially in scenarios with unknown gold-standard labels and high dimensionality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf614cea92ffef23dcd05fbea55a21a48d81f7c" target='_blank'>
              MMIL: A novel algorithm for disease associated cell type discovery.
              </a>
            </td>
          <td>
            Erin Craig, Timothy Keyes, J. Sarno, Maxim E. Zaslavsky, Garry Nolan, Kara Davis, Trevor Hastie, R. Tibshirani
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>153</td>
        </tr>

        <tr id="Machine learning (ML) is changing the world of computational protein design, with data- driven methods surpassing biophysical-based methods in experimental success rates. However, they are most often reported as case studies, lack integration and standardization across platforms, and are therefore hard to objectively compare. In this study, we established a streamlined and diverse toolbox for methods that predict amino acid probabilities inside the Rosetta software framework that allows for the side-by-side comparison of these models. Subsequently, existing protein fitness landscapes were used to benchmark novel self- supervised machine learning methods in realistic protein design settings. We focused on the traditional problems of protein sequence design: sampling and scoring. A major finding of our study is that novel ML approaches are better at purging the sampling space from deleterious mutations. Nevertheless, scoring resulting mutations without model fine-tuning showed no clear improvement over scoring with Rosetta. This study fills an important gap in the field and allows for the first time a comprehensive head-to-head comparison of different ML and biophysical methods. We conclude that ML currently acts as a complement to, rather than a replacement for, biophysical methods in protein design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d74eec4ceff71302ee342c63682798408eacb28" target='_blank'>
              Self-supervised machine learning methods for protein design improve sampling, but not the identification of high-fitness variants
              </a>
            </td>
          <td>
            Moritz Ertelt, Rocco Moretti, Jens Meiler, Clara T. Schoeder
          </td>
          <td>2024-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b393bee7cf5ce703bc767943b9da4324d84f453c" target='_blank'>
              CPMI: comprehensive neighborhood-based perturbed mutual information for identifying critical states of complex biological processes
              </a>
            </td>
          <td>
            Jing Ren, Peiluan Li, Jinling Yan
          </td>
          <td>2024-06-15</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction & Objective: High-risk single-nucleotide polymorphisms (SNPs) account for only ~20% of type 2 diabetes heritability, implying islet genes greatly influence one another to alter diabetes risk. Correlative analysis does not reveal conditional dependencies between key players regulating islet function whereas machine learning (ML) may do so.
 Methods: Using data from 374 genetically diverse mice, we derived models predicting islet function from protein abundance with gradient-boosted decision tree algorithms. Some mice (70%) were used to train the models and the rest were used to validate them. We also applied our models to similar data from other mouse studies and to data from humans to see if the models’ accuracies replicated. We then determined if the models included any known islet regulators, proteins with human orthologues possessing glycemia-related SNPs, and enriched for islet-relevant functional pathways. Finally, using the proteins’ predictive influences as meta-traits, we ran quantitative trait locus (QTL) scans to identify genomic regions altering the influence of proteins on islet function.
 Results: Our analysis revealed < 150 of the ~ 5000 detected proteins sufficiently modeled any one functional trait, with > 90% of a model’s prediction influenced by < 50 proteins. Some models predicted as well or better in the other mouse sets (R > 0.6) and reasonably well in human data (R > 0.5). ML analysis correctly identified the direction of effect for candidates (e.g. DPP8) where prior studies using correlation alone failed. Models for islet traits enriched for pathways potentially relevant for islet function and several unstudied proteins (e.g. CRELD2) present in multiple models have SNPs for glycemia-related traits. Finally, QTL scans revealed genomic regions that may alter the influence of key proteins on islet function and are far from those proteins’ genomic positions.
 Conclusions: ML can be used to identify conditional dependencies regulating islet function.


 C. Emfinger: None. E. Freiberger: Employee; AbbVie Inc. M. Rabaglia: None. J. Kolic: None. S. Simonett: Employee; Exact Sciences. M. Shortreed: None. L. Clark: None. D. Stapleton: None. K. Schueler: None. K. Mitok: None. T.R. Price: None. J.J. Coon: Research Support; Agilent, Eli Lilly and Company, Merck & Co., Inc. L. Smith: None. M.J. Merrins: None. J.D. Johnson: None. M. Keller: None. A.D. Attie: None.



 American Diabetes Association (7-21-PDF-157); National Institutes of Health (R01-DK101573-06); National Institutes of Health (GM070683); National Institutes of Health (P41-GM108538); National Institutes of Health (R35GM126914); National Institutes of Health (R35-GM118110); National Institutes of Health (T32-HL007936); National Institutes of Health (R01-DK113103); National Institutes of Health (R01DK127637); Canadian Institutes for Health Research (CIHR) operating grant 168857 CIHR Team Grant (ASD-179092/5-SRA-2021-1149-S-B); CIHR-JDRF Team (ASD-173663/5-SRA-2020-1059-S-B); CIHR Banting fellowship United States Department of Veterans Affairs Biomedical Laboratory Research and Development Service (I01BX005113)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae9b0cfbbd675841b7923b5ba1ab6fcfd03c60f2" target='_blank'>
              1483-P: Identifying Conditional Dependence of Proteins Regulating Islet Biology with Machine Learning
              </a>
            </td>
          <td>
            Christopher H Emfinger, Elyse C. Freiberger, Mary R. Rabaglia, Jelena Kolic, Shane P. Simonett, Michael Shortreed, Lauren E. Clark, Donnie Stapleton, Kathryn Schueler, Kelly A. Mitok, Tara R. Price, Joshua J. Coon, Lloyd Smith, M. Merrins, James D. Johnson, Mark Keller, A. Attie
          </td>
          <td>2024-06-14</td>
          <td>Diabetes</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In vitro cell biology experiments are routinely used to characterize cell migration properties under various experimental conditions. These experiments can be interpreted using lattice-based random walk models to provide insight into underlying biological mechanisms, and continuum limit partial differential equation (PDE) descriptions of the stochastic models can be used to efficiently explore model properties instead of relying on repeated stochastic simulations. Working with efficient PDE models is of high interest for parameter estimation algorithms that typically require a large number of forward model simulations. Quantitative data from cell biology experiments usually involves non-negative cell counts in different regions of the experimental images, and it is not obvious how to relate finite, noisy count data to the solutions of continuous PDE models that correspond to noise-free density profiles. In this work we illustrate how to develop and implement likelihood-based methods for parameter estimation, parameter identifiability and model prediction for lattice-based models describing collective migration with an arbitrary number of interacting subpopulations. We implement a standard additive Gaussian measurement error model as well as a new physically-motivated multinomial measurement error model that relates noisy count data with the solution of continuous PDE models. Both measurement error models lead to similar outcomes for parameter estimation and parameter identifiability, whereas the standard additive Gaussian measurement error model leads to non-physical prediction outcomes. In contrast, the new multinomial measurement error model involves a lower computational overhead for parameter estimation and identifiability analysis, as well as leading to physically meaningful model predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7501b082e09fd3e991e69561d882bcf1981f2a48" target='_blank'>
              Likelihood-based inference, identifiability and prediction using count data from lattice-based random walk models
              </a>
            </td>
          <td>
            Yihan Liu, D. Warne, Matthew J. Simpson
          </td>
          <td>2024-06-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/480567bb55e580af5f22e324481c693a5881a9b8" target='_blank'>
              Unveiling gene regulatory networks during cellular state transitions without linkage across time points
              </a>
            </td>
          <td>
            Ruosi Wan, Yuhao Zhang, Yongli Peng, Feng Tian, Ge Gao, Fuchou Tang, Jinzhu Jia, Hao Ge
          </td>
          <td>2024-05-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
Machine learning (ML) has been harnessed as a promising modelling tool for materials research. However, small data, or data scarcity, is a bottleneck when incorporating ML in studies involving experimentation. Current experiment planning methods show several disadvantages: one‐factor‐at‐a‐time (OFAT) experimentation became impractical due to limited laboratory resources; conventional design of experiments (DoE) failed to incorporate high‐dimensional features in ML; Surrogate‐based or Bayesian optimization (BO) shifted the goal to optimize material properties rather than guiding training data accumulation. The present research proposes leveraging active learning (AL) to strategically select critical data for experimentation. Two AL strategies, query‐by‐Committee (QBC) algorithm and Greedy method, are benchmarked against random query baseline on various materials datasets. AL is shown to efficiently reduce model prediction errors with minimal additional experiment data. Investigation of hyperparameters revealed benefits of applying AL at an early stage of experimental dataset construction. Moreover, AL is implemented and validated for an in‐house materials development task ‐ electrospray modelling. AL exploration as a paradigm is highlighted to guide experiment design for efficient data accumulation purposes, and its potential for further ML modelling. In doing so, the power of ML is expected to be fully unleashed to experimental researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41751ad3a09025a836bdcc2a49f5c274ea5e9855" target='_blank'>
              Tackling Data Scarcity Challenge through Active Learning in Materials Processing with Electrospray
              </a>
            </td>
          <td>
            Fanjin Wang, A. Harker, Mohan Edirisinghe, M. Parhizkar
          </td>
          <td>2024-05-23</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="We study the problem of network regression, where one is interested in how the topology of a network changes as a function of Euclidean covariates. We build upon recent developments in generalized regression models on metric spaces based on Fr\'echet means and propose a network regression method using the Wasserstein metric. We show that when representing graphs as multivariate Gaussian distributions, the network regression problem requires the computation of a Riemannian center of mass (i.e., Fr\'echet means). Fr\'echet means with non-negative weights translates into a barycenter problem and can be efficiently computed using fixed point iterations. Although the convergence guarantees of fixed-point iterations for the computation of Wasserstein affine averages remain an open problem, we provide evidence of convergence in a large number of synthetic and real-data scenarios. Extensive numerical results show that the proposed approach improves existing procedures by accurately accounting for graph size, topology, and sparsity in synthetic experiments. Additionally, real-world experiments using the proposed approach result in higher Coefficient of Determination ($R^{2}$) values and lower mean squared prediction error (MSPE), cementing improved prediction capabilities in practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa8448a82eadc3cf80edcd1ffef9d6d4063adc2" target='_blank'>
              An Optimal Transport Approach for Network Regression
              </a>
            </td>
          <td>
            Alex G. Zalles, Kai M. Hung, Ann E. Finneran, L. Beaudrot, C'esar A. Uribe
          </td>
          <td>2024-06-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Modelling gene expression is a central problem in systems biology. Single-cell technologies have revolutionised the field by enabling sequencing at the resolution of individual cells. This results in a much richer data compared to what is obtained by bulk technologies, offering new possibilities and challenges for gene regulatory network inference. Here we introduce GRIT — a method to fit a differential equation model and to infer gene regulatory networks from single-cell data using the theory of optimal transport. The idea consists in tracking the evolution of the cell distribution over time and finding the system that minimises the transport cost between consecutive time points.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0edd4d732cd966d7162a3543e7c8bfa54031593f" target='_blank'>
              Gene regulatory network inference from single-cell data using optimal transport
              </a>
            </td>
          <td>
            François Lamoline, Isabel Haasler, Johan Karlsson, Jorge Gonçalves, A. Aalto
          </td>
          <td>2024-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Time-series experiments are crucial for understanding the transient and dynamic nature of biological phenomena. These experiments, leveraging advanced classification and clustering algorithms, allow for a deep dive into the cellular processes. However, while these approaches effectively identify patterns and trends within data, they often need to improve in elucidating the causal mechanisms behind these changes. Building on this foundation, our study introduces a novel algorithm for temporal causal signaling modeling, integrating established knowledge networks with sequential gene expression data to elucidate signal transduction pathways over time. Focusing on Escherichia coli’s (E. coli) aerobic to anaerobic transition (AAT), this research marks a significant leap in understanding the organism’s metabolic shifts. By applying our algorithm to a comprehensive E. coli regulatory network and a time-series microarray dataset, we constructed the cross-time point core signaling and regulatory processes of E. coli’s AAT. Through gene expression analysis, we validated the primary regulatory interactions governing this process. We identified a novel regulatory scheme wherein environmentally responsive genes, soxR and oxyR, activate fur, modulating the nitrogen metabolism regulators fnr and nac. This regulatory cascade controls the stress regulators ompR and lrhA, ultimately affecting the cell motility gene flhD, unveiling a novel regulatory axis that elucidates the complex regulatory dynamics during the AAT process. Our approach, merging empirical data with prior knowledge, represents a significant advance in modeling cellular signaling processes, offering a deeper understanding of microbial physiology and its applications in biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03f1ae2c4960a846a7510e092a37140e059d408" target='_blank'>
              A Causal Regulation Modeling Algorithm for Temporal Events with Application to Escherichia coli’s Aerobic to Anaerobic Transition
              </a>
            </td>
          <td>
            Yigang Chen, Runbo Mao, Jiatong Xu, Yixian Huang, Jingyi Xu, Shidong Cui, Zihao Zhu, Xiang Ji, Shenghan Huang, Yanzhe Huang, Hsi-Yuan Huang, Shih-Chung Yen, Yang-Chi-Dung Lin, Hsien-Da Huang
          </td>
          <td>2024-05-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="In eukaryotic cells, DNA replication is organised both spatially and temporally, as evidenced by the stage-specific spatial distribution of replication foci in the nucleus. Despite the genetic association of aberrant DNA replication with numerous human diseases, the labour-intensive methods employed to study DNA replication have hindered large-scale analyses of its roles in pathological processes. In this study, we first demonstrate that a convolutional neural network trained to classify S-phase stages based on DAPI and EdU patterns could identify altered replication dynamics in Rif1-deficient mouse embryonic stem cells (mESCs), revealing a skewed distribution across the various S-phase stages. Given the possible practical limitations associated with a supervised framework, we proceed to show that the abnormal replication profile of Rif1-deficient mESCs could further be detected by an unsupervised approach (based on self-supervised representation learning), which could additionally reconstruct progression through S-phase. Finally, we extend our approach to a well-characterised cellular model of inducible deregulated origin firing, involving cyclin E overexpression. Through parallel EdU- and PCNA-based analyses, we demonstrate the potential applicability of our method to patient samples, offering a means to identify the contribution of deregulated DNA replication to a plethora of pathogenic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c624ffd1fffa3270b4fe27637084d11a9b502abd" target='_blank'>
              Supervised and unsupervised deep learning-based approaches for studying DNA replication spatiotemporal dynamics
              </a>
            </td>
          <td>
            Julian Ng-Kee-Kwong, Ben Philps, Fiona N. C. Smith, Aleksandra Sobieska, Naiming Chen, Constance Alabert, Hakan Bilen, S. B. Buonomo
          </td>
          <td>2024-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ischemic stroke, a significant threat to human life and health, refers to a class of conditions where brain tissue damage is induced following decreased cerebral blood flow. The incidence of ischemic stroke has been steadily increasing globally, and its disease mechanisms are highly complex and involve a multitude of biological mechanisms at various scales from genes all the way to the human body system that can affect the stroke onset, progression, treatment, and prognosis. To complement conventional experimental research methods, computational systems biology modeling can integrate and describe the pathogenic mechanisms of ischemic stroke across multiple biological scales and help identify emergent modulatory principles that drive disease progression and recovery. In addition, by running virtual experiments and trials in computers, these models can efficiently predict and evaluate outcomes of different treatment methods and thereby assist clinical decision-making. In this review, we summarize the current research and application of systems-level computational modeling in the field of ischemic stroke from the multiscale mechanism-based, physics-based and omics-based perspectives and discuss how modeling-driven research frameworks can deliver insights for future stroke research and drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dcd2e371d77918b7b6e34831a5f22fe63d490e" target='_blank'>
              Systems-level computational modeling in ischemic stroke: from cells to patients
              </a>
            </td>
          <td>
            Ge-li Li, Yanyong Zhao, Wen Ma, Yuan Gao, Chen Zhao
          </td>
          <td>2024-07-02</td>
          <td>Frontiers in Physiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="It is becoming clear that bulk gene expression measurements represent an average over very different cells. Elucidating the expression and abundance of each of the encompassed cells is key to disease understanding and precision medicine approaches. A first step in any such deconvolution is the inference of cell type abundances in the given mixture. Numerous approaches to cell-type deconvolution have been proposed, yet very few take advantage of the emerging discipline of deep learning and most approaches are limited to input data regarding the expression profiles of the cell types in question. Here we present DECODE, a deep learning method for the task that is data-driven and does not depend on input expression profiles. DECODE builds on a deep unfolded non-negative matrix factorization technique. It is shown to outperform previous approaches on a range of synthetic and real data sets, producing abundance estimates that are closer to and better correlated with the real values.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e613190af39a3bdbca89af001530f5a32a0af36c" target='_blank'>
              Deep Cell-Type Deconvolution from Bulk Gene Expression Data Using DECODE
              </a>
            </td>
          <td>
            Eran Hermush, R. Sharan
          </td>
          <td>2024-06-18</td>
          <td>Digital Medicine and Healthcare Technology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/152fe4e5299071fe03c972227009ecff75e0a855" target='_blank'>
              A guided network estimation approach using multi-omic information
              </a>
            </td>
          <td>
            G. Bartzis, C. Peeters, W. Ligterink, Fred A. van Eeuwijk
          </td>
          <td>2024-05-30</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="In various biological systems, analyzing how cell behaviors are coordinated over time would enable a deeper understanding of tissue-scale response to physiologic or superphysiologic stimuli. Such data is necessary for establishing both normal tissue function and the sequence of events after injury that lead to chronic disease. However, collecting and analyzing these large datasets presents a challenge—such systems are time-consuming to process, and the overwhelming scale of data makes it difficult to parse overall behaviors. This problem calls for an analysis technique that can quickly provide an overview of the groups present in the entire system and also produce meaningful categorization of cell behaviors. Here, we demonstrate the application of an unsupervised method—the Variational Autoencoder (VAE)—to learn the features of cells in cartilage tissue after impact-induced injury and identify meaningful clusters of chondrocyte behavior. This technique quickly generated new insights into the spatial distribution of specific cell behavior phenotypes and connected specific peracute calcium signaling timeseries with long term cellular outcomes, demonstrating the value of the VAE technique.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b92339847fe62e1bb5e9e8b25a050f9e1f85707" target='_blank'>
              Application of a variational autoencoder for clustering and analyzing in situ articular cartilage cellular response to mechanical stimuli
              </a>
            </td>
          <td>
            Jingyang Zheng, Han Kheng Teoh, M. Delco, L. Bonassar, I. Cohen
          </td>
          <td>2024-05-20</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Many problems in physical sciences are characterized by the prediction of space-time sequences. Such problems range from weather prediction to the analysis of disease propagation and video prediction. Modern techniques for the solution of these problems typically combine Convolution Neural Networks (CNN) architecture with a time prediction mechanism. However, oftentimes, such approaches underperform in the long-range propagation of information and lack explainability. In this work, we introduce a physically inspired architecture for the solution of such problems. Namely, we propose to augment CNNs with advection by designing a novel semi-Lagrangian push operator. We show that the proposed operator allows for the non-local transformation of information compared with standard convolutional kernels. We then complement it with Reaction and Diffusion neural components to form a network that mimics the Reaction-Advection-Diffusion equation, in high dimensions. We demonstrate the effectiveness of our network on a number of spatio-temporal datasets that show their merit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc41c2840d718cf94ea811d2a43c01d4849949a" target='_blank'>
              Advection Augmented Convolutional Neural Networks
              </a>
            </td>
          <td>
            N. Zakariaei, Siddharth Rout, Eldad Haber, Moshe Eliasof
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="G-protein coupled receptors (GPCRs) are transmembrane proteins that transmit signals from the extracellular environment to the inside of the cells. Their ability to adopt various conformational states, which influence their function, makes them crucial in pharmacoproteomic studies. While many drugs target specific GPCR states to exert their effects—thereby regulating the protein’s activity—unraveling the activation pathway remains challenging due to the multitude of intermediate transformations occurring throughout this process, and intrinsically influencing the dynamics of the receptors. In this context, computational modeling, particularly molecular dynamics (MD) simulations, may offer valuable insights into the dynamics and energetics of GPCR transformations, especially when combined with machine learning (ML) methods and techniques for achieving model interpretability for knowledge generation. The current study builds upon previous work in which the layer relevance propagation (LRP) technique was employed to interpret the predictions in a multi-class classification problem concerning the conformational states of the β2-adrenergic (β2AR) receptor from MD simulations. Here, we address the challenges posed by class imbalance and extend previous analyses by evaluating the robustness and stability of deep learning (DL)-based predictions under different imbalance mitigation techniques. By meticulously evaluating explainability and imbalance strategies, we aim to produce reliable and robust insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa30cc0ae2a6e6381d995fe687640fc3f469403" target='_blank'>
              A Study on the Robustness and Stability of Explainable Deep Learning in an Imbalanced Setting: The Exploration of the Conformational Space of G Protein-Coupled Receptors
              </a>
            </td>
          <td>
            M. A. Gutierrez-Mondragon, Alfredo Vellido, Caroline König
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Understanding the mechanisms underlying crystal formation is crucial. For most systems, crystallization typically goes through a nucleation process that involves dynamics that happen at short time and length scales. Due to this, molecular dynamics serves as a powerful tool to study this phenomenon. Existing approaches to study the mechanism often focus analysis on static snapshots of the global configuration, potentially overlooking subtle local fluctuations and history of the atoms involved in the formation of solid nuclei. To address this limitation, we propose a methodology that categorizes nucleation pathways into reactive pathways based on the time evolution of constituent atoms. Our approach effectively captures the diverse structural pathways explored by crystallizing Lennard-Jones-like particles and solidifying Ni$_3$Al, providing a more nuanced understanding of nucleating pathways. Moreover, our methodology enables the prediction of the resulting polymorph from each reactive trajectory. This deep learning-assisted comprehensive analysis offers an alternative view of crystal nucleation mechanisms and pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/734e1f4d145aa409813ced436949cacec2844770" target='_blank'>
              LeaPP: Learning Pathways to Polymorphs through machine learning analysis of atomic trajectories
              </a>
            </td>
          <td>
            Steven W Hall, Porhouy Minh, Sapna Sarupria
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The rise of single-cell RNA-sequencing (scRNA-seq) and evolved computational algorithms have significantly advanced biomedical science by revealing and visualizing the multifaceted and diverse nature of single cells. These technical advancements have also highlighted the pivotal role of cell clusters as representations of biologically universal entities such as cell types and cell states. However, to some extent, these clusterings remain dataset-specific and method-dependent. To improve comparability across different datasets or compositions, we previously introduced a graph-based representation of cell collections that captures the statistical dependencies of their characteristic genes. While our earlier work focused on theoretical insights, it was not sufficiently adapted and fine-tuned for practical implementation. To address this, the present paper introduces an improved practice to define and evaluate cellular identities based on our theory. First, we provide a concise summary of our previous theory and workflow. Then, point-by-point, we highlight the issues that needed fixing and propose solutions. The framework’s utility was enhanced by leveraging alternative formats of cellular features such as gene ontology (GO) terms and effectively handling dropouts. Supplemental techniques are offered to reinforce the versatility and robustness of our method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da477f921cb27ea88031ef50da4dab1e779172ca" target='_blank'>
              A graph-based practice of evaluating collective identities of cell clusters
              </a>
            </td>
          <td>
            Yuji Okano, Y. Kase, Hideyuki Okano
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite the advancements in learning governing differential equations from observations of dynamical systems, data-driven methods are often unaware of fundamental physical laws, such as frame invariance. As a result, these algorithms may search an unnecessarily large space and discover equations that are less accurate or overly complex. In this paper, we propose to leverage symmetry in automated equation discovery to compress the equation search space and improve the accuracy and simplicity of the learned equations. Specifically, we derive equivariance constraints from the time-independent symmetries of ODEs. Depending on the types of symmetries, we develop a pipeline for incorporating symmetry constraints into various equation discovery algorithms, including sparse regression and genetic programming. In experiments across a diverse range of dynamical systems, our approach demonstrates better robustness against noise and recovers governing equations with significantly higher probability than baselines without symmetry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/294f1e8ba8fdeee906321b73f3e14bd0b704a7e0" target='_blank'>
              Symmetry-Informed Governing Equation Discovery
              </a>
            </td>
          <td>
            Jianwei Yang, Wang Rao, Nima Dehmamy, R. Walters, Rose Yu
          </td>
          <td>2024-05-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Dynamic processes in complex networks are crucial for better understanding collective behavior in human societies, biological systems, and the internet. In this paper, we first focus on the continuous Markov-based modeling of evolving networks with the birth-death of individuals. A new individual arrives at the group by the Poisson process, while new links are established in the network through either uniform connection or preferential attachment. Moreover, an existing individual has a limited lifespan before leaving the network. We determine stationary topological properties of these networks, including their size and mean degree. To address the effect of the birth-death evolution, we further study the information dynamics in the proposed network model from the random drift and natural selection perspective, based on assumptions of total-stochastic and fitness-driven evolution, respectively. In simulations, we analyze the fixation probability of individual information and find that means of new connections affect the random drift process but do not affect the natural selection process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cc97d3b6a2007d5cad5a44d3da12a6a5c9c0d40" target='_blank'>
              Information Dynamics in Evolving Networks Based on the Birth-Death Process: Random Drift and Natural Selection Perspective
              </a>
            </td>
          <td>
            Minyu Feng, Ziyan Zeng, Qin Li, M. Perc, Jürgen Kurths
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>109</td>
        </tr>

        <tr id="Phylodynamics inference (PI) is a powerful approach for quantifying population dynamics and evolutionary trajectories of natural species based on phylogenetic trees. The emergence of single-cell lineage tracing technologies now enables the reconstruction of phylogenetic trees for thousands of individual cells within a multicellular organism, opening avenues for employing PI methodologies at the cellular level. However, the intricate process of cell differentiation poses challenges for directly applying current PI frameworks in somatic tissues. Here, we introduce a novel computational approach called single-cell phylodynamic explorer (scPhyloX), designed to model structured cell populations in various cell states, by leveraging single-cell phylogenetic trees to infer dynamics of tissue development and tumor evolution. Our comprehensive simulations demonstrate the high accuracy of scPhyloX across various biological scenarios. Application of scPhyloX to three real datasets of single-cell lineage tracing unveils novel insights into somatic dynamics, such as the overshoot of cycling stem cell populations in fly organ development, clonal expansion of multipotent progenitors of hematopoiesis during human aging, and pronounced subclonal selection in early colorectal tumorigenesis. Thus, scPhyloX is an innovative computational method for investigating the development and evolution of somatic tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eee3dcf68183af1b991832dcd1fd9e4bb3a0730f" target='_blank'>
              Single-cell phylodynamic inference of tissue development and tumor evolution with scPhyloX
              </a>
            </td>
          <td>
            Kun Wang, Zhaolian Lu, Zeqi Yao, Xionglei He, Zheng Hu, Da Zhou
          </td>
          <td>2024-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Deep learning with physics-informed neural networks (PINNs) has emerged as a highly popular and effective approach for solving partial differential equations(PDEs). In this paper, we first investigate the extrapolation capability of the PINN method for time-dependent PDEs. Taking advantage of this extrapolation property, we can generalize the training result obtained in the time subinterval to the large interval by adding a correction term to the network parameters of the subinterval. The correction term is determined by further training with the sample points in the added subinterval. Secondly, by designing an extrapolation control function with special characteristics and combining it with the correction term, we construct a new neural network architecture whose network parameters are coupled with the time variable, which we call the extrapolation-driven network architecture. Based on this architecture, using a single neural network, we can obtain the overall PINN solution of the whole domain with the following two characteristics: (1) it completely inherits the local solution of the interval obtained from the previous training, (2) at the interval node, it strictly maintains the continuity and smoothness that the true solution has. The extrapolation-driven network architecture allows us to divide a large time domain into multiple subintervals and solve the time-dependent PDEs one by one in chronological order. This training scheme respects the causality principle and effectively overcomes the difficulties of the conventional PINN method in solving the evolution equation on a large time domain. Numerical experiments verify the performance of our proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e945aa09792681fe44ec2d98a12595cf885296" target='_blank'>
              An extrapolation-driven network architecture for physics-informed deep learning
              </a>
            </td>
          <td>
            Yong Wang, Yanzhong Yao, Zhiming Gao
          </td>
          <td>2024-06-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="5065 Background: Overall survival (OS) remains the gold standard endpoint for clinical trials in metastatic prostate cancer (metPC) yet requires extensive follow-up. We sought to develop a model utilizing short term (≤ 4 months) prostate specific antigen (PSA) kinetic data to predict the OS readout from phase 3 clinical trials, with the goal of accelerating readout time of trials for patients (pts) with metPC. Methods: Clinical and PSA data were obtained from 7 metPC trials: 6 randomized, double-blind, phase 3 trials (TITAN, COU-AA-301, COU-AA-302, LATITUDE, ACIS, and MAGNITUDE), and one multicenter, phase II trial (GALAHAD). We developed 18 PSA kinetic variables from the first 4 months of enrollment including: 50% decline in PSA (PSA50), 90% decline in PSA, PSA=0.1, and PSA=0.2 within 1, 2, 3, and 4 months; and slope and base of the exponential fit of PSA over the first 4 months on study). To balance pt and disease characteristics, TITAN, COU-AA-301, and ACIS were used to train the intermediate endpoint, with remaining studies held out as an external testing cohort. A random 60/40% split of the pooled pt-level data was performed as a sensitivity analysis. Adaptive least absolute shrinkage and selection operator (aLASSO)-based Cox proportional hazards models were applied to select previously identified prognostic baseline clinical variables with and without PSA kinetic variables most predictive of OS on five-fold cross validation. Performance with and without PSA kinetics was assessed using C-index, integrated Brier score (IBS), and time-dependent area under the receiver operating characteristic curve (tAUC) at 12, 24, 36, and 48 months. Results: The study included 6,451 pts with median follow-up of 22 months and 85,785 PSA values. aLASSO selected Eastern Cooperative Oncology Group performance status, disease extent, lactate dehydrogenase, albumin, hemoglobin, PSA, and alkaline phosphatase as baseline variables for the prognostic comparator model. In the model including PSA kinetics, aLASSO identified PSA50 at 4 months, PSA0.1 at 1 month, and PSA slope as kinetics providing additional intra-treatment information to improve prediction of OS. The model with PSA kinetics showed improved performance on the test trials: C-index (0.72 vs. 0.66), IBS (0.158 vs. 0.173), and tAUC (0.84, 0.78, 0.78, and 0.76 vs. 0.78, 0.71, 0.69, and 0.65 at times 12, 24, 36, and 48 months, respectively; p < 0.05.) Sensitivity analysis using a random split of the pooled pt-level data supported the importance of PSA50 at 4 months and PSA slope. Conclusions: Data from 6,451 mPC pts enrolled on prospective clinical trials were used to develop a machine learning model which included kinetic PSA data and predicted the long-term OS readout in Phase 3 trials with just the first four months of data. This model will be validated in independent data sets comprised of other completed phase 3 trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dac296b786d21d2a4c231f4e0d52d3f32e79335" target='_blank'>
              Development of a machine learning model to predict overall survival results from randomized clinical trials of patients with metastatic prostate cancer.
              </a>
            </td>
          <td>
            A. Sabbagh, David A Quigley, Nicholas Lillis, Li Zhang, I. Friesner, A. Bailey, Rahul R Aggarwal, Meera Reddy Chappidi, Hari Singhal, Ke Zhang, J. Greshock, Jinhui Li, Margaret K. Yu, Eric J. Small, Julian C. Hong
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Developing and optimizing models for complex systems poses challenges due to the inherent complexity introduced by multiple types of input information and sources of uncertainty. In this study, we utilize Bayesian formalism to analytically examine the propagation of probability in the modeling process and propose quantitative assessments for it. Upon which, we develop a method for optimizing models for complex systems by (i) minimizing model uncertainty; (ii) maximizing model consistency; and (iii) minimizing model complexity, following the Bayesian Occam’s razor rationale. We showcase the benefits of this method by optimizing the modeling of the dynamic system of glucose-stimulated insulin secretion in pancreatic β-cells, leading to an optimized model that demonstrates better alignment with experimental observations compared to the non-optimized one. We anticipate that this method will facilitate the construction of accurate, precise, and sufficiently simple models for diverse complex systems. It is implemented in our open-source software Integrative Modeling Platform (IMP), ensuring its broad applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2852aa1c64c573614d3968195a3e4de98331119b" target='_blank'>
              Bayesian Occam’s Razor to Optimize Models for Complex Systems
              </a>
            </td>
          <td>
            Chenxi Wang, Jihui Zhao, Jingjing Zheng, Barak Raveh, Xuming He, Liping Sun
          </td>
          <td>2024-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In recent years, the integration of single-cell multi-omics data has provided a more comprehensive understanding of cell functions and internal regulatory mechanisms from a non-single omics perspective, but it still suffers many challenges, such as omics-variance, sparsity, cell heterogeneity, and confounding factors. As it is known, the cell cycle is regarded as a confounder when analyzing other factors in single-cell RNA-seq data, but it is not clear how it will work on the integrated single-cell multi-omics data. Here, a cell cycle-aware network (CCAN) is developed to remove cell cycle effects from the integrated single-cell multi-omics data while keeping the cell type-specific variations. This is the first computational model to study the cell-cycle effects in the integration of single-cell multi-omics data. Validations on several benchmark datasets show the outstanding performance of CCAN in a variety of downstream analyses and applications, including removing cell cycle effects and batch effects of scRNA-seq datasets from different protocols, integrating paired and unpaired scRNA-seq and scATAC-seq data, accurately transferring cell type labels from scRNA-seq to scATAC-seq data, and characterizing the differentiation process from hematopoietic stem cells to different lineages in the integration of differentiation data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/349bd78a45070086917086f9be8bf7faa3646057" target='_blank'>
              A Cell Cycle-Aware Network for Data Integration and Label Transferring of Single-Cell RNA-Seq and ATAC-Seq.
              </a>
            </td>
          <td>
            Jiajia Liu, Jian Ma, Jianguo Wen, Xiaobo Zhou
          </td>
          <td>2024-06-17</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Being able to infer the interactions between a set of species from observations of the system is of paramount importance to obtain explaining and predictive models in ecology. We tackled this challenge by employing qualitative modeling frameworks and logic methods for the synthesis of mathematical models that can integrate both observations and expert knowledge on the system. After devising a formal link between ecological networks and the causal structure of Boolean networks, we applied a generic model synthesis engine to infer Boolean models that are able to reproduce the observed dynamics of a protist community. Our inference method supports optimization criteria to derive most parsimonious and most precise models. It is also able to integrate prior knowledge on the ecological network, adding constraints on impossible interactions, which is necessary to obtain realistic predictions. Such constraints may however prove to be too strict, in which case our method is able to conclude on the absence of a model compatible with both the observations and the input hypotheses. We demonstrated our methodology on experimental data of a protist system, and showed its ability to recover essential and sufficient ecological interactions to explain the observed dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62558d01baf45fe3497544b482909271335ce476" target='_blank'>
              Inference of ecological networks and possibilistic dynamics based on Boolean networks from observations and prior knowledge
              </a>
            </td>
          <td>
            Loïc Paulevé, Cédric Gaucherel
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6155f5d70357f51adefc8adbbf3af90792e5537c" target='_blank'>
              SEEI: spherical evolution with feedback mechanism for identifying epistatic interactions
              </a>
            </td>
          <td>
            Deyu Tang, Yi-jun Mao, Jie Zhao, Jin Yang, Shiyin Li, Fu-xiang Ren, Junxi Zheng
          </td>
          <td>2024-05-13</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [16],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>