<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Latent%20space%20simulators/">
      
      
        <link rel="next" href="../PINNs/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Parametrizing using ML
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Time-series%20forecasting/" class="md-tabs__link">
        
  
    
  
  Time-series forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Symbolic%20regression/" class="md-tabs__link">
        
  
    
  
  Symbolic regression

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Neural%20ODEs/" class="md-tabs__link">
        
  
    
  
  Neural ODEs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Physics-based%20GNNs/" class="md-tabs__link">
        
  
    
  
  Physics-based GNNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Latent%20space%20simulators/" class="md-tabs__link">
        
  
    
  
  Latent space simulators

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Parametrizing using ML

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../PINNs/" class="md-tabs__link">
        
  
    
  
  PINNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Koopman%20operator/" class="md-tabs__link">
        
  
    
  
  Koopman operator

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Time-series%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Time-series forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Symbolic%20regression/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Symbolic regression
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Neural%20ODEs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Neural ODEs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Physics-based%20GNNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Physics-based GNNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Latent%20space%20simulators/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Latent space simulators
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Parametrizing using ML
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../PINNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    PINNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Koopman%20operator/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Koopman operator
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Parametrizing using ML</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-07-04 13:11:20 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Parametrizing using ML</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Parametrizing using ML</a><br>
      <a href="#recommended_articles">3. Recommended articles on Parametrizing using ML</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Parametrizing using ML</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Parametrizing using ML</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e837965494c4edbec4d30832d31ba5639996da8" target='_blank'>
                Deep learning prediction of patient response time course from early data via neural-pharmacokinetic/pharmacodynamic modelling
                </a>
              </td>
          <td>
            James Lu, B. Bender, Jin Y. Jin, Y. Guan
          </td>
          <td>2020-10-22</td>
          <td>Nature Machine Intelligence</td>
          <td>34</td>
          <td>38</td>

            <td><a href='../recommendations/4e837965494c4edbec4d30832d31ba5639996da8' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Many real-world systems such as social networks and moving planets are dynamic in nature, where a set of coupled objects are connected via the interaction graph and exhibit complex behavior along the time. For example, the COVID-19 pandemic can be considered as a dynamical system, where objects represent geographical locations (e.g., states) whose daily confirmed cases of infection evolve over time. Outbreak at one location may influence another location as people travel between these locations, forming a graph. Thus, how to model and predict the complex dynamics for these systems becomes a critical research problem. Existing work on modeling graph-structured data mostly assumes a static setting. How to handle dynamic graphs remains to be further explored. On one hand, features of objects change over time, influenced by the linked objects in the interaction graph. On the other hand, the graph itself can also evolve, where new interactions (links) may form and existing links may drop, which may in turn be affected by the dynamic features of objects. In this paper, we propose coupled graph ODE: a novel latent ordinary differential equation (ODE) generative model that learns the coupled dynamics of nodes and edges with a graph neural network (GNN) based ODE in a continuous manner. Our model consists of two coupled ODE functions for modeling the dynamics of edges and nodes based on their latent representations respectively. It employs a novel encoder parameterized by a GNN for inferring the initial states from historical data, which serves as the starting point of the predicted latent trajectories. Experiment results on the COVID-19 dataset and the simulated social network dataset demonstrate the effectiveness of our proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf2145f9998f304513c0c9b530ee9f7750c6f55" target='_blank'>
                Coupled Graph ODE for Learning Interacting System Dynamics
                </a>
              </td>
          <td>
            Zijie Huang, Yizhou Sun, Wei Wang
          </td>
          <td>2021-08-14</td>
          <td>Knowledge Discovery and Data Mining, Proceedings of the 27th ACM SIGKDD Conference on Knowledge Discovery & Data Mining</td>
          <td>50</td>
          <td>57</td>

            <td><a href='../recommendations/aaf2145f9998f304513c0c9b530ee9f7750c6f55' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd28606fbae3449f831248804264c9885e992f9" target='_blank'>
                CellBox: Interpretable Machine Learning for Perturbation Biology with Application to the Design of Cancer Combination Therapy.
                </a>
              </td>
          <td>
            Bo Yuan, Ciyue Shen, Augustin Luna, Anil Korkut, D. Marks, John Ingraham, C. Sander
          </td>
          <td>2020-12-21</td>
          <td>Cell systems</td>
          <td>70</td>
          <td>153</td>

            <td><a href='../recommendations/6bd28606fbae3449f831248804264c9885e992f9' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="We introduce a flexible, scalable Bayesian inference framework for nonlinear dynamical systems characterised by distinct and hierarchical variability at the individual, group, and population levels. Our model class is a generalisation of nonlinear mixed-effects (NLME) dynamical systems, the statistical workhorse for many experimental sciences. We cast parameter inference as stochastic optimisation of an end-to-end differentiable, block-conditional variational autoencoder. We specify the dynamics of the data-generating process as an ordinary differential equation (ODE) such that both the ODE and its solver are fully differentiable. This model class is highly flexible: the ODE right-hand sides can be a mixture of user-prescribed or “white-box” sub-components and neural network or “black-box” sub-components. Using stochastic optimisation, our amortised inference algorithm could seamlessly scale up to massive data collection pipelines (common in labs with robotic automation). Finally, our framework supports interpretability with respect to the underlying dynamics, as well as predictive generalization to unseen combinations of group components (also called “zero-shot” learning). We empirically validate our method by predicting the dynamic behaviour of bacteria that were genetically engineered to function as biosensors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309c5ae93a4cabfd37747abd130866240e265b2d" target='_blank'>
                Efficient Amortised Bayesian Inference for Hierarchical and Nonlinear Dynamical Systems
                </a>
              </td>
          <td>
            Geoffrey Roeder, Paul K. Grant, Andrew Phillips, Neil Dalchau, Edward Meeds
          </td>
          <td>2019-05-24</td>
          <td>ArXiv, International Conference on Machine Learning</td>
          <td>21</td>
          <td>28</td>

            <td><a href='../recommendations/309c5ae93a4cabfd37747abd130866240e265b2d' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Parametrizing using ML'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Parametrizing using ML</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="A variety of classical machine learning approaches have been developed over the past ten years with the aim to individualize drug dosages based on measured plasma concentrations. However, the interpretability of these models is challenging as they do not incorporate information on pharmacokinetic (PK) drug disposition. In this work we compare well-known population PK modelling with classical and a newly proposed scientific machine learning (SciML) framework, which combines knowledge on drug disposition with data-driven modelling. Our approach lets us estimate population PK parameters and their inter-individual variability (IIV) using multimodal covariate data of each patient. A dataset of 549 fluorouracil (5FU) plasma concentrations as example for an intravenously administered drug and a dataset of 308 sunitinib concentrations as example for an orally administered drug were used for analysis. Whereas classical machine learning models were not able to describe the data sufficiently, the proposed model allowed us to obtain highly accurate predictions even for new patients. Additionally, we demonstrated that our model could outperform traditional population PK models in terms of accuracy and greater flexibility when learning population parameters if given enough training data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01334b780295132c89212673bbf083ad5a36343e" target='_blank'>
              Scientific machine learning for predicting plasma concentrations in anti-cancer therapy
              </a>
            </td>
          <td>
            Diego Valderrama, O. Teplytska, Luca Marie, Koltermann, Elena Trunz, Eduard Schmulenson, Achim Fritsch, Ulrich Jaehde, Holger Fröhlich, Schloss Birlinghoven, Sankt Augustin, Germany, An der Immenburg, D. Bonn
          </td>
          <td>2024-05-08</td>
          <td>None</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de2e8855c3ac85fb287f6356a1f811bb41e6472" target='_blank'>
              Mechanism-based organization of neural networks to emulate systems biology and pharmacology models
              </a>
            </td>
          <td>
            John Mann, Hamed Meshkin, Joel Zirkle, Xiaomei Han, Bradlee Thrasher, Anik Chaturbedi, Ghazal Arabidarrehdor, Zhihua Li
          </td>
          <td>2024-05-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="From pedestrians to Kuramoto oscillators, interactions between agents govern how a multitude of dynamical systems evolve in space and time. Discovering how these agents relate to each other can improve our understanding of the often complex dynamics that underlie these systems. Recent works learn to categorize relationships between agents based on observations of their physical behavior. These approaches are limited in that the relationship categories are modelled as independent and mutually exclusive, when in real world systems categories are often interacting. In this work, we introduce a level of abstraction between the physical behavior of agents and the categories that define their behavior. To do this, we learn a mapping from the agents' states to their affinities for each category in a graph neural network. We integrate the physical proximity of agents and their affinities in a nonlinear opinion dynamics model which provides a mechanism to identify mutually exclusive categories, predict an agent's evolution in time, and control an agent's behavior. We demonstrate the utility of our model for learning interpretable categories for mechanical systems, and demonstrate its efficacy on several long-horizon trajectory prediction benchmarks where we consistently out perform existing methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667ae92834a3c09ccd46dc0cf9ad000cab20dfe8" target='_blank'>
              Relational Reasoning On Graphs Using Opinion Dynamics
              </a>
            </td>
          <td>
            Yulong Yang, Bowen Feng, Keqin Wang, Naomi Leonard, Adji B. Dieng, Christine Allen-Blanchette
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multi-factor screenings are commonly used in diverse applications in medicine and bioengineering, including optimizing combination drug treatments and microbiome engineering. Despite the advances in high-throughput technologies, large-scale experiments typically remain prohibitively expensive. Here we introduce a machine learning platform, structure-augmented regression (SAR), that exploits the intrinsic structure of each biological system to learn a high-accuracy model with minimal data requirement. Under different environmental perturbations, each biological system exhibits a unique, structured phenotypic response. This structure can be learned based on limited data and once learned, can constrain subsequent quantitative predictions. We demonstrate that SAR requires significantly fewer data comparing to other existing machine-learning methods to achieve a high prediction accuracy, first on simulated data, then on experimental data of various systems and input dimensions. We then show how a learned structure can guide effective design of new experiments. Our approach has implications for predictive control of biological systems and an integration of machine learning prediction and experimental design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c960a00bc91e7ef861762c26cbaf5fb088878ab0" target='_blank'>
              Data-driven learning of structure augments quantitative prediction of biological responses.
              </a>
            </td>
          <td>
            Yuanchi Ha, Helena R. Ma, Feilun Wu, Andrea Weiss, Katherine Duncker, Helen Xu, Jia Lu, Max Golovsky, Daniel Reker, Lingchong You
          </td>
          <td>2024-06-03</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c53a64b37da477c06468da0f1b359b77e2e273e9" target='_blank'>
              Biologically informed NeuralODEs for genome-wide regulatory dynamics
              </a>
            </td>
          <td>
            Intekhab Hossain, Viola Fanfani, Jonas Fischer, John Quackenbush, R. Burkholz
          </td>
          <td>2024-05-21</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Introduction Prediction of pharmacokinetic (PK) properties is crucial for drug discovery and development. Machine-learning (ML) models, which use statistical pattern recognition to learn correlations between input features (such as chemical structures) and target variables (such as PK parameters), are being increasingly used for this purpose. To embed ML models for PK prediction into workflows and to guide future development, a solid understanding of their applicability, advantages, limitations, and synergies with other approaches is necessary. Areas covered This narrative review discusses the design and application of ML models to predict PK parameters of small molecules, especially in light of established approaches including in vitro-in vivo extrapolation (IVIVE) and physiologically based pharmacokinetic (PBPK) models. The authors illustrate scenarios in which the three approaches are used and emphasize how they enhance and complement each other. In particular, they highlight achievements, the state of the art and potentials of applying machine learning for PK prediction through a comphrehensive literature review. Expert opinion ML models, when carefully crafted, regularly updated, and appropriately used, empower users to prioritize molecules with favorable PK properties. Informed practitioners can leverage these models to improve the efficiency of drug discovery and development process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4407f5485a20d2a2a26036c5e168a7a01ce78c14" target='_blank'>
              Another string to your bow: machine learning prediction of the pharmacokinetic properties of small molecules
              </a>
            </td>
          <td>
            Davide Bassani, Neil John Parrott, Nenad Manevski, Jitao David Zhang
          </td>
          <td>2024-05-10</td>
          <td>Expert Opinion on Drug Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Modeling complex physical dynamics is a fundamental task in science and engineering. Traditional physics-based models are first-principled, explainable, and sample-efficient. However, they often rely on strong modeling assumptions and expensive numerical integration, requiring significant computational resources and domain expertise. While deep learning (DL) provides efficient alternatives for modeling complex dynamics, they require a large amount of labeled training data. Furthermore, its predictions may disobey the governing physical laws and are difficult to interpret. Physics-guided DL aims to integrate first-principled physical knowledge into data-driven methods. It has the best of both worlds and is well equipped to better solve scientific problems. Recently, this field has gained great progress and has drawn considerable interest across discipline Here, we introduce the framework of physics-guided DL with a special emphasis on learning dynamical systems. We describe the learning pipeline and categorize state-of-the-art methods under this framework. We also offer our perspectives on the open challenges and emerging opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60d721e89c2f9c549241a4982b77f9c752b34460" target='_blank'>
              Learning dynamical systems from data: An introduction to physics-guided deep learning.
              </a>
            </td>
          <td>
            Rose Yu, Rui Wang
          </td>
          <td>2024-06-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Personalized cancer treatment requires a thorough understanding of complex interactions between drugs and cancer cell lines in varying genetic and molecular contexts. To address this, high-throughput screening has been used to generate large-scale drug response data, facilitating data-driven computational models. Such models can capture complex drug–cell line interactions across various contexts in a fully data-driven manner. However, accurately prioritizing the most effective drugs for each cell line still remains a significant challenge. To address this, we developed multiple neural ranking approaches that leverage large-scale drug response data across multiple cell lines from diverse cancer types. Unlike existing approaches that primarily utilize regression and classification techniques for drug response prediction, we formulated the objective of drug selection and prioritization as a drug ranking problem. In this work, we proposed multiple pairwise and listwise neural ranking methods that learn latent representations of drugs and cell lines and then use those representations to score drugs in each cell line via a learnable scoring function. Specifically, we developed neural pairwise and listwise ranking methods, Pair-PushC and List-One on top of the existing methods, pLETORg and ListNet, respectively. Additionally, we proposed a novel listwise ranking method, List-All, that focuses on all the effective drugs instead of the top effective drug, unlike List-One. We also provide an exhaustive empirical evaluation with state-of-the-art regression and ranking baselines on large-scale data sets across multiple experimental settings. Our results demonstrate that our proposed ranking methods mostly outperform the best baselines with significant improvements of as much as 25.6% in terms of selecting truly effective drugs within the top 20 predicted drugs (i.e., hit@20) across 50% test cell lines. Furthermore, our analyses suggest that the learned latent spaces from our proposed methods demonstrate informative clustering structures and capture relevant underlying biological features. Moreover, our comprehensive evaluation provides a thorough and objective comparison of the performance of different methods (including our proposed ones).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c0f5fb3d038f7182389a924d6a9c7f0a1db1816" target='_blank'>
              Improving Anticancer Drug Selection and Prioritization via Neural Learning to Rank
              </a>
            </td>
          <td>
            Vishal Dey, Xia Ning
          </td>
          <td>2024-05-13</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Machine learning (ML) has been applied to predict the efficacy of biologic agents in ulcerative colitis (UC). ML can offer precision, personalization, efficiency, and automation. Moreover, it can improve decision support in predicting clinical outcomes. However, it faces challenges related to data quality and quantity, overfitting, generalization, and interpretability. This paper comments on two recent ML models that predict the efficacy of vedolizumab and ustekinumab in UC. Models that consider multiple pathways, multiple ethnicities, and combinations of real-world and clinical trial data are required for optimal shared decision-making and precision medicine. This paper also highlights the potential of combining ML with computational models to enhance clinical outcomes and personalized healthcare. Key Insights: (1) ML offers precision, personalization, efficiency, and decision support for predicting the efficacy of biologic agents in UC. (2) Challenging aspects in ML prediction include data quality, overfitting, and interpretability. (3) Multiple pathways, multiple ethnicities, and combinations of real-world and clinical trial data should be considered in predictive models for optimal decision-making. (4) Combining ML with computational models may improve clinical outcomes and personalized healthcare.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fd22b7c6f685cae80958a1bc2dd8ae70573c792" target='_blank'>
              Machine Learning for Predicting Biologic Agent Efficacy in Ulcerative Colitis: An Analysis for Generalizability and Combination with Computational Models
              </a>
            </td>
          <td>
            Philippe Pinton
          </td>
          <td>2024-06-21</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Predicting cancer dynamics under treatment is challenging due to high inter-patient heterogeneity, lack of predictive biomarkers, and sparse and noisy longitudinal data. Mathematical models can summarize cancer dynamics by a few interpretable parameters per patient. Machine learning methods can then be trained to predict the model parameters from baseline covariates, but do not account for uncertainty in the parameter estimates. Instead, hierarchical Bayesian modeling can model the relationship between baseline covariates to longitudinal measurements via mechanistic parameters while accounting for uncertainty in every part of the model. The mapping from baseline covariates to model parameters can be modeled in several ways. A linear mapping simplifies inference but fails to capture nonlinear covariate effects and scale poorly for interaction modeling when the number of covariates is large. In contrast, Bayesian neural networks can potentially discover interactions between covariates automatically, but at a substantial cost in computational complexity. In this work, we develop a hierarchical Bayesian model of subpopulation dynamics that uses baseline covariate information to predict cancer dynamics under treatment, inspired by cancer dynamics in multiple myeloma (MM), where serum M protein is a well-known proxy of tumor burden. As a working example, we apply the model to a simulated dataset and compare its ability to predict M protein trajectories to a model with linear covariate effects. Our results show that the Bayesian neural network covariate effect model predicts cancer dynamics more accurately than a linear covariate effect model when covariate interactions are present. The framework can also be applied to other types of cancer or other time series prediction problems that can be described with a parametric model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ecd81405c7df09ef0fe2550029e319785f4a7b" target='_blank'>
              Prediction of cancer dynamics under treatment using Bayesian neural networks: A simulated study
              </a>
            </td>
          <td>
            E. M. Myklebust, A. Frigessi, Fredrik Schjesvold, J. Foo, K. Leder, Alvaro Kohn-Luque
          </td>
          <td>2024-05-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="SUMMARY
Drug response is conventionally measured at the cell level, often quantified by metrics like IC50. However, to gain a deeper understanding of drug response, cellular outcomes need to be understood in terms of pathway perturbation. This perspective leads us to recognize a challenge posed by the gap between two widely used large-scale databases, LINCS L1000 and GDSC, measuring drug response at different levels-L1000 captures information at the gene expression level, while GDSC operates at the cell line level. Our study aims to bridge this gap by integrating the two databases through transfer learning, focusing on condition-specific perturbations in gene interactions from L1000 to interpret drug response integrating both gene and cell levels in GDSC. This transfer learning strategy involves pretraining on the transcriptomic-level L1000 dataset, with parameter-frozen fine-tuning to cell line-level drug response. Our novel condition-specific gene-gene attention (CSG2A) mechanism dynamically learns gene interactions specific to input conditions, guided by both data and biological network priors. The CSG2A network, equipped with transfer learning strategy, achieves state-of-the-art performance in cell line-level drug response prediction. In two case studies, well-known mechanisms of drugs are well represented in both the learned gene-gene attention and the predicted transcriptomic profiles. This alignment supports the modeling power in terms of interpretability and biological relevance. Furthermore, our model's unique capacity to capture drug response in terms of both pathway perturbation and cell viability extends predictions to the patient level using TCGA data, demonstrating its expressive power obtained from both gene and cell levels.


AVAILABILITY AND IMPLEMENTATION
The source code for the CSG2A network is available at https://github.com/eugenebang/CSG2A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33a0e168c290638894183c35c3f97f05d25895b1" target='_blank'>
              Transfer learning of condition-specific perturbation in gene interactions improves drug response prediction.
              </a>
            </td>
          <td>
            D. Bang, Bonil Koo, Sun Kim
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In pharmacometrics, developing and selecting models is crucial for quantitatively assessing drug-biological interactions, treatment planning, and gaining insights into underlying processes. These validated models are essential for predictive analytics and strategic decision-making in drug development and clinical practice. Unlike traditional methods, machine learning (ML) offers a data-driven alternative to conventional, first-principle approaches. This paper presents an automatic method to derive unknown or uncertain (sub)models using longitudinal, heterogeneous data. Initially, we employ deep nonlinear mixed effect (DeepNLME) to train a neural network as a universal approximator, which then generates a parameterized representation of the underlying process. Subsequently, we apply symbolic regression to identify a set of potential models expressed as equations. Within the study parameters, the proposed method outperforms the baseline models and demonstrates the validity of both the DeepNLME approach and the symbolic regression approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e126fc4a7b791eeb4ca10543f627ff0e54a67c1" target='_blank'>
              Data-Driven Discovery of Feedback Mechanisms in Acute Myeloid Leukaemia: Alternatives to classical models using Deep Nonlinear Mixed Effect modeling and Symbolic Regression
              </a>
            </td>
          <td>
            Carl Julius Martensen, Niklas Korsbo, V. Ivaturi, Sebastian Sager
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Modeling dynamical systems is a fundamental task in scientific and engineering fields, often accomplished by applying theory-based models with mathematical equations. Yet, in cases where these equations cannot be established or parameterized properly, theory-based models are not applicable. Instead, a viable alternative is to learn the system dynamics directly from data, for example with deep learning models. However, traditional deep learning models often produce physically inconsistent results and struggle to generalize to unseen data, especially when training data is limited. One solution to this shortcoming is knowledge-guided deep learning, leveraging prior knowledge about the expected behavior of a dynamical system. In this work, we identify and formalize permissible system states, a novel type of prior knowledge that is often available for systems in the context of temporal dynamics modeling. This prior knowledge describes dynamic states that the system is allowed to take during its operation. We propose a knowledge-guided multi-state constraint to encode this type of prior knowledge through a loss function, making it applicable to any deep learning model. This approach allows to create an accurate data-driven model with minimal effort and data requirements. We validate the effectiveness of our method by applying it to model the temporal behavior of a gas turbine in response to an input control signal. Our results indicate that the proposed method reduces the prediction error by up to 40%. In addition to reducing the dependency on extensive training data, our method mitigates training randomness and enhances the consistency of predictions with the expected behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba124a5ffc81be0320f4553459c69561edd30f5" target='_blank'>
              Knowledge-Guided Learning of Temporal Dynamics and its Application to Gas Turbines
              </a>
            </td>
          <td>
            Pawel Bielski, Aleksandr Eismont, Jakob Bach, Florian Leiser, D. Kottonau, Klemens Böhm
          </td>
          <td>2024-06-04</td>
          <td>Proceedings of the 15th ACM International Conference on Future and Sustainable Energy Systems</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Combination therapy is an important direction of continuous exploration in the field of medicine, with the core goals of improving treatment efficacy, reducing adverse reactions, and optimizing clinical outcomes. Machine learning technology holds great promise in improving the prediction of drug synergy combinations. However, most studies focus on single disease-oriented collaborative predictive models or involve excessive feature categories, making it challenging to predict the majority of new drugs. To address these challenges, the DrugSK comprehensive model was developed, which utilizes SMILES-BERT to extract structural information from 3492 drugs and trains on reactions from 48,756 drug combinations. DrugSK is an integrated learning model capable of predicting interactions among various drug categories. First, the primary learner is trained from the initial data set. Random forest, support vector machine, and XGboost model are selected as primary learners and logistic regression as secondary learners. A new data set is then "generated" to train level 2 learners, which can be thought of as a prediction for each model. Finally, the results are filtered using logistic regression. Furthermore, the combination of the new antibacterial drug Drafloxacin with other antibacterial agents was tested. The synergistic effect of Drafloxacin and Isavuconazonium in the fight against Candida albicans has been confirmed, providing enlightenment for the clinical treatment of skin infection. DrugSK's prediction is accurate in practical application and can also predict the probability of the outcome. In addition, the tendency of Drafloxacin and antifungal drugs to be synergistic was found. The development of DrugSK will provide a new blueprint for predicting drug combination synergies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055325b217f8174f457e2c50697ff15bdc4f66ea" target='_blank'>
              DrugSK: A Stacked Ensemble Learning Framework for Predicting Drug Combinations of Multiple Diseases.
              </a>
            </td>
          <td>
            Siqi Chen, Nana Gao, Chunzhi Li, Fei Zhai, Xiwei Jiang, Peng Zhang, Jibin Guan, Kefeng Li, Rongwu Xiang, Guixia Ling
          </td>
          <td>2024-06-20</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Physics informed neural networks have been gaining popularity due to their unique ability to incorporate physics laws into data-driven models, ensuring that the predictions are not only consistent with empirical data but also align with domain-specific knowledge in the form of physics equations. The integration of physics principles enables the method to require less data while maintaining the robustness of deep learning in modeling complex dynamical systems. However, current PINN frameworks are not sufficiently mature for real-world ODE systems, especially those with extreme multi-scale behavior such as mosquito population dynamical modelling. In this research, we propose a PINN framework with several improvements for forward and inverse problems for ODE systems with a case study application in modelling the dynamics of mosquito populations. The framework tackles the gradient imbalance and stiff problems posed by mosquito ordinary differential equations. The method offers a simple but effective way to resolve the time causality issue in PINNs by gradually expanding the training time domain until it covers entire domain of interest. As part of a robust evaluation, we conduct experiments using simulated data to evaluate the effectiveness of the approach. Preliminary results indicate that physics-informed machine learning holds significant potential for advancing the study of ecological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf1c97b75409ff1ee6e5c4298dd05295c9b772df" target='_blank'>
              Adapting Physics-Informed Neural Networks To Optimize ODEs in Mosquito Population Dynamics
              </a>
            </td>
          <td>
            D. V. Cuong, Branislava Lali'c, Mina Petri'c, Binh Nguyen, M. Roantree
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In this article, a distributed neural network modeling framework including a novel neural hybrid system model is proposed for enhancing the scalability of neural network models in modeling dynamical systems. First, high-dimensional training data samples will be mapped to a low-dimensional feature space through the principal component analysis (PCA) featuring process. Following that, the feature space is bisected into multiple partitions based on the variation of the Shannon entropy under the maximum entropy (ME) bisecting process. The behavior of subsystems in the prespecified state space partitions will then be approximated using a group of shallow neural networks (SNNs) known as extreme learning machines (ELMs), and then it can further simplify the model by merging the redundant lattices based on their training error performance. The proposed modeling framework can handle high-dimensional dynamical system modeling problems with the advantages of reducing model complexity and improving model performance in training and verification. To demonstrate the effectiveness of the proposed modeling framework, examples of modeling the LASA dataset and an industrial robot are presented.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a2a2b97f2bbf0735400cc24119a8d6645252543" target='_blank'>
              A Distributed Neural Hybrid System Learning Framework in Modeling Complex Dynamical Systems.
              </a>
            </td>
          <td>
            Yejiang Yang, Tao Wang, Weiming Xiang
          </td>
          <td>2024-06-28</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The MRI-derived brain network serves as a pivotal instrument in elucidating both the structural and functional aspects of the brain, encompassing the ramifications of diseases and developmental processes. However, prevailing methodologies, often focusing on synchronous BOLD signals from functional MRI (fMRI), may not capture directional influences among brain regions and rarely tackle temporal functional dynamics. In this study, we first construct the brain-effective network via the dynamic causal model. Subsequently, we introduce an interpretable graph learning framework termed Spatio-Temporal Embedding ODE (STE-ODE). This framework incorporates specifically designed directed node embedding layers, aiming at capturing the dynamic interplay between structural and effective networks via an ordinary differential equation (ODE) model, which characterizes spatial-temporal brain dynamics. Our framework is validated on several clinical phenotype prediction tasks using two independent publicly available datasets (HCP and OASIS). The experimental results clearly demonstrate the advantages of our model compared to several state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/922be272d7098c93f25b6b7c27aad33f566c9494" target='_blank'>
              Interpretable Spatio-Temporal Embedding for Brain Structural-Effective Network with Ordinary Differential Equation
              </a>
            </td>
          <td>
            Haoteng Tang, Guodong Liu, Siyuan Dai, Kai Ye, Kun Zhao, Wenlu Wang, Carl Yang, Lifang He, Alex Leow, Paul M. Thompson, Heng Huang, Liang Zhan
          </td>
          <td>2024-05-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Motivation In recent years, applying computational modeling to systems biology has caused a substantial surge in both discovery and practical applications and a significant shift in our understanding of the complexity inherent in biological systems. Results In this perspective article, we briefly overview computational modeling in biology, highlighting recent advancements such as multi-scale modeling due to the omics revolution, single-cell technology, and integration of artificial intelligence and machine learning approaches. We also discuss the primary challenges faced: integration, standardization, model complexity, scalability, and interdisciplinary collaboration. Lastly, we highlight the contribution made by the Computational Modeling of Biological Systems (SysMod) Community of Special Interest (COSI) associated with the International Society of Computational Biology (ISCB) in driving progress within this rapidly evolving field through community engagement (via both in person and virtual meetings, social media interactions), webinars, and conferences. Availability and implementation Additional information about SysMod is available at https://sysmod.info.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b174dea12ab92a7f7139cb9122517aefde98aa6" target='_blank'>
              Perspectives on computational modeling of biological systems and the significance of the SysMod community
              </a>
            </td>
          <td>
            B. L. Puniya, Meghna Verma, Chiara Damiani, Shaimaa Bakr, Andreas Dräger
          </td>
          <td>2024-06-26</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Machine learning has been proposed as an alternative to theoretical modeling when dealing with complex problems in biological physics. However, in this perspective, we argue that a more successful approach is a proper combination of these two methodologies. We discuss how ideas coming from physical modeling neuronal processing led to early formulations of computational neural networks, e.g., Hopfield networks. We then show how modern learning approaches like Potts models, Boltzmann machines, and the transformer architecture are related to each other, specifically, through a shared energy representation. We summarize recent efforts to establish these connections and provide examples on how each of these formulations integrating physical modeling and machine learning have been successful in tackling recent problems in biomolecular structure, dynamics, function, evolution, and design. Instances include protein structure prediction; improvement in computational complexity and accuracy of molecular dynamics simulations; better inference of the effects of mutations in proteins leading to improved evolutionary modeling and finally how machine learning is revolutionizing protein engineering and design. Going beyond naturally existing protein sequences, a connection to protein design is discussed where synthetic sequences are able to fold to naturally occurring motifs driven by a model rooted in physical principles. We show that this model is "learnable" and propose its future use in the generation of unique sequences that can fold into a target structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/857c85e6a44f8c49c7d41155c041eb3e9361ca60" target='_blank'>
              Machine learning in biological physics: From biomolecular prediction to design.
              </a>
            </td>
          <td>
            Jonathan Martin, Marcos Lequerica Mateos, J. Onuchic, Ivan Coluzza, F. Morcos
          </td>
          <td>2024-06-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="Mammalian cells are commonly used as hosts in cell culture for biologics production in the pharmaceutical industry. Structured mechanistic models of metabolism have been used to capture complex cellular mechanisms that contribute to varying metabolic shifts in different cell lines. However, little research has focused on the impact of temporal changes in enzyme abundance and activity on the modeling of cell metabolism. In this work, we present a framework for constructing mechanistic models of metabolism that integrate growth-signaling control of enzyme activity and transcript dynamics. The proposed approach is applied to build models for three Chinese hamster ovary (CHO) cell lines using fed-batch culture data and time-series transcript profiles. Leveraging information from the transcriptome data, we develop a parameter estimation approach based on multi-cell-line (MCL) learning, which combines data sets from different cell lines and trains the individual cell-line models jointly to improve model accuracy. The computational results demonstrate the important role of growth signaling and transcript variability in metabolic models as well as the virtue of the MCL approach for constructing cell-line models with a limited amount of data. The resulting models exhibit a high level of accuracy in predicting distinct metabolic behaviors in the different cell lines; these models can potentially be used to accelerate the process and cell-line development for the biomanufacturing of new protein therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e85b0ee10ff62351e38806eafe8cf7db9748e226" target='_blank'>
              Multi-cell-line learning for the data-driven construction of mechanistic metabolic models.
              </a>
            </td>
          <td>
            Yen-An Lu, Meghan G. McCann, Wei-Shou Hu, Qi Zhang
          </td>
          <td>2024-06-03</td>
          <td>Biotechnology and bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hybrid dynamical systems are prevalent in science and engineering to express complex systems with continuous and discrete states. To learn the laws of systems, all previous methods for equation discovery in hybrid systems follow a two-stage paradigm, i.e. they first group time series into small cluster fragments and then discover equations in each fragment separately through methods in non-hybrid systems. Although effective, these methods do not fully take advantage of the commonalities in the shared dynamics of multiple fragments that are driven by the same equations. Besides, the two-stage paradigm breaks the interdependence between categorizing and representing dynamics that jointly form hybrid systems. In this paper, we reformulate the problem and propose an end-to-end learning framework, i.e. Amortized Equation Discovery (AMORE), to jointly categorize modes and discover equations characterizing the dynamics of each mode by all segments of the mode. Experiments on four hybrid and six non-hybrid systems show that our method outperforms previous methods on equation discovery, segmentation, and forecasting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f40610f6f31195a8a68f3802b6cc12d6503f9bae" target='_blank'>
              Amortized Equation Discovery in Hybrid Dynamical Systems
              </a>
            </td>
          <td>
            Yongtuo Liu, Sara Magliacane, Miltiadis Kofinas, E. Gavves
          </td>
          <td>2024-06-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Introduction: Heterogeneity between patients and within tumors poses challenges to effective cancer treatment. Drug combinations can overcome heterogeneity by increasing the probability that a patient will have a response to one or more drugs. This approach is successful in numerous disease settings including multiple lymphomas, but technological advances have led to a new approach to addressing heterogeneity: precision approaches that stratify patients into subpopulations with more uniform, stronger responses to specific treatments. These strategies can be implemented together to improve patient outcomes. We have developed computational mechanistic models of cancer treatment in the context of both sources of heterogeneity which explain and predict clinical outcomes of treatment of Large B-cell lymphoma (LBCL) with combination chemoimmunotherapy and CAR T-cell therapy.
 Combination therapy: Our novel population tumor kinetics (pop-TK) model simulates clinical combination therapy outcomes by implementing multidrug dose response functions in heterogeneous populations of tumor cells, within heterogeneous cohorts of patients. Informed by the outcomes of novel drugs in phase 2 trials in relapsed/refractory (r/r) patients, the pop-TK model accurately predicted of outcomes of seven first-line phase 3 trials of novel combinations. Most notably, informed by treatment of r/r LBCL with polatuzumab-vedotin (pola), the pop-TK predicted the success of modifying the standard treatment regimen (RCHOP) with pola before the clinical trial results were published. The pola combination improves overall survival (OS) and PFS in Activated B-Cell (ABC) LBCL (hazard ratio (HR): OS 0.3, PFS 0.4), but reduces OS (HR: 1.6) and does not improve PFS (HR: 1.0) in Germinal Center B-Cell (GCB) LBCL. The subtype-specific difference in PFS was also predictable based on subtype results of r/r trials (predicted HR: ABC 0.3, GCB 0.9).
 CAR T-cell therapy: We used a population pharmacokinetic/pharmacodynamic (pop-PK/PD) model of CAR T-cell therapy to reproduce the PFS distribution and the distribution CAR T-cell abundance for CAR T therapy in r/r LBCL patients. The impact of tumor size on patient outcomes in our model and in clinical studies led us to computationally explore the impact of treating smaller tumors with CAR T-cell therapy on treatment outcomes. We found that using detectable circulating tumor DNA (ctDNA) as a biomarker for administering CAR T-cell therapy immediately after RCHOP increased the proportion of simulated patients without a second disease progression event by 30% (HR: 0.43 [0.34, 0.55]).
 Conclusions: The pop-TK model supports LBCL cell-of-origin subtype as a biomarker for pola performance, and the pop-PK/PD model demonstrates that detectable ctDNA is a potential biomarker for early CAR T-cell administration. These examples of model-driven precision trial design have the potential to improve outcomes using readily available biomarkers and existing therapies.
 Citation Format: Amy E. Pomeroy, Adam C. Palmer. Mechanistic models of cancer heterogeneity explain and predict clinical outcomes of Large B-Cell Lymphoma (LBCL) treatment [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PO-005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d895a967e89449a96dfc98b6bea1d188560e4b" target='_blank'>
              Abstract PO-005: Mechanistic models of cancer heterogeneity explain and predict clinical outcomes of Large B-Cell Lymphoma (LBCL) treatment
              </a>
            </td>
          <td>
            Amy E. Pomeroy, Adam C Palmer
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Systems biology tackles the challenge of understanding the high complexity in the internal regulation of homeostasis in the human body through mathematical modelling. These models can aid in the discovery of disease mechanisms and potential drug targets. However, on one hand the development and validation of knowledge-based mechanistic models is time-consuming and does not scale well with increasing features in medical data. On the other hand, more data-driven approaches such as machine learning models require large volumes of data to produce generalizable models. The integration of neural networks and mechanistic models, forming universal differential equation (UDE) models, enables the automated learning of unknown model terms with less data than the neural network alone. Nevertheless, estimating parameters for these hybrid models remains difficult with sparse data and limited sampling durations that are common in biological applications. In this work, we propose the use of physiology-informed regularization, penalizing biologically implausible model behavior to guide the UDE towards more physiologically plausible regions of the solution space. In a simulation study we show that physiology-informed regularization not only results in a more accurate forecasting of model behaviour, but also supports training with less data. We also applied this technique to learn a representation of the rate of glucose appearance in the glucose minimal model using meal response data measured in healthy people. In that case, the inclusion of regularization reduces variability between UDE-embedded neural networks that were trained from different initial parameter guesses. Author summary Systems biology concerns the modelling and analysis of biological processes, by viewing these as interconnected systems. Modelling is typically done either using mechanistic differential equations that are derived from experiments and known biology, or using machine learning on large biological datasets. While mathematical modelling from biological experiments can provide useful insights with limited data, building and validating these models takes a long time and often requires highly invasive measurements in humans. Efforts to combine this classical technique with machine learning have resulted in a framework termed universal differential equations, where the model equations contain a neural network to describe unknown biological interactions. While these methods have shown success in numerous fields, applications in biology are more challenging due to limited data-availability, high data sparsity. In this work, we have introduced physiology-informed regularization to overcome these instabilities and to constrain the model to biologically plausible behavior. Our results show that by using physiology-informed regularization, we can accurately predict future unseen observations in a simulated example, with much more limited data than a similar model without regularization. Additionally, we show an application of this technique on human data, applying a neural network to learn the appearance of glucose in the blood plasma after a meal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf0dda21d3a8fbe599f98eed4fe943aa342f41" target='_blank'>
              Physiology-informed regularization enables training of universal differential equation systems for biological applications
              </a>
            </td>
          <td>
            Max de Rooij, Balázs Erdős, N. V. van Riel, Shauna D. O’Donovan
          </td>
          <td>2024-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Representation learning for tumor gene expression (GEx) data with deep neural networks is limited by the large gene feature space and the scarcity of available clinical and preclinical data. The translation of the learned representation between these data sources is further hindered by inherent molecular differences. To address these challenges, we propose GExMix (Gene Expression Mixup), a data augmentation method, which extends the Mixup concept to generate training samples accounting for the imbalance in both data classes and data sources. We leverage the GExMix-augmented training set in encoder-decoder models to learn a GEx latent representation. Subsequently, we combine the learned representation with drug chemical features in a dual-objective enhanced gene-centric drug response prediction, i.e., reconstruction of GEx latent embeddings and drug response classification. This dual-objective design strategically prioritizes gene-centric information to enhance the final drug response prediction. We demonstrate that augmenting training samples improves the GEx representation, benefiting the gene-centric drug response prediction model. Our findings underscore the effectiveness of our proposed GExMix in enriching GEx data for deep neural networks. Moreover, our proposed gene-centricity further improves drug response prediction when translating preclinical to clinical datasets. This highlights the untapped potential of the proposed framework for GEx data analysis, paving the way toward precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d8245b02f26c5698605a2d1bfdadb5793c2f155" target='_blank'>
              Enhancing Gene Expression Representation and Drug Response Prediction with Data Augmentation and Gene Emphasis
              </a>
            </td>
          <td>
            Diyuan Lu, Daksh P.S. Pamar, AJ Ohnmacht, G. Kutkaite, Michael P. Menden
          </td>
          <td>2024-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Based on multi-omics data and drug information, predicting the response of cancer cell lines to drugs is a crucial area of research in modern oncology, as it can promote the development of personalized treatments. Despite the promising performance achieved by existing models, most of them overlook the variations among different omics and lack effective integration of multi-omics data. Moreover, the explicit modeling of cell line/drug attribute and cell line-drug association has not been thoroughly investigated in existing approaches. To address these issues, we propose RedCDR, a dual relation distillation model for cancer drug response (CDR) prediction. Specifically, a parallel dual-branch architecture is designed to enable both the independent learning and interactive fusion feasible for cell line/drug attribute and cell line-drug association information. To facilitate the adaptive interacting integration of multi-omics data, the proposed multi-omics encoder introduces the multiple similarity relations between cell lines and takes the importance of different omics data into account. To accomplish knowledge transfer from the two independent attribute and association branches to their fusion, a dual relation distillation mechanism consisting of representation distillation and prediction distillation is presented. Experiments conducted on the GDSC and CCLE datasets show that RedCDR outperforms previous state-of-the-art approaches in CDR prediction. The source code is available at https://github.com/mhxu1998/RedCDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1d774b32aa1d353908c13e835eb7b0507d698d" target='_blank'>
              RedCDR: Dual Relation Distillation for Cancer Drug Response Prediction.
              </a>
            </td>
          <td>
            Muhao Xu, Zhenfeng Zhu, Yawei Zhao, Kunlun He, Qinghua Huang, Yao Zhao
          </td>
          <td>2024-05-22</td>
          <td>IEEE/ACM transactions on computational biology and bioinformatics</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="Pancreatic cancer is often diagnosed at an advanced stage when treatment options are limited. Being one of the deadliest cancers that mandates longer medication and treatment phases, there is an inevitable need to have the knowledge of drug response of anti-pancreatic cancer drugs before it is recommended for a patient. AI-driven drug response prediction has proven potential to personalize treatment strategies, improve therapeutic outcomes, and reduce adverse effects and treatment costs for cancer patients. In this research work, we have accounted for the use of different drug descriptors and their core structures known as scaffolds along with three cell line features, chromatin profiling, reverse phase protein array, and metabolomics data to build a feature engineered dataset for drug response prediction tested on various computational learning models. The 53 unique drugs against 18 unique pancreatic cancer cell lines were taken as the raw dataset. The initial dataset having a large dimension was feature selected using an ensemble method derived from five different techniques. The dataset was evaluated on various computational methods and an accuracy of 89% was achieved using the TabNet architecture. Furthermore, the common scaffolds that were persistently found among the drugs that possess high IC50-valued drug clusters were also recorded.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f23eae37f7bd87a4aefe75268ff15ffb52910e9" target='_blank'>
              Tailoring therapies: a frontier approach to pancreatic cancer with AI-driven multiomics profiling
              </a>
            </td>
          <td>
            Janiel Jawahar, Paramasivan Selvi Rajendran
          </td>
          <td>2024-08-01</td>
          <td>Indonesian Journal of Electrical Engineering and Computer Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multi-omic characterization and integration remains a challenge due to data complexity and sparsity. Addressing this, our study introduces an unsupervised deep learning model, MOVE (Multi-Omic Variational Encoder), specifically designed to integrate and augment the Cancer Dependency Map (DepMap). Harnessing orthogonal multi-omic information, this model successfully generates molecular and phenotypic profiles, resulting in an increase of 32.7% in the number of multi-omic profiles and thereby generating a complete DepMap for 1,523 cancer cell lines. The synthetically enhanced data increases statistical power, uncovering less studied mechanisms associated with drug resistance, and refines the identification of genetic associations and clustering of cancer cell lines. By applying SHAP for model interpretation, MOVE reveals multi-omic features essential for cell clustering and biomarker identification related to drug and gene dependencies. This understanding is crucial for the development of much-needed, effective strategies in prioritizing cancer targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5f98740294adae572cbbfc27a5b1f5503d6c2b3" target='_blank'>
              Synthetic augmentation of cancer cell line multi-omic datasets using unsupervised deep learning
              </a>
            </td>
          <td>
            Zhaoxiang Cai, Sofia Apolinário, A. R. Baião, Clare Pacini, , Susana Vinga, R. Reddel, Phillip J Robinson, Mathew J. Garnett, Qing Zhong, Emanuel Gonçalves
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction: Synergistic medication, a crucial therapeutic strategy in cancer treatment, involves combining multiple drugs to enhance therapeutic effectiveness and mitigate side effects. Current research predominantly employs deep learning models for extracting features from cell line and cancer drug structure data. However, these methods often overlook the intricate nonlinear relationships within the data, neglecting the distribution characteristics and weighted probability densities of gene expression data in multi-dimensional space. It also fails to fully exploit the structural information of cancer drugs and the potential interactions between drug molecules. Methods: To overcome these challenges, we introduce an innovative end-to-end learning model specifically tailored for cancer drugs, named Dual Kernel Density and Positional Encoding (DKPE) for Graph Synergy Representation Network (DKPEGraphSYN). This model is engineered to refine the prediction of drug combination synergy effects in cancer. DKPE-GraphSYN utilizes Dual Kernel Density Estimation and Positional Encoding techniques to effectively capture the weighted probability density and spatial distribution information of gene expression, while exploring the interactions and potential relationships between cancer drug molecules via a graph neural network. Results: Experimental results show that our prediction model achieves significant performance enhancements in forecasting drug synergy effects on a comprehensive cancer drug and cell line synergy dataset, achieving an AUPR of 0.969 and an AUC of 0.976. Discussion: These results confirm our model’s superior accuracy in predicting cancer drug combinations, providing a supportive method for clinical medication strategy in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282777dc9478ed17a298154b1e985c4aec15b5e9" target='_blank'>
              DKPE-GraphSYN: a drug synergy prediction model based on joint dual kernel density estimation and positional encoding for graph representation
              </a>
            </td>
          <td>
            Yunyun Dong, Yujie Bai, Haitao Liu, Ziting Yang, Yunqing Chang, Jianguang Li, Qixuan Han, Xiufang Feng, Xiaole Fan, Xiaoqiang Ren
          </td>
          <td>2024-06-14</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In oncology drug development, tumor dynamics modeling is widely applied to predict patients' overall survival (OS) via parametric models. However, the current modeling paradigm, which assumes a disease-specific link between tumor dynamics and survival, has its limitations. This is particularly evident in drug development scenarios where the clinical trial under consideration contains patients with tumor types for which there is little to no prior institutional data. In this work, we propose the use of a pan-indication solid tumor machine learning (ML) approach whereby all three tumor metrics (tumor shrinkage rate, tumor regrowth rate and time to tumor growth) are simultaneously used to predict patients' OS in a tumor type independent manner. We demonstrate the utility of this approach in a clinical trial of cancer patients treated with the tyrosine kinase inhibitor, pralsetinib. We compared the parametric and ML models and the results showed that the proposed ML approach is able to adequately predict patient OS across RET-altered solid tumors, including non-small cell lung cancer, medullary thyroid cancer as well as other solid tumors. While the findings of this study are promising, further research is needed for evaluating the generalizability of the ML model to other solid tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f96bc459032f6a187b68e4d09690f24f0e08e461" target='_blank'>
              Predicting overall survival from tumor dynamics metrics using parametric statistical and machine learning models: application to patients with RET-altered solid tumors
              </a>
            </td>
          <td>
            Erick Velasquez, Nastya Kassir, S. Cheeti, Denison Kuruvilla, Rucha Sane, Steve Dang, Dale Miles, James Lu
          </td>
          <td>2024-06-11</td>
          <td>Frontiers in Artificial Intelligence</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The mammalian genome organizes into complex three-dimensional structures, where interactions among chromatin regulatory elements play a pivotal role in mediating biological functions, highlighting the significance of genomic region interactions in biological research. Traditional biological sequencing techniques like HiC and MicroC, commonly employed to estimate these interactions, are resource-intensive and time-consuming, especially given the vast array of cell lines and tissues involved. With the advent of advanced machine learning (ML) methodologies, there has been a push towards developing ML models to predict genomic interactions. However, while these models excel in predicting interactions for cell lines similar to their training data, they often fail to generalize across distantly related cell lines or accurately predict interactions specific to certain cell lines. Identifying the potential oversight of excluding example genomic region interaction information from model inputs as a fundamental limitation, this paper introduces GRACHIP, a model rooted in graph neural network technology aiming to address this issue by incorporating detailed interaction information as a hint. Through extensive testing across various cell lines, GRACHIP not only demonstrates exceptional accuracy in predicting chromatin interaction intensity but showcases remarkable generalizability to cell lines not encountered during training. Consequently, GRACHIP emerges as a potent research tool, offering a viable alternative to conventional sequencing methods for analyzing the interactions and three-dimensional organization of mammalian genomes, thus alleviating the dependency on expensive and time-consuming biological sequencing techniques. It also offers an alternative way for researchers to investigate 3D chromatin interactions and simulate their changes in model systems to test their hypotheses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92aaa802a4f13bf6fc0e8fce14687506f2b37a22" target='_blank'>
              Improving Cell-type-specific 3D Genome Architectures Prediction Leveraging Graph Neural Networks
              </a>
            </td>
          <td>
            Ruoyun Wang, Weicheng Ma, Aryan Soltani Mohammadi, Saba Shahsavari, Soroush Vosoughi, Xiaofeng Wang
          </td>
          <td>2024-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Motivation Anti-cancer drug response prediction (DRP) using cancer cell lines plays a vital role in stratified medicine and drug discovery. Recently there has been a surge of new deep learning (DL) models for DRP that show promising performance improvements. However, different models use different input data modalities and neural network architectures making it hard to find the source of these improvements. Results We consider three DL DRP models with reported state-of-the-art performance and use genomics or transcriptomics profiles of cell lines and chemical profiles of drugs as input data. We also construct null hypothesis benchmarks that simply leverage patterns in truth values and thus do not use omics or chemical profiles. By experimenting with these DL models and comparing them with our null hypothesis benchmarks we show that for multiple testing types, much or all of the current reported performance is a property of the training truth values. Furthermore, we show that for cancer blind testing (testing on unseen cell lines) no performance comes from drug features, instead, performance is due to the transcriptomics cell line profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c5705a239471a8f4b914d01d1c42bcefb5e083" target='_blank'>
              Understanding the Sources of Performance in Deep Learning Drug Response Prediction Models
              </a>
            </td>
          <td>
            Nikhil Branson, Pedro R. Cutillas, Conrad Besseant
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The evaluation of drug-induced Torsades de pointes (TdP) risks is crucial in drug safety assessment. In this study, we discuss machine learning approaches in the prediction of drug-induced TdP risks using preclinical data. Specifically, a random forest model was trained on the dataset generated by the rabbit ventricular wedge assay. The model prediction performance was measured on 28 drugs from the Comprehensive In Vitro Proarrhythmia Assay initiative. Leave-one-drug-out cross-validation provided an unbiased estimation of model performance. Stratified bootstrap revealed the uncertainty in the asymptotic model prediction. Our study validated the utility of machine learning approaches in predicting drug-induced TdP risks from preclinical data. Our methods can be extended to other preclinical protocols and serve as a supplementary evaluation in drug safety assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aff64c32acf59cf216b66c0fc2cd3375c21ab4" target='_blank'>
              Drug safety assessment by machine learning models.
              </a>
            </td>
          <td>
            Nan Miles Xi, D. Huang
          </td>
          <td>2024-06-18</td>
          <td>Journal of biopharmaceutical statistics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d88d1ee244b5bbb95c70dc410ecf9f282eee608" target='_blank'>
              Prediction of anticancer drug sensitivity using an interpretable model guided by deep learning
              </a>
            </td>
          <td>
            Weixiong Pang, Ming Chen, Yufang Qin
          </td>
          <td>2024-05-09</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer, a leading cause of death globally, occurs due to genomic changes and manifests heterogeneously across patients. To advance research on personalized treatment strategies, the effectiveness of various drugs on cells derived from cancers (`cell lines') is experimentally determined in laboratory settings. Nevertheless, variations in the distribution of genomic data and drug responses between cell lines and humans arise due to biological and environmental differences. Moreover, while genomic profiles of many cancer patients are readily available, the scarcity of corresponding drug response data limits the ability to train machine learning models that can predict drug response in patients effectively. Recent cancer drug response prediction methods have largely followed the paradigm of unsupervised domain-invariant representation learning followed by a downstream drug response classification step. Introducing supervision in both stages is challenging due to heterogeneous patient response to drugs and limited drug response data. This paper addresses these challenges through a novel representation learning method in the first phase and weak supervision in the second. Experimental results on real patient data demonstrate the efficacy of our method (WISER) over state-of-the-art alternatives on predicting personalized drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28ee9b9ea8cc0a9f28f3c21fa707bc8f6007d90" target='_blank'>
              WISER: Weak supervISion and supErvised Representation learning to improve drug response prediction in cancer
              </a>
            </td>
          <td>
            Kumar Shubham, A. Jayagopal, Syed Mohammed Danish, AP Prathosh, V. Rajan
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Drug development is a lengthy process with a high failure rate. Increasingly, machine learning is utilized to facilitate the drug development processes. These models aim to enhance our understanding of drug characteristics, including their activity in biological contexts. However, a major challenge in drug response (DR) prediction is model interpretability as it aids in the validation of findings. This is important in biomedicine, where models need to be understandable in comparison with established knowledge of drug interactions with proteins. drGAT, a graph deep learning model, leverages a heterogeneous graph composed of relationships between proteins, cell lines, and drugs. drGAT is designed with two objectives: DR prediction as a binary sensitivity prediction and elucidation of drug mechanism from attention coefficients. drGAT has demonstrated superior performance over existing models, achieving 78% accuracy (and precision), and 76% F1 score for 269 DNA-damaging compounds of the NCI60 drug response dataset. To assess the model’s interpretability, we conducted a review of drug-gene co-occurrences in Pubmed abstracts in comparison to the top 5 genes with the highest attention coefficients for each drug. We also examined whether known relationships were retained in the model by inspecting the neighborhoods of topoisomerase-related drugs. For example, our model retained TOP1 as a highly weighted predictive feature for irinotecan and topotecan, in addition to other genes that could potentially be regulators of the drugs. Our method can be used to accurately predict sensitivity to drugs and may be useful in the identification of biomarkers relating to the treatment of cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23626c19de7aea3fe09bc589778f03d93292a717" target='_blank'>
              drGAT: Attention-Guided Gene Assessment of Drug Response Utilizing a Drug-Cell-Gene Heterogeneous Network
              </a>
            </td>
          <td>
            Yoshitaka Inoue, Hunmin Lee, Tianfan Fu, Augustin Luna
          </td>
          <td>2024-05-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Aim: Despite the advances in oncology and cancer treatment over the past decades, cancer remains one of the deadliest diseases. This study focuses on further understanding the complex nature of cancer by using mathematical tumor modeling to understand, capture as best as possible, and describe its complex dynamics under chemotherapy treatment. Materials and Methods: Focusing on autoregressive with exogenous inputs, i.e., ARX, and adaptive neuro-fuzzy inference system, i.e., ANFIS, models, this work investigates tumor growth dynamics under both single and combination anticancer agent chemotherapy treatments using chemotherapy treatment data on xenografted mice. Results: Four ARX and ANFIS models for tumor growth inhibition were developed, estimated, and evaluated, demonstrating a strong correlation with tumor weight data, with ANFIS models showing superior performance in handling the multi-agent tumor growth complexities. These findings suggest potential clinical applications of the ANFIS models through further testing. Both types of models were also tested for their prediction capabilities across different chemotherapy schedules, with accurate forecasting of tumor growth up to five days in advance. The use of adaptive prediction and sliding (moving) data window techniques allowed for continuous model updating, ensuring more robust predictive capabilities. However, long-term forecasting remains a challenge, with accuracy declining over longer prediction horizons. Conclusion: While ANFIS models showed greater reliability in predictions, the simplicity and rapid deployment of ARX models offer advantages in situations requiring immediate approximations. Future research with larger, more diverse datasets and by exploring varying model complexities is recommended to improve the models’ reliability and applicability in clinical decision-making, thereby aiding the development of personalized chemotherapy regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e4427fcccf3edcdd11ca0d2dc97f640c7df5d5" target='_blank'>
              A Comparative Analysis of ARX and ANFIS Models for Tumor Growth Prediction Under Single and Multi-agent Chemotherapy
              </a>
            </td>
          <td>
            S. Liliopoulos, G. Stavrakakis, Konstantinos S Dimas
          </td>
          <td>2024-05-31</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Introduction Mpox (formerly monkeypox) is an infectious disease that spreads mostly through direct contact with infected animals or people's blood, bodily fluids, or cutaneous or mucosal lesions. In light of the global outbreak that occurred in 2022–2023, in this paper, we analyzed global Mpox univariate time series data and provided a comprehensive analysis of disease outbreaks across the world, including the USA with Brazil and three continents: North America, South America, and Europe. The novelty of this study is that it delved into the Mpox time series data by implementing the data-driven methods and a mathematical model concurrently—an aspect not typically addressed in the existing literature. The study is also important because implementing these models concurrently improved our predictions' reliability for infectious diseases. Methods We proposed a traditional compartmental model and also implemented deep learning models (1D- convolutional neural network (CNN), long-short term memory (LSTM), bidirectional LSTM (BiLSTM), hybrid CNN-LSTM, and CNN-BiLSTM) as well as statistical time series models: autoregressive integrated moving average (ARIMA) and exponential smoothing on the Mpox data. We also employed the least squares method fitting to estimate the essential epidemiological parameters in the proposed deterministic model. Results The primary finding of the deterministic model is that vaccination rates can flatten the curve of infected dynamics and influence the basic reproduction number. Through the numerical simulations, we determined that increased vaccination among the susceptible human population is crucial to control disease transmission. Moreover, in case of an outbreak, our model showed the potential for epidemic control by adjusting the key epidemiological parameters, namely the baseline contact rate and the proportion of contacts within the human population. Next, we analyzed data-driven models that contribute to a comprehensive understanding of disease dynamics in different locations. Additionally, we trained models to provide short-term (eight-week) predictions across various geographical locations, and all eight models produced reliable results. Conclusion This study utilized a comprehensive framework to investigate univariate time series data to understand the dynamics of Mpox transmission. The prediction showed that Mpox is in its die-out situation as of July 29, 2023. Moreover, the deterministic model showed the importance of the Mpox vaccination in mitigating the Mpox transmission and highlighted the significance of effectively adjusting key epidemiological parameters during outbreaks, particularly the contact rate in high-risk groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302a4b8ee78aaa9f3ebf07b199bea13110d35c6e" target='_blank'>
              Exploring the dynamics of monkeypox transmission with data-driven methods and a deterministic model
              </a>
            </td>
          <td>
            Haridas K. Das
          </td>
          <td>2024-05-22</td>
          <td>Frontiers in Epidemiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dose-response prediction in cancer is an active application field in machine learning. Using large libraries of \textit{in-vitro} drug sensitivity screens, the goal is to develop accurate predictive models that can be used to guide experimental design or inform treatment decisions. Building on previous work that makes use of permutation invariant multi-output Gaussian Processes in the context of dose-response prediction for drug combinations, we develop a variational approximation to these models. The variational approximation enables a more scalable model that provides uncertainty quantification and naturally handles missing data. Furthermore, we propose using a deep generative model to encode the chemical space in a continuous manner, enabling prediction for new drugs and new combinations. We demonstrate the performance of our model in a simple setting using a high-throughput dataset and show that the model is able to efficiently borrow information across outputs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54716d9548a4c571e30130186ea85e7d8aec3712" target='_blank'>
              Permutation invariant multi-output Gaussian Processes for drug combination prediction in cancer
              </a>
            </td>
          <td>
            Leiv Rønneberg, V. Lalchand, Paul D. W. Kirk
          </td>
          <td>2024-06-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Targeted monotherapies for cancer often fail due to inherent or acquired drug resistance. By aiming at multiple targets simultaneously, drug combinations can produce synergistic interactions that increase drug effectiveness and reduce resistance. Computational models based on the integration of omics data have been used to identify synergistic combinations, but predicting drug synergy remains a challenge. Here, we introduce DIPx, an algorithm for personalized prediction of drug synergy based on biologically motivated tumor- and drug-specific pathway activation scores (PASs). We trained and validated DIPx in the AstraZeneca-Sanger (AZS) DREAM Challenge dataset using two separate test sets: Test Set 1 comprised the combinations already present in the training set, while Test Set 2 contained combinations absent from the training set, thus indicating the model’s ability to handle novel combinations. The Spearman correlation coefficients between predicted and observed drug synergy were 0.50 (95% CI: 0.47–0.53) in Test Set 1 and 0.26 (95% CI: 0.22–0.30) in Test Set 2, compared to 0.38 (95% CI: 0.34–0.42) and 0.18 (95% CI: 0.16–0.20), respectively, for the best performing method in the Challenge. We show evidence that higher synergy is associated with higher functional interaction between the drug targets, and this functional interaction information is captured by PAS. We illustrate the use of PAS to provide a potential biological explanation in terms of activated pathways that mediate the synergistic effects of combined drugs. In summary, DIPx can be a useful tool for personalized prediction of drug synergy and exploration of activated pathways related to the effects of combined drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/967bc228620e27010cd037753de4c0665790671f" target='_blank'>
              Pathway activation model for personalized prediction of drug synergy
              </a>
            </td>
          <td>
            Quang Thinh Trac, Yue Huang, Tom Erkers, P. Östling, Anna Bohlin, A. Österroos, Mattias Vesterlund, R. Jafari, I. Siavelis, Helena Bäckvall, Santeri Kiviluoto, L. Orre, Mattias Rantalainen, Janne Lehtiö, Sören Lehmann, Olli-P. Kallioniemi, Y. Pawitan, T. Vu
          </td>
          <td>2024-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Analyzing the motion of multiple biological agents, be it cells or individual animals, is pivotal for the understanding of complex collective behaviors. With the advent of advanced microscopy, detailed images of complex tissue formations involving multiple cell types have become more accessible in recent years. However, deciphering the underlying rules that govern cell movements is far from trivial. Here, we present a novel deep learning framework to estimate the underlying equations of motion from observed trajectories, a pivotal step in decoding such complex dynamics. Our framework integrates graph neural networks with neural differential equations, enabling effective prediction of two-body interactions based on the states of the interacting entities. We demonstrate the efficacy of our approach through two numerical experiments. First, we used a simulated data from a toy model to tune the hyperparameters. Based on the obtained hyperparameters, we then applied this approach to a more complex model that describes interacting cells of cellular slime molds. Our results show that the proposed method can accurately estimate the function of two-body interactions, thereby precisely replicating both individual and collective behaviors within these systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bbd03a34ea3ecbe6332287f4b5f0d43638d30f4" target='_blank'>
              Integrating GNN and Neural ODEs for Estimating Two-Body Interactions in Mixed-Species Collective Motion
              </a>
            </td>
          <td>
            Masahito Uwamichi, S. Schnyder, Tetsuya J. Kobayashi, Satoshi Sawai
          </td>
          <td>2024-05-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Modeling complex systems using standard neural ordinary differential equations (NODEs) often faces some essential challenges, including high computational costs and susceptibility to local optima. To address these challenges, we propose a simulation-free framework, called Fourier NODEs (FNODEs), that effectively trains NODEs by directly matching the target vector field based on Fourier analysis. Specifically, we employ the Fourier analysis to estimate temporal and potential high-order spatial gradients from noisy observational data. We then incorporate the estimated spatial gradients as additional inputs to a neural network. Furthermore, we utilize the estimated temporal gradient as the optimization objective for the output of the neural network. Later, the trained neural network generates more data points through an ODE solver without participating in the computational graph, facilitating more accurate estimations of gradients based on Fourier analysis. These two steps form a positive feedback loop, enabling accurate dynamics modeling in our framework. Consequently, our approach outperforms state-of-the-art methods in terms of training time, dynamics prediction, and robustness. Finally, we demonstrate the superior performance of our framework using a number of representative complex systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bedb3203006239c598b64a69777f9f9b9613ed" target='_blank'>
              From Fourier to Neural ODEs: Flow Matching for Modeling Complex Systems
              </a>
            </td>
          <td>
            Xin Li, Jingdong Zhang, Qunxi Zhu, Chengli Zhao, Xue Zhang, Xiaojun Duan, Wei Lin
          </td>
          <td>2024-05-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Virtual clinical trials (VCTs) are growing in popularity as a tool for quantitatively predicting heterogeneous treatment responses across a population. In the context of a VCT, a plausible patient is an instance of a mathematical model with parameter (or attribute) values chosen to reflect features of the disease and response to treatment for that particular patient. A number of techniques have been introduced to determine the set of model parametrizations to include in a virtual patient cohort. These methodologies generally start with a prior distribution for each model parameter and utilize some criteria to determine whether a parameter set sampled from the priors should be included or excluded from the plausible population. No standard technique exists, however, for generating these prior distributions and choosing the inclusion/exclusion criteria. In this work, we rigorously quantify the impact that VCT design choices have on VCT predictions. Rather than use real data and a complex mathematical model, a spatial model of radiotherapy is used to generate simulated patient data and the mathematical model used to describe the patient data is a two-parameter ordinary differential equations model. This controlled setup allows us to isolate the impact of both the prior distribution and the inclusion/exclusion criteria on both the heterogeneity of plausible populations and on predicted treatment response. We find that the prior distribution, rather than the inclusion/exclusion criteria, has a larger impact on the heterogeneity of the plausible population. Yet, the percent of treatment responders in the plausible population was more sensitive to the inclusion/exclusion criteria utilized. This foundational understanding of the role of virtual clinical trial design should help inform the development of future VCTs that use more complex models and real data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231e7bc7a40e4003493fc06a15b04cc9b5fdabe2" target='_blank'>
              Assessing the Role of Patient Generation Techniques in Virtual Clinical Trial Outcomes
              </a>
            </td>
          <td>
            J. L. Gevertz, J. R. Wares
          </td>
          <td>2024-06-12</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="INTRODUCTION
This review explores the transformative impact of machine learning (ML) on carcinogenicity prediction within drug development. It discusses the historical context and recent advancements, emphasizing the significance of ML methodologies in overcoming challenges related to data interpretation, ethical considerations, and regulatory acceptance.


AREAS COVERED
The review comprehensively examines the integration of ML, deep learning, and diverse artificial intelligence (AI) approaches in various aspects of drug development safety assessments. It explores applications ranging from early-phase compound screening to clinical trial optimization, highlighting the versatility of ML in enhancing predictive accuracy and efficiency.


EXPERT OPINION
Through the analysis of traditional approaches such as in vivo rodent bioassays and in vitro assays, the review underscores the limitations and resource intensity associated with these methods. It provides expert insights into how ML offers innovative solutions to address these challenges, revolutionizing safety assessments in drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61374a9fb6566a155d65ae679b0e9a760d4938b3" target='_blank'>
              Recent progress in machine learning approaches for predicting carcinogenicity in drug development.
              </a>
            </td>
          <td>
            Nguyen Quoc Khanh Le, Thi-Xuan Tran, Phung-Anh Nguyen, Trang-Thi Ho, Van-Nui Nguyen
          </td>
          <td>2024-05-14</td>
          <td>Expert opinion on drug metabolism & toxicology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor growth inhibition (TGI) modeling attempts to describe the time course changes in tumor size for patients undergoing cancer therapy. TGI models present several advantages over traditional exposure-response models that are based explicitly on clinical end points and have become a popular tool in the pharmacometrics community. Unfortunately, the data required to fit TGI models, namely longitudinal tumor measurements, are sparse or often not available in literature or publicly accessible databases. On the contrary, common end points such as progression-free survival (PFS) and objective response rate (ORR) are directly derived from longitudinal tumor measurements and are routinely published. To this end, a Bayesian generative model relating underlying tumor dynamics to summary PFS and ORR data is introduced to learn TGI model parameters using only published summary data. The parameterized model can describe the tumor dynamics, quantify treatment effect, and account for differences in the study population. The utility of this model is shown by applying it to several published studies, and learned parameters are combined to simulate an in silico trial of a novel combination therapy in a novel setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5334e76ce88316ffc777a0522ed6c120271486bb" target='_blank'>
              A novel Bayesian generative approach for estimating tumor dynamics from published studies.
              </a>
            </td>
          <td>
            Arya Pourzanjani, Saurabh Modi, Jamie Connarn, Xinwen Zhang, V. Upreti, Chih-Wei Lin, Khamir Mehta
          </td>
          <td>2024-05-22</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Serum creatinine in neonates follows complex dynamics due to maturation processes, most pronounced in the first few weeks of life. The development of a mechanism-based model describing complex dynamics requires high expertise in pharmacometric (PMX) modeling and substantial model development time. A recently published machine learning (ML) approach of low-dimensional neural ordinary differential equations (NODEs) is capable of modeling such data from newborns automatically. However, this efficient data-driven approach in itself does not result in a clinically interpretable model. In this work, an approach to deriving an interpretable model with reasonable PMX-type functions is presented. This "translation" was applied to derive a PMX model for serum creatinine in neonates considering maturation processes and covariates. The developed model was compared to a previously published mechanism-based PMX model whereas both models had similar mechanistic structures. The developed model was then utilized to simulate serum creatinine concentrations in the first few weeks of life considering different covariate values for gestational age and birth weight. The reference serum creatinine values derived from these simulations are consistent with observed serum creatinine values and previously published reference values. Thus, the presented NODE-based ML approach to model complex serum creatinine dynamics in newborns and derive interpretable, mathematical-statistical components similar to those in a conventional PMX model demonstrates a novel, viable approach to facilitate the modeling of complex dynamics in clinical settings and pediatric drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fae58343bde99a83efb245281ab69b262f4cd7f" target='_blank'>
              Applying Neural ODEs to Derive a Mechanism-Based Model for Characterizing Maturation-Related Serum Creatinine Dynamics in Preterm Newborns.
              </a>
            </td>
          <td>
            Dominic Stefan Bräm, Gilbert Koch, Karel Allegaert, J. van den Anker, M. Pfister
          </td>
          <td>2024-05-16</td>
          <td>Journal of clinical pharmacology</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f0f239ff184cb06fa871df53cfa4741c1a7fab" target='_blank'>
              Deep Kernel learning for reaction outcome prediction and optimization
              </a>
            </td>
          <td>
            Sukriti Singh, José Miguel Hernández-Lobato
          </td>
          <td>2024-06-14</td>
          <td>Communications Chemistry</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Classical compartmental models in epidemiology often struggle to accurately capture real-world dynamics due to their inability to address the inherent heterogeneity of populations. In this paper, we introduce a novel approach that incorporates heterogeneity through a mobility variable, transforming the traditional ODE system into a system of integro-differential equations that describe the dynamics of population densities across different compartments. Our results show that, for the same basic reproduction number, our mobility-based model predicts a smaller final pandemic size compared to classic compartmental models, whose population densities are represented as Dirac delta functions in our density-based framework. This addresses the overestimation issue common in many classical models. Additionally, we demonstrate that the time series of the infected population is sufficient to uniquely identify the mobility distribution. We reconstruct this distribution using a machine-learning-based framework, providing both theoretical and algorithmic support to effectively constrain the mobility-based model with real-world data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c03a14a48a310a736f8149b03d7cfacc5775bf6" target='_blank'>
              Modeling, Inference, and Prediction in Mobility-Based Compartmental Models for Epidemiology
              </a>
            </td>
          <td>
            Ning Jiang, Weiqi Chu, Yao Li
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In recent years, the escalating environmental challenges have contributed to a rising incidence of cancer. The precise anticipation of cancer incidence and mortality rates has emerged as a pivotal focus in scientific inquiry, exerting a profound impact on the formulation of public health policies. This investigation adopts a pioneering machine learning framework to address this critical issue, utilizing a dataset encompassing 72,591 comprehensive records that include essential variables such as age, case count, population size, race, gender, site, and year of diagnosis. Diverse machine learning algorithms, including decision trees, random forests, logistic regression, support vector machines, and neural networks, were employed in this study. The ensuing analysis revealed testing accuracies of 62.17%, 61.92%, 54.53%, 55.72%, and 62.30% for the respective models. This state-of-the-art model not only enhances our understanding of cancer dynamics but also equips researchers and policymakers with the capability of making meticulous projections concerning forthcoming cancer incidence and mortality rates. Considering sustainability, the application of this advanced machine learning framework emphasizes the importance of judiciously utilizing extensive and intricate databases. By doing so, it facilitates a more sustainable approach to healthcare planning, allowing for informed decision-making that takes into account the long-term ecological and societal impacts of cancer-related policies. This integrative perspective underscores the broader commitment to sustainable practices in both health research and public policy formulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4560de26fc63a5fa7d9e33f55b5a31acaa915143" target='_blank'>
              Machine Learning Model Construction and Testing: Anticipating Cancer Incidence and Mortality
              </a>
            </td>
          <td>
            Yuanzhao Ding
          </td>
          <td>2024-06-30</td>
          <td>Diseases</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbc8395840f2621fd3a1b982cf278edaa1f8cb" target='_blank'>
              A review of machine learning methods for cancer characterization from microbiome data
              </a>
            </td>
          <td>
            Marco Teixeira, Francisco Silva, R. Ferreira, Tânia Pereira, Ceu Figueiredo, Hélder P. Oliveira
          </td>
          <td>2024-05-30</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55d2566d48623ecfc68341b1562423b2d582647" target='_blank'>
              Deep learning-based approach for COVID-19 spread prediction
              </a>
            </td>
          <td>
            S. Cumbane, Gyözö Gidófalvi
          </td>
          <td>2024-06-10</td>
          <td>International Journal of Data Science and Analytics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The ever-expanding landscape of pharmaceutical compounds and medical treatments has underscored the critical need for an advanced system to predict drug-drug interactions (DDIs) using machine learning. This paper delves into the development of a novel predictive model that leverages diverse data sources, including chemical structures, pharmacological properties, and scientific literature, to comprehensively analyse and foresee potential DDIs. The current state of DDIs is rife with limitations in scope and accuracy, making the development of an advanced system not only a necessity for enhancing patient safety but also a boon to pharmaceutical research and regulatory oversight. The relationship between two medications in which one drug’s pharmacological actions are altered by another is known as a drug-drug interaction (DDI). Favourable drug-drug interactions (DDIs) generally contribute to improved therapeutic results for patients. On the contrary, adverse DDIs pose a significant risk, often resulting in undesirable drug reactions that can potentially lead to the withdrawal of a drug from the market and even patient fatalities. Consequently, the identification of DDIs is essential for advancing new drug development and effectively managing diseases. Keywords: Drug, Drug-drug interaction, Machine learning, Bioinformatics, Drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc31ce4df8b41262c53f949e913ee6c901766ec" target='_blank'>
              PREDICTING DRUG-DRUG INTERACTION USING MACHINE LEARNING
              </a>
            </td>
          <td>

          </td>
          <td>2024-05-29</td>
          <td>INTERANTIONAL JOURNAL OF SCIENTIFIC RESEARCH IN ENGINEERING AND MANAGEMENT</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A Marked Temporal Point Process (MTPP) is a stochastic process whose realization is a set of event-time data. MTPP is often used to understand complex dynamics of asynchronous temporal events such as money transaction, social media, healthcare, etc. Recent studies have utilized deep neural networks to capture complex temporal dependencies of events and generate embedding that aptly represent the observed events. While most previous studies focus on the inter-event dependencies and their representations, how individual events influence the overall dynamics over time has been under-explored. In this regime, we propose a Decoupled MTPP framework that disentangles characterization of a stochastic process into a set of evolving influences from different events. Our approach employs Neural Ordinary Differential Equations (Neural ODEs) to learn flexible continuous dynamics of these influences while simultaneously addressing multiple inference problems, such as density estimation and survival rate computation. We emphasize the significance of disentangling the influences by comparing our framework with state-of-the-art methods on real-life datasets, and provide analysis on the model behavior for potential applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01528db95c399e0822e827b108710fdf76fd5f21" target='_blank'>
              Decoupled Marked Temporal Point Process using Neural Ordinary Differential Equations
              </a>
            </td>
          <td>
            Yujee Song, Donghyun Lee, Rui Meng, Won Hwa Kim
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mathematical models are indispensable to the system biology toolkit for studying the structure and behavior of intracellular signaling networks. A common approach to modeling is to develop a system of equations that encode the known biology using approximations and simplifying assumptions. As a result, the same signaling pathway can be represented by multiple models, each with its set of underlying assumptions, which opens up challenges for model selection and decreases certainty in model predictions. Here, we use Bayesian multimodel inference to develop a framework to increase certainty in systems biology models. Using models of the extracellular regulated kinase (ERK) pathway, we first show that multimodel inference increases predictive certainty and yields predictors that are robust to changes in the set of available models. We then show that predictions made with multi-model inference are robust to data uncertainties introduced by decreasing the measurement duration and reducing the sample size. Finally, we use multimodel inference to identify a new model to explain experimentally measured sub-cellular location-specific ERK activity dynamics. In summary, our framework highlights multimodel inference as a disciplined approach to increasing the certainty of intracellular signaling activity predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e6724b6c05e22b755836f73b25e0ce04843d4" target='_blank'>
              Increasing certainty in systems biology models using Bayesian multimodel inference
              </a>
            </td>
          <td>
            Nathaniel Linden-Santangeli, Jin Zhang, Boris Kramer, P. Rangamani
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Machine-learning (ML) and deep-learning (DL) approaches to predict the molecular properties of small molecules are increasingly deployed within the design-make-test-analyze (DMTA) drug design cycle to predict molecular properties of interest. Despite this uptake, there are only a few automated packages to aid their development and deployment that also support uncertainty estimation, model explainability, and other key aspects of model usage. This represents a key unmet need within the field, and the large number of molecular representations and algorithms (and associated parameters) means it is nontrivial to robustly optimize, evaluate, reproduce, and deploy models. Here, we present QSARtuna, a molecule property prediction modeling pipeline, written in Python and utilizing the Optuna, Scikit-learn, RDKit, and ChemProp packages, which enables the efficient and automated comparison between molecular representations and machine learning models. The platform was developed by considering the increasingly important aspect of model uncertainty quantification and explainability by design. We provide details for our framework and provide illustrative examples to demonstrate the capability of the software when applied to simple molecular property, reaction/reactivity prediction, and DNA encoded library enrichment classification. We hope that the release of QSARtuna will further spur innovation in automatic ML modeling and provide a platform for education of best practices in molecular property modeling. The code for the QSARtuna framework is made freely available via GitHub.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d79165e7710fcacddea5e1ab595a6e3520735bc2" target='_blank'>
              QSARtuna: An Automated QSAR Modeling Platform for Molecular Property Prediction in Drug Design.
              </a>
            </td>
          <td>
            Lewis Mervin, Alexey Voronov, Mikhail Kabeshov, O. Engkvist
          </td>
          <td>2024-07-01</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In the complex landscape of hematologic samples such as peripheral blood or bone marrow derived from flow cytometry (FC) data, cell-level prediction presents profound challenges. This work explores injecting hierarchical prior knowledge into graph neural networks (GNNs) for single-cell multi-class classification of tabular cellular data. By representing the data as graphs and encoding hierarchical relationships between classes, we propose our hierarchical plug-in method to be applied to several GNN models, namely, FCHC-GNN, and effectively designed to capture neighborhood information crucial for single-cell FC domain. Extensive experiments on our cohort of 19 distinct patients, demonstrate that incorporating hierarchical biological constraints boosts performance significantly across multiple metrics compared to baseline GNNs without such priors. The proposed approach highlights the importance of structured inductive biases for gaining improved generalization in complex biological prediction tasks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e0fdaea8a065eda5545038bbdb830b51b0c573" target='_blank'>
              Injecting Hierarchical Biological Priors into Graph Neural Networks for Flow Cytometry Prediction
              </a>
            </td>
          <td>
            Fatemeh Nassajian Mojarrad, Lorenzo Bini, Thomas Matthes, Stéphane Marchand-Maillet
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MOTIVATION
The rise of single-cell RNA sequencing (scRNA-seq) technology presents new opportunities for constructing detailed cell type-specific gene regulatory networks (GRNs) to study cell heterogeneity. However, challenges caused by noises, technical errors, and dropout phenomena in scRNA-seq data pose significant obstacles to GRN inference, making the design of accurate GRN inference algorithms still essential. The recent growth of both single-cell and spatial transcriptomic sequencing data enables the development of supervised deep learning methods to infer GRNs on these diverse single-cell datasets.


RESULTS
In this study, we introduce a novel deep learning framework based on shared factor neighborhood and integrated neural network (SFINN) for inferring potential interactions and causalities between transcription factors and target genes from single-cell and spatial transcriptomic data. SFINN utilizes shared factor neighborhood to construct cellular neighborhood network based on gene expression data and additionally integrates cellular network generated from spatial location information. Subsequently, the cell adjacency matrix and gene pair expression are fed into an integrated neural network framework consisting of a graph convolutional neural network and a fully-connected neural network to determine whether the genes interact. Performance evaluation in the tasks of gene interaction and causality prediction against the existing GRN reconstruction algorithms demonstrates the usability and competitiveness of SFINN across different kinds of data. SFINN can be applied to infer gene regulatory networks from conventional single-cell sequencing data and spatial transcriptomic data.


AVAILABILITY
SFINN can be accessed at GitHub: https://github.com/JGuan-lab/SFINN.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52724caebdd9eb86a3aab7e26584a02fd43a37c9" target='_blank'>
              SFINN: inferring gene regulatory network from single-cell and spatial transcriptomic data with shared factor neighborhood and integrated neural network.
              </a>
            </td>
          <td>
            Yongjie Wang, Fengfan Zhou, Jinting Guan
          </td>
          <td>2024-07-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Predicting drug efficacy and safety in vivo requires information on biological responses (e.g., cell morphology and gene expression) to small molecule perturbations. However, current molecular representation learning methods do not provide a comprehensive view of cell states under these perturbations and struggle to remove noise, hindering model generalization. We introduce the Information Alignment (InfoAlign) approach to learn molecular representations through the information bottleneck method in cells. We integrate molecules and cellular response data as nodes into a context graph, connecting them with weighted edges based on chemical, biological, and computational criteria. For each molecule in a training batch, InfoAlign optimizes the encoder's latent representation with a minimality objective to discard redundant structural information. A sufficiency objective decodes the representation to align with different feature spaces from the molecule's neighborhood in the context graph. We demonstrate that the proposed sufficiency objective for alignment is tighter than existing encoder-based contrastive methods. Empirically, we validate representations from InfoAlign in two downstream tasks: molecular property prediction against up to 19 baseline methods across four datasets, plus zero-shot molecule-morphology matching.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db33b94c598876b05b7af83901a3a69d60eb47d" target='_blank'>
              Learning Molecular Representation in a Cell
              </a>
            </td>
          <td>
            Gang Liu, Srijit Seal, John Arevalo, Zhenwen Liang, A. Carpenter, Meng Jiang, Shantanu Singh
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Parameter estimation is one of the central problems in computational modeling of biological systems. Typically, scientists must fully specify the mathematical structure of the model, often expressed as a system of ordinary differential equations, to estimate the parameters. This process poses significant challenges due to the necessity for a detailed understanding of the underlying biological mechanisms. In this paper, we present an approach for estimating model parameters and assessing their identifiability in situations where only partial knowledge of the system structure is available. The partially known model is extended into a system of Hybrid Neural Ordinary Differential Equations, which captures the unknown portions of the system using neural networks. Integrating neural networks into the model structure introduces two primary challenges for parameter estimation: the need to globally explore the search space while employing gradient-based optimization, and the assessment of parameter identifiability, which may be hindered by the expressive nature of neural networks. To overcome the first issue, we treat biological parameters as hyperparameters in the extended model, exploring the parameter search space during hyperparameter tuning. The second issue is then addressed by an a posteriori analysis of parameter identifiability, computed by introducing a variant of a well-established approach for mechanistic models. These two components are integrated into an end-to-end pipeline that is thoroughly described in the paper. We assess the effectiveness of the proposed workflow on test cases derived from three different benchmark models. These test cases have been designed to mimic real-world conditions, including the presence of noise in the training data and various levels of data availability for the system variables. Author summary Parameter estimation is a central challenge in modeling biological systems. Typically, scientists calibrate the parameters by aligning model predictions with measured data once the model structure is defined. Our paper introduces a workflow that leverages the integration between mechanistic modeling and machine learning to estimate model parameters when the model structure is not fully known. We focus mainly on analyzing the identifiability of the model parameters, which measures how confident we can be in the parameter estimates given the available experimental data and partial mechanistic understanding of the system. We assessed the effectiveness of our approach in various in silico scenarios. Our workflow represents a first step to adapting traditional methods used in fully mechanistic models to the scenario of hybrid modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a064c473effb10441bd8fe1c6bde24ed0162e1e3" target='_blank'>
              Robust parameter estimation and identifiability analysis with Hybrid Neural Ordinary Differential Equations in Computational Biology
              </a>
            </td>
          <td>
            Stefano Giampiccolo, Federico Reali, Anna Fochesato, Giovanni Iacca, Luca Marchetti
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Artificial intelligence (AI) is increasingly used in every stage of drug development. Continuing breakthroughs in AI-based methods for drug discovery require the creation, improvement, and refinement of drug discovery data. We posit a new data challenge that slows the advancement of drug discovery AI: datasets are often collected independently from each other, often with little overlap, creating data sparsity. Data sparsity makes data curation difficult for researchers looking to answer key research questions requiring values posed across multiple datasets. We propose a novel diffusion GNN model Syngand capable of generating ligand and pharmacokinetic data end-to-end. We show and provide a methodology for sampling pharmacokinetic data for existing ligands using our Syngand model. We show the initial promising results on the efficacy of the Syngand-generated synthetic target property data on downstream regression tasks with AqSolDB, LD50, and hERG central. Using our proposed model and methodology, researchers can easily generate synthetic ligand data to help them explore research questions that require data spanning multiple datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309b0837f1ce764a9066e2d9555ed9883956e6f0" target='_blank'>
              Synthetic Data from Diffusion Models Improve Drug Discovery Prediction
              </a>
            </td>
          <td>
            Bing Hu, Ashish Saragadam, Anita Layton, Helen Chen
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dynamic Positron Emission Tomography (dPET) imaging and Time-Activity Curve (TAC) analyses are essential for understanding and quantifying the biodistribution of radiopharmaceuticals over time and space. Traditional compartmental modeling, while foundational, commonly struggles to fully capture the complexities of biological systems, including non-linear dynamics and variability. This study introduces an innovative data-driven neural network-based framework, inspired by Reaction Diffusion systems, designed to address these limitations. Our approach, which adaptively fits TACs from dPET, enables the direct calibration of diffusion coefficients and reaction terms from observed data, offering significant improvements in predictive accuracy and robustness over traditional methods, especially in complex biological scenarios. By more accurately modeling the spatio-temporal dynamics of radiopharmaceuticals, our method advances modeling of pharmacokinetic and pharmacodynamic processes, enabling new possibilities in quantitative nuclear medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7e97f4a77c199435d67bf7917712e6bcfb5bc2" target='_blank'>
              Beyond Conventional Parametric Modeling: Data-Driven Framework for Estimation and Prediction of Time Activity Curves in Dynamic PET Imaging
              </a>
            </td>
          <td>
            N. Zakariaei, Arman Rahmim, Eldad Haber
          </td>
          <td>2024-05-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Advances in molecular technologies underlie an enormous growth in the size of data sets pertaining to biology and biomedicine. These advances parallel those in the deep learning subfield of machine learning. Components in the differentiable programming toolbox that makes deep learning possible are allowing computer scientists to address an increasingly large array of problems with flexible and effective tools. However many of these tools have not fully proliferated into the computational biology and bioinformatics fields. In this perspective we survey some of these advances and highlight exemplary examples of their utilization in the biosciences, with the goal of increasing awareness among practitioners of emerging opportunities to blend expert knowledge with newly emerging deep learning architectural tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dab377c45c143c62caa8200f42e162d634ff1e2" target='_blank'>
              Opportunities in deep learning methods development for computational biology
              </a>
            </td>
          <td>
            Alex J. Lee, Reza Abbasi-Asl
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The live identification of emergent behavior in complex systems with little a-priori information is a challenging task and existing approaches are either applicable to a small subset of models or do not scale well. In contrast, post-mortem approaches that have a more in-depth understanding of the characteristics of emergent properties often struggle with analyzing a large amount of data to extract relationships between the variables, events, and entities whose interaction eventually leads to emergent behavior. Machine learning approaches have been promoted as potential replacements of existing approaches, due to their ability to analyze large amounts of data without a-priori knowledge of existing relationships. In this paper, we present a first step towards the use of supervised learning approaches to identify and predict emergent behavior. Our hybrid approach unifies live and post-mortem perspectives by relying on a visual inspection of the simulation run and the simulation data set to identify a set of features that are more likely to generate emergent behavior (post-mortem) which are then used by a machine learning module to predict emergent behavior (live). Our analysis shows the potential of such approaches but also highlights challenges and future avenues of research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c22c6b6693586120f80e593dda61dfc8e28ccd" target='_blank'>
              Detecting Emergent Behavior in Complex Systems: A Machine Learning Approach
              </a>
            </td>
          <td>
            Simranjeet Singh Dahia, Claudia Szabo
          </td>
          <td>2024-06-24</td>
          <td>Proceedings of the 38th ACM SIGSIM Conference on Principles of Advanced Discrete Simulation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Modeling the dynamics of flexible objects has become an emerging topic in the community as these objects become more present in many applications, e.g., soft robotics. Due to the properties of flexible materials, the movements of soft objects are often highly nonlinear and, thus, complex to predict. Data-driven approaches seem promising for modeling those complex dynamics but often neglect basic physical principles, which consequently makes them untrustworthy and limits generalization. To address this problem, we propose a physics-constrained learning method that combines powerful learning tools and reliable physical models. Our method leverages the data collected from observations by sending them into a Gaussian process that is physically constrained by a distributed Port-Hamiltonian model. Based on the Bayesian nature of the Gaussian process, we not only learn the dynamics of the system, but also enable uncertainty quantification. Furthermore, the proposed approach preserves the compositional nature of Port-Hamiltonian systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020e63b8b17cdec54c056d8a1edc98770c8fa7ab" target='_blank'>
              Physics-Constrained Learning for PDE Systems with Uncertainty Quantified Port-Hamiltonian Models
              </a>
            </td>
          <td>
            Kaiyuan Tan, Peilun Li, Thomas Beckers
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In a landscape where scientific discovery is increasingly driven by data, the integration of machine learning (ML) with traditional scientific methodologies has emerged as a transformative approach. This paper introduces a novel, data-driven framework that synergizes physics-based priors with advanced ML techniques to address the computational and practical limitations inherent in first-principle-based methods and brute-force machine learning methods. Our framework showcases four algorithms, each embedding a specific physics-based prior tailored to a particular class of nonlinear systems, including separable and nonseparable Hamiltonian systems, hyperbolic partial differential equations, and incompressible fluid dynamics. The intrinsic incorporation of physical laws preserves the system's intrinsic symmetries and conservation laws, ensuring solutions are physically plausible and computationally efficient. The integration of these priors also enhances the expressive power of neural networks, enabling them to capture complex patterns typical in physical phenomena that conventional methods often miss. As a result, our models outperform existing data-driven techniques in terms of prediction accuracy, robustness, and predictive capability, particularly in recognizing features absent from the training set, despite relying on small datasets, short training periods, and small sample sizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba424643899decf4b48792c380cc0c66abe63c7" target='_blank'>
              Data-Driven Computing Methods for Nonlinear Physics Systems with Geometric Constraints
              </a>
            </td>
          <td>
            Yunjin Tong
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Most approaches for assessing causality in complex dynamical systems fail when the interactions between variables are inherently non-linear and non-stationary. Here we introduce Temporal Autoencoders for Causal Inference (TACI), a methodology that combines a new surrogate data metric for assessing causal interactions with a novel two-headed machine learning architecture to identify and measure the direction and strength of time-varying causal interactions. Through tests on both synthetic and real-world datasets, we demonstrate TACI's ability to accurately quantify dynamic causal interactions across a variety of systems. Our findings display the method's effectiveness compared to existing approaches and also highlight our approach's potential to build a deeper understanding of the mechanisms that underlie time-varying interactions in physical and biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d22ad3492ed41cdaeed3d49e84841e11089e47" target='_blank'>
              Inferring the time-varying coupling of dynamical systems with temporal convolutional autoencoders
              </a>
            </td>
          <td>
            Josuan Calderon, Gordon J. Berman
          </td>
          <td>2024-06-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose: Artificial Intelligence (AI), which mimics the human brain structure and operation, simulates intelligence. The aim of Machine Learning (ML), which is a branch of artificial intelligence, is to create models by analyzing data. Another type of artificial intelligence, Deep Learning (DL), depicts geometric changes using several layers of model representations. Since DL broke the computational analysis record, AI has advanced in many areas. 
Materials and Methods: Contrary to the widespread use of conventional ML methodologies, there is still a need to promote the use and popularity of DL for pharmaceutical research and development. Drug discovery and design have been enhanced by ML and DL in major research projects. To fully realize its potential, drug design must overcome many challenges and issues. Various aspects of medication design must be considered to successfully address these concerns and challenges. This review article explains DL's significance both in technological breakthroughs and in effective medications. 
Results: There are numerous barriers and substantial challenges associated with drug design associated with DL architectures and key application domains. The article discusses several elements of medication development that have been influenced by existing research. Two widely used and efficient Neural Network (NN) designs are discussed in this article: Convolutional Neural Networks (CNNs) and Recurrent Neural Networks (RNNs). 
Conclusion: It is described how these tools can be utilized to design and discover small molecules for drug discovery. They are also given an overview of the history of DL approaches, as well as a discussion of some of their drawbacks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0fdca353e21ebc2e1d16a9b8425165c55ad670" target='_blank'>
              Deep Learning in Drug Design—Progress, Methods, and Challenges
              </a>
            </td>
          <td>
            Seyyed Saeid Masoomkhah, Khosro Rezaee, Mojtaba Ansari, Hossein Eslami
          </td>
          <td>2024-06-24</td>
          <td>Frontiers in Biomedical Technologies</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The rapid evolution of single-cell sequencing technologies has facilitated precise transcriptomics profiling at the single-cell level, shedding light on the intricate heterogeneity within cellular populations. Despite these advances, the inherent diversity of cells and data challenges such as noise, batch effects, and sparsity, underscores the pressing need for a unified model to learn and represent cellular states effectively. Single-cell Large Language Models (LLMs) have been crafted to bridge this gap yet exhibit limited performance on human cells. This short-fall may stem from the confounding effects of training data from diverse species, partly because of limited cells for the single species. Here, we have compiled a dataset of approximately 100 million human cells sequenced by multiple technolo-gies from human single-cell datasets with various file types deposited in public databases and websites. Leveraging these extensive data cohorts, we developed CellFM, a robust single-cell foundation model with an impressive 800 million parameters, marking an eight-fold increase over the current largest single-species model. To ensure the training of CellFM on the MindSpore AI framework from Huawei, we have integrated RetNet, a Transformer architecture variant with lin-ear complexity for a balance between efficiency and performance, serving as the backbone of our model. Our comprehensive experiments have shown that CellFM outperforms existing models across diverse applications, such as cell annotation, perturbation prediction, and gene function prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd45aed312f5393a6aaf2e612c46c292cadd436e" target='_blank'>
              CellFM: a large-scale foundation model pre-trained on transcriptomics of 100 million human cells
              </a>
            </td>
          <td>
            Yuansong Zeng, Jiancong Xie, Zhuoyi Wei, Yun Su, Ningyuan Shangguan, Shuangyu Yang, Chengyang Zhang, Wenbing Li, Jinbo Zhang, Nan Fang, Hongyu Zhang, Huiying Zhao, Yutong Lu, Jue Fan, Weijiang Yu, Yuedong Yang
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Autoimmune rheumatic diseases are often characterised by heterogeneity in presentation. The traditional approach to diseases guided by their phenotype may be suboptimal with the advent of precision medicine. Precision medicine is the integration and application of multiomics to predict the best-performing drug and its toxicity profile to derive optimal benefits. With novel drug discoveries and an expanding therapeutic armamentarium, it potentially aids in clinical and therapeutic decision-making, while saving time and averting adverse events. However, multiomics comes with ‘big data’, and owing to the costs, the sample size is usually small. Machine learning (ML) plays an important role in these scenarios where conventional statistics fall short. So, by integrating clinical data with the data from -omics, ML models can be built, which can accurately predict the clinical factors or even novel biomarkers that predict response. This approach has a potential for great benefit as valuable time or the ‘therapeutic window of opportunity’ would be saved, with fewer adverse events, eventually translating to lower damage accrual and better outcomes. Most of the evidence for the use of ML in precision rheumatology comes from rheumatoid arthritis and the factors predicting response to various drugs, including tumour necrosis factor inhibitors. This approach also has its limitations such as the lack of generalizability and the current scarcity of longitudinal data. These models must be tested in larger cohorts and population-based studies for validation, failing which there is a risk of apparent identification of multiple ‘novel’ biomarkers that may or may not be mechanistic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f607769c09b08a0d2145eee4ed39d8f624f5aad6" target='_blank'>
              Current Status of Machine Learning for Precision Rheumatology: Are We There Yet?
              </a>
            </td>
          <td>
            Prakashini Mruthyunjaya, Avarna Agarwal, Sakir Ahmed
          </td>
          <td>2024-06-16</td>
          <td>Indian Journal of Rheumatology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Deep learning approaches have demonstrated remarkable potential in predicting cancer drug responses (CDRs), using cell line and drug features. However, existing methods predominantly rely on single-omics data of cell lines, potentially overlooking the complex biological mechanisms governing cell line responses. This paper introduces DeepFusionCDR, a novel approach employing unsupervised contrastive learning to amalgamate multi-omics features, including mutation, transcriptome, methylome, and copy number variation data, from cell lines. Furthermore, we incorporate molecular SMILES-specific transformers to derive drug features from their chemical structures. The unified multi-omics and drug signatures are combined, and a multi-layer perceptron (MLP) is applied to predict IC50 values for cell line-drug pairs. Moreover, this MLP can discern whether a cell line is resistant or sensitive to a particular drug. We assessed DeepFusionCDR's performance on the GDSC dataset and juxtaposed it against cutting-edge methods, demonstrating its superior performance in regression and classification tasks. We also conducted ablation studies and case analyses to exhibit the effectiveness and versatility of our proposed approach. Our results underscore the potential of DeepFusionCDR to enhance CDR predictions by harnessing the power of multi-omics fusion and molecular-specific transformers. The prediction of DeepFusionCDR on TCGA patient data and case study highlight the practical application scenarios of DeepFusionCDR in real-world environments. Source code and datasets can be available on https://github.com/altriavin/DeepFusionCDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55237e128fd3db51c88289d302cb3a3283ce281e" target='_blank'>
              DeepFusionCDR: Employing Multi-Omics Integration and Molecule-Specific Transformers for Enhanced Prediction of Cancer Drug Responses.
              </a>
            </td>
          <td>
            Xiaowen Hu, Pan Zhang, Jiaxuan Zhang, Lei Deng
          </td>
          <td>2024-06-27</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Developing new pharmaceuticals is a costly and time-consuming endeavor fraught with significant safety risks. A critical aspect of drug research and disease therapy is discerning the existence of interactions between drugs and proteins. The evolution of deep learning (DL) in computer science has been remarkably aided in this regard in recent years. Yet, two challenges remain: (i) balancing the extraction of profound, local cohesive characteristics while warding off gradient disappearance and (ii) globally representing and understanding the interactions between the drug and target local attributes, which is vital for delivering molecular level insights indispensable to drug development. In response to these challenges, we propose a DL network structure, MolLoG, primarily comprising two modules: local feature encoders (LFE) and global interactive learning (GIL). Within the LFE module, graph convolution networks and leap blocks capture the local features of drug and protein molecules, respectively. The GIL module enables the efficient amalgamation of feature information, facilitating the global learning of feature structural semantics and procuring multihead attention weights for abstract features stemming from two modalities, providing biologically pertinent explanations for black-box results. Finally, predictive outcomes are achieved by decoding the unified representation via a multilayer perceptron. Our experimental analysis reveals that MolLoG outperforms several cutting-edge baselines across four data sets, delivering superior overall performance and providing satisfactory results when elucidating various facets of drug-target interaction predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47d7134e78582b908d74226e67442dfea752621" target='_blank'>
              MolLoG: A Molecular Level Interpretability Model Bridging Local to Global for Predicting Drug Target Interactions
              </a>
            </td>
          <td>
            Bao-Ming Feng, Yuanyuan Zhang, Xiao-Chen Zhou, Jin-Long Wang, Yinfei Feng
          </td>
          <td>2024-05-06</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="e13603 Background: Oncology therapeutic development has high failure rates with substantial costs. Advances in technology, including molecular characterisations of cancer and computational power, provide the opportunity to better model therapeutic response for existing and emerging therapies. Methods: Concr adapted Bayesian statistical principles used by astrophysicists to effectively integrate multi-dimensional data (genomic & transcriptomic) for accurate modelling of cancer biology. Our proprietary computational framework was applied to predict clinical response and survival through construction of digital twins and in silico simulations of clinical trials. Predicted log odds ratios (LOR) were generated for drug response and compared to published data from 8 suitable historical clinical trials. Three distinct models underpin this framework: Drug Efficacy, Treatment Response & Overall Survival (OS). Three publicly available datasets were used for training: Cancer Therapeutic Response Portal ( in vitro dose-response data), Cancer Cell Line Encyclopaedia and TCGA. Results: For all 8 clinical studies, the digital twin model accurately simulated both trial arms, compared drug efficacy across arms and predicted which treatment was most active. Unblinded evaluation: Our model accurately predicted gemcitabine would have greater clinical benefit than 5-FU (LOR -0.10, P <0.0001) in metastatic pancreatic cancer. In advanced breast cancer, the model predicted docetaxel had a higher response rate than doxorubicin, and carboplatin response rates were similar to docetaxel in BRCA wild-type patients. In platinum-sensitive recurrent ovarian cancer, the model predicted cisplatin-based combination had a higher response rate than paclitaxel. Blinded evaluation: Using data for paclitaxel, the model correctly predicted that nab-paclitaxel+gemcitabine response rates were higher than gemcitabine in metastatic pancreatic cancer (predicted LOR -0.090, p = <0.001). In all 3 early breast cancer studies, predictions mirrored clinical trial results, adjuvant ECF > CMF, adjuvant TC > AC and adjuvant tamoxifen was superior to capecitabine. The underlying pan-tumour Overall Survival model used a Random Survival Forest approach, trained on 23 cancers with predictive accuracy of 0.78 AUC ROC and C-index 0.71. To enable cohort enrichment for drug response, the model segmented patient cohorts into treatment responder and non-responder. Using this approach, the log odds increased for drug response rate for 16 of the 19 different cancers and 14 of the different 17 drugs. Conclusions: Our digital twin model represents transformative technology for clinical trial simulation with strong potential for accelerating and de-risking clinical drug development, and utility for synthetic control arms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a659d949c4ba645ce412729470f4959a538035" target='_blank'>
              Prediction of therapeutic response and cancer outcomes in solid tumours via in silico clinical trials.
              </a>
            </td>
          <td>
            Uzma Asghar, M. Griffiths, Amanzhol Kubeyev, Jordan Laurie, Andrea Giorni, Luiz A Zillmann da Silva, Prabu Sivasubramaniam, Matthew Foster, Andrew Biankin
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Temporal link prediction is one of the most important tasks for predicting time-varying links by capturing dynamics within complex networks. However, it suffers from difficulties such as vulnerability to adversarial attacks and inadaptation to distinct evolutionary patterns. In this article, we propose a robust temporal link prediction architecture via stable gated models with reinforcement learning (SAGE-RL) consisting of a state encoding network (SEN) and a self-adaptive policy network (SPN). The former is utilized to capture network dynamics, while the latter helps the former adapt to distinct evolutionary patterns across various time periods. Within the SEN, a novel stable gate is introduced to ensure multiple spatiotemporal dependency paths and defend against adversarial attacks. An SPN is proposed to select different SEN instances by approximating the optimal action function, thereby adapting to various evolutionary patterns to learn the robust temporal and structural features from dynamic complex networks. It is proven that SAGE-LR with integral Lipschitz graph convolution is stable to relative perturbations in dynamic complex networks. With the aid of extensive experiments on five real-world graph benchmarks, SAGE-LR is shown to substantially outperform current state-of-the-art approaches in terms of precision and stability of temporal link prediction and ability to successfully defend against various attacks. We also implement the temporal link prediction in shipping transaction networks, which forecast effectively its potential transaction risks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc8eab5464296d20d63e8638a6cc53aff28a678d" target='_blank'>
              Robust Temporal Link Prediction in Dynamic Complex Networks via Stable Gated Models With Reinforcement Learning.
              </a>
            </td>
          <td>
            Liping Yang, Hongbo Liu, Daoqiang Sun, Seán F. McLoone, Kai Liu, C. L. P. Chen
          </td>
          <td>2024-05-14</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Addressing the charged particle beam diagnostics in accelerators poses a formidable challenge, demanding high-fidelity simulations in limited computational time. Machine learning (ML) based surrogate models have emerged as a promising tool for non-invasive charged particle beam diagnostics. Trained ML models can make predictions much faster than computationally expensive physics simulations. In this work, we have proposed a temporally structured variational autoencoder model to autoregressively forecast the spatiotemporal dynamics of the 15 unique 2D projections of 6D phase space of charged particle beam as it travels through the LANSCE linear accelerator. In the model, VAE embeds the phase space projections into a lower dimensional latent space. A long-short-term memory network then learns the temporal correlations in the latent space. The trained network can evolve the phase space projections across further modules provided the first few modules as inputs. The model predicts all the projections across different modules with low mean squared error and high structural similarity index.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0389e136b98aa7fd4f3803f851e928b030ad8c1d" target='_blank'>
              Towards latent space evolution of spatiotemporal dynamics of six-dimensional phase space of charged particle beams
              </a>
            </td>
          <td>
            M. Rautela, Alan Williams, A. Scheinker
          </td>
          <td>2024-06-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Deep generative models are becoming a tool of choice for exploring the molecular space. One important application area of deep generative models is the reverse design of drug compounds for given attributes (solubility, ease of synthesis, etc.). Although there are many generative models, these models cannot generate specific intervals of attributes. This paper proposes a AC-ModNet model that effectively combines VAE with AC-GAN to generate molecular structures in specific attribute intervals. The AC-ModNet is trained and evaluated using the open 250K ZINC dataset. In comparison with related models, our method performs best in the FCD and Frag model evaluation indicators. Moreover, we prove the AC-ModNet created molecules have potential application value in drug design by comparing and analyzing them with medical records in the PubChem database. The results of this paper will provide a new method for machine learning drug reverse design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2a0c9044c8aa0c08c17758aa4b9b3d3d82c4c7b" target='_blank'>
              AC-ModNet: Molecular Reverse Design Network Based on Attribute Classification
              </a>
            </td>
          <td>
            Wei Wei, Jun Fang, Ning Yang, Qi Li, Lin Hu, Lanbo Zhao, Jie Han
          </td>
          <td>2024-06-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract A major challenge of precision oncology is the identification and prioritization of suitable treatment options based on molecular biomarkers of the considered tumor. In pursuit of this goal, large cancer cell line panels have successfully been studied to elucidate the relationship between cellular features and treatment response. Due to the high dimensionality of these datasets, machine learning (ML) is commonly used for their analysis. However, choosing a suitable algorithm and set of input features can be challenging. We performed a comprehensive benchmarking of ML methods and dimension reduction (DR) techniques for predicting drug response metrics. Using the Genomics of Drug Sensitivity in Cancer cell line panel, we trained random forests, neural networks, boosting trees and elastic nets for 179 anti-cancer compounds with feature sets derived from nine DR approaches. We compare the results regarding statistical performance, runtime and interpretability. Additionally, we provide strategies for assessing model performance compared with a simple baseline model and measuring the trade-off between models of different complexity. Lastly, we show that complex ML models benefit from using an optimized DR strategy, and that standard models—even when using considerably fewer features—can still be superior in performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2eaacf9f8df2242ab429c53e183de03dc2d3222" target='_blank'>
              A comprehensive benchmarking of machine learning algorithms and dimensionality reduction methods for drug sensitivity prediction
              </a>
            </td>
          <td>
            Lea Eckhart, Kerstin Lenhof, Lisa-Marie Rolli, H-P. Lenhof
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1491d337e12daab70edf38ee62d8f8ee586b8e98" target='_blank'>
              Learning nonlinear operators in latent spaces for real-time predictions of complex dynamics in physical systems
              </a>
            </td>
          <td>
            Katiana Kontolati, S. Goswami, G. Em Karniadakis, Michael D Shields
          </td>
          <td>2024-06-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84a5d0b1c26573809df096903d64fc1da14e7531" target='_blank'>
              Advancing cancer drug development with mechanistic mathematical modeling: bridging the gap between theory and practice.
              </a>
            </td>
          <td>
            Alexander Kulesza, C. Couty, Paul Lemarre, Craig J Thalhauser, Yanguang Cao
          </td>
          <td>2024-06-21</td>
          <td>Journal of pharmacokinetics and pharmacodynamics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="After the coronavirus disease 2019 (COVID-19) outbreak erupted, it swiftly spread globally and triggered a severe public health crisis in 2019. To contain the virus’s spread, several countries implemented various lockdown measures. As the governments faced this unprecedented challenge, understanding the impact of lockdown policies became paramount. The goal of addressing the pandemic crisis is to devise prudent policies that strike a balance between safeguarding lives and maintaining economic stability. Traditional mathematical and statistical models for studying virus transmission only offer macro-level predictions of epidemic development and often overlook individual variations’ impact, therefore failing to reflect the role of government decisions. To address this challenge, we propose an integrated framework that combines agent-based modeling (ABM) and deep Q-network (DQN) techniques. This framework enables a more comprehensive analysis and optimization of epidemic control strategies while considering real human behavior. We construct a pandemic simulator based on the ABM method, accurately simulating agents’ daily activities, interactions, and the dynamic spread of the virus. Additionally, we employ a data-driven approach and adjust the model through real statistical data to enhance its effectiveness. Subsequently, we integrated ABM into a decision-making framework using reinforcement learning techniques to explore the most effective strategies. In experiments, we validated the model’s effectiveness by simulating virus transmission across different countries globally. In this model, we obtained decision outcomes when governments focused on various factors. Our research findings indicate that our model serves as a valuable tool for decision-makers, enabling them to formulate prudent and rational policies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce211bae4071f08f917efe2eb6cd613c89e2263" target='_blank'>
              A Data-Driven Pandemic Simulator with Reinforcement Learning
              </a>
            </td>
          <td>
            Yuting Zhang, Biyang Ma, Langcai Cao, Yanyu Liu
          </td>
          <td>2024-06-27</td>
          <td>Electronics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract The inherent heterogeneity of cancer contributes to highly variable responses to any anticancer treatments. This underscores the need to first identify precise biomarkers through complex multi-omics datasets that are now available. Although much research has focused on this aspect, identifying biomarkers associated with distinct drug responders still remains a major challenge. Here, we develop MOMLIN, a multi-modal and -omics machine learning integration framework, to enhance drug-response prediction. MOMLIN jointly utilizes sparse correlation algorithms and class–specific feature selection algorithms, which identifies multi-modal and -omics–associated interpretable components. MOMLIN was applied to 147 patients’ breast cancer datasets (clinical, mutation, gene expression, tumor microenvironment cells and molecular pathways) to analyze drug-response class predictions for non-responders and variable responders. Notably, MOMLIN achieves an average AUC of 0.989, which is at least 10% greater when compared with current state-of-the-art (data integration analysis for biomarker discovery using latent components, multi-omics factor analysis, sparse canonical correlation analysis). Moreover, MOMLIN not only detects known individual biomarkers such as genes at mutation/expression level, most importantly, it correlates multi-modal and -omics network biomarkers for each response class. For example, an interaction between ER-negative-HMCN1-COL5A1 mutations-FBXO2-CSF3R expression-CD8 emerge as a multimodal biomarker for responders, potentially affecting antimicrobial peptides and FLT3 signaling pathways. In contrast, for resistance cases, a distinct combination of lymph node-TP53 mutation-PON3-ENSG00000261116 lncRNA expression-HLA-E-T-cell exclusions emerged as multimodal biomarkers, possibly impacting neurotransmitter release cycle pathway. MOMLIN, therefore, is expected advance precision medicine, such as to detect context–specific multi-omics network biomarkers and better predict drug-response classifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c9478bdc170073aa91d05881b2bed54efbf90f" target='_blank'>
              Advancing drug-response prediction using multi-modal and -omics machine learning integration (MOMLIN): a case study on breast cancer clinical data
              </a>
            </td>
          <td>
            Md Mamunur Rashid, Kumar Selvarajoo
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="EpiLearn is a Python toolkit developed for modeling, simulating, and analyzing epidemic data. Although there exist several packages that also deal with epidemic modeling, they are often restricted to mechanistic models or traditional statistical tools. As machine learning continues to shape the world, the gap between these packages and the latest models has become larger. To bridge the gap and inspire innovative research in epidemic modeling, EpiLearn not only provides support for evaluating epidemic models based on machine learning, but also incorporates comprehensive tools for analyzing epidemic data, such as simulation, visualization, transformations, etc. For the convenience of both epidemiologists and data scientists, we provide a unified framework for training and evaluation of epidemic models on two tasks: Forecasting and Source Detection. To facilitate the development of new models, EpiLearn follows a modular design, making it flexible and easy to use. In addition, an interactive web application is also developed to visualize the real-world or simulated epidemic data. Our package is available at https://github.com/Emory-Melody/EpiLearn.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e0a4bf71b4cf7906c2ed42050b45a3e4b7e895" target='_blank'>
              EpiLearn: A Python Library for Machine Learning in Epidemic Modeling
              </a>
            </td>
          <td>
            Zewen Liu, Yunxiao Li, Mingyang Wei, Guancheng Wan, Max S. Y. Lau, Wei Jin
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In this work, we introduce a novel application of foundation models in the domain of nonlinear chemical process modeling. Given the challenges of obtaining accurate first-principles models for real-world chemical processes and the inefficiency of rebuilding and retraining models for new chemical processes, we pose a pivotal question: What if we could develop a single, universal neural network (i.e., foundation model) capable of rapidly adapting to modeling any new chemical process? To address this question, we propose a meta-learning-based approach using Reptile to construct the foundation model, followed by physics-informed adaptation to fine-tune it to new modeling tasks using only a few data samples. To assess the effectiveness of our methodology, we construct a foundation model for various chemical reactions in three classical generic reactors, including continuous stirred tank reactors (CSTRs), batch reactors (BRs), and plug flow reactors (PFRs). Our approach outperforms conventional methods such as data-driven learning, physics-informed learning, transfer learning, and pure meta-learning in a few-shot setting. Furthermore, our method achieves rapid adaptation to new CSTRs, BRs, and PFRs using only a few data samples from the designated tasks. Source code is available at https://github.com/killingbear999/chemical-process-foundation-model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4edf55b1063ea6f1fd059221cbe47191bb3da351" target='_blank'>
              Foundation Model for Chemical Process Modeling: Meta-Learning with Physics-Informed Adaptation
              </a>
            </td>
          <td>
            Zihao Wang, Zhen Wu
          </td>
          <td>2024-05-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In the current field of clinical medicine, traditional methods for analyzing recurrent events have limitations when dealing with complex time-dependent data. This study combines Long Short-Term Memory networks (LSTM) with the Cox model to enhance the model's performance in analyzing recurrent events with dynamic temporal information. Compared to classical models, the LSTM-Cox model significantly improves the accuracy of extracting clinical risk features and exhibits lower Akaike Information Criterion (AIC) values, while maintaining good performance on simulated datasets. In an empirical analysis of bladder cancer recurrence data, the model successfully reduced the mean squared error during the training phase and achieved a Concordance index of up to 0.90 on the test set. Furthermore, the model effectively distinguished between high and low-risk patient groups, and the identified recurrence risk features such as the number of tumor recurrences and maximum size were consistent with other research and clinical trial results. This study not only provides a straightforward and efficient method for analyzing recurrent data and extracting features but also offers a convenient pathway for integrating deep learning techniques into clinical risk prediction systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e83aa2b5db279cd8e060c5ac6ab858f591cf20e" target='_blank'>
              LSTM-COX Model: A Concise and Efficient Deep Learning Approach for Handling Recurrent Events
              </a>
            </td>
          <td>
            Runquan Huang, Xiaoping Shi
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The principal goal of this research is to use regular imaging to diagnose and classify several lung disorders, including pneumonia, COVID-19 and tuberculosis.A novel Bi-directional Recurrent Neural Network (BRNN) model is implemented in this work to categorize the disease according to the input samples given. With the help of the feature extractor, prediction model, and classification model, the suggested model performs feature analysis. The E2E model that has been suggested carries out knowledge acquisition using an efficient computational and memory approach. The benchmark dataset, made accessible online, was used to test the proposed model. When networks are trained to extract characteristics, the developments in deep learning techniques yield encouraging results and offer higher potential efficiency than biomedical applications. The main goal is to evaluate the significance of current methods and validate the BRNN model to address the concerns that are currently present. The suggested model offers better efficiency and performance for better detection. The results reveal that the model with a recurrent network works better than the current methods, with an accuracy of 71% for all diseases, whereas the traditional network models perform worse. The suggested model has less computational overhead and fewer training parameters. When compared to alternative methods, the model offers a superior trade-off. ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/367442c73e761e05e7edc9056ec2df9cf67276f3" target='_blank'>
              Designing a Novel Learning Approach for Infectious Lung Disease Prediction using Bi-Directional Recurrent Neural Network (BRNN)
              </a>
            </td>
          <td>
            Gireesh T K
          </td>
          <td>2024-06-17</td>
          <td>Journal of Electrical Systems</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The development of models to predict sensitivity to anticancer drugs is an area of significant interest, given the diverse responses to treatment among patients and the considerable expense and time involved in anticancer drug development. Leveraging “omic” data and anticancer response information from the Cancer Cell Line Encyclopedia, we propose a novel approach utilizing multitask learning to enhance prediction accuracy and inference. We extended a multitask learning framework called the Data Shared Lasso to develop the Data Shared Elastic Net. This enabled the construction of tissue-specific models with information sharing while maintaining the attractive properties of Elastic Net regression. By employing this approach, we observed improvements in prediction accuracy compared to single-task Elastic Net models, particularly for cell lines displaying high sensitivity to treatment. Furthermore, the Data Shared Elastic Net facilitated the identification of predictors for anticancer drug sensitivity within specific tissue types, shedding light on cellular pathways targeted by these drugs across tissues. We also investigated the impact of data leakage on modeling outcomes from previous studies, which led to underestimating prediction error and erroneous inferences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83664bfa905ff011a2bcad47d103dd20fcdcd259" target='_blank'>
              Improving Drug Sensitivity Prediction and Inference by Multi-Task Learning
              </a>
            </td>
          <td>
            Jared Strauch, Amir Asiaee
          </td>
          <td>2024-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="5583 Background: Tumors with homologous recombination repair deficiencies (HRD+) are susceptible to PARP inhibitors (PARPi) which exploit a synthetic lethality by targeting an essential base excision repair (BER) pathway. Current biomarkers designed to predict efficacy of PARPi (BRCA1/2mut, HRD+) are ineffective, particularly in advanced stages of therapy.There is an urgent unmet need for improved PARPi biomarkers that is particularly acute for patients who fail multiple lines of therapy. We sought to develop and validate a predictive signature for identifying olaparib-sensitive patients that uses real-world data (RWD) as input. Methods: We developed a machine learning (ML) model, called DrugBERT, that learns a relationship between 1) drug structure, 2) DNA alterations, 3) drug response and 4) binding affinity between a drug and its purported target. DrugBERT utilizes genetic information from clinical commercial NGS panels and a subset of clinical features, and outputs a predictive signature for a drug of interest, without need for additional training. Each prediction is accompanied by a Vulnerability Network (VN) which represents a predicted latent genetic sensitivity in tumors and provides biological interpretability to model outputs. Because our model uses RWD as input, we were able to retrospectively evaluate our signature in a RWD ovarian cancer cohort treated with the PARPi, olaparib. Samples were filtered to biopsies taken < 2 years prior to olaparib treatment (N= 48 samples). Drug combinations were analyzed by processing each drug separately and stratifying patients based on sensitivity predictions to all drugs in the regimen. Results: Zephyr's signature outperformed existing PARPi biomarkers in a complex real-world olaparib-treated ovarian cancer cohort. OS and PFS were not significantly different when stratifying patients by BRCA1/2 mutations or HRD+. For Zephyr’s signature, both OS (p-value < 5x10-3; HR = 3.37, 95% CI: 1.27-5.46) and PFS (p-value < 10-3; HR = 2.04, 95% CI: 1.35-3.10), were statistically and clinically significantly prolonged. Characterization of VNs enriched in olaparib-predicted sensitive patient tumors indicated enhanced sensitivity to perturbing HR and cell cycle pathways. Conversely, olaparib-predicted non-sensitive ovarian tumors were characterized by VNs indicating perturbation sensitivity to cell motility and stress response pathways and displayed higher expression of DNA repair, drug resistance, and cancer stem cell pathways. Conclusions: We present a novel and enhanced approach for stratifying patients for PARPi therapy, retrospectively validated in a complex real-world ovarian cancer cohort treated with olaparib. Our interpretable model not only offers a biological hypothesis for predicted responses but is also readily applicable in clinical settings and compatible with standard multi-gene commercial NGS panels for immediate use.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d277e5ee1c329dece0aff0db401470ab93c64da" target='_blank'>
              Evaluation of a novel signature for PARP inhibitor sensitivity prediction using real-world data.
              </a>
            </td>
          <td>
            Emily Vucic, Felicia Kuperwaser, Sunil Kumar, Dillon Tracy, Olena Marchenko, Zong Miao, Jacob Kaffey, Samantha Pindak, Nathaniel Tann, Jordan Wolinsky, Sean Klei, Andrey Chursov, Fahad Khan, Jean Michel Rouly, G. Komatsoulis, Jeff Sherman, Maayan Baron
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In the trend of hybrid Artificial Intelligence (AI) techniques, Physic Informed Machine Learning has seen a growing interest. It operates mainly by imposing a data, learning or inductive bias with simulation data, Partial Differential Equations or equivariance and invariance properties. While these models have shown great success on tasks involving one physical domain such as fluid dynamics, existing methods still struggle on tasks with complex multi-physical and multi-domain phenomena. To address this challenge, we propose to leverage Bond Graphs, a multi-physics modeling approach together with Graph Neural Network. We thus propose Neural Bond Graph Encoder (NBgE), a model agnostic physical-informed encoder tailored for multi-physics systems. It provides an unified framework for any multi-physics informed AI with a graph encoder readable for any deep learning model. Our experiments on two challenging multi-domain physical systems - a Direct Current Motor and the Respiratory system - demonstrate the effectiveness of our approach on a multi-variate time series forecasting task.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5d7393bdb45ad9ec2527ea020c7d99f6d06d06" target='_blank'>
              Bond Graphs for multi-physics informed Neural Networks for multi-variate time series
              </a>
            </td>
          <td>
            Alexis-Raja Brachet, Pierre-Yves Richard, C'eline Hudelot
          </td>
          <td>2024-05-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Survival prediction is an important branch of cancer prognosis analysis. The model that predicts survival risk through TCGA genomics data can discover genes related to cancer and provide diagnosis and treatment recommendations based on patient characteristics. We found that deep learning models based on Cox proportional hazards often suffer from overfitting when dealing with high-throughput data. Moreover, we found that as the number of network layers increases, the experimental results will not get better, and network degradation will occur. Based on this problem, we propose a new framework based on Deep Residual Learning. Combine the ideas of Cox proportional hazards and Residual. And name it ResSurv. First, ResSurv is a feed-forward deep learning network stacked by multiple basic ResNet Blocks. In each ResNet Block, we add a Normalization Layer to prevent gradient disappearance and gradient explosion. Secondly, for the loss function of the neural network, we inherited the Cox proportional hazards methods, applied the semi-parametric of the CPH model to the neural network, combined with the partial likelihood model, established the loss function, and performed backpropagation and gradient update. Finally, we compared ResSurv networks of different depths and found that we can effectively extract high-dimensional features. Ablation experiments and comparative experiments prove that our model has reached SOTA(state of the art) in the field of deep learning, and our network can effectively extract deep information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2cbfe31657721f4bf6c2ca44aa455fb795d7549" target='_blank'>
              ResSurv: Cancer Survival Analysis Prediction Model Based on Residual Networks
              </a>
            </td>
          <td>
            Wankang Zhai
          </td>
          <td>2024-05-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance in cancer treatment presents a significant challenge, necessitating innovative approaches to improve therapeutic efficacy. Integrating machine learning (ML) in cancer research is promising as ML algorithms outrival in analysing complex datasets, identifying patterns, and predicting treatment outcomes. Leveraging diverse data sources such as genomic profiles, clinical records, and drug response assays, ML uncovers molecular mechanisms of drug resistance, enabling personalised treatment, maximising efficacy and minimising adverse effects. Various ML algorithms contribute to the drug discovery process- Random Forest and Decision Trees predict drug-target interactions and aid in virtual screening, and SVM classify leads on bioactivity data. Neural Networks model QSAR to optimise lead compounds and K-means clustering group compounds with similar chemical properties aiding compound selection. Gaussian Processes predict drug responses, Bayesian Networks infer causal relationships, Autoencoders generate novel compounds, and Genetic Algorithms optimise molecular structures. These algorithms collectively enhance efficiency and success rates in drug design endeavours, from lead identification to optimisation and are cost-effective, empowering clinicians with real-time treatment monitoring and improving patient outcomes. This review highlights the immense potential of ML in revolutionising cancer care through effective drug design to reduce drug resistance, and we have also discussed various limitations and research gaps to understand better.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e16966bea454fb58dc07ad8a5bb340ce722c1069" target='_blank'>
              Harnessing Machine Learning Potential for Personalised Drug Design and Overcoming Drug Resistance.
              </a>
            </td>
          <td>
            Mohammed AgeeliHakami
          </td>
          <td>2024-06-06</td>
          <td>Journal of drug targeting</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Ovarian cancer (OC) is the most lethal gynecological cancer in the United States. Among the different types of OC, serous ovarian cancer (SOC) stands out as the most prevalent. Transcriptomics techniques generate extensive gene expression data, yet only a few of these genes are relevant to clinical diagnosis. Methods: Methods for feature selection (FS) address the challenges of high dimensionality in extensive datasets. This study proposes a computational framework that applies FS techniques to identify genes highly associated with platinum-based chemotherapy response on SOC patients. Using SOC datasets from the Gene Expression Omnibus (GEO) database, LASSO and varSelRF FS methods were employed. Machine learning classification algorithms such as random forest (RF) and support vector machine (SVM) were also used to evaluate the performance of the models. Results: The proposed framework has identified biomarkers panels with 9 and 10 genes that are highly correlated with platinum–paclitaxel and platinum-only response in SOC patients, respectively. The predictive models have been trained using the identified gene signatures and accuracy of above 90% was achieved. Conclusions: In this study, we propose that applying multiple feature selection methods not only effectively reduces the number of identified biomarkers, enhancing their biological relevance, but also corroborates the efficacy of drug response prediction models in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3840852461c3aaf6b8f84cbd6005d49de7fd0573" target='_blank'>
              Evaluating Ovarian Cancer Chemotherapy Response Using Gene Expression Data and Machine Learning
              </a>
            </td>
          <td>
            Soukaina Amniouel, Keertana Yalamanchili, Sreenidhi Sankararaman, M. S. Jafri
          </td>
          <td>2024-05-22</td>
          <td>BioMedInformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Physiological and pathological processes are governed by a network of genes called gene regulatory networks (GRNs). By reconstructing GRNs, we can accurately model how cells behave in their natural state and predict how genetic changes will affect them. Transcriptomic data of single cells are now available for a wide range of cellular processes in multiple species. Thus, a method building predictive GRNs from single-cell RNA sequencing (scRNA-seq) data, without any additional prior knowledge, could have a great impact on our understanding of biological processes and the genes playing a key role in them. To this aim, we developed IGNITE (Inference of Gene Networks using Inverse kinetic Theory and Experiments), an unsupervised machine learning framework designed to infer directed, weighted, and signed GRNs directly from unperturbed single-cell RNA sequencing data. IGNITE uses the GRNs to generate gene expression data upon single and multiple genetic perturbations. IGNITE is based on the inverse problem for a kinetic Ising model, a model from statistical physics that has been successfully applied to biological networks. We tested IGNITE on murine pluripotent stem cells (PSCs) transitioning from the naïve to formative states. Using as input only scRNA-seq data of unperturbed PSCs, IGNITE simulated single and triple gene knockouts. Comparison with experimental data revealed high accuracy, up to 74%, outperforming currently available methods. In sum, IGNITE identifies predictive GRNs from scRNA-seq data without additional prior knowledge and faithfully simulates single and multiple gene perturbations. Applications of IGNITE range from studying cell differentiation to identifying genes specifically active under pathological conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f91d1ee7feee1c23aecfd8641c38451501491d9" target='_blank'>
              Unveiling Gene Perturbation Effects through Gene Regulatory Networks Inference from single-cell transcriptomic data
              </a>
            </td>
          <td>
            Clelia Corridori, Merrit Romeike, Giorgio Nicoletti, C. Buecker, S. Suweis, S. Azaele, Graziano Martello
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="e15064 Background: This real-world study underscores the transformative potential of multidimensional tumor profiling, including tissue and blood-based markers, in guiding personalized oncology treatments. We established a robust link between comprehensive diagnostics and improved clinical outcomes by correlating extensive profiling data with progression-free survival (PFS), the ratio of PFS2 to PFS1, and comparing these with treatment modalities. Methods: Over five years, 44 patients with advanced-stage solid tumors underwent comprehensive tumor profiling, including genetic markers from tissue and blood, mRNA expression, circulating tumor cells, pharmacogenomics, and immunohistochemistry, to tailor therapies. These multidimensional datasets were rigorously analyzed applying machine learning algorithms against responses to target, chemo, and combo therapies, using robust statistical frameworks to elucidate the impact of advanced diagnostics on therapy effectiveness. Results: Multidimensional tumor testing markedly extended progression-free survival, with median PFS1 and PFS2 at 22.5 and 30 weeks, respectively t(43) = 1.76, p = 0.043, and a significant median PFS2/1 ratio of 1.31. Targeted therapies achieved the highest median PFS2 at 61 weeks, surpassing chemo alone (10.5 weeks) and combination therapy (29.5 weeks) significantly. Patients treated based on gene expression, including mRNA profiling and immunohistochemistry/immunocytochemistry (n = 22), had a median PFS of 37.5 weeks, outperforming those treated based on NGS (n = 12) with 27.5 weeks significantly. Those not fully adhering to guideline treatments due to divergent findings had a median PFS2 of 16.5 weeks. Notably, 26 out of 44 patients with critical illness had a PFS2 over 6 months. Conclusions: A clear linkage was established between comprehensive profiling indicators, including genetic, molecular, and cellular markers, and the observed therapeutic outcomes. The study reaffirms the critical importance of multidimensional tumor profiling in the era of precision oncology. However, we proceed monitoring this growing group of patients to establish a clearer picture how to adapt complex diagnostic tools into modern oncological practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d850c6e4f18d0de72ca354916f7a112b7597673" target='_blank'>
              Multidimensional tumor profiling and its meaningful impact on personalized therapy outcomes: Insights from a real-world study.
              </a>
            </td>
          <td>
            Dörthe Schaffrin-Nabe, Timothy Crook, S. Limaye, S. Schuster, A. Josten-Nabe, D. Reismann, D. Patil, V. Datta, Adrian Heinze, M. Schaffrin, R. Voigtmann, R. Kurzrock
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="By using omics, we can now examine all components of biological systems simultaneously. Deep learning-based drug prediction methods have shown promise by integrating cancer-related multi-omics data. However, the complex interaction between genes poses challenges in accurately projecting multi-omics data. In this research, we present a predictive model for drug response that incorporates diverse types of omics data, comprising genetic mutation, copy number variation, methylation, and gene expression data. This study proposes latent alignment for information mismatch in integration, which is achieved through an attention module capturing interactions among diverse types of omics data. The latent alignment and attention modules significantly improve predictions, outperforming the baseline model, with MSE = 1.1333, F1-score = 0.5342, and AUROC = 0.5776. High accuracy was achieved in predicting drug responses for piplartine and tenovin-6, while the accuracy was comparatively lower for mitomycin-C and obatoclax. The latent alignment module exclusively outperforms the baseline model, enhancing the MSE by 0.2375, the F1-score by 4.84%, and the AUROC by 6.1%. Similarly, the attention module only improves these metrics by 0.1899, 2.88%, and 2.84%, respectively. In the interpretability case study, panobinostat exhibited the most effective predicted response, with a value of −4.895. We provide reliable insights for drug selection in personalized medicine by identifying crucial genetic factors influencing drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f40ba043b018c0ffb309d5df62d65dac2ff38ebf" target='_blank'>
              An Innovative Multi-Omics Model Integrating Latent Alignment and Attention Mechanism for Drug Response Prediction
              </a>
            </td>
          <td>
            Hui-O Chen, Yuan-Chi Cui, Peng-Chan Lin, Jung-Hsien Chiang
          </td>
          <td>2024-06-27</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Reconstructing the topology of gene regulatory network from gene expression data has been extensively studied. With the abundance functional transcriptomic data available, it is now feasible to systematically decipher regulatory interaction dynamics in a logic form such as a Boolean network (BN) framework, which qualitatively indicates how multiple regulators aggregated to affect a common target gene. However, inferring both the network topology and gene interaction dynamics simultaneously is still a challenging problem since gene expression data are typically noisy and data discretization is prone to information loss. We propose a new method for BN inference from time-series transcriptional profiles, called LogicGep. LogicGep formulates the identification of Boolean functions as a symbolic regression problem that learns the Boolean function expression and solve it efficiently through multi-objective optimization using an improved gene expression programming algorithm. To avoid overly emphasizing dynamic characteristics at the expense of topology structure ones, as traditional methods often do, a set of promising Boolean formulas for each target gene is evolved firstly, and a feed-forward neural network trained with continuous expression data is subsequently employed to pick out the final solution. We validated the efficacy of LogicGep using multiple datasets including both synthetic and real-world experimental data. The results elucidate that LogicGep adeptly infers accurate BN models, outperforming other representative BN inference algorithms in both network topology reconstruction and the identification of Boolean functions. Moreover, the execution of LogicGep is hundreds of times faster than other methods, especially in the case of large network inference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81189774cd4d14cf52c8efb5374bdb7e2ca817d4" target='_blank'>
              LogicGep: Boolean networks inference using symbolic regression from time-series transcriptomic profiling data
              </a>
            </td>
          <td>
            Dezhen Zhang, Shuhua Gao, Zhiping Liu, Rui Gao
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="10061 Background: Genomics precision medicine, deployed via tumor panel sequencing, now assists in deploying targeted therapies to cancer patients. Numerous clinical trials have investigated the utility and benefit of genomics precision medicine in multiple cancer indications. Current large-scale studies report actionability rates from ~35% to ~60%, although clinical benefit rates have been shown to be closer to 10%. While this has positively impacted patients in need, the gap between actionability and benefit remains a clinical challenge attributed to multiple factors including the complex, multi-factorial relationship between molecular status and response to therapy. These differences go beyond simple disease states and may be reflective of multiple clinically relevant features including age, sex, and race/ethnicity. Methods: We implemented a functional precision medicine (FPM) program where patients with advanced pediatric cancers were prospectively profiled via high-throughput drug sensitivity testing (DST) of FDA-approved agents on patient-derived tumor cells as well as genomics testing. The objective was to investigate the clinical utility and benefit of FPM guidance in the treatment of pediatric cancer and elucidate the relationship between molecular status of patients’ diseases and treatment responses. We generated DST data (n = 21 patients) and genomic profiling data (n = 20 patients) on pediatric cancer patients in Miami, FL, as well as post-hoc whole exome and transcriptome sequencing data (n = 13 patients) and investigated three specific relationships. Results: We analyzed the relationship between racial/ethnic background and functional response to anti-cancer agents, determining potential differences in response to therapeutic classes. Next, we examined relationships between functional response and cancer type, identifying an unanticipated lack of clustering between disease indications in patients with advanced pediatric cancers. Finally, we applied an explainable machine learning (xML) framework to the functional genomic dataset to develop multi-omics biomarker hypotheses for the chemotherapy agent idarubicin, pinpointing a potential multi-cancer relationship between response to idarubicin and known disease mechanisms in acute myeloid leukemia (AML), the sole indication where idarubicin is approved. We further present additional proof-of-concept studies generating biomarker hypotheses via xML, demonstrating a framework for development of multi-omics biomarkers. Conclusions: We are now expanding our pan-pediatric cancer functional genomics dataset through an NIMHD-funded expansion cohort (NCT05857969, n = 65 patients) to further investigate multi-omics relationships between functional and molecular characteristics and understand the role of race/ethnicity in the complex relationship.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/786847ce4af81d26fccb1bc7ad4a99896c1c5825" target='_blank'>
              Biomarker development from functional precision medicine datasets via explainable machine learning.
              </a>
            </td>
          <td>
            Noah Berlow, Arlet M Acanda De La Rocha, Maggie Fader, E. Coats, Cima Saghira, P. Espinal, Jeanette Galano, Z. Khatib, Haneen Y. Abdella, O. Maher, Cristina M. Andrade-Feraud, Baylee Holcomb, Yasmin Ghurani, Lilliam Rimblas, Tom á s R. Guilarte, Nan Hu, Daria Salyakina, Diana J Azzam
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/985fb213001f71393d7581e0584e1c2910789136" target='_blank'>
              CVGAE: A Self-Supervised Generative Method for Gene Regulatory Network Inference Using Single-Cell RNA Sequencing Data.
              </a>
            </td>
          <td>
            Wei Liu, Zhijie Teng, Zejun Li, Jing Chen
          </td>
          <td>2024-05-23</td>
          <td>Interdisciplinary sciences, computational life sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The complexity, scale, and uncertainty in regulatory networks (e.g., gene regulatory networks and microbial networks) regularly pose a huge uncertainty in their models. These uncertainties often cannot be entirely reduced using limited and costly data acquired from the normal condition of systems. Meanwhile, regulatory networks often suffer from the non-identifiability issue, which refers to scenarios where the true underlying network model cannot be clearly distinguished from other possible models. Perturbation or excitation is a well-known process in systems biology for acquiring targeted data to reveal the complex underlying mechanisms of regulatory networks and overcome the non-identifiability issue. We consider a general class of Boolean network models for capturing the activation and inactivation of components and their complex interactions. Assuming partial available knowledge about the interactions between components of the networks, this paper formulates the inference process through the maximum aposteriori (MAP) criterion. We develop a Bayesian lookahead policy that systematically perturbs regulatory networks to maximize the performance of MAP inference under the perturbed data. This is achieved by optimally formulating the perturbation process in a reinforcement learning context and deriving a scalable deep reinforcement learning perturbation policy to compute near-optimal Bayesian policy. The proposed method learns the perturbation policy through planning without the need for any real data. The high performance of the proposed approach is demonstrated by comprehensive numerical experiments using the well-known mammalian cell cycle and gut microbial community networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13661311b83454b1f32c13b0d86a4299e417da66" target='_blank'>
              Bayesian Lookahead Perturbation Policy for Inference of Regulatory Networks.
              </a>
            </td>
          <td>
            Mohammad Alali, Mahdi Imani
          </td>
          <td>2024-05-17</td>
          <td>IEEE/ACM transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In recent decades, the utilization of machine learning (ML) and artificial intelligence (AI) approaches have been explored for process modelling applications. However, different types of ML models may have contrasting advantages and disadvantages, which become critical during the optimal selection of a specific data‐driven model for a particular application as well as estimation of parameters during model training. This paper compares and contrasts two different types of data‐driven modelling approaches, namely the series/parallel all‐nonlinear static‐dynamic neural network models and models from a Bayesian ML approach. Both types of AI modelling approaches considered in this work have shown to significantly outperform several state‐of‐the‐art steady‐state and dynamic data‐driven modelling techniques for various performance measures, specifically, model sparsity, predictive capabilities, and computational expense. The performances of the proposed model structures and algorithms have been evaluated for two nonlinear dynamic chemical engineering systems—a plug‐flow reactor for vapour phase cracking of acetone for production of acetic anhydride and a pilot‐plant for post‐combustion CO2 capture using monoethanolamine as the solvent. For the validation data from the CO2 capture pilot plant, root mean squared error (RMSE) for flue gas outlet temperature, flowrate and CO2 concentration is 0.05%, 1.07%, and 5.0%, respectively, for the all‐nonlinear static‐dynamic neural networks and 0.1%, 1.75%, and 14.14%, respectively, for the Bayesian ML models. For the plug flow reactor data, the Bayesian ML models yield superior RMSE compared to the all‐nonlinear static‐dynamic neural networks when the measurement data are corrupted with Gaussian, auto‐correlated, or cross‐correlated noise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/056ceb214531414b85db491a54767e185fb5c38a" target='_blank'>
              All‐nonlinear static‐dynamic neural networks versus Bayesian machine learning for data‐driven modelling of chemical processes
              </a>
            </td>
          <td>
            Angan Mukherjee, Samuel Adeyemo, D. Bhattacharyya
          </td>
          <td>2024-06-24</td>
          <td>The Canadian Journal of Chemical Engineering</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Developing new drugs is laborious and costly, demanding extensive time investment. In this study, we introduce an innovative de-novo drug design strategy, which harnesses the capabilities of language models to devise targeted drugs for specific proteins. Employing a Reinforcement Learning (RL) framework utilizing Proximal Policy Optimization (PPO), we refine the model to acquire a policy for generating drugs tailored to protein targets. Our method integrates a composite reward function, combining considerations of drug-target interaction and molecular validity. Following RL fine-tuning, our approach demonstrates promising outcomes, yielding notable improvements in molecular validity, interaction efficacy, and critical chemical properties, achieving 65.37 for Quantitative Estimation of Drug-likeness (QED), 321.55 for Molecular Weight (MW), and 4.47 for Octanol-Water Partition Coefficient (logP), respectively. Furthermore, out of the generated drugs, only 0.041\% do not exhibit novelty.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a86c4941d3f844efdeba838a2b0581bd4da69c84" target='_blank'>
              Improving Targeted Molecule Generation through Language Model Fine-Tuning Via Reinforcement Learning
              </a>
            </td>
          <td>
            Salma J. Ahmed, Mustafa A. Alattar
          </td>
          <td>2024-05-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9165fe7b8af9c461e14f5fa3a2cd250b938e0d" target='_blank'>
              Optimizing clinico-genomic disease prediction across ancestries: a machine learning strategy with Pareto improvement
              </a>
            </td>
          <td>
            Yan Gao, Yan Cui
          </td>
          <td>2024-06-04</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/620803fdb5f14b75e2d7602b6c40e8374eb91f5a" target='_blank'>
              Enhancing drug and cell line representations via contrastive learning for improved anti-cancer drug prioritization
              </a>
            </td>
          <td>
            Patrick J. Lawrence, Benjamin Burns, Xia Ning
          </td>
          <td>2024-05-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We consider the problem of learning the dynamics in the topology of time-evolving point clouds, the prevalent spatiotemporal model for systems exhibiting collective behavior, such as swarms of insects and birds or particles in physics. In such systems, patterns emerge from (local) interactions among self-propelled entities. While several well-understood governing equations for motion and interaction exist, they are difficult to fit to data due to the often large number of entities and missing correspondences between the observation times, which may also not be equidistant. To evade such confounding factors, we investigate collective behavior from a \textit{topological perspective}, but instead of summarizing entire observation sequences (as in prior work), we propose learning a latent dynamical model from topological features \textit{per time point}. The latter is then used to formulate a downstream regression task to predict the parametrization of some a priori specified governing equation. We implement this idea based on a latent ODE learned from vectorized (static) persistence diagrams and show that this modeling choice is justified by a combination of recent stability results for persistent homology. Various (ablation) experiments not only demonstrate the relevance of each individual model component, but provide compelling empirical evidence that our proposed model -- \textit{neural persistence dynamics} -- substantially outperforms the state-of-the-art across a diverse set of parameter regression tasks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49c1f339720cec5e7fa7b827d7664d093032838" target='_blank'>
              Neural Persistence Dynamics
              </a>
            </td>
          <td>
            Sebastian Zeng, Florian Graf, M. Uray, Stefan Huber, R. Kwitt
          </td>
          <td>2024-05-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Physics-informed deep learning has been developed as a novel paradigm for learning physical dynamics recently. While general physics-informed deep learning methods have shown early promise in learning fluid dynamics, they are difficult to generalize in arbitrary time instants in real-world scenario, where the fluid motion can be considered as a time-variant trajectory involved large-scale particles. Inspired by the advantage of diffusion model in learning the distribution of data, we first propose Pi-fusion, a physics-informed diffusion model for predicting the temporal evolution of velocity and pressure field in fluid dynamics. Physics-informed guidance sampling is proposed in the inference procedure of Pi-fusion to improve the accuracy and interpretability of learning fluid dynamics. Furthermore, we introduce a training strategy based on reciprocal learning to learn the quasiperiodical pattern of fluid motion and thus improve the generalizability of the model. The proposed approach are then evaluated on both synthetic and real-world dataset, by comparing it with state-of-the-art physics-informed deep learning methods. Experimental results show that the proposed approach significantly outperforms existing methods for predicting temporal evolution of velocity and pressure field, confirming its strong generalization by drawing probabilistic inference of forward process and physics-informed guidance sampling. The proposed Pi-fusion can also be generalized in learning other physical dynamics governed by partial differential equations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d533bf917f7e4f8a81a52e26c206fadbb3f08b" target='_blank'>
              Pi-fusion: Physics-informed diffusion model for learning fluid dynamics
              </a>
            </td>
          <td>
            Jing Qiu, Jiancheng Huang, Xiangdong Zhang, Zeng Lin, Minglei Pan, Zengding Liu, F. Miao
          </td>
          <td>2024-06-06</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="In recent years, alternative animal testing methods such as computational and machine learning approaches have become increasingly crucial for toxicity testing. However, the complexity and scarcity of available biomedical data challenge the development of predictive models. Combining nonlinear machine learning together with multicondition descriptors offers a solution for using data from various assays to create a robust model. This work applies multicondition descriptors (MCDs) to develop a QSTR (Quantitative Structure-Toxicity Relationship) model based on a large toxicity data set comprising more than 80,000 compounds and 59 different end points (122,572 data points). The prediction capabilities of developed single-task multi-end point machine learning models as well as a novel data analysis approach with the use of Convolutional Neural Networks (CNN) are discussed. The results show that using MCDs significantly improves the model and using them with CNN-1D yields the best result (R2train = 0.93, R2ext = 0.70). Several structural features showed a high level of contribution to the toxicity, including van der Waals surface area (VSA), number of nitrogen-containing fragments (nN+), presence of S-P fragments, ionization potential, and presence of C-N fragments. The developed models can be very useful tools to predict the toxicity of various compounds under different conditions, enabling quick toxicity assessment of new compounds.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87156d13c3b859bac22e2f79aaa50bb299a8a8aa" target='_blank'>
              Multi-Endpoint Acute Toxicity Assessment of Organic Compounds Using Large-Scale Machine Learning Modeling.
              </a>
            </td>
          <td>
            Amirreza Daghighi, Gerardo Casanola-Martin, Kweeni Iduoku, H. Kušić, Humberto González-Díaz, Bakhtiyor Rasulev
          </td>
          <td>2024-05-26</td>
          <td>Environmental science & technology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Heterogeneous, interconnected, systems-level, molecular data have become increasingly available and key in precision medicine. We need to utilize them to better stratify patients into risk groups, discover new biomarkers and targets, repurpose known and discover new drugs to personalize medical treatment. Existing methodologies are limited and a paradigm shift is needed to achieve quantitative and qualitative breakthroughs. In this perspective paper, we survey the literature and argue for the development of a comprehensive, general framework for embedding of multi-scale molecular network data that would enable their explainable exploitation in precision medicine in linear time. Network embedding methods map nodes to points in low-dimensional space, so that proximity in the learned space reflects the network's topology-function relationships. They have recently achieved unprecedented performance on hard problems of utilizing few omic data in various biomedical applications. However, research thus far has been limited to special variants of the problems and data, with the performance depending on the underlying topology-function network biology hypotheses, the biomedical applications and evaluation metrics. The availability of multi-omic data, modern graph embedding paradigms and compute power call for a creation and training of efficient, explainable and controllable models, having no potentially dangerous, unexpected behaviour, that make a qualitative breakthrough. We propose to develop a general, comprehensive embedding framework for multi-omic network data, from models to efficient and scalable software implementation, and to apply it to biomedical informatics. It will lead to a paradigm shift in computational and biomedical understanding of data and diseases that will open up ways to solving some of the major bottlenecks in precision medicine and other domains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2728c79f21b67623d3fdbf31a0f92bf8fe73b0aa" target='_blank'>
              Simplicity within biological complexity
              </a>
            </td>
          <td>
            Nataša Pržulj, N. Malod-Dognin
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Optimal control of multiple heating, ventilation, and air-conditioning units in an open-plan space demands fast and accurate thermodynamic modeling. While computational fluid dynamics is precise, it is computationally impractical for real-time control. Lumped thermal models divide the room into cells and explicitly model inter-cell air-mixing. However, solving the coupled differential equations is still computationally challenging. Data-driven methods like long-short term memory may handle non-linear thermal dynamics but face limitations in observability, adherence to physical constraints, and generalization. Physics-informed neural networks (PINN) impose physical constraints even while learning from data. However, PINNs still lack the scalability required for effective control in large open-plan offices, primarily due to air-mixing interactions Our approach, PhyGICS, combines graph neural networks with physics-informed learning (PI-GNN) to overcome these challenges. Specifically, we model the thermodynamic interactions as edges between nodes that represent cells. Further, the modeling approach allows explicit modeling of wall and window surface temperatures that are commonly ignored. PhyGICS, utilizing PI-GNN as a state-estimator, employs a receding-horizon approach for optimal HVAC control. We adapt PI-GNNs for building HVAC control by incorporating a time-resetting strategy to handle time-dependent ambient conditions and therefore set-points. Evaluations in a simulated environment with real-world data show PhyGICS outperforms a regular PINN model and other baseline control strategies on: 1) thermal model accuracy; 2) computation time; 3) energy consumption; and 4) user comfort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b9dc12e24efd410665c6606c578c428451e250" target='_blank'>
              PhyGICS – A Physics-informed Graph Neural Network-based Intelligent HVAC Controller for Open-plan Spaces
              </a>
            </td>
          <td>
            S. Nagarathinam, Arunchandar Vasan
          </td>
          <td>2024-06-04</td>
          <td>Proceedings of the 15th ACM International Conference on Future and Sustainable Energy Systems</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The intricacies of the human genome, manifested as a complex network of genes, transcend conventional representations in text or numerical matrices. The intricate gene-to-gene relationships inherent in this complexity find a more suitable depiction in graph structures. In the pursuit of predicting gene expression, an endeavor shared by predecessors like the L1000 and Enformer methods, we introduce a novel spatial graph-neural network (GNN) approach. This innovative strategy incorporates graph features, encompassing both regulatory and structural elements. The regulatory elements include pair-wise gene correlation, biological pathways, protein-protein interaction networks, and transcription factor regulation. The spatial structural elements include chromosomal distance, histone modification and Hi-C inferred 3D genomic features. Principal Node Aggregation models, validated independently, emerge as frontrunners, demonstrating superior performance compared to traditional regression and other deep learning models. By embracing the spatial GNN paradigm, our method significantly advances the description of the intricate network of gene interactions, surpassing the performance, predictable scope, and initial requirements set by previous methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f51120aeb183a8a8caaf3a4cca0f7e47b5212e" target='_blank'>
              Reinventing gene expression connectivity through regulatory and spatial structural empowerment via principal node aggregation graph neural network.
              </a>
            </td>
          <td>
            Fengyao Yan, Limin Jiang, Danqian Chen, M. Ceccarelli, Yan Guo
          </td>
          <td>2024-06-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Molecular analytics increasingly utilize machine learning (ML) for predictive modeling based on data acquired through molecular profiling technologies. However, developing robust models that accurately capture physiological phenotypes is challenged by a multitude of factors. These include the dynamics inherent to biological systems, variability stemming from analytical procedures, and the resource-intensive nature of obtaining sufficiently representative datasets. Here, we propose and evaluate a new method: Contextual Out-of-Distribution Integration (CODI). Based on experimental observations, CODI generates synthetic data that integrate unrepresented sources of variation encountered in real-world applications into a given molecular fingerprint dataset. By augmenting a dataset with out-of-distribution variance, CODI enables an ML model to better generalize to samples beyond the initial training data. Using three independent longitudinal clinical studies and a case-control study, we demonstrate CODI’s application to several classification scenarios involving vibrational spectroscopy of human blood. We showcase our approach’s ability to enable personalized fingerprinting for multi-year longitudinal molecular monitoring and enhance the robustness of trained ML models for improved disease detection. Our comparative analyses revealed that incorporating CODI into the classification workflow consistently led to significantly improved classification accuracy while minimizing the requirement of collecting extensive experimental observations. SIGNIFICANCE STATEMENT Analyzing molecular fingerprint data is challenging due to multiple sources of biological and analytical variability. This variability hinders the capacity to collect sufficiently large and representative datasets that encompass realistic data distributions. Consequently, the development of machine learning models that generalize to unseen, independently collected samples is often compromised. Here, we introduce CODI, a versatile framework that enhances traditional classifier training methodologies. CODI is a general framework that incorporates information about possible out-of-distribution variations into a given training dataset, augmenting it with simulated samples that better capture the true distribution of the data. This allows the classification to achieve improved predictive performance on samples beyond the original distribution of the training data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f5a765a1c0f589cfa9eccf1b65dd12320bb5114" target='_blank'>
              CODI: Enhancing machine learning-based molecular profiling through contextual out-of-distribution integration
              </a>
            </td>
          <td>
            Tarek Eissa, Marinus Huber, Barbara Obermayer-Pietsch, Birgit Linkohr, A. Peters, Frank Fleischmann, M. Žigman
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Protein-protein bindings play a key role in a variety of fundamental biological processes, and thus predicting the effects of amino acid mutations on protein-protein binding is crucial. To tackle the scarcity of annotated mutation data, pre-training with massive unlabeled data has emerged as a promising solution. However, this process faces a series of challenges: (1) complex higher-order dependencies among multiple (more than paired) structural scales have not yet been fully captured; (2) it is rarely explored how mutations alter the local conformation of the surrounding microenvironment; (3) pre-training is costly, both in data size and computational burden. In this paper, we first construct a hierarchical prompt codebook to record common microenvironmental patterns at different structural scales independently. Then, we develop a novel codebook pre-training task, namely masked microenvironment modeling, to model the joint distribution of each mutation with their residue types, angular statistics, and local conformational changes in the microenvironment. With the constructed prompt codebook, we encode the microenvironment around each mutation into multiple hierarchical prompts and combine them to flexibly provide information to wild-type and mutated protein complexes about their microenvironmental differences. Such a hierarchical prompt learning framework has demonstrated superior performance and training efficiency over state-of-the-art pre-training-based methods in mutation effect prediction and a case study of optimizing human antibodies against SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d0d461d9e042af32d1b4135610c40d13c0d751" target='_blank'>
              Learning to Predict Mutation Effects of Protein-Protein Interactions by Microenvironment-aware Hierarchical Prompt Learning
              </a>
            </td>
          <td>
            Lirong Wu, Yijun Tian, Haitao Lin, Yufei Huang, Siyuan Li, N. Chawla, Stan Z. Li
          </td>
          <td>2024-05-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Background Recent advancements in computational pathology have introduced deep learning methods to predict genomic, transcriptomic and molecular biomarkers from routine histology whole slide images (WSIs) for cancer diagnosis, prognosis, and treatment. However, existing methods often overlook the critical role of co-dependencies among biomarker statuses during training and inference. We hypothesize that this oversight results in models that predict the combined effect of multiple interdependent biomarkers rather than individual statuses independently, akin to attributing the quality of an orchestral symphony to a single instrument, highlighting limitations of current predictors. Methods Using large datasets (n = 8,221 patients), we conducted statistical co-dependence testing to demonstrate significant interdependencies among biomarker statuses in training datasets. Following standard protocols, we trained two machine learning models to predict biomarkers from WSIs achieving or matching state-of-the-art predictive performance. We then employed permutation testing and stratification analysis to evaluate their predictive quality based on the principle of conditional independence, i.e., if a model accurately captures the phenotypic influence of a specific biomarker independent of other biomarkers, its performance should remain consistent across subgroups of patients stratified by other biomarkers, aligning with its overall performance on the entire dataset. Findings Our statistical analysis reveals significant interdependencies among biomarkers, reflecting expected co-occurrence and mutual exclusivity patterns influenced by pathological and biological processes that are consistent across datasets, as well as sampling artefacts that can be different across datasets. Our results indicate that the predictive quality of an image-based predictor for a biomarker is contingent on the status of other biomarkers, revealing that models capture aggregated influences rather than predicting individual statuses independently. For example, mutation predictions are confounded by the overall tumour mutation burden. We also show that, due to the presence of such correlations, deep learning models may not offer significant advantages in predicting certain biomarkers in comparison to simply using pathologist-assigned grades for their prediction. Interpretation We show that current deep learning models in computational pathology fall short in isolating individual biomarker effects, leading to confounded and less precise predictions. Our findings suggest revisiting model training protocols to recognize and adjust for biomarker interdependencies at all development stages—from problem definition to usage guidelines. This involves selecting diverse datasets to reflect clinical heterogeneity, defining prediction variables or grouping patients based on co-dependencies, designing models to disentangle complex relationships, and stringent stratification testing. Clinically, failure to account for interdependencies may lead to suboptimal decisions, necessitating appropriate usage guidelines for predictive models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a318afdaf3385141a6d7bb9fce1bc08ba68f39c4" target='_blank'>
              Buyer Beware: confounding factors and biases abound when predicting omics-based biomarkers from histological images
              </a>
            </td>
          <td>
            Muhammad Dawood, Kim Branson, Sabine Tejpar, N. Rajpoot, F. Minhas
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="This study proposes a new method to improve the efficiency of drug discovery by repurposing existing drugs, aiming to reduce the time and costs associated with traditional drug development processes, which can span 10 to 15 years and cost billions of dollars. Current approaches focus on leveraging heterogeneous data, such as drug-protein and disease-protein interactions, to construct complex networks that link drugs, proteins, and diseases. However, a significant challenge is the imbalance in data, where numerous unconfirmed potential drug-disease interactions (the majority class) outnumber approved drugs (the minority class), severely impacting the predictive performance of machine learning models. Previous attempts to address this issue have shown limited success. This study introduces a novel approach that integrates meta-paths in heterogeneous information networks with data balancing techniques to tackle this imbalance. Experimental results demonstrate that the proposed method enhances model performance and reliability in identifying new relationships between drugs and diseases. This research represents a promising advancement by leveraging network-based strategies and data balancing techniques to facilitate the rediscovery of drug applications, thereby potentially revolutionizing the pharmaceutical industry’s approach to drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25438f7cc83aaed0d4842e6721d7f9054712298b" target='_blank'>
              ENHANCING DRUG DISCOVERY THROUGH A META-PATH BASED OVERSAMPLING APPROACH FOR IMBALANCED DATA
              </a>
            </td>
          <td>
            Manh Hung Le
          </td>
          <td>2024-06-30</td>
          <td>Journal of Science and Technique</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Capturing the time evolution and predicting future kinetic states of physicochemical systems present significant challenges due to the precision and computational effort required. In this study, we demonstrate that the transformer, a machine learning model renowned for machine translation and natural language processing, can be effectively adapted to predict the dynamical state-to-state transition kinetics of biologically relevant physicochemical systems. Specifically, by using sequences of time-discretized states from Molecular Dynamics (MD) simulation trajectories as input, we show that a transformer can learn the complex syntactic and semantic relationships within the trajectory. This enables this generative pre-trained transformer (GPT) to predict kinetically accurate sequences of future states for a diverse set of models and biomolecules of varying complexity. Remarkably, the GPT can predict future states much faster than traditional MD simulations. We show that it is particularly adept at forecasting the time evolution of an out-of-equilibrium active system that do not maintain detailed balance. An analysis of self-attention mechanism inherent in transformers is found to hold crucial role for capturing the long-range correlations necessary for accurate state-to-state transition predictions. Together, our results highlight the ability of transformer based machine learning model in generating future states of physicochemical systems with statistical precision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f68df42b37752017ad05438942ecb38a7e24c6e" target='_blank'>
              Predicting Future Kinetic States of Physicochemical Systems Using Generative Pre-trained Transformer
              </a>
            </td>
          <td>
            Palash Bera, Jagannath Mondal
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="We address the problem of model misspecification in population pharmacokinetics (PopPK), by modeling residual unexplained variability (RUV) by machine learning (ML) methods in a postprocessing step after conventional model building. The practical purpose of the method is the generation of realistic virtual patient profiles and the quantification of the extent of model misspecification, by introducing an appropriate metric, to be used as an additional diagnostic of model quality. The proposed methodology consists of the following steps: After developing a PopPK model, the individual residual errors IRES = DV-IPRED, are computed, where DV are the observations and IPRED the individual predictions and are modeled by ML to obtain IRESML. Correction of the IPREDs can then be carried out as DVML = IPRED + IRESML. The methodology was tested in a PK study of ropinirole, for which a PopPK model was developed while a second deliberately misspecified model was also considered. Various supervised ML algorithms were tested, among which Random Forest gave the best results. The ML model was able to correct individual predictions as inspected in diagnostic plots and most importantly it simulated realistic profiles that resembled the real data and canceled out the artifacts introduced by the elevated RUV, even in the case of the heavily misspecified model. Furthermore, a metric to quantify the extent of model misspecification was introduced based on the R2 between IRES and IRESML, following the rationale that the greater the extent of variability explained by the ML model, the higher the degree of model misspecification present in the original model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69296a29a3089d2ff48e2bc6b1e62f03d5894045" target='_blank'>
              Simulating realistic patient profiles from pharmacokinetic models by a machine learning postprocessing correction of residual variability.
              </a>
            </td>
          <td>
            Christos Kaikousidis, Robert R Bies, A. Dokoumetzidis
          </td>
          <td>2024-06-14</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55aa55ce9bae4e173f97d3be1d9cc988e4eeeb71" target='_blank'>
              Property-guided generation of complex polymer topologies using variational autoencoders
              </a>
            </td>
          <td>
            Shengli Jiang, Adji B. Dieng, Michael A. Webb
          </td>
          <td>2024-06-29</td>
          <td>npj Computational Materials</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Influenced by breakthroughs in LLMs, single-cell foundation models are emerging. While these models show successful performance in cell type clustering, phenotype classification, and gene perturbation response prediction, it remains to be seen if a simpler model could achieve comparable or better results, especially with limited data. This is important, as the quantity and quality of single-cell data typically fall short of the standards in textual data used for training LLMs. Single-cell sequencing often suffers from technical artifacts, dropout events, and batch effects. These challenges are compounded in a weakly supervised setting, where the labels of cell states can be noisy, further complicating the analysis. To tackle these challenges, we present sc-OTGM, streamlined with less than 500K parameters, making it approximately 100x more compact than the foundation models, offering an efficient alternative. sc-OTGM is an unsupervised model grounded in the inductive bias that the scRNAseq data can be generated from a combination of the finite multivariate Gaussian distributions. The core function of sc-OTGM is to create a probabilistic latent space utilizing a GMM as its prior distribution and distinguish between distinct cell populations by learning their respective marginal PDFs. It uses a Hit-and-Run Markov chain sampler to determine the OT plan across these PDFs within the GMM framework. We evaluated our model against a CRISPR-mediated perturbation dataset, called CROP-seq, consisting of 57 one-gene perturbations. Our results demonstrate that sc-OTGM is effective in cell state classification, aids in the analysis of differential gene expression, and ranks genes for target identification through a recommender system. It also predicts the effects of single-gene perturbations on downstream gene regulation and generates synthetic scRNA-seq data conditioned on specific cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17340a9e8433044198b5184a2bd57e4cd4f8ad2b" target='_blank'>
              sc-OTGM: Single-Cell Perturbation Modeling by Solving Optimal Mass Transport on the Manifold of Gaussian Mixtures
              </a>
            </td>
          <td>
            Andac Demir, E. Solovyeva, James Boylan, Mei Xiao, Fabrizio Serluca, S. Hoersch, Jeremy L Jenkins, Murthy S. Devarakonda, B. Kiziltan
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="When studying ecosystems, hierarchical trees are often used to organize entities based on proximity criteria, such as the taxonomy in microbiology, social classes in geography, or product types in retail businesses, offering valuable insights into entity relationships. Despite their significance, current count-data models do not leverage this structured information. In particular, the widely used Poisson log-normal (PLN) model, known for its ability to model interactions between entities from count data, lacks the possibility to incorporate such hierarchical tree structures, limiting its applicability in domains characterized by such complexities. To address this matter, we introduce the PLN-Tree model as an extension of the PLN model, specifically designed for modeling hierarchical count data. By integrating structured variational inference techniques, we propose an adapted training procedure and establish identifiability results, enhancisng both theoretical foundations and practical interpretability. Additionally, we extend our framework to classification tasks as a preprocessing pipeline, showcasing its versatility. Experimental evaluations on synthetic datasets as well as real-world microbiome data demonstrate the superior performance of the PLN-Tree model in capturing hierarchical dependencies and providing valuable insights into complex data structures, showing the practical interest of knowledge graphs like the taxonomy in ecosystems modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86dd1a5a64cacf81f6895c4656147dbabe2e7e27" target='_blank'>
              Tree-based variational inference for Poisson log-normal models
              </a>
            </td>
          <td>
            Alexandre Chaussard, Anna Bonnet, Elisabeth Gassiat, S. Corff
          </td>
          <td>2024-06-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Computational chemistry is an indispensable tool for understanding molecules and predicting chemical properties. However, traditional computational methods face significant challenges due to the difficulty of solving the Schrödinger equations and the increasing computational cost with the size of the molecular system. In response, there has been a surge of interest in leveraging artificial intelligence (AI) and machine learning (ML) techniques to in silico experiments. Integrating AI and ML into computational chemistry increases the scalability and speed of the exploration of chemical space. However, challenges remain, particularly regarding the reproducibility and transferability of ML models. This review highlights the evolution of ML in learning from, complementing, or replacing traditional computational chemistry for energy and property predictions. Starting from models trained entirely on numerical data, a journey set forth toward the ideal model incorporating or learning the physical laws of quantum mechanics. This paper also reviews existing computational methods and ML models and their intertwining, outlines a roadmap for future research, and identifies areas for improvement and innovation. Ultimately, the goal is to develop AI architectures capable of predicting accurate and transferable solutions to the Schrödinger equation, thereby revolutionizing in silico experiments within chemistry and materials science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c965a92a4418b12137dd3227bb9be135171b111" target='_blank'>
              In Silico Chemical Experiments in the Age of AI: From Quantum Chemistry to Machine Learning and Back.
              </a>
            </td>
          <td>
            Abdulrahman Aldossary, J. A. Campos-Gonzalez-Angulo, Sergio Pablo-García, Shi Xuan Leong, Ella Miray Rajaonson, Luca Thiede, Gary Tom, Andrew Wang, Davide Avagliano, Alán Aspuru-Guzik
          </td>
          <td>2024-05-25</td>
          <td>Advanced materials</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The ability to accurately predict disease progression is paramount for optimizing multiple myeloma patient care. This study introduces a hybrid neural network architecture, combining Long Short-Term Memory networks with a Conditional Restricted Boltzmann Machine, to predict future blood work of affected patients from a series of historical laboratory results. We demonstrate that our model can replicate the statistical moments of the time series ($0.95~\pm~0.01~\geq~R^2~\geq~0.83~\pm~0.03$) and forecast future blood work features with high correlation to actual patient data ($0.92\pm0.02~\geq~r~\geq~0.52~\pm~0.09$). Subsequently, a second Long Short-Term Memory network is employed to detect and annotate disease progression events within the forecasted blood work time series. We show that these annotations enable the prediction of progression events with significant reliability (AUROC$~=~0.88~\pm~0.01$), up to 12 months in advance (AUROC($t+12~$mos)$~=0.65~\pm~0.01$). Our system is designed in a modular fashion, featuring separate entities for forecasting and progression event annotation. This structure not only enhances interpretability but also facilitates the integration of additional modules to perform subsequent operations on the generated outputs. Our approach utilizes a minimal set of routine blood work measurements, which avoids the need for expensive or resource-intensive tests and ensures accessibility of the system in clinical routine. This capability allows for individualized risk assessment and making informed treatment decisions tailored to a patient's unique disease kinetics. The represented approach contributes to the development of a scalable and cost-effective virtual human twin system for optimized healthcare resource utilization and improved patient outcomes in multiple myeloma care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720eddebaebf55df59d7ccef7e7c2748b0a79603" target='_blank'>
              Predicting Progression Events in Multiple Myeloma from Routine Blood Work
              </a>
            </td>
          <td>
            Maximilian Ferle, N. Grieb, Markus Kreuz, U. Platzbecker, Thomas Neumuth, Kristin Reiche, Alexander Oeser, M. Merz
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Models have always been central to inferring molecular evolution and to reconstructing phylogenetic trees. Their use typically involves the development of a mechanistic framework reflecting our understanding of the underlying biological processes, such as nucleotide substitu- tions, and the estimation of model parameters by maximum likelihood or Bayesian inference. However, deriving and optimizing the likelihood of the data is not always possible under complex evolutionary scenarios or even tractable for large datasets, often leading to unrealistic simplifying assumptions in the fitted models. To overcome this issue, we coupled stochastic simulations of genome evolution with a new supervised deep learning model to infer key parameters of molecular evolution. Our model is designed to directly analyze multiple sequence alignments and estimate per-site evolutionary rates and divergence, without requiring a known phylogenetic tree. The accuracy of our predictions matched that of likelihood-based phylogenetic inference, when rate heterogeneity followed a simple gamma distribution, but it strongly exceeded it under more complex patterns of rate variation, such as codon models. Our approach is highly scalable and can be efficiently applied to genomic data, as we showed on a dataset of 26 million nucleotides from the clownfish clade. Our simulations also showed that the integration of per-site rates obtained by deep learning within a Bayesian framework led to significantly more accu- rate phylogenetic inference, particularly with respect to the estimated branch lengths. We thus propose that future advancements in phylogenetic analysis will benefit from a semi-supervised learning approach that combines deep-learning estimation of substitution rates, which allows for more flexible models of rate variation, and probabilistic inference of the phylogenetic tree, which guarantees interpretability and a rigorous assessment of statistical support.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e5726c0265f212865a58fafea78c3fde46d235" target='_blank'>
              Toward a semi-supervised learning approach to phylogenetic estimation.
              </a>
            </td>
          <td>
            Daniele Silvestro, Thibault Latrille, N. Salamin
          </td>
          <td>2024-06-25</td>
          <td>Systematic biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="MOTIVATION
Accurately predicting the driver genes of cancer is of great significance for carcinogenesis progress research and cancer treatment. In recent years, more and more deep-learning-based methods have been used for predicting cancer driver genes. However, deep learning algorithms often have black box properties and cannot interpret the output results. Here, we propose a novel cancer driver gene mining method based on heterogeneous network metapaths (MCDHGN), which uses meta-path aggregation to enhance the interpretability of predictions.


RESULTS
MCDHGN constructs a heterogeneous network by using several types of multi-omics data that are biologically linked to genes. And the differential probabilities of SNV, DNA methylation, and gene expression data between cancerous tissues and normal tissues are extracted as initial features of genes. Nine meta-paths are manually selected, and the representation vectors obtained by aggregating information within and across metapath nodes are used as new features for subsequent classification and prediction tasks. By comparing with eight homogeneous and heterogeneous network models on two pan-cancer datasets, MCDHGN has better performance on AUC and AUPR values. Additionally, MCDHGN provides interpretability of predicted cancer driver genes through the varying weights of biologically meaningful meta-paths.


AVAILABILITY
https://github.com/1160300611/MCDHGN.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b32a7b3611c1e300542c76e55e6a0bb0480476b" target='_blank'>
              MCDHGN: Heterogeneous network-based cancer driver gene prediction and interpretability analysis.
              </a>
            </td>
          <td>
            Lexiang Wang, Jingli Zhou, Xuan Wang, Yadong Wang, Junyi Li
          </td>
          <td>2024-06-12</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Scientific Machine Learning is a new class of approaches that integrate physical knowledge and mechanistic models with data-driven techniques for uncovering governing equations of complex processes. Among the available approaches, Universal Differential Equations (UDEs) are used to combine prior knowledge in the form of mechanistic formulations with universal function approximators, like neural networks. Integral to the efficacy of UDEs is the joint estimation of parameters within mechanistic formulations and the universal function approximators using empirical data. The robustness and applicability of resultant models, however, hinge upon the rigorous quantification of uncertainties associated with these parameters, as well as the predictive capabilities of the overall model or its constituent components. With this work, we provide a formalisation of uncertainty quantification (UQ) for UDEs and investigate important frequentist and Bayesian methods. By analysing three synthetic examples of varying complexity, we evaluate the validity and efficiency of ensembles, variational inference and Markov chain Monte Carlo sampling as epistemic UQ methods for UDEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320dbbdecad0c6895c5ce6a4b03811b5cb0b85c8" target='_blank'>
              Assessment of Uncertainty Quantification in Universal Differential Equations
              </a>
            </td>
          <td>
            Nina Schmid, David Fernandes del Pozo, Willem Waegeman, Jan Hasenauer
          </td>
          <td>2024-06-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the context of transcriptional dynamics modeled by ordinary differential equations (ODEs), the RNA level in a single cell is controlled by specific RNA kinetics parameters, which include transcription rate, splicing rate, and degradation rate. Investigating these single-cell RNA kinetics rates is pivotal for understanding RNA metabolism and the heterogeneity of complex tissues. Although metabolic labeling is an effective method to estimate these kinetics rates experimentally, it is not suitable for current large-scale conventional single-cell RNA sequencing (scRNA-seq) data. Moreover, existing methods for scRNA-seq often either neglect certain specific kinetics parameters or use inappropriate ways to fit the parameters. To address these issues, we introduce scRNAkinetics, a parallelized method that fits the kinetics parameters of the ODE for each cell using pseudo-time derived from various biological priors (e.g. cell lineage tree and differentiation potential). This approach allows for the estimation of the relative kinetics of each cell and gene in a scRNA-seq dataset. Validated on simulated datasets, scRNAkinetics can accurately infer the kinetics rates of transcription boosting, multi-branch, and time-dependent RNA degradation systems. Nevertheless, the inferred kinetics trends are concordant with previous studies on metabolic labeling and conventional scRNA-seq datasets. Furthermore, we show that scRNAkinetics can provide valuable insights into different regulatory schemes and validate the coupling between transcription and splicing in RNA metabolism. The open-source implementation of scRNAkinetics is available at https://github.com/poseidonchan/scRNAkinetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94f71a1f0958ca99a7ef6222f0f0c7006b2e729" target='_blank'>
              Inferring Single-Cell RNA Kinetics from Various Biological Priors
              </a>
            </td>
          <td>
            Yanshuo Chen, Zhengmian Hu, Ke Ni, Site Feng, Wei Chen, Heng Huang
          </td>
          <td>2024-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Machine learning (ML) algorithms are necessary to efficiently identify potent drug combinations within a large candidate space to combat drug resistance. However, existing ML approaches cannot be applied to emerging and under-studied pathogens with limited training data. To address this, we developed a transfer learning and crowdsourcing framework (TACTIC) to train ML models on data from multiple bacteria. TACTIC was built using 2,965 drug interactions from 12 bacterial strains and outperformed traditional ML models in predicting drug interaction outcomes for species that lack training data. Top TACTIC model features revealed genetic and metabolic factors that influence cross- species and species-specific drug interaction outcomes. Upon analyzing ∼600,000 predicted drug interactions across 9 metabolic environments and 18 bacterial strains, we identified a small set of drug interactions that are selectively synergistic against Gram- negative (e.g., A. baumannii) and non-tuberculous mycobacteria (NTM) pathogens. We experimentally validated synergistic drug combinations containing clarithromycin, ampicillin, and mecillinam against M. abscessus, an emerging pathogen with growing levels of antibiotic resistance. Lastly, we leveraged TACTIC to propose selectively synergistic drug combinations to treat bacterial eye infections (endophthalmitis).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5017bc92dbb38d49a788006dff57d7610ff49b95" target='_blank'>
              Transfer learning predicts species-specific drug interactions in emerging pathogens
              </a>
            </td>
          <td>
            Carolina H. Chung, David C. Chang, Nicole M. Rhoads, Madeline R. Shay, Karthik Srinivasan, M. Okezue, Ashlee D. Brunaugh, Sriram Chandrasekaran
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Determining the aqueous solubility of the chemical compound is of great importance in-silico drug discovery. However, correctly and rapidly predicting the aqueous solubility remains a challenging task. This paper explores and evaluates the predictability of multiple machine learning models in the aqueous solubility of compounds. Specifically, we apply a series of machine learning algorithms, including Random Forest, XG-Boost, LightGBM, and CatBoost, on a well-established aqueous solubility dataset (i. e., the Huuskonen dataset) of over 1200 compounds. Experimental results show that even traditional machine learning algorithms can achieve satisfactory performance with high accuracy. In addition, our investigation goes beyond mere prediction accuracy, delving into the interpretability of models to identify key features and understand the molecular properties that influence the predicted outcomes. This study sheds light on the ability to use machine learning approaches to predict compound solubility, significantly shortening the time that researchers spend on new drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edea23b8b0e1d8e76000d6da3d598ce6e99db3ee" target='_blank'>
              Evaluation of Machine Learning Models for Aqueous Solubility Prediction in Drug Discovery
              </a>
            </td>
          <td>
            Nian Xue, Yuzhu Zhang, Sensen Liu
          </td>
          <td>2024-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The study of effective connectivity (EC) is essential in understanding how the brain integrates and responds to various sensory inputs. Model-driven estimation of EC is a powerful approach that requires estimating global and local parameters of a generative model of neural activity. Insights gathered through this process can be used in various applications, such as studying neurodevelopmental disorders. However, accurately determining EC through generative models remains a significant challenge due to the complexity of brain dynamics and the inherent noise in neural recordings, e.g., in electroencephalography (EEG). Current model-driven methods to study EC are computationally complex and cannot scale to all brain regions as required by whole-brain analyses. To facilitate EC assessment, an inference algorithm must exhibit reliable prediction of parameters in the presence of noise. Further, the relationship between the model parameters and the neural recordings must be learnable. To progress toward these objectives, we benchmarked the performance of a Bi-LSTM model for parameter inference from the Jansen-Rit neural mass model (JR-NMM) simulated EEG under various noise conditions. Additionally, our study explores how the JR-NMM reacts to changes in key biological parameters (i.e., sensitivity analysis) like synaptic gains and time constants, a crucial step in understanding the connection between neural mechanisms and observed brain activity. Our results indicate that we can predict the local JR-NMM parameters from EEG, supporting the feasibility of our deep-learning-based inference approach. In future work, we plan to extend this framework to estimate local and global parameters from real EEG in clinically relevant applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd046ab8f0278c72f414f856472d80db3160b67" target='_blank'>
              Benchmarking Deep Jansen-Rit Parameter Inference: An in Silico Study
              </a>
            </td>
          <td>
            Deepa Tilwani, Christian O'Reilly
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell time series data frequently display considerable variability across a cell population. The current gold standard for inferring parameter distributions across cell populations is the Global Two Stage (GTS) approach for nonlinear mixed-effects (NLME) models. However, this method is computationally intensive, as it makes repeated use of non-convex optimization that in turn requires numerical integration of the underlying system. Here, we propose the Gradient Matching GTS (GMGTS) method as an efficient alternative to GTS. Gradient matching offers an integration-free approach to parameter estimation that is particularly powerful for dynamical systems that are linear in the unknown parameters, such as biochemical networks modeled by mass action kinetics. Here, we harness the power of gradient matching by integrating it into the GTS framework. To this end, we significantly expand the capabilities of gradient matching via uncertainty propagation calculations and the development of an iterative estimation scheme for partially observed systems. Through comparisons of GMGTS with GTS in different inference setups, we demonstrate that our method provides a significant computational advantage, thereby facilitating the use of complex NLME models in systems biology applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dced58f5c5737541e67c9ad6ec12731d8224877" target='_blank'>
              Gradient matching accelerates mixed-effects inference for biochemical networks
              </a>
            </td>
          <td>
            Yulan B van Oppen, Andreas Milias-Argeitis
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Drug safety relies on advanced methods for timely and accurate prediction of side effects. To tackle this requirement, this scoping review examines machine-learning approaches for predicting drug-related side effects with a particular focus on chemical, biological, and phenotypical features. Methods: This was a scoping review in which a comprehensive search was conducted in various databases from 1 January 2013 to 31 December 2023. Results: The results showed the widespread use of Random Forest, k-nearest neighbor, and support vector machine algorithms. Ensemble methods, particularly random forest, emphasized the significance of integrating chemical and biological features in predicting drug-related side effects. Conclusions: This review article emphasized the significance of considering a variety of features, datasets, and machine learning algorithms for predicting drug-related side effects. Ensemble methods and Random Forest showed the best performance and combining chemical and biological features improved prediction. The results suggested that machine learning techniques have some potential to improve drug development and trials. Future work should focus on specific feature types, selection techniques, and graph-based methods for even better prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b346d3270a77c7b4d467e797b36b5b738915e328" target='_blank'>
              Machine Learning Techniques for Predicting Drug-Related Side Effects: A Scoping Review
              </a>
            </td>
          <td>
            Esmaeel Toni, H. Ayatollahi, Reza Abbaszadeh, Alireza Fotuhi Siahpirani
          </td>
          <td>2024-06-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cardiomyopathies, often caused by mutations in genes encoding muscle proteins, are traditionally treated by phenotyping hearts and addressing symptoms post irreversible damage. With advancements in genotyping, early diagnosis is now possible, potentially preventing such damage. However, the intricate structure of muscle and its myriad proteins make treatment predictions challenging. Here we approach the problem of estimating therapeutic targets for a mutation in mouse muscle using a spatially explicit half sarcomere muscle model. We selected 9 rate parameters in our model linked to both small molecules and cardiomyopathy-causing mutations. We then randomly varied these rate parameters and simulated an isometric twitch for each combination to generate a large training dataset. We used this dataset to train a Conditional Variational Autoencoder (CVAE), a technique used in Bayesian parameter estimation. Given simulated or experimental isometric twitches, this machine learning model is able to then predict the set of rate parameters which are most likely to yield that result. We then predict the set of rate parameters associated with both control and the cardiac Troponin C (cTnC) I61Q variant in mouse trabeculae and model parameters that recover the abnormal I61Q cTnC twitches. SIGNIFICANCE Machine learning techniques have potential to accelerate discoveries in biologically complex systems. However, they require large data sets and can be challenging in high dimensional systems such as cardiac muscle. In this study, we combined experimental measures of cardiac muscle twitch forces with mechanistic simulations and a newly developed mixture of Bayesian inference with neural networks (in autoencoders) to solve the inverse problem of determining the underlying kinetics for observed force generation by cardiac muscle. The autoencoders are trained on millions of simulations spanning parameter spaces that correspond to the mechanochemistry of cardiac sarcomeres. We apply the trained model to experimental data in order to infer parameters that can explain a diseased twitch and ways to recover it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/898beeadaf2f7a68b5dd4751723ce9ccd2389b02" target='_blank'>
              Identifying mechanisms and therapeutic targets in muscle using Bayesian parameter estimation with conditional variational autoencoders
              </a>
            </td>
          <td>
            Travis Tune, Kristina B. Kooiker, Jennifer Davis, Thomas L. Daniel, F. Moussavi-Harami
          </td>
          <td>2024-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="We propose a novel method, scTree, for single-cell Tree Variational Autoencoders, extending a hierarchical clustering approach to single-cell RNA sequencing data. scTree corrects for batch effects while simultaneously learning a tree-structured data representation. This VAE-based method allows for a more in-depth understanding of complex cellular landscapes independently of the biasing effects of batches. We show empirically on seven datasets that scTree discovers the underlying clusters of the data and the hierarchical relations between them, as well as outperforms established baseline methods across these datasets. Additionally, we analyze the learned hierarchy to understand its biological relevance, thus underpinning the importance of integrating batch correction directly into the clustering procedure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0110e13e861d81d662028615d0d5e03a00bed2d" target='_blank'>
              scTree: Discovering Cellular Hierarchies in the Presence of Batch Effects in scRNA-seq Data
              </a>
            </td>
          <td>
            Moritz Vandenhirtz, Florian Barkmann, Laura Manduchi, Julia E. Vogt, Valentina Boeva
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Despite recent biotechnological breakthroughs, cancer risk prediction remains a formidable computational and experimental challenge. Addressing it is critical in order to improve prevention, early detection and survival rates. Here, I briefly summarize some key emerging theoretical and computational challenges as well as recent computational advances that promise to help realize the goals of cancer-risk prediction. The focus is on computational strategies based on single-cell data, in particular on bottom-up network modeling approaches that aim to estimate cancer stemness and dedifferentiation at single-cell resolution from a systems-biological perspective. I will describe two promising methods, a tissue and cell-lineage independent one based on the concept of diffusion network entropy, and a tissue and cell-lineage specific one that uses transcription factor regulons. Application of these tools to single-cell and single-nucleus RNA-seq data from stages prior to invasive cancer reveal that they can successfully delineate the heterogeneous inter-cellular cancer-risk landscape, identifying those cells that are more likely to turn cancerous. Bottom-up systems biological modeling of single-cell omic data is a novel computational analysis paradigm that promises to facilitate the development of preventive, early detection and cancer-risk prediction strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d7a13e6b2f47623a33253ae61889093d174b00d" target='_blank'>
              Computational single-cell methods for predicting cancer risk.
              </a>
            </td>
          <td>
            Andrew E Teschendorff
          </td>
          <td>2024-06-10</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute Myocardial Infarction (AMI), a common disease that can have serious consequences, occurs when myocardial blood flow stops due to occlusion of the coronary artery. Early and accurate prediction of AMI is critical for rapid prognosis and improved patient outcomes. Metabolomics, the study of small molecules within biological systems, is an effective tool used to discover biomarkers associated with many diseases. This study intended to construct a predictive model for AMI utilizing metabolomics data and an explainable machine learning approach called Explainable Boosting Machines (EBM). The EBM model was trained on a dataset of 102 prognostic metabolites gathered from 99 individuals, including 34 healthy controls and 65 AMI patients. After a comprehensive data preprocessing, 21 metabolites were determined as the candidate predictors to predict AMI. The EBM model displayed satisfactory performance in predicting AMI, with various classification performance metrics. The model’s predictions were based on the combined effects of individual metabolites and their interactions. In this context, the results obtained in two different EBM modeling, including both only individual metabolite features and their interaction effects, were discussed. The most important predictors included creatinine, nicotinamide, and isocitrate. These metabolites are involved in different biological activities, such as energy metabolism, DNA repair, and cellular signaling. The results demonstrate the potential of the combination of metabolomics and the EBM model in constructing reliable and interpretable prediction outputs for AMI. The discussed metabolite biomarkers may assist in early diagnosis, risk assessment, and personalized treatment methods for AMI patients. This study successfully developed a pipeline incorporating extensive data preprocessing and the EBM model to identify potential metabolite biomarkers for predicting AMI. The EBM model, with its ability to incorporate interaction terms, demonstrated satisfactory classification performance and revealed significant metabolite interactions that could be valuable in assessing AMI risk. However, the results obtained from this study should be validated with studies to be carried out in larger and well-defined samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2eefc7a5a6dd6ee4a7ea1bd2688c9f1437cfb7" target='_blank'>
              Combining the Strengths of the Explainable Boosting Machine and Metabolomics Approaches for Biomarker Discovery in Acute Myocardial Infarction
              </a>
            </td>
          <td>
            A. Arslan, F. Yağın, Abdulmohsen Algarni, F. Al-Hashem, L. P. Ardigò
          </td>
          <td>2024-06-26</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Reconstructing gene regulatory networks (GRNs) using single-cell RNA sequencing (scRNA-seq) data holds great promise for unraveling cellular fate development and heterogeneity. While numerous machine-learning methods have been proposed to infer GRNs from scRNA-seq gene expression data, many of them operate solely in a statistical or black box manner, limiting their capacity for making causal inferences between genes. In this study, we introduce GRN inference with Accuracy and Causal Explanation (GRACE), a novel graph-based causal autoencoder framework that combines a structural causal model (SCM) with graph neural networks (GNNs) to enable GRN inference and gene causal reasoning from scRNA-seq data. By explicitly modeling causal relationships between genes, GRACE facilitates the learning of regulatory context and gene embeddings. With the learned gene signals, our model successfully decoding the causal structures and alleviates the accurate determination of multiple attributes of gene regulation that is important to determine the regulatory levels. Through extensive evaluations on seven benchmarks, we demonstrate that GRACE outperforms 14 state-of-the-art GRN inference methods, with the incorporation of causal mechanisms significantly enhancing the accuracy of GRN and gene causality inference. Furthermore, the application to human peripheral blood mononuclear cell (PBMC) samples reveals cell type-specific regulators in monocyte phagocytosis and immune regulation, validated through network analysis and functional enrichment analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78acc67c6cfb1020b9bf2b2c2f841815e9417f9d" target='_blank'>
              GRACE: Unveiling Gene Regulatory Networks With Causal Mechanistic Graph Neural Networks in Single-Cell RNA-Sequencing Data.
              </a>
            </td>
          <td>
            Jiacheng Wang, Yao-Jia Chen, Quan Zou
          </td>
          <td>2024-06-19</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Liquid biopsy demonstrates excellent potential in patient management by providing a minimally invasive and cost-effective approach to detecting and monitoring cancer, even at its early stages. Due to the complexity of liquid biopsy data, machine-learning techniques are increasingly gaining attention in sample analysis, especially for multidimensional data such as RNA expression profiles. Yet, there is no agreement in the community on which methods are the most effective or how to process the data. To circumvent this, we performed a large-scale study using various machine-learning techniques. First, we took a closer look at existing datasets and filtered out some patients to assert data collection quality. The final data collection included platelet RNA samples acquired from 1397 cancer patients (17 types of cancer) and 354 asymptomatic, presumed healthy, donors. Then, we assessed an array of different machine-learning models and techniques (e.g., feature selection of RNA transcripts) in pan-cancer detection and multiclass classification. Our results show that simple logistic regression performs the best, reaching a 68% cancer detection rate at a 99% specificity level, and multiclass classification accuracy of 79.38% when distinguishing between five cancer types. In summary, by revisiting classical machine-learning models, we have exceeded the previously used method by 5% and 9.65% in cancer detection and multiclass classification, respectively. To ease further research, we open-source our code and data processing pipelines (https://gitlab.com/jopekmaksym/improving-platelet-rna-based-diagnostics), which we hope will serve the community as a strong baseline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/479013c1f2cb89512c4830381ffb51b56ae291e2" target='_blank'>
              Improving platelet-RNA-based diagnostics: a comparative analysis of machine learning models for cancer detection and multiclass classification.
              </a>
            </td>
          <td>
            M. Jopek, Krzysztof Pastuszak, Michal Sieczczynski, Sebastian Cygert, A. Żaczek, Matthew T Rondina, A. Supernat
          </td>
          <td>2024-06-17</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The assessment of compound blood-brain barrier (BBB) permeability poses a significant challenge in the discovery of drugs targeting the central nervous system. Conventional experimental approaches to measure BBB permeability are labor-intensive, cost-ineffective, and time-consuming. In this study, we constructed six machine learning classification models by combining various machine learning algorithms and molecular representations. The model based on ExtraTree algorithm and random partitioning strategy obtains the best prediction result, with AUC value of 0.932±0.004 and balanced accuracy (BA) of 0.837±0.010 for the test set. We employed the SHAP method to identify important features associated with BBB permeability. In addition, matched molecular pair (MMP) analysis and representative substructure derivation method were utilized to uncover the transformation rules and distinctive structural features of BBB permeable compounds. The machine learning models proposed in this work can serve as an effective tool for assessing BBB permeability in the drug discovery for central nervous system disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56e1049bac83cc9b11daadbd100b52480b127ec9" target='_blank'>
              Prediction of blood-brain barrier permeability using machine learning approaches based on various molecular representation.
              </a>
            </td>
          <td>
            Li Liang, Zhiwen Liu, Xinyi Yang, Yanmin Zhang, Haichun Liu, Yadong Chen
          </td>
          <td>2024-06-12</td>
          <td>Molecular informatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Recently, Conditional Neural Fields (NeFs) have emerged as a powerful modelling paradigm for PDEs, by learning solutions as flows in the latent space of the Conditional NeF. Although benefiting from favourable properties of NeFs such as grid-agnosticity and space-time-continuous dynamics modelling, this approach limits the ability to impose known constraints of the PDE on the solutions -- e.g. symmetries or boundary conditions -- in favour of modelling flexibility. Instead, we propose a space-time continuous NeF-based solving framework that - by preserving geometric information in the latent space - respects known symmetries of the PDE. We show that modelling solutions as flows of pointclouds over the group of interest $G$ improves generalization and data-efficiency. We validated that our framework readily generalizes to unseen spatial and temporal locations, as well as geometric transformations of the initial conditions - where other NeF-based PDE forecasting methods fail - and improve over baselines in a number of challenging geometries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8295fdfb6de078027cabc7d49b319df8abb0ee" target='_blank'>
              Space-Time Continuous PDE Forecasting using Equivariant Neural Fields
              </a>
            </td>
          <td>
            David M. Knigge, David R. Wessels, Riccardo Valperga, Samuele Papa, J. Sonke, E. Gavves, E. J. Bekkers
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>56</td>
        </tr>

        <tr id="The obscure drug response continues to be a limiting factor for accurate cures for cancer. Next generation sequencing technologies have propelled the pharmacogenomic studies with characterized large panels of cancer cell line at multi‐omics level. However, the sufficient integration of the multi‐omics data and the efficient prediction for drug response and synergy still remain a challenge. To address these problems, ECFD is designed, an ensemble cascade forest‐based framework that predicts drug response and synergy using five types of omics data. Experimental results show the significant advantages of the ECFD model over existing models. The best integration of feature extraction is determined and the superiorities of robust stability in the face of new and small samples are highlighted. In addition, the methodological framework highlights the explainability of the model, the mechanisms of drug resistance and drug combination treatment strategies based on explainable analyses and biological networks. In sum, ECFD may facilitate the evaluation of drug response and speculation of potential synergy therapies in personalized and precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202be07a703e58d34b9d528b6a979074db9400b4" target='_blank'>
              An Ensemble Cascade Forest‐Based Framework for Multi‐Omics Drug Response and Synergy Prediction
              </a>
            </td>
          <td>
            Ruijiang Li, Binsheng Sui, Dongjing Leng, Song He, Kunhong Liu, Xiaochen Bo
          </td>
          <td>2024-06-09</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="MOTIVATION
Predicting cancer drug response requires a comprehensive assessment of many mutations present across a tumor genome. While current drug response models generally use a binary mutated/unmutated indicator for each gene, not all mutations in a gene are equivalent.


RESULTS
Here, we construct and evaluate a series of predictive models based on leading methods for quantitative mutation scoring. Such methods include VEST4 and CADD, which score the impact of a mutation on gene function, and CHASMplus, which scores the likelihood a mutation drives cancer. The resulting predictive models capture cellular responses to dabrafenib, which targets BRAF-V600 mutations, whereas models based on binary mutation status do not. Performance improvements generalize to other drugs, extending genetic indications for PIK3CA, ERBB2, EGFR, PARP1, and ABL1 inhibitors. Introducing quantitative mutation features in drug response models increases performance and mechanistic understanding.


AVAILABILITY AND IMPLEMENTATION
Code and example datasets are available at https://github.com/pgwall/qms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b78691ef04081b6ccf1c8699cd4747a45bf1d4f" target='_blank'>
              Representing mutations for predicting cancer drug response.
              </a>
            </td>
          <td>
            Patrick Wall, T. Ideker
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="e13641 Background: Patient eligibility is a routine concept in clinical research, comprised of simple or complex set of inclusion or exclusion criteria applied in a specific sequence. Our methodology introduces the concept of ‘availability’ as a narrower term that is closer to the actual number that will be considered for a specific study and who may decide to participate. This specific research develops Machine Learning (ML) model that dynamically predicts likelihood of Multiple Myeloma progression and departure from targeted outcomes to sustain therapy and to indicate potential availability for clinical trials – to reduce screening efforts and improve screening precision of clinical research organizations. Methods: The cohort for training these models was drawn from the ConcertAI Oncology Research database, further enriched by key variables derived from unstructured data sources. Eligible patients for the analysis had a diagnosis of Multiple Myeloma made after Jan 1, 2020. Progressions were labelled using International Myeloma Working Group Criteria. Patients that did not satisfy the criteria were censored. Labels were assigned to patients after initial diagnosis of MM, and the duration between diagnosis to label was computed to build time to event (TTE) models. Over 2000 features were created to build this model, e.g., laboratory values, vitals, biomarkers, medications, ECOG, stage, etc. Variable creation preserved temporality of lab variables and vitals and noted presence of other variables. Several ML methods were employed to build the TTE models, with gradient boosting resulting in the highest c-index (0.8) based on 12 clinically relevant features. A temporal validation of the model was performed on a simulated index date. Results: Cohort used for training the model comprised of N = 4315 patients with average age at diagnosis of 72.3 years, and ~11% of the patients labelled as progressing. Temporal validation was performed on N = 1874 patients, with ~9% progression rate. The area under the curve (AUC) and weighted F1 score for this cohort was 87% and 89% respectively. Conclusions: Patient availability assessments are complex, not supported effectively by existing solutions and often require 100’s of hours from highly trained clinical personnel. An AI model predicting Multiple Myeloma progression built using a large-scale real-world dataset demonstrated 2-4x improvement in forecasting patient availability over other approaches. Integrating these solutions into clinical workflows can improve the effectiveness of research teams; enable more patients to have clinical trials as viable options at the right time; and allow care teams to have forward visibility to choices of standard-of-care and clinical trial alternatives in the best interest of patient outcomes. [Table: see text]">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71f96261be2e68d4c685fffbbf7d6bf43b30dfae" target='_blank'>
              Dynamic machine learning model to forecast patient availability for clinical trials.
              </a>
            </td>
          <td>
            Rajeev Kulkarni, Vivek Prabhakar Vaidya, Dhaval Parmar, Abhishek Tibrewal, R. B. Parikh
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Exploring the degree to which phenotypic variation, influenced by intrinsic nonlinear biological mechanisms, can be accurately captured using statistical methods is essential for advancing our comprehension of complex biological systems and predicting their functionality. Here, we examine this issue by combining a computational model of gene regulation networks with a linear additive prediction model, akin to polygenic scores utilized in genetic analyses. Inspired by the variational framework of quantitative genetics, we create a population of individual networks possessing identical topology yet showcasing diversity in regulatory strengths. By discerning which regulatory connections determine the prediction of phenotypes, we contextualize our findings within the framework of core and peripheral causal determinants, as proposed by the omnigenic model of complex traits. We establish connections between our results and concepts such as global sensitivity and local stability in dynamical systems, alongside the notion of sloppy parameters in biological models. Furthermore, we explore the implications of our investigation for the broader discourse surrounding the role of epistatic interactions in the prediction of complex phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9d655e857d6cdd755c46f5fbfbbbb85e241f74" target='_blank'>
              Polygenic prediction and gene regulation networks
              </a>
            </td>
          <td>
            J. F. Poyatos
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction With the increasing prevalence of type 2 diabetes mellitus (T2DM), there is an urgent need to discover effective therapeutic targets for this complex condition. Coding and non-coding RNAs, with traditional biochemical parameters, have shown promise as viable targets for therapy. Machine learning (ML) techniques have emerged as powerful tools for predicting drug responses. Method In this study, we developed an ML-based model to identify the most influential features for drug response in the treatment of type 2 diabetes using three medicinal plant-based drugs (Rosavin, Caffeic acid, and Isorhamnetin), and a probiotics drug (Z-biotic), at different doses. A hundred rats were randomly assigned to ten groups, including a normal group, a streptozotocin-induced diabetic group, and eight treated groups. Serum samples were collected for biochemical analysis, while liver tissues (L) and adipose tissues (A) underwent histopathological examination and molecular biomarker extraction using quantitative PCR. Utilizing five machine learning algorithms, we integrated 32 molecular features and 12 biochemical features to select the most predictive targets for each model and the combined model. Results and discussion Our results indicated that high doses of the selected drugs effectively mitigated liver inflammation, reduced insulin resistance, and improved lipid profiles and renal function biomarkers. The machine learning model identified 13 molecular features, 10 biochemical features, and 20 combined features with an accuracy of 80% and AUC (0.894, 0.93, and 0.896), respectively. This study presents an ML model that accurately identifies effective therapeutic targets implicated in the molecular pathways associated with T2DM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884985e25d267f735c330d0f1ae8ec579a0bd05e" target='_blank'>
              Comprehensive machine learning models for predicting therapeutic targets in type 2 diabetes utilizing molecular and biochemical features in rats
              </a>
            </td>
          <td>
            M. Matboli, Hiba Al-Amodi, Abdelrahman Khaled, Radwa Khaled, M. Roushdy, Marwa Ali, Gouda Ibrahim Diab, Mahmoud Fawzy Elnagar, R. A. Elmansy, Hagir H. TAhmed, Enshrah M. E. Ahmed, Doaa M Elzoghby, Hala F. M.Kamel, Mohamed F. Farag, Hind A. ELsawi, Laila M. Farid, M. B. Abouelkhair, Eman K. Habib, H. Fikry, Lobna A Saleh, Ibrahim H. Aboughaleb
          </td>
          <td>2024-05-24</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Motivation In drug discovery, it is crucial to assess the drug–target binding affinity (DTA). Although molecular docking is widely used, computational efficiency limits its application in large-scale virtual screening. Deep learning-based methods learn virtual scoring functions from labeled datasets and can quickly predict affinity. However, there are three limitations. First, existing methods only consider the atom-bond graph or one-dimensional sequence representations of compounds, ignoring the information about functional groups (pharmacophores) with specific biological activities. Second, relying on limited labeled datasets fails to learn comprehensive embedding representations of compounds and proteins, resulting in poor generalization performance in complex scenarios. Third, existing feature fusion methods cannot adequately capture contextual interaction information. Results Therefore, we propose a novel DTA prediction method named HeteroDTA. Specifically, a multi-view compound feature extraction module is constructed to model the atom–bond graph and pharmacophore graph. The residue concat graph and protein sequence are also utilized to model protein structure and function. Moreover, to enhance the generalization capability and reduce the dependence on task-specific labeled data, pre-trained models are utilized to initialize the atomic features of the compounds and the embedding representations of the protein sequence. A context-aware nonlinear feature fusion method is also proposed to learn interaction patterns between compounds and proteins. Experimental results on public benchmark datasets show that HeteroDTA significantly outperforms existing methods. In addition, HeteroDTA shows excellent generalization performance in cold-start experiments and superiority in the representation learning ability of drug–target pairs. Finally, the effectiveness of HeteroDTA is demonstrated in a real-world drug discovery study. Availability and implementation The source code and data are available at https://github.com/daydayupzzl/HeteroDTA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38cae834827755fe52cdc2576ba9cfde2dc9d11b" target='_blank'>
              Enhancing generalizability and performance in drug–target interaction identification by integrating pharmacophore and pre-trained models
              </a>
            </td>
          <td>
            Zuolong Zhang, Xin He, Dazhi Long, Gang Luo, Shengbo Chen
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4042b81dac3cbb2b1c6eb651f000e5a8614aac9d" target='_blank'>
              Machine learning identifies prognostic subtypes of the tumor microenvironment of NSCLC
              </a>
            </td>
          <td>
            Duo Yu, M. Kane, Eugene J. Koay, I. Wistuba, Brian,P. Hobbs
          </td>
          <td>2024-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="There is increasing interest in using microbial data diagnostically for tissue health monitoring such as in early cancer detection. To build such models, we need to understand whether normal tissue microbiomes can also be predictive of tissue of origin, and importantly ask how contaminants may contribute to model performance. In this study, using the Tabula Sapiens Microbiome dataset, we built machine learning models of human tissue microbiomes that can predict tissue of origin. This may in part explain how tumor types can be predicted based on the tumor microbiomes. We also demonstrate that machine learning models built using contaminants alone, though not as powerful as those built on true signal, can still predict tissue of origin. Reassuringly, the addition of contaminants to true signal does not increase the performance over models built on true signal. Overall, our findings raise the burden of proof for predictive models of the human tissue and tumor microbiomes. Without addressing the magnitude of contribution from contaminants to model performance, a model’s reproducibility and its clinical value becomes questionable. We also discuss the optimal microbial taxonomic resolution for building these models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231eee45deea2a0ced68f4fbb30aa4882e1d581b" target='_blank'>
              Machine learning models of human tissue microbiomes for tissue-of-origin prediction
              </a>
            </td>
          <td>
            Gita Mahmoudabadi, Stephen R. Quake
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In brain neural networks, Local Field Potential (LFP) signals represent the dynamic flow of information. Analyzing LFP clinical data plays a critical role in improving our understanding of brain mechanisms. One way to enhance our understanding of these mechanisms is to identify a global model to predict brain signals in different situations. This paper identifies a global data-driven based on LFP recordings of the Nucleus Accumbens and Hippocampus regions in freely moving rats. The LFP is recorded from each rat in two different situations: before and after the process of getting a reward which can be either a drug (Morphine) or natural food (like popcorn or biscuit). A comparison of five machine learning methods including Long Short Term Memory (LSTM), Echo State Network (ESN), Deep Echo State Network (DeepESN), Radial Basis Function (RBF), and Local Linear Model Tree (LLM) is conducted to develop this model. LoLiMoT was chosen with the best performance among all methods. This model can predict the future states of these regions with one pre-trained model. Identifying this model showed that Morphine and natural rewards do not change the dynamic features of neurons in these regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f91bc2c074768dec31cf981006a514eae67279b" target='_blank'>
              A Global Data-Driven Model for The Hippocampus and Nucleus Accumbens of Rat From The Local Field Potential Recordings (LFP)
              </a>
            </td>
          <td>
            Maedeh Sadeghi, M. A. Shoorehdeli, Shoele Jamali, Abbas Haghparast
          </td>
          <td>2024-05-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Drug discovery is a complex process that involves sequentially screening and examining a vast array of molecules to identify those with the target properties. This process, also referred to as sequential experimentation, faces challenges due to the vast search space, the rarity of target molecules, and constraints imposed by limited data and experimental budgets. To address these challenges, we introduce a human-in-the-loop framework for sequential experiments in drug discovery. This collaborative approach combines human expert knowledge with deep learning algorithms, enhancing the discovery of target molecules within a specified experimental budget. The proposed algorithm processes experimental data to recommend both promising molecules and those that could improve its performance to human experts. Human experts retain the final decision-making authority based on these recommendations and their domain expertise, including the ability to override algorithmic recommendations. We applied our method to drug discovery tasks using real-world data and found that it consistently outperforms all baseline methods, including those which rely solely on human or algorithmic input. This demonstrates the complementarity between human experts and the algorithm. Our results provide key insights into the levels of humans' domain knowledge, the importance of meta-knowledge, and effective work delegation strategies. Our findings suggest that such a framework can significantly accelerate the development of new vaccines and drugs by leveraging the best of both human and artificial intelligence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb16349d53232ad4fb92ecb11572a1778c99da9" target='_blank'>
              Collaborative Intelligence in Sequential Experiments: A Human-in-the-Loop Framework for Drug Discovery
              </a>
            </td>
          <td>
            Jinghai He, Cheng Hua, Yingfei Wang, Zeyu Zheng
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The integration of Graph Neural Networks (GNNs) and Neural Ordinary and Partial Differential Equations has been extensively studied in recent years. GNN architectures powered by neural differential equations allow us to reason about their behavior, and develop GNNs with desired properties such as controlled smoothing or energy conservation. In this paper we take inspiration from Turing instabilities in a Reaction Diffusion (RD) system of partial differential equations, and propose a novel family of GNNs based on neural RD systems. We \textcolor{black}{demonstrate} that our RDGNN is powerful for the modeling of various data types, from homophilic, to heterophilic, and spatio-temporal datasets. We discuss the theoretical properties of our RDGNN, its implementation, and show that it improves or offers competitive performance to state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57254c8083a6fe4f39ea4b61b190c97bb047ab60" target='_blank'>
              Graph Neural Reaction Diffusion Models
              </a>
            </td>
          <td>
            Moshe Eliasof, Eldad Haber, Eran Treister
          </td>
          <td>2024-06-16</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Supervised machine learning models rely on training datasets with positive (target class) and negative examples. Therefore, the composition of the training dataset has a direct influence on model performance. Specifically, negative sample selection bias, concerning samples not representing the target class, presents challenges across a range of domains such as text classification and protein-protein interaction prediction. Machine-learning-based immunotherapeutics design is an increasingly important area of research, focusing on designing antibodies or T-cell receptors (TCRs) that can bind to their target molecules with high specificity and affinity. Given the biomedical importance of immunotherapeutics, there is a need to address the unresolved question of how negative training set composition impacts model generalization and biological rule discovery to enable rational and safe drug design. We set out to study this question in the context of the antibody-antigen binding prediction problem by varying the negative class, encompassing a binding affinity gradient. We based our investigation on large synthetic datasets that provide ground truth structure-based antibody-antigen binding data, allowing access to residue-wise binding energy on the binding interface. We found that both out-of-distribution generalization and binding rule discovery depended on the type of negative dataset used. Importantly, we discovered that a model’s capacity to learn the binding rules of the positive dataset is not a trivial correlate of its classification accuracy. We confirmed our findings with real-world relevant experimental data. Our work highlights the importance of considering training dataset composition for achieving optimal out-of-distribution performance and rule learning in machine-learning-based research. Significance Statement The effectiveness of supervised machine learning models hinges on the composition of their training datasets, particularly the inclusion of negative examples. This bias in negative sample selection can greatly impact model performance. As the development of immunotherapeutic agents using machine learning is becoming increasingly crucial in biomedicine, understanding the impact of negative training set composition is imperative. Our study, focused on the antibody-antigen binding prediction problem, reveals that the choice of negative dataset significantly affects both out-of-distribution generalization and binding rule discovery across synthetic and experimental data. These findings underscore the necessity of carefully considering training dataset composition in machine-learning-driven biomedical research for optimal performance, robustness and meaningful rule acquisition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23c2f1758056f3fe1173a00d561a41e8889e3482" target='_blank'>
              Training data composition determines machine learning generalization and biological rule discovery
              </a>
            </td>
          <td>
            E. Ursu, Aygul Minnegalieva, Puneet Rawat, Maria Chernigovskaya, Robi Tacutu, G. K. Sandve, Philippe A. Robert, Victor Greiff
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cancers are commonly characterised by a complex pathology encompassing genetic, microscopic and macroscopic features, which can be probed individually using imaging and omics technologies. Integrating this data to obtain a full understanding of pathology remains challenging. We introduce a new method called Deep Latent Variable Path Modelling (DLVPM), which combines the representational power of deep learning with the capacity of path modelling to identify relationships between interacting elements in a complex system. To evaluate the capabilities of DLVPM, we initially trained a foundational model to map dependencies between SNV, Methylation, miRNA-Seq, RNA-Seq and Histological data using Breast Cancer data from The Cancer Genome Atlas (TCGA). This method exhibited superior performance in mapping associations between data types compared to classical path modelling. We additionally performed successful applications of the model to: stratify single-cell data, identify synthetic lethal interactions using CRISPR-Cas9 screens derived from cell-lines, and detect histologic-transcriptional associations using spatial transcriptomic data. Results from each of these data types can then be understood with reference to the same holistic model of illness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36a969a03ce741afe8ca32855f97d3ec725e814d" target='_blank'>
              Integrating Multi-Modal Cancer Data Using Deep Latent Variable Path Modelling
              </a>
            </td>
          <td>
            Alex Ing, Alvaro Andrades, M. R. Cosenza, J. Korbel
          </td>
          <td>2024-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="8614 Background: Advanced lung adenocarcinoma (aLUAD) is divided into multiple molecularly defined subsets, each with specific biological characteristics and responses to targeted therapy. Consequently, conducting practice-changing and cost-effective randomized trials is challenging. In silico clinical trials, utilizing mechanistic computational models, offer a potential solution, yet their ability to predict clinical trial results has not been proven. Methods: We updated a previously published version of the computational mechanistic model of EGFR-mutated aLUAD, notably by incorporating metastases. Disease model was integrated with physiologically based pharmacokinetic (PBPK) models of cisplatin, pemetrexed, and osimertinib, offering a more comprehensive understanding of the disease's dynamics. These models are founded on causal connections between biological entities and phenomena, establishing a link between the specific biological behaviors of EGFR-mutated aLUAD and the evolution of tumor size and progression over time. We have independently and prospectively simulated the outcomes of the FLAURA2 and MARIPOSA randomized trials dedicated to EGFR-mutated aLUAD. This was achieved by replicating the experimental protocols and generating cohorts of virtual twins of patients using publicly available data. Results: In silico results were released before the actual outcomes of the FLAURA2 and MARIPOSA trials were publicly available. The simulations yielded predictions with overlapping confidence intervals, similar hazard ratios, median survivals, and shapes of curves compared to the actual trial data. Bootstrapped weighted log-rank tests were used to compare the simulated and observed Kaplan-Meier curves for the two arms of FLAURA2 and for the osimertinib arm of MARIPOSA. Between 94 and 98% of the tests were statistically non-significant (α=0.05), confirming the accuracy of the predictions. Conclusions: This first-of-its-kind prediction of clinical trial outcomes demonstrates that in silico trials, when based on robust mechanistic models, can be a reliable tool for enhancing the design of actual trials, particularly for EGFR-mutated aLUAD. They offer a promising avenue for overcoming the challenges of comparator arms in single-arm trials and may serve as a new standard for formulating statistical hypotheses in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76086ed48f1470fe5129435b0503e60cd9f7700" target='_blank'>
              Prospective validation of in silico clinical trials prediction using an EGFR-mutated NSCLC mechanistic computational disease model.
              </a>
            </td>
          <td>
            Michaël Duruisseaux, P. Masson, Arnaud Nativel, Nicolas Girard, J. Cadranel, A. Swalduz, Matthieu Coudron, Guillaume Bouchard, R. Kahoul, Evgueni Jacob, J. Bosley, J. Palgen, A. L’Hostis, C. Monteiro
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="We present the first use of machine learning tools to predict multistationarity of reaction networks. Chemical Reaction Networks (CRNs) are the mathematical formulation of how the quantities associated to a set of species (molecules, proteins, cells, or animals) vary as time passes with respect to their interactions with each other. Their mathematics does not describe just chemical reactions but many other areas of the life sciences such as ecology, epidemiology, and population dynamics. We say a CRN is at a steady state when the concentration (or number) of species do not vary anymore. Some CRNs do not attain a steady state while some others may have more than one possible steady state. The CRNs in the later group are called multistationary. Multistationarity is an important property, e.g. switch-like behaviour in cells needs multistationarity to occur. Existing algorithms to detect whether a CRN is multistationary or not are either extremely expensive or restricted in the type of CRNs they can be used on, motivating a new machine learning approach. We address the problem of representing variable-length CRN data to machine learning models by developing a new graph representation of CRNs for use with graph learning algorithms. We contribute a large dataset of labelled fully open CRNs whose production necessitated the development of new CRN theory. Then we present experimental results on the training and testing of a graph attention network model on this dataset, showing excellent levels of performance. We finish by testing the model predictions on validation data produced independently, demonstrating generalisability of the model to different types of CRN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd097d9cd785834b1edb4051c13330afba0eaa4" target='_blank'>
              Designing Machine Learning Tools to Characterize Multistationarity of Fully Open Reaction Networks
              </a>
            </td>
          <td>
            Shenghao Yao, AmirHosein Sadeghimanesh, Matthew England
          </td>
          <td>2024-07-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="RNA sequencing (RNA-seq) has become a key technology in precision medicine, especially for cancer prognosis. However, the high dimensionality of such data may restrict classic statistical methods, thus raising the need to learn dense representations from them. Transformers models have exhibited capacities in providing representations for long sequences and thus are well suited for transcriptomics data. In this paper, we develop a pre-trained transformer-based language model through self-supervised learning using bulk RNA-seq from both non-cancer and cancer tissues, following BERT’s masking method. By probing learned embeddings from the model or using parameter-efficient fine-tuning, we then build downstream models for cancer type classification and survival time prediction. Leveraging the TCGA dataset, we demonstrate the performance of our method, BulkRNABert, on both tasks, with significant improvement compared to state-of-the-art methods in the pan-cancer setting for classification and survival analysis. We also show the transfer-learning capabilities of the model in the survival analysis setting on unseen cohorts. Code available at https://github.com/instadeepai/multiomics-open-research">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/124956adf9ab74435727398d4933f5b39b67b2ea" target='_blank'>
              BulkRNABert: Cancer prognosis from bulk RNA-seq based language models
              </a>
            </td>
          <td>
            Maxence Gélard, Guillaume Richard, Thomas Pierrot, Paul-Henri Cournède
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Many causal systems such as biological processes in cells can only be observed indirectly via measurements, such as gene expression. Causal representation learning -- the task of correctly mapping low-level observations to latent causal variables -- could advance scientific understanding by enabling inference of latent variables such as pathway activation. In this paper, we develop methods for inferring latent variables from multiple related datasets (environments) and tasks. As a running example, we consider the task of predicting a phenotype from gene expression, where we often collect data from multiple cell types or organisms that are related in known ways. The key insight is that the mapping from latent variables driven by gene expression to the phenotype of interest changes sparsely across closely related environments. To model sparse changes, we introduce Tree-Based Regularization (TBR), an objective that minimizes both prediction error and regularizes closely related environments to learn similar predictors. We prove that under assumptions about the degree of sparse changes, TBR identifies the true latent variables up to some simple transformations. We evaluate the theory empirically with both simulations and ground-truth gene expression data. We find that TBR recovers the latent causal variables better than related methods across these settings, even under settings that violate some assumptions of the theory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/828daf97066cdcac9612888fea4854de9c05ee25" target='_blank'>
              Sparsity regularization via tree-structured environments for disentangled representations
              </a>
            </td>
          <td>
            Elliot Layne, Dhanya Sridhar, Jason S. Hartford, M. Blanchette
          </td>
          <td>2024-05-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Machine learning is increasingly employed in engineering, with one of its primary applications being the construction of surrogate models to replace computationally expensive physical models for analysis and design, especially control co-design which integrates physical and control system design. In cases where generating labels for training through physical models is computationally intensive, label-free machine learning offers a viable alternative. However, surrogate models built from label-free machine learning typically have prediction errors, which can be characterized and quantified through epistemic uncertainty, representing model-form uncertainty. Moreover, when these surrogate models are used in optimization design for real-world applications, inherent random variables introduce aleatory uncertainty. This study introduces a robust design optimization method that addresses the intertwined epistemic and aleatory uncertainty. By optimizing both the average product performance and reducing uncertainty stemming from the coupled uncertainty, the method achieves improved robustness. A four-bar linkage mechanism design serves as a demonstration of this approach. The surrogate model for the design is constructed using label-free neural network, accommodating a system of physical equations. The error of the surrogate model is assessed through Gaussian Process regression, using existing training points and derivatives of the physical equations at these points. The design of the four-bar linkage aims to minimize both its average motion error and the variability of the error attributed to coupled uncertainty.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef8bdfc8b842dd50054b97d50b7f9695cf4df0d9" target='_blank'>
              Optimization Under Uncertainty Using Physics-Based Label-Free Machine Learning
              </a>
            </td>
          <td>
            Xiaoping Du
          </td>
          <td>2024-05-23</td>
          <td>2024 International Congress on Human-Computer Interaction, Optimization and Robotic Applications (HORA)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intratumour phenotypic heterogeneity is nowadays understood to play a critical role in disease progression and treatment failure. Accordingly, there has been increasing interest in the development of mathematical models capable of capturing its role in cancer cell adaptation. This can be systematically achieved by means of models comprising phenotype-structured nonlocal partial differential equations, tracking the evolution of the phenotypic density distribution of the cell population, which may be compared to gene and protein expression distributions obtained experimentally. Nevertheless, given the high analytical and computational cost of solving these models, much is to be gained from reducing them to systems of ordinary differential equations for the moments of the distribution. We propose a generalised method of model-reduction, relying on the use of a moment generating function, Taylor series expansion and truncation closure, to reduce a nonlocal reaction-advection-diffusion equation, with general phenotypic drift and proliferation rate functions, to a system of moment equations up to arbitrary order. Our method extends previous results in the literature, which we address via two examples, by removing any \textit{a priori} assumption on the shape of the distribution, and provides a flexible framework for mathematical modellers to account for the role of phenotypic heterogeneity in cancer adaptive dynamics, in a simpler mathematical framework.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d951ebfbeb9789f55de6ae13ba4fa4d0ddb0abc" target='_blank'>
              Reducing phenotype-structured PDE models of cancer evolution to systems of ODEs: a generalised moment dynamics approach
              </a>
            </td>
          <td>
            Chiara Villa, P. K. Maini, Alexander P Browning, A. Jenner, Sara Hamis, Tyler Cassidy
          </td>
          <td>2024-06-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In the present article, we propose a paradigm shift on evolving Artificial Neural Networks (ANNs) towards a new bio-inspired design that is grounded on the structural properties, interactions, and dynamics of protein networks (PNs): the Artificial Protein Network (APN). This introduces several advantages previously unrealized by state-of-the-art approaches in NE: (1) We can draw inspiration from how nature, thanks to millions of years of evolution, efficiently encodes protein interactions in the DNA to translate our APN to silicon DNA. This helps bridge the gap between syntax and semantics observed in current NE approaches. (2) We can learn from how nature builds networks in our genes, allowing us to design new and smarter networks through EA evolution. (3) We can perform EA crossover/mutation operations and evolution steps, replicating the operations observed in nature directly on the genotype of networks, thus exploring and exploiting the phenotypic space, such that we avoid getting trapped in sub-optimal solutions. (4) Our novel definition of APN opens new ways to leverage our knowledge about different living things and processes from biology. (5) Using biologically inspired encodings, we can model more complex demographic and ecological relationships (e.g., virus-host or predator-prey interactions), allowing us to optimise for multiple, often conflicting objectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdbe8cd6d06b80d544472cbb9efc805782d0782" target='_blank'>
              Protein pathways as a catalyst to directed evolution of the topology of artificial neural networks
              </a>
            </td>
          <td>
            Oscar Lao, Konstantinos Zacharopoulos, A. Fournaris, Rossano Schifanella, Ioannis Arapakis
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Co-folding models are the newest innovation in deep-learning-based protein-ligand structure prediction. The recent publications of RoseTTAFold All-Atom and AlphaFold 3 have shown high-quality results on predicting the structures of proteins interacting with small-molecules, other proteins and nucleic-acids. Despite these advanced capabilities and broad potential, the current study presents critical findings that question these models’ adherence to fundamental physical principles and its susceptibility to overfitting specific protein families. Through adversarial examples based on established physical, chemical, and biological principles, we demonstrate notable discrepancies in protein-ligand structural predictions when subjected to biologically plausible perturbations. These discrepancies reveal a significant divergence from expected physical behaviors, indicating potential overfitting to particular data subsets within its training corpus. Our findings underscore the models’ limitations in generalizing effectively across diverse biological structures and highlight the necessity of integrating robust physical and chemical priors in the development of such predictive tools. The results advocate a measured reliance on deep-learning-based models for critical applications in drug discovery and protein engineering, where a deep understanding of the underlying physical and chemical properties is crucial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246574ae80d8bdbec4e6397732952f7f393a2387" target='_blank'>
              Do Deep Learning Models for Co-Folding Learn the Physics of Protein-Ligand Interactions?
              </a>
            </td>
          <td>
            Matthew Masters, Amr H. Mahmoud, Markus A. Lill
          </td>
          <td>2024-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="A network, whose nodes are genes and whose directed edges represent positive or negative influences of a regulatory gene and its targets, is often used as a representation of causality. To infer a network, researchers often develop a machine learning model and then evaluate the model based on its match with experimentally verified “gold standard” edges. The desired result of such a model is a network that may extend the gold standard edges. Since networks are a form of visual representation, one can compare their utility with architectural or machine blueprints. Blueprints are clearly useful because they provide precise guidance to builders in construction. If the primary role of gene regulatory networks is to characterize causality, then such networks should be good tools of prediction because prediction is the actionable benefit of knowing causality. But are they? In this paper, we compare prediction quality based on “gold standard” regulatory edges from previous experimental work with non-linear models inferred from time series data across four different species. We show that the same non-linear machine learning models have better predictive performance, with improvements from 5.3% to 25.3% in terms of the reduction in the root mean square error (RMSE) compared with the same models based on the gold standard edges. Having established that networks fail to characterize causality properly, we suggest that causality research should focus on four goals: (i) predictive accuracy; (ii) a parsimonious enumeration of predictive regulatory genes for each target gene g; (iii) the identification of disjoint sets of predictive regulatory genes for each target g of roughly equal accuracy; and (iv) the construction of a bipartite network (whose node types are genes and models) representation of causality. We provide algorithms for all goals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd7eda0fbce6aa023e877e8ea11a162f2aca8b4" target='_blank'>
              Bipartite networks represent causality better than simple networks: evidence, algorithms, and applications
              </a>
            </td>
          <td>
            Bingran Shen, Gloria M Coruzzi, Dennis Shasha
          </td>
          <td>2024-05-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In the field of drug discovery, a proliferation of pre-trained models has surfaced, exhibiting exceptional performance across a variety of tasks. However, the extensive size of these models, coupled with the limited interpretative capabilities of current fine-tuning methods, impedes the integration of pre-trained models into the drug discovery process. This paper pushes the boundaries of pre-trained models in drug discovery by designing a novel fine-tuning paradigm known as the Head Feature Parallel Adapter (HFPA), which is highly interpretable, high-performing, and has fewer parameters than other widely used methods. Specifically, this approach enables the model to consider diverse information across representation subspaces concurrently by strategically using Adapters, which can operate directly within the model's feature space. Our tactic freezes the backbone model and forces various small-size Adapters' corresponding subspaces to focus on exploring different atomic and chemical bond knowledge, thus maintaining a small number of trainable parameters and enhancing the interpretability of the model. Moreover, we furnish a comprehensive interpretability analysis, imparting valuable insights into the chemical area. HFPA outperforms over seven physiology and toxicity tasks and achieves state-of-the-art results in three physical chemistry tasks. We also test ten additional molecular datasets, demonstrating the robustness and broad applicability of HFPA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89c47edeb51c005371552573aa512d2ca9791cf9" target='_blank'>
              Advancing the Boundary of Pre-trained Models for Drug Discovery: Interpretable Fine-Tuning Empowered by Molecular Physicochemical Properties.
              </a>
            </td>
          <td>
            Xiaoqing Lian, Jie Zhu, Tianxu Lv, Xiaoyan Hong, Longzhen Ding, Wei Chu, Jianming Ni, Xiang Pan
          </td>
          <td>2024-06-18</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract In precision medicine, both predicting the disease susceptibility of an individual and forecasting its disease-free survival are areas of key research. Besides the classical epidemiological predictor variables, data from multiple (omic) platforms are increasingly available. To integrate this wealth of information, we propose new methodology to combine both cooperative learning, a recent approach to leverage the predictive power of several datasets, and polygenic hazard score models. Polygenic hazard score models provide a practitioner with a more differentiated view of the predicted disease-free survival than the one given by merely a point estimate, for instance computed with a polygenic risk score. Our aim is to leverage the advantages of cooperative learning for the computation of polygenic hazard score models via Cox’s proportional hazard model, thereby improving the prediction of the disease-free survival. In our experimental study, we apply our methodology to forecast the disease-free survival for Alzheimer’s disease (AD) using three layers of data. One layer contains epidemiological variables such as sex, APOE (apolipoprotein E, a genetic risk factor for AD) status and 10 leading principal components. Another layer contains selected genomic loci, and the last layer contains methylation data for selected CpG sites. We demonstrate that the survival curves computed via cooperative learning yield an AUC of around \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $0.7$\end{document}, above the state-of-the-art performance of its competitors. Importantly, the proposed methodology returns (1) a linear score that can be easily interpreted (in contrast to machine learning approaches), and (2) a weighting of the predictive power of the involved data layers, allowing for an assessment of the importance of each omic (or other) platform. Similarly to polygenic hazard score models, our methodology also allows one to compute individual survival curves for each patient.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9168bf7c51756564400a8c3a96e0936a62287e5" target='_blank'>
              Prediction of disease-free survival for precision medicine using cooperative learning on multi-omic data
              </a>
            </td>
          <td>
            Georg Hahn, D. Prokopenko, J. Hecker, S. M. Lutz, K. Mullin, Leinal Sejour, Winston Hide, Ioannis Vlachos, Stacia DeSantis, Rudolph E. Tanzi, Christoph Lange
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Deep learning has emerged as a transformative technology in data science, revolutionizing various domains through its powerful capabilities. This paper explores the theoretical foundations, practical applications, and comparative analysis of deep learning models. The theoretical foundations section discusses key neural network architectures such as Convolutional Neural Networks (CNNs), Recurrent Neural Networks (RNNs), and Transformers, highlighting their unique capabilities in processing different types of data. Optimization algorithms crucial for effective training, including Stochastic Gradient Descent (SGD) and Adam, are examined. Regularization techniques for preventing overfitting and enhancing generalization are also addressed. Practical applications in healthcare, finance, and retail showcase the real-world impact of deep learning. A comparative analysis of performance metrics demonstrates the superiority of deep learning models over traditional methods. Despite their advantages, deep learning models face limitations and challenges, including data dependency and interpretability issues. The paper concludes by emphasizing the ongoing research efforts to mitigate these challenges and ensure the continued advancement of deep learning in data science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b019728ec1ec6b00f0a46d426f7a25c881bbf25" target='_blank'>
              Deep learning in data science: Theoretical foundations, practical applications, and comparative analysis
              </a>
            </td>
          <td>
            Yingxuan Chai, Liangning Jin
          </td>
          <td>2024-06-21</td>
          <td>Applied and Computational Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Accurate prediction of the future evolution of observational time series is a paramount challenge in current data‐driven research. While existing techniques struggle to learn useful representations from the temporal correlations, the high dimensionality in spatial domain is always considered as obstacle, leading to the curse of dimensionality and excessive resource consumption. This work designs a novel structure‐aware reservoir computing aiming at enhancing the predictability of coupled time series, by incorporating their historical dynamics as well as structural information. Paralleled reservoir computers with redesigned mixing inputs based on spatial relationships are implemented to cope with the multiple time series, whose core idea originates from the principle of the celebrated Granger causality. Representative numerical simulations and comparisons demonstrate the superior performance of the approach over the traditional ones. This work provides valuable insights into deeply mining both temporal and spatial information to enhance the representation learning of data in various machine learning techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3af7d40101accb02cd7367635eb89b2f0f3967db" target='_blank'>
              Enhancing Time Series Predictability via Structure‐Aware Reservoir Computing
              </a>
            </td>
          <td>
            Suzhen Guo, Chun Guan, Siyang Leng
          </td>
          <td>2024-06-02</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Agent-based models (ABM) provide an excellent framework for modeling outbreaks and interventions in epidemiology by explicitly accounting for diverse individual interactions and environments. However, these models are usually stochastic and highly parametrized, requiring precise calibration for predictive performance. When considering realistic numbers of agents and properly accounting for stochasticity, this high dimensional calibration can be computationally prohibitive. This paper presents a random forest based surrogate modeling technique to accelerate the evaluation of ABMs and demonstrates its use to calibrate an epidemiological ABM named CityCOVID via Markov chain Monte Carlo (MCMC). The technique is first outlined in the context of CityCOVID's quantities of interest, namely hospitalizations and deaths, by exploring dimensionality reduction via temporal decomposition with principal component analysis (PCA) and via sensitivity analysis. The calibration problem is then presented and samples are generated to best match COVID-19 hospitalization and death numbers in Chicago from March to June in 2020. These results are compared with previous approximate Bayesian calibration (IMABC) results and their predictive performance is analyzed showing improved performance with a reduction in computation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ba5a747ad383f428c440aad51df2e9d201eb52" target='_blank'>
              Bayesian calibration of stochastic agent based model via random forest
              </a>
            </td>
          <td>
            Connor Robertson, C. Safta, Nicholson T. Collier, J. Ozik, Jaideep Ray
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Biological neural networks can perform complex computations to predict their environment, far above the limited predictive capabilities of individual neurons. While conventional approaches to understanding these computations often focus on isolating the contributions of single neurons, here we argue that a deeper understanding requires considering emergent dynamics - dynamics that make the whole system"more than the sum of its parts". Specifically, we examine the relationship between prediction performance and emergence by leveraging recent quantitative metrics of emergence, derived from Partial Information Decomposition, and by modelling the prediction of environmental dynamics in a bio-inspired computational framework known as reservoir computing. Notably, we reveal a bidirectional coupling between prediction performance and emergence, which generalises across task environments and reservoir network topologies, and is recapitulated by three key results: 1) Optimising hyperparameters for performance enhances emergent dynamics, and vice versa; 2) Emergent dynamics represent a near sufficient criterion for prediction success in all task environments, and an almost necessary criterion in most environments; 3) Training reservoir computers on larger datasets results in stronger emergent dynamics, which contain task-relevant information crucial for performance. Overall, our study points to a pivotal role of emergence in facilitating environmental predictions in a bio-inspired computational architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2572feb06e7ea060ecb305043a5d4b589af5886d" target='_blank'>
              Evolving reservoir computers reveals bidirectional coupling between predictive power and emergent dynamics
              </a>
            </td>
          <td>
            Hanna M. Tolle, A. Luppi, Anil K. Seth, P. Mediano
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="

 In the rapidly evolving pharmaceutical industry, drug efficacy and safety stand as critical concerns. The vast accumulation of data, including customer feedback, drug popularity, and usage details, offers a rich resource for improving healthcare outcomes.



 The primary aim of this study is to harness machine learning and Natural Language Processing (NLP) techniques to sift through extensive pharmaceutical data, identifying the most effective drugs for various conditions and uncovering patterns that could guide better decision-making in drug efficacy and safety.



 This research seeks to construct a sophisticated model capable of analyzing diverse data points to pinpoint the most efficacious drugs for specific health conditions, thereby providing pharmaceutical companies with data-driven insights to optimize drug safety and effectiveness.



 Employing a blend of Natural Language Processing (NLP) and machine learning strategies, the study analyzes a comprehensive dataset featuring customer reviews, drug popularity metrics, usage information, and other relevant data collected over an extended period. This methodological approach aims to reveal latent trends and patterns that are crucial for assessing drug performance.



 The developed model adeptly identifies leading medications for various conditions, elucidating efficacy and safety profiles derived from patient reviews and drug utilization trends. These findings furnish pharmaceutical companies with actionable intelligence for enhancing drug development and patient care strategies.



 The integration of machine learning and NLP for the analysis of vast drug-related datasets presents a powerful method for advancing drug efficacy and safety. The insights yielded by the proposed model significantly empower the decision-making processes of the pharmaceutical industry, ultimately fostering improved health outcomes for patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa74cbe78ea6c68e73d842096a74cc272cfa464" target='_blank'>
              Leveraging Machine Learning and Patient Reviews for Developing a Drug Recommendation System to Reduce Medical Errors
              </a>
            </td>
          <td>
            K. P. Swain, S.K. Mohapatra, Vinayakumar Ravi, S. R. Nayak, Tahani Jaser Alahmadi, Prabhishek Singh, M. Diwakar
          </td>
          <td>2024-06-25</td>
          <td>The Open Bioinformatics Journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Biological sequences encode fundamental instructions for the building blocks of life, in the form of DNA, RNA, and proteins. Modeling these sequences is key to understand disease mechanisms and is an active research area in computational biology. Recently, Large Language Models have shown great promise in solving certain biological tasks but current approaches are limited to a single sequence modality (DNA, RNA, or protein). Key problems in genomics intrinsically involve multiple modalities, but it remains unclear how to adapt general-purpose sequence models to those cases. In this work we propose a multi-modal model that connects DNA, RNA, and proteins by leveraging information from different pre-trained modality-specific encoders. We demonstrate its capabilities by applying it to the largely unsolved problem of predicting how multiple RNA transcript isoforms originate from the same gene (i.e. same DNA sequence) and map to different transcription expression levels across various human tissues. We show that our model, dubbed IsoFormer, is able to accurately predict differential transcript expression, outperforming existing methods and leveraging the use of multiple modalities. Our framework also achieves efficient transfer knowledge from the encoders pre-training as well as in between modalities. We open-source our model, paving the way for new multi-modal gene expression approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c94cf63c86cbeefe668b6cb6b118506e8e144b7a" target='_blank'>
              Multi-modal Transfer Learning between Biological Foundation Models
              </a>
            </td>
          <td>
            Juan Jose Garau-Luis, Patrick Bordes, Liam Gonzalez, Masa Roller, Bernardo P. de Almeida, Lorenz Hexemer, Christopher Blum, Stefan Laurent, Jan Grzegorzewski, Maren Lang, Thomas Pierrot, Guillaume Richard
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Wildland fires pose terrifying natural hazards, underscoring the urgent need to develop data-driven and physics-informed digital twins for wildfire prevention, monitoring, intervention, and response. In this direction of research, this work introduces a physics-informed neural network (PiNN) to learn the unknown parameters of an interpretable wildfire spreading model. The considered wildfire spreading model integrates fundamental physical laws articulated by key model parameters, essential for capturing the complex behavior of wildfires. The proposed machine learning approach leverages the theory of artificial neural networks with the physical constraints governing wildfire dynamics, such as the first principles of mass and energy conservation. Training of the PiNN for physics-informed parameter identification is realized using data of the temporal evolution of one- and two-dimensional (plane surface) fire fronts that have been obtained from a high-fidelity simulator of the wildfire spreading model under consideration. The parameter learning results demonstrate the remarkable predictive ability of the proposed PiNN in uncovering the unknown coefficients in both the one- and two-dimensional fire spreading scenarios. Additionally, this methodology exhibits robustness by identifying the same parameters in the presence of noisy data. The proposed framework is envisioned to be incorporated in a physics-informed digital twin for intelligent wildfire management and risk assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62a95332da195a72205da26346da59e96afa55ce" target='_blank'>
              Physics-informed neural networks for parameter learning of wildfire spreading
              </a>
            </td>
          <td>
            K. Vogiatzoglou, C. Papadimitriou, V. Bontozoglou, Konstantinos Ampountolas
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The dynamics of biological systems, from proteins to cells to organisms, is complex and stochastic. To decipher their physical laws, we need to bridge between experimental observations and theoretical modeling. Thanks to progress in microscopy and tracking, there is today an abundance of experimental trajectories reflecting these dynamical laws. Inferring physical models from noisy and imperfect experimental data, however, is challenging. Because there are no inference methods that are robust and efficient, model reconstruction from experimental trajectories is a bottleneck to data-driven biophysics. In this Thesis, I present a set of tools developed to bridge this gap and permit robust and universal inference of stochastic dynamical models from experimental trajectories. These methods are rooted in an information-theoretical framework that quantifies how much can be inferred from trajectories that are short, partial and noisy. They permit the efficient inference of dynamical models for overdamped and underdamped Langevin systems, as well as the inference of entropy production rates. I finally present early applications of these techniques, as well as future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5a9b10d628b238c485c3e97f0ad456101dedb" target='_blank'>
              Learning dynamical models from stochastic trajectories
              </a>
            </td>
          <td>
            Pierre Ronceray
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Message passing neural networks have demonstrated significant efficacy in predicting molecular interactions. Introducing equivariant vectorial representations augments expressivity by capturing geometric data symmetries, thereby improving model accuracy. However, two-body bond vectors in opposition may cancel each other out during message passing, leading to the loss of directional information on their shared node. In this study, we develop Equivariant N-body Interaction Networks (ENINet) that explicitly integrates equivariant many-body interactions to preserve directional information in the message passing scheme. Experiments indicate that integrating many-body equivariant representations enhances prediction accuracy across diverse scalar and tensorial quantum chemical properties. Ablation studies show an average performance improvement of 7.9% across 11 out of 12 properties in QM9, 27.9% in forces in MD17, and 11.3% in polarizabilities (CCSD) in QM7b.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecd2f7b7cabcfbabeac06fdb9a5f94402110188" target='_blank'>
              Molecule Graph Networks with Many-body Equivariant Interactions
              </a>
            </td>
          <td>
            Zetian Mao, Jiawen Li, Chen Liang, Diptesh Das, Masato Sumita, Koji Tsuda
          </td>
          <td>2024-06-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Motivation Complex diseases are often caused and characterized by misregulation of multiple biological pathways. Differential network analysis aims to detect significant rewiring of biological network structures under different conditions and has become an important tool for understanding the molecular etiology of disease progression and therapeutic response. With few exceptions, most existing differential network analysis tools perform differential tests on separately learned network structures that are computationally expensive and prone to collapse when grouped samples are limited or less consistent. Results We previously developed an accurate differential network analysis method—differential dependency networks (DDN), that enables joint learning of common and rewired network structures under different conditions. We now introduce the DDN3.0 tool that improves this framework with three new and highly efficient algorithms, namely, unbiased model estimation with a weighted error measure applicable to imbalance sample groups, multiple acceleration strategies to improve learning efficiency, and data-driven determination of proper hyperparameters. The comparative experimental results obtained from both realistic simulations and case studies show that DDN3.0 can help biologists more accurately identify, in a study-specific and often unknown conserved regulatory circuitry, a network of significantly rewired molecular players potentially responsible for phenotypic transitions. Availability and implementation The Python package of DDN3.0 is freely available at https://github.com/cbil-vt/DDN3. A user’s guide and a vignette are provided at https://ddn-30.readthedocs.io/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f7f53646bc589a6aedd4807e452f6e2c56e13c4" target='_blank'>
              DDN3.0: determining significant rewiring of biological network structure with differential dependency networks
              </a>
            </td>
          <td>
            Yi Fu, Yingzhou Lu, Yizhi Wang, Bai Zhang, Zhen Zhang, Guoqiang Yu, Chunyu Liu, Robert Clarke, David M. Herrington, Yue Wang
          </td>
          <td>2024-06-01</td>
          <td>Bioinformatics</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="Chronic Kidney Disease (CKD) is a global health issue impacting over 800 million people, characterized by a gradual loss of kidney function leading to severe complications. Traditional diagnostic methods, relying on laboratory tests and clinical assessments, have limitations in sensitivity and are prone to human error, particularly in the early stages of CKD. Recent advances in machine learning (ML) offer promising tools for disease diagnosis, but a lack of interpretability often hinders their adoption in clinical practice. Gaussian Processes (GP) provide a flexible ML model capable of delivering predictions and uncertainty estimates, essential for high-stakes medical applications. However, the integration of GP with interpretable methods remains underexplored. We developed an interpretable CKD classification model to address this knowledge gap by combining GP with Shapley Additive Explanations (SHAP). We assessed the model's performance using three GP kernels (Radial Basis Function, Matern, and Rational Quadratic). The results show that the Rational Quadratic kernel outperforms the other kernels, achieving an accuracy of 98.75%, precision of 100%, sensitivity of 97.87%, specificity of 100%, and an F1-score of 98.51%. SHAP values indicate that haemoglobin and specific gravity are the most influential features. The results demonstrate that the Rational Quadratic kernel enhances predictive accuracy and provides robust uncertainty estimates and interpretable explanations. This combination of accuracy and interpretability supports clinicians in making informed decisions and improving patient management and outcomes in CKD. Our study connects advanced ML techniques with practical medical applications, leading to more effective and reliable ML-driven healthcare solutions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adf6d004ccf334eefb041c57041421906b537c8e" target='_blank'>
              Interpretable Machine Learning for Chronic Kidney Disease Diagnosis: A Gaussian Processes Approach
              </a>
            </td>
          <td>
            T. R. Noviandy, Ghifari Maulana Idroes, M. Syukri, R. Idroes
          </td>
          <td>2024-06-29</td>
          <td>Indonesian Journal of Case Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Temporal data modelling techniques with neural networks are useful in many domain applications, including time-series forecasting and control engineering. This paper aims at developing a recurrent version of stochastic configuration networks (RSCNs) for problem solving, where we have no underlying assumption on the dynamic orders of the input variables. Given a collection of historical data, we first build an initial RSCN model in the light of a supervisory mechanism, followed by an online update of the output weights by using a projection algorithm. Some theoretical results are established, including the echo state property, the universal approximation property of RSCNs for both the offline and online learnings, and the convergence of the output weights. The proposed RSCN model is remarkably distinguished from the well-known echo state networks (ESNs) in terms of the way of assigning the input random weight matrix and a special structure of the random feedback matrix. A comprehensive comparison study among the long short-term memory (LSTM) network, the original ESN, and several state-of-the-art ESN methods such as the simple cycle reservoir (SCR), the polynomial ESN (PESN), the leaky-integrator ESN (LIESN) and RSCN is carried out. Numerical results clearly indicate that the proposed RSCN performs favourably over all of the datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e3c55f0b0e77192fb6cc422ecf599a47334d73" target='_blank'>
              Recurrent Stochastic Configuration Networks for Temporal Data Analytics
              </a>
            </td>
          <td>
            Dianhui Wang, Gang Dang
          </td>
          <td>2024-06-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The estimation of directed couplings between the nodes of a network from indirect measurements is a central methodological challenge in scientific fields such as neuroscience, systems biology and economics. Unfortunately, the problem is generally ill-posed due to the possible presence of unknown delays in the measurements. In this paper, we offer a solution of this problem by using a variational Bayes framework, where the uncertainty over the delays is marginalized in order to obtain conservative coupling estimates. To overcome the well-known overconfidence of classical variational methods, we use a hybrid-VI scheme where the (possibly flat or multimodal) posterior over the measurement parameters is estimated using a forward KL loss while the (nearly convex) conditional posterior over the couplings is estimated using the highly scalable gradient-based VI. In our ground-truth experiments, we show that the network provides reliable and conservative estimates of the couplings, greatly outperforming similar methods such as regression DCM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a02bb50bd2145ddb88212879e50602c28462b00e" target='_blank'>
              Robust and highly scalable estimation of directional couplings from time-shifted signals
              </a>
            </td>
          <td>
            Luca Ambrogioni, Louis Rouillard, Demian Wassermann
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Resting-state functional magnetic resonance imaging (rs-fMRI) is a noninvasive technique pivotal for understanding human neural mechanisms of intricate cognitive processes. Most rs-fMRI studies compute a single static functional connectivity matrix across brain regions of interest, or dynamic functional connectivity matrices with a sliding window approach. These approaches are at risk of oversimplifying brain dynamics and lack proper consideration of the goal at hand. While deep learning has gained substantial popularity for modeling complex relational data, its application to uncovering the spatiotemporal dynamics of the brain is still limited. We propose a novel interpretable deep learning framework that learns goal-specific functional connectivity matrix directly from time series and employs a specialized graph neural network for the final classification. Our model, DSAM, leverages temporal causal convolutional networks to capture the temporal dynamics in both low- and high-level feature representations, a temporal attention unit to identify important time points, a self-attention unit to construct the goal-specific connectivity matrix, and a novel variant of graph neural network to capture the spatial dynamics for downstream classification. To validate our approach, we conducted experiments on the Human Connectome Project dataset with 1075 samples to build and interpret the model for the classification of sex group, and the Adolescent Brain Cognitive Development Dataset with 8520 samples for independent testing. Compared our proposed framework with other state-of-art models, results suggested this novel approach goes beyond the assumption of a fixed connectivity matrix and provides evidence of goal-specific brain connectivity patterns, which opens up the potential to gain deeper insights into how the human brain adapts its functional connectivity specific to the task at hand.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8de1b9499bdea8b4a5599a305fd5adae62c3ed0" target='_blank'>
              DSAM: A Deep Learning Framework for Analyzing Temporal and Spatial Dynamics in Brain Networks
              </a>
            </td>
          <td>
            Bishal Thapaliya, Robyn Miller, Jiayu Chen, Yu-Ping Wang, Esra Akbas, Ram Sapkota, Bhaskar Ray, Pranav Suresh, Santosh Ghimire, V. Calhoun, Jingyu Liu
          </td>
          <td>2024-05-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Single-cell technologies enable comprehensive profiling of diverse immune cell-types through the measurement of multiple genes or proteins per cell. In order to translate data from immune profiling assays into powerful diagnostics, machine learning approaches are used to compute per-sample immunological summaries, or featurizations that can be used as inputs to models for outcomes of interest. Current supervised learning approaches for computing per-sample representations are optimized based only on the outcome variable to be predicted and do not take into account clinically-relevant covariates that are likely to also be measured. Here we expand the optimization problem to also take into account such additional patient covariates to directly inform the learned per-sample representations. To do this, we introduce CytoCoSet, a set-based encoding method, which formulates a loss function with an additional triplet term penalizing samples with similar covariates from having disparate embedding results in per-sample representations. Overall, incorporating clinical covariates leads to improved prediction of clinical phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca96995fe3a8574dda746ff07923576169480ac" target='_blank'>
              Conditional Similarity Triplets Enable Covariate-Informed Representations of Single-Cell Data
              </a>
            </td>
          <td>
            Chi-Jane Chen, Haidong Yi, Natalie Stanley
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Stepwise covariate modeling (SCM) has a high computational burden and can select the wrong covariates. Machine learning (ML) has been proposed as a screening tool to improve the efficiency of covariate selection, but little is known about how to apply ML on actual clinical data. First, we simulated datasets based on clinical data to compare the performance of various ML and traditional pharmacometrics (PMX) techniques with and without accounting for highly-correlated covariates. This simulation step identified the ML algorithm and the number of top covariates to select when using the actual clinical data. A previously developed desipramine population-pharmacokinetic model was used to simulate virtual subjects. Fifteen covariates were considered with four having an effect included. Based on the F1 score (an accuracy measure), ridge regression was the most accurate ML technique on 200 simulated datasets (F1 score = 0.475 ± 0.231), a performance which almost doubled when highly-correlated covariates were accounted for (F1 score = 0.860 ± 0.158). These performances were better than forwards selection with SCM (F1 score = 0.251 ± 0.274 and 0.499 ± 0.381 without/with correlations respectively). In terms of computational cost, ridge regression (0.42 ± 0.07 seconds/simulated dataset, 1 thread) was ~20,000 times faster than SCM (2.30 ± 2.29 hours, 15 threads). On the clinical dataset, prescreening with the selected ML algorithm reduced SCM runtime by 42.86% (from 1.75 to 1.00 days) and produced the same final model as SCM only. In conclusion, we have demonstrated that accounting for highly-correlated covariates improves ML prescreening accuracy. The choice of ML method and the proportion of important covariates (unknown a priori) can be guided by simulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6e659fbccaa48361b3511d11da2b5a70822817" target='_blank'>
              Machine-Learning Assisted Screening of Correlated Covariates: Application to Clinical Data of Desipramine.
              </a>
            </td>
          <td>
            Innocent Gerald Asiimwe, Bonginkosi S'fiso Ndzamba, Samer Mouksassi, G. Pillai, Aurelie Lombard, Jennifer Lang
          </td>
          <td>2024-05-30</td>
          <td>The AAPS journal</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d03a9ff2ae1d1c085223172f82f70126105827e" target='_blank'>
              Discrete latent embedding of single-cell chromatin accessibility sequencing data for uncovering cell heterogeneity
              </a>
            </td>
          <td>
            Xuejian Cui, Xiaoyang Chen, Zhen Li, Zijing Gao, Shengquan Chen, Rui Jiang
          </td>
          <td>2024-05-10</td>
          <td>Nature computational science</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Modern recordings of neural activity provide diverse observations of neurons across brain areas, behavioral conditions, and subjects -- thus presenting an exciting opportunity to reveal the fundamentals of brain-wide dynamics underlying cognitive function. Current methods, however, often fail to fully harness the richness of such data as they either provide an uninterpretable representation (e.g., via"black box"deep networks) or over-simplify the model (e.g., assume stationary dynamics or analyze each session independently). Here, instead of regarding asynchronous recordings that lack alignment in neural identity or brain areas as a limitation, we exploit these diverse views of the same brain system to learn a unified model of brain dynamics. We assume that brain observations stem from the joint activity of a set of functional neural ensembles (groups of co-active neurons) that are similar in functionality across recordings, and propose to discover the ensemble and their non-stationary dynamical interactions in a new model we term CrEIMBO (Cross-Ensemble Interactions in Multi-view Brain Observations). CrEIMBO identifies the composition of the per-session neural ensembles through graph-driven dictionary learning and models the ensemble dynamics as a latent sparse time-varying decomposition of global sub-circuits, thereby capturing non-stationary dynamics. CrEIMBO identifies multiple co-active sub-circuits while maintaining representation interpretability due to sharing sub-circuits across sessions. CrEIMBO distinguishes session-specific from global (session-invariant) computations by exploring when distinct sub-circuits are active. We demonstrate CrEIMBO's ability to recover ground truth components in synthetic data and uncover meaningful brain dynamics, capturing cross-subject and inter- and intra-area variability, in high-density electrode recordings of humans performing a memory task.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d7b62d73540f0ee3170538475eecbf533fc4b56" target='_blank'>
              CrEIMBO: Cross Ensemble Interactions in Multi-view Brain Observations
              </a>
            </td>
          <td>
            Noga Mudrik, Ryan Ly, Oliver Ruebel, Adam S. Charles
          </td>
          <td>2024-05-27</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Interactions between drugs can lead to adverse side effects for patients taking combination therapies to treat complex diseases such as cancer. Knowledge of drug-action towards a receptor would allow these drug-drug interactions to be predicted, and in this study, we trained a total of 5 different machine learning models to classify whether a given drug was an agonist (activator), antagonist (blocker), or a decoy (non-binder) to each of the androgen, estrogen, glucocorticoid, and progesterone receptors. The classification performance and efficiency, measured in training time, of the decision tree, naïve Bayes, neural network, random forest, and support vector machine models for each receptor were then compared. The results showed that the decision tree and naïve Bayes models were best suited for drug-action prediction across all receptors while only requiring minutes of training time at most. Future work will focus on increasing the prediction accuracy of antagonist drugs, integrating experimental data during training, and using other targets outside of nuclear type I receptors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3f1fd185e5d36f4f38477342cd4d936ce7439" target='_blank'>
              Application of Supervised Machine Learning Models for Drug-Action Prediction Towards Nuclear Type I Receptors
              </a>
            </td>
          <td>
            Rajeev Jaundoo, J. Tuszynski, T. Craddock
          </td>
          <td>2024-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="We present an application of Physics-Informed Neural Networks to handle MultiPhase-Field simulations of microstructure evolution. It has been showcased that a combination of optimization techniques extended and adapted from the PINNs literature, and the introduction of specific techniques inspired by the MPF Method background, is required. The numerical resolution is realized through a multi-variable time-series problem by using fully discrete resolution. Within each interval, space, time, and phases are treated separately, constituting discrete subdomains. An extended multi-networking concept is implemented to subdivide the simulation domain into multiple batches, with each batch associated with an independent Neural Network trained to predict the solution. To ensure efficient interaction across different phasesand in the spatio-temporal-phasic subdomain, a Master NN handles efficient interaction among the multiple networks, as well as the transfer of learning in different directions. A set of systematic simulations with increasing complexity was performed, that benchmarks various critical aspects of MPF simulations, including different geometries, types of interface dynamics and the evolution of an interfacial triple junction. A comprehensive approach is adopted to specifically focus the attention on the interfacial regions through an automatic and dynamic meshing process, significantly simplifying the tuning of hyper-parameters and serving as a fundamental key for addressing MPF problems using Machine Learning. The pyramidal training approach is proposed to the PINN community as a dual-impact method: it facilitates the initialization of training and allows an extended transfer of learning. The proposed PINNs-MPF framework successfully reproduces benchmark tests with high fidelity and Mean Squared Error loss values ranging from 10$^{-4}$ to 10$^{-6}$ compared to ground truth solutions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f31d909eac4f45aba2c5d7eda80b9159e9f0772f" target='_blank'>
              PINNs-MPF: A Physics-Informed Neural Network Framework for Multi-Phase-Field Simulation of Interface Dynamics
              </a>
            </td>
          <td>
            Seifallah Elfetni, R. D. Kamachali
          </td>
          <td>2024-07-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In this study, we develop and implement the Extended Kalman Filter (EKF) to forecast the rate of change in tumor cells, healthy host cells, and effector immune cells within the Itik-Banks model. This novel application of EKF in cancer dynamics modeling aims to provide precise real-time estimations of cellular interactions, especially in constructing a new state space representation from the Itik-Banks model. We use a first-order Taylor series to linearize the model. The numerical simulations were performed to analyze the accuracy of this new state space with data from William Gilpin’s GitHub repository. The results show that the EKF predictions strongly align with actual data, i.e., in the prior and posterior steps for tumor and healthy host cells, there is a strong agreement between the predictions and the actual data. The EKF captures the oscillatory nature of the tumor and healthy host cell population well. The peaks and troughs of the predictions align closely with the actual data, indicating the EKF’s effectiveness in modeling the dynamic behavior of the tumor and healthy host cells. However, for effector immune cells, the oscillatory nature of the data in these cells gives rise to slight deviations. This represents a significant challenge in the future for updating the state space representations. Despite minor discrepancies, the EKF demonstrates a strong performance in both the training and testing data, with the posterior step estimates significantly improving the prior step accuracy. This study emphasizes the importance of data availability for accurate predictions, noting a symmetric Mean Absolute Percentage Error (sMAPE) of 35.92% when data is unavailable. Prompt correction with new data is essential to maintain accuracy. This research underscores the EKF’s potential for real-time monitoring and prediction in complex biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c47c21057eed2151b03ae81e7e3e1609e8f172d" target='_blank'>
              Prediction of the Change Rate of Tumor Cells, Healthy Host Cells, and Effector Immune Cells in a Three-Dimensional Cancer Model using Extended Kalman Filter
              </a>
            </td>
          <td>
            Nina Fitriyati, Salma Abidah Faizah, Taufik Edy Sutanto
          </td>
          <td>2024-06-27</td>
          <td>Jambura Journal of Biomathematics (JJBM)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Real-world networks are sparse. As we show in this article, even when a large number of interactions is observed most node pairs remain disconnected. We demonstrate that classical multi-edge network models, such as the $G(N,p)$, configuration models, and stochastic block models, fail to accurately capture this phenomenon. To mitigate this issue, zero-inflation must be integrated into these traditional models. Through zero-inflation, we incorporate a mechanism that accounts for the excess number of zeroes (disconnected pairs) observed in empirical data. By performing an analysis on all the datasets from the Sociopatterns repository, we illustrate how zero-inflated models more accurately reflect the sparsity and heavy-tailed edge count distributions observed in empirical data. Our findings underscore that failing to account for these ubiquitous properties in real-world networks inadvertently leads to biased models which do not accurately represent complex systems and their dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f81d421fba2326c997d828fe267b1ec0356ef368" target='_blank'>
              Empirical Networks are Sparse: Enhancing Multi-Edge Models with Zero-Inflation
              </a>
            </td>
          <td>
            Giona Casiraghi, George Andres
          </td>
          <td>2024-06-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Deterministic Neural Networks (NNs) have been widely adopted to represent system behaviors in various engineering problems, which have been criticized for providing overconfident predictions. This challenge, i.e., overfitting, refers to a model that captures data patterns and random noise during the learning process, leading to an inferior predictive capability in the unobserved domains. As known to all, Bayesian inference offers an effective way for uncertainty quantification (UQ) to help enhance model predictions and offer informed decision-making. Integrating deterministic NN models with Bayesian inference provides a way to fast simulate and quantify both data and model uncertainty simultaneously. Thereby, Bayesian Neural Networks (BNNs), a combination of standard NNs with Bayesian philosophy, introduce probability to the network parameters, i.e., biases and weights. In BNNs, predictive uncertainty reflects how confident model predictions are regarding the corresponding credible intervals, which helps identify overfitting and interpreting abnormal predictions. Another advantage is that BNNs naturally implement regularization by integrating over the weights such that BNNs can reach good generalization when data availability is limited. BNNs rely heavily on observations as a purely data-driven approach. Besides, standard BNNs have relatively complex network structures, which makes it hard to provide detailed interpretability on model predictions. Researchers share concerns about their predictions that may violate real-world phenomena. The combination of physics knowledge with the optimization objective can improve the adherence of predictions with the known physical principles. Physics laws are usually embedded as constraint functions in the process of optimization. With the help of constrained optimization, model predictions keep consistent with the known physical principles.
 The authors refine BNNs with physics-guided information, i.e., Physics-Guided BNNs (PG-BNNs), to reduce dependence on data, improve model interpretability, and offer reliable model predictions. PG-BNNs interpret the known physics knowledge as constraint functions to better guide optimization processes, especially for case studies with sparse and limited data sets. A case study of an ordinary differential equation (ODE) problem is carried out to compare the performance of PG-BNN and unconstrained BNN models and demonstrate the effectiveness of PG-BNN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd95612093b011b8ce714901de80f0888b466c28" target='_blank'>
              Physics-Guided Bayesian Neural Networks and Their Application in ODE Problems
              </a>
            </td>
          <td>
            Xinyue Xu, Suman Paneru, Sez A. Russcher, Julian Wang
          </td>
          <td>2024-05-15</td>
          <td>ASME 2024 Verification, Validation, and Uncertainty Quantification Symposium</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human induced pluripotent stem cells (hiPSCs)-derived neurons offer a valuable platform for studying neurological disorders in a patient-specific manner. These neurons can be rapidly differentiated into excitatory neuronal networks, whose activity is measurable using multi-electrode arrays (MEAs). Neuronal networks derived from patients exhibit distinct characteristics, reflecting underlying pathological molecular mechanisms. However, elucidating these mechanisms traditionally requires extensive and hypothesis-driven additional experiments. Computational models can link observable network activity to underlying molecular mechanisms by identifying biophysical model parameters that simulate the activity, but this identification process is challenging. Here, we address this challenge by using simulation-based inference (SBI), a machine-learning approach, to automatically identify the full range of model parameters able to explain the patient-derived MEA activity. Our study demonstrates that SBI can accurately identify ground-truth parameters in simulated data, and successfully estimate the parameters that replicate the network activity of healthy hiPSC-derived neuronal networks. Furthermore, we show that SBI can pinpoint molecular mechanisms affected by pharmacological agents and identify key disease mechanisms in neuronal networks derived from patients. These findings underscore the potential of SBI to automate and enhance the identification of disease mechanisms from MEA measurements, offering a robust and scalable method for advancing research with hiPSC-derived neuronal networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c641603a2ff64bd27f582b8d6f3dd5aa1161d658" target='_blank'>
              Automated inference of disease mechanisms in patient-hiPSC-derived neuronal networks
              </a>
            </td>
          <td>
            Nina Doorn, Michel J.A.M. van Putten, Monica Frega
          </td>
          <td>2024-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The disease known as breast cancer continues to be one of the most common and potentially fatal diseases that affect women all over the world. In order to provide effective therapy and more favorable outcomes for patients, early detection and correct diagnosis are absolutely necessary. This research explores the integration of operational research (OR) methodologies and advanced statistical methods within machine learning frameworks to enhance the prediction accuracy of breast cancer outcomes. By harnessing the power of nonlinear optimization and statistical techniques, including regression analysis and probability distribution models, we aim to refine the predictive capabilities of existing algorithms. The research employs a comprehensive dataset derived from clinical trials and patient records, analyzed through a series of machine learning models that incorporate elements of combinatorial optimization, decision analysis, and stochastic modeling. Key performance metrics, such as accuracy, sensitivity, and specificity, are evaluated against standard benchmarks to determine the efficacy of the integrated approaches. Preliminary results indicate that incorporating OR and statistical methods significantly improves model robustness and predictive accuracy. The study not only demonstrates the potential of applied mathematics in medical diagnostics but also provides a framework for future research in enhancing machine learning models for health outcomes prediction through mathematical innovations. This investigation contributes to the field of mathematical oncology by demonstrating how applied nonlinear analysis can bridge the gap between theoretical mathematical approaches and practical clinical applications, offering new pathways for early and more accurate detection of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91e2ad7ffea30ec7bd6b8cdf76f263552d6be77" target='_blank'>
              Integration of Complex Mathematical Approaches and Statistical Methodologies in Machine Learning for Design of Robust Prediction Models
              </a>
            </td>
          <td>
            Sarita Silaich, Dr Rajesh yadav
          </td>
          <td>2024-05-17</td>
          <td>Communications on Applied Nonlinear Analysis</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Peptide- and protein-based therapeutics are becoming a promising treatment regimen for myriad diseases. Toxicity of proteins is the primary hurdle for protein-based therapies. Thus, there is an urgent need for accurate in silico methods for determining toxic proteins to filter the pool of potential candidates. At the same time, it is imperative to precisely identify non-toxic proteins to expand the possibilities for protein-based biologics. To address this challenge, we proposed an ensemble framework, called VISH-Pred, comprising models built by fine-tuning ESM2 transformer models on a large, experimentally validated, curated dataset of protein and peptide toxicities. The primary steps in the VISH-Pred framework are to efficiently estimate protein toxicities taking just the protein sequence as input, employing an under sampling technique to handle the humongous class-imbalance in the data and learning representations from fine-tuned ESM2 protein language models which are then fed to machine learning techniques such as Lightgbm and XGBoost. The VISH-Pred framework is able to correctly identify both peptides/proteins with potential toxicity and non-toxic proteins, achieving a Matthews correlation coefficient of 0.737, 0.716 and 0.322 and F1-score of 0.759, 0.696 and 0.713 on three non-redundant blind tests, respectively, outperforming other methods by over \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $10\%$\end{document} on these quality metrics. Moreover, VISH-Pred achieved the best accuracy and area under receiver operating curve scores on these independent test sets, highlighting the robustness and generalization capability of the framework. By making VISH-Pred available as an easy-to-use web server, we expect it to serve as a valuable asset for future endeavors aimed at discerning the toxicity of peptides and enabling efficient protein-based therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/830f079e12a11f4914fe454d00e34c6bd71b06d6" target='_blank'>
              VISH-Pred: an ensemble of fine-tuned ESM models for protein toxicity prediction
              </a>
            </td>
          <td>
            Raghvendra Mall, Ankita Singh, Chirag N Patel, Gregory Guirimand, Filippo Castiglione
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Summary The retarded transient function (RTF) approach has been introduced as a complementary approach to employing ordinary differential equations (ODEs) for modelling dynamics typically observed for cellular signalling processes. Here, we introduce the R package to the RTF approach, which has originally been implemented within the MATLAB-based Data2Dynamics modelling framework. The package enables modelling of time and dose dependencies and includes model reduction to minimize overfitting. It can be applied not only to experimental data but also to trajectories of ODE models, to characterize the dynamics or to identify major targets of experimental perturbations in the low-dimensional representations calculated by the package. Availability and Implementation The R package RTF is available at https://github.com/kreutz-lab/RTF. Contact clemens.kreutz@uniklinik-freiburg.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0620e9d50945b256e9edbd53674e984230196e5b" target='_blank'>
              RTF: An R package for modelling time course data
              </a>
            </td>
          <td>
            Eva Brombacher, Clemens Kreutz
          </td>
          <td>2024-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recurrent Neural Networks excel at predicting and generating complex high-dimensional temporal patterns. Due to their inherent nonlinear dynamics and memory, they can learn unbounded temporal dependencies from data. In a Machine Learning setting, the network's parameters are adapted during a training phase to match the requirements of a given task/problem increasing its computational capabilities. After the training, the network parameters are kept fixed to exploit the learned computations. The static parameters thereby render the network unadaptive to changing conditions, such as external or internal perturbation. In this manuscript, we demonstrate how keeping parts of the network adaptive even after the training enhances its functionality and robustness. Here, we utilize the conceptor framework and conceptualize an adaptive control loop analyzing the network's behavior continuously and adjusting its time-varying internal representation to follow a desired target. We demonstrate how the added adaptivity of the network supports the computational functionality in three distinct tasks: interpolation of temporal patterns, stabilization against partial network degradation, and robustness against input distortion. Our results highlight the potential of adaptive networks in machine learning beyond training, enabling them to not only learn complex patterns but also dynamically adjust to changing environments, ultimately broadening their applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d77920a1f3057b33f9ab48db38a16dc2b0f292" target='_blank'>
              Adaptive control of recurrent neural networks using conceptors
              </a>
            </td>
          <td>
            Guillaume Pourcel, Mirko Goldmann, Ingo Fischer, Miguel C. Soriano
          </td>
          <td>2024-05-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Drug-side effect prediction has become an essential area of research in the field of pharmacology. As the use of medications continues to rise, so does the importance of understanding and mitigating the potential risks associated with them. At present, researchers have turned to data-driven methods to predict drug-side effects. Drug-side effect prediction is a link prediction problem, and the related data can be described from various perspectives. To process these kinds of data, a multi-view method, called Multiple Kronecker RLS fusion-based link propagation (MKronRLSF-LP), is proposed. MKronRLSF-LP extends the Kron-RLS by finding the consensus partitions and multiple graph Laplacian constraints in the multi-view setting. Both of these multi-view settings contribute to a higher quality result. Extensive experiments have been conducted on drug-side effect datasets, and our empirical results provide evidence that our approach is effective and robust.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63527bfc5bdbb286aecbc16d31c2e4de057109d" target='_blank'>
              Multiple Kronecker RLS fusion-based link propagation for drug-side effect prediction
              </a>
            </td>
          <td>
            Yuqing Qian, Ziyu Zheng, Prayag Tiwari, Yijie Ding, Quan Zou
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1446b534858fc3fc6d9406920450b5c3ca87ee13" target='_blank'>
              Predicting equilibrium distributions for molecular systems with deep learning
              </a>
            </td>
          <td>
            Shuxin Zheng, Jiyan He, Chang Liu, Yu Shi, Ziheng Lu, Weitao Feng, Fusong Ju, Jiaxi Wang, Jianwei Zhu, Yaosen Min, He Zhang, Shidi Tang, Hongxia Hao, Peiran Jin, Chi Chen, Frank Noé, Haiguang Liu, Tie-Yan Liu
          </td>
          <td>2024-05-08</td>
          <td>Nature Machine Intelligence</td>
          <td>5</td>
          <td>9</td>
        </tr>

        <tr id="A growing number of mathematical models have been developed, providing a powerful means to forecast tumor growth. These models hold great promise as tools in the advancement of personalized medical care. Yet, their clinical relevance often remains restricted. Adequate parameter estimation and integration of existing clinical data from the patient is crucial for individualized predictions. Since CT scans form an integral part of radiotherapy, a widely used cancer treatment, we propose the use of CT-derived radiomic features to attempt to predict the course of tumor growth curves. Therefore, in this exploratory study, we examine the association between radiomic features and the three parameters of the Gompertz tumor growth model. As a proof of concept, murine tumor growth (CT26) is monitored and the Gompertz model is fitted to each animal's data to obtain the parameters. Radiomic features of the tumors were then obtained via CT, which were subsequently analyzed using exploratory factor analysis. Finally, correlations between the two were computed. We were able to discover latent factors within the radiomic features that revealed significant correlations with the parameters of the Gompertz model. Additionally, distinct features exhibiting strong correlations with the factors were identified. These may be instrumental in predicting future tumor growth. These findings offer encouraging preliminary evidence for supporting the notion that CT scans contain information that can estimate the future course of tumor growth curves. This approach opens up promising possibilities for personalized medicine in oncology, by facilitating the development of treatment plans tailored to individual patients' tumor growth patterns, right from the time of diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c33415d539a913b222861f5873ec836c14602c7" target='_blank'>
              Exploring the Potential of Radiomic Features in Predictive Murine Tumor Growth Modeling
              </a>
            </td>
          <td>
            Assia Hamitou, A. Rifi, I. Dufait, Camille Raets, M. De Ridder, Kurt Barbé
          </td>
          <td>2024-05-20</td>
          <td>2024 IEEE International Instrumentation and Measurement Technology Conference (I2MTC)</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Predicting drug toxicity is a critical aspect of ensuring patient safety during the drug design process. Although conventional machine learning techniques have shown some success in this field, the scarcity of annotated toxicity data poses a significant challenge in enhancing models’ performance. In this study, we explore the potential of leveraging large unlabeled datasets using semi-supervised learning to improve predictive performance for cardiotoxicity across three targets: the voltage-gated potassium channel (hERG), the voltage-gated calcium channel (Cav1.2), and the voltage-gated sodium channel (Nav1.5). We extensively mined the ChEMBL database, comprising approximately 2 million small molecules, then employed semi-supervised learning to construct robust classification models for this purpose. We achieved a performance boost on highly diverse (i.e. structurally dissimilar) test datasets across all three targets. Using our built models, we screened the whole ChEMBL database and a large set of FDA-approved drugs, identifying several compounds with potential cardiac channel activity. To ensure broad accessibility and usability for both technical and non-technical users, we developed a cross-platform graphical user interface that allows users to make predictions and gain insights into the cardiotoxicity of drugs and other small molecules. The software is made available as open source under the permissive MIT license at https://github.com/issararab/CToxPred2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f0e09b146e54215719071ab7b9f648e458c75b" target='_blank'>
              Semi-Supervised Learning to Boost Cardiotoxicity Prediction by Mining a Large Unlabeled Small Molecule Dataset
              </a>
            </td>
          <td>
            Issar Arab, Kris Laukens, Wout Bittremieux
          </td>
          <td>2024-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The field of machine learning has been evolving and applied in medical applications. We utilised a public dataset, MIMIC-III, to develop compact models that can accurately predict the outcome of mechanically ventilated patients in the first 24 h of first-time hospital admission.67 predictive features, grouped into 6 categories, were selected for the classification and prediction task. 4 tree-based algorithms (Decision Tree, Bagging, eXtreme Gradient Boosting and Random Forest), and 5 non-tree-based algorithms (Logistic Regression, K-Nearest Neighbour, Linear Discriminant Analysis, Support Vector Machine and Naïve Bayes), were employed to predict the outcome of 18,883 mechanically ventilated patients. 5 scenarios were crafted to mirror the target population as per existing literature. S1.1 reflected an imbalanced situation, with significantly fewer mortality cases than survival ones, and both the training and test sets played similar target class distributions. S1.2 and S2.2 featured balanced classes; however, instances from the majority class were removed from the test set and/or the training set. S1.3 and S 2.3 generated additional instances of the minority class via the Synthetic Minority Over-sampling Technique. Standard evaluation metrics were used to determine the best-performing models for each scenario. With the best performers, Autofeat, an automated feature engineering library, was used to eliminate less important features per scenario.Tree-based models generally outperformed the non-tree-based ones. Moreover, XGB consistently yielded the highest AUC score (between 0.91 and 0.97), while exhibiting relatively high Sensitivity (between 0.58 and 0.88) on 4 scenarios (1.2, 2.2, 1.3, and 2.3). After reducing a significant number of predictors, the selected calibrated ML models were still able to achieve similar AUC and MCC scores across those scenarios. The calibration curves of the XGB and BG models, both prior to and post dimension reduction in Scenario 2.2, showed better alignment to the perfect calibration line than curves produced from other algorithms.This study demonstrated that dimension-reduced models can perform well and are able to retain the important features for the classification tasks. Deploying a compact machine learning model into production helps reduce costs in terms of computational resources and monitoring changes in input data over time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/287a2fc8b7b19b61518b06105a0998ead22cc3db" target='_blank'>
              Compact machine learning model for the accurate prediction of first 24-hour survival of mechanically ventilated patients
              </a>
            </td>
          <td>
            Quynh T. Nguyen, Mai P. Tran, Vishnu Prabhakaran, Andrew Liu, Ghi H Nguyen
          </td>
          <td>2024-06-20</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In chemical engineering, process data is often expensive to acquire, and complex phenomena are difficult to model rigorously, rendering both entirely data-driven and purely mechanistic modeling approaches impractical. We explore using physics-informed neural networks (PINNs) for modeling dynamic processes governed by differential-algebraic equation systems when process data is scarce and complete mechanistic knowledge is missing. In particular, we focus on estimating states for which neither direct observational data nor constitutive equations are available. For demonstration purposes, we study a continuously stirred tank reactor and a liquid-liquid separator. We find that PINNs can infer unmeasured states with reasonable accuracy, and they generalize better in low-data scenarios than purely data-driven models. We thus show that PINNs, similar to hybrid mechanistic/data-driven models, are capable of modeling processes when relatively few experimental data and only partially known mechanistic descriptions are available, and conclude that they constitute a promising avenue that warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4e0161b8d6befa3c8a0687b96bbfc9b32e40f7" target='_blank'>
              Physics-Informed Neural Networks for Dynamic Process Operations with Limited Physical Knowledge and Data
              </a>
            </td>
          <td>
            M. Velioglu, Song Zhai, Sophia Rupprecht, Alexander Mitsos, Andreas Jupke, M. Dahmen
          </td>
          <td>2024-06-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Bayesian graphical models are powerful tools to infer complex relationships in high dimension, yet are often fraught with computational and statistical challenges. If exploited in a principled way, the increasing information collected alongside the data of primary interest constitutes an opportunity to mitigate these difficulties by guiding the detection of dependence structures. For instance, gene network inference may be informed by the use of publicly available summary statistics on the regulation of genes by genetic variants. Here we present a novel Gaussian graphical modeling framework to identify and leverage information on the centrality of nodes in conditional independence graphs. Specifically, we consider a fully joint hierarchical model to simultaneously infer (i) sparse precision matrices and (ii) the relevance of node-level information for uncovering the sought-after network structure. We encode such information as candidate auxiliary variables using a spike-and-slab submodel on the propensity of nodes to be hubs, which allows hypothesis-free selection and interpretation of a sparse subset of relevant variables. As efficient exploration of large posterior spaces is needed for real-world applications, we develop a variational expectation conditional maximization algorithm that scales inference to hundreds of samples, nodes and auxiliary variables. We illustrate and exploit the advantages of our approach in simulations and in a gene network study which identifies hub genes involved in biological pathways relevant to immune-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfbfe802cb5f3c9c2e7e580929c13fe21fe00c25" target='_blank'>
              A modeling framework for detecting and leveraging node-level information in Bayesian network inference.
              </a>
            </td>
          <td>
            Xiaoyue Xi, H. Ruffieux
          </td>
          <td>2024-06-25</td>
          <td>Biostatistics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Interpretable mathematical expressions defining discrete-time dynamical systems (iterated maps) can model many phenomena of scientific interest, enabling a deeper understanding of system behaviors. Since formulating governing expressions from first principles can be difficult, it is of particular interest to identify expressions for iterated maps given only their data streams. In this work, we consider a modified Symbolic Artificial Neural Network-Trained Expressions (SymANNTEx) architecture for this task, an architecture more expressive than others in the literature. We make a modification to the model pipeline to optimize the regression, then characterize the behavior of the adjusted model in identifying several classical chaotic maps. With the goal of parsimony, sparsity-inducing weight regularization and information theory-informed simplification are implemented. We show that our modified SymANNTEx model properly identifies single-state maps and achieves moderate success in approximating a dual-state attractor. These performances offer significant promise for data-driven scientific discovery and interpretation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3502a1e8d5f9351f334ca3982d09f0f0f0433666" target='_blank'>
              Expressive Symbolic Regression for Interpretable Models of Discrete-Time Dynamical Systems
              </a>
            </td>
          <td>
            Adarsh Iyer, N. Boddupalli, Jeff Moehlis
          </td>
          <td>2024-06-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Optimizing complex and high-dimensional black-box functions is ubiquitous in science and engineering fields. Unfortunately, the online evaluation of these functions is restricted due to time and safety constraints in most cases. In offline model-based optimization (MBO), we aim to find a design that maximizes the target function using only a pre-existing offline dataset. While prior methods consider forward or inverse approaches to address the problem, these approaches are limited by conservatism and the difficulty of learning highly multi-modal mappings. Recently, there has been an emerging paradigm of learning to improve solutions with synthetic trajectories constructed from the offline dataset. In this paper, we introduce a novel conditional generative modeling approach to produce trajectories toward high-scoring regions. First, we construct synthetic trajectories toward high-scoring regions using the dataset while injecting locality bias for consistent improvement directions. Then, we train a conditional diffusion model to generate trajectories conditioned on their scores. Lastly, we sample multiple trajectories from the trained model with guidance to explore high-scoring regions beyond the dataset and select high-fidelity designs among generated trajectories with the proxy function. Extensive experiment results demonstrate that our method outperforms competitive baselines on Design-Bench and its practical variants. The code is publicly available in \texttt{https://github.com/dbsxodud-11/GTG}.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fa109cf8ef9888e5f0dbfe9a6a6d7918594856e" target='_blank'>
              Guided Trajectory Generation with Diffusion Models for Offline Model-based Optimization
              </a>
            </td>
          <td>
            Taeyoung Yun, Sujin Yun, Jaewoo Lee, Jinkyoo Park
          </td>
          <td>2024-06-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c1cd074b744953a14cf72dff23f50b500e57211" target='_blank'>
              APF2: an improved ensemble method for pharmacogenomic variant effect prediction
              </a>
            </td>
          <td>
            Yitian Zhou, Sebastian Pirmann, V. Lauschke
          </td>
          <td>2024-05-27</td>
          <td>The Pharmacogenomics Journal</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983a84ba84d4e4978cb8f3fe20f03553417b6b7e" target='_blank'>
              DeepARV: ensemble deep learning to predict drug-drug interaction of clinical relevance with antiretroviral therapy
              </a>
            </td>
          <td>
            Thao Pham, Mohamed Ghafoor, Sandra Grañana-Castillo, C. Marzolini, Sara Gibbons, Saye Khoo, J. Chiong, Dennis Wang, Marco Siccardi
          </td>
          <td>2024-05-06</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Due to the lack of experimental toxicity data for environmental chemicals, there arises a need to fill data gaps by in silico approaches. One of the most commonly used in silico approaches for toxicity assessment of small datasets is the Quantitative Structure-Activity Relationship (QSAR), which generates predictive models for the efficient prediction of query compounds. However, the reliability of the predictions from QSARs derived from small datasets is often questionable from a statistical point of view. This is due to the presence of a larger number of descriptors as compared to the number of training compounds, which reduces the degree of freedom of the developed model. To reduce the overall prediction error for a particular QSAR model, we have proposed here the computation of the novel Arithmetic Residuals in K-groups Analysis (ARKA) descriptors. We have reduced the number of modeling descriptors in a supervised manner by partitioning them into K classes (K = 2 here) depending on the higher mean normalized values of the descriptors to a particular response class, thus preventing the loss of chemical information. A scatter plot of the data points using the values of two ARKA descriptors (ARKA_2 vs. ARKA_1) can potentially identify activity cliffs, less confident data points, and less modelable data points. We have used here five representative environmentally relevant endpoints (skin sensitization, earthworm toxicity, milk/plasma partitioning, algal toxicity, and rodent carcinogenicity of hazardous chemicals) with graded responses to which the ARKA framework was applied for classification modeling. On comparing the performance of the models generated using conventional QSAR descriptors and the ARKA descriptors, the prediction quality of the models derived from ARKA descriptors was found, based on multiple graded-data validation metrics-derived decision criteria, much better than the models derived from QSAR descriptors signifying the potential of ARKA descriptors in ecotoxicological classification modeling of small data sets. Additionally, this holds true for the Read-Across approach as well, since the Read-Across predictions using ARKA descriptors supersede the predictions generated from QSAR descriptors. For the ease of users, a Java-based expert system has been developed that computes the ARKA descriptors from the input of QSAR descriptors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/543e009fa9a7316f55e90d44767e72704b6417c1" target='_blank'>
              ARKA: a framework of dimensionality reduction for machine-learning classification modeling, risk assessment, and data gap-filling of sparse environmental toxicity data.
              </a>
            </td>
          <td>
            Arkaprava Banerjee, Kunal Roy
          </td>
          <td>2024-05-14</td>
          <td>Environmental science. Processes & impacts</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In the past two decades, machine learning potentials (MLPs) have driven significant developments in chemical, biological, and material sciences. The construction and training of MLPs enable fast and accurate simulations and analysis of thermodynamic and kinetic properties. This review focuses on the application of MLPs to reaction systems with consideration of bond breaking and formation. We review the development of MLP models, primarily with neural network and kernel-based algorithms, and recent applications of reactive MLPs (RMLPs) to systems at different scales. We show how RMLPs are constructed, how they speed up the calculation of reactive dynamics, and how they facilitate the study of reaction trajectories, reaction rates, free energy calculations, and many other calculations. Different data sampling strategies applied in building RMLPs are also discussed with a focus on how to collect structures for rare events and how to further improve their performance with active learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe08859388b658711cbf4ac096f214e5d2eca580" target='_blank'>
              Machine Learning of Reactive Potentials.
              </a>
            </td>
          <td>

          </td>
          <td>2024-06-01</td>
          <td>Annual review of physical chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Our interest is in replicated network data with multiple networks observed across the same set of nodes. Examples include brain connection networks, in which nodes corresponds to brain regions and replicates to different individuals, and ecological networks, in which nodes correspond to species and replicates to samples collected at different locations and/or times. Our goal is to infer a hierarchical structure of the nodes at a population level, while performing multi-resolution clustering of the individual replicates. In brain connectomics, the focus is on inferring common relationships among the brain regions, while characterizing inter-individual variability in an easily interpretable manner. To accomplish this, we propose a Bayesian hierarchical model, while providing theoretical support in terms of identifiability and posterior consistency, and design efficient methods for posterior computation. We provide novel technical tools for proving model identifiability, which are of independent interest. Our simulations and application to brain connectome data provide support for the proposed methodology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a92bd9ad8c8ee3e22f5fcf7a819cb2d7dd4b471" target='_blank'>
              Bayesian Deep Generative Models for Replicated Networks with Multiscale Overlapping Clusters
              </a>
            </td>
          <td>
            Yuren Zhou, Yuqi Gu, D. Dunson
          </td>
          <td>2024-05-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="One of the central goals of neuroscience is to gain a mechanistic understanding of how the dynamics of neural circuits give rise to their observed function. A popular approach towards this end is to train recurrent neural networks (RNNs) to reproduce experimental recordings of neural activity. These trained RNNs are then treated as surrogate models of biological neural circuits, whose properties can be dissected via dynamical systems analysis. How reliable are the mechanistic insights derived from this procedure? While recent advances in population-level recording technologies have allowed simultaneous recording of up to tens of thousands of neurons, this represents only a tiny fraction of most cortical circuits. Here we show that observing only a subset of neurons in a circuit can create mechanistic mismatches between a simulated teacher network and a data-constrained student, even when the two networks have matching single-unit dynamics. In particular, partial observation of models of low-dimensional cortical dynamics based on functionally feedforward or low-rank connectivity can lead to surrogate models with spurious attractor structure. Our results illustrate the challenges inherent in accurately uncovering neural mechanisms from single-trial data, and suggest the need for new methods of validating data-constrained models for neural dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ff706d2507265f316f84e0d354f6a7ce5c6f8a" target='_blank'>
              Partial observation can induce mechanistic mismatches in data-constrained models of neural dynamics
              </a>
            </td>
          <td>
            William Qian, Jacob A. Zavatone-Veth, Benjamin S. Ruben, C. Pehlevan
          </td>
          <td>2024-05-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Despite innumerable efforts to characterize C. elegans neural activity and behavior, the roles of each individual neuron and the compositional mechanisms of all their interactions are still not completely known. In particular, there is evidence that C. elegans neurons perform changing roles over time, in different behavioral contexts, etc.; however, existing stationary anatomical and functional connectivity can average across time and obfuscate the nonstationary nature of the neural dynamics. We contribute to these efforts by leveraging recent advances in decomposed linear dynamical systems (dLDS) models. dLDS models neural dynamics in a latent space with a set of linear operators that can be recombined and reused over time, enabling the discovery of multiple parallel neural processes on different timescales. We leverage the ability to identify reused patterns of dynamical neural interactions, which we call “dynamical connectivity,” to 1) identify contextually dependent roles of neurons; 2) discover the underlying variability of neural representations even under discrete behaviors; 3) quantify differences between anatomical, functional, and dynamical connectivity; and 4) learn a single aligned latent space underlying multiple individual worms’ activity. These results highlight the importance of defining a neuron’s functions not solely by its internal activity or place in a time-averaged network, but by its evolving interactions across context-dependent circuits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165b84af20920579578d673faaf369e271200b1a" target='_blank'>
              Decomposed Linear Dynamical Systems (dLDS) models reveal context-dependent dynamic connectivity in C. elegans
              </a>
            </td>
          <td>
            Eva Yezerets, Noga Mudrik, Adam S. Charles
          </td>
          <td>2024-06-01</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Cardiovascular diseases (CVDs) pose a significant threat to global public health, affecting individuals across various age groups. Factors such as cholesterol levels, smoking, alcohol consumption, and physical inactivity contribute to their onset and progression. Enhancing our understanding of CVD etiology and informing targeted interventions for disease prevention and management remains a critical challenge. In this study, we address the task of predicting the likelihood of individuals developing CVDs using machine learning techniques. Specifically, we explore three approaches: the k-nearest neighbors (KNN) algorithm, logistic regression, and the random forest algorithm. Leveraging a comprehensive dataset sourced from Kaggle, encompassing 11 relevant factors, we conduct a series of experiments to identify the most influential predictors of CVDs. Our analysis aims not only to forecast disease occurrence but also to elucidate the primary determinants contributing to its manifestation. Through comparative analysis of the three methodologies, we demonstrate that the random forest algorithm exhibits superior performance in terms of predictive accuracy. This research represents a significant step towards leveraging machine learning techniques to enhance our understanding of CVD dynamics and inform targeted interventions for disease prevention and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2894a49a6f2f572cc91952d4b7583ed1d008b074" target='_blank'>
              Machine Learning Models for Cardiovascular Disease Prediction: A Comparative Study
              </a>
            </td>
          <td>
            Chao Yan, Yiluan Xing, Sensen Liu, Erdi Gao, Jinyin Wang
          </td>
          <td>2024-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This study assesses the predictive performance of six machine learning models and a 1D Convolutional Neural Network (CNN) in forecasting tumor dynamics within three months following Gamma Knife radiosurgery (GKRS) in 77 brain metastasis (BM) patients. The analysis meticulously evaluates each model before and after hyperparameter tuning, utilizing accuracy, AUC, and other metrics derived from confusion matrices. The CNN model showcased notable performance with an accuracy of 98% and an AUC of 0.97, effectively complementing the broader model analysis. Initial findings highlighted that XGBoost significantly outperformed other models with an accuracy of 0.95 and an AUC of 0.95 before tuning. Post-tuning, the Support Vector Machine (SVM) demonstrated the most substantial improvement, achieving an accuracy of 0.98 and an AUC of 0.98. Conversely, XGBoost showed a decline in performance after tuning, indicating potential overfitting. The study also explores feature importance across models, noting that features like “control at one year”, “age of the patient”, and “beam-on time for volume V1 treated” were consistently influential across various models, albeit their impacts were interpreted differently depending on the model’s underlying mechanics. This comprehensive evaluation not only underscores the importance of model selection and hyperparameter tuning but also highlights the practical implications in medical diagnostic scenarios, where the accuracy of positive predictions can be crucial. Our research explores the effects of staged Gamma Knife radiosurgery (GKRS) on larger tumors, revealing no significant outcome differences across protocols. It uniquely considers the impact of beam-on time and fraction intervals on treatment efficacy. However, the investigation is limited by a small patient cohort and data from a single institution, suggesting the need for future multicenter research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1145edccd36b1fcabce5d3713aae781c5134f5c4" target='_blank'>
              Predicting Tumor Dynamics Post-Staged GKRS: Machine Learning Models in Brain Metastases Prognosis
              </a>
            </td>
          <td>
            A. Trofin, C. Buzea, Razvan-Cristian Buga, M. Agop, L. Ochiuz, D. Iancu, Lucian Eva
          </td>
          <td>2024-06-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8112c18b6d184d24cba044589140338bfc4b2d5" target='_blank'>
              Advanced methods for gene network identification and noise decomposition from single-cell data
              </a>
            </td>
          <td>
            Zhou Fang, Ankit Gupta, Sant Kumar, Mustafa Khammash
          </td>
          <td>2024-06-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Many clinical trials generate both longitudinal biomarker and time-to-event data. We might be interested in their relationship, as in the case of tumor size and overall survival in oncology drug development. Many well-established methods exist for analyzing such data either sequentially (two-stage models) or simultaneously (joint models). Two-stage modeling (2stgM) has been challenged (i) for not acknowledging that biomarkers are endogenous covariable to the survival submodel and (ii) for not propagating the uncertainty of the longitudinal biomarker submodel to the survival submodel. On the other hand, joint modeling (JM), which properly circumvents both problems, has been criticized for being time-consuming, and difficult to use in practice. In this paper, we explore a third approach, referred to as a novel two-stage modeling (N2stgM). This strategy reduces the model complexity without compromising the parameter estimate accuracy. The three approaches (2stgM, JM, and N2stgM) are formulated, and a Bayesian framework is considered for their implementation. Both real and simulated data were used to analyze the performance of such approaches. In all scenarios, our proposal estimated the parameters approximately as JM but without being computationally expensive, while 2stgM produced biased results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1431f8841ec1dea194d5b4d31717f328d35e5b2" target='_blank'>
              Bridging the gap between two-stage and joint models: The case of tumor growth inhibition and overall survival models.
              </a>
            </td>
          <td>
            Danilo Alvares, F. Mercier
          </td>
          <td>2024-06-03</td>
          <td>Statistics in medicine</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Gene expression programs that establish and maintain specific cellular states are orchestrated through a regulatory network composed of transcription factors, cofactors, and chromatin regulators. Dysregulation of this network can lead to a broad range of diseases by altering gene programs. This article presents LaGrACE, a novel method designed to estimate dysregulation of gene programs utilizing omics data with clinical information. This approach facilitates grouping of samples exhibiting similar patterns of gene program dysregulation, thereby enhancing the discovery of underlying molecular mechanisms. We rigorously evaluated LaGrACE’s performance using synthetic data, bulk RNA-seq clinical datasets (breast cancer, chronic obstructive pulmonary disease (COPD)), and single-cell RNA-seq drug perturbation datasets. Our findings demonstrate that LaGrACE is exceptionally robust in identifying biologically meaningful and prognostic molecular subtypes. Additionally, it effectively discerns drug-response signals at a single-cell resolution. Moreover, the COPD analysis revealed a new association between LEF1 and COPD molecular mechanisms and mortality. Collectively, these results underscore the utility of LaGrACE as a valuable tool for elucidating underlying mechanisms of diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2d4a72bbe23e694e15804f601164485fcfac620" target='_blank'>
              LaGrACE: Estimating gene program dysregulation using latent gene regulatory network for biomedical discovery
              </a>
            </td>
          <td>
            Minxue Jia, Haiyi Mao, Mengli Zhou, Yu-Chih Chen, P. Benos
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="A central aim in computational neuroscience is to relate the activity of large populations of neurons to an underlying dynamical system. Models of these neural dynamics should ideally be both interpretable and fit the observed data well. Low-rank recurrent neural networks (RNNs) exhibit such interpretability by having tractable dynamics. However, it is unclear how to best fit low-rank RNNs to data consisting of noisy observations of an underlying stochastic system. Here, we propose to fit stochastic low-rank RNNs with variational sequential Monte Carlo methods. We validate our method on several datasets consisting of both continuous and spiking neural data, where we obtain lower dimensional latent dynamics than current state of the art methods. Additionally, for low-rank models with piecewise linear nonlinearities, we show how to efficiently identify all fixed points in polynomial rather than exponential cost in the number of units, making analysis of the inferred dynamics tractable for large RNNs. Our method both elucidates the dynamical systems underlying experimental recordings and provides a generative model whose trajectories match observed trial-to-trial variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579f56813a03517163b86fa89044dc78505bf2cb" target='_blank'>
              Inferring stochastic low-rank recurrent neural networks from neural data
              </a>
            </td>
          <td>
            Matthijs Pals, A. E. Saugtekin, Felix Pei, Manuel Gloeckler, J. H. Macke
          </td>
          <td>2024-06-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af19be9576fba29fc6ce10addc184ed0663d1348" target='_blank'>
              A Computational Predictor for Accurate Identification of Tumor Homing Peptides by Integrating Sequential and Deep BiLSTM Features.
              </a>
            </td>
          <td>
            Roha Arif, Sameera Kanwal, Saeed Ahmed, Muhammad Kabir
          </td>
          <td>2024-05-11</td>
          <td>Interdisciplinary sciences, computational life sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="In this paper, we present a guide to the foundations of learning Dynamic Bayesian Networks (DBNs) from data in the form of multiple samples of trajectories for some length of time. We present the formalism for a generic as well as a set of common types of DBNs for particular variable distributions. We present the analytical form of the models, with a comprehensive discussion on the interdependence between structure and weights in a DBN model and their implications for learning. Next, we give a broad overview of learning methods and describe and categorize them based on the most important statistical features, and how they treat the interplay between learning structure and weights. We give the analytical form of the likelihood and Bayesian score functions, emphasizing the distinction from the static case. We discuss functions used in optimization to enforce structural requirements. We briefly discuss more complex extensions and representations. Finally we present a set of comparisons in different settings for various distinct but representative algorithms across the variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c93e8d8300c924f0796c2d1e335ce010a7267e" target='_blank'>
              Learning Dynamic Bayesian Networks from Data: Foundations, First Principles and Numerical Comparisons
              </a>
            </td>
          <td>
            Vyacheslav Kungurtsev, Petr Rysavy, Fadwa Idlahcen, Pavel Rytir, Ales Wodecki
          </td>
          <td>2024-06-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In the real world, many dynamic behaviors can be explained by the propagation of perturbations, such as the transfer of chemical signals and the spread of infectious diseases. Previous researchers have achieved excellent results in approximating the global propagation time, revealing the mechanism of signal propagation through multiple paths. However, the known frameworks rely on the extension of physical concepts rather than mathematically rigorous derivations. As a result, they may not perfectly predict time or explain the underlying physical significance in certain specific cases. In this paper, we propose a novel method for decomposing network topology, focusing on two modules: the tree-like module and the path-module. Subsequently, we introduce a new framework for signal propagation analysis, which can be applied to estimate the propagation time for two fundamental global topology modules and provide a rigorous proof for the propagation time in global topology. Compared to previous work, our results are not only more concise, clearly defined, efficient, but also are more powerful in predicting propagation time which outperforms some known results in some cases, for example, biochemical dynamics.Additionally, the proposed framework is based on information transfer pathways, which can be also applied to other physical fields, such as network stability, network controlling and network resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5df6abe3831bf4c489162c2dc445d5e085e2f6f" target='_blank'>
              Information transfer pathways: signal propagation in complex global topologies
              </a>
            </td>
          <td>
            Qitong Hu, Xiao-Dong Zhang
          </td>
          <td>2024-05-20</td>
          <td>Physica Scripta</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Data science is playing an increasingly important role in the design and analysis of engineered biology. This has been fueled by the development of high-throughput methods like massively parallel reporter assays, data-rich microscopy techniques, computational protein structure prediction and design, and the development of whole-cell models able to generate huge volumes of data. Although the ability to apply data-centric analyses in these contexts is appealing and increasingly simple to do, it comes with potential risks. For example, how might biases in the underlying data affect the validity of a result and what might the environmental impact of large-scale data analyses be? Here, we present a community-developed framework for assessing data hazards to help address these concerns and demonstrate its application to two synthetic biology case studies. We show the diversity of considerations that arise in common types of bioengineering projects and provide some guidelines and mitigating steps. Understanding potential issues and dangers when working with data and proactively addressing them will be essential for ensuring the appropriate use of emerging data-intensive AI methods and help increase the trustworthiness of their applications in synthetic biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/993a0859db6bdf23a69bbfc26332b2731030205e" target='_blank'>
              Data hazards in synthetic biology
              </a>
            </td>
          <td>
            Natalie R Zelenka, Nina Di Cara, Kieren Sharma, Seeralan Sarvaharman, Jasdeep S. Ghataora, Fabio Parmeggiani, Jeff Nivala, Z. Abdallah, Lucia Marucci, T. Gorochowski
          </td>
          <td>2024-06-21</td>
          <td>Synthetic Biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Proteins are complex molecules responsible for different functions in nature. Enhancing the functionality of proteins and cellular fitness can significantly impact various industries. However, protein optimization using computational methods remains challenging, especially when starting from low-fitness sequences. We propose LatProtRL, an optimization method to efficiently traverse a latent space learned by an encoder-decoder leveraging a large protein language model. To escape local optima, our optimization is modeled as a Markov decision process using reinforcement learning acting directly in latent space. We evaluate our approach on two important fitness optimization tasks, demonstrating its ability to achieve comparable or superior fitness over baseline methods. Our findings and in vitro evaluation show that the generated sequences can reach high-fitness regions, suggesting a substantial potential of LatProtRL in lab-in-the-loop scenarios.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9589c7a5062134012d56bf3ea6aedd1d76bfb1d" target='_blank'>
              Robust Optimization in Protein Fitness Landscapes Using Reinforcement Learning in Latent Space
              </a>
            </td>
          <td>
            Minji Lee, L. Vecchietti, Hyunkyu Jung, Hyun Joo Ro, Meeyoung Cha, Ho Min Kim
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In neuroscience, understanding how single-neuron firing contributes to distributed neural ensembles is crucial. Traditional methods of analysis have been limited to descriptions of whole population activity, or, when analyzing individual neurons, criteria for response categorization varied significantly across experiments. Current methods lack scalability for large datasets, fail to capture temporal changes and rely on parametric assumptions. There’s a need for a robust, scalable, and non-parametric functional clustering approach to capture interpretable dynamics. To address this challenge, we developed a model-based, statistical framework for unsupervised clustering of multiple time series datasets that exhibit nonlinear dynamics into an a-priori-unknown number of parameterized ensembles called Functional Encoding Units (FEUs). FEU outperforms existing techniques in accuracy and benchmark scores. Here, we apply this FEU formalism to single-unit recordings collected during social behaviors in rodents and primates and demonstrate its hypothesis-generating and testing capacities. This novel pipeline serves as an analytic bridge, translating neural ensemble codes across model systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5685eae99f09bcd7815de6d87c0b5c7c06c96bb4" target='_blank'>
              An unbiased method to partition diverse neuronal responses into functional ensembles reveals interpretable population dynamics during innate social behavior
              </a>
            </td>
          <td>
            Alexander Lin, Cyril Akafia, O. Dal Monte, Siqi Fan, Nicholas A. Fagan, Philip Putnam, Kay M. Tye, Steve Chang, Demba E. Ba, A. Allsop
          </td>
          <td>2024-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="To make accurate predictions, understand mechanisms, and design interventions in systems of many variables, we wish to learn causal graphs from large scale data. Unfortunately the space of all possible causal graphs is enormous so scalably and accurately searching for the best fit to the data is a challenge. In principle we could substantially decrease the search space, or learn the graph entirely, by testing the conditional independence of variables. However, deciding if two variables are adjacent in a causal graph may require an exponential number of tests. Here we build a scalable and flexible method to evaluate if two variables are adjacent in a causal graph, the Differentiable Adjacency Test (DAT). DAT replaces an exponential number of tests with a provably equivalent relaxed problem. It then solves this problem by training two neural networks. We build a graph learning method based on DAT, DAT-Graph, that can also learn from data with interventions. DAT-Graph can learn graphs of 1000 variables with state of the art accuracy. Using the graph learned by DAT-Graph, we also build models that make much more accurate predictions of the effects of interventions on large scale RNA sequencing data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aba9e483fc80b2246e0043768867a60815d1c888" target='_blank'>
              Scalable and Flexible Causal Discovery with an Efficient Test for Adjacency
              </a>
            </td>
          <td>
            Alan Nawzad Amin, Andrew Gordon Wilson
          </td>
          <td>2024-06-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The key goals in learning Bayesian networks (BNs) from data are to identify significant statistical relationships between variables and to build a Directed Acyclic Graph (DAG) that represents these relationships through Joint Probability Distributions. Most research relies on score-based or conditional test methods for model selection. However, when using real-world data, it can be challenging to identify whether the learned DAG represents the underlying relations inherent in the limited datasets, particularly when evaluating data obtained from multiple independent sources. This study presents a methodology to assess the credible interval for both the existence and direction of each edge within Bayesian networks derived from data. Furthermore, it explores the fusion of models acquired from distinct and independent datasets. This approach enables the Bayesian learning of Bayesian Networks (BNs) from data by treating the uncertainty associated with the existence and orientation of each edge as a random variable. By evaluating the probability of the orientation of each edge, it is possible to suggest the existence of a potential latent variable within the dataset. If an edge exhibits equiprobable directions and is verified to exist, it becomes a plausible hypothesis for a latent variable. The Fast Causal Algorithm, originally introduced by [1], is the foundation of this approach. Finally, by employing a maximum a posteriori estimation, the most prominent edges and their respective orientations are identified and employed to create the leading DAG. We present our findings in simulated datasets with different length sizes. By comparing the structure of the learned DAGs with existing structures and evaluating the inference capabilities of the final BN, we establish that our approach achieves results comparable to the most recent studies in the field, while offering insights into the model’s reliability and improving the use of the data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e44ff83d27d27148c5c9f26a576e56ad85ceed40" target='_blank'>
              Quantifying uncertainty in Bayesian Networks structural learning
              </a>
            </td>
          <td>
            Vitor O. Barth, H. O. Caetano, Carlos D. Maciel, Marco Aiello
          </td>
          <td>2024-05-23</td>
          <td>2024 IEEE International Conference on Evolving and Adaptive Intelligent Systems (EAIS)</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Through integrating the evolutionary correlations across global states in the bidirectional recursion, an explainable Bayesian recurrent neural smoother (EBRNS) is proposed for offline data-assisted fixed-interval state smoothing. At first, the proposed model, containing global states in the evolutionary interval, is transformed into an equivalent model with bidirectional memory. This transformation incorporates crucial global state information with support for bi-directional recursive computation. For the transformed model, the joint state-memory-trend Bayesian filtering and smoothing frameworks are derived by introducing the bidirectional memory iteration mechanism and offline data into Bayesian estimation theory. The derived frameworks are implemented using the Gaussian approximation to ensure analytical properties and computational efficiency. Finally, the neural network modules within EBRNS and its two-stage training scheme are designed. Unlike most existing approaches that artificially combine deep learning and model-based estimation, the bidirectional recursion and internal gated structures of EBRNS are naturally derived from Bayesian estimation theory, explainably integrating prior model knowledge, online measurement, and offline data. Experiments on representative real-world datasets demonstrate that the high smoothing accuracy of EBRNS is accompanied by data efficiency and a lightweight parameter scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57349533774d9106325c9d199314485305e9ade6" target='_blank'>
              Explainable Bayesian Recurrent Neural Smoother to Capture Global State Evolutionary Correlations
              </a>
            </td>
          <td>
            Shi Yan, Yan Liang, Huayu Zhang, Le Zheng, Difan Zou, Binglu Wang
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The increasing prevalence of Diabetes Mellitus (DM) as a global health concern highlights the paramount importance of accurately predicting its progression. This necessity has propelled the use of deep learning’s advanced analytical and predictive capabilities to the forefront of current research. However, this approach is confronted with significant challenges, notably the prevalence of incomplete data and the need for more robust predictive models. Our research aims to address these critical issues, leveraging deep learning to enhance the precision and reliability of diabetes progression predictions. We address the issue of missing data by first locating individuals with data gaps within specific patient clusters, and then applying targeted imputation strategies for effective data imputation. To enhance the robustness of our model, we implement strategies such as data augmentation and the development of advanced group-level feature analysis. A cornerstone of our approach is the implementation of a deep attentive transformer that is sensitive to group characteristics. This framework excels in processing a wide array of data, including clinical and physical examination information, to accurately predict the progression of DM. Beyond its predictive capabilities, our model is engineered to perform advanced feature selection and reasoning. This is crucial for understanding the impact of both individual and group-level factors on deep models’ predictions, providing invaluable insights into the dynamics of DM progression. Our approach not only marks a significant advancement in the prediction of diabetes progression but also contributes to a deeper understanding of the multifaceted factors influencing this chronic disease, thereby aiding in more effective diabetes management and research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a6aee01e185686bab2a1ba25742acdd41abd0a" target='_blank'>
              Group-informed attentive framework for enhanced diabetes mellitus progression prediction
              </a>
            </td>
          <td>
            Changting Sheng, Luyao Wang, Caiyi Long, Rensong Yue
          </td>
          <td>2024-06-24</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The aging process is a complex and multifaceted phenomenon affecting all living organisms. It involves a gradual deterioration of tissue and cellular function, leading to a higher risk of developing various age-related diseases (ARDs), including cancer, neurodegenerative, and cardiovascular diseases. The gene regulatory networks (GRNs) and their respective niches are crucial in determining the aging rate. Unveiling these GRNs holds promise for developing novel therapies and diagnostic tools to enhance healthspan and longevity. This review examines GRN modeling approaches in aging, encompassing differential equations, Boolean/fuzzy logic decision trees, Bayesian networks, mutual information, and regression clustering. These approaches provide nuanced insights into the intricate gene-protein interactions in aging, unveiling potential therapeutic targets and ARD biomarkers. Nevertheless, outstanding challenges persist, demanding more comprehensive datasets and advanced algorithms to comprehend and predict GRN behavior accurately. Despite these hurdles, identifying GRNs associated with aging bears immense potential and is poised to transform our comprehension of human health and aging. This review aspires to stimulate further research in aging, fostering the innovation of computational approaches for promoting healthspan and longevity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126e6aff53e70f3203e5ef0b41bbc26dab6f16fc" target='_blank'>
              Computational modeling of aging-related gene networks: a review
              </a>
            </td>
          <td>
            José Américo Nabuco Leva Ferreira Freitas, Oliver Bischof
          </td>
          <td>2024-05-28</td>
          <td>Frontiers in Applied Mathematics and Statistics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Predicting cancer-associated clinical events is challenging in oncology. In Multiple Myeloma (MM), a cancer of plasma cells, disease progression is determined by changes in biomarkers, such as serum concentration of the paraprotein secreted by plasma cells (M-protein). Therefore, the time-dependent behaviour of M-protein and the transition across lines of therapy (LoT) that may be a consequence of disease progression should be accounted for in statistical models to predict relevant clinical outcomes. Furthermore, it is important to understand the contribution of the patterns of longitudinal biomarkers, upon each LoT initiation, to time-to-death or time-to-next-LoT. Motivated by these challenges, we propose a Bayesian joint model for trajectories of multiple longitudinal biomarkers, such as M-protein, and the competing risks of death and transition to next LoT. Additionally, we explore two estimation approaches for our joint model: simultaneous estimation of all parameters (joint estimation) and sequential estimation of parameters using a corrected two-stage strategy aiming to reduce computational time. Our proposed model and estimation methods are applied to a retrospective cohort study from a real-world database of patients diagnosed with MM in the US from January 2015 to February 2022. We split the data into training and test sets in order to validate the joint model using both estimation approaches and make dynamic predictions of times until clinical events of interest, informed by longitudinally measured biomarkers and baseline variables available up to the time of prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55580b4051352431cf661d36a73d6403047a57db" target='_blank'>
              A Bayesian joint model of multiple nonlinear longitudinal and competing risks outcomes for dynamic prediction in multiple myeloma: joint estimation and corrected two-stage approaches
              </a>
            </td>
          <td>
            Danilo Alvares, Jessica K. Barrett, Franccois Mercier, S. Roumpanis, Sean Yiu, Felipe Castro, Jochen Schulze, Yajing Zhu
          </td>
          <td>2024-05-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938f503cc0bdd3355c300a529506c4292f5f9afd" target='_blank'>
              Biologically meaningful genome interpretation models to address data underdetermination for the leaf and seed ionome prediction in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Daniele Raimondi, Antoine Passemiers, N. Verplaetse, Massimiliano Corso, Ángel Ferrero-Serrano, Nelson Nazzicari, Filippo Biscarini, Piero Fariselli, Yves Moreau
          </td>
          <td>2024-06-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5916a879490d67fb48ad872b52d27539ca4a54" target='_blank'>
              Deep representation learning of chemical-induced transcriptional profile for phenotype-based drug discovery
              </a>
            </td>
          <td>
            X. Tong, Ning Qu, Xiangtai Kong, Shengkun Ni, Jingyi Zhou, Kun Wang, Lehan Zhang, Yiming Wen, Jiangshan Shi, Sulin Zhang, Xutong Li, Mingyue Zheng
          </td>
          <td>2024-06-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="With the development of high-throughput technologies, genomics datasets rapidly grow in size, including functional genomics data. This has allowed the training of large Deep Learning (DL) models to predict epigenetic readouts, such as protein binding or histone modifications, from genome sequences. However, large dataset sizes come at a price of data consistency, often aggregating results from a large number of studies, conducted under varying experimental conditions. While data from large-scale consortia are useful as they allow studying the effects of different biological conditions, they can also contain unwanted biases from confounding experimental factors. Here, we introduce Metadata-guided Feature Disentanglement (MFD) - an approach that allows disentangling biologically relevant features from potential technical biases. MFD incorporates target metadata into model training, by conditioning weights of the model output layer on different experimental factors. It then separates the factors into disjoint groups and enforces independence of the corresponding feature subspaces with an adversarially learned penalty. We show that the metadata-driven disentanglement approach allows for better model introspection, by connecting latent features to experimental factors, without compromising, or even improving performance in downstream tasks, such as enhancer prediction, or genetic variant discovery. The code for our implemementation is available at https://github.com/HealthML/MFD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32dd14e31f771ca226153c9210adaac58da256a4" target='_blank'>
              Metadata-guided Feature Disentanglement for Functional Genomics
              </a>
            </td>
          <td>
            Alexander Rakowski, Remo Monti, Viktoriia Huryn, Marta Lemanczyk, Uwe Ohler, Christoph Lippert
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Techniques to autonomously drive research have been prominent in Computational Scientific Discovery, while Synthetic Biology is a field of science that focuses on designing and constructing new biological systems for useful purposes. Here we seek to apply logic-based machine learning techniques to facilitate cellular engineering and drive biological discovery. Comprehensive databases of metabolic processes called genome-scale metabolic network models (GEMs) are often used to evaluate cellular engineering strategies to optimise target compound production. However, predicted host behaviours are not always correctly described by GEMs, often due to errors in the models. The task of learning the intricate genetic interactions within GEMs presents computational and empirical challenges. To address these, we describe a novel approach called Boolean Matrix Logic Programming (BMLP) by leveraging boolean matrices to evaluate large logic programs. We introduce a new system, $BMLP_{active}$, which efficiently explores the genomic hypothesis space by guiding informative experimentation through active learning. In contrast to sub-symbolic methods, $BMLP_{active}$ encodes a state-of-the-art GEM of a widely accepted bacterial host in an interpretable and logical representation using datalog logic programs. Notably, $BMLP_{active}$ can successfully learn the interaction between a gene pair with fewer training examples than random experimentation, overcoming the increase in experimental design space. $BMLP_{active}$ enables rapid optimisation of metabolic models to reliably engineer biological systems for producing useful compounds. It offers a realistic approach to creating a self-driving lab for microbial engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4217fa5bc20619984d78d30c100e2b19faa0091" target='_blank'>
              Boolean matrix logic programming for active learning of gene functions in genome-scale metabolic network models
              </a>
            </td>
          <td>
            L. Ai, Stephen Muggleton, Shishun Liang, Geoff S. Baldwin
          </td>
          <td>2024-05-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genotype-to-phenotype mapping is an essential problem in the current genomic era. While qualitative case-control predictions have received significant attention, less emphasis has been placed on predicting quantitative phenotypes. This emerging field holds great promise in revealing intricate connections between microbial communities and host health. However, the presence of heterogeneity in microbiome datasets poses a substantial challenge to the accuracy of predictions and undermines the reproducibility of models. To tackle this challenge, we investigated 22 normalization methods that aimed at removing heterogeneity across multiple datasets, conducted a comprehensive review of them, and evaluated their effectiveness in predicting quantitative phenotypes in three simulation scenarios and 31 real datasets. The results indicate that none of these methods demonstrate significant superiority in predicting quantitative phenotypes or attain a noteworthy reduction in Root Mean Squared Error (RMSE) of the predictions. Given the frequent occurrence of batch effects and the satisfactory performance of batch correction methods in predicting datasets affected by these effects, we strongly recommend utilizing batch correction methods as the initial step in predicting quantitative phenotypes. In summary, the performance of normalization methods in predicting metagenomic data remains a dynamic and ongoing research area. Our study contributes to this field by undertaking a comprehensive evaluation of diverse methods and offering valuable insights into their effectiveness in predicting quantitative phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a680651a3cf41ba9343ce9f9c1efc81ca6d5163" target='_blank'>
              Evaluation of normalization methods for predicting quantitative phenotypes in metagenomic data analysis
              </a>
            </td>
          <td>
            Beibei Wang, Yihui Luan
          </td>
          <td>2024-06-05</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Inferring causal protein signalling networks from human immune system cell data is a promising approach to unravel the underlying tissue signalling biology and dysfunction in diseased cells, which has attracted considerable attention within the bioinformatics field. Recently, Bayesian network (BN) techniques have gained significant popularity in inferring causal protein signalling networks from multiparameter single‐cell data. However, current BN methods may exhibit high computational complexity and ignore interactions among protein signalling molecules from different single cells. A novel BN method is presented for learning causal protein signalling networks based on parallel discrete artificial bee colony (PDABC), named PDABC. Specifically, PDABC is a score‐based BN method that utilises the parallel artificial bee colony to search for the global optimal causal protein signalling networks with the highest discrete K2 metric. The experimental results on several simulated datasets, as well as a previously published multi‐parameter fluorescence‐activated cell sorter dataset, indicate that PDABC surpasses the existing state‐of‐the‐art methods in terms of performance and computational efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45fc9744b37a980e5e05a5eeb12a6c928df4c0fc" target='_blank'>
              Inferring causal protein signalling networks from single‐cell data based on parallel discrete artificial bee colony algorithm
              </a>
            </td>
          <td>
            Jinduo Liu, Jihao Zhai, Junzhong Ji
          </td>
          <td>2024-05-11</td>
          <td>CAAI Transactions on Intelligence Technology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Our paper deals with a collection of networks on a common set of nodes, where some of the networks are associated with responses. Assuming that the networks correspond to points on a one-dimensional manifold in a higher dimensional ambient space, we propose an algorithm to consistently predict the response at an unlabeled network. Our model involves a specific multiple random network model, namely the common subspace independent edge model, where the networks share a common invariant subspace, and the heterogeneity amongst the networks is captured by a set of low dimensional matrices. Our algorithm estimates these low dimensional matrices that capture the heterogeneity of the networks, learns the underlying manifold by isomap, and consistently predicts the response at an unlabeled network. We provide theoretical justifications for the use of our algorithm, validated by numerical simulations. Finally, we demonstrate the use of our algorithm on larval Drosophila connectome data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb592d6fdf3ce2b11fe06b667a08556edadacce2" target='_blank'>
              Consistent response prediction for multilayer networks on unknown manifolds
              </a>
            </td>
          <td>
            Aranyak Acharyya, Jes'us Arroyo Reli'on, Michael Clayton, Marta Zlatic, Youngser Park, Carey E. Priebe
          </td>
          <td>2024-05-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In recent years, reinforcement learning (RL) has emerged as a valuable tool in drug design, offering the potential to propose and optimize molecules with desired properties. However, striking a balance between capabilities, flexibility, reliability, and efficiency remains challenging due to the complexity of advanced RL algorithms and the significant reliance on specialized code. In this work, we introduce ACEGEN, a comprehensive and streamlined toolkit tailored for generative drug design, built using TorchRL, a modern RL library that offers thoroughly tested reusable components. We validate ACEGEN by benchmarking against other published generative modeling algorithms and show comparable or improved performance. We also show examples of ACEGEN applied in multiple drug discovery case studies. ACEGEN is accessible at \url{https://github.com/acellera/acegen-open} and available for use under the MIT license.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/005ddb8c11113c508f104b731c8bae247ccf51e9" target='_blank'>
              ACEGEN: Reinforcement learning of generative chemical agents for drug discovery
              </a>
            </td>
          <td>
            Albert Bou, Morgan Thomas, Sebastian Dittert, Carles Navarro Ram'irez, Maciej Majewski, Ye Wang, Shivam Patel, Gary Tresadern, Mazen Ahmad, Vincent Moens, Woody Sherman, Simone Sciabola, G. D. Fabritiis
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Deep learning has been increasingly popular in omics data analysis. Recent works incorporating variable selection into deep learning have greatly enhanced the model’s interpretability. However, because deep learning desires a large sample size, the existing methods may result in uncertain findings when the dataset has a small sample size, commonly seen in omics data analysis. With the explosion and availability of omics data from multiple populations/studies, the existing methods naively pool them into one dataset to enhance the sample size while ignoring that variable structures can differ across datasets, which might lead to inaccurate variable selection results. We propose a penalized integrative deep neural network (PIN) to simultaneously select important variables from multiple datasets. PIN directly aggregates multiple datasets as input and considers both homogeneity and heterogeneity situations among multiple datasets in an integrative analysis framework. Results from extensive simulation studies and applications of PIN to gene expression datasets from elders with different cognitive statuses or ovarian cancer patients at different stages demonstrate that PIN outperforms existing methods with considerably improved performance among multiple datasets. The source code is freely available on Github (rucliyang/PINFunc). We speculate that the proposed PIN method will promote the identification of disease‐related important variables based on multiple studies/datasets from diverse origins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1667454b3841428b4c868841dbc397f6a3e10d6" target='_blank'>
              A penalized integrative deep neural network for variable selection among multiple omics datasets
              </a>
            </td>
          <td>
            Yang Li, Xiaonan Ren, Haochen Yu, Tao Sun, Shuangge Ma
          </td>
          <td>2024-06-07</td>
          <td>Quantitative Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129758877c0cd26445a33434e51b6c889c6028cd" target='_blank'>
              Exploration and verification a 13-gene diagnostic framework for ulcerative colitis across multiple platforms via machine learning algorithms
              </a>
            </td>
          <td>
            Jing Wang, Lin Li, Pingbo Chen, Chiyi He, Xiaoping Niu
          </td>
          <td>2024-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="3064 Background: In recent years, the use of tumor molecular profiling in clinical settings has enhanced cancer diagnostics, as well as the delivery of precision oncology. Recently, several methods for predicting gene expression directly from Haematoxylin-and-Eosin-stained (H&E) histology images have offered a new way to leverage the easily obtainable and cost-effective H&E images for multiple precision oncology applications. We previously introduced such a method – DeepPT – and demonstrated how we can leverage its imputed gene expression for successful prediction of drug response, through our ENLIGHT-DeepPT platform. In our previous publication and in an independent study comparing six different methods, DeepPT exhibited the best overall gene expression prediction accuracy. Methods: Here we present a new version, DeepPT 2.0, with improved architecture, including multi-task learning and a feature space based on a self-supervised pre-trained deep network. We tested both versions of DeepPT and the leading competing methods on patient data from 17 cancer subtypes included in TCGA. In addition, we obtained a new dataset collected at the Medical University of Bialystok (UMB) consisting of matched slide images and mRNA expression from 151 cases representing 7 cancer subtypes. These serve as an external validation to demonstrate that DeepPT generalizes well. Results: Our findings indicate that both versions of DeepPT show statistically significant improvement compared with the other methods in terms of median Pearson correlation of top predicted genes - the only metric available for all methods. Moreover, DeepPT 2.0 significantly improves upon version 1.0, demonstrating up to a 3-fold increase in the number of well-predicted genes (defined as genes with Pearson 𝜌 > 0.4 between actual and predicted mRNA expressions) for 14 of the 17 cancer subtypes tested. On the external validation data from UMB, DeepPT 2.0 improves gene expression prediction in 6 of 7 of the tested cancer subtypes, with up to a 7-fold increase in the number of well-predicted genes, thus mitigating the concern of overfitting on the training set. Immune genes are particularly well predicted, as we previously observed: using a set of 826 hallmark immune genes, DeepPT 2.0 exhibits up to 3.5-fold and 2-fold increase in the percentage of well-predicted genes in the TCGA and UMB data, respectively, compared to all other genes. Conclusions: DeepPT 2.0 significantly improves upon competing methods for predicting mRNA expression from H&E slides across multiple metrics and diverse cancer types. Furthermore, it demonstrates robust generalization to slides from sources not previously seen. The method’s good ability to predict the expression of immune genes suggests a potential benefit in predicting response to immunotherapy using the ENLIGHT-DeepPT platform, as indeed demonstrated elsewhere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c75aa806813e51a7f41b199e9391b3e01188e73" target='_blank'>
              Development, comparative study, and external validation of a new deep learning model for predicting genome-wide gene expression from histopathology slides.
              </a>
            </td>
          <td>
            Leon Gugel, O. Tirosh, Yaron Kinar, Anna Michalska-Falkowska, Joanna Reszeć-Giełażyn, Witold Bauer, Magdalena Niemira, Boris Temkin, T. Beker, R. Aharonov, Gal Dinstag
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
Fusing known physics into data-driven learners allows modelling practitioners to combine the expressive power of traditional machine learning with known mechanistic laws, where the objective is to enhance predictive performance, interpretability, and model generalisation. A core consideration that must be made when implementing a physics-informed learning architecture is how relevant knowledge will be embedded into the model structure, which, generally, is informed by the type of physics that is available. Frequently this knowledge may not be complete, with only a partial understanding of the governing physics available. 

In this work, possible paths for deriving Gaussian process kernels that are representative of partial knowledge will be considered. How the type of knowledge that is possessed influences the derivation will be explored, particularly when there is the potential for some aspect of misspecified physics. An example of deriving partially structured kernels will be investigated for modelling the decoupled response of a GARTEUR laboratory aircraft structure, where the derived kernels are used to decompose the dynamics of the aircraft into modal contributions. 

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da9bbf07470b8cc6d6606c4b51a040b8e45ce3d6" target='_blank'>
              Gaussian process kernels for partial physical insight
              </a>
            </td>
          <td>
            Matthew R. Jones, D. J. Pitchforth, E. J. Cross
          </td>
          <td>2024-07-01</td>
          <td>e-Journal of Nondestructive Testing</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: Synthetic essentiality represents a promising therapeutic approach by identifying genes that are necessary for the proliferation and survival of tumors harboring hard-to-target gene alterations. Understanding and accurately predicting synthetic essential genes, through genetic dependencies, may reveal therapeutically effective drug targets in a specific molecular context. Deep learning, as exemplified by our published DeepDEP model, has the potential to capture intricate multi-omic profiles for such prediction tasks. However, the validity of such tools in specific biological contexts remains to be fully examined and presents a major obstacle to adoption by researchers. Materials and Methods: To address this gap, we conducted a case study in which we screened for synthetic essential genes for one of the most frequently mutated and yet undruggable genes, CTNNB1, in hepatocellular carcinoma (HCC). Specifically, we predicted the genetic dependencies of each HCC patient in The Cancer Genome Atlas (TCGA; n=346) by DeepDEP and identified potential dependencies that were intensified with the presence of CTNNB1 mutations. The top 10 genes, ranked by p-value of differential gene-effect scores for CTNNB1-mutated (n=92) versus CTNNB1-WT HCC (n=254), were reviewed in the literature to validate their essentiality in CTNNB1-mutated HCC as well as their potential for pharmacologic inhibition. Survival analysis was performed using published data from the IMBrave150 trial to validate one of the findings. Results: Experimental evidence in the literature supported the essentiality of many of the top 10 predicted genes for CTNNB1-mutated HCC, including one gene with mechanistic evidence of being a transcriptional co-activator of β-catenin target genes. Furthermore, several of these genes have known pharmacologic inhibitors which are either natural compounds or FDA-approved drugs. One example was PDGFB, which encodes a ligand activating the PDGF signaling pathway. PDGF signaling is targeted by sorafenib, an FDA-approved first line drug for HCC. Survival analysis of the sorafenib-treated arm of the IMBrave150 trial showed that patients with mutated CTNNB1 had improved progression-free survival compared to those with wild-type CTNNB1 (p = 0.044). Conclusions: Our study illustrates a potential application of deep learning to identify synthetic essential genes, including genes with readily available pharmacologic inhibitors, for targeting challenging gene alterations. Remarkably, our tool demonstrates the ability to predict cancer dependencies with molecular subtype specificity, suggesting a potential for in silico screening of gene dependencies to facilitate drug discovery and personalized medicine approaches. Our current efforts are focused on optimizing this computational pipeline and making it publicly available for cancer researchers.
 Citation Format: Tyler M. Yasaka, Michael Kasper, Li-Ju Wang, Michael Ning, Yufei Huang, Satdarshan P Monga, Yu-Chiao Chiu. Deep learning-based prediction of synthetic essentialities in CTNNB1-mutated hepatocellular carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B013.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3559381603c0ae065121569bbed37a8a776050a2" target='_blank'>
              Abstract B013: Deep learning-based prediction of synthetic essentialities in CTNNB1-mutated hepatocellular carcinoma
              </a>
            </td>
          <td>
            Tyler M. Yasaka, Michael J. Kasper, Li-Ju Wang, Michael Ning, Yufei Huang, S. Monga, Yu-Chiao Chiu
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="In dynamical systems reconstruction (DSR) we seek to infer from time series measurements a generative model of the underlying dynamical process. This is a prime objective in any scientific discipline, where we are particularly interested in parsimonious models with a low parameter load. A common strategy here is parameter pruning, removing all parameters with small weights. However, here we find this strategy does not work for DSR, where even low magnitude parameters can contribute considerably to the system dynamics. On the other hand, it is well known that many natural systems which generate complex dynamics, like the brain or ecological networks, have a sparse topology with comparatively few links. Inspired by this, we show that geometric pruning, where in contrast to magnitude-based pruning weights with a low contribution to an attractor's geometrical structure are removed, indeed manages to reduce parameter load substantially without significantly hampering DSR quality. We further find that the networks resulting from geometric pruning have a specific type of topology, and that this topology, and not the magnitude of weights, is what is most crucial to performance. We provide an algorithm that automatically generates such topologies which can be used as priors for generative modeling of dynamical systems by RNNs, and compare it to other well studied topologies like small-world or scale-free networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7262300a7dc61b5b9020753dc9d2bbd63a06cc2" target='_blank'>
              Optimal Recurrent Network Topologies for Dynamical Systems Reconstruction
              </a>
            </td>
          <td>
            Christoph Jurgen Hemmer, Manuel Brenner, Florian Hess, Daniel Durstewitz
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Computational pathology, utilizing whole slide image (WSI) for pathological diagnosis, has advanced the development of intelligent healthcare. However, the scarcity of annotated data and histological differences hinder the general application of existing methods. Extensive histopathological data and the robustness of self-supervised models in small-scale data demonstrate promising prospects for developing foundation pathology models. In this work, we propose the BEPH (BEiT-based model Pre-training on Histopathological image), a general method that leverages self-supervised learning to learn meaningful representations from 11 million unlabeled histopathological images. These representations are then efficiently adapted to various tasks, including patch-level cancer recognition, WSI-level cancer classification, and survival prediction for multiple cancer subtypes. Experimental results demonstrate that our model consistently outperforms several comparative models, even with limited training data reduced to 50%. Especially when the downstream structure is the same, the model can improve ResNet and DINO by up to a maximum increase of 8.8% and 7.2% (WSI level classification), and 6.44% and 3.28% on average (survival prediction), respectively. Therefore, BEPH offers a universal solution to enhance model performance, reduce the burden of expert annotations, and enable widespread clinical applications of artificial intelligence. The code and models can be obtained at https://github.com/Zhcyoung/BEPH. And currently, online fine-tuning of WSI classification tasks is available for use on http://yulab-sjtu.natapp1.cc/BEPH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6572398e4b60a8f1eaab9108d27cf5629267e013" target='_blank'>
              A foundation model for generalizable cancer diagnosis and survival prediction from histopathological images
              </a>
            </td>
          <td>
            Zhaochang Yang, Ting Wei, Ying Liang, Xin Yuan, Ruitian Gao, Yujia Xia, Jie Zhou, Yue Zhang, Zhangsheng Yu
          </td>
          <td>2024-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recurrent neural networks (RNNs) are a widely used tool for sequential data analysis, however, they are still often seen as black boxes of computation. Understanding the functional principles of these networks is critical to developing ideal model architectures and optimization strategies. Previous studies typically only emphasize the network representation post-training, overlooking their evolution process throughout training. Here, we present Multiway Multislice PHATE (MM-PHATE), a novel method for visualizing the evolution of RNNs' hidden states. MM-PHATE is a graph-based embedding using structured kernels across the multiple dimensions spanned by RNNs: time, training epoch, and units. We demonstrate on various datasets that MM-PHATE uniquely preserves hidden representation community structure among units and identifies information processing and compression phases during training. The embedding allows users to look under the hood of RNNs across training and provides an intuitive and comprehensive strategy to understanding the network's internal dynamics and draw conclusions, e.g., on why and how one model outperforms another or how a specific architecture might impact an RNN's learning ability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c93f4f438902f06a5f77105696c09ab5c831cf6f" target='_blank'>
              Multiway Multislice PHATE: Visualizing Hidden Dynamics of RNNs through Training
              </a>
            </td>
          <td>
            Jiancheng Xie, Lou C. Kohler Voinov, Noga Mudrik, Gal Mishne, Adam S. Charles
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Radiobiological data, whether obtained at the clinical, biological or molecular level has significantly contributed to a better description and prediction of the individual dose-response to ionizing radiation and a better estimation of the radiation-induced risks. Particularly, over the last seventy years, the amount of radiobiological data has considerably increased, and permitted the mathematical formulas describing dose-response to become less empirical. A better understanding of the basic radiobiological mechanisms has also contributed to establish quantitative inter-correlations between clinical, biological and molecular biomarkers, refining again the mathematical models of description. Today, big data approaches and, more recently, artificial intelligence may finally complete and secure this long process of thinking from the multi-scale description of radiation-induced events to their prediction. Here, we reviewed the major dose-response models applied in radiobiology for quantifying molecular and cellular radiosensitivity and aimed to explain their evolution: Specifically, we highlighted the advances concerning the target theory with the cell survival models and the progressive introduction of the DNA repair process in the mathematical models. Furthermore, we described how the technological advances have changed the description of DNA double-strand break (DSB) repair kinetics by introducing the important notion of DSB recognition, independent of that of DSB repair. Initially developed separately, target theory on one hand and, DSB recognition and repair, on the other hand may be now fused into a unified model involving the cascade of phosphorylations mediated by the ATM kinase in response to any genotoxic stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cdbdf7288e7e261d821e90454903709c5fbfc0e" target='_blank'>
              Seventy Years of Dose-response Models: From the Target Theory to the Use of Big Databases Involving Cell Survival and DNA Repair.
              </a>
            </td>
          <td>
            L. Bodgi, Laurent Pujo-Menjouet, Audrey Bouchet, Michel Bourguignon, N. Foray
          </td>
          <td>2024-05-28</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The growth of bacterial gene expression datasets has offered unprecedented coverage of achievable transcriptomes, reflecting diverse activity states of the transcription regulatory network. Machine learning methods like Independent Component Analysis (ICA) can decompose gene expression datasets into regulatory modules and condition-specific regulator activities. Here, we present a workflow to utilize inferred regulator activities to construct quantitative models of promoter regulation in E. coli. Resulting models are validated by predicting condition-specific TF effector concentrations and binding site motif strength based on differential gene expression data alone. We show how reconstructed promoter models can capture multi-scale regulation and disentangle regulator interactions, including resolving the apparent paradox where argR expression is positively correlated with its regulon despite being a repressor. We applied the workflow for all regulator-linked components extracted by ICA, demonstrating the scalability of the workflow to capture the E. coli TRN. This work suggests a path toward systematic, quantitative reconstruction of transcription regulatory networks driven by the large-scale databases that are now available for many organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54beacc49acede7e1ab1316132bfe8fb330701f" target='_blank'>
              Data-driven modeling of E. coli transcriptional regulation
              </a>
            </td>
          <td>
            Christopher Dalldorf, Griffith Hughes, Gaoyuan Li, Bernhard O. Palsson, Daniel C. Zielinski
          </td>
          <td>2024-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Chaos presents complex dynamics arising from nonlinearity and a sensitivity to initial states. These characteristics suggest a depth of expressivity that underscores their potential for advanced computational applications. However, strategies to effectively exploit chaotic dynamics for information processing have largely remained elusive. In this study, we reveal that the essence of chaos can be found in various state-of-the-art deep neural networks. Drawing inspiration from this revelation, we propose a novel method that directly leverages chaotic dynamics for deep learning architectures. Our approach is systematically evaluated across distinct chaotic systems. In all instances, our framework presents superior results to conventional deep neural networks in terms of accuracy, convergence speed, and efficiency. Furthermore, we found an active role of transient chaos formation in our scheme. Collectively, this study offers a new path for the integration of chaos, which has long been overlooked in information processing, and provides insights into the prospective fusion of chaotic dynamics within the domains of machine learning and neuromorphic computation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df45f19a69c7d8fe529ae6dce6555e7faac04eb9" target='_blank'>
              Exploiting Chaotic Dynamics as Deep Neural Networks
              </a>
            </td>
          <td>
            Shuhong Liu, Nozomi Akashi, Qingyao Huang, Yasuo Kuniyoshi, Kohei Nakajima
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Uncertainty in parameter estimates from fitting within-host models to empirical data limits the model’s ability to uncover mechanisms of infection, disease progression, and to guide pharmaceutical interventions. Understanding the effect of model structure and data availability on model predictions is important for informing model development and experimental design. To address sources of uncertainty in parameter estimation, we use four mathematical models of influenza A infection with increased degrees of biological realism. We test the ability of each model to reveal its parameters in the presence of unlimited data by performing structural identifiability analyses. We then refine the results by predicting practical identifiability of parameters under daily influenza A virus titers alone or together with daily adaptive immune cell data. Using these approaches, we present insight into the sources of uncertainty in parameter estimation and provide guidelines for the types of model assumptions, optimal experimental design, and biological information needed for improved predictions. Author summary Within-host models of virus infections fitted to data have improved our understanding of mechanisms of infection, disease progression, and allowed us to provide guidelines for pharmaceutical interventions. Given their predictive power, it is essential that we properly uncover and address uncertainty in model predictions and parameter estimation. Here, we focus on the effect of model structure and data availability on our ability to uncover unknown parameters. To address these questions, we use four mathematical models of influenza A infection with increased degrees of biological realism. We test the ability of each model to reveal its parameters in the presence of unlimited data by performing structural identifiability analysis. We then refine the results by predicting practical identifiability of parameters under daily influenza A virus titers alone or together with daily adaptive immune cell data. Using these approaches, we present insights into the sources of uncertainty in parameter estimations and provide guidelines for the types of model assumptions, optimal experimental design, and biological information needed for improved predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08312e1a270ca4ad6a36e75e2b5550f77112dd89" target='_blank'>
              Identifiability investigation of within-host models of acute virus infection
              </a>
            </td>
          <td>
            Yuganthi R. Liyanage, Nora Heitzman-Breen, N. Tuncer, Stanca M. Ciupe
          </td>
          <td>2024-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Purpose: To evaluate the efficacy of prominent machine learning algorithms in predicting normal tissue complication probability utilizing clinical data obtained from two distinct disease sites, and to create a software tool that facilitates the automatic determination of the optimal algorithm to model any given labeled dataset. Methods and Materials: We obtained 3 sets of radiation toxicity data (478 patients) from our clinic, gastrointestinal toxicity (GIT), radiation pneumonitis (RP), and radiation esophagitis (RE). These data comprised clinicopathological and dosimetric information for patients diagnosed with non-small cell lung cancer and anal squamous cell carcinoma. Each dataset was modeled using ten commonly employed machine learning algorithms (elastic net, LASSO, random forest, regression forest, support vector machine, XGBoost, k-nearest-neighbors, neural network, Bayesian-LASSO, and Bayesian neural network) by randomly dividing the dataset into a training and test set. The training set was used to create and tune the model, and the test set served to assess it by calculating performance metrics. This process was repeated 100 times by each algorithm for each dataset. Figures were generated to visually compare the performance of the algorithms. A graphical user interface was developed to automate this whole process. Results: LASSO achieved the highest area under the precision-recall curve (AUPRC) (0.807{+/-}0.067) for RE, random forest for GIT (0.726{+/-}0.096), and the neural network for RP (0.878{+/-}0.060). Area-under-the-curve was 0.754{+/-}0.069, 0.889{+/-}0.043, and 0.905{+/-}0.045, respectively. The graphical user interface was used to compare all algorithms for each dataset automatically. When averaging AUPRC across all toxicities, Bayesian-LASSO was the best model. Conclusion: Our results show that there is no best algorithm for all datasets. Therefore, it is important to compare multiple algorithms when training an outcome prediction model on a new dataset. The graphical user interface created for this study automatically compares the performance of these ten algorithms for any dataset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5e39f5ff1a961de94f4b53bfa977bf8735ef11" target='_blank'>
              Performance comparison of ten state-of-the-art machine learning algorithms for outcome prediction modeling of radiation-induced toxicity
              </a>
            </td>
          <td>
            PhD Ramon M. Salazar, MSc Saurabh S. Nair, Mbs Alexandra O. Leone, PhD Ting Xu, BS Raymond P. Mumme, BA Jack D. Duryea, MD Brian De, MD Kelsey L. Corrigan, MD Michael K. Rooney, MD Matthew S. Ning, MD Prajnan Das, MD Emma B. Holliday, MD Zhongxing Liao, PhD Laurence E. Court, PhD Joshua S. Niedzielski
          </td>
          <td>2024-05-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Modeling dynamical systems, e.g. in climate and engineering sciences, often necessitates solving partial differential equations. Neural operators are deep neural networks designed to learn nontrivial solution operators of such differential equations from data. As for all statistical models, the predictions of these models are imperfect and exhibit errors. Such errors are particularly difficult to spot in the complex nonlinear behaviour of dynamical systems. We introduce a new framework for approximate Bayesian uncertainty quantification in neural operators using function-valued Gaussian processes. Our approach can be interpreted as a probabilistic analogue of the concept of currying from functional programming and provides a practical yet theoretically sound way to apply the linearized Laplace approximation to neural operators. In a case study on Fourier neural operators, we show that, even for a discretized input, our method yields a Gaussian closure--a structured Gaussian process posterior capturing the uncertainty in the output function of the neural operator, which can be evaluated at an arbitrary set of points. The method adds minimal prediction overhead, can be applied post-hoc without retraining the neural operator, and scales to large models and datasets. We showcase the efficacy of our approach through applications to different types of partial differential equations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb48eb416a7b362eb444cb5d9c111a6939db5fe" target='_blank'>
              Linearization Turns Neural Operators into Function-Valued Gaussian Processes
              </a>
            </td>
          <td>
            Emilia Magnani, Marvin Pfortner, Tobias Weber, Philipp Hennig
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Allostery is a fundamental mechanism driving biomolecular processes that holds significant therapeutic concern. Our study rigorously investigates how two distinct machine-learning algorithms uniquely classify two already close-to-active DFG-in states of TAK1, differing just by the presence or absence of its allosteric activator TAB1, from an ensemble mixture of conformations (obtained from 2.4 μs molecular dynamics (MD) simulations). The novelty, however, lies in understanding the deeper algorithmic potentials to systematically derive a diverse set of differential residue connectivity features that reconstruct the essential mechanistic architecture for TAK1-TAB1 allostery in such a close-to-active biochemical scenario. While the recursive, random forest-based workflow displays the potential of conducting discretized, hierarchical derivation of allosteric features, a multilayer perceptron-based approach gains considerable efficacy in revealing fluid connected patterns of features when hybridized with mutual information scoring. Interestingly, both pipelines benchmark similar directions of functional conformational changes for TAK1's activation. The findings significantly advance the depth of mechanistic understanding by highlighting crucial activation signatures along a directed C-lobe → activation loop → ATP pocket channel of information flow, including (1) the αF-αE biterminal alignments and (2) the "catalytic" drift of the activation loop toward kinase active site. Besides, some novel allosteric hotspots (K253, Y206, N189, etc.) are further recognized as TAB1 sensors, transducers, and responders, including a benchmark E70 mutation site, precisely mapping the important structural segments for sequential allosteric execution. Hence, our work demonstrates how to navigate through greater structural depths and dimensions of dynamic allosteric machineries just by leveraging standard ML methods in suitable streamlined workflows adaptive to the specific system and objectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e656737754f86ab9ba75a485691b12e7213422c" target='_blank'>
              Adaptive Workflows of Machine Learning Illuminate the Sequential Operation Mechanism of the TAK1's Allosteric Network.
              </a>
            </td>
          <td>
            Nibedita Ray Chaudhuri, Shubhra Ghosh Dastidar
          </td>
          <td>2024-05-14</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f007b787d6d9ef2d12cb5a178877181131370a1f" target='_blank'>
              Nonlinear compartmental modeling to monitor ovarian follicle population dynamics on the whole lifespan.
              </a>
            </td>
          <td>
            Guillaume Ballif, Frédérique Clément, Romain Yvinec
          </td>
          <td>2024-06-06</td>
          <td>Journal of mathematical biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Drug allergy is responsible for a huge burden on public healthcare systems, representing in some instances a threat for patient's life. Diagnosis is complex due to the heterogeneity of clinical phenotypes and mechanisms involved, the limitations of in vitro tests, and the associated risk to in vivo tests. Predictive models, including those using recent advances in artificial intelligence, may circumvent these drawbacks, leading to an appropriate classification of patients and improving their management in clinical settings.


RECENT FINDINGS
Scores and predictive models to assess drug allergy development, including patient risk stratification, are scarce and usually apply logistic regression analysis. Over recent years, different methods encompassed under the general umbrella of artificial intelligence, including machine and deep learning, and artificial neural networks, are emerging as powerful tools to provide reliable and optimal models for clinical diagnosis, prediction, and precision medicine in different types of drug allergy.


SUMMARY
This review provides general concepts and current evidence supporting the potential utility of predictive models and artificial intelligence branches in drug allergy diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8035591f56be14f579fd4dad147b5d984378cff" target='_blank'>
              Predictive models and applicability of artificial intelligence-based approaches in drug allergy.
              </a>
            </td>
          <td>
            Rafael Núñez, Inmaculada Doña, J. Cornejo‐García
          </td>
          <td>2024-05-23</td>
          <td>Current opinion in allergy and clinical immunology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Biologically detailed models of brain circuitry are challenging to build and simulate due to the large number of neurons, their complex interactions, and the many unknown physiological parameters. Simplified mathematical models are more tractable, but harder to evaluate when too far removed from neuroanatomy/physiology. We propose that a multiscale model, coarse-grained (CG) while preserving local biological details, offers the best balance between biological realism and computability. This paper presents such a model. Generally, CG models focus on the interaction between groups of neurons-here termed "pixels"-rather than individual cells. In our case, dynamics are alternately updated at intra- and interpixel scales, with one informing the other, until convergence to equilibrium is achieved on both scales. An innovation is how we exploit the underlying biology: Taking advantage of the similarity in local anatomical structures across large regions of the cortex, we model intrapixel dynamics as a single dynamical system driven by "external" inputs. These inputs vary with events external to the pixel, but their ranges can be estimated a priori. Precomputing and tabulating all potential local responses speed up the updating procedure significantly compared to direct multiscale simulation. We illustrate our methodology using a model of the primate visual cortex. Except for local neuron-to-neuron variability (necessarily lost in any CG approximation) our model reproduces various features of large-scale network models at a tiny fraction of the computational cost. These include neuronal responses as a consequence of their orientation selectivity, a primary function of visual neurons.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e473f5dcdb151d8bc6c17393e9489ea4f23400e" target='_blank'>
              Efficient models of cortical activity via local dynamic equilibria and coarse-grained interactions.
              </a>
            </td>
          <td>
            Zhuo-Cheng Xiao, Kevin K Lin, Lai-Sang Young
          </td>
          <td>2024-06-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Kolmogorov-Arnold Networks (KAN) is a groundbreaking model recently proposed by the MIT team, representing a revolutionary approach with the potential to be a game-changer in the field. This innovative concept has rapidly garnered worldwide interest within the AI community. Inspired by the Kolmogorov-Arnold representation theorem, KAN utilizes spline-parametrized univariate functions in place of traditional linear weights, enabling them to dynamically learn activation patterns and significantly enhancing interpretability. In this paper, we explore the application of KAN to time series forecasting and propose two variants: T-KAN and MT-KAN. T-KAN is designed to detect concept drift within time series and can explain the nonlinear relationships between predictions and previous time steps through symbolic regression, making it highly interpretable in dynamically changing environments. MT-KAN, on the other hand, improves predictive performance by effectively uncovering and leveraging the complex relationships among variables in multivariate time series. Experiments validate the effectiveness of these approaches, demonstrating that T-KAN and MT-KAN significantly outperform traditional methods in time series forecasting tasks, not only enhancing predictive accuracy but also improving model interpretability. This research opens new avenues for adaptive forecasting models, highlighting the potential of KAN as a powerful and interpretable tool in predictive analytics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10145b2238569436754c4d9be3f9c7db501cc65c" target='_blank'>
              Kolmogorov-Arnold Networks for Time Series: Bridging Predictive Power and Interpretability
              </a>
            </td>
          <td>
            Kunpeng Xu, Lifei Chen, Shengrui Wang
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>6</td>
          <td>4</td>
        </tr>

        <tr id="Integrable partial differential equation (PDE) systems are of great interest in natural science, but are exceedingly rare and difficult to discover. To solve this, we introduce OptPDE, a first-of-its-kind machine learning approach that Optimizes PDEs' coefficients to maximize their number of conserved quantities, $n_{\rm CQ}$, and thus discover new integrable systems. We discover four families of integrable PDEs, one of which was previously known, and three of which have at least one conserved quantity but are new to the literature to the best of our knowledge. We investigate more deeply the properties of one of these novel PDE families, $u_t = (u_x+a^2u_{xxx})^3$. Our paper offers a promising schema of AI-human collaboration for integrable system discovery: machine learning generates interpretable hypotheses for possible integrable systems, which human scientists can verify and analyze, to truly close the discovery loop.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6cc89b3ade1d5ca2f3035e1d7a3f55890ce38c3" target='_blank'>
              OptPDE: Discovering Novel Integrable Systems via AI-Human Collaboration
              </a>
            </td>
          <td>
            Subhash Kantamneni, Ziming Liu, Max Tegmark
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Single-cell datasets often lack individual cell labels, making it challenging to identify cells associated with disease. To address this, we introduce Mixture Modeling for Multiple Instance Learning (MMIL), an expectation maximization method that enables the training and calibration of cell-level classifiers using patient-level labels. Our approach can be used to train e.g. lasso logistic regression models, gradient boosted trees, and neural networks. When applied to clinically-annotated, primary patient samples in Acute Myeloid Leukemia (AML) and Acute Lymphoblastic Leukemia (ALL), our method accurately identifies cancer cells, generalizes across tissues and treatment timepoints, and selects biologically relevant features. In addition, MMIL is capable of incorporating cell labels into model training when they are known, providing a powerful framework for leveraging both labeled and unlabeled data simultaneously. Mixture Modeling for MIL offers a novel approach for cell classification, with significant potential to advance disease understanding and management, especially in scenarios with unknown gold-standard labels and high dimensionality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf614cea92ffef23dcd05fbea55a21a48d81f7c" target='_blank'>
              MMIL: A novel algorithm for disease associated cell type discovery.
              </a>
            </td>
          <td>
            Erin Craig, Timothy Keyes, J. Sarno, Maxim E. Zaslavsky, Garry Nolan, Kara Davis, Trevor Hastie, R. Tibshirani
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>153</td>
        </tr>

        <tr id="Machine learning (ML) is changing the world of computational protein design, with data- driven methods surpassing biophysical-based methods in experimental success rates. However, they are most often reported as case studies, lack integration and standardization across platforms, and are therefore hard to objectively compare. In this study, we established a streamlined and diverse toolbox for methods that predict amino acid probabilities inside the Rosetta software framework that allows for the side-by-side comparison of these models. Subsequently, existing protein fitness landscapes were used to benchmark novel self- supervised machine learning methods in realistic protein design settings. We focused on the traditional problems of protein sequence design: sampling and scoring. A major finding of our study is that novel ML approaches are better at purging the sampling space from deleterious mutations. Nevertheless, scoring resulting mutations without model fine-tuning showed no clear improvement over scoring with Rosetta. This study fills an important gap in the field and allows for the first time a comprehensive head-to-head comparison of different ML and biophysical methods. We conclude that ML currently acts as a complement to, rather than a replacement for, biophysical methods in protein design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d74eec4ceff71302ee342c63682798408eacb28" target='_blank'>
              Self-supervised machine learning methods for protein design improve sampling, but not the identification of high-fitness variants
              </a>
            </td>
          <td>
            Moritz Ertelt, Rocco Moretti, Jens Meiler, Clara T. Schoeder
          </td>
          <td>2024-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Introduction & Objective: High-risk single-nucleotide polymorphisms (SNPs) account for only ~20% of type 2 diabetes heritability, implying islet genes greatly influence one another to alter diabetes risk. Correlative analysis does not reveal conditional dependencies between key players regulating islet function whereas machine learning (ML) may do so.
 Methods: Using data from 374 genetically diverse mice, we derived models predicting islet function from protein abundance with gradient-boosted decision tree algorithms. Some mice (70%) were used to train the models and the rest were used to validate them. We also applied our models to similar data from other mouse studies and to data from humans to see if the models’ accuracies replicated. We then determined if the models included any known islet regulators, proteins with human orthologues possessing glycemia-related SNPs, and enriched for islet-relevant functional pathways. Finally, using the proteins’ predictive influences as meta-traits, we ran quantitative trait locus (QTL) scans to identify genomic regions altering the influence of proteins on islet function.
 Results: Our analysis revealed < 150 of the ~ 5000 detected proteins sufficiently modeled any one functional trait, with > 90% of a model’s prediction influenced by < 50 proteins. Some models predicted as well or better in the other mouse sets (R > 0.6) and reasonably well in human data (R > 0.5). ML analysis correctly identified the direction of effect for candidates (e.g. DPP8) where prior studies using correlation alone failed. Models for islet traits enriched for pathways potentially relevant for islet function and several unstudied proteins (e.g. CRELD2) present in multiple models have SNPs for glycemia-related traits. Finally, QTL scans revealed genomic regions that may alter the influence of key proteins on islet function and are far from those proteins’ genomic positions.
 Conclusions: ML can be used to identify conditional dependencies regulating islet function.


 C. Emfinger: None. E. Freiberger: Employee; AbbVie Inc. M. Rabaglia: None. J. Kolic: None. S. Simonett: Employee; Exact Sciences. M. Shortreed: None. L. Clark: None. D. Stapleton: None. K. Schueler: None. K. Mitok: None. T.R. Price: None. J.J. Coon: Research Support; Agilent, Eli Lilly and Company, Merck & Co., Inc. L. Smith: None. M.J. Merrins: None. J.D. Johnson: None. M. Keller: None. A.D. Attie: None.



 American Diabetes Association (7-21-PDF-157); National Institutes of Health (R01-DK101573-06); National Institutes of Health (GM070683); National Institutes of Health (P41-GM108538); National Institutes of Health (R35GM126914); National Institutes of Health (R35-GM118110); National Institutes of Health (T32-HL007936); National Institutes of Health (R01-DK113103); National Institutes of Health (R01DK127637); Canadian Institutes for Health Research (CIHR) operating grant 168857 CIHR Team Grant (ASD-179092/5-SRA-2021-1149-S-B); CIHR-JDRF Team (ASD-173663/5-SRA-2020-1059-S-B); CIHR Banting fellowship United States Department of Veterans Affairs Biomedical Laboratory Research and Development Service (I01BX005113)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae9b0cfbbd675841b7923b5ba1ab6fcfd03c60f2" target='_blank'>
              1483-P: Identifying Conditional Dependence of Proteins Regulating Islet Biology with Machine Learning
              </a>
            </td>
          <td>
            Christopher H Emfinger, Elyse C. Freiberger, Mary R. Rabaglia, Jelena Kolic, Shane P. Simonett, Michael Shortreed, Lauren E. Clark, Donnie Stapleton, Kathryn Schueler, Kelly A. Mitok, Tara R. Price, Joshua J. Coon, Lloyd Smith, M. Merrins, James D. Johnson, Mark Keller, A. Attie
          </td>
          <td>2024-06-14</td>
          <td>Diabetes</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In vitro cell biology experiments are routinely used to characterize cell migration properties under various experimental conditions. These experiments can be interpreted using lattice-based random walk models to provide insight into underlying biological mechanisms, and continuum limit partial differential equation (PDE) descriptions of the stochastic models can be used to efficiently explore model properties instead of relying on repeated stochastic simulations. Working with efficient PDE models is of high interest for parameter estimation algorithms that typically require a large number of forward model simulations. Quantitative data from cell biology experiments usually involves non-negative cell counts in different regions of the experimental images, and it is not obvious how to relate finite, noisy count data to the solutions of continuous PDE models that correspond to noise-free density profiles. In this work we illustrate how to develop and implement likelihood-based methods for parameter estimation, parameter identifiability and model prediction for lattice-based models describing collective migration with an arbitrary number of interacting subpopulations. We implement a standard additive Gaussian measurement error model as well as a new physically-motivated multinomial measurement error model that relates noisy count data with the solution of continuous PDE models. Both measurement error models lead to similar outcomes for parameter estimation and parameter identifiability, whereas the standard additive Gaussian measurement error model leads to non-physical prediction outcomes. In contrast, the new multinomial measurement error model involves a lower computational overhead for parameter estimation and identifiability analysis, as well as leading to physically meaningful model predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7501b082e09fd3e991e69561d882bcf1981f2a48" target='_blank'>
              Likelihood-based inference, identifiability and prediction using count data from lattice-based random walk models
              </a>
            </td>
          <td>
            Yihan Liu, D. Warne, Matthew J. Simpson
          </td>
          <td>2024-06-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/480567bb55e580af5f22e324481c693a5881a9b8" target='_blank'>
              Unveiling gene regulatory networks during cellular state transitions without linkage across time points
              </a>
            </td>
          <td>
            Ruosi Wan, Yuhao Zhang, Yongli Peng, Feng Tian, Ge Gao, Fuchou Tang, Jinzhu Jia, Hao Ge
          </td>
          <td>2024-05-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
Machine learning (ML) has been harnessed as a promising modelling tool for materials research. However, small data, or data scarcity, is a bottleneck when incorporating ML in studies involving experimentation. Current experiment planning methods show several disadvantages: one‐factor‐at‐a‐time (OFAT) experimentation became impractical due to limited laboratory resources; conventional design of experiments (DoE) failed to incorporate high‐dimensional features in ML; Surrogate‐based or Bayesian optimization (BO) shifted the goal to optimize material properties rather than guiding training data accumulation. The present research proposes leveraging active learning (AL) to strategically select critical data for experimentation. Two AL strategies, query‐by‐Committee (QBC) algorithm and Greedy method, are benchmarked against random query baseline on various materials datasets. AL is shown to efficiently reduce model prediction errors with minimal additional experiment data. Investigation of hyperparameters revealed benefits of applying AL at an early stage of experimental dataset construction. Moreover, AL is implemented and validated for an in‐house materials development task ‐ electrospray modelling. AL exploration as a paradigm is highlighted to guide experiment design for efficient data accumulation purposes, and its potential for further ML modelling. In doing so, the power of ML is expected to be fully unleashed to experimental researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41751ad3a09025a836bdcc2a49f5c274ea5e9855" target='_blank'>
              Tackling Data Scarcity Challenge through Active Learning in Materials Processing with Electrospray
              </a>
            </td>
          <td>
            Fanjin Wang, A. Harker, Mohan Edirisinghe, M. Parhizkar
          </td>
          <td>2024-05-23</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Modelling gene expression is a central problem in systems biology. Single-cell technologies have revolutionised the field by enabling sequencing at the resolution of individual cells. This results in a much richer data compared to what is obtained by bulk technologies, offering new possibilities and challenges for gene regulatory network inference. Here we introduce GRIT — a method to fit a differential equation model and to infer gene regulatory networks from single-cell data using the theory of optimal transport. The idea consists in tracking the evolution of the cell distribution over time and finding the system that minimises the transport cost between consecutive time points.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0edd4d732cd966d7162a3543e7c8bfa54031593f" target='_blank'>
              Gene regulatory network inference from single-cell data using optimal transport
              </a>
            </td>
          <td>
            François Lamoline, Isabel Haasler, Johan Karlsson, Jorge Gonçalves, A. Aalto
          </td>
          <td>2024-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Diagnosis and prognosis are especially difficult areas of medical research related to cancer due to the high incidence of breast cancer, which has surpassed all other cancers in terms of female mortality. Another factor that has a substantial influence on the quality of life of cancer patients is the fear that they may experience a relapse of their disease. The objective of the study is to give medical practitioners a more effective strategy for using ensemble learning techniques to forecast when breast cancer may recur. This research aimed to investigate the usage of deep neural networks (DNNs) and artificial neural networks (ANNs) in addition to machine learning (ML) based approaches, including bagging, averaging, and voting, to enhance the efficacy of breast cancer relapse diagnosis on two breast cancer relapse datasets. Results from the empirical study demonstrate that the proposed ensemble learning-enabled approach improves accuracies by 96.31% and 95.81%, precisions by 96.70% and 96.15%, sensitivities by 98.88% and 98.68%, specificities by 84.62% in both, F1-scores by 97.78% and 97.40%, and area under the curve (AUCs) of 0.987 and 0.978, with University Medical Centre, Institute of Oncology (UMCIO) and Wisconsin prognostic breast cancer (WPBC) datasets respectively. Consequently, these improved disease outcomes may encourage physicians to use this model to make better treatment choices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cda0380ec84db5b7b900c42f0f27833d1ab99f3" target='_blank'>
              Breast cancer relapse disease prediction improvements with ensemble learning approaches
              </a>
            </td>
          <td>
            Ghanashyam Sahoo, Ajit Kumar Nayak, Pradyumna Kumar Tripathy, Abhilash Pati, Amrutanshu Panigrahi, Adyasha Rath, Bhimasen Moharana
          </td>
          <td>2024-07-01</td>
          <td>Indonesian Journal of Electrical Engineering and Computer Science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Several recent studies have optimized deep neural networks to learn high-dimensional relationships linking structural and functional connectivity across the human connectome. However, the extent to which these models recapitulate individual-specific characteristics of resting-state functional brain networks remains unclear. A core concern relates to whether current individual predictions outperform simple benchmarks such as group averages and null conditions. Here, we consider two measures to statistically evaluate whether functional connectivity predictions capture individual effects. We revisit our previously published functional connectivity predictions for 1000 healthy adults and provide multiple lines of evidence supporting that our predictions successfully capture subtle individual-specific variation in connectivity. While predicted individual effects are statistically significant and outperform several benchmarks, we find that effect sizes are small (i.e., 8–11% improvement relative to group-average benchmarks). As such, initial expectations about individual prediction performance expressed by us and others may require moderation. We conclude that individual predictions can significantly outperform appropriate benchmark conditions and we provide several recommendations for future studies in this area. Future studies should statistically assess the individual prediction performance of their models using one of the measures and benchmarks provided here.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3816457b161bdd6ea07785b4cfcadeb0d0b69b13" target='_blank'>
              Predicting an individual’s functional connectivity from their structural connectome: Evaluation of evidence, recommendations and future prospects
              </a>
            </td>
          <td>
            Andrew Zalesky, Tabinda Sarwar, Y. Tian, Yuanzhe Liu, B. Yeo, K. Ramamohanarao
          </td>
          <td>2024-06-24</td>
          <td>Network Neuroscience</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="It is becoming clear that bulk gene expression measurements represent an average over very different cells. Elucidating the expression and abundance of each of the encompassed cells is key to disease understanding and precision medicine approaches. A first step in any such deconvolution is the inference of cell type abundances in the given mixture. Numerous approaches to cell-type deconvolution have been proposed, yet very few take advantage of the emerging discipline of deep learning and most approaches are limited to input data regarding the expression profiles of the cell types in question. Here we present DECODE, a deep learning method for the task that is data-driven and does not depend on input expression profiles. DECODE builds on a deep unfolded non-negative matrix factorization technique. It is shown to outperform previous approaches on a range of synthetic and real data sets, producing abundance estimates that are closer to and better correlated with the real values.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e613190af39a3bdbca89af001530f5a32a0af36c" target='_blank'>
              Deep Cell-Type Deconvolution from Bulk Gene Expression Data Using DECODE
              </a>
            </td>
          <td>
            Eran Hermush, R. Sharan
          </td>
          <td>2024-06-18</td>
          <td>Digital Medicine and Healthcare Technology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Certain cancer types, namely pancreatic cancer is difficult to detect at an early stage; sparking the importance of discovering the causal relationship between biomarkers and cancer to identify cancer efficiently. By allowing for the detection and monitoring of specific biomarkers through a non-invasive method, liquid biopsies enhance the precision and efficacy of medical interventions, advocating the move towards personalized healthcare. Several machine learning algorithms such as Random Forest, SVM are utilized for classification, yet causing inefficiency due to the need for conducting hyperparameter tuning. We leverage a meta-trained Hyperfast model for classifying cancer, accomplishing the highest AUC of 0.9929 and simultaneously achieving robustness especially on highly imbalanced datasets compared to other ML algorithms in several binary classification tasks (e.g. breast invasive carcinoma; BRCA vs. non-BRCA). We also propose a novel ensemble model combining pre-trained Hyperfast model, XGBoost, and LightGBM for multi-class classification tasks, achieving an incremental increase in accuracy (0.9464) while merely using 500 PCA features; distinguishable from previous studies where they used more than 2,000 features for similar results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c9894bd9c35a06d4073a4397d4705474cc1e65b" target='_blank'>
              Biomarker based Cancer Classification using an Ensemble with Pre-trained Models
              </a>
            </td>
          <td>
            Chongmin Lee, Jihie Kim
          </td>
          <td>2024-06-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="We investigate trends in the data-error scaling behavior of machine learning (ML) models trained on discrete combinatorial spaces that are prone-to-mutation, such as proteins or organic small molecules. We trained and evaluated kernel ridge regression machines using variable amounts of computationally generated training data. Our synthetic datasets comprise i) two na\"ive functions based on many-body theory; ii) binding energy estimates between a protein and a mutagenised peptide; and iii) solvation energies of two 6-heavy atom structural graphs. In contrast to typical data-error scaling, our results showed discontinuous monotonic phase transitions during learning, observed as rapid drops in the test error at particular thresholds of training data. We observed two learning regimes, which we call saturated and asymptotic decay, and found that they are conditioned by the level of complexity (i.e. number of mutations) enclosed in the training set. We show that during training on this class of problems, the predictions were clustered by the ML models employed in the calibration plots. Furthermore, we present an alternative strategy to normalize learning curves (LCs) and the concept of mutant based shuffling. This work has implications for machine learning on mutagenisable discrete spaces such as chemical properties or protein phenotype prediction, and improves basic understanding of concepts in statistical learning theory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ffa62d0de0dd69c6cbdb95f7d32cb464245d35" target='_blank'>
              Data-Error Scaling in Machine Learning on Natural Discrete Combinatorial Mutation-prone Sets: Case Studies on Peptides and Small Molecules
              </a>
            </td>
          <td>
            Vanni Doffini, O. V. Lilienfeld, Michael A. Nash
          </td>
          <td>2024-05-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Motivated by the pressing challenges in the digital twin development for biomanufacturing systems, we introduce an adjoint sensitivity analysis (SA) approach to expedite the learning of mechanistic model parameters. In this paper, we consider enzymatic stochastic reaction networks representing a multi-scale bioprocess mechanistic model that allows us to integrate disparate data from diverse production processes and leverage the information from existing macro-kinetic and genome-scale models. To support forward prediction and backward reasoning, we develop a convergent adjoint SA algorithm studying how the perturbations of model parameters and inputs (e.g., initial state) propagate through enzymatic reaction networks and impact on output trajectory predictions. This SA can provide a sample efficient and interpretable way to assess the sensitivities between inputs and outputs accounting for their causal dependencies. Our empirical study underscores the resilience of these sensitivities and illuminates a deeper comprehension of the regulatory mechanisms behind bioprocess through sensitivities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa83caf636ea9a9577cbd13662fa8d1e926e34f" target='_blank'>
              Adjoint Sensitivity Analysis on Multi-Scale Bioprocess Stochastic Reaction Network
              </a>
            </td>
          <td>
            Keilung Choy, Wei Xie
          </td>
          <td>2024-05-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Many problems in physical sciences are characterized by the prediction of space-time sequences. Such problems range from weather prediction to the analysis of disease propagation and video prediction. Modern techniques for the solution of these problems typically combine Convolution Neural Networks (CNN) architecture with a time prediction mechanism. However, oftentimes, such approaches underperform in the long-range propagation of information and lack explainability. In this work, we introduce a physically inspired architecture for the solution of such problems. Namely, we propose to augment CNNs with advection by designing a novel semi-Lagrangian push operator. We show that the proposed operator allows for the non-local transformation of information compared with standard convolutional kernels. We then complement it with Reaction and Diffusion neural components to form a network that mimics the Reaction-Advection-Diffusion equation, in high dimensions. We demonstrate the effectiveness of our network on a number of spatio-temporal datasets that show their merit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc41c2840d718cf94ea811d2a43c01d4849949a" target='_blank'>
              Advection Augmented Convolutional Neural Networks
              </a>
            </td>
          <td>
            N. Zakariaei, Siddharth Rout, Eldad Haber, Moshe Eliasof
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="G-protein coupled receptors (GPCRs) are transmembrane proteins that transmit signals from the extracellular environment to the inside of the cells. Their ability to adopt various conformational states, which influence their function, makes them crucial in pharmacoproteomic studies. While many drugs target specific GPCR states to exert their effects—thereby regulating the protein’s activity—unraveling the activation pathway remains challenging due to the multitude of intermediate transformations occurring throughout this process, and intrinsically influencing the dynamics of the receptors. In this context, computational modeling, particularly molecular dynamics (MD) simulations, may offer valuable insights into the dynamics and energetics of GPCR transformations, especially when combined with machine learning (ML) methods and techniques for achieving model interpretability for knowledge generation. The current study builds upon previous work in which the layer relevance propagation (LRP) technique was employed to interpret the predictions in a multi-class classification problem concerning the conformational states of the β2-adrenergic (β2AR) receptor from MD simulations. Here, we address the challenges posed by class imbalance and extend previous analyses by evaluating the robustness and stability of deep learning (DL)-based predictions under different imbalance mitigation techniques. By meticulously evaluating explainability and imbalance strategies, we aim to produce reliable and robust insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa30cc0ae2a6e6381d995fe687640fc3f469403" target='_blank'>
              A Study on the Robustness and Stability of Explainable Deep Learning in an Imbalanced Setting: The Exploration of the Conformational Space of G Protein-Coupled Receptors
              </a>
            </td>
          <td>
            M. A. Gutierrez-Mondragon, Alfredo Vellido, Caroline König
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The precise prediction of molecular properties can greatly accelerate the development of new drugs. However, in silico molecular property prediction approaches have been limited so far to assays for which large amounts of data are available. In this study, we develop a new computational approach leveraging both the textual description of the assay of interest and the chemical structure of target compounds. By combining these two sources of information via self-supervised learning, our tool can provide accurate predictions for assays where no measurements are available. Remarkably, our approach achieves state-of-the-art performance on the FS-Mol benchmark for zero-shot prediction, outperforming a wide variety of deep learning approaches. Additionally, we demonstrate how our tool can be used for tailoring screening libraries for the assay of interest, showing promising performance in a retrospective case study on a high-throughput screening campaign. By accelerating the early identification of active molecules in drug discovery and development, this method has the potential to streamline the identification of novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65027730b047f692f4242f772033d2fe6821a424" target='_blank'>
              Synergizing Chemical Structures and Bioassay Descriptions for Enhanced Molecular Property Prediction in Drug Discovery
              </a>
            </td>
          <td>
            Maximilian G Schuh, Davide Boldini, Stephan A Sieber
          </td>
          <td>2024-06-05</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Preventing, diagnosing, and treating diseases requires accurate clinical biomarkers, which remains challenging. Recently, advanced computational approaches have accelerated the discovery of promising biomarkers from high-dimensional multimodal data. Although machine-learning methods have greatly contributed to the research fields, handling data sparseness, which is not unusual in research settings, is still an issue as it leads to limited interpretability and performance in the presence of missing information. Here, we propose a novel pipeline integrating joint non-negative matrix factorization (JNMF), identifying key features within sparse high-dimensional heterogeneous data, and a biological pathway analysis, interpreting the functionality of features by detecting activated signaling pathways. By applying our pipeline to large-scale public cancer datasets, we identified sets of genomic features relevant to specific cancer types as common pattern modules (CPMs) of JNMF. We further detected COPS5 as a potential upstream regulator of pathways associated with diffuse large B-cell lymphoma (DLBCL). COPS5 exhibited co-overexpression with MYC, TP53, and BCL2, known DLBCL marker genes, and its high expression was correlated with a lower survival probability of DLBCL patients. Using the CRISPR-Cas9 system, we confirmed the tumor growth effect of COPS5, which suggests it as a novel prognostic biomarker for DLBCL. Our results highlight that integrating multiple high-dimensional data and effectively decomposing them to interpretable dimensions unravels hidden biological importance, which enhances the discovery of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ab81c7bf9d4d54c741aeecc346cc29faef5638f" target='_blank'>
              Integrative analysis of cancer multimodality data identifying COPS5 as a novel biomarker of diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Yutong Dai, Jingmei Li, Keita Yamamoto, Susumu Goyama, Martin Loza, Sung-Joon Park, Kenta Nakai
          </td>
          <td>2024-06-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65f70fe6d698072acfae68cb9be3a90281df6d72" target='_blank'>
              Application of physics encoded neural networks to improve predictability of properties of complex multi-scale systems
              </a>
            </td>
          <td>
            M. Meinders, Jack Yang, Erik van der Linden
          </td>
          <td>2024-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Modern generative models demonstrate impressive capabilities, likely stemming from an ability to identify and manipulate abstract concepts underlying their training data. However, fundamental questions remain: what determines the concepts a model learns, the order in which it learns them, and its ability to manipulate those concepts? To address these questions, we propose analyzing a model's learning dynamics via a framework we call the concept space, where each axis represents an independent concept underlying the data generating process. By characterizing learning dynamics in this space, we identify how the speed at which a concept is learned, and hence the order of concept learning, is controlled by properties of the data we term concept signal. Further, we observe moments of sudden turns in the direction of a model's learning dynamics in concept space. Surprisingly, these points precisely correspond to the emergence of hidden capabilities, i.e., where latent interventions show the model possesses the capability to manipulate a concept, but these capabilities cannot yet be elicited via naive input prompting. While our results focus on synthetically defined toy datasets, we hypothesize a general claim on emergence of hidden capabilities may hold: generative models possess latent capabilities that emerge suddenly and consistently during training, though a model might not exhibit these capabilities under naive input prompting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a6c5534663a7858630ef0147815718535a1269" target='_blank'>
              Emergence of Hidden Capabilities: Exploring Learning Dynamics in Concept Space
              </a>
            </td>
          <td>
            Core Francisco Park, Maya Okawa, Andrew Lee, Ekdeep Singh Lubana, Hidenori Tanaka
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="10043 Background: Prediction, understanding, and management, of neuroblastoma (NBL) outcomes, from spontaneous regression to relapse and death, remain limited, and rely mostly on the International Neuroblastoma Staging System (INSS) stage, age, and the one-gene test for MYCN amplification. The entire multi-ome affects every aspect of the disease. But typical artificial intelligence and machine learning (AI/ML) cannot discover effective biomarkers from real-life, small-cohort, multi-omic, and noisy data. Methods: We use our AI/ML to identify two whole-genome biomarkers from open-source pre-treatment NBL patient-matched primary tumor and whole blood DNA copy numbers, and primary tumor mRNA expression. Our data-agnostic, unsupervised algorithms extend the mathematics that underlies quantum mechanics to overcome the limitations of typical AI/ML [1,2]. Our platform- and reference genome-agnostic biomarkers in astrocytoma, including glioblastoma, and lung, ovarian, and uterine adenocarcinomas, were mathematically discovered, and computationally and experimentally validated, repeatedly, in federated, imbalanced datasets from as few as 50–100 patients [3]. Results: We discover the biomarkers in “skinny” datasets of minimally preprocessed ≈3M-bin whole-genome sequences from tumors and blood of a set of 101 patients. Blindly, i.e., label-free, we separate both biomarkers from the normal X chromosome-number variation, demonstrating sex-agnostic learning. Combined, the biomarkers are statistically independent of all standard-of-care indicators, with the univariate Cox hazard ratio of 4.0 (log-rank P-value=2.3×10–5) within the 95% confidence intervals of the bivariate ratios. The risk that the tumor’s whole genome confers upon outcome, as is reflected by the bivariate ratios, is greater than that conferred by all standard-of-care indicators, including DNA ploidy, the Children’s Oncology Group (COG) risk, and the mitosis-karyorrhexis index (MKI), except for histopathology. We validate the biomarkers in ≈10K-bin target-capture sequences of a mutually-exclusive set of 419 patients, demonstrating generalizability as well as site-, platform-, and protocol-agnostic transfer learning. At 73–80% concordance, in both the tumor and blood profiles of both the discovery and validation sets, the biomarkers combined are more accurate than MYCN. We show that the biomarkers identify known and previously unrecognized disease mechanisms and druggable gene alterations, including co-amplification of MYCN with genes encoding for extra-embryonic transcripts, as well as the hijacking of embryonic development toward aneuploidy, which can spontaneously regress via embryonic self-correction. Conclusions: Our AI/ML is uniquely suited for personalized medicine. 1. doi: 10.1063/1.5099268 2. doi: 10.1073/pnas.0530258100 3. doi: 10.1063/1.5142559.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0234e4be40726e2547947696f74629005c40508" target='_blank'>
              AI/ML-derived mechanistically interpretable whole-genome biomarkers of patient survival in pre-treatment primary neuroblastoma tumors and whole blood.
              </a>
            </td>
          <td>
            Orly Alter, Elizabeth Newman, Sri Priya Ponnapalli, Jessica W. Tsai
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction: This study bridges traditional remedies and modern pharmacology by exploring the synergy between natural compounds and Ceritinib in treating Non-Small Cell Lung Cancer (NSCLC), aiming to enhance efficacy and reduce toxicities. Methods: Using a combined approach of computational analysis, machine learning, and experimental procedures, we identified and analyzed PD173074, Isoquercitrin, and Rhapontin as potential inhibitors of fibroblast growth factor receptor 3 (FGFR3). Machine learning algorithms guided the initial selection, followed by Quantitative Structure-Activity Relationship (QSAR) modeling and molecular dynamics simulations to evaluate the interaction dynamics and stability of Rhapontin. Physicochemical assessments further verified its drug-like properties and specificity. Results: Our experiments demonstrate that Rhapontin, when combined with Ceritinib, significantly suppresses tumor activity in NSCLC while sparing healthy cells. The molecular simulations and physicochemical evaluations confirm Rhapontin’s stability and favorable interaction with FGFR3, highlighting its potential as an effective adjunct in NSCLC therapy. Discussion: The integration of natural compounds with established cancer therapies offers a promising avenue for enhancing treatment outcomes in NSCLC. By combining the ancient wisdom of natural remedies with the precision of modern science, this study contributes to evolving cancer treatment paradigms, potentially mitigating the side effects associated with current therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fb4b90bbdc60b8cb3c0cd01d57b8e7e7c0b535a" target='_blank'>
              Active and machine learning-enhanced discovery of new FGFR3 inhibitor, Rhapontin, through virtual screening of receptor structures and anti-cancer activity assessment
              </a>
            </td>
          <td>
            Qingxin Zeng, Haichuan Hu, Zhengwei Huang, Aotian Guo, Sheng Lu, Wenbin Tong, Zhongheng Zhang, Tao Shen
          </td>
          <td>2024-06-11</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Understanding the mechanisms underlying crystal formation is crucial. For most systems, crystallization typically goes through a nucleation process that involves dynamics that happen at short time and length scales. Due to this, molecular dynamics serves as a powerful tool to study this phenomenon. Existing approaches to study the mechanism often focus analysis on static snapshots of the global configuration, potentially overlooking subtle local fluctuations and history of the atoms involved in the formation of solid nuclei. To address this limitation, we propose a methodology that categorizes nucleation pathways into reactive pathways based on the time evolution of constituent atoms. Our approach effectively captures the diverse structural pathways explored by crystallizing Lennard-Jones-like particles and solidifying Ni$_3$Al, providing a more nuanced understanding of nucleating pathways. Moreover, our methodology enables the prediction of the resulting polymorph from each reactive trajectory. This deep learning-assisted comprehensive analysis offers an alternative view of crystal nucleation mechanisms and pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/734e1f4d145aa409813ced436949cacec2844770" target='_blank'>
              LeaPP: Learning Pathways to Polymorphs through machine learning analysis of atomic trajectories
              </a>
            </td>
          <td>
            Steven W Hall, Porhouy Minh, Sapna Sarupria
          </td>
          <td>2024-05-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Task-specific functional MRI (fMRI) images provide excellent modalities for studying the neuronal basis of cognitive processes. We use fMRI data to formulate and solve the problem of deconvolving task-specific aggregate neuronal networks into a set of basic building blocks called canonical networks, to use these networks for functional characterization, and to characterize the physiological basis of these responses by mapping them to regions of the brain. Our results show excellent task-specificity of canonical networks, i.e., the expression of a small number of canonical networks can be used to accurately predict tasks; generalizability across cohorts, i.e., canonical networks are conserved across diverse populations, studies, and acquisition protocols; and that canonical networks have strong anatomical and physiological basis. From a methods perspective, the problem of identifying these canonical networks poses challenges rooted in the high dimensionality, small sample size, acquisition variability, and noise. Our deconvolution technique is based on non-negative matrix factorization (NMF) that identifies canonical networks as factors of a suitably constructed matrix. We demonstrate that our method scales to large datasets, yields stable and accurate factors, and is robust to noise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3130e7a08afa6272e164684a7f59dab312eb1de7" target='_blank'>
              Deconvolving Complex Neuronal Networks into Interpretable Task-Specific Connectomes
              </a>
            </td>
          <td>
            Yifan Wang, V. Ravindra, A. Grama
          </td>
          <td>2024-06-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Spiking Neural Networks (SNNs) emulate the integrated-fire-leak mechanism found in biological neurons, offering a compelling combination of biological realism and energy efficiency. In recent years, they have gained considerable research interest. However, existing SNNs predominantly rely on the Leaky Integrate-and-Fire (LIF) model and are primarily suited for simple, static tasks. They lack the ability to effectively model long-term temporal dependencies and facilitate spatial information interaction, which is crucial for tackling complex, dynamic spatio-temporal prediction tasks. To tackle these challenges, this paper draws inspiration from the concept of autaptic synapses in biology and proposes a novel Spatio-Temporal Circuit (STC) model. The STC model integrates two learnable adaptive pathways, enhancing the spiking neurons' temporal memory and spatial coordination. We conduct a theoretical analysis of the dynamic parameters in the STC model, highlighting their contribution in establishing long-term memory and mitigating the issue of gradient vanishing. Through extensive experiments on multiple spatio-temporal prediction datasets, we demonstrate that our model outperforms other adaptive models. Furthermore, our model is compatible with existing spiking neuron models, thereby augmenting their dynamic representations. In essence, our work enriches the specificity and topological complexity of SNNs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12dfecc261785b9363f0b41e9e2529956b457e1f" target='_blank'>
              Autaptic Synaptic Circuit Enhances Spatio-temporal Predictive Learning of Spiking Neural Networks
              </a>
            </td>
          <td>
            Lihao Wang, Zhaofei Yu
          </td>
          <td>2024-06-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite the advancements in learning governing differential equations from observations of dynamical systems, data-driven methods are often unaware of fundamental physical laws, such as frame invariance. As a result, these algorithms may search an unnecessarily large space and discover equations that are less accurate or overly complex. In this paper, we propose to leverage symmetry in automated equation discovery to compress the equation search space and improve the accuracy and simplicity of the learned equations. Specifically, we derive equivariance constraints from the time-independent symmetries of ODEs. Depending on the types of symmetries, we develop a pipeline for incorporating symmetry constraints into various equation discovery algorithms, including sparse regression and genetic programming. In experiments across a diverse range of dynamical systems, our approach demonstrates better robustness against noise and recovers governing equations with significantly higher probability than baselines without symmetry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/294f1e8ba8fdeee906321b73f3e14bd0b704a7e0" target='_blank'>
              Symmetry-Informed Governing Equation Discovery
              </a>
            </td>
          <td>
            Jianwei Yang, Wang Rao, Nima Dehmamy, R. Walters, Rose Yu
          </td>
          <td>2024-05-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Dynamic processes in complex networks are crucial for better understanding collective behavior in human societies, biological systems, and the internet. In this paper, we first focus on the continuous Markov-based modeling of evolving networks with the birth-death of individuals. A new individual arrives at the group by the Poisson process, while new links are established in the network through either uniform connection or preferential attachment. Moreover, an existing individual has a limited lifespan before leaving the network. We determine stationary topological properties of these networks, including their size and mean degree. To address the effect of the birth-death evolution, we further study the information dynamics in the proposed network model from the random drift and natural selection perspective, based on assumptions of total-stochastic and fitness-driven evolution, respectively. In simulations, we analyze the fixation probability of individual information and find that means of new connections affect the random drift process but do not affect the natural selection process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cc97d3b6a2007d5cad5a44d3da12a6a5c9c0d40" target='_blank'>
              Information Dynamics in Evolving Networks Based on the Birth-Death Process: Random Drift and Natural Selection Perspective
              </a>
            </td>
          <td>
            Minyu Feng, Ziyan Zeng, Qin Li, M. Perc, Jürgen Kurths
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>109</td>
        </tr>

        <tr id="Deep learning with physics-informed neural networks (PINNs) has emerged as a highly popular and effective approach for solving partial differential equations(PDEs). In this paper, we first investigate the extrapolation capability of the PINN method for time-dependent PDEs. Taking advantage of this extrapolation property, we can generalize the training result obtained in the time subinterval to the large interval by adding a correction term to the network parameters of the subinterval. The correction term is determined by further training with the sample points in the added subinterval. Secondly, by designing an extrapolation control function with special characteristics and combining it with the correction term, we construct a new neural network architecture whose network parameters are coupled with the time variable, which we call the extrapolation-driven network architecture. Based on this architecture, using a single neural network, we can obtain the overall PINN solution of the whole domain with the following two characteristics: (1) it completely inherits the local solution of the interval obtained from the previous training, (2) at the interval node, it strictly maintains the continuity and smoothness that the true solution has. The extrapolation-driven network architecture allows us to divide a large time domain into multiple subintervals and solve the time-dependent PDEs one by one in chronological order. This training scheme respects the causality principle and effectively overcomes the difficulties of the conventional PINN method in solving the evolution equation on a large time domain. Numerical experiments verify the performance of our proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e945aa09792681fe44ec2d98a12595cf885296" target='_blank'>
              An extrapolation-driven network architecture for physics-informed deep learning
              </a>
            </td>
          <td>
            Yong Wang, Yanzhong Yao, Zhiming Gao
          </td>
          <td>2024-06-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="5065 Background: Overall survival (OS) remains the gold standard endpoint for clinical trials in metastatic prostate cancer (metPC) yet requires extensive follow-up. We sought to develop a model utilizing short term (≤ 4 months) prostate specific antigen (PSA) kinetic data to predict the OS readout from phase 3 clinical trials, with the goal of accelerating readout time of trials for patients (pts) with metPC. Methods: Clinical and PSA data were obtained from 7 metPC trials: 6 randomized, double-blind, phase 3 trials (TITAN, COU-AA-301, COU-AA-302, LATITUDE, ACIS, and MAGNITUDE), and one multicenter, phase II trial (GALAHAD). We developed 18 PSA kinetic variables from the first 4 months of enrollment including: 50% decline in PSA (PSA50), 90% decline in PSA, PSA=0.1, and PSA=0.2 within 1, 2, 3, and 4 months; and slope and base of the exponential fit of PSA over the first 4 months on study). To balance pt and disease characteristics, TITAN, COU-AA-301, and ACIS were used to train the intermediate endpoint, with remaining studies held out as an external testing cohort. A random 60/40% split of the pooled pt-level data was performed as a sensitivity analysis. Adaptive least absolute shrinkage and selection operator (aLASSO)-based Cox proportional hazards models were applied to select previously identified prognostic baseline clinical variables with and without PSA kinetic variables most predictive of OS on five-fold cross validation. Performance with and without PSA kinetics was assessed using C-index, integrated Brier score (IBS), and time-dependent area under the receiver operating characteristic curve (tAUC) at 12, 24, 36, and 48 months. Results: The study included 6,451 pts with median follow-up of 22 months and 85,785 PSA values. aLASSO selected Eastern Cooperative Oncology Group performance status, disease extent, lactate dehydrogenase, albumin, hemoglobin, PSA, and alkaline phosphatase as baseline variables for the prognostic comparator model. In the model including PSA kinetics, aLASSO identified PSA50 at 4 months, PSA0.1 at 1 month, and PSA slope as kinetics providing additional intra-treatment information to improve prediction of OS. The model with PSA kinetics showed improved performance on the test trials: C-index (0.72 vs. 0.66), IBS (0.158 vs. 0.173), and tAUC (0.84, 0.78, 0.78, and 0.76 vs. 0.78, 0.71, 0.69, and 0.65 at times 12, 24, 36, and 48 months, respectively; p < 0.05.) Sensitivity analysis using a random split of the pooled pt-level data supported the importance of PSA50 at 4 months and PSA slope. Conclusions: Data from 6,451 mPC pts enrolled on prospective clinical trials were used to develop a machine learning model which included kinetic PSA data and predicted the long-term OS readout in Phase 3 trials with just the first four months of data. This model will be validated in independent data sets comprised of other completed phase 3 trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dac296b786d21d2a4c231f4e0d52d3f32e79335" target='_blank'>
              Development of a machine learning model to predict overall survival results from randomized clinical trials of patients with metastatic prostate cancer.
              </a>
            </td>
          <td>
            A. Sabbagh, David A Quigley, Nicholas Lillis, Li Zhang, I. Friesner, A. Bailey, Rahul R Aggarwal, Meera Reddy Chappidi, Hari Singhal, Ke Zhang, J. Greshock, Jinhui Li, Margaret K. Yu, Eric J. Small, Julian C. Hong
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Developing and optimizing models for complex systems poses challenges due to the inherent complexity introduced by multiple types of input information and sources of uncertainty. In this study, we utilize Bayesian formalism to analytically examine the propagation of probability in the modeling process and propose quantitative assessments for it. Upon which, we develop a method for optimizing models for complex systems by (i) minimizing model uncertainty; (ii) maximizing model consistency; and (iii) minimizing model complexity, following the Bayesian Occam’s razor rationale. We showcase the benefits of this method by optimizing the modeling of the dynamic system of glucose-stimulated insulin secretion in pancreatic β-cells, leading to an optimized model that demonstrates better alignment with experimental observations compared to the non-optimized one. We anticipate that this method will facilitate the construction of accurate, precise, and sufficiently simple models for diverse complex systems. It is implemented in our open-source software Integrative Modeling Platform (IMP), ensuring its broad applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2852aa1c64c573614d3968195a3e4de98331119b" target='_blank'>
              Bayesian Occam’s Razor to Optimize Models for Complex Systems
              </a>
            </td>
          <td>
            Chenxi Wang, Jihui Zhao, Jingjing Zheng, Barak Raveh, Xuming He, Liping Sun
          </td>
          <td>2024-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="A deep latent variable model is a powerful method for capturing complex distributions. These models assume that underlying structures, but unobserved, are present within the data. In this dissertation, we explore high-dimensional problems related to physiological monitoring using latent variable models. First, we present a novel deep state-space model to generate electrical waveforms of the heart using optically obtained signals as inputs. This can bring about clinical diagnoses of heart disease via simple assessment through wearable devices. Second, we present a brain signal modeling scheme that combines the strengths of probabilistic graphical models and deep adversarial learning. The structured representations can provide interpretability and encode inductive biases to reduce the data complexity of neural oscillations. The efficacy of the learned representations is further studied in epilepsy seizure detection formulated as an unsupervised learning problem. Third, we propose a framework for the joint modeling of physiological measures and behavior. Existing methods to combine multiple sources of brain data provided are limited. Direct analysis of the relationship between different types of physiological measures usually does not involve behavioral data. Our method can identify the unique and shared contributions of brain regions to behavior and can be used to discover new functions of brain regions. The success of these innovative computational methods would allow the translation of biomarker findings across species and provide insight into neurocognitive analysis in numerous biological studies and clinical diagnoses, as well as emerging consumer applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf07061c3b5a4943a43f226d9cab1e727639eec0" target='_blank'>
              Deep Latent Variable Modeling of Physiological Signals
              </a>
            </td>
          <td>
            Khuong Vo
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Diffusion regulates a phenomenal number of natural processes and the dynamics of many successful generative models. Existing models to learn the diffusion terms from observational data rely on complex bilevel optimization problems and properly model only the drift of the system. We propose a new simple model, JKOnet*, which bypasses altogether the complexity of existing architectures while presenting significantly enhanced representational capacity: JKOnet* recovers the potential, interaction, and internal energy components of the underlying diffusion process. JKOnet* minimizes a simple quadratic loss, runs at lightspeed, and drastically outperforms other baselines in practice. Additionally, JKOnet* provides a closed-form optimal solution for linearly parametrized functionals. Our methodology is based on the interpretation of diffusion processes as energy-minimizing trajectories in the probability space via the so-called JKO scheme, which we study via its first-order optimality conditions, in light of few-weeks-old advancements in optimization in the probability space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e136b8e7261e1835acc89bf0402e956bd4ee5225" target='_blank'>
              Learning Diffusion at Lightspeed
              </a>
            </td>
          <td>
            Antonio Terpin, Nicolas Lanzetti, Florian Dorfler
          </td>
          <td>2024-06-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Under normal conditions the principal cells of the striatum, medium spiny neurons (MSNs), show structured cell assembly activity patterns which alternate sequentially over exceedingly long timescales of many minutes. It is important to understand this activity since it is characteristically disrupted in multiple pathologies, such as Parkinson's disease and dyskinesia, and thought to be caused by alterations in the MSN to MSN lateral inhibitory connections and in the strength and distribution of cortical excitation to MSNs. To understand how these long timescales arise we extended a previous network model of MSN cells to include synapses with short-term plasticity, with parameters taken from a recent detailed striatal connectome study. We first confirmed the presence of sequentially switching cell clusters using the non-linear dimensionality reduction technique, Uniform Manifold Approximation and Projection (UMAP). We found that the network could generate non-stationary activity patterns varying extremely slowly on the order of minutes under biologically realistic conditions. Next we used Simulation Based Inference (SBI) to train a deep net to map features of the MSN network generated cell assembly activity to MSN network parameters. We used the trained SBI model to estimate MSN network parameters from ex-vivo brain slice calcium imaging data. We found that best fit network parameters were very close to their physiologically observed values. On the other hand network parameters estimated from Parkinsonian, decorticated and dyskinetic ex-vivo slice preparations were different. Our work may provide a pipeline for diagnosis of basal ganglia pathology from spiking data as well as for the design pharmacological treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1c16b19e154300ba2b803533fad1e8ee97c186" target='_blank'>
              Pathological cell assembly dynamics in a striatal MSN network model
              </a>
            </td>
          <td>
            Astrid Correa, Adam Ponzi, Vladimir M. Calderón, Rosanna Migliore
          </td>
          <td>2024-06-06</td>
          <td>Frontiers in Computational Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [26],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>