<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="red" data-md-color-accent="purple">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (Simulator)" class="md-header__button md-logo" aria-label="Literature Survey (Simulator)" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (Simulator)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              58912e2c2aaa77d1448d51e9d9460e06a5b924b9
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="red" data-md-color-accent="purple"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="red" data-md-color-accent="lime"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Symbolic%20regression/" class="md-tabs__link">
        
  
    
  
  Symbolic regression

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Koopman%20operator%20theory/" class="md-tabs__link">
        
  
    
  
  Koopman operator theory

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../PINNs/" class="md-tabs__link">
        
  
    
  
  PINNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Parametrizing%20of%20ODEs%20using%20ML/" class="md-tabs__link">
        
  
    
  
  Parametrizing of ODEs using ML

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Latent%20space%20simulators%20%28VAMP%29/" class="md-tabs__link">
        
  
    
  
  Latent space simulators (VAMP)

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Physics-based%20GNNs/" class="md-tabs__link">
        
  
    
  
  Physics-based GNNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Neural%20ODEs/" class="md-tabs__link">
        
  
    
  
  Neural ODEs

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (Simulator)" class="md-nav__button md-logo" aria-label="Literature Survey (Simulator)" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    Literature Survey (Simulator)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Symbolic%20regression/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Symbolic regression
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Koopman%20operator%20theory/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Koopman operator theory
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../PINNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    PINNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Parametrizing%20of%20ODEs%20using%20ML/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Parametrizing of ODEs using ML
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Latent%20space%20simulators%20%28VAMP%29/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Latent space simulators (VAMP)
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Physics-based%20GNNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Physics-based GNNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Neural%20ODEs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Neural ODEs
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>58912e2c2aaa77d1448d51e9d9460e06a5b924b9</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-05-13 06:08:01 UTC</i>
  </p>


  <p>
  <h3>Recommendations for the article <i>VAMPnets for deep learning of molecular kinetics</i></h3>
  <table id="table1" class="display" style="width:100%">
  <thead>
    <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f8a6341496701a0d7fdaa50e6a5aa157236a0ff8" target='_blank'>Author Correction: VAMPnets for deep learning of molecular kinetics</a></td>
          <td>

              Andreas Mardt,

              Luca Pasquali,

              Hao Wu,

              F. Noé,

          </td>
          <td>2018-10-22</td>
          <td>Nature Communications</td>
          <td>24</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c4d9b7166a2ac580c195d72f64489b3f4e0b9c23" target='_blank'>DeepWEST: Deep Learning of Kinetic Models with the Weighted Ensemble Simulation Toolkit for Enhanced Sampling.</a></td>
          <td>

              Anupam Anand Ojha,

              Saumya Thakur,

              Surl-Hee Ahn,

              Rommie E. Amaro,

          </td>
          <td>2023-01-31</td>
          <td>Journal of chemical theory and computation</td>
          <td>7</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e15c87cfb4d2b1f82076331f9c58085598440e46" target='_blank'>GraphVAMPNet, using graph neural networks and variational approach to markov processes for dynamical modeling of biomolecules</a></td>
          <td>

              Mahdi Ghorbani,

              Samarjeet Prasad,

              Jeffery B. Klauda,

              B. Brooks,

          </td>
          <td>2022-01-12</td>
          <td>The Journal of chemical physics</td>
          <td>19</td>
          <td>64</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/86de5fbe09c4381f548e1312d53f15d898c76c78" target='_blank'>Mind reading of the proteins: Deep-learning to forecast molecular dynamics</a></td>
          <td>

              C. Gupta,

              J. Cava,

              Daipayan Sarkar,

              Eric A. Wilson,

              J. Vant,

              Steven Murray,

              A. Singharoy,

              S. Karmaker,

          </td>
          <td>2020-07-29</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>23</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2e7163e31e9b32cec11005678bae9e1dbeb6d573" target='_blank'>Nonlinear Discovery of Slow Molecular Modes using Hierarchical Dynamics Encoders</a></td>
          <td>

              ,

              Hythem Sidky,

              Andrew L. Ferguson,

          </td>
          <td>2019-02-09</td>
          <td>The Journal of chemical physics</td>
          <td>74</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2d3000d245988a02d3c1060211e9d89c67147b49" target='_blank'>Molecular latent space simulators</a></td>
          <td>

              Hythem Sidky,

              Wei Chen,

              Andrew L. Ferguson,

          </td>
          <td>2020-07-01</td>
          <td>Chemical Science</td>
          <td>30</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/50c74a10fcddb0e41a5a00b965383fdc52534eec" target='_blank'>Machine Learning in Molecular Dynamics Simulations of Biomolecular Systems</a></td>
          <td>

              Christopher Kolloff,

              Simon Olsson,

          </td>
          <td>2022-05-06</td>
          <td>None</td>
          <td>2</td>
          <td>4</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/13b52cd927d3daa5c95528b987734d97474e790c" target='_blank'>Markov field models: Scaling molecular kinetics approaches to large molecular machines.</a></td>
          <td>

              Tim Hempel,

              Simon Olsson,

              Frank No'e,

          </td>
          <td>2022-06-23</td>
          <td>Current opinion in structural biology</td>
          <td>6</td>
          <td>9</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/260f212da0cf8b3309f0ab3a130a0ad22993892e" target='_blank'>Markov State Models: From an Art to a Science.</a></td>
          <td>

              B. Husic,

              V. Pande,

          </td>
          <td>2018-02-02</td>
          <td>Journal of the American Chemical Society</td>
          <td>557</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8271ca0056e9a3a09b33963054d04b7c14c9ed69" target='_blank'>Active Learning of the Conformational Ensemble of Proteins using Maximum Entropy VAMPNets</a></td>
          <td>

              D. Kleiman,

              D. Shukla,

          </td>
          <td>2023-01-13</td>
          <td>bioRxiv</td>
          <td>9</td>
          <td>7</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/41c0783182ca3e23732b95ddfdb6654cd23a4b45" target='_blank'>Deep learning to decompose macromolecules into independent Markovian domains</a></td>
          <td>

              Andreas Mardt,

              Tim Hempel,

              C. Clementi,

              F. Noé,

          </td>
          <td>2022-10-13</td>
          <td>Nature Communications</td>
          <td>13</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bc4e32c1b7fe39a37b21ec4c0624cfa45a6b36df" target='_blank'>Achieving Reversible Ligand-Protein Unbinding with Deep Learning and Molecular Dynamics through RAVE</a></td>
          <td>

              João Marcelo Lamim Ribeiro,

              P. Tiwary,

          </td>
          <td>2018-08-25</td>
          <td>bioRxiv</td>
          <td>6</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bab880c698b850ba18c08b9d93b73ffb538a417b" target='_blank'>Accelerated Simulations of Molecular Systems through Learning of their Effective Dynamics</a></td>
          <td>

              Pantelis R. Vlachas,

              J. Zavadlav,

              M. Praprotnik,

              P. Koumoutsakos,

          </td>
          <td>2021-02-17</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>77</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/51894ffcf778630e7b021bc473d13d028a3b9158" target='_blank'>DynamicBind: predicting ligand-specific protein-ligand complex structure with a deep equivariant generative model</a></td>
          <td>

              Wei Lu,

              Jixian Zhang,

              Weifeng Huang,

              Ziqiao Zhang,

              Xiangyu Jia,

              Zhenyu Wang,

              Leilei Shi,

              Chengtao Li,

              Peter G Wolynes,

              Shuangjia Zheng,

          </td>
          <td>2024-02-05</td>
          <td>Nature Communications</td>
          <td>6</td>
          <td>11</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0107d0f47f7d1ccf290538922728dcece452c0b5" target='_blank'>Note: Variational Encoding of Protein Dynamics Benefits from Maximizing Latent Autocorrelation</a></td>
          <td>

              H. Wayment-Steele,

              V. Pande,

          </td>
          <td>2018-03-17</td>
          <td>The Journal of chemical physics</td>
          <td>6</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/204cedd41db9db3623226e9dfce6d82aaa3f0f41" target='_blank'>Machine learning for collective variable discovery and enhanced sampling in biomolecular simulation</a></td>
          <td>

              Hythem Sidky,

              Wei Chen,

              Andrew L. Ferguson,

          </td>
          <td>2020-03-03</td>
          <td>Molecular Physics</td>
          <td>87</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b674a7aee72e9b9cc5390eca13f9c5c7812f2ba0" target='_blank'>Machine learning for molecular simulation</a></td>
          <td>

              F. Noé,

              A. Tkatchenko,

              K. Müller,

              C. Clementi,

          </td>
          <td>2019-11-07</td>
          <td>Annual review of physical chemistry</td>
          <td>486</td>
          <td>127</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9b70bf13b098d5388ed3acbaedfc7835c2de06fd" target='_blank'>An Efficient Path Classification Algorithm Based on Variational Autoencoder to Identify Metastable Path Channels for Complex Conformational Changes.</a></td>
          <td>

              Yunrui Qiu,

              Michael S O'Connor,

              Mingyi Xue,

              Bojun Liu,

              Xuhui Huang,

          </td>
          <td>2023-06-29</td>
          <td>Journal of chemical theory and computation</td>
          <td>8</td>
          <td>6</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/915f23b0ab17625cec9863dc2eee1d603dcac80c" target='_blank'>Learning Markovian dynamics with spectral maps.</a></td>
          <td>

              Jakub Rydzewski,

              Tuğçe Gökdemir,

          </td>
          <td>2023-11-28</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6db34306fd106dee45a5f252e9f2fc8ed11c80f7" target='_blank'>Unsupervised deep learning for molecular dynamics simulations: a novel analysis of protein–ligand interactions in SARS-CoV-2 Mpro</a></td>
          <td>

              Jessica Mustali,

              Ikki Yasuda,

              Yoshinori Hirano,

              Kenji Yasuoka,

              Alfonso Gautieri,

              Noriyoshi Arai,

          </td>
          <td>2023-11-16</td>
          <td>RSC Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d03d605aa500b66f4ac1f9ae486c56474ae62c94" target='_blank'>Markov State Models to Study the Functional Dynamics of Proteins in the Wake of Machine Learning</a></td>
          <td>

              K. Konovalov,

              I. C. Unarta,

              Siqin Cao,

              Eshani C. Goonetilleke,

              Xuhui Huang,

          </td>
          <td>2021-08-04</td>
          <td>JACS Au</td>
          <td>47</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fd1fb1d3e31260eaa72e3bff278b0cf837989a80" target='_blank'>RevGraphVAMP: A protein molecular simulation analysis model combining graph convolutional neural networks and physical constraints</a></td>
          <td>

              Ying Huang,

              Huiling Zhang,

              Zhenli Lin,

              Yanjie Wei,

              Wenhui Xi,

          </td>
          <td>2024-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5c6ee980b791766ba036cba282bcbcbcb2e5309e" target='_blank'>Deep Generative Markov State Models</a></td>
          <td>

              Hao Wu,

              Andreas Mardt,

              Luca Pasquali,

              F. Noé,

          </td>
          <td>2018-05-19</td>
          <td>ArXiv</td>
          <td>53</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/404b4fbf580f6c9dfb340603618e9840cd5e760f" target='_blank'>Chapter 12. Machine Learning Algorithms for the Analysis of Molecular Dynamics Trajectories</a></td>
          <td>

              Abbas Saleem Khan,

              B. Sim,

              Dongqing Wei,

          </td>
          <td>2021-09-24</td>
          <td>Theoretical and Computational Chemistry Series</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8f771f2fc3b8f446e7a1f1a15fb158a34966e294" target='_blank'>Explore Protein Conformational Space With Variational Autoencoder</a></td>
          <td>

              Hao Tian,

              Xi Jiang,

              Francesco Trozzi,

              Sian Xiao,

              Eric C. Larson,

              Peng Tao,

          </td>
          <td>2021-09-23</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>25</td>
          <td>23</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9ebf5c7ac8fc22961e7dc2130776a9b7d92d6d94" target='_blank'>Learning reduced kinetic Monte Carlo models of complex chemistry from molecular dynamics</a></td>
          <td>

              Qian Yang,

              C. Sing-Long,

              E. Reed,

          </td>
          <td>2017-06-19</td>
          <td>Chemical Science</td>
          <td>19</td>
          <td>41</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d8d8e2c04ca47bd628bd2a499e03ad7cd29633da" target='_blank'>Time-lagged autoencoders: Deep learning of slow collective variables for molecular kinetics</a></td>
          <td>

              C. Wehmeyer,

              F. Noé,

          </td>
          <td>2017-10-30</td>
          <td>The Journal of chemical physics</td>
          <td>327</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4813cd2feebbcef748d23902884261f10fb553e2" target='_blank'>Machine Learning Deciphered Molecular Mechanistics with Accurate Kinetic and Thermodynamic Prediction.</a></td>
          <td>

              Junlin Dong,

              Shiyu Wang,

              Wenqiang Cui,

              Xiaolin Sun,

              Haojie Guo,

              Hailu Yan,

              Horst Vogel,

              Zhi Wang,

              Shuguang Yuan,

          </td>
          <td>2024-02-23</td>
          <td>Journal of chemical theory and computation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9868cca5b562f4ea41bff4b5ee4ea63e62006430" target='_blank'>Learning dynamical information from static protein and sequencing data</a></td>
          <td>

              P. Pearce,

              Francis G. Woodhouse,

              Aden Forrow,

              Ashley J. Kelly,

              H. Kusumaatmaja,

              J. Dunkel,

          </td>
          <td>2018-08-27</td>
          <td>Nature Communications</td>
          <td>12</td>
          <td>40</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1c4c2983bc3719c80d93e39e59c58dd4c5a99afb" target='_blank'>Variational deep learning of equilibrium transition path ensembles.</a></td>
          <td>

              A. Singh,

              David T. Limmer,

          </td>
          <td>2023-02-28</td>
          <td>The Journal of chemical physics</td>
          <td>3</td>
          <td>31</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f8701f745cb0a5871c21f86e2045b190103c230c" target='_blank'>Deeptime: a Python library for machine learning dynamical models from time series data</a></td>
          <td>

              Moritz Hoffmann,

              Martin K. Scherer,

              Tim Hempel,

              Andreas Mardt,

              Brian de Silva,

              B. Husic,

              Stefan Klus,

              Hao Wu,

              N. Kutz,

              S. Brunton,

              F. Noé,

          </td>
          <td>2021-10-28</td>
          <td>Machine Learning: Science and Technology</td>
          <td>56</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cb4d8f8b2b1169d210d289c6b2deac0ccbbc34fe" target='_blank'>An Information Bottleneck Approach for Markov Model Construction</a></td>
          <td>

              Dedi Wang,

              Yunrui Qiu,

              E. Beyerle,

              Xuhui Huang,

              P. Tiwary,

          </td>
          <td>2024-04-03</td>
          <td>None</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/83e8e8d89e64de17d3d4709051a1f799c0b9ae22" target='_blank'>Deep learning for variational multiscale molecular modeling.</a></td>
          <td>

              Jun Zhang,

              Yao-Kun Lei,

              Yi Isaac Yang,

              Y. Gao,

          </td>
          <td>2020-06-29</td>
          <td>The Journal of chemical physics</td>
          <td>11</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ec461f9fd4b19b52508e85f3ffb7b7e70fc5d67" target='_blank'>Chasing collective variables using temporal data-driven strategies</a></td>
          <td>

              Haochuan Chen,

              C. Chipot,

          </td>
          <td>2023-01-06</td>
          <td>QRB Discovery</td>
          <td>9</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/68c0f4d535078674fd2ce738306eb72171a0677d" target='_blank'>Accelerated Simulations of Molecular Systems through Learning of Effective Dynamics.</a></td>
          <td>

              Pantelis R. Vlachas,

              J. Zavadlav,

              M. Praprotnik,

              P. Koumoutsakos,

          </td>
          <td>2021-12-10</td>
          <td>Journal of chemical theory and computation</td>
          <td>26</td>
          <td>77</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d00252df8465a93ba68dda823d8adb8e36418560" target='_blank'>Analyzing Molecular Dynamics Trajectories Thermodynamically through Artificial Intelligence.</a></td>
          <td>

              Xuyang Liu,

              Jingya Xing,

              Haohao Fu,

              Xueguang Shao,

              W. Cai,

          </td>
          <td>2024-01-09</td>
          <td>Journal of chemical theory and computation</td>
          <td>2</td>
          <td>42</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8c9f4af01a841b431917e605a85746154d49529d" target='_blank'>Spectral Maps for Learning Reduced Representations of Molecular Systems</a></td>
          <td>

              Tuugcce Gokdemir,

              Jakub Rydzewski,

          </td>
          <td>2023-11-07</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2646b021a73a0651c4dac42e99d757319bfa0832" target='_blank'>Learning Protein Dynamics with Metastable Switching Systems</a></td>
          <td>

              Bharath Ramsundar,

              V. Pande,

          </td>
          <td>2016-10-05</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bef7c61e1d111ab3d4b0ad39291346181aeabfe6" target='_blank'>Coupling Molecular Dynamics and Deep Learning to Mine Protein Conformational Space</a></td>
          <td>

              M. Degiacomi,

          </td>
          <td>2019-06-01</td>
          <td>Z--BiochemRN: Protein Folding (Proteins) (Topic)</td>
          <td>77</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d3d51ccff69b92a784a2e234d39f1f2c950bf1e1" target='_blank'>Machine learning approaches for analyzing and enhancing molecular dynamics simulations.</a></td>
          <td>

              Yihang Wang,

              João Marcelo Lamim Ribeiro,

              P. Tiwary,

          </td>
          <td>2019-09-25</td>
          <td>Current opinion in structural biology</td>
          <td>168</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5b10349eade93332fbff9127b5273382103fdf27" target='_blank'>Deep Learning for Multi-Scale Molecular Modeling</a></td>
          <td>

              ,

              Yao-Kun Lei,

              Yi Isaac Yang,

              Y. Gao,

          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4af4996d0dd209282e25f88ef7f280ef54b21be7" target='_blank'>Reaction coordinate flows for model reduction of molecular kinetics</a></td>
          <td>

              Hao Wu,

              Frank No'e,

          </td>
          <td>2023-09-11</td>
          <td>The Journal of chemical physics</td>
          <td>2</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/538d0f3f70c90dfd8d9f6e7bba3f50ea4058c34c" target='_blank'>High-Resolution Markov State Models for the Dynamics of Trp-Cage Miniprotein Constructed over Slow Folding Modes Identified by State-Free Reversible VAMPnets.</a></td>
          <td>

              Hythem Sidky,

              ,

              Andrew L. Ferguson,

          </td>
          <td>2019-06-12</td>
          <td>The journal of physical chemistry. B</td>
          <td>44</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/47171992793ccd578bd34a222537e71bc8d0def8" target='_blank'>Learning protein conformational space by enforcing physics with convolutions and latent interpolations</a></td>
          <td>

              Venkatakrishnan Ramaswamy,

              Chris G. Willcocks,

              M. Degiacomi,

          </td>
          <td>2019-10-10</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5fdcaee10a43c2f4df8b44e64042ec0ff0d40a5f" target='_blank'>Deep learning of dynamically responsive chemical Hamiltonians with semiempirical quantum mechanics</a></td>
          <td>

              Guoqing Zhou,

              N. Lubbers,

              K. Barros,

              S. Tretiak,

              B. Nebgen,

          </td>
          <td>2022-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>20</td>
          <td>69</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/267db6e5e036b0f4bccc9e9b06085288638e31f2" target='_blank'>Deep learning path-like collective variable for enhanced sampling molecular dynamics</a></td>
          <td>

              Thorben Frohlking,

              Luigi Bonati,

              Valerio Rizzi,

              Francesco L. Gervasio,

          </td>
          <td>2024-02-02</td>
          <td>None</td>
          <td>2</td>
          <td>11</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e1834d63294230b37fb4e0709b8d9bfca640d70c" target='_blank'>Deep clustering of protein folding simulations</a></td>
          <td>

              D. Bhowmik,

              Shang Gao,

              M. T. Young,

              A. Ramanathan,

          </td>
          <td>2018-06-12</td>
          <td>BMC Bioinformatics</td>
          <td>65</td>
          <td>45</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f3eae9b9dc159bd46e1669cf26cc5c71ae2f32f" target='_blank'>Using VAEs to Learn Latent Variables: Observations on Applications in cryo-EM</a></td>
          <td>

              Daniel G Edelberg,

              Roy R. Lederman,

          </td>
          <td>2023-03-13</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>14</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e4140d67d59c61de86d2a9de31380e01a07ef85b" target='_blank'>Past–future information bottleneck for sampling molecular reaction coordinate simultaneously with thermodynamics and kinetics</a></td>
          <td>

              Yihang Wang,

              João Marcelo Lamim Ribeiro,

              P. Tiwary,

          </td>
          <td>2019-08-08</td>
          <td>Nature Communications</td>
          <td>118</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d1516aeab1a504cc02237fdd5957c9b421498110" target='_blank'>Deep Learning Unravels a Dynamic Hierarchy While Empowering Molecular Dynamics Simulations</a></td>
          <td>

              Ariel Fernández,

          </td>
          <td>2020-02-13</td>
          <td>Annalen der Physik</td>
          <td>2</td>
          <td>29</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/21e8bbd2846539c180b901b747e579f82027d375" target='_blank'>Pre-training Protein Models with Molecular Dynamics Simulations for Drug Binding</a></td>
          <td>

              Fang Wu,

              Q. Zhang,

              Dragomir R. Radev,

              Yuyang Wang,

              Xurui Jin,

              Yinghui Jiang,

              Z. Niu,

              Stan Z. Li,

          </td>
          <td>None</td>
          <td>None</td>
          <td>4</td>
          <td>73</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dfec6b86a6080a722f96c73bc557169380204137" target='_blank'>DeepCV: A Deep Learning Framework for Blind Search of Collective Variables in Expanded Configurational Space</a></td>
          <td>

              R. Ketkaew,

              S. Luber,

          </td>
          <td>2022-11-29</td>
          <td>Journal of chemical information and modeling</td>
          <td>10</td>
          <td>28</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3999faeea4b6fc58615245072ce010a0e6f641ea" target='_blank'>Systematic control of collective variables learned from variational autoencoders.</a></td>
          <td>

              Jacob I Monroe,

              V. K. Shen,

          </td>
          <td>2022-08-09</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ec72a00ab118f77bf9161b0558dbc0866bf3a1b" target='_blank'>A Perspective on Deep Learning for Molecular Modeling and Simulations</a></td>
          <td>

              Jun Zhang,

              Yao-Kun Lei,

              Zhen Zhang,

              Junhan Chang,

              Maodong Li,

              Xu Han,

              Lijiang Yang,

              Yi Isaac Yang,

              Y. Gao,

          </td>
          <td>2020-04-25</td>
          <td>The journal of physical chemistry. B</td>
          <td>5</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aae6576b6f449e115f5c4e1aa3759593239f7b11" target='_blank'>A Perspective on Deep Learning for Molecular Modeling and Simulations.</a></td>
          <td>

              Jun Zhang,

              Yao-Kun Lei,

              Zhen Zhang,

              Junhan Chang,

              Maodong Li,

              Xu Han,

              Lijiang Yang,

              Yi Isaac Yang,

              Y. Gao,

          </td>
          <td>2020-07-14</td>
          <td>The journal of physical chemistry. A</td>
          <td>27</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4244b9b2515e37a20b254fa8f9ed7305f2f64d7e" target='_blank'>Interpretable embeddings from molecular simulations using Gaussian mixture variational autoencoders</a></td>
          <td>

              Yasemin Bozkurt Varolgunes,

              T. Bereau,

              J. F. Rudzinski,

          </td>
          <td>2019-12-22</td>
          <td>Machine Learning: Science and Technology</td>
          <td>36</td>
          <td>22</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c9561b15c25ca781ccf1a384e978e28341bac0e4" target='_blank'>State-specific protein–ligand complex structure prediction with a multiscale deep generative model</a></td>
          <td>

              Zhuoran Qiao,

              Weili Nie,

              Arash Vahdat,

              Thomas F. Miller,

              Anima Anandkumar,

          </td>
          <td>2022-09-30</td>
          <td>Nature Machine Intelligence</td>
          <td>16</td>
          <td>60</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/744249d8846dad1cccaeb835119c39c9a10f079b" target='_blank'>A deep encoder-decoder framework for identifying distinct ligand binding pathways.</a></td>
          <td>

              Satyabrata Bandyopadhyay,

              Jagannath Mondal,

          </td>
          <td>2023-05-15</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/510a041ffcc7dcccb23cf4a23f0992e5de6b8f9e" target='_blank'>Autoencoders for dimensionality reduction in molecular dynamics: Collective variable dimension, biasing, and transition states.</a></td>
          <td>

              Zineb Belkacemi,

              M. Bianciotto,

              H. Minoux,

              T. Lelièvre,

              G. Stoltz,

              P. Gkeka,

          </td>
          <td>2023-06-05</td>
          <td>The Journal of chemical physics</td>
          <td>3</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bbd38193f26f04e021bff620878b90ec0e91b23c" target='_blank'>MLCV: Bridging Machine-Learning-Based Dimensionality Reduction and Free-Energy Calculation</a></td>
          <td>

              Haochuan Chen,

              Han Liu,

              Heying Feng,

              Haohao Fu,

              W. Cai,

              Xueguang Shao,

              C. Chipot,

          </td>
          <td>2021-12-23</td>
          <td>Journal of chemical information and modeling</td>
          <td>16</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e89b8240b54e2beccb99d7f6c82ee369b4af91c4" target='_blank'>Learning generative models of molecular dynamics</a></td>
          <td>

              N. Razavian,

              Hetunandan Kamisetty,

              C. Langmead,

          </td>
          <td>2012-01-17</td>
          <td>BMC Genomics</td>
          <td>8</td>
          <td>45</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e89b8240b54e2beccb99d7f6c82ee369b4af91c4" target='_blank'>Learning generative models of molecular dynamics</a></td>
          <td>

              N. Razavian,

              Hetunandan Kamisetty,

              C. Langmead,

          </td>
          <td>2012-01-17</td>
          <td>BMC Genomics</td>
          <td>8</td>
          <td>45</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7f02647a9029d7f65da697a8bc943971c034695b" target='_blank'>P RO GAE: A G EOMETRIC A UTOENCODER - BASED G ENERATIVE M ODEL FOR D ISENTANGLING P ROTEIN C ONFORMATIONAL S PACE</a></td>
          <td>

          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ac055a76189bda2cd0c47998a74bcfc9bfa4a7cb" target='_blank'>Transferable deep generative modeling of intrinsically disordered protein conformations</a></td>
          <td>

              Giacomo Janson,

              M. Feig,

          </td>
          <td>2024-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/55705bec7256bb2f3ceec805823de4e87bee87f1" target='_blank'>Machine Learning for Molecular Dynamics on Long Timescales</a></td>
          <td>

              F. Noé,

          </td>
          <td>2018-12-18</td>
          <td>ArXiv</td>
          <td>30</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/01fafc7042cd2a1c9ec2d3832ffd6d7e0d93e9b6" target='_blank'>Multitask machine learning of collective variables for enhanced sampling of rare events</a></td>
          <td>

              Lixin Sun,

              Jonathan Vandermause,

              Simon L. Batzner,

              Yu Xie,

              David J. Clark,

              Wei Chen,

              B. Kozinsky,

          </td>
          <td>2020-12-07</td>
          <td>Journal of chemical theory and computation</td>
          <td>27</td>
          <td>36</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2b624accabd4b40bb341ba70038686c043efcace" target='_blank'>Multiensemble Markov models of molecular thermodynamics and kinetics</a></td>
          <td>

              Hao Wu,

              Fabian Paul,

              C. Wehmeyer,

              Frank Noé,

          </td>
          <td>2016-05-25</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>64</td>
          <td>15</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0215dd9f346534bf4c4247220501d7ab7d7715c6" target='_blank'>A Multi-Grained Symmetric Differential Equation Model for Learning Protein-Ligand Binding Dynamics</a></td>
          <td>

              Shengchao Liu,

              Weitao Du,

              Yanjing Li,

              Zhuoxinran Li,

              Vignesh C. Bhethanabotla,

              Nakul Rampal,

              O. Yaghi,

              C. Borgs,

              A. Anandkumar,

              Hongyu Guo,

              J. Chayes,

          </td>
          <td>2024-01-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c871d2dc802d276608a6734637f8bc9e6da0d837" target='_blank'>GeoDiff: a Geometric Diffusion Model for Molecular Conformation Generation</a></td>
          <td>

              Minkai Xu,

              Lantao Yu,

              Yang Song,

              Chence Shi,

              Stefano Ermon,

              Jian Tang,

          </td>
          <td>2022-03-06</td>
          <td>ArXiv</td>
          <td>310</td>
          <td>73</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3b97d1e21e19862a1372cdfc3a910e83d6c3354e" target='_blank'>Prediction of good reaction coordinates and future evolution of MD trajectories using Regularized Sparse Autoencoders: A novel deep learning approach</a></td>
          <td>

              Abhijit Gupta,

          </td>
          <td>2022-08-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/82ee174bc312d3960b865b0946ce9c86f1a10154" target='_blank'>Phanto-IDP: compact model for precise intrinsically disordered protein backbone generation and enhanced sampling</a></td>
          <td>

              Junjie Zhu,

              Zhengxin Li,

              Haowei Tong,

              Zhouyu Lu,

              Ningjie Zhang,

              Ting Wei,

              Hai-Feng Chen,

          </td>
          <td>2023-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b44d831a3e6e394d6db92139f4f2fba5bbe5b6f3" target='_blank'>GLOW: A Workflow Integrating Gaussian-Accelerated Molecular Dynamics and Deep Learning for Free Energy Profiling.</a></td>
          <td>

              H. Do,

              Jinan Wang,

              A. Bhattarai,

              Yinglong Miao,

          </td>
          <td>2022-02-24</td>
          <td>Journal of chemical theory and computation</td>
          <td>24</td>
          <td>34</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/31d62c95e128caa934ca6a6c5cd5c766c6a9d541" target='_blank'>Molecular dynamics without molecules: searching the conformational space of proteins with generative neural networks</a></td>
          <td>

              Gregory Schwing,

              L. Palese,

              Ariel Fern'andez,

              L. Schwiebert,

              D. Gatti,

          </td>
          <td>2022-06-09</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ea8ae9fda8336d6f288ee21ce947ed567c6dee3" target='_blank'>RPnet: A Reverse Projection Based Neural Network for Coarse-graining Metastable Conformational States for Protein Dynamics</a></td>
          <td>

              Hanlin Gu,

              Wei Wang,

              Siqin Cao,

              I. C. Unarta,

              Y. Yao,

              F. Sheong,

              Xuhui Huang,

          </td>
          <td>2021-08-06</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c89a40750265888d7f9c1c557675e1656108cf7e" target='_blank'>Machine learning heralding a new development phase in molecular dynamics simulations</a></td>
          <td>

              Eva Prasnikar,

              M. Ljubic,

              Andrej Perdih,

              J. Borišek,

          </td>
          <td>2024-03-29</td>
          <td>Artif. Intell. Rev.</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe16255bbbfbd74b63b4bbf72899069830ff27d3" target='_blank'>An Interpretable Convolutional Neural Network Framework for Analyzing Molecular Dynamics Trajectories: a Case Study on Functional States for G-Protein-Coupled Receptors</a></td>
          <td>

              Chuan Li,

              Jiangting Liu,

              Jianfang Chen,

              Yuan Yuan,

              Jin Yu,

              Qiaolin Gou,

              Yanzhi Guo,

              X. Pu,

          </td>
          <td>2022-03-08</td>
          <td>Journal of chemical information and modeling</td>
          <td>9</td>
          <td>23</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/464a4804060a4692e2d7a72be0042a2ea621e3b8" target='_blank'>LAST: Latent Space-Assisted Adaptive Sampling for Protein Trajectories</a></td>
          <td>

              Hao Tian,

              Xi Jiang,

              Sian Xiao,

              Hunter La Force,

              Eric C. Larson,

              Peng Tao,

          </td>
          <td>2022-04-27</td>
          <td>Journal of chemical information and modeling</td>
          <td>5</td>
          <td>23</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/355cb24f5e3dff4c50df682ece55fefa76d7bf0b" target='_blank'>Editorial: Machine Learning in Biomolecular Simulations</a></td>
          <td>

              Gennady M. Verkhivker,

              V. Spiwok,

              F. Gervasio,

          </td>
          <td>2019-08-29</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fc0aa1d384633d4b47abf854383faac21cf72a92" target='_blank'>Approximating solutions of the Chemical Master equation using neural networks</a></td>
          <td>

              A. Sukys,

              Kaan Öcal,

              R. Grima,

          </td>
          <td>2022-04-27</td>
          <td>iScience</td>
          <td>19</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/02a0504b526587d96e40d2b7d8d384385762ff2d" target='_blank'>PyEMMA 2: A Software Package for Estimation, Validation, and Analysis of Markov Models.</a></td>
          <td>

              Martin K. Scherer,

              Benjamin Trendelkamp-Schroer,

              Fabian Paul,

              G. Pérez-Hernández,

              Moritz Hoffmann,

              Nuria Plattner,

              C. Wehmeyer,

              Jan-Hendrik Prinz,

              F. Noé,

          </td>
          <td>2015-10-14</td>
          <td>Journal of chemical theory and computation</td>
          <td>813</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a2fcf672d2ac57fb45318b3351c3051df93895f5" target='_blank'>P RO GAE: A G EOMETRIC G ENERATIVE M ODEL FOR D ISENTANGLING P ROTEIN C ONFORMATIONAL S PACE</a></td>
          <td>

              N. Joseph Tatro,

              Payel Das,

              Pin-Yu Chen,

              V. Chenthamarakshan,

              Rongjie Lai,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9896835f050121358372254a9bce57ca117672c8" target='_blank'>Boltzmann Generators - Sampling Equilibrium States of Many-Body Systems with Deep Learning</a></td>
          <td>

              F. Noé,

              Hao Wu,

          </td>
          <td>2018-12-04</td>
          <td>ArXiv</td>
          <td>20</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1af26b2f66408bda852f8abfb9912b21ed075a19" target='_blank'>Introduction to Markov state modeling with the PyEMMA software [Article v1.0]</a></td>
          <td>

              C. Wehmeyer,

              Martin K. Scherer,

              Tim Hempel,

              B. Husic,

              S. Olsson,

              F. Noé,

          </td>
          <td>2018-11-01</td>
          <td>Living Journal of Computational Molecular Science</td>
          <td>44</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/12ba80c62d718cfee6f5dffa356178cbd408e397" target='_blank'>Predictive Collective Variable Discovery with Deep Bayesian Models</a></td>
          <td>

              M. Schöberl,

              N. Zabaras,

              P. Koutsourelakis,

          </td>
          <td>2018-09-18</td>
          <td>The Journal of chemical physics</td>
          <td>32</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f3ebef5245ffe1c8e4ccfa334c86cf2f0d9f4ed5" target='_blank'>An Introduction to Markov State Models and Their Application to Long Timescale Molecular Simulation</a></td>
          <td>

              G. Bowman,

              V. Pande,

              F. Noé,

          </td>
          <td>None</td>
          <td></td>
          <td>486</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/df7bc9ea5e0b1af5c7a675e6dfd9a6983f6d9d2e" target='_blank'>Supervised Learning on Markov States as a Method for Gaining Insight into Protein Allostery</a></td>
          <td>

              Mohammad M. Sultan,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/41cf570d66b907eb5b2446fceed6c9c97dc52f25" target='_blank'>Understanding Protein Dynamics with L1-Regularized Reversible Hidden Markov Models</a></td>
          <td>

              R. McGibbon,

              Bharath Ramsundar,

              Mohammad M. Sultan,

              G. Kiss,

              V. Pande,

          </td>
          <td>2014-05-06</td>
          <td>None</td>
          <td>19</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69ccf867833b3564c2d2f26c49915b444eadb268" target='_blank'>Transferable Neural Networks for Enhanced Sampling of Protein Dynamics.</a></td>
          <td>

              Mohammad M. Sultan,

              H. Wayment-Steele,

              V. Pande,

          </td>
          <td>2018-01-02</td>
          <td>Journal of chemical theory and computation</td>
          <td>92</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3f3ee15122e47f65ac0580ec8b00e9cedac75983" target='_blank'>Restricted Boltzmann machines : from compositional representations to protein sequence analysis</a></td>
          <td>

              J. Tubiana,

          </td>
          <td>2018-11-29</td>
          <td></td>
          <td>8</td>
          <td>11</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3d50464ec004730c9bb79f461b1eccf124d42e6d" target='_blank'>Toward Achieving Efficient and Accurate Ligand-Protein Unbinding with Deep Learning and Molecular Dynamics through RAVE.</a></td>
          <td>

              João Marcelo Lamim Ribeiro,

              P. Tiwary,

          </td>
          <td>2018-12-07</td>
          <td>Journal of chemical theory and computation</td>
          <td>56</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/446681b1a3b3cafc7522cffdc3550348480d4085" target='_blank'>Operator Autoencoders: Learning Physical Operations on Encoded Molecular Graphs</a></td>
          <td>

              Willis Hoke,

              D. Shea,

              S. Casey,

          </td>
          <td>2021-05-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a81be5b6594d95431aab852bfa568ef6123d667e" target='_blank'>FlexiDock: Compositional diffusion models for flexible molecular docking</a></td>
          <td>

              Zichen Wang,

              Balasubramaniam Srinivasan,

              Zhengyuan Shen,

              G. Karypis,

              H. Rangwala,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1f9c63e957adb671e52a8a4368dbab25faefc428" target='_blank'>Variational embedding of protein folding simulations using gaussian mixture variational autoencoders</a></td>
          <td>

              Mahdi Ghorbani,

              Samarjeet Prasad,

              Jeffery B. Klauda,

              B. Brooks,

          </td>
          <td>2021-08-27</td>
          <td>The Journal of chemical physics</td>
          <td>11</td>
          <td>64</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/67226aecde573785d39d59f100c0c183d2ed5292" target='_blank'>Analysing protein dynamics using machine learning based generative models</a></td>
          <td>

              Alexandra-Ioana Albu,

              G. Czibula,

          </td>
          <td>2020-05-01</td>
          <td>2020 IEEE 14th International Symposium on Applied Computational Intelligence and Informatics (SACI)</td>
          <td>2</td>
          <td>13</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7cb50b79a1eec42efd2fec611b9cdc5d08ca936c" target='_blank'>Machine learning transforms how microstates are sampled</a></td>
          <td>

              M. Tuckerman,

          </td>
          <td>2019-09-05</td>
          <td>Science</td>
          <td>10</td>
          <td>70</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/90e7dfe131b21c12c2998ec0b87010ac6eb4d337" target='_blank'>Two for One: Diffusion Models and Force Fields for Coarse-Grained Molecular Dynamics</a></td>
          <td>

              Marloes Arts,

              Victor Garcia Satorras,

              Chin-Wei Huang,

              Daniel Zuegner,

              M. Federici,

              C. Clementi,

              Frank No'e,

              Robert Pinsler,

              Rianne van den Berg,

          </td>
          <td>2023-02-01</td>
          <td>Journal of chemical theory and computation</td>
          <td>34</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/88a277bbc18ad0eed29ea02bef0219b1c1a111ab" target='_blank'>Variational encoding of complex dynamics.</a></td>
          <td>

              Carlos X. Hernández,

              H. Wayment-Steele,

              Mohammad M. Sultan,

              B. Husic,

              V. Pande,

          </td>
          <td>2017-11-23</td>
          <td>Physical review. E</td>
          <td>134</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c86006be8cd1c5ee6193ee31cf31d44449b9c082" target='_blank'>Molecular enhanced sampling with autoencoders: On‐the‐fly collective variable discovery and accelerated free energy landscape exploration</a></td>
          <td>

              Wei Chen,

              Andrew L. Ferguson,

          </td>
          <td>2017-12-30</td>
          <td>Journal of Computational Chemistry</td>
          <td>165</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/827ca2f16ded73b2df1f760eebb2f1c5c9e5b44b" target='_blank'>Timewarp: Transferable Acceleration of Molecular Dynamics by Learning Time-Coarsened Dynamics</a></td>
          <td>

              Leon Klein,

              Andrew Y. K. Foong,

              T. E. Fjelde,

              Bruno Mlodozeniec,

              Marc Brockschmidt,

              Sebastian Nowozin,

              Frank No'e,

              Ryota Tomioka,

          </td>
          <td>2023-02-02</td>
          <td>ArXiv</td>
          <td>15</td>
          <td>50</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0cd3637bf739405263a78687d2f1ddaaa801cd89" target='_blank'>Latent Space Simulator for Unveiling Molecular Free Energy Landscapes and Predicting Transition Dynamics</a></td>
          <td>

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/22bd75994b94a00cf2a9697a28a805e35da30b1f" target='_blank'>Reweighted autoencoded variational Bayes for enhanced sampling (RAVE).</a></td>
          <td>

              João Marcelo Lamim Ribeiro,

              P. Bravo,

              Yihang Wang,

              P. Tiwary,

          </td>
          <td>2018-02-09</td>
          <td>The Journal of chemical physics</td>
          <td>240</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69e96a63ffd74a6c816203329c8a54794ac1ca00" target='_blank'>Transferable Kinetic Monte Carlo Models with Thousands of Reactions Learned from Molecular Dynamics Simulations.</a></td>
          <td>

              Enze Chen,

              Qian Yang,

              Vincent Dufour-Décieux,

              C. Sing-Long,

              Rodrigo Freitas,

              E. Reed,

          </td>
          <td>2019-02-08</td>
          <td>The journal of physical chemistry. A</td>
          <td>8</td>
          <td>41</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/07b4a3910273a7122a5dedbced31ea5b877eb5b2" target='_blank'>Enhancing Conformational Sampling for Intrinsically Disordered and Ordered Proteins by Variational Autoencoder</a></td>
          <td>

              Junjie Zhu,

              Ningjie Zhang,

              Ting Wei,

              Haifeng Chen,

          </td>
          <td>2023-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>7</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0129beb25ff82f128116b35cdb45aff8f1456255" target='_blank'>Top-Down Machine Learning of Coarse-Grained Protein Force Fields</a></td>
          <td>

              Carles Navarro,

              Maciej Majewski,

              G. D. Fabritiis,

          </td>
          <td>2023-06-20</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>3</td>
          <td>28</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/728649fcdf03de6cbb023c2ecb6cd132019cbadf" target='_blank'>Long-time methods for molecular dynamics simulations: Markov State Models and Milestoning.</a></td>
          <td>

              Brajesh Narayan,

              Ye Yuan,

              A. Fathizadeh,

              R. Elber,

              Nicolae-Viorel Buchete,

          </td>
          <td>None</td>
          <td>Progress in molecular biology and translational science</td>
          <td>5</td>
          <td>55</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8bd7b6f622911c5f68e7dd19fc002074bde9722e" target='_blank'>Representation of Protein Dynamics Disentangled by Time-structure-based Prior</a></td>
          <td>

              Tsuyoshi Ishizone,

              Yasuhiro Matsunaga,

              Sotaro Fuchigami,

              Kazuyuki Nakamura,

          </td>
          <td>2023-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92da30c4137270487bc51fec1aaee97b17f4e2ac" target='_blank'>Machine Learning Generation of Dynamic Protein Conformational Ensembles</a></td>
          <td>

              Li-E Zheng,

              Shrishti Barethiya,

              Erik B. Nordquist,

              Jianhan Chen,

          </td>
          <td>2023-05-01</td>
          <td>Molecules</td>
          <td>5</td>
          <td>33</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78042c8517d29ca1890d02635ce75edfc4ce1c7f" target='_blank'>Deep Generative Model Driven Protein Folding Simulations</a></td>
          <td>

              Heng Ma,

              D. Bhowmik,

              Hyungro Lee,

              M. Turilli,

              M. T. Young,

              S. Jha,

              A. Ramanathan,

          </td>
          <td>2019-08-01</td>
          <td>None</td>
          <td>9</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/65aaf814b43994af4bb920982a2c496a011cb4a0" target='_blank'>MSMBuilder2: Modeling Conformational Dynamics at the Picosecond to Millisecond Scale.</a></td>
          <td>

              Kyle A. Beauchamp,

              G. Bowman,

              T. Lane,

              L. Maibaum,

              I. Haque,

              V. Pande,

          </td>
          <td>2011-10-11</td>
          <td>Journal of chemical theory and computation</td>
          <td>358</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/947f16a3a3eedf07cde5b787882d3d5ff2d46d8a" target='_blank'>Scalable Normalizing Flows Enable Boltzmann Generators for Macromolecules</a></td>
          <td>

              Joseph C. Kim,

              David Bloore,

              Karan Kapoor,

              Jun Feng,

              Ming-Hong Hao,

              Mengdi Wang,

          </td>
          <td>2024-01-08</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>1</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/eaad1865050f8213ecae404d5e8b014441b520fd" target='_blank'>Neural relational inference to learn long-range allosteric interactions in proteins from molecular dynamics simulations</a></td>
          <td>

              Jingxuan Zhu,

              Juexin Wang,

              Weiwei Han,

              Dong Xu,

          </td>
          <td>2022-03-29</td>
          <td>Nature Communications</td>
          <td>38</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/52d15f2e931259d2f9e5056ba80e34e5bcc40983" target='_blank'>Neural relational inference to learn long-range allosteric interactions in proteins from molecular dynamics simulations</a></td>
          <td>

              Jingxuan Zhu,

              Juexin Wang,

              Weiwei Han,

              Dong Xu,

          </td>
          <td>2022-03-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/868ee34bc7bda4ff9652fd9e739e093d6ad163c9" target='_blank'>Dynamic Molecular Graph-based Implementation for Biophysical Properties Prediction</a></td>
          <td>

              C. Knutson,

              G. Panapitiya,

              R. Varikoti,

              Neeraj Kumar,

          </td>
          <td>2022-12-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/76317336bf971d2006683e6dba463cd85154efff" target='_blank'>Learning Kinetic Distance Metrics for Markov State Models of Protein Conformational Dynamics.</a></td>
          <td>

              R. McGibbon,

              V. Pande,

          </td>
          <td>2013-06-06</td>
          <td>Journal of chemical theory and computation</td>
          <td>41</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fd11b7e8bbc5498f79c6478529c21604630abac4" target='_blank'>EMMA 1.3 Markov Model Algorithms Tutorial and Documentation</a></td>
          <td>

              Martin Senne,

              Benjamin Trendelkamp-Schroer,

              A. Mey,

              Frank Noé,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0187ec0ad130049a72e9cfc50c6160437c3fb2aa" target='_blank'>Understanding recent deep‐learning techniques for identifying collective variables of molecular dynamics</a></td>
          <td>

              W. Zhang,

              Christof Schutte,

          </td>
          <td>2023-07-01</td>
          <td>PAMM</td>
          <td>2</td>
          <td>25</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fadebc6c400cb504812b31e795fb8d33cbe05b06" target='_blank'>An Integrated Markov State Model and Path Metadynamics Approach to Characterize Drug Binding Processes.</a></td>
          <td>

              Mattia Bernetti,

              Matteo Masetti,

              M. Recanatini,

              Rommie E. Amaro,

              A. Cavalli,

          </td>
          <td>2019-09-05</td>
          <td>Journal of chemical theory and computation</td>
          <td>42</td>
          <td>73</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/755dc5e951f01bb6fd648fcebe84803584d54fb2" target='_blank'>Detection of molecular behavior that characterizes systems using a deep learning approach.</a></td>
          <td>

              Katsuhiro Endo,

              Daisuke Yuhara,

              Katsufumi Tomobe,

              K. Yasuoka,

          </td>
          <td>2019-05-23</td>
          <td>Nanoscale</td>
          <td>4</td>
          <td>40</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4cad5547c77b9243c5db342534e6536adb466fc0" target='_blank'>Linking time-series of single-molecule experiments with molecular dynamics simulations by machine learning</a></td>
          <td>

          </td>
          <td>2018-05-03</td>
          <td>eLife</td>
          <td>38</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/17c80d75ea490a12f54d166717e9d745f7129f8d" target='_blank'>Spectral Map: Embedding Slow Kinetics in Collective Variables</a></td>
          <td>

              J. Rydzewski,

          </td>
          <td>2023-06-01</td>
          <td>The Journal of Physical Chemistry Letters</td>
          <td>5</td>
          <td>9</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5b85e2ad38f6b262cc0267bf96d45dd93e587184" target='_blank'>Machine Learning in Molecular Simulations of Biomolecules</a></td>
          <td>

              Guan Xing-Yue,

              Huang Heng-Yan,

              Peng Hua-Qi,

              Liu Yan-Hang,

              Li Wen-Fei,

              Wang Wei,

          </td>
          <td>None</td>
          <td>Acta Physica Sinica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5303dffbc43db94c5acc804b87b76d84eb81dd13" target='_blank'>Reinforcement Learning for Multi-Scale Molecular Modeling</a></td>
          <td>

              Jun Zhang,

              Yao-Kun Lei,

              Yi Isaac Yang,

              Y. Gao,

          </td>
          <td>2019-08-19</td>
          <td>None</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f047a957ff17bc9c01c20c5e37723f5206e58e7a" target='_blank'>Data2Dynamics: a modeling environment tailored to parameter estimation in dynamical systems</a></td>
          <td>

              A. Raue,

              Bernhard Steiert,

              Max Schelker,

              C. Kreutz,

              T. Maiwald,

              Helge Hass,

              J. Vanlier,

              C. Tönsing,

              L. Adlung,

              R. Engesser,

              W. Mader,

              T. Heinemann,

              J. Hasenauer,

              M. Schilling,

              T. Höfer,

              E. Klipp,

              Fabian J Theis,

              U. Klingmüller,

              B. Schoeberl,

              J. Timmer,

          </td>
          <td>2015-11-01</td>
          <td>Bioinformatics</td>
          <td>199</td>
          <td>95</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6b506cab4e4bd23ef4d751e97592457a4dbb370c" target='_blank'>Recent advances in protein conformation sampling by combining machine learning with molecular simulation</a></td>
          <td>

              Yiming Tang,

              Zhongyuan Yang,

              Yifei Yao,

              Yun Zhou,

              Yuan Tan,

              Zichao Wang,

              Tong Pan,

              Rui Xiong,

              Junli Sun,

              Guanghong Wei,

          </td>
          <td>2024-01-04</td>
          <td>Chinese Physics B</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/98243acffb45975b0fe73763b41316c520a7037a" target='_blank'>Progress in deep Markov state modeling: Coarse graining and experimental data restraints.</a></td>
          <td>

              Andreas Mardt,

              Frank Noé,

          </td>
          <td>2021-08-04</td>
          <td>The Journal of chemical physics</td>
          <td>10</td>
          <td>7</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a86717a0af0e90830183b221a9473d235da96698" target='_blank'>Data-driven learning of the generalized Langevin equation with state-dependent memory</a></td>
          <td>

              Pei Ge,

              Zhongqiang Zhang,

              Huan Lei,

          </td>
          <td>2023-10-28</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/39fd60798d23d5514548f92fa881f520905f0bd0" target='_blank'>Learning Compositional Representations of Interacting Systems with Restricted Boltzmann Machines: Comparative Study of Lattice Proteins</a></td>
          <td>

              J. Tubiana,

              S. Cocco,

              R. Monasson,

          </td>
          <td>2019-02-18</td>
          <td>Neural Computation</td>
          <td>24</td>
          <td>40</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cd01f8a5a5aa4f735d47fe3761e43665e42f2784" target='_blank'>ViSNet: a scalable and accurate geometric deep learning potential for molecular dynamics simulation</a></td>
          <td>

              Yusong Wang,

              Shaoning Li,

              Xinheng He,

              Mingyu Li,

              Zun Wang,

              Nanning Zheng,

              Bin Shao,

              Tong Wang,

              Tie-Yan Liu,

          </td>
          <td>None</td>
          <td>None</td>
          <td>7</td>
          <td>12</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1af26b2f66408bda852f8abfb9912b21ed075a19" target='_blank'>Introduction to Markov state modeling with the PyEMMA software [Article v1.0]</a></td>
          <td>

              C. Wehmeyer,

              Martin K. Scherer,

              Tim Hempel,

              B. Husic,

              S. Olsson,

              F. Noé,

          </td>
          <td>2018-11-01</td>
          <td>Living Journal of Computational Molecular Science</td>
          <td>44</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6582d19655b187195965e9ba7c4d93b193b649d2" target='_blank'>ALGORITHMS FOR MOLECULAR KINETICS</a></td>
          <td>

              F. Qin,

          </td>
          <td>2011-04-18</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7410966762efc567ca6410e0b826201f0c330f70" target='_blank'>Efficient Parameter Estimation for DNA Kinetics Modeled as Continuous-Time Markov Chains</a></td>
          <td>

              Sedigheh Zolaktaf,

              Frits Dannenberg,

              E. Winfree,

              A. Bouchard-Côté,

              Mark W. Schmidt,

              A. Condon,

          </td>
          <td>2019-08-05</td>
          <td>None</td>
          <td>5</td>
          <td>58</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/732d462555dd5c9d6c639e89eb9a21e3d133bdc3" target='_blank'>DeepDriveMD: Deep-Learning Driven Adaptive Molecular Simulations for Protein Folding</a></td>
          <td>

              Hyungro Lee,

              Heng Ma,

              M. Turilli,

              D. Bhowmik,

              S. Jha,

              A. Ramanathan,

          </td>
          <td>2019-09-17</td>
          <td>2019 IEEE/ACM Third Workshop on Deep Learning on Supercomputers (DLS)</td>
          <td>54</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58257756dc1b2136850735a7b693f789d68680d2" target='_blank'>Deep Learning for Molecular Thermodynamics</a></td>
          <td>

              Hassaan Malik,

              Muhammad Umar Chaudhry,

              M. Jasinski,

          </td>
          <td>2022-12-09</td>
          <td>Energies</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca246ca0069caf6a95d3f8dbf6a4409fc380e2ea" target='_blank'>An Accurate Computation of a Physical Quantity of a Protein with a Markov State Model Constructed using a Manifold-Learning Technique</a></td>
          <td>

              Reika Ito,

              Takashi Yoshidome,

          </td>
          <td>2018-02-01</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5355e9d96db7298620b8ffa73a2f7b74f90c8d2e" target='_blank'>Protein Ensemble Generation Through Variational Autoencoder Latent Space Sampling</a></td>
          <td>

              Sanaa Mansoor,

              M. Baek,

              Hahnbeom Park,

              Gyu Rie Lee,

              D. Baker,

          </td>
          <td>2023-08-01</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>6</td>
          <td>31</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8ed31496eea24fb64e465cea58f1a0a4a92b4470" target='_blank'>Neural Langevin Dynamics: towards interpretable Neural Stochastic Differential Equations</a></td>
          <td>

              Simon Koop,

              M. Peletier,

              J. Portegies,

              V. Menkovski,

          </td>
          <td>2022-11-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/33b5a4bb1027bfc15dc04d3fcf6e609266fb5379" target='_blank'>Modeling of Dynamic Allostery in Proteins Enabled by Machine Learning</a></td>
          <td>

              Mohsen Botlani-Esfahani,

          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e70ed4cec2382a01413e6353cc8406cc0faa5993" target='_blank'>Markov State Models: To Optimize or Not to Optimize</a></td>
          <td>

              Robert E Arbon,

              Yanchen Zhu,

              A. Mey,

          </td>
          <td>2024-01-01</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dcebb0579b327d8250f37c013be13906f3c79978" target='_blank'>Diffusion-Driven Generative Framework for Molecular Conformation Prediction</a></td>
          <td>

              Bobin Yang,

              Jie Deng,

              Zhenghan Chen,

              Ruoxue Wu,

          </td>
          <td>2023-12-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0377d2e18facfaa8df108a6980a81fed5dde750b" target='_blank'>Top-down machine learning approach for high-throughput single-molecule analysis</a></td>
          <td>

              D. S. White,

              Marcel P. Goldschen-Ohm,

              R. Goldsmith,

              B. Chanda,

          </td>
          <td>2020-04-08</td>
          <td>eLife</td>
          <td>23</td>
          <td>32</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/471e083e9b938854d403b950378f6248b49d7e49" target='_blank'>A Machine Learning Approach for the Discovery of Ligand-Specific Functional Mechanisms of GPCRs</a></td>
          <td>

              Ambrose Plante,

              Derek M. Shore,

              G. Morra,

              G. Khelashvili,

              H. Weinstein,

          </td>
          <td>2019-04-22</td>
          <td>Molecules</td>
          <td>40</td>
          <td>71</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0a22a48825230b81221c5bfc0abb04594c7b7297" target='_blank'>Accelerating All-Atom Simulations and Gaining Mechanistic Understanding of Biophysical Systems through State Predictive Information Bottleneck.</a></td>
          <td>

              S. Mehdi,

              Dedi Wang,

              S. Pant,

              P. Tiwary,

          </td>
          <td>2021-12-21</td>
          <td>Journal of chemical theory and computation</td>
          <td>30</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5d310f49cd92b2b3a0c716592f6903fd9deb55c1" target='_blank'>Progress and challenges in the automated construction of Markov state models for full protein systems.</a></td>
          <td>

              G. Bowman,

              Kyle A. Beauchamp,

              G. Boxer,

              V. Pande,

          </td>
          <td>2009-09-28</td>
          <td>The Journal of chemical physics</td>
          <td>359</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8e7a1024ac46c43d3728f9d731c48d0311957a2e" target='_blank'>Generating Physically-Realistic Tertiary Protein Structures with Deep Latent Variable Models Learning Over Experimentally-available Structures</a></td>
          <td>

              F. Alam,

              Amarda Shehu,

          </td>
          <td>2021-12-09</td>
          <td>2021 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>2</td>
          <td>27</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6263c9498dc41e9f466478bda26996b3f3bbb70e" target='_blank'>Kinetic Machine Learning Unravels Ligand-Directed Conformational Change of μ Opioid Receptor</a></td>
          <td>

              Evan N. Feinberg,

              A. Farimani,

              Carlos X. Hernández,

              V. Pande,

          </td>
          <td>2017-07-31</td>
          <td>bioRxiv</td>
          <td>10</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6bec80cf9dcdd2719ff738a3ea91d14837882256" target='_blank'>Enhanced modeling via network theory: Adaptive sampling of Markov state models.</a></td>
          <td>

              G. Bowman,

              D. Ensign,

              V. Pande,

          </td>
          <td>2010-02-17</td>
          <td>Journal of chemical theory and computation</td>
          <td>207</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe220dded3b8540c19fdebe7dc95332e3db4f81e" target='_blank'>The bigger picture</a></td>
          <td>

              John Urbanic,

              Chris Olah,

          </td>
          <td>2021-11-15</td>
          <td>Nature</td>
          <td>13</td>
          <td>1</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f023a46dd0866693cc5f7986f18761dbeb26bb2" target='_blank'>Dynamics-Based Peptide-MHC Binding Optimization by a Convolutional Variational Autoencoder: A Use-Case Model for CASTELO.</a></td>
          <td>

              David R. Bell,

              Giacomo Domeniconi,

              Chih-Chieh Yang,

              R. Zhou,

              Leili Zhang,

              Guojing Cong,

          </td>
          <td>2020-11-29</td>
          <td>Journal of chemical theory and computation</td>
          <td>6</td>
          <td>73</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e88cac1fabc8c8633db4b8601ec3c4fbf56dd5d7" target='_blank'>Neural relational inference to learn allosteric long-range interactions in proteins from molecular dynamics simulations</a></td>
          <td>

              Jingxuan Zhu,

              Juexin Wang,

              Weiwei Han,

              Dong Xu,

          </td>
          <td>2021-01-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>67</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/390c937c44ffb10a9552d275fe8bf6d869b8a1cd" target='_blank'>Enhanced Sampling with Machine Learning: A Review</a></td>
          <td>

              S. Mehdi,

              Zachary Smith,

              Lukas Herron,

              Ziyue Zou,

              P. Tiwary,

          </td>
          <td>2023-06-15</td>
          <td>Annual review of physical chemistry</td>
          <td>7</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5f57848726a0ecc12bd0420e23a745109c404b84" target='_blank'>Physics-Inspired Protein Encoder Pre-Training via Siamese Sequence-Structure Diffusion Trajectory Prediction</a></td>
          <td>

              Zuobai Zhang,

              Minghao Xu,

              A. Lozano,

              V. Chenthamarakshan,

              Payel Das,

              Jian Tang,

          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>5</td>
          <td>33</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92e8942259e1b34172f8dbe5818575b43e89977e" target='_blank'>Concurrent parametrization against static and kinetic information leads to more robust coarse-grained force fields</a></td>
          <td>

              J. F. Rudzinski,

              T. Bereau,

          </td>
          <td>2016-07-18</td>
          <td>The European Physical Journal Special Topics</td>
          <td>17</td>
          <td>22</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9cbfa7ef13b37a3f14747ea914fbf6c1935fd48d" target='_blank'>Automatic state partitioning for multibody systems (APM): an efficient algorithm for constructing Markov state models to elucidate conformational dynamics of multibody systems.</a></td>
          <td>

              F. Sheong,

              Daniel‐Adriano Silva,

              Luming Meng,

              Yutong Zhao,

              Xuhui Huang,

          </td>
          <td>2015-01-13</td>
          <td>Journal of chemical theory and computation</td>
          <td>40</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d5a2ae8f201dd9f48062ad001f14514000acf359" target='_blank'>MSMBuilder: Statistical Models for Biomolecular Dynamics</a></td>
          <td>

              M. Harrigan,

              Mohammad M. Sultan,

              Carlos X. Hernández,

              B. Husic,

              P. Eastman,

              Christian Schwantes,

              Kyle A. Beauchamp,

              R. McGibbon,

              V. Pande,

          </td>
          <td>2016-10-30</td>
          <td>bioRxiv</td>
          <td>215</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7937fc2e974c74d7bed6b0b05ab4aacfeef7ca4b" target='_blank'>An extended autoencoder model for reaction coordinate discovery in rare event molecular dynamics datasets.</a></td>
          <td>

              M. Frassek,

              A. Arjun,

              P. Bolhuis,

          </td>
          <td>2021-08-14</td>
          <td>The Journal of chemical physics</td>
          <td>24</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/873545c5aa7506ae15e7045232b57b3dceaa62a7" target='_blank'>KINNTREX: a neural network to unveil protein mechanisms from time-resolved X-ray crystallography</a></td>
          <td>

              Gabriel Biener,

              T. Malla,

              P. Schwander,

              Marius Schmidt,

          </td>
          <td>2024-04-25</td>
          <td>IUCrJ</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/87dab6c203b173dc821f657190e91296a3dc8bea" target='_blank'>Data-driven prediction of αIIbβ3 integrin activation pathways using nonlinear manifold learning and deep generative modeling.</a></td>
          <td>

              Siva Dasetty,

              T. Bidone,

              Andrew L. Ferguson,

          </td>
          <td>2023-12-01</td>
          <td>Biophysical journal</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bf80771cb5426a41402f1bf5c6cdae6df2bef0c3" target='_blank'>End-to-End Differentiable , Hardware Accelerated , Molecular Dynamics in Pure Python</a></td>
          <td>

              S. Schoenholz,

          </td>
          <td>None</td>
          <td>None</td>
          <td>18</td>
          <td>38</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/de2d3e12be8f97504b6f4ee194e08df25a8d8ea9" target='_blank'>Computational methods for exploring protein conformations</a></td>
          <td>

              Jane R. Allison,

          </td>
          <td>2020-08-05</td>
          <td>Biochemical Society Transactions</td>
          <td>34</td>
          <td>20</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3f2e2970002f60ab6b34d8f3708cd5185b8363bf" target='_blank'>Rescuing off-equilibrium simulation data through dynamic experimental data with dynAMMo</a></td>
          <td>

              Christopher Kolloff,

              Simon Olsson,

          </td>
          <td>2023-11-08</td>
          <td>Machine Learning: Science and Technology</td>
          <td>3</td>
          <td>4</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6408d2f50d8b2ed2dd9446b1a97d32b29a76ef68" target='_blank'>Accurate Sampling of Macromolecular Conformations Using Adaptive Deep Learning and Coarse-Grained Representation</a></td>
          <td>

              Amr H. Mahmoud,

              Matthew Masters,

              Soo-Jung Lee,

              M. Lill,

          </td>
          <td>2022-03-30</td>
          <td>Journal of chemical information and modeling</td>
          <td>9</td>
          <td>32</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/028969f5b5135797ba0690b46aaa3042a7720f6a" target='_blank'>Deep Molecular Representation Learning via Fusing Physical and Chemical Information</a></td>
          <td>

              Shuwen Yang,

              Ziyao Li,

              Guojie Song,

              Lingsheng Cai,

          </td>
          <td>2021-11-28</td>
          <td>None</td>
          <td>18</td>
          <td>28</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/68b03ad0ed6c1c9ac968fa4c5ae66c3020dc783e" target='_blank'>Multiscale simulations of complex systems by learning their effective dynamics</a></td>
          <td>

              Pantelis R. Vlachas,

              G. Arampatzis,

              Caroline Uhler,

              P. Koumoutsakos,

          </td>
          <td>2020-06-24</td>
          <td>Nature Machine Intelligence</td>
          <td>87</td>
          <td>77</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a4be57f8248d031dae67def54a0f952e98105a41" target='_blank'>Machine learning dynamic correlation in chemical kinetics.</a></td>
          <td>

              C. Kim,

              Nathan D. Ricke,

              T. Van Voorhis,

          </td>
          <td>2021-08-05</td>
          <td>The Journal of chemical physics</td>
          <td>3</td>
          <td>55</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5885b18d331c869060129f479570f134c67f58ee" target='_blank'>Machine Learning of Biomolecular Reaction Coordinates.</a></td>
          <td>

              S. Brandt,

              F. Sittel,

              M. Ernst,

              G. Stock,

          </td>
          <td>2018-04-09</td>
          <td>The journal of physical chemistry letters</td>
          <td>58</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ac6b10fcb6869769d4f7c75a4ad8f46bba65e99f" target='_blank'>Learning Geometrically Disentangled Representations of Protein Folding Simulations</a></td>
          <td>

              N. Joseph Tatro,

              Payel Das,

              Pin-Yu Chen,

              V. Chenthamarakshan,

              Rongjie Lai,

          </td>
          <td>2022-05-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca17f2865ee2459f41c221164fb193502948693b" target='_blank'>An overview and practical guide to building Markov state models.</a></td>
          <td>

              G. Bowman,

          </td>
          <td>None</td>
          <td>Advances in experimental medicine and biology</td>
          <td>21</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4ee2669028bd6280d640b567f5f33dbf6e621b37" target='_blank'>An Integrated Machine Learning Approach Delineates Entropic Modulation of α -synuclein by Small Molecule</a></td>
          <td>

              Sneha Menon,

              ⊥. SubinoyAdhikari,

              Jagannath Mondal,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/268fb2625b2497d81079456b51f9f1a413d753fa" target='_blank'>Making high-dimensional molecular distribution functions tractable through Belief Propagation on Factor Graphs</a></td>
          <td>

              Zachary Smith,

              P. Tiwary,

          </td>
          <td>2021-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/073dc917746394b7edfab461afd3e36e605dc5ba" target='_blank'>May the Force be with You: Unified Force-Centric Pre-Training for 3D Molecular Conformations</a></td>
          <td>

              Rui Feng,

              Qi Zhu,

              Huan Tran,

              Binghong Chen,

              Aubrey Toland,

              R. Ramprasad,

              Chao Zhang,

          </td>
          <td>2023-08-24</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>57</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/33c04427b6fcd265da08f80fe8f279486f11d44a" target='_blank'>Machine learning of microscopic structure-dynamics relationships in complex molecular systems</a></td>
          <td>

              M. Crippa,

              Annalisa Cardellini,

              M. Cioni,

              Gábor Csányi,

              G. Pavan,

          </td>
          <td>2023-08-31</td>
          <td>Machine Learning: Science and Technology</td>
          <td>1</td>
          <td>65</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4afa2c93c386bc87c4fa4339709e2b788a6ed32a" target='_blank'>Everything you wanted to know about Markov State Models but were afraid to ask.</a></td>
          <td>

              V. Pande,

              Kyle A. Beauchamp,

              G. Bowman,

          </td>
          <td>2010-09-01</td>
          <td>Methods</td>
          <td>578</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ba4973004694ed2684b0ddc518ff47cf7fb79e83" target='_blank'>Deep-Learning-Assisted Enhanced Sampling for Exploring Molecular Conformational Changes.</a></td>
          <td>

              Haohao Fu,

              Han Liu,

              Jingya Xing,

              Tong Zhao,

              Xueguang Shao,

              W. Cai,

          </td>
          <td>2023-11-10</td>
          <td>The journal of physical chemistry. B</td>
          <td>3</td>
          <td>42</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d09f8cd520789fbd378a7bd90dcafc2544b6fcfc" target='_blank'>Simple synthetic molecular dynamics for efficient trajectory generation</a></td>
          <td>

              John D. Russo,

              D. Zuckerman,

          </td>
          <td>2022-04-09</td>
          <td>None</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3e2c07792a60338f29a5cfe012da2bdc99a69687" target='_blank'>A Machine Learning Approach to Solving Biochemical Systems</a></td>
          <td>

              Joshua Ologbonyo,

              Emmanuel O. Adara,

              R. Sidje,

          </td>
          <td>2024-02-05</td>
          <td>2024 IEEE First International Conference on Artificial Intelligence for Medicine, Health and Care (AIMHC)</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7415c2ae40e1a84b0fe47f94908474ec54af915d" target='_blank'>Deep Learning Dynamic Allostery of G-Protein-Coupled Receptors</a></td>
          <td>

              H. Do,

              Jinan Wang,

              Yinglong Miao,

          </td>
          <td>2023-01-18</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e12ebec869b56d7d0b265027509b9e644958daae" target='_blank'>Optimizing Physics-Informed Neural Network in Dynamic System Simulation and Learning of Parameters</a></td>
          <td>

              Ebenezer O. Oluwasakin,

              Abdul Q. M. Khaliq,

          </td>
          <td>2023-11-28</td>
          <td>Algorithms</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2c58f68b87493a209d9b56574f3c232296fdcbe3" target='_blank'>Combining experimental and simulation data of molecular processes via augmented Markov models</a></td>
          <td>

              S. Olsson,

              Hao Wu,

              Fabian Paul,

              C. Clementi,

              F. Noé,

          </td>
          <td>2017-07-17</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>77</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/37a5855400d64a3836c73321d25412a90535e04e" target='_blank'>Biomolecular dynamics with machine-learned quantum-mechanical force fields trained on diverse chemical fragments.</a></td>
          <td>

              Oliver T. Unke,

              M. Stöhr,

              Stefan Ganscha,

              Thomas Unterthiner,

              Hartmut Maennel,

              S. Kashubin,

              Daniel Ahlin,

              M. Gastegger,

              L. Medrano Sandonas,

              J. T. Berryman,

              Alexandre Tkatchenko,

              Klaus-Robert Müller,

          </td>
          <td>2024-04-05</td>
          <td>Science advances</td>
          <td>2</td>
          <td>24</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92081440bf32d43143a1304ec738e720ca4d2901" target='_blank'>D YNAMICS -I NFORMED P ROTEIN D ESIGN WITH S TRUCTURE C ONDITIONING</a></td>
          <td>

              Urszula Julia Komorowska,

              Simon V Mathis,

              Kieran Didi,

              Francisco Vargas,

              Pietro Lio,

              M. Jamnik,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/59de694649b05d03e65b89af1606cd33e5bc9624" target='_blank'>An efficient Bayesian kinetic lumping algorithm to identify metastable conformational states via Gibbs sampling.</a></td>
          <td>

              Wei Wang,

              Tong Liang,

              F. Sheong,

              Xiaodan Fan,

              Xuhui Huang,

          </td>
          <td>2018-08-21</td>
          <td>The Journal of chemical physics</td>
          <td>20</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/402717ce4e4339fd5b1e97c2cd9d41c5273f01d7" target='_blank'>Molecular relaxation by reverse diffusion with time step prediction</a></td>
          <td>

              Khaled Kahouli,

              Stefaan S. P. Hessmann,

              Klaus-Robert Muller,

              Shinichi Nakajima,

              Stefan Gugler,

              Niklas Wolf Andreas Gebauer,

          </td>
          <td>2024-04-16</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/712080b7b8d296e61827d6c738303ec6ed0d44c1" target='_blank'>Generation of conformational ensembles of small molecules via surrogate model-assisted molecular dynamics</a></td>
          <td>

              Juan Viguera Diez,

              Sara Romeo Atance,

              Ola Engkvist,

              Simon Olsson,

          </td>
          <td>2024-04-05</td>
          <td>Mach. Learn. Sci. Technol.</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/761320b0617a8925c391ae12c556cd463bc159c4" target='_blank'>Trajectory-based training enables protein simulations with accurate folding and Boltzmann ensembles in cpu-hours</a></td>
          <td>

              J. Jumper,

              N. Faruk,

              K. Freed,

              T. Sosnick,

          </td>
          <td>2018-12-01</td>
          <td>PLoS Computational Biology</td>
          <td>31</td>
          <td>66</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/25f9e33fae4be74e769b65524071e4fd60b3781c" target='_blank'>Machine-guided path sampling to discover mechanisms of molecular self-organization</a></td>
          <td>

              Hendrik Jung,

              Roberto Covino,

              A. Arjun,

              Christian Leitold,

              Christoph Dellago,

              P. Bolhuis,

              G. Hummer,

          </td>
          <td>2021-05-14</td>
          <td>Nature Computational Science</td>
          <td>42</td>
          <td>95</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/259c36d340e7e5a0d33b772faad981c64befce4b" target='_blank'>JAX, M.D.: End-to-End Differentiable, Hardware Accelerated, Molecular Dynamics in Pure Python</a></td>
          <td>

              S. Schoenholz,

              E. D. Cubuk,

          </td>
          <td>2019-12-09</td>
          <td>arXiv: Computational Physics</td>
          <td>34</td>
          <td>50</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/31739ea55a7c86ed38a1030bb99625ef96f9fc7a" target='_blank'>Manifold learning in atomistic simulations: a conceptual review</a></td>
          <td>

              J. Rydzewski,

              Ming Chen,

              O. Valsson,

          </td>
          <td>2023-03-15</td>
          <td>Machine Learning: Science and Technology</td>
          <td>7</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4275490b2442d0f4bfd56ec5f494e8c81543c0bb" target='_blank'>Past–future information bottleneck framework for simultaneously sampling biomolecular reaction coordinate, thermodynamics and kinetics</a></td>
          <td>

              Yihang Wang,

              João Marcelo Lamim Ribeiro,

              P. Tiwary,

          </td>
          <td>2018-12-28</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f7df8a93bf727598e0225a6c9cbbdc5463e139ba" target='_blank'>Predicting the protein-ligand affinity from molecular dynamics trajectories</a></td>
          <td>

              Yaosen Min,

              Ye Wei,

              Peizhuo Wang,

              N. Wu,

              Stefan Bauer,

              Shuxin Zheng,

              Yu Shi,

              Yingheng Wang,

              Dan Zhao,

              Ji Wu,

              Jianyang Zeng,

          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>29</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5711400d6fbba28fc776a3b85b6890b62934f64f" target='_blank'>End-to-end protein–ligand complex structure generation with diffusion-based generative models</a></td>
          <td>

              Shuya Nakata,

              Yoshiharu Mori,

              S. Tanaka,

          </td>
          <td>2022-12-21</td>
          <td>BMC Bioinformatics</td>
          <td>17</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c63657b2e54419a908221ea878df708d78a6409c" target='_blank'>Denoising diﬀusion probabilistic models for replica exchange</a></td>
          <td>

              Yihang Wang,

              P. Tiwary,

          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ddd8d63db17e6e6df42042054e360a18daf7e18b" target='_blank'>Tyrosine kinases: complex molecular systems challenging computational methodologies</a></td>
          <td>

              Trayder Thomas,

              B. Roux,

          </td>
          <td>2021-10-01</td>
          <td>The European Physical Journal B</td>
          <td>4</td>
          <td>117</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fa25e6dbbf460affab05c322decf7ef5fc1142f7" target='_blank'>Markov State Models Provide Insights into Dynamic Modulation of Protein Function</a></td>
          <td>

              D. Shukla,

              Carlos X. Hernández,

              J. Weber,

              V. Pande,

          </td>
          <td>2015-01-03</td>
          <td>Accounts of Chemical Research</td>
          <td>220</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f1748ef585758622ea82d64e4ba5674ec0636d2b" target='_blank'>Exploring the conformational ensembles of protein-protein complex with transformer-based generative model</a></td>
          <td>

              Jianmin Wang,

              Xun Wang,

              Yanyi Chu,

              Chunyan Li,

              Xue Li,

              Xiangyu Meng,

              Yitian Fang,

              Kyoung Tai No,

              Jiashun Mao,

              Xiangxiang Zeng,

          </td>
          <td>2024-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/48753fd45da00aa39a4a14841683158f73625d38" target='_blank'>SurfDock is a Surface-Informed Diffusion Generative Model for Reliable and Accurate Protein-ligand Complex Prediction</a></td>
          <td>

              Duanhua Cao,

              Mingan Chen,

              Runze Zhang,

              Jie Yu,

              Xinyu Jiang,

              Zhehuan Fan,

              Wei Zhang,

              Mingyue Zheng,

          </td>
          <td>2023-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/62c7155cc3c25d7b9a3790b5170bec9fbcb79356" target='_blank'>EMMA: A Software Package for Markov Model Building and Analysis.</a></td>
          <td>

              Martin Senne,

              Benjamin Trendelkamp-Schroer,

              A. Mey,

              C. Schütte,

              F. Noé,

          </td>
          <td>2012-06-18</td>
          <td>Journal of chemical theory and computation</td>
          <td>136</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/535803259d7b37e7f024c97dc15f48884bb6ba80" target='_blank'>Stochastic Lag Time Parameterization for Markov State Models of Protein Dynamics.</a></td>
          <td>

              Shiqi Gong,

              Xinheng He,

              Qi Meng,

              Zhiming Ma,

              Bin Shao,

              Tong Wang,

              Tie-Yan Liu,

          </td>
          <td>2022-11-08</td>
          <td>The journal of physical chemistry. B</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/240f93655c42fff4a40aa62c11dfde1248641126" target='_blank'>Efficient estimation of rare-event kinetics</a></td>
          <td>

              Benjamin Trendelkamp-Schroer,

              F. Noé,

          </td>
          <td>2014-09-23</td>
          <td>arXiv: Chemical Physics</td>
          <td>34</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6b8840350f5d4a41d678977f61aab85232c91bcf" target='_blank'>A Latent Diffusion Model for Protein Structure Generation</a></td>
          <td>

              Cong Fu,

              Keqiang Yan,

              Limei Wang,

              Wing Yee Au,

              Michael McThrow,

              Tao Komikado,

              Koji Maruhashi,

              Kanji Uchino,

              Xiaoning Qian,

              Shuiwang Ji,

          </td>
          <td>2023-05-06</td>
          <td>ArXiv</td>
          <td>10</td>
          <td>57</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4e3b2f677e03c624d9aed77a58fd596cebf452b3" target='_blank'>Sequence network 1 D 2 D Structure network Invariant features A C OrientationsDistances B weights features Energy</a></td>
          <td>

              John Ingraham,

              Adam J. Riesselman,

              C. Sander,

              D. Marks,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>152</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/731ef6c7b8761074d9c4b8d10bc9ded124bb00b0" target='_blank'>Employing artificial neural networks to find reaction coordinates and pathways for self-assembly</a></td>
          <td>

              Jörn H Appeldorn,

              Simon Lemcke,

              T. Speck,

              A. Nikoubashman,

          </td>
          <td>2021-08-31</td>
          <td>ChemRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/27fc4138289273c311dbfe6d26486521d7544396" target='_blank'>Toward a structural identification of metastable molecular conformations.</a></td>
          <td>

              Simon Lemcke,

              Jörn H Appeldorn,

              Michael Wand,

              Thomas Speck,

          </td>
          <td>2023-09-15</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3dbd0ff97315f3d70f5c0209068e394aa5476c6a" target='_blank'>Score Dynamics: Scaling Molecular Dynamics with Picoseconds Time Steps via Conditional Diffusion Model</a></td>
          <td>

              Tim Hsu,

              Babak Sadigh,

              Vasily V. Bulatov,

              Fei Zhou,

          </td>
          <td>2023-10-02</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bb3d0da51adfe51393e9952503f0722e7d4d1f2b" target='_blank'>Continuous Graphical Models for Static and Dynamic Distributions : Application to Structural Biology</a></td>
          <td>

              N. Razavian,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aaa66e8729fbd071781f23c4239eff9995054019" target='_blank'>Simulations meet machine learning in structural biology.</a></td>
          <td>

              Adrià Pérez,

              Gerard Martínez-Rosell,

              G. de Fabritiis,

          </td>
          <td>2018-04-01</td>
          <td>Current opinion in structural biology</td>
          <td>35</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/925edfff606df315acd5d2fa1db693dd4dbe91fa" target='_blank'>Automated detection of many-particle solvation states for accurate characterizations of diffusion kinetics.</a></td>
          <td>

              J. F. Rudzinski,

              M. Radu,

              T. Bereau,

          </td>
          <td>2018-10-08</td>
          <td>The Journal of chemical physics</td>
          <td>2</td>
          <td>22</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6bb5eeb7728656d61987b98aceb9cc4579a6f705" target='_blank'>A deep autoencoder framework for discovery of metastable ensembles in biomacromolecules.</a></td>
          <td>

              Satyabrata Bandyopadhyay,

              Jagannath Mondal,

          </td>
          <td>2021-06-01</td>
          <td>The Journal of chemical physics</td>
          <td>10</td>
          <td>25</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7e0c748c252fedd9b4aa01670c2069ae273a41cc" target='_blank'>Mapping the Ligand Binding Landscape</a></td>
          <td>

              Alex Dickson,

          </td>
          <td>2018-06-13</td>
          <td>Biophysical Journal</td>
          <td>38</td>
          <td>22</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d84e2baf19aa1a3d231bafb0143cf6cddf9c86d4" target='_blank'>Efficient Neural Network Models of Chemical Kinetics Using a Latent asinh Rate Transformation</a></td>
          <td>

              Felix A. Döppel,

              M. Votsmeier,

          </td>
          <td>None</td>
          <td>Reaction Chemistry &amp; Engineering</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f3eebe7cf446380258a42654978a9483e0325edf" target='_blank'>A Note on Learning Rare Events in Molecular Dynamics using LSTM and Transformer</a></td>
          <td>

              Wenqi Zeng,

              Siqin Cao,

              Xuhui Huang,

              Y. Yao,

          </td>
          <td>2021-07-14</td>
          <td>ArXiv</td>
          <td>9</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4229e3ba5f02221c22ef55e6be2825a08488d051" target='_blank'>SEEKR2: Versatile Multiscale Milestoning Utilizing the OpenMM Molecular Dynamics Engine</a></td>
          <td>

              L. Votapka,

              Andy Stokely,

              Anupam Anand Ojha,

              Rommie E. Amaro,

          </td>
          <td>2021-08-16</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>16</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f97964fd13dfccf136dfca52a9d9e4840e34211b" target='_blank'>Predicting new protein conformations from molecular dynamics simulation conformational landscapes and machine learning</a></td>
          <td>

              Yiming Jin,

              L. Johannissen,

              Sam Hay,

          </td>
          <td>2020-08-06</td>
          <td>Proteins: Structure</td>
          <td>20</td>
          <td>34</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf03b2599bbc48cf80eae312c95d8c5ebb45c125" target='_blank'>An Empirical Study of Graphormer on Large-Scale Molecular Modeling Datasets</a></td>
          <td>

              Yu Shi,

              Shuxin Zheng,

              Guolin Ke,

              Yifei Shen,

              Jiacheng You,

              ,

              Shengjie Luo,

              Chang Liu,

              Di He,

              Tie-Yan Liu,

          </td>
          <td>2022-02-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f327f32fb45ed0fde1b5fc0315f1da4c3da7a0f6" target='_blank'>Deciphering complex mechanisms of resistance and loss of potency through coupled molecular dynamics and machine learning</a></td>
          <td>

              F. Leidner,

              N. K. Yilmaz,

              C. Schiffer,

          </td>
          <td>2020-06-09</td>
          <td>bioRxiv</td>
          <td>8</td>
          <td>62</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6396ad9290c82a7ae1c7c138c9304e3ce62513f7" target='_blank'>Correction to “Exposing the Limitations of Molecular Machine Learning with Activity Cliffs”</a></td>
          <td>

              D. V. Tilborg,

              Alisa Alenicheva,

              F. Grisoni,

          </td>
          <td>2023-03-30</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c6f2f5df6f749df2fe4e7ad099b42d943e4d8a26" target='_blank'>Trajectory Statistical Learning of the Potential Mean of Force and Diffusion Coefficient from Molecular Dynamics Simulations.</a></td>
          <td>

              Yi-Tsao Chen,

              Haw Yang,

              J. Chu,

          </td>
          <td>2024-01-02</td>
          <td>The journal of physical chemistry. B</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/be563ac0b8eb00e23104b3dc11e8ab7144501065" target='_blank'>The prediction of protein–ligand unbinding for modern drug discovery</a></td>
          <td>

              Qianqian Zhang,

              Nannan Zhao,

              Xiaoxiao Meng,

              Fansen Yu,

              X. Yao,

              Huanxiang Liu,

          </td>
          <td>2021-11-03</td>
          <td>Expert Opinion on Drug Discovery</td>
          <td>8</td>
          <td>41</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/064ed7634c8ec92790accd4d1e107ebbb9d4e1ba" target='_blank'>Understanding what drives proteins</a></td>
          <td>

              V. Vinson,

          </td>
          <td>2020-11-27</td>
          <td>Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/125b2ac4050584a85eac86c9f85399f1ef85d71c" target='_blank'>Learning molecular dynamics with simple language model built upon long short-term memory neural network</a></td>
          <td>

              Sun-Ting Tsai,

              En-Jui Kuo,

              P. Tiwary,

          </td>
          <td>2020-04-26</td>
          <td>Nature Communications</td>
          <td>71</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1a3c895ac5d996424fac3a12011386c427195abf" target='_blank'>Development of Range-Corrected Deep Learning Potentials for Fast, Accurate Quantum Mechanical/Molecular Mechanical Simulations of Chemical Reactions in Solution.</a></td>
          <td>

              Jinzhe Zeng,

              Giese Tj,

              S. Ekesan,

              D. York,

          </td>
          <td>2021-02-26</td>
          <td>Journal of chemical theory and computation</td>
          <td>38</td>
          <td>49</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/18d50aee34266bbe7d77e3b50bd0c38f729c8437" target='_blank'>Molecular Free Energies, Rates, and Mechanisms from Data-Efficient Path Sampling Simulations</a></td>
          <td>

              Gianmarco Lazzeri,

              Hendrik Jung,

              P. Bolhuis,

              Roberto Covino,

          </td>
          <td>2023-07-20</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>7</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e3fd228d6b288db9b30f968697e345ab9954dc7e" target='_blank'>Navigating protein landscapes with a machine-learned transferable coarse-grained model</a></td>
          <td>

              N. Charron,

              Felix Musil,

              Andrea Guljas,

              Yaoyi Chen,

              Klara Bonneau,

              Aldo S. Pasos-Trejo,

              Jacopo Venturin,

              Daria Gusew,

              Iryna Zaporozhets,

              Andreas Kramer,

              Clark Templeton,

              Atharva S Kelkar,

              Aleksander E. P. Durumeric,

              Simon Olsson,

              Adria P'erez,

              Maciej Majewski,

              B. Husic,

              Ankit Patel,

              G. D. Fabritiis,

              Frank No'e,

              C. Clementi,

          </td>
          <td>2023-10-27</td>
          <td>None</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cfbdb4cb9ae6d79d0f41fc12ace856be86976cfc" target='_blank'>Enspara: Modeling molecular ensembles with scalable data structures and parallel computing</a></td>
          <td>

              J. Porter,

              Maxwell I. Zimmerman,

              Gregory R. Bowman,

          </td>
          <td>2018-09-29</td>
          <td>bioRxiv</td>
          <td>46</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/09b2d52317e91ab3757f8df4b1655d062ba330f8" target='_blank'>Neural networks learn the motions of molecular machines</a></td>
          <td>

              Timothy Grant,

          </td>
          <td>2021-07-29</td>
          <td>Nature Methods</td>
          <td>4</td>
          <td>4</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2a944e76958c0a1449a1110a2eb6df89301dc6ae" target='_blank'>A universal framework for accurate and efficient geometric deep learning of molecular systems</a></td>
          <td>

              Shuo Zhang,

              Yang Liu,

              Lei Xie,

          </td>
          <td>2023-11-06</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3502af36a980d8b99e8f462c2441006b98470df1" target='_blank'>Coarse-grained Markov chains capture molecular thermodynamics and kinetics in no uncertain terms</a></td>
          <td>

              G. Bowman,

          </td>
          <td>2012-01-18</td>
          <td>arXiv: Biomolecules</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/80d28e926b3cd25b9c298e82c404b20aa0b15789" target='_blank'>Embedded-physics machine learning for coarse-graining and collective variable discovery without data</a></td>
          <td>

              M. Schöberl,

              N. Zabaras,

              P. Koutsourelakis,

          </td>
          <td>2020-02-24</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d7bb3a1118c85c4a948df1eeb93a4dd93d92f49a" target='_blank'>Deciphering protein evolution and fitness landscapes with latent space models</a></td>
          <td>

              Xinqiang Ding,

              Zhengting Zou,

              Charles L. Brooks III,

          </td>
          <td>2019-12-01</td>
          <td>Nature Communications</td>
          <td>68</td>
          <td>11</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7562e5747ce73b36b57f51d53f7e25d6875ab57a" target='_blank'>Simulate Time-integrated Coarse-grained Molecular Dynamics with Geometric Machine Learning</a></td>
          <td>

              Xiang Fu,

              T. Xie,

              Nathan J. Rebello,

              B. Olsen,

              T. Jaakkola,

          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>14</td>
          <td>94</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aa1146c1db4b205be456323fe657ae54848f9dd4" target='_blank'>Building insightful, memory-enriched models to capture long-time biochemical processes from short-time simulations</a></td>
          <td>

              Anthony J. Dominic,

              T. Sayer,

              Siqin Cao,

              T. Markland,

              Xuhui Huang,

              Andrés Montoya-Castillo,

          </td>
          <td>2022-10-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>14</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3333f169bd4815f92c03c9445761d8267c4e9ef2" target='_blank'>Simulate Time-integrated Coarse-grained Molecular Dynamics with Multi-Scale Graph Networks</a></td>
          <td>

              Xiang Fu,

              T. Xie,

              Nathan J. Rebello,

              B. Olsen,

              T. Jaakkola,

          </td>
          <td>2022-04-21</td>
          <td>None</td>
          <td>3</td>
          <td>94</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/132df201f03dd6c0523c642d3cd7a4de6278e435" target='_blank'>Deep Autoencoders for Additional Insight into Protein Dynamics</a></td>
          <td>

              Mihai Teletin,

              G. Czibula,

              Maria-Iuliana Bocicor,

              S. Albert,

              A. Pandini,

          </td>
          <td>2018-10-04</td>
          <td>None</td>
          <td>8</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8d0a923ee5493276b887965020ecf1b4e80731c6" target='_blank'>A tutorial on building markov state models with MSMBuilder and coarse-graining them with BACE.</a></td>
          <td>

              G. Bowman,

          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>8</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f954c3e5262104a383fc64adcb57fb800d7ac221" target='_blank'>Deep Representation Learning for Complex Free Energy Landscapes.</a></td>
          <td>

              Jun Zhang,

              Yao-Kun Lei,

              X. Che,

              Zhen Zhang,

              Yi Isaac Yang,

              Y. Gao,

          </td>
          <td>2019-09-05</td>
          <td>The journal of physical chemistry letters</td>
          <td>14</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e753d1f39d086b536f0375a847057199f26dcc3f" target='_blank'>Nonlinear intrinsic variables and state reconstruction in multiscale simulations.</a></td>
          <td>

              Carmeline J. Dsilva,

              R. Talmon,

              N. Rabin,

              R. Coifman,

              I. Kevrekidis,

          </td>
          <td>2013-07-29</td>
          <td>The Journal of chemical physics</td>
          <td>40</td>
          <td>76</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/606590d67a28180297ad65ae3f067564553a5a33" target='_blank'>Discovery of multiple hidden allosteric sites by combining Markov state models and experiments</a></td>
          <td>

              G. Bowman,

              E. Bolin,

              Kathryn M Hart,

              B. Maguire,

              S. Marqusee,

          </td>
          <td>2015-02-17</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>164</td>
          <td>50</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a422712a543a43b3c64be2505aa8435333270b1d" target='_blank'>LagNet: Deep Lagrangian Mechanics for Plug-and-Play Molecular Representation Learning</a></td>
          <td>

              Chunyan Li,

              Junfeng Yao,

              Jinsong Su,

              Zhaoyang Liu,

              Xiangxiang Zeng,

              Chenxi Huang,

          </td>
          <td>2023-06-26</td>
          <td>None</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1f385d1785cc97103fc5e65b08969407ba348677" target='_blank'>Accurate Machine Learned Quantum-Mechanical Force Fields for Biomolecular Simulations</a></td>
          <td>

              Oliver T. Unke,

              M. Stohr,

              Stefan Ganscha,

              Thomas Unterthiner,

              Hartmut Maennel,

              S. Kashubin,

              Daniel Ahlin,

              M. Gastegger,

              L. M. Sandonas,

              A. Tkatchenko,

              Klaus-Robert Muller,

          </td>
          <td>2022-05-17</td>
          <td>ArXiv</td>
          <td>17</td>
          <td>76</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a252e00c006100bca62ee54174435b2283b886bb" target='_blank'>Coarse-Grained Modeling of the SARS-CoV-2 Spike Glycoprotein by Physics-Informed Machine Learning</a></td>
          <td>

              David Liang,

              Ziji Zhang,

              M. Rafailovich,

              Marcia Simon,

              Yuefan Deng,

              Peng Zhang,

          </td>
          <td>2023-02-02</td>
          <td>Comput.</td>
          <td>2</td>
          <td>20</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/936594a9115e6a3cac5e6208e294d32c6d030db7" target='_blank'>Barriers and Dynamical Paths in Alternating Gibbs Sampling of Restricted Boltzmann Machines</a></td>
          <td>

              C. Roussel,

              S. Cocco,

              R. Monasson,

          </td>
          <td>2021-07-13</td>
          <td>Physical review. E</td>
          <td>12</td>
          <td>40</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/490ad1648f79dea701d0e717977990c888c8bbc3" target='_blank'>DP Train, then DP Compress: Model Compression in Deep Potential Molecular Dynamics</a></td>
          <td>

              Denghui Lu,

              Wanrun Jiang,

              Yixiao Chen,

              Linfeng Zhang,

              Weile Jia,

              Han Wang,

              Mohan Chen,

          </td>
          <td>None</td>
          <td>None</td>
          <td>10</td>
          <td>29</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6941a3030881bb16393cb1b264aa3972b43a2bf8" target='_blank'>Scalable Multimer Structure Prediction using Diffusion Models</a></td>
          <td>

              Peter Pao-Huang,

              Bowen Jing,

              Bonnie Berger,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/678b1a05fb2df99b0c055bfaf9da61be0847c0c3" target='_blank'>Learning Small Molecule Energies and Interatomic Forces with an Equivariant Transformer on the ANI-1x Dataset</a></td>
          <td>

              Bryce Hedelius,

              F. Fuchs,

              Dennis Della Corte,

          </td>
          <td>2022-01-03</td>
          <td>None</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b20bd30a1274ac83242c2cbd74f452857a5f86e4" target='_blank'>Multiscale Reweighted Stochastic Embedding: Deep Learning of Collective Variables for Enhanced Sampling</a></td>
          <td>

              J. Rydzewski,

              O. Valsson,

          </td>
          <td>2020-07-13</td>
          <td>The Journal of Physical Chemistry. a</td>
          <td>24</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9a4400208b778f30ef77b2ba7600232f463dde59" target='_blank'>Markov State Models to Elucidate Ligand Binding Mechanism.</a></td>
          <td>

              Yunhui Ge,

              Vincent A. Voelz,

          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/06a090117138a4866b34f7debba950545eab2cce" target='_blank'>Stability-Aware Training of Neural Network Interatomic Potentials with Differentiable Boltzmann Estimators</a></td>
          <td>

              Sanjeev Raja,

              Ishan Amin,

              Fabian Pedregosa,

              Aditi S. Krishnapriyan,

          </td>
          <td>2024-02-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2f31bd26e2e34aa580344578af6c9e3c2151664c" target='_blank'>Themed collection on Insightful Machine Learning for Physical Chemistry.</a></td>
          <td>

              A. Clark,

              Pavlo O. Dral,

              I. Tamblyn,

              O. Isayev,

          </td>
          <td>2023-07-13</td>
          <td>Physical chemistry chemical physics : PCCP</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/923fab47b5c4e5a90219322ed0e7b1b7b94adc7b" target='_blank'>Variational approximation of molecular kinetics from short off-equilibrium simulations</a></td>
          <td>

              Hao Wu,

              F. Nüske,

              Fabian Paul,

              Stefan Klus,

              P. Koltai,

              F. Noé,

          </td>
          <td>2016-10-20</td>
          <td></td>
          <td>3</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2eb1ed9d4baf548391d8220c0a8a3d0be79845ed" target='_blank'>Boltzmann generators: Sampling equilibrium states of many-body systems with deep learning</a></td>
          <td>

              F. Noé,

              Jonas Köhler,

              Hao Wu,

          </td>
          <td>2019-09-05</td>
          <td>Science</td>
          <td>491</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4e2c97a0cd1db189787f1ed886ebd319fe13e770" target='_blank'>Hidden dynamics of proteasome autoregulation discovered by cryo-EM data-driven deep learning</a></td>
          <td>

              Zhaolong Wu,

              Enbo Chen,

              Shuwen Zhang,

              Wei Wang,

              Yinping Ma,

              Yuanchen Dong,

              Congcong Liu,

              C. Yin,

              Youdong Mao,

          </td>
          <td>2022-03-03</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/06b0a8c66ba5a4da7363116e7f4d65f1703d7e31" target='_blank'>Deep learning the slow modes for rare events sampling</a></td>
          <td>

              L. Bonati,

              G. Piccini,

              M. Parrinello,

          </td>
          <td>2021-07-08</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>107</td>
          <td>123</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bccfbf2a9a02fa3e2a99f95a1fdae7a0f7d4e6c3" target='_blank'>Implicit Transfer Operator Learning: Multiple Time-Resolution Surrogates for Molecular Dynamics</a></td>
          <td>

              M. Schreiner,

              O. Winther,

              Simon Olsson,

          </td>
          <td>2023-05-29</td>
          <td>None</td>
          <td>4</td>
          <td>48</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2f534901ac9b3616de5581fea82789afbb22722b" target='_blank'>Unsupervised Protein-Ligand Binding Energy Prediction via Neural Euler's Rotation Equation</a></td>
          <td>

              Wengong Jin,

              Siranush Sarkizova,

              Xun Chen,

              N. Hacohen,

              Caroline Uhler,

          </td>
          <td>2023-01-25</td>
          <td>ArXiv</td>
          <td>8</td>
          <td>105</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c2f962fea9a69cd42c93c85f1c01078c2232358e" target='_blank'>Application of Molecular-Dynamics Based Markov State Models to Functional Proteins</a></td>
          <td>

              R. D. Malmstrom,

              Christopher T. Lee,

              Adam T. Van Wart,

              Rommie E. Amaro,

          </td>
          <td>2014-06-04</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>87</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a6e2e1189e3f888d1a01ca535ff9b3dad560b6d0" target='_blank'>High-fidelity molecular dynamics trajectory reconstruction with bi-directional neural networks</a></td>
          <td>

              Ludwig Winkler,

              Klaus-Robert Müller,

              H. E. Sauceda,

          </td>
          <td>2022-01-02</td>
          <td>Machine Learning: Science and Technology</td>
          <td>8</td>
          <td>18</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/042e1fc051725f52a427c68f5af4309e122b7591" target='_blank'>Artificial Intelligence Resolves Kinetic Pathways of Magnesium Binding to RNA</a></td>
          <td>

              Jan Neumann,

              N. Schwierz,

          </td>
          <td>2021-07-26</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>8</td>
          <td>17</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4b20dc3bd249bc43b97a1efdd513a589d68c38fa" target='_blank'>Deep Molecular Programming: A Natural Implementation of Binary-Weight ReLU Neural Networks</a></td>
          <td>

              Marko Vasić,

              Cameron T. Chalk,

              S. Khurshid,

              D. Soloveichik,

          </td>
          <td>2020-03-30</td>
          <td>None</td>
          <td>16</td>
          <td>49</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0a76e5107737cd3017a880f52b91ccade4504bf3" target='_blank'>DockGame: Cooperative Games for Multimeric Rigid Protein Docking</a></td>
          <td>

              Vignesh Ram Somnath,

              Pier Giuseppe Sessa,

              Maria Rodriguez Martinez,

              Andreas Krause,

          </td>
          <td>2023-10-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1e14c666775a9b4b52ebc55c31f5c80b38915acc" target='_blank'>Analysing ConformationalEnsembles using Machine Learning</a></td>
          <td>

              Christina M. Friberg,

          </td>
          <td>None</td>
          <td></td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f6c635ccce0c3ed7b229e36364f7f4494e8b666c" target='_blank'>Metastability and Markov State Models in Molecular Dynamics: Modeling, Analysis, Algorithmic Approaches</a></td>
          <td>

              Christof Schütte,

              M. Sarich,

          </td>
          <td>2013-12-26</td>
          <td></td>
          <td>8</td>
          <td>12</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dbc2cc6e14a0fa04879764a9cd7fc1d4242be3cc" target='_blank'>Machine-learned molecular mechanics force field for the simulation of protein-ligand systems and beyond</a></td>
          <td>

              K. Takaba,

              Iván Pulido,

              P. Behara,

              Chapin E. Cavender,

              Anika J. Friedman,

              Mike Henry,

              Hugo MacDermott Opeskin,

              C. Iacovella,

              Arnav M. Nagle,

              A. M. Payne,

              Michael R. Shirts,

              David L. Mobley,

              J. Chodera,

              Yuanqing Wang,

          </td>
          <td>2023-07-13</td>
          <td>None</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bf136065e0aed4271f8ad3ff7e4e3305e9ba03c6" target='_blank'>Statistical model selection for Markov models of biomolecular dynamics.</a></td>
          <td>

              R. McGibbon,

              Christian Schwantes,

              V. Pande,

          </td>
          <td>2014-06-19</td>
          <td>The journal of physical chemistry. B</td>
          <td>23</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3c64b1d8a81ab02c1b4c097cf3a0329df20c1840" target='_blank'>Dynamic graphical models of molecular kinetics</a></td>
          <td>

              S. Olsson,

              F. Noé,

          </td>
          <td>2018-11-09</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>26</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/66cdb2c89bb00dada7fbc6a1ac28661dbe6e5158" target='_blank'>Towards equilibrium molecular conformation generation with GFlowNets</a></td>
          <td>

              Alexandra Volokhova,

              Michal Koziarski,

              Alex Hernández-García,

              Cheng-Hao Liu,

              Santiago Miret,

              Pablo Lemos,

              Luca Thiede,

              Zichao Yan,

              Alán Aspuru-Guzik,

              Y. Bengio,

          </td>
          <td>2023-10-20</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dcbcc1b8405ce2fb72d00243e8be343de9bcb977" target='_blank'>adabmDCA: adaptive Boltzmann machine learning for biological sequences</a></td>
          <td>

              A. Muntoni,

              A. Pagnani,

              M. Weigt,

              F. Zamponi,

          </td>
          <td>2021-09-09</td>
          <td>BMC Bioinformatics</td>
          <td>15</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a9730366151ae58da7d1ede9f55244501c397e11" target='_blank'>Towards Finding a Better Reaction Coordinate: NtrC and DIMS</a></td>
          <td>

              J. Perilla,

              T. Woolf,

          </td>
          <td>2011-02-02</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c046774b24be185a9fa47a538df0b23089015e44" target='_blank'>DROIDS 3.0—Detecting Genetic and Drug Class Variant Impact on Conserved Protein Binding Dynamics</a></td>
          <td>

              G. Babbitt,

              Ernest Fokoue,

              Joshua R. Evans,

              Kyle I. Diller,

              Lily E. Adams,

          </td>
          <td>2019-12-18</td>
          <td>Biophysical Journal</td>
          <td>19</td>
          <td>13</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c70ff8d7903100cfb6fe14622b254c640f58fce6" target='_blank'>models : Protein dynamics Chemical Theory and Computation Special Feature : Comparative study of generalized Born</a></td>
          <td>

              Hao Fan,

              A. Mark,

              Jiang Zhu,

              B. Honig,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3f698eea76327bdbdffc10a3e2fa124c75609c3f" target='_blank'>Growing ecosystem of deep learning methods for modeling protein-protein interactions</a></td>
          <td>

              Julia R. Rogers,

              Gergo Nikolényi,

              Mohammed Alquraishi,

          </td>
          <td>2023-10-10</td>
          <td>Protein engineering, design & selection : PEDS</td>
          <td>2</td>
          <td>17</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6e6b7ff27cfd2baf49181ef5f4f782328b38f8f0" target='_blank'>Tutorial on how to build non-Markovian dynamic models from molecular dynamics simulations for studying protein conformational changes.</a></td>
          <td>

              Yue Wu,

              Siqin Cao,

              Yunrui Qiu,

              Xuhui Huang,

          </td>
          <td>2024-03-22</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dfd128998d8f69beb10efe0ea2ecdc7aa8819551" target='_blank'>Predicting mutational effects on protein-protein binding via a side-chain diffusion probabilistic model</a></td>
          <td>

              Shiwei Liu,

              Tian Zhu,

              Milong Ren,

              Chungong Yu,

              Dongbo Bu,

              Haicang Zhang,

          </td>
          <td>2023-10-30</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>11</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3eed492986e7a0c03362118daead192076d46446" target='_blank'>Supervised machine learning approach to molecular dynamics forecast of SARS-CoV-2 spike glycoproteins at varying temperatures</a></td>
          <td>

              David Liang,

              Meichen Song,

              Ziyuan Niu,

              Peng Zhang,

              M. Rafailovich,

              Yuefan Deng,

          </td>
          <td>2021-02-17</td>
          <td>Mrs Advances</td>
          <td>8</td>
          <td>60</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9cc9740a1c791664183a08d1a836a3def87bb580" target='_blank'>Deep Neural Network-Based Simulation of Sel’kov Model in Glycolysis: A Comprehensive Analysis</a></td>
          <td>

              Jamshaid Ul Rahman,

              Sana Danish,

              D. Lu,

          </td>
          <td>2023-07-21</td>
          <td>Mathematics</td>
          <td>1</td>
          <td>52</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fab22c08f44bd3e71bc45b0e87155df0f3f4e691" target='_blank'>Extraction of protein dynamics information from cryo-EM maps using deep learning</a></td>
          <td>

              S. Matsumoto,

              Shoichi Ishida,

              M. Araki,

              Takayuki Kato,

              Kei Terayama,

              Y. Okuno,

          </td>
          <td>2021-02-01</td>
          <td>Nature Machine Intelligence</td>
          <td>49</td>
          <td>45</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/df7b1ea2a9d6174ce83a0b04a709e2e5ea0a89e7" target='_blank'>Independent Markov decomposition: Toward modeling kinetics of biomolecular complexes</a></td>
          <td>

              Tim Hempel,

              Mauricio J. del Razo,

              Christopher T. Lee,

              Bryn C. Taylor,

              Rommie E. Amaro,

              F. Noé,

          </td>
          <td>2021-03-25</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>15</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e9cf45ce86339359aaee5f9c5e904a4c7ac0b68b" target='_blank'>Active learning of neural network potentials for rare events</a></td>
          <td>

              Gang Seob Jung,

              Jong Youl Choi,

              S. Lee,

          </td>
          <td>None</td>
          <td>Digital Discovery</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/38fc5852128337711554595ecf8de1cf66afe82e" target='_blank'>Caliber Corrected Markov Modeling (C2M2): Correcting Equilibrium Markov Models.</a></td>
          <td>

              P. Dixit,

              K. Dill,

          </td>
          <td>2017-11-08</td>
          <td>Journal of chemical theory and computation</td>
          <td>25</td>
          <td>92</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/385f4b919171f35824f45469a4de56cc915b7970" target='_blank'>Learning Large-Time-Step Molecular Dynamics with Graph Neural Networks</a></td>
          <td>

              Tian Zheng,

              Weihao Gao,

              Chong Wang,

          </td>
          <td>2021-11-30</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>14</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe3850f70016ae3a31767166fb996ed0f0325fd4" target='_blank'>Capabilities and Limitations of Time-lagged Autoencoders for Slow Mode Discovery in Dynamical Systems</a></td>
          <td>

              ,

              Hythem Sidky,

              Andrew L. Ferguson,

          </td>
          <td>2019-06-02</td>
          <td>ArXiv</td>
          <td>31</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/03f062024f5212a0cdb048a927ec0c5aae2fd757" target='_blank'>Extract Latent Features of Single Particle Trajectories with Historical Experience Learning.</a></td>
          <td>

              Yongyu Zhang,

              Feng Ge,

              Xijian Lin,

              Jianfeng Xue,

              Yuxin Song,

              Hao Xie,

              Yan He,

          </td>
          <td>2023-10-01</td>
          <td>Biophysical journal</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1a14185615a3d142f5205edbb2535f74d4f4575b" target='_blank'>Generating, Reconstructing, and Representing Discrete and Continuous Data: Generalized Diffusion with Learnable Encoding-Decoding</a></td>
          <td>

              Guangyi Liu,

              Yu Wang,

              Zeyu Feng,

              Qiyu Wu,

              Liping Tang,

              Yuan Gao,

              Zhen Li,

              Shuguang Cui,

              Julian McAuley,

              Eric P. Xing,

              Zichao Yang,

              Zhiting Hu,

          </td>
          <td>2024-02-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e8445fd57ba138f1fef0695f7257786bf0e47c67" target='_blank'>Correction: Elucidation of molecular kinetic schemes from macroscopic traces using system identification</a></td>
          <td>

              M. Fribourg,

              D. Logothetis,

              J. González-Maeso,

              S. Sealfon,

              Belén Galocha-Iragüen,

              F. Andrés,

              V. Brezina,

          </td>
          <td>2018-10-01</td>
          <td>PLoS Computational Biology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/313cb2cfec8d36aac197681cf500128b8c2e8b19" target='_blank'>A dual diffusion model enables 3D molecule generation and lead optimization based on target pockets</a></td>
          <td>

              Lei Huang,

              Tingyang Xu,

              Yang Yu,

              Peilin Zhao,

              Xingjian Chen,

              Jing Han,

              Zhi Xie,

              Hailong Li,

              Wenge Zhong,

              Ka-Chun Wong,

              Hengtong Zhang,

          </td>
          <td>2024-03-26</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f74efb7f8b18ac1c6bdbafc5a6966e2917624a7b" target='_blank'>Machine Learning of Coarse-Grained Molecular Dynamics Force Fields</a></td>
          <td>

              Jiang Wang,

              C. Wehmeyer,

              F. Noé,

              C. Clementi,

          </td>
          <td>2018-12-04</td>
          <td>ACS Central Science</td>
          <td>326</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3db6ec823a29d80d23cb195c8223fc01161fc469" target='_blank'>Grappa -- A Machine Learned Molecular Mechanics Force Field</a></td>
          <td>

              Leif Seute,

              Eric Hartmann,

              Jan Stuhmer,

              Frauke Grater,

          </td>
          <td>2024-03-25</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1e7e3c4408c0a73430e8904c8ea4d65dd5814e3e" target='_blank'>Comparative Study and Analysis of Methods and Models</a></td>
          <td>

              D. Kulasiri,

              R. Kosarwal,

          </td>
          <td>None</td>
          <td>Chemical Master Equation for Large Biological Networks</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7cd70c6493caeafb8ff9be638229a14f878c8d97" target='_blank'>Analyzing Multimodal Probability Measures with Autoencoders.</a></td>
          <td>

              Tony Lelièvre,

              Thomas Pigeon,

              G. Stoltz,

              Wei Zhang,

          </td>
          <td>2023-10-05</td>
          <td>The journal of physical chemistry. B</td>
          <td>2</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5e660b3335f6a2a1d8592c6bfd048b1d1aaa6021" target='_blank'>Learning minimal representations of stochastic processes with variational autoencoders</a></td>
          <td>

              Gabriel Fern'andez-Fern'andez,

              Carlo Manzo,

              M. Lewenstein,

              A. Dauphin,

              Gorka Muñoz-Gil,

          </td>
          <td>2023-07-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c47e8168158a85f81d39dc57387b35d934514c6e" target='_blank'>Latent Models of Molecular Dynamics Data: Automatic Order Parameter Generation for Peptide Fibrillization.</a></td>
          <td>

              Nathaniel Charest,

              M. Tro,

              M. Bowers,

              J. Shea,

          </td>
          <td>2020-08-11</td>
          <td>The journal of physical chemistry. B</td>
          <td>7</td>
          <td>80</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/56de4646fd7409be7ec212ab6dba058b4a25586f" target='_blank'>Extraction of Protein Dynamics Information Hidden in Cryogenic Electron Microscopy Maps Using Deep Learning</a></td>
          <td>

              S. Matsumoto,

              Shoichi Ishida,

              M. Araki,

              Takayuki Kato,

              Kei Terayama,

              Y. Okuno,

          </td>
          <td>2020-02-18</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>45</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7811dc0675cff432796f6b09a8d1a40df28026dd" target='_blank'>Effective Molecular Dynamics from Neural-Network Based Structure Prediction Models</a></td>
          <td>

              Alexander Jussupow,

              Ville R. I. Kaila,

          </td>
          <td>2022-10-18</td>
          <td>bioRxiv</td>
          <td>11</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e58139737979e1ba99fbce12a47a86edae958418" target='_blank'>NNP/MM: Accelerating Molecular Dynamics Simulations with Machine Learning Potentials and Molecular Mechanics</a></td>
          <td>

              Raimondas Galvelis,

              Alejandro Varela-Rial,

              Stefan Doerr,

              R. Fino,

              P. Eastman,

              T. Markland,

              J. Chodera,

              G. D. Fabritiis,

          </td>
          <td>2022-01-20</td>
          <td>Journal of chemical information and modeling</td>
          <td>12</td>
          <td>55</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf6a85689c63810951d5f5efc1d488a1e8a1c96d" target='_blank'>Leveraging attention-enhanced variational autoencoders: Novel approach for investigating latent space of aptamer sequences.</a></td>
          <td>

              Abbas Salimi,

              Jee Hwan Jang,

              Jin Yong Lee,

          </td>
          <td>2023-11-01</td>
          <td>International journal of biological macromolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/853b4194775c1a5ed68a8c29a4f1976c74a4f66d" target='_blank'>ACE: adaptive cluster expansion for maximum entropy graphical model inference</a></td>
          <td>

              J. Barton,

              E. D. Leonardis,

              A. Coucke,

              S. Cocco,

          </td>
          <td>2016-03-18</td>
          <td>bioRxiv</td>
          <td>78</td>
          <td>31</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/426fa8decdfaffbe7d9299b400d669750c02a662" target='_blank'>: Extraction of Protein Dynamics Information Hidden in Cryo-EM Map Using Deep Learning</a></td>
          <td>

              S. Matsumoto,

              Shoichi Ishida,

              M. Araki,

              Takayuki Kato,

              Kei Terayama,

              Yasushi Okuno,

          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a4ac048edff84a0d87a14943042f0edf46316da4" target='_blank'>Bayesian Abstraction of Markov Population Models</a></td>
          <td>

              L. Bortolussi,

              Francesca Cairoli,

          </td>
          <td>2019-09-10</td>
          <td>None</td>
          <td>4</td>
          <td>27</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dec1deaaa4969250ae7c43206dee2cdea0958952" target='_blank'>Modelling the Dynamics of Biological Systems with the Geometric Hidden Markov Model</a></td>
          <td>

              Borislav Vangelov,

              Mauricio Barahona,

          </td>
          <td>2017-11-22</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>39</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7cf2f104088f930ae07cfedfca9fe09690024f5d" target='_blank'>Efficient Deep Molecular Dynamic Model Training on Heterogeneous System</a></td>
          <td>

              Shaokang Du,

              Xin You,

              Hailong Yang,

              Jing Shang,

              Zhiwen Xiao,

              Zhihui Wu,

              Zhongzhi Luan,

              Depei Qian,

          </td>
          <td>2023-12-17</td>
          <td>2023 IEEE 29th International Conference on Parallel and Distributed Systems (ICPADS)</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dc873c5cc7dfa9f6e565094c8c63ad4c56abf907" target='_blank'>Towards simple kinetic models of functional dynamics for a kinase subfamily</a></td>
          <td>

              Mohammad M. Sultan,

              G. Kiss,

              V. Pande,

          </td>
          <td>2017-12-03</td>
          <td>Nature Chemistry</td>
          <td>51</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/48ce3fc51e894a4867b0fd7c989b94e784578e68" target='_blank'>Trajectory-based machine learning method and its application to molecular dynamics</a></td>
          <td>

              R. Han,

              Sandra Luber,

          </td>
          <td>2020-07-04</td>
          <td>Molecular Physics</td>
          <td>3</td>
          <td>3</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/377b191510fe23f3bc1728dff00ed3db55852efb" target='_blank'>RiteWeight: Randomized Iterative Trajectory Reweighting for Steady-State Distributions Without Discretization Error</a></td>
          <td>

              Sagar Kania,

              D. Aristoff,

              Daniel M. Zuckerman,

          </td>
          <td>2024-01-11</td>
          <td>None</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c6cd47907f65a507baf89532c0aa0a851e2b93d0" target='_blank'>Computational prediction of ω-transaminase selectivity by deep learning analysis of molecular dynamics trajectories</a></td>
          <td>

              Carlos Ramírez-Palacios,

              S. Marrink,

          </td>
          <td>2022-12-12</td>
          <td>QRB Discovery</td>
          <td>2</td>
          <td>97</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0fabd55d41e116f50b02e6e3144adaa4fe84591f" target='_blank'>Graph Neural Networks Accelerated Molecular Dynamics</a></td>
          <td>

              Zijie Li,

              Kazem Meidani,

              Prakarsh Yadav,

              A. Farimani,

          </td>
          <td>2021-12-06</td>
          <td>The Journal of chemical physics</td>
          <td>34</td>
          <td>33</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d7de3d18e174fb9ac9049d260882afd7bb0a814b" target='_blank'>Author Correction: Optimization of Molecules via Deep Reinforcement Learning</a></td>
          <td>

              Zhenpeng Zhou,

              S. Kearnes,

              Li Li,

              R. Zare,

              Patrick F. Riley,

          </td>
          <td>2020-06-23</td>
          <td>Scientific Reports</td>
          <td>2</td>
          <td>115</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b8afcdf7a74fcf6ac434ea0809e672171518b648" target='_blank'>A Score-based Geometric Model for Molecular Dynamics Simulations</a></td>
          <td>

              Fang Wu,

              Qian Zhang,

              Xurui Jin,

              Yinghui Jiang,

              Stan Z. Li,

          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>7</td>
          <td>97</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7b4ceb9763e3a897db4e1b12f2b01e4d3ead696c" target='_blank'>Improved coarse-graining of Markov state models via explicit consideration of statistical uncertainty.</a></td>
          <td>

              G. Bowman,

          </td>
          <td>2012-01-18</td>
          <td>The Journal of chemical physics</td>
          <td>72</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ea3201dba38ddcac8fb78fb0c126cd059e3822dc" target='_blank'>Pre‐Training of Equivariant Graph Matching Networks with Conformation Flexibility for Drug Binding</a></td>
          <td>

              Fang Wu,

              Shuting Jin,

              Xurui Jin,

              Xiangrong Liu,

              Yinghui Jiang,

              Z. Niu,

              Qian Zhang,

              Stan Z. Li,

          </td>
          <td>2022-04-19</td>
          <td>Advanced Science</td>
          <td>14</td>
          <td>97</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8461c8b7dbbf71c0f3aea3e6a5852bcc21620401" target='_blank'>Ancestral protein sequence reconstruction using a tree-structured Ornstein-Uhlenbeck variational autoencoder</a></td>
          <td>

              L. S. Moreta,

              Ahmad Salim Al-Sibahi,

              J. Hein,

          </td>
          <td>None</td>
          <td>None</td>
          <td>2</td>
          <td>42</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c5190554d9d124de8c12640d368548bef0d74578" target='_blank'>Learning molecular dynamics: predicting the dynamics of glasses by a machine learning simulator.</a></td>
          <td>

              Han Liu,

              Zijie Huang,

              S. Schoenholz,

              E. D. Cubuk,

              M. Smedskjaer,

              Yizhou Sun,

              W. Wang,

              M. Bauchy,

          </td>
          <td>2023-06-29</td>
          <td>Materials horizons</td>
          <td>2</td>
          <td>65</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f5431452e9558f29207dcc66b1e8d639c52acd2c" target='_blank'>Machine learning implicit solvation for molecular dynamics.</a></td>
          <td>

              Yaoyi Chen,

              Andreas Krämer,

              N. Charron,

              B. Husic,

              C. Clementi,

              Frank No'e,

          </td>
          <td>2021-06-14</td>
          <td>The Journal of chemical physics</td>
          <td>36</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ad97f5fb9d261252b3ffd132d5c69f9f204aec5b" target='_blank'>Mixup-Augmented Meta-Learning for Sample-Efficient Fine-Tuning of Protein Simulators</a></td>
          <td>

              Jingbang Chen,

              Yian Wang,

              Xingwei Qu,

              Shuangjia Zheng,

              Yao-Chun Yang,

              Hao Dong,

              Jie Fu,

          </td>
          <td>2023-08-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9a1c517e3648c85f8b3f39679b4fadf153a19916" target='_blank'>Free energy along drug-protein binding pathways interactively sampled in virtual reality</a></td>
          <td>

              Helen M. Deeks,

              Kirill Zinovjev,

              J. Barnoud,

              A. Mulholland,

              M. W. van der Kamp,

              David R Glowacki,

          </td>
          <td>2023-10-04</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>59</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/91721e0801310a00f3e750505e61519e1e0f985a" target='_blank'>Learning Latent Dynamics via Invariant Decomposition and (Spatio-)Temporal Transformers</a></td>
          <td>

              Kai Lagemann,

              C. Lagemann,

              Swarnava Mukherjee,

          </td>
          <td>2023-06-21</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>10</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/751910f192146126ec050ac5c45a8f3416d58387" target='_blank'>The RED scheme: Rate-constant estimation from pre-steady state weighted ensemble simulations.</a></td>
          <td>

              A. DeGrave,

              Anthony T. Bogetti,

              L. Chong,

          </td>
          <td>2021-03-21</td>
          <td>The Journal of chemical physics</td>
          <td>11</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/93013a0ad444641953c01cd8182a84e1dff378a0" target='_blank'>What Markov state models can and cannot do: Correlation versus path-based observables in protein folding models</a></td>
          <td>

              Ernesto Suárez,

              Rafal P. Wiewiora,

              C. Wehmeyer,

              F. Noé,

              J. Chodera,

              D. Zuckerman,

          </td>
          <td>2020-11-09</td>
          <td>bioRxiv</td>
          <td>38</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c9295b260154dc8b4d95bc4a19da9d297dcf0d26" target='_blank'>Uncertainty-driven dynamics for active learning of interatomic potentials</a></td>
          <td>

              M. Kulichenko,

              K. Barros,

              N. Lubbers,

              Ying Wai Li,

              Richard A. Messerly,

              S. Tretiak,

              Justin S. Smith,

              B. Nebgen,

          </td>
          <td>2023-03-01</td>
          <td>Nature Computational Science</td>
          <td>32</td>
          <td>69</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/934a6ec4163084915801c1a7e054cb826e1bff31" target='_blank'>Markov models for the elucidation of allosteric regulation</a></td>
          <td>

              Ushnish Sengupta,

              B. Strodel,

          </td>
          <td>2018-06-19</td>
          <td>Philosophical Transactions of the Royal Society B: Biological Sciences</td>
          <td>27</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/86e3440fd4fdf47fd93779678c5d6d01b1b246ab" target='_blank'>Transition Path Sampling with Boltzmann Generator-based MCMC Moves</a></td>
          <td>

              Michael Plainer,

              Hannes Stärk,

              Charlotte Bunne,

              Stephan Günnemann,

          </td>
          <td>2023-12-08</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/79714b9ee1abe7f5e73947235392a57d15dc54a2" target='_blank'>∆ -Learning for Coarse-Grained Potentials</a></td>
          <td>

              Aditi Khot,

              B. Savoie,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6df0c65a803c91a84ab42a8ea114049165e3e737" target='_blank'>FAST Conformational Searches by Balancing Exploration/Exploitation Trade-Offs.</a></td>
          <td>

              Maxwell I. Zimmerman,

              G. Bowman,

          </td>
          <td>2015-11-20</td>
          <td>Journal of chemical theory and computation</td>
          <td>135</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a69482738401055aa4c73201000599d4fbe4741f" target='_blank'>3DFlex: determining structure and motion of flexible proteins from cryo-EM</a></td>
          <td>

              A. Punjani,

              David J. Fleet,

          </td>
          <td>2023-05-11</td>
          <td>Nature Methods</td>
          <td>53</td>
          <td>71</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1f5b19a1f10b8a7836d6e9b7e57bd5acc6b00f85" target='_blank'>Toward a direct and scalable identification of reduced models for categorical processes</a></td>
          <td>

              Susanne Gerber,

              I. Horenko,

          </td>
          <td>2017-04-21</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>19</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9af91d9abf67a3f4dcfe3f15561119beaf9a0cc9" target='_blank'>A kernel-based approach to molecular conformation analysis</a></td>
          <td>

              Stefan Klus,

              A. Bittracher,

              Ingmar Schuster,

              C. Schütte,

          </td>
          <td>2018-09-28</td>
          <td>The Journal of chemical physics</td>
          <td>22</td>
          <td>41</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/02a6b974397afa97ee32a32b7a655f2aed513313" target='_blank'>Hidden GPCR structural transitions addressed by multiple walker supervised molecular dynamics (mwSuMD)</a></td>
          <td>

              Giuseppe Deganutti,

              Ludovico Pipitò,

              Roxana-Maria Rujan,

              T. Weizmann,

              Peter Griffin,

              Antonella Ciancetta,

              S. Moro,

              C. Reynolds,

          </td>
          <td>2024-01-25</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>52</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/060d35318a5ef9637b5c1be4c2917b14277bd338" target='_blank'>Modeling Ensembles of Enzyme Reaction Pathways with Hi-MSM Reveals the Importance of Accounting for Pathway Diversity.</a></td>
          <td>

              Joseph R Persichetti,

              Yang Jiang,

              Phillip S. Hudson,

              E. O’Brien,

          </td>
          <td>2022-11-16</td>
          <td>The journal of physical chemistry. B</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cd571fd71d3b4b87b7cd6453813169aaa72e0e30" target='_blank'>Learning generative models for protein fold families</a></td>
          <td>

              Sivaraman Balakrishnan,

              Hetunandan Kamisetty,

              J. Carbonell,

              Su-In Lee,

              C. Langmead,

          </td>
          <td>2011-04-01</td>
          <td>Proteins: Structure</td>
          <td>318</td>
          <td>72</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8917ffec1627ad5ad2bc543b3c80aafc25b36cbf" target='_blank'>Path lumping: An efficient algorithm to identify metastable path channels for conformational dynamics of multi-body systems.</a></td>
          <td>

              Luming Meng,

              F. Sheong,

              Xiangze Zeng,

              Lizhe Zhu,

              Xuhui Huang,

          </td>
          <td>2017-07-31</td>
          <td>The Journal of chemical physics</td>
          <td>10</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cee36473ff970b98a36543bf2997360abdcead50" target='_blank'>Machine learning coarse grained models for water</a></td>
          <td>

              Henry Chan,

              M. Cherukara,

              B. Narayanan,

              T. Loeffler,

              C. Benmore,

              S. Gray,

              S. Sankaranarayanan,

          </td>
          <td>2019-01-22</td>
          <td>Nature Communications</td>
          <td>111</td>
          <td>64</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8120c229a6c660c64fdd72221f868abcff31f3c1" target='_blank'>Ensemble learning of coarse-grained molecular dynamics force fields with a kernel approach.</a></td>
          <td>

              Jiang Wang,

              Stefan Chmiela,

              K. Müller,

              F. Noé,

              C. Clementi,

          </td>
          <td>2020-05-04</td>
          <td>The Journal of chemical physics</td>
          <td>41</td>
          <td>127</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/38309ab72b0dbdef7d39faeff8e4df59cadaeabd" target='_blank'>High-dimensional structure learning of binary pairwise Markov networks: A comparative numerical study</a></td>
          <td>

              J. Pensar,

              Y. Xu,

              S. Puranen,

              M. Pesonen,

              Y. Kabashima,

              J. Corander,

          </td>
          <td>2019-01-14</td>
          <td>Comput. Stat. Data Anal.</td>
          <td>2</td>
          <td>70</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a76b6396a6f1326a3afdefc850f2129dad7c3720" target='_blank'>Neural Networks In Chemical Reaction Dynamics</a></td>
          <td>

              K. Baumgartner,

          </td>
          <td>None</td>
          <td></td>
          <td>28</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e563c5cde23f5557763be84e660e3635b0a02778" target='_blank'>Data-driven classification of ligand unbinding pathways.</a></td>
          <td>

              Dhiman Ray,

              M. Parrinello,

          </td>
          <td>2023-08-02</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>2</td>
          <td>123</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e9527775263020013e141fd6a99d9c22ae1380cd" target='_blank'>Adaptive Sampling Methods for Molecular Dynamics in the Era of Machine Learning.</a></td>
          <td>

              D. Kleiman,

              Hassan Nadeem,

              D. Shukla,

          </td>
          <td>2023-07-18</td>
          <td>The journal of physical chemistry. B</td>
          <td>7</td>
          <td>33</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d1806f70ee304004a32f068d014ec5db665964c5" target='_blank'>Long Short-Term Memory to predict 3D Amino acids Positions in GPCR Molecular Dynamics</a></td>
          <td>

              J. López-Correa,

              Caroline König,

              A. Vellido,

          </td>
          <td>2022-06-02</td>
          <td>None</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7ed7686f52f68686e734ad4a459ded57aff00cb7" target='_blank'>Towards Molecular Machine Learning for the IoBNT</a></td>
          <td>

              Stefan Angerbauer,

              Franz Enzenhofer,

              Tobias Pankratz,

              W. Haselmayr,

          </td>
          <td>2023-09-20</td>
          <td>Proceedings of the 10th ACM International Conference on Nanoscale Computing and Communication</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1e186ceeff16b19d1f705edef2d662d20fc5ad39" target='_blank'>RosENet: Improving Binding Affinity Prediction by Leveraging Molecular Mechanics Energies with an Ensemble of 3D Convolutional Neural Networks</a></td>
          <td>

              Hussein Hassan-Harrirou,

              Ce Zhang,

              Thomas Lemmin,

          </td>
          <td>2020-05-11</td>
          <td>Journal of chemical information and modeling</td>
          <td>56</td>
          <td>24</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/117987ac9bea8eaf6b5d43e73a67ba5cf8203f8f" target='_blank'>Accelerated Construction of Kinetic Network Model of Biomolecules Using Steered Molecular Dynamics.</a></td>
          <td>

              Susmita Ghosh,

              A. Chatterjee,

              S. Bhattacharya,

          </td>
          <td>2018-09-13</td>
          <td>Journal of chemical theory and computation</td>
          <td>3</td>
          <td>31</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c846862843769707ee676f9ecd892cfc90afe887" target='_blank'>On-the-fly machine learning of quantum mechanical forces and its potential applications for large scale molecular dynamics</a></td>
          <td>

              Zhenwei Li,

          </td>
          <td>None</td>
          <td></td>
          <td>3</td>
          <td>3</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d757b4604edd1ee395fcda54d166c906e080cff9" target='_blank'>Exploring glassy dynamics with Markov state models from graph dynamical neural networks.</a></td>
          <td>

              S. Soltani,

              C. Sinclair,

              Joerg Rottler,

          </td>
          <td>2021-11-19</td>
          <td>Physical review. E</td>
          <td>3</td>
          <td>31</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1d3aebc51df18ba1dd32c232c6535de4262e473d" target='_blank'>Graphical continuous Lyapunov models</a></td>
          <td>

              Gherardo Varando,

              N. Hansen,

          </td>
          <td>2020-05-21</td>
          <td>None</td>
          <td>9</td>
          <td>13</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/42f7dd636e82a04984f65a23e31dc2d0f3d03145" target='_blank'>Trajectory-Based Parameterization of a Coarse-Grained Forcefield for High-Thoughput Protein Simulation</a></td>
          <td>

              J. Jumper,

              K. Freed,

              T. Sosnick,

          </td>
          <td>2017-07-27</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>66</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b5a29e6fc8dea483e7c7ff1091e8c15fe9058ff9" target='_blank'>Exploring Chemical and Conformational Spaces by Batch Mode Deep Active Learning</a></td>
          <td>

              Viktor Zaverkin,

              David Holzmüller,

              Ingo Steinwart,

              Johannes Kästner,

          </td>
          <td>None</td>
          <td>Digital Discovery</td>
          <td>15</td>
          <td>9</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3fcb9b705c27dc8b062c255ba59940c4e8bd15ca" target='_blank'>Set-free Markov state model building.</a></td>
          <td>

              Marcus Weber,

              K. Fackeldey,

              C. Schütte,

          </td>
          <td>2017-03-31</td>
          <td>The Journal of chemical physics</td>
          <td>18</td>
          <td>41</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/33e92d69d7dd3bd7593aff044059d747e906388c" target='_blank'>Set-free Markov State Building</a></td>
          <td>

              Marcus Weber,

              K. Fackeldey,

              C. Schütte,

          </td>
          <td>2017-03-03</td>
          <td></td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e88a72d6df0f548b1ef4b1b431d644ffc2db0ac9" target='_blank'>Unravelling the Kinetic Model of Photochemical Reactions via Deep Learning.</a></td>
          <td>

              Philipp Kollenz,

              D. Herten,

              T. Buckup,

          </td>
          <td>2020-06-26</td>
          <td>The journal of physical chemistry. B</td>
          <td>13</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/587000ac3883fef394c1dc1dda01e895ee9f8f64" target='_blank'>Boosting performance of generative diffusion model for molecular docking by training on artificial binding pockets</a></td>
          <td>

              Taras Voitsitskyi,

              Volodymyr Bdzhola,

              Roman Stratiichuk,

              Ihor Koleiev,

              Zakhar Ostrovsky,

              Volodymyr Vozniak,

              Ivan Khropachov,

              Pavlo Henitsoi,

              Leonid Popryho,

              Roman Zhytar,

              S. Yesylevskyy,

              Alan Nafiiev,

              Serhii Starosyla,

          </td>
          <td>2023-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f61aa289bdafe7902580ce49981f334056079c6f" target='_blank'>The emergence of machine learning force fields in drug design.</a></td>
          <td>

              Mingan Chen,

              Xinyu Jiang,

              Lehan Zhang,

              Xiaoxu Chen,

              Yiming Wen,

              Zhiyong Gu,

              Xutong Li,

              Mingyue Zheng,

          </td>
          <td>2024-01-03</td>
          <td>Medicinal research reviews</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8bc613de1d0e6efff5317a2e7495a6d4f4791934" target='_blank'>3D Flexible Refinement: Structure and Motion of Flexible Proteins from Cryo-EM</a></td>
          <td>

              A. Punjani,

              David J. Fleet,

          </td>
          <td>2021-04-22</td>
          <td>bioRxiv</td>
          <td>52</td>
          <td>71</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f2eafeb800f34ec0da8c3eb82809087047db54ac" target='_blank'>Interpolating Nonadiabatic Molecular Dynamics Hamiltonian with Bidirectional Long Short-Term Memory Networks</a></td>
          <td>

              Bipeng Wang,

              Ludwig Winkler,

              Yifan Wu,

              Klaus-Robert Müller,

              H. E. Sauceda,

              O. Prezhdo,

          </td>
          <td>2023-08-02</td>
          <td>The Journal of Physical Chemistry Letters</td>
          <td>2</td>
          <td>81</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d76b2ac177b43425ed403587a74dbb6a4856b3f8" target='_blank'>AI2BMD: efficient characterization of protein dynamics with ab initio accuracy</a></td>
          <td>

              Tong Wang,

              Xinheng He,

              Mingyu Li,

              Yusong Wang,

              Zun Wang,

              Shaoning Li,

              Bin Shao,

              Tie-Yan Liu,

          </td>
          <td>2023-07-13</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>9</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b418bc0d7ba63f948050e15de8d362de4db2580e" target='_blank'>Stochastic embeddings of dynamical phenomena through variational autoencoders</a></td>
          <td>

              C. A. García,

              P. Félix,

              J. Presedo,

              A. Otero,

          </td>
          <td>2020-10-13</td>
          <td>J. Comput. Phys.</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2e02bf6c25cde6b660dc479cc524ada87db85f8f" target='_blank'>Accelerating the Generation of Molecular Conformations with Progressive Distillation of Equivariant Latent Diffusion Models</a></td>
          <td>

              Romain Lacombe,

              Neal Vaidya,

          </td>
          <td>2024-04-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/de279b452b98d27e53e34bde08bfcc0e213f7e5a" target='_blank'>Local Kernel Regression and Neural Network Approaches to the Conformational Landscapes of Oligopeptides</a></td>
          <td>

              Raimon Fabregat,

              Alberto Fabrizio,

              Edgar A. Engel,

              Benjamin Meyer,

              V. Jurásková,

              M. Ceriotti,

              C. Corminboeuf,

          </td>
          <td>2022-02-18</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>7</td>
          <td>55</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6fa357cbd86e5f6dccbebed602c2e0e57dee6bb9" target='_blank'>Predicting the dynamic interaction of intrinsically disordered proteins</a></td>
          <td>

              Yuchuan Zheng,

              Qixiu Li,

              M. Freiberger,

              Haoyu Song,

              Guorong Hu,

              Moxin Zhang,

              Ruoxu Gu,

              Jingyuan Li,

          </td>
          <td>2024-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/26b6f84e9e909a24375b8a89bbf3c40d245e9019" target='_blank'>Machine Learning Force Fields with Data Cost Aware Training</a></td>
          <td>

              Alexander W. Bukharin,

              Tianyi Liu,

              Sheng Wang,

              Simiao Zuo,

              Weihao Gao,

              Wen Yan,

              Tuo Zhao,

          </td>
          <td>2023-06-05</td>
          <td>None</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5864926d0af9ddafb8b036a70f74667b4643eeeb" target='_blank'>Encoding protein dynamic information in graph representation for functional residue identification</a></td>
          <td>

              Chiang Yuan,

              Weiwei Hui,

              Shu-Wei Chang,

          </td>
          <td>2021-12-15</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>7</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4ffe5fa8540bb66354f75ca1d16de4b0db37e89b" target='_blank'>Algorithmic models for the natural sciences</a></td>
          <td>

              Hinsen Konrad,

          </td>
          <td>2014-06-05</td>
          <td></td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/334dc849e21285fa661a75bb896590b6e4f36e42" target='_blank'>Hierarchical variational models for statistical physics</a></td>
          <td>

              Jaan Altosaar,

          </td>
          <td>None</td>
          <td>None</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ddf67d3d35632cc73cf7612a0fd5a8161d51071" target='_blank'>An inductive transfer learning force field (ITLFF) protocol builds protein force fields in seconds</a></td>
          <td>

              Yanqiang Han,

              Zhilong Wang,

              An Chen,

              Imran Ali,

              Junfei Cai,

              Simin Ye,

              Jinjin Li,

          </td>
          <td>2022-01-17</td>
          <td>Briefings in bioinformatics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/24faddd8ef452e190b30210e4cc9f2277787da12" target='_blank'>Automatic Feature Selection in Markov State Models Using Genetic Algorithm</a></td>
          <td>

              Qihua Chen,

              Jiangyan Feng,

              S. Mittal,

              D. Shukla,

          </td>
          <td>2018-06-01</td>
          <td>The Journal of Computational Science Education</td>
          <td>6</td>
          <td>33</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/182afd2cfc31e8d283c86228c03da091fc2ff067" target='_blank'>Machine learning of an implicit solvent for dynamic Monte Carlo simulations.</a></td>
          <td>

              Ankush Checkervarty,

              J. Sommer,

              M. Werner,

          </td>
          <td>2023-01-24</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bfc3fe8484bf3ea63dfaf4a3b9ff0855ac1253c1" target='_blank'>Efficient Continuous-Time Markov Chain Estimation</a></td>
          <td>

              Monir Hajiaghayi,

              Bonnie Kirkpatrick,

              Liangliang Wang,

              A. Bouchard-Côté,

          </td>
          <td>2013-09-12</td>
          <td>None</td>
          <td>30</td>
          <td>25</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7e6deb5b57e72a1053de6166c718bf9174ce7dad" target='_blank'>Driving Conformational Transitions in the Feature Space of Autoencoder Neural Network</a></td>
          <td>

              Bumjoon Seo,

              Seulwoo Kim,

              Minhwan Lee,

              Youn-Woo Lee,

              Won Bo Lee,

          </td>
          <td>2018-09-14</td>
          <td>The Journal of Physical Chemistry C</td>
          <td>6</td>
          <td>15</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7ece6935e1ceed2efe6941be01cd347fc53b62a9" target='_blank'>Monte Carlo sampling for stochastic weight functions</a></td>
          <td>

              D. Frenkel,

              K. J. Schrenk,

              Stefano Martiniani,

          </td>
          <td>2016-12-19</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>15</td>
          <td>97</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/54783aeec96d075fc03fd256ca9d3e3c0372906a" target='_blank'>MSM/RD: Coupling Markov state models of molecular kinetics with reaction-diffusion simulations.</a></td>
          <td>

              Manuel Dibak,

              Mauricio J. del Razo,

              David de Sancho,

              C. Schütte,

              F. Noé,

          </td>
          <td>2017-12-21</td>
          <td>The Journal of chemical physics</td>
          <td>27</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/500513997028c264c661f06493b1091ceb2cd1ea" target='_blank'>Geometric learning of the conformational dynamics of molecules using dynamic graph neural networks</a></td>
          <td>

              Michael Ashby,

              Jenna A. Bilbrey,

          </td>
          <td>2021-06-24</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>11</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/374313c2fc407655ae05eed61005a29cfaa7460b" target='_blank'>Towards exact molecular dynamics simulations with machine-learned force fields</a></td>
          <td>

              Stefan Chmiela,

              H. E. Sauceda,

              K. Müller,

              A. Tkatchenko,

          </td>
          <td>2018-02-26</td>
          <td>Nature Communications</td>
          <td>516</td>
          <td>127</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe19179c54d1abcb79eca24a765214e910915e44" target='_blank'>Expectation maximized molecular dynamics: Toward efficient learning of rarely sampled features in free energy surfaces from unbiased simulations.</a></td>
          <td>

              Pallab Dutta,

              Neelanjana Sengupta,

          </td>
          <td>2020-10-15</td>
          <td>The Journal of chemical physics</td>
          <td>3</td>
          <td>16</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/290716662cbc93de08dfdc34c55be2741c235773" target='_blank'>Coarse-Graining of Molecular Dynamics Using Neural Ordinary Differential Equations</a></td>
          <td>

              Jakub Lála,

              Stefano Angiolleti-Uberti,

          </td>
          <td>2023-07-06</td>
          <td>IEEE EUROCON 2023 - 20th International Conference on Smart Technologies</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b78620184e43c6609b1c2df240fad080460de420" target='_blank'>The combined force field-sampling problem in simulations of disordered amyloid-β peptides</a></td>
          <td>

              A. Vitalis,

              M. Bacci,

              A. Caflisch,

          </td>
          <td>None</td>
          <td>None</td>
          <td>4</td>
          <td>74</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2ebc00784cae401010ba9c97a7821c5819f420cd" target='_blank'>Efficient Bayesian estimation of Markov model transition matrices with given stationary distribution.</a></td>
          <td>

              Benjamin Trendelkamp-Schroer,

              F. Noé,

          </td>
          <td>2013-01-10</td>
          <td>The Journal of chemical physics</td>
          <td>34</td>
          <td>61</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2c6d4cf66e67cef04be2e61f27f7332817e03ada" target='_blank'>Improving machine learning force fields for molecular dynamics simulations with fine-grained force metrics.</a></td>
          <td>

              Zun Wang,

              Hongfei Wu,

              Lixin Sun,

              Xinheng He,

              Zhirong Liu,

              Bin Shao,

              Tong Wang,

              Tie-Yan Liu,

          </td>
          <td>2023-07-17</td>
          <td>The Journal of chemical physics</td>
          <td>10</td>
          <td>37</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d40b083ac2c6f1776b188293f5cf78fdbd50cabf" target='_blank'>From Peptides to Nanostructures: A Euclidean Transformer for Fast and Stable Machine Learned Force Fields</a></td>
          <td>

              J. Frank,

              Oliver T. Unke,

              Klaus-Robert Müller,

              Stefan Chmiela,

          </td>
          <td>2023-09-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6c3123c197ae144fce79b6c008ad2de45863af4b" target='_blank'>Long-Short-Range Message-Passing: A Physics-Informed Framework to Capture Non-Local Interaction for Scalable Molecular Dynamics Simulation</a></td>
          <td>

              Yunyang Li,

              Yusong Wang,

              Lin Huang,

              Hang Yang,

              Xinran Wei,

              Jia Zhang,

              Tong Wang,

              Zun Wang,

              Bin Shao,

              Tie-Yan Liu,

          </td>
          <td>2023-04-26</td>
          <td>None</td>
          <td>5</td>
          <td>9</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0b502f9628475bc17df2b46d7149e74fccc59d5c" target='_blank'>Protein Conformational States—A First Principles Bayesian Method †</a></td>
          <td>

              D. Rogers,

          </td>
          <td>2020-08-05</td>
          <td>Entropy</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/582fd49161c3b6360091d65c25c26392b44807bd" target='_blank'>Activation Pathways of Neurotensin Receptor 1 Elucidated Using Statistical Machine Learning.</a></td>
          <td>

              Prakarsh Yadav,

              Amir Barati Farimani,

          </td>
          <td>2022-04-05</td>
          <td>ACS chemical neuroscience</td>
          <td>3</td>
          <td>17</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/43def8db9cd59e5a3b5bc7a2852f693c36e53ee6" target='_blank'>Unfolding Hidden Barriers by Active Enhanced Sampling</a></td>
          <td>

              Jing Zhang,

              Ming Chen,

          </td>
          <td>2017-05-21</td>
          <td>Physical review letters</td>
          <td>30</td>
          <td>7</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cae2adb6642a38e2dd80ee975a65b26788a891c6" target='_blank'>From Latent Dynamics to Meaningful Representations.</a></td>
          <td>

              Dedi Wang,

              Yihang Wang,

              Luke J. Evans,

              P. Tiwary,

          </td>
          <td>2022-09-02</td>
          <td>Journal of chemical theory and computation</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4bfcf09bd7280528cd87b7aed7a8b91fdbd6a176" target='_blank'>A physics-informed neural network method for the approximation of slow invariant manifolds for the general class of stiff systems of ODEs</a></td>
          <td>

              Dimitrios G. Patsatzis,

              Lucia Russo,

              C. Siettos,

          </td>
          <td>2024-03-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7ced62abf6087405892594b8d07b6b53a6e1ea86" target='_blank'>Guiding Protein Conformation Sampling with Conformation Space Maps</a></td>
          <td>

              Ahmed Bin Zaman,

              K. De Jong,

              Amarda Shehu,

          </td>
          <td>None</td>
          <td>EPiC Series in Computing</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/49da7e115031ddf0a44b730e99f737fb2fc9fafa" target='_blank'>Discovering Protein Conformational Flexibility Through Artificial Intelligence Aided Molecular Dynamics.</a></td>
          <td>

              Zachary Smith,

              P. Ravindra,

              Yihang Wang,

              Rory Cooley,

              P. Tiwary,

          </td>
          <td>2020-08-25</td>
          <td>The journal of physical chemistry. B</td>
          <td>20</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5ba2cacf5cf13544964a7dc146d531cb8328ce6d" target='_blank'>Alecsandre Sauvé-Lacoursière</a></td>
          <td>

              Normand Mousseau,

          </td>
          <td>2018-06-06</td>
          <td></td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d4705791a22794c2a0df05da139a81647d518f71" target='_blank'>Using Neural Architectures to Model Complex Dynamical Systems</a></td>
          <td>

              N. Gabriel,

              Neil F Johnson,

          </td>
          <td>None</td>
          <td>Adv. Artif. Intell. Mach. Learn.</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/52437307ca098da3c7b3771464fec78fc26b0963" target='_blank'>Embedding of Molecular Structure Using Molecular Hypergraph Variational Autoencoder with Metric Learning</a></td>
          <td>

              Daiki Koge,

              N. Ono,

              Ming Huang,

              M. Altaf-Ul-Amin,

              S. Kanaya,

          </td>
          <td>2020-11-08</td>
          <td>Molecular Informatics</td>
          <td>16</td>
          <td>47</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a27e0bfd608a11ff40d59b41ae6e27414863a719" target='_blank'>Neural network learns physical rules for copolymer translocation through amphiphilic barriers</a></td>
          <td>

              M. Werner,

              Yachong Guo,

              V. Baulin,

          </td>
          <td>2019-04-30</td>
          <td>npj Computational Materials</td>
          <td>6</td>
          <td>32</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/59e61d648de2b4f104e8f82bbcae1445e219cded" target='_blank'>Physical Markov model for protein structure prediction</a></td>
          <td>

              Yeona Kang,

              C. Fortmann,

          </td>
          <td>2009-11-01</td>
          <td>2009 IEEE International Conference on Bioinformatics and Biomedicine Workshop</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d530099f6c8815080fa8860ab3a72c1edaafd88d" target='_blank'>A Group Symmetric Stochastic Differential Equation Model for Molecule Multi-modal Pretraining</a></td>
          <td>

              Shengchao Liu,

              Weitao Du,

              Zhiming Ma,

              Hongyu Guo,

              Jian Tang,

          </td>
          <td>2023-05-28</td>
          <td>ArXiv</td>
          <td>12</td>
          <td>22</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ef85de5122736a603650263f9c39939f55b6a5df" target='_blank'>Centroid Molecular Dynamics Can Be Greatly Accelerated through Neural Network Learned Centroid Forces Derived from Path Integral Molecular Dynamics</a></td>
          <td>

              Timothy D Loose,

              Patrick G. Sahrmann,

              G. Voth,

          </td>
          <td>2022-08-16</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>5</td>
          <td>103</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3cdd0500a082a838b9b945d76e305c99ab0ad938" target='_blank'>GemNet: Universal Directional Graph Neural Networks for Molecules</a></td>
          <td>

              Johannes Klicpera,

              Florian Becker,

              Stephan Gunnemann,

          </td>
          <td>2021-06-02</td>
          <td>None</td>
          <td>297</td>
          <td>16</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e66a427214f6a68d0fae58182ede6cb6ba79167c" target='_blank'>Enhancing geometric representations for molecules with equivariant vector-scalar interactive message passing</a></td>
          <td>

              Yusong Wang,

              Tong Wang,

              Shaoning Li,

              Xinheng He,

              Mingyu Li,

              Zun Wang,

              Nanning Zheng,

              Bin Shao,

              Tie-Yan Liu,

          </td>
          <td>2024-01-05</td>
          <td>Nature Communications</td>
          <td>4</td>
          <td>9</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/73f8535cd9d86de196545b4ba563b4bb5f9496af" target='_blank'>Heterogeneous relational message passing networks for molecular dynamics simulations</a></td>
          <td>

              Zun Wang,

              Chong Wang,

              Sibo Zhao,

              Yong Xu,

              Shaogang Hao,

              Chang-Yu Hsieh,

              B. Gu,

              W. Duan,

          </td>
          <td>2021-09-02</td>
          <td>npj Computational Materials</td>
          <td>20</td>
          <td>71</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/360054585b0c4cf0b1a7cba955544d77401e4d39" target='_blank'>Active Learning the Potential Energy Landscape for Water Clusters from Sparse Training Data</a></td>
          <td>

              T. Loeffler,

              T. Patra,

              Henry Chan,

              M. Cherukara,

              S. Sankaranarayanan,

          </td>
          <td>2020-01-31</td>
          <td>Journal of Physical Chemistry C</td>
          <td>22</td>
          <td>42</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ba712d7a1bb988644c952589059625c623a57120" target='_blank'>De novo protein design using geometric vector field networks</a></td>
          <td>

              Weian Mao,

              Muzhi Zhu,

              Zheng Sun,

              Shuaike Shen,

              Lin Yuanbo Wu,

              Hao Chen,

              Chunhua Shen,

          </td>
          <td>2023-10-18</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>3</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/83ea81255d3ad1b92b201662ac1e71e652e3f309" target='_blank'>Analysing ill-conditioned Markov chains</a></td>
          <td>

              Esmae J. Woods,

              Deepti Kannan,

              Daniel J. Sharpe,

              T. Swinburne,

              D. Wales,

          </td>
          <td>2023-05-22</td>
          <td>Philosophical transactions. Series A, Mathematical, physical, and engineering sciences</td>
          <td>2</td>
          <td>76</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/90cd845ab02cbfeb0d33d981ba099516dc9c64f5" target='_blank'>MSMExplorer: visualizing Markov state models for biomolecule folding simulations</a></td>
          <td>

              Bryce W. Cronkite-Ratcliff,

              V. Pande,

          </td>
          <td>2013-04-01</td>
          <td>Bioinformatics</td>
          <td>21</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0eb4a78e199c2801cf9aa8d3ec1a31f9220f05a7" target='_blank'>Influence of Mg2+ Distribution on the Stability of Folded States of the Twister Ribozyme Revealed Using Grand Canonical Monte Carlo and Generative Deep Learning Enhanced Sampling</a></td>
          <td>

              M. Sengul,

              A. MacKerell,

          </td>
          <td>2023-05-24</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ff75eeba95ac02994bb6a4cf609df14e8698a1e" target='_blank'>Uncertainty Quantification for Molecular Models via Stochastic Gradient MCMC</a></td>
          <td>

              Stephan Thaler,

              J. Zavadlav,

          </td>
          <td>None</td>
          <td>MATHMOD 2022 Discussion Contribution Volume</td>
          <td>2</td>
          <td>14</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69bab843432803a8f69ffa7c33779f205e843841" target='_blank'>HamNet: Conformation-Guided Molecular Representation with Hamiltonian Neural Networks</a></td>
          <td>

              Ziyao Li,

              Shuwen Yang,

              Guojie Song,

              Lingsheng Cai,

          </td>
          <td>2021-05-08</td>
          <td>ArXiv</td>
          <td>21</td>
          <td>28</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bd7c9b2a6880a3c624fe9ef8f48412f9755ba3fe" target='_blank'>Neural Networks for Chemists: An Introduction</a></td>
          <td>

              J. Zupan,

              J. Gasteiger,

          </td>
          <td>None</td>
          <td></td>
          <td>510</td>
          <td>72</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b937df713d1ea7057861e7872fa37b9148d3544" target='_blank'>Efficient and exact sampling of transition path ensembles on Markovian networks.</a></td>
          <td>

              Daniel J. Sharpe,

              D. Wales,

          </td>
          <td>2020-07-13</td>
          <td>The Journal of chemical physics</td>
          <td>14</td>
          <td>76</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/33eb7177d80860bc5ba1034c4937b8fd7df1ee80" target='_blank'>Decoding Molecular Graph Embeddings with Reinforcement Learning</a></td>
          <td>

              S. Kearnes,

              Li Li,

              Patrick F. Riley,

          </td>
          <td>2019-04-18</td>
          <td>ArXiv</td>
          <td>25</td>
          <td>16</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ff3a543e2fca4940eac26245b059436da3ce9dcf" target='_blank'>Discovering loop conformational flexibility in T4 lysozyme mutants through artificial intelligence aided molecular dynamics</a></td>
          <td>

              Zachary Smith,

              P. Ravindra,

              Yihang Wang,

              Rory Cooley,

              P. Tiwary,

          </td>
          <td>2020-04-09</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/68d2d3bb133dfa29a9dfadccd19321eb2602d925" target='_blank'>ViSNet: an equivariant geometry-enhanced graph neural network with vector-scalar interactive message passing for molecules</a></td>
          <td>

              Yusong Wang,

              Shaoning Li,

              Xinheng He,

              Mingyu Li,

              Zun Wang,

              Nanning Zheng,

              Bin Shao,

              Tie-Yan Liu,

              Tong Wang,

          </td>
          <td>2022-10-29</td>
          <td>None</td>
          <td>4</td>
          <td>12</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8e3ca5951f705406d5a89bf4cbd4d4c436cdd8f0" target='_blank'>Equivariant vector field network for many-body system modeling</a></td>
          <td>

              Weitao Du,

              He Zhang,

              Yuanqi Du,

              Qi Meng,

              Wei Chen,

              Bin Shao,

              Tie-Yan Liu,

          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>8</td>
          <td>81</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/20a6548e83fc464931ab8845c0f5e5d41ac6d640" target='_blank'>F$^3$low: Frame-to-Frame Coarse-grained Molecular Dynamics with SE(3) Guided Flow Matching</a></td>
          <td>

              Shaoning Li,

              Yusong Wang,

              Mingyu Li,

              Jian Zhang,

              Bin Shao,

              Nanning Zheng,

              Jian Tang,

          </td>
          <td>2024-05-01</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3ea192f15e8573f54c54c7338a95032377927149" target='_blank'>Efficiency and Parameter Selection of a micro-macro Markov chain Monte Carlo method for molecular dynamics</a></td>
          <td>

              H. Vandecasteele,

              G. Samaey,

          </td>
          <td>2022-09-26</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>25</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2cd290c76a7ab0f6b2daa0dc718279b36e927ea8" target='_blank'>Molecular Geometry-aware Transformer for accurate 3D Atomic System modeling</a></td>
          <td>

              Zheng Yuan,

              Yaoyun Zhang,

              Chuanqi Tan,

              Wei Wang,

              Feiran Huang,

              Songfang Huang,

          </td>
          <td>2023-02-02</td>
          <td>ArXiv</td>
          <td>5</td>
          <td>31</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/25c822868c3b1d9d1b84ae198e69e158bb33ba6c" target='_blank'>Towards exact molecular dynamics simulations with invariant machine-learned models</a></td>
          <td>

              Stefan Chmiela,

          </td>
          <td>None</td>
          <td></td>
          <td>8</td>
          <td>15</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f64c988795cbb0371832d52be41b91b68e22af9f" target='_blank'>Espaloma-0.3.0: Machine-learned molecular mechanics force field for the simulation of protein-ligand systems and beyond</a></td>
          <td>

              K. Takaba,

              Iván Pulido,

              Mike Henry,

              Hugo MacDermott-Opeskin,

              J. Chodera,

              Yuanqing Wang,

          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>55</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fa8e09205060f93b8a72dcd9bed7aeff8a9ae416" target='_blank'>A General Deep Learning Framework for Structure and Dynamics Reconstruction from Time Series Data</a></td>
          <td>

              Zhang Zhang,

              ,

              Shuo Wang,

              Ruyue Xin,

              Jiang Zhang,

          </td>
          <td>2018-12-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58c00995549392fe03fd495de30e5f022dc1065b" target='_blank'>Hybrid computational methods combining experimental information with molecular dynamics.</a></td>
          <td>

              Arup Mondal,

              Stefan Lenz,

              J. MacCallum,

              Alberto Perez,

          </td>
          <td>2023-05-22</td>
          <td>Current opinion in structural biology</td>
          <td>3</td>
          <td>28</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f5ce9b10c2c677576f15a7c9ad6ec156dbcb7fb" target='_blank'>Lightweight and effective tensor sensitivity for atomistic neural networks.</a></td>
          <td>

              Michael Chigaev,

              Justin S. Smith,

              Steven Anaya,

              B. Nebgen,

              M. Bettencourt,

              K. Barros,

              N. Lubbers,

          </td>
          <td>2022-12-06</td>
          <td>The Journal of chemical physics</td>
          <td>5</td>
          <td>27</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3819fae661099ee01322b6c5624da017b402f530" target='_blank'>Gramian Angular Fields for leveraging pretrained computer vision models with anomalous diffusion trajectories</a></td>
          <td>

              Òscar Garibo i Orts,

              Nicolás Firbas,

              L. Sebastiá,

              J. A. Conejero,

          </td>
          <td>2023-03-01</td>
          <td>Physical review. E</td>
          <td>5</td>
          <td>14</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a452aff825339a1917b6529e4056242e58acbf67" target='_blank'>Understanding protein folding using Markov state models.</a></td>
          <td>

              V. Pande,

          </td>
          <td>None</td>
          <td>Advances in experimental medicine and biology</td>
          <td>11</td>
          <td>102</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9b449b4824f5f2f11e75f2e892e57062a029c382" target='_blank'>On the NS-DSSB unidirectional estimates in the SAMPL6 SAMPLing challenge</a></td>
          <td>

              P. Procacci,

              Marina Macchiagodena,

          </td>
          <td>2021-10-01</td>
          <td>Journal of Computer-Aided Molecular Design</td>
          <td>3</td>
          <td>35</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fa4af359c3de8f8c047ce7a2d9fc48074afcca92" target='_blank'>Mapping energy landscapes of homopolymeric RNAs via simulated tempering and deep unsupervised learning</a></td>
          <td>

              Vysakh Ramachandran,

              D. Potoyan,

          </td>
          <td>2023-10-10</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b96802440ece1f2012b3f28c757ad98eff23bd14" target='_blank'>EspcTM: Kinetic Transition Network Based on Trajectory Mapping in Effective Energy Rescaling Space</a></td>
          <td>

              Zhenyu Wang,

              Xin Zhou,

              Guanghong Zuo,

          </td>
          <td>2020-10-27</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6957a4fcf53e4cfd3933909bcff67601f2c144eb" target='_blank'>Advancing molecular simulation with equivariant interatomic potentials</a></td>
          <td>

              Simon L. Batzner,

              Albert Musaelian,

              B. Kozinsky,

          </td>
          <td>2023-07-19</td>
          <td>Nature Reviews Physics</td>
          <td>7</td>
          <td>36</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5fec27e7ea87d5c7712aaffbdc8d43b3282dc5d5" target='_blank'>RotNet: A Rotationally Invariant Graph Neural Network for Quantum Mechanical Calculations.</a></td>
          <td>

              Hongwei Tu,

              Yanqiang Han,

              Zhilong Wang,

              An Chen,

              Kehao Tao,

              Simin Ye,

              Shiwei Wang,

              Zhiyun Wei,

              Jinjin Li,

          </td>
          <td>2023-09-19</td>
          <td>Small methods</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b068c83aa1dc6de42005e0bed649ac205f0fa30b" target='_blank'>Application of artificial neural networks for modeling of electronic excitation dynamics in 2D lattice: Direct and inverse problems</a></td>
          <td>

              Pranas Juknevicius,

              J. Chmeliov,

              L. Valkunas,

              A. Gelžinis,

          </td>
          <td>2023-03-01</td>
          <td>AIP Advances</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/00f39c314542902cd1b03842069e5d3ed441a5c3" target='_blank'>Equivariant message passing for the prediction of tensorial properties and molecular spectra</a></td>
          <td>

              Kristof T. Schütt,

              Oliver T. Unke,

              M. Gastegger,

          </td>
          <td>2021-02-05</td>
          <td>None</td>
          <td>344</td>
          <td>24</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a46b9fe9d83f21b18cf18b350be6484828f7f3e8" target='_blank'>Energy-Inspired Molecular Conformation Optimization</a></td>
          <td>

              Jiaqi Guan,

              Wesley Wei Qian,

              Qiang Liu,

              W. Ma,

              Jianzhu Ma,

              Jian-wei Peng,

          </td>
          <td>None</td>
          <td>None</td>
          <td>13</td>
          <td>29</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f0ba021f6df7cf9578f91449cabb43b6ca310efe" target='_blank'>Transferable Implicit Solvation via Contrastive Learning of Graph Neural Networks</a></td>
          <td>

              J. Airas,

              Xinqiang Ding,

              Bin Zhang,

          </td>
          <td>2023-11-16</td>
          <td>ACS Central Science</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3911bd43fe270429541741b7b5629a4b5ca370f3" target='_blank'>End-to-End Differentiable Molecular Mechanics Force Field Construction</a></td>
          <td>

              Yuanqing Wang,

              Josh Fass,

              J. Chodera,

          </td>
          <td>2020-10-02</td>
          <td>ArXiv</td>
          <td>14</td>
          <td>55</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c210ce694fb7d7c9d1841a0acacc8f472d74137f" target='_blank'>CRANKITE: A fast polypeptide backbone conformation sampler</a></td>
          <td>

              A. Podtelezhnikov,

              D. Wild,

          </td>
          <td>2008-06-24</td>
          <td>Source Code for Biology and Medicine</td>
          <td>17</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0d034a08c130691d20c06618bedc0a950b275489" target='_blank'>Weighted Distance Functions Improve Analysis of High-Dimensional Data: Application to Molecular Dynamics Simulations.</a></td>
          <td>

              Nicolas Blöchliger,

              A. Caflisch,

              A. Vitalis,

          </td>
          <td>2015-11-02</td>
          <td>Journal of chemical theory and computation</td>
          <td>11</td>
          <td>74</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/037f82fa06421cf3a054dac5f97fb002dc0b7f4f" target='_blank'>Learning effective stochastic differential equations from microscopic simulations: combining stochastic numerics and deep learning</a></td>
          <td>

              Felix Dietrich,

              A. Makeev,

              George A. Kevrekidis,

              N. Evangelou,

              Tom S. Bertalan,

              S. Reich,

              I. Kevrekidis,

          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>17</td>
          <td>84</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/28b32daa8d35f44efd38adfb725398cfaa16df8b" target='_blank'>Computational Recipe for Efficient Description of Large-Scale Conformational Changes in Biomolecular Systems</a></td>
          <td>

              Mahmoud Moradi,

              E. Tajkhorshid,

          </td>
          <td>2014-06-03</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>64</td>
          <td>72</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1068197434a8e303802bb090be7284b7bba83a96" target='_blank'>WESTPA 2.0: High-performance upgrades for weighted ensemble simulations and analysis of longer-timescale applications</a></td>
          <td>

              John D. Russo,

              She Zhang,

              Jeremy M. G. Leung,

              Anthony T. Bogetti,

              Jeff P. Thompson,

              A. DeGrave,

              Paul A. Torrillo,

              A. Pratt,

              Kim F. Wong,

              Junchao Xia,

              J. Copperman,

              J. Adelman,

              M. Zwier,

              David N. LeBard,

              D. Zuckerman,

              L. Chong,

          </td>
          <td>2021-12-06</td>
          <td>bioRxiv</td>
          <td>26</td>
          <td>38</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69bab843432803a8f69ffa7c33779f205e843841" target='_blank'>HamNet: Conformation-Guided Molecular Representation with Hamiltonian Neural Networks</a></td>
          <td>

              Ziyao Li,

              Shuwen Yang,

              Guojie Song,

              Lingsheng Cai,

          </td>
          <td>2021-05-08</td>
          <td>ArXiv</td>
          <td>21</td>
          <td>28</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d471bd0b4ad51521932be88a8b0e3c150a2cd78d" target='_blank'>Computational Methods And Experimental Measurements</a></td>
          <td>

              Yvonne Neudorf,

          </td>
          <td>None</td>
          <td></td>
          <td>4</td>
          <td>22</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/379c7e09059cda7f552ba7ec9b42235d8ed9bd72" target='_blank'>Improving molecular machine learning through adaptive subsampling with active learning</a></td>
          <td>

              Yujing Wen,

              Zhixiong Li,

              Yan Xiang,

              D. Reker,

          </td>
          <td>None</td>
          <td>Digital Discovery</td>
          <td>3</td>
          <td>27</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/995bbf6fb5924643786a14271b208a84d2719281" target='_blank'>Utilizing Perturbation of Atoms’ Positions for Equivariant Pre-Training in 3D Molecular Analysis</a></td>
          <td>

              Tal Kiani,

              Avi Caciularu,

              Shani Zev,

              D. Major,

              Jacob Goldberger,

          </td>
          <td>2023-09-17</td>
          <td>2023 IEEE 33rd International Workshop on Machine Learning for Signal Processing (MLSP)</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b82814549a8a0b6e3075952a9b89690812d9cb0" target='_blank'>Exploration of Protein Conformational Change with PELE and Meta-Dynamics.</a></td>
          <td>

              Benjamin P. Cossins,

              A. Hosseini,

              V. Guallar,

          </td>
          <td>2012-02-09</td>
          <td>Journal of chemical theory and computation</td>
          <td>40</td>
          <td>45</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/736598876cb9b057e7206ba04d6a9d6385a89dce" target='_blank'>Efficient Chemical Space Exploration Using Active Learning Based on Marginalized Graph Kernel: an Application for Predicting the Thermodynamic Properties of Alkanes with Molecular Simulation</a></td>
          <td>

              Yan Xiang,

              Yunhao Tang,

              Zheng Gong,

              ,

              Liang Wu,

              Guang Lin,

              Huai Sun,

          </td>
          <td>2022-09-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/be5077a111e8d0ed76b64cfacaaacef126943f10" target='_blank'>Machine-learned dynamic disorder of electron transfer coupling.</a></td>
          <td>

              Yi-Siang Wang,

              Chun-I Wang,

              Chou-hsun Yang,

              Chao‐Ping Hsu,

          </td>
          <td>2023-07-17</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6a6474d5202178a7d4d29c3682c9edd08142b52b" target='_blank'>Machine learning for predicting thermodynamic properties of pure fluids and their mixtures</a></td>
          <td>

              Yuanbin Liu,

              Weixiang Hong,

              B. Cao,

          </td>
          <td>2019-09-09</td>
          <td>Energy</td>
          <td>50</td>
          <td>38</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e9c63cf22cd2504505a0f603a6f0af443f00bbdd" target='_blank'>Dynamics of DNA Breathing and Folding for Molecular Recognition and Computation</a></td>
          <td>

              G. Bonnet,

          </td>
          <td>None</td>
          <td></td>
          <td>2</td>
          <td>5</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bbef626717242515b839b44a2e424bda8cd6d885" target='_blank'>Achieving Rapid Free Energy Estimates from λ-Dynamics with Bias Updated Gibbs Sampling</a></td>
          <td>

              Michael T. Robo,

              Ryan L Hayes,

              Brian Pulawski,

              Jonah Z. Vilseck,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/47883928702efd21c23a4092e65cbcf9f29970aa" target='_blank'>GeoMol: Torsional Geometric Generation of Molecular 3D Conformer Ensembles</a></td>
          <td>

              O. Ganea,

              L. Pattanaik,

              Connor W. Coley,

              R. Barzilay,

              K. Jensen,

              W. Green,

              T. Jaakkola,

          </td>
          <td>2021-06-08</td>
          <td>ArXiv</td>
          <td>91</td>
          <td>111</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3d582fdf2ced2e92832c4ea726f314cdc6def1bd" target='_blank'>Multiscale simulation of plasma flows using active learning.</a></td>
          <td>

              A. Diaw,

              K. Barros,

              J. Haack,

              Christoph Junghans,

              B. Keenan,

              Y. Li,

              D. Livescu,

              N. Lubbers,

              M. McKerns,

              R. Pavel,

              D. Rosenberger,

              I. Sagert,

              T. Germann,

          </td>
          <td>2020-08-01</td>
          <td>Physical review. E</td>
          <td>8</td>
          <td>50</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e337bbc6e92e37d088c89fcd5db06a345d3ee67b" target='_blank'>Machine learning surrogates for molecular dynamics simulations of soft materials</a></td>
          <td>

              J. Kadupitiya,

              Fanbo Sun,

              G. Fox,

              V. Jadhao,

          </td>
          <td>2020-03-12</td>
          <td>J. Comput. Sci.</td>
          <td>30</td>
          <td>55</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2626c0e7e9a35df5c08fbbf2ef057ff809003372" target='_blank'>Enhanced force-field calibration via machine learning</a></td>
          <td>

              Aykut Argun,

              T. Thalheim,

              Stefano Bo,

              F. Cichos,

              G. Volpe,

          </td>
          <td>2020-06-16</td>
          <td>Applied Physics Reviews</td>
          <td>11</td>
          <td>45</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b66fd689144a7ee2a84da624e68e39ca7038111c" target='_blank'>Multiplex Decomposition of Non-Markovian Dynamics and the Hidden Layer Reconstruction Problem</a></td>
          <td>

              L. Lacasa,

              I. P. Mariño,

              J. Míguez,

              V. Nicosia,

              E. Rold'an,

              Ana Lisica,

              S. Grill,

              J. G'omez-Gardenes,

          </td>
          <td>2017-05-12</td>
          <td>Physical Review X</td>
          <td>22</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/05624fbfd7a4359f367f0844f57a4eefc852aaec" target='_blank'>Tensor improve equivariant graph neural network for molecular dynamics prediction.</a></td>
          <td>

              Chi Jiang,

              Yi Zhang,

              Yang Liu,

              Jing Peng,

          </td>
          <td>2024-03-01</td>
          <td>Computational biology and chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/32fd29dc21a187295f366921ff5039ef23280079" target='_blank'>Energy-conserving molecular dynamics is not energy conserving.</a></td>
          <td>

              Lina Zhang,

              Yi-Fan Hou,

              Fuchun Ge,

              Pavlo O. Dral,

          </td>
          <td>2023-08-22</td>
          <td>Physical chemistry chemical physics : PCCP</td>
          <td>3</td>
          <td>23</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7901217fc5f3b690a836274f46a3f06193f646de" target='_blank'>Information bottleneck learns dominant transfer operator eigenfunctions in dynamical systems</a></td>
          <td>

              Matthew S. Schmitt,

              Maciej Koch-Janusz,

              Michel Fruchart,

              Daniel S. Seara,

              Vincenzo Vitelli,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0e067d857a38b0c9cb948fa8da4a82eab383a5bb" target='_blank'>TensorNet: Cartesian Tensor Representations for Efficient Learning of Molecular Potentials</a></td>
          <td>

              Guillem Simeon,

              G. D. Fabritiis,

          </td>
          <td>2023-06-10</td>
          <td>ArXiv</td>
          <td>10</td>
          <td>28</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/735d4099d3be0d919ddedb054043e6763205e0f7" target='_blank'>Energy landscapes for machine learning.</a></td>
          <td>

              A. J. Ballard,

              R. Das,

              Stefano Martiniani,

              D. Mehta,

              Levent Sagun,

              J. Stevenson,

              D. Wales,

          </td>
          <td>2017-05-24</td>
          <td>Physical chemistry chemical physics : PCCP</td>
          <td>88</td>
          <td>76</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ed27c0a1df82dae1550bb083f9a6a3f2ca159212" target='_blank'>Multisecond ligand dissociation dynamics from atomistic simulations</a></td>
          <td>

              Steffen Wolf,

              Benjamin Lickert,

              Simon A. Bray,

              G. Stock,

          </td>
          <td>2020-01-13</td>
          <td>Nature Communications</td>
          <td>48</td>
          <td>53</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/89a1b991a28ce671ecf36d6a4addd284bd846f0e" target='_blank'>Learning Harmonic Molecular Representations on Riemannian Manifold</a></td>
          <td>

              Yiqun Wang,

              Yuning Shen,

              Shih‐Ya Chen,

              Lihao Wang,

              Fei Ye,

              Hao Zhou,

          </td>
          <td>2023-03-27</td>
          <td>ArXiv</td>
          <td>6</td>
          <td>15</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4528895a427d37af5efb5c1d490fe7019ee19d69" target='_blank'>Graph Neural Network with Local Frame for Molecular Potential Energy Surface</a></td>
          <td>

              Xiyuan Wang,

              Muhan Zhang,

          </td>
          <td>2022-08-01</td>
          <td>ArXiv</td>
          <td>7</td>
          <td>18</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e15c46cd5ec654a7a5ff9a19e0943c1b7d32cdf7" target='_blank'>Learning Reactive Islands of the Voter97 System</a></td>
          <td>

              Alexander Hind,

              Stephen Wiggins,

          </td>
          <td>2023-02-01</td>
          <td>Int. J. Bifurc. Chaos</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f6b19727e2901195e9f0c190b552d14b5e06642f" target='_blank'>Bayesian Learning in Undirected Graphical Models: Approximate MCMC Algorithms</a></td>
          <td>

              Iain Murray,

              Zoubin Ghahramani,

          </td>
          <td>2004-07-07</td>
          <td>ArXiv</td>
          <td>115</td>
          <td>110</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/32e65f1af0bb8841787cabe52b548821b6ab8c8a" target='_blank'>Accurate Molecular Dynamics Enabled by Efficient Physically Constrained Machine Learning Approaches</a></td>
          <td>

              Stefan Chmiela,

              H. E. Sauceda,

              A. Tkatchenko,

              Klaus-Robert Muller,

          </td>
          <td>2019-12-13</td>
          <td>Machine Learning Meets Quantum Physics</td>
          <td>9</td>
          <td>76</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/290d0b33a7d52a8c5c3e73fc6e626f3b08ecb3ad" target='_blank'>HAGO-Net: Hierarchical Geometric Massage Passing for Molecular Representation Learning</a></td>
          <td>

              Hongbin Pei,

              Taile Chen,

              Chen A,

              Huiqi Deng,

              Jing Tao,

              Pinghui Wang,

              Xiaohong Guan,

          </td>
          <td>2024-03-24</td>
          <td>None</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/636838221cae02fdb98419bae56b304704b75efe" target='_blank'>Automating the search of molecular motor templates by evolutionary methods</a></td>
          <td>

              Jose D. Fernández,

              F. Vico,

          </td>
          <td>2011-11-01</td>
          <td>Bio Systems</td>
          <td>4</td>
          <td>11</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c31e79a7ad8f4332dc2f3a0a9789911092011f93" target='_blank'>Transfer learning of memory kernels for transferable coarse-graining of polymer dynamics.</a></td>
          <td>

              Zhan Ma,

              Shu Wang,

              Minhee Kim,

              Kaibo Liu,

              Chun-Long Chen,

              Wenxiao Pan,

          </td>
          <td>2021-06-07</td>
          <td>Soft matter</td>
          <td>11</td>
          <td>21</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/369227538d5f6398a09271c8adadc8e4c19270c6" target='_blank'>Learning Invariant Representations of Molecules for Atomization Energy Prediction</a></td>
          <td>

              G. Montavon,

              K. Hansen,

              S. Fazli,

              M. Rupp,

              Franziska Biegler,

              A. Ziehe,

              A. Tkatchenko,

              A. V. Lilienfeld,

              K. Müller,

          </td>
          <td>2012-12-03</td>
          <td>None</td>
          <td>118</td>
          <td>127</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ef8ae5ef66f2e2cf624ca7be272f3bae1e6fdd7e" target='_blank'>A Hierarchical Approach to Scaling Batch Active Search Over Structured Data</a></td>
          <td>

              Vivek Myers,

              Peyton Greenside,

          </td>
          <td>2020-07-20</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/15fd3c562901994dde05ef53b737b948187ec3aa" target='_blank'>Molecular Dynamics with Neural Network Potentials</a></td>
          <td>

              M. Gastegger,

              P. Marquetand,

          </td>
          <td>2018-12-18</td>
          <td>Machine Learning Meets Quantum Physics</td>
          <td>19</td>
          <td>36</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cee2c50c124b5158adcc1ebcb093bd6928665648" target='_blank'>Predicting the entire glassy dynamics from static structure by machine learning relative motion</a></td>
          <td>

              H. Shiba,

              Masatoshi Hanai,

              T. Suzumura,

              T. Shimokawabe,

          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d09cca82577b9c35040af9ea4bc128ad8c867479" target='_blank'>CTAGE: Curvature-Based Topology-Aware Graph Embedding for Learning Molecular Representations</a></td>
          <td>

              Yili Chen,

              Zhengyu Li,

              Zheng Wan,

              Hui Yu,

              Xian Wei,

          </td>
          <td>2023-07-25</td>
          <td>None</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c1357f16a25bcc980469583e7ba1460dd15e68e1" target='_blank'>A Plug-and-Play Quaternion Message-Passing Module for Molecular Conformation Representation</a></td>
          <td>

              Angxiao Yue,

              Dixin Luo,

              Hongteng Xu,

          </td>
          <td>2024-03-24</td>
          <td>None</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8e97ca7244f077dbda40a2264ef8e4eea2ce514d" target='_blank'>3D Molecular Geometry Analysis with 2D Graphs</a></td>
          <td>

              Zhao Xu,

              Yaochen Xie,

              Youzhi Luo,

              Xuan Zhang,

              Xinyi Xu,

              Meng Liu,

              Kaleb Dickerson,

              Cheng Deng,

              Maho Nakata,

              Shuiwang Ji,

          </td>
          <td>2023-05-01</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3b891ce94aa29efbec6bbec6e312e5bd79adfb62" target='_blank'>BOTAN: BOnd TArgeting Network for prediction of slow glassy dynamics by machine learning relative motion.</a></td>
          <td>

              H. Shiba,

              Masatoshi Hanai,

              T. Suzumura,

              T. Shimokawabe,

          </td>
          <td>2022-06-28</td>
          <td>The Journal of chemical physics</td>
          <td>12</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0bb1683f6dea536f228540ffbdd5026ac2c9ea46" target='_blank'>Analysis of Atomistic Representations Using Weighted Skip-Connections</a></td>
          <td>

              K. Nicoli,

              P. Kessel,

              M. Gastegger,

              K. T. Schutt,

          </td>
          <td>2018-10-23</td>
          <td>arXiv: Computational Physics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fa18c4664a007de8cc253c50acb7c166de723187" target='_blank'>Exploring the Transferability of Replica Exchange Structure Reservoirs to Accelerate Generation of Ensembles for Alternate Hamiltonians or Protein Mutations.</a></td>
          <td>

              Koushik Kasavajhala,

              Carlos Simmerling,

          </td>
          <td>2023-03-02</td>
          <td>Journal of chemical theory and computation</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca32b1cd3507cc1953e951ecfb3c1c8bd7646662" target='_blank'>AdaptiveBandit: A multi-armed bandit framework for adaptive sampling in molecular simulations.</a></td>
          <td>

              Adrià Pérez,

              Pablo Herrera-Nieto,

              Stefan Doerr,

              G. de Fabritiis,

          </td>
          <td>2020-02-28</td>
          <td>Journal of chemical theory and computation</td>
          <td>22</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/741835ed93c57a852f2b923ba86e21ce8a2a07c6" target='_blank'>Accelerating Dissociative Events in Molecular Dynamics Simulations by Selective Potential Scaling</a></td>
          <td>

              I. Deb,

              A. Frank,

          </td>
          <td>2019-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/125f4d9b2c004a2b8d9dd2fc5f541805700c7604" target='_blank'>Bayesian Parameter Estimation in Ising and Potts Models : A Comparative Study with Applications to Protein Modeling</a></td>
          <td>

              Xiang-Jun Zhou,

              S. Schmidler,

          </td>
          <td>None</td>
          <td>None</td>
          <td>17</td>
          <td>20</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bcade0826f7ff1b1732cdf05f0a92fe714b9dea2" target='_blank'>The construction of general basis functions in reweighting ensemble dynamics simulations: Reproduce equilibrium distribution in complex systems from multiple short simulation trajectories</a></td>
          <td>

              C. Zhang 张,

              Ming 明 Li 黎,

              X. Zhou 周,

          </td>
          <td>2015-10-20</td>
          <td>Chinese Physics B</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/60e92c63badb9c69bf3fa8a9a2aa753f8dfc6597" target='_blank'>MolNet_Equi: A Chemically Intuitive, Rotation-Equivariant Graph Neural Network.</a></td>
          <td>

              Jihoo Kim,

              Yoonho Jeong,

              W. Kim,

              E. Lee,

              Insung S Choi,

          </td>
          <td>2023-11-12</td>
          <td>Chemistry, an Asian journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cc694135dcb3a9f5948b89fa7531453e463d757e" target='_blank'>ACCELERATING CONVERGENCE OF MOLECULAR DYNAMICS-BASED STRUCTURAL RELAXATION</a></td>
          <td>

              A. Christensen,

          </td>
          <td>2005-02-01</td>
          <td>International Journal of Modern Physics C</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/984756c4091a759a6f982208dd448afbdcfa81ea" target='_blank'>Stochastic Quasi-Newton: application to minimal model for proteins</a></td>
          <td>

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2363d2a5d0749f7b2e5342513e9041cdd6066bb7" target='_blank'>Markov Methods for Hierarchical Coarse-Graining of Large Protein Dynamics</a></td>
          <td>

              C. Chennubhotla,

              I. Bahar,

          </td>
          <td>2006-04-02</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>29</td>
          <td>80</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d633a97b6fb0227eb38c6987d819085132d97c54" target='_blank'>Equivariant Scalar Fields for Molecular Docking with Fast Fourier Transforms</a></td>
          <td>

              Bowen Jing,

              T. Jaakkola,

              Bonnie Berger,

          </td>
          <td>2023-12-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e6cf9471ddc2be7ea523c32f1eab5476776c9fbc" target='_blank'>Greedy Algorithms for Fast Discovery of Macrostates from Molecular Dynamics Simulations</a></td>
          <td>

              Haoyun Feng,

              J. Izaguirre,

              D. Thain,

              Badi Abdul-Wahid,

          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/79410b75e1d2a134ebe436e943e513a004efc4eb" target='_blank'>Euclidean Transformers for Macromolecular Structures: Lessons Learned</a></td>
          <td>

              David Liu,

              L. Melo,

              Allan Costa,

              Martin V¨ogele,

              Raphael J. L. Townshend,

              R. Dror,

          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>78</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8bee16dca1282a6dd43bbf2be3db01154c5a652c" target='_blank'>Geometric Prediction: Moving Beyond Scalars</a></td>
          <td>

              Raphael J. L. Townshend,

              Brent Townshend,

              Stephan Eismann,

              R. Dror,

          </td>
          <td>2020-06-25</td>
          <td>ArXiv</td>
          <td>8</td>
          <td>78</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/271b425f60d895ed3fbf041005937b70a52f6d0a" target='_blank'>Simultaneous Computation of Dynamical and Equilibrium Information Using a Weighted Ensemble of Trajectories</a></td>
          <td>

              Ernesto Suárez,

              S. Lettieri,

              M. Zwier,

              C. Stringer,

              S. Subramanian,

              L. Chong,

              D. Zuckerman,

          </td>
          <td>2012-10-11</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>83</td>
          <td>38</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/33e87d1f2f40edbca1698915bed866fffcddb67f" target='_blank'>Equilibrium distribution from distributed computing (simulations of protein folding).</a></td>
          <td>

              Riccardo Scalco,

              A. Caflisch,

          </td>
          <td>2011-04-25</td>
          <td>The journal of physical chemistry. B</td>
          <td>20</td>
          <td>74</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/af503820df2306d0db32fb358ac8a21b91c4962f" target='_blank'>Equivariant Graph Neural Networks for Toxicity Prediction</a></td>
          <td>

              Julian Cremer,

              L. Medrano Sandonas,

              Alexandre Tkatchenko,

              Djork-Arné Clevert,

              G. de Fabritiis,

          </td>
          <td>2023-09-10</td>
          <td>Chemical Research in Toxicology</td>
          <td>8</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5e92c2fa618f7606ee71a9c72bee53d1b6fa8b5d" target='_blank'>An enumerative stepwise ansatz enables atomic-accuracy RNA loop modeling</a></td>
          <td>

              Parin Sripakdeevong,

              W. Kladwang,

              Rhiju Das,

          </td>
          <td>2011-12-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>69</td>
          <td>62</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/99f319ab72ae43f0ac94fe7ff1891be524611483" target='_blank'>Relevance of Rotationally Equivariant Convolutions for Predicting Molecular Properties</a></td>
          <td>

              Benjamin Kurt Miller,

              M. Geiger,

              T. Smidt,

              Frank No'e,

          </td>
          <td>2020-08-19</td>
          <td>ArXiv</td>
          <td>66</td>
          <td>22</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9d34a6d0e77834548c60b23f719677333e62b38f" target='_blank'>Numerical computation of effective thermal equilibria in stochastically switching Langevin systems.</a></td>
          <td>

              ,

              K. Newhall,

          </td>
          <td>2021-04-27</td>
          <td>Physical review. E</td>
          <td>7</td>
          <td>8</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0c8265a9543f6172c2980579dceedf188b20b520" target='_blank'>3D Equivariant Molecular Graph Pretraining</a></td>
          <td>

              Rui Jiao,

              Jiaqi Han,

              Wenbing Huang,

              Yu Rong,

              Yang Liu,

          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>14</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bce034b098679aaee6535ef8964abe6b88c89384" target='_blank'>Accelerating chemical similarity search using GPUs and metric embeddings</a></td>
          <td>

              I. Haque,

          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5645dfeae81d6d5bc7c5648267837e1e0696100d" target='_blank'>Molecular Geometry Pretraining with SE(3)-Invariant Denoising Distance Matching</a></td>
          <td>

              Shengchao Liu,

              Hongyu Guo,

              Jian Tang,

          </td>
          <td>2022-06-27</td>
          <td>ArXiv</td>
          <td>50</td>
          <td>39</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/330e6f3ffe59faeb042ae94be1d56ab1cea148f3" target='_blank'>Efficient and Equivariant Graph Networks for Predicting Quantum Hamiltonian</a></td>
          <td>

              Haiyang Yu,

              Zhao Xu,

              X. Qian,

              Xiaoning Qian,

              Shuiwang Ji,

          </td>
          <td>2023-06-08</td>
          <td>None</td>
          <td>8</td>
          <td>57</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c9c2e461a57b7b5e51a547f739174760f2ad7c2e" target='_blank'>Model-free inference of memory in conformational dynamics of a multi-domain protein</a></td>
          <td>

              L. Vollmar,

              Rick Bebon,

              J. Schimpf,

              Bastian Flietel,

              Sirin Celiksoy,

              Carsten Sonnichsen,

              Aljavz Godec,

              Thorsten Hugel,

          </td>
          <td>2024-04-25</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9453ff830c6593d2283aa2b7e02e05f81cef8320" target='_blank'>Is Distance Matrix Enough for Geometric Deep Learning?</a></td>
          <td>

              Zian Li,

              Xiyuan Wang,

              Yinan Huang,

              Muhan Zhang,

          </td>
          <td>2023-02-11</td>
          <td>ArXiv</td>
          <td>6</td>
          <td>18</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f8079b38314b9a78e2558d26126ed649e9c3d2dc" target='_blank'>Spherical Message Passing for 3D Molecular Graphs</a></td>
          <td>

              Yi Liu,

              Limei Wang,

              Meng Liu,

              Xuan Zhang,

              Bora Oztekin,

              Shuiwang Ji,

          </td>
          <td>2021-02-09</td>
          <td>None</td>
          <td>149</td>
          <td>57</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6bc4004347cdf76d84597210796f38fcf7a01a80" target='_blank'>Directional Message Passing for Molecular Graphs</a></td>
          <td>

              Johannes Klicpera,

              Janek Groß,

              Stephan Günnemann,

          </td>
          <td>2020-03-06</td>
          <td>ArXiv</td>
          <td>654</td>
          <td>45</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ab233d1a846eefe42639b30f6d701abc05aba170" target='_blank'>Multi-Agent Reinforcement Learning-based Adaptive Sampling for Conformational Sampling of Proteins</a></td>
          <td>

              D. Kleiman,

              D. Shukla,

          </td>
          <td>2022-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d4a3580e391b625fd081c8d297a18f99f11e86cb" target='_blank'>Multiagent Reinforcement Learning-Based Adaptive Sampling for Conformational Dynamics of Proteins.</a></td>
          <td>

              D. Kleiman,

              D. Shukla,

          </td>
          <td>2022-08-31</td>
          <td>Journal of chemical theory and computation</td>
          <td>14</td>
          <td>33</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/124ec00db85c0cc7059f6aba76764d22d3f5d463" target='_blank'>Equivariant Graph Hierarchy-Based Neural Networks</a></td>
          <td>

              J. Han,

              Yu Rong,

              Tingyang Xu,

              Fuchun Sun,

              Wenbing Huang,

          </td>
          <td>2022-02-22</td>
          <td>ArXiv</td>
          <td>10</td>
          <td>30</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b61fe62b2cf63a499f96205d6314f6bf70762a6d" target='_blank'>Combining the multicanonical ensemble with generative probabilistic models of local biomolecular structure</a></td>
          <td>

              J. Frellsen,

              T. Hamelryck,

              J. Ferkinghoff-Borg,

          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr>
          <td><a href="https://www.semanticscholar.org/paper/498fa3707397a05fdfff97bd23bab039622d9a94" target='_blank'>Computing the binding affinity of a ligand buried deep inside a protein with the hybrid steered molecular dynamics.</a></td>
          <td>

              Oscar D Villarreal,

              Lili Yu,

              Roberto A. Rodriguez,

              Liao Y. Chen,

          </td>
          <td>2017-01-29</td>
          <td>Biochemical and biophysical research communications</td>
          <td>14</td>
          <td>14</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    // columnDefs: [
    //     {"className": "dt-center", "targets": "_all"},
    //     // set width for both columns 0 and 1 as 25%
    //     { width: '40%', targets: 4 }

    //   ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
</script>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '25%', targets: 1 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>