---
hide:
 - navigation
---
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-05-13 06:07:43 UTC</i>
  </p>
  

  <p>
  <h3>Recommendations for the article <i>Molecular latent space simulators</i></h3>
  <table id="table1" class="display" style="width:100%">
  <thead>
    <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bab880c698b850ba18c08b9d93b73ffb538a417b" target='_blank'>Accelerated Simulations of Molecular Systems through Learning of their Effective Dynamics</a></td>
          <td>
            
              Pantelis R. Vlachas,
            
              J. Zavadlav,
            
              M. Praprotnik,
            
              P. Koumoutsakos,
            
          </td>
          <td>2021-02-17</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>77</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/68c0f4d535078674fd2ce738306eb72171a0677d" target='_blank'>Accelerated Simulations of Molecular Systems through Learning of Effective Dynamics.</a></td>
          <td>
            
              Pantelis R. Vlachas,
            
              J. Zavadlav,
            
              M. Praprotnik,
            
              P. Koumoutsakos,
            
          </td>
          <td>2021-12-10</td>
          <td>Journal of chemical theory and computation</td>
          <td>26</td>
          <td>77</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d09f8cd520789fbd378a7bd90dcafc2544b6fcfc" target='_blank'>Simple synthetic molecular dynamics for efficient trajectory generation</a></td>
          <td>
            
              John D. Russo,
            
              D. Zuckerman,
            
          </td>
          <td>2022-04-09</td>
          <td>None</td>
          <td>1</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0cd3637bf739405263a78687d2f1ddaaa801cd89" target='_blank'>Latent Space Simulator for Unveiling Molecular Free Energy Landscapes and Predicting Transition Dynamics</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5c6ee980b791766ba036cba282bcbcbcb2e5309e" target='_blank'>Deep Generative Markov State Models</a></td>
          <td>
            
              Hao Wu,
            
              Andreas Mardt,
            
              Luca Pasquali,
            
              F. Noé,
            
          </td>
          <td>2018-05-19</td>
          <td>ArXiv</td>
          <td>53</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58912e2c2aaa77d1448d51e9d9460e06a5b924b9" target='_blank'>VAMPnets for deep learning of molecular kinetics</a></td>
          <td>
            
              Andreas Mardt,
            
              Luca Pasquali,
            
              Hao Wu,
            
              F. Noé,
            
          </td>
          <td>2017-10-16</td>
          <td>Nature Communications</td>
          <td>452</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9896835f050121358372254a9bce57ca117672c8" target='_blank'>Boltzmann Generators - Sampling Equilibrium States of Many-Body Systems with Deep Learning</a></td>
          <td>
            
              F. Noé,
            
              Hao Wu,
            
          </td>
          <td>2018-12-04</td>
          <td>ArXiv</td>
          <td>20</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/13b52cd927d3daa5c95528b987734d97474e790c" target='_blank'>Markov field models: Scaling molecular kinetics approaches to large molecular machines.</a></td>
          <td>
            
              Tim Hempel,
            
              Simon Olsson,
            
              Frank No'e,
            
          </td>
          <td>2022-06-23</td>
          <td>Current opinion in structural biology</td>
          <td>6</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/31d62c95e128caa934ca6a6c5cd5c766c6a9d541" target='_blank'>Molecular dynamics without molecules: searching the conformational space of proteins with generative neural networks</a></td>
          <td>
            
              Gregory Schwing,
            
              L. Palese,
            
              Ariel Fern'andez,
            
              L. Schwiebert,
            
              D. Gatti,
            
          </td>
          <td>2022-06-09</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cb4d8f8b2b1169d210d289c6b2deac0ccbbc34fe" target='_blank'>An Information Bottleneck Approach for Markov Model Construction</a></td>
          <td>
            
              Dedi Wang,
            
              Yunrui Qiu,
            
              E. Beyerle,
            
              Xuhui Huang,
            
              P. Tiwary,
            
          </td>
          <td>2024-04-03</td>
          <td>None</td>
          <td>0</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92da30c4137270487bc51fec1aaee97b17f4e2ac" target='_blank'>Machine Learning Generation of Dynamic Protein Conformational Ensembles</a></td>
          <td>
            
              Li-E Zheng,
            
              Shrishti Barethiya,
            
              Erik B. Nordquist,
            
              Jianhan Chen,
            
          </td>
          <td>2023-05-01</td>
          <td>Molecules</td>
          <td>5</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4afa2c93c386bc87c4fa4339709e2b788a6ed32a" target='_blank'>Everything you wanted to know about Markov State Models but were afraid to ask.</a></td>
          <td>
            
              V. Pande,
            
              Kyle A. Beauchamp,
            
              G. Bowman,
            
          </td>
          <td>2010-09-01</td>
          <td>Methods</td>
          <td>578</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/86de5fbe09c4381f548e1312d53f15d898c76c78" target='_blank'>Mind reading of the proteins: Deep-learning to forecast molecular dynamics</a></td>
          <td>
            
              C. Gupta,
            
              J. Cava,
            
              Daipayan Sarkar,
            
              Eric A. Wilson,
            
              J. Vant,
            
              Steven Murray,
            
              A. Singharoy,
            
              S. Karmaker,
            
          </td>
          <td>2020-07-29</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/03f062024f5212a0cdb048a927ec0c5aae2fd757" target='_blank'>Extract Latent Features of Single Particle Trajectories with Historical Experience Learning.</a></td>
          <td>
            
              Yongyu Zhang,
            
              Feng Ge,
            
              Xijian Lin,
            
              Jianfeng Xue,
            
              Yuxin Song,
            
              Hao Xie,
            
              Yan He,
            
          </td>
          <td>2023-10-01</td>
          <td>Biophysical journal</td>
          <td>1</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c4d9b7166a2ac580c195d72f64489b3f4e0b9c23" target='_blank'>DeepWEST: Deep Learning of Kinetic Models with the Weighted Ensemble Simulation Toolkit for Enhanced Sampling.</a></td>
          <td>
            
              Anupam Anand Ojha,
            
              Saumya Thakur,
            
              Surl-Hee Ahn,
            
              Rommie E. Amaro,
            
          </td>
          <td>2023-01-31</td>
          <td>Journal of chemical theory and computation</td>
          <td>7</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/31739ea55a7c86ed38a1030bb99625ef96f9fc7a" target='_blank'>Manifold learning in atomistic simulations: a conceptual review</a></td>
          <td>
            
              J. Rydzewski,
            
              Ming Chen,
            
              O. Valsson,
            
          </td>
          <td>2023-03-15</td>
          <td>Machine Learning: Science and Technology</td>
          <td>7</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9868cca5b562f4ea41bff4b5ee4ea63e62006430" target='_blank'>Learning dynamical information from static protein and sequencing data</a></td>
          <td>
            
              P. Pearce,
            
              Francis G. Woodhouse,
            
              Aden Forrow,
            
              Ashley J. Kelly,
            
              H. Kusumaatmaja,
            
              J. Dunkel,
            
          </td>
          <td>2018-08-27</td>
          <td>Nature Communications</td>
          <td>12</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/538d0f3f70c90dfd8d9f6e7bba3f50ea4058c34c" target='_blank'>High-Resolution Markov State Models for the Dynamics of Trp-Cage Miniprotein Constructed over Slow Folding Modes Identified by State-Free Reversible VAMPnets.</a></td>
          <td>
            
              Hythem Sidky,
            
              ,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2019-06-12</td>
          <td>The journal of physical chemistry. B</td>
          <td>44</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/41c0783182ca3e23732b95ddfdb6654cd23a4b45" target='_blank'>Deep learning to decompose macromolecules into independent Markovian domains</a></td>
          <td>
            
              Andreas Mardt,
            
              Tim Hempel,
            
              C. Clementi,
            
              F. Noé,
            
          </td>
          <td>2022-10-13</td>
          <td>Nature Communications</td>
          <td>13</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aa1146c1db4b205be456323fe657ae54848f9dd4" target='_blank'>Building insightful, memory-enriched models to capture long-time biochemical processes from short-time simulations</a></td>
          <td>
            
              Anthony J. Dominic,
            
              T. Sayer,
            
              Siqin Cao,
            
              T. Markland,
            
              Xuhui Huang,
            
              Andrés Montoya-Castillo,
            
          </td>
          <td>2022-10-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>14</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f3ebef5245ffe1c8e4ccfa334c86cf2f0d9f4ed5" target='_blank'>An Introduction to Markov State Models and Their Application to Long Timescale Molecular Simulation</a></td>
          <td>
            
              G. Bowman,
            
              V. Pande,
            
              F. Noé,
            
          </td>
          <td>None</td>
          <td></td>
          <td>486</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/93013a0ad444641953c01cd8182a84e1dff378a0" target='_blank'>What Markov state models can and cannot do: Correlation versus path-based observables in protein folding models</a></td>
          <td>
            
              Ernesto Suárez,
            
              Rafal P. Wiewiora,
            
              C. Wehmeyer,
            
              F. Noé,
            
              J. Chodera,
            
              D. Zuckerman,
            
          </td>
          <td>2020-11-09</td>
          <td>bioRxiv</td>
          <td>38</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2e7163e31e9b32cec11005678bae9e1dbeb6d573" target='_blank'>Nonlinear Discovery of Slow Molecular Modes using Hierarchical Dynamics Encoders</a></td>
          <td>
            
              ,
            
              Hythem Sidky,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2019-02-09</td>
          <td>The Journal of chemical physics</td>
          <td>74</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1c4c2983bc3719c80d93e39e59c58dd4c5a99afb" target='_blank'>Variational deep learning of equilibrium transition path ensembles.</a></td>
          <td>
            
              A. Singh,
            
              David T. Limmer,
            
          </td>
          <td>2023-02-28</td>
          <td>The Journal of chemical physics</td>
          <td>3</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d03d605aa500b66f4ac1f9ae486c56474ae62c94" target='_blank'>Markov State Models to Study the Functional Dynamics of Proteins in the Wake of Machine Learning</a></td>
          <td>
            
              K. Konovalov,
            
              I. C. Unarta,
            
              Siqin Cao,
            
              Eshani C. Goonetilleke,
            
              Xuhui Huang,
            
          </td>
          <td>2021-08-04</td>
          <td>JACS Au</td>
          <td>47</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/260f212da0cf8b3309f0ab3a130a0ad22993892e" target='_blank'>Markov State Models: From an Art to a Science.</a></td>
          <td>
            
              B. Husic,
            
              V. Pande,
            
          </td>
          <td>2018-02-02</td>
          <td>Journal of the American Chemical Society</td>
          <td>557</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e4140d67d59c61de86d2a9de31380e01a07ef85b" target='_blank'>Past–future information bottleneck for sampling molecular reaction coordinate simultaneously with thermodynamics and kinetics</a></td>
          <td>
            
              Yihang Wang,
            
              João Marcelo Lamim Ribeiro,
            
              P. Tiwary,
            
          </td>
          <td>2019-08-08</td>
          <td>Nature Communications</td>
          <td>118</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78042c8517d29ca1890d02635ce75edfc4ce1c7f" target='_blank'>Deep Generative Model Driven Protein Folding Simulations</a></td>
          <td>
            
              Heng Ma,
            
              D. Bhowmik,
            
              Hyungro Lee,
            
              M. Turilli,
            
              M. T. Young,
            
              S. Jha,
            
              A. Ramanathan,
            
          </td>
          <td>2019-08-01</td>
          <td>None</td>
          <td>9</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/25f9e33fae4be74e769b65524071e4fd60b3781c" target='_blank'>Machine-guided path sampling to discover mechanisms of molecular self-organization</a></td>
          <td>
            
              Hendrik Jung,
            
              Roberto Covino,
            
              A. Arjun,
            
              Christian Leitold,
            
              Christoph Dellago,
            
              P. Bolhuis,
            
              G. Hummer,
            
          </td>
          <td>2021-05-14</td>
          <td>Nature Computational Science</td>
          <td>42</td>
          <td>95</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/18d50aee34266bbe7d77e3b50bd0c38f729c8437" target='_blank'>Molecular Free Energies, Rates, and Mechanisms from Data-Efficient Path Sampling Simulations</a></td>
          <td>
            
              Gianmarco Lazzeri,
            
              Hendrik Jung,
            
              P. Bolhuis,
            
              Roberto Covino,
            
          </td>
          <td>2023-07-20</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>7</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b674a7aee72e9b9cc5390eca13f9c5c7812f2ba0" target='_blank'>Machine learning for molecular simulation</a></td>
          <td>
            
              F. Noé,
            
              A. Tkatchenko,
            
              K. Müller,
            
              C. Clementi,
            
          </td>
          <td>2019-11-07</td>
          <td>Annual review of physical chemistry</td>
          <td>486</td>
          <td>127</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/55705bec7256bb2f3ceec805823de4e87bee87f1" target='_blank'>Machine Learning for Molecular Dynamics on Long Timescales</a></td>
          <td>
            
              F. Noé,
            
          </td>
          <td>2018-12-18</td>
          <td>ArXiv</td>
          <td>30</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/68b03ad0ed6c1c9ac968fa4c5ae66c3020dc783e" target='_blank'>Multiscale simulations of complex systems by learning their effective dynamics</a></td>
          <td>
            
              Pantelis R. Vlachas,
            
              G. Arampatzis,
            
              Caroline Uhler,
            
              P. Koumoutsakos,
            
          </td>
          <td>2020-06-24</td>
          <td>Nature Machine Intelligence</td>
          <td>87</td>
          <td>77</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5b10349eade93332fbff9127b5273382103fdf27" target='_blank'>Deep Learning for Multi-Scale Molecular Modeling</a></td>
          <td>
            
              ,
            
              Yao-Kun Lei,
            
              Yi Isaac Yang,
            
              Y. Gao,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/02a0504b526587d96e40d2b7d8d384385762ff2d" target='_blank'>PyEMMA 2: A Software Package for Estimation, Validation, and Analysis of Markov Models.</a></td>
          <td>
            
              Martin K. Scherer,
            
              Benjamin Trendelkamp-Schroer,
            
              Fabian Paul,
            
              G. Pérez-Hernández,
            
              Moritz Hoffmann,
            
              Nuria Plattner,
            
              C. Wehmeyer,
            
              Jan-Hendrik Prinz,
            
              F. Noé,
            
          </td>
          <td>2015-10-14</td>
          <td>Journal of chemical theory and computation</td>
          <td>813</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/47171992793ccd578bd34a222537e71bc8d0def8" target='_blank'>Learning protein conformational space by enforcing physics with convolutions and latent interpolations</a></td>
          <td>
            
              Venkatakrishnan Ramaswamy,
            
              Chris G. Willcocks,
            
              M. Degiacomi,
            
          </td>
          <td>2019-10-10</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/83e8e8d89e64de17d3d4709051a1f799c0b9ae22" target='_blank'>Deep learning for variational multiscale molecular modeling.</a></td>
          <td>
            
              Jun Zhang,
            
              Yao-Kun Lei,
            
              Yi Isaac Yang,
            
              Y. Gao,
            
          </td>
          <td>2020-06-29</td>
          <td>The Journal of chemical physics</td>
          <td>11</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/377b191510fe23f3bc1728dff00ed3db55852efb" target='_blank'>RiteWeight: Randomized Iterative Trajectory Reweighting for Steady-State Distributions Without Discretization Error</a></td>
          <td>
            
              Sagar Kania,
            
              D. Aristoff,
            
              Daniel M. Zuckerman,
            
          </td>
          <td>2024-01-11</td>
          <td>None</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca17f2865ee2459f41c221164fb193502948693b" target='_blank'>An overview and practical guide to building Markov state models.</a></td>
          <td>
            
              G. Bowman,
            
          </td>
          <td>None</td>
          <td>Advances in experimental medicine and biology</td>
          <td>21</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d1516aeab1a504cc02237fdd5957c9b421498110" target='_blank'>Deep Learning Unravels a Dynamic Hierarchy While Empowering Molecular Dynamics Simulations</a></td>
          <td>
            
              Ariel Fernández,
            
          </td>
          <td>2020-02-13</td>
          <td>Annalen der Physik</td>
          <td>2</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d00252df8465a93ba68dda823d8adb8e36418560" target='_blank'>Analyzing Molecular Dynamics Trajectories Thermodynamically through Artificial Intelligence.</a></td>
          <td>
            
              Xuyang Liu,
            
              Jingya Xing,
            
              Haohao Fu,
            
              Xueguang Shao,
            
              W. Cai,
            
          </td>
          <td>2024-01-09</td>
          <td>Journal of chemical theory and computation</td>
          <td>2</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/267db6e5e036b0f4bccc9e9b06085288638e31f2" target='_blank'>Deep learning path-like collective variable for enhanced sampling molecular dynamics</a></td>
          <td>
            
              Thorben Frohlking,
            
              Luigi Bonati,
            
              Valerio Rizzi,
            
              Francesco L. Gervasio,
            
          </td>
          <td>2024-02-02</td>
          <td>None</td>
          <td>2</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69e96a63ffd74a6c816203329c8a54794ac1ca00" target='_blank'>Transferable Kinetic Monte Carlo Models with Thousands of Reactions Learned from Molecular Dynamics Simulations.</a></td>
          <td>
            
              Enze Chen,
            
              Qian Yang,
            
              Vincent Dufour-Décieux,
            
              C. Sing-Long,
            
              Rodrigo Freitas,
            
              E. Reed,
            
          </td>
          <td>2019-02-08</td>
          <td>The journal of physical chemistry. A</td>
          <td>8</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3f3ee15122e47f65ac0580ec8b00e9cedac75983" target='_blank'>Restricted Boltzmann machines : from compositional representations to protein sequence analysis</a></td>
          <td>
            
              J. Tubiana,
            
          </td>
          <td>2018-11-29</td>
          <td></td>
          <td>8</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/728649fcdf03de6cbb023c2ecb6cd132019cbadf" target='_blank'>Long-time methods for molecular dynamics simulations: Markov State Models and Milestoning.</a></td>
          <td>
            
              Brajesh Narayan,
            
              Ye Yuan,
            
              A. Fathizadeh,
            
              R. Elber,
            
              Nicolae-Viorel Buchete,
            
          </td>
          <td>None</td>
          <td>Progress in molecular biology and translational science</td>
          <td>5</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/df7b1ea2a9d6174ce83a0b04a709e2e5ea0a89e7" target='_blank'>Independent Markov decomposition: Toward modeling kinetics of biomolecular complexes</a></td>
          <td>
            
              Tim Hempel,
            
              Mauricio J. del Razo,
            
              Christopher T. Lee,
            
              Bryn C. Taylor,
            
              Rommie E. Amaro,
            
              F. Noé,
            
          </td>
          <td>2021-03-25</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>15</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2eb1ed9d4baf548391d8220c0a8a3d0be79845ed" target='_blank'>Boltzmann generators: Sampling equilibrium states of many-body systems with deep learning</a></td>
          <td>
            
              F. Noé,
            
              Jonas Köhler,
            
              Hao Wu,
            
          </td>
          <td>2019-09-05</td>
          <td>Science</td>
          <td>491</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fad4374a6cf7ca4f4f0af89bfd6519a4747efd61" target='_blank'>Generative models of conformational dynamics.</a></td>
          <td>
            
              C. Langmead,
            
          </td>
          <td>None</td>
          <td>Advances in experimental medicine and biology</td>
          <td>3</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9b70bf13b098d5388ed3acbaedfc7835c2de06fd" target='_blank'>An Efficient Path Classification Algorithm Based on Variational Autoencoder to Identify Metastable Path Channels for Complex Conformational Changes.</a></td>
          <td>
            
              Yunrui Qiu,
            
              Michael S O'Connor,
            
              Mingyi Xue,
            
              Bojun Liu,
            
              Xuhui Huang,
            
          </td>
          <td>2023-06-29</td>
          <td>Journal of chemical theory and computation</td>
          <td>8</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8c9f4af01a841b431917e605a85746154d49529d" target='_blank'>Spectral Maps for Learning Reduced Representations of Molecular Systems</a></td>
          <td>
            
              Tuugcce Gokdemir,
            
              Jakub Rydzewski,
            
          </td>
          <td>2023-11-07</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cfb331b1cb54859cc236fc797e99d5886e055e81" target='_blank'>Learning the intrinsic dynamics of spatio-temporal processes through Latent Dynamics Networks.</a></td>
          <td>
            
              Francesco Regazzoni,
            
              S. Pagani,
            
              M. Salvador,
            
              L. Dede’,
            
              A. Quarteroni,
            
          </td>
          <td>2024-02-28</td>
          <td>Nature communications</td>
          <td>1</td>
          <td>78</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6eb318112bafc0c1d2dbcda0b4b34812ee4c92b9" target='_blank'>Latent Dynamics Networks (LDNets): learning the intrinsic dynamics of spatio-temporal processes</a></td>
          <td>
            
              Francesco Regazzoni,
            
              S. Pagani,
            
              M. Salvador,
            
              L. Dede’,
            
              A. Quarteroni,
            
          </td>
          <td>2023-04-28</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>78</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2646b021a73a0651c4dac42e99d757319bfa0832" target='_blank'>Learning Protein Dynamics with Metastable Switching Systems</a></td>
          <td>
            
              Bharath Ramsundar,
            
              V. Pande,
            
          </td>
          <td>2016-10-05</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e89b8240b54e2beccb99d7f6c82ee369b4af91c4" target='_blank'>Learning generative models of molecular dynamics</a></td>
          <td>
            
              N. Razavian,
            
              Hetunandan Kamisetty,
            
              C. Langmead,
            
          </td>
          <td>2012-01-17</td>
          <td>BMC Genomics</td>
          <td>8</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e89b8240b54e2beccb99d7f6c82ee369b4af91c4" target='_blank'>Learning generative models of molecular dynamics</a></td>
          <td>
            
              N. Razavian,
            
              Hetunandan Kamisetty,
            
              C. Langmead,
            
          </td>
          <td>2012-01-17</td>
          <td>BMC Genomics</td>
          <td>8</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/38596c90a7da2fd6544d46a1136630ccbc26234b" target='_blank'>Boltzmann Generators and the New Frontier of Computational Sampling in Many-Body Systems</a></td>
          <td>
            
              Alessandro Coretti,
            
              Sebastian Falkner,
            
              J. Weinreich,
            
              Christoph Dellago,
            
              O. V. Lilienfeld,
            
          </td>
          <td>2024-04-25</td>
          <td>None</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5d310f49cd92b2b3a0c716592f6903fd9deb55c1" target='_blank'>Progress and challenges in the automated construction of Markov state models for full protein systems.</a></td>
          <td>
            
              G. Bowman,
            
              Kyle A. Beauchamp,
            
              G. Boxer,
            
              V. Pande,
            
          </td>
          <td>2009-09-28</td>
          <td>The Journal of chemical physics</td>
          <td>359</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f8701f745cb0a5871c21f86e2045b190103c230c" target='_blank'>Deeptime: a Python library for machine learning dynamical models from time series data</a></td>
          <td>
            
              Moritz Hoffmann,
            
              Martin K. Scherer,
            
              Tim Hempel,
            
              Andreas Mardt,
            
              Brian de Silva,
            
              B. Husic,
            
              Stefan Klus,
            
              Hao Wu,
            
              N. Kutz,
            
              S. Brunton,
            
              F. Noé,
            
          </td>
          <td>2021-10-28</td>
          <td>Machine Learning: Science and Technology</td>
          <td>56</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8bd7b6f622911c5f68e7dd19fc002074bde9722e" target='_blank'>Representation of Protein Dynamics Disentangled by Time-structure-based Prior</a></td>
          <td>
            
              Tsuyoshi Ishizone,
            
              Yasuhiro Matsunaga,
            
              Sotaro Fuchigami,
            
              Kazuyuki Nakamura,
            
          </td>
          <td>2023-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/712080b7b8d296e61827d6c738303ec6ed0d44c1" target='_blank'>Generation of conformational ensembles of small molecules via surrogate model-assisted molecular dynamics</a></td>
          <td>
            
              Juan Viguera Diez,
            
              Sara Romeo Atance,
            
              Ola Engkvist,
            
              Simon Olsson,
            
          </td>
          <td>2024-04-05</td>
          <td>Mach. Learn. Sci. Technol.</td>
          <td>1</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ea8ae9fda8336d6f288ee21ce947ed567c6dee3" target='_blank'>RPnet: A Reverse Projection Based Neural Network for Coarse-graining Metastable Conformational States for Protein Dynamics</a></td>
          <td>
            
              Hanlin Gu,
            
              Wei Wang,
            
              Siqin Cao,
            
              I. C. Unarta,
            
              Y. Yao,
            
              F. Sheong,
            
              Xuhui Huang,
            
          </td>
          <td>2021-08-06</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/39fd60798d23d5514548f92fa881f520905f0bd0" target='_blank'>Learning Compositional Representations of Interacting Systems with Restricted Boltzmann Machines: Comparative Study of Lattice Proteins</a></td>
          <td>
            
              J. Tubiana,
            
              S. Cocco,
            
              R. Monasson,
            
          </td>
          <td>2019-02-18</td>
          <td>Neural Computation</td>
          <td>24</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0129beb25ff82f128116b35cdb45aff8f1456255" target='_blank'>Top-Down Machine Learning of Coarse-Grained Protein Force Fields</a></td>
          <td>
            
              Carles Navarro,
            
              Maciej Majewski,
            
              G. D. Fabritiis,
            
          </td>
          <td>2023-06-20</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>3</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/12ba80c62d718cfee6f5dffa356178cbd408e397" target='_blank'>Predictive Collective Variable Discovery with Deep Bayesian Models</a></td>
          <td>
            
              M. Schöberl,
            
              N. Zabaras,
            
              P. Koutsourelakis,
            
          </td>
          <td>2018-09-18</td>
          <td>The Journal of chemical physics</td>
          <td>32</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4244b9b2515e37a20b254fa8f9ed7305f2f64d7e" target='_blank'>Interpretable embeddings from molecular simulations using Gaussian mixture variational autoencoders</a></td>
          <td>
            
              Yasemin Bozkurt Varolgunes,
            
              T. Bereau,
            
              J. F. Rudzinski,
            
          </td>
          <td>2019-12-22</td>
          <td>Machine Learning: Science and Technology</td>
          <td>36</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9ebf5c7ac8fc22961e7dc2130776a9b7d92d6d94" target='_blank'>Learning reduced kinetic Monte Carlo models of complex chemistry from molecular dynamics</a></td>
          <td>
            
              Qian Yang,
            
              C. Sing-Long,
            
              E. Reed,
            
          </td>
          <td>2017-06-19</td>
          <td>Chemical Science</td>
          <td>19</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/464a4804060a4692e2d7a72be0042a2ea621e3b8" target='_blank'>LAST: Latent Space-Assisted Adaptive Sampling for Protein Trajectories</a></td>
          <td>
            
              Hao Tian,
            
              Xi Jiang,
            
              Sian Xiao,
            
              Hunter La Force,
            
              Eric C. Larson,
            
              Peng Tao,
            
          </td>
          <td>2022-04-27</td>
          <td>Journal of chemical information and modeling</td>
          <td>5</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ac055a76189bda2cd0c47998a74bcfc9bfa4a7cb" target='_blank'>Transferable deep generative modeling of intrinsically disordered protein conformations</a></td>
          <td>
            
              Giacomo Janson,
            
              M. Feig,
            
          </td>
          <td>2024-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2cfc26b4af99f195b433fa7aa00f221b111c7cd4" target='_blank'>Illuminating protein space with a programmable generative model</a></td>
          <td>
            
              John Ingraham,
            
              Max Baranov,
            
              Zak Costello,
            
              Vincent Frappier,
            
              Ahmed Ismail,
            
              Shan Tie,
            
              Wujie Wang,
            
              Vincent Xue,
            
              F. Obermeyer,
            
              Andrew L Beam,
            
              G. Grigoryan,
            
          </td>
          <td>2022-12-02</td>
          <td>Nature</td>
          <td>133</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0107d0f47f7d1ccf290538922728dcece452c0b5" target='_blank'>Note: Variational Encoding of Protein Dynamics Benefits from Maximizing Latent Autocorrelation</a></td>
          <td>
            
              H. Wayment-Steele,
            
              V. Pande,
            
          </td>
          <td>2018-03-17</td>
          <td>The Journal of chemical physics</td>
          <td>6</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7cb50b79a1eec42efd2fec611b9cdc5d08ca936c" target='_blank'>Machine learning transforms how microstates are sampled</a></td>
          <td>
            
              M. Tuckerman,
            
          </td>
          <td>2019-09-05</td>
          <td>Science</td>
          <td>10</td>
          <td>70</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/915f23b0ab17625cec9863dc2eee1d603dcac80c" target='_blank'>Learning Markovian dynamics with spectral maps.</a></td>
          <td>
            
              Jakub Rydzewski,
            
              Tuğçe Gökdemir,
            
          </td>
          <td>2023-11-28</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2b624accabd4b40bb341ba70038686c043efcace" target='_blank'>Multiensemble Markov models of molecular thermodynamics and kinetics</a></td>
          <td>
            
              Hao Wu,
            
              Fabian Paul,
            
              C. Wehmeyer,
            
              Frank Noé,
            
          </td>
          <td>2016-05-25</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>64</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6209957354367e3da4c9f978d93b0acfbf4d3dc8" target='_blank'>Deep Momentum Multi-Marginal Schrödinger Bridge</a></td>
          <td>
            
              T. Chen,
            
              Guan-Horng Liu,
            
              Molei Tao,
            
              E. Theodorou,
            
          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/925edfff606df315acd5d2fa1db693dd4dbe91fa" target='_blank'>Automated detection of many-particle solvation states for accurate characterizations of diffusion kinetics.</a></td>
          <td>
            
              J. F. Rudzinski,
            
              M. Radu,
            
              T. Bereau,
            
          </td>
          <td>2018-10-08</td>
          <td>The Journal of chemical physics</td>
          <td>2</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f96f0a5039a1dce39c08b65f27176251e539ec29" target='_blank'>Learning Coarse-Grained Particle Latent Space with Auto-Encoders</a></td>
          <td>
            
              Wujie Wang,
            
              Rafael Gómez-Bombarelli,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/390c937c44ffb10a9552d275fe8bf6d869b8a1cd" target='_blank'>Enhanced Sampling with Machine Learning: A Review</a></td>
          <td>
            
              S. Mehdi,
            
              Zachary Smith,
            
              Lukas Herron,
            
              Ziyue Zou,
            
              P. Tiwary,
            
          </td>
          <td>2023-06-15</td>
          <td>Annual review of physical chemistry</td>
          <td>7</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4275490b2442d0f4bfd56ec5f494e8c81543c0bb" target='_blank'>Past–future information bottleneck framework for simultaneously sampling biomolecular reaction coordinate, thermodynamics and kinetics</a></td>
          <td>
            
              Yihang Wang,
            
              João Marcelo Lamim Ribeiro,
            
              P. Tiwary,
            
          </td>
          <td>2018-12-28</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/90e7dfe131b21c12c2998ec0b87010ac6eb4d337" target='_blank'>Two for One: Diffusion Models and Force Fields for Coarse-Grained Molecular Dynamics</a></td>
          <td>
            
              Marloes Arts,
            
              Victor Garcia Satorras,
            
              Chin-Wei Huang,
            
              Daniel Zuegner,
            
              M. Federici,
            
              C. Clementi,
            
              Frank No'e,
            
              Robert Pinsler,
            
              Rianne van den Berg,
            
          </td>
          <td>2023-02-01</td>
          <td>Journal of chemical theory and computation</td>
          <td>34</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a1625eaec70f1efce37a4d0c251e8e5df4cfadeb" target='_blank'>Fusing Neural and Physical: Augment Protein Conformation Sampling with Tractable Simulations</a></td>
          <td>
            
              Jiarui Lu,
            
              Zuobai Zhang,
            
              Bozitao Zhong,
            
              Chence Shi,
            
              Jian Tang,
            
          </td>
          <td>2024-02-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5303dffbc43db94c5acc804b87b76d84eb81dd13" target='_blank'>Reinforcement Learning for Multi-Scale Molecular Modeling</a></td>
          <td>
            
              Jun Zhang,
            
              Yao-Kun Lei,
            
              Yi Isaac Yang,
            
              Y. Gao,
            
          </td>
          <td>2019-08-19</td>
          <td>None</td>
          <td>1</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e70ed4cec2382a01413e6353cc8406cc0faa5993" target='_blank'>Markov State Models: To Optimize or Not to Optimize</a></td>
          <td>
            
              Robert E Arbon,
            
              Yanchen Zhu,
            
              A. Mey,
            
          </td>
          <td>2024-01-01</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/82ee174bc312d3960b865b0946ce9c86f1a10154" target='_blank'>Phanto-IDP: compact model for precise intrinsically disordered protein backbone generation and enhanced sampling</a></td>
          <td>
            
              Junjie Zhu,
            
              Zhengxin Li,
            
              Haowei Tong,
            
              Zhouyu Lu,
            
              Ningjie Zhang,
            
              Ting Wei,
            
              Hai-Feng Chen,
            
          </td>
          <td>2023-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>1</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/26a186b235fe13fec788cb5a3129a38e55e0be1e" target='_blank'>Generative modeling of short, disordered proteins with homogeneous sequence composition</a></td>
          <td>
            
              Ishan Taneja,
            
              Keren Lasker,
            
          </td>
          <td>2023-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/50c74a10fcddb0e41a5a00b965383fdc52534eec" target='_blank'>Machine Learning in Molecular Dynamics Simulations of Biomolecular Systems</a></td>
          <td>
            
              Christopher Kolloff,
            
              Simon Olsson,
            
          </td>
          <td>2022-05-06</td>
          <td>None</td>
          <td>2</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4af4996d0dd209282e25f88ef7f280ef54b21be7" target='_blank'>Reaction coordinate flows for model reduction of molecular kinetics</a></td>
          <td>
            
              Hao Wu,
            
              Frank No'e,
            
          </td>
          <td>2023-09-11</td>
          <td>The Journal of chemical physics</td>
          <td>2</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/446681b1a3b3cafc7522cffdc3550348480d4085" target='_blank'>Operator Autoencoders: Learning Physical Operations on Encoded Molecular Graphs</a></td>
          <td>
            
              Willis Hoke,
            
              D. Shea,
            
              S. Casey,
            
          </td>
          <td>2021-05-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94ce2a11cb11cd4c1a467c1796b3680e857c51c5" target='_blank'>Conformation Clustering of Long MD Protein Dynamics with an Adversarial Autoencoder</a></td>
          <td>
            
              Yunlong Liu,
            
              L. Amzel,
            
          </td>
          <td>2018-05-31</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cdae73da6322cbb9f4a0945aa4e68055184d466d" target='_blank'>Neural Continuous-Time Markov Models</a></td>
          <td>
            
              Majerle Reeves,
            
              H. Bhat,
            
          </td>
          <td>2022-12-11</td>
          <td>2023 SICE International Symposium on Control Systems (SICE ISCS)</td>
          <td>0</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/86e3440fd4fdf47fd93779678c5d6d01b1b246ab" target='_blank'>Transition Path Sampling with Boltzmann Generator-based MCMC Moves</a></td>
          <td>
            
              Michael Plainer,
            
              Hannes Stärk,
            
              Charlotte Bunne,
            
              Stephan Günnemann,
            
          </td>
          <td>2023-12-08</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ba4973004694ed2684b0ddc518ff47cf7fb79e83" target='_blank'>Deep-Learning-Assisted Enhanced Sampling for Exploring Molecular Conformational Changes.</a></td>
          <td>
            
              Haohao Fu,
            
              Han Liu,
            
              Jingya Xing,
            
              Tong Zhao,
            
              Xueguang Shao,
            
              W. Cai,
            
          </td>
          <td>2023-11-10</td>
          <td>The journal of physical chemistry. B</td>
          <td>3</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/80d28e926b3cd25b9c298e82c404b20aa0b15789" target='_blank'>Embedded-physics machine learning for coarse-graining and collective variable discovery without data</a></td>
          <td>
            
              M. Schöberl,
            
              N. Zabaras,
            
              P. Koutsourelakis,
            
          </td>
          <td>2020-02-24</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92e8942259e1b34172f8dbe5818575b43e89977e" target='_blank'>Concurrent parametrization against static and kinetic information leads to more robust coarse-grained force fields</a></td>
          <td>
            
              J. F. Rudzinski,
            
              T. Bereau,
            
          </td>
          <td>2016-07-18</td>
          <td>The European Physical Journal Special Topics</td>
          <td>17</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b167898f46597d3944a7bb611f53ea9110d68b53" target='_blank'>Modeling Global Dynamics from Local Snapshots with Deep Generative Neural Networks</a></td>
          <td>
            
              Scott A. Gigante,
            
              David van Dijk,
            
              Kevin R. Moon,
            
              Alexander Strzalkowski,
            
              Guy Wolf,
            
              Smita Krishnaswamy,
            
          </td>
          <td>2018-02-10</td>
          <td>2019 13th International conference on Sampling Theory and Applications (SampTA)</td>
          <td>1</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/204cedd41db9db3623226e9dfce6d82aaa3f0f41" target='_blank'>Machine learning for collective variable discovery and enhanced sampling in biomolecular simulation</a></td>
          <td>
            
              Hythem Sidky,
            
              Wei Chen,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2020-03-03</td>
          <td>Molecular Physics</td>
          <td>87</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/29a5becddc15df49b06dcff52d919d84d3998175" target='_blank'>Machine learning phase space quantum dynamics approaches.</a></td>
          <td>
            
              Xinzijian Liu,
            
              Linfeng Zhang,
            
              Jian Liu,
            
          </td>
          <td>2021-05-11</td>
          <td>The Journal of chemical physics</td>
          <td>9</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c2f962fea9a69cd42c93c85f1c01078c2232358e" target='_blank'>Application of Molecular-Dynamics Based Markov State Models to Functional Proteins</a></td>
          <td>
            
              R. D. Malmstrom,
            
              Christopher T. Lee,
            
              Adam T. Van Wart,
            
              Rommie E. Amaro,
            
          </td>
          <td>2014-06-04</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>87</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3f98344f6fc1fe8402a3f24943590e37a74873d9" target='_blank'>Deep learning Markov and Koopman models with physical constraints</a></td>
          <td>
            
              Andreas Mardt,
            
              Luca Pasquali,
            
              Frank No'e,
            
              Hao Wu,
            
          </td>
          <td>2019-12-16</td>
          <td>None</td>
          <td>25</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f3eebe7cf446380258a42654978a9483e0325edf" target='_blank'>A Note on Learning Rare Events in Molecular Dynamics using LSTM and Transformer</a></td>
          <td>
            
              Wenqi Zeng,
            
              Siqin Cao,
            
              Xuhui Huang,
            
              Y. Yao,
            
          </td>
          <td>2021-07-14</td>
          <td>ArXiv</td>
          <td>9</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cae2adb6642a38e2dd80ee975a65b26788a891c6" target='_blank'>From Latent Dynamics to Meaningful Representations.</a></td>
          <td>
            
              Dedi Wang,
            
              Yihang Wang,
            
              Luke J. Evans,
            
              P. Tiwary,
            
          </td>
          <td>2022-09-02</td>
          <td>Journal of chemical theory and computation</td>
          <td>0</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3999faeea4b6fc58615245072ce010a0e6f641ea" target='_blank'>Systematic control of collective variables learned from variational autoencoders.</a></td>
          <td>
            
              Jacob I Monroe,
            
              V. K. Shen,
            
          </td>
          <td>2022-08-09</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/923fab47b5c4e5a90219322ed0e7b1b7b94adc7b" target='_blank'>Variational approximation of molecular kinetics from short off-equilibrium simulations</a></td>
          <td>
            
              Hao Wu,
            
              F. Nüske,
            
              Fabian Paul,
            
              Stefan Klus,
            
              P. Koltai,
            
              F. Noé,
            
          </td>
          <td>2016-10-20</td>
          <td></td>
          <td>3</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7f02647a9029d7f65da697a8bc943971c034695b" target='_blank'>P RO GAE: A G EOMETRIC A UTOENCODER - BASED G ENERATIVE M ODEL FOR D ISENTANGLING P ROTEIN C ONFORMATIONAL S PACE</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/23d06160c110fb916901768f964ffe53d1ffc762" target='_blank'>Conditional Normalizing Flows for Active Learning of Coarse-Grained Molecular Representations</a></td>
          <td>
            
              Henrik Schopmans,
            
              Pascal Friederich,
            
          </td>
          <td>2024-02-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/14e2346e73e34683ee27c0abfeb1112d82d9c2f6" target='_blank'>Action Matching: Learning Stochastic Dynamics from Samples</a></td>
          <td>
            
              Kirill Neklyudov,
            
              Rob Brekelmans,
            
              Daniel de Souza Severo,
            
              Alireza Makhzani,
            
          </td>
          <td>2022-10-13</td>
          <td>None</td>
          <td>14</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/744249d8846dad1cccaeb835119c39c9a10f079b" target='_blank'>A deep encoder-decoder framework for identifying distinct ligand binding pathways.</a></td>
          <td>
            
              Satyabrata Bandyopadhyay,
            
              Jagannath Mondal,
            
          </td>
          <td>2023-05-15</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f1748ef585758622ea82d64e4ba5674ec0636d2b" target='_blank'>Exploring the conformational ensembles of protein-protein complex with transformer-based generative model</a></td>
          <td>
            
              Jianmin Wang,
            
              Xun Wang,
            
              Yanyi Chu,
            
              Chunyan Li,
            
              Xue Li,
            
              Xiangyu Meng,
            
              Yitian Fang,
            
              Kyoung Tai No,
            
              Jiashun Mao,
            
              Xiangxiang Zeng,
            
          </td>
          <td>2024-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4813cd2feebbcef748d23902884261f10fb553e2" target='_blank'>Machine Learning Deciphered Molecular Mechanistics with Accurate Kinetic and Thermodynamic Prediction.</a></td>
          <td>
            
              Junlin Dong,
            
              Shiyu Wang,
            
              Wenqiang Cui,
            
              Xiaolin Sun,
            
              Haojie Guo,
            
              Hailu Yan,
            
              Horst Vogel,
            
              Zhi Wang,
            
              Shuguang Yuan,
            
          </td>
          <td>2024-02-23</td>
          <td>Journal of chemical theory and computation</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/59de694649b05d03e65b89af1606cd33e5bc9624" target='_blank'>An efficient Bayesian kinetic lumping algorithm to identify metastable conformational states via Gibbs sampling.</a></td>
          <td>
            
              Wei Wang,
            
              Tong Liang,
            
              F. Sheong,
            
              Xiaodan Fan,
            
              Xuhui Huang,
            
          </td>
          <td>2018-08-21</td>
          <td>The Journal of chemical physics</td>
          <td>20</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f0c7016f60733519429247cefdedde5ae00daf6f" target='_blank'>Machine learning approaches for biological function and dynamics</a></td>
          <td>
            
              P. Schwander,
            
          </td>
          <td>2022-07-29</td>
          <td>Acta Crystallographica Section A Foundations and Advances</td>
          <td>0</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a86717a0af0e90830183b221a9473d235da96698" target='_blank'>Data-driven learning of the generalized Langevin equation with state-dependent memory</a></td>
          <td>
            
              Pei Ge,
            
              Zhongqiang Zhang,
            
              Huan Lei,
            
          </td>
          <td>2023-10-28</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3c64b1d8a81ab02c1b4c097cf3a0329df20c1840" target='_blank'>Dynamic graphical models of molecular kinetics</a></td>
          <td>
            
              S. Olsson,
            
              F. Noé,
            
          </td>
          <td>2018-11-09</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>26</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/21a4afa74caf78e28880b7fe4c30a09a5f9c14e3" target='_blank'>Direct generation of protein conformational ensembles via machine learning</a></td>
          <td>
            
              G. Janson,
            
              Gilberto Valdés-García,
            
              Lim Heo,
            
              M. Feig,
            
          </td>
          <td>2022-06-19</td>
          <td>Nature Communications</td>
          <td>40</td>
          <td>56</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/827ca2f16ded73b2df1f760eebb2f1c5c9e5b44b" target='_blank'>Timewarp: Transferable Acceleration of Molecular Dynamics by Learning Time-Coarsened Dynamics</a></td>
          <td>
            
              Leon Klein,
            
              Andrew Y. K. Foong,
            
              T. E. Fjelde,
            
              Bruno Mlodozeniec,
            
              Marc Brockschmidt,
            
              Sebastian Nowozin,
            
              Frank No'e,
            
              Ryota Tomioka,
            
          </td>
          <td>2023-02-02</td>
          <td>ArXiv</td>
          <td>15</td>
          <td>50</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d59b094223687b58007e00de8c01fabab8b6c04b" target='_blank'>Conditioning Boltzmann generators for rare event sampling</a></td>
          <td>
            
              S. Falkner,
            
              Alessandro Coretti,
            
              Salvatore Romano,
            
              P. Geissler,
            
              C. Dellago,
            
          </td>
          <td>2023-08-30</td>
          <td>Machine Learning: Science and Technology</td>
          <td>4</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/125b2ac4050584a85eac86c9f85399f1ef85d71c" target='_blank'>Learning molecular dynamics with simple language model built upon long short-term memory neural network</a></td>
          <td>
            
              Sun-Ting Tsai,
            
              En-Jui Kuo,
            
              P. Tiwary,
            
          </td>
          <td>2020-04-26</td>
          <td>Nature Communications</td>
          <td>71</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/65aaf814b43994af4bb920982a2c496a011cb4a0" target='_blank'>MSMBuilder2: Modeling Conformational Dynamics at the Picosecond to Millisecond Scale.</a></td>
          <td>
            
              Kyle A. Beauchamp,
            
              G. Bowman,
            
              T. Lane,
            
              L. Maibaum,
            
              I. Haque,
            
              V. Pande,
            
          </td>
          <td>2011-10-11</td>
          <td>Journal of chemical theory and computation</td>
          <td>358</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/984ca90a7b3912db88b02a4cf8fdfef4626f7752" target='_blank'>Modeling Molecular Structures with Intrinsic Diffusion Models</a></td>
          <td>
            
              Gabriele Corso,
            
          </td>
          <td>2023-02-23</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e1834d63294230b37fb4e0709b8d9bfca640d70c" target='_blank'>Deep clustering of protein folding simulations</a></td>
          <td>
            
              D. Bhowmik,
            
              Shang Gao,
            
              M. T. Young,
            
              A. Ramanathan,
            
          </td>
          <td>2018-06-12</td>
          <td>BMC Bioinformatics</td>
          <td>65</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a083d9b69ac6d5d6fc25487401529bbe15f443ee" target='_blank'>On the Approximation Quality of Markov State Models</a></td>
          <td>
            
              M. Sarich,
            
              F. Noé,
            
              C. Schütte,
            
          </td>
          <td>2010-05-28</td>
          <td>Multiscale Model. Simul.</td>
          <td>170</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3333f169bd4815f92c03c9445761d8267c4e9ef2" target='_blank'>Simulate Time-integrated Coarse-grained Molecular Dynamics with Multi-Scale Graph Networks</a></td>
          <td>
            
              Xiang Fu,
            
              T. Xie,
            
              Nathan J. Rebello,
            
              B. Olsen,
            
              T. Jaakkola,
            
          </td>
          <td>2022-04-21</td>
          <td>None</td>
          <td>3</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/885fc7dde40ccada93f28c560af8e6d2a1531262" target='_blank'>Statistically Optimal Force Aggregation for Coarse-Graining Molecular Dynamics.</a></td>
          <td>
            
              Andreas Krämer,
            
              Aleksander E. P. Durumeric,
            
              N. Charron,
            
              Yaoyi Chen,
            
              C. Clementi,
            
              Frank No'e,
            
          </td>
          <td>2023-02-14</td>
          <td>The journal of physical chemistry letters</td>
          <td>11</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cfbdb4cb9ae6d79d0f41fc12ace856be86976cfc" target='_blank'>Enspara: Modeling molecular ensembles with scalable data structures and parallel computing</a></td>
          <td>
            
              J. Porter,
            
              Maxwell I. Zimmerman,
            
              Gregory R. Bowman,
            
          </td>
          <td>2018-09-29</td>
          <td>bioRxiv</td>
          <td>46</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8f771f2fc3b8f446e7a1f1a15fb158a34966e294" target='_blank'>Explore Protein Conformational Space With Variational Autoencoder</a></td>
          <td>
            
              Hao Tian,
            
              Xi Jiang,
            
              Francesco Trozzi,
            
              Sian Xiao,
            
              Eric C. Larson,
            
              Peng Tao,
            
          </td>
          <td>2021-09-23</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>25</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5028002b12d9a83e297f040517ea412240559f18" target='_blank'>Abstraction of Markov Population Dynamics via Generative Adversarial Nets</a></td>
          <td>
            
              Francesca Cairoli,
            
              Ginevra Carbone,
            
              L. Bortolussi,
            
          </td>
          <td>2021-06-24</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5b8403be6597123db420112c1b1e6bca6351e475" target='_blank'>Algorithms for building models of molecular motion from simulations</a></td>
          <td>
            
              N. Hinrichs,
            
          </td>
          <td>None</td>
          <td></td>
          <td>5</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d3d51ccff69b92a784a2e234d39f1f2c950bf1e1" target='_blank'>Machine learning approaches for analyzing and enhancing molecular dynamics simulations.</a></td>
          <td>
            
              Yihang Wang,
            
              João Marcelo Lamim Ribeiro,
            
              P. Tiwary,
            
          </td>
          <td>2019-09-25</td>
          <td>Current opinion in structural biology</td>
          <td>168</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6b506cab4e4bd23ef4d751e97592457a4dbb370c" target='_blank'>Recent advances in protein conformation sampling by combining machine learning with molecular simulation</a></td>
          <td>
            
              Yiming Tang,
            
              Zhongyuan Yang,
            
              Yifei Yao,
            
              Yun Zhou,
            
              Yuan Tan,
            
              Zichao Wang,
            
              Tong Pan,
            
              Rui Xiong,
            
              Junli Sun,
            
              Guanghong Wei,
            
          </td>
          <td>2024-01-04</td>
          <td>Chinese Physics B</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f8a6341496701a0d7fdaa50e6a5aa157236a0ff8" target='_blank'>Author Correction: VAMPnets for deep learning of molecular kinetics</a></td>
          <td>
            
              Andreas Mardt,
            
              Luca Pasquali,
            
              Hao Wu,
            
              F. Noé,
            
          </td>
          <td>2018-10-22</td>
          <td>Nature Communications</td>
          <td>24</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bfc3fe8484bf3ea63dfaf4a3b9ff0855ac1253c1" target='_blank'>Efficient Continuous-Time Markov Chain Estimation</a></td>
          <td>
            
              Monir Hajiaghayi,
            
              Bonnie Kirkpatrick,
            
              Liangliang Wang,
            
              A. Bouchard-Côté,
            
          </td>
          <td>2013-09-12</td>
          <td>None</td>
          <td>30</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3706ddb1442605d71180833f73aee9a07c6f8412" target='_blank'>Data-Efficient Generation of Protein Conformational Ensembles with Backbone-to-Side-Chain Transformers.</a></td>
          <td>
            
              Shriram Chennakesavalu,
            
              Grant M. Rotskoff,
            
          </td>
          <td>2023-11-19</td>
          <td>The journal of physical chemistry. B</td>
          <td>2</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6408d2f50d8b2ed2dd9446b1a97d32b29a76ef68" target='_blank'>Accurate Sampling of Macromolecular Conformations Using Adaptive Deep Learning and Coarse-Grained Representation</a></td>
          <td>
            
              Amr H. Mahmoud,
            
              Matthew Masters,
            
              Soo-Jung Lee,
            
              M. Lill,
            
          </td>
          <td>2022-03-30</td>
          <td>Journal of chemical information and modeling</td>
          <td>9</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6bec80cf9dcdd2719ff738a3ea91d14837882256" target='_blank'>Enhanced modeling via network theory: Adaptive sampling of Markov state models.</a></td>
          <td>
            
              G. Bowman,
            
              D. Ensign,
            
              V. Pande,
            
          </td>
          <td>2010-02-17</td>
          <td>Journal of chemical theory and computation</td>
          <td>207</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4e3b2f677e03c624d9aed77a58fd596cebf452b3" target='_blank'>Sequence network 1 D 2 D Structure network Invariant features A C OrientationsDistances B weights features Energy</a></td>
          <td>
            
              John Ingraham,
            
              Adam J. Riesselman,
            
              C. Sander,
            
              D. Marks,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/732d462555dd5c9d6c639e89eb9a21e3d133bdc3" target='_blank'>DeepDriveMD: Deep-Learning Driven Adaptive Molecular Simulations for Protein Folding</a></td>
          <td>
            
              Hyungro Lee,
            
              Heng Ma,
            
              M. Turilli,
            
              D. Bhowmik,
            
              S. Jha,
            
              A. Ramanathan,
            
          </td>
          <td>2019-09-17</td>
          <td>2019 IEEE/ACM Third Workshop on Deep Learning on Supercomputers (DLS)</td>
          <td>54</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f97964fd13dfccf136dfca52a9d9e4840e34211b" target='_blank'>Predicting new protein conformations from molecular dynamics simulation conformational landscapes and machine learning</a></td>
          <td>
            
              Yiming Jin,
            
              L. Johannissen,
            
              Sam Hay,
            
          </td>
          <td>2020-08-06</td>
          <td>Proteins: Structure</td>
          <td>20</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/404b4fbf580f6c9dfb340603618e9840cd5e760f" target='_blank'>Chapter 12. Machine Learning Algorithms for the Analysis of Molecular Dynamics Trajectories</a></td>
          <td>
            
              Abbas Saleem Khan,
            
              B. Sim,
            
              Dongqing Wei,
            
          </td>
          <td>2021-09-24</td>
          <td>Theoretical and Computational Chemistry Series</td>
          <td>0</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ede032d42942ba24a39696e61cc65294e1142737" target='_blank'>Differentiable rotamer sampling with molecular force fields</a></td>
          <td>
            
              Congzhou M. Sha,
            
              Jian Wang,
            
              N. Dokholyan,
            
          </td>
          <td>2023-02-22</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1182b9d85119627ceafc8a73c05a319810a89b56" target='_blank'>Automated Deep Abstractions for Stochastic Chemical Reaction Networks</a></td>
          <td>
            
              Tatjana Petrov,
            
              D. Repin,
            
          </td>
          <td>2020-01-30</td>
          <td>ArXiv</td>
          <td>5</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/efabfeafb866f7922986a1d7b44d406341c19a72" target='_blank'>TrustMol: Trustworthy Inverse Molecular Design via Alignment with Molecular Dynamics</a></td>
          <td>
            
              Kevin Tirta Wijaya,
            
              Navid Ansari,
            
              Hans-Peter Seidel,
            
              Vahid Babaei,
            
          </td>
          <td>2024-02-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/33c04427b6fcd265da08f80fe8f279486f11d44a" target='_blank'>Machine learning of microscopic structure-dynamics relationships in complex molecular systems</a></td>
          <td>
            
              M. Crippa,
            
              Annalisa Cardellini,
            
              M. Cioni,
            
              Gábor Csányi,
            
              G. Pavan,
            
          </td>
          <td>2023-08-31</td>
          <td>Machine Learning: Science and Technology</td>
          <td>1</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6bb5eeb7728656d61987b98aceb9cc4579a6f705" target='_blank'>A deep autoencoder framework for discovery of metastable ensembles in biomacromolecules.</a></td>
          <td>
            
              Satyabrata Bandyopadhyay,
            
              Jagannath Mondal,
            
          </td>
          <td>2021-06-01</td>
          <td>The Journal of chemical physics</td>
          <td>10</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3e2c07792a60338f29a5cfe012da2bdc99a69687" target='_blank'>A Machine Learning Approach to Solving Biochemical Systems</a></td>
          <td>
            
              Joshua Ologbonyo,
            
              Emmanuel O. Adara,
            
              R. Sidje,
            
          </td>
          <td>2024-02-05</td>
          <td>2024 IEEE First International Conference on Artificial Intelligence for Medicine, Health and Care (AIMHC)</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3dbd0ff97315f3d70f5c0209068e394aa5476c6a" target='_blank'>Score Dynamics: Scaling Molecular Dynamics with Picoseconds Time Steps via Conditional Diffusion Model</a></td>
          <td>
            
              Tim Hsu,
            
              Babak Sadigh,
            
              Vasily V. Bulatov,
            
              Fei Zhou,
            
          </td>
          <td>2023-10-02</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/936594a9115e6a3cac5e6208e294d32c6d030db7" target='_blank'>Barriers and Dynamical Paths in Alternating Gibbs Sampling of Restricted Boltzmann Machines</a></td>
          <td>
            
              C. Roussel,
            
              S. Cocco,
            
              R. Monasson,
            
          </td>
          <td>2021-07-13</td>
          <td>Physical review. E</td>
          <td>12</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/90fecaa7210c2c1321b478c1bf782a675c0e5d17" target='_blank'>Continuous Graphical Models for Static and Dynamic Distributions: Application to Structural Biology</a></td>
          <td>
            
              N. Razavian,
            
          </td>
          <td>None</td>
          <td></td>
          <td>2</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/32882132f465a4f93f0112028fe4cbbe3d2279f9" target='_blank'>Steps and bumps: precision extraction of discrete states of molecular machines.</a></td>
          <td>
            
              Max A. Little,
            
              B. Steel,
            
              F. Bai,
            
              Y. Sowa,
            
              Thomas Bilyard,
            
              D. Mueller,
            
              R. Berry,
            
              N. S. Jones,
            
          </td>
          <td>2010-04-08</td>
          <td>Biophysical journal</td>
          <td>31</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/56ef38a3e74aabfa0bbd161896a581f0c7e6d218" target='_blank'>Learning dynamic Boltzmann distributions as reduced models of spatial chemical kinetics.</a></td>
          <td>
            
              Oliver K. Ernst,
            
              T. Bartol,
            
              T. Sejnowski,
            
              E. Mjolsness,
            
          </td>
          <td>2018-03-02</td>
          <td>The Journal of chemical physics</td>
          <td>9</td>
          <td>156</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a452aff825339a1917b6529e4056242e58acbf67" target='_blank'>Understanding protein folding using Markov state models.</a></td>
          <td>
            
              V. Pande,
            
          </td>
          <td>None</td>
          <td>Advances in experimental medicine and biology</td>
          <td>11</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d5a2ae8f201dd9f48062ad001f14514000acf359" target='_blank'>MSMBuilder: Statistical Models for Biomolecular Dynamics</a></td>
          <td>
            
              M. Harrigan,
            
              Mohammad M. Sultan,
            
              Carlos X. Hernández,
            
              B. Husic,
            
              P. Eastman,
            
              Christian Schwantes,
            
              Kyle A. Beauchamp,
            
              R. McGibbon,
            
              V. Pande,
            
          </td>
          <td>2016-10-30</td>
          <td>bioRxiv</td>
          <td>215</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c871d2dc802d276608a6734637f8bc9e6da0d837" target='_blank'>GeoDiff: a Geometric Diffusion Model for Molecular Conformation Generation</a></td>
          <td>
            
              Minkai Xu,
            
              Lantao Yu,
            
              Yang Song,
            
              Chence Shi,
            
              Stefano Ermon,
            
              Jian Tang,
            
          </td>
          <td>2022-03-06</td>
          <td>ArXiv</td>
          <td>310</td>
          <td>73</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c63657b2e54419a908221ea878df708d78a6409c" target='_blank'>Denoising diﬀusion probabilistic models for replica exchange</a></td>
          <td>
            
              Yihang Wang,
            
              P. Tiwary,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/06b0a8c66ba5a4da7363116e7f4d65f1703d7e31" target='_blank'>Deep learning the slow modes for rare events sampling</a></td>
          <td>
            
              L. Bonati,
            
              G. Piccini,
            
              M. Parrinello,
            
          </td>
          <td>2021-07-08</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>107</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c16df0a90a1d4480ccd41f43ec175d0bc39ca830" target='_blank'>Statistical Computation for Problems in Dynamic Systems and Protein Folding</a></td>
          <td>
            
              Samuel W. K. Wong,
            
          </td>
          <td>2013-08-21</td>
          <td></td>
          <td>2</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6cd2dd33c7bdad838f93d3fa23f3b758423b8228" target='_blank'>Fast and Functional Structured Data Generators Rooted in Out-of-Equilibrium Physics</a></td>
          <td>
            
              A. Carbone,
            
              A. Decelle,
            
              Lorenzo Rosset,
            
              B. Seoane,
            
          </td>
          <td>2023-07-13</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b77379f7a2d28804a9021bf7cc18982e86d1c415" target='_blank'>Generative Coarse-Graining of Molecular Conformations</a></td>
          <td>
            
              Wujie Wang,
            
              Minkai Xu,
            
              Chen Cai,
            
              Benjamin Kurt Miller,
            
              T. Smidt,
            
              Yusu Wang,
            
              Jian Tang,
            
              R. G'omez-Bombarelli,
            
          </td>
          <td>2022-01-28</td>
          <td>None</td>
          <td>23</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/060d35318a5ef9637b5c1be4c2917b14277bd338" target='_blank'>Modeling Ensembles of Enzyme Reaction Pathways with Hi-MSM Reveals the Importance of Accounting for Pathway Diversity.</a></td>
          <td>
            
              Joseph R Persichetti,
            
              Yang Jiang,
            
              Phillip S. Hudson,
            
              E. O’Brien,
            
          </td>
          <td>2022-11-16</td>
          <td>The journal of physical chemistry. B</td>
          <td>0</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/08e1e14df71651bb15302fdbc1a9ef7b31705630" target='_blank'>Modeling Dynamics of Biological Systems with Deep Generative Neural Networks</a></td>
          <td>
            
              Scott A. Gigante,
            
              D. V. Dijk,
            
              Kevin R. Moon,
            
              Alexander Strzalkowski,
            
              Guy Wolf,
            
              Smita Krishnaswamy,
            
          </td>
          <td>2018-09-27</td>
          <td>arXiv: Learning</td>
          <td>3</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7562e5747ce73b36b57f51d53f7e25d6875ab57a" target='_blank'>Simulate Time-integrated Coarse-grained Molecular Dynamics with Geometric Machine Learning</a></td>
          <td>
            
              Xiang Fu,
            
              T. Xie,
            
              Nathan J. Rebello,
            
              B. Olsen,
            
              T. Jaakkola,
            
          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>14</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/535803259d7b37e7f024c97dc15f48884bb6ba80" target='_blank'>Stochastic Lag Time Parameterization for Markov State Models of Protein Dynamics.</a></td>
          <td>
            
              Shiqi Gong,
            
              Xinheng He,
            
              Qi Meng,
            
              Zhiming Ma,
            
              Bin Shao,
            
              Tong Wang,
            
              Tie-Yan Liu,
            
          </td>
          <td>2022-11-08</td>
          <td>The journal of physical chemistry. B</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7869928bc7c5809b05760367acc2eb5fd2a0e8c7" target='_blank'>Physics-aware, probabilistic model order reduction with guaranteed stability</a></td>
          <td>
            
              Sebastian Kaltenbach,
            
              P. Koutsourelakis,
            
          </td>
          <td>2021-01-14</td>
          <td>ArXiv</td>
          <td>12</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a6e2e1189e3f888d1a01ca535ff9b3dad560b6d0" target='_blank'>High-fidelity molecular dynamics trajectory reconstruction with bi-directional neural networks</a></td>
          <td>
            
              Ludwig Winkler,
            
              Klaus-Robert Müller,
            
              H. E. Sauceda,
            
          </td>
          <td>2022-01-02</td>
          <td>Machine Learning: Science and Technology</td>
          <td>8</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4c1aca2b6c597bcb913d36c4624ada4bf9c4b2aa" target='_blank'>Deep Learning Moment Closure Approximations using Dynamic Boltzmann Distributions</a></td>
          <td>
            
              Oliver K. Ernst,
            
              T. Bartol,
            
              T. Sejnowski,
            
              E. Mjolsness,
            
          </td>
          <td>2019-05-28</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>156</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4321a76a5a4ac4b06763a80773fa00df4fdc2376" target='_blank'>Learning stochastic dynamics and predicting emergent behavior using transformers</a></td>
          <td>
            
              Corneel Casert,
            
              Isaac Tamblyn,
            
              S. Whitelam,
            
          </td>
          <td>2022-02-17</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3b97d1e21e19862a1372cdfc3a910e83d6c3354e" target='_blank'>Prediction of good reaction coordinates and future evolution of MD trajectories using Regularized Sparse Autoencoders: A novel deep learning approach</a></td>
          <td>
            
              Abhijit Gupta,
            
          </td>
          <td>2022-08-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8bfb1b701c801fc8fa470379c4a86341a9ea36e0" target='_blank'>Protein Structure and Sequence Generation with Equivariant Denoising Diffusion Probabilistic Models</a></td>
          <td>
            
              N. Anand,
            
              Tudor Achim,
            
          </td>
          <td>2022-05-26</td>
          <td>ArXiv</td>
          <td>132</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/91e8d167b9a27d3489b0de38ba4f8cb2d46db432" target='_blank'>SGOOP-d: Estimating Kinetic Distances and Reaction Coordinate Dimensionality for Rare Event Systems from Biased/Unbiased Simulations.</a></td>
          <td>
            
              Sun-Ting Tsai,
            
              Zachary Smith,
            
              P. Tiwary,
            
          </td>
          <td>2021-04-28</td>
          <td>Journal of chemical theory and computation</td>
          <td>26</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5c8fca05881025024452f677b6528570d54c83a3" target='_blank'>Linear Transformations in Autoencoder Latent Space Predict Time Translations in Active Matter System</a></td>
          <td>
            
              E. Amaya,
            
              S. Shadkhoo,
            
              Dominik Schildknecht,
            
              Matt Thomson,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7c04a1c7e16e7e6c9a35c3b18c4110acb1aab2b0" target='_blank'>DRAFT Efficient Learning of Sparse Gaussian Mixture Models of Protein Conformational Substates</a></td>
          <td>
            
              J. Yoo,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fd11b7e8bbc5498f79c6478529c21604630abac4" target='_blank'>EMMA 1.3 Markov Model Algorithms Tutorial and Documentation</a></td>
          <td>
            
              Martin Senne,
            
              Benjamin Trendelkamp-Schroer,
            
              A. Mey,
            
              Frank Noé,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e396698359d0ecb525dbfaf46e4fc841a08c18d4" target='_blank'>Variational cross-validation of slow dynamical modes in molecular kinetics.</a></td>
          <td>
            
              R. McGibbon,
            
              V. Pande,
            
          </td>
          <td>2014-07-30</td>
          <td>The Journal of chemical physics</td>
          <td>165</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b937df713d1ea7057861e7872fa37b9148d3544" target='_blank'>Efficient and exact sampling of transition path ensembles on Markovian networks.</a></td>
          <td>
            
              Daniel J. Sharpe,
            
              D. Wales,
            
          </td>
          <td>2020-07-13</td>
          <td>The Journal of chemical physics</td>
          <td>14</td>
          <td>76</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69ccf867833b3564c2d2f26c49915b444eadb268" target='_blank'>Transferable Neural Networks for Enhanced Sampling of Protein Dynamics.</a></td>
          <td>
            
              Mohammad M. Sultan,
            
              H. Wayment-Steele,
            
              V. Pande,
            
          </td>
          <td>2018-01-02</td>
          <td>Journal of chemical theory and computation</td>
          <td>92</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b138eee8467f2d9e871331871e95c3749dc5e18b" target='_blank'>Coarse-graining auto-encoders for molecular dynamics</a></td>
          <td>
            
              Wujie Wang,
            
              R. G'omez-Bombarelli,
            
          </td>
          <td>2018-12-06</td>
          <td>npj Computational Materials</td>
          <td>141</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b20bd30a1274ac83242c2cbd74f452857a5f86e4" target='_blank'>Multiscale Reweighted Stochastic Embedding: Deep Learning of Collective Variables for Enhanced Sampling</a></td>
          <td>
            
              J. Rydzewski,
            
              O. Valsson,
            
          </td>
          <td>2020-07-13</td>
          <td>The Journal of Physical Chemistry. a</td>
          <td>24</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d767444574a2b9bb0dc6ca2b5e0f6f2f0fbfaf68" target='_blank'>Chemically Transferable Generative Backmapping of Coarse-Grained Proteins</a></td>
          <td>
            
              Soojung Yang,
            
              Rafael G'omez-Bombarelli,
            
          </td>
          <td>2023-03-02</td>
          <td>ArXiv</td>
          <td>9</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/54ab829a326f0e7f6328ea8451700570d66f186f" target='_blank'>Landmark Kernel tICA for Conformational Dynamics</a></td>
          <td>
            
              M. Harrigan,
            
              V. Pande,
            
          </td>
          <td>2017-04-04</td>
          <td>bioRxiv</td>
          <td>15</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/84b36724dfe9ff42b657e9ee06baf0254f9b8e32" target='_blank'>Potential-based dynamical reweighting for Markov state models of protein dynamics.</a></td>
          <td>
            
              J. Weber,
            
              V. Pande,
            
          </td>
          <td>2015-06-09</td>
          <td>Journal of chemical theory and computation</td>
          <td>4</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c6f2f5df6f749df2fe4e7ad099b42d943e4d8a26" target='_blank'>Trajectory Statistical Learning of the Potential Mean of Force and Diffusion Coefficient from Molecular Dynamics Simulations.</a></td>
          <td>
            
              Yi-Tsao Chen,
            
              Haw Yang,
            
              J. Chu,
            
          </td>
          <td>2024-01-02</td>
          <td>The journal of physical chemistry. B</td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aaa66e8729fbd071781f23c4239eff9995054019" target='_blank'>Simulations meet machine learning in structural biology.</a></td>
          <td>
            
              Adrià Pérez,
            
              Gerard Martínez-Rosell,
            
              G. de Fabritiis,
            
          </td>
          <td>2018-04-01</td>
          <td>Current opinion in structural biology</td>
          <td>35</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c89a40750265888d7f9c1c557675e1656108cf7e" target='_blank'>Machine learning heralding a new development phase in molecular dynamics simulations</a></td>
          <td>
            
              Eva Prasnikar,
            
              M. Ljubic,
            
              Andrej Perdih,
            
              J. Borišek,
            
          </td>
          <td>2024-03-29</td>
          <td>Artif. Intell. Rev.</td>
          <td>1</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/755dc5e951f01bb6fd648fcebe84803584d54fb2" target='_blank'>Detection of molecular behavior that characterizes systems using a deep learning approach.</a></td>
          <td>
            
              Katsuhiro Endo,
            
              Daisuke Yuhara,
            
              Katsufumi Tomobe,
            
              K. Yasuoka,
            
          </td>
          <td>2019-05-23</td>
          <td>Nanoscale</td>
          <td>4</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bccfbf2a9a02fa3e2a99f95a1fdae7a0f7d4e6c3" target='_blank'>Implicit Transfer Operator Learning: Multiple Time-Resolution Surrogates for Molecular Dynamics</a></td>
          <td>
            
              M. Schreiner,
            
              O. Winther,
            
              Simon Olsson,
            
          </td>
          <td>2023-05-29</td>
          <td>None</td>
          <td>4</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1a6555a8134c4c6b65c318bb6258e8735f5525e3" target='_blank'>Characterization and rapid sampling of protein folding Markov state model topologies.</a></td>
          <td>
            
              J. Weber,
            
              V. Pande,
            
          </td>
          <td>2011-10-11</td>
          <td>Journal of chemical theory and computation</td>
          <td>53</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7c720dd228c54829e4ed94a2b28c776f6529524f" target='_blank'>State predictive information bottleneck.</a></td>
          <td>
            
              Dedi Wang,
            
              P. Tiwary,
            
          </td>
          <td>2020-11-19</td>
          <td>The Journal of chemical physics</td>
          <td>61</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cbc86e3ad6f3856033f31e7e99dca323ce184d99" target='_blank'>Prediction of Molecular Conformation using Deep Generative Neural Networks</a></td>
          <td>
            
              Congsheng Xu,
            
              Yi Lu,
            
              Xiaomei Deng,
            
              Peiyuan Yu,
            
          </td>
          <td>2023-08-03</td>
          <td>Chinese Journal of Chemistry</td>
          <td>1</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/42f7c7e514a37d3943421e9463b0c65ad38be488" target='_blank'>Molearn: a Python package streamlining the design of generative models of biomolecular dynamics</a></td>
          <td>
            
              Samuel C. Musson,
            
              M. Degiacomi,
            
          </td>
          <td>2023-09-05</td>
          <td>J. Open Source Softw.</td>
          <td>0</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/268fb2625b2497d81079456b51f9f1a413d753fa" target='_blank'>Making high-dimensional molecular distribution functions tractable through Belief Propagation on Factor Graphs</a></td>
          <td>
            
              Zachary Smith,
            
              P. Tiwary,
            
          </td>
          <td>2021-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e15c87cfb4d2b1f82076331f9c58085598440e46" target='_blank'>GraphVAMPNet, using graph neural networks and variational approach to markov processes for dynamical modeling of biomolecules</a></td>
          <td>
            
              Mahdi Ghorbani,
            
              Samarjeet Prasad,
            
              Jeffery B. Klauda,
            
              B. Brooks,
            
          </td>
          <td>2022-01-12</td>
          <td>The Journal of chemical physics</td>
          <td>19</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7389d3fc2e8abc43964c96c16a94d96a04573f96" target='_blank'>Deep Generative Models for Molecular Science</a></td>
          <td>
            
              P. B. Jørgensen,
            
              Mikkel N. Schmidt,
            
              O. Winther,
            
          </td>
          <td>2018-01-01</td>
          <td>Molecular Informatics</td>
          <td>61</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b390f269e2f4d27bb5b27577c47fae2156cf38c8" target='_blank'>Generative models for protein sequence modeling: recent advances and future directions</a></td>
          <td>
            
              Mehrsa Mardikoraem,
            
              Zirui Wang,
            
              Nathaniel Pascual,
            
              Daniel R. Woldring,
            
          </td>
          <td>2023-09-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>5</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/293362ef34159c843f51bbfb7f17ca804c63468a" target='_blank'>An Extended Ensemble Approach for Protein Sequence Variation</a></td>
          <td>
            
              Hoda Akl,
            
              Brooke Emison,
            
              Xiaochuan Zhao,
            
              P. Dixit,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8184145c0c4660b073606f8c21fb97f931b66699" target='_blank'>Annealed fractional Lévy–Itō diffusion models for protein generation</a></td>
          <td>
            
              Eric Paquet,
            
              Farzan Soleymani,
            
              H. Viktor,
            
              Wojtek Michalowski,
            
          </td>
          <td>2024-04-01</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>0</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fcf26f700dc4459df01cd0c6e680a53f431f5c40" target='_blank'>Simple few-state models reveal hidden complexity in protein folding</a></td>
          <td>
            
              Kyle A. Beauchamp,
            
              R. McGibbon,
            
              Yu-Shan Lin,
            
              V. Pande,
            
          </td>
          <td>2012-07-09</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>148</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c86006be8cd1c5ee6193ee31cf31d44449b9c082" target='_blank'>Molecular enhanced sampling with autoencoders: On‐the‐fly collective variable discovery and accelerated free energy landscape exploration</a></td>
          <td>
            
              Wei Chen,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2017-12-30</td>
          <td>Journal of Computational Chemistry</td>
          <td>165</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bef7c61e1d111ab3d4b0ad39291346181aeabfe6" target='_blank'>Coupling Molecular Dynamics and Deep Learning to Mine Protein Conformational Space</a></td>
          <td>
            
              M. Degiacomi,
            
          </td>
          <td>2019-06-01</td>
          <td>Z--BiochemRN: Protein Folding (Proteins) (Topic)</td>
          <td>77</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/677bb2eb608075c2fb546fa88b7525f11fe8e79c" target='_blank'>Reinforcement learning of rare diffusive dynamics</a></td>
          <td>
            
              Avishek Das,
            
              Dominic C. Rose,
            
              J. P. Garrahan,
            
              David T. Limmer,
            
          </td>
          <td>2021-05-10</td>
          <td>The Journal of chemical physics</td>
          <td>19</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fd592e870345200c837c851770a2ef7097dd69e8" target='_blank'>Rapid Prediction of a Liquid Structure from a Single Molecular Configuration Using Deep Learning</a></td>
          <td>
            
              Chunhui Li,
            
              B. Gilbert,
            
              S. Farrell,
            
              P. Zarzycki,
            
          </td>
          <td>2023-06-12</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>2</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/43b2f986dfa28579fe24b37d4b4534b8dc1552b9" target='_blank'>Capturing Protein-Ligand Recognition Pathways in Coarse-grained Simulation</a></td>
          <td>
            
              B. Dandekar,
            
              Jagannath Mondal,
            
          </td>
          <td>2019-12-08</td>
          <td>bioRxiv</td>
          <td>21</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/01aea9c70456baa828642398c1a1d9985ddd69e5" target='_blank'>Discovering Reaction Pathways, Slow Variables, and Committor Probabilities with Machine Learning.</a></td>
          <td>
            
              Haochuan Chen,
            
              Benoı T Roux,
            
              C. Chipot,
            
          </td>
          <td>2023-05-24</td>
          <td>Journal of chemical theory and computation</td>
          <td>9</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a078463c48fc988e0d9b24b219804179f25fb254" target='_blank'>Towards Predicting Equilibrium Distributions for Molecular Systems with Deep Learning</a></td>
          <td>
            
              Shuxin Zheng,
            
              Jiyan He,
            
              Chang Liu,
            
              Yu Shi,
            
              Ziheng Lu,
            
              Weitao Feng,
            
              Fusong Ju,
            
              Jiaxian Wang,
            
              Jianwei Zhu,
            
              Yaosen Min,
            
              He Zhang,
            
              Shidi Tang,
            
              Hongxia Hao,
            
              Peiran Jin,
            
              Chi Chen,
            
              Frank No'e,
            
              Haiguang Liu,
            
              Tie-Yan Liu,
            
          </td>
          <td>2023-06-08</td>
          <td>ArXiv</td>
          <td>29</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/510a041ffcc7dcccb23cf4a23f0992e5de6b8f9e" target='_blank'>Autoencoders for dimensionality reduction in molecular dynamics: Collective variable dimension, biasing, and transition states.</a></td>
          <td>
            
              Zineb Belkacemi,
            
              M. Bianciotto,
            
              H. Minoux,
            
              T. Lelièvre,
            
              G. Stoltz,
            
              P. Gkeka,
            
          </td>
          <td>2023-06-05</td>
          <td>The Journal of chemical physics</td>
          <td>3</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e10ccaa71794bfd4df18785aa0f57b853fe4939d" target='_blank'>Learned Reconstruction of Protein Folding Trajectories from Noisy Single-Molecule Time Series.</a></td>
          <td>
            
              Maximilian Topel,
            
              A. Ejaz,
            
              A. Squires,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2023-01-26</td>
          <td>Journal of chemical theory and computation</td>
          <td>1</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3502af36a980d8b99e8f462c2441006b98470df1" target='_blank'>Coarse-grained Markov chains capture molecular thermodynamics and kinetics in no uncertain terms</a></td>
          <td>
            
              G. Bowman,
            
          </td>
          <td>2012-01-18</td>
          <td>arXiv: Biomolecules</td>
          <td>1</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2acfaad1fe42948a23db1fc4943e86002971b1bb" target='_blank'>Data-driven molecular modeling with the generalized Langevin equation</a></td>
          <td>
            
              Francesca Grogan,
            
              H. Lei,
            
              Xiantao Li,
            
              Nathan A. Baker,
            
          </td>
          <td>2019-11-30</td>
          <td>Journal of computational physics</td>
          <td>23</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c22344d86794d63c90c20e9c61e4cd97c8cf81c9" target='_blank'>SE(3)-Stochastic Flow Matching for Protein Backbone Generation</a></td>
          <td>
            
              A. Bose,
            
              Tara Akhound-Sadegh,
            
              Kilian Fatras,
            
              Guillaume Huguet,
            
              Jarrid Rector-Brooks,
            
              Cheng-Hao Liu,
            
              A. Nica,
            
              Maksym Korablyov,
            
              Michael M. Bronstein,
            
              Alexander Tong,
            
          </td>
          <td>2023-10-03</td>
          <td>ArXiv</td>
          <td>13</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fd1fb1d3e31260eaa72e3bff278b0cf837989a80" target='_blank'>RevGraphVAMP: A protein molecular simulation analysis model combining graph convolutional neural networks and physical constraints</a></td>
          <td>
            
              Ying Huang,
            
              Huiling Zhang,
            
              Zhenli Lin,
            
              Yanjie Wei,
            
              Wenhui Xi,
            
          </td>
          <td>2024-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bf136065e0aed4271f8ad3ff7e4e3305e9ba03c6" target='_blank'>Statistical model selection for Markov models of biomolecular dynamics.</a></td>
          <td>
            
              R. McGibbon,
            
              Christian Schwantes,
            
              V. Pande,
            
          </td>
          <td>2014-06-19</td>
          <td>The journal of physical chemistry. B</td>
          <td>23</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2c58f68b87493a209d9b56574f3c232296fdcbe3" target='_blank'>Combining experimental and simulation data of molecular processes via augmented Markov models</a></td>
          <td>
            
              S. Olsson,
            
              Hao Wu,
            
              Fabian Paul,
            
              C. Clementi,
            
              F. Noé,
            
          </td>
          <td>2017-07-17</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>77</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5d78c202c817e18ec5ef1692c193c97a07445ed6" target='_blank'>Learning high-dimensional reaction coordinates of fast-folding proteins using State Predictive information bottleneck and Bias Exchange Metadynamics</a></td>
          <td>
            
              Nancy D. Pomarici,
            
              S. Mehdi,
            
              Patrick K. Quoika,
            
              Suemin Lee,
            
              J. R. Loeffler,
            
              K. Liedl,
            
              P. Tiwary,
            
              Monica L. Fernández-Quintero,
            
          </td>
          <td>2023-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f1222bfe8c0cdeca72b595f0cae8921ebfc514a1" target='_blank'>Unbiased computation of transition times by pathway recombination.</a></td>
          <td>
            
              J. Kuipers,
            
              G. Barkema,
            
          </td>
          <td>2008-02-28</td>
          <td>The Journal of chemical physics</td>
          <td>4</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/87dab6c203b173dc821f657190e91296a3dc8bea" target='_blank'>Data-driven prediction of αIIbβ3 integrin activation pathways using nonlinear manifold learning and deep generative modeling.</a></td>
          <td>
            
              Siva Dasetty,
            
              T. Bidone,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2023-12-01</td>
          <td>Biophysical journal</td>
          <td>1</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/07b4a3910273a7122a5dedbced31ea5b877eb5b2" target='_blank'>Enhancing Conformational Sampling for Intrinsically Disordered and Ordered Proteins by Variational Autoencoder</a></td>
          <td>
            
              Junjie Zhu,
            
              Ningjie Zhang,
            
              Ting Wei,
            
              Haifeng Chen,
            
          </td>
          <td>2023-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>7</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4e2c97a0cd1db189787f1ed886ebd319fe13e770" target='_blank'>Hidden dynamics of proteasome autoregulation discovered by cryo-EM data-driven deep learning</a></td>
          <td>
            
              Zhaolong Wu,
            
              Enbo Chen,
            
              Shuwen Zhang,
            
              Wei Wang,
            
              Yinping Ma,
            
              Yuanchen Dong,
            
              Congcong Liu,
            
              C. Yin,
            
              Youdong Mao,
            
          </td>
          <td>2022-03-03</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6db34306fd106dee45a5f252e9f2fc8ed11c80f7" target='_blank'>Unsupervised deep learning for molecular dynamics simulations: a novel analysis of protein–ligand interactions in SARS-CoV-2 Mpro</a></td>
          <td>
            
              Jessica Mustali,
            
              Ikki Yasuda,
            
              Yoshinori Hirano,
            
              Kenji Yasuoka,
            
              Alfonso Gautieri,
            
              Noriyoshi Arai,
            
          </td>
          <td>2023-11-16</td>
          <td>RSC Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a29887ecce2e2a62fd2e1e8920b9cd1cc9719578" target='_blank'>Time-Varying Gaussian Graphical Models of Molecular Dynamics Data</a></td>
          <td>
            
              N. Razavian,
            
              Subhodeep Moitra,
            
              Hetunandan Kamisetty,
            
              A. Ramanathan,
            
              C. Langmead,
            
          </td>
          <td>None</td>
          <td></td>
          <td>9</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bc4e32c1b7fe39a37b21ec4c0624cfa45a6b36df" target='_blank'>Achieving Reversible Ligand-Protein Unbinding with Deep Learning and Molecular Dynamics through RAVE</a></td>
          <td>
            
              João Marcelo Lamim Ribeiro,
            
              P. Tiwary,
            
          </td>
          <td>2018-08-25</td>
          <td>bioRxiv</td>
          <td>6</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e231b3ac28bf6db4483426b4a3c24dd8e5cbfe86" target='_blank'>Artificial intelligence guided conformational mining of intrinsically disordered proteins</a></td>
          <td>
            
              Aayush Gupta,
            
              Souvik Dey,
            
              Alan Hicks,
            
              Huan‐Xiang Zhou,
            
          </td>
          <td>2021-11-21</td>
          <td>Communications Biology</td>
          <td>18</td>
          <td>67</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6bc6d859785aa237858d3a1ed7ae62a3a7b73536" target='_blank'>Protein Conformational Dynamics</a></td>
          <td>
            
              Keli Han,
            
              Xin Zhang,
            
              Mingjun Yang,
            
          </td>
          <td>None</td>
          <td></td>
          <td>30</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/64e66318f7d408e1192b93d15bba021447925522" target='_blank'>Automatic discovery of metastable states for the construction of Markov models of macromolecular conformational dynamics.</a></td>
          <td>
            
              J. Chodera,
            
              Nina Singhal,
            
              V. Pande,
            
              K. Dill,
            
              William Swope,
            
          </td>
          <td>2007-04-19</td>
          <td>The Journal of chemical physics</td>
          <td>566</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5d8ac66fa6425354f4f6e4b22e0364bafd3249e0" target='_blank'>Multi-Step Time Series Generator for Molecular Dynamics</a></td>
          <td>
            
              Katsuhiro Endo,
            
              Katsufumi Tomobe,
            
              K. Yasuoka,
            
          </td>
          <td>2018-04-26</td>
          <td>None</td>
          <td>18</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2c04651ea938fab3fc032ccff8fd0a2ef93675bf" target='_blank'>Transition Path Sampling as Markov Chain Monte Carlo of Trajectories: Recent Algorithms, Software, Applications, and Future Outlook</a></td>
          <td>
            
              P. Bolhuis,
            
              David W. H. Swenson,
            
          </td>
          <td>2021-03-09</td>
          <td>Advanced Theory and Simulations</td>
          <td>37</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d8d8e2c04ca47bd628bd2a499e03ad7cd29633da" target='_blank'>Time-lagged autoencoders: Deep learning of slow collective variables for molecular kinetics</a></td>
          <td>
            
              C. Wehmeyer,
            
              F. Noé,
            
          </td>
          <td>2017-10-30</td>
          <td>The Journal of chemical physics</td>
          <td>327</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/67226aecde573785d39d59f100c0c183d2ed5292" target='_blank'>Analysing protein dynamics using machine learning based generative models</a></td>
          <td>
            
              Alexandra-Ioana Albu,
            
              G. Czibula,
            
          </td>
          <td>2020-05-01</td>
          <td>2020 IEEE 14th International Symposium on Applied Computational Intelligence and Informatics (SACI)</td>
          <td>2</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1c92eff274561620931cf10d45597f9e09dcc91d" target='_blank'>Determination of Markov Model Dynamics from Equilibrium Data Snapshots</a></td>
          <td>
            
              Austen Mazenko,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d4f2a6e3aaaa099c3570f6f9523b5550217f9798" target='_blank'>Generative methods for sampling transition paths in molecular dynamics</a></td>
          <td>
            
              T. Lelièvre,
            
              Geneviève Robin,
            
              Inass Sekkat,
            
              G. Stoltz,
            
              Gabriel Victorino Cardoso,
            
          </td>
          <td>2022-05-05</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92081440bf32d43143a1304ec738e720ca4d2901" target='_blank'>D YNAMICS -I NFORMED P ROTEIN D ESIGN WITH S TRUCTURE C ONDITIONING</a></td>
          <td>
            
              Urszula Julia Komorowska,
            
              Simon V Mathis,
            
              Kieran Didi,
            
              Francisco Vargas,
            
              Pietro Lio,
            
              M. Jamnik,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/51894ffcf778630e7b021bc473d13d028a3b9158" target='_blank'>DynamicBind: predicting ligand-specific protein-ligand complex structure with a deep equivariant generative model</a></td>
          <td>
            
              Wei Lu,
            
              Jixian Zhang,
            
              Weifeng Huang,
            
              Ziqiao Zhang,
            
              Xiangyu Jia,
            
              Zhenyu Wang,
            
              Leilei Shi,
            
              Chengtao Li,
            
              Peter G Wolynes,
            
              Shuangjia Zheng,
            
          </td>
          <td>2024-02-05</td>
          <td>Nature Communications</td>
          <td>6</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b0a4098efda1b4a7e4397395a1616a7ccef57fa7" target='_blank'>Generative learning for nonlinear dynamics</a></td>
          <td>
            
              William Gilpin,
            
          </td>
          <td>2023-11-07</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a4ac048edff84a0d87a14943042f0edf46316da4" target='_blank'>Bayesian Abstraction of Markov Population Models</a></td>
          <td>
            
              L. Bortolussi,
            
              Francesca Cairoli,
            
          </td>
          <td>2019-09-10</td>
          <td>None</td>
          <td>4</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/013c63fe00c5e270a60a6ed6289663707d9562f7" target='_blank'>Fitting Stochastic Lattice Models Using Approximate Gradients</a></td>
          <td>
            
              Jan Schering,
            
              Sander Keemink,
            
              Johannes Textor,
            
          </td>
          <td>2023-10-12</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/76317336bf971d2006683e6dba463cd85154efff" target='_blank'>Learning Kinetic Distance Metrics for Markov State Models of Protein Conformational Dynamics.</a></td>
          <td>
            
              R. McGibbon,
            
              V. Pande,
            
          </td>
          <td>2013-06-06</td>
          <td>Journal of chemical theory and computation</td>
          <td>41</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5fdcaee10a43c2f4df8b44e64042ec0ff0d40a5f" target='_blank'>Deep learning of dynamically responsive chemical Hamiltonians with semiempirical quantum mechanics</a></td>
          <td>
            
              Guoqing Zhou,
            
              N. Lubbers,
            
              K. Barros,
            
              S. Tretiak,
            
              B. Nebgen,
            
          </td>
          <td>2022-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>20</td>
          <td>69</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8d0a923ee5493276b887965020ecf1b4e80731c6" target='_blank'>A tutorial on building markov state models with MSMBuilder and coarse-graining them with BACE.</a></td>
          <td>
            
              G. Bowman,
            
          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>8</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/290716662cbc93de08dfdc34c55be2741c235773" target='_blank'>Coarse-Graining of Molecular Dynamics Using Neural Ordinary Differential Equations</a></td>
          <td>
            
              Jakub Lála,
            
              Stefano Angiolleti-Uberti,
            
          </td>
          <td>2023-07-06</td>
          <td>IEEE EUROCON 2023 - 20th International Conference on Smart Technologies</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/35d98c8dc32c111edeeb0ae838fba2a31d27aecd" target='_blank'>PathFlow: A normalizing flow generator that finds transition paths</a></td>
          <td>
            
              Tianyi Liu,
            
              Weihao Gao,
            
              Zhirui Wang,
            
              Chong Wang,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>7</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/62ce907ba34c53a6790ce6f0ed9fbd440dd11f9d" target='_blank'>Characterizing the Latent Space of Molecular Deep Generative Models with Persistent Homology Metrics</a></td>
          <td>
            
              Yair Schiff,
            
              Vijil Chenthamarakshan,
            
              K. Ramamurthy,
            
              Payel Das,
            
          </td>
          <td>2020-10-18</td>
          <td>ArXiv</td>
          <td>6</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/27fc4138289273c311dbfe6d26486521d7544396" target='_blank'>Toward a structural identification of metastable molecular conformations.</a></td>
          <td>
            
              Simon Lemcke,
            
              Jörn H Appeldorn,
            
              Michael Wand,
            
              Thomas Speck,
            
          </td>
          <td>2023-09-15</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bd12dcecd73cc1613ca3c5aa69ae8d1da9ff6224" target='_blank'>Markov state models of protein misfolding.</a></td>
          <td>
            
              Anshul Sirur,
            
              David de Sancho,
            
              R. Best,
            
          </td>
          <td>2016-02-17</td>
          <td>The Journal of chemical physics</td>
          <td>28</td>
          <td>67</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/761320b0617a8925c391ae12c556cd463bc159c4" target='_blank'>Trajectory-based training enables protein simulations with accurate folding and Boltzmann ensembles in cpu-hours</a></td>
          <td>
            
              J. Jumper,
            
              N. Faruk,
            
              K. Freed,
            
              T. Sosnick,
            
          </td>
          <td>2018-12-01</td>
          <td>PLoS Computational Biology</td>
          <td>31</td>
          <td>66</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/02c251e00b24cf6c94cb060a1f6da6c262edc9f7" target='_blank'>Learning nonequilibrium control forces to characterize dynamical phase transitions.</a></td>
          <td>
            
              Jia-Feng Yan,
            
              H. Touchette,
            
              Grant M. Rotskoff,
            
          </td>
          <td>2021-07-07</td>
          <td>Physical review. E</td>
          <td>22</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/67a86e09f3e60f4af19d6c4c3b0c723835f9090f" target='_blank'>EPR-Net: Constructing non-equilibrium potential landscape via a variational force projection formulation</a></td>
          <td>
            
              Yue Zhao,
            
              W. Zhang,
            
              Tiejun Li,
            
          </td>
          <td>2023-01-05</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/acd233193196afa4f6bb8eabfa16e885fd21cd54" target='_blank'>From data to noise to data for mixing physics across temperatures with generative artificial intelligence</a></td>
          <td>
            
              Yihang Wang,
            
              Lukas Herron,
            
              P. Tiwary,
            
          </td>
          <td>2021-07-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>20</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/83ea81255d3ad1b92b201662ac1e71e652e3f309" target='_blank'>Analysing ill-conditioned Markov chains</a></td>
          <td>
            
              Esmae J. Woods,
            
              Deepti Kannan,
            
              Daniel J. Sharpe,
            
              T. Swinburne,
            
              D. Wales,
            
          </td>
          <td>2023-05-22</td>
          <td>Philosophical transactions. Series A, Mathematical, physical, and engineering sciences</td>
          <td>2</td>
          <td>76</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/53a0bca5a24cba98d8c658c59ab11ffd13d8b4e1" target='_blank'>Generating Novel, Designable, and Diverse Protein Structures by Equivariantly Diffusing Oriented Residue Clouds</a></td>
          <td>
            
              Yeqing Lin,
            
              Mohammed Alquraishi,
            
          </td>
          <td>2023-01-29</td>
          <td>None</td>
          <td>37</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/98243acffb45975b0fe73763b41316c520a7037a" target='_blank'>Progress in deep Markov state modeling: Coarse graining and experimental data restraints.</a></td>
          <td>
            
              Andreas Mardt,
            
              Frank Noé,
            
          </td>
          <td>2021-08-04</td>
          <td>The Journal of chemical physics</td>
          <td>10</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/21e8bbd2846539c180b901b747e579f82027d375" target='_blank'>Pre-training Protein Models with Molecular Dynamics Simulations for Drug Binding</a></td>
          <td>
            
              Fang Wu,
            
              Q. Zhang,
            
              Dragomir R. Radev,
            
              Yuyang Wang,
            
              Xurui Jin,
            
              Yinghui Jiang,
            
              Z. Niu,
            
              Stan Z. Li,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>4</td>
          <td>73</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d40b083ac2c6f1776b188293f5cf78fdbd50cabf" target='_blank'>From Peptides to Nanostructures: A Euclidean Transformer for Fast and Stable Machine Learned Force Fields</a></td>
          <td>
            
              J. Frank,
            
              Oliver T. Unke,
            
              Klaus-Robert Müller,
            
              Stefan Chmiela,
            
          </td>
          <td>2023-09-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/073dc917746394b7edfab461afd3e36e605dc5ba" target='_blank'>May the Force be with You: Unified Force-Centric Pre-Training for 3D Molecular Conformations</a></td>
          <td>
            
              Rui Feng,
            
              Qi Zhu,
            
              Huan Tran,
            
              Binghong Chen,
            
              Aubrey Toland,
            
              R. Ramprasad,
            
              Chao Zhang,
            
          </td>
          <td>2023-08-24</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bf8483f233ab19f600de940a1bfce6cc323cb91d" target='_blank'>Learning Neural Generative Dynamics for Molecular Conformation Generation</a></td>
          <td>
            
              Minkai Xu,
            
              Shitong Luo,
            
              Y. Bengio,
            
              Jian Peng,
            
              Jian Tang,
            
          </td>
          <td>2021-02-20</td>
          <td>ArXiv</td>
          <td>94</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/021fbc740911eff5d51b08905fc82d046c740ef0" target='_blank'>Diffusion Methods for Generating Transition Paths</a></td>
          <td>
            
              Luke Triplett,
            
              Jianfeng Lu,
            
          </td>
          <td>2023-09-19</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/91e2c0b61ceb898383927b39e4436541cac2ddb0" target='_blank'>A molecular simulation protocol to avoid sampling redundancy and discover new states.</a></td>
          <td>
            
              M. Bacci,
            
              A. Vitalis,
            
              A. Caflisch,
            
          </td>
          <td>2015-05-01</td>
          <td>Biochimica et biophysica acta</td>
          <td>30</td>
          <td>74</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b78620184e43c6609b1c2df240fad080460de420" target='_blank'>The combined force field-sampling problem in simulations of disordered amyloid-β peptides</a></td>
          <td>
            
              A. Vitalis,
            
              M. Bacci,
            
              A. Caflisch,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>4</td>
          <td>74</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/037f82fa06421cf3a054dac5f97fb002dc0b7f4f" target='_blank'>Learning effective stochastic differential equations from microscopic simulations: combining stochastic numerics and deep learning</a></td>
          <td>
            
              Felix Dietrich,
            
              A. Makeev,
            
              George A. Kevrekidis,
            
              N. Evangelou,
            
              Tom S. Bertalan,
            
              S. Reich,
            
              I. Kevrekidis,
            
          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>17</td>
          <td>84</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0187ec0ad130049a72e9cfc50c6160437c3fb2aa" target='_blank'>Understanding recent deep‐learning techniques for identifying collective variables of molecular dynamics</a></td>
          <td>
            
              W. Zhang,
            
              Christof Schutte,
            
          </td>
          <td>2023-07-01</td>
          <td>PAMM</td>
          <td>2</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5812a54cb511aba5f4d47cf3af5509e98a7ee0b1" target='_blank'>ScMiles2: A Script to Conduct and Analyze Milestoning Trajectories for Long Time Dynamics.</a></td>
          <td>
            
              A. Cardenas,
            
              Allison Hunter,
            
              Hao Wang,
            
              R. Elber,
            
          </td>
          <td>2022-10-03</td>
          <td>Journal of chemical theory and computation</td>
          <td>1</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/06a090117138a4866b34f7debba950545eab2cce" target='_blank'>Stability-Aware Training of Neural Network Interatomic Potentials with Differentiable Boltzmann Estimators</a></td>
          <td>
            
              Sanjeev Raja,
            
              Ishan Amin,
            
              Fabian Pedregosa,
            
              Aditi S. Krishnapriyan,
            
          </td>
          <td>2024-02-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/947f16a3a3eedf07cde5b787882d3d5ff2d46d8a" target='_blank'>Scalable Normalizing Flows Enable Boltzmann Generators for Macromolecules</a></td>
          <td>
            
              Joseph C. Kim,
            
              David Bloore,
            
              Karan Kapoor,
            
              Jun Feng,
            
              Ming-Hong Hao,
            
              Mengdi Wang,
            
          </td>
          <td>2024-01-08</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/15586fec836ffcb60eba81491e04c225e6914aac" target='_blank'>Protein structure generation via folding diffusion</a></td>
          <td>
            
              Kevin E. Wu,
            
              Kevin Kaichuang Yang,
            
              Rianne van den Berg,
            
              James Zou,
            
              Alex X. Lu,
            
              Ava P. Amini,
            
          </td>
          <td>2022-09-30</td>
          <td>Nature Communications</td>
          <td>95</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4cad5547c77b9243c5db342534e6536adb466fc0" target='_blank'>Linking time-series of single-molecule experiments with molecular dynamics simulations by machine learning</a></td>
          <td>
            
          </td>
          <td>2018-05-03</td>
          <td>eLife</td>
          <td>38</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8271ca0056e9a3a09b33963054d04b7c14c9ed69" target='_blank'>Active Learning of the Conformational Ensemble of Proteins using Maximum Entropy VAMPNets</a></td>
          <td>
            
              D. Kleiman,
            
              D. Shukla,
            
          </td>
          <td>2023-01-13</td>
          <td>bioRxiv</td>
          <td>9</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dc873c5cc7dfa9f6e565094c8c63ad4c56abf907" target='_blank'>Towards simple kinetic models of functional dynamics for a kinase subfamily</a></td>
          <td>
            
              Mohammad M. Sultan,
            
              G. Kiss,
            
              V. Pande,
            
          </td>
          <td>2017-12-03</td>
          <td>Nature Chemistry</td>
          <td>51</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7700e6b622338e4fbdb707b29d6404aa865dcfd6" target='_blank'>Molecular Conformer Search with Low-Energy Latent Space</a></td>
          <td>
            
              Xiaomi Guo,
            
              Lincan Fang,
            
              Yong Xu,
            
              W. Duan,
            
              P. Rinke,
            
              M. Todorović,
            
              Xi Chen,
            
          </td>
          <td>2022-03-26</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>1</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7b4ceb9763e3a897db4e1b12f2b01e4d3ead696c" target='_blank'>Improved coarse-graining of Markov state models via explicit consideration of statistical uncertainty.</a></td>
          <td>
            
              G. Bowman,
            
          </td>
          <td>2012-01-18</td>
          <td>The Journal of chemical physics</td>
          <td>72</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/20a6548e83fc464931ab8845c0f5e5d41ac6d640" target='_blank'>F$^3$low: Frame-to-Frame Coarse-grained Molecular Dynamics with SE(3) Guided Flow Matching</a></td>
          <td>
            
              Shaoning Li,
            
              Yusong Wang,
            
              Mingyu Li,
            
              Jian Zhang,
            
              Bin Shao,
            
              Nanning Zheng,
            
              Jian Tang,
            
          </td>
          <td>2024-05-01</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/02de449f85841d342cd2d30e7bc4f85e049842a1" target='_blank'>Addressing the Embeddability Problem in Transition Rate Estimation</a></td>
          <td>
            
              C. Goolsby,
            
              James W. Losey,
            
              ,
            
              Marie-Christine Düker,
            
              Mila Getmansky Sherman,
            
              D. Matteson,
            
              Mahmoud Moradi,
            
          </td>
          <td>2019-08-02</td>
          <td>The Journal of Physical Chemistry. a</td>
          <td>2</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8120c229a6c660c64fdd72221f868abcff31f3c1" target='_blank'>Ensemble learning of coarse-grained molecular dynamics force fields with a kernel approach.</a></td>
          <td>
            
              Jiang Wang,
            
              Stefan Chmiela,
            
              K. Müller,
            
              F. Noé,
            
              C. Clementi,
            
          </td>
          <td>2020-05-04</td>
          <td>The Journal of chemical physics</td>
          <td>41</td>
          <td>127</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a69482738401055aa4c73201000599d4fbe4741f" target='_blank'>3DFlex: determining structure and motion of flexible proteins from cryo-EM</a></td>
          <td>
            
              A. Punjani,
            
              David J. Fleet,
            
          </td>
          <td>2023-05-11</td>
          <td>Nature Methods</td>
          <td>53</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5b85e2ad38f6b262cc0267bf96d45dd93e587184" target='_blank'>Machine Learning in Molecular Simulations of Biomolecules</a></td>
          <td>
            
              Guan Xing-Yue,
            
              Huang Heng-Yan,
            
              Peng Hua-Qi,
            
              Liu Yan-Hang,
            
              Li Wen-Fei,
            
              Wang Wei,
            
          </td>
          <td>None</td>
          <td>Acta Physica Sinica</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ae1cc243ff4f2e84ee36530cfabc76fbff7dbe6" target='_blank'>Variational algorithms for analyzing noisy multi-state diffusion trajectories</a></td>
          <td>
            
              M. Lindén,
            
              J. Elf,
            
          </td>
          <td>2018-03-08</td>
          <td>bioRxiv</td>
          <td>22</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d85d5be05014ce94888ea85962b3dc2d11e1639f" target='_blank'>Efficient inference of protein structural ensembles</a></td>
          <td>
            
              T. Lane,
            
              Christian Schwantes,
            
              Kyle A. Beauchamp,
            
              V. Pande,
            
          </td>
          <td>2014-08-01</td>
          <td>arXiv: Biological Physics</td>
          <td>3</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e7cfb758b6c200f47f00fbba310d505b32f68379" target='_blank'>Efficient Construction of Mesostate Networks from Molecular Dynamics Trajectories.</a></td>
          <td>
            
              A. Vitalis,
            
              Amedeo Caflisch,
            
          </td>
          <td>2012-02-10</td>
          <td>Journal of chemical theory and computation</td>
          <td>4</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f954c3e5262104a383fc64adcb57fb800d7ac221" target='_blank'>Deep Representation Learning for Complex Free Energy Landscapes.</a></td>
          <td>
            
              Jun Zhang,
            
              Yao-Kun Lei,
            
              X. Che,
            
              Zhen Zhang,
            
              Yi Isaac Yang,
            
              Y. Gao,
            
          </td>
          <td>2019-09-05</td>
          <td>The journal of physical chemistry letters</td>
          <td>14</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b0751ec97d4d106b4abf4ff0a822fd12e7a665ed" target='_blank'>Overcoming the Size Limit of First Principles Molecular Dynamics Simulations with an In-Distribution Substructure Embedding Active Learner</a></td>
          <td>
            
              Lingyu Kong,
            
              Jielan Li,
            
              Lixin Sun,
            
              Han Yang,
            
              Hongxia Hao,
            
              Chi Chen,
            
              Nongnuch Artrith,
            
              Jose Antonio Garrido Torres,
            
              Ziheng Lu,
            
              Yichi Zhou,
            
          </td>
          <td>2023-11-14</td>
          <td>None</td>
          <td>0</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8daf91622ec05492a52427b7666a4fa859ff2811" target='_blank'>Generative Adversarial Learning of Protein Tertiary Structures</a></td>
          <td>
            
              Taseef Rahman,
            
              Yuanqi Du,
            
              Liang Zhao,
            
              Amarda Shehu,
            
          </td>
          <td>2021-02-24</td>
          <td>Molecules</td>
          <td>15</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a5b9aec1885f12ef93aeeb3fd6ec000771c1d574" target='_blank'>Learning deterministic hydrodynamic equations from stochastic active particle dynamics</a></td>
          <td>
            
              S. Maddu,
            
              Q. Vagne,
            
              I. Sbalzarini,
            
          </td>
          <td>2022-01-21</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d9f236925cfbb97625682891adf1570a3964bad9" target='_blank'>Pathway Histogram Analysis of Trajectories: A general strategy for quantification of molecular mechanisms</a></td>
          <td>
            
              Ernesto Su'arez,
            
              D. Zuckerman,
            
          </td>
          <td>2018-10-24</td>
          <td>arXiv: Chemical Physics</td>
          <td>5</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94815cffb3269740c3d9521d5c368ad1ae56ed33" target='_blank'>Von Mises Mixture Distributions for Molecular Conformation Generation</a></td>
          <td>
            
              K. Swanson,
            
              J. Williams,
            
              Eric Jonas,
            
          </td>
          <td>2023-06-13</td>
          <td>None</td>
          <td>2</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bf80771cb5426a41402f1bf5c6cdae6df2bef0c3" target='_blank'>End-to-End Differentiable , Hardware Accelerated , Molecular Dynamics in Pure Python</a></td>
          <td>
            
              S. Schoenholz,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>18</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1f385d1785cc97103fc5e65b08969407ba348677" target='_blank'>Accurate Machine Learned Quantum-Mechanical Force Fields for Biomolecular Simulations</a></td>
          <td>
            
              Oliver T. Unke,
            
              M. Stohr,
            
              Stefan Ganscha,
            
              Thomas Unterthiner,
            
              Hartmut Maennel,
            
              S. Kashubin,
            
              Daniel Ahlin,
            
              M. Gastegger,
            
              L. M. Sandonas,
            
              A. Tkatchenko,
            
              Klaus-Robert Muller,
            
          </td>
          <td>2022-05-17</td>
          <td>ArXiv</td>
          <td>17</td>
          <td>76</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f0f0b680ac74960922634e3da5cf1e6f6883c840" target='_blank'>Protein generation with evolutionary diffusion: sequence is all you need</a></td>
          <td>
            
              Sarah Alamdari,
            
              Nitya Thakkar,
            
              Rianne van den Berg,
            
              Alex X. Lu,
            
              Nicolo Fusi,
            
              Ava P. Amini,
            
              Kevin Kaichuang Yang,
            
          </td>
          <td>2023-09-12</td>
          <td>bioRxiv</td>
          <td>30</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/259c36d340e7e5a0d33b772faad981c64befce4b" target='_blank'>JAX, M.D.: End-to-End Differentiable, Hardware Accelerated, Molecular Dynamics in Pure Python</a></td>
          <td>
            
              S. Schoenholz,
            
              E. D. Cubuk,
            
          </td>
          <td>2019-12-09</td>
          <td>arXiv: Computational Physics</td>
          <td>34</td>
          <td>50</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/de07678108c3635f6f8ab26b0edf4311d07a8124" target='_blank'>Exploring particle dynamics during self-organization processes via rotationally invariant latent representations</a></td>
          <td>
            
              Sergei V. Kalinin,
            
              Shuai Zhang,
            
              Mani Valleti,
            
              Harley Pyles,
            
              D. Baker,
            
              J. Yoreo,
            
              M. Ziatdinov,
            
          </td>
          <td>2020-09-02</td>
          <td>arXiv: Soft Condensed Matter</td>
          <td>3</td>
          <td>145</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/66cdb2c89bb00dada7fbc6a1ac28661dbe6e5158" target='_blank'>Towards equilibrium molecular conformation generation with GFlowNets</a></td>
          <td>
            
              Alexandra Volokhova,
            
              Michal Koziarski,
            
              Alex Hernández-García,
            
              Cheng-Hao Liu,
            
              Santiago Miret,
            
              Pablo Lemos,
            
              Luca Thiede,
            
              Zichao Yan,
            
              Alán Aspuru-Guzik,
            
              Y. Bengio,
            
          </td>
          <td>2023-10-20</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e8ae5f6ec788460983768998811a5bf6095e18d6" target='_blank'>Mcamc: An Advanced Algorithm for Kinetic Monte Carlo Simulations from Magnetization Switching to Protein Folding</a></td>
          <td>
            
              M. Novotny,
            
              S. Wheeler,
            
          </td>
          <td>2002-11-02</td>
          <td>arXiv: Materials Science</td>
          <td>1</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2d63090895bbdeec76895197310fedb5be9a559b" target='_blank'>GRadient Adaptive Decomposition (GRAD) Method: Optimized Refinement Along Macrostate Borders in Markov State Models</a></td>
          <td>
            
              P. Romano,
            
              M. Guenza,
            
          </td>
          <td>2017-10-31</td>
          <td>Journal of chemical information and modeling</td>
          <td>2</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ec72a00ab118f77bf9161b0558dbc0866bf3a1b" target='_blank'>A Perspective on Deep Learning for Molecular Modeling and Simulations</a></td>
          <td>
            
              Jun Zhang,
            
              Yao-Kun Lei,
            
              Zhen Zhang,
            
              Junhan Chang,
            
              Maodong Li,
            
              Xu Han,
            
              Lijiang Yang,
            
              Yi Isaac Yang,
            
              Y. Gao,
            
          </td>
          <td>2020-04-25</td>
          <td>The journal of physical chemistry. B</td>
          <td>5</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/28ae29bdc2d01fcff4f3549d54c44ba18a67d355" target='_blank'>A Primer on the Bayesian Approach to High-Density Single-Molecule Trajectories Analysis.</a></td>
          <td>
            
              Mohamed El Beheiry,
            
              S. Türkcan,
            
              Maximilian U. Richly,
            
              A. Triller,
            
              A. Alexandrou,
            
              M. Dahan,
            
              J. Masson,
            
          </td>
          <td>2016-03-29</td>
          <td>Biophysical journal</td>
          <td>24</td>
          <td>78</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ec461f9fd4b19b52508e85f3ffb7b7e70fc5d67" target='_blank'>Chasing collective variables using temporal data-driven strategies</a></td>
          <td>
            
              Haochuan Chen,
            
              C. Chipot,
            
          </td>
          <td>2023-01-06</td>
          <td>QRB Discovery</td>
          <td>9</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b4ecde129fb11af90e691f4bc1f023f23a6f950b" target='_blank'>Towards Conditional Generation of Minimal Action Potential Pathways for Molecular Dynamics</a></td>
          <td>
            
              J. Cava,
            
              J. Vant,
            
              Nicholas Ho,
            
              Ankita Shulka,
            
              P. Turaga,
            
              Ross Maciejewski,
            
              A. Singharoy,
            
          </td>
          <td>2021-11-28</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b9dfe2a7b6a5daff279f56c17fd77abd8d92b158" target='_blank'>Active learning accelerates ab initio molecular dynamics on reactive energy surfaces</a></td>
          <td>
            
              Shi Jun Ang,
            
              Wujie Wang,
            
              Daniel Schwalbe-Koda,
            
              Simon Axelrod,
            
              Rafael Gómez-Bombarelli,
            
          </td>
          <td>2021-03-11</td>
          <td>Chem</td>
          <td>40</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6195b623a13e861741ac1e80a7a8b774d5332236" target='_blank'>Fast and Uncertainty-Aware Directional Message Passing for Non-Equilibrium Molecules</a></td>
          <td>
            
              Johannes Klicpera,
            
              Shankari Giri,
            
              Johannes T. Margraf,
            
              Stephan Gunnemann,
            
          </td>
          <td>2020-11-28</td>
          <td>ArXiv</td>
          <td>231</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/24726572ccb2e0daea9c7a7007a431da122e9f5d" target='_blank'>Perspective: Identification of collective variables and metastable states of protein dynamics.</a></td>
          <td>
            
              F. Sittel,
            
              G. Stock,
            
          </td>
          <td>2018-10-15</td>
          <td>The Journal of chemical physics</td>
          <td>98</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e06d9dd9385d6eaacc561756155a5cc481775f30" target='_blank'>PyCGTOOL: Automated Generation of Coarse-Grained Molecular Dynamics Models from Atomistic Trajectories</a></td>
          <td>
            
              James A. Graham,
            
              J. Essex,
            
              S. Khalid,
            
          </td>
          <td>2017-04-04</td>
          <td>Journal of chemical information and modeling</td>
          <td>60</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dec1deaaa4969250ae7c43206dee2cdea0958952" target='_blank'>Modelling the Dynamics of Biological Systems with the Geometric Hidden Markov Model</a></td>
          <td>
            
              Borislav Vangelov,
            
              Mauricio Barahona,
            
          </td>
          <td>2017-11-22</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/38cdb6fc3f8548facad8d6a9c8952a5878b8ee6d" target='_blank'>Proximal Optimal Transport Modeling of Population Dynamics</a></td>
          <td>
            
              Charlotte Bunne,
            
              Laetitia Meng-Papaxanthos,
            
              Andreas Krause,
            
              Marco Cuturi,
            
          </td>
          <td>2021-06-11</td>
          <td>None</td>
          <td>57</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5711400d6fbba28fc776a3b85b6890b62934f64f" target='_blank'>End-to-end protein–ligand complex structure generation with diffusion-based generative models</a></td>
          <td>
            
              Shuya Nakata,
            
              Yoshiharu Mori,
            
              S. Tanaka,
            
          </td>
          <td>2022-12-21</td>
          <td>BMC Bioinformatics</td>
          <td>17</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c864473ff9664e9f25ad0c1359e8f9cbd0749880" target='_blank'>Shadow hybrid Monte Carlo: an efficient propagator in phase space of macromolecules</a></td>
          <td>
            
              J. Izaguirre,
            
              Scott S. Hampton,
            
          </td>
          <td>2004-11-01</td>
          <td>Journal of Computational Physics</td>
          <td>101</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/13dc2b43477777f6a174c282477310e067263c32" target='_blank'>Characterizing metastable states with the help of machine learning</a></td>
          <td>
            
              P. Novelli,
            
              L. Bonati,
            
              M. Pontil,
            
              M. Parrinello,
            
          </td>
          <td>2022-04-15</td>
          <td>Journal of chemical theory and computation</td>
          <td>12</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aae6576b6f449e115f5c4e1aa3759593239f7b11" target='_blank'>A Perspective on Deep Learning for Molecular Modeling and Simulations.</a></td>
          <td>
            
              Jun Zhang,
            
              Yao-Kun Lei,
            
              Zhen Zhang,
            
              Junhan Chang,
            
              Maodong Li,
            
              Xu Han,
            
              Lijiang Yang,
            
              Yi Isaac Yang,
            
              Y. Gao,
            
          </td>
          <td>2020-07-14</td>
          <td>The journal of physical chemistry. A</td>
          <td>27</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d757b4604edd1ee395fcda54d166c906e080cff9" target='_blank'>Exploring glassy dynamics with Markov state models from graph dynamical neural networks.</a></td>
          <td>
            
              S. Soltani,
            
              C. Sinclair,
            
              Joerg Rottler,
            
          </td>
          <td>2021-11-19</td>
          <td>Physical review. E</td>
          <td>3</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7479ab45c58e86de621b7c5483d811dd85ebdc85" target='_blank'>Optical lattice experiments at unobserved conditions with generative adversarial deep learning</a></td>
          <td>
            
              Corneel Casert,
            
              Kyle Mills,
            
              Tom Vieijra,
            
              J. Ryckebusch,
            
              Isaac Tamblyn,
            
          </td>
          <td>2020-02-17</td>
          <td>Physical Review Research</td>
          <td>2</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d1b0f8f9ec911b3129deeed602a975fe89c8a357" target='_blank'>Computing the transition state populations in simple protein models.</a></td>
          <td>
            
              S. Ozkan,
            
              S. Ozkan,
            
              K. Dill,
            
              I. Bahar,
            
              I. Bahar,
            
          </td>
          <td>None</td>
          <td>Biopolymers</td>
          <td>44</td>
          <td>92</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe16255bbbfbd74b63b4bbf72899069830ff27d3" target='_blank'>An Interpretable Convolutional Neural Network Framework for Analyzing Molecular Dynamics Trajectories: a Case Study on Functional States for G-Protein-Coupled Receptors</a></td>
          <td>
            
              Chuan Li,
            
              Jiangting Liu,
            
              Jianfang Chen,
            
              Yuan Yuan,
            
              Jin Yu,
            
              Qiaolin Gou,
            
              Yanzhi Guo,
            
              X. Pu,
            
          </td>
          <td>2022-03-08</td>
          <td>Journal of chemical information and modeling</td>
          <td>9</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/57f40c18e45c2571312493df612113d413a44ed4" target='_blank'>UniCon3D: de novo protein structure prediction using united-residue conformational search via stepwise, probabilistic sampling</a></td>
          <td>
            
              Debswapna Bhattacharya,
            
              Renzhi Cao,
            
              Jianlin Cheng,
            
          </td>
          <td>2016-06-03</td>
          <td>Bioinformatics</td>
          <td>43</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a1b36c08f6f04550f1dc48d84bb74ad3a8e0aaa5" target='_blank'>Learning Clustered Representation for Complex Free Energy Landscapes</a></td>
          <td>
            
              Jun Zhang,
            
              Yao-Kun Lei,
            
              X. Che,
            
              Zhen Zhang,
            
              Yi Isaac Yang,
            
              Y. Gao,
            
          </td>
          <td>2019-06-07</td>
          <td>arXiv: Statistical Mechanics</td>
          <td>0</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7cd70c6493caeafb8ff9be638229a14f878c8d97" target='_blank'>Analyzing Multimodal Probability Measures with Autoencoders.</a></td>
          <td>
            
              Tony Lelièvre,
            
              Thomas Pigeon,
            
              G. Stoltz,
            
              Wei Zhang,
            
          </td>
          <td>2023-10-05</td>
          <td>The journal of physical chemistry. B</td>
          <td>2</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aeb87738cf0058e1c85cad38d5cda241d420a455" target='_blank'>Learning local equivariant representations for large-scale atomistic dynamics</a></td>
          <td>
            
              Albert Musaelian,
            
              Simon L. Batzner,
            
              A. Johansson,
            
              Lixin Sun,
            
              Cameron J. Owen,
            
              M. Kornbluth,
            
              B. Kozinsky,
            
          </td>
          <td>2022-04-11</td>
          <td>Nature Communications</td>
          <td>205</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a804be91dbda6618c549fa5e6a898bd1a16ce180" target='_blank'>Detailed Balanced Chemical Reaction Networks as Generalized Boltzmann Machines</a></td>
          <td>
            
              William G. Poole,
            
              T. Ouldridge,
            
              Manoj Gopalkrishnan,
            
              E. Winfree,
            
          </td>
          <td>2022-05-12</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>58</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a9730366151ae58da7d1ede9f55244501c397e11" target='_blank'>Towards Finding a Better Reaction Coordinate: NtrC and DIMS</a></td>
          <td>
            
              J. Perilla,
            
              T. Woolf,
            
          </td>
          <td>2011-02-02</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/38fc5852128337711554595ecf8de1cf66afe82e" target='_blank'>Caliber Corrected Markov Modeling (C2M2): Correcting Equilibrium Markov Models.</a></td>
          <td>
            
              P. Dixit,
            
              K. Dill,
            
          </td>
          <td>2017-11-08</td>
          <td>Journal of chemical theory and computation</td>
          <td>25</td>
          <td>92</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/609a034e0cfdf0ae10ccf82668adb44687756566" target='_blank'>Neural-network solutions to stochastic reaction networks</a></td>
          <td>
            
              Ying Tang,
            
              Jiayu Weng,
            
              Pan Zhang,
            
          </td>
          <td>2022-09-29</td>
          <td>Nature Machine Intelligence</td>
          <td>11</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/22bd75994b94a00cf2a9697a28a805e35da30b1f" target='_blank'>Reweighted autoencoded variational Bayes for enhanced sampling (RAVE).</a></td>
          <td>
            
              João Marcelo Lamim Ribeiro,
            
              P. Bravo,
            
              Yihang Wang,
            
              P. Tiwary,
            
          </td>
          <td>2018-02-09</td>
          <td>The Journal of chemical physics</td>
          <td>240</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3b8fb7d210d5c44b54206db0544eb063e63631e3" target='_blank'>Probabilistic models for protein conformational changes</a></td>
          <td>
            
              Chuong Nguyen,
            
          </td>
          <td>2020-06-09</td>
          <td></td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2dcef55a07f8607a819c21fe84131ea269cc2e3c" target='_blank'>Deep Unsupervised Learning using Nonequilibrium Thermodynamics</a></td>
          <td>
            
              Jascha Narain Sohl-Dickstein,
            
              Eric A. Weiss,
            
              Niru Maheswaranathan,
            
              S. Ganguli,
            
          </td>
          <td>2015-03-11</td>
          <td>ArXiv</td>
          <td>3649</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2bde1c477d5d159842ae46f3bfd24fde1c71a814" target='_blank'>Recent Applications of Dynamical Mean-Field Methods</a></td>
          <td>
            
              L. Cugliandolo,
            
          </td>
          <td>2023-05-02</td>
          <td>Annual Review of Condensed Matter Physics</td>
          <td>8</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e60d09b0b23844d9dc58f4a780915b44d5b996da" target='_blank'>Machine learning based methods to generate conformational ensembles of disordered proteins.</a></td>
          <td>
            
              Ishan Taneja,
            
              Keren Lasker,
            
          </td>
          <td>2023-12-01</td>
          <td>Biophysical journal</td>
          <td>1</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/182afd2cfc31e8d283c86228c03da091fc2ff067" target='_blank'>Machine learning of an implicit solvent for dynamic Monte Carlo simulations.</a></td>
          <td>
            
              Ankush Checkervarty,
            
              J. Sommer,
            
              M. Werner,
            
          </td>
          <td>2023-01-24</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/09b2d52317e91ab3757f8df4b1655d062ba330f8" target='_blank'>Neural networks learn the motions of molecular machines</a></td>
          <td>
            
              Timothy Grant,
            
          </td>
          <td>2021-07-29</td>
          <td>Nature Methods</td>
          <td>4</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69c8e33fb6266ffd0b4156b9fdd784ba9d28e415" target='_blank'>Particle Mobility Analysis Using Deep Learning and the Moment Scaling Spectrum</a></td>
          <td>
            
              Marloes Arts,
            
              Ihor Smal,
            
              M. Paul,
            
              C. Wyman,
            
              E. Meijering,
            
          </td>
          <td>2019-11-05</td>
          <td>Scientific Reports</td>
          <td>36</td>
          <td>49</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8bc613de1d0e6efff5317a2e7495a6d4f4791934" target='_blank'>3D Flexible Refinement: Structure and Motion of Flexible Proteins from Cryo-EM</a></td>
          <td>
            
              A. Punjani,
            
              David J. Fleet,
            
          </td>
          <td>2021-04-22</td>
          <td>bioRxiv</td>
          <td>52</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1f5b19a1f10b8a7836d6e9b7e57bd5acc6b00f85" target='_blank'>Toward a direct and scalable identification of reduced models for categorical processes</a></td>
          <td>
            
              Susanne Gerber,
            
              I. Horenko,
            
          </td>
          <td>2017-04-21</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>19</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/102d5b795ee1777c0631e5c38f5415951123c8fd" target='_blank'>BuRNN: A Buffer Region Neural Network Approach for QM/MM Simulations of Complex Systems</a></td>
          <td>
            
              Bettina Lier,
            
              Peter Poliak,
            
              P. Marquetand,
            
              Julia Westermayr,
            
              C. Oostenbrink,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>47</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c080faa28b5aa2a2aa0f4e52836cb55dcb30ff39" target='_blank'>Modeling Dynamics with Deep Transition-Learning Networks</a></td>
          <td>
            
              D. V. Dijk,
            
              Scott A. Gigante,
            
              Alexander Strzalkowski,
            
              Guy Wolf,
            
              Smita Krishnaswamy,
            
          </td>
          <td>2018-02-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6f629ccf5d25e47d7d5942f417bf12a2b1342bce" target='_blank'>Binding-and-Folding Recognition of an Intrinsically Disordered Protein Using Online Learning Molecular Dynamics</a></td>
          <td>
            
              Pablo Herrera-Nieto,
            
              Adria P'erez,
            
              G. D. Fabritiis,
            
          </td>
          <td>2023-02-20</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>1</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94800faf04d01d15a154fe1bf79ec38f7c0406d2" target='_blank'>HEroBM: a deep equivariant graph neural network for universal backmapping from coarse-grained to all-atom representations</a></td>
          <td>
            
              Daniele Angioletti,
            
              S. Raniolo,
            
              V. Limongelli,
            
          </td>
          <td>2024-04-25</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/853b4194775c1a5ed68a8c29a4f1976c74a4f66d" target='_blank'>ACE: adaptive cluster expansion for maximum entropy graphical model inference</a></td>
          <td>
            
              J. Barton,
            
              E. D. Leonardis,
            
              A. Coucke,
            
              S. Cocco,
            
          </td>
          <td>2016-03-18</td>
          <td>bioRxiv</td>
          <td>78</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8992115e1aae09e50b41518014db049cf1bd1cb4" target='_blank'>Learning protein constitutive motifs from sequence data</a></td>
          <td>
            
              J. Tubiana,
            
              S. Cocco,
            
              R. Monasson,
            
          </td>
          <td>2018-03-23</td>
          <td>eLife</td>
          <td>85</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b27667a5901213b207ddecc1d4e648ca9919c370" target='_blank'>ExTrack characterizes transition kinetics and diffusion in noisy single-particle tracks</a></td>
          <td>
            
              François Simon,
            
              J. Tinevez,
            
              S. van Teeffelen,
            
          </td>
          <td>2022-07-14</td>
          <td>The Journal of Cell Biology</td>
          <td>7</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b794176493f02f655f7f0bb32c4584ff171363a2" target='_blank'>An all-atom protein generative model</a></td>
          <td>
            
              Alexander E. Chu,
            
              Lucy Cheng,
            
              Gina El Nesr,
            
              Minkai Xu,
            
              Po-Ssu Huang,
            
          </td>
          <td>2023-05-25</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/433fbaae85d5cdd75292450e11e259bf916bd49c" target='_blank'>SMAUG: Analyzing single-molecule tracks with nonparametric Bayesian statistics</a></td>
          <td>
            
              J. Karslake,
            
              E. Donarski,
            
              S. Shelby,
            
              L. Demey,
            
              V. DiRita,
            
              S. Veatch,
            
              J. Biteen,
            
          </td>
          <td>2019-03-14</td>
          <td>bioRxiv</td>
          <td>50</td>
          <td>50</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0215dd9f346534bf4c4247220501d7ab7d7715c6" target='_blank'>A Multi-Grained Symmetric Differential Equation Model for Learning Protein-Ligand Binding Dynamics</a></td>
          <td>
            
              Shengchao Liu,
            
              Weitao Du,
            
              Yanjing Li,
            
              Zhuoxinran Li,
            
              Vignesh C. Bhethanabotla,
            
              Nakul Rampal,
            
              O. Yaghi,
            
              C. Borgs,
            
              A. Anandkumar,
            
              Hongyu Guo,
            
              J. Chayes,
            
          </td>
          <td>2024-01-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>56</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f293dd554326b93786cbd4d4c956141db055929c" target='_blank'>Neural mode jump Monte Carlo.</a></td>
          <td>
            
              Luigi Sbailò,
            
              Manuel Dibak,
            
              Frank No'e,
            
          </td>
          <td>2019-12-11</td>
          <td>The Journal of chemical physics</td>
          <td>9</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/20e17a844a4cc2bf0e58eab1474679d4899d0638" target='_blank'>Denoise Pretraining on Nonequilibrium Molecules for Accurate and Transferable Neural Potentials</a></td>
          <td>
            
              Yuyang Wang,
            
              Chang Xu,
            
              Zijie Li,
            
              A. Farimani,
            
          </td>
          <td>2023-03-03</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>4</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/89d2082786cac743b961e642d18085d0634b4f6e" target='_blank'>MIT Open Access Articles Efficient Traversal of Beta-Sheet Protein Folding Pathways Using Ensemble Models</a></td>
          <td>
            
              S. Shenker,
            
              Charles W. O’Donnell,
            
              Srinivas Devadas,
            
              Bonnie Berger,
            
              JE´ROˆME,
            
              WALDISPU¨HL,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2f328f1c2c7ef798cc9f8540c56bfa940d97ceed" target='_blank'>Diffusion models in protein structure and docking</a></td>
          <td>
            
              Jason Yim,
            
              Hannes Stärk,
            
              Gabriele Corso,
            
              Bowen Jing,
            
              R. Barzilay,
            
              T. Jaakkola,
            
          </td>
          <td>2024-03-01</td>
          <td>WIREs Computational Molecular Science</td>
          <td>0</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9e250bb14bad3f93572cc906dc96d358a91a1da2" target='_blank'>Kino‐Geometric Modeling: Insights into Protein Molecular Mechanisms</a></td>
          <td>
            
              D. Budday,
            
              S. Leyendecker,
            
              H. van den Bedem,
            
          </td>
          <td>2019-11-01</td>
          <td>PAMM</td>
          <td>0</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/88350ce03452678c05f7e1717e10ad19a408b79e" target='_blank'>Deep generative models create new and diverse protein structures</a></td>
          <td>
            
              Zeming Lin,
            
              Tom Sercu,
            
              Alexander Rives,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>15</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/71c6a07fce01386c14e4aacb068a87021bb8c643" target='_blank'>Prediction of protein continuum secondary structure with probabilistic models based on NMR solved structures</a></td>
          <td>
            
              Mikael Bodén,
            
              Zheng Yuan,
            
              Timothy L. Bailey,
            
          </td>
          <td>2006-02-14</td>
          <td>BMC Bioinformatics</td>
          <td>37</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/90cd845ab02cbfeb0d33d981ba099516dc9c64f5" target='_blank'>MSMExplorer: visualizing Markov state models for biomolecule folding simulations</a></td>
          <td>
            
              Bryce W. Cronkite-Ratcliff,
            
              V. Pande,
            
          </td>
          <td>2013-04-01</td>
          <td>Bioinformatics</td>
          <td>21</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3a1c9acd38c438bacd92788f22d60b27b0e6ab72" target='_blank'>A library-based Monte Carlo technique enables rapid equilibrium sampling of a protein model with atomistic components</a></td>
          <td>
            
              A. Mamonov,
            
              D. Bhatt,
            
              D. Cashman,
            
              D. Zuckerman,
            
          </td>
          <td>2008-09-22</td>
          <td>arXiv: Biological Physics</td>
          <td>0</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d0d24e6ec5cc692702afacc201f3d343a8d9ea49" target='_blank'>Using Brownian Dynamics Simulation in the Construction of Benchmark Trajectories for Use in Optimizing Markov State Models Generation</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b8871be8abc8035e9c3e4c29b02505182806d771" target='_blank'>Learning effective stochastic differential equations from microscopic simulations: Linking stochastic numerics to deep learning.</a></td>
          <td>
            
              Felix Dietrich,
            
              A. Makeev,
            
              George A. Kevrekidis,
            
              N. Evangelou,
            
              Tom S. Bertalan,
            
              S. Reich,
            
              I. Kevrekidis,
            
          </td>
          <td>2021-06-10</td>
          <td>Chaos</td>
          <td>19</td>
          <td>84</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/05ad5de8401411a7fdf53120bf90b0f370970ce4" target='_blank'>Inferring pointwise diffusion properties of single trajectories with deep learning.</a></td>
          <td>
            
              Borja Requena,
            
              Sergi Mas'o,
            
              Joan Bertran,
            
              M. Lewenstein,
            
              Carlo Manzo,
            
              Gorka Muñoz-Gil,
            
          </td>
          <td>2023-02-01</td>
          <td>Biophysical journal</td>
          <td>5</td>
          <td>92</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0915bc6adb2fee9a14716d117ddd5ec148869d2a" target='_blank'>Sampling a Rare Protein Transition Using Quantum Annealing.</a></td>
          <td>
            
              Danial Ghamari,
            
              Roberto Covino,
            
              ,
            
          </td>
          <td>2024-04-08</td>
          <td>Journal of chemical theory and computation</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b96802440ece1f2012b3f28c757ad98eff23bd14" target='_blank'>EspcTM: Kinetic Transition Network Based on Trajectory Mapping in Effective Energy Rescaling Space</a></td>
          <td>
            
              Zhenyu Wang,
            
              Xin Zhou,
            
              Guanghong Zuo,
            
          </td>
          <td>2020-10-27</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/79714b9ee1abe7f5e73947235392a57d15dc54a2" target='_blank'>∆ -Learning for Coarse-Grained Potentials</a></td>
          <td>
            
              Aditi Khot,
            
              B. Savoie,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3eed492986e7a0c03362118daead192076d46446" target='_blank'>Supervised machine learning approach to molecular dynamics forecast of SARS-CoV-2 spike glycoproteins at varying temperatures</a></td>
          <td>
            
              David Liang,
            
              Meichen Song,
            
              Ziyuan Niu,
            
              Peng Zhang,
            
              M. Rafailovich,
            
              Yuefan Deng,
            
          </td>
          <td>2021-02-17</td>
          <td>Mrs Advances</td>
          <td>8</td>
          <td>60</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1627305bd3c6e01b343325ca6b465e9ef744b3b7" target='_blank'>Modelling sequential protein folding under kinetic control</a></td>
          <td>
            
              Fabien P. E. Huard,
            
              C. Deane,
            
              G. Wood,
            
          </td>
          <td>2006-07-10</td>
          <td>Bioinformatics</td>
          <td>30</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9ee4f8beb4155b889e8f4f2aaa3253f0d379760f" target='_blank'>Machine Learning-Assisted Discovery of Hidden States in Expanded Free Energy Space.</a></td>
          <td>
            
              R. Ketkaew,
            
              Fabrizio Creazzo,
            
              S. Luber,
            
          </td>
          <td>2022-02-16</td>
          <td>The journal of physical chemistry letters</td>
          <td>6</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78d1c58b634ae3f62bba7e7e085fadb529b12a49" target='_blank'>Predicting Real-time Scientific Experiments Using Transformer models and Reinforcement Learning</a></td>
          <td>
            
              J. M. Parrilla-Gutierrez,
            
          </td>
          <td>2021-12-01</td>
          <td>2021 20th IEEE International Conference on Machine Learning and Applications (ICMLA)</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fab22c08f44bd3e71bc45b0e87155df0f3f4e691" target='_blank'>Extraction of protein dynamics information from cryo-EM maps using deep learning</a></td>
          <td>
            
              S. Matsumoto,
            
              Shoichi Ishida,
            
              M. Araki,
            
              Takayuki Kato,
            
              Kei Terayama,
            
              Y. Okuno,
            
          </td>
          <td>2021-02-01</td>
          <td>Nature Machine Intelligence</td>
          <td>49</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b86b08ccc78b8c19fa462dff789433daf088c50a" target='_blank'>Efficient generative modeling of protein sequences using simple autoregressive models</a></td>
          <td>
            
              J. Trinquier,
            
              Guido Uguzzoni,
            
              A. Pagnani,
            
              F. Zamponi,
            
              M. Weigt,
            
          </td>
          <td>2021-03-04</td>
          <td>Nature Communications</td>
          <td>56</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/751910f192146126ec050ac5c45a8f3416d58387" target='_blank'>The RED scheme: Rate-constant estimation from pre-steady state weighted ensemble simulations.</a></td>
          <td>
            
              A. DeGrave,
            
              Anthony T. Bogetti,
            
              L. Chong,
            
          </td>
          <td>2021-03-21</td>
          <td>The Journal of chemical physics</td>
          <td>11</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1f121c55fe314bc53430a01531549ab2f010827a" target='_blank'>Beyond rotamers: a generative, probabilistic model of side chains in proteins</a></td>
          <td>
            
              T. Harder,
            
              Wouter Boomsma,
            
              M. Paluszewski,
            
              J. Frellsen,
            
              K. E. Johansson,
            
              T. Hamelryck,
            
          </td>
          <td>2010-06-05</td>
          <td>BMC Bioinformatics</td>
          <td>65</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ad97f5fb9d261252b3ffd132d5c69f9f204aec5b" target='_blank'>Mixup-Augmented Meta-Learning for Sample-Efficient Fine-Tuning of Protein Simulators</a></td>
          <td>
            
              Jingbang Chen,
            
              Yian Wang,
            
              Xingwei Qu,
            
              Shuangjia Zheng,
            
              Yao-Chun Yang,
            
              Hao Dong,
            
              Jie Fu,
            
          </td>
          <td>2023-08-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fd90f8444b050a0fa2415ab911ce13be8b140dfc" target='_blank'>Describing Protein Folding Kinetics by Molecular Dynamics Simulations. 1. Theory</a></td>
          <td>
            
              William Swope,
            
              J. Pitera,
            
              F. Suits,
            
          </td>
          <td>2004-04-22</td>
          <td>Journal of Physical Chemistry B</td>
          <td>400</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/490ad1648f79dea701d0e717977990c888c8bbc3" target='_blank'>DP Train, then DP Compress: Model Compression in Deep Potential Molecular Dynamics</a></td>
          <td>
            
              Denghui Lu,
            
              Wanrun Jiang,
            
              Yixiao Chen,
            
              Linfeng Zhang,
            
              Weile Jia,
            
              Han Wang,
            
              Mohan Chen,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>10</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a31aeeddae54b3d030778aea6bfe5e4f032773ea" target='_blank'>Computing Absolute Free Energy with Deep Generative Models</a></td>
          <td>
            
              Xinqiang Ding,
            
              Bin Zhang,
            
          </td>
          <td>2020-05-01</td>
          <td>The journal of physical chemistry. B</td>
          <td>15</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f2eafeb800f34ec0da8c3eb82809087047db54ac" target='_blank'>Interpolating Nonadiabatic Molecular Dynamics Hamiltonian with Bidirectional Long Short-Term Memory Networks</a></td>
          <td>
            
              Bipeng Wang,
            
              Ludwig Winkler,
            
              Yifan Wu,
            
              Klaus-Robert Müller,
            
              H. E. Sauceda,
            
              O. Prezhdo,
            
          </td>
          <td>2023-08-02</td>
          <td>The Journal of Physical Chemistry Letters</td>
          <td>2</td>
          <td>81</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7082dbb60ee3c3d9c3c15b0395664b6e0ae62033" target='_blank'>Intrinsically Disordered Proteins: Where Computation Meets Experiment</a></td>
          <td>
            
              Virginia M. Burger,
            
              Thomas Gurry,
            
              Collin M. Stultz,
            
          </td>
          <td>2014-10-01</td>
          <td></td>
          <td>47</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3d50464ec004730c9bb79f461b1eccf124d42e6d" target='_blank'>Toward Achieving Efficient and Accurate Ligand-Protein Unbinding with Deep Learning and Molecular Dynamics through RAVE.</a></td>
          <td>
            
              João Marcelo Lamim Ribeiro,
            
              P. Tiwary,
            
          </td>
          <td>2018-12-07</td>
          <td>Journal of chemical theory and computation</td>
          <td>56</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c046774b24be185a9fa47a538df0b23089015e44" target='_blank'>DROIDS 3.0—Detecting Genetic and Drug Class Variant Impact on Conserved Protein Binding Dynamics</a></td>
          <td>
            
              G. Babbitt,
            
              Ernest Fokoue,
            
              Joshua R. Evans,
            
              Kyle I. Diller,
            
              Lily E. Adams,
            
          </td>
          <td>2019-12-18</td>
          <td>Biophysical Journal</td>
          <td>19</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3ad4b34e8eaa58491e73dbd07ed03f9ada4b0ea5" target='_blank'>Characterisation of anomalous diﬀusion statistics powered by deep learning</a></td>
          <td>
            
              A. Gentili,
            
              G. Volpe,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1e46586086ff17ddba194288e712dbc04c4fb447" target='_blank'>TRamWAy: mapping physical properties of individual biomolecule random motion in large-scale single-particle tracking experiments</a></td>
          <td>
            
              François Laurent,
            
              Hippolyte Verdier,
            
              M. Duval,
            
              A. Serov,
            
              Christian L. Vestergaard,
            
              J. Masson,
            
          </td>
          <td>2022-04-28</td>
          <td>Bioinformatics</td>
          <td>4</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/086d883a039b76c43fe54f5843ad4b2103ec57b9" target='_blank'>Hidden Markov modeling of single-particle diffusion with stochastic tethering.</a></td>
          <td>
            
              Amit Federbush,
            
              Amit Moscovich,
            
              Yohai Bar-Sinai,
            
          </td>
          <td>2023-08-02</td>
          <td>Physical review. E</td>
          <td>1</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d36c692ab8b3c24028b2d87162a04d7d7296bccb" target='_blank'>High-order polytomous Boltzmann machine and forecasting time series I</a></td>
          <td>
            
              Y. Liu,
            
          </td>
          <td>1997-04-04</td>
          <td>None</td>
          <td>1</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a1da55ff96d015b9d3f27c743890e15a9442330e" target='_blank'>The structural de-correlation time: A robust statistical measure of convergence of biomolecular simulations</a></td>
          <td>
            
              E. Lyman,
            
              D. Zuckerman,
            
          </td>
          <td>2006-07-21</td>
          <td>arXiv: Quantitative Methods</td>
          <td>3</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ad15fbef83bad74d7022cefbb4629432613d974" target='_blank'>Surprisal Metrics for Quantifying Perturbed Conformational Dynamics in Markov State Models.</a></td>
          <td>
            
              Vincent A. Voelz,
            
              B. Elman,
            
              Asghar M. Razavi,
            
              Guangfeng Zhou,
            
          </td>
          <td>2014-11-21</td>
          <td>Journal of chemical theory and computation</td>
          <td>21</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/56de4646fd7409be7ec212ab6dba058b4a25586f" target='_blank'>Extraction of Protein Dynamics Information Hidden in Cryogenic Electron Microscopy Maps Using Deep Learning</a></td>
          <td>
            
              S. Matsumoto,
            
              Shoichi Ishida,
            
              M. Araki,
            
              Takayuki Kato,
            
              Kei Terayama,
            
              Y. Okuno,
            
          </td>
          <td>2020-02-18</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/678b1a05fb2df99b0c055bfaf9da61be0847c0c3" target='_blank'>Learning Small Molecule Energies and Interatomic Forces with an Equivariant Transformer on the ANI-1x Dataset</a></td>
          <td>
            
              Bryce Hedelius,
            
              F. Fuchs,
            
              Dennis Della Corte,
            
          </td>
          <td>2022-01-03</td>
          <td>None</td>
          <td>1</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/426fa8decdfaffbe7d9299b400d669750c02a662" target='_blank'>: Extraction of Protein Dynamics Information Hidden in Cryo-EM Map Using Deep Learning</a></td>
          <td>
            
              S. Matsumoto,
            
              Shoichi Ishida,
            
              M. Araki,
            
              Takayuki Kato,
            
              Kei Terayama,
            
              Yasushi Okuno,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/132df201f03dd6c0523c642d3cd7a4de6278e435" target='_blank'>Deep Autoencoders for Additional Insight into Protein Dynamics</a></td>
          <td>
            
              Mihai Teletin,
            
              G. Czibula,
            
              Maria-Iuliana Bocicor,
            
              S. Albert,
            
              A. Pandini,
            
          </td>
          <td>2018-10-04</td>
          <td>None</td>
          <td>8</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/445db716ea486b89fac622fcc46a5ee9a3326308" target='_blank'>The Role of Force Fields and Water Models in Protein Folding and Unfolding Dynamics</a></td>
          <td>
            
              Anna-Lena M. Fischer,
            
              Anna Tichy,
            
              Janik Kokot,
            
              Valentin J. Hoerschinger,
            
              Robert F Wild,
            
              Jakob R. Riccabona,
            
              J. R. Loeffler,
            
              Franz Waibl,
            
              Patrick K. Quoika,
            
              Philipp Gschwandtner,
            
              Stefano Forli,
            
              A. Ward,
            
              K. Liedl,
            
              Martin Zacharias,
            
              Monica L. Fernández-Quintero,
            
          </td>
          <td>2024-02-19</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>1</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/808702fab27f8a85344bfe2804e455ac5f034696" target='_blank'>A Generative Angular Model of Protein Structure Evolution</a></td>
          <td>
            
              Michael Golden,
            
              E. García-Portugués,
            
              Michael Sørensen,
            
              K. Mardia,
            
              T. Hamelryck,
            
              J. Hein,
            
          </td>
          <td>2016-12-30</td>
          <td>Molecular Biology and Evolution</td>
          <td>17</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ca6454c2b6694a5e3c65c9229bb947e3d6eca4d" target='_blank'>ATRANET - Automated generation of transition networks for the structural characterization of intrinsically disordered proteins.</a></td>
          <td>
            
              Moritz Schäffler,
            
              Mohammed Khaled,
            
              B. Strodel,
            
          </td>
          <td>2022-08-01</td>
          <td>Methods</td>
          <td>5</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3819fae661099ee01322b6c5624da017b402f530" target='_blank'>Gramian Angular Fields for leveraging pretrained computer vision models with anomalous diffusion trajectories</a></td>
          <td>
            
              Òscar Garibo i Orts,
            
              Nicolás Firbas,
            
              L. Sebastiá,
            
              J. A. Conejero,
            
          </td>
          <td>2023-03-01</td>
          <td>Physical review. E</td>
          <td>5</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/26d2619f35d99024b81971c552a3098afa3d60b5" target='_blank'>Estimation of diffusive states from single-particle trajectory in heterogeneous medium using machine-learning methods.</a></td>
          <td>
            
              Y. Matsuda,
            
              I. Hanasaki,
            
              Ryo Iwao,
            
              H. Yamaguchi,
            
              T. Niimi,
            
          </td>
          <td>2018-09-26</td>
          <td>Physical chemistry chemical physics : PCCP</td>
          <td>21</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d415bdf861e3e10535415495d58cc015b36d5ccf" target='_blank'>Nonequilibrium candidate Monte Carlo: A new tool for equilibrium simulation</a></td>
          <td>
            
              Jerome P. Nilmeier,
            
              G. Crooks,
            
              David D. L. Minh,
            
              J. Chodera,
            
          </td>
          <td>2011-06-21</td>
          <td></td>
          <td>1</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7811dc0675cff432796f6b09a8d1a40df28026dd" target='_blank'>Effective Molecular Dynamics from Neural-Network Based Structure Prediction Models</a></td>
          <td>
            
              Alexander Jussupow,
            
              Ville R. I. Kaila,
            
          </td>
          <td>2022-10-18</td>
          <td>bioRxiv</td>
          <td>11</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5ff7ce809f3304de96567133b1f69b346edfcba1" target='_blank'>Learning to evolve structural ensembles of unfolded and disordered proteins using experimental solution data.</a></td>
          <td>
            
              Oufan Zhang,
            
              Mojtaba Haghighatlari,
            
              Jie Li,
            
              Ziqing Liu,
            
              Ashley Namini,
            
              J. C. Teixeira,
            
              J. Forman-Kay,
            
              T. Head‐Gordon,
            
          </td>
          <td>2022-06-25</td>
          <td>The Journal of chemical physics</td>
          <td>9</td>
          <td>80</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/067c7f5d3da411e407abfcde5707a266c6ac7872" target='_blank'>Markov state models from hierarchical density-based assignment</a></td>
          <td>
            
              I. Mitxelena,
            
              X. López,
            
              David de Sancho,
            
          </td>
          <td>2021-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1469204da0c3ca2baddee7d592233ef68e5452ef" target='_blank'>Biomolecular modeling and simulation: a field coming of age</a></td>
          <td>
            
              T. Schlick,
            
              R. Collepardo-Guevara,
            
              L. Halvorsen,
            
              Segun Jung,
            
              Xia Xiao,
            
          </td>
          <td>2011-01-12</td>
          <td>Quarterly Reviews of Biophysics</td>
          <td>132</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/41cf6bf31afd91e24548c26cb3cbfcbd68fd7f37" target='_blank'>Constrained hidden Markov models reveal further Hsp90 protein states</a></td>
          <td>
            
              Riccardo Tancredi,
            
              Antonio Feltrin,
            
              Giosue Sardo Infirri,
            
              Simone Toso,
            
              L. Vollmar,
            
              Thorsten Hugel,
            
              Marco Baiesi,
            
          </td>
          <td>2024-02-29</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf143a4be3617352a98132cdbe37b84a8e5acaa3" target='_blank'>De-biasing Particle Filtering for a Continuous Time Hidden Markov Model With a Cox Process Observation Model</a></td>
          <td>
            
              Rui Jin,
            
              Sumeetpal S. Singh,
            
              N. Chopin,
            
          </td>
          <td>2022-06-21</td>
          <td>Statistica Sinica</td>
          <td>2</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7e631773b14d7203eb0d85742b2a850f51eb1616" target='_blank'>GEOM: Energy-annotated molecular conformations for property prediction and molecular generation</a></td>
          <td>
            
              Simon Axelrod,
            
              Rafael Gómez-Bombarelli,
            
          </td>
          <td>2020-06-09</td>
          <td>ArXiv</td>
          <td>133</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ee1bd2de94882940da12f09d18380a51bcf8727" target='_blank'>Statistical methods for large ensemble of super-resolution stochastic single particle trajectories</a></td>
          <td>
            
              N. Hozé,
            
              D. Holcman,
            
          </td>
          <td>2017-11-30</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f0f134bc53f9aea130f1a561acb5fa6b08ae4ce" target='_blank'>Iterated Denoising Energy Matching for Sampling from Boltzmann Densities</a></td>
          <td>
            
              Tara Akhound-Sadegh,
            
              Jarrid Rector-Brooks,
            
              A. Bose,
            
              Sarthak Mittal,
            
              Pablo Lemos,
            
              Cheng-Hao Liu,
            
              Marcin Sendera,
            
              Siamak Ravanbakhsh,
            
              Gauthier Gidel,
            
              Y. Bengio,
            
              Nikolay Malkin,
            
              Alexander Tong,
            
          </td>
          <td>2024-02-09</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/117987ac9bea8eaf6b5d43e73a67ba5cf8203f8f" target='_blank'>Accelerated Construction of Kinetic Network Model of Biomolecules Using Steered Molecular Dynamics.</a></td>
          <td>
            
              Susmita Ghosh,
            
              A. Chatterjee,
            
              S. Bhattacharya,
            
          </td>
          <td>2018-09-13</td>
          <td>Journal of chemical theory and computation</td>
          <td>3</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e44b3bd5b4c544a3b5ffc47f7c2ac58a1633d569" target='_blank'>Generative models for protein structures and sequences</a></td>
          <td>
            
              Chloe Hsu,
            
              Clara Fannjiang,
            
              Jennifer Listgarten,
            
          </td>
          <td>2024-02-01</td>
          <td>Nature Biotechnology</td>
          <td>1</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dbfa557b80aa4a227a6419ce5c69d6ba3f2366ef" target='_blank'>PyVisA: Visualization and Analysis of path sampling trajectories</a></td>
          <td>
            
              Ola Aarøen,
            
              Henrik Kiær,
            
              E. Riccardi,
            
          </td>
          <td>2020-12-13</td>
          <td>Journal of Computational Chemistry</td>
          <td>4</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2363d2a5d0749f7b2e5342513e9041cdd6066bb7" target='_blank'>Markov Methods for Hierarchical Coarse-Graining of Large Protein Dynamics</a></td>
          <td>
            
              C. Chennubhotla,
            
              I. Bahar,
            
          </td>
          <td>2006-04-02</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>29</td>
          <td>80</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c297f2b893dc8bc93333d8d7e236a2d43a0b1838" target='_blank'>SAPPHIRE-based clustering.</a></td>
          <td>
            
              F. Cocina,
            
              A. Vitalis,
            
              A. Caflisch,
            
          </td>
          <td>2020-09-09</td>
          <td>Journal of chemical theory and computation</td>
          <td>10</td>
          <td>74</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2ce762211753c97a5cf1b7ed52c9d5b24f4ee558" target='_blank'>A Probabilistic Framework for Constructing Temporal Relations in Replica Exchange Molecular Trajectories.</a></td>
          <td>
            
              Aditya Chattopadhyay,
            
              Min Zheng,
            
              M. Waller,
            
              U. Priyakumar,
            
          </td>
          <td>2018-05-23</td>
          <td>Journal of chemical theory and computation</td>
          <td>5</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e7672751b91f34a236d7bbc19afefcb2546842e6" target='_blank'>ChemoVerse: Manifold Traversal of Latent Spaces for Novel Molecule Discovery</a></td>
          <td>
            
              Harshdeep Singh,
            
              Nicholas McCarthy,
            
              Q. Ain,
            
              Jeremiah Hayes,
            
          </td>
          <td>2020-09-29</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9b54ccdb90d03a49d667e78937aed18381c57d3f" target='_blank'>Detecting directed motion and confinement in single-particle trajectories using hidden variables</a></td>
          <td>
            
              François Simon,
            
              Guillaume Ramadier,
            
              Inès Fonquernie,
            
              Janka Zsok,
            
              S. Patskovsky,
            
              Michel Meunier,
            
              Caroline Boudoux,
            
              E. Dultz,
            
              Lucien E. Weiss,
            
          </td>
          <td>2024-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aba18bdf3f4be858879820adcecea4034d492bb9" target='_blank'>Pacific Symposium on Biocomputing 15:228-239(2010) CONSTRUCTING MULTI-RESOLUTION MARKOV STATE MODELS (MSMS) TO ELUCIDATE RNA HAIRPIN FOLDING MECHANISMS</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>59</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8878f984a720dbd946ff6af5b9cf4d029b96905f" target='_blank'>Bayesian deep learning for error estimation in the analysis of anomalous diffusion</a></td>
          <td>
            
              Henrik Seckler,
            
              R. Metzler,
            
          </td>
          <td>2022-11-07</td>
          <td>Nature Communications</td>
          <td>26</td>
          <td>80</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ff1ef27425a0d8c9b493808e204e89f71b8e2aff" target='_blank'>Dynamic Boltzmann Machines for Second Order Moments and Generalized Gaussian Distributions</a></td>
          <td>
            
              Raymond H. Putra,
            
              T. Osogami,
            
              Sakyasingha Dasgupta,
            
          </td>
          <td>2017-12-17</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/61bc6f304e9ca8274a6ff5c65e0b7bbd5d3fa46d" target='_blank'>Using Dimensionality Reduction to Systematically Expand Conformational Sampling of Intrinsically Disordered Peptides.</a></td>
          <td>
            
              O. Kukharenko,
            
              Kevin Sawade,
            
              Jakob Steuer,
            
              C. Peter,
            
          </td>
          <td>2016-09-13</td>
          <td>Journal of chemical theory and computation</td>
          <td>27</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1068197434a8e303802bb090be7284b7bba83a96" target='_blank'>WESTPA 2.0: High-performance upgrades for weighted ensemble simulations and analysis of longer-timescale applications</a></td>
          <td>
            
              John D. Russo,
            
              She Zhang,
            
              Jeremy M. G. Leung,
            
              Anthony T. Bogetti,
            
              Jeff P. Thompson,
            
              A. DeGrave,
            
              Paul A. Torrillo,
            
              A. Pratt,
            
              Kim F. Wong,
            
              Junchao Xia,
            
              J. Copperman,
            
              J. Adelman,
            
              M. Zwier,
            
              David N. LeBard,
            
              D. Zuckerman,
            
              L. Chong,
            
          </td>
          <td>2021-12-06</td>
          <td>bioRxiv</td>
          <td>26</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4fda3d9952bf2c779e3d6571955bc4f0ab0f2884" target='_blank'>Accurate Free Energy Estimations of Molecular Systems Via Flow-based Targeted Free Energy Perturbation</a></td>
          <td>
            
              Soo-Jung Lee,
            
              Amr H. Mahmoud,
            
              M. Lill,
            
          </td>
          <td>2023-02-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ef85de5122736a603650263f9c39939f55b6a5df" target='_blank'>Centroid Molecular Dynamics Can Be Greatly Accelerated through Neural Network Learned Centroid Forces Derived from Path Integral Molecular Dynamics</a></td>
          <td>
            
              Timothy D Loose,
            
              Patrick G. Sahrmann,
            
              G. Voth,
            
          </td>
          <td>2022-08-16</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>5</td>
          <td>103</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6ca95861a225854f32cc25f67fc9452ba429d7f0" target='_blank'>Deep learning-based multiple particle tracking in complex system</a></td>
          <td>
            
              Xiaoming Xu,
            
              Jianjun Wei,
            
              Sheng Sang,
            
          </td>
          <td>2024-01-01</td>
          <td>AIP Advances</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/46d7ad5a6bc488e580db86cb13df8c378ddc80f1" target='_blank'>Revisiting Hybridization Kinetics with Improved Elementary Step Simulation</a></td>
          <td>
            
              Jordan Lovrod,
            
              Boyan Beronov,
            
              Chenwei Zhang,
            
              Erik Winfree,
            
              Anne Condon,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/85c64b9fe6c955cd961da38135f2d743337921aa" target='_blank'>John von Neumann Institute for Computing Sequential Monte Carlo Methods for Protein Folding</a></td>
          <td>
            
              P. Grassberger,
            
              D. Wolf,
            
              G. Münster,
            
              M. Kremer,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>9</td>
          <td>70</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9bd16217a295cfc0d496a7e9bce793ff28223b62" target='_blank'>A generative model to synthetize spatio-temporal dynamics of biomolecules in cells</a></td>
          <td>
            
              Lisa Balsollier,
            
              F. Lavancier,
            
              J. Salamero,
            
              Charles Kervrann,
            
          </td>
          <td>2023-03-13</td>
          <td>None</td>
          <td>1</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3c0ee52b6eebb908f508ea64ca3d1f5912b88b17" target='_blank'>On-line Optimization of Hamiltonian Replica Exchange Simulations</a></td>
          <td>
            
              J. MacCallum,
            
              Mir Ishruna Muniyat,
            
              Kari Gaalswyk,
            
          </td>
          <td>2017-12-03</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5ea78363da6c7d74de12ede29b0ce4caaea10c0a" target='_blank'>Finite Temperature String Method with Umbrella Sampling: Application on a Side Chain Flipping in Mhp1 Transporter.</a></td>
          <td>
            
              H. Song,
            
              Fangqiang Zhu,
            
          </td>
          <td>2017-04-20</td>
          <td>The journal of physical chemistry. B</td>
          <td>8</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bcade0826f7ff1b1732cdf05f0a92fe714b9dea2" target='_blank'>The construction of general basis functions in reweighting ensemble dynamics simulations: Reproduce equilibrium distribution in complex systems from multiple short simulation trajectories</a></td>
          <td>
            
              C. Zhang 张,
            
              Ming 明 Li 黎,
            
              X. Zhou 周,
            
          </td>
          <td>2015-10-20</td>
          <td>Chinese Physics B</td>
          <td>1</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/873545c5aa7506ae15e7045232b57b3dceaa62a7" target='_blank'>KINNTREX: a neural network to unveil protein mechanisms from time-resolved X-ray crystallography</a></td>
          <td>
            
              Gabriel Biener,
            
              T. Malla,
            
              P. Schwander,
            
              Marius Schmidt,
            
          </td>
          <td>2024-04-25</td>
          <td>IUCrJ</td>
          <td>0</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/696e0bf23f20245de860d9bcf50d4b241f7eaeeb" target='_blank'>A Likelihood based Approach for Building Trajectories from Intermittent Observations</a></td>
          <td>
            
              Peter K. Relich,
            
              Patrick J. Cutler,
            
              K. Lidke,
            
          </td>
          <td>2013-01-29</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8d384f01f557aa823310b7264babf322854c42b9" target='_blank'>A hybrid MD-kMC algorithm for folding proteins in explicit solvent.</a></td>
          <td>
            
              E. Peter,
            
              J. Shea,
            
          </td>
          <td>2014-03-12</td>
          <td>Physical chemistry chemical physics : PCCP</td>
          <td>28</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3358d4ea9f2e03c33b543e123e680d1756f4a55b" target='_blank'>Single-particle diffusional fingerprinting: A machine-learning framework for quantitative analysis of heterogeneous diffusion</a></td>
          <td>
            
              H. Pinholt,
            
              S. Bohr,
            
              Josephine F. Iversen,
            
              Wouter Boomsma,
            
              N. Hatzakis,
            
          </td>
          <td>2021-07-28</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>34</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6c1dd20526cf1095d26c30c88ba9e80891a24cf1" target='_blank'>Single particle diffusion characterization by deep learning</a></td>
          <td>
            
              Naor Granik,
            
              E. Nehme,
            
              Lucien E. Weiss,
            
              Maayan Levin,
            
              Michael Chein,
            
              E. Perlson,
            
              Y. Roichman,
            
              Y. Shechtman,
            
          </td>
          <td>2019-03-26</td>
          <td>bioRxiv</td>
          <td>110</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/82242d4b28cd557b91703d43726eabbbf54a4e27" target='_blank'>General, PDB-based collective variables for protein folding</a></td>
          <td>
            
              A. Ardèvol,
            
              Ferruccio Palazzesi,
            
              G. A. Tribello,
            
              M. Parrinello,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0d034a08c130691d20c06618bedc0a950b275489" target='_blank'>Weighted Distance Functions Improve Analysis of High-Dimensional Data: Application to Molecular Dynamics Simulations.</a></td>
          <td>
            
              Nicolas Blöchliger,
            
              A. Caflisch,
            
              A. Vitalis,
            
          </td>
          <td>2015-11-02</td>
          <td>Journal of chemical theory and computation</td>
          <td>11</td>
          <td>74</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c9c2e461a57b7b5e51a547f739174760f2ad7c2e" target='_blank'>Model-free inference of memory in conformational dynamics of a multi-domain protein</a></td>
          <td>
            
              L. Vollmar,
            
              Rick Bebon,
            
              J. Schimpf,
            
              Bastian Flietel,
            
              Sirin Celiksoy,
            
              Carsten Sonnichsen,
            
              Aljavz Godec,
            
              Thorsten Hugel,
            
          </td>
          <td>2024-04-25</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/271b425f60d895ed3fbf041005937b70a52f6d0a" target='_blank'>Simultaneous Computation of Dynamical and Equilibrium Information Using a Weighted Ensemble of Trajectories</a></td>
          <td>
            
              Ernesto Suárez,
            
              S. Lettieri,
            
              M. Zwier,
            
              C. Stringer,
            
              S. Subramanian,
            
              L. Chong,
            
              D. Zuckerman,
            
          </td>
          <td>2012-10-11</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>83</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4cbbde90af35257091bee4d3c109ac5df76950d7" target='_blank'>Geometric deep learning of particle motion by MAGIK</a></td>
          <td>
            
              Bahare Fatemi,
            
              Jonathan Halcrow,
            
              K. Jaqaman,
            
          </td>
          <td>2023-05-01</td>
          <td>Nature Machine Intelligence</td>
          <td>0</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ff75eeba95ac02994bb6a4cf609df14e8698a1e" target='_blank'>Uncertainty Quantification for Molecular Models via Stochastic Gradient MCMC</a></td>
          <td>
            
              Stephan Thaler,
            
              J. Zavadlav,
            
          </td>
          <td>None</td>
          <td>MATHMOD 2022 Discussion Contribution Volume</td>
          <td>2</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d23a35967f522244f5d74b1940a9062aec04c658" target='_blank'>A generative model to simulate spatiotemporal dynamics of biomolecules in cells</a></td>
          <td>
            
              Lisa Balsollier,
            
              F. Lavancier,
            
              J. Salamero,
            
              Charles Kervrann,
            
          </td>
          <td>2023-11-13</td>
          <td>Biological Imaging</td>
          <td>1</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f0e707152713e17696ba2e26927582e5e54dd904" target='_blank'>A critical assessment of hidden markov model sub‐optimal sampling strategies applied to the generation of peptide 3D models</a></td>
          <td>
            
              Alexis Lamiable,
            
              P. Thévenet,
            
              P. Tufféry,
            
          </td>
          <td>2016-08-05</td>
          <td>Journal of Computational Chemistry</td>
          <td>11</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/16bff30945258bdb414bcf603ab04d52ebd33814" target='_blank'>Imaging Metastable States and Transitions in Proteins by Trajectory Map.</a></td>
          <td>
            
              Chuanbiao Zhang,
            
              Jin Yu,
            
              Xin Zhou,
            
          </td>
          <td>2017-04-28</td>
          <td>The journal of physical chemistry. B</td>
          <td>5</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/170d64b44cf3fa61ec368d775a9ba49cd61e864f" target='_blank'>Classification of diffusion modes in single-particle tracking data: Feature-based versus deep-learning approach.</a></td>
          <td>
            
              Patrycja Kowalek,
            
              Hanna Loch-Olszewska,
            
              J. Szwabiński,
            
          </td>
          <td>2019-02-21</td>
          <td>Physical review. E</td>
          <td>77</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e6cf9471ddc2be7ea523c32f1eab5476776c9fbc" target='_blank'>Greedy Algorithms for Fast Discovery of Macrostates from Molecular Dynamics Simulations</a></td>
          <td>
            
              Haoyun Feng,
            
              J. Izaguirre,
            
              D. Thain,
            
              Badi Abdul-Wahid,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9d34a6d0e77834548c60b23f719677333e62b38f" target='_blank'>Numerical computation of effective thermal equilibria in stochastically switching Langevin systems.</a></td>
          <td>
            
              ,
            
              K. Newhall,
            
          </td>
          <td>2021-04-27</td>
          <td>Physical review. E</td>
          <td>7</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e563c5cde23f5557763be84e660e3635b0a02778" target='_blank'>Data-driven classification of ligand unbinding pathways.</a></td>
          <td>
            
              Dhiman Ray,
            
              M. Parrinello,
            
          </td>
          <td>2023-08-02</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>2</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3387172d924dfd4ba4d95cca9ca5f5e381ab808e" target='_blank'>Statistical Characterization of Protein Ensembles</a></td>
          <td>
            
              D. Rother,
            
              G. Sapiro,
            
              V. Pande,
            
          </td>
          <td>None</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>13</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/717fa5ba837da94dc7ac5dedc7e66639be19fa43" target='_blank'>Variational analysis of molecular free energy landscapes and AI implementations</a></td>
          <td>
            
              Bochuan Du,
            
              Pu Tian,
            
          </td>
          <td>None</td>
          <td>Acta Physica Sinica</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/334dc849e21285fa661a75bb896590b6e4f36e42" target='_blank'>Hierarchical variational models for statistical physics</a></td>
          <td>
            
              Jaan Altosaar,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>2</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ac387b5d68eaea83fe8d32273392d3ce4f17ef29" target='_blank'>Advances in machine learning for directed evolution.</a></td>
          <td>
            
              Bruce J. Wittmann,
            
              Kadina E. Johnston,
            
              Zachary Wu,
            
              F. Arnold,
            
          </td>
          <td>2021-02-26</td>
          <td>Current opinion in structural biology</td>
          <td>88</td>
          <td>111</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ea5e95c4b673ba3e10b234e10d96fd9843b7ae28" target='_blank'>A recurrent Markov state-space generative model for sequences</a></td>
          <td>
            
              Anand Ramachandran,
            
              S. Lumetta,
            
              E. Klee,
            
              Deming Chen,
            
          </td>
          <td>2019-04-11</td>
          <td>None</td>
          <td>1</td>
          <td>47</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/818605ad3cd1b50ddb2f6b1816913b443b291bf9" target='_blank'>Toward a Global View of the Conformational Landscape of the Human Kinome</a></td>
          <td>
            
              Daniel L. Parton,
            
              D. Shukla,
            
              Yutong Zhao,
            
              V. Pande,
            
              J. Chodera,
            
          </td>
          <td>2014-01-28</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/125f4d9b2c004a2b8d9dd2fc5f541805700c7604" target='_blank'>Bayesian Parameter Estimation in Ising and Potts Models : A Comparative Study with Applications to Protein Modeling</a></td>
          <td>
            
              Xiang-Jun Zhou,
            
              S. Schmidler,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>17</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d1806f70ee304004a32f068d014ec5db665964c5" target='_blank'>Long Short-Term Memory to predict 3D Amino acids Positions in GPCR Molecular Dynamics</a></td>
          <td>
            
              J. López-Correa,
            
              Caroline König,
            
              A. Vellido,
            
          </td>
          <td>2022-06-02</td>
          <td>None</td>
          <td>1</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7e1b07efa85ce354675644ab8f1dc45a02c0ac48" target='_blank'>Deep learning assisted single particle tracking for automated correlation between diffusion and function</a></td>
          <td>
            
              Jacob Kæstel-Hansen,
            
              Marilina de Sautu,
            
              A. Saminathan,
            
              Gustavo Scanavachi,
            
              Ricardo F. Bango Da Cunha Correia,
            
              Annette Juma Nielsen,
            
              Sara Vogt Bleshøy,
            
              Wouter Boomsma,
            
              Tom Kirchhausen,
            
              N. Hatzakis,
            
          </td>
          <td>2023-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/243469bfdee86e9de3894f5d87686b23b7f6366f" target='_blank'>Speeding Up Particle Trajectory Simulations under Moving Force Fields using GPUs</a></td>
          <td>
            
              Robert Patro,
            
              John P. Dickerson,
            
              Sujal Bista,
            
              S. Gupta,
            
              Amitabh Varshney,
            
          </td>
          <td>None</td>
          <td></td>
          <td>2</td>
          <td>60</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9e2fcbf361e3d89b10f1ae2e3d9c49876db42976" target='_blank'>Variational Neural Cellular Automata</a></td>
          <td>
            
              Rasmus Berg Palm,
            
              Miguel González Duque,
            
              Shyam Sudhakaran,
            
              S. Risi,
            
          </td>
          <td>2022-01-28</td>
          <td>ArXiv</td>
          <td>9</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6d076a7c4c92e2a5787187b40e383a3b53c06cce" target='_blank'>Single-Particle Diffusion Characterization by Deep Learning</a></td>
          <td>
            
              He Li,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>3</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d64aa4daca616d113e1279abc16158994503728c" target='_blank'>Scalable approximate Bayesian inference for particle tracking data</a></td>
          <td>
            
              Ruoxi Sun,
            
              L. Paninski,
            
          </td>
          <td>2018-03-05</td>
          <td>bioRxiv</td>
          <td>8</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ccdb6598c04c5750e9d4b29aae056ca9d03436c" target='_blank'>Ab initio simulations of the thermodynamic properties and phase transition of Fermi systems based on fictitious identical particles and physics-informed neural networks</a></td>
          <td>
            
              Yunuo Xiong,
            
              Hongwei Xiong,
            
          </td>
          <td>2024-02-11</td>
          <td>None</td>
          <td>1</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2cd290c76a7ab0f6b2daa0dc718279b36e927ea8" target='_blank'>Molecular Geometry-aware Transformer for accurate 3D Atomic System modeling</a></td>
          <td>
            
              Zheng Yuan,
            
              Yaoyun Zhang,
            
              Chuanqi Tan,
            
              Wei Wang,
            
              Feiran Huang,
            
              Songfang Huang,
            
          </td>
          <td>2023-02-02</td>
          <td>ArXiv</td>
          <td>5</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bfd37eb6b57586d510e8c0d2ed559f738871461e" target='_blank'>SHOT-R: A next generation algorithm for particle kinematics analysis</a></td>
          <td>
            
              Eloina Corradi,
            
              W. Boscheri,
            
              M. Baudet,
            
          </td>
          <td>2021-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/da96bb3eed082348dff760addae80dd02fbc47ef" target='_blank'>An Estimation Algorithm for General Linear Single Particle Tracking Models with Time-Varying Parameters</a></td>
          <td>
            
              Boris I. Godoy,
            
              Nicholas A. Vickers,
            
              S. Andersson,
            
          </td>
          <td>2021-02-01</td>
          <td>Molecules</td>
          <td>5</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c31e79a7ad8f4332dc2f3a0a9789911092011f93" target='_blank'>Transfer learning of memory kernels for transferable coarse-graining of polymer dynamics.</a></td>
          <td>
            
              Zhan Ma,
            
              Shu Wang,
            
              Minhee Kim,
            
              Kaibo Liu,
            
              Chun-Long Chen,
            
              Wenxiao Pan,
            
          </td>
          <td>2021-06-07</td>
          <td>Soft matter</td>
          <td>11</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4389fe25762e50aa668e0fced4de50d8cce0f917" target='_blank'>Sequence-to-Sequence Change-Point Detection in Single-Particle Trajectories via Recurrent Neural Network for Measuring Self-Diffusion</a></td>
          <td>
            
              Q. Martinez,
            
              C. Chen,
            
              J. Xia,
            
              H. Bahai,
            
          </td>
          <td>2023-03-15</td>
          <td>Transport in Porous Media</td>
          <td>2</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/57bac1883f2fd848ffeb1be3ca74c083d11e9cc6" target='_blank'>Efficient recurrent neural network methods for anomalously diffusing single particle short and noisy trajectories</a></td>
          <td>
            
              Òscar Garibo i Orts,
            
              M. Garcia-March,
            
              J. A. Conejero,
            
          </td>
          <td>2021-08-05</td>
          <td>Journal of Physics A: Mathematical and Theoretical</td>
          <td>11</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aad557289124b52793178fdf4668b18d711191ba" target='_blank'>Computationally Projecting the Influence of Nucleic Acid on Pathways of Nucleation-Limited Virus Capsid Assembly</a></td>
          <td>
            
              Gregory R. Smith,
            
              Lu Xie,
            
              R. Schwartz,
            
          </td>
          <td>2015-01-27</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f6b19727e2901195e9f0c190b552d14b5e06642f" target='_blank'>Bayesian Learning in Undirected Graphical Models: Approximate MCMC Algorithms</a></td>
          <td>
            
              Iain Murray,
            
              Zoubin Ghahramani,
            
          </td>
          <td>2004-07-07</td>
          <td>ArXiv</td>
          <td>115</td>
          <td>110</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f8ea5821491bef73aa31a6ca00d450bd4053b8ae" target='_blank'>A hidden Markov model approach to characterizing the photo-switching behavior of fluorophores</a></td>
          <td>
            
              Lekha Patel,
            
              N. Gustafsson,
            
              Yu Lin,
            
              R. Ober,
            
              Ricardo Henriques,
            
              E. Cohen,
            
          </td>
          <td>2017-11-22</td>
          <td>bioRxiv</td>
          <td>17</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b61fe62b2cf63a499f96205d6314f6bf70762a6d" target='_blank'>Combining the multicanonical ensemble with generative probabilistic models of local biomolecular structure</a></td>
          <td>
            
              J. Frellsen,
            
              T. Hamelryck,
            
              J. Ferkinghoff-Borg,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fb1b1d5ccdc893cd5aa03a7ed3cc5c7ba7056dca" target='_blank'>Single-shot self-supervised particle tracking</a></td>
          <td>
            
              Benjamin Midtvedt,
            
              Jesús Pineda,
            
              Fredrik Skärberg,
            
              Erik Ols'en,
            
              Harshith Bachimanchi,
            
              Emelie Wesén,
            
              E. K. Esbjörner,
            
              Erik Selander,
            
              F. Höök,
            
              Daniel Midtvedt,
            
              G. Volpe,
            
          </td>
          <td>2022-02-28</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>58</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58a530813dfcd75ebefa3640c5a2f768bf0b6ccf" target='_blank'>Computer Simulations of Protein Folding</a></td>
          <td>
            
              V. Pande,
            
              Eric J. Sorin,
            
              C. Snow,
            
              Y. M. Rhee,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1221a02080ddb019021c02ccc3060b1ddebbd14c" target='_blank'>Semantic segmentation of anomalous diffusion using deep convolutional networks</a></td>
          <td>
            
              Xiang Qu,
            
              Yidan Hu,
            
              Wenjie Cai,
            
              Yang Xu,
            
              Hu Ke,
            
              Guolong Zhu,
            
              Zihan Huang,
            
          </td>
          <td>2023-04-29</td>
          <td>Physical Review Research</td>
          <td>3</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/73d4ce2cdef026eb3f080aa075c93b7131fb5ad1" target='_blank'>A Continuous Restricted Boltzmann Machine with a Hardware-Amenable Learning Algorithm</a></td>
          <td>
            
              Hsin Chen,
            
              A. Murray,
            
          </td>
          <td>2002-08-28</td>
          <td>None</td>
          <td>33</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e60df5f640e047a6095a17b6e1c625b4d308b55e" target='_blank'>A review for cell and particle tracking on microscopy images using algorithms and deep learning technologies</a></td>
          <td>
            
              Hui-Jun Cheng,
            
              Ching-Hsien Hsu,
            
              Che-Lun Hung,
            
              Chun-Yuan Lin,
            
          </td>
          <td>2021-10-01</td>
          <td>Biomedical Journal</td>
          <td>8</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8f3461546103c9e65381097f68b7732416787ea6" target='_blank'>Statistical Characterization of Protein Ensembles (PREPRINT)</a></td>
          <td>
            
              D. Rother,
            
              G. Sapiro,
            
              V. Pande,
            
          </td>
          <td>2006-03-01</td>
          <td></td>
          <td>0</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1eeb028909089975227e80d9cabc68ae32703284" target='_blank'>Deep-learning method for data association in particle tracking</a></td>
          <td>
            
              Yao Yao,
            
              Ihor Smal,
            
              I. Grigoriev,
            
              A. Akhmanova,
            
              E. Meijering,
            
          </td>
          <td>2020-07-06</td>
          <td>Bioinformatics</td>
          <td>24</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ea55154e321ef96d806c2291ea6195ad1655866e" target='_blank'>Attention-based LSTM for Motion Switching Detection of Particles in Living Cells</a></td>
          <td>
            
              Bo You,
            
              Ge Yang,
            
          </td>
          <td>2021-07-18</td>
          <td>2021 International Joint Conference on Neural Networks (IJCNN)</td>
          <td>3</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8141a9bc98b8aaf9a83f2a3c47efef022c9d3b04" target='_blank'>Searching for Low Probability Opening Events in a DNA Sliding Clamp</a></td>
          <td>
            
              Reza Esmaeeli,
            
              B. Andal,
            
              Alberto Pérez,
            
          </td>
          <td>2022-02-01</td>
          <td>Life</td>
          <td>0</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9ec29c53e65b7511e48a94415476390ce93c8de7" target='_blank'>Machine learning classification of particle trajectories in living cells: machine learning versus statistical testing hypothesis</a></td>
          <td>
            
              J. Janczura,
            
              Patrycja Kowalek,
            
              Hanna Loch-Olszewska,
            
              Janusz Szwabi'nski,
            
              A. Weron,
            
          </td>
          <td>2020-05-13</td>
          <td>arXiv: Quantitative Methods</td>
          <td>1</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ef3ae8843c44234d99e3923ba787600fa1b2d4cf" target='_blank'>Optimal Kullback–Leibler Aggregation via Information Bottleneck</a></td>
          <td>
            
              B. Geiger,
            
              Tatjana Petrov,
            
              G. Kubin,
            
              H. Koeppl,
            
          </td>
          <td>2013-04-24</td>
          <td>IEEE Transactions on Automatic Control</td>
          <td>25</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e7dc4ad91cf200280a328bc83282004e88d03d68" target='_blank'>Tracking droplets in soft granular flows with deep learning techniques</a></td>
          <td>
            
              M. Durve,
            
              F. Bonaccorso,
            
              A. Montessori,
            
              M. Lauricella,
            
              A. Tiribocchi,
            
              S. Succi,
            
          </td>
          <td>2021-07-04</td>
          <td>European Physical Journal plus</td>
          <td>9</td>
          <td>67</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7ed454c27e7a64149bd434ec7d5d2a61dd3cde1b" target='_blank'>Dense particle tracking using a learned predictive model</a></td>
          <td>
            
              K. Mallery,
            
              Siyao Shao,
            
              Jiarong Hong,
            
          </td>
          <td>2020-10-01</td>
          <td>Experiments in Fluids</td>
          <td>13</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/31f2fefbad5a6b339c2871e7e1ef6ecee7c849f5" target='_blank'>Deep Gaussian Markov random fields</a></td>
          <td>
            
              Per Sid'en,
            
              F. Lindsten,
            
          </td>
          <td>2020-02-18</td>
          <td>None</td>
          <td>20</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aba24ee466471d1206c3fc868645aa20ed81e382" target='_blank'>FSATOOL: A useful tool to do the conformational sampling and trajectory analysis work for biomolecules</a></td>
          <td>
            
              Haomiao Zhang,
            
              Qiankun Gong,
            
              Haozhe Zhang,
            
              Changjun Chen,
            
          </td>
          <td>2019-10-11</td>
          <td>Journal of Computational Chemistry</td>
          <td>5</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f2606da69262a9eb0f3624e2de37e6cecb87472a" target='_blank'>Inference in particle tracking experiments by passing messages between images</a></td>
          <td>
            
              M. Chertkov,
            
              L. Kroc,
            
              Florent Krzakala,
            
              M. Vergassola,
            
              L. Zdeborová,
            
          </td>
          <td>2009-09-23</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>62</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a65b95912eab3d2ec10f7a21f4b33aff5eb50287" target='_blank'>Time Varying Particle Data Feature Extraction and Tracking with Neural Networks</a></td>
          <td>
            
              Haoyu Li,
            
              Han-Wei Shen,
            
          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/15385a00d87665f11bbc9da245e3ff564b1bc500" target='_blank'>Konrad-zuse-zentrum F ¨ Ur Informationstechnik Berlin Metastable Conformations via Successive Perron-cluster Cluster Analysis of Dihedrals Metastable Conformations via Successive Perron-cluster Cluster Analysis of Dihedrals</a></td>
          <td>
            
              F. Cordes,
            
              Marcus Weber,
            
              Johannes Schmidt-Ehrenberg,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>17</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/862ba8ed1ea3db329450578e338fb79c45e97143" target='_blank'>Deep neural networks for data association in particle tracking</a></td>
          <td>
            
              Yao Yao,
            
              Ihor Smal,
            
              E. Meijering,
            
          </td>
          <td>2018-04-04</td>
          <td>2018 IEEE 15th International Symposium on Biomedical Imaging (ISBI 2018)</td>
          <td>23</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5793a031c405548de6914f3174991c291f7e3c5e" target='_blank'>Classification of particle trajectories in living cells: Machine learning versus statistical testing hypothesis for fractional anomalous diffusion.</a></td>
          <td>
            
              J. Janczura,
            
              Patrycja Kowalek,
            
              Hanna Loch-Olszewska,
            
              Janusz Szwabi'nski,
            
              A. Weron,
            
          </td>
          <td>2020-09-01</td>
          <td>Physical review. E</td>
          <td>38</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/01977b4ceeeed8a57cf1b4263be3b13e0b8d8c98" target='_blank'>Super-resolution of spin configurations based on flow-based generative models</a></td>
          <td>
            
              K. Shiina,
            
              Lee Hwee Kuan,
            
              H. Mori,
            
              Y. Okabe,
            
              Y. Tomita,
            
          </td>
          <td>2021-08-25</td>
          <td>None</td>
          <td>0</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/feb249fd86dea82b7c50ce8cd6eaea6d66db5c1a" target='_blank'>Conformational Ensemble Generation via Constraint-based Rigid-body Dynamics Guided by the Elastic Network Model</a></td>
          <td>
            
              K. Borowski,
            
          </td>
          <td>2011-08-29</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/39cbb3a0c09ff8bb30437dd8503935ca2aa07d91" target='_blank'>Single trajectory characterization via machine learning</a></td>
          <td>
            
              Gorka Muñoz-Gil,
            
              M. Garcia-March,
            
              C. Manzo,
            
              J. Martín-Guerrero,
            
              M. Lewenstein,
            
          </td>
          <td>2020-01-14</td>
          <td>New Journal of Physics</td>
          <td>65</td>
          <td>92</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dd20edb9b6a45769bc4b2e83a80eb818be805502" target='_blank'>Temporal analysis of relative distances (TARDIS) is a robust, parameter-free alternative to single-particle tracking</a></td>
          <td>
            
              Koen J. A. Martens,
            
              Bartosz Turkowyd,
            
              J. Hohlbein,
            
              Ulrike Endesfelder,
            
          </td>
          <td>2023-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/741835ed93c57a852f2b923ba86e21ce8a2a07c6" target='_blank'>Accelerating Dissociative Events in Molecular Dynamics Simulations by Selective Potential Scaling</a></td>
          <td>
            
              I. Deb,
            
              A. Frank,
            
          </td>
          <td>2019-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1967f8312cbbfad8de03c3f3a3a9fe9de3b7b169" target='_blank'>Change Point Detection of Events in Molecular Simulations using dupin</a></td>
          <td>
            
              Brandon L. Butler,
            
              Domagoj Fijan,
            
              S. Glotzer,
            
          </td>
          <td>2023-12-22</td>
          <td>None</td>
          <td>0</td>
          <td>79</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1077c1ee5ccbc8063294ba4705bb8696eebfd32a" target='_blank'>Modeling mass spectrometry-based protein analysis.</a></td>
          <td>
            
              J. Eriksson,
            
              D. Fenyö,
            
          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>3</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d5369c30fd773ceb3a4278a53b816a33d4aab84e" target='_blank'>Identification of Diffusive States in Tracking Applications Using Unsupervised Deep Learning Methods</a></td>
          <td>
            
              Hélène Kabbech,
            
              Ihor Smal,
            
          </td>
          <td>2022-03-28</td>
          <td>2022 IEEE 19th International Symposium on Biomedical Imaging (ISBI)</td>
          <td>3</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5b605a00ab8eca8a3bd8423d73a4fcac356a7d6e" target='_blank'>Tracking Intermittent Particles with Self-Learned Visual Features</a></td>
          <td>
            
              Raphael Reme,
            
              Victor Piriou,
            
              Alison Hanson,
            
              R. Yuste,
            
              A. Newson,
            
              E. Angelini,
            
              Jean-Christophe Olivo-Marin,
            
              T. Lagache,
            
          </td>
          <td>2023-04-18</td>
          <td>2023 IEEE 20th International Symposium on Biomedical Imaging (ISBI)</td>
          <td>2</td>
          <td>107</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a95aea866dd04a544e50f60925466a75535c2a17" target='_blank'>Deep learning based object tracking in walking droplet and granular intruder experiments</a></td>
          <td>
            
              Erdi Kara,
            
              George Zhang,
            
              Joseph J. Williams,
            
              Gonzalo Ferrandez-Quinto,
            
              Leviticus J. Rhoden,
            
              Maximilian Kim,
            
              J. Kutz,
            
              Aminur Rahman,
            
          </td>
          <td>2023-01-27</td>
          <td>Journal of Real-Time Image Processing</td>
          <td>3</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2909dd831795ce2ebbf891d3b323049e91e1ec5c" target='_blank'>An ergodic theorem for the weighted ensemble method</a></td>
          <td>
            
              D. Aristoff,
            
          </td>
          <td>2019-06-03</td>
          <td>Journal of Applied Probability</td>
          <td>3</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0b48d278467e20ece2d8f1d03f4a5b74555c2095" target='_blank'>Avoiding False Positive Conclusions in Molecular Simulation: The Importance of Replicas.</a></td>
          <td>
            
              B. Knapp,
            
              Luis Ospina,
            
              C. Deane,
            
          </td>
          <td>2018-10-24</td>
          <td>Journal of chemical theory and computation</td>
          <td>180</td>
          <td>48</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    // columnDefs: [
    //     {"className": "dt-center", "targets": "_all"},
    //     // set width for both columns 0 and 1 as 25%
    //     { width: '40%', targets: 4 }

    //   ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
</script>
</html>