---
hide:
 - navigation
---
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-05-06 06:07:17 UTC</i>
  </p>
  

  <p>
  <h3>Recommendations for the article <i>VAMPnets for deep learning of molecular kinetics</i></h3>
  <table id="table1" class="display" style="width:100%">
  <thead>
    <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f8a6341496701a0d7fdaa50e6a5aa157236a0ff8" target='_blank'>Author Correction: VAMPnets for deep learning of molecular kinetics</a></td>
          <td>
            
              Andreas Mardt,
            
              Luca Pasquali,
            
              Hao Wu,
            
              F. Noé,
            
          </td>
          <td>2018-10-22</td>
          <td>Nature Communications</td>
          <td>24</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c4d9b7166a2ac580c195d72f64489b3f4e0b9c23" target='_blank'>DeepWEST: Deep Learning of Kinetic Models with the Weighted Ensemble Simulation Toolkit for Enhanced Sampling.</a></td>
          <td>
            
              Anupam Anand Ojha,
            
              Saumya Thakur,
            
              Surl-Hee Ahn,
            
              Rommie E. Amaro,
            
          </td>
          <td>2023-01-31</td>
          <td>Journal of chemical theory and computation</td>
          <td>7</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e15c87cfb4d2b1f82076331f9c58085598440e46" target='_blank'>GraphVAMPNet, using graph neural networks and variational approach to markov processes for dynamical modeling of biomolecules</a></td>
          <td>
            
              Mahdi Ghorbani,
            
              Samarjeet Prasad,
            
              Jeffery B. Klauda,
            
              B. Brooks,
            
          </td>
          <td>2022-01-12</td>
          <td>The Journal of chemical physics</td>
          <td>18</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/86de5fbe09c4381f548e1312d53f15d898c76c78" target='_blank'>Mind reading of the proteins: Deep-learning to forecast molecular dynamics</a></td>
          <td>
            
              C. Gupta,
            
              J. Cava,
            
              Daipayan Sarkar,
            
              Eric A. Wilson,
            
              J. Vant,
            
              Steven Murray,
            
              A. Singharoy,
            
              S. Karmaker,
            
          </td>
          <td>2020-07-29</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2e7163e31e9b32cec11005678bae9e1dbeb6d573" target='_blank'>Nonlinear Discovery of Slow Molecular Modes using Hierarchical Dynamics Encoders</a></td>
          <td>
            
              ,
            
              Hythem Sidky,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2019-02-09</td>
          <td>The Journal of chemical physics</td>
          <td>74</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2d3000d245988a02d3c1060211e9d89c67147b49" target='_blank'>Molecular latent space simulators</a></td>
          <td>
            
              Hythem Sidky,
            
              Wei Chen,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2020-07-01</td>
          <td>Chemical Science</td>
          <td>30</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/50c74a10fcddb0e41a5a00b965383fdc52534eec" target='_blank'>Machine Learning in Molecular Dynamics Simulations of Biomolecular Systems</a></td>
          <td>
            
              Christopher Kolloff,
            
              Simon Olsson,
            
          </td>
          <td>2022-05-06</td>
          <td>None</td>
          <td>2</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/13b52cd927d3daa5c95528b987734d97474e790c" target='_blank'>Markov field models: Scaling molecular kinetics approaches to large molecular machines.</a></td>
          <td>
            
              Tim Hempel,
            
              Simon Olsson,
            
              Frank No'e,
            
          </td>
          <td>2022-06-23</td>
          <td>Current opinion in structural biology</td>
          <td>6</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/260f212da0cf8b3309f0ab3a130a0ad22993892e" target='_blank'>Markov State Models: From an Art to a Science.</a></td>
          <td>
            
              B. Husic,
            
              V. Pande,
            
          </td>
          <td>2018-02-02</td>
          <td>Journal of the American Chemical Society</td>
          <td>552</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8271ca0056e9a3a09b33963054d04b7c14c9ed69" target='_blank'>Active Learning of the Conformational Ensemble of Proteins using Maximum Entropy VAMPNets</a></td>
          <td>
            
              D. Kleiman,
            
              D. Shukla,
            
          </td>
          <td>2023-01-13</td>
          <td>bioRxiv</td>
          <td>7</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/41c0783182ca3e23732b95ddfdb6654cd23a4b45" target='_blank'>Deep learning to decompose macromolecules into independent Markovian domains</a></td>
          <td>
            
              Andreas Mardt,
            
              Tim Hempel,
            
              C. Clementi,
            
              F. Noé,
            
          </td>
          <td>2022-10-13</td>
          <td>Nature Communications</td>
          <td>13</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bc4e32c1b7fe39a37b21ec4c0624cfa45a6b36df" target='_blank'>Achieving Reversible Ligand-Protein Unbinding with Deep Learning and Molecular Dynamics through RAVE</a></td>
          <td>
            
              João Marcelo Lamim Ribeiro,
            
              P. Tiwary,
            
          </td>
          <td>2018-08-25</td>
          <td>bioRxiv</td>
          <td>6</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bab880c698b850ba18c08b9d93b73ffb538a417b" target='_blank'>Accelerated Simulations of Molecular Systems through Learning of their Effective Dynamics</a></td>
          <td>
            
              Pantelis R. Vlachas,
            
              J. Zavadlav,
            
              M. Praprotnik,
            
              P. Koumoutsakos,
            
          </td>
          <td>2021-02-17</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>77</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/51894ffcf778630e7b021bc473d13d028a3b9158" target='_blank'>DynamicBind: predicting ligand-specific protein-ligand complex structure with a deep equivariant generative model</a></td>
          <td>
            
              Wei Lu,
            
              Jixian Zhang,
            
              Weifeng Huang,
            
              Ziqiao Zhang,
            
              Xiangyu Jia,
            
              Zhenyu Wang,
            
              Leilei Shi,
            
              Chengtao Li,
            
              Peter G Wolynes,
            
              Shuangjia Zheng,
            
          </td>
          <td>2024-02-05</td>
          <td>Nature Communications</td>
          <td>6</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0107d0f47f7d1ccf290538922728dcece452c0b5" target='_blank'>Note: Variational Encoding of Protein Dynamics Benefits from Maximizing Latent Autocorrelation</a></td>
          <td>
            
              H. Wayment-Steele,
            
              V. Pande,
            
          </td>
          <td>2018-03-17</td>
          <td>The Journal of chemical physics</td>
          <td>6</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/204cedd41db9db3623226e9dfce6d82aaa3f0f41" target='_blank'>Machine learning for collective variable discovery and enhanced sampling in biomolecular simulation</a></td>
          <td>
            
              Hythem Sidky,
            
              Wei Chen,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2020-03-03</td>
          <td>Molecular Physics</td>
          <td>86</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b674a7aee72e9b9cc5390eca13f9c5c7812f2ba0" target='_blank'>Machine learning for molecular simulation</a></td>
          <td>
            
              F. Noé,
            
              A. Tkatchenko,
            
              K. Müller,
            
              C. Clementi,
            
          </td>
          <td>2019-11-07</td>
          <td>Annual review of physical chemistry</td>
          <td>480</td>
          <td>126</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9b70bf13b098d5388ed3acbaedfc7835c2de06fd" target='_blank'>An Efficient Path Classification Algorithm Based on Variational Autoencoder to Identify Metastable Path Channels for Complex Conformational Changes.</a></td>
          <td>
            
              Yunrui Qiu,
            
              Michael S O'Connor,
            
              Mingyi Xue,
            
              Bojun Liu,
            
              Xuhui Huang,
            
          </td>
          <td>2023-06-29</td>
          <td>Journal of chemical theory and computation</td>
          <td>8</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/915f23b0ab17625cec9863dc2eee1d603dcac80c" target='_blank'>Learning Markovian dynamics with spectral maps.</a></td>
          <td>
            
              Jakub Rydzewski,
            
              Tuğçe Gökdemir,
            
          </td>
          <td>2023-11-28</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6db34306fd106dee45a5f252e9f2fc8ed11c80f7" target='_blank'>Unsupervised deep learning for molecular dynamics simulations: a novel analysis of protein–ligand interactions in SARS-CoV-2 Mpro</a></td>
          <td>
            
              Jessica Mustali,
            
              Ikki Yasuda,
            
              Yoshinori Hirano,
            
              Kenji Yasuoka,
            
              Alfonso Gautieri,
            
              Noriyoshi Arai,
            
          </td>
          <td>2023-11-16</td>
          <td>RSC Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d03d605aa500b66f4ac1f9ae486c56474ae62c94" target='_blank'>Markov State Models to Study the Functional Dynamics of Proteins in the Wake of Machine Learning</a></td>
          <td>
            
              K. Konovalov,
            
              I. C. Unarta,
            
              Siqin Cao,
            
              Eshani C. Goonetilleke,
            
              Xuhui Huang,
            
          </td>
          <td>2021-08-04</td>
          <td>JACS Au</td>
          <td>47</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fd1fb1d3e31260eaa72e3bff278b0cf837989a80" target='_blank'>RevGraphVAMP: A protein molecular simulation analysis model combining graph convolutional neural networks and physical constraints</a></td>
          <td>
            
              Ying Huang,
            
              Huiling Zhang,
            
              Zhenli Lin,
            
              Yanjie Wei,
            
              Wenhui Xi,
            
          </td>
          <td>2024-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5c6ee980b791766ba036cba282bcbcbcb2e5309e" target='_blank'>Deep Generative Markov State Models</a></td>
          <td>
            
              Hao Wu,
            
              Andreas Mardt,
            
              Luca Pasquali,
            
              F. Noé,
            
          </td>
          <td>2018-05-19</td>
          <td>ArXiv</td>
          <td>53</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/404b4fbf580f6c9dfb340603618e9840cd5e760f" target='_blank'>Chapter 12. Machine Learning Algorithms for the Analysis of Molecular Dynamics Trajectories</a></td>
          <td>
            
              Abbas Saleem Khan,
            
              B. Sim,
            
              Dongqing Wei,
            
          </td>
          <td>2021-09-24</td>
          <td>Theoretical and Computational Chemistry Series</td>
          <td>0</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8f771f2fc3b8f446e7a1f1a15fb158a34966e294" target='_blank'>Explore Protein Conformational Space With Variational Autoencoder</a></td>
          <td>
            
              Hao Tian,
            
              Xi Jiang,
            
              Francesco Trozzi,
            
              Sian Xiao,
            
              Eric C. Larson,
            
              Peng Tao,
            
          </td>
          <td>2021-09-23</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>24</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9ebf5c7ac8fc22961e7dc2130776a9b7d92d6d94" target='_blank'>Learning reduced kinetic Monte Carlo models of complex chemistry from molecular dynamics</a></td>
          <td>
            
              Qian Yang,
            
              C. Sing-Long,
            
              E. Reed,
            
          </td>
          <td>2017-06-19</td>
          <td>Chemical Science</td>
          <td>19</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d8d8e2c04ca47bd628bd2a499e03ad7cd29633da" target='_blank'>Time-lagged autoencoders: Deep learning of slow collective variables for molecular kinetics</a></td>
          <td>
            
              C. Wehmeyer,
            
              F. Noé,
            
          </td>
          <td>2017-10-30</td>
          <td>The Journal of chemical physics</td>
          <td>328</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4813cd2feebbcef748d23902884261f10fb553e2" target='_blank'>Machine Learning Deciphered Molecular Mechanistics with Accurate Kinetic and Thermodynamic Prediction.</a></td>
          <td>
            
              Junlin Dong,
            
              Shiyu Wang,
            
              Wenqiang Cui,
            
              Xiaolin Sun,
            
              Haojie Guo,
            
              Hailu Yan,
            
              Horst Vogel,
            
              Zhi Wang,
            
              Shuguang Yuan,
            
          </td>
          <td>2024-02-23</td>
          <td>Journal of chemical theory and computation</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9868cca5b562f4ea41bff4b5ee4ea63e62006430" target='_blank'>Learning dynamical information from static protein and sequencing data</a></td>
          <td>
            
              P. Pearce,
            
              Francis G. Woodhouse,
            
              Aden Forrow,
            
              Ashley J. Kelly,
            
              H. Kusumaatmaja,
            
              J. Dunkel,
            
          </td>
          <td>2018-08-27</td>
          <td>Nature Communications</td>
          <td>12</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1c4c2983bc3719c80d93e39e59c58dd4c5a99afb" target='_blank'>Variational deep learning of equilibrium transition path ensembles.</a></td>
          <td>
            
              A. Singh,
            
              David T. Limmer,
            
          </td>
          <td>2023-02-28</td>
          <td>The Journal of chemical physics</td>
          <td>3</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f8701f745cb0a5871c21f86e2045b190103c230c" target='_blank'>Deeptime: a Python library for machine learning dynamical models from time series data</a></td>
          <td>
            
              Moritz Hoffmann,
            
              Martin K. Scherer,
            
              Tim Hempel,
            
              Andreas Mardt,
            
              Brian de Silva,
            
              B. Husic,
            
              Stefan Klus,
            
              Hao Wu,
            
              N. Kutz,
            
              S. Brunton,
            
              F. Noé,
            
          </td>
          <td>2021-10-28</td>
          <td>Machine Learning: Science and Technology</td>
          <td>56</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cb4d8f8b2b1169d210d289c6b2deac0ccbbc34fe" target='_blank'>An Information Bottleneck Approach for Markov Model Construction</a></td>
          <td>
            
              Dedi Wang,
            
              Yunrui Qiu,
            
              E. Beyerle,
            
              Xuhui Huang,
            
              P. Tiwary,
            
          </td>
          <td>2024-04-03</td>
          <td>None</td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/83e8e8d89e64de17d3d4709051a1f799c0b9ae22" target='_blank'>Deep learning for variational multiscale molecular modeling.</a></td>
          <td>
            
              Jun Zhang,
            
              Yao-Kun Lei,
            
              Yi Isaac Yang,
            
              Y. Gao,
            
          </td>
          <td>2020-06-29</td>
          <td>The Journal of chemical physics</td>
          <td>11</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ec461f9fd4b19b52508e85f3ffb7b7e70fc5d67" target='_blank'>Chasing collective variables using temporal data-driven strategies</a></td>
          <td>
            
              Haochuan Chen,
            
              C. Chipot,
            
          </td>
          <td>2023-01-06</td>
          <td>QRB Discovery</td>
          <td>9</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/68c0f4d535078674fd2ce738306eb72171a0677d" target='_blank'>Accelerated Simulations of Molecular Systems through Learning of Effective Dynamics.</a></td>
          <td>
            
              Pantelis R. Vlachas,
            
              J. Zavadlav,
            
              M. Praprotnik,
            
              P. Koumoutsakos,
            
          </td>
          <td>2021-12-10</td>
          <td>Journal of chemical theory and computation</td>
          <td>27</td>
          <td>77</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d00252df8465a93ba68dda823d8adb8e36418560" target='_blank'>Analyzing Molecular Dynamics Trajectories Thermodynamically through Artificial Intelligence.</a></td>
          <td>
            
              Xuyang Liu,
            
              Jingya Xing,
            
              Haohao Fu,
            
              Xueguang Shao,
            
              W. Cai,
            
          </td>
          <td>2024-01-09</td>
          <td>Journal of chemical theory and computation</td>
          <td>2</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8c9f4af01a841b431917e605a85746154d49529d" target='_blank'>Spectral Maps for Learning Reduced Representations of Molecular Systems</a></td>
          <td>
            
              Tuugcce Gokdemir,
            
              Jakub Rydzewski,
            
          </td>
          <td>2023-11-07</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2646b021a73a0651c4dac42e99d757319bfa0832" target='_blank'>Learning Protein Dynamics with Metastable Switching Systems</a></td>
          <td>
            
              Bharath Ramsundar,
            
              V. Pande,
            
          </td>
          <td>2016-10-05</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bef7c61e1d111ab3d4b0ad39291346181aeabfe6" target='_blank'>Coupling Molecular Dynamics and Deep Learning to Mine Protein Conformational Space</a></td>
          <td>
            
              M. Degiacomi,
            
          </td>
          <td>2019-06-01</td>
          <td>Z--BiochemRN: Protein Folding (Proteins) (Topic)</td>
          <td>77</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/984ca90a7b3912db88b02a4cf8fdfef4626f7752" target='_blank'>Modeling Molecular Structures with Intrinsic Diffusion Models</a></td>
          <td>
            
              Gabriele Corso,
            
          </td>
          <td>2023-02-23</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d3d51ccff69b92a784a2e234d39f1f2c950bf1e1" target='_blank'>Machine learning approaches for analyzing and enhancing molecular dynamics simulations.</a></td>
          <td>
            
              Yihang Wang,
            
              João Marcelo Lamim Ribeiro,
            
              P. Tiwary,
            
          </td>
          <td>2019-09-25</td>
          <td>Current opinion in structural biology</td>
          <td>168</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5b10349eade93332fbff9127b5273382103fdf27" target='_blank'>Deep Learning for Multi-Scale Molecular Modeling</a></td>
          <td>
            
              ,
            
              Yao-Kun Lei,
            
              Yi Isaac Yang,
            
              Y. Gao,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4af4996d0dd209282e25f88ef7f280ef54b21be7" target='_blank'>Reaction coordinate flows for model reduction of molecular kinetics</a></td>
          <td>
            
              Hao Wu,
            
              Frank No'e,
            
          </td>
          <td>2023-09-11</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/538d0f3f70c90dfd8d9f6e7bba3f50ea4058c34c" target='_blank'>High-Resolution Markov State Models for the Dynamics of Trp-Cage Miniprotein Constructed over Slow Folding Modes Identified by State-Free Reversible VAMPnets.</a></td>
          <td>
            
              Hythem Sidky,
            
              ,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2019-06-12</td>
          <td>The journal of physical chemistry. B</td>
          <td>44</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/47171992793ccd578bd34a222537e71bc8d0def8" target='_blank'>Learning protein conformational space by enforcing physics with convolutions and latent interpolations</a></td>
          <td>
            
              Venkatakrishnan Ramaswamy,
            
              Chris G. Willcocks,
            
              M. Degiacomi,
            
          </td>
          <td>2019-10-10</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5fdcaee10a43c2f4df8b44e64042ec0ff0d40a5f" target='_blank'>Deep learning of dynamically responsive chemical Hamiltonians with semiempirical quantum mechanics</a></td>
          <td>
            
              Guoqing Zhou,
            
              N. Lubbers,
            
              K. Barros,
            
              S. Tretiak,
            
              B. Nebgen,
            
          </td>
          <td>2022-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>20</td>
          <td>69</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/267db6e5e036b0f4bccc9e9b06085288638e31f2" target='_blank'>Deep learning path-like collective variable for enhanced sampling molecular dynamics</a></td>
          <td>
            
              Thorben Frohlking,
            
              Luigi Bonati,
            
              Valerio Rizzi,
            
              Francesco L. Gervasio,
            
          </td>
          <td>2024-02-02</td>
          <td>None</td>
          <td>2</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e1834d63294230b37fb4e0709b8d9bfca640d70c" target='_blank'>Deep clustering of protein folding simulations</a></td>
          <td>
            
              D. Bhowmik,
            
              Shang Gao,
            
              M. T. Young,
            
              A. Ramanathan,
            
          </td>
          <td>2018-06-12</td>
          <td>BMC Bioinformatics</td>
          <td>65</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f3eae9b9dc159bd46e1669cf26cc5c71ae2f32f" target='_blank'>Using VAEs to Learn Latent Variables: Observations on Applications in cryo-EM</a></td>
          <td>
            
              Daniel G Edelberg,
            
              Roy R. Lederman,
            
          </td>
          <td>2023-03-13</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e4140d67d59c61de86d2a9de31380e01a07ef85b" target='_blank'>Past–future information bottleneck for sampling molecular reaction coordinate simultaneously with thermodynamics and kinetics</a></td>
          <td>
            
              Yihang Wang,
            
              João Marcelo Lamim Ribeiro,
            
              P. Tiwary,
            
          </td>
          <td>2019-08-08</td>
          <td>Nature Communications</td>
          <td>118</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d1516aeab1a504cc02237fdd5957c9b421498110" target='_blank'>Deep Learning Unravels a Dynamic Hierarchy While Empowering Molecular Dynamics Simulations</a></td>
          <td>
            
              Ariel Fernández,
            
          </td>
          <td>2020-02-13</td>
          <td>Annalen der Physik</td>
          <td>2</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/21e8bbd2846539c180b901b747e579f82027d375" target='_blank'>Pre-training Protein Models with Molecular Dynamics Simulations for Drug Binding</a></td>
          <td>
            
              Fang Wu,
            
              Q. Zhang,
            
              Dragomir R. Radev,
            
              Yuyang Wang,
            
              Xurui Jin,
            
              Yinghui Jiang,
            
              Z. Niu,
            
              Stan Z. Li,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>4</td>
          <td>73</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dfec6b86a6080a722f96c73bc557169380204137" target='_blank'>DeepCV: A Deep Learning Framework for Blind Search of Collective Variables in Expanded Configurational Space</a></td>
          <td>
            
              R. Ketkaew,
            
              S. Luber,
            
          </td>
          <td>2022-11-29</td>
          <td>Journal of chemical information and modeling</td>
          <td>10</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3999faeea4b6fc58615245072ce010a0e6f641ea" target='_blank'>Systematic control of collective variables learned from variational autoencoders.</a></td>
          <td>
            
              Jacob I Monroe,
            
              V. K. Shen,
            
          </td>
          <td>2022-08-09</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4244b9b2515e37a20b254fa8f9ed7305f2f64d7e" target='_blank'>Interpretable embeddings from molecular simulations using Gaussian mixture variational autoencoders</a></td>
          <td>
            
              Yasemin Bozkurt Varolgunes,
            
              T. Bereau,
            
              J. F. Rudzinski,
            
          </td>
          <td>2019-12-22</td>
          <td>Machine Learning: Science and Technology</td>
          <td>36</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c9561b15c25ca781ccf1a384e978e28341bac0e4" target='_blank'>State-specific protein–ligand complex structure prediction with a multiscale deep generative model</a></td>
          <td>
            
              Zhuoran Qiao,
            
              Weili Nie,
            
              Arash Vahdat,
            
              Thomas F. Miller,
            
              Anima Anandkumar,
            
          </td>
          <td>2022-09-30</td>
          <td>Nature Machine Intelligence</td>
          <td>16</td>
          <td>60</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/744249d8846dad1cccaeb835119c39c9a10f079b" target='_blank'>A deep encoder-decoder framework for identifying distinct ligand binding pathways.</a></td>
          <td>
            
              Satyabrata Bandyopadhyay,
            
              Jagannath Mondal,
            
          </td>
          <td>2023-05-15</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bbd38193f26f04e021bff620878b90ec0e91b23c" target='_blank'>MLCV: Bridging Machine-Learning-Based Dimensionality Reduction and Free-Energy Calculation</a></td>
          <td>
            
              Haochuan Chen,
            
              Han Liu,
            
              Heying Feng,
            
              Haohao Fu,
            
              W. Cai,
            
              Xueguang Shao,
            
              C. Chipot,
            
          </td>
          <td>2021-12-23</td>
          <td>Journal of chemical information and modeling</td>
          <td>16</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e89b8240b54e2beccb99d7f6c82ee369b4af91c4" target='_blank'>Learning generative models of molecular dynamics</a></td>
          <td>
            
              N. Razavian,
            
              Hetunandan Kamisetty,
            
              C. Langmead,
            
          </td>
          <td>2012-01-17</td>
          <td>BMC Genomics</td>
          <td>8</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7f02647a9029d7f65da697a8bc943971c034695b" target='_blank'>P RO GAE: A G EOMETRIC A UTOENCODER - BASED G ENERATIVE M ODEL FOR D ISENTANGLING P ROTEIN C ONFORMATIONAL S PACE</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ac055a76189bda2cd0c47998a74bcfc9bfa4a7cb" target='_blank'>Transferable deep generative modeling of intrinsically disordered protein conformations</a></td>
          <td>
            
              Giacomo Janson,
            
              M. Feig,
            
          </td>
          <td>2024-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/55705bec7256bb2f3ceec805823de4e87bee87f1" target='_blank'>Machine Learning for Molecular Dynamics on Long Timescales</a></td>
          <td>
            
              F. Noé,
            
          </td>
          <td>2018-12-18</td>
          <td>ArXiv</td>
          <td>30</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/01fafc7042cd2a1c9ec2d3832ffd6d7e0d93e9b6" target='_blank'>Multitask machine learning of collective variables for enhanced sampling of rare events</a></td>
          <td>
            
              Lixin Sun,
            
              Jonathan Vandermause,
            
              Simon L. Batzner,
            
              Yu Xie,
            
              David J. Clark,
            
              Wei Chen,
            
              B. Kozinsky,
            
          </td>
          <td>2020-12-07</td>
          <td>Journal of chemical theory and computation</td>
          <td>27</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2b624accabd4b40bb341ba70038686c043efcace" target='_blank'>Multiensemble Markov models of molecular thermodynamics and kinetics</a></td>
          <td>
            
              Hao Wu,
            
              Fabian Paul,
            
              C. Wehmeyer,
            
              Frank Noé,
            
          </td>
          <td>2016-05-25</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>64</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/82ee174bc312d3960b865b0946ce9c86f1a10154" target='_blank'>Phanto-IDP: compact model for precise intrinsically disordered protein backbone generation and enhanced sampling</a></td>
          <td>
            
              Jun-Jie Zhu,
            
              Zhengxin Li,
            
              Haowei Tong,
            
              Zhouyu Lu,
            
              Ningjie Zhang,
            
              Ting Wei,
            
              Hai-Feng Chen,
            
          </td>
          <td>2023-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>1</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b44d831a3e6e394d6db92139f4f2fba5bbe5b6f3" target='_blank'>GLOW: A Workflow Integrating Gaussian-Accelerated Molecular Dynamics and Deep Learning for Free Energy Profiling.</a></td>
          <td>
            
              H. Do,
            
              Jinan Wang,
            
              A. Bhattarai,
            
              Yinglong Miao,
            
          </td>
          <td>2022-02-24</td>
          <td>Journal of chemical theory and computation</td>
          <td>23</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/31d62c95e128caa934ca6a6c5cd5c766c6a9d541" target='_blank'>Molecular dynamics without molecules: searching the conformational space of proteins with generative neural networks</a></td>
          <td>
            
              Gregory Schwing,
            
              L. Palese,
            
              Ariel Fern'andez,
            
              L. Schwiebert,
            
              D. Gatti,
            
          </td>
          <td>2022-06-09</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ea8ae9fda8336d6f288ee21ce947ed567c6dee3" target='_blank'>RPnet: A Reverse Projection Based Neural Network for Coarse-graining Metastable Conformational States for Protein Dynamics</a></td>
          <td>
            
              Hanlin Gu,
            
              Wei Wang,
            
              Siqin Cao,
            
              I. C. Unarta,
            
              Y. Yao,
            
              F. Sheong,
            
              Xuhui Huang,
            
          </td>
          <td>2021-08-06</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/15d9e0ffe0d154c41dd1902b028393745d33c781" target='_blank'>Calculating Protein-Ligand Residence Times Through State Predictive Information Bottleneck based Enhanced Sampling</a></td>
          <td>
            
              Suemin Lee,
            
              Dedi Wang,
            
              Markus A. Seeliger,
            
              P. Tiwary,
            
          </td>
          <td>2024-04-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c89a40750265888d7f9c1c557675e1656108cf7e" target='_blank'>Machine learning heralding a new development phase in molecular dynamics simulations</a></td>
          <td>
            
              Eva Prasnikar,
            
              M. Ljubic,
            
              Andrej Perdih,
            
              J. Borišek,
            
          </td>
          <td>2024-03-29</td>
          <td>Artif. Intell. Rev.</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe16255bbbfbd74b63b4bbf72899069830ff27d3" target='_blank'>An Interpretable Convolutional Neural Network Framework for Analyzing Molecular Dynamics Trajectories: a Case Study on Functional States for G-Protein-Coupled Receptors</a></td>
          <td>
            
              Chuan Li,
            
              Jiangting Liu,
            
              Jianfang Chen,
            
              Yuan Yuan,
            
              Jin Yu,
            
              Qiaolin Gou,
            
              Yanzhi Guo,
            
              X. Pu,
            
          </td>
          <td>2022-03-08</td>
          <td>Journal of chemical information and modeling</td>
          <td>9</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/464a4804060a4692e2d7a72be0042a2ea621e3b8" target='_blank'>LAST: Latent Space-Assisted Adaptive Sampling for Protein Trajectories</a></td>
          <td>
            
              Hao Tian,
            
              Xi Jiang,
            
              Sian Xiao,
            
              Hunter La Force,
            
              Eric C. Larson,
            
              Peng Tao,
            
          </td>
          <td>2022-04-27</td>
          <td>Journal of chemical information and modeling</td>
          <td>5</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/56ef38a3e74aabfa0bbd161896a581f0c7e6d218" target='_blank'>Learning dynamic Boltzmann distributions as reduced models of spatial chemical kinetics.</a></td>
          <td>
            
              Oliver K. Ernst,
            
              T. Bartol,
            
              T. Sejnowski,
            
              E. Mjolsness,
            
          </td>
          <td>2018-03-02</td>
          <td>The Journal of chemical physics</td>
          <td>9</td>
          <td>155</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fc0aa1d384633d4b47abf854383faac21cf72a92" target='_blank'>Approximating solutions of the Chemical Master equation using neural networks</a></td>
          <td>
            
              A. Sukys,
            
              Kaan Öcal,
            
              R. Grima,
            
          </td>
          <td>2022-04-27</td>
          <td>iScience</td>
          <td>19</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/02a0504b526587d96e40d2b7d8d384385762ff2d" target='_blank'>PyEMMA 2: A Software Package for Estimation, Validation, and Analysis of Markov Models.</a></td>
          <td>
            
              Martin K. Scherer,
            
              Benjamin Trendelkamp-Schroer,
            
              Fabian Paul,
            
              G. Pérez-Hernández,
            
              Moritz Hoffmann,
            
              Nuria Plattner,
            
              C. Wehmeyer,
            
              Jan-Hendrik Prinz,
            
              F. Noé,
            
          </td>
          <td>2015-10-14</td>
          <td>Journal of chemical theory and computation</td>
          <td>811</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a2fcf672d2ac57fb45318b3351c3051df93895f5" target='_blank'>P RO GAE: A G EOMETRIC G ENERATIVE M ODEL FOR D ISENTANGLING P ROTEIN C ONFORMATIONAL S PACE</a></td>
          <td>
            
              N. Joseph Tatro,
            
              Payel Das,
            
              Pin-Yu Chen,
            
              V. Chenthamarakshan,
            
              Rongjie Lai,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9896835f050121358372254a9bce57ca117672c8" target='_blank'>Boltzmann Generators - Sampling Equilibrium States of Many-Body Systems with Deep Learning</a></td>
          <td>
            
              F. Noé,
            
              Hao Wu,
            
          </td>
          <td>2018-12-04</td>
          <td>ArXiv</td>
          <td>19</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1af26b2f66408bda852f8abfb9912b21ed075a19" target='_blank'>Introduction to Markov state modeling with the PyEMMA software [Article v1.0]</a></td>
          <td>
            
              C. Wehmeyer,
            
              Martin K. Scherer,
            
              Tim Hempel,
            
              B. Husic,
            
              S. Olsson,
            
              F. Noé,
            
          </td>
          <td>2018-11-01</td>
          <td>Living Journal of Computational Molecular Science</td>
          <td>44</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cdae73da6322cbb9f4a0945aa4e68055184d466d" target='_blank'>Neural Continuous-Time Markov Models</a></td>
          <td>
            
              Majerle Reeves,
            
              H. Bhat,
            
          </td>
          <td>2022-12-11</td>
          <td>2023 SICE International Symposium on Control Systems (SICE ISCS)</td>
          <td>0</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/12ba80c62d718cfee6f5dffa356178cbd408e397" target='_blank'>Predictive Collective Variable Discovery with Deep Bayesian Models</a></td>
          <td>
            
              M. Schöberl,
            
              N. Zabaras,
            
              P. Koutsourelakis,
            
          </td>
          <td>2018-09-18</td>
          <td>The Journal of chemical physics</td>
          <td>32</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f3ebef5245ffe1c8e4ccfa334c86cf2f0d9f4ed5" target='_blank'>An Introduction to Markov State Models and Their Application to Long Timescale Molecular Simulation</a></td>
          <td>
            
              G. Bowman,
            
              V. Pande,
            
              F. Noé,
            
          </td>
          <td>None</td>
          <td></td>
          <td>483</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4dcae326c8ec623662de72de38d44d8bc79f30a4" target='_blank'>Variational Autoencoding Molecular Graphs with Denoising Diffusion Probabilistic Model</a></td>
          <td>
            
              Daiki Koge,
            
              N. Ono,
            
              Shigehiko Kanaya,
            
          </td>
          <td>2023-07-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/df7bc9ea5e0b1af5c7a675e6dfd9a6983f6d9d2e" target='_blank'>Supervised Learning on Markov States as a Method for Gaining Insight into Protein Allostery</a></td>
          <td>
            
              Mohammad M. Sultan,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/41cf570d66b907eb5b2446fceed6c9c97dc52f25" target='_blank'>Understanding Protein Dynamics with L1-Regularized Reversible Hidden Markov Models</a></td>
          <td>
            
              R. McGibbon,
            
              Bharath Ramsundar,
            
              Mohammad M. Sultan,
            
              G. Kiss,
            
              V. Pande,
            
          </td>
          <td>2014-05-06</td>
          <td>None</td>
          <td>19</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/28bc6d94e78ffc3cf338294b4698502160d644b8" target='_blank'>Differences in ligand-induced protein dynamics extracted from an unsupervised deep learning approach correlate with protein–ligand binding affinities</a></td>
          <td>
            
              Ikki Yasuda,
            
              Katsuhiro Endo,
            
              Eiji Yamamoto,
            
              Y. Hirano,
            
              K. Yasuoka,
            
          </td>
          <td>2021-09-03</td>
          <td>Communications Biology</td>
          <td>10</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69ccf867833b3564c2d2f26c49915b444eadb268" target='_blank'>Transferable Neural Networks for Enhanced Sampling of Protein Dynamics.</a></td>
          <td>
            
              Mohammad M. Sultan,
            
              H. Wayment-Steele,
            
              V. Pande,
            
          </td>
          <td>2018-01-02</td>
          <td>Journal of chemical theory and computation</td>
          <td>91</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3d50464ec004730c9bb79f461b1eccf124d42e6d" target='_blank'>Toward Achieving Efficient and Accurate Ligand-Protein Unbinding with Deep Learning and Molecular Dynamics through RAVE.</a></td>
          <td>
            
              João Marcelo Lamim Ribeiro,
            
              P. Tiwary,
            
          </td>
          <td>2018-12-07</td>
          <td>Journal of chemical theory and computation</td>
          <td>56</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/446681b1a3b3cafc7522cffdc3550348480d4085" target='_blank'>Operator Autoencoders: Learning Physical Operations on Encoded Molecular Graphs</a></td>
          <td>
            
              Willis Hoke,
            
              D. Shea,
            
              S. Casey,
            
          </td>
          <td>2021-05-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a81be5b6594d95431aab852bfa568ef6123d667e" target='_blank'>FlexiDock: Compositional diffusion models for flexible molecular docking</a></td>
          <td>
            
              Zichen Wang,
            
              Balasubramaniam Srinivasan,
            
              Zhengyuan Shen,
            
              G. Karypis,
            
              H. Rangwala,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>95</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1f9c63e957adb671e52a8a4368dbab25faefc428" target='_blank'>Variational embedding of protein folding simulations using gaussian mixture variational autoencoders</a></td>
          <td>
            
              Mahdi Ghorbani,
            
              Samarjeet Prasad,
            
              Jeffery B. Klauda,
            
              B. Brooks,
            
          </td>
          <td>2021-08-27</td>
          <td>The Journal of chemical physics</td>
          <td>11</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0022cd247a3f8e22faa0eb00f9068d4f792168a8" target='_blank'>LINES: Log-Probability Estimation via Invertible Neural Networks for Enhanced Sampling.</a></td>
          <td>
            
              Ryan E Odstrcil,
            
              P. Dutta,
            
              Jin Liu,
            
          </td>
          <td>2022-09-13</td>
          <td>Journal of chemical theory and computation</td>
          <td>5</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/67226aecde573785d39d59f100c0c183d2ed5292" target='_blank'>Analysing protein dynamics using machine learning based generative models</a></td>
          <td>
            
              Alexandra-Ioana Albu,
            
              G. Czibula,
            
          </td>
          <td>2020-05-01</td>
          <td>2020 IEEE 14th International Symposium on Applied Computational Intelligence and Informatics (SACI)</td>
          <td>2</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b76b33666a3e4bb048078454fd8d1d3ac0f27305" target='_blank'>Enhanced Sampling of Nucleic Acids’ Structures Using Deep-Learning-Derived Biasing Forces</a></td>
          <td>
            
              E. Salawu,
            
          </td>
          <td>2020-12-01</td>
          <td>2020 IEEE Symposium Series on Computational Intelligence (SSCI)</td>
          <td>1</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6b22986a544efc35506ea736d884c19d4ba7108c" target='_blank'>Estimation of binding rates and affinities from multiensemble Markov models and ligand decoupling.</a></td>
          <td>
            
              Yunhui Ge,
            
              Vincent A. Voelz,
            
          </td>
          <td>2021-06-04</td>
          <td>The Journal of chemical physics</td>
          <td>2</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7cb50b79a1eec42efd2fec611b9cdc5d08ca936c" target='_blank'>Machine learning transforms how microstates are sampled</a></td>
          <td>
            
              M. Tuckerman,
            
          </td>
          <td>2019-09-05</td>
          <td>Science</td>
          <td>10</td>
          <td>70</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/90e7dfe131b21c12c2998ec0b87010ac6eb4d337" target='_blank'>Two for One: Diffusion Models and Force Fields for Coarse-Grained Molecular Dynamics</a></td>
          <td>
            
              Marloes Arts,
            
              Victor Garcia Satorras,
            
              Chin-Wei Huang,
            
              Daniel Zuegner,
            
              M. Federici,
            
              C. Clementi,
            
              Frank No'e,
            
              Robert Pinsler,
            
              Rianne van den Berg,
            
          </td>
          <td>2023-02-01</td>
          <td>Journal of chemical theory and computation</td>
          <td>32</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/88a277bbc18ad0eed29ea02bef0219b1c1a111ab" target='_blank'>Variational encoding of complex dynamics.</a></td>
          <td>
            
              Carlos X. Hernández,
            
              H. Wayment-Steele,
            
              Mohammad M. Sultan,
            
              B. Husic,
            
              V. Pande,
            
          </td>
          <td>2017-11-23</td>
          <td>Physical review. E</td>
          <td>134</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c86006be8cd1c5ee6193ee31cf31d44449b9c082" target='_blank'>Molecular enhanced sampling with autoencoders: On‐the‐fly collective variable discovery and accelerated free energy landscape exploration</a></td>
          <td>
            
              Wei Chen,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2017-12-30</td>
          <td>Journal of Computational Chemistry</td>
          <td>165</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/827ca2f16ded73b2df1f760eebb2f1c5c9e5b44b" target='_blank'>Timewarp: Transferable Acceleration of Molecular Dynamics by Learning Time-Coarsened Dynamics</a></td>
          <td>
            
              Leon Klein,
            
              Andrew Y. K. Foong,
            
              T. E. Fjelde,
            
              Bruno Mlodozeniec,
            
              Marc Brockschmidt,
            
              Sebastian Nowozin,
            
              Frank No'e,
            
              Ryota Tomioka,
            
          </td>
          <td>2023-02-02</td>
          <td>ArXiv</td>
          <td>15</td>
          <td>50</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1284a3eac1be38226f47e8ea53f25bb904b50c45" target='_blank'>FusionDock: Physics-informed Diffusion Model for Molecular Docking</a></td>
          <td>
            
              Matthew Masters,
            
              Amr H. Mahmoud,
            
              Markus A. Lill,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0cd3637bf739405263a78687d2f1ddaaa801cd89" target='_blank'>Latent Space Simulator for Unveiling Molecular Free Energy Landscapes and Predicting Transition Dynamics</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/22bd75994b94a00cf2a9697a28a805e35da30b1f" target='_blank'>Reweighted autoencoded variational Bayes for enhanced sampling (RAVE).</a></td>
          <td>
            
              João Marcelo Lamim Ribeiro,
            
              P. Bravo,
            
              Yihang Wang,
            
              P. Tiwary,
            
          </td>
          <td>2018-02-09</td>
          <td>The Journal of chemical physics</td>
          <td>239</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/23318a709145ebf46b23c348293580fadd339f75" target='_blank'>Artificial Intelligence Assists Discovery of Reaction Coordinates and Mechanisms from Molecular Dynamics Simulations</a></td>
          <td>
            
              Hendrik Jung,
            
              Roberto Covino,
            
              G. Hummer,
            
          </td>
          <td>2019-01-14</td>
          <td>arXiv: Chemical Physics</td>
          <td>39</td>
          <td>95</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69e96a63ffd74a6c816203329c8a54794ac1ca00" target='_blank'>Transferable Kinetic Monte Carlo Models with Thousands of Reactions Learned from Molecular Dynamics Simulations.</a></td>
          <td>
            
              Enze Chen,
            
              Qian Yang,
            
              Vincent Dufour-Décieux,
            
              C. Sing-Long,
            
              Rodrigo Freitas,
            
              E. Reed,
            
          </td>
          <td>2019-02-08</td>
          <td>The journal of physical chemistry. A</td>
          <td>8</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/07b4a3910273a7122a5dedbced31ea5b877eb5b2" target='_blank'>Enhancing Conformational Sampling for Intrinsically Disordered and Ordered Proteins by Variational Autoencoder</a></td>
          <td>
            
              Jun-Jie Zhu,
            
              Ningjie Zhang,
            
              Ting Wei,
            
              Haifeng Chen,
            
          </td>
          <td>2023-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>6</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0129beb25ff82f128116b35cdb45aff8f1456255" target='_blank'>Top-Down Machine Learning of Coarse-Grained Protein Force Fields</a></td>
          <td>
            
              Carles Navarro,
            
              Maciej Majewski,
            
              G. D. Fabritiis,
            
          </td>
          <td>2023-06-20</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>3</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/728649fcdf03de6cbb023c2ecb6cd132019cbadf" target='_blank'>Long-time methods for molecular dynamics simulations: Markov State Models and Milestoning.</a></td>
          <td>
            
              Brajesh Narayan,
            
              Ye Yuan,
            
              A. Fathizadeh,
            
              R. Elber,
            
              Nicolae-Viorel Buchete,
            
          </td>
          <td>None</td>
          <td>Progress in molecular biology and translational science</td>
          <td>5</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8bd7b6f622911c5f68e7dd19fc002074bde9722e" target='_blank'>Representation of Protein Dynamics Disentangled by Time-structure-based Prior</a></td>
          <td>
            
              Tsuyoshi Ishizone,
            
              Yasuhiro Matsunaga,
            
              Sotaro Fuchigami,
            
              Kazuyuki Nakamura,
            
          </td>
          <td>2023-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92da30c4137270487bc51fec1aaee97b17f4e2ac" target='_blank'>Machine Learning Generation of Dynamic Protein Conformational Ensembles</a></td>
          <td>
            
              Li-E Zheng,
            
              Shrishti Barethiya,
            
              Erik B. Nordquist,
            
              Jianhan Chen,
            
          </td>
          <td>2023-05-01</td>
          <td>Molecules</td>
          <td>3</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/65aaf814b43994af4bb920982a2c496a011cb4a0" target='_blank'>MSMBuilder2: Modeling Conformational Dynamics at the Picosecond to Millisecond Scale.</a></td>
          <td>
            
              Kyle A. Beauchamp,
            
              G. Bowman,
            
              T. Lane,
            
              L. Maibaum,
            
              I. Haque,
            
              V. Pande,
            
          </td>
          <td>2011-10-11</td>
          <td>Journal of chemical theory and computation</td>
          <td>358</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/868ee34bc7bda4ff9652fd9e739e093d6ad163c9" target='_blank'>Dynamic Molecular Graph-based Implementation for Biophysical Properties Prediction</a></td>
          <td>
            
              C. Knutson,
            
              G. Panapitiya,
            
              R. Varikoti,
            
              Neeraj Kumar,
            
          </td>
          <td>2022-12-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4dce31663630b64546c7168ae7076a40ca41ca62" target='_blank'>Accelerating Inference in Molecular Diffusion Models with Latent Representations of Protein Structure</a></td>
          <td>
            
              Ian Dunn,
            
              D. Koes,
            
          </td>
          <td>2023-11-22</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/76317336bf971d2006683e6dba463cd85154efff" target='_blank'>Learning Kinetic Distance Metrics for Markov State Models of Protein Conformational Dynamics.</a></td>
          <td>
            
              R. McGibbon,
            
              V. Pande,
            
          </td>
          <td>2013-06-06</td>
          <td>Journal of chemical theory and computation</td>
          <td>41</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fd11b7e8bbc5498f79c6478529c21604630abac4" target='_blank'>EMMA 1.3 Markov Model Algorithms Tutorial and Documentation</a></td>
          <td>
            
              Martin Senne,
            
              Benjamin Trendelkamp-Schroer,
            
              A. Mey,
            
              Frank Noé,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0187ec0ad130049a72e9cfc50c6160437c3fb2aa" target='_blank'>Understanding recent deep‐learning techniques for identifying collective variables of molecular dynamics</a></td>
          <td>
            
              W. Zhang,
            
              Christof Schutte,
            
          </td>
          <td>2023-07-01</td>
          <td>PAMM</td>
          <td>1</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fadebc6c400cb504812b31e795fb8d33cbe05b06" target='_blank'>An Integrated Markov State Model and Path Metadynamics Approach to Characterize Drug Binding Processes.</a></td>
          <td>
            
              Mattia Bernetti,
            
              Matteo Masetti,
            
              M. Recanatini,
            
              Rommie E. Amaro,
            
              A. Cavalli,
            
          </td>
          <td>2019-09-05</td>
          <td>Journal of chemical theory and computation</td>
          <td>42</td>
          <td>73</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/755dc5e951f01bb6fd648fcebe84803584d54fb2" target='_blank'>Detection of molecular behavior that characterizes systems using a deep learning approach.</a></td>
          <td>
            
              Katsuhiro Endo,
            
              Daisuke Yuhara,
            
              Katsufumi Tomobe,
            
              K. Yasuoka,
            
          </td>
          <td>2019-05-23</td>
          <td>Nanoscale</td>
          <td>4</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/17c80d75ea490a12f54d166717e9d745f7129f8d" target='_blank'>Spectral Map: Embedding Slow Kinetics in Collective Variables</a></td>
          <td>
            
              J. Rydzewski,
            
          </td>
          <td>2023-06-01</td>
          <td>The Journal of Physical Chemistry Letters</td>
          <td>5</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5303dffbc43db94c5acc804b87b76d84eb81dd13" target='_blank'>Reinforcement Learning for Multi-Scale Molecular Modeling</a></td>
          <td>
            
              Jun Zhang,
            
              Yao-Kun Lei,
            
              Yi Isaac Yang,
            
              Y. Gao,
            
          </td>
          <td>2019-08-19</td>
          <td>None</td>
          <td>1</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6b506cab4e4bd23ef4d751e97592457a4dbb370c" target='_blank'>Recent advances in protein conformation sampling by combining machine learning with molecular simulation</a></td>
          <td>
            
              Yiming Tang,
            
              Zhongyuan Yang,
            
              Yifei Yao,
            
              Yun Zhou,
            
              Yuan Tan,
            
              Zichao Wang,
            
              Tong Pan,
            
              Rui Xiong,
            
              Junli Sun,
            
              Guanghong Wei,
            
          </td>
          <td>2024-01-04</td>
          <td>Chinese Physics B</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8d96e0b4a9e603b6c03142269f606fe11a0a45d1" target='_blank'>Understanding chemical reactions via variational autoencoder and atomic representations</a></td>
          <td>
            
              Martin vS'ipka,
            
              A. Erlebach,
            
              L. Grajciar,
            
          </td>
          <td>2022-03-15</td>
          <td>None</td>
          <td>1</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/98243acffb45975b0fe73763b41316c520a7037a" target='_blank'>Progress in deep Markov state modeling: Coarse graining and experimental data restraints.</a></td>
          <td>
            
              Andreas Mardt,
            
              Frank Noé,
            
          </td>
          <td>2021-08-04</td>
          <td>The Journal of chemical physics</td>
          <td>10</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a86717a0af0e90830183b221a9473d235da96698" target='_blank'>Data-driven learning of the generalized Langevin equation with state-dependent memory</a></td>
          <td>
            
              Pei Ge,
            
              Zhongqiang Zhang,
            
              Huan Lei,
            
          </td>
          <td>2023-10-28</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/39fd60798d23d5514548f92fa881f520905f0bd0" target='_blank'>Learning Compositional Representations of Interacting Systems with Restricted Boltzmann Machines: Comparative Study of Lattice Proteins</a></td>
          <td>
            
              J. Tubiana,
            
              S. Cocco,
            
              R. Monasson,
            
          </td>
          <td>2019-02-18</td>
          <td>Neural Computation</td>
          <td>24</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6582d19655b187195965e9ba7c4d93b193b649d2" target='_blank'>ALGORITHMS FOR MOLECULAR KINETICS</a></td>
          <td>
            
              F. Qin,
            
          </td>
          <td>2011-04-18</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7410966762efc567ca6410e0b826201f0c330f70" target='_blank'>Efficient Parameter Estimation for DNA Kinetics Modeled as Continuous-Time Markov Chains</a></td>
          <td>
            
              Sedigheh Zolaktaf,
            
              Frits Dannenberg,
            
              E. Winfree,
            
              A. Bouchard-Côté,
            
              Mark W. Schmidt,
            
              A. Condon,
            
          </td>
          <td>2019-08-05</td>
          <td>None</td>
          <td>5</td>
          <td>58</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca246ca0069caf6a95d3f8dbf6a4409fc380e2ea" target='_blank'>An Accurate Computation of a Physical Quantity of a Protein with a Markov State Model Constructed using a Manifold-Learning Technique</a></td>
          <td>
            
              Reika Ito,
            
              Takashi Yoshidome,
            
          </td>
          <td>2018-02-01</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9ee4f8beb4155b889e8f4f2aaa3253f0d379760f" target='_blank'>Machine Learning-Assisted Discovery of Hidden States in Expanded Free Energy Space.</a></td>
          <td>
            
              R. Ketkaew,
            
              Fabrizio Creazzo,
            
              S. Luber,
            
          </td>
          <td>2022-02-16</td>
          <td>The journal of physical chemistry letters</td>
          <td>6</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cc57d60098fa1a5deb4d73a0b9b4d29e2bbf6f36" target='_blank'>StochNetV2: A Tool for Automated Deep Abstractions for Stochastic Reaction Networks</a></td>
          <td>
            
              D. Repin,
            
              Nhat-Huy Phung,
            
              Tatjana Petrov,
            
          </td>
          <td>2020-08-31</td>
          <td>None</td>
          <td>1</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e70ed4cec2382a01413e6353cc8406cc0faa5993" target='_blank'>Markov State Models: To Optimize or Not to Optimize</a></td>
          <td>
            
              Robert E Arbon,
            
              Yanchen Zhu,
            
              A. Mey,
            
          </td>
          <td>2024-01-01</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0377d2e18facfaa8df108a6980a81fed5dde750b" target='_blank'>Top-down machine learning approach for high-throughput single-molecule analysis</a></td>
          <td>
            
              D. S. White,
            
              Marcel P. Goldschen-Ohm,
            
              R. Goldsmith,
            
              B. Chanda,
            
          </td>
          <td>2020-04-08</td>
          <td>eLife</td>
          <td>23</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0a22a48825230b81221c5bfc0abb04594c7b7297" target='_blank'>Accelerating All-Atom Simulations and Gaining Mechanistic Understanding of Biophysical Systems through State Predictive Information Bottleneck.</a></td>
          <td>
            
              S. Mehdi,
            
              Dedi Wang,
            
              S. Pant,
            
              P. Tiwary,
            
          </td>
          <td>2021-12-21</td>
          <td>Journal of chemical theory and computation</td>
          <td>30</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/323461e6f2d2f1e7db71d77cf1a37488a6153078" target='_blank'>MLMOD: Machine Learning Methods for Data-Driven Modeling in LAMMPS</a></td>
          <td>
            
              P. Atzberger,
            
          </td>
          <td>2021-07-29</td>
          <td>J. Open Source Softw.</td>
          <td>1</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/26a186b235fe13fec788cb5a3129a38e55e0be1e" target='_blank'>Generative modeling of short, disordered proteins with homogeneous sequence composition</a></td>
          <td>
            
              Ishan Taneja,
            
              Keren Lasker,
            
          </td>
          <td>2023-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8e7a1024ac46c43d3728f9d731c48d0311957a2e" target='_blank'>Generating Physically-Realistic Tertiary Protein Structures with Deep Latent Variable Models Learning Over Experimentally-available Structures</a></td>
          <td>
            
              F. Alam,
            
              Amarda Shehu,
            
          </td>
          <td>2021-12-09</td>
          <td>2021 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>2</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6263c9498dc41e9f466478bda26996b3f3bbb70e" target='_blank'>Kinetic Machine Learning Unravels Ligand-Directed Conformational Change of μ Opioid Receptor</a></td>
          <td>
            
              Evan N. Feinberg,
            
              A. Farimani,
            
              Carlos X. Hernández,
            
              V. Pande,
            
          </td>
          <td>2017-07-31</td>
          <td>bioRxiv</td>
          <td>10</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6bec80cf9dcdd2719ff738a3ea91d14837882256" target='_blank'>Enhanced modeling via network theory: Adaptive sampling of Markov state models.</a></td>
          <td>
            
              G. Bowman,
            
              D. Ensign,
            
              V. Pande,
            
          </td>
          <td>2010-02-17</td>
          <td>Journal of chemical theory and computation</td>
          <td>206</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe220dded3b8540c19fdebe7dc95332e3db4f81e" target='_blank'>The bigger picture</a></td>
          <td>
            
              John Urbanic,
            
              Chris Olah,
            
          </td>
          <td>2021-11-15</td>
          <td>Nature</td>
          <td>13</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e88cac1fabc8c8633db4b8601ec3c4fbf56dd5d7" target='_blank'>Neural relational inference to learn allosteric long-range interactions in proteins from molecular dynamics simulations</a></td>
          <td>
            
              Jingxuan Zhu,
            
              Juexin Wang,
            
              Weiwei Han,
            
              Dong Xu,
            
          </td>
          <td>2021-01-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>67</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5ab9f247f0278df388377e3cd5f5311297e73235" target='_blank'>PepFlow: direct conformational sampling from peptide energy landscapes through hypernetwork-conditioned diffusion</a></td>
          <td>
            
              Osama Abdin,
            
              Philip M. Kim,
            
          </td>
          <td>2023-06-26</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e99604e2da48483b633247c13dd4ad5f46196562" target='_blank'>DiffDock: Diffusion Steps, Twists, and Turns for Molecular Docking</a></td>
          <td>
            
              Gabriele Corso,
            
              Hannes Stärk,
            
              Bowen Jing,
            
              R. Barzilay,
            
              T. Jaakkola,
            
          </td>
          <td>2022-10-04</td>
          <td>ArXiv</td>
          <td>218</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/390c937c44ffb10a9552d275fe8bf6d869b8a1cd" target='_blank'>Enhanced Sampling with Machine Learning: A Review</a></td>
          <td>
            
              S. Mehdi,
            
              Zachary Smith,
            
              Lukas Herron,
            
              Ziyue Zou,
            
              P. Tiwary,
            
          </td>
          <td>2023-06-15</td>
          <td>Annual review of physical chemistry</td>
          <td>7</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5f57848726a0ecc12bd0420e23a745109c404b84" target='_blank'>Physics-Inspired Protein Encoder Pre-Training via Siamese Sequence-Structure Diffusion Trajectory Prediction</a></td>
          <td>
            
              Zuobai Zhang,
            
              Minghao Xu,
            
              A. Lozano,
            
              V. Chenthamarakshan,
            
              Payel Das,
            
              Jian Tang,
            
          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>5</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a33d2c5b230cf154c8fe796e381112d8a80ee7b5" target='_blank'>A combination of machine learning and infrequent metadynamics to efficiently predict kinetic rates, transition states, and molecular determinants of drug dissociation from G protein-coupled receptors</a></td>
          <td>
            
              João Marcelo Lamim Ribeiro,
            
              D. Provasi,
            
              M. Filizola,
            
          </td>
          <td>2020-09-23</td>
          <td>The Journal of Chemical Physics</td>
          <td>30</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92e8942259e1b34172f8dbe5818575b43e89977e" target='_blank'>Concurrent parametrization against static and kinetic information leads to more robust coarse-grained force fields</a></td>
          <td>
            
              J. F. Rudzinski,
            
              T. Bereau,
            
          </td>
          <td>2016-07-18</td>
          <td>The European Physical Journal Special Topics</td>
          <td>17</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92e8942259e1b34172f8dbe5818575b43e89977e" target='_blank'>Concurrent parametrization against static and kinetic information leads to more robust coarse-grained force fields</a></td>
          <td>
            
              J. F. Rudzinski,
            
              T. Bereau,
            
          </td>
          <td>2016-07-18</td>
          <td>The European Physical Journal Special Topics</td>
          <td>17</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9cbfa7ef13b37a3f14747ea914fbf6c1935fd48d" target='_blank'>Automatic state partitioning for multibody systems (APM): an efficient algorithm for constructing Markov state models to elucidate conformational dynamics of multibody systems.</a></td>
          <td>
            
              F. Sheong,
            
              Daniel‐Adriano Silva,
            
              Luming Meng,
            
              Yutong Zhao,
            
              Xuhui Huang,
            
          </td>
          <td>2015-01-13</td>
          <td>Journal of chemical theory and computation</td>
          <td>40</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7937fc2e974c74d7bed6b0b05ab4aacfeef7ca4b" target='_blank'>An extended autoencoder model for reaction coordinate discovery in rare event molecular dynamics datasets.</a></td>
          <td>
            
              M. Frassek,
            
              A. Arjun,
            
              P. Bolhuis,
            
          </td>
          <td>2021-08-14</td>
          <td>The Journal of chemical physics</td>
          <td>24</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/87dab6c203b173dc821f657190e91296a3dc8bea" target='_blank'>Data-driven prediction of αIIbβ3 integrin activation pathways using nonlinear manifold learning and deep generative modeling.</a></td>
          <td>
            
              Siva Dasetty,
            
              T. Bidone,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2023-12-01</td>
          <td>Biophysical journal</td>
          <td>1</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/de2d3e12be8f97504b6f4ee194e08df25a8d8ea9" target='_blank'>Computational methods for exploring protein conformations</a></td>
          <td>
            
              Jane R. Allison,
            
          </td>
          <td>2020-08-05</td>
          <td>Biochemical Society Transactions</td>
          <td>32</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/314193c238cce350aa14c76eed89eccf4baf3744" target='_blank'>Deep Learning: Current and Emerging Applications in Medicine and Technology</a></td>
          <td>
            
              A. Akay,
            
              H. Hess,
            
          </td>
          <td>2019-01-23</td>
          <td>IEEE Journal of Biomedical and Health Informatics</td>
          <td>58</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bd8c74f9c67ec07605eac970928782d5cb47b8b3" target='_blank'>Learning Efficient, Collective Monte Carlo Moves with Variational Autoencoders.</a></td>
          <td>
            
              Jacob I Monroe,
            
              V. K. Shen,
            
          </td>
          <td>2022-05-25</td>
          <td>Journal of chemical theory and computation</td>
          <td>9</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/68b03ad0ed6c1c9ac968fa4c5ae66c3020dc783e" target='_blank'>Multiscale simulations of complex systems by learning their effective dynamics</a></td>
          <td>
            
              Pantelis R. Vlachas,
            
              G. Arampatzis,
            
              Caroline Uhler,
            
              P. Koumoutsakos,
            
          </td>
          <td>2020-06-24</td>
          <td>Nature Machine Intelligence</td>
          <td>88</td>
          <td>77</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a4be57f8248d031dae67def54a0f952e98105a41" target='_blank'>Machine learning dynamic correlation in chemical kinetics.</a></td>
          <td>
            
              C. Kim,
            
              Nathan D. Ricke,
            
              T. Van Voorhis,
            
          </td>
          <td>2021-08-05</td>
          <td>The Journal of chemical physics</td>
          <td>3</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aa0b401f21ee75696bb7260423fc72a9a8fb716b" target='_blank'>Deep Learning Predicts Protein-Ligand Interactions</a></td>
          <td>
            
              J. Balma,
            
              A. Vose,
            
              Y. Peterson,
            
              A. Chittiboyina,
            
              P. Pandey,
            
              C. R. Yates,
            
              I. Khan,
            
              S. Sukumar,
            
          </td>
          <td>2020-12-10</td>
          <td>2020 IEEE International Conference on Big Data (Big Data)</td>
          <td>0</td>
          <td>60</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca17f2865ee2459f41c221164fb193502948693b" target='_blank'>An overview and practical guide to building Markov state models.</a></td>
          <td>
            
              G. Bowman,
            
          </td>
          <td>None</td>
          <td>Advances in experimental medicine and biology</td>
          <td>21</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d6458b29132c5fa32037e6ad19d93c855474e399" target='_blank'>Methodological Advances for Drug Discovery and ProteinEngineering</a></td>
          <td>
            
              Xinqiang Ding,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4ee2669028bd6280d640b567f5f33dbf6e621b37" target='_blank'>An Integrated Machine Learning Approach Delineates Entropic Modulation of α -synuclein by Small Molecule</a></td>
          <td>
            
              Sneha Menon,
            
              ⊥. SubinoyAdhikari,
            
              Jagannath Mondal,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/268fb2625b2497d81079456b51f9f1a413d753fa" target='_blank'>Making high-dimensional molecular distribution functions tractable through Belief Propagation on Factor Graphs</a></td>
          <td>
            
              Zachary Smith,
            
              P. Tiwary,
            
          </td>
          <td>2021-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2cfc26b4af99f195b433fa7aa00f221b111c7cd4" target='_blank'>Illuminating protein space with a programmable generative model</a></td>
          <td>
            
              John Ingraham,
            
              Max Baranov,
            
              Zak Costello,
            
              Vincent Frappier,
            
              Ahmed Ismail,
            
              Shan Tie,
            
              Wujie Wang,
            
              Vincent Xue,
            
              F. Obermeyer,
            
              Andrew L Beam,
            
              G. Grigoryan,
            
          </td>
          <td>2022-12-02</td>
          <td>Nature</td>
          <td>131</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3ae9bf6b23d92fb1164271f06fa6365d120dae9f" target='_blank'>Machine learning in solid-state physics and statistical physics</a></td>
          <td>
            
              Lovro Vrček,
            
          </td>
          <td>2018-07-05</td>
          <td></td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/33c04427b6fcd265da08f80fe8f279486f11d44a" target='_blank'>Machine learning of microscopic structure-dynamics relationships in complex molecular systems</a></td>
          <td>
            
              M. Crippa,
            
              Annalisa Cardellini,
            
              M. Cioni,
            
              Gábor Csányi,
            
              G. Pavan,
            
          </td>
          <td>2023-08-31</td>
          <td>Machine Learning: Science and Technology</td>
          <td>1</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4afa2c93c386bc87c4fa4339709e2b788a6ed32a" target='_blank'>Everything you wanted to know about Markov State Models but were afraid to ask.</a></td>
          <td>
            
              V. Pande,
            
              Kyle A. Beauchamp,
            
              G. Bowman,
            
          </td>
          <td>2010-09-01</td>
          <td>Methods</td>
          <td>576</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6ecaefd40cc4a81bf1124f039b36862cf02f2d88" target='_blank'>Learning heterogenous reaction rates from stochastic simulations.</a></td>
          <td>
            
              A. Torres-Knoop,
            
              I. Kryven,
            
          </td>
          <td>2018-10-09</td>
          <td>Physical review. E</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d09f8cd520789fbd378a7bd90dcafc2544b6fcfc" target='_blank'>Simple synthetic molecular dynamics for efficient trajectory generation</a></td>
          <td>
            
              John D. Russo,
            
              D. Zuckerman,
            
          </td>
          <td>2022-04-09</td>
          <td>None</td>
          <td>1</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3e2c07792a60338f29a5cfe012da2bdc99a69687" target='_blank'>A Machine Learning Approach to Solving Biochemical Systems</a></td>
          <td>
            
              Joshua Ologbonyo,
            
              Emmanuel O. Adara,
            
              R. Sidje,
            
          </td>
          <td>2024-02-05</td>
          <td>2024 IEEE First International Conference on Artificial Intelligence for Medicine, Health and Care (AIMHC)</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e12ebec869b56d7d0b265027509b9e644958daae" target='_blank'>Optimizing Physics-Informed Neural Network in Dynamic System Simulation and Learning of Parameters</a></td>
          <td>
            
              Ebenezer O. Oluwasakin,
            
              Abdul Q. M. Khaliq,
            
          </td>
          <td>2023-11-28</td>
          <td>Algorithms</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2c58f68b87493a209d9b56574f3c232296fdcbe3" target='_blank'>Combining experimental and simulation data of molecular processes via augmented Markov models</a></td>
          <td>
            
              S. Olsson,
            
              Hao Wu,
            
              Fabian Paul,
            
              C. Clementi,
            
              F. Noé,
            
          </td>
          <td>2017-07-17</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>77</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92081440bf32d43143a1304ec738e720ca4d2901" target='_blank'>D YNAMICS -I NFORMED P ROTEIN D ESIGN WITH S TRUCTURE C ONDITIONING</a></td>
          <td>
            
              Urszula Julia Komorowska,
            
              Simon V Mathis,
            
              Kieran Didi,
            
              Francisco Vargas,
            
              Pietro Lio,
            
              M. Jamnik,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/59de694649b05d03e65b89af1606cd33e5bc9624" target='_blank'>An efficient Bayesian kinetic lumping algorithm to identify metastable conformational states via Gibbs sampling.</a></td>
          <td>
            
              Wei Wang,
            
              Tong Liang,
            
              F. Sheong,
            
              Xiaodan Fan,
            
              Xuhui Huang,
            
          </td>
          <td>2018-08-21</td>
          <td>The Journal of chemical physics</td>
          <td>20</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c895fe350d8750a718e5a1742c1c440389bc1ef2" target='_blank'>Interpreting Molecular Dynamics Forces as Deep Learning Gradients Improves Quality Of Predicted Protein Structures</a></td>
          <td>
            
              J. E. King,
            
              D. Koes,
            
          </td>
          <td>2023-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/402717ce4e4339fd5b1e97c2cd9d41c5273f01d7" target='_blank'>Molecular relaxation by reverse diffusion with time step prediction</a></td>
          <td>
            
              Khaled Kahouli,
            
              Stefaan S. P. Hessmann,
            
              Klaus-Robert Muller,
            
              Shinichi Nakajima,
            
              Stefan Gugler,
            
              Niklas Wolf Andreas Gebauer,
            
          </td>
          <td>2024-04-16</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/345eb3195aac43eb5dcaacc0a7fd288be0fa4491" target='_blank'>DiffDock-PP: Rigid Protein-Protein Docking with Diffusion Models</a></td>
          <td>
            
              Mohamed Amine Ketata,
            
              Cedrik Laue,
            
              R. Mammadov,
            
              Hannes Stärk,
            
              Menghua Wu,
            
              Gabriele Corso,
            
              Ćeline Marquet,
            
              R. Barzilay,
            
              T. Jaakkola,
            
          </td>
          <td>2023-04-08</td>
          <td>ArXiv</td>
          <td>15</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/712080b7b8d296e61827d6c738303ec6ed0d44c1" target='_blank'>Generation of conformational ensembles of small molecules via Surrogate Model-Assisted Molecular Dynamics</a></td>
          <td>
            
              Juan Viguera Diez,
            
              Sara Romeo Atance,
            
              Ola Engkvist,
            
              Simon Olsson,
            
          </td>
          <td>2024-04-05</td>
          <td>Machine Learning: Science and Technology</td>
          <td>1</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/761320b0617a8925c391ae12c556cd463bc159c4" target='_blank'>Trajectory-based training enables protein simulations with accurate folding and Boltzmann ensembles in cpu-hours</a></td>
          <td>
            
              J. Jumper,
            
              N. Faruk,
            
              K. Freed,
            
              T. Sosnick,
            
          </td>
          <td>2018-12-01</td>
          <td>PLoS Computational Biology</td>
          <td>31</td>
          <td>66</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/df9ec47c2e33646d1a37476584ad78620be168b9" target='_blank'>Machine Learning in Molecular Sciences</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>Machine Learning in Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/31739ea55a7c86ed38a1030bb99625ef96f9fc7a" target='_blank'>Manifold learning in atomistic simulations: a conceptual review</a></td>
          <td>
            
              J. Rydzewski,
            
              Ming Chen,
            
              O. Valsson,
            
          </td>
          <td>2023-03-15</td>
          <td>Machine Learning: Science and Technology</td>
          <td>7</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4275490b2442d0f4bfd56ec5f494e8c81543c0bb" target='_blank'>Past–future information bottleneck framework for simultaneously sampling biomolecular reaction coordinate, thermodynamics and kinetics</a></td>
          <td>
            
              Yihang Wang,
            
              João Marcelo Lamim Ribeiro,
            
              P. Tiwary,
            
          </td>
          <td>2018-12-28</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5711400d6fbba28fc776a3b85b6890b62934f64f" target='_blank'>End-to-end protein–ligand complex structure generation with diffusion-based generative models</a></td>
          <td>
            
              Shuya Nakata,
            
              Yoshiharu Mori,
            
              S. Tanaka,
            
          </td>
          <td>2022-12-21</td>
          <td>BMC Bioinformatics</td>
          <td>17</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c63657b2e54419a908221ea878df708d78a6409c" target='_blank'>Denoising diﬀusion probabilistic models for replica exchange</a></td>
          <td>
            
              Yihang Wang,
            
              P. Tiwary,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ddd8d63db17e6e6df42042054e360a18daf7e18b" target='_blank'>Tyrosine kinases: complex molecular systems challenging computational methodologies</a></td>
          <td>
            
              Trayder Thomas,
            
              B. Roux,
            
          </td>
          <td>2021-10-01</td>
          <td>The European Physical Journal B</td>
          <td>4</td>
          <td>117</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0dfec6265a347e9a7f077ca00dda79763eab0b1f" target='_blank'>Statistical Learning Theory of Protein Dynamics</a></td>
          <td>
            
              K. Haas,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fa25e6dbbf460affab05c322decf7ef5fc1142f7" target='_blank'>Markov State Models Provide Insights into Dynamic Modulation of Protein Function</a></td>
          <td>
            
              D. Shukla,
            
              Carlos X. Hernández,
            
              J. Weber,
            
              V. Pande,
            
          </td>
          <td>2015-01-03</td>
          <td>Accounts of Chemical Research</td>
          <td>220</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f1748ef585758622ea82d64e4ba5674ec0636d2b" target='_blank'>Exploring the conformational ensembles of protein-protein complex with transformer-based generative model</a></td>
          <td>
            
              Jianmin Wang,
            
              Xun Wang,
            
              Yanyi Chu,
            
              Chunyan Li,
            
              Xue Li,
            
              Xiangyu Meng,
            
              Yitian Fang,
            
              Kyoung Tai No,
            
              Jiashun Mao,
            
              Xiangxiang Zeng,
            
          </td>
          <td>2024-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/48753fd45da00aa39a4a14841683158f73625d38" target='_blank'>SurfDock is a Surface-Informed Diffusion Generative Model for Reliable and Accurate Protein-ligand Complex Prediction</a></td>
          <td>
            
              Duanhua Cao,
            
              Mingan Chen,
            
              Runze Zhang,
            
              Jie Yu,
            
              Xinyu Jiang,
            
              Zhehuan Fan,
            
              Wei Zhang,
            
              Mingyue Zheng,
            
          </td>
          <td>2023-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94ce2a11cb11cd4c1a467c1796b3680e857c51c5" target='_blank'>Conformation Clustering of Long MD Protein Dynamics with an Adversarial Autoencoder</a></td>
          <td>
            
              Yunlong Liu,
            
              L. Amzel,
            
          </td>
          <td>2018-05-31</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/535803259d7b37e7f024c97dc15f48884bb6ba80" target='_blank'>Stochastic Lag Time Parameterization for Markov State Models of Protein Dynamics.</a></td>
          <td>
            
              Shiqi Gong,
            
              Xinheng He,
            
              Qi Meng,
            
              Zhiming Ma,
            
              Bin Shao,
            
              Tong Wang,
            
              Tie-Yan Liu,
            
          </td>
          <td>2022-11-08</td>
          <td>The journal of physical chemistry. B</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/240f93655c42fff4a40aa62c11dfde1248641126" target='_blank'>Efficient estimation of rare-event kinetics</a></td>
          <td>
            
              Benjamin Trendelkamp-Schroer,
            
              F. Noé,
            
          </td>
          <td>2014-09-23</td>
          <td>arXiv: Chemical Physics</td>
          <td>34</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c4cd9da3045afd35938156980b80302966150cd7" target='_blank'>KINNTREX: A Neural Network Unveils Protein Mechanism from Time Resolved X-ray Crystallography</a></td>
          <td>
            
              Gabriel Biener,
            
              T. Malla,
            
              P. Schwander,
            
              Marius Schmidt,
            
          </td>
          <td>2023-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6b8840350f5d4a41d678977f61aab85232c91bcf" target='_blank'>A Latent Diffusion Model for Protein Structure Generation</a></td>
          <td>
            
              Cong Fu,
            
              Keqiang Yan,
            
              Limei Wang,
            
              Wing Yee Au,
            
              Michael McThrow,
            
              Tao Komikado,
            
              Koji Maruhashi,
            
              Kanji Uchino,
            
              Xiaoning Qian,
            
              Shuiwang Ji,
            
          </td>
          <td>2023-05-06</td>
          <td>ArXiv</td>
          <td>10</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bb580a55427341298007649714d434d0b5f2fd6c" target='_blank'>Large Scale Training of Graph Neural Networks for Optimal Markov-Chain Partitioning Using the Kemeny Constant</a></td>
          <td>
            
              S. Martino,
            
              Joao Morado,
            
              Chenghao Li,
            
              Zhenghao Lu,
            
              E. Rosta,
            
          </td>
          <td>2023-12-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4e3b2f677e03c624d9aed77a58fd596cebf452b3" target='_blank'>Sequence network 1 D 2 D Structure network Invariant features A C OrientationsDistances B weights features Energy</a></td>
          <td>
            
              John Ingraham,
            
              Adam J. Riesselman,
            
              C. Sander,
            
              D. Marks,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/731ef6c7b8761074d9c4b8d10bc9ded124bb00b0" target='_blank'>Employing artificial neural networks to find reaction coordinates and pathways for self-assembly</a></td>
          <td>
            
              Jörn H Appeldorn,
            
              Simon Lemcke,
            
              T. Speck,
            
              A. Nikoubashman,
            
          </td>
          <td>2021-08-31</td>
          <td>ChemRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/27fc4138289273c311dbfe6d26486521d7544396" target='_blank'>Toward a structural identification of metastable molecular conformations.</a></td>
          <td>
            
              Simon Lemcke,
            
              Jörn H Appeldorn,
            
              Michael Wand,
            
              Thomas Speck,
            
          </td>
          <td>2023-09-15</td>
          <td>The Journal of chemical physics</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3dbd0ff97315f3d70f5c0209068e394aa5476c6a" target='_blank'>Score Dynamics: Scaling Molecular Dynamics with Picoseconds Time Steps via Conditional Diffusion Model</a></td>
          <td>
            
              Tim Hsu,
            
              Babak Sadigh,
            
              Vasily V. Bulatov,
            
              Fei Zhou,
            
          </td>
          <td>2023-10-02</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bb3d0da51adfe51393e9952503f0722e7d4d1f2b" target='_blank'>Continuous Graphical Models for Static and Dynamic Distributions : Application to Structural Biology</a></td>
          <td>
            
              N. Razavian,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/925edfff606df315acd5d2fa1db693dd4dbe91fa" target='_blank'>Automated detection of many-particle solvation states for accurate characterizations of diffusion kinetics.</a></td>
          <td>
            
              J. F. Rudzinski,
            
              M. Radu,
            
              T. Bereau,
            
          </td>
          <td>2018-10-08</td>
          <td>The Journal of chemical physics</td>
          <td>2</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d791cdb5ee0c1ae8012d7c832620b941f618c8c6" target='_blank'>Efficient sampling in stochastic biological models</a></td>
          <td>
            
              Rory M. Donovan-Maiye,
            
          </td>
          <td>2016-09-07</td>
          <td></td>
          <td>0</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6bb5eeb7728656d61987b98aceb9cc4579a6f705" target='_blank'>A deep autoencoder framework for discovery of metastable ensembles in biomacromolecules.</a></td>
          <td>
            
              Satyabrata Bandyopadhyay,
            
              Jagannath Mondal,
            
          </td>
          <td>2021-06-01</td>
          <td>The Journal of chemical physics</td>
          <td>10</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7e0c748c252fedd9b4aa01670c2069ae273a41cc" target='_blank'>Mapping the Ligand Binding Landscape</a></td>
          <td>
            
              Alex Dickson,
            
          </td>
          <td>2018-06-13</td>
          <td>Biophysical Journal</td>
          <td>37</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f3eebe7cf446380258a42654978a9483e0325edf" target='_blank'>A Note on Learning Rare Events in Molecular Dynamics using LSTM and Transformer</a></td>
          <td>
            
              Wenqi Zeng,
            
              Siqin Cao,
            
              Xuhui Huang,
            
              Y. Yao,
            
          </td>
          <td>2021-07-14</td>
          <td>ArXiv</td>
          <td>8</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4229e3ba5f02221c22ef55e6be2825a08488d051" target='_blank'>SEEKR2: Versatile Multiscale Milestoning Utilizing the OpenMM Molecular Dynamics Engine</a></td>
          <td>
            
              L. Votapka,
            
              Andy Stokely,
            
              Anupam Anand Ojha,
            
              Rommie E. Amaro,
            
          </td>
          <td>2021-08-16</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>15</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f327f32fb45ed0fde1b5fc0315f1da4c3da7a0f6" target='_blank'>Deciphering complex mechanisms of resistance and loss of potency through coupled molecular dynamics and machine learning</a></td>
          <td>
            
              F. Leidner,
            
              N. K. Yilmaz,
            
              C. Schiffer,
            
          </td>
          <td>2020-06-09</td>
          <td>bioRxiv</td>
          <td>8</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6396ad9290c82a7ae1c7c138c9304e3ce62513f7" target='_blank'>Correction to “Exposing the Limitations of Molecular Machine Learning with Activity Cliffs”</a></td>
          <td>
            
              D. V. Tilborg,
            
              Alisa Alenicheva,
            
              F. Grisoni,
            
          </td>
          <td>2023-03-30</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>0</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e396698359d0ecb525dbfaf46e4fc841a08c18d4" target='_blank'>Variational cross-validation of slow dynamical modes in molecular kinetics.</a></td>
          <td>
            
              R. McGibbon,
            
              V. Pande,
            
          </td>
          <td>2014-07-30</td>
          <td>The Journal of chemical physics</td>
          <td>165</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c6f2f5df6f749df2fe4e7ad099b42d943e4d8a26" target='_blank'>Trajectory Statistical Learning of the Potential Mean of Force and Diffusion Coefficient from Molecular Dynamics Simulations.</a></td>
          <td>
            
              Yi-Tsao Chen,
            
              Haw Yang,
            
              J. Chu,
            
          </td>
          <td>2024-01-02</td>
          <td>The journal of physical chemistry. B</td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/be563ac0b8eb00e23104b3dc11e8ab7144501065" target='_blank'>The prediction of protein–ligand unbinding for modern drug discovery</a></td>
          <td>
            
              Qianqian Zhang,
            
              Nannan Zhao,
            
              Xiaoxiao Meng,
            
              Fansen Yu,
            
              X. Yao,
            
              Huanxiang Liu,
            
          </td>
          <td>2021-11-03</td>
          <td>Expert Opinion on Drug Discovery</td>
          <td>8</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7301b65b5c2370222f36559bac5c9d274b681976" target='_blank'>Building Markov state models using optimal transport theory.</a></td>
          <td>
            
              P. Dixit,
            
              K. Dill,
            
          </td>
          <td>2018-12-21</td>
          <td>The Journal of chemical physics</td>
          <td>7</td>
          <td>91</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/064ed7634c8ec92790accd4d1e107ebbb9d4e1ba" target='_blank'>Understanding what drives proteins</a></td>
          <td>
            
              V. Vinson,
            
          </td>
          <td>2020-11-27</td>
          <td>Science</td>
          <td>0</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1a3c895ac5d996424fac3a12011386c427195abf" target='_blank'>Development of Range-Corrected Deep Learning Potentials for Fast, Accurate Quantum Mechanical/Molecular Mechanical Simulations of Chemical Reactions in Solution.</a></td>
          <td>
            
              Jinzhe Zeng,
            
              Giese Tj,
            
              S. Ekesan,
            
              D. York,
            
          </td>
          <td>2021-02-26</td>
          <td>Journal of chemical theory and computation</td>
          <td>38</td>
          <td>49</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/89e93fdefd95b7147a784a92b598ef37961862e5" target='_blank'>Employing Artificial Neural Networks to Identify Reaction Coordinates and Pathways for Self-Assembly.</a></td>
          <td>
            
              Jörn H Appeldorn,
            
              Simon Lemcke,
            
              T. Speck,
            
              A. Nikoubashman,
            
          </td>
          <td>2022-07-06</td>
          <td>The journal of physical chemistry. B</td>
          <td>5</td>
          <td>56</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e3fd228d6b288db9b30f968697e345ab9954dc7e" target='_blank'>Navigating protein landscapes with a machine-learned transferable coarse-grained model</a></td>
          <td>
            
              N. Charron,
            
              Felix Musil,
            
              Andrea Guljas,
            
              Yaoyi Chen,
            
              Klara Bonneau,
            
              Aldo S. Pasos-Trejo,
            
              Jacopo Venturin,
            
              Daria Gusew,
            
              Iryna Zaporozhets,
            
              Andreas Kramer,
            
              Clark Templeton,
            
              Atharva S Kelkar,
            
              Aleksander E. P. Durumeric,
            
              Simon Olsson,
            
              Adria P'erez,
            
              Maciej Majewski,
            
              B. Husic,
            
              Ankit Patel,
            
              G. D. Fabritiis,
            
              Frank No'e,
            
              C. Clementi,
            
          </td>
          <td>2023-10-27</td>
          <td>None</td>
          <td>1</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/694c206ec85579d658af1a8ec04594eb29ad789d" target='_blank'>Deep learning collective variables from transition path ensemble.</a></td>
          <td>
            
              Dhiman Ray,
            
              Enrico Trizio,
            
              M. Parrinello,
            
          </td>
          <td>2023-03-02</td>
          <td>The Journal of chemical physics</td>
          <td>15</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/09b2d52317e91ab3757f8df4b1655d062ba330f8" target='_blank'>Neural networks learn the motions of molecular machines</a></td>
          <td>
            
              Timothy Grant,
            
          </td>
          <td>2021-07-29</td>
          <td>Nature Methods</td>
          <td>4</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/80d28e926b3cd25b9c298e82c404b20aa0b15789" target='_blank'>Embedded-physics machine learning for coarse-graining and collective variable discovery without data</a></td>
          <td>
            
              M. Schöberl,
            
              N. Zabaras,
            
              P. Koutsourelakis,
            
          </td>
          <td>2020-02-24</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca12d5de475be5acf8fda5e7797245897903ceca" target='_blank'>Dynamic-Backbone Protein-Ligand Structure Prediction with Multiscale Generative Diffusion Models</a></td>
          <td>
            
              Zhuoran Qiao,
            
              Weili Nie,
            
              Arash Vahdat,
            
              Thomas F. Miller,
            
              Anima Anandkumar,
            
          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>22</td>
          <td>60</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d7bb3a1118c85c4a948df1eeb93a4dd93d92f49a" target='_blank'>Deciphering protein evolution and fitness landscapes with latent space models</a></td>
          <td>
            
              Xinqiang Ding,
            
              Zhengting Zou,
            
              Charles L. Brooks III,
            
          </td>
          <td>2019-12-01</td>
          <td>Nature Communications</td>
          <td>68</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aa1146c1db4b205be456323fe657ae54848f9dd4" target='_blank'>Building insightful, memory-enriched models to capture long-time biochemical processes from short-time simulations</a></td>
          <td>
            
              Anthony J. Dominic,
            
              T. Sayer,
            
              Siqin Cao,
            
              T. Markland,
            
              Xuhui Huang,
            
              Andrés Montoya-Castillo,
            
          </td>
          <td>2022-10-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>13</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3333f169bd4815f92c03c9445761d8267c4e9ef2" target='_blank'>Simulate Time-integrated Coarse-grained Molecular Dynamics with Multi-Scale Graph Networks</a></td>
          <td>
            
              Xiang Fu,
            
              T. Xie,
            
              Nathan J. Rebello,
            
              B. Olsen,
            
              T. Jaakkola,
            
          </td>
          <td>2022-04-21</td>
          <td>None</td>
          <td>3</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/132df201f03dd6c0523c642d3cd7a4de6278e435" target='_blank'>Deep Autoencoders for Additional Insight into Protein Dynamics</a></td>
          <td>
            
              Mihai Teletin,
            
              G. Czibula,
            
              Maria-Iuliana Bocicor,
            
              S. Albert,
            
              A. Pandini,
            
          </td>
          <td>2018-10-04</td>
          <td>None</td>
          <td>8</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e753d1f39d086b536f0375a847057199f26dcc3f" target='_blank'>Nonlinear intrinsic variables and state reconstruction in multiscale simulations.</a></td>
          <td>
            
              Carmeline J. Dsilva,
            
              R. Talmon,
            
              N. Rabin,
            
              R. Coifman,
            
              I. Kevrekidis,
            
          </td>
          <td>2013-07-29</td>
          <td>The Journal of chemical physics</td>
          <td>40</td>
          <td>76</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f08060342a5fd4ffd65521422925dc584d8a6d02" target='_blank'>A multiscale approach for computing gated ligand binding from molecular dynamics and Brownian dynamics simulations</a></td>
          <td>
            
              S. Sadiq,
            
              Abraham Muñiz Chicharro,
            
              Patrick Friedrich,
            
              R. Wade,
            
          </td>
          <td>2021-06-23</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f24482261646024cc6fb5353c6a64e3b5e20e657" target='_blank'>Application of artiﬁcial neural networks to identify equilibration in computer simulations</a></td>
          <td>
            
              Mitchell H. Leibowitz,
            
              Evan D Miller,
            
              Michael M. Henry,
            
              Eric Jankowski,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>2</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/56be80957e4ddbb5c6ecdb3aa60741ecd178a106" target='_blank'>A versatile deep learning-based protein–ligand interaction prediction model for accurate binding aﬃnity scoring and virtual screening †</a></td>
          <td>
            
              Seokhyun Moon,
            
              Sang-Yeon Hwang,
            
              Jaechang Lim,
            
              Woo Youn Kim,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>2</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1f385d1785cc97103fc5e65b08969407ba348677" target='_blank'>Accurate Machine Learned Quantum-Mechanical Force Fields for Biomolecular Simulations</a></td>
          <td>
            
              Oliver T. Unke,
            
              M. Stohr,
            
              Stefan Ganscha,
            
              Thomas Unterthiner,
            
              Hartmut Maennel,
            
              S. Kashubin,
            
              Daniel Ahlin,
            
              M. Gastegger,
            
              L. M. Sandonas,
            
              A. Tkatchenko,
            
              Klaus-Robert Muller,
            
          </td>
          <td>2022-05-17</td>
          <td>ArXiv</td>
          <td>17</td>
          <td>76</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9989ae61660bd8e2ea95d144ca8768cfa734dc43" target='_blank'>Binding-Adaptive Diffusion Models for Structure-Based Drug Design</a></td>
          <td>
            
              Zhilin Huang,
            
              Ling Yang,
            
              Zaixi Zhang,
            
              Xiangxin Zhou,
            
              Yu Bao,
            
              Xiawu Zheng,
            
              Yuwei Yang,
            
              Yu Wang,
            
              Wenming Yang,
            
          </td>
          <td>2024-01-15</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6209957354367e3da4c9f978d93b0acfbf4d3dc8" target='_blank'>Deep Momentum Multi-Marginal Schrödinger Bridge</a></td>
          <td>
            
              T. Chen,
            
              Guan-Horng Liu,
            
              Molei Tao,
            
              E. Theodorou,
            
          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4d3c7cdbbb80131465c78e26b227b296089c8751" target='_blank'>Biomolecular Modeling and Simulation: A Prospering Multidisciplinary Field.</a></td>
          <td>
            
              T. Schlick,
            
              Stephanie Portillo-Ledesma,
            
              Christopher G. Myers,
            
              Lauren Beljak,
            
              Justin Chen,
            
              Sami Dakhel,
            
              D. Darling,
            
              Sayak Ghosh,
            
              Joseph Hall,
            
              Mikaeel Jan,
            
              Emily Liang,
            
              Sera Saju,
            
              Mackenzie Vohr,
            
              Chris Wu,
            
              ,
            
              Eva Xue,
            
          </td>
          <td>2021-02-19</td>
          <td>Annual review of biophysics</td>
          <td>31</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6941a3030881bb16393cb1b264aa3972b43a2bf8" target='_blank'>Scalable Multimer Structure Prediction using Diffusion Models</a></td>
          <td>
            
              Peter Pao-Huang,
            
              Bowen Jing,
            
              Bonnie Berger,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b20bd30a1274ac83242c2cbd74f452857a5f86e4" target='_blank'>Multiscale Reweighted Stochastic Embedding: Deep Learning of Collective Variables for Enhanced Sampling</a></td>
          <td>
            
              J. Rydzewski,
            
              O. Valsson,
            
          </td>
          <td>2020-07-13</td>
          <td>The Journal of Physical Chemistry. a</td>
          <td>24</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9a4400208b778f30ef77b2ba7600232f463dde59" target='_blank'>Markov State Models to Elucidate Ligand Binding Mechanism.</a></td>
          <td>
            
              Yunhui Ge,
            
              Vincent A. Voelz,
            
          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>1</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ea43533335fe627d8b9638e0c57d24464d25a30a" target='_blank'>Ig-VAE: Generative modeling of protein structure by direct 3D coordinate generation</a></td>
          <td>
            
              Raphael R. Eguchi,
            
              Christian A. Choe,
            
              Po-Ssu Huang,
            
          </td>
          <td>2020-08-10</td>
          <td>PLoS Computational Biology</td>
          <td>76</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ad98cd7c1964f09558ba914de240289c4906b1ca" target='_blank'>Machine Learning in Transition State Searching for Complex Biomolecules</a></td>
          <td>
            
              Yang Jian-Yu,
            
              Xi Kun,
            
              Zhu Li-Zhe,
            
          </td>
          <td>None</td>
          <td>Acta Physica Sinica</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/06a090117138a4866b34f7debba950545eab2cce" target='_blank'>Stability-Aware Training of Neural Network Interatomic Potentials with Differentiable Boltzmann Estimators</a></td>
          <td>
            
              Sanjeev Raja,
            
              Ishan Amin,
            
              Fabian Pedregosa,
            
              Aditi S. Krishnapriyan,
            
          </td>
          <td>2024-02-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cc327acf0fc22137b03dc714bb93df2c715377b6" target='_blank'>Kinetic and thermodynamic insights into sodium ion translocation through the μ-opioid receptor from molecular dynamics and machine learning analysis</a></td>
          <td>
            
              Xiaohu Hu,
            
              Yibo Wang,
            
              Amanda Hunkele,
            
              D. Provasi,
            
              G. Pasternak,
            
              M. Filizola,
            
          </td>
          <td>2019-01-01</td>
          <td>PLoS Computational Biology</td>
          <td>45</td>
          <td>87</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2f31bd26e2e34aa580344578af6c9e3c2151664c" target='_blank'>Themed collection on Insightful Machine Learning for Physical Chemistry.</a></td>
          <td>
            
              A. Clark,
            
              Pavlo O. Dral,
            
              I. Tamblyn,
            
              O. Isayev,
            
          </td>
          <td>2023-07-13</td>
          <td>Physical chemistry chemical physics : PCCP</td>
          <td>0</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78bbdec22c5465698ac761886cd9b0be90ce47a6" target='_blank'>Utilizing Machine Learning for Efficient Parameterization of Coarse Grained Molecular Force Fields</a></td>
          <td>
            
              J. McDonagh,
            
              Ardita Shkurti,
            
              D. J. Bray,
            
              Richard L. Anderson,
            
              Edward O. Pyzer-Knapp,
            
          </td>
          <td>2019-07-17</td>
          <td>Journal of chemical information and modeling</td>
          <td>40</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6195b623a13e861741ac1e80a7a8b774d5332236" target='_blank'>Fast and Uncertainty-Aware Directional Message Passing for Non-Equilibrium Molecules</a></td>
          <td>
            
              Johannes Klicpera,
            
              Shankari Giri,
            
              Johannes T. Margraf,
            
              Stephan Gunnemann,
            
          </td>
          <td>2020-11-28</td>
          <td>ArXiv</td>
          <td>228</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/923fab47b5c4e5a90219322ed0e7b1b7b94adc7b" target='_blank'>Variational approximation of molecular kinetics from short off-equilibrium simulations</a></td>
          <td>
            
              Hao Wu,
            
              F. Nüske,
            
              Fabian Paul,
            
              Stefan Klus,
            
              P. Koltai,
            
              F. Noé,
            
          </td>
          <td>2016-10-20</td>
          <td></td>
          <td>3</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2eb1ed9d4baf548391d8220c0a8a3d0be79845ed" target='_blank'>Boltzmann generators: Sampling equilibrium states of many-body systems with deep learning</a></td>
          <td>
            
              F. Noé,
            
              Jonas Köhler,
            
              Hao Wu,
            
          </td>
          <td>2019-09-05</td>
          <td>Science</td>
          <td>484</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/56db7a4959393d13aa4da01c34e8dc06fc4468cf" target='_blank'>Modeling biomolecular kinetics with large-scale simulation.</a></td>
          <td>
            
              P. Kasson,
            
          </td>
          <td>2021-09-27</td>
          <td>Current opinion in structural biology</td>
          <td>0</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/06b0a8c66ba5a4da7363116e7f4d65f1703d7e31" target='_blank'>Deep learning the slow modes for rare events sampling</a></td>
          <td>
            
              L. Bonati,
            
              G. Piccini,
            
              M. Parrinello,
            
          </td>
          <td>2021-07-08</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>107</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bccfbf2a9a02fa3e2a99f95a1fdae7a0f7d4e6c3" target='_blank'>Implicit Transfer Operator Learning: Multiple Time-Resolution Surrogates for Molecular Dynamics</a></td>
          <td>
            
              M. Schreiner,
            
              O. Winther,
            
              Simon Olsson,
            
          </td>
          <td>2023-05-29</td>
          <td>None</td>
          <td>4</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4068dd0b78ea1a6f3669028d1073f892e70857bd" target='_blank'>Modeling Changes in Molecular Dynamics Time Series as Wasserstein Barycentric Interpolations</a></td>
          <td>
            
              J. Damjanovic,
            
              Yu-Shan Lin,
            
              James M. Murphy,
            
          </td>
          <td>2023-07-10</td>
          <td>2023 International Conference on Sampling Theory and Applications (SampTA)</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c2f962fea9a69cd42c93c85f1c01078c2232358e" target='_blank'>Application of Molecular-Dynamics Based Markov State Models to Functional Proteins</a></td>
          <td>
            
              R. D. Malmstrom,
            
              Christopher T. Lee,
            
              Adam T. Van Wart,
            
              Rommie E. Amaro,
            
          </td>
          <td>2014-06-04</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>87</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a6e2e1189e3f888d1a01ca535ff9b3dad560b6d0" target='_blank'>High-fidelity molecular dynamics trajectory reconstruction with bi-directional neural networks</a></td>
          <td>
            
              Ludwig Winkler,
            
              Klaus-Robert Müller,
            
              H. E. Sauceda,
            
          </td>
          <td>2022-01-02</td>
          <td>Machine Learning: Science and Technology</td>
          <td>8</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/042e1fc051725f52a427c68f5af4309e122b7591" target='_blank'>Artificial Intelligence Resolves Kinetic Pathways of Magnesium Binding to RNA</a></td>
          <td>
            
              Jan Neumann,
            
              N. Schwierz,
            
          </td>
          <td>2021-07-26</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>8</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/554088d36d26052b46fc74aa305e85d377f9fa62" target='_blank'>Complex-based Ligand-Binding Proteins Redesign by Equivariant Diffusion-based Generative Models</a></td>
          <td>
            
              Viet Thanh Duy Nguyen,
            
              Nhan D. Nguyen,
            
              T. Hy,
            
          </td>
          <td>2024-04-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f6c635ccce0c3ed7b229e36364f7f4494e8b666c" target='_blank'>Metastability and Markov State Models in Molecular Dynamics: Modeling, Analysis, Algorithmic Approaches</a></td>
          <td>
            
              Christof Schütte,
            
              M. Sarich,
            
          </td>
          <td>2013-12-26</td>
          <td></td>
          <td>8</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bf136065e0aed4271f8ad3ff7e4e3305e9ba03c6" target='_blank'>Statistical model selection for Markov models of biomolecular dynamics.</a></td>
          <td>
            
              R. McGibbon,
            
              Christian Schwantes,
            
              V. Pande,
            
          </td>
          <td>2014-06-19</td>
          <td>The journal of physical chemistry. B</td>
          <td>23</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/544591d69f429b3b7031bbd202e09766e8057ff5" target='_blank'>Predicting improved protein conformations with a temporal deep recurrent neural network</a></td>
          <td>
            
              E. Pfeiffenberger,
            
              P. Bates,
            
          </td>
          <td>2018-03-03</td>
          <td>PLoS ONE</td>
          <td>14</td>
          <td>60</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0b62f4f4130bf7127678c536630b1f17bc1bcef3" target='_blank'>Maximizing Kinetic Information Gain of Markov State Models for Optimal Design of Spectroscopy Experiments.</a></td>
          <td>
            
              S. Mittal,
            
              D. Shukla,
            
          </td>
          <td>2018-10-17</td>
          <td>The journal of physical chemistry. B</td>
          <td>9</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/065f53dcdddb99b9fa049f4694a54a2b0ee1af0f" target='_blank'>Hierarchical, rotation-equivariant neural networks to predict the structure of protein complexes</a></td>
          <td>
            
              Stephan Eismann,
            
              Raphael J. L. Townshend,
            
              Nathaniel Thomas,
            
              Milind Jagota,
            
              Bowen Jing,
            
              R. Dror,
            
          </td>
          <td>2020-06-05</td>
          <td>ArXiv</td>
          <td>8</td>
          <td>78</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ea7ab102e770c40266c4ca8498d689c2a11ccd05" target='_blank'>DiAMoNDBack: Diffusion-denoising Autoregressive Model for Non-Deterministic Backmapping of Cα Protein Traces</a></td>
          <td>
            
              Michael S. Jones,
            
              Kirill Shmilovich,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2023-07-23</td>
          <td>Journal of chemical theory and computation</td>
          <td>2</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3c64b1d8a81ab02c1b4c097cf3a0329df20c1840" target='_blank'>Dynamic graphical models of molecular kinetics</a></td>
          <td>
            
              S. Olsson,
            
              F. Noé,
            
          </td>
          <td>2018-11-09</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>26</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c70ff8d7903100cfb6fe14622b254c640f58fce6" target='_blank'>models : Protein dynamics Chemical Theory and Computation Special Feature : Comparative study of generalized Born</a></td>
          <td>
            
              Hao Fan,
            
              A. Mark,
            
              Jiang Zhu,
            
              B. Honig,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>107</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a15ae2ed3a3394f65512577b9a95f39ff1a1b33c" target='_blank'>Physics-Guided Deep Generative Model For New Ligand Discovery</a></td>
          <td>
            
              Dikshant Sagar,
            
              Ali Risheh,
            
              Nida Sheikh,
            
              Negin Forouzesh,
            
          </td>
          <td>2023-09-03</td>
          <td>Proceedings of the 14th ACM International Conference on Bioinformatics, Computational Biology, and Health Informatics</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3f698eea76327bdbdffc10a3e2fa124c75609c3f" target='_blank'>Growing ecosystem of deep learning methods for modeling protein-protein interactions</a></td>
          <td>
            
              Julia R. Rogers,
            
              Gergo Nikolényi,
            
              Mohammed Alquraishi,
            
          </td>
          <td>2023-10-10</td>
          <td>Protein engineering, design & selection : PEDS</td>
          <td>2</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3eed492986e7a0c03362118daead192076d46446" target='_blank'>Supervised machine learning approach to molecular dynamics forecast of SARS-CoV-2 spike glycoproteins at varying temperatures</a></td>
          <td>
            
              David Liang,
            
              Meichen Song,
            
              Ziyuan Niu,
            
              Peng Zhang,
            
              M. Rafailovich,
            
              Yuefan Deng,
            
          </td>
          <td>2021-02-17</td>
          <td>Mrs Advances</td>
          <td>8</td>
          <td>60</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9cc9740a1c791664183a08d1a836a3def87bb580" target='_blank'>Deep Neural Network-Based Simulation of Sel’kov Model in Glycolysis: A Comprehensive Analysis</a></td>
          <td>
            
              Jamshaid Ul Rahman,
            
              Sana Danish,
            
              D. Lu,
            
          </td>
          <td>2023-07-21</td>
          <td>Mathematics</td>
          <td>1</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fdc1045cb29d704785f50dc3ff2162daaa9a9107" target='_blank'>A Deep Learning-Driven Sampling Technique to Explore the Phase Space of an RNA Stem-Loop</a></td>
          <td>
            
              Ayush Gupta,
            
              Heng Ma,
            
              Arvind Ramanathan,
            
              Gül H. Zerze,
            
          </td>
          <td>2024-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/df7b1ea2a9d6174ce83a0b04a709e2e5ea0a89e7" target='_blank'>Independent Markov decomposition: Toward modeling kinetics of biomolecular complexes</a></td>
          <td>
            
              Tim Hempel,
            
              Mauricio J. del Razo,
            
              Christopher T. Lee,
            
              Bryn C. Taylor,
            
              Rommie E. Amaro,
            
              F. Noé,
            
          </td>
          <td>2021-03-25</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>15</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b0d79612eec9b4d00a79747bed3fe592270ebdc7" target='_blank'>Mean Field Variational Approximations in Continuous-Time Markov Processes</a></td>
          <td>
            
              Ido Cohn,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>5</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e9cf45ce86339359aaee5f9c5e904a4c7ac0b68b" target='_blank'>Active learning of neural network potentials for rare events</a></td>
          <td>
            
              Gang Seob Jung,
            
              Jong Youl Choi,
            
              S. Lee,
            
          </td>
          <td>None</td>
          <td>Digital Discovery</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ede6ac519e2a62ebd81c2320a66eb0372130bef4" target='_blank'>Automated placement of interfaces in conformational kinetics calculations using machine learning.</a></td>
          <td>
            
              Gianmarc Grazioli,
            
              C. Butts,
            
              I. Andricioaei,
            
          </td>
          <td>2017-10-17</td>
          <td>The Journal of chemical physics</td>
          <td>4</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8acfcf199eff589b0396e5d6e4d8e1a2c03fa912" target='_blank'>Computational methods to examine conformational changes and ligand-binding properties: Examples in neurobiology</a></td>
          <td>
            
              M. A. Dämgen,
            
              P. Biggin,
            
          </td>
          <td>2019-05-01</td>
          <td>Neuroscience Letters</td>
          <td>13</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/38fc5852128337711554595ecf8de1cf66afe82e" target='_blank'>Caliber Corrected Markov Modeling (C2M2): Correcting Equilibrium Markov Models.</a></td>
          <td>
            
              P. Dixit,
            
              K. Dill,
            
          </td>
          <td>2017-11-08</td>
          <td>Journal of chemical theory and computation</td>
          <td>25</td>
          <td>91</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ec310088c4ed7a4665511e44ce169dffd19e4f9b" target='_blank'>Backdiff: a diffusion model for generalized transferable protein backmapping</a></td>
          <td>
            
              Yikai Liu,
            
              Ming Chen,
            
              Guang Lin,
            
          </td>
          <td>2023-10-03</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/385f4b919171f35824f45469a4de56cc915b7970" target='_blank'>Learning Large-Time-Step Molecular Dynamics with Graph Neural Networks</a></td>
          <td>
            
              Tian Zheng,
            
              Weihao Gao,
            
              Chong Wang,
            
          </td>
          <td>2021-11-30</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe3850f70016ae3a31767166fb996ed0f0325fd4" target='_blank'>Capabilities and Limitations of Time-lagged Autoencoders for Slow Mode Discovery in Dynamical Systems</a></td>
          <td>
            
              ,
            
              Hythem Sidky,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2019-06-02</td>
          <td>ArXiv</td>
          <td>31</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/03f062024f5212a0cdb048a927ec0c5aae2fd757" target='_blank'>Extract Latent Features of Single Particle Trajectories with Historical Experience Learning.</a></td>
          <td>
            
              Yongyu Zhang,
            
              Feng Ge,
            
              Xijian Lin,
            
              Jianfeng Xue,
            
              Yuxin Song,
            
              Hao Xie,
            
              Yan He,
            
          </td>
          <td>2023-10-01</td>
          <td>Biophysical journal</td>
          <td>1</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/815f7fa8739a869eac6ed18145ba215302ceaeb6" target='_blank'>Augmenting a training dataset of the generative diffusion model for molecular docking with artificial binding pockets</a></td>
          <td>
            
              Taras Voitsitskyi,
            
              Volodymyr Bdzhola,
            
              Roman Stratiichuk,
            
              Ihor Koleiev,
            
              Zakhar Ostrovsky,
            
              Volodymyr Vozniak,
            
              Ivan Khropachov,
            
              Pavlo Henitsoi,
            
              Leonid Popryho,
            
              Roman Zhytar,
            
              S. Yesylevskyy,
            
              Alan Nafiiev,
            
              Serhii Starosyla,
            
          </td>
          <td>2024-01-02</td>
          <td>RSC Advances</td>
          <td>1</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dfb473833f5f7e0677e08e1439fff618077cc311" target='_blank'>Using classifiers to understand coarse-grained models and their fidelity with the underlying all-atom systems.</a></td>
          <td>
            
              Aleksander E. P. Durumeric,
            
              G. Voth,
            
          </td>
          <td>2023-06-15</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>103</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7cd70c6493caeafb8ff9be638229a14f878c8d97" target='_blank'>Analyzing Multimodal Probability Measures with Autoencoders.</a></td>
          <td>
            
              Tony Lelièvre,
            
              Thomas Pigeon,
            
              G. Stoltz,
            
              Wei Zhang,
            
          </td>
          <td>2023-10-05</td>
          <td>The journal of physical chemistry. B</td>
          <td>1</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5e660b3335f6a2a1d8592c6bfd048b1d1aaa6021" target='_blank'>Learning minimal representations of stochastic processes with variational autoencoders</a></td>
          <td>
            
              Gabriel Fern'andez-Fern'andez,
            
              Carlo Manzo,
            
              M. Lewenstein,
            
              A. Dauphin,
            
              Gorka Muñoz-Gil,
            
          </td>
          <td>2023-07-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>93</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c47e8168158a85f81d39dc57387b35d934514c6e" target='_blank'>Latent Models of Molecular Dynamics Data: Automatic Order Parameter Generation for Peptide Fibrillization.</a></td>
          <td>
            
              Nathaniel Charest,
            
              M. Tro,
            
              M. Bowers,
            
              J. Shea,
            
          </td>
          <td>2020-08-11</td>
          <td>The journal of physical chemistry. B</td>
          <td>7</td>
          <td>79</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78a4edd4f2d62d7fcb1a32e02cac7a51cb87500c" target='_blank'>Markovian Weighted Ensemble Milestoning (M-WEM): Long-time Kinetics from Short Trajectories</a></td>
          <td>
            
              Dhiman Ray,
            
              S. E. Stone,
            
              I. Andricioaei,
            
          </td>
          <td>2021-06-28</td>
          <td>bioRxiv</td>
          <td>20</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/286f8e41b003ea1589362df451ae3db24e6facb8" target='_blank'>Reconstruction of protein structures from single-molecule time series.</a></td>
          <td>
            
              Maximilian Topel,
            
              Andrew L. Ferguson,
            
          </td>
          <td>2020-07-30</td>
          <td>The Journal of chemical physics</td>
          <td>5</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e58139737979e1ba99fbce12a47a86edae958418" target='_blank'>NNP/MM: Accelerating Molecular Dynamics Simulations with Machine Learning Potentials and Molecular Mechanics</a></td>
          <td>
            
              Raimondas Galvelis,
            
              Alejandro Varela-Rial,
            
              Stefan Doerr,
            
              R. Fino,
            
              P. Eastman,
            
              T. Markland,
            
              J. Chodera,
            
              G. D. Fabritiis,
            
          </td>
          <td>2022-01-20</td>
          <td>Journal of chemical information and modeling</td>
          <td>11</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d767444574a2b9bb0dc6ca2b5e0f6f2f0fbfaf68" target='_blank'>Chemically Transferable Generative Backmapping of Coarse-Grained Proteins</a></td>
          <td>
            
              Soojung Yang,
            
              Rafael G'omez-Bombarelli,
            
          </td>
          <td>2023-03-02</td>
          <td>ArXiv</td>
          <td>9</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9163d323d874125c0375307918395523bea3e02a" target='_blank'>Machine learning coarse-grained potentials of protein thermodynamics</a></td>
          <td>
            
              Maciej Majewski,
            
              Adriana P'erez,
            
              Philipp Thölke,
            
              Stefan Doerr,
            
              N. Charron,
            
              T. Giorgino,
            
              B. Husic,
            
              C. Clementi,
            
              Frank No'e,
            
              G. D. Fabritiis,
            
          </td>
          <td>2022-12-14</td>
          <td>Nature Communications</td>
          <td>23</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ad07d3499faade81e6c33069902c45b13ba90c44" target='_blank'>Broadly applicable and accurate protein design by integrating structure prediction networks and diffusion generative models</a></td>
          <td>
            
              Joseph L. Watson,
            
              David Juergens,
            
              N. Bennett,
            
              Brian L. Trippe,
            
              Jason Yim,
            
              Helen E. Eisenach,
            
              Woody Ahern,
            
              Andrew J. Borst,
            
              R. Ragotte,
            
              L. Milles,
            
              B. Wicky,
            
              Nikita Hanikel,
            
              S. Pellock,
            
              A. Courbet,
            
              W. Sheffler,
            
              Jue Wang,
            
              Preetham Venkatesh,
            
              Isaac Sappington,
            
              Susana Vázquez Torres,
            
              A. Lauko,
            
              Valentin De Bortoli,
            
              Emile Mathieu,
            
              R. Barzilay,
            
              T. Jaakkola,
            
              F. DiMaio,
            
              M. Baek,
            
              D. Baker,
            
          </td>
          <td>2022-12-14</td>
          <td>bioRxiv</td>
          <td>134</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf6a85689c63810951d5f5efc1d488a1e8a1c96d" target='_blank'>Leveraging attention-enhanced variational autoencoders: Novel approach for investigating latent space of aptamer sequences.</a></td>
          <td>
            
              Abbas Salimi,
            
              Jee Hwan Jang,
            
              Jin Yong Lee,
            
          </td>
          <td>2023-11-01</td>
          <td>International journal of biological macromolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/426fa8decdfaffbe7d9299b400d669750c02a662" target='_blank'>: Extraction of Protein Dynamics Information Hidden in Cryo-EM Map Using Deep Learning</a></td>
          <td>
            
              S. Matsumoto,
            
              Shoichi Ishida,
            
              M. Araki,
            
              Takayuki Kato,
            
              Kei Terayama,
            
              Yasushi Okuno,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c077f609041111d24e5163c2e3556ef158c46e3d" target='_blank'>A dual diffusion model enables 3D binding bioactive molecule generation and lead optimization given target pockets</a></td>
          <td>
            
              Lei Huang,
            
              Tingyang Xu,
            
              Yang Yu,
            
              P. Zhao,
            
              Ka-chun Wong,
            
              Heng Zhang,
            
          </td>
          <td>2023-01-30</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8cf33b84937ebba66749c4bd470aea8fb4b448e5" target='_blank'>Explore Conformational Space of Proteins with Supervised Auto-Encoder</a></td>
          <td>
            
              Chen Guanglin,
            
              Zhang Zhiyong,
            
          </td>
          <td>None</td>
          <td>Acta Physica Sinica</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a4ac048edff84a0d87a14943042f0edf46316da4" target='_blank'>Bayesian Abstraction of Markov Population Models</a></td>
          <td>
            
              L. Bortolussi,
            
              Francesca Cairoli,
            
          </td>
          <td>2019-09-10</td>
          <td>None</td>
          <td>4</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5ee2916e52bec10294deacb09e73c924d95908c5" target='_blank'>Targeted Adversarial Learning Optimized Sampling.</a></td>
          <td>
            
              Jun Zhang,
            
              Yi Isaac Yang,
            
              F. Noé,
            
          </td>
          <td>2019-04-02</td>
          <td>The journal of physical chemistry letters</td>
          <td>49</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dec1deaaa4969250ae7c43206dee2cdea0958952" target='_blank'>Modelling the Dynamics of Biological Systems with the Geometric Hidden Markov Model</a></td>
          <td>
            
              Borislav Vangelov,
            
              Mauricio Barahona,
            
          </td>
          <td>2017-11-22</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/da7766c76f560d043e443738168ce649470ce851" target='_blank'>High Performance Computing aspects of Single Particle Machine Learning</a></td>
          <td>
            
              M. Näslund,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6f3752c5d5ace3041c4cdf43b7c4e9c1145199e6" target='_blank'>Malramsay64/Mlcrystals-Tutorial: Release Version 1.0 Of Tutorial</a></td>
          <td>
            
              M. Ramsay,
            
              Jwood,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d7de3d18e174fb9ac9049d260882afd7bb0a814b" target='_blank'>Author Correction: Optimization of Molecules via Deep Reinforcement Learning</a></td>
          <td>
            
              Zhenpeng Zhou,
            
              S. Kearnes,
            
              Li Li,
            
              R. Zare,
            
              Patrick F. Riley,
            
          </td>
          <td>2020-06-23</td>
          <td>Scientific Reports</td>
          <td>2</td>
          <td>115</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/25b70f092725bf78fad2243de88cbd68159186c9" target='_blank'>Decoding Interaction Patterns from the Chemical Sequence of Polymers Using Neural Networks</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>4</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8907524fc56b8194103f3c179a1af90073a3971f" target='_blank'>Target-aware Variational Auto-encoders for Ligand Generation with Multimodal Protein Representation Learning</a></td>
          <td>
            
              Nhat-Khang Ngô,
            
              T. Hy,
            
          </td>
          <td>2023-08-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ea3201dba38ddcac8fb78fb0c126cd059e3822dc" target='_blank'>Pre‐Training of Equivariant Graph Matching Networks with Conformation Flexibility for Drug Binding</a></td>
          <td>
            
              Fang Wu,
            
              Shuting Jin,
            
              Xurui Jin,
            
              Xiangrong Liu,
            
              Yinghui Jiang,
            
              Z. Niu,
            
              Qian Zhang,
            
              Stan Z. Li,
            
          </td>
          <td>2022-04-19</td>
          <td>Advanced Science</td>
          <td>14</td>
          <td>97</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8461c8b7dbbf71c0f3aea3e6a5852bcc21620401" target='_blank'>Ancestral protein sequence reconstruction using a tree-structured Ornstein-Uhlenbeck variational autoencoder</a></td>
          <td>
            
              L. S. Moreta,
            
              Ahmad Salim Al-Sibahi,
            
              J. Hein,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>2</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/13cbfaf3058d4c2302e619cbaf7c3fcb0a77069f" target='_blank'>Partitioned integrators for thermodynamic parameterization of neural networks</a></td>
          <td>
            
              B. Leimkuhler,
            
              Charles Matthews,
            
              Tiffany J. Vlaar,
            
          </td>
          <td>2019-08-30</td>
          <td>ArXiv</td>
          <td>19</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/522df16f12bde5b06e6568b74a23efe4f29773e9" target='_blank'>The confluence of machine learning and multiscale simulations.</a></td>
          <td>
            
              H. Bhatia,
            
              Fikret Aydin,
            
              Timothy S. Carpenter,
            
              F. Lightstone,
            
              P. Bremer,
            
              Helgi I. Ingólfsson,
            
              D. Nissley,
            
              F. Streitz,
            
          </td>
          <td>2023-03-23</td>
          <td>Current opinion in structural biology</td>
          <td>5</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c5190554d9d124de8c12640d368548bef0d74578" target='_blank'>Learning molecular dynamics: predicting the dynamics of glasses by a machine learning simulator.</a></td>
          <td>
            
              Han Liu,
            
              Zijie Huang,
            
              S. Schoenholz,
            
              E. D. Cubuk,
            
              M. Smedskjaer,
            
              Yizhou Sun,
            
              W. Wang,
            
              M. Bauchy,
            
          </td>
          <td>2023-06-29</td>
          <td>Materials horizons</td>
          <td>2</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/596e2415ccaea299b19e38f3085fcf59f3d39b81" target='_blank'>Machine learning enables long time scale molecular photodynamics simulations† †Electronic supplementary information (ESI) available. See DOI: 10.1039/c9sc01742a</a></td>
          <td>
            
              Julia Westermayr,
            
              M. Gastegger,
            
              M. Menger,
            
              ,
            
              L. González,
            
              Philipp,
            
              Marquetand,
            
          </td>
          <td>2018-11-22</td>
          <td>Chemical Science</td>
          <td>57</td>
          <td>49</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ad97f5fb9d261252b3ffd132d5c69f9f204aec5b" target='_blank'>Mixup-Augmented Meta-Learning for Sample-Efficient Fine-Tuning of Protein Simulators</a></td>
          <td>
            
              Jingbang Chen,
            
              Yian Wang,
            
              Xingwei Qu,
            
              Shuangjia Zheng,
            
              Yao-Chun Yang,
            
              Hao Dong,
            
              Jie Fu,
            
          </td>
          <td>2023-08-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9a1c517e3648c85f8b3f39679b4fadf153a19916" target='_blank'>Free energy along drug-protein binding pathways interactively sampled in virtual reality</a></td>
          <td>
            
              Helen M. Deeks,
            
              Kirill Zinovjev,
            
              J. Barnoud,
            
              A. Mulholland,
            
              M. W. van der Kamp,
            
              David R Glowacki,
            
          </td>
          <td>2023-10-04</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>58</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/751910f192146126ec050ac5c45a8f3416d58387" target='_blank'>The RED scheme: Rate-constant estimation from pre-steady state weighted ensemble simulations.</a></td>
          <td>
            
              A. DeGrave,
            
              Anthony T. Bogetti,
            
              L. Chong,
            
          </td>
          <td>2021-03-21</td>
          <td>The Journal of chemical physics</td>
          <td>11</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/68981c9aba4c06176f3c062b94c3e6861371bdb6" target='_blank'>Re-Dock: Towards Flexible and Realistic Molecular Docking with Diffusion Bridge</a></td>
          <td>
            
              Yufei Huang,
            
              Odin Zhang,
            
              Lirong Wu,
            
              Cheng Tan,
            
              Haitao Lin,
            
              Zhangyang Gao,
            
              Siyuan Li,
            
              Stan Z. Li,
            
          </td>
          <td>2024-02-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/93013a0ad444641953c01cd8182a84e1dff378a0" target='_blank'>What Markov state models can and cannot do: Correlation versus path-based observables in protein folding models</a></td>
          <td>
            
              Ernesto Suárez,
            
              Rafal P. Wiewiora,
            
              C. Wehmeyer,
            
              F. Noé,
            
              J. Chodera,
            
              D. Zuckerman,
            
          </td>
          <td>2020-11-09</td>
          <td>bioRxiv</td>
          <td>38</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c9295b260154dc8b4d95bc4a19da9d297dcf0d26" target='_blank'>Uncertainty-driven dynamics for active learning of interatomic potentials</a></td>
          <td>
            
              M. Kulichenko,
            
              K. Barros,
            
              N. Lubbers,
            
              Ying Wai Li,
            
              Richard A. Messerly,
            
              S. Tretiak,
            
              Justin S. Smith,
            
              B. Nebgen,
            
          </td>
          <td>2023-03-01</td>
          <td>Nature Computational Science</td>
          <td>32</td>
          <td>69</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/84e07c84a411ec5ed0828049d777b9cb8129c14d" target='_blank'>DiffBP: Generative Diffusion of 3D Molecules for Target Protein Binding</a></td>
          <td>
            
              Haitao Lin,
            
              Yufei Huang,
            
              Meng Liu,
            
              X. Li,
            
              Shuiwang Ji,
            
              Stan Z. Li,
            
          </td>
          <td>2022-11-21</td>
          <td>ArXiv</td>
          <td>35</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c94bd71338acf8a4f759c45587c85b0b03c79f0e" target='_blank'>Physics-informed machine learning for automatic model reduction in chemical reaction networks</a></td>
          <td>
            
              J. Pateras,
            
              Colin Zhang,
            
              Shriya Majumdar,
            
              Ayush Pal,
            
              Preetam Ghosh,
            
          </td>
          <td>2024-03-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/de3d1b8a3551553b621dd0ea3feb91cf013b694b" target='_blank'>Introducing prescribed biases in out of equilibrium Markov models</a></td>
          <td>
            
              P. Dixit,
            
          </td>
          <td>2017-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d40c02d28475b42d17ba4e06714da440e9f9c846" target='_blank'>An efficient strategy to estimate thermodynamics and kinetics of G protein-coupled receptor activation using metadynamics and maximum caliber</a></td>
          <td>
            
              Derya Meral,
            
              D. Provasi,
            
              M. Filizola,
            
          </td>
          <td>2018-07-12</td>
          <td>The Journal of Chemical Physics</td>
          <td>33</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5b8403be6597123db420112c1b1e6bca6351e475" target='_blank'>Algorithms for building models of molecular motion from simulations</a></td>
          <td>
            
              N. Hinrichs,
            
          </td>
          <td>None</td>
          <td></td>
          <td>5</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5ba1cc65d3cabc7061db491d6f45d950a60a57c9" target='_blank'>Overcoming the timescale barrier in molecular dynamics: Transfer operators, variational principles and machine learning</a></td>
          <td>
            
              C. Schütte,
            
              Stefan Klus,
            
              C. Hartmann,
            
          </td>
          <td>2023-05-01</td>
          <td>Acta Numerica</td>
          <td>11</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4b525c91a75f90c3ecd478324c0c77e0f839c675" target='_blank'>Generating 3D Molecular Structures Conditional on a Receptor Binding Site with Deep Generative Models</a></td>
          <td>
            
              Tomohide Masuda,
            
              Matthew Ragoza,
            
              D. Koes,
            
          </td>
          <td>2020-10-16</td>
          <td>ArXiv</td>
          <td>44</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/79714b9ee1abe7f5e73947235392a57d15dc54a2" target='_blank'>∆ -Learning for Coarse-Grained Potentials</a></td>
          <td>
            
              Aditi Khot,
            
              B. Savoie,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1f5b19a1f10b8a7836d6e9b7e57bd5acc6b00f85" target='_blank'>Toward a direct and scalable identification of reduced models for categorical processes</a></td>
          <td>
            
              Susanne Gerber,
            
              I. Horenko,
            
          </td>
          <td>2017-04-21</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>19</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a6310d27a05de6b90763205b2ab808cdb7ebfba4" target='_blank'>SSnet: A Deep Learning Approach for Protein-Ligand Interaction Prediction</a></td>
          <td>
            
              Niraj Verma,
            
              Xingming Qu,
            
              Francesco Trozzi,
            
              M. Elsaied,
            
              Nischal Karki,
            
              Yunwen Tao,
            
              B. Zoltowski,
            
              Eric C. Larson,
            
              E. Kraka,
            
          </td>
          <td>2019-12-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>29</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/170afbeba3b354451730e6f8b520fdc34208d79c" target='_blank'>Generating intrinsically disordered protein conformational ensembles from a Markov chain.</a></td>
          <td>
            
              R. Cukier,
            
          </td>
          <td>2018-03-14</td>
          <td>The Journal of chemical physics</td>
          <td>3</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b7e1d24048310c5822bd64bba1e9ec1b8d17593a" target='_blank'>New Machine Learning Methods for the Prediction of Protein Topologies</a></td>
          <td>
            
              P. Baldi,
            
              G. Pollastri,
            
              P. Frasconi,
            
              A. Vullo,
            
          </td>
          <td>None</td>
          <td></td>
          <td>4</td>
          <td>114</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/49d86747498c956aa35a2d3b7a7f1e6e1180586a" target='_blank'>Long Short-Term Memory Neural Network on the Trajectory Computing of Direct Dynamics Simulation</a></td>
          <td>
            
              Fred Y. Wu,
            
              ,
            
              Colmenares Eduardo,
            
              ,
            
              ,
            
              ,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cee36473ff970b98a36543bf2997360abdcead50" target='_blank'>Machine learning coarse grained models for water</a></td>
          <td>
            
              Henry Chan,
            
              M. Cherukara,
            
              B. Narayanan,
            
              T. Loeffler,
            
              C. Benmore,
            
              S. Gray,
            
              S. Sankaranarayanan,
            
          </td>
          <td>2019-01-22</td>
          <td>Nature Communications</td>
          <td>110</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a083d9b69ac6d5d6fc25487401529bbe15f443ee" target='_blank'>On the Approximation Quality of Markov State Models</a></td>
          <td>
            
              M. Sarich,
            
              F. Noé,
            
              C. Schütte,
            
          </td>
          <td>2010-05-28</td>
          <td>Multiscale Model. Simul.</td>
          <td>170</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d7dfae44bec49719061c7780c0d18168983741ab" target='_blank'>Efficient and scalable prediction of spatio-temporal stochastic gene expression in cells and tissues using graph neural networks</a></td>
          <td>
            
              Zhixing Cao,
            
              Rui Chen,
            
              Libin Xu,
            
              Xinyi Zhou,
            
              Xiao-jin Fu,
            
              Weimin Zhong,
            
              R. Grima,
            
          </td>
          <td>2023-03-01</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7d496d478d1e2a28194b23f42d632b8f9a923c84" target='_blank'>Fast dynamics and stationary averages of PDMPs using parallel replica dynamics</a></td>
          <td>
            
              D. Aristoff,
            
              Peter Christman,
            
          </td>
          <td>2018-02-26</td>
          <td>arXiv: Numerical Analysis</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8120c229a6c660c64fdd72221f868abcff31f3c1" target='_blank'>Ensemble learning of coarse-grained molecular dynamics force fields with a kernel approach.</a></td>
          <td>
            
              Jiang Wang,
            
              Stefan Chmiela,
            
              K. Müller,
            
              F. Noé,
            
              C. Clementi,
            
          </td>
          <td>2020-05-04</td>
          <td>The Journal of chemical physics</td>
          <td>41</td>
          <td>126</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/32882132f465a4f93f0112028fe4cbbe3d2279f9" target='_blank'>Steps and bumps: precision extraction of discrete states of molecular machines.</a></td>
          <td>
            
              Max A. Little,
            
              B. Steel,
            
              F. Bai,
            
              Y. Sowa,
            
              Thomas Bilyard,
            
              D. Mueller,
            
              R. Berry,
            
              N. S. Jones,
            
          </td>
          <td>2010-04-08</td>
          <td>Biophysical journal</td>
          <td>31</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5b261adbd2518e9c93ad62a4d14b80af79990061" target='_blank'>A Deep Learning Bioinformatics Approach to Modeling Protein-Ligand Interaction with cryo-EM Data in 2021 Ligand Model Challenge</a></td>
          <td>
            
              Nabin Giri,
            
              Jianlin Cheng,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/52b22e6103f96f58dcd721cc89a5d4bebe1cb7bb" target='_blank'>Allegro-Legato: Scalable, Fast, and Robust Neural-Network Quantum Molecular Dynamics via Sharpness-Aware Minimization</a></td>
          <td>
            
              Hikaru Ibayashi,
            
              Taufeq Mohammed Razakh,
            
              Liqiu Yang,
            
              Thomas M Linker,
            
              M. Olguin,
            
              Shinnosuke Hattori,
            
              Ye Luo,
            
              R. Kalia,
            
              A. Nakano,
            
              K. Nomura,
            
              P. Vashishta,
            
          </td>
          <td>2023-03-14</td>
          <td>None</td>
          <td>5</td>
          <td>60</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/38309ab72b0dbdef7d39faeff8e4df59cadaeabd" target='_blank'>High-dimensional structure learning of binary pairwise Markov networks: A comparative numerical study</a></td>
          <td>
            
              J. Pensar,
            
              Y. Xu,
            
              S. Puranen,
            
              M. Pesonen,
            
              Y. Kabashima,
            
              J. Corander,
            
          </td>
          <td>2019-01-14</td>
          <td>Comput. Stat. Data Anal.</td>
          <td>2</td>
          <td>70</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c6858f3930e558d27e6c00fe65ea4dc558ff7bd9" target='_blank'>Structure-based, deep-learning models for protein-ligand binding affinity prediction</a></td>
          <td>
            
              Debby D. Wang,
            
              Wenhui Wu,
            
              Ran Wang,
            
          </td>
          <td>2024-01-03</td>
          <td>Journal of Cheminformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8dddbcbdbeb3ac9be7c77ccd2001fa713eed2dca" target='_blank'>Fast and accurate Ab Initio Protein structure prediction using deep learning potentials</a></td>
          <td>
            
              Robin Pearce,
            
              Yang Li,
            
              G. Omenn,
            
              Yang Zhang,
            
          </td>
          <td>2022-09-01</td>
          <td>PLoS Computational Biology</td>
          <td>8</td>
          <td>84</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f545b28a376979ac157a4ae96d1ec7b04fc5ff1" target='_blank'>Improving ΔΔG Predictions with a Multitask Convolutional Siamese Network</a></td>
          <td>
            
              Andrew T. McNutt,
            
              D. Koes,
            
          </td>
          <td>2021-12-14</td>
          <td>Journal of chemical information and modeling</td>
          <td>13</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a76b6396a6f1326a3afdefc850f2129dad7c3720" target='_blank'>Neural Networks In Chemical Reaction Dynamics</a></td>
          <td>
            
              K. Baumgartner,
            
          </td>
          <td>None</td>
          <td></td>
          <td>28</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/100147e735fe400a66844dfece80e3dcbdf04eef" target='_blank'>Calculation of Protein-Ligand Binding Free Energy Using a Physics-Guided Neural Network</a></td>
          <td>
            
              Sahar Cain,
            
              Ali Risheh,
            
              Negin Forouzesh,
            
          </td>
          <td>2021-12-09</td>
          <td>2021 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>2</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e9527775263020013e141fd6a99d9c22ae1380cd" target='_blank'>Adaptive Sampling Methods for Molecular Dynamics in the Era of Machine Learning.</a></td>
          <td>
            
              D. Kleiman,
            
              Hassan Nadeem,
            
              D. Shukla,
            
          </td>
          <td>2023-07-18</td>
          <td>The journal of physical chemistry. B</td>
          <td>5</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5123ee0f8b2bc47278e5d87731f53d1e98191d09" target='_blank'>Harnessing protein folding neural networks for peptide–protein docking</a></td>
          <td>
            
              Tomer Tsaban,
            
              Julia K. Varga,
            
              O. Avraham,
            
              Ziv Ben-Aharon,
            
              A. Khramushin,
            
              Ora Schueler‐Furman,
            
          </td>
          <td>2021-08-02</td>
          <td>Nature Communications</td>
          <td>163</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5123ee0f8b2bc47278e5d87731f53d1e98191d09" target='_blank'>Harnessing protein folding neural networks for peptide–protein docking</a></td>
          <td>
            
              Tomer Tsaban,
            
              Julia K. Varga,
            
              O. Avraham,
            
              Ziv Ben-Aharon,
            
              A. Khramushin,
            
              Ora Schueler‐Furman,
            
          </td>
          <td>2021-08-02</td>
          <td>Nature Communications</td>
          <td>163</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/023e66706204102bddbbe9a9521e7e0c0bdc5ee0" target='_blank'>ScanNet: A Web Server for Structure-based Prediction of Protein Binding Sites with Geometric Deep Learning.</a></td>
          <td>
            
              J. Tubiana,
            
              D. Schneidman-Duhovny,
            
              H. Wolfson,
            
          </td>
          <td>2022-07-01</td>
          <td>Journal of molecular biology</td>
          <td>8</td>
          <td>70</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/53281dba07603d4819ad54051150989a789ccd94" target='_blank'>Editorial: Decision making from the perspective of neural thermodynamics and molecular information processing</a></td>
          <td>
            
              R. Rastmanesh,
            
              Eva K. Deli,
            
              Sisir Roy,
            
              Brent Vogt,
            
          </td>
          <td>2022-08-16</td>
          <td>Frontiers in Neuroscience</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7ed7686f52f68686e734ad4a459ded57aff00cb7" target='_blank'>Towards Molecular Machine Learning for the IoBNT</a></td>
          <td>
            
              Stefan Angerbauer,
            
              Franz Enzenhofer,
            
              Tobias Pankratz,
            
              W. Haselmayr,
            
          </td>
          <td>2023-09-20</td>
          <td>Proceedings of the 10th ACM International Conference on Nanoscale Computing and Communication</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c49772678e60d2d972e0cdd25bc383b3cf6910ae" target='_blank'>Toward generalizable structure‐based deep learning models for protein–ligand interaction prediction: Challenges and strategies</a></td>
          <td>
            
              Seokhyun Moon,
            
              Wonho Zhung,
            
              Woo Youn Kim,
            
          </td>
          <td>2024-01-01</td>
          <td>WIREs Computational Molecular Science</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5c8fca05881025024452f677b6528570d54c83a3" target='_blank'>Linear Transformations in Autoencoder Latent Space Predict Time Translations in Active Matter System</a></td>
          <td>
            
              E. Amaya,
            
              S. Shadkhoo,
            
              Dominik Schildknecht,
            
              Matt Thomson,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3e26142d5b3e0a9ec2f13ece949c255d62bfd59e" target='_blank'>Kinetic transition network based on trajectory mapping.</a></td>
          <td>
            
              Linchen Gong,
            
              Xin Zhou,
            
          </td>
          <td>2010-07-23</td>
          <td>The journal of physical chemistry. B</td>
          <td>17</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/29b894452af4b0fea4c49de5b685f47531798107" target='_blank'>An Analog Chemical Circuit with Parallel-Accessible Delay Line for Learning Temporal Tasks</a></td>
          <td>
            
              Peter Banda,
            
              C. Teuscher,
            
          </td>
          <td>2014-07-30</td>
          <td>None</td>
          <td>8</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4e62888cbdbf318a8a0d20f660a75ff135fa2b1a" target='_blank'>Improving Protein–Ligand Interaction Modeling with cryo-EM Data, Templates, and Deep Learning in 2021 Ligand Model Challenge</a></td>
          <td>
            
              Nabin Giri,
            
              Jianlin Cheng,
            
          </td>
          <td>2022-12-23</td>
          <td>Biomolecules</td>
          <td>11</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e76e0d777a96cf216e35ba552a27b3db33ffddeb" target='_blank'>3D pride without 2D prejudice: Bias-controlled multi-level generative models for structure-based ligand design</a></td>
          <td>
            
              Lucian Chan,
            
              Rajendra Kumar,
            
              M. Verdonk,
            
              C. Poelking,
            
          </td>
          <td>2022-04-22</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d757b4604edd1ee395fcda54d166c906e080cff9" target='_blank'>Exploring glassy dynamics with Markov state models from graph dynamical neural networks.</a></td>
          <td>
            
              S. Soltani,
            
              C. Sinclair,
            
              Joerg Rottler,
            
          </td>
          <td>2021-11-19</td>
          <td>Physical review. E</td>
          <td>3</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1d3aebc51df18ba1dd32c232c6535de4262e473d" target='_blank'>Graphical continuous Lyapunov models</a></td>
          <td>
            
              Gherardo Varando,
            
              N. Hansen,
            
          </td>
          <td>2020-05-21</td>
          <td>None</td>
          <td>9</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b5a29e6fc8dea483e7c7ff1091e8c15fe9058ff9" target='_blank'>Exploring Chemical and Conformational Spaces by Batch Mode Deep Active Learning</a></td>
          <td>
            
              Viktor Zaverkin,
            
              David Holzmüller,
            
              Ingo Steinwart,
            
              Johannes Kästner,
            
          </td>
          <td>None</td>
          <td>Digital Discovery</td>
          <td>15</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/43b2f986dfa28579fe24b37d4b4534b8dc1552b9" target='_blank'>Capturing Protein-Ligand Recognition Pathways in Coarse-grained Simulation</a></td>
          <td>
            
              B. Dandekar,
            
              Jagannath Mondal,
            
          </td>
          <td>2019-12-08</td>
          <td>bioRxiv</td>
          <td>21</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3fcb9b705c27dc8b062c255ba59940c4e8bd15ca" target='_blank'>Set-free Markov state model building.</a></td>
          <td>
            
              Marcus Weber,
            
              K. Fackeldey,
            
              C. Schütte,
            
          </td>
          <td>2017-03-31</td>
          <td>The Journal of chemical physics</td>
          <td>18</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/76bbb8ea587008ec5f7db32524fa0e707ac6a90f" target='_blank'>Predicting the Conformational Variability of Abl Tyrosine Kinase using Molecular Dynamics Simulations and Markov State Models.</a></td>
          <td>
            
              Yilin Meng,
            
              Cen Gao,
            
              D. Clawson,
            
              S. Atwell,
            
              M. Russell,
            
              M. Vieth,
            
              B. Roux,
            
          </td>
          <td>2018-02-23</td>
          <td>Journal of chemical theory and computation</td>
          <td>39</td>
          <td>117</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d7bf97418bd22f4b3444bb0d364e4e6cf00d3165" target='_blank'>A general deep learning framework for network reconstruction and dynamics learning</a></td>
          <td>
            
              Zhang Zhang,
            
              Yi Zhao,
            
              Jing Liu,
            
              Shuo Wang,
            
              Ruyi Tao,
            
              Ruyue Xin,
            
              Jiang Zhang,
            
          </td>
          <td>2018-12-30</td>
          <td>Applied Network Science</td>
          <td>47</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2582151d5accc2521974c0f4afcbfe50b06338f0" target='_blank'>Deep Hidden Physics Modeling of Cell Signaling Networks</a></td>
          <td>
            
              M. Seeger,
            
              J. Longden,
            
              E. Klipp,
            
              R. Linding,
            
          </td>
          <td>2021-06-14</td>
          <td>Current Genomics</td>
          <td>1</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e6210a777d7d38506ae6fd3236edf1c105fa8cf3" target='_blank'>An Integrated Deep Learning and Molecular Dynamics Simulation-Based Screening Pipeline Identifies Inhibitors of a New Cancer Drug Target TIPE2</a></td>
          <td>
            
              Haiping Zhang,
            
              Junxin Li,
            
              K. Saravanan,
            
              Hao Wu,
            
              Zhichao Wang,
            
              Du Wu,
            
              Yanjie Wei,
            
              Z. Lu,
            
              Youhai H. Chen,
            
              X. Wan,
            
              Yi Pan,
            
          </td>
          <td>2021-11-23</td>
          <td>Frontiers in Pharmacology</td>
          <td>16</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/66035191b72d4c74d789c167306e8f1ef670873c" target='_blank'>Multiscale Enhanced Sampling Using Machine Learning</a></td>
          <td>
            
              K. Moritsugu,
            
          </td>
          <td>2021-10-01</td>
          <td>Life</td>
          <td>8</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d76b2ac177b43425ed403587a74dbb6a4856b3f8" target='_blank'>AI2BMD: efficient characterization of protein dynamics with ab initio accuracy</a></td>
          <td>
            
              Tong Wang,
            
              Xinheng He,
            
              Mingyu Li,
            
              Yusong Wang,
            
              Zun Wang,
            
              Shaoning Li,
            
              Bin Shao,
            
              Tie-Yan Liu,
            
          </td>
          <td>2023-07-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/65e9bcf8a83ecf6ede4203f3af895245852aee37" target='_blank'>Elucidating Ligand-Modulated Conformational Landscape of GPCRs Using Cloud-Computing Approaches.</a></td>
          <td>
            
              D. Shukla,
            
              M. Lawrenz,
            
              V. Pande,
            
          </td>
          <td>None</td>
          <td>Methods in enzymology</td>
          <td>10</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8c8528e5005f6ac55dfbc6408bff395b0186c6d4" target='_blank'>M2 internship (+ PhD) “Eﬃcient computation of linear response of nonequilibrium stochastic dynamics”</a></td>
          <td>
            
              G. Stoltz,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c5bc32fe955a8b86eda53d7c4fda2c294c39c4c1" target='_blank'>Stochastic variational inference for hidden Markov models</a></td>
          <td>
            
              N. Foti,
            
              Jason Xu,
            
              Dillon Laird,
            
              E. Fox,
            
          </td>
          <td>2014-11-06</td>
          <td>None</td>
          <td>88</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b418bc0d7ba63f948050e15de8d362de4db2580e" target='_blank'>Stochastic embeddings of dynamical phenomena through variational autoencoders</a></td>
          <td>
            
              C. A. García,
            
              P. Félix,
            
              J. Presedo,
            
              A. Otero,
            
          </td>
          <td>2020-10-13</td>
          <td>J. Comput. Phys.</td>
          <td>1</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fd310b582bef058ea7f450e8c60e9b4f57d963fa" target='_blank'>Nearly reducible finite Markov chains: Theory and algorithms.</a></td>
          <td>
            
              Daniel J. Sharpe,
            
              D. Wales,
            
          </td>
          <td>2021-10-14</td>
          <td>The Journal of chemical physics</td>
          <td>13</td>
          <td>76</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/de279b452b98d27e53e34bde08bfcc0e213f7e5a" target='_blank'>Local Kernel Regression and Neural Network Approaches to the Conformational Landscapes of Oligopeptides</a></td>
          <td>
            
              Raimon Fabregat,
            
              Alberto Fabrizio,
            
              Edgar A. Engel,
            
              Benjamin Meyer,
            
              V. Jurásková,
            
              M. Ceriotti,
            
              C. Corminboeuf,
            
          </td>
          <td>2022-02-18</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>6</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b7bb6aa4a3f2e087542760226fdd93072420ef69" target='_blank'>LS-MolGen: Ligand-and-Structure Dual-Driven Deep Reinforcement Learning for Target-Specific Molecular Generation Improves Binding Affinity and Novelty</a></td>
          <td>
            
              Song-bo Li,
            
              Chao Hu,
            
              Song Ke,
            
              Chenxing Yang,
            
              Jun Chen,
            
              Y. Xiong,
            
              H. Liu,
            
              Liang Hong,
            
          </td>
          <td>2023-06-21</td>
          <td>Journal of chemical information and modeling</td>
          <td>1</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1c92eff274561620931cf10d45597f9e09dcc91d" target='_blank'>Determination of Markov Model Dynamics from Equilibrium Data Snapshots</a></td>
          <td>
            
              Austen Mazenko,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5864926d0af9ddafb8b036a70f74667b4643eeeb" target='_blank'>Encoding protein dynamic information in graph representation for functional residue identification</a></td>
          <td>
            
              Chiang Yuan,
            
              Weiwei Hui,
            
              Shu-Wei Chang,
            
          </td>
          <td>2021-12-15</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f0e541a19452201f2e53652237826f4c10458146" target='_blank'>In-Pocket 3D Graphs Enhance Ligand-Target Compatibility in Generative Small-Molecule Creation</a></td>
          <td>
            
              Seung-gu Kang,
            
              J. Weber,
            
              Joseph A. Morrone,
            
              Leili Zhang,
            
              T. Huynh,
            
              W. Cornell,
            
          </td>
          <td>2022-04-05</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d0977d87c2fd7017cded2584ef82fb9cf0e6a293" target='_blank'>Arbitrarily accurate, nonparametric coarse graining with Markov renewal processes and the Mori–Zwanzig formulation</a></td>
          <td>
            
              D. Aristoff,
            
              Mats Johnson,
            
              D. Perez,
            
          </td>
          <td>2023-05-31</td>
          <td>AIP Advances</td>
          <td>1</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ec0624e0edfa931e7f36e85961e3598ee34d14b9" target='_blank'>Unraveling hidden interactions in complex systems with deep learning</a></td>
          <td>
            
              Seungwoong Ha,
            
              Hawoong Jeong,
            
          </td>
          <td>2020-12-21</td>
          <td>Scientific Reports</td>
          <td>19</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/24faddd8ef452e190b30210e4cc9f2277787da12" target='_blank'>Automatic Feature Selection in Markov State Models Using Genetic Algorithm</a></td>
          <td>
            
              Qihua Chen,
            
              Jiangyan Feng,
            
              S. Mittal,
            
              D. Shukla,
            
          </td>
          <td>2018-06-01</td>
          <td>The Journal of Computational Science Education</td>
          <td>6</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/17a7aa0741d5db0627b038754220552a930c7252" target='_blank'>Thermodynamically Optimized Machine-Learned Reaction Coordinates for Hydrophobic Ligand Dissociation.</a></td>
          <td>
            
              E. Beyerle,
            
              P. Tiwary,
            
          </td>
          <td>2023-10-05</td>
          <td>The journal of physical chemistry. B</td>
          <td>2</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6a6998abf867bcfe183fa22e69bd2452c6a14db9" target='_blank'>Biomolecular transitions: efficient computation of pathways, free energies, and rates</a></td>
          <td>
            
              D. Bhatt,
            
              I. Bahar,
            
          </td>
          <td>2011-09-04</td>
          <td>arXiv: Biological Physics</td>
          <td>0</td>
          <td>80</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1a6f965fcbfb9695de196591b54e5cd53dead490" target='_blank'>A Suite of Advanced Tutorials for the WESTPA 2.0 Rare-Events Sampling Software [Article v0.1]</a></td>
          <td>
            
              Anthony T. Bogetti,
            
              Jeremy M. G. Leung,
            
              John D. Russo,
            
              She Zhang,
            
              Jeff P. Thompson,
            
              A. S. Saglam,
            
              Dhiman Ray,
            
              Rhea C. Abraham,
            
              J. Faeder,
            
              I. Andricioaei,
            
              J. Adelman,
            
              M. Zwier,
            
              David N. LeBard,
            
              D. Zuckerman,
            
              L. Chong,
            
          </td>
          <td>2022-10-07</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/182afd2cfc31e8d283c86228c03da091fc2ff067" target='_blank'>Machine learning of an implicit solvent for dynamic Monte Carlo simulations.</a></td>
          <td>
            
              Ankush Checkervarty,
            
              J. Sommer,
            
              M. Werner,
            
          </td>
          <td>2023-01-24</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ac4d8b8fc34b31692c9f53b60e84fde14728854c" target='_blank'>Deep Learning Method for Computing Committor Functions with Adaptive Sampling</a></td>
          <td>
            
              Bo Lin,
            
              Weiqing Ren,
            
          </td>
          <td>2024-04-09</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/54783aeec96d075fc03fd256ca9d3e3c0372906a" target='_blank'>MSM/RD: Coupling Markov state models of molecular kinetics with reaction-diffusion simulations.</a></td>
          <td>
            
              Manuel Dibak,
            
              Mauricio J. del Razo,
            
              David de Sancho,
            
              C. Schütte,
            
              F. Noé,
            
          </td>
          <td>2017-12-21</td>
          <td>The Journal of chemical physics</td>
          <td>27</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/500513997028c264c661f06493b1091ceb2cd1ea" target='_blank'>Geometric learning of the conformational dynamics of molecules using dynamic graph neural networks</a></td>
          <td>
            
              Michael Ashby,
            
              Jenna A. Bilbrey,
            
          </td>
          <td>2021-06-24</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe19179c54d1abcb79eca24a765214e910915e44" target='_blank'>Expectation maximized molecular dynamics: Toward efficient learning of rarely sampled features in free energy surfaces from unbiased simulations.</a></td>
          <td>
            
              Pallab Dutta,
            
              Neelanjana Sengupta,
            
          </td>
          <td>2020-10-15</td>
          <td>The Journal of chemical physics</td>
          <td>3</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6c3123c197ae144fce79b6c008ad2de45863af4b" target='_blank'>Long-Short-Range Message-Passing: A Physics-Informed Framework to Capture Non-Local Interaction for Scalable Molecular Dynamics Simulation</a></td>
          <td>
            
              Yunyang Li,
            
              Yusong Wang,
            
              Lin Huang,
            
              Hang Yang,
            
              Xinran Wei,
            
              Jia Zhang,
            
              Tong Wang,
            
              Zun Wang,
            
              Bin Shao,
            
              Tie-Yan Liu,
            
          </td>
          <td>2023-04-26</td>
          <td>None</td>
          <td>4</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dcd50d82995b361509b5c01f31a9e864dd167ac4" target='_blank'>Exploring protein–ligand binding affinity prediction with electron density-based geometric deep learning</a></td>
          <td>
            
              Clemens Isert,
            
              Kenneth Atz,
            
              Sereina Riniker,
            
              Gisbert Schneider,
            
          </td>
          <td>2024-01-31</td>
          <td>RSC Advances</td>
          <td>4</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/75384ccf628a986ed357d743ea9462c6567e92bd" target='_blank'>Multimodal Protein Representation Learning and Target-aware Variational Auto-encoders for Protein-binding Ligand Generation</a></td>
          <td>
            
              Nhat-Khang Ngô,
            
              T. Hy,
            
          </td>
          <td>2024-04-15</td>
          <td>Machine Learning: Science and Technology</td>
          <td>1</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/37488634c91507955c5df9e4f11338b02e5de586" target='_blank'>Modeling and enhanced sampling of molecular systems with smooth and nonlinear data-driven collective variables.</a></td>
          <td>
            
              Behrooz Hashemian,
            
              Daniel Millán,
            
              M. Arroyo,
            
          </td>
          <td>2013-12-02</td>
          <td>The Journal of chemical physics</td>
          <td>52</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bf1e445071fe32e78abf34f1b6fb89a16c2f7464" target='_blank'>Markov state models from short non-equilibrium simulations—Analysis and correction of estimation bias</a></td>
          <td>
            
              Feliks Nuske,
            
              Hao Wu,
            
              Jan-Hendrik Prinz,
            
              C. Wehmeyer,
            
              C. Clementi,
            
              Frank No'e,
            
          </td>
          <td>2017-01-06</td>
          <td>Journal of Chemical Physics</td>
          <td>52</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a7336fca259c75b8ba03d52ecedf9de46d112ca6" target='_blank'>FlexVDW: A machine learning approach to account for protein flexibility in ligand docking</a></td>
          <td>
            
              Patricia Suriana,
            
              Joseph M. Paggi,
            
              R. Dror,
            
          </td>
          <td>2023-03-20</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>78</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/91fc9a5d13cb3ef058519c9adfc8b4e6c526f0ee" target='_blank'>CHAPTER 22:Machine-learning Routes to Dynamics, Thermodynamics and Work Cycles of Biological Nanomachines</a></td>
          <td>
            
              A. Ourmazd,
            
          </td>
          <td>2017-08-11</td>
          <td></td>
          <td>2</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2658595d06055d1951e4a8c90d9ce8bb754e5163" target='_blank'>Characterizing protein-ligand binding using atomistic simulation and machine learning: Application to drug resistance in HIV-1 protease.</a></td>
          <td>
            
              T. Whitfield,
            
              Debra A. Ragland,
            
              K. Zeldovich,
            
              C. Schiffer,
            
          </td>
          <td>2019-12-26</td>
          <td>Journal of chemical theory and computation</td>
          <td>16</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5ef9f2c027728ee556f0cee5856b6489637f372a" target='_blank'>Protein-Ligand Binding Site Prediction and De Novo Ligand Generation from Cryo-EM Maps</a></td>
          <td>
            
              Chunyang Lu,
            
              Kaustav Mitra,
            
              Kiran Mitra,
            
              Hanze Meng,
            
              Shane T. Rich-New,
            
              Fengbin Wang,
            
              Dong Si,
            
          </td>
          <td>2023-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/528d49cac01600cc4d53e03bbe09d64671923baf" target='_blank'>Harmonic Prior Self-conditioned Flow Matching for Multi-Ligand Docking and Binding Site Design</a></td>
          <td>
            
              Hannes Stark,
            
              Bowen Jing,
            
              R. Barzilay,
            
              T. Jaakkola,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3d2cd2e98b8c0ad116f74af97b8583a17b803b5f" target='_blank'>Designing a set of criteria for evaluating artificial neural networks trained with physics-based data to replicate molecular dynamics and other particle method trajectories</a></td>
          <td>
            
              Alessio Alexiadis,
            
          </td>
          <td>2024-04-10</td>
          <td>Frontiers in Nanotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ad56584b822792e38f2d70f526afe2985c70699" target='_blank'>3D molecular generative framework for interaction-guided drug design</a></td>
          <td>
            
              Wonho Zhung,
            
              Hyeongwoo Kim,
            
              Woo Youn Kim,
            
          </td>
          <td>2024-03-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/13fe74329a30372f220d9749e54918e2d3ee81d8" target='_blank'>Effective equations in complex systems: from Langevin to machine learning</a></td>
          <td>
            
              A. Vulpiani,
            
              Marco Baldovin,
            
          </td>
          <td>2019-11-19</td>
          <td>Journal of Statistical Mechanics: Theory and Experiment</td>
          <td>6</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cbba6806a864086b5ddf56a0db7462eab2271f30" target='_blank'>Improved Protein-ligand Binding Affinity Prediction with Structure-Based Deep Fusion Inference</a></td>
          <td>
            
              Derek Jones,
            
              Hyojin Kim,
            
              Xiaohua Zhang,
            
              A. Zemla,
            
              Garrett A. Stevenson,
            
              W. D. Bennett,
            
              D. Kirshner,
            
              Sergio E. Wong,
            
              F. Lightstone,
            
              Jonathan E. Allen,
            
          </td>
          <td>2020-05-17</td>
          <td>Journal of chemical information and modeling</td>
          <td>144</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/49da7e115031ddf0a44b730e99f737fb2fc9fafa" target='_blank'>Discovering Protein Conformational Flexibility Through Artificial Intelligence Aided Molecular Dynamics.</a></td>
          <td>
            
              Zachary Smith,
            
              P. Ravindra,
            
              Yihang Wang,
            
              Rory Cooley,
            
              P. Tiwary,
            
          </td>
          <td>2020-08-25</td>
          <td>The journal of physical chemistry. B</td>
          <td>20</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f16f1126053423cfb892ee3178a2b293685875ed" target='_blank'>Exploring Successful Parameter Region for Coarse-Grained Simulation of Biomolecules by Bayesian Optimization and Active Learning</a></td>
          <td>
            
              R. Kanada,
            
              Atsushi Tokuhisa,
            
              Koji Tsuda,
            
              Yasushi Okuno,
            
              Kei Terayama,
            
          </td>
          <td>2020-03-01</td>
          <td>Biomolecules</td>
          <td>3</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/31a248d6a7ea2f0d357ab23e97fdf1e8779130da" target='_blank'>Factorization in molecular modeling and belief propagation algorithms.</a></td>
          <td>
            
              P. Tian,
            
          </td>
          <td>2021-10-07</td>
          <td>Mathematical biosciences and engineering : MBE</td>
          <td>0</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/60722f4e9fc9a1028957fb3bcacd0b6b09801ad8" target='_blank'>Deep learning reveals hidden interactions in complex systems</a></td>
          <td>
            
              Seungwoong Ha,
            
              Hawoong Jeong,
            
          </td>
          <td>2020-01-03</td>
          <td>arXiv: Statistical Mechanics</td>
          <td>3</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/36a10aecb4bc960006b5d3f6346577c4d8547e8a" target='_blank'>HydraScreen: A Generalizable Structure-Based Deep Learning Approach to Drug Discovery</a></td>
          <td>
            
              Alvaro Prat,
            
              Hisham Abdel-Aty,
            
              Gintautas Kamuntavicius,
            
              Tanya Paquet,
            
              P. Norvaisas,
            
              Piero Gasparotto,
            
              Roy Tal,
            
          </td>
          <td>2023-09-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/047922dcebe9ee9f11348e10b0a54594e0425d37" target='_blank'>Phase diagrams of polymer-containing liquid mixtures with a theory-embedded neural network</a></td>
          <td>
            
              I. Nakamura,
            
          </td>
          <td>2019-08-16</td>
          <td>New Journal of Physics</td>
          <td>7</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5ba2cacf5cf13544964a7dc146d531cb8328ce6d" target='_blank'>Alecsandre Sauvé-Lacoursière</a></td>
          <td>
            
              Normand Mousseau,
            
          </td>
          <td>2018-06-06</td>
          <td></td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d4705791a22794c2a0df05da139a81647d518f71" target='_blank'>Using Neural Architectures to Model Complex Dynamical Systems</a></td>
          <td>
            
              N. Gabriel,
            
              Neil F Johnson,
            
          </td>
          <td>None</td>
          <td>Adv. Artif. Intell. Mach. Learn.</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/39eee116bcc6590ac081ec2b080d5fa51aa8ee05" target='_blank'>Towards Automated Statistical Physics : Data-driven Modeling of Complex Systems with Deep Learning</a></td>
          <td>
            
              Seungwoong Ha,
            
              Hawoong Jeong,
            
          </td>
          <td>2020-01-03</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/790e0a160fff355ce2fbb51beb4362dfcc58a830" target='_blank'>Machine-learning methods for ligand-protein molecular docking.</a></td>
          <td>
            
              Kevin Crampon,
            
              Alexis Giorkallos,
            
              Myrtille Deldossi,
            
              S. Baud,
            
              Luiz Angelo Steffenel,
            
          </td>
          <td>2021-09-01</td>
          <td>Drug discovery today</td>
          <td>99</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9cc6aeff7ada5c284f047cad292ee6a41b77fbd7" target='_blank'>Machine Learning Nucleation Collective Variables with Graph Neural Networks</a></td>
          <td>
            
              Florian M Dietrich,
            
              Xavier R Advincula,
            
              Gianpaolo Gobbo,
            
              Michael A. Bellucci,
            
              M. Salvalaglio,
            
          </td>
          <td>2023-10-25</td>
          <td>Journal of Chemical Theory and Computation</td>
          <td>3</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3d6e10fc047f168464d498729d7ff5a02cd703b0" target='_blank'>Statistical and machine learning approaches to predicting protein-ligand interactions.</a></td>
          <td>
            
              Lucy J. Colwell,
            
          </td>
          <td>2018-04-01</td>
          <td>Current opinion in structural biology</td>
          <td>53</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2bec1b31a54755c99d2b9f564a89a766d275b4fc" target='_blank'>Deciphering the co-evolutionary dynamics of L2 β-lactamases via Deep learning</a></td>
          <td>
            
              Yu Zhu,
            
              Jing Gu,
            
              ,
            
              A. W. E. Chan,
            
              M. Mojica,
            
              A. Hujer,
            
              R. Bonomo,
            
              S. Haider,
            
          </td>
          <td>2024-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8ec4b3934b6f6de10a59caca2aed959a0f41f716" target='_blank'>Analysis of Kinetic Models for Label Switching and Stochastic Gradient Descent</a></td>
          <td>
            
              M. Burger,
            
              Alex Rossi,
            
          </td>
          <td>2022-07-01</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/333e250797f9b04c00c3fd88fd041bc720ec9bb6" target='_blank'>Single Molecule Analysis Research Tool (SMART): An Integrated Approach for Analyzing Single Molecule Data</a></td>
          <td>
            
              Max Greenfeld,
            
              Dmitri S. Pavlichin,
            
              H. Mabuchi,
            
              D. Herschlag,
            
          </td>
          <td>2012-02-20</td>
          <td>PLoS ONE</td>
          <td>81</td>
          <td>73</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/04196d0f89b387a202a7f66e72afd0219b7d4f30" target='_blank'>Integrating physics in deep learning algorithms: a force field as a PyTorch module</a></td>
          <td>
            
              Gabriele Orlando,
            
              Luis Serrano,
            
              J. Schymkowitz,
            
              F. Rousseau,
            
          </td>
          <td>2023-01-16</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/341bb656ac46e4927d63a1e3c7923ba04962cb66" target='_blank'>Accelerating Stochastic Simulation with Spatiotemporal Neural Processes</a></td>
          <td>
            
              D. Wu,
            
              Matteo Chinazzi,
            
              A. Vespignani,
            
              Yi-An Ma,
            
              Rose Yu,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a27e0bfd608a11ff40d59b41ae6e27414863a719" target='_blank'>Neural network learns physical rules for copolymer translocation through amphiphilic barriers</a></td>
          <td>
            
              M. Werner,
            
              Yachong Guo,
            
              V. Baulin,
            
          </td>
          <td>2019-04-30</td>
          <td>npj Computational Materials</td>
          <td>6</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dc21978ef59ffa63d0f7754bdf7d360c91fb4b8d" target='_blank'>Ward Clustering Improves Cross-Validated Markov State Models of Protein Folding.</a></td>
          <td>
            
              B. Husic,
            
              V. Pande,
            
          </td>
          <td>2017-02-17</td>
          <td>Journal of chemical theory and computation</td>
          <td>40</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/122a1743b8481874cbab7238ccd6fd7bbea6ebc0" target='_blank'>Str2Str: A Score-based Framework for Zero-shot Protein Conformation Sampling</a></td>
          <td>
            
              Jiarui Lu,
            
              Bozitao Zhong,
            
              Zuobai Zhang,
            
              Jian Tang,
            
          </td>
          <td>2023-06-05</td>
          <td>None</td>
          <td>5</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/360054585b0c4cf0b1a7cba955544d77401e4d39" target='_blank'>Active Learning the Potential Energy Landscape for Water Clusters from Sparse Training Data</a></td>
          <td>
            
              T. Loeffler,
            
              T. Patra,
            
              Henry Chan,
            
              M. Cherukara,
            
              S. Sankaranarayanan,
            
          </td>
          <td>2020-01-31</td>
          <td>Journal of Physical Chemistry C</td>
          <td>22</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/83ea81255d3ad1b92b201662ac1e71e652e3f309" target='_blank'>Analysing ill-conditioned Markov chains</a></td>
          <td>
            
              Esmae J. Woods,
            
              Deepti Kannan,
            
              Daniel J. Sharpe,
            
              T. Swinburne,
            
              D. Wales,
            
          </td>
          <td>2023-05-22</td>
          <td>Philosophical transactions. Series A, Mathematical, physical, and engineering sciences</td>
          <td>2</td>
          <td>76</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3f652b9076bcce684e28275bc7c5e27feda96926" target='_blank'>An adaptive solution to the chemical master equation using quantized tensor trains with sliding windows</a></td>
          <td>
            
              Trang Dinh,
            
              R. Sidje,
            
          </td>
          <td>2020-07-01</td>
          <td>Physical Biology</td>
          <td>7</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ef5fceee2925cb8441bf1de100b67a33eeeef3a3" target='_blank'>FABind: Fast and Accurate Protein-Ligand Binding</a></td>
          <td>
            
              Qizhi Pei,
            
              Kaiyuan Gao,
            
              Lijun Wu,
            
              Jinhua Zhu,
            
              Yingce Xia,
            
              Shufang Xie,
            
              Tao Qin,
            
              Kun He,
            
              Tie-Yan Liu,
            
              Rui Yan,
            
          </td>
          <td>2023-10-10</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1352c0f369e3f9acef9f5378d2ce972c21bd8edf" target='_blank'>DEELIG: A Deep Learning-based approach to predict protein-ligand binding affinity</a></td>
          <td>
            
              Asad Ahmed,
            
              Bhavika Mam,
            
              R. Sowdhamini,
            
          </td>
          <td>2020-09-28</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aa105dff412c040a857df54200cfb5e3f1fab4af" target='_blank'>Machine Learning in Chemical Dynamics</a></td>
          <td>
            
              Rupayan Biswas,
            
              Richa Rashmi,
            
              U. Lourderaj,
            
          </td>
          <td>2020-01-01</td>
          <td>Resonance</td>
          <td>4</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/576bcad1ee5a1f194a605c0d705111c6ef38b092" target='_blank'>Deep-LASI: deep-learning assisted, single-molecule imaging analysis of multi-color DNA origami structures</a></td>
          <td>
            
              Simon Wanninger,
            
              Pooyeh Asadiatouei,
            
              J. Bohlen,
            
              Clemens-Bässem Salem,
            
              P. Tinnefeld,
            
              E. Ploetz,
            
              D. Lamb,
            
          </td>
          <td>2023-02-03</td>
          <td>Nature Communications</td>
          <td>3</td>
          <td>58</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/255375123b94d949d0dca02bf04d8a9c7d862def" target='_blank'>Conserve Water: A Method for the Analysis of Solvent in Molecular Dynamics.</a></td>
          <td>
            
              M. Harrigan,
            
              D. Shukla,
            
              V. Pande,
            
          </td>
          <td>2015-02-27</td>
          <td>Journal of chemical theory and computation</td>
          <td>16</td>
          <td>102</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6c1f1f6c14981c593fd5d86a00f6900485adb2db" target='_blank'>DEELIG: A Deep Learning Approach to Predict Protein-Ligand Binding Affinity</a></td>
          <td>
            
              Asad Ahmed,
            
              Bhavika Mam,
            
              R. Sowdhamini,
            
          </td>
          <td>2021-01-01</td>
          <td>Bioinformatics and Biology Insights</td>
          <td>33</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2b0047e4090870475b8b3249c84e91cb3812aa36" target='_blank'>Computational modeling aids in linking structure, dynamics, and function of neural systems: A commentary on Wright, J.J., & Bourke, P.D. "The growth of cognition: Free energy minimization and the embryogenesis of cortical computation", Physics of Life Reviews.</a></td>
          <td>
            
              H. Liljenström,
            
          </td>
          <td>2020-11-16</td>
          <td>Physics of life reviews</td>
          <td>1</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2f82c8bf13da6f07a53c70ccff46eca481f971a4" target='_blank'>End-to-End Protein Normal Mode Frequency Predictions Using Language and Graph Models and Application to Sonification.</a></td>
          <td>
            
              Yiwen Hu,
            
              M. Buehler,
            
          </td>
          <td>2022-11-23</td>
          <td>ACS nano</td>
          <td>11</td>
          <td>104</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ff3a543e2fca4940eac26245b059436da3ce9dcf" target='_blank'>Discovering loop conformational flexibility in T4 lysozyme mutants through artificial intelligence aided molecular dynamics</a></td>
          <td>
            
              Zachary Smith,
            
              P. Ravindra,
            
              Yihang Wang,
            
              Rory Cooley,
            
              P. Tiwary,
            
          </td>
          <td>2020-04-09</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/387e21ac1d0add4be9494ddbcc9a77296f3a2c32" target='_blank'>Machine Learning Models for Predicting Molecular Diffusion in Metal–Organic Frameworks Accounting for the Impact of Framework Flexibility</a></td>
          <td>
            
              Yuhan Yang,
            
              Zhenzi Yu,
            
              D. Sholl,
            
          </td>
          <td>2023-11-22</td>
          <td>Chemistry of Materials</td>
          <td>0</td>
          <td>87</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/861e72f3ce6335d1f22125895f478a50c1434f14" target='_blank'>Flexible Protein-Protein Docking with a Multi-Track Iterative Transformer</a></td>
          <td>
            
              Lee-Shin Chu,
            
              Jeffrey A. Ruffolo,
            
              Ameya Harmalkar,
            
              Jeffrey J. Gray,
            
          </td>
          <td>2023-07-01</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d81e78b507b47303ca94122ace6818e4a184b647" target='_blank'>Exploring a Sub-optimal Hidden Markov Model Sampling Approach for De Novo Peptide Structure Modeling</a></td>
          <td>
            
              P. Thévenet,
            
              P. Tufféry,
            
          </td>
          <td>2014-03-03</td>
          <td>None</td>
          <td>0</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3403956f4fc402fe2ae21058689dfb8710df9d5c" target='_blank'>In silico evolution of protein binders with deep learning models for structure prediction and sequence design</a></td>
          <td>
            
              Odessa J Goudy,
            
              Amrita Nallathambi,
            
              Tomoaki Kinjo,
            
              Nicholas Z Randolph,
            
              B. Kuhlman,
            
          </td>
          <td>2023-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7b5c9830f6b22848cdff6ce5522cd4384f7ec317" target='_blank'>Deep neural network analysis models for complex random telegraph signals</a></td>
          <td>
            
              Marcel Robitaille,
            
              HeeBong Yang,
            
              Lu Wang,
            
              Bowen Deng,
            
              Na Young Kim,
            
          </td>
          <td>2023-06-27</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fa8e09205060f93b8a72dcd9bed7aeff8a9ae416" target='_blank'>A General Deep Learning Framework for Structure and Dynamics Reconstruction from Time Series Data</a></td>
          <td>
            
              Zhang Zhang,
            
              ,
            
              Shuo Wang,
            
              Ruyue Xin,
            
              Jiang Zhang,
            
          </td>
          <td>2018-12-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58c00995549392fe03fd495de30e5f022dc1065b" target='_blank'>Hybrid computational methods combining experimental information with molecular dynamics.</a></td>
          <td>
            
              Arup Mondal,
            
              Stefan Lenz,
            
              J. MacCallum,
            
              Alberto Perez,
            
          </td>
          <td>2023-05-22</td>
          <td>Current opinion in structural biology</td>
          <td>3</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b8c9731b2461d44861a1972d387c9db4fb7fcc8e" target='_blank'>Binding Modes of Ligands Using Enhanced Sampling (BLUES): Rapid Decorrelation of Ligand Binding Modes via Nonequilibrium Candidate Monte Carlo.</a></td>
          <td>
            
              S. Gill,
            
              N. Lim,
            
              Patrick B. Grinaway,
            
              Ariën S. Rustenburg,
            
              Josh Fass,
            
              Gregory A Ross,
            
              J. Chodera,
            
              D. Mobley,
            
          </td>
          <td>2018-02-27</td>
          <td>The journal of physical chemistry. B</td>
          <td>10</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/290d40f4d8fd906dc4a41275124af6e04a5dc6a7" target='_blank'>Extensive Study of Multiple Deep Neural Networks for Complex Random Telegraph Signals</a></td>
          <td>
            
              Marcel Robitaille,
            
              HeeBong Yang,
            
              Lu Wang,
            
              Na Young Kim,
            
          </td>
          <td>2022-05-31</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a9ec03dbe702f6909acd1f1f14a3395d0141043b" target='_blank'>Generative Models for Graph-Based Protein Design</a></td>
          <td>
            
              John Ingraham,
            
              Vikas K. Garg,
            
              R. Barzilay,
            
              T. Jaakkola,
            
          </td>
          <td>2019-03-27</td>
          <td>None</td>
          <td>370</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f2c01a216fa2acbfb8774ae9da553b85aad5565b" target='_blank'>Monte Carlo, harmonic approximation, and coarse-graining approaches for enhanced sampling of biomolecular structure</a></td>
          <td>
            
              Tamar Schlick,
            
          </td>
          <td>2009-06-29</td>
          <td>F1000 Biology Reports</td>
          <td>13</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b068c83aa1dc6de42005e0bed649ac205f0fa30b" target='_blank'>Application of artificial neural networks for modeling of electronic excitation dynamics in 2D lattice: Direct and inverse problems</a></td>
          <td>
            
              Pranas Juknevicius,
            
              J. Chmeliov,
            
              L. Valkunas,
            
              A. Gelžinis,
            
          </td>
          <td>2023-03-01</td>
          <td>AIP Advances</td>
          <td>0</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e11ffa44708ce3e7e179e6d23e352c82d921c3f6" target='_blank'>Unsupervised machine learning for supercooled liquids</a></td>
          <td>
            
              Yunrui Qiu,
            
              Inhyuk Jang,
            
              Xuhui Huang,
            
              Arun Yethiraj,
            
          </td>
          <td>2024-04-06</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3d9147a6b8d05e8ce25a11e446159c3484a39027" target='_blank'>Data driven flexible backbone protein design</a></td>
          <td>
            
              Mark G. F. Sun,
            
              Philip M. Kim,
            
          </td>
          <td>2017-08-01</td>
          <td>PLoS Computational Biology</td>
          <td>12</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3498fc4c26f301125ff93d1e2f1e92caeec2472e" target='_blank'>Solving the Side-Chain Packing Arrangement of Proteins from Reinforcement Learned Stochastic Decision Making</a></td>
          <td>
            
              Chandrajit L. Bajaj,
            
              Conrad Li,
            
              Minh Hoai Nguyen,
            
          </td>
          <td>2022-12-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4df2f4030c0515657d553a5a79bf7efbe204772b" target='_blank'>Learnt representations of proteins can be used for accurate prediction of small molecule binding sites on experimentally determined and predicted protein structures</a></td>
          <td>
            
              A. Carbery,
            
              Martin Buttenschoen,
            
              R. Skyner,
            
              Frank von Delft,
            
              Charlotte M. Deane,
            
          </td>
          <td>2023-09-08</td>
          <td>Journal of Cheminformatics</td>
          <td>1</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/037f82fa06421cf3a054dac5f97fb002dc0b7f4f" target='_blank'>Learning effective stochastic differential equations from microscopic simulations: combining stochastic numerics and deep learning</a></td>
          <td>
            
              Felix Dietrich,
            
              A. Makeev,
            
              George A. Kevrekidis,
            
              N. Evangelou,
            
              Tom S. Bertalan,
            
              S. Reich,
            
              I. Kevrekidis,
            
          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>17</td>
          <td>84</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1068197434a8e303802bb090be7284b7bba83a96" target='_blank'>WESTPA 2.0: High-performance upgrades for weighted ensemble simulations and analysis of longer-timescale applications</a></td>
          <td>
            
              John D. Russo,
            
              She Zhang,
            
              Jeremy M. G. Leung,
            
              Anthony T. Bogetti,
            
              Jeff P. Thompson,
            
              A. DeGrave,
            
              Paul A. Torrillo,
            
              A. Pratt,
            
              Kim F. Wong,
            
              Junchao Xia,
            
              J. Copperman,
            
              J. Adelman,
            
              M. Zwier,
            
              David N. LeBard,
            
              D. Zuckerman,
            
              L. Chong,
            
          </td>
          <td>2021-12-06</td>
          <td>bioRxiv</td>
          <td>26</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/95c69570d6a80c8251a611156826f8486a74dc26" target='_blank'>Non-Markovian properties and multiscale hidden Markovian network buried in single molecule time series.</a></td>
          <td>
            
              T. Sultana,
            
              H. Takagi,
            
              M. Morimatsu,
            
              H. Teramoto,
            
              Chun-Biu Li,
            
              Y. Sako,
            
              T. Komatsuzaki,
            
          </td>
          <td>2013-12-28</td>
          <td>The Journal of chemical physics</td>
          <td>9</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/379c7e09059cda7f552ba7ec9b42235d8ed9bd72" target='_blank'>Improving molecular machine learning through adaptive subsampling with active learning</a></td>
          <td>
            
              Yujing Wen,
            
              Zhixiong Li,
            
              Yan Xiang,
            
              D. Reker,
            
          </td>
          <td>None</td>
          <td>Digital Discovery</td>
          <td>3</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b886797dc34c91f186629403d7c7e2092fc25083" target='_blank'>TANKBind: Trigonometry-Aware Neural NetworKs for Drug-Protein Binding Structure Prediction</a></td>
          <td>
            
              Wei Lu,
            
              Qifeng Wu,
            
              Jixian Zhang,
            
              Jiahua Rao,
            
              Chengtao Li,
            
              Shuangjia Zheng,
            
          </td>
          <td>2022-10-25</td>
          <td>bioRxiv</td>
          <td>80</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7a180797af78526767d535762d5c16af9ab5c267" target='_blank'>Efficient and accurate prediction of protein structures and interactions using RoseTTAFold</a></td>
          <td>
            
              M. Baek,
            
          </td>
          <td>2022-07-29</td>
          <td>Acta Crystallographica Section A Foundations and Advances</td>
          <td>0</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/300101de022314cbe70aa0712e47558594425e17" target='_blank'>Emerging frontiers in virtual drug discovery: From quantum mechanical methods to deep learning approaches.</a></td>
          <td>
            
              C. Gorgulla,
            
              A. Jayaraj,
            
              K. Fackeldey,
            
              H. Arthanari,
            
          </td>
          <td>2022-05-13</td>
          <td>Current opinion in chemical biology</td>
          <td>13</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/736598876cb9b057e7206ba04d6a9d6385a89dce" target='_blank'>Efficient Chemical Space Exploration Using Active Learning Based on Marginalized Graph Kernel: an Application for Predicting the Thermodynamic Properties of Alkanes with Molecular Simulation</a></td>
          <td>
            
              Yan Xiang,
            
              Yunhao Tang,
            
              Zheng Gong,
            
              ,
            
              Liang Wu,
            
              Guang Lin,
            
              Huai Sun,
            
          </td>
          <td>2022-09-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/230b250a4b8224c5e65ae096f427d6d953342744" target='_blank'>Modeling Molecular Dynamics from Simulations</a></td>
          <td>
            
              N. Hinrichs,
            
          </td>
          <td>2009-01-28</td>
          <td></td>
          <td>0</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e9c63cf22cd2504505a0f603a6f0af443f00bbdd" target='_blank'>Dynamics of DNA Breathing and Folding for Molecular Recognition and Computation</a></td>
          <td>
            
              G. Bonnet,
            
          </td>
          <td>None</td>
          <td></td>
          <td>2</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8019fb16bbb35f60d5d9c4052b91739e24879be3" target='_blank'>PIGNet: a physics-informed deep learning model toward generalized drug–target interaction predictions</a></td>
          <td>
            
              Seokhyun Moon,
            
              Wonho Zhung,
            
              Soojung Yang,
            
              Jaechang Lim,
            
              W. Kim,
            
          </td>
          <td>2020-08-22</td>
          <td>Chemical Science</td>
          <td>63</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7a75afea237d4bbb478aef8baa3ab9229b0d7e56" target='_blank'>Convenient Interface to Inverse Ising (ConIII): A Python package for solving maximum entropy models</a></td>
          <td>
            
              Edward D. Lee,
            
              Bryan C. Daniels,
            
          </td>
          <td>2018-01-24</td>
          <td>arXiv: Quantitative Methods</td>
          <td>2</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/de3f2b94d5928389781e9d9cee7f399350675464" target='_blank'>Certiﬁcates of many-body properties assisted by machine learning</a></td>
          <td>
            
              Borja Requena,
            
              Gorka Muñoz-Gil,
            
              M. Lewenstein,
            
              V. Dunjko,
            
              Jordi Tura i Brugués,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>93</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/30c53eb5fc0d2b957b0c6bc21fe67b44994c1ee7" target='_blank'>BiRDS - Binding Residue Detection from Protein Sequences Using Deep ResNets</a></td>
          <td>
            
              Vineeth R. Chelur,
            
              U. Priyakumar,
            
          </td>
          <td>2021-07-15</td>
          <td>Journal of chemical information and modeling</td>
          <td>7</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/44d5938e55c6f4eba3c97d9fcfae060f0735f9cd" target='_blank'>A critical perspective on Markov state model treatments of protein–protein association using coarse-grained simulations</a></td>
          <td>
            
              Ziwei He,
            
              Fabian Paul,
            
              B. Roux,
            
          </td>
          <td>2021-02-22</td>
          <td>The Journal of Chemical Physics</td>
          <td>12</td>
          <td>117</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/24110157f3247109c51cef3e0a33f65d650d86e4" target='_blank'>A micro-macro Markov chain Monte Carlo method for molecular dynamics using reaction coordinate proposals I: direct reconstruction</a></td>
          <td>
            
              H. Vandecasteele,
            
              G. Samaey,
            
          </td>
          <td>2020-02-21</td>
          <td>SIAM J. Sci. Comput.</td>
          <td>3</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3d582fdf2ced2e92832c4ea726f314cdc6def1bd" target='_blank'>Multiscale simulation of plasma flows using active learning.</a></td>
          <td>
            
              A. Diaw,
            
              K. Barros,
            
              J. Haack,
            
              Christoph Junghans,
            
              B. Keenan,
            
              Y. Li,
            
              D. Livescu,
            
              N. Lubbers,
            
              M. McKerns,
            
              R. Pavel,
            
              D. Rosenberger,
            
              I. Sagert,
            
              T. Germann,
            
          </td>
          <td>2020-08-01</td>
          <td>Physical review. E</td>
          <td>8</td>
          <td>50</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e337bbc6e92e37d088c89fcd5db06a345d3ee67b" target='_blank'>Machine learning surrogates for molecular dynamics simulations of soft materials</a></td>
          <td>
            
              J. Kadupitiya,
            
              Fanbo Sun,
            
              G. Fox,
            
              V. Jadhao,
            
          </td>
          <td>2020-03-12</td>
          <td>J. Comput. Sci.</td>
          <td>30</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/558c499a1f6b8b24fdf1ca33f5a4c1964d8be8c7" target='_blank'>DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes</a></td>
          <td>
            
              Jonas Pfab,
            
              Dong Si,
            
          </td>
          <td>2020-02-13</td>
          <td>bioRxiv</td>
          <td>7</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6d3799e070b376b5cb793b23c675f3f074293ff2" target='_blank'>PocketGen: Generating Full-Atom Ligand-Binding Protein Pockets</a></td>
          <td>
            
              Zaixin Zhang,
            
              Wanxiang Shen,
            
              Qi Liu,
            
              M. Zitnik,
            
          </td>
          <td>2024-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a01a4e7b98c7c785ff9beaf763ab722eac852b73" target='_blank'>Efficient Construction of Mesostate Networks from Molecular Dynamics Trajectories</a></td>
          <td>
            
              A. Vitalis,
            
              Amedeo Caflisch,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>10</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/70f87df008e275e1859dbd08004877e4230f6a79" target='_blank'>DeepSurf: A surface-based deep learning approach for the prediction of ligand binding sites on proteins</a></td>
          <td>
            
              Stelios K. Mylonas,
            
              A. Axenopoulos,
            
              P. Daras,
            
          </td>
          <td>2020-02-13</td>
          <td>Bioinformatics</td>
          <td>68</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3196163821908f3ba50c12bd73236abd9f239acc" target='_blank'>Exploring the computational methods for protein-ligand binding site prediction</a></td>
          <td>
            
              Jingtian Zhao,
            
              Yang Cao,
            
              Le Zhang,
            
          </td>
          <td>2020-02-17</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>99</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c606795056e1f9db0924b28c0b2080ef05511d35" target='_blank'>Accelerating Systems Biology Computation: Enhanced Sampling of Spatially Realistic Stochastic Models using the Weighted Ensemble Approach</a></td>
          <td>
            
              Rory M. Donovan,
            
          </td>
          <td>2015-01-27</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/41b9ecfed92d3f3779bbfcc63a3e3c5fb3718235" target='_blank'>Dropout Regularization in Deep Neural Networks Used in Atomic Simulations</a></td>
          <td>
            
              Ljubinka Sandjakoska,
            
          </td>
          <td>2018-10-05</td>
          <td>PROCEEDINGS OF THE 8th INTERNATIONAL CONFERENCE ON APPLIED INTERNET AND INFORMATION TECHNOLOGIES</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b66fd689144a7ee2a84da624e68e39ca7038111c" target='_blank'>Multiplex Decomposition of Non-Markovian Dynamics and the Hidden Layer Reconstruction Problem</a></td>
          <td>
            
              L. Lacasa,
            
              I. P. Mariño,
            
              J. Míguez,
            
              V. Nicosia,
            
              E. Rold'an,
            
              Ana Lisica,
            
              S. Grill,
            
              J. G'omez-Gardenes,
            
          </td>
          <td>2017-05-12</td>
          <td>Physical Review X</td>
          <td>22</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5493ee22622bba51405d9451b5fa56b6ef54c63c" target='_blank'>Reinforcement Learning Based Approach for Ligand Pose Prediction</a></td>
          <td>
            
              Justin Jose,
            
              Kritika Gupta,
            
              Ujjaini Alam,
            
              N. Jatana,
            
              Pooja Arora,
            
          </td>
          <td>2021-04-11</td>
          <td>bioRxiv</td>
          <td>7</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/663d7f9d2b4b8fcbb488fdb07a7458f734e62726" target='_blank'>Deep Learning for Flexible and Site-Specific Protein Docking and Design</a></td>
          <td>
            
              Matthew McPartlon,
            
              Jinbo Xu,
            
          </td>
          <td>2023-04-02</td>
          <td>bioRxiv</td>
          <td>8</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ade35eb1cea5e858a8bb03947a021c8be6639db9" target='_blank'>New Strategies in Conformation Dynamics</a></td>
          <td>
            
              Martina Klimm,
            
              K. Fackeldey,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/37a83637b136c20f7ffb494ac8b8065f34782286" target='_blank'>Graph ODE with Factorized Prototypes for Modeling Complicated Interacting Dynamics</a></td>
          <td>
            
              Xiao Luo,
            
              Yiyang Gu,
            
              Huiyu Jiang,
            
              Jinsheng Huang,
            
              Wei Ju,
            
              Ming Zhang,
            
              Yizhou Sun,
            
          </td>
          <td>2023-11-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9c3e49e6fb981b8c88bc19f62df6e7276eb38399" target='_blank'>Using attribution to decode binding mechanism in neural network models for chemistry</a></td>
          <td>
            
              Kevin McCloskey,
            
              Ankur Taly,
            
              Federico Monti,
            
              M. Brenner,
            
              Lucy J. Colwell,
            
          </td>
          <td>2018-11-28</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>67</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/32fd29dc21a187295f366921ff5039ef23280079" target='_blank'>Energy-conserving molecular dynamics is not energy conserving.</a></td>
          <td>
            
              Lina Zhang,
            
              Yi-Fan Hou,
            
              Fuchun Ge,
            
              Pavlo O. Dral,
            
          </td>
          <td>2023-08-22</td>
          <td>Physical chemistry chemical physics : PCCP</td>
          <td>3</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f5337217aa6867d8b6586ebc8b58a30d0dce889a" target='_blank'>Efficient sampling of high-energy states by machine learning force fields.</a></td>
          <td>
            
              W. Płaziński,
            
              A. Płazińska,
            
              Agnieszka Brzyska,
            
          </td>
          <td>2020-06-22</td>
          <td>Physical chemistry chemical physics : PCCP</td>
          <td>2</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5a14646725cea74443463f6ae9f84ffe6c833f13" target='_blank'>Preliminary results in using Deep Learning to emulate BLOB, a nuclear interaction model.</a></td>
          <td>
            
              A. Ciardiello,
            
              M. Asai,
            
              B. Caccia,
            
              G. Cirrone,
            
              M. Colonna,
            
              A. Dotti,
            
              R. Faccini,
            
              S. Giagu,
            
              A. Messina,
            
              P. Napolitani,
            
              L. Pandola,
            
              D. Wright,
            
              C. Mancini-Terracciano,
            
          </td>
          <td>2020-04-10</td>
          <td>Physica medica : PM : an international journal devoted to the applications of physics to medicine and biology : official journal of the Italian Association of Biomedical Physics</td>
          <td>5</td>
          <td>87</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e88242a424f734e17b4e5625c6f0a0bfc8277467" target='_blank'>Data-driven prediction of molecular function</a></td>
          <td>
            
              Thomas Litfin,
            
          </td>
          <td>2020-07-06</td>
          <td></td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6e682197df6b899bceb585cf664e8a39b98a7310" target='_blank'>Reinforcement Learning reveals fundamental limits on the mixing of active particles</a></td>
          <td>
            
              Dominik Schildknecht,
            
              A. N. Popova,
            
              J. Stellwagen,
            
              Matt Thomson,
            
          </td>
          <td>2021-05-28</td>
          <td>Soft matter</td>
          <td>3</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ed27c0a1df82dae1550bb083f9a6a3f2ca159212" target='_blank'>Multisecond ligand dissociation dynamics from atomistic simulations</a></td>
          <td>
            
              Steffen Wolf,
            
              Benjamin Lickert,
            
              Simon A. Bray,
            
              G. Stock,
            
          </td>
          <td>2020-01-13</td>
          <td>Nature Communications</td>
          <td>45</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/12f90a54c1f94981ba4f129be8b220b4b8282a6b" target='_blank'>CarsiDock: a deep learning paradigm for accurate protein–ligand docking and screening based on large-scale pre-training</a></td>
          <td>
            
              Heng Cai,
            
              Chao Shen,
            
              Tianye Jian,
            
              Xujun Zhang,
            
              Tong Chen,
            
              Xiaoqi Han,
            
              Zhuo Yang,
            
              Wei Dang,
            
              Chang-Yu Hsieh,
            
              Yu Kang,
            
              P. Pan,
            
              Xiangyang Ji,
            
              Jianfei Song,
            
              Tingjun Hou,
            
              Yafeng Deng,
            
          </td>
          <td>2023-12-19</td>
          <td>Chemical Science</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b74601b0d18dd5eb1d928512f7bcf69a8953ae17" target='_blank'>Dataset Augmentation Allows Deep Learning-Based Virtual Screening To Better Generalize To Unseen Target Classes, And Highlight Important Binding Interactions</a></td>
          <td>
            
              Jack Scantlebury,
            
              Nathan Brown,
            
              F. von Delft,
            
              C. Deane,
            
          </td>
          <td>2020-03-06</td>
          <td>bioRxiv</td>
          <td>34</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4850728254bc6a2b8edcaf149113ed9a1f037266" target='_blank'>Computational methods for protein structure prediction and energy minimization</a></td>
          <td>
            
              George Karypis,
            
              Andy Blair,
            
              Greg Williams,
            
              Jeff Hedman,
            
              Chelsea Griffin,
            
              Brad Lemke,
            
              Scott Agster,
            
              Getiria Onsongo,
            
              E. DeJong,
            
              Matt Hildebrand,
            
              John Stever,
            
              Seong Oh,
            
              Eliana Salvemini,
            
              Ikumi Suzuki,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f2633ad706a8537bd7d8d89561d4bbad71373380" target='_blank'>Equivariant Flexible Modeling of the Protein-Ligand Binding Pose with Geometric Deep Learning.</a></td>
          <td>
            
              Tiejun Dong,
            
              Ziduo Yang,
            
              Jun Zhou,
            
              C. Chen,
            
          </td>
          <td>2023-11-08</td>
          <td>Journal of chemical theory and computation</td>
          <td>7</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bd3618c6cac7aaac3c4a1de648b4ebbf9bcc9079" target='_blank'>Transfer Learning of Memory Kernels in Coarse-grained Modeling</a></td>
          <td>
            
              Zhan Ma,
            
              Shu Wang,
            
              Minhee Kim,
            
              Kaibo Liu,
            
              Chun-Long Chen,
            
              Wenxiao Pan,
            
          </td>
          <td>2021-03-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/becb9114114ed910f0504ca0770b6ccd2e1e3dba" target='_blank'>Few-shot deep learning for AFM force curve characterization of single-molecule interactions</a></td>
          <td>
            
              Joshua R. Waite,
            
              Sin Yong Tan,
            
              Homagni Saha,
            
              S. Sarkar,
            
              A. Sarkar,
            
          </td>
          <td>2023-01-01</td>
          <td>Patterns</td>
          <td>5</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bc5d1bbc34396e6dcf20c5e26cf2cc1fe437bbc3" target='_blank'>Machine Learning for Sequence and Structure-Based Protein-Ligand Interaction Prediction</a></td>
          <td>
            
              Yunjiang Zhang,
            
              Shuyuan Li,
            
              Kong Meng,
            
              Shaorui Sun,
            
          </td>
          <td>2024-02-22</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8c245602d3c5a7f864f86196c1c3b618875dc574" target='_blank'>Machine Learning-Assisted Phase Transition Temperatures from Generalized Replica Exchange Simulations of Dry Martini Lipid Bilayers.</a></td>
          <td>
            
              Zeke A. Piskulich,
            
              Q. Cui,
            
          </td>
          <td>2022-07-12</td>
          <td>The journal of physical chemistry letters</td>
          <td>3</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/52531cafbfd5b78ee17e41c7a9ddea4f05c794ee" target='_blank'>A Physics-Infused Deep Learning Model for the Prediction of Refractive Indices and Its Use for the Large-Scale Screening of Organic Compound Space</a></td>
          <td>
            
              Mojtaba Haghighatlari,
            
              Gaurav Vishwakarma,
            
              M. A. F. Afzal,
            
              J. Hachmann,
            
          </td>
          <td>2019-07-08</td>
          <td>None</td>
          <td>9</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/597c6c4b574de2c8f2b653b9aa59b2bb0b538242" target='_blank'>A new paradigm for applying deep learning to protein–ligand interaction prediction</a></td>
          <td>
            
              Zechen Wang,
            
              Sheng Wang,
            
              Yangyang Li,
            
              Jingjing Guo,
            
              Yanjie Wei,
            
              Yuguang Mu,
            
              Liangzhen Zheng,
            
              Weifeng Li,
            
          </td>
          <td>2023-11-03</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5c32e5ff7772775707805b5607bb0deedbdc051a" target='_blank'>Dynamic Partitioning of Stochastic Networks of Molecular Interactions</a></td>
          <td>
            
              M. Griffith,
            
          </td>
          <td>2006-06-01</td>
          <td></td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d450cc13a7ed3f9fc1b4ed3902df4959a91dbf28" target='_blank'>Calibration of force fields using recurrent neural networks</a></td>
          <td>
            
              Aykut Argun,
            
              T. Thalheim,
            
              F. Cichos,
            
              G. Volpe,
            
          </td>
          <td>2020-08-20</td>
          <td></td>
          <td>0</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/636838221cae02fdb98419bae56b304704b75efe" target='_blank'>Automating the search of molecular motor templates by evolutionary methods</a></td>
          <td>
            
              Jose D. Fernández,
            
              F. Vico,
            
          </td>
          <td>2011-11-01</td>
          <td>Bio Systems</td>
          <td>4</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3ea3346dcde3f62fe93789245d044f30241af989" target='_blank'>A Markov Random Field Framework for Protein Side-Chain Resonance Assignment ? ( Supporting Material )</a></td>
          <td>
            
              Jianyang Zeng,
            
              Pei Zhou,
            
              B. Donald,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/32f3d308cb0fcb5877d55db7a8488399b9fa4370" target='_blank'>Modeling Functional Motions of Biological Systems by Customized Natural Moves</a></td>
          <td>
            
              Samuel Demharter,
            
              B. Knapp,
            
              C. Deane,
            
              P. Minary,
            
          </td>
          <td>2016-08-23</td>
          <td>Biophysical Journal</td>
          <td>6</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0434a4aa47c9f4ee816ef692d3d894d241df4222" target='_blank'>Regression Algorithm Based on Self-Distillation and Ensemble Learning</a></td>
          <td>
            
              Yaqi Li,
            
              Qiwen Dong,
            
              Gang Liu,
            
          </td>
          <td>2021-12-04</td>
          <td>Proceedings of the 2021 5th International Conference on Computer Science and Artificial Intelligence</td>
          <td>0</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ef8ae5ef66f2e2cf624ca7be272f3bae1e6fdd7e" target='_blank'>A Hierarchical Approach to Scaling Batch Active Search Over Structured Data</a></td>
          <td>
            
              Vivek Myers,
            
              Peyton Greenside,
            
          </td>
          <td>2020-07-20</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/66533fd636b9f0755254622f0d58f2a03ddddd7d" target='_blank'>A Bayesian Framework for Cryo-EM Heterogeneity Analysis using Regularized Covariance Estimation</a></td>
          <td>
            
              M. A. Gilles,
            
              Amit Singer,
            
          </td>
          <td>2023-11-01</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bcade0826f7ff1b1732cdf05f0a92fe714b9dea2" target='_blank'>The construction of general basis functions in reweighting ensemble dynamics simulations: Reproduce equilibrium distribution in complex systems from multiple short simulation trajectories</a></td>
          <td>
            
              C. Zhang 张,
            
              Ming 明 Li 黎,
            
              X. Zhou 周,
            
          </td>
          <td>2015-10-20</td>
          <td>Chinese Physics B</td>
          <td>1</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cee2c50c124b5158adcc1ebcb093bd6928665648" target='_blank'>Predicting the entire glassy dynamics from static structure by machine learning relative motion</a></td>
          <td>
            
              H. Shiba,
            
              Masatoshi Hanai,
            
              T. Suzumura,
            
              T. Shimokawabe,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b696a5c8411d81c795b48d5aa5abb730f150931c" target='_blank'>tinyIFD: A High-Throughput Binding Pose Refinement Workflow Through Induced-Fit Ligand Docking</a></td>
          <td>
            
              Darren J. Hsu,
            
              Russell B. Davidson,
            
              A. Sedova,
            
              Jens Glaser,
            
          </td>
          <td>2023-05-19</td>
          <td>Journal of chemical information and modeling</td>
          <td>6</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bb42bc744325cf1c526c89c1f09131648de66c34" target='_blank'>Ultrahigh Throughput Protein-Ligand Docking with Deep Learning.</a></td>
          <td>
            
              Austin R. Clyde,
            
          </td>
          <td>2021-11-04</td>
          <td>Methods in molecular biology</td>
          <td>1</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bedc29f7b553bd322df265e2c5a215f1dfd19b4e" target='_blank'>Multi-scale Iterative Refinement towards Robust and Versatile Molecular Docking</a></td>
          <td>
            
              Jiaxian Yan,
            
              Zaixin Zhang,
            
              Kai Zhang,
            
              Qi Liu,
            
          </td>
          <td>2023-11-30</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>9</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    // columnDefs: [
    //     {"className": "dt-center", "targets": "_all"},
    //     // set width for both columns 0 and 1 as 25%
    //     { width: '40%', targets: 4 }

    //   ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
</script>
</html>