---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-07-15 06:06:26 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Systems biology informed neural networks (SBINN) predict response and novel combinations for PD-1 checkpoint blockade</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d54a4ef82873fd3a0e857ad2c136d65fa17db8" target='_blank'>Systems biology informed neural networks (SBINN) predict response and novel combinations for PD-1 checkpoint blockade</a></td>
          <td>
            M. Przedborski, Munisha Smalley, S. Thiyagarajan, A. Goldman, M. Kohandel
          </td>
          <td>2021-07-15</td>
          <td>Communications Biology</td>
          <td>9</td>
          <td>28</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4624f66c438a80c62525b6af71f20d1edb159bd" target='_blank'>Integrating Systems Biology and an Ex Vivo Human Tumor Model Elucidates PD-1 Blockade Response Dynamics</a></td>
          <td>
            Munisha Smalley, Munisha Smalley, M. Przedborski, S. Thiyagarajan, Moriah Pellowe, A. Verma, N. Brijwani, Debika Datta, Misti Jain, Basavaraja U. Shanthappa, Vidushi Kapoor, K. Gopinath, D. C. Doval, K. Sabitha, G. Taroncher-Oldenburg, B. Majumder, P. Majumder, M. Kohandel, Aaron Goldman, Aaron Goldman
          </td>
          <td>2020-06-01</td>
          <td>iScience</td>
          <td>6</td>
          <td>73</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acdf4e69bc25f2c08197466cbe432159a3f675d2" target='_blank'>Network-based machine learning approach to predict immunotherapy response in cancer patients</a></td>
          <td>
            JungHo Kong, Doyeon Ha, Juhun Lee, Inhae Kim, Minhyuk Park, S. Im, Kunyoo Shin, Sanguk Kim
          </td>
          <td>2022-06-28</td>
          <td>Nature Communications</td>
          <td>63</td>
          <td>45</td>
        </tr>
    
        <tr id="3136 Background: Tremendous progress towards understanding the molecular underpinnings of cancer has ushered in an era of precision medicine, where actionable biomarkers inform rational design of novel drugs, repurposing of existing medications or new combination therapies, agnostic of tumor type. However less than 20% of cancer patients match to therapies using existing predictive biomarkers. There is therefore an acute need for new methods to expand the population of patients who could benefit from existing drugs and improve patient selection and design of clinical trials evaluating new or repurposed drugs. Methods: We developed systems biology-informed AI methods trained on vast multi-omics and multi-modal datasets from DepMap, TCGA and other publicly available sources, to uncover latent vulnerabilities existing in tumors. These vulnerabilities map to pharmacological sensitivities and can be assigned to individual patients using clinically available molecular data. To evaluate our model, we focused on predicting sensitivities to a third-generation tyrosine kinase inhibitor (TKI), which we assessed in non-small cell lung cancer (NSCLC) cell lines, patient derived xenografts (PDXs) and a clinicogenomics cohort from AACR Project GENIE (n=2004 patients, n=273 patients treated with TKI of interest). Where applicable, whole exome data used as input for validation model predictions were filtered to ~300 genes included on commercial NGS panels to reflect data available in RWD cohorts. Results: Our model accurately predicts response to the TKI across diverse cancer cell lines with high accuracy (r2= 0.768; rho= 0.83; mean absolute error = 0.065). Using NSCLC cancer cell lines and PDX model systems heterogeneous for this TKI’s current FDA molecular label (but within the scope of the drug’s current cancer indication), we were able to further validate our model’s predictions, show they were not dependent on obvious molecular confounders, and demonstrate a potential label expansion opportunity for this drug beyond the current narrow molecular label. While all NSCLC patients from AACR Project GENIE treated with this TKI were “on label”, NSCLC patients our model predicted to be sensitive to the TKI had a 4-month increased median PFS compared to patients predicted to be less sensitive (Log-rank test p-value< 0.05; HR= 1.64; 1.23-2.22, 95% CI). The PFS partition was not due to obvious clinical confounders such as stage (p= 0.91), age (p=0.86), sex (p=0.63), smoking status (p=0.57) or on-target mutations (p=0.70). Biological networks associated with each prediction provide rich context and highlight a putative rational drug combination strategy for this drug. Conclusions: We have developed an ML model that uses clinically obtainable NGS data to accurately predict increased sensitivity to a third-generation TKI validated in NSCLC cell lines, PDX models and a RWD patient cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4346310495ab63af1d7f88c90bf128d0bd32a82" target='_blank'>Use of a systems-biology informed machine learning model to predict drug response using clinically available NGS data.</a></td>
          <td>
            Maayan Baron, Andrey Chursov, Brandon Funkhouser, Jacob Kaffey, S. Sushanth Kumar, G. Komatsoulis, Felicia Kuperwaser, M. Ramchandran, J. Sherman, E. Vucic
          </td>
          <td>2023-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>
    
        <tr id="Cancer cells can leverage several cell-intrinsic and -extrinsic mechanisms to escape immune system recognition. The inherent complexity of the tumor microenvironment, with its multicellular and dynamic nature, poses great challenges for the extraction of biomarkers of immune response and immunotherapy efficacy. Here, we use RNA-seq data combined with different sources of prior-knowledge to derive system-based signatures of the tumor microenvironment, quantifying immune-cell composition and intra- and inter-cellular communications. We applied multi-task learning to these signatures to predict different hallmarks of immune responses and derive cancer-type-specific models based on interpretable systems biomarkers. By applying our models to independent RNA-seq data from cancer patients treated with PD-1 inhibitors, we demonstrated that our method to Estimate Systems Immune Response (EaSIeR) accurately predicts therapeutic outcome. We anticipate that EaSIeR will be a valuable tool to provide a holistic description of immune responses in complex and dynamic systems such as tumors using available RNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2aaa9d42938b19deef4e268b53afddc2ccc6e8" target='_blank'>Predictive systems biomarkers of response to immune checkpoint inhibitors</a></td>
          <td>
            Óscar Lapuente-Santana, Maisa van Genderen, P. Hilbers, F. Finotello, Federica Eduati
          </td>
          <td>2021-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>
    
        <tr id="Immunotherapy and immune checkpoint blocking antibodies such as anti-PD-1 are approved and significantly improve the survival of advanced non-small cell lung cancer (NSCLC) patients, but there has been little success in identifying biomarkers capable of separating the responders from non-responders before the onset of the therapy. In this study, we developed a quantitative system pharmacology (QSP) model to represent the anti-tumor immune response in human NSCLC that integrated our knowledge of tumor growth, antigen processing and presentation, T cell activation and distribution, antibody pharmacokinetics, and immune checkpoint dynamics. The model was calibrated with the available data and was used to identify potential biomarkers as well as patient-specific response based on the patient parameters. The model predicted that in addition to tumor mutational burden (TMB), a known biomarker for anti-PD-1 therapy in NSCLC, the number of effector T cells and regulatory T cells in the tumor and blood is a predictor of the responders. Furthermore, the model simulated a set of 12 patients with known TMB and MHC/antigen-binding affinity from a recent clinical trial (ClinicalTrials.gov number, NCT02259621) on neoadjuvant nivolumab therapy in resectable lung cancer and predicted an augmented durable response in patients with adjuvant nivolumab treatment in addition to the clinical trial protocol of neoadjuvant nivolumab treatment followed by resection. Overall, the model provides a valuable framework to model tumor immunity and response to immune checkpoint blockers to enhance biomarker discovery and performing virtual clinical trials to aid in design and interpretation of the current trials with fewer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6107c61042002f08fd5b87757a1769200845b10" target='_blank'>A Computational Model of Neoadjuvant PD-1 Inhibition in Non-Small Cell Lung Cancer</a></td>
          <td>
            M. Jafarnejad, Chang Gong, E. Gabrielson, I. Bartelink, P. Vicini, B. Wang, R. Narwal, L. Roskos, A. Popel
          </td>
          <td>2019-06-24</td>
          <td>The AAPS Journal</td>
          <td>49</td>
          <td>69</td>
        </tr>
    
        <tr id="We present a study where predictive mechanistic modeling is used in combination with deep learning methods to predict individual patient survival probabilities under immune checkpoint inhibitor (ICI) therapy. This hybrid approach enables prediction based on both measures that are calculable from mechanistic models (but may not be directly measurable in the clinic) and easily measurable quantities or characteristics (that are not always readily incorporated into predictive mechanistic models). The mechanistic model we have applied here can predict tumor response from CT or MRI imaging based on key mechanisms underlying checkpoint inhibitor therapy, and in the present work, its parameters were combined with readily-available clinical measures from 93 patients into a hybrid training set for a deep learning time-to-event predictive model. Analysis revealed that training an artificial neural network with both mechanistic modeling-derived and clinical measures achieved higher per-patient predictive accuracy based on event-time concordance, Brier score, and negative binomial log-likelihood-based criteria than when only mechanistic model-derived values or only clinical data were used. Feature importance analysis revealed that both clinical and model-derived parameters play prominent roles in neural network decision making, and in increasing prediction accuracy, further supporting the advantage of our hybrid approach. We anticipate that many existing mechanistic models may be hybridized with deep learning methods in a similar manner to improve predictive accuracy through addition of additional data that may not be readily implemented in mechanistic descriptions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7b3da8173ee0e944bcc41b5f5c4c90ff6088d34" target='_blank'>Hybridizing mechanistic mathematical modeling with deep learning methods to predict individual cancer patient survival after immune checkpoint inhibitor therapy</a></td>
          <td>
            J. D. Butner, P. Dogra, Caroline Chung, Eugene J Koay, James Welsh, David Hong, Vittorio Cristini, Zhihui Wang
          </td>
          <td>2024-03-29</td>
          <td>Research Square</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73679d28d06ad0d4ebed1bb9857e3cbc975de458" target='_blank'>Biology-aware mutation-based deep learning for outcome prediction of cancer immunotherapy with immune checkpoint inhibitors</a></td>
          <td>
            Junyan Liu, Md Tauhidul Islam, Shengtian Sang, Liang Qiu, Lei Xing
          </td>
          <td>2023-11-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c74be92bfee5ad8a4f41543b36dd3d2c83e2bf" target='_blank'>Interpretable predictions of cellular behavior</a></td>
          <td>
            Ananya Rastogi
          </td>
          <td>2021-03-01</td>
          <td>Nature Computational Science</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr id="Abstract Understanding the intricate interactions of cancer cells with the tumor microenvironment (TME) is a pre-requisite for the optimization of immunotherapy. Mechanistic models such as quantitative systems pharmacology (QSP) provide insights into the TME dynamics and predict the efficacy of immunotherapy in virtual patient populations/digital twins but require vast amounts of multimodal data for parameterization. Large-scale datasets characterizing the TME are available due to recent advances in bioinformatics for multi-omics data. Here, we discuss the perspectives of leveraging omics-derived bioinformatics estimates to inform QSP models and circumvent the challenges of model calibration and validation in immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e6ee84353828e640c230c07c64013c8803ab5" target='_blank'>Leveraging multi-omics data to empower quantitative systems pharmacology in immuno-oncology</a></td>
          <td>
            Theinmozhi Arulraj, Hanwen Wang, Alberto Ippolito, Shuming Zhang, E. Fertig, Aleksander S. Popel
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>43</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>