---
hide:
 - navigation
---
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-04-21 21:32:38 UTC</i>
  </p>
  

  <p>
  <h3>Recommendations for the article <i>CellBox: Interpretable Machine Learning for Perturbation Biology with Application to the Design of Cancer Combination Therapy.</i></h3>
  <table id="table1" class="display" style="width:100%">
  <thead>
    <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/250017f99bdff84eb0a2ac65698224b4abe7c8a4" target='_blank'>Interpretable Machine Learning for Perturbation Biology</a></td>
          <td>
            
              Bo Yuan,
            
              Ciyue Shen,
            
              Augustin Luna,
            
              Anil Korkut,
            
              D. Marks,
            
              John Ingraham,
            
              C. Sander,
            
          </td>
          <td>2019-08-28</td>
          <td>bioRxiv</td>
          <td>6</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e6f21c31f5c969c279bb555f1dff954d04e64763" target='_blank'>Abstract 2102: Interpretable machine learning for perturbation biology</a></td>
          <td>
            
              Judy Shen,
            
              Bo Yuan,
            
              Augustin Luna,
            
              Anil Korkut,
            
              D. Marks,
            
              John Ingraham,
            
              C. Sander,
            
          </td>
          <td>2020-08-13</td>
          <td>Clinical Research (Excluding Clinical Trials)</td>
          <td>0</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/44c74be92bfee5ad8a4f41543b36dd3d2c83e2bf" target='_blank'>Interpretable predictions of cellular behavior</a></td>
          <td>
            
              Ananya Rastogi,
            
          </td>
          <td>2021-03-01</td>
          <td>Nature Computational Science</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1e881a1d1a8c8f35ba366d63693c2ac0094344e4" target='_blank'>Perturbation Biology: Inferring Signaling Networks in Cellular Systems</a></td>
          <td>
            
              Evan J. Molinelli,
            
              Anil Korkut,
            
              Weiqing Wang,
            
              Martin L. Miller,
            
              N. Gauthier,
            
              Xiaohong Jing,
            
              Poorvi Kaushik,
            
              Qin He,
            
              Gordon Mills,
            
              D. Solit,
            
              C. Pratilas,
            
              M. Weigt,
            
              A. Braunstein,
            
              A. Pagnani,
            
              R. Zecchina,
            
              C. Sander,
            
          </td>
          <td>2013-08-23</td>
          <td>PLoS Computational Biology</td>
          <td>128</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/14c762a67f5a9622f2d23ad99cb08992af10b2d5" target='_blank'>Perturbation biology links temporal protein changes to drug responses in a melanoma cell line</a></td>
          <td>
            
              Elin Nyman,
            
              R. Stein,
            
              Xiaohong Jing,
            
              Weiqing Wang,
            
              Benjamin Marks,
            
              I. Zervantonakis,
            
              Anil Korkut,
            
              N. Gauthier,
            
              C. Sander,
            
          </td>
          <td>2019-03-06</td>
          <td>PLoS Computational Biology</td>
          <td>12</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fbff9ee6f0bdabf0ef4cd37032398f2655399604" target='_blank'>Causal Models, Prediction, and Extrapolation in Cell Line Perturbation Experiments</a></td>
          <td>
            
              J. Long,
            
              Yumeng Yang,
            
              K. Do,
            
          </td>
          <td>2022-07-20</td>
          <td>None</td>
          <td>0</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/219ca99cb481dd119358086b5ced78904cf72071" target='_blank'>Interpretable Modeling of Single-cell perturbation Responses to Novel Drugs Using Cycle Consistence Learning</a></td>
          <td>
            
              Wei Huang,
            
              Aichun Zhu,
            
              Hui Liu,
            
          </td>
          <td>2023-11-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/39f3131ed92cb56b1c5af701ea805e6fd9bbe339" target='_blank'>Predicting dynamic signaling network response under unseen perturbations</a></td>
          <td>
            
              Fan Zhu,
            
              Y. Guan,
            
          </td>
          <td>2014-10-01</td>
          <td>Bioinformatics</td>
          <td>15</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1a792cb6bdc69d6f3bb2c8def0809b7dbd05d8f2" target='_blank'>CODEX: COunterfactual Deep learning for the in-silico EXploration of cancer cell line perturbations</a></td>
          <td>
            
              Stefan Schrod,
            
              Tim Beißbarth,
            
              Helena U. Zacharias,
            
              Anne-Christin Hauschild,
            
              M. Altenbuchinger,
            
          </td>
          <td>2024-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d60bdd96b46505ab91ee904706c4a5cf4772b6d0" target='_blank'>The CausalBench challenge: A machine learning contest for gene network inference from single-cell perturbation data</a></td>
          <td>
            
              Mathieu Chevalley,
            
              Jacob A. Sackett-Sanders,
            
              Yusuf H. Roohani,
            
              Pascal Notin,
            
              A. Bakulin,
            
              D. Brzeziński,
            
              Kaiwen Deng,
            
              Y. Guan,
            
              Justin Hong,
            
              Michael Ibrahim,
            
              W. Kotłowski,
            
              Marcin Kowiel,
            
              Panagiotis Misiakos,
            
              Achille Nazaret,
            
              M. Puschel,
            
              Chris Wendler,
            
              Arash Mehrjou,
            
              P. Schwab,
            
          </td>
          <td>2023-08-29</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c5bc5d88ddc9537c194384668aefbb99268f1d44" target='_blank'>Learning interpretable cellular responses to complex perturbations in high-throughput screens</a></td>
          <td>
            
              M. Lotfollahi,
            
              Anna Klimovskaia Susmelj,
            
              C. De Donno,
            
              Yuge Ji,
            
              I. Ibarra,
            
              F. A. Wolf,
            
              N. Yakubova,
            
              Fabian J Theis,
            
              David Lopez-Paz,
            
          </td>
          <td>2021-04-15</td>
          <td>bioRxiv</td>
          <td>41</td>
          <td>95</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e7b46dd8732444e1453773ae3e26cae69554a9d3" target='_blank'>Data-driven reverse engineering of signaling pathways using ensembles of dynamic models</a></td>
          <td>
            
              David Henriques,
            
              A. F. Villaverde,
            
              Miguel Rocha,
            
              J. Sáez-Rodríguez,
            
              J. Banga,
            
          </td>
          <td>2017-02-01</td>
          <td>PLoS Computational Biology</td>
          <td>41</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b993ca25b69f33a600d79449330321e036281b5" target='_blank'>Leveraging multi-way interactions for systematic prediction of pre-clinical drug combination effects</a></td>
          <td>
            
              H. Julkunen,
            
              A. Cichońska,
            
              Prson Gautam,
            
              S. Szedmák,
            
              Jane Douat,
            
              T. Pahikkala,
            
              T. Aittokallio,
            
              Juho Rousu,
            
          </td>
          <td>2020-12-01</td>
          <td>Nature Communications</td>
          <td>58</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/08f6856e3bac8359095bb47767c4531ec31f564e" target='_blank'>Abstract A24: Network models of signaling and drug response in melanoma.</a></td>
          <td>
            
              Anil Korkut,
            
              Weiqing Wang,
            
              Evan J. Molinelli,
            
              Martin L. Miller,
            
              Poorvi Kaushik,
            
              Arman Aksoy,
            
              Xiaohong Jing,
            
              N. Gauthier,
            
              C. Sander,
            
          </td>
          <td>2013-11-01</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2432177fcc1ec5b5126043bc973f2239ce50e756" target='_blank'>comboFM: leveraging multi-way interactions for systematic prediction of drug combination effects</a></td>
          <td>
            
              H. Julkunen,
            
              A. Cichońska,
            
              Prson Gautam,
            
              S. Szedmák,
            
              Jane Douat,
            
              T. Pahikkala,
            
              T. Aittokallio,
            
              Juho Rousu,
            
          </td>
          <td>2020-09-02</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69345d79ff4c375b54d763a2037d2ca9eb40812d" target='_blank'>Embedding single-cell experimental conditions to reveal manifold structure of cancer drug perturbation effects</a></td>
          <td>
            
              William S. Chen,
            
              Nevena Zivanovic,
            
              D. V. Dijk,
            
              Guy Wolf,
            
              B. Bodenmiller,
            
              Smita Krishnaswamy,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/79a2df200d2aa14a19cd5c25fb0db7833cdfb3f6" target='_blank'>MECHANISTIC AND DATA-DRIVEN MODELS OF CELL SIGNALING: TOOLS FOR FUNDAMENTAL DISCOVERY AND RATIONAL DESIGN OF THERAPY.</a></td>
          <td>
            
              Paul J. Myers,
            
              Sung Hyun Lee,
            
              M. Lazzara,
            
          </td>
          <td>2021-06-01</td>
          <td>Current opinion in systems biology</td>
          <td>6</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f5e380b3b0534a5e89ba5338fb4dce4b84ec6d13" target='_blank'>Predicting cellular responses to complex perturbations in high‐throughput screens</a></td>
          <td>
            
              M. Lotfollahi,
            
              Anna Klimovskaia Susmelj,
            
              C. De Donno,
            
              Leon Hetzel,
            
              Yuge Ji,
            
              Ignacio L Ibarra,
            
              Sanjay R Srivatsan,
            
              Mohsen Naghipourfar,
            
              R. Daza,
            
              Beth K. Martin,
            
              J. Shendure,
            
              José L. McFaline-Figueroa,
            
              Pierre Boyeau,
            
              F. A. Wolf,
            
              N. Yakubova,
            
              Stephan Günnemann,
            
              C. Trapnell,
            
              David Lopez-Paz,
            
              Fabian J Theis,
            
          </td>
          <td>2023-05-08</td>
          <td>Molecular Systems Biology</td>
          <td>40</td>
          <td>142</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b44faa0d29a0a0a5c8a1f0d1fa52c4b8b2d33eb0" target='_blank'>Learning single-cell perturbation responses using neural optimal transport</a></td>
          <td>
            
              Charlotte Bunne,
            
              Stefan G. Stark,
            
              Gabriele Gut,
            
              Jacobo Sarabia Del Castillo,
            
              K. Lehmann,
            
              L. Pelkmans,
            
              Andreas Krause,
            
              G. Rätsch,
            
          </td>
          <td>2021-12-15</td>
          <td>Nature Methods</td>
          <td>45</td>
          <td>69</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/500b567ec04a718411eb4a59b5b37f83439cb68d" target='_blank'>Capturing Biomarkers and Molecular Targets in Cellular Landscapes From Dynamic Reaction Network Models and Machine Learning</a></td>
          <td>
            
              S. Mertins,
            
          </td>
          <td>2022-01-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f7be3f64b10c688aabb1fccc48bd867ba6ecadf1" target='_blank'>Efficient parameterization of large-scale mechanistic models enables drug response prediction for cancer cell lines</a></td>
          <td>
            
              Fabian Froehlich,
            
              T. Kessler,
            
              Daniel Weindl,
            
              A. Shadrin,
            
              Leonard Schmiester,
            
              Hendrik Hache,
            
              A. Muradyan,
            
              M. Schuette,
            
              J. Lim,
            
              M. Heinig,
            
              Fabian J Theis,
            
              H. Lehrach,
            
              C. Wierling,
            
              B. Lange,
            
              J. Hasenauer,
            
          </td>
          <td>2017-08-09</td>
          <td>bioRxiv</td>
          <td>10</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f64ee6a55b38ec3bed3836d96a73e4dc8508ffc" target='_blank'>CellNOptR: a flexible toolkit to train protein signaling networks to data using multiple logic formalisms</a></td>
          <td>
            
              Camille Terfve,
            
              T. Cokelaer,
            
              David Henriques,
            
              Aidan MacNamara,
            
              E. Gonçalves,
            
              Melody K. Morris,
            
              M. V. Iersel,
            
              D. Lauffenburger,
            
              J. Sáez-Rodríguez,
            
          </td>
          <td>2012-10-01</td>
          <td>BMC Systems Biology</td>
          <td>198</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b05d741198474f367df5476c770f0bf45625d17" target='_blank'>Converting networks to predictive logic models from perturbation signalling data with CellNOpt</a></td>
          <td>
            
              Enio Gjerga,
            
              P. Trairatphisan,
            
              A. Gábor,
            
              H. Koch,
            
              C. Chevalier,
            
              Francesco Ceccarelli,
            
              Aurélien Dugourd,
            
              A. Mitsos,
            
              J. Saez-Rodriguez,
            
          </td>
          <td>2020-03-05</td>
          <td>Bioinformatics</td>
          <td>15</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/23532e195730c4f109f158a7a56f2f9f8ee215e1" target='_blank'>Network-driven cancer cell avatars for combination discovery and biomarker identification for DNA Damage Response inhibitors</a></td>
          <td>
            
              O. Papp,
            
              Viktória Jordán,
            
              Szabolcs Hetey,
            
              Róbert Balázs,
            
              Árpád Bartha,
            
              Nóra N. Ordasi,
            
              Sebestyén Kamp,
            
              Bálint Farkas,
            
              Jay Mettetal,
            
              J. Dry,
            
              Duncan Young,
            
              Ben Sidders,
            
              K. Bulusu,
            
              D. Veres,
            
          </td>
          <td>2023-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/95d29815ae80adf97b5be8fddd1f37e415ce96c1" target='_blank'>Data used in "comboFM: leveraging multi-way interactions for systematic prediction of drug combination effects"</a></td>
          <td>
            
              H. Julkunen,
            
              A. Cichońska,
            
              Prson Gautam,
            
              S. Szedmák,
            
              Jane Douat,
            
              T. Pahikkala,
            
              T. Aittokallio,
            
          </td>
          <td>2020-05-02</td>
          <td></td>
          <td>0</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5cb58a880a3a63f790270c7e9c0886da57fa90d1" target='_blank'>Modeling interpretable correspondence between cell state and perturbation response with CellCap</a></td>
          <td>
            
              Yang Xu,
            
              Stephen Fleming,
            
              M. Tegtmeyer,
            
              S. Mccarroll,
            
              M. Babadi,
            
          </td>
          <td>2024-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2aac28eedb11ab7a15d2b095b276547fa03db580" target='_blank'>GEARS: Predicting transcriptional outcomes of novel multi-gene perturbations</a></td>
          <td>
            
              Yusuf H. Roohani,
            
              Kexin Huang,
            
              J. Leskovec,
            
          </td>
          <td>2022-07-15</td>
          <td>bioRxiv</td>
          <td>18</td>
          <td>132</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca6f0093c3490e7b937a7e37c5add468630618bd" target='_blank'>Abstract 5227: Quantitative network models of signaling and drug response in melanoma.</a></td>
          <td>
            
              Anil Korkut,
            
              Weiqing Wang,
            
              Evan J. Molinelli,
            
              Martin L. Miller,
            
              Poorvi Kaushik,
            
              Arman Aksoy,
            
              C. Sander,
            
          </td>
          <td>2013-04-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6cab87236aa24a5b487870d090335444ce2806d8" target='_blank'>Predicting Cellular Responses to Novel Drug Perturbations at a Single-Cell Resolution</a></td>
          <td>
            
              Leon Hetzel,
            
              Simon Böhm,
            
              Niki Kilbertus,
            
              Stephan Günnemann,
            
              M. Lotfollahi,
            
              F. Theis,
            
          </td>
          <td>2022-04-28</td>
          <td>None</td>
          <td>16</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/85906593661911c9a076cbb96391b9bb61a996bc" target='_blank'>Predicting transcriptional outcomes of novel multigene perturbations with GEARS.</a></td>
          <td>
            
              Yusuf H. Roohani,
            
              Kexin Huang,
            
              J. Leskovec,
            
          </td>
          <td>2023-08-17</td>
          <td>Nature biotechnology</td>
          <td>22</td>
          <td>132</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/111bc6c6e664a124d2cdf18572b0cdaf714ca0f1" target='_blank'>Predicting single-cell perturbation responses for unseen drugs</a></td>
          <td>
            
              Leon Hetzel,
            
              Simon Böhm,
            
              Niki Kilbertus,
            
              Stephan Günnemann,
            
              M. Lotfollahi,
            
              F. Theis,
            
          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>7</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ee25812a45c166d4bda91a4d6196677577af78a8" target='_blank'>Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-Cancer Pathway Model.</a></td>
          <td>
            
              Fabian Fröhlich,
            
              T. Kessler,
            
              Daniel Weindl,
            
              A. Shadrin,
            
              Leonard Schmiester,
            
              Hendrik Hache,
            
              A. Muradyan,
            
              M. Schütte,
            
              J. Lim,
            
              M. Heinig,
            
              Fabian J Theis,
            
              H. Lehrach,
            
              C. Wierling,
            
              B. Lange,
            
              J. Hasenauer,
            
          </td>
          <td>2018-12-01</td>
          <td>Cell systems</td>
          <td>94</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/eefeef4ec68a6eef64a8f476f48831037bd03b85" target='_blank'>Quantitative Learning of Cellular Features From Single-cell Transcriptomics Data Facilitates Effective Drug Repurposing</a></td>
          <td>
            
              Jianmei Zhong,
            
              Junyao Yang,
            
              Yinghui Song,
            
              Zhihua Zhang,
            
              Chunming Wang,
            
              Renyang Tong,
            
              Chenglong Li,
            
              Nanhui Yu,
            
              L. Zou,
            
              Sulai Liu,
            
              Jun Pu,
            
              Wei Lin,
            
          </td>
          <td>2023-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4a3ba00d086079efd37f45e95d0ff4cc1155a08c" target='_blank'>Learning Perturbation-specific Cell Representations for Prediction of Transcriptional Response across Cellular Contexts</a></td>
          <td>
            
              Gal Keinan,
            
              K. Sayal,
            
              Alon Gonen,
            
              Jiang Zhu,
            
              L. Granovsky,
            
              Jeremy England,
            
          </td>
          <td>2023-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8d19c68cada5e1d50d801a06b3acae0ab32c451a" target='_blank'>Predicting Drug Response and Synergy Using a Deep Learning Model of Human Cancer Cells.</a></td>
          <td>
            
              B. Kuenzi,
            
              Jisoo Park,
            
              Samson H. Fong,
            
              Kyle S. Sanchez,
            
              John Lee,
            
              J. Kreisberg,
            
              Jianzhu Ma,
            
              T. Ideker,
            
          </td>
          <td>2020-10-21</td>
          <td>Cancer cell</td>
          <td>201</td>
          <td>79</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ed6bdc8d57a8547923d000da4cc87c7013332ab1" target='_blank'>PerturbNet predicts single-cell responses to unseen chemical and genetic perturbations</a></td>
          <td>
            
              Hengshi Yu,
            
              Joshua D. Welch,
            
          </td>
          <td>2022-07-21</td>
          <td>bioRxiv</td>
          <td>10</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f7102460a7c1c25f8491fc3b49ed44b276f9e4f" target='_blank'>ComboPath: An ML system for predicting drug combination effects with superior model specification</a></td>
          <td>
            
              D. Ranasinghe,
            
              Nathan Sanders,
            
              Hok Hei Tam,
            
              Changchang Liu,
            
              Dan Spitz,
            
          </td>
          <td>2024-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/86854a163a91862ad25919cbd970fd100a4b815e" target='_blank'>Models from experiments: combinatorial drug perturbations of cancer cells</a></td>
          <td>
            
              Sven Nelander,
            
              Weiqing Wang,
            
              B. Nilsson,
            
              Q. She,
            
              C. Pratilas,
            
              N. Rosen,
            
              P. Gennemark,
            
              C. Sander,
            
          </td>
          <td>2008-09-02</td>
          <td>Molecular Systems Biology</td>
          <td>195</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/80440d644237300f0163522ecd6327a9ce4184e3" target='_blank'>Abstract 1925: DrugCell: A visible neural network to guide personalized medicine</a></td>
          <td>
            
              B. Kuenzi,
            
              Jisoo Park,
            
              Samson H. Fong,
            
              Kyle S. Sanchez,
            
              J. Kreisberg,
            
              Jianzhu Ma,
            
              T. Ideker,
            
          </td>
          <td>2020-08-13</td>
          <td>Experimental and Molecular Therapeutics</td>
          <td>0</td>
          <td>79</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/75691e7eb6ec25249c8628104bba4f9dfe263b2f" target='_blank'>Cancer drug discovery as a low rank tensor completion problem</a></td>
          <td>
            
              V. S. Murali,
            
              Didem Ağaç Çobanoğlu,
            
              Michael Hsieh,
            
              Meyer Zinn,
            
              V. Malladi,
            
              J. Gesell,
            
              N. Williams,
            
              Erik S. Welf,
            
              G. Raj,
            
              M. Çobanoğlu,
            
          </td>
          <td>2021-03-09</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d130ad02e49f5e2d7ef0a024f965e2c0adada58f" target='_blank'>MEMMAL: A tool for expanding large-scale mechanistic models with machine learned associations and big datasets</a></td>
          <td>
            
              Cemal Erdem,
            
              M. Birtwistle,
            
          </td>
          <td>2023-03-09</td>
          <td>Frontiers in systems biology</td>
          <td>1</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2469f98d9f3edae53e295a96083f4d0b3bb474c3" target='_blank'>Inferring perturbation profiles of cancer samples</a></td>
          <td>
            
              M. Pirkl,
            
              N. Beerenwinkel,
            
          </td>
          <td>2020-12-10</td>
          <td>Bioinformatics</td>
          <td>1</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c69c5c3591914755d090a34a21005ca228613c55" target='_blank'>Abstract C006: DrugCell: A visible neural network to guide precision medicine</a></td>
          <td>
            
              B. Kuenzi,
            
              Jisoo Park,
            
              Samson H. Fong,
            
              J. Kreisberg,
            
              T. Ideker,
            
          </td>
          <td>2019-12-01</td>
          <td>Pharmacogenetics, Pharmacogenomics, and Therapeutic Response</td>
          <td>0</td>
          <td>79</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a9c18fd2ea917e6bd9c3ae0dad21ffdcf793d877" target='_blank'>Integrating network reconstruction with mechanistic modeling to predict cancer therapies</a></td>
          <td>
            
              M. Halasz,
            
              B. Kholodenko,
            
              W. Kolch,
            
              T. Santra,
            
          </td>
          <td>2016-11-22</td>
          <td>Science Signaling</td>
          <td>51</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6d687d12821b42f2e63d93c7838343a822b3b630" target='_blank'>Neural optimal transport predicts perturbation responses at the single-cell level</a></td>
          <td>
            
          </td>
          <td>2023-09-28</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f86fe76b2eafd6a25499a2f6b022f10b26513dc9" target='_blank'>Predicting Drug Mechanics by Deep Learning on Gene and Cell Activities</a></td>
          <td>
            
              Abhishek Dutta,
            
          </td>
          <td>2022-07-01</td>
          <td>2022 44th Annual International Conference of the IEEE Engineering in Medicine & Biology Society (EMBC)</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a9c18fd2ea917e6bd9c3ae0dad21ffdcf793d877" target='_blank'>Integrating network reconstruction with mechanistic modeling to predict cancer therapies</a></td>
          <td>
            
              M. Halasz,
            
              B. Kholodenko,
            
              W. Kolch,
            
              T. Santra,
            
          </td>
          <td>2016-11-22</td>
          <td>Science Signaling</td>
          <td>51</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/43471f39f0d6bc8ea5aba99851cc98d35af550af" target='_blank'>Predicting synergistic target and drug target combinations in a large-scale data-driven simulation of cancer cells.</a></td>
          <td>
            
              T. Haberichter,
            
              Rina Gindelman,
            
              R. Christopher,
            
              A. Dhiman,
            
              C. Hill,
            
              I. Khalil,
            
          </td>
          <td>2004-04-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5d82430a94a815be7801006aa8cf759bb0c3fc36" target='_blank'>Running title : A framework for pathway inference and modelling Character count : 48856</a></td>
          <td>
            
              M. Halasz,
            
              B. Kholodenko,
            
              W. Kolch,
            
              T. Santra,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ebc9ddd822239cc4e456b720a62084216223d598" target='_blank'>Cell reprogramming design by transfer learning of functional transcriptional networks.</a></td>
          <td>
            
              T. Wytock,
            
              A. Motter,
            
          </td>
          <td>2024-03-04</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/516c13660aa9eadff6cefbce4ac0292281fdc217" target='_blank'>Reconstructing and comparing signal transduction networks from single cell protein quantification data</a></td>
          <td>
            
              Tim Stohn,
            
              Roderick van Eijl,
            
              Klaas W. Mulder,
            
              L. Wessels,
            
              Evert Bosdriesz,
            
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e91e7987688bdf0c489f65475d14e6e0c48fd69c" target='_blank'>Personalized logical models to investigate cancer response to BRAF treatments in melanomas and colorectal cancers</a></td>
          <td>
            
              Jonas Béal,
            
              L. Pantolini,
            
              Vincent Noël,
            
              E. Barillot,
            
              Laurence Calzone,
            
          </td>
          <td>2020-05-27</td>
          <td>bioRxiv</td>
          <td>29</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf7bdef0a58cda1802097f3b9a57e950a4613b78" target='_blank'>kboolnet: a toolkit for the verification, validation, and visualization of reaction-contingency (rxncon) models</a></td>
          <td>
            
              Willow Carretero Chavez,
            
              Marcus Krantz,
            
              E. Klipp,
            
              I. Kufareva,
            
          </td>
          <td>2023-06-12</td>
          <td>BMC Bioinformatics</td>
          <td>1</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b83bb2aa35405d23f9bea3e6411e6d9a634e4355" target='_blank'>Logic-based modeling and drug repurposing for the prediction of novel therapeutic targets and combination regimens against E2F1-driven melanoma progression</a></td>
          <td>
            
              Nivedita Singh,
            
              Faiz Khan,
            
              Lakshmi Bala,
            
              Julio Vera,
            
              Olaf Wolkenhauer,
            
              B. Pützer,
            
              Stella Logotheti,
            
              Shailendra Gupta,
            
          </td>
          <td>2023-11-22</td>
          <td>BMC Chemistry</td>
          <td>1</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9c97d7ec2f117ff2be64946f4410f4cfc78a9720" target='_blank'>Computational discovery of dynamic cell line specific Boolean networks from multiplex time-course data</a></td>
          <td>
            
              Misbah Razzaq,
            
              Loïc Paulevé,
            
              A. Siegel,
            
              J. Saez-Rodriguez,
            
              Jérémie Bourdon,
            
              Carito Guziolowski,
            
          </td>
          <td>2018-10-01</td>
          <td>PLoS Computational Biology</td>
          <td>22</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3cca65618f8cf2bf9c4135eb9bef0d9f8b6278db" target='_blank'>De novo Prediction of Cell-Drug Sensitivities Using Deep Learning-based Graph Regularized Matrix Factorization</a></td>
          <td>
            
              Shuangxia Ren,
            
              Yifeng Tao,
            
              Ke Yu,
            
              Yifan Xue,
            
              R. Schwartz,
            
              Xinghua Lu,
            
          </td>
          <td>2021-09-30</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>4</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/437cdce6f393d0b2c68769382f9accba3b03668e" target='_blank'>Combinatorial Optimization for Predicting Optimal Cell-state Conversion Paths</a></td>
          <td>
            
              Shaurya Agrawal,
            
              M. Kellis,
            
              Martin Wohlwend,
            
          </td>
          <td>2023-10-28</td>
          <td>2023 16th International Congress on Image and Signal Processing, BioMedical Engineering and Informatics (CISP-BMEI)</td>
          <td>0</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5cca6f8e0035ee95765a9fed931549a300e3f6ec" target='_blank'>Graph Structured Neural Networks for Perturbation Biology</a></td>
          <td>
            
              Nathaniel Evans,
            
              Gordon B. Mills,
            
              Guanming Wu,
            
              Xubo Song,
            
              S. McWeeney,
            
          </td>
          <td>2024-02-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e7b9460f53b1393fa834eeb46b9c94177968a96a" target='_blank'>Abstract 1568: Predicting stochastic proliferation and death in response to drugs with mechanistic models tailored to genomic, transcriptomic, and proteomic data</a></td>
          <td>
            
              M. Bouhaddou,
            
              A. Barrette,
            
              R. J. Koch,
            
              M. Birtwistle,
            
          </td>
          <td>2017-07-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/918f7f7a0a07ef22413c8d3e174a52a8cf64bfd0" target='_blank'>TumFlow: An AI Model for Predicting New Anticancer Molecules</a></td>
          <td>
            
              Davide Rigoni,
            
              Sachithra Yaddehige,
            
              Nicoletta Bianchi,
            
              A. Sperduti,
            
              Stefano Moro,
            
              Cristian Taccioli,
            
          </td>
          <td>2024-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/488815adde83ad267b507082359af3dfc5928d8c" target='_blank'>CausalBench: A Large-scale Benchmark for Network Inference from Single-cell Perturbation Data</a></td>
          <td>
            
              Mathieu Chevalley,
            
              Yusuf H. Roohani,
            
              Arash Mehrjou,
            
              J. Leskovec,
            
              P. Schwab,
            
          </td>
          <td>2022-10-31</td>
          <td>ArXiv</td>
          <td>13</td>
          <td>132</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/739a16a818bb82059328645b53e7ba825c3f3812" target='_blank'>Perturbation-based gene regulatory network inference to unravel oncogenic mechanisms</a></td>
          <td>
            
              D. Morgan,
            
              Matthew Studham,
            
              Andreas Tjärnberg,
            
              Holger Weishaupt,
            
              F. Swartling,
            
              Torbjörn E. M. Nordling,
            
              E. Sonnhammer,
            
          </td>
          <td>2019-08-15</td>
          <td>Scientific Reports</td>
          <td>5</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b1736febf869c5e02668b650231ddf148ed92a49" target='_blank'>Molecular mechanism matters: Benefits of mechanistic computational models for drug development.</a></td>
          <td>
            
              Lindsay E. Clegg,
            
              F. Mac Gabhann,
            
          </td>
          <td>2015-09-01</td>
          <td>Pharmacological research</td>
          <td>28</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/67a2a552ea17043d93abe625434e942f88995cbb" target='_blank'>Personalized signaling models for personalized treatments</a></td>
          <td>
            
              J. Saez-Rodriguez,
            
              N. Blüthgen,
            
          </td>
          <td>2020-01-01</td>
          <td>Molecular Systems Biology</td>
          <td>29</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e75a189762eb72580797b7115fc0982618113005" target='_blank'>Data-driven learning how oncogenic gene expression locally alters heterocellular networks</a></td>
          <td>
            
              D. Klinke,
            
              Audry Fernández,
            
              Wentao Deng,
            
              A. Razazan,
            
              Habibolla Latifizadeh,
            
              Anika C Pirkey,
            
          </td>
          <td>2020-05-05</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/760b038e5750152cdae63e55a88943bd1c787b5f" target='_blank'>Logic-Based Modeling and Virtual Drug Screening for the Prediction of Novel Therapeutic Targets and Combination Regimens Against E2F1-driven Melanoma Progression</a></td>
          <td>
            
              Nivedita Singh,
            
              Faiz M. Khan,
            
              L. Bala,
            
              J. Vera,
            
              O. Wolkenhauer,
            
              B. Pützer,
            
              Stella Logotheti,
            
              S. Gupta,
            
          </td>
          <td>2021-08-16</td>
          <td></td>
          <td>0</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cbe976be11c8973fb9bcd5921423f2e10c5afe43" target='_blank'>Prediction of Whole-Cell Transcriptional Response with Machine Learning</a></td>
          <td>
            
              Mohammed Eslami,
            
              Amin Espah Borujeni,
            
              Hamid Doosthosseini,
            
              M. Vaughn,
            
              Hamed Eramian,
            
              Katie J. Clowers,
            
              D. B. Gordon,
            
              N. Gaffney,
            
              Mark Weston,
            
              Diveena Becker,
            
              Yuval Dorfan,
            
              John M. Fonner,
            
              Joshua Urrutia,
            
              Carolyn Corbet,
            
              G. Zheng,
            
              Joe Stubbs,
            
              Alex Cristofaro,
            
              Paul Maschhoff,
            
              Jed Singer,
            
              Christopher A. Voigt,
            
              Enoch Yeung,
            
          </td>
          <td>2021-04-30</td>
          <td>bioRxiv</td>
          <td>9</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/499ad7d88c7bee176b658d70e6cf7b947863eda4" target='_blank'>Machine learning for perturbational single-cell omics.</a></td>
          <td>
            
              Yuge Ji,
            
              M. Lotfollahi,
            
              F. A. Wolf,
            
              Fabian J Theis,
            
          </td>
          <td>2021-06-16</td>
          <td>Cell systems</td>
          <td>57</td>
          <td>95</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/901ecbc48c1208da81d971b259a10214099763b6" target='_blank'>Fine tuning a logical model of cancer cells to predict drug synergies: combining manual curation and automated parameterization</a></td>
          <td>
            
              Åsmund Flobak,
            
              John Zobolas,
            
              Miguel Vazquez,
            
              T. S. Steigedal,
            
              L. Thommesen,
            
              Asle Grislingås,
            
              B. Niederdorfer,
            
              Evelina Folkesson,
            
              Martin Kuiper,
            
          </td>
          <td>2023-11-20</td>
          <td>Frontiers in Systems Biology</td>
          <td>1</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ebced1f95674e41423a3ad8b11091ad5bb874d7" target='_blank'>A scalable, open-source implementation of a large-scale mechanistic model for single cell proliferation and death signaling</a></td>
          <td>
            
              Cemal Erdem,
            
              Arnab Mutsuddy,
            
              Ethan M. Bensman,
            
              William B. Dodd,
            
              Michael M. Saint-Antoine,
            
              M. Bouhaddou,
            
              R. Blake,
            
              Sean M. Gross,
            
              Laura M. Heiser,
            
              F. Feltus,
            
              M. Birtwistle,
            
          </td>
          <td>2022-06-21</td>
          <td>Nature Communications</td>
          <td>9</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/be687744152afab7c2ddb01709a001ccb60917bb" target='_blank'>Causal identification of single-cell experimental perturbation effects with CINEMA-OT</a></td>
          <td>
            
              Mingze Dong,
            
              Bao Wang,
            
              Jessica Wei,
            
              Antonio H. de O. Fonseca,
            
              Curt Perry,
            
              Alexander Frey,
            
              Feriel Ouerghi,
            
              E. Foxman,
            
              J. Ishizuka,
            
              R. M. Dhodapkar,
            
              David van Dijk,
            
          </td>
          <td>2023-04-03</td>
          <td>Nature Methods</td>
          <td>7</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/36f0071697a0bf4d8ef408e9428ec7b87f3f155a" target='_blank'>Deriving executable models of biochemical network dynamics from qualitative data</a></td>
          <td>
            
              L. Nakhleh,
            
              Derek A. Ruths,
            
          </td>
          <td>None</td>
          <td></td>
          <td>2</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d80e004f3217d435786b3d2091a2b00e9d7a5b59" target='_blank'>Inference of tumor inhibition pathways from drug perturbation data</a></td>
          <td>
            
              Saad Haider,
            
              R. Pal,
            
          </td>
          <td>2013-12-01</td>
          <td>2013 IEEE Global Conference on Signal and Information Processing</td>
          <td>5</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6718e83fe2443acf30a1a9ee7b9accb540c77c28" target='_blank'>Simple Causal Relationships in Gene Expression Discovered through Deep Learned Collective Variables</a></td>
          <td>
            
              Ching-Hao Wang,
            
              Kalin H. Vetsigian,
            
              Chris Lin,
            
              Finnian Firth,
            
              G. Bradley,
            
              Lena Granovsky,
            
              Jeremy L. England,
            
          </td>
          <td>2023-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf606a3f90e22221c43516e8c4dee7391d95b64e" target='_blank'>Capturing cell heterogeneity in representations of cell populations for image-based profiling using contrastive learning</a></td>
          <td>
            
              Robert van Dijk,
            
              John Arevalo,
            
              M. Babadi,
            
              A. Carpenter,
            
              Shantanu Singh,
            
          </td>
          <td>2023-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/675d66951c0e0f08ceac004dd79db0e81f8dc389" target='_blank'>Stabilized Reconstruction of Signaling Networks from Single-Cell Cue-Response Data</a></td>
          <td>
            
              Sunil Kumar,
            
              Xiao-Kang Lun,
            
              B. Bodenmiller,
            
              M. Rodríguez Martínez,
            
              H. Koeppl,
            
          </td>
          <td>2020-01-27</td>
          <td>Scientific Reports</td>
          <td>3</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/25239424da5948ae4915a5568b8270636c0d8bc0" target='_blank'>A Boolean Approach to Linear Prediction for Signaling Network Modeling</a></td>
          <td>
            
              Federica Eduati,
            
              A. Corradin,
            
              B. Di Camillo,
            
              G. Toffolo,
            
          </td>
          <td>2010-09-16</td>
          <td>PLoS ONE</td>
          <td>22</td>
          <td>56</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b2551be88e4d5d23e724cdcba34eb11c0e1887d1" target='_blank'>Inferring Intracellular Signal Transduction Circuitry from Molecular Perturbation Experiments</a></td>
          <td>
            
              Michelle L. Wynn,
            
              M. Egbert,
            
              N. Consul,
            
              Jungsoo Chang,
            
              Zhiqian Wu,
            
              Sofia D. Meravjer,
            
              S. Schnell,
            
          </td>
          <td>2017-02-10</td>
          <td>Bulletin of Mathematical Biology</td>
          <td>3</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b2551be88e4d5d23e724cdcba34eb11c0e1887d1" target='_blank'>Inferring Intracellular Signal Transduction Circuitry from Molecular Perturbation Experiments</a></td>
          <td>
            
              Michelle L. Wynn,
            
              M. Egbert,
            
              N. Consul,
            
              Jungsoo Chang,
            
              Zhiqian Wu,
            
              Sofia D. Meravjer,
            
              S. Schnell,
            
          </td>
          <td>2017-02-10</td>
          <td>Bulletin of Mathematical Biology</td>
          <td>3</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7c7c22f81b27dfc622adbe84aee66b666a8f8154" target='_blank'>CellFateScout – a bioinformatics tool for elucidating small molecule signaling pathways that drive cells in a specific direction</a></td>
          <td>
            
              Marcin Siatkowski,
            
              V. Liebscher,
            
              G. Fuellen,
            
          </td>
          <td>2013-11-08</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>3</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf6574c6ab1b8f89c7d9ffc8f025aaf2517b1f79" target='_blank'>Identifying Biological Network Structure, Predicting Network Behavior, and Classifying Network State With High Dimensional Model Representation (HDMR)</a></td>
          <td>
            
              Miles A. Miller,
            
              Xiao-Jiang Feng,
            
              Genyuan Li,
            
              H. Rabitz,
            
          </td>
          <td>2012-06-18</td>
          <td>PLoS ONE</td>
          <td>24</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca90d5ff200d1a8e3ecb19f3b2e70cb9be7545c5" target='_blank'>Identification of Significant Gene Expression Changes in Multiple Perturbation Experiments using Knockoffs</a></td>
          <td>
            
              Ting Zhao,
            
              Guangyu Zhu,
            
              Patrick Flaherty,
            
          </td>
          <td>2021-10-19</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b2179f14d12a3b6f8b220dc8f28fe2de311fe5ee" target='_blank'>Cellograph: a semi-supervised approach to analyzing multi-condition single-cell RNA-sequencing data using graph neural networks</a></td>
          <td>
            
              Jamshaid A. Shahir,
            
              Natalie Stanley,
            
              J. Purvis,
            
          </td>
          <td>2023-02-27</td>
          <td>BMC Bioinformatics</td>
          <td>1</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7f0f4d46da1eb223cbfac1ecc3c19f71f64f3f3f" target='_blank'>Multi-ContrastiveVAE disentangles perturbation effects in single cell images from optical pooled screens</a></td>
          <td>
            
              Zitong Jerry Wang,
            
              Romain Lopez,
            
              Jan-Christian Hütter,
            
              Takamasa Kudo,
            
              Heming Yao,
            
              Philipp Hanslovsky,
            
              Burkhard Höckendorf,
            
              Rahul Moran,
            
              David Richmond,
            
              Aviv Regev,
            
          </td>
          <td>2024-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4e43c617b070d8f8962d3421414badabdcb73e0d" target='_blank'>Inference of drug off-target effects on cellular signaling using interactome-based deep learning</a></td>
          <td>
            
              N. Meimetis,
            
              D. Lauffenburger,
            
              Avlant Nilsson,
            
          </td>
          <td>2023-10-15</td>
          <td>iScience</td>
          <td>0</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/543e0f84cb1b21cd4baa90a0bd84fdd643063ae2" target='_blank'>Machine learning of stochastic gene network phenotypes</a></td>
          <td>
            
              Kyemyung Park,
            
              T. Prüstel,
            
              Yong Lu,
            
              J. Tsang,
            
          </td>
          <td>2019-10-31</td>
          <td>bioRxiv</td>
          <td>6</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0038cc391a39ac6f8a7c39b6e5411aef245c08aa" target='_blank'>Systematic reconstruction of molecular pathway signatures using scalable single-cell perturbation screens</a></td>
          <td>
            
              Longda Jiang,
            
              Carol Dalgarno,
            
              Efthymia Papalexi,
            
              Isabella Mascio,
            
              H. Wessels,
            
              Huiyoung Yun,
            
              Nika Iremadze,
            
              G. Lithwick-Yanai,
            
              D. Lipson,
            
              R. Satija,
            
          </td>
          <td>2024-01-30</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fed989fd87c7141bf188cf54581345f04a17c5b1" target='_blank'>Predictive Modeling of Drug Treatment in the Area of Personalized Medicine</a></td>
          <td>
            
              L. Ogilvie,
            
              C. Wierling,
            
              T. Kessler,
            
              H. Lehrach,
            
              B. Lange,
            
          </td>
          <td>2015-12-06</td>
          <td>Cancer Informatics</td>
          <td>21</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d2d0dd850123e4fbe298ac00382236dcbf2ab1a8" target='_blank'>Predicting cancer drug mechanisms of action using molecular network signatures.</a></td>
          <td>
            
              J. Pritchard,
            
              P. M. Bruno,
            
              M. Hemann,
            
              D. Lauffenburger,
            
          </td>
          <td>2013-06-04</td>
          <td>Molecular bioSystems</td>
          <td>28</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b52aeb43131c06a2c335bd79f3b88903567cddfd" target='_blank'>Inference of cell dynamics on perturbation data using adjoint sensitivity</a></td>
          <td>
            
              Weiqi Ji,
            
              Bo Yuan,
            
              Ciyue Shen,
            
              A. Regev,
            
              C. Sander,
            
              Sili Deng,
            
          </td>
          <td>2021-04-13</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>185</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a6d0e837a9dcdbc43d9a4a2226afb42531ce5dbb" target='_blank'>Dissecting cell state transitions by causal inference</a></td>
          <td>
            
              Longchen Xu,
            
              Tingting Cong,
            
              Hengyi Xu,
            
              Naixue Yang,
            
              Chen Tian,
            
              Sijie Yang,
            
              Ming Zhu,
            
              Rahul Sinha,
            
              Ryo Yamamoto,
            
              Wei Zhang,
            
              Jianwei Wang,
            
              Xun Lan,
            
          </td>
          <td>2023-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/10ad33cfd2737f28f296070cb3cb2fc7e4156e90" target='_blank'>PHOCOS: inferring multi-feature phenotypic crosstalk networks</a></td>
          <td>
            
              Yue Deng,
            
              S. Altschuler,
            
              Lani F. Wu,
            
          </td>
          <td>2016-06-11</td>
          <td>Bioinformatics</td>
          <td>1</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6f4b1caaec3342214fc92c5e6161daa75ffd9ea8" target='_blank'>Modelling Cellular Perturbations with the Sparse Additive Mechanism Shift Variational Autoencoder</a></td>
          <td>
            
              Michael D. Bereket,
            
              Theofanis Karaletsos,
            
          </td>
          <td>2023-11-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d9376a5aba08c5400b327c29af9c6107626e055f" target='_blank'>Predicting cell morphological responses to perturbations using generative modeling</a></td>
          <td>
            
              A. Palma,
            
              F. Theis,
            
              M. Lotfollahi,
            
          </td>
          <td>2023-07-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6e5e9ec96195300822f18549b285b771362a78c2" target='_blank'>Approaches to the analysis of cell signaling networks and their application in drug discovery.</a></td>
          <td>
            
              E. Berg,
            
              E. Hytopoulos,
            
              I. Plavec,
            
              E. Kunkel,
            
          </td>
          <td>None</td>
          <td>Current opinion in drug discovery & development</td>
          <td>11</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3a9a0b2e48180081316331ab76704fc68f0f04fb" target='_blank'>ChemPert: mapping between chemical perturbation and transcriptional response for non-cancer cells</a></td>
          <td>
            
              Menglin Zheng,
            
              S. Okawa,
            
              M. Bravo,
            
              Fei Chen,
            
              M. Martínez‐Chantar,
            
              A. del Sol,
            
          </td>
          <td>2022-05-01</td>
          <td>Nucleic Acids Research</td>
          <td>6</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/199da934ab811e693733e7fd43331f520646f774" target='_blank'>NeuMF: Predicting anti-cancer drug response through a neural matrix factorization model</a></td>
          <td>
            
              Hui Liu,
            
              Jian Yu,
            
              Xiangzhi Chen,
            
              Lin Zhang,
            
          </td>
          <td>2022-06-09</td>
          <td>Current Bioinformatics</td>
          <td>3</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/af786ba590cb9db62ff31318c785685ade68bfd9" target='_blank'>Genome-scale mechanistic modeling of signaling pathways made easy: A bioconductor/cytoscape/web server framework for the analysis of omic data</a></td>
          <td>
            
              Kinza Rian,
            
              Marta R. Hidalgo,
            
              C. Çubuk,
            
              M. M. Falco,
            
              C. Loucera,
            
              M. Esteban-Medina,
            
              Inmaculada Álamo-Álvarez,
            
              M. Peña-Chilet,
            
              J. Dopazo,
            
          </td>
          <td>2021-05-15</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>8</td>
          <td>77</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/810f39fee2ea4ba27ab948178ab11876565d03fe" target='_blank'>Sparse Regression Based Structure Learning of Stochastic Reaction Networks from Single Cell Snapshot Time Series</a></td>
          <td>
            
              A. Klimovskaia,
            
              Stefan Ganscha,
            
              M. Claassen,
            
          </td>
          <td>2016-12-01</td>
          <td>PLoS Computational Biology</td>
          <td>19</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0c515bf7948cdf79f0a7fbf3196ce79e08a5b9bc" target='_blank'>Network medicine</a></td>
          <td>
            
              T. Pawson,
            
              R. Linding,
            
          </td>
          <td>None</td>
          <td>FEBS Letters</td>
          <td>173</td>
          <td>151</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/20d0e30a85cd97e0d2c4e938a40b388b3e088fa2" target='_blank'>Modeling transcriptional profiles of gene perturbation with deep neural network</a></td>
          <td>
            
              Wenke Liu,
            
              Xuya Wang,
            
              D. Mani,
            
              D. Fenyö,
            
          </td>
          <td>2021-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2e023e852082a713445ec3354b84096a4e2f50ea" target='_blank'>Modeling drug combination effects via latent tensor reconstruction</a></td>
          <td>
            
              Tianduanyi Wang,
            
              S. Szedmák,
            
              Haishan Wang,
            
              T. Aittokallio,
            
              T. Pahikkala,
            
              A. Cichońska,
            
              Juho Rousu,
            
          </td>
          <td>2021-04-19</td>
          <td>Bioinformatics</td>
          <td>11</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4dfb2fb4a9f004b39a375d0f2d55f5e4ab225d10" target='_blank'>The DIONESUS algorithm provides scalable and accurate reconstruction of dynamic phosphoproteomic networks to reveal new drug targets.</a></td>
          <td>
            
              Mark F. Ciaccio,
            
              Vincent C. Chen,
            
              R. Jones,
            
              N. Bagheri,
            
          </td>
          <td>2015-07-06</td>
          <td>Integrative biology : quantitative biosciences from nano to macro</td>
          <td>13</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a900d13ad7b43388f35b94daae1431ae172efe88" target='_blank'>Logic-based modeling of biological networks with Netflux</a></td>
          <td>
            
              Alexander P. Clark,
            
              Mukti Chowkwale,
            
              Alexander M. Paap,
            
              Stephen T. Dang,
            
              J. Saucerman,
            
          </td>
          <td>2024-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a3a2c0d275e4d995f629a49ea94c7859f8796f5d" target='_blank'>Identifying mutant-specific multi-drug combinations using comparative network reconstruction</a></td>
          <td>
            
              Evert Bosdriesz,
            
              J. M. Fernandes Neto,
            
              A. Sieber,
            
              R. Bernards,
            
              N. Blüthgen,
            
              L. Wessels,
            
          </td>
          <td>2020-12-18</td>
          <td>iScience</td>
          <td>1</td>
          <td>85</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2582151d5accc2521974c0f4afcbfe50b06338f0" target='_blank'>Deep Hidden Physics Modeling of Cell Signaling Networks</a></td>
          <td>
            
              M. Seeger,
            
              J. Longden,
            
              E. Klipp,
            
              R. Linding,
            
          </td>
          <td>2021-06-14</td>
          <td>Current Genomics</td>
          <td>1</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/74f7f895e5af5af4665da721e85d55a55f0fa021" target='_blank'>Applications of machine learning in computational biology</a></td>
          <td>
            
              Edouard Pauwels,
            
          </td>
          <td>2013-11-14</td>
          <td></td>
          <td>3</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/73856a7b883350f38c5c824eb1a8a2e8d0bec432" target='_blank'>CellDrift: inferring perturbation responses in temporally sampled single-cell data</a></td>
          <td>
            
              Kang Jin,
            
              Daniel J. Schnell,
            
              Guangyuan Li,
            
              N. Salomonis,
            
              V. B. Surya Prasath,
            
              R. Szczesniak,
            
              B. Aronow,
            
          </td>
          <td>2022-04-14</td>
          <td>Briefings in Bioinformatics</td>
          <td>4</td>
          <td>91</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2f89b7cc6d2bc61afa6e537388a2ce3d94814a90" target='_blank'>Explainable machine learning prediction of synergistic drug combinations for precision cancer medicine</a></td>
          <td>
            
              Joseph D. Janizek,
            
              Safiye Celik,
            
              Su-In Lee,
            
          </td>
          <td>2018-05-27</td>
          <td>bioRxiv</td>
          <td>33</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e1bd658dcec793d941873bfb8d1a29c01d59eeb5" target='_blank'>Cell-specific prediction and application of drug-induced gene expression profiles</a></td>
          <td>
            
              R. Hodos,
            
              Ping Zhang,
            
              Hao-Chih Lee,
            
              Qiaonan Duan,
            
              Zichen Wang,
            
              N. R. Clark,
            
              Avi Ma’ayan,
            
              Fei Wang,
            
              B. Kidd,
            
              Jianying Hu,
            
              D. Sontag,
            
              J. Dudley,
            
          </td>
          <td>2017-11-17</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>26</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2d6dbb19e42b7a66ce2daaa8d8f0f50ee5999758" target='_blank'>How Artificial Intelligence Unravels the Complex Web of Cancer Drug Response.</a></td>
          <td>
            
              Olivier Elemento,
            
          </td>
          <td>2024-04-08</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e77623b8586bad8623c76900f8d98a5ac3954aea" target='_blank'>Deep neural networks identify signaling mechanisms of ErbB-family drug resistance from a continuous cell morphology space.</a></td>
          <td>
            
              J. Longden,
            
              X. Robin,
            
              Mathias Engel,
            
              J. Ferkinghoff-Borg,
            
              I. Kjær,
            
              I. Horak,
            
              M. W. Pedersen,
            
              R. Linding,
            
          </td>
          <td>2021-01-19</td>
          <td>Cell reports</td>
          <td>9</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/184d07b3a1b80feebef29a192bd27f319319180b" target='_blank'>Biologically-informed neural networks guide mechanistic modeling from sparse experimental data</a></td>
          <td>
            
              John H. Lagergren,
            
              John T. Nardini,
            
              R. Baker,
            
              M. Simpson,
            
              K. Flores,
            
          </td>
          <td>2020-05-26</td>
          <td>PLoS Computational Biology</td>
          <td>70</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0238db2a09463e1111b8a7452d56292ea4679196" target='_blank'>Reverse engineering biological networks: computational approaches for modeling biological systems from perturbation data</a></td>
          <td>
            
              Yungil Kim,
            
          </td>
          <td>2013-09-01</td>
          <td></td>
          <td>0</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe5339d9a9ea4e1b61c19559f4c3ea528d718664" target='_blank'>Predicting Cellular Drug Sensitivity using Conditional Modulation of Gene Expression</a></td>
          <td>
            
              Will Connell,
            
              Michael J. Keiser,
            
          </td>
          <td>2021-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/73ba88115d9f51e7b0d325cd76d33ce096de9c5f" target='_blank'>Exploring genetic interaction manifolds constructed from rich single-cell phenotypes</a></td>
          <td>
            
              Thomas M. Norman,
            
              Max A. Horlbeck,
            
              J. Replogle,
            
              A. Ge,
            
              Albert Xu,
            
              M. Jost,
            
              Luke A. Gilbert,
            
              J. Weissman,
            
          </td>
          <td>2019-08-08</td>
          <td>Science</td>
          <td>164</td>
          <td>133</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7346343516805497ff4b8abfb76efd7213fc8d80" target='_blank'>Cross-domain feature disentanglement for interpretable modeling of tumor microenvironment impact on drug response</a></td>
          <td>
            
              Jia Zhai,
            
              Hui Liu,
            
          </td>
          <td>2023-11-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/888ce556d85d3908b3febbdc50a1911a0f8028ef" target='_blank'>Cell-Vision Fusion: A Swin Transformer-based Approach to Predicting Kinase Inhibitor Mechanism of Action from Cell Painting Data</a></td>
          <td>
            
              William Dee,
            
              Ines Sequeira,
            
              Anna Lobley,
            
              Gregory Slabaugh,
            
          </td>
          <td>2023-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/16a77da1c7e35ad72fd9aa3aecae2ed6ae15675b" target='_blank'>An interpretable deep learning framework for genome-informed precision oncology</a></td>
          <td>
            
              Shuangxia Ren,
            
              Gregory F. Cooper,
            
              Lujia Chen,
            
              Xinghua Lu,
            
          </td>
          <td>2023-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cace16dff9e4a5062c4400f50727ffc44e264a6d" target='_blank'>Systematic quantitative characterization of cellular responses induced by multiple signals</a></td>
          <td>
            
              Ibrahim Al-Shyoukh,
            
              Fuqu Yu,
            
              Jiaying Feng,
            
              Karen Yan,
            
              S. Dubinett,
            
              Chih-Ming Ho,
            
              J. Shamma,
            
              R. Sun,
            
          </td>
          <td>2011-05-30</td>
          <td>BMC Systems Biology</td>
          <td>73</td>
          <td>76</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f489770b414403bd0cf8aa558b15c2836308c23f" target='_blank'>Sorad: a systems biology approach to predict and modulate dynamic signaling pathway response from phosphoproteome time-course measurements</a></td>
          <td>
            
              Tarmo Äijö,
            
              Kirsi J. Granberg,
            
              H. Lähdesmäki,
            
          </td>
          <td>2013-05-15</td>
          <td>Bioinformatics</td>
          <td>12</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a8fe764ee996495b47e9ad27e1376a171892626e" target='_blank'>Causal network inference using biochemical kinetics</a></td>
          <td>
            
              Chris J. Oates,
            
              F. Dondelinger,
            
              N. Bayani,
            
              J. Korkola,
            
              J. Gray,
            
              S. Mukherjee,
            
          </td>
          <td>2014-05-31</td>
          <td>Bioinformatics</td>
          <td>53</td>
          <td>104</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7ebd7a3de31056b4fc912b115aa45bcf96989f0a" target='_blank'>Prediction of kinase inhibitor response using activity profiling, in vitro screening, and elastic net regression</a></td>
          <td>
            
              T. Tran,
            
              Edison Ong,
            
              Andrew P. Hodges,
            
              G. Paternostro,
            
              C. Piermarocchi,
            
          </td>
          <td>2013-07-17</td>
          <td>BMC Systems Biology</td>
          <td>21</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0503969a52a41ce25bc7d79bb7dfa6a958779cb0" target='_blank'>Drug response prediction by inferring pathway-response associations with kernelized Bayesian matrix factorization</a></td>
          <td>
            
              Muhammad Ammad-ud-din,
            
              Suleiman A. Khan,
            
              D. Malani,
            
              A. Murumägi,
            
              O. Kallioniemi,
            
              T. Aittokallio,
            
              Samuel Kaski,
            
          </td>
          <td>2016-06-11</td>
          <td>Bioinformatics</td>
          <td>82</td>
          <td>107</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/09762a704630c719061d3b8d2b2a51b23223f5ac" target='_blank'>Learning Pathway Dynamics from Single‐Cell Proteomic Data: A Comparative Study</a></td>
          <td>
            
              Kleio-Maria Verrou,
            
              I. Tsamardinos,
            
              G. Papoutsoglou,
            
          </td>
          <td>2020-02-25</td>
          <td>Cytometry</td>
          <td>3</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7ad7516eff9fc696646a72f93252f46f481996f6" target='_blank'>Comparative Network Reconstruction using mixed integer programming</a></td>
          <td>
            
              Evert Bosdriesz,
            
              A. Prahallad,
            
              B. Klinger,
            
              A. Sieber,
            
              A. Bosma,
            
              R. Bernards,
            
              N. Blüthgen,
            
              L. Wessels,
            
          </td>
          <td>2018-01-05</td>
          <td>Bioinformatics</td>
          <td>8</td>
          <td>85</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf0c4565167c4fe995eb2dab34efc016fdc4bb96" target='_blank'>Navigating the genetic perturbation landscape: Multi-modal, causal representation learning for target discovery.</a></td>
          <td>
            
              Professor Philip Torr,
            
              Novo Nordisk Associate Professor,
            
              Jesper Ferkinghoff-Borg,
            
              Dr Robert Kitchen,
            
              Dr Cesar Medina,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/68d54a4ef82873fd3a0e857ad2c136d65fa17db8" target='_blank'>Systems biology informed neural networks (SBINN) predict response and novel combinations for PD-1 checkpoint blockade</a></td>
          <td>
            
              M. Przedborski,
            
              Munisha Smalley,
            
              S. Thiyagarajan,
            
              A. Goldman,
            
              M. Kohandel,
            
          </td>
          <td>2021-07-15</td>
          <td>Communications Biology</td>
          <td>8</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/68d54a4ef82873fd3a0e857ad2c136d65fa17db8" target='_blank'>Systems biology informed neural networks (SBINN) predict response and novel combinations for PD-1 checkpoint blockade</a></td>
          <td>
            
              M. Przedborski,
            
              Munisha Smalley,
            
              S. Thiyagarajan,
            
              A. Goldman,
            
              M. Kohandel,
            
          </td>
          <td>2021-07-15</td>
          <td>Communications Biology</td>
          <td>8</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e2a92780976449d08dbc2406577d5c4baa9e540d" target='_blank'>Abstract 2447: Accurate prediction of the signaling pathway activity elicited by variants of uncertain significance (VUS) and their drug sensitivity using deep convolutional neural networks on florescent microscopy images of cells</a></td>
          <td>
            
              Lior Zimmerman,
            
              Arie Aizenman,
            
              Y. Daitsh,
            
              Z. Barbash,
            
              G. Tarcic,
            
              M. Vidne,
            
          </td>
          <td>2019-07-01</td>
          <td>Bioinformatics, Convergence Science, and Systems Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dc80f608750f8ca33b1d0928a49559b562c964b6" target='_blank'>Nonlinear mixed effects dose response modeling in high throughput drug screens: application to melanoma cell line analysis</a></td>
          <td>
            
              Kuan-Fu Ding,
            
              E. Petricoin,
            
              D. Finlay,
            
              H. Yin,
            
              William P. D. Hendricks,
            
              C. Sereduk,
            
              J. Kiefer,
            
              A. Sekulic,
            
              P. LoRusso,
            
              K. Vuori,
            
              J. Trent,
            
              N. Schork,
            
          </td>
          <td>2017-12-15</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>125</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/79a94a8237b7c28336c2f183f4a4fb3339e77641" target='_blank'>Static network structure can be used to model the phenotypic effects of perturbations in regulatory networks</a></td>
          <td>
            
              Ariel Feiglin,
            
              Adar Hacohen,
            
              Avital Sarusi-Portuguez,
            
              J. Fisher,
            
              R. Unger,
            
              Yanay Ofran,
            
          </td>
          <td>2012-11-01</td>
          <td>Bioinformatics</td>
          <td>21</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/377f7126d7b8b2631ac0ebbe3b2d1f87e8179879" target='_blank'>Network dynamics-based cancer panel stratification for systemic prediction of anticancer drug response</a></td>
          <td>
            
              Minsoo Choi,
            
              Jue Shi,
            
              Yan-ting Zhu,
            
              Ruizhen Yang,
            
              Kwang-Hyun Cho,
            
          </td>
          <td>2017-12-01</td>
          <td>Nature Communications</td>
          <td>38</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/377f7126d7b8b2631ac0ebbe3b2d1f87e8179879" target='_blank'>Network dynamics-based cancer panel stratification for systemic prediction of anticancer drug response</a></td>
          <td>
            
              Minsoo Choi,
            
              Jue Shi,
            
              Yan-ting Zhu,
            
              Ruizhen Yang,
            
              Kwang-Hyun Cho,
            
          </td>
          <td>2017-12-01</td>
          <td>Nature Communications</td>
          <td>38</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ab7876236250c23376a91855071a7847b11714ef" target='_blank'>Causal machine learning for single-cell genomics</a></td>
          <td>
            
              Alejandro Tejada-Lapuerta,
            
              Paul Bertin,
            
              Stefan Bauer,
            
              H. Aliee,
            
              Y. Bengio,
            
              Fabian J. Theis,
            
          </td>
          <td>2023-10-23</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/514b3b15114767c3969c9b5144d73f4676fa5314" target='_blank'>Interpretable Drug Response Prediction using a Knowledge-based Neural Network</a></td>
          <td>
            
              Oliver Snow,
            
              Hossein Sharifi-Noghabi,
            
              Jialin Lu,
            
              O. Zolotareva,
            
              Mark Lee,
            
              M. Ester,
            
          </td>
          <td>2021-08-14</td>
          <td>Proceedings of the 27th ACM SIGKDD Conference on Knowledge Discovery & Data Mining</td>
          <td>5</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/385ecf17e4b8589c0b54575cc27c3b02323fabbb" target='_blank'>Biologically Interpretable VAE with Supervision for Transcriptomics Data Under Ordinal Perturbations</a></td>
          <td>
            
              Seyednami Niyakan,
            
              Byung-Jun Yoon,
            
              Xiaoning Qian,
            
              Xihaier Luo,
            
          </td>
          <td>2024-03-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a752b4aaddbbd93b174c693ca797329dfa915b90" target='_blank'>Network inference from perturbation time course data</a></td>
          <td>
            
              Deepraj Sarmah,
            
              Gregory R. Smith,
            
              M. Bouhaddou,
            
              Alan D. Stern,
            
              James Erskine,
            
              M. Birtwistle,
            
          </td>
          <td>2018-06-07</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>6</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/eb116844b3648debb85e2f3d42e92d4236fe05cb" target='_blank'>Drug combination prioritization for cancer treatment using single-cell RNA-seq based transfer learning</a></td>
          <td>
            
              Daniel Osório,
            
              D. McGrail,
            
              Nidhi Sahni,
            
              S. Yi,
            
          </td>
          <td>2022-04-09</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/194dc65d583a9464023f82846ef3c6501bfdaaa0" target='_blank'>lpNet: a linear programming approach to reconstruct signal transduction networks</a></td>
          <td>
            
              Marta R. A. Matos,
            
              Bettina Knapp,
            
              L. Kaderali,
            
          </td>
          <td>2015-10-01</td>
          <td>Bioinformatics</td>
          <td>3</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ef0933787aee0990325e4b4d80906ec4c1713785" target='_blank'>Extensions of ℓ1 regularization increase detection specificity for cell-type specific parameters in dynamic models</a></td>
          <td>
            
              Pascal Dolejsch,
            
              Helge Hass,
            
              J. Timmer,
            
          </td>
          <td>2019-07-16</td>
          <td>BMC Bioinformatics</td>
          <td>9</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7fb2bbe2d1bae2a2de8eb8030724557293a80534" target='_blank'>Analysis of cancer signaling networks by systems biology to develop therapies.</a></td>
          <td>
            
              Rexxi D. Prasasya,
            
              D. Tian,
            
              P. Kreeger,
            
          </td>
          <td>2011-06-01</td>
          <td>Seminars in cancer biology</td>
          <td>28</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c5b3349cc79f09ecee3d3da88bea9fb8a0d5ebf1" target='_blank'>Generative Modeling of Single Cell Gene Expression for Dose-Dependent Chemical Perturbations</a></td>
          <td>
            
              Omar Kana,
            
              R. Nault,
            
              D. Filipovic,
            
              Daniel Marri,
            
              T. Zacharewski,
            
              S. Bhattacharya,
            
          </td>
          <td>2022-10-07</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6ef46e9d9b89a966fd61169c4d736ebe77ac4e4f" target='_blank'>Quantifying heterogeneity to drug response in cancer–stroma kinetics</a></td>
          <td>
            
              Francesco Alemanno,
            
              M. Cavo,
            
              Donatella Delle Cave,
            
              A. Fachechi,
            
              Riccardo Rizzo,
            
              E. D'Amone,
            
              G. Gigli,
            
              E. Lonardo,
            
              A. Barra,
            
              L. L. del Mercato,
            
          </td>
          <td>2023-03-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>2</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/48092d6982a36fb3377367ef9c51e539910b5bee" target='_blank'>Inference of dynamic biological networks based on perturbation data</a></td>
          <td>
            
              R. Pal,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b49bc5b101b13b27f16b4faff42eeb4610f9d976" target='_blank'>Design of perturbations is the key to inference of tumour specific gene regulation</a></td>
          <td>
            
              Torbjörn E. M. Nordling,
            
              E. W. Jacobsen,
            
          </td>
          <td>None</td>
          <td></td>
          <td>1</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9ab85e79a97ff7df23065cd68653e633a4f1713f" target='_blank'>Virtual perturbations to assess explainability of deep-learning based cell fate predictors</a></td>
          <td>
            
              Christopher J. Soelistyo,
            
              G. Charras,
            
              Alan R. Lowe,
            
          </td>
          <td>2023-07-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>58</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0eb3e57ce69d1ae3d6814a3d72ea43785e97889b" target='_blank'>Learning to encode cellular responses to systematic perturbations with deep generative models</a></td>
          <td>
            
              Yifan Xue,
            
              Michael Q. Ding,
            
              Xinghua Lu,
            
          </td>
          <td>2020-01-14</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>5</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0eb3e57ce69d1ae3d6814a3d72ea43785e97889b" target='_blank'>Learning to encode cellular responses to systematic perturbations with deep generative models</a></td>
          <td>
            
              Yifan Xue,
            
              Michael Q. Ding,
            
              Xinghua Lu,
            
          </td>
          <td>2020-01-14</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>5</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d1a9962c0d8dad044099e4cc4844653bb366f9c6" target='_blank'>Quantifying differences in cell line population dynamics using CellPD</a></td>
          <td>
            
              Edwin F. Juarez,
            
              R. Lau,
            
              Samuel H. Friedman,
            
              Ahmadreza Ghaffarizadeh,
            
              E. Jonckheere,
            
              D. Agus,
            
              S. Mumenthaler,
            
              P. Macklin,
            
          </td>
          <td>2016-09-21</td>
          <td>BMC Systems Biology</td>
          <td>20</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0d44ff6e5fa6058296c6eb485bde75df869f55df" target='_blank'>Analytic formulation of optimal structural intervention strategies in Boolean Networks</a></td>
          <td>
            
              Xiaoning Qian,
            
              E. Dougherty,
            
          </td>
          <td>2008-06-08</td>
          <td>2008 IEEE International Workshop on Genomic Signal Processing and Statistics</td>
          <td>0</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/46117124daca3ecb291aabe3600c96baee745993" target='_blank'>Abstract 636: Prediction of therapeutic response via data-driven maps of tumor cell architecture</a></td>
          <td>
            
              Erica Silva,
            
              Akshat Singhal,
            
              Sungjoon Park,
            
              J. Kreisberg,
            
              T. Ideker,
            
          </td>
          <td>2022-06-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>79</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6092e82c0889c7266d300d32543c588b122bf2c2" target='_blank'>NIRest: A Tool for Gene Network and Mode of Action Inference</a></td>
          <td>
            
              Mario Lauria,
            
              Francesco Iorio,
            
              D. di Bernardo,
            
          </td>
          <td>2009-03-01</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>14</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2521c3d76bc439c961b7003080f4a7a661949547" target='_blank'>Causal Protein-Signaling Networks Derived from Multiparameter Single-Cell Data</a></td>
          <td>
            
              K. Sachs,
            
              O. Perez,
            
              D. Pe’er,
            
              D. Lauffenburger,
            
              G. Nolan,
            
          </td>
          <td>2005-04-22</td>
          <td>Science</td>
          <td>1761</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ee43f8dc76a2590e1ac367a7fa51fc7ee49b7e92" target='_blank'>Application of modular response analysis to medium- to large-size biological systems</a></td>
          <td>
            
              Meriem Mekedem,
            
              P. Ravel,
            
              J. Colinge,
            
          </td>
          <td>2021-07-27</td>
          <td>PLoS Computational Biology</td>
          <td>2</td>
          <td>47</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e0b92168fd136f2e5f3d3697ce9f73c94b04cd43" target='_blank'>Comparison of Chemical Structure and Cell Morphology Information for Multitask Bioactivity Predictions</a></td>
          <td>
            
              Maria-Anna Trapotsi,
            
              Lewis H. Mervin,
            
              Avid M. Afzal,
            
              Noé Sturm,
            
              O. Engkvist,
            
              I. Barrett,
            
              A. Bender,
            
          </td>
          <td>2021-03-04</td>
          <td>Journal of chemical information and modeling</td>
          <td>21</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c5731dc2144693fe24fe5b5a783b38bc48b495ce" target='_blank'>Predictive modeling based on functional and genomic characterizations</a></td>
          <td>
            
              R. Pal,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a0841bbac5d01181960b36435c88f22bd9678da5" target='_blank'>Equation learning to identify nano-engineered particle-cell interactions: an interpretable machine learning approach.</a></td>
          <td>
            
              S. Johnston,
            
              Matthew Faria,
            
          </td>
          <td>2022-10-31</td>
          <td>Nanoscale</td>
          <td>4</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/abf14809e2e66e11662e980c6738c2b23bd6f329" target='_blank'>Exploring genetic interaction manifolds constructed from rich phenotypes</a></td>
          <td>
            
              Thomas M. Norman,
            
              Max A. Horlbeck,
            
              J. Replogle,
            
              A. Ge,
            
              Albert Xu,
            
              M. Jost,
            
              Luke A. Gilbert,
            
              J. Weissman,
            
          </td>
          <td>2019-04-07</td>
          <td>bioRxiv</td>
          <td>7</td>
          <td>133</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a4040560bed2818f9e64293c462627858d99cf3e" target='_blank'>RxRx3: Phenomics Map of Biology</a></td>
          <td>
            
              Marta M. Fay,
            
              Oren Z. Kraus,
            
              Mason L. Victors,
            
              Lakshmanan Arumugam,
            
              Kamal Vuggumudi,
            
              John Urbanik,
            
              Kyle Hansen,
            
              Safiye Celik,
            
              Nico Cernek,
            
              Ganesh Jagannathan,
            
              Jordan Christensen,
            
              Berton A. Earnshaw,
            
              I. Haque,
            
              Ben Mabey,
            
          </td>
          <td>2023-02-08</td>
          <td>bioRxiv</td>
          <td>9</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bf54125d5afea1e14f5d83f6dd89a302faed02a3" target='_blank'>PHENSIM: Phenotype Simulator</a></td>
          <td>
            
              S. Alaimo,
            
              Rosaria Valentina Rapicavoli,
            
              G. P. Marceca,
            
              Alessandro La Ferlita,
            
              O. Serebrennikova,
            
              P. Tsichlis,
            
              B. Mishra,
            
              A. Pulvirenti,
            
              A. Ferro,
            
          </td>
          <td>2020-01-21</td>
          <td>PLoS Computational Biology</td>
          <td>7</td>
          <td>80</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/41ab4770fbf5191a6e62e1db41fc3742e6bef409" target='_blank'>Perturbation Detection Through Modeling of Gene Expression on a Latent Biological Pathway Network: A Bayesian Hierarchical Approach</a></td>
          <td>
            
              Lisa Pham,
            
              Luis Carvalho,
            
              S. Schaus,
            
              E. Kolaczyk,
            
          </td>
          <td>2014-09-01</td>
          <td>Journal of the American Statistical Association</td>
          <td>7</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/53ba6cb0bc400a0701fccb960b58ac355135112d" target='_blank'>Population-level comparisons of gene regulatory networks modeled on high-throughput single-cell transcriptomics data</a></td>
          <td>
            
              Daniel Osório,
            
              A. Capasso,
            
              S. Eckhardt,
            
              U. Giri,
            
              Alex Somma,
            
              T. Pitts,
            
              C. Lieu,
            
              W. Messersmith,
            
              S. Bagby,
            
              Harinder Singh,
            
              Jishnu Das,
            
              Nidhi Sahni,
            
              S. S. Yi,
            
              M. Kuijjer,
            
          </td>
          <td>2023-01-20</td>
          <td>Nature Computational Science</td>
          <td>1</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/39b85b8aebc0242791c8944fd3435e2529943c50" target='_blank'>A Computational Framework for Prediction and Analysis of Cancer Signaling Dynamics from RNA Sequencing Data—Application to the ErbB Receptor Signaling Pathway</a></td>
          <td>
            
              Hiroaki Imoto,
            
              Suxiang Zhang,
            
              Mariko Okada,
            
          </td>
          <td>2020-10-01</td>
          <td>Cancers</td>
          <td>17</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d2c12b2e57d1262a3c8beb119bbadf596d7a116a" target='_blank'>Next-Generation Machine Learning for Biological Networks</a></td>
          <td>
            
              Diogo M. Camacho,
            
              K. M. Collins,
            
              Rani K. Powers,
            
              J. Costello,
            
              J. Collins,
            
          </td>
          <td>2018-06-01</td>
          <td>Cell</td>
          <td>617</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/87d9a8ecdbe88827648236d1407034238e4b6f3b" target='_blank'>Learning signaling networks from combinatorial perturbations by exploiting siRNA off-target effects</a></td>
          <td>
            
              J. Tiuryn,
            
              E. Szczurek,
            
          </td>
          <td>2019-07-01</td>
          <td>Bioinformatics</td>
          <td>5</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8ef2787ed860af376ae583b1492007cd37876690" target='_blank'>Network Pharmacology Strategies Toward Multi-Target Anticancer Therapies: From Computational Models to Experimental Design Principles</a></td>
          <td>
            
              Jing Tang,
            
              T. Aittokallio,
            
          </td>
          <td>2014-01-01</td>
          <td>Current Pharmaceutical Design</td>
          <td>121</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58ec472ddfdd86e34f573d35350e65391bd867c9" target='_blank'>Toward subtask decomposition-based learning and benchmarking for genetic perturbation outcome prediction and beyond</a></td>
          <td>
            
              Yicheng Gao,
            
              Zhiting Wei,
            
              Kejing Dong,
            
              Jingya Yang,
            
              Guohui Chuai,
            
              Qi Liu,
            
          </td>
          <td>2024-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3bcc6b6b7ad2a5fd0b693d18fd0a0a6c6bebd007" target='_blank'>Predictive Analytics of Phosphoproteins in Breast Cancer Cells</a></td>
          <td>
            
              S. Opiyo,
            
          </td>
          <td>2016-11-14</td>
          <td></td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9944ac68965dc5c11917e08c5b48887d5bf40eb4" target='_blank'>Combinatorial prediction of therapeutic perturbations using causally-inspired neural networks</a></td>
          <td>
            
              Guadalupe Gonzalez,
            
              Isuru Herath,
            
              Kirill Veselkov,
            
              Michael Bronstein,
            
              M. Zitnik,
            
          </td>
          <td>2024-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6241769c55cbe48d2e27def8eb7f5803add638d1" target='_blank'>Predictive Modeling of Multiplex Chemical Phenomics for Novel Cells and Patients: Applied to Personalized Alzheimer’s Disease Drug Repurposing</a></td>
          <td>
            
              Qiao Liu,
            
              Yue Qiu,
            
              Lei Xie,
            
          </td>
          <td>2021-08-10</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/faece74722f29827ea979c8878669ccabe045ffc" target='_blank'>Generative modeling and latent space arithmetics predict single-cell perturbation response across cell types, studies and species</a></td>
          <td>
            
              M. Lotfollahi,
            
              F. A. Wolf,
            
              Fabian J Theis,
            
          </td>
          <td>2018-11-29</td>
          <td>bioRxiv</td>
          <td>21</td>
          <td>95</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/211e36e0acc01778c91b27f2f135e4e6ac7d3d80" target='_blank'>Machine-learning model makes predictions about network biology</a></td>
          <td>
            
          </td>
          <td>2023-05-31</td>
          <td>Nature</td>
          <td>1</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4588ed2b6771693f49090af31eedc1e60a68d633" target='_blank'>Predicting Cellular Responses with Variational Causal Inference and Refined Relational Information</a></td>
          <td>
            
              Yulun Wu,
            
              Robert A. Barton,
            
              Zichen Wang,
            
              V. Ioannidis,
            
              Carlo De Donno,
            
              Layne Price,
            
              L. Voloch,
            
              G. Karypis,
            
          </td>
          <td>2022-09-30</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>95</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/470ddf945ed118f04b78fe56683288202ff7d56f" target='_blank'>Learning time-varying information flow from single-cell epithelial to mesenchymal transition data</a></td>
          <td>
            
              Smita Krishnaswamy,
            
              Nevena Zivanovic,
            
              Roshan Sharma,
            
              D. Pe’er,
            
              B. Bodenmiller,
            
          </td>
          <td>2018-10-29</td>
          <td>PLoS ONE</td>
          <td>17</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94e0c5b57a795f63319fb45e2716e458007292e9" target='_blank'>Integrating pharmacogenomics and cheminformatics with diverse disease phenotypes for cell type-guided drug discovery</a></td>
          <td>
            
              A. Halu,
            
              J. Decano,
            
              J. Matamalas,
            
              Mary Whelan,
            
              Takaharu Asano,
            
              Namitra Kalicharran,
            
              Sasha A. Singh,
            
              J. Loscalzo,
            
              M. Aikawa,
            
          </td>
          <td>2022-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>137</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5233a066fa24cb9ac285768e198c3e42ec0525b5" target='_blank'>TranSynergy: Mechanism-driven interpretable deep neural network for the synergistic prediction and pathway deconvolution of drug combinations</a></td>
          <td>
            
              Qiao Liu,
            
              Lei Xie,
            
          </td>
          <td>2020-07-10</td>
          <td>PLoS Computational Biology</td>
          <td>80</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b85624cb339f019cfcd8d3c609f17b18c23655ad" target='_blank'>Predicting mechanism of action of cellular perturbations with pathway activity signatures</a></td>
          <td>
            
              Yan Ren,
            
              S. Sivaganesan,
            
              Nicholas A. Clark,
            
              Lixia Zhang,
            
              J. Biesiada,
            
              W. Niu,
            
              D. Plas,
            
              M. Medvedovic,
            
          </td>
          <td>2019-07-16</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>49</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/826ecb9ee14f06b6b3b341db488a16cd2170890c" target='_blank'>Experimental design schemes for learning Boolean network models</a></td>
          <td>
            
              N. Atias,
            
              M. Gershenzon,
            
              Katia Labazin,
            
              R. Sharan,
            
          </td>
          <td>2014-08-22</td>
          <td>Bioinformatics</td>
          <td>13</td>
          <td>68</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ee320fb74e14f0baa7f492bbf9fa2da3c8106a62" target='_blank'>Interpretable deep learning architectures for improving drug response prediction performance: myth or reality?</a></td>
          <td>
            
              Yihui Li,
            
              D. Hostallero,
            
              A. Emad,
            
          </td>
          <td>2023-02-26</td>
          <td>Bioinformatics</td>
          <td>2</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a349e801b1ab0baab515f790c948d51421d9a4ac" target='_blank'>Interpretable deep learning of label-free live cell images uncovers functional hallmarks of highly-metastatic melanoma</a></td>
          <td>
            
              A. Zaritsky,
            
              A. Jamieson,
            
              Erik S. Welf,
            
              Andres Nevarez,
            
              Justin Cillay,
            
              U. Eskiocak,
            
              B. Cantarel,
            
              G. Danuser,
            
          </td>
          <td>2020-05-15</td>
          <td>bioRxiv</td>
          <td>11</td>
          <td>76</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c55e590f7a8a711282e4c359acea97fd05ee2e35" target='_blank'>Identifying Drug Effects via Pathway Alterations using an Integer Linear Programming Optimization Formulation on Phosphoproteomic Data</a></td>
          <td>
            
              A. Mitsos,
            
              Ioannis N. Melas,
            
              Paraskeuas Siminelakis,
            
              A. Chairakaki,
            
              J. Sáez-Rodríguez,
            
              L. Alexopoulos,
            
          </td>
          <td>2009-12-01</td>
          <td>PLoS Computational Biology</td>
          <td>133</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6d0d7ca8fdbc726fad17ba68be1dc6875b02c3d1" target='_blank'>BioXNet: a biologically inspired neural network for deciphering anti-cancer drug response in precision medicine</a></td>
          <td>
            
              Jiannan Yang,
            
              William Ka Kei Wu,
            
              Rina Yee Man Hui,
            
              Ian Chi Kei Wong,
            
              Qingpeng Zhang,
            
          </td>
          <td>2024-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f4542df1391d52171e6193c5d4b5e313b6d1f8fc" target='_blank'>Data-Driven Identiﬁcation of Cancer Cell Population Dynamics Leveraging the Effect of Pre-Treatment for Drug Schedule Design</a></td>
          <td>
            
              Marius Wiggert,
            
              Z. Cohen,
            
              Megan Turnidge,
            
              Ellen M. Langer,
            
              R. Sears,
            
              Margaret P. Chapman,
            
              C. Tomlin,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>78</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/593d758a97cdca686117bc33080e1561ea0705f2" target='_blank'>MultiCPA: Multimodal Compositional Perturbation Autoencoder</a></td>
          <td>
            
              Kemal Inecik,
            
              Andreas Uhlmann,
            
              M. Lotfollahi,
            
              F. Theis,
            
          </td>
          <td>2022-07-10</td>
          <td>bioRxiv</td>
          <td>6</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3b3ac5127e0d6d665398157f9621b19db7769537" target='_blank'>Cancer: A Computational Disease that AI Can Cure</a></td>
          <td>
            
              J. Tenenbaum,
            
              Jeff Shrager,
            
          </td>
          <td>2011-06-05</td>
          <td>AI Mag.</td>
          <td>15</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/45f13af763f21f322be1383ebd6a0fd02d49c134" target='_blank'>Paradoxical results in perturbation-based signaling network reconstruction.</a></td>
          <td>
            
              S. Prabakaran,
            
              J. Gunawardena,
            
              Eduardo Sontag,
            
          </td>
          <td>2014-06-01</td>
          <td>Biophysical journal</td>
          <td>30</td>
          <td>95</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/733a5c37291732e0d4d3c42e02327e428a07439b" target='_blank'>Causal Interactions from Proteomic Profiles: Molecular Data Meets Pathway Knowledge</a></td>
          <td>
            
              Özgün Babur,
            
              Augustin Luna,
            
              Anil Korkut,
            
              Funda Durupinar,
            
              M. Siper,
            
              U. Dogrusoz,
            
              Alvaro Sebastian Vaca Jacome,
            
              Ryan Peckner,
            
              K. E. Christianson,
            
              Jacob D. Jaffe,
            
              Paul T. Spellman,
            
              Joseph E. Aslan,
            
              C. Sander,
            
              E. Demir,
            
          </td>
          <td>2020-10-23</td>
          <td>SSRN Electronic Journal</td>
          <td>0</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bb451794313b932064dd550f5a6e928eb21d64df" target='_blank'>Integration of Drug Perturbed Dose Dependent Response Data in Predictive Computational Models of Chemotherapeutic Response Using Multiple‐Kernel‐Learning</a></td>
          <td>
            
              Joshua D. Mannheimer,
            
              A. Prasad,
            
              D. Gustafson,
            
          </td>
          <td>2020-04-01</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7411b1effb6d89c638ee8e1aa6142d3e48609297" target='_blank'>Using computational modeling and experimental synthetic perturbations to probe biological circuits.</a></td>
          <td>
            
              Joshua R. Porter,
            
              Eric Batchelor,
            
          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>4</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/93da388399e3aad95c8bb229f8f792d9ea241d80" target='_blank'>Robust network inference using response logic</a></td>
          <td>
            
              Torsten Gross,
            
              M. Wongchenko,
            
              Yibing Yan,
            
              Nils Blüthgen,
            
          </td>
          <td>2019-02-12</td>
          <td>Bioinformatics</td>
          <td>7</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d14407c282b9df499c35323b905fde042c651558" target='_blank'>Decoding Heterogenous Single-cell Perturbation Responses</a></td>
          <td>
            
              Bicna Song,
            
              Dingyu Liu,
            
              Weiwei Dai,
            
              Natalie F. McMyn,
            
              Qingyang Wang,
            
              Dapeng Yang,
            
              Adam Krejci,
            
              A. Vasilyev,
            
              Nicole Untermoser,
            
              Anke Loregger,
            
              Dongyuan Song,
            
              Breanna Williams,
            
              Bess P. Rosen,
            
              Xiaolong Cheng,
            
              Lumen Chao,
            
              Hanuman Kale,
            
              Hao Zhang,
            
              Yarui Diao,
            
              T. Bürckstümmer,
            
              Jenet M. Siliciano,
            
              J. J. Li,
            
              R. Siliciano,
            
              Danwei Huangfu,
            
              Wei Li,
            
          </td>
          <td>2023-11-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>110</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0cff4da1b6d07f709aa4bccc536dee490961087e" target='_blank'>Integrating Transcriptomic Data with Mechanistic Systems Pharmacology Models for Virtual Drug Combination Trials.</a></td>
          <td>
            
              A. Barrette,
            
              M. Bouhaddou,
            
              M. Birtwistle,
            
          </td>
          <td>2017-10-06</td>
          <td>ACS chemical neuroscience</td>
          <td>18</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/335745be9347aab6067811287f1f1ede86b7f9da" target='_blank'>DeepNoise: Disentanglement of Experimental Noise from Real Biological Signals based on Fluorescent Microscopy Image Classiﬁcation via Deep Learning</a></td>
          <td>
            
              Sen Yang,
            
              Tao Shen,
            
              Yu-Qiang Fang,
            
              Xiyue Wang,
            
              Jun Zhang,
            
              Wei Yang,
            
              Junzhou Huang,
            
              Xiao Han,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>73</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/17c5519680e8756bc84fe34f73e47bb94fd131e5" target='_blank'>L1 regularization facilitates detection of cell type-specific parameters in dynamical systems</a></td>
          <td>
            
              Bernhard Steiert,
            
              J. Timmer,
            
              C. Kreutz,
            
          </td>
          <td>2016-08-29</td>
          <td>Bioinformatics</td>
          <td>20</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a02ba4850f89eb5c7e49854ac0753f4cedd17e0b" target='_blank'>Personalized single-cell networks: a framework to predict the response of any gene to any drug for any patient</a></td>
          <td>
            
              Haripriya Harikumar,
            
              T. Quinn,
            
              Santu Rana,
            
              Sunil Gupta,
            
              S. Venkatesh,
            
          </td>
          <td>2020-08-24</td>
          <td>BioData Mining</td>
          <td>1</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e6de630ffb762090f2eafd68f1e4dc3b97cf5240" target='_blank'>Dynamic probabilistic threshold networks to infer signaling pathways from time-course perturbation data</a></td>
          <td>
            
              N. Kiani,
            
              L. Kaderali,
            
          </td>
          <td>2014-07-22</td>
          <td>BMC Bioinformatics</td>
          <td>14</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e6de630ffb762090f2eafd68f1e4dc3b97cf5240" target='_blank'>Dynamic probabilistic threshold networks to infer signaling pathways from time-course perturbation data</a></td>
          <td>
            
              N. Kiani,
            
              L. Kaderali,
            
          </td>
          <td>2014-07-22</td>
          <td>BMC Bioinformatics</td>
          <td>14</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/050dcd5bdd87453ed8e64ee017bbf1fe0971646b" target='_blank'>Improving Accuracy in Cell-Perturbation Experiments by Leveraging Auxiliary Information</a></td>
          <td>
            
              J. Loper,
            
              Noam Solomon,
            
              J. Regier,
            
          </td>
          <td>2023-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4fb773807d137be9f32cf9cbcdffe2acd7f6d815" target='_blank'>Efficient computation of minimal perturbation sets in gene regulatory networks</a></td>
          <td>
            
              A. Garg,
            
              K. Mohanram,
            
              Alessandro Di Cara,
            
              G. Degueurce,
            
              M. Ibberson,
            
              Julien Dorier,
            
              I. Xenarios,
            
          </td>
          <td>2013-08-31</td>
          <td>Frontiers in Physiology</td>
          <td>10</td>
          <td>75</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78a259b7fe1cfc80201024e23e308961dd1024b1" target='_blank'>BDKANN – Biological Domain Knowledge-based Artificial Neural Network for drug response prediction</a></td>
          <td>
            
              Oliver Snow,
            
              Hossein Sharifi-Noghabi,
            
              Jialin Lu,
            
              O. Zolotareva,
            
              Mark Lee,
            
              M. Ester,
            
          </td>
          <td>2019-11-13</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94635adf1310128039a73d1b311b9ae88e4f0f1e" target='_blank'>Computational models of melanoma</a></td>
          <td>
            
              M. Albrecht,
            
              P. Lucarelli,
            
              D. Kulms,
            
              T. Sauter,
            
          </td>
          <td>2020-05-14</td>
          <td>Theoretical Biology & Medical Modelling</td>
          <td>9</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94635adf1310128039a73d1b311b9ae88e4f0f1e" target='_blank'>Computational models of melanoma</a></td>
          <td>
            
              M. Albrecht,
            
              P. Lucarelli,
            
              D. Kulms,
            
              T. Sauter,
            
          </td>
          <td>2020-05-14</td>
          <td>Theoretical Biology & Medical Modelling</td>
          <td>9</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8f8e502757b08ccf9384000c5ff7a28a9e45df22" target='_blank'>Discovering the mechanism of action of drugs with a sparse explainable network</a></td>
          <td>
            
              Katyna Sada Del Real,
            
              A. Rubio,
            
          </td>
          <td>2023-08-24</td>
          <td>eBioMedicine</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0937219158373bdfc74db0e4bcf254133e08411f" target='_blank'>Learning a nonlinear dynamical system model of gene regulation: A perturbed steady-state approach</a></td>
          <td>
            
              Arwen V. Bradley,
            
              Ye Henry Li,
            
              Bokyung Choi,
            
              W. Wong,
            
          </td>
          <td>2012-07-13</td>
          <td>arXiv: Molecular Networks</td>
          <td>19</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2efc878148dcec184b46a5eef061d772272652d3" target='_blank'>Deterministic Effects Propagation Networks for reconstructing protein signaling networks from multiple interventions</a></td>
          <td>
            
              H. Fröhlich,
            
              Ö. Sahin,
            
              D. Arlt,
            
              C. Bender,
            
              T. Beissbarth,
            
          </td>
          <td>2009-10-08</td>
          <td>BMC Bioinformatics</td>
          <td>30</td>
          <td>50</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cc9916e2fc6064bdb5ef11f4f091e9d45235adbd" target='_blank'>Reconstruction of Cellular Signal Transduction Networks Using Perturbation Assays and Linear Programming</a></td>
          <td>
            
              Bettina Knapp,
            
              L. Kaderali,
            
          </td>
          <td>2013-07-30</td>
          <td>PLoS ONE</td>
          <td>13</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f95f5827a8af08ab6f2d79b5afa1d707af9950de" target='_blank'>Prediction in the face of uncertainty: a Monte Carlo-based approach for systems biology of cancer treatment.</a></td>
          <td>
            
              C. Wierling,
            
              Alexander Kühn,
            
              Hendrik Hache,
            
              A. Daskalaki,
            
              Elisabeth Maschke-Dutz,
            
              S. Peycheva,
            
              Jian Li,
            
              R. Herwig,
            
              H. Lehrach,
            
          </td>
          <td>2012-08-15</td>
          <td>Mutation research</td>
          <td>30</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1bd61a55ff09cce9029c90dd6900d4b45278f0b4" target='_blank'>In silico modeling predicts drug sensitivity of patient-derived cancer cells</a></td>
          <td>
            
              S. Pingle,
            
              Zeba Sultana,
            
              S. Pastorino,
            
              Pengfei Jiang,
            
              Rajesh Mukthavaram,
            
              Ying S. Chao,
            
              I. Bharati,
            
              Natsuko Nomura,
            
              M. Makale,
            
              T. Abbasi,
            
              S. Kapoor,
            
              Ansu Kumar,
            
              Shahabuddin Usmani,
            
              A. Agrawal,
            
              S. Vali,
            
              S. Kesari,
            
          </td>
          <td>2014-05-21</td>
          <td>Journal of Translational Medicine</td>
          <td>28</td>
          <td>70</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/748f9da063fa715fefeaa0bfbccf264963bf4d60" target='_blank'>Learning and actioning general principles of cancer cell drug sensitivity</a></td>
          <td>
            
              Francesco Carli,
            
              Pierluigi Di Chiaro,
            
              M. Morelli,
            
              Chakit Arora,
            
              Luisa Bisceglia,
            
              Natalia De Oliveira Rosa,
            
              Alice Cortesi,
            
              S. Franceschi,
            
              F. Lessi,
            
              A. D. Di Stefano,
            
              O. Santonocito,
            
              Francesco Pasqualetti,
            
              P. Aretini,
            
              Pasquale Miglionico,
            
              G. Diaferia,
            
              Fosca Giannotti,
            
              Pietro Liò,
            
              Miquel Duran-Frigola,
            
              C. M. Mazzanti,
            
              Gioacchino Natoli,
            
              Francesco Raimondi,
            
          </td>
          <td>2024-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f95550e17d2863ce352419b4c2f62ecca35a535" target='_blank'>Abstract B2-41: Inference of differential dependency networks between tumor and stroma by integrated UNDO-KDDN software tool</a></td>
          <td>
            
              Lulu Chen,
            
              Niya Wang,
            
              R. Clarke,
            
              Zhen Zhang,
            
              Yue Wang,
            
          </td>
          <td>2015-11-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aed89b87e3cc585d11f5420aea7683c047a83cb6" target='_blank'>Improvement of experimental testing and network training conditions with genome-wide microarrays for more accurate predictions of drug gene targets</a></td>
          <td>
            
              Lisa M. Christadore,
            
              Lisa Pham,
            
              E. Kolaczyk,
            
              S. Schaus,
            
          </td>
          <td>2014-01-20</td>
          <td>BMC Systems Biology</td>
          <td>2</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ab17295acf0d21092af0fd7d4621a2c9eced393" target='_blank'>Abstract C176: Cancer-specific AI identifies multi-modal biomarkers of therapeutic response for 1,951 drugs including TNG348, a highly selective USP1 inhibitor</a></td>
          <td>
            
              Adam Yaari,
            
              Lee McDaniel,
            
              Antoine Simoneau,
            
              Samuel R Meier,
            
              Oliver Priebe,
            
              Eduardo Farias,
            
              A. Huang,
            
              Jannik N Andersen,
            
              Yi Yu,
            
              Maxwell A. Sherman,
            
          </td>
          <td>2023-12-01</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6271a8d1a9451e165feb2a8b3c46db75b899f55f" target='_blank'>Distangling Biological Noise in Cellular Images with a focus on Explainability</a></td>
          <td>
            
              Manik Sharma,
            
              Ganapathy Krishnamurthi,
            
          </td>
          <td>2020-07-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7666bbb23f840fd20fd9ba11365aa9e2dcc38cec" target='_blank'>Revealing the Impact of Genomic Alterations on Cancer Cell Signaling with an Interpretable Deep Learning Model</a></td>
          <td>
            
              Jonathan D. Young,
            
              Shuangxia Ren,
            
              Lujia Chen,
            
              Xinghua Lu,
            
          </td>
          <td>2023-07-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ea5af003d4cfdb49de74af10115d114b397967c" target='_blank'>Predicting activatory and inhibitory drug–target interactions based on structural compound representations and genetically perturbed transcriptomes</a></td>
          <td>
            
              Won-Yung Lee,
            
              Choong-Yeol Lee,
            
              C. Kim,
            
          </td>
          <td>2023-04-12</td>
          <td>PLOS ONE</td>
          <td>1</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ee6f6351a8308cbb254858debbab622196798436" target='_blank'>Expectation propagation for large scale Bayesian inference of non-linear molecular networks from perturbation data</a></td>
          <td>
            
              Z. Narimani,
            
              H. Beigy,
            
              Ashar Ahmad,
            
              A. Masoudi-Nejad,
            
              H. Fröhlich,
            
          </td>
          <td>2017-02-06</td>
          <td>PLoS ONE</td>
          <td>5</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b6eb03c66d700dbbea0473e7de6144bf037edc0" target='_blank'>Computationally predicting clinical drug combination efficacy with cancer cell line screens and independent drug action</a></td>
          <td>
            
              Alexander L. Ling,
            
              R. Huang,
            
          </td>
          <td>2020-11-17</td>
          <td>Nature Communications</td>
          <td>32</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e19082c5e115a6c5dadc6055444e9505afb315e1" target='_blank'>Classification of Single-Cell Gene Expression Trajectories from Incomplete and Noisy Data</a></td>
          <td>
            
              Alireza Karbalayghareh,
            
              U. Braga-Neto,
            
              E. Dougherty,
            
          </td>
          <td>2019-01-01</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>10</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9b4488d13f45fd490582e6d6ad558b49a1162460" target='_blank'>Data-Derived Modeling Characterizes Plasticity of MAPK Signaling in Melanoma</a></td>
          <td>
            
              Martí Bernardo-Faura,
            
              S. Massen,
            
              C. Falk,
            
              N. Brady,
            
              R. Eils,
            
          </td>
          <td>2014-09-01</td>
          <td>PLoS Computational Biology</td>
          <td>20</td>
          <td>96</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/01e80c3379b36d1216af733a2dccc03124f45526" target='_blank'>NEMix: Single-cell Nested Effects Models for Probabilistic Pathway Stimulation</a></td>
          <td>
            
              Juliane Siebourg-Polster,
            
              D. Mudrak,
            
              M. Emmenlauer,
            
              Pauli Rämö,
            
              C. Dehio,
            
              U. Greber,
            
              H. Fröhlich,
            
              N. Beerenwinkel,
            
          </td>
          <td>2015-04-01</td>
          <td>PLoS Computational Biology</td>
          <td>17</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ab7248d46ef069e3633f0114b71e9a4e8722b09b" target='_blank'>Success in the DREAM3 Signaling Response Challenge Using Simple Weighted-Average Imputation: Lessons for Community-Wide Experiments in Systems Biology</a></td>
          <td>
            
              N. Clarke,
            
              G. Bourque,
            
          </td>
          <td>2010-01-26</td>
          <td>PLoS ONE</td>
          <td>3</td>
          <td>68</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4a4c4caba7eb86c63e0c07ee1a475ef8ee660247" target='_blank'>Predicting drug polypharmacology from cell morphology readouts using variational autoencoder latent space arithmetic</a></td>
          <td>
            
              Yuen Ler Chow,
            
              Shantanu Singh,
            
              Anne E Carpenter,
            
              G. Way,
            
          </td>
          <td>2021-09-03</td>
          <td>PLoS Computational Biology</td>
          <td>17</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d59c26f7643ea7ffa2302cef38d761dea15a0da4" target='_blank'>rediction in the face of uncertainty : A Monte Carlo-based approach for systems iology of cancer treatment</a></td>
          <td>
            
              hristoph Wierling,
            
              Alexander Kühn,
            
              Hendrik Hache,
            
              A. Daskalaki,
            
              Elisabeth Maschke-Dutz,
            
              V. Peycheva,
            
              Jian Li,
            
              R. Herwig,
            
              H. Lehrach,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a195afa11b49103054c41ab700a3ae097fccc647" target='_blank'>DeepNoise: Signal and Noise Disentanglement Based on Classifying Fluorescent Microscopy Images via Deep Learning</a></td>
          <td>
            
              Sen Yang,
            
              Tao Shen,
            
              Yu-Qiang Fang,
            
              Xiyue Wang,
            
              Jun Zhang,
            
              Wei Yang,
            
              Junzhou Huang,
            
              Xiao Han,
            
          </td>
          <td>2022-09-13</td>
          <td>Genomics, Proteomics & Bioinformatics</td>
          <td>2</td>
          <td>73</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/771e6ee84353828e640c230c07c64013c8803ab5" target='_blank'>Leveraging multi-omics data to empower quantitative systems pharmacology in immuno-oncology</a></td>
          <td>
            
              Theinmozhi Arulraj,
            
              Hanwen Wang,
            
              Alberto Ippolito,
            
              Shuming Zhang,
            
              E. Fertig,
            
              Aleksander S Popel,
            
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9c2744b348ddc2249768eee40b7f73ce2479c807" target='_blank'>A subcomponent-guided deep learning method for interpretable cancer drug response prediction</a></td>
          <td>
            
              Xuan Liu,
            
              Wen Zhang,
            
          </td>
          <td>2023-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>5</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/042824e893f3eaefbb19f8bce7552c903746a523" target='_blank'>Enhancing drug and cell line representations via contrastive learning for improved anti-cancer drug prioritization</a></td>
          <td>
            
              Patrick J. Lawrence,
            
              Xia Ning,
            
          </td>
          <td>2023-10-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c11e11ef48b7a694b15aa88af2b752dafa0c85e6" target='_blank'>Learning a nonlinear gene regulation model from perturbed steady-state data</a></td>
          <td>
            
              Arwen Bradley,
            
              Ye,
            
              Li,
            
              Bokyung Choi,
            
              W. Wong,
            
          </td>
          <td>2012-07-13</td>
          <td>None</td>
          <td>1</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9b1aa0257278c653d78d48548bba075f76242a29" target='_blank'>A genome-scale deep learning model to predict gene expression changes of genetic perturbations from multiplex biological networks</a></td>
          <td>
            
              Lingmin Zhan,
            
              Yuanyuan Zhang,
            
              Yingdong Wang,
            
              Aoyi Wang,
            
              Caiping Cheng,
            
              Jinzhong Zhao,
            
              Wuxia Zhang,
            
              Peng Lia,
            
              Jianxin Chen,
            
          </td>
          <td>2024-03-05</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8992a8e22af28309ac95a3afacd04d412d27a28c" target='_blank'>Pisces: A multi-modal data augmentation approach for drug combination synergy prediction</a></td>
          <td>
            
              Jiacheng Lin,
            
              Hanwen Xu,
            
              Addie Woicik,
            
              Jianzhu Ma,
            
              Sheng Wang,
            
          </td>
          <td>2023-08-10</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/881b3732b72fc42968e8ccda0c256f3f5d3a9540" target='_blank'>Analysis and modeling of cancer drug responses using cell cycle phase-specific rate effects</a></td>
          <td>
            
              Sean M. Gross,
            
              Farnaz Mohammadi,
            
              Crystal Sanchez-Aguila,
            
              Paulina J. Zhan,
            
              T. Liby,
            
              Mark A. Dane,
            
              Aaron S. Meyer,
            
              Laura M. Heiser,
            
          </td>
          <td>2020-07-25</td>
          <td>Nature Communications</td>
          <td>6</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ab92f29852d3119f63ff317276753feed93d951b" target='_blank'>SpaRx: Elucidate single-cell spatial heterogeneity of drug responses for personalized treatment</a></td>
          <td>
            
              Ziyang Tang,
            
              Xiang Liu,
            
              Zuotian Li,
            
              Tonglin Zhang,
            
              B. Yang,
            
              Jing Su,
            
              Qianqian Song,
            
          </td>
          <td>2023-08-06</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ab353723dbf21ca46b048821a572c5a18cc80468" target='_blank'>Better Accuracy in Predicting Efficacy of Drug Combinations Using Novel Algorithm</a></td>
          <td>
            
              Santeri Mentu,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/450f5f482ea3fb72782a376883f3744087aa3279" target='_blank'>Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data</a></td>
          <td>
            
              Aditya Pratapa,
            
              A. Jalihal,
            
              Jeffrey N. Law,
            
              Aditya Bharadwaj,
            
              T. Murali,
            
          </td>
          <td>2019-05-20</td>
          <td>Nature methods</td>
          <td>388</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f945e25ec7be816d9c187081af84c5ef6dd91193" target='_blank'>DIVERSE: Bayesian Data IntegratiVE Learning for Precise Drug ResponSE Prediction</a></td>
          <td>
            
              Betul Guvencpaltun,
            
              Samuel Kaski,
            
              Hiroshi Mamitsuka,
            
          </td>
          <td>2021-03-11</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>7</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5917e98fde69eab5466607e591165a98856a0b7f" target='_blank'>From Years to Hours: Accelerating Model Refinement</a></td>
          <td>
            
              Kyu Hyong Park,
            
              Jordan C. Rozum,
            
              Réka Albert,
            
          </td>
          <td>2023-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d8b53c25d331df61bf601a33636784d04ce05d68" target='_blank'>Learning interpretable cellular embedding for inferring biological mechanisms underlying single-cell transcriptomics</a></td>
          <td>
            
              k.-l. Hsieh,
            
              Y. Chu,
            
              P. G. Pilie,
            
              K. Zhang,
            
          </td>
          <td>2024-04-01</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6ecd06ade454c2179638f764083b994b87b0b456" target='_blank'>Manifold learning-based methods for analyzing single-cell RNA-sequencing data</a></td>
          <td>
            
              Kevin R. Moon,
            
              Jay S. Stanley,
            
              Daniel B. Burkhardt,
            
              D. V. Dijk,
            
              Guy Wolf,
            
              Smita Krishnaswamy,
            
          </td>
          <td>2018-02-01</td>
          <td>Current Opinion in Systems Biology</td>
          <td>119</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/146b4e73d93713b9c0fdbadee586337f5405e90c" target='_blank'>A new approach for prediction of tumor sensitivity to targeted drugs based on functional data</a></td>
          <td>
            
              Noah E. Berlow,
            
              L. Davis,
            
              Emma L. Cantor,
            
              B. Séguin,
            
              C. Keller,
            
              R. Pal,
            
          </td>
          <td>2013-07-29</td>
          <td>BMC Bioinformatics</td>
          <td>38</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4b4fd91ed2aa02549d7afea9ab48049f0c6cabcc" target='_blank'>Interpretable Drug Response Prediction through Molecule Structure-aware and Knowledge-Guided Visible Neural Network</a></td>
          <td>
            
              Jiancong Xie,
            
              Zhe Zhang,
            
              Youyou Li,
            
              Jiahua Rao,
            
              Yuedong Yang,
            
          </td>
          <td>2024-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f9b4ba528526bb4328fc81d6fd71855cf3d45b87" target='_blank'>WENDY: Gene Regulatory Network Inference with Covariance Dynamics</a></td>
          <td>
            
              Yue Wang,
            
              Peng Zheng,
            
              ,
            
              Zikun Wang,
            
              Aleksandr Y Aravkin,
            
          </td>
          <td>2024-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b6e5742338cbe9f6e02c880cf25e77d846a16982" target='_blank'>Effective Combination Therapies for B-cell Lymphoma Predicted by a Virtual Disease Model.</a></td>
          <td>
            
              W. Du,
            
              R. Goldstein,
            
              Yanwen Jiang,
            
              Omar Aly,
            
              L. Cerchietti,
            
              A. Melnick,
            
              O. Elemento,
            
          </td>
          <td>2017-04-15</td>
          <td>Cancer research</td>
          <td>11</td>
          <td>106</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/983711ff24b453116f9f5fd32ac75ea0440f3281" target='_blank'>A deep learning framework for high-throughput mechanism-driven phenotype compound screening and its application to COVID-19 drug repurposing</a></td>
          <td>
            
              Thai-Hoang Pham,
            
              Yue Qiu,
            
              Jucheng Zeng,
            
              Lei Xie,
            
              Ping Zhang,
            
          </td>
          <td>2021-02-01</td>
          <td>Nature machine intelligence</td>
          <td>104</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8ad6da1c4a699af5694d6c6c8e6af1cf5d3855ab" target='_blank'>Genomic data integration by WON-PARAFAC identifies interpretable factors for predicting drug-sensitivity in vivo</a></td>
          <td>
            
              Yongsoo Kim,
            
              Tycho Bismeijer,
            
              W. Zwart,
            
              L. Wessels,
            
              D. Vis,
            
          </td>
          <td>2019-11-06</td>
          <td>Nature Communications</td>
          <td>11</td>
          <td>66</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/498f2bc0606e297f071fdba36db552b3bf11792a" target='_blank'>A systems biology method to predict oncogenes and perturbation targets on a cellular level Technology</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0bdb6a4cfe1eb89e95f23275c2b3b48a8784f5ef" target='_blank'>Abstract 3115: Predicting in-depth mechanism of action of cancer drugs</a></td>
          <td>
            
              Sanju Sinha,
            
              Neelam Sinha,
            
              E. Ruppin,
            
          </td>
          <td>2023-04-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>84</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/00d54fdde2e7f199813e86ab03a7ab6805f23394" target='_blank'>Inferring Signaling Pathway Topologies from Multiple Perturbation Measurements of Specific Biochemical Species</a></td>
          <td>
            
              Tian-Rui Xu,
            
              Vladislav Vyshemirsky,
            
              Amelie Gormand,
            
              Alex von Kriegsheim,
            
              M. Girolami,
            
              G. Baillie,
            
              Dominic Ketley,
            
              Allan J. Dunlop,
            
              Graeme Milligan,
            
              M. Houslay,
            
              Walter Kolch,
            
          </td>
          <td>2010-03-16</td>
          <td>Science Signaling</td>
          <td>133</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2bd89a32034fd6d35c445f014cecc7addf92f16c" target='_blank'>Integration of transcriptomics data into agent-based models of solid tumor metastasis</a></td>
          <td>
            
              Jimmy Retzlaff,
            
              Xin Lai,
            
              C. Berking,
            
              J. Vera,
            
          </td>
          <td>2023-01-10</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>1</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/64c13037fea56c2c15ed18994be3b593d9c5beca" target='_blank'>Massively multiplex chemical transcriptomics at single-cell resolution</a></td>
          <td>
            
              S. Srivatsan,
            
              José L. McFaline-Figueroa,
            
              V. Ramani,
            
              Lauren M. Saunders,
            
              Junyue Cao,
            
              Jonathan S. Packer,
            
              Hannah A. Pliner,
            
              Dana L. Jackson,
            
              R. Daza,
            
              Lena Christiansen,
            
              Fan Zhang,
            
              F. Steemers,
            
              J. Shendure,
            
              C. Trapnell,
            
          </td>
          <td>2019-12-05</td>
          <td>Science</td>
          <td>196</td>
          <td>142</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/95d039d13b016696484b2fc193d1d20de3e9df88" target='_blank'>Modeling of signal-response cascades using decision tree analysis</a></td>
          <td>
            
              S. Hautaniemi,
            
              Sourabh Kharait,
            
              Akihiro Iwabu,
            
              A. Wells,
            
              D. Lauffenburger,
            
          </td>
          <td>2005-05-01</td>
          <td>Bioinformatics</td>
          <td>70</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c00b3fb240007ac9db87c89bbfb3dec6bbfd7f53" target='_blank'>Many are better than one - next generation multivariate biomarkers for precision oncology</a></td>
          <td>
            
              Jinyan Du,
            
              D. Kirouac,
            
              B. Schoeberl,
            
          </td>
          <td>2017-01-09</td>
          <td>Cancer cell & microenvironment</td>
          <td>0</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8eb67e99e76235ceeed881994c5518e18f76a75c" target='_blank'>caspo: a toolbox for automated reasoning on the response of logical signaling networks families</a></td>
          <td>
            
              S. Videla,
            
              J. Sáez-Rodríguez,
            
              Carito Guziolowski,
            
              A. Siegel,
            
          </td>
          <td>2016-11-23</td>
          <td>Bioinformatics</td>
          <td>19</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/488815adde83ad267b507082359af3dfc5928d8c" target='_blank'>CausalBench: A Large-scale Benchmark for Network Inference from Single-cell Perturbation Data</a></td>
          <td>
            
              Mathieu Chevalley,
            
              Yusuf H. Roohani,
            
              Arash Mehrjou,
            
              J. Leskovec,
            
              P. Schwab,
            
          </td>
          <td>2022-10-31</td>
          <td>ArXiv</td>
          <td>13</td>
          <td>132</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/021816e7c8e48f65621ca0035f81296609d11189" target='_blank'>Factoring single-cell perturbations</a></td>
          <td>
            
              Bicna Song,
            
              Wei Li,
            
          </td>
          <td>2023-09-28</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/eb0991882a6b874ca139f6e657cddd25bc92d096" target='_blank'>Deep learning prediction of chemical-induced dose-dependent and context-specific multiplex phenotype responses and its application to personalized alzheimer’s disease drug repurposing</a></td>
          <td>
            
              You Wu,
            
              Qiao Liu,
            
              Yue Qiu,
            
              Lei Xie,
            
          </td>
          <td>2022-08-01</td>
          <td>PLoS Computational Biology</td>
          <td>5</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b9733bae80b05e065eb7ef9698ef331b1d9f378c" target='_blank'>Detecting and Removing Inconsistencies between Experimental Data and Signaling Network Topologies Using Integer Linear Programming on Interaction Graphs</a></td>
          <td>
            
              Ioannis N. Melas,
            
              Regina Samaga,
            
              L. Alexopoulos,
            
              Steffen Klamt,
            
          </td>
          <td>2013-09-01</td>
          <td>PLoS Computational Biology</td>
          <td>40</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f34f3f17f559c88b561928a179247e8c428d6bf3" target='_blank'>Discrete Logic Modelling Optimization to Contextualize Prior Knowledge Networks Using PRUNET</a></td>
          <td>
            
              Ana Rodriguez,
            
              I. Crespo,
            
              Anna Fournier,
            
              A. del Sol,
            
          </td>
          <td>2015-06-09</td>
          <td>PLOS ONE</td>
          <td>18</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f61827b05e3b39d1a6c6640e56addecb0b990c6a" target='_blank'>Systems Perturbation Analysis of a Large-Scale Signal Transduction Model Reveals Potentially Influential Candidates for Cancer Therapeutics</a></td>
          <td>
            
              B. L. Puniya,
            
              Laura Allen,
            
              C. Hochfelder,
            
              M. Majumder,
            
              T. Helikar,
            
          </td>
          <td>2015-11-10</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>18</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fbf7f988fa139f63575bf182bc2f39a8a5ad7dc0" target='_blank'>Drepmel—A Multi-Omics Melanoma Drug Repurposing Resource for Prioritizing Drug Combinations and Understanding Tumor Microenvironment</a></td>
          <td>
            
              Z. Thompson,
            
              J. Teer,
            
              Jiannong Li,
            
              Zhihua Chen,
            
              E. Welsh,
            
              Yonghong Zhang,
            
              N. Ayoubi,
            
              Z. Eroglu,
            
              A. Tan,
            
              K. Smalley,
            
              Y. Chen,
            
          </td>
          <td>2022-09-01</td>
          <td>Cells</td>
          <td>2</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7315a031786d41325855e8fdd3e3e19d12f61053" target='_blank'>Tensor clustering with algebraic constraints gives interpretable groups of crosstalk mechanisms in breast cancer</a></td>
          <td>
            
              A. Seigal,
            
              Mariano Beguerisse-Díaz,
            
              B. Schoeberl,
            
              M. Niepel,
            
              H. Harrington,
            
          </td>
          <td>2016-12-24</td>
          <td>Journal of the Royal Society Interface</td>
          <td>4</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ff7368a1e0e4b7d9367eb3662bef352351c50484" target='_blank'>ADRML: anticancer drug response prediction using manifold learning</a></td>
          <td>
            
              Fatemeh Ahmadi Moughari,
            
              C. Eslahchi,
            
          </td>
          <td>2020-08-28</td>
          <td>Scientific Reports</td>
          <td>9</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/03f97b75d30666cf4e68c25eaea12f1276068bc4" target='_blank'>Systems Perturbation Analysis of a Large-Scale Signal Systems Perturbation Analysis of a Large-Scale Signal Transduction Model Reveals Potentially Influential Candidates for Transduction Model Reveals Potentially Influential Candidates for Cancer Therapeutics Cancer Therapeutics</a></td>
          <td>
            
              D. Rosenblueth,
            
              T. Helikar,
            
              B. L. Puniya,
            
              Laura Allen,
            
              C. Hochfelder,
            
              Mahbubul Majumder,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78ff3585f4dc208ce372acd896a86df979e84b04" target='_blank'>Integrating knowledge and omics to decipher mechanisms via large‐scale models of signaling networks</a></td>
          <td>
            
              Martín Garrido-Rodriguez,
            
              Katharina Zirngibl,
            
              Olga Ivanova,
            
              Sebastian Lobentanzer,
            
              J. Saez-Rodriguez,
            
          </td>
          <td>2022-07-01</td>
          <td>Molecular Systems Biology</td>
          <td>20</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8e7823044dce780bf904b69a96ec44192259f255" target='_blank'>MOViDA: multiomics visible drug activity prediction with a biologically informed neural network model</a></td>
          <td>
            
              Luigi Ferraro,
            
              G. Scala,
            
              L. Cerulo,
            
              E. Carosati,
            
              M. Ceccarelli,
            
          </td>
          <td>2023-04-08</td>
          <td>Bioinformatics</td>
          <td>1</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ce2f254c7a49e1050d4b89c0370089c6605682af" target='_blank'>DTF: Deep Tensor Factorization for predicting anticancer drug synergy</a></td>
          <td>
            
              Zexuan Sun,
            
              Shujun Huang,
            
              Peiran Jiang,
            
              P. Hu,
            
              Zhiyong Lu,
            
          </td>
          <td>2019-11-23</td>
          <td>Bioinformatics</td>
          <td>32</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6657d442f660cdc2a368869097b286ce92592c24" target='_blank'>Abstract 901: BATCHIE: An active learning platform for scalable combination drug screens</a></td>
          <td>
            
              Christopher Tosh,
            
              Mauricio Tec,
            
              Jessica White,
            
              Jeffrey F. Quinn,
            
              G. I. Sanchez,
            
              Paul Calder,
            
              Andrew L Kung,
            
              F. D. Dela Cruz,
            
              Wesley Tansey,
            
          </td>
          <td>2024-03-22</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8024ca052f96abc813fc6e1422975a655c3bf430" target='_blank'>Anti-cancer Drug Synergy Prediction in Understudied Tissues using Transfer Learning</a></td>
          <td>
            
              Yejin Kim,
            
              Shuyu Zheng,
            
              Jing Tang,
            
              W. Jim Zheng,
            
              Z. Li,
            
              Xiaoqian Jiang,
            
          </td>
          <td>2020-02-10</td>
          <td>bioRxiv</td>
          <td>28</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/02c83d0264d5b40f55eecf7088c2b9be85cdc1e6" target='_blank'>GRAND: a database of gene regulatory network models across human conditions</a></td>
          <td>
            
              Marouen Ben Guebila,
            
              C. Lopes-Ramos,
            
              Deborah A. Weighill,
            
              Abhijeet R. Sonawane,
            
              R. Burkholz,
            
              Behrouz Shamsaei,
            
              John Platig,
            
              Kimberly Glass,
            
              M. Kuijjer,
            
              J. Quackenbush,
            
          </td>
          <td>2021-06-18</td>
          <td>Nucleic Acids Research</td>
          <td>33</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/568e9e382ee444e6af59ff748b8852dfde0083da" target='_blank'>SynPathy: Predicting Drug Synergy through Drug-Associated Pathways Using Deep Learning.</a></td>
          <td>
            
              Yi-Ching Tang,
            
              A. Gottlieb,
            
          </td>
          <td>2022-01-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>13</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/51252ab25f8dddb97f2968843edc3ff9df40d369" target='_blank'>AttentionPert: Accurately Modeling Multiplexed Genetic Perturbations with Multi-scale Effects</a></td>
          <td>
            
              Ding Bai,
            
              Caleb N. Ellington,
            
              Shentong Mo,
            
              Le Song,
            
              Eric Xing,
            
          </td>
          <td>2024-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2789cec330a2710c3c420eec8a18d9d536bc4476" target='_blank'>DeepCellState: An autoencoder-based framework for predicting cell type specific transcriptional states induced by drug treatment</a></td>
          <td>
            
              Ramzan Umarov,
            
              Yu Li,
            
              E. Arner,
            
          </td>
          <td>2020-12-15</td>
          <td>PLoS Computational Biology</td>
          <td>10</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/39fc30019bae9a55c3cb4868bc8ed84c9c961a32" target='_blank'>TMOD-04. IMAGE-BASED DRUG RESPONSE PROFILING FROM PEDIATRIC TUMOR CELL SPHEROIDS USING PATIENT-BY-PATIENT DEEP TRANSFER LEARNING</a></td>
          <td>
            
              Yannick Berker,
            
              Dina Elharouni,
            
              H. Peterziel,
            
              I. Oehme,
            
              M. Schlesner,
            
              O. Witt,
            
              S. Oppermann,
            
          </td>
          <td>2021-06-01</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/360fb8ff8dd2318e5f1a96e5150459e24a8e5f06" target='_blank'>Dr.VAE: improving drug response prediction via modeling of drug perturbation effects</a></td>
          <td>
            
              Ladislav Rampášek,
            
              Daniel Hidru,
            
              P. Smirnov,
            
              B. Haibe-Kains,
            
              A. Goldenberg,
            
          </td>
          <td>2019-03-08</td>
          <td>Bioinformatics</td>
          <td>107</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6b87b9af14033e28c6cf715b0c208f557fc365a8" target='_blank'>TRIOMPHE: Transcriptome-Based Inference and Generation of Molecules with Desired Phenotypes by Machine Learning</a></td>
          <td>
            
              Kazuma Kaitoh,
            
              Yoshihiro Yamanishi,
            
          </td>
          <td>2021-09-16</td>
          <td>Journal of chemical information and modeling</td>
          <td>10</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bc07dfb613ccafbc973aca41dde40873fb19fcc2" target='_blank'>Systematic Analysis of Network-driven Adaptive Resistance to CDK4/6 and Estrogen Receptor Inhibition using Meta-Dynamic Network Modelling</a></td>
          <td>
            
              Anthony Hart,
            
              Sung-young Shin,
            
              Leann Nguyen,
            
          </td>
          <td>2023-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3b0b67cec80455c8c57eb95d3c47b569fe510913" target='_blank'>Generative modeling of single-cell gene expression for dose-dependent chemical perturbations</a></td>
          <td>
            
              Omar Kana,
            
              R. Nault,
            
              D. Filipovic,
            
              Daniel Marri,
            
              T. R. Zacharewski,
            
              S. Bhattacharya,
            
          </td>
          <td>2023-08-01</td>
          <td>Patterns</td>
          <td>1</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/28342c03bc2b1a6ad6c9813d920108ed08dcfb6c" target='_blank'>Modeling Signaling Networks</a></td>
          <td>
            
          </td>
          <td>2005-04-26</td>
          <td>Science's STKE</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4fdc98c482246018e95728f46bc4f1cf33a81584" target='_blank'>Decoding Drug Response with Structurized Gridding Map-based Cell Representation.</a></td>
          <td>
            
              Jiayi Yin,
            
              Hanyu Zhang,
            
              Xiuna Sun,
            
              Nanxin You,
            
              M. Mou,
            
              Mingkun Lu,
            
              Ziqi Pan,
            
              Fengcheng Li,
            
              Hongli Li,
            
              Su Zeng,
            
              F. Zhu,
            
          </td>
          <td>2023-12-13</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c15025b2adb84a86f475c1b56dd3fd524013d17b" target='_blank'>Boosting an operator-valued kernel-based model for gene regulatory network inference</a></td>
          <td>
            
              Néhémy Lim,
            
              Y. Şenbabaoğlu,
            
              G. Michailidis,
            
              Florence d'Alché-Buc,
            
          </td>
          <td>2013-06-25</td>
          <td></td>
          <td>0</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/99b84ce390b4d74e38658380a51fcba4ab303453" target='_blank'>UvA-DARE ( Digital Academic Repository ) Cyclic causal discovery from continuous equilibrium data</a></td>
          <td>
            
              Mooij,
            
              A. Nicholson,
            
              Padhraic Smyth,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>74</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ad2a166c1d4fa695617ebab9aae2c11886b6c300" target='_blank'>Pathway-guided deep neural network toward interpretable and predictive modeling of drug sensitivity</a></td>
          <td>
            
              L. Deng,
            
              Yideng Cai,
            
              Wenhao Zhang,
            
              Wenyi Yang,
            
              Bo Gao,
            
              Hui Liu,
            
          </td>
          <td>2020-02-07</td>
          <td>bioRxiv</td>
          <td>27</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/76e33d0ae43f2975586e25f3a91b591e8f0f75c5" target='_blank'>Inferring the Impact of Regulatory Mechanisms that Underpin CD8+ T Cell Control of B16 Tumor Growth In vivo Using Mechanistic Models and Simulation</a></td>
          <td>
            
              D. Klinke,
            
              Qing Wang,
            
          </td>
          <td>2017-01-04</td>
          <td>Frontiers in Pharmacology</td>
          <td>4</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6fc30204f7f20fb3ac4d3ad6f5d04c9cd1102c53" target='_blank'>Signaling network prediction by the Ontology Fingerprint enhanced Bayesian network</a></td>
          <td>
            
              T. Qin,
            
              L. Tsoi,
            
              Kellie J. Sims,
            
              Xinghua Lu,
            
              Jim Zheng,
            
          </td>
          <td>2012-12-17</td>
          <td>BMC Systems Biology</td>
          <td>23</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0babdc5493acddb6091a2c52b73fd07c66b6c1d6" target='_blank'>Community assessment to advance computational prediction of cancer drug combinations in a pharmacogenomic screen</a></td>
          <td>
            
              M. Menden,
            
              Dennis Wang,
            
              M. Mason,
            
              B. Szalai,
            
              K. Bulusu,
            
              Y. Guan,
            
              Thomas V Yu,
            
              Jaewoo Kang,
            
              Minji Jeon,
            
              R. Wolfinger,
            
              Tin Nguyen,
            
              M. Zaslavskiy,
            
              I. S. Jang,
            
              Z. Ghazoui,
            
              M. Ahsen,
            
              Robert M. Vogel,
            
              E. C. Neto,
            
              Thea C. Norman,
            
              E. Tang,
            
              M. Garnett,
            
              G. Y. Veroli,
            
              S. Fawell,
            
              G. Stolovitzky,
            
              J. Guinney,
            
              J. Dry,
            
              J. Saez-Rodriguez,
            
          </td>
          <td>2019-06-17</td>
          <td>Nature Communications</td>
          <td>219</td>
          <td>60</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aec710312a3c7e5e2ef80a2329cf50192cee794c" target='_blank'>Artificial intelligence and machine learning methods in predicting anti-cancer drug combination effects</a></td>
          <td>
            
              Kunjie Fan,
            
              Lijun Cheng,
            
              Lang Li,
            
          </td>
          <td>2021-08-04</td>
          <td>Briefings in bioinformatics</td>
          <td>15</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/84ce858182e9aaf67d340b7ba0e45b93ad39db98" target='_blank'>scPreGAN, a deep generative model for predicting the response of single-cell expression to perturbation</a></td>
          <td>
            
              Xiajie Wei,
            
              Jiayi Dong,
            
              Fei Wang,
            
          </td>
          <td>2022-05-26</td>
          <td>Bioinformatics</td>
          <td>9</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2fe64c91814341724c21c20e658fd22703247518" target='_blank'>Computational Analyses Connect Small-Molecule Sensitivity to Cellular Features Using Large Panels of Cancer Cell Lines.</a></td>
          <td>
            
              M. Rees,
            
              B. Seashore-Ludlow,
            
              P. Clemons,
            
          </td>
          <td>2018-12-06</td>
          <td>Methods in molecular biology</td>
          <td>3</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a6d6a10ed918603f5e7a996fdba1959832bcec0f" target='_blank'>Using interpretable deep learning to model cancer dependencies</a></td>
          <td>
            
              Chih-Hsu Lin,
            
              O. Lichtarge,
            
          </td>
          <td>2021-05-27</td>
          <td>Bioinformatics</td>
          <td>10</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3e476957d77a3dde1b293f3149d34085b8ee0ebc" target='_blank'>Abstract 357: A systems biology approach for rational molecular network inference</a></td>
          <td>
            
              M. Egbert,
            
              Michelle L. Wynn,
            
              Zhi-fen Wu,
            
              R. Gilani,
            
              S. Schnell,
            
              S. Merajver,
            
          </td>
          <td>2014-10-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7080c36f133967910374641c206e7ed321c2dd8c" target='_blank'>Improved prediction of gene expression through integrating cell signalling models with machine learning</a></td>
          <td>
            
              Nada Al taweraqi,
            
              R. King,
            
          </td>
          <td>2022-08-06</td>
          <td>BMC Bioinformatics</td>
          <td>1</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e14e509ee0d9711fbc383045b40b46a8c850df24" target='_blank'>Abstract SY17-02: A systems approach to understanding tumor specific drug response</a></td>
          <td>
            
              U. Akavia,
            
              O. Litvin,
            
              Felix Sanchez-Garcia,
            
              Helen C. Dylan Kotliar,
            
              Jessica J. Kim,
            
              L. Garraway,
            
              D. Pe’er,
            
          </td>
          <td>2011-04-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>130</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/50711745edbaf746356e2ac92a1859a74032430e" target='_blank'>Automation of Biological Model Learning, Design and Analysis</a></td>
          <td>
            
              Nataša Miškov-Živanov,
            
          </td>
          <td>2015-05-20</td>
          <td>Proceedings of the 25th edition on Great Lakes Symposium on VLSI</td>
          <td>8</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/852bcc3cc41df93e079582a2ef3c40689ff8411a" target='_blank'>Black Box FDR</a></td>
          <td>
            
              Wesley Tansey,
            
              Yixin Wang,
            
              D. Blei,
            
              R. Rabadán,
            
          </td>
          <td>2018-06-08</td>
          <td>None</td>
          <td>18</td>
          <td>96</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/329354c7d3e48d085e2e40d22c8a53393d595443" target='_blank'>Abstract A14: Modeling drug combination sensitivity with Hopfield networks and transcriptomics data</a></td>
          <td>
            
              C. Piermarocchi,
            
              Sergii Domanskyi,
            
              A. Hakansson,
            
              G. Paternostro,
            
          </td>
          <td>2020-06-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/61938257dcd466f868c5c81ac5b2eebec0fdd0ee" target='_blank'>Deep Learning Applications for Predicting Pharmacological Properties of Drugs and Drug Repurposing Using Transcriptomic Data.</a></td>
          <td>
            
              A. Aliper,
            
              S. Plis,
            
              Artem V. Artemov,
            
              Alvaro E. Ulloa,
            
              Polina Mamoshina,
            
              A. Zhavoronkov,
            
          </td>
          <td>2016-06-08</td>
          <td>Molecular pharmaceutics</td>
          <td>381</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/63e033d7b61a3a772056fbd0a97aa5f7a9a75815" target='_blank'>Assembling, analyzing, refining, and cataloging molecular interaction networks</a></td>
          <td>
            
              H. C. Mak,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/71474075e227985f9331a2ff965c3e5557d1bab5" target='_blank'>Automated Design of Synthetic Cell Classifier Circuits Using a Two-Step Optimization Strategy.</a></td>
          <td>
            
              P. Mohammadi,
            
              N. Beerenwinkel,
            
              Y. Benenson,
            
          </td>
          <td>2017-02-22</td>
          <td>Cell systems</td>
          <td>33</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7929604c7e4f58db52122f7895e19a4a46d7d0ec" target='_blank'>Identification of optimal drug combinations targeting cellular networks: integrating phospho-proteomics and computational network analysis.</a></td>
          <td>
            
              S. Iadevaia,
            
              Yiling Lu,
            
              F. C. Morales,
            
              G. Mills,
            
              Prahlad T. Ram,
            
          </td>
          <td>2010-09-01</td>
          <td>Cancer research</td>
          <td>221</td>
          <td>184</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6e42b1abaf396b64593e432eb236c0268a99d43b" target='_blank'>Erratum to: GraphSynergy: a network-inspired deep learning model for anticancer drug combination prediction</a></td>
          <td>
            
              Jiannan Yang,
            
              Zhongzhi Xu,
            
              W. Wu,
            
              Qian Chu,
            
              Qingpeng Zhang,
            
          </td>
          <td>2021-10-15</td>
          <td>Journal of the American Medical Informatics Association : JAMIA</td>
          <td>3</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d77cc87500842f15984fa04644f5b52f8b1e4367" target='_blank'>Network-Based Systems Analysis Explains Sequence-Dependent Synergism of Bortezomib and Vorinostat in Multiple Myeloma</a></td>
          <td>
            
              Charvi Nanavati,
            
              D. Mager,
            
          </td>
          <td>2021-08-17</td>
          <td>The AAPS Journal</td>
          <td>2</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/873fccf9e6cd078a3443bc485c0869ff9b3e92d4" target='_blank'>Homologous control of protein signaling networks.</a></td>
          <td>
            
              D. Napoletani,
            
              Michele Signore,
            
              T. Sauer,
            
              Lance Liotta,
            
              E. Petricoin,
            
          </td>
          <td>2010-12-26</td>
          <td>Journal of theoretical biology</td>
          <td>2</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/16c4bf7b7b63ac6d23884d07c5f5850b3a00a1d6" target='_blank'>Signaling Networks in Biology</a></td>
          <td>
            
              Anamika Sarkar,
            
              A. D. Wist,
            
              R. Iyengar,
            
          </td>
          <td>2008-05-15</td>
          <td></td>
          <td>0</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/00ad36682d669b61aa3a972e0388e7a463c26765" target='_blank'>Network Propagation Predicts Drug Synergy in Cancers.</a></td>
          <td>
            
              Hongyang Li,
            
              Tingyang Li,
            
              Daniel Quang,
            
              Y. Guan,
            
          </td>
          <td>2018-07-27</td>
          <td>Cancer research</td>
          <td>69</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b12ab9d5d1223f95a6f0627a977840985d1ef53a" target='_blank'>Drug Response Prediction as a Link Prediction Problem</a></td>
          <td>
            
              Zachary Stanfield,
            
              Mustafa Coşkun,
            
              Mehmet Koyutürk,
            
          </td>
          <td>2017-01-09</td>
          <td>Scientific Reports</td>
          <td>97</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/90ecc3eb280fb929a693cb181dbdb70d750dd481" target='_blank'>A Machine Learning Approach to Predict Cellular Mechanical Stresses in Response to Chemical Perturbation</a></td>
          <td>
            
              V. Subramanianbalachandar,
            
              Md. Mydul Islam,
            
              R. Steward,
            
          </td>
          <td>2023-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/331f67f6ac3756584eee11ddec666655d8a1228b" target='_blank'>Predicting Tumor Cell Response to Synergistic Drug Combinations Using a Novel Simplified Deep Learning Model</a></td>
          <td>
            
              Heming Zhang,
            
              Jiarui Feng,
            
              Amanda Zeng,
            
              Philip R. O. Payne,
            
              Fuhai Li,
            
          </td>
          <td>2020-04-13</td>
          <td>bioRxiv</td>
          <td>11</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/77bb19de772ad66fcf101138a8a3347d9b6cfb8d" target='_blank'>Integrated Graph Propagation and Optimization with Biological Applications.</a></td>
          <td>
            
              Krithika Krishnan,
            
              Tian Shi,
            
              Han Yu,
            
              Rachael Hageman Blair,
            
          </td>
          <td>2022-11-24</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>1</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a595193f78e429286f6816e227a16b9753d835b3" target='_blank'>Towards a Rigorous Assessment of Systems Biology Models: The DREAM3 Challenges</a></td>
          <td>
            
              R. Prill,
            
              D. Marbach,
            
              J. Sáez-Rodríguez,
            
              P. Sorger,
            
              L. Alexopoulos,
            
              Xiaowei Xue,
            
              N. Clarke,
            
              G. Altan-Bonnet,
            
              G. Stolovitzky,
            
          </td>
          <td>2010-02-23</td>
          <td>PLoS ONE</td>
          <td>436</td>
          <td>104</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fa68093ce64240e73b94545459c36af8255c511e" target='_blank'>Inferring Gene Networks: Dream or Nightmare?</a></td>
          <td>
            
              Angela Baralla,
            
              Wieslawa I. Mentzen,
            
              A. G. de la Fuente,
            
          </td>
          <td>2009-03-01</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>34</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0b9b36c2ebe192d5c290f9f2f2aa9466f33b9357" target='_blank'>PaccMann: Designing anticancer drugs from transcriptomic data via reinforcement learning</a></td>
          <td>
            
              Jannis Born,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>2</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f22bb4508e66f9ed4c214cdae0c6c462e3c50e96" target='_blank'>Predicting Causal Relationships from Biological Data: Applying Automated Causal Discovery on Mass Cytometry Data of Human Immune Cells</a></td>
          <td>
            
              Sofia Triantafillou,
            
              V. Lagani,
            
              C. Heinze-Deml,
            
              A. Schmidt,
            
              J. Tegnér,
            
              I. Tsamardinos,
            
          </td>
          <td>2017-03-30</td>
          <td>Scientific Reports</td>
          <td>36</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f8db6316f4db1f1ffe34ddc038f91c6124f577ab" target='_blank'>Deep Learning for Drug Combination Synergy Prediction</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6660a7050e9ac22177c31af492ec143c577dacae" target='_blank'>Single-cell morphological data reveals signaling network architecture</a></td>
          <td>
            
              O. Nir,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2c492cc5d9f4d89a77f6be66447ef55b0202d88e" target='_blank'>Estimation of Drug Treatment Effects on the Signaling Pathways</a></td>
          <td>
            
              Seçkin Erce,
            
              Z. Işik,
            
          </td>
          <td>None</td>
          <td>International Journal of Bioscience, Biochemistry and Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c09fa4f63cf7efb2eace65b7fa78f3ec62deee08" target='_blank'>Cyclic Causal Discovery from Continuous Equilibrium Data</a></td>
          <td>
            
              J. Mooij,
            
              T. Heskes,
            
          </td>
          <td>2013-08-11</td>
          <td>ArXiv</td>
          <td>71</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/11e63502a75da64ec9817ccb005647d5b3ef9451" target='_blank'>PaccMann$^{RL}$: Designing anticancer drugs from transcriptomic data via reinforcement learning</a></td>
          <td>
            
              Jannis Born,
            
              Matteo Manica,
            
              Ali Oskooei,
            
              Joris Cadow,
            
              K. Borgwardt,
            
              Mar'ia Rodr'iguez Mart'inez,
            
          </td>
          <td>2019-08-29</td>
          <td>arXiv: Biomolecules</td>
          <td>15</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf2785f9de26afd8ad2c2f823bf9e7d3f74957ac" target='_blank'>Predicting gene targets of perturbations via network-based filtering of mRNA expression compendia</a></td>
          <td>
            
              Elissa J. Cosgrove,
            
              Yingchun Zhou,
            
              T. Gardner,
            
              E. Kolaczyk,
            
          </td>
          <td>2008-11-01</td>
          <td>Bioinformatics</td>
          <td>41</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca6410a37106ae5d68d99c17f729e603b5c1642a" target='_blank'>Efficient parameterization of large-scale dynamic models based on relative measurements</a></td>
          <td>
            
              Leonard Schmiester,
            
              Yannik Schälte,
            
              Fabian Froehlich,
            
              J. Hasenauer,
            
              Daniel Weindl,
            
          </td>
          <td>2019-03-16</td>
          <td>Bioinformatics</td>
          <td>29</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f0063f4c59becc261188fc5a820fb6bdb55f691d" target='_blank'>Uncovering expression signatures of synergistic drug responses via ensembles of explainable machine-learning models</a></td>
          <td>
            
              Joseph D. Janizek,
            
              A. Dincer,
            
              Safiye Celik,
            
              Hugh Chen,
            
              William Chen,
            
              K. Naxerova,
            
              Su-In Lee,
            
          </td>
          <td>2023-05-01</td>
          <td>Nature Biomedical Engineering</td>
          <td>9</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92e3a7754d090a24e6d1a9fb5a486e177656f4b3" target='_blank'>Causal interactions from proteomic profiles: Molecular data meet pathway knowledge</a></td>
          <td>
            
              O. Babur,
            
              Augustin Luna,
            
              Anil Korkut,
            
              Funda Durupinar,
            
              M. Siper,
            
              U. Dogrusoz,
            
              Alvaro Sebastian Vaca Jacome,
            
              Ryan Peckner,
            
              K. E. Christianson,
            
              Jacob D. Jaffe,
            
              Paul T. Spellman,
            
              J. Aslan,
            
              C. Sander,
            
              E. Demir,
            
          </td>
          <td>2021-05-12</td>
          <td>Patterns</td>
          <td>56</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d6d0b08c8768d86315016aed23ead36d3882c41b" target='_blank'>HiDi: an efficient reverse engineering schema for large‐scale dynamic regulatory network reconstruction using adaptive differentiation</a></td>
          <td>
            
              Yue Deng,
            
              H. Zenil,
            
              J. Tegnér,
            
              N. Kiani,
            
          </td>
          <td>2017-06-05</td>
          <td>Bioinformatics</td>
          <td>11</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f089e9451548d55c7f1d1a6b5de67e7698b63178" target='_blank'>Directed Discovery of Novel Drug Cocktails 1</a></td>
          <td>
            
              J. Miller,
            
              R. Zinner,
            
              B. Barrett,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>2</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f746febb4c253dbf9fa2bcb37e3ebbf95804141b" target='_blank'>Estimating information in time-varying signals</a></td>
          <td>
            
              S. A. Cepeda-Humerez,
            
              J. Ruess,
            
              G. Tkačik,
            
          </td>
          <td>2018-12-31</td>
          <td>PLoS Computational Biology</td>
          <td>18</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a6f5bbea83a666ec329b10fd7fbfa733b0ad25b8" target='_blank'>Similarity metric learning on perturbational datasets improves functional identification of perturbations</a></td>
          <td>
            
              Ian Smith,
            
              P. Smirnov,
            
              B. Haibe-Kains,
            
          </td>
          <td>2023-06-11</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/90b8b9cbb55743ceace091f0e0fa5e4ce6dd7637" target='_blank'>Genomic Network Tomography</a></td>
          <td>
            
              M. Rabbat,
            
              Mário A. T. Figueiredo,
            
              R. Nowak,
            
          </td>
          <td>2007-04-15</td>
          <td>2007 IEEE International Conference on Acoustics, Speech and Signal Processing - ICASSP '07</td>
          <td>0</td>
          <td>77</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8e133d29b87cf2a314abfa9308ee05434a58d6cf" target='_blank'>Computational modeling of drug response with applications to neuroscience</a></td>
          <td>
            
              R. Herwig,
            
          </td>
          <td>2014-12-01</td>
          <td>Dialogues in Clinical Neuroscience</td>
          <td>3</td>
          <td>49</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dd8014ff010a5db61ef5859baa794303182af45a" target='_blank'>ASGARD: A Single-cell Guided Pipeline to Aid Repurposing of Drugs</a></td>
          <td>
            
              Bing He,
            
              Yao Xiao,
            
              Haodong Liang,
            
              Qianhui Huang,
            
              Yuheng Du,
            
              Yijun Li,
            
              D. Garmire,
            
              Duxin Sun,
            
              L. Garmire,
            
          </td>
          <td>2021-09-14</td>
          <td>ArXiv</td>
          <td>16</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0d9d669c81ec5bced289ac5f8387e066422624c9" target='_blank'>De Novo Design of Molecules with Multiaction Potential from Differential Gene Expression using Variational Autoencoder</a></td>
          <td>
            
              Nutaya Pravalphruekul,
            
              Maytus Piriyajitakonkij,
            
              P. Phunchongharn,
            
              Supanida Piyayotai,
            
          </td>
          <td>2023-06-22</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/db5d6be77a6b4b3473497dec2a49a12b7888ccd2" target='_blank'>Dynamic Control of Stochastic Evolution: A Deep Reinforcement Learning Approach to Adaptively Targeting Emergent Drug Resistance</a></td>
          <td>
            
              D. Engelhardt,
            
          </td>
          <td>2019-03-27</td>
          <td>ArXiv</td>
          <td>9</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8d12c67d2dcf5fb28473d6f76aba74d92a9e8cc4" target='_blank'>XGraphCDS: An explainable deep learning model for predicting drug sensitivity from gene pathways and chemical structures</a></td>
          <td>
            
              Yimeng Wang,
            
              Xinxin Yu,
            
              Yaxin Gu,
            
              Weihua Li,
            
              Keyun Zhu,
            
              Long Chen,
            
              Yun Tang,
            
              Guixia Liu,
            
          </td>
          <td>2023-11-01</td>
          <td>Computers in biology and medicine</td>
          <td>0</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/814563494f0b39231d02f0fb8a20587e7da38438" target='_blank'>Inferring Broad Regulatory Biology from Time Course Data: Have We Reached an Upper Bound under Constraints Typical of In Vivo Studies?</a></td>
          <td>
            
              Saurabh Vashishtha,
            
              G. Broderick,
            
              T. Craddock,
            
              M. Fletcher,
            
              N. Klimas,
            
          </td>
          <td>2015-05-18</td>
          <td>PLoS ONE</td>
          <td>8</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a1fd94059691e61a9bb5723af85213955c5f3c8f" target='_blank'>Computing cancer's weak spots.</a></td>
          <td>
            
              Roxanne Khamsi,
            
          </td>
          <td>2020-06-11</td>
          <td>Science</td>
          <td>2</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6b779878af4311a8141206571a79190fdfcae1b6" target='_blank'>The Multidimensional Perturbation Value</a></td>
          <td>
            
              J. Hutz,
            
              Thomas Nelson,
            
              Hua‐Lang Wu,
            
              Gregory McAllister,
            
              I. Moutsatsos,
            
              S. Jaeger,
            
              Somnath Bandyopadhyay,
            
              Florian Nigsch,
            
              B. Cornett,
            
              J. Jenkins,
            
              D. Selinger,
            
          </td>
          <td>2013-04-01</td>
          <td>Journal of Biomolecular Screening</td>
          <td>20</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7929604c7e4f58db52122f7895e19a4a46d7d0ec" target='_blank'>Identification of optimal drug combinations targeting cellular networks: integrating phospho-proteomics and computational network analysis.</a></td>
          <td>
            
              S. Iadevaia,
            
              Yiling Lu,
            
              F. C. Morales,
            
              G. Mills,
            
              Prahlad T. Ram,
            
          </td>
          <td>2010-09-01</td>
          <td>Cancer research</td>
          <td>221</td>
          <td>184</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/324d3e186193186f6ea885f56c6fc0e4f26a2c90" target='_blank'>Causal network perturbations for instance-specific analysis of single cell and disease samples</a></td>
          <td>
            
              K. Buschur,
            
              M. Chikina,
            
              P. Benos,
            
          </td>
          <td>2019-05-16</td>
          <td>Bioinformatics</td>
          <td>17</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a7ce58d22d6d2b87a20f251a53f88b6e054bc474" target='_blank'>Delineating feedback activity in the MAPK and AKT pathways using feedback-enabled Inference of Signaling Activity</a></td>
          <td>
            
              B. Thijssen,
            
              K. Jastrzebski,
            
              R. Beijersbergen,
            
              L. Wessels,
            
          </td>
          <td>2018-02-20</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>66</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8dc4a9931fdd5bf87408a0a55efaa720e7a84e0f" target='_blank'>Robust Prediction of Patient-Specific Clinical Response to Unseen Drugs From in vitro Screens Using Context-aware Deconfounding Autoencoder</a></td>
          <td>
            
              Di He,
            
              Qiao Liu,
            
              Lei Xie,
            
          </td>
          <td>2021-05-21</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/df0f4fb984bf383056c581d009bc19c75dc7a748" target='_blank'>Evaluation of Machine Learning Classifiers to Predict Compound Mechanism of Action When Transferred across Distinct Cell Lines</a></td>
          <td>
            
              Scott Warchal,
            
              John C. Dawson,
            
              N. Carragher,
            
          </td>
          <td>2019-01-29</td>
          <td>Slas Discovery</td>
          <td>12</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/84181e92eb0fd35d69a4d3610aafbdde43349a5d" target='_blank'>RECOVER: sequential model optimization platform for combination drug repurposing identifies novel synergistic compounds in vitro</a></td>
          <td>
            
              Paul Bertin,
            
              Jarrid Rector-Brooks,
            
              Deepak Sharma,
            
              Thomas Gaudelet,
            
              A. Anighoro,
            
              Torsten Gross,
            
              Francisco Martínez-Peña,
            
              Eileen L. Tang,
            
              S. SurajM,
            
              Cristian Regep,
            
              Jeremy B. R. Hayter,
            
              Maksym Korablyov,
            
              N. Valiante,
            
              A. V. D. Sloot,
            
              M. Tyers,
            
              Charlie Roberts,
            
              M. Bronstein,
            
              L. Lairson,
            
              J. Taylor-King,
            
              Y. Bengio,
            
          </td>
          <td>2022-02-07</td>
          <td>ArXiv</td>
          <td>9</td>
          <td>97</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3cb0eaa2bdcacdb8afd04ac30edacd4b4a0f572a" target='_blank'>Integrative and Personalized QSAR Analysis in Cancer by Kernelized Bayesian Matrix Factorization</a></td>
          <td>
            
              Muhammad Ammad-ud-din,
            
              Elisabeth Georgii,
            
              M. Gönen,
            
              T. Laitinen,
            
              O. Kallioniemi,
            
              K. Wennerberg,
            
              A. Poso,
            
              Samuel Kaski,
            
          </td>
          <td>2014-08-06</td>
          <td>Journal of chemical information and modeling</td>
          <td>92</td>
          <td>107</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/004e8bf4058e6cfcf48d74aeed4ae6c6bf449b88" target='_blank'>scTenifoldKnk: a machine learning workflow performing virtual knockout experiments on single-cell gene regulatory networks</a></td>
          <td>
            
              Daniel Osório,
            
              Yan Zhong,
            
              Guanxun Li,
            
              Qian Xu,
            
              A. Hillhouse,
            
              Jingshu Chen,
            
              A. Laurie,
            
              Davidson,
            
              Yanan Tian,
            
              R. Chapkin,
            
              Jianhua Huang,
            
              James J. Cai,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>2</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ddb9ba8651e39d3c2e7a990c4b27049c20aff228" target='_blank'>Machine learning, artificial intelligence, and data science breaking into drug design and neglected diseases</a></td>
          <td>
            
              José Peña-Guerrero,
            
              P. Nguewa,
            
              A. García‐Sosa,
            
          </td>
          <td>2021-01-05</td>
          <td>Wiley Interdisciplinary Reviews: Computational Molecular Science</td>
          <td>23</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d7a3ef65e4516088eaacf8310aa9cb1ba35db1ca" target='_blank'>Abstract LB_B19: Utilizing genome-informed modeling to assess CDK4/6 inhibitor response in breast cancer patients, simulate clinical trials, and result in novel CDK4/6 inhibitor treatment for chordoma patients</a></td>
          <td>
            
              Mei Yang,
            
              Yuhan Liu,
            
              Yi-Ching Hsueh,
            
              Qiangzu Zhang,
            
              Yanhui Fan,
            
              Juntao Xu,
            
              Min Huang,
            
              Xu Li,
            
              Su Chen,
            
              Jianfei Yang,
            
              G. Niu,
            
          </td>
          <td>2023-12-01</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bceaf703938d9a36435d575866a495d261076d88" target='_blank'>Generating executable models from signaling network connectivity and semi-quantitative proteomic measurements</a></td>
          <td>
            
              Derek A. Ruths,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1a75e68fe9829d4c857ded00d56300e5c241cc26" target='_blank'>Predicting anti-cancer drug synergy using extended drug similarity profiles</a></td>
          <td>
            
              Sayed-Rzgar Hosseini,
            
              Xiaobo Zhou,
            
          </td>
          <td>2022-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/456f22d6e3dcf323438f139400f40c612c0438df" target='_blank'>PTML Modeling for Pancreatic Cancer Research: In Silico Design of Simultaneous Multi-Protein and Multi-Cell Inhibitors</a></td>
          <td>
            
              V. V. Kleandrova,
            
              A. Speck-Planche,
            
          </td>
          <td>2022-02-01</td>
          <td>Biomedicines</td>
          <td>11</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3726aa0b96f28293a88619a630bca2d3ec45f9dd" target='_blank'>CCSynergy: an integrative deep-learning framework enabling context-aware prediction of anti-cancer drug synergy</a></td>
          <td>
            
              Sayed-Rzgar Hosseini,
            
              Xiaobo Zhou,
            
          </td>
          <td>2022-12-23</td>
          <td>Briefings in bioinformatics</td>
          <td>10</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/025dab3894ae7b94bcb84b2ed07d6fbc4d1569a1" target='_blank'>Predicting drug response of tumors from integrated genomic profiles by deep neural networks</a></td>
          <td>
            
              Yu-Chiao Chiu,
            
              H. Chen,
            
              Tinghe Zhang,
            
              Song-Seon Zhang,
            
              Aparna Gorthi,
            
              Li-Ju Wang,
            
              Yufei Huang,
            
              Yidong Chen,
            
          </td>
          <td>2018-05-20</td>
          <td>BMC Medical Genomics</td>
          <td>141</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dfbf73b4d67dd34c855cf8e556071b7e092bb704" target='_blank'>Input–output behavior of ErbB signaling pathways as revealed by a mass action model trained against dynamic data</a></td>
          <td>
            
              William W. Chen,
            
              B. Schoeberl,
            
              P. Jasper,
            
              M. Niepel,
            
              U. Nielsen,
            
              D. Lauffenburger,
            
              P. Sorger,
            
          </td>
          <td>2009-01-20</td>
          <td>Molecular Systems Biology</td>
          <td>423</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3da82248f0c26eda16729a986a9c351ef5d938b7" target='_blank'>Determining Dose-Response Characteristics of Molecular Perturbations in Whole-Organism Assays Using Biological Imaging and Machine Learning</a></td>
          <td>
            
              D. Asarnow,
            
              Rahul Singh,
            
          </td>
          <td>2018-12-01</td>
          <td>2018 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>3</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1e07bf5fc284d9b70bcbbd63cc2210862d30301c" target='_blank'>Predicting genetic interactions, cell line dependencies and drug sensitivities with variational graph auto-encoder</a></td>
          <td>
            
              Asia Gervits,
            
              R. Sharan,
            
          </td>
          <td>2022-12-02</td>
          <td>Frontiers in Bioinformatics</td>
          <td>1</td>
          <td>68</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/08116bce9f7e3a123ddca0637cbf66278ed600b6" target='_blank'>Reverse Engineering of Gene Networks Reverse Engineering of Gene Networks with LASSO and Non-Linear Basis Functions</a></td>
          <td>
            
              M. Gustafsson,
            
              Michael Hörnquist,
            
              Jesper Lundstrom,
            
              Johan L M Bjorkegren,
            
              J. Tegnér,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7888fdafa30838321ae6b503c28981fb0b1149b1" target='_blank'>Predictive Modeling of Drug Effects on Signaling Pathways in Diverse Cancer Cell Lines</a></td>
          <td>
            
              S. K. Mishra,
            
              S. Bhowmick,
            
              Huey-Eng Chua,
            
              Jie Zheng,
            
          </td>
          <td>2016-10-02</td>
          <td>Proceedings of the 7th ACM International Conference on Bioinformatics, Computational Biology, and Health Informatics</td>
          <td>0</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/865139e38c1c44b60d21a2131fc5f7e6cf009088" target='_blank'>An enhanced cascade-based deep forest model for drug combination prediction</a></td>
          <td>
            
              Weiping Lin,
            
              Lianlian Wu,
            
              Yixin Zhang,
            
              Yuqi Wen,
            
              Bowei Yan,
            
              Chong Dai,
            
              Kun-hong Liu,
            
              Song He,
            
              Xiaochen Bo,
            
          </td>
          <td>2022-01-21</td>
          <td>Briefings in bioinformatics</td>
          <td>7</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7442eb37810048fb64b23b53aca660c7b381242d" target='_blank'>Dose–response prediction for in-vitro drug combination datasets: a probabilistic approach</a></td>
          <td>
            
              Leiv Rønneberg,
            
              Paul D. W. Kirk,
            
              M. Zucknick,
            
          </td>
          <td>2023-04-21</td>
          <td>BMC Bioinformatics</td>
          <td>2</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/777fe6e367aa7a01a9e4731974ddcb6fa8a5421b" target='_blank'>SynergyX: a multi-modality mutual attention network for interpretable drug synergy prediction</a></td>
          <td>
            
              Yue Guo,
            
              Haitao Hu,
            
              Wenbo Chen,
            
              Hao Yin,
            
              Jian Wu,
            
              Chang-Yu Hsieh,
            
              Qiaojun He,
            
              Ji Cao,
            
          </td>
          <td>2024-01-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/99d13534f4a428159022376d33a35366a86b8b95" target='_blank'>Reverse causal reasoning: applying qualitative causal knowledge to the interpretation of high-throughput data</a></td>
          <td>
            
              Natalie L. Catlett,
            
              Anthony J. Bargnesi,
            
              Stephen Ungerer,
            
              Toby Seagaran,
            
              W. Ladd,
            
              Keith Owen Elliston,
            
              Dexter Pratt,
            
          </td>
          <td>2013-11-23</td>
          <td>BMC Bioinformatics</td>
          <td>94</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/99d13534f4a428159022376d33a35366a86b8b95" target='_blank'>Reverse causal reasoning: applying qualitative causal knowledge to the interpretation of high-throughput data</a></td>
          <td>
            
              Natalie L. Catlett,
            
              Anthony J. Bargnesi,
            
              Stephen Ungerer,
            
              Toby Seagaran,
            
              W. Ladd,
            
              Keith Owen Elliston,
            
              Dexter Pratt,
            
          </td>
          <td>2013-11-23</td>
          <td>BMC Bioinformatics</td>
          <td>94</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d391f3ea4ba237cda17d6ff0c1cfdfeab8b7d363" target='_blank'>A functional module states framework reveals cell states for drug and target prediction</a></td>
          <td>
            
              Guangrong Qin,
            
              T. Knijnenburg,
            
              David L. Gibbs,
            
              R. Moser,
            
              R. Monnat,
            
              C. Kemp,
            
              I. Shmulevich,
            
          </td>
          <td>2020-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>81</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/02108c118b34acb56cb5e41d224cf0b1f8d2fbec" target='_blank'>Compound activity prediction with dose-dependent transcriptomic profiles and deep learning</a></td>
          <td>
            
              William J. Godinez,
            
              Vladimir Trifonov,
            
              Bin Fang,
            
              Guray Kuzu,
            
              Luying Pei,
            
              W. A. Guiguemde,
            
              E. Martin,
            
              Frederick J. King,
            
              J. Jenkins,
            
              Peter Skewes-Cox,
            
          </td>
          <td>2023-08-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/99cad9acf47b3f3cdd2459d3b550f4fa34eedb09" target='_blank'>Extracting functional insights from loss-of-function screens using deep link prediction</a></td>
          <td>
            
              Pieter-Paul Strybol,
            
              M. Larmuseau,
            
              Louise de Schaetzen van Brienen,
            
              Tim Van den Bulcke,
            
              K. Marchal,
            
          </td>
          <td>2022-02-01</td>
          <td>Cell Reports Methods</td>
          <td>1</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6066680b4266566576bc91fc28a95f18b898f502" target='_blank'>Accelerating the Development of Personalized Cancer Immunotherapy by Integrating Molecular Patients' Profiles with Dynamic Mathematical Models.</a></td>
          <td>
            
              Z. Agur,
            
              M. Elishmereni,
            
              U. Foryś,
            
              Y. Kogan,
            
          </td>
          <td>2020-06-14</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>10</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6aadbfd598a53108a324ea9d6f68aeffe855d167" target='_blank'>Logic models to predict continuous outputs based on binary inputs with an application to personalized cancer therapy</a></td>
          <td>
            
              T. Knijnenburg,
            
              G. Klau,
            
              Francesco Iorio,
            
              M. Garnett,
            
              U. McDermott,
            
              I. Shmulevich,
            
              L. Wessels,
            
          </td>
          <td>2016-01-15</td>
          <td>Scientific Reports</td>
          <td>50</td>
          <td>81</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8810af2ce3a872ab86fd29e7aef2f8a3122afb9e" target='_blank'>MARSY: a multitask deep-learning framework for prediction of drug combination synergy scores</a></td>
          <td>
            
              Mohamed Reda El Khili,
            
              Safyan Aman Memon,
            
              A. Emad,
            
          </td>
          <td>2023-01-05</td>
          <td>Bioinformatics</td>
          <td>10</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/02a54d9853706abc5e88e724dfd7d3639e38ba71" target='_blank'>Targeted Therapy Database (TTD): A Model to Match Patient's Molecular Profile with Current Knowledge on Cancer Biology</a></td>
          <td>
            
              S. Mocellin,
            
              Jeff Shrager,
            
              R. Scolyer,
            
              S. Pasquali,
            
              Daunia Verdi,
            
              F. Marincola,
            
              M. Briarava,
            
              Randy Gobbel,
            
              C. Rossi,
            
              D. Nitti,
            
          </td>
          <td>2010-08-10</td>
          <td>PLoS ONE</td>
          <td>18</td>
          <td>110</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/05e5f3ec73acdf864ee4f1cc17751fe9c775af92" target='_blank'>Learning feedback molecular network models using integer linear programming</a></td>
          <td>
            
              M. Ozen,
            
              A. Abdi,
            
              E. Emamian,
            
          </td>
          <td>2021-11-10</td>
          <td>Physical Biology</td>
          <td>0</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/442d06ef258ccf43633ae9654d097a8bb99af573" target='_blank'>Assessing Combinational Drug Efficacy in Cancer Cells by Using Image-based Dynamic Response Analysis</a></td>
          <td>
            
              Chao Sima,
            
              Jianping Hua,
            
              M. Cypert,
            
              T. Miller,
            
              H. Wilson-Robles,
            
              J. Trent,
            
              E. Dougherty,
            
              M. Bittner,
            
          </td>
          <td>2015-01-01</td>
          <td>Cancer Informatics</td>
          <td>4</td>
          <td>107</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ec29a85fa243cc233fa19a11be6113e5b3c1ad13" target='_blank'>GexMolGen: Cross-modal Generation of Hit-like Molecules via Large Language Model Encoding of Gene Expression Signatures</a></td>
          <td>
            
              Jiabei Cheng,
            
              Xiaoyong Pan,
            
              Kaiyuan Yang,
            
              Shenghao Cao,
            
              Bin Liu,
            
              Qingran Yan,
            
              Ye Yuan,
            
          </td>
          <td>2024-02-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7e4407cdea481018cf79932ee44555257b0ac2ed" target='_blank'>Intrinsically Bayesian Robust Classifier for Single-Cell Gene Expression Time Series in Gene Regulatory Networks</a></td>
          <td>
            
              Alireza Karbalayghareh,
            
              U. Braga-Neto,
            
              E. Dougherty,
            
          </td>
          <td>2017-08-20</td>
          <td>Proceedings of the 8th ACM International Conference on Bioinformatics, Computational Biology,and Health Informatics</td>
          <td>2</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/90ce6e4448a1b178af1bbcb0eef95afb1502845d" target='_blank'>Artificial intelligence to guide precision anticancer therapy with multitargeted kinase inhibitors</a></td>
          <td>
            
              Manali Singha,
            
              Limeng Pu,
            
              Brent A. Stanfield,
            
              Ifeanyi K. Uche,
            
              Paul J. F. Rider,
            
              K. Kousoulas,
            
              J. Ramanujam,
            
              Michal Brylinski,
            
          </td>
          <td>2022-11-24</td>
          <td>BMC Cancer</td>
          <td>4</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f1599e311548811f0c063fe988ae4cf1ead08122" target='_blank'>ResponseNet2.0: revealing signaling and regulatory pathways connecting your proteins and genes—now with human data</a></td>
          <td>
            
              Omer Basha,
            
              Shoval Tirman,
            
              Amir Eluk,
            
              Esti Yeger Lotem,
            
          </td>
          <td>2013-06-11</td>
          <td>Nucleic Acids Research</td>
          <td>23</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b1b8834d34b6fcc076ca62afd924d20b3d555141" target='_blank'>Abstract 3106: Integrated in vitro and machine learning approaches for targeted drug screening in gliomas with concurrent NF1 and ATRX loss</a></td>
          <td>
            
              Swati Dubey,
            
              Somesh Mohapatra,
            
              M. Yuan,
            
              C. Eberhart,
            
              Fausto J. Rodriguez,
            
          </td>
          <td>2024-03-22</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aed5d73322b026e81679d90a5b93550793ba4b19" target='_blank'>Robust Prediction of Personalized In Vivo Response to Unseen Drugs From In Vitro Screens Using a Novel Context-Aware Deconfounding Autoencoder</a></td>
          <td>
            
              Di He,
            
              Qiao Liu,
            
              Lei Xie,
            
          </td>
          <td>2021-11-12</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/729e7b79d8240a360cec4aae90c56ec35de1541c" target='_blank'>Uncovering expression signatures of synergistic drug response using an ensemble of explainable AI models</a></td>
          <td>
            
              Joseph D. Janizek,
            
              A. Dincer,
            
              Safiye Celik,
            
              Hugh Chen,
            
              William Chen,
            
              K. Naxerova,
            
              Su-In Lee,
            
          </td>
          <td>2021-10-07</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0e3449a69a538559f349b5f025ab83a43208a57e" target='_blank'>Longitudinal drug synergy assessment using convolutional neural network image-decoding of glioblastoma single-spheroid cultures</a></td>
          <td>
            
              Anna Giczewska,
            
              Krzysztof Pastuszak,
            
              M. Houweling,
            
              Kulsoom U Abdul,
            
              Noa Faaij,
            
              L. Wedekind,
            
              David Noske,
            
              Tom Wurdinger,
            
              A. Supernat,
            
              Bart A Westerman,
            
          </td>
          <td>2023-11-05</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fc583004d4583e52c3849627848ea2d290c65e66" target='_blank'>Intrinsically Bayesian robust classifier for single-cell gene expression trajectories in gene regulatory networks</a></td>
          <td>
            
              Alireza Karbalayghareh,
            
              U. Braga-Neto,
            
              E. Dougherty,
            
          </td>
          <td>2018-03-21</td>
          <td>BMC Systems Biology</td>
          <td>10</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4fa5f172b476e7f0de6bac14ca3fd21980ddb5e6" target='_blank'>Computational Model of Heterogeneity in Melanoma: Designing Therapies and Predicting Outcomes</a></td>
          <td>
            
              A. Hodgkinson,
            
              D. Trucu,
            
              M. Lacroix,
            
              L. Le Cam,
            
              O. Radulescu,
            
          </td>
          <td>2022-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>5</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cfb747db00c6e98936d0749d962f04b796ea7036" target='_blank'>Systems analysis of apoptosis protein expression allows the case-specific prediction of cell death responsiveness of melanoma cells</a></td>
          <td>
            
              Egle Passante,
            
              Maximilian L. Würstle,
            
              C. Hellwig,
            
              Martin Leverkus,
            
              M. Rehm,
            
          </td>
          <td>2013-08-09</td>
          <td>Cell Death and Differentiation</td>
          <td>35</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7c06bf1cdd1e95eab750295a1d4e7f12bc04e874" target='_blank'>Time-of-day effects of drugs revealed by high-throughput deep phenotyping</a></td>
          <td>
            
              Carolin Ector,
            
              Christoph Schmal,
            
              Jeff Didier,
            
              Sébastien De Landtsheer,
            
              Anna-Marie Finger,
            
              Francesca Müller-Marquardt,
            
              Johannes Schulte,
            
              Thomas Sauter,
            
              Ulrich Keilholz,
            
              H. Herzel,
            
              Achim Kramer,
            
              Adrián E. Granada,
            
          </td>
          <td>2023-12-01</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>69</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/51c089fef5e76e99ca2fcdf036d63de7ca809cbb" target='_blank'>A reverse-engineering approach to dissect post-translational modulators of transcription factor’s activity from transcriptional data</a></td>
          <td>
            
              G. Gambardella,
            
              I. Peluso,
            
              S. Montefusco,
            
              M. Bansal,
            
              D. Medina,
            
              Neil D. Lawrence,
            
              D. Bernardo,
            
          </td>
          <td>2015-09-03</td>
          <td>BMC Bioinformatics</td>
          <td>7</td>
          <td>68</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7cd32e8383dc06d79ebf6494e88ef8eaf5c1e23b" target='_blank'>TranSiGen: Deep representation learning of chemical-induced transcriptional profile</a></td>
          <td>
            
              X. Tong,
            
              Ning Qu,
            
              Xiangtai Kong,
            
              Shengkun Ni,
            
              Kun Wang,
            
              Lehan Zhang,
            
              Yiming Wen,
            
              Sulin Zhang,
            
              Xutong Li,
            
              Mingyue Zheng,
            
          </td>
          <td>2023-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f87189c3da0abe61e47ccc6636e1effaa403de02" target='_blank'>DREAM4: Combining Genetic and Dynamic Information to Identify Biological Networks and Dynamical Models</a></td>
          <td>
            
              Alex Greenfield,
            
              Aviv Madar,
            
              H. Ostrer,
            
              Richard Bonneau,
            
          </td>
          <td>2010-10-25</td>
          <td>PLoS ONE</td>
          <td>238</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/96645e644150fae6be9b711f54a840afd7aee085" target='_blank'>A Non-Negative Matrix Tri-Factorization based Method for Predicting Antitumor Drug Sensitivity</a></td>
          <td>
            
              C. Testa,
            
              S. Pidó,
            
              Pietro Pinoli,
            
          </td>
          <td>2021-12-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c1e80dc09804074285b6292f54ec55c68c55410d" target='_blank'>Identifying chemogenetic interactions from CRISPR screens with drugZ</a></td>
          <td>
            
              Medina Colic,
            
              Gang Wang,
            
              Michal Zimmermann,
            
              Keith S. Mascall,
            
              M. McLaughlin,
            
              Lori L. Bertolet,
            
              W. Lenoir,
            
              J. Moffat,
            
              S. Angers,
            
              D. Durocher,
            
              D. Durocher,
            
              Traver Hart,
            
          </td>
          <td>2019-08-22</td>
          <td>Genome Medicine</td>
          <td>115</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2f9169a9604184a634feb6d610ab87d2a22ab5bf" target='_blank'>Training set expansion: an approach to improving the reconstruction of biological networks from limited and uneven reliable interactions</a></td>
          <td>
            
              Kevin Y. Yip,
            
              M. Gerstein,
            
          </td>
          <td>2008-11-17</td>
          <td>Bioinformatics</td>
          <td>34</td>
          <td>165</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3b7a47c3bf7bae09d7eec168709fd507b8b22100" target='_blank'>ChemPrint: An AI-Driven Framework for Enhanced Drug Discovery</a></td>
          <td>
            
              Tyler Umansky,
            
              Navya Ramesh,
            
              Virgil Woods,
            
              Sean M. Russell,
            
              Davey Smith,
            
              Daniel Haders,
            
          </td>
          <td>2024-03-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6f180a748d3a1545fd6b93f0b99c308bd20edf73" target='_blank'>Stratification and prediction of drug synergy based on target functional similarity</a></td>
          <td>
            
              Mi Yang,
            
              P. Jaaks,
            
              J. Dry,
            
              M. Garnett,
            
              M. Menden,
            
              J. Sáez-Rodríguez,
            
          </td>
          <td>2019-03-22</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>29</td>
          <td>50</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0d28b2c4ab559ba9d75a767d948e96abdc7643e1" target='_blank'>Signaling pathway models as biomarkers: Patient-specific simulations of JNK activity predict the survival of neuroblastoma patients</a></td>
          <td>
            
              D. Fey,
            
              M. Halasz,
            
              D. Dreidax,
            
              S. Kennedy,
            
              Jordan F Hastings,
            
              Nora Rauch,
            
              Amaya Garcia Munoz,
            
              R. Pilkington,
            
              M. Fischer,
            
              F. Westermann,
            
              W. Kolch,
            
              B. Kholodenko,
            
              David R. Croucher,
            
          </td>
          <td>2015-12-22</td>
          <td>Science Signaling</td>
          <td>136</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7c84fa54613ee0b265a3ee5acce402c83f93ef5b" target='_blank'>Meta-Dynamic Network Modelling for Biochemical Networks.</a></td>
          <td>
            
              Anthony Hart,
            
              Leann Nguyen,
            
          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0b8fb690e3f8d41a5e207ed329811c23a0069b2b" target='_blank'>Few-Shot Drug Synergy Prediction With a Prior-Guided Hypernetwork Architecture</a></td>
          <td>
            
              Qingqing Zhang,
            
              Shaowu Zhang,
            
              Yue-Hua Feng,
            
              Jianyu Shi,
            
          </td>
          <td>2023-02-23</td>
          <td>IEEE Transactions on Pattern Analysis and Machine Intelligence</td>
          <td>4</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/23421c6b6b1ab55e6b5369e83167832050578a40" target='_blank'>Building predictive signaling models by perturbing yeast cells with time-varying stimulations resulting in distinct signaling responses</a></td>
          <td>
            
              Hossein Jashnsaz,
            
              Z. Fox,
            
              B. Munsky,
            
              G. Neuert,
            
          </td>
          <td>2021-07-07</td>
          <td>STAR Protocols</td>
          <td>2</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/51573a2ecaf1a9addac19b76d1cf85161a3bbccc" target='_blank'>Biologically Informed Neural Networks Predict Drug Responses.</a></td>
          <td>
            
              C. Greene,
            
              J. Costello,
            
          </td>
          <td>2020-10-21</td>
          <td>Cancer cell</td>
          <td>8</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e67af7add8f12d3f49a425a63da0662117fd02f0" target='_blank'>Benchmarking causal reasoning algorithms for gene expression-based compound mechanism of action analysis</a></td>
          <td>
            
              Layla Hosseini-Gerami,
            
              I. Higgins,
            
              D. A. Collier,
            
              Emma Laing,
            
              David Evans,
            
              Howard Broughton,
            
              A. Bender,
            
          </td>
          <td>2022-01-19</td>
          <td>BMC Bioinformatics</td>
          <td>6</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/322cae8262c7252d6b4bac82025185293face74f" target='_blank'>Identifying biomarkers of anti-cancer drug synergy using multi-task learning</a></td>
          <td>
            
              N. Aben,
            
              J. D. de Ruiter,
            
              Evert Bosdriesz,
            
              Yongsoo Kim,
            
              G. Bounova,
            
              D. Vis,
            
              L. Wessels,
            
              M. Michaut,
            
          </td>
          <td>2018-01-09</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>66</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/03c120e239a6f65d88045fcdde4e6fcfe9d4bdaf" target='_blank'>GeneSPIDER - gene regulatory network inference benchmarking with controlled network and data properties.</a></td>
          <td>
            
              Andreas Tjärnberg,
            
              D. Morgan,
            
              Matthew Studham,
            
              Torbjörn E. M. Nordling,
            
              E. Sonnhammer,
            
          </td>
          <td>2017-06-27</td>
          <td>Molecular bioSystems</td>
          <td>14</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f67a3e326d08d43d2b5878f56ff5ff13842f410e" target='_blank'>PRODeepSyn: predicting anticancer synergistic drug combinations by embedding cell lines with protein–protein interaction network</a></td>
          <td>
            
              Xiaowen Wang,
            
              Hongming Zhu,
            
              Yizhi Jiang,
            
              Yulong Li,
            
              Chen Tang,
            
              Xiaohan Chen,
            
              Yunjie Li,
            
              Qi Liu,
            
              Qin Liu,
            
          </td>
          <td>2022-01-19</td>
          <td>Briefings in Bioinformatics</td>
          <td>12</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8d6d144f935235fb1c886ec43fbefcf71bbfda38" target='_blank'>PathwayMap: Molecular Pathway Association with Self-Normalizing Neural Networks</a></td>
          <td>
            
              J. Jiménez,
            
              Davide Sabbadin,
            
              Alberto Cuzzolin,
            
              Gerard Martínez-Rosell,
            
              Jacob Gora,
            
              J. Manchester,
            
              J. Duca,
            
              G. D. Fabritiis,
            
          </td>
          <td>2018-12-26</td>
          <td>Journal of chemical information and modeling</td>
          <td>21</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58e533bc46cb27eb3860281a30f7c22f912426aa" target='_blank'>Genetic function approximation in the molecular pharmacology of cancer</a></td>
          <td>
            
              Leming Shi,
            
              Yi Fan,
            
              T. Myers,
            
              J. Weinstein,
            
          </td>
          <td>1997-06-09</td>
          <td>Proceedings of International Conference on Neural Networks (ICNN'97)</td>
          <td>3</td>
          <td>122</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/95f75baf3f1cb48108d2823a2289c2c328109163" target='_blank'>DOSE-L1000: unveiling the intricate landscape of compound-induced transcriptional changes</a></td>
          <td>
            
              Junmin Wang,
            
              Steven Novick,
            
          </td>
          <td>2023-11-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e8a71562fd515b200d22ce0ceab058be500c70da" target='_blank'>Optimal Experimental Design for Parameter Estimation of a Cell Signaling Model</a></td>
          <td>
            
              Samuel Bandara,
            
              J. Schlöder,
            
              R. Eils,
            
              H. Bock,
            
              T. Meyer,
            
          </td>
          <td>2009-11-01</td>
          <td>PLoS Computational Biology</td>
          <td>128</td>
          <td>96</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/82729dc0ce591f4c6b7f084bab932f14b419f2a2" target='_blank'>Individualized Prediction of Drug Response and Rational Combination Therapy in NSCLC Using Artificial Intelligence–Enabled Studies of Acute Phosphoproteomic Changes</a></td>
          <td>
            
              E. Coker,
            
              A. Stewart,
            
              Bugra Ozer,
            
              A. Minchom,
            
              L. Pickard,
            
              R. Ruddle,
            
              S. Carreira,
            
              S. Popat,
            
              M. O'Brien,
            
              F. Raynaud,
            
              J. D. de Bono,
            
              B. Al-Lazikani,
            
              U. Banerji,
            
          </td>
          <td>2022-04-03</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>4</td>
          <td>116</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aa76568358955b754566591bd497ed51a8b52daa" target='_blank'>Reconstructing gene-regulatory networks from time series, knock-out data, and prior knowledge</a></td>
          <td>
            
              Florian Geier,
            
              J. Timmer,
            
              Christian Fleck,
            
          </td>
          <td>2007-02-02</td>
          <td>BMC Systems Biology</td>
          <td>110</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f348cef1025adac984ab9becbe84d2dbf3ea3e72" target='_blank'>Parameter optimization of linear ordinary differential equations with application in gene regulatory network inference problems</a></td>
          <td>
            
              Yue Deng,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4ac8c553765495313df0c13a9c57e5f618166194" target='_blank'>Predicting clinically promising therapeutic hypotheses using tensor factorization</a></td>
          <td>
            
              Jin Yao,
            
              M. Hurle,
            
              M. Nelson,
            
              Pankaj Agarwal,
            
          </td>
          <td>2018-02-27</td>
          <td>BMC Bioinformatics</td>
          <td>9</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c65cc3ce601427b7fa9002f4e84d8dd437052412" target='_blank'>Crosstalk between Plk1, p53, cell cycle, and G2/M DNA damage checkpoint regulation in cancer: computational modeling and analysis</a></td>
          <td>
            
              Yongwoon Jung,
            
              P. Kraikivski,
            
              Sajad Shafiekhani,
            
              S. Terhune,
            
              R. Dash,
            
          </td>
          <td>2021-12-01</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>22</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ce306f505f830a5a4ae7bc8a76ca5b76f2c13ed6" target='_blank'>Statistical Applications in Genetics and Molecular Biology Weighted-LASSO for Structured Network Inference from Time Course Data</a></td>
          <td>
            
              J. Chiquet,
            
              C. Ambroise,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>58</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ce306f505f830a5a4ae7bc8a76ca5b76f2c13ed6" target='_blank'>Statistical Applications in Genetics and Molecular Biology Weighted-LASSO for Structured Network Inference from Time Course Data</a></td>
          <td>
            
              J. Chiquet,
            
              C. Ambroise,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>58</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e94872b51e519d3887310522a621b3132f179864" target='_blank'>The iceberg plot, improving the visualisation of therapy response in oncology in the era of sequence-directed therapy.</a></td>
          <td>
            
              M. Lythgoe,
            
              T. Olivier,
            
              V. Prasad,
            
          </td>
          <td>2021-11-01</td>
          <td>European journal of cancer</td>
          <td>1</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2a0f2725f5b499c31267ec5d4e123a5e941ca626" target='_blank'>DCE-DForest: A Deep Forest Model for the Prediction of Anticancer Drug Combination Effects</a></td>
          <td>
            
              Wei Zhang,
            
              Ziyun Xue,
            
              Zhong Li,
            
              Huichao Yin,
            
          </td>
          <td>2022-06-09</td>
          <td>Computational and Mathematical Methods in Medicine</td>
          <td>4</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58472b88e77316e307943e685836dca035a39667" target='_blank'>Abstract 43: Using matrix factorization to discover clinically-relevant molecular signatures across cancers</a></td>
          <td>
            
              C. Szeto,
            
              Jingchun Zhu,
            
              James Durbin,
            
              S. Benz,
            
              Charles J. Vaske,
            
              Joshua M. Stuart,
            
              D. Haussler,
            
          </td>
          <td>2011-04-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>166</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d81874b97ba756da778ce3f5b67e8e378556e11e" target='_blank'>Predicting drug activity by image encoded gene expression profiles</a></td>
          <td>
            
              O. Özgül,
            
              Batuhan Bardak,
            
              Mehmet Tan,
            
          </td>
          <td>2018-05-01</td>
          <td>2018 26th Signal Processing and Communications Applications Conference (SIU)</td>
          <td>1</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94aef05cf8ef71ddfd9b8af79c809ba929f83728" target='_blank'>Predictive power of non-identifiable models</a></td>
          <td>
            
              F. Grabowski,
            
              Paweł Nałęcz-Jawecki,
            
              T. Lipniacki,
            
          </td>
          <td>2023-04-08</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4d5e8b5e0f04450eed81e58d731e3545d491661f" target='_blank'>Prediction of synergistic drug combinations using PCA-initialized deep learning</a></td>
          <td>
            
              Jun Ma,
            
              A. Motsinger-Reif,
            
          </td>
          <td>2021-10-20</td>
          <td>BioData Mining</td>
          <td>9</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ea96bb25a3d6b2119c909bda771bf01f73f17fdc" target='_blank'>Deep learning-based multi-drug synergy prediction model for individually tailored anti-cancer therapies</a></td>
          <td>
            
              Shengnan She,
            
              He-Gang Chen,
            
              Wei Ji,
            
              Mengqiu Sun,
            
              Jiaxi Cheng,
            
              Mengjie Rui,
            
              Chunlai Feng,
            
          </td>
          <td>2022-12-15</td>
          <td>Frontiers in Pharmacology</td>
          <td>3</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4aa16b6840474281e19ccebf78588f97e88f5440" target='_blank'>Convolutional Neural Network Can Recognize Drug Resistance of Single Cancer Cells</a></td>
          <td>
            
              Kiminori Yanagisawa,
            
              Masayasu Toratani,
            
              Ayumu Asai,
            
              M. Konno,
            
              H. Niioka,
            
              T. Mizushima,
            
              T. Satoh,
            
              J. Miyake,
            
              K. Ogawa,
            
              A. Vecchione,
            
              Y. Doki,
            
              H. Eguchi,
            
              H. Ishii,
            
          </td>
          <td>2020-04-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>11</td>
          <td>89</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5cf23bfc01b960926b574a122fd7b6591d06555c" target='_blank'>Multi-task learning predicts drug combination synergy in cells and in the clinic</a></td>
          <td>
            
              Coryandar Gilvary,
            
              J. Dry,
            
              O. Elemento,
            
          </td>
          <td>2019-03-13</td>
          <td>bioRxiv</td>
          <td>15</td>
          <td>100</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/05e4382614c79b87689f82e88e59e9b432d75dd6" target='_blank'>A context-aware deconfounding autoencoder for robust prediction of personalized clinical drug response from cell-line compound screening</a></td>
          <td>
            
              Di He,
            
              Qiao Liu,
            
              You Wu,
            
              Linfu Xie,
            
          </td>
          <td>2022-10-01</td>
          <td>Nature Machine Intelligence</td>
          <td>24</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c83d1e8c76022ff680d177ca3348933b6cfe624e" target='_blank'>Application of machine learning to large in vitro databases to identify drug–cancer cell interactions: azithromycin and KLK6 mutation status</a></td>
          <td>
            
              J. Sherman,
            
              Grant Verstandig,
            
              John W. Rowe,
            
              Yisroel Brumer,
            
          </td>
          <td>2021-05-05</td>
          <td>Oncogene</td>
          <td>7</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4f55e3db89685674c93300a11b867a758206582f" target='_blank'>Abstract 278: Artificial intelligence accelerate drug discovery</a></td>
          <td>
            
              Weidong Xie,
            
              Xiaoyan Cheng,
            
              Zhengfang Ding,
            
              R. Deng,
            
              D. Gu,
            
          </td>
          <td>2021-07-01</td>
          <td>Cancer Chemistry</td>
          <td>0</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/926c40504ecd230f9100f3f44beaa63011824120" target='_blank'>Clinical translation pathway for identifying patient-specific drugs based on predictive simulation outcomes with ex vivo validation.</a></td>
          <td>
            
              N. Doudican,
            
              S. Vali,
            
              A. Meredith,
            
              Anay A Talawdekar,
            
              Ansu Kumar,
            
              N. Singh,
            
              A. Tyagi,
            
              S. Kapoor,
            
              Zeba Sultana,
            
              T. Abbasi,
            
              A. Mazumder,
            
          </td>
          <td>2014-05-20</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ccfbbb4569576e6bdf81ea3255bd572b9d7d53e3" target='_blank'>Unleashing the Power of OpenAI in Shaping the Future of Cancer Research</a></td>
          <td>
            
              Ramyar M. Abdullah,
            
              Hawro D. I. Masseh,
            
              Abbas Salihi,
            
              Bawar Mohammed Faraj,
            
          </td>
          <td>2023-06-30</td>
          <td>BIOMED TARGET JOURNAL</td>
          <td>1</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7d1886668644cb7f87104f111cf506e2f13fa396" target='_blank'>In-silico Prediction of Synergistic Anti-Cancer Drug Combinations Using Multi-omics Data</a></td>
          <td>
            
              R. Çelebi,
            
              Oliver J Bear Don't Walk,
            
              Rajiv Movva,
            
              S. Alpsoy,
            
              M. Dumontier,
            
          </td>
          <td>2019-06-20</td>
          <td>Scientific Reports</td>
          <td>53</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/692e0e6f2473f62032a0bded3d08d247fb4514c3" target='_blank'>Interrogation of genome-wide networks in biology: comparison of knowledge-based and statistical methods</a></td>
          <td>
            
              Sathyabaarathi Ravichandran,
            
              N. Chandra,
            
          </td>
          <td>2019-01-01</td>
          <td>International Journal of Advances in Engineering Sciences and Applied Mathematics</td>
          <td>6</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/72f0e88f83171cd1673ec0a9e3f3d25dd207eff8" target='_blank'>From Cell-Lines to Cancer Patients: Personalized Drug Synergy Prediction</a></td>
          <td>
            
              Halil Ibrahim Kuru,
            
              A. E. Cicek,
            
              Oznur Tastan,
            
          </td>
          <td>2023-02-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/21880cde1cf543b6b8f3e159b08ea6d569561168" target='_blank'>Artificial Intelligence for Drug Discovery, Biomarker Development, and Generation of Novel Chemistry.</a></td>
          <td>
            
              A. Zhavoronkov,
            
          </td>
          <td>2018-10-01</td>
          <td>Molecular pharmaceutics</td>
          <td>97</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d942623eb6e5abb0113b4b67848107215da590b0" target='_blank'>Molecular generation strategy and optimization based on A2C reinforcement learning in de novo drug design</a></td>
          <td>
            
              Qian Wang,
            
              Zhiqiang Wei,
            
              Xiaotong Hu,
            
              Zhuoya Wang,
            
              Yujie Dong,
            
              Hao Liu,
            
          </td>
          <td>2023-11-01</td>
          <td>Bioinformatics</td>
          <td>1</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a359e0b47079d85bfe988545b1f972acf381533b" target='_blank'>Artificial intelligence techniques for prediction of drug synergy in malignant diseases: Past, present, and future</a></td>
          <td>
            
              P. Rani,
            
              K. Dutta,
            
              Vijay Kumar,
            
          </td>
          <td>2022-03-08</td>
          <td>Computers in biology and medicine</td>
          <td>6</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/48622f7caf0372a0e046b4a3deff8cb92b21ba48" target='_blank'>Targeting the Untargetable: Predicting Pramlintide Resistance Using a Neural Network Based Cellular Automata</a></td>
          <td>
            
              Eunjung Kim,
            
              R. Schenck,
            
              J. West,
            
              W. Cross,
            
              Valerie K. Harris,
            
              Joseph K McKenna,
            
              Heyrim Cho,
            
              E. Coker,
            
              Steven Lee-Kramer Kenneth Y. Tsai,
            
              E. Flores,
            
              Chandler D. Gatenbee,
            
          </td>
          <td>2017-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4d3e6f21fd428ef70ebfaec3144541679f16da42" target='_blank'>Innovative Drug-like Molecule Generation from Flow-based Generative Model</a></td>
          <td>
            
              Haotian Zhang,
            
              Lin Wan,
            
          </td>
          <td>2022-11-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b3be66ef0e461c8915807a6ce667b1c9afc2fc8" target='_blank'>DrugCom: Synergistic Discovery of Drug Combinations Using Tensor Decomposition</a></td>
          <td>
            
              Huiyuan Chen,
            
              Jing Li,
            
          </td>
          <td>2018-11-01</td>
          <td>2018 IEEE International Conference on Data Mining (ICDM)</td>
          <td>23</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6874f67fa13ae14208f2edaf4ed6bb8ccdcd632a" target='_blank'>Understanding Cancer Phenomenon at Gene Expression Level by using a Shallow Neural Network Chain</a></td>
          <td>
            
              Pietro Barbiero,
            
              A. Bertotti,
            
              Gabriele Ciravegna,
            
              G. Cirrincione,
            
              Elio Piccolo,
            
              A. Tonda,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>3</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ee807ca47062c79f73f6fe79dd21932be70ec75a" target='_blank'>Explaining Black Box Drug Target Prediction Through Model Agnostic Counterfactual Samples</a></td>
          <td>
            
              T. Nguyen,
            
              Thomas P. Quinn,
            
              Thin Nguyen,
            
              T. Tran,
            
          </td>
          <td>2022-07-12</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>4</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2f4b864db579dd094bc65955127f0ab74f14172c" target='_blank'>AI in drug discovery and its clinical relevance</a></td>
          <td>
            
              Rizwan Qureshi,
            
              Muhammad Irfan,
            
              Taimoor Muzaffar Gondal,
            
              Sheheryar Khan,
            
              Jia Wu,
            
              Muhammad Usman Hadi,
            
              J. Heymach,
            
              X. Le,
            
              Hong Yan,
            
              T. Alam,
            
          </td>
          <td>2023-06-26</td>
          <td>Heliyon</td>
          <td>29</td>
          <td>103</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/18ba1e1edf549a4e505189cc44685fcbab07876e" target='_blank'>PANCDR: precise medicine prediction using an adversarial network for cancer drug response</a></td>
          <td>
            
              Juyeon Kim,
            
              Sung-Hye Park,
            
              Hyunju Lee,
            
          </td>
          <td>2024-01-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8cc943fb1fc82322b1a4139f3a95df4307cf7aef" target='_blank'>AI identifies potent inducers of breast cancer stem cell differentiation based on adversarial learning from gene expression data</a></td>
          <td>
            
              Zhongxiao Li,
            
              A. Napolitano,
            
              M. Fedele,
            
              Xin Gao,
            
              F. Napolitano,
            
          </td>
          <td>2023-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7487897b97a4ac0c6a5bec0e5eb0b9066993494d" target='_blank'>Abstract P31: Graph-Based Deep Learning for Predicting Synergy in Multi-Drug Treatments for Leukemia</a></td>
          <td>
            
              Archishma Marrapu,
            
          </td>
          <td>2024-03-04</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f5a6b6c44e823a7689ccaa84f67f4d4a93748960" target='_blank'>A Hybrid Model Integrating Multi-Omic and Topological Information of PPI Network for Drug Synergism Prediction</a></td>
          <td>
            
              Giang T. T. Nguyen,
            
              Kien T. Phuong,
            
              Khanh Nguyen-Trong,
            
              Duc-Hau Le,
            
          </td>
          <td>2023-12-23</td>
          <td>2023 RIVF International Conference on Computing and Communication Technologies (RIVF)</td>
          <td>0</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0b309c0725c717989f6d6ae485fbab521e72d903" target='_blank'>The cornucopia of meaningful leads: Applying deep adversarial autoencoders for new molecule development in oncology</a></td>
          <td>
            
              Artur Kadurin,
            
              A. Aliper,
            
              A. Kazennov,
            
              Polina Mamoshina,
            
              Q. Vanhaelen,
            
              Kuzma Khrabrov,
            
              A. Zhavoronkov,
            
          </td>
          <td>2016-12-22</td>
          <td>Oncotarget</td>
          <td>276</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a07d5c391636b8179b61e9fcdf65779b964bc31e" target='_blank'>Predicting Drug Synergism by Means of Non-Negative Matrix Tri-Factorization</a></td>
          <td>
            
              Pietro Pinoli,
            
              G. Ceddia,
            
              S. Ceri,
            
              M. Masseroli,
            
          </td>
          <td>2021-06-24</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>5</td>
          <td>69</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/61f6fa295a9c2687cf2b7802565d170a3b356e80" target='_blank'>PiLSL: pairwise interaction learning-based graph neural network for synthetic lethality prediction in human cancers.</a></td>
          <td>
            
              Xin Liu,
            
              Jiale Yu,
            
              Siyu Tao,
            
              Beiyuan Yang,
            
              Shike Wang,
            
              Lin Wang,
            
              Fang Bai,
            
              Jie Zheng,
            
          </td>
          <td>2022-09-16</td>
          <td>Bioinformatics</td>
          <td>10</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cb81d359a8f4a00ca2593d46d29500e186f29312" target='_blank'>Abstract 4921: Dynamic and quantitative modeling of drug effect</a></td>
          <td>
            
              Xiangfang Li,
            
              Lijun Qian,
            
              M. Bittner,
            
              E. Dougherty,
            
          </td>
          <td>2011-04-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/91cd4fb4daba66066d58684e3eee294c40f013ea" target='_blank'>Inferring the Effects of Honokiol on the Notch Signaling Pathway in SW480 Colon Cancer Cells</a></td>
          <td>
            
              Michelle L. Wynn,
            
              N. Consul,
            
              S. Merajver,
            
              Santiago Schnell,
            
          </td>
          <td>2014-01-01</td>
          <td>Cancer Informatics</td>
          <td>2</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c31d121a7975d3f9f022e281d80400f3ea03d7d8" target='_blank'>Abstract 7373: Generative Bayesian networks for augmentation of molecular data from commercial genetics panels</a></td>
          <td>
            
              Dillon Tracy,
            
              Jeff Sherman,
            
              Maayan Baron,
            
          </td>
          <td>2024-03-22</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3d7dfa3edccebddf8c1d7ac000eabbd23ad5159a" target='_blank'>Inferring Gene Networks: Dream or Nightmare?</a></td>
          <td>
            
              A. Scheinine,
            
              Wieslawa I. Mentzen,
            
              G. Fotia,
            
              E. Pieroni,
            
              F. Maggio,
            
              G. Mancosu,
            
              A. G. de la Fuente,
            
          </td>
          <td>2009-03-01</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>14</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/56cb130a58fc544097f60dc2c75d6a300bf62397" target='_blank'>AI3SD Video: Machine Learning for Early Stage Drug Discovery</a></td>
          <td>
            
              C. Deane,
            
          </td>
          <td>2020-09-04</td>
          <td></td>
          <td>0</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/85f9785117850e2a7e56382297a23748ee4b049c" target='_blank'>Viral pandemic preparedness: A pluripotent stem cell‐based machine‐learning platform for simulating SARS‐CoV‐2 infection to enable drug discovery and repurposing</a></td>
          <td>
            
              S. Esmail,
            
              W. Danter,
            
          </td>
          <td>2020-09-22</td>
          <td>Stem Cells Translational Medicine</td>
          <td>19</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/66ca99e65476ce67ab99ce86af6fee547a3c1983" target='_blank'>Predicting the Effects of Drug Combinations Using Probabilistic Matrix Factorization</a></td>
          <td>
            
              Ron Nafshi,
            
              T. Lezon,
            
          </td>
          <td>2021-05-04</td>
          <td>Frontiers in Bioinformatics</td>
          <td>7</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fda524f3c8a733c747aa17d00127c339659accf7" target='_blank'>From intuition to AI: evolution of small molecule representations in drug discovery</a></td>
          <td>
            
              Miles McGibbon,
            
              S. Shave,
            
              Jie Dong,
            
              Yumiao Gao,
            
              Douglas R. Houston,
            
              Jiancong Xie,
            
              Yuedong Yang,
            
              P. Schwaller,
            
              Vincent Blay,
            
          </td>
          <td>2023-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>1</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d9b9460fbf943826cad599c25ba40395a39e91d7" target='_blank'>Reverse engineering of logic-based differential equation models using a mixed-integer dynamic optimization approach</a></td>
          <td>
            
              David Henriques,
            
              Miguel Rocha,
            
              J. Sáez-Rodríguez,
            
              J. Banga,
            
          </td>
          <td>2015-05-21</td>
          <td>Bioinformatics</td>
          <td>25</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d911eba195fe26e44f418c5dc8b6f64546679ccd" target='_blank'>Inference of Genetic Networks from Time Course Expression Data Using Functional Regression with Lasso Penalty</a></td>
          <td>
            
              Zhaoping Hong,
            
              H. Lian,
            
          </td>
          <td>2008-04-04</td>
          <td>Communications in Statistics - Theory and Methods</td>
          <td>14</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f145d87d9d6f8905892d8a538b0914aad1364c9c" target='_blank'>Dynamical structure function identifiability conditions enabling signal structure reconstruction</a></td>
          <td>
            
              J. Adebayo,
            
              T. Southwick,
            
              V. Chetty,
            
              Enoch Yeung,
            
              Ye Yuan,
            
              J. Gonçalves,
            
              J. Grose,
            
              J. Prince,
            
              G. Stan,
            
              S. Warnick,
            
          </td>
          <td>2012-12-01</td>
          <td>2012 IEEE 51st IEEE Conference on Decision and Control (CDC)</td>
          <td>54</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8cef4bd7e0883de1974515ff8fff9f19037fb862" target='_blank'>Network Reconstruction From High-Dimensional Ordinary Differential Equations</a></td>
          <td>
            
              Shizhe Chen,
            
              A. Shojaie,
            
              D. Witten,
            
          </td>
          <td>2016-10-11</td>
          <td>Journal of the American Statistical Association</td>
          <td>60</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/30fb5f7109b79d49706e8046c59865ffdf67c3ff" target='_blank'>An In-depth Summary of Recent Artificial Intelligence Applications in Drug Design</a></td>
          <td>
            
              Yi Zhang,
            
          </td>
          <td>2021-10-10</td>
          <td>ArXiv</td>
          <td>4</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3c353d680e899f86c341dd82c70bf6a0532deba4" target='_blank'>RediscMol: Benchmarking Molecular Generation Models in Biological Properties.</a></td>
          <td>
            
              Gaoqi Weng,
            
              Huifeng Zhao,
            
              Dou Nie,
            
              Haotian Zhang,
            
              Liwei Liu,
            
              Tingjun Hou,
            
              Yu Kang,
            
          </td>
          <td>2024-01-05</td>
          <td>Journal of medicinal chemistry</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/461fe23938c6cf74b7144ade64889ad8b1da3c38" target='_blank'>1279 A novel graphical deep neural network learning approach utilizing molecular data for optimizing patient selection for treatment with immune checkpoint inhibitors: An ORIEN pan-cancer study</a></td>
          <td>
            
              I. E. Naqa,
            
              ,
            
              A. Tan,
            
              James Chen,
            
              A. Ratan,
            
              M. McCarter,
            
              J. Carpten,
            
              H. Colman,
            
              A. Ikeguchi,
            
              I. Puzanov,
            
              S. Arnold,
            
              M. Churchman,
            
              P. Hwu,
            
              J. Conejo-Garcia,
            
              W. Dalton,
            
              G. Weiner,
            
              A. Tarhini,
            
          </td>
          <td>2022-11-01</td>
          <td>Regular and Young Investigator Award Abstracts</td>
          <td>0</td>
          <td>106</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e62c6a6187c95010492409238187eff0ba62436a" target='_blank'>A hybrid deep forest-based method for predicting synergistic drug combinations</a></td>
          <td>
            
              Lianlian Wu,
            
              Jie Gao,
            
              Yixin Zhang,
            
              Binsheng Sui,
            
              Yuqi Wen,
            
              Qingqiang Wu,
            
              Kunhong Liu,
            
              Song He,
            
              Xiaochen Bo,
            
          </td>
          <td>2023-02-01</td>
          <td>Cell Reports Methods</td>
          <td>9</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/da4c745d005bfaeff422b267c1fbf6226f153ae4" target='_blank'>[Prediction of drug-induced cell viability by SAE-XGBoost algorithm based on LINCS-L1000 perturbation signal].</a></td>
          <td>
            
              Jiaxing Lu,
            
              Ming Chen,
            
              Yufang Qin,
            
              Xiaoqing Yu,
            
          </td>
          <td>2021-04-25</td>
          <td>Sheng wu gong cheng xue bao = Chinese journal of biotechnology</td>
          <td>2</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/65dd5d5c72fc1eb9c4b573dfb1ba26eab831dcce" target='_blank'>SynAI: an AI-driven cancer drugs synergism prediction platform</a></td>
          <td>
            
              Kuan Yan,
            
              Runjun Jia,
            
              Sheng Guo,
            
          </td>
          <td>2023-11-10</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a6f98dd6e74e26b2cb814179f2117e6f01ebfba3" target='_blank'>FastFlows: Flow-Based Models for Molecular Graph Generation</a></td>
          <td>
            
              Nathan C Frey,
            
              V. Gadepally,
            
              Bharath Ramsundar,
            
          </td>
          <td>2022-01-28</td>
          <td>ArXiv</td>
          <td>6</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0dd77ad0509f3eb7ee1d9295221205798596e8de" target='_blank'>Gene Regulatory Networks: Improving Inferences with Transfer Learning.</a></td>
          <td>
            
              Marcelo Tigre Moura,
            
          </td>
          <td>2023-12-01</td>
          <td>Cellular reprogramming</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/55089f41b566a36dd346aaa09c6f6dc7bb51b7be" target='_blank'>Abstract 370: Modeling signaling networks in tumor immunology</a></td>
          <td>
            
              T. Clancy,
            
              E. Hovig,
            
          </td>
          <td>2014-10-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e6d6b08393c8621286f345b33a84a5665a780ffc" target='_blank'>Generative Network Complex for the Automated Generation of Drug-like Molecules</a></td>
          <td>
            
              Kaifu Gao,
            
              D. Nguyen,
            
              M. Tu,
            
              G. Wei,
            
          </td>
          <td>2020-05-28</td>
          <td>Journal of chemical information and modeling</td>
          <td>71</td>
          <td>56</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ed152e3e47524ef43a9aedc39a96365433384535" target='_blank'>Generative artificial intelligence empowers digital twins in drug discovery and clinical trials</a></td>
          <td>
            
              Maria Bordukova,
            
              Nikita Makarov,
            
              Raul Rodriguez-Esteban,
            
              Fabian Schmich,
            
              M. Menden,
            
          </td>
          <td>2023-10-27</td>
          <td>Expert Opinion on Drug Discovery</td>
          <td>4</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3919fd288507f0b456d2d12bc60cb9ebfc6f6a75" target='_blank'>Use of deep learning frameworks to detect super-responder and super-survivor stage IV squamous non-small-cell lung cancer (NSCLC) patients treated with a gemcitabine and cisplatin combination.</a></td>
          <td>
            
              C. Galmarini,
            
              M. Lucius,
            
          </td>
          <td>2021-05-20</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e8485a000603bb15d2382067114ca3f16414da74" target='_blank'>Cell system perturbation for time-resolved quantification of tyrosine phosphorylation in complex samples</a></td>
          <td>
            
              F. White,
            
          </td>
          <td>None</td>
          <td>2006 International Conference of the IEEE Engineering in Medicine and Biology Society</td>
          <td>0</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2b18251e7f43157132993961a9a79fbf7e8b5525" target='_blank'>Multiplying insights from perturbation experiments: predicting new perturbation combinations</a></td>
          <td>
            
              Joshua D. Welch,
            
          </td>
          <td>2023-05-11</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f604725106b9990702a3b081cb47f726c6113c0c" target='_blank'>Development and evaluation of a java-based deep neural network method for drug response predictions</a></td>
          <td>
            
              Beibei Huang,
            
              L. W. Fong,
            
              R. Chaudhari,
            
              Shuxing Zhang,
            
          </td>
          <td>2023-03-23</td>
          <td>Frontiers in Artificial Intelligence</td>
          <td>2</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/04fa5b69349b7a4ba1f060b69aff34639e57bbf8" target='_blank'>A molecular generative model with genetic algorithm and tree search for cancer samples</a></td>
          <td>
            
              Sejin Park,
            
              Hyunju Lee,
            
          </td>
          <td>2021-12-16</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/67a585a6dccdca81fc0cc59879eb06732b4d076d" target='_blank'>Abstract A35: Computer modeling of nocodazole exposure on cell cultures in vitro</a></td>
          <td>
            
              Eric Fernandez,
            
              G. Johnson,
            
              D. Orrell,
            
              D. Fell,
            
              S. Doak,
            
              C. Chassagnole,
            
          </td>
          <td>2009-12-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1237e86bc9d35cc5a1406f8a7df3ac4186258470" target='_blank'>The Concilium of Information Processing Networks of Chemical Oscillators for Determining Drug Response in Patients With Multiple Myeloma</a></td>
          <td>
            
              Ashmita Bose,
            
              P. Dittrich,
            
              J. Górecki,
            
          </td>
          <td>2022-07-08</td>
          <td>Frontiers in Chemistry</td>
          <td>1</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/153251c984fa453d9afb42ba6956ae17efe5fd06" target='_blank'>Protocol to use TopNet for gene regulatory network modeling using gene expression data from perturbation experiments</a></td>
          <td>
            
              H. McMurray,
            
              H. Stern,
            
              Aslihan Ambeskovic,
            
              Hartmut Land,
            
              M. Mccall,
            
          </td>
          <td>2022-09-30</td>
          <td>STAR Protocols</td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/646160e3596dba1480df71318fd9fb6022afc916" target='_blank'>Optimizing Drug Design by Merging Generative AI With Active Learning Frameworks</a></td>
          <td>
            
              Isaac Filella-Merce,
            
              Alexis Molina,
            
              Marek Orzechowski,
            
              Luc'ia D'iaz,
            
              Yangnv Zhu,
            
              Julia Vilalta Mor,
            
              Laura Malo,
            
              Ajay Yekkirala,
            
              Soumya S. Ray,
            
              V. Guallar,
            
          </td>
          <td>2023-05-04</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0900126a594d07ac519299b559b21b73d2e4ee5a" target='_blank'>Artificial Intelligence for Drug Discovery: Are We There Yet?</a></td>
          <td>
            
              C. Hasselgren,
            
              Tudor I. Oprea,
            
          </td>
          <td>2023-07-13</td>
          <td>Annual review of pharmacology and toxicology</td>
          <td>10</td>
          <td>75</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fb78e130b45b54e555ad434cd79cfb86fab4ceb0" target='_blank'>State-of-the-art of artificial intelligence in medicinal chemistry</a></td>
          <td>
            
              J. Bajorath,
            
          </td>
          <td>2021-03-29</td>
          <td>Future Science OA</td>
          <td>20</td>
          <td>67</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/393cbc262575fa28a2f8b853e23f70d04cc21d54" target='_blank'>UNRAVELLING THE INTERACTION BETWEEN GARCINISIDONE-A AND HER2 PROTEIN IN BREAST CANCER: A COMPUTATIONAL STUDY</a></td>
          <td>
            
              Mainal Furqan,
            
              Dachriyanus,
            
              Meri Susanti,
            
              Purnawan Pontana Putra,
            
              F. Wahyuni,
            
          </td>
          <td>2024-02-15</td>
          <td>International Journal of Applied Pharmaceutics</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/27cc8cc3c816c5ccaff972ad1c067a7ae349bbfb" target='_blank'>Logistic matrix factorisation and generative adversarial neural network-based method for predicting drug-target interactions</a></td>
          <td>
            
              Sarra Itidal Abbou,
            
              Hafida Bouziane,
            
              Abdallah Chouarfia,
            
          </td>
          <td>2021-07-23</td>
          <td>Molecular Diversity</td>
          <td>3</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/145f1d6b5fb152cc2b043240cef8c7a50c3a7f29" target='_blank'>Bridging in vitro and in vivo research via an agent-based modelling approach: predicting tumour responses to an ATR-inhibiting drug</a></td>
          <td>
            
              Sara Hamis,
            
              James Yates,
            
              Mark A. J. Chaplain,
            
              Gibin G. Powathil,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a01d196472c9d5335c21eaecf9653c3160adec74" target='_blank'>Abstract PR12: A model of HGF and EGF dual ligand stimulation in ACHN cells constructed with the aid of a novel model merging heuristic</a></td>
          <td>
            
              Andrew Matteson,
            
          </td>
          <td>2012-07-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cc6b644f1c8657dbc1bf64635c86052f1dc1e0a8" target='_blank'>Chemical Induced Differential Gene Expression Prediction on LINCS Database</a></td>
          <td>
            
              Riza Isik,
            
              Isiksu Eksioglu,
            
              Bahattin Can Maral,
            
              Benan Bardak,
            
              Mehmet Tan,
            
          </td>
          <td>2020-10-01</td>
          <td>2020 IEEE 20th International Conference on Bioinformatics and Bioengineering (BIBE)</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b2b3ab7e4f81ce7d09a8c387af70017a4e734bb2" target='_blank'>Benchmarking network algorithms for contextualizing genes of interest</a></td>
          <td>
            
              Abby Hill,
            
              S. Gleim,
            
              Florian Kiefer,
            
              Frederic D. Sigoillot,
            
              J. Loureiro,
            
              J. Jenkins,
            
              Melody K. Morris,
            
          </td>
          <td>2019-12-01</td>
          <td>PLoS Computational Biology</td>
          <td>5</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dfe72973324814adba51fac772c9a52ca53024cd" target='_blank'>Transformer-Based Generative Model Accelerating the Development of Novel BRAF Inhibitors</a></td>
          <td>
            
              Lijuan Yang,
            
              Guanghui Yang,
            
              Zhitong Bing,
            
              Yuan Tian,
            
              Yuzhen Niu,
            
              Liang Huang,
            
              Lei Yang,
            
          </td>
          <td>2021-12-01</td>
          <td>ACS Omega</td>
          <td>22</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0f63bc016a5effd897abf3ea7d759d646ba614c9" target='_blank'>Deep generative models in the quest for anticancer drugs: ways forward</a></td>
          <td>
            
              Virgilio Romanelli,
            
              Carmen Cerchia,
            
              A. Lavecchia,
            
          </td>
          <td>2024-02-02</td>
          <td>Frontiers in Drug Discovery</td>
          <td>0</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4f4e46c60c3f75bfed23be152dddb1beca797a26" target='_blank'>PDCD5 interacts with p53 and functions as a regulator of p53 dynamics in the DNA damage response</a></td>
          <td>
            
              Changjing Zhuge,
            
              Xiaojuan Sun,
            
              Yingyu Chen,
            
              J. Lei,
            
          </td>
          <td>2015-03-28</td>
          <td>arXiv: Molecular Networks</td>
          <td>0</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c01089451be26600b0988f4a5d8f49768e9d8642" target='_blank'>AI-Driven Drug Discovery: Unravelling the Potential of Generative Adversarial Networks (GANs) in Pharmaceutical Research</a></td>
          <td>
            
              Srinivasa Rao Burri,
            
              Mamadou Yero Diallo,
            
              Lakshay Sharma,
            
              Vishal Dutt,
            
          </td>
          <td>2023-11-01</td>
          <td>2023 3rd International Conference on Technological Advancements in Computational Sciences (ICTACS)</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bb6f75a3b81294379ff044d64c58e4dda9d3539f" target='_blank'>A Survey of Generative AI for De Novo Drug Design: New Frontiers in Molecule and Protein Generation</a></td>
          <td>
            
              Xiangru Tang,
            
              Howard Dai,
            
              Elizabeth Knight,
            
              Fang Wu,
            
              Yunyang Li,
            
              Tianxiao Li,
            
              Mark Gerstein,
            
          </td>
          <td>2024-02-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ef83973b1d91bd524308ec60ae821fbeaa515c21" target='_blank'>The Hitchhiker’s Guide to Deep Learning Driven Generative Chemistry</a></td>
          <td>
            
              Y. Ivanenkov,
            
              B. Zagribelnyy,
            
              Alex Malyshev,
            
              Sergei Evteev,
            
              V. Terentiev,
            
              Petrina Kamya,
            
              Dmitry Bezrukov,
            
              A. Aliper,
            
              Fengzhi Ren,
            
              A. Zhavoronkov,
            
          </td>
          <td>2023-06-29</td>
          <td>ACS Medicinal Chemistry Letters</td>
          <td>7</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f0afbce39d2d0249673b17dadd78ed31698a59c5" target='_blank'>Introduction: An Overview of AI in Oncology Drug Discovery and Development</a></td>
          <td>
            
              K. Linton-Reid,
            
          </td>
          <td>2020-09-09</td>
          <td></td>
          <td>4</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ba5eeaa9b1bd2feb431953aa1b868303776af3bf" target='_blank'>De novo drug design through artificial intelligence: an introduction</a></td>
          <td>
            
              Davide Crucitti,
            
              Carlos Pérez Míguez,
            
              J. D. Díaz Arias,
            
              Diego Beltrán Fernandez Prada,
            
              A. Mosquera Orgueira,
            
          </td>
          <td>2024-01-25</td>
          <td>Frontiers in Hematology</td>
          <td>0</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/50e1e8e609fd2a7ae4667ff94619dfc3b6064b7d" target='_blank'>Synergistic drug combinations prediction by integrating pharmacological data</a></td>
          <td>
            
              ,
            
              Guiying Yan,
            
          </td>
          <td>2019-02-07</td>
          <td>Synthetic and Systems Biotechnology</td>
          <td>18</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d2bf5811dedd4393a3f2340ee0995e638ee27805" target='_blank'>Revolutionizing Medicinal Chemistry: The Application of Artificial Intelligence (AI) in Early Drug Discovery</a></td>
          <td>
            
              Ri Han,
            
              Hongryul Yoon,
            
              Gahee Kim,
            
              Hyundo Lee,
            
              Yoonji Lee,
            
          </td>
          <td>2023-09-01</td>
          <td>Pharmaceuticals</td>
          <td>7</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8e4e379bebebcd91c6c4250aba38dea5569bb4ae" target='_blank'>A novel molecule generative model of VAE combined with Transformer</a></td>
          <td>
            
              Yasuhiro Yoshikai,
            
              T. Mizuno,
            
              Shumpei Nemoto,
            
              Hiroyuki Kusuhara,
            
          </td>
          <td>2024-02-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/56c03c1dca5ebe5b4d139fee23a42090e24643ab" target='_blank'>Abstract ED3-2: Understanding melanoma using GWAS and systems biology</a></td>
          <td>
            
              J. A. Bishop,
            
          </td>
          <td>2010-04-01</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    // columnDefs: [
    //     {"className": "dt-center", "targets": "_all"},
    //     // set width for both columns 0 and 1 as 25%
    //     { width: '40%', targets: 4 }

    //   ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
</script>
</html>