---
hide:
 - navigation
---
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-05-06 06:08:01 UTC</i>
  </p>
  

  <p>
  <h3>Recommendations for the article <i>CellBox: Interpretable Machine Learning for Perturbation Biology with Application to the Design of Cancer Combination Therapy.</i></h3>
  <table id="table1" class="display" style="width:100%">
  <thead>
    <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/250017f99bdff84eb0a2ac65698224b4abe7c8a4" target='_blank'>Interpretable Machine Learning for Perturbation Biology</a></td>
          <td>
            
              Bo Yuan,
            
              Ciyue Shen,
            
              Augustin Luna,
            
              Anil Korkut,
            
              D. Marks,
            
              John Ingraham,
            
              C. Sander,
            
          </td>
          <td>2019-08-28</td>
          <td>bioRxiv</td>
          <td>6</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e6f21c31f5c969c279bb555f1dff954d04e64763" target='_blank'>Abstract 2102: Interpretable machine learning for perturbation biology</a></td>
          <td>
            
              Judy Shen,
            
              Bo Yuan,
            
              Augustin Luna,
            
              Anil Korkut,
            
              D. Marks,
            
              John Ingraham,
            
              C. Sander,
            
          </td>
          <td>2020-08-13</td>
          <td>Clinical Research (Excluding Clinical Trials)</td>
          <td>0</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/44c74be92bfee5ad8a4f41543b36dd3d2c83e2bf" target='_blank'>Interpretable predictions of cellular behavior</a></td>
          <td>
            
              Ananya Rastogi,
            
          </td>
          <td>2021-03-01</td>
          <td>Nature Computational Science</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1e881a1d1a8c8f35ba366d63693c2ac0094344e4" target='_blank'>Perturbation Biology: Inferring Signaling Networks in Cellular Systems</a></td>
          <td>
            
              Evan J. Molinelli,
            
              Anil Korkut,
            
              Weiqing Wang,
            
              Martin L. Miller,
            
              N. Gauthier,
            
              Xiaohong Jing,
            
              Poorvi Kaushik,
            
              Qin He,
            
              Gordon Mills,
            
              D. Solit,
            
              C. Pratilas,
            
              M. Weigt,
            
              A. Braunstein,
            
              A. Pagnani,
            
              R. Zecchina,
            
              C. Sander,
            
          </td>
          <td>2013-08-23</td>
          <td>PLoS Computational Biology</td>
          <td>129</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/14c762a67f5a9622f2d23ad99cb08992af10b2d5" target='_blank'>Perturbation biology links temporal protein changes to drug responses in a melanoma cell line</a></td>
          <td>
            
              Elin Nyman,
            
              R. Stein,
            
              Xiaohong Jing,
            
              Weiqing Wang,
            
              Benjamin Marks,
            
              I. Zervantonakis,
            
              Anil Korkut,
            
              N. Gauthier,
            
              C. Sander,
            
          </td>
          <td>2019-03-06</td>
          <td>PLoS Computational Biology</td>
          <td>12</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fbff9ee6f0bdabf0ef4cd37032398f2655399604" target='_blank'>Causal Models, Prediction, and Extrapolation in Cell Line Perturbation Experiments</a></td>
          <td>
            
              J. Long,
            
              Yumeng Yang,
            
              K. Do,
            
          </td>
          <td>2022-07-20</td>
          <td>None</td>
          <td>0</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/219ca99cb481dd119358086b5ced78904cf72071" target='_blank'>Interpretable Modeling of Single-cell perturbation Responses to Novel Drugs Using Cycle Consistence Learning</a></td>
          <td>
            
              Wei Huang,
            
              Aichun Zhu,
            
              Hui Liu,
            
          </td>
          <td>2023-11-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/39f3131ed92cb56b1c5af701ea805e6fd9bbe339" target='_blank'>Predicting dynamic signaling network response under unseen perturbations</a></td>
          <td>
            
              Fan Zhu,
            
              Y. Guan,
            
          </td>
          <td>2014-10-01</td>
          <td>Bioinformatics</td>
          <td>15</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1a792cb6bdc69d6f3bb2c8def0809b7dbd05d8f2" target='_blank'>CODEX: COunterfactual Deep learning for the in-silico EXploration of cancer cell line perturbations</a></td>
          <td>
            
              Stefan Schrod,
            
              Tim Beißbarth,
            
              Helena U. Zacharias,
            
              Anne-Christin Hauschild,
            
              M. Altenbuchinger,
            
          </td>
          <td>2024-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d60bdd96b46505ab91ee904706c4a5cf4772b6d0" target='_blank'>The CausalBench challenge: A machine learning contest for gene network inference from single-cell perturbation data</a></td>
          <td>
            
              Mathieu Chevalley,
            
              Jacob A. Sackett-Sanders,
            
              Yusuf H. Roohani,
            
              Pascal Notin,
            
              A. Bakulin,
            
              D. Brzeziński,
            
              Kaiwen Deng,
            
              Y. Guan,
            
              Justin Hong,
            
              Michael Ibrahim,
            
              W. Kotłowski,
            
              Marcin Kowiel,
            
              Panagiotis Misiakos,
            
              Achille Nazaret,
            
              M. Puschel,
            
              Chris Wendler,
            
              Arash Mehrjou,
            
              P. Schwab,
            
          </td>
          <td>2023-08-29</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c5bc5d88ddc9537c194384668aefbb99268f1d44" target='_blank'>Learning interpretable cellular responses to complex perturbations in high-throughput screens</a></td>
          <td>
            
              M. Lotfollahi,
            
              Anna Klimovskaia Susmelj,
            
              C. De Donno,
            
              Yuge Ji,
            
              I. Ibarra,
            
              F. A. Wolf,
            
              N. Yakubova,
            
              Fabian J Theis,
            
              David Lopez-Paz,
            
          </td>
          <td>2021-04-15</td>
          <td>bioRxiv</td>
          <td>41</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e7b46dd8732444e1453773ae3e26cae69554a9d3" target='_blank'>Data-driven reverse engineering of signaling pathways using ensembles of dynamic models</a></td>
          <td>
            
              David Henriques,
            
              A. F. Villaverde,
            
              Miguel Rocha,
            
              J. Sáez-Rodríguez,
            
              J. Banga,
            
          </td>
          <td>2017-02-01</td>
          <td>PLoS Computational Biology</td>
          <td>41</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b993ca25b69f33a600d79449330321e036281b5" target='_blank'>Leveraging multi-way interactions for systematic prediction of pre-clinical drug combination effects</a></td>
          <td>
            
              H. Julkunen,
            
              A. Cichońska,
            
              Prson Gautam,
            
              S. Szedmák,
            
              Jane Douat,
            
              T. Pahikkala,
            
              T. Aittokallio,
            
              Juho Rousu,
            
          </td>
          <td>2020-12-01</td>
          <td>Nature Communications</td>
          <td>60</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/08f6856e3bac8359095bb47767c4531ec31f564e" target='_blank'>Abstract A24: Network models of signaling and drug response in melanoma.</a></td>
          <td>
            
              Anil Korkut,
            
              Weiqing Wang,
            
              Evan J. Molinelli,
            
              Martin L. Miller,
            
              Poorvi Kaushik,
            
              Arman Aksoy,
            
              Xiaohong Jing,
            
              N. Gauthier,
            
              C. Sander,
            
          </td>
          <td>2013-11-01</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2432177fcc1ec5b5126043bc973f2239ce50e756" target='_blank'>comboFM: leveraging multi-way interactions for systematic prediction of drug combination effects</a></td>
          <td>
            
              H. Julkunen,
            
              A. Cichońska,
            
              Prson Gautam,
            
              S. Szedmák,
            
              Jane Douat,
            
              T. Pahikkala,
            
              T. Aittokallio,
            
              Juho Rousu,
            
          </td>
          <td>2020-09-02</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/69345d79ff4c375b54d763a2037d2ca9eb40812d" target='_blank'>Embedding single-cell experimental conditions to reveal manifold structure of cancer drug perturbation effects</a></td>
          <td>
            
              William S. Chen,
            
              Nevena Zivanovic,
            
              D. V. Dijk,
            
              Guy Wolf,
            
              B. Bodenmiller,
            
              Smita Krishnaswamy,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/79a2df200d2aa14a19cd5c25fb0db7833cdfb3f6" target='_blank'>MECHANISTIC AND DATA-DRIVEN MODELS OF CELL SIGNALING: TOOLS FOR FUNDAMENTAL DISCOVERY AND RATIONAL DESIGN OF THERAPY.</a></td>
          <td>
            
              Paul J. Myers,
            
              Sung Hyun Lee,
            
              M. Lazzara,
            
          </td>
          <td>2021-06-01</td>
          <td>Current opinion in systems biology</td>
          <td>6</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f5e380b3b0534a5e89ba5338fb4dce4b84ec6d13" target='_blank'>Predicting cellular responses to complex perturbations in high‐throughput screens</a></td>
          <td>
            
              M. Lotfollahi,
            
              Anna Klimovskaia Susmelj,
            
              C. De Donno,
            
              Leon Hetzel,
            
              Yuge Ji,
            
              Ignacio L Ibarra,
            
              Sanjay R Srivatsan,
            
              Mohsen Naghipourfar,
            
              R. Daza,
            
              Beth K. Martin,
            
              J. Shendure,
            
              José L. McFaline-Figueroa,
            
              Pierre Boyeau,
            
              F. A. Wolf,
            
              N. Yakubova,
            
              Stephan Günnemann,
            
              C. Trapnell,
            
              David Lopez-Paz,
            
              Fabian J Theis,
            
          </td>
          <td>2023-05-08</td>
          <td>Molecular Systems Biology</td>
          <td>44</td>
          <td>142</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b44faa0d29a0a0a5c8a1f0d1fa52c4b8b2d33eb0" target='_blank'>Learning single-cell perturbation responses using neural optimal transport</a></td>
          <td>
            
              Charlotte Bunne,
            
              Stefan G. Stark,
            
              Gabriele Gut,
            
              Jacobo Sarabia Del Castillo,
            
              K. Lehmann,
            
              L. Pelkmans,
            
              Andreas Krause,
            
              G. Rätsch,
            
          </td>
          <td>2021-12-15</td>
          <td>Nature Methods</td>
          <td>47</td>
          <td>69</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/500b567ec04a718411eb4a59b5b37f83439cb68d" target='_blank'>Capturing Biomarkers and Molecular Targets in Cellular Landscapes From Dynamic Reaction Network Models and Machine Learning</a></td>
          <td>
            
              S. Mertins,
            
          </td>
          <td>2022-01-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f7be3f64b10c688aabb1fccc48bd867ba6ecadf1" target='_blank'>Efficient parameterization of large-scale mechanistic models enables drug response prediction for cancer cell lines</a></td>
          <td>
            
              Fabian Froehlich,
            
              T. Kessler,
            
              Daniel Weindl,
            
              A. Shadrin,
            
              Leonard Schmiester,
            
              Hendrik Hache,
            
              A. Muradyan,
            
              M. Schuette,
            
              J. Lim,
            
              M. Heinig,
            
              Fabian J Theis,
            
              H. Lehrach,
            
              C. Wierling,
            
              B. Lange,
            
              J. Hasenauer,
            
          </td>
          <td>2017-08-09</td>
          <td>bioRxiv</td>
          <td>10</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f64ee6a55b38ec3bed3836d96a73e4dc8508ffc" target='_blank'>CellNOptR: a flexible toolkit to train protein signaling networks to data using multiple logic formalisms</a></td>
          <td>
            
              Camille Terfve,
            
              T. Cokelaer,
            
              David Henriques,
            
              Aidan MacNamara,
            
              E. Gonçalves,
            
              Melody K. Morris,
            
              M. V. Iersel,
            
              D. Lauffenburger,
            
              J. Sáez-Rodríguez,
            
          </td>
          <td>2012-10-01</td>
          <td>BMC Systems Biology</td>
          <td>198</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b05d741198474f367df5476c770f0bf45625d17" target='_blank'>Converting networks to predictive logic models from perturbation signalling data with CellNOpt</a></td>
          <td>
            
              Enio Gjerga,
            
              P. Trairatphisan,
            
              A. Gábor,
            
              H. Koch,
            
              C. Chevalier,
            
              Francesco Ceccarelli,
            
              Aurélien Dugourd,
            
              A. Mitsos,
            
              J. Saez-Rodriguez,
            
          </td>
          <td>2020-03-05</td>
          <td>Bioinformatics</td>
          <td>15</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/23532e195730c4f109f158a7a56f2f9f8ee215e1" target='_blank'>Network-driven cancer cell avatars for combination discovery and biomarker identification for DNA Damage Response inhibitors</a></td>
          <td>
            
              O. Papp,
            
              Viktória Jordán,
            
              Szabolcs Hetey,
            
              Róbert Balázs,
            
              Árpád Bartha,
            
              Nóra N. Ordasi,
            
              Sebestyén Kamp,
            
              Bálint Farkas,
            
              Jay Mettetal,
            
              J. Dry,
            
              Duncan Young,
            
              Ben Sidders,
            
              K. Bulusu,
            
              D. Veres,
            
          </td>
          <td>2023-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/95d29815ae80adf97b5be8fddd1f37e415ce96c1" target='_blank'>Data used in "comboFM: leveraging multi-way interactions for systematic prediction of drug combination effects"</a></td>
          <td>
            
              H. Julkunen,
            
              A. Cichońska,
            
              Prson Gautam,
            
              S. Szedmák,
            
              Jane Douat,
            
              T. Pahikkala,
            
              T. Aittokallio,
            
          </td>
          <td>2020-05-02</td>
          <td></td>
          <td>0</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5cb58a880a3a63f790270c7e9c0886da57fa90d1" target='_blank'>Modeling interpretable correspondence between cell state and perturbation response with CellCap</a></td>
          <td>
            
              Yang Xu,
            
              Stephen J. Fleming,
            
              M. Tegtmeyer,
            
              S. Mccarroll,
            
              M. Babadi,
            
          </td>
          <td>2024-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2aac28eedb11ab7a15d2b095b276547fa03db580" target='_blank'>GEARS: Predicting transcriptional outcomes of novel multi-gene perturbations</a></td>
          <td>
            
              Yusuf H. Roohani,
            
              Kexin Huang,
            
              J. Leskovec,
            
          </td>
          <td>2022-07-15</td>
          <td>bioRxiv</td>
          <td>18</td>
          <td>132</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca6f0093c3490e7b937a7e37c5add468630618bd" target='_blank'>Abstract 5227: Quantitative network models of signaling and drug response in melanoma.</a></td>
          <td>
            
              Anil Korkut,
            
              Weiqing Wang,
            
              Evan J. Molinelli,
            
              Martin L. Miller,
            
              Poorvi Kaushik,
            
              Arman Aksoy,
            
              C. Sander,
            
          </td>
          <td>2013-04-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6cab87236aa24a5b487870d090335444ce2806d8" target='_blank'>Predicting Cellular Responses to Novel Drug Perturbations at a Single-Cell Resolution</a></td>
          <td>
            
              Leon Hetzel,
            
              Simon Böhm,
            
              Niki Kilbertus,
            
              Stephan Günnemann,
            
              M. Lotfollahi,
            
              F. Theis,
            
          </td>
          <td>2022-04-28</td>
          <td>None</td>
          <td>18</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/85906593661911c9a076cbb96391b9bb61a996bc" target='_blank'>Predicting transcriptional outcomes of novel multigene perturbations with GEARS.</a></td>
          <td>
            
              Yusuf H. Roohani,
            
              Kexin Huang,
            
              J. Leskovec,
            
          </td>
          <td>2023-08-17</td>
          <td>Nature biotechnology</td>
          <td>26</td>
          <td>132</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/111bc6c6e664a124d2cdf18572b0cdaf714ca0f1" target='_blank'>Predicting single-cell perturbation responses for unseen drugs</a></td>
          <td>
            
              Leon Hetzel,
            
              Simon Böhm,
            
              Niki Kilbertus,
            
              Stephan Günnemann,
            
              M. Lotfollahi,
            
              F. Theis,
            
          </td>
          <td>None</td>
          <td>ArXiv</td>
          <td>7</td>
          <td>45</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ee25812a45c166d4bda91a4d6196677577af78a8" target='_blank'>Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-Cancer Pathway Model.</a></td>
          <td>
            
              Fabian Fröhlich,
            
              T. Kessler,
            
              Daniel Weindl,
            
              A. Shadrin,
            
              Leonard Schmiester,
            
              Hendrik Hache,
            
              A. Muradyan,
            
              M. Schütte,
            
              J. Lim,
            
              M. Heinig,
            
              Fabian J Theis,
            
              H. Lehrach,
            
              C. Wierling,
            
              B. Lange,
            
              J. Hasenauer,
            
          </td>
          <td>2018-12-01</td>
          <td>Cell systems</td>
          <td>94</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/eefeef4ec68a6eef64a8f476f48831037bd03b85" target='_blank'>Quantitative Learning of Cellular Features From Single-cell Transcriptomics Data Facilitates Effective Drug Repurposing</a></td>
          <td>
            
              Jianmei Zhong,
            
              Junyao Yang,
            
              Yinghui Song,
            
              Zhihua Zhang,
            
              Chunming Wang,
            
              Renyang Tong,
            
              Chenglong Li,
            
              Nanhui Yu,
            
              L. Zou,
            
              Sulai Liu,
            
              Jun Pu,
            
              Wei Lin,
            
          </td>
          <td>2023-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4a3ba00d086079efd37f45e95d0ff4cc1155a08c" target='_blank'>Learning Perturbation-specific Cell Representations for Prediction of Transcriptional Response across Cellular Contexts</a></td>
          <td>
            
              Gal Keinan,
            
              K. Sayal,
            
              Alon Gonen,
            
              Jiang Zhu,
            
              L. Granovsky,
            
              Jeremy England,
            
          </td>
          <td>2023-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8d19c68cada5e1d50d801a06b3acae0ab32c451a" target='_blank'>Predicting Drug Response and Synergy Using a Deep Learning Model of Human Cancer Cells.</a></td>
          <td>
            
              B. Kuenzi,
            
              Jisoo Park,
            
              Samson H. Fong,
            
              Kyle S. Sanchez,
            
              John Lee,
            
              J. Kreisberg,
            
              Jianzhu Ma,
            
              T. Ideker,
            
          </td>
          <td>2020-10-21</td>
          <td>Cancer cell</td>
          <td>202</td>
          <td>80</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ed6bdc8d57a8547923d000da4cc87c7013332ab1" target='_blank'>PerturbNet predicts single-cell responses to unseen chemical and genetic perturbations</a></td>
          <td>
            
              Hengshi Yu,
            
              Joshua D. Welch,
            
          </td>
          <td>2022-07-21</td>
          <td>bioRxiv</td>
          <td>10</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f7102460a7c1c25f8491fc3b49ed44b276f9e4f" target='_blank'>ComboPath: An ML system for predicting drug combination effects with superior model specification</a></td>
          <td>
            
              D. Ranasinghe,
            
              Nathan Sanders,
            
              Hok Hei Tam,
            
              Changchang Liu,
            
              Dan Spitz,
            
          </td>
          <td>2024-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/86854a163a91862ad25919cbd970fd100a4b815e" target='_blank'>Models from experiments: combinatorial drug perturbations of cancer cells</a></td>
          <td>
            
              Sven Nelander,
            
              Weiqing Wang,
            
              B. Nilsson,
            
              Q. She,
            
              C. Pratilas,
            
              N. Rosen,
            
              P. Gennemark,
            
              C. Sander,
            
          </td>
          <td>2008-09-02</td>
          <td>Molecular Systems Biology</td>
          <td>196</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/80440d644237300f0163522ecd6327a9ce4184e3" target='_blank'>Abstract 1925: DrugCell: A visible neural network to guide personalized medicine</a></td>
          <td>
            
              B. Kuenzi,
            
              Jisoo Park,
            
              Samson H. Fong,
            
              Kyle S. Sanchez,
            
              J. Kreisberg,
            
              Jianzhu Ma,
            
              T. Ideker,
            
          </td>
          <td>2020-08-13</td>
          <td>Experimental and Molecular Therapeutics</td>
          <td>0</td>
          <td>80</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/75691e7eb6ec25249c8628104bba4f9dfe263b2f" target='_blank'>Cancer drug discovery as a low rank tensor completion problem</a></td>
          <td>
            
              V. S. Murali,
            
              Didem Ağaç Çobanoğlu,
            
              Michael Hsieh,
            
              Meyer Zinn,
            
              V. Malladi,
            
              J. Gesell,
            
              N. Williams,
            
              Erik S. Welf,
            
              G. Raj,
            
              M. Çobanoğlu,
            
          </td>
          <td>2021-03-09</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d130ad02e49f5e2d7ef0a024f965e2c0adada58f" target='_blank'>MEMMAL: A tool for expanding large-scale mechanistic models with machine learned associations and big datasets</a></td>
          <td>
            
              Cemal Erdem,
            
              M. Birtwistle,
            
          </td>
          <td>2023-03-09</td>
          <td>Frontiers in systems biology</td>
          <td>1</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2469f98d9f3edae53e295a96083f4d0b3bb474c3" target='_blank'>Inferring perturbation profiles of cancer samples</a></td>
          <td>
            
              M. Pirkl,
            
              N. Beerenwinkel,
            
          </td>
          <td>2020-12-10</td>
          <td>Bioinformatics</td>
          <td>1</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c69c5c3591914755d090a34a21005ca228613c55" target='_blank'>Abstract C006: DrugCell: A visible neural network to guide precision medicine</a></td>
          <td>
            
              B. Kuenzi,
            
              Jisoo Park,
            
              Samson H. Fong,
            
              J. Kreisberg,
            
              T. Ideker,
            
          </td>
          <td>2019-12-01</td>
          <td>Pharmacogenetics, Pharmacogenomics, and Therapeutic Response</td>
          <td>0</td>
          <td>80</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a9c18fd2ea917e6bd9c3ae0dad21ffdcf793d877" target='_blank'>Integrating network reconstruction with mechanistic modeling to predict cancer therapies</a></td>
          <td>
            
              M. Halasz,
            
              B. Kholodenko,
            
              W. Kolch,
            
              T. Santra,
            
          </td>
          <td>2016-11-22</td>
          <td>Science Signaling</td>
          <td>51</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6d687d12821b42f2e63d93c7838343a822b3b630" target='_blank'>Neural optimal transport predicts perturbation responses at the single-cell level</a></td>
          <td>
            
          </td>
          <td>2023-09-28</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f86fe76b2eafd6a25499a2f6b022f10b26513dc9" target='_blank'>Predicting Drug Mechanics by Deep Learning on Gene and Cell Activities</a></td>
          <td>
            
              Abhishek Dutta,
            
          </td>
          <td>2022-07-01</td>
          <td>2022 44th Annual International Conference of the IEEE Engineering in Medicine & Biology Society (EMBC)</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a9c18fd2ea917e6bd9c3ae0dad21ffdcf793d877" target='_blank'>Integrating network reconstruction with mechanistic modeling to predict cancer therapies</a></td>
          <td>
            
              M. Halasz,
            
              B. Kholodenko,
            
              W. Kolch,
            
              T. Santra,
            
          </td>
          <td>2016-11-22</td>
          <td>Science Signaling</td>
          <td>51</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/43471f39f0d6bc8ea5aba99851cc98d35af550af" target='_blank'>Predicting synergistic target and drug target combinations in a large-scale data-driven simulation of cancer cells.</a></td>
          <td>
            
              T. Haberichter,
            
              Rina Gindelman,
            
              R. Christopher,
            
              A. Dhiman,
            
              C. Hill,
            
              I. Khalil,
            
          </td>
          <td>2004-04-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5d82430a94a815be7801006aa8cf759bb0c3fc36" target='_blank'>Running title : A framework for pathway inference and modelling Character count : 48856</a></td>
          <td>
            
              M. Halasz,
            
              B. Kholodenko,
            
              W. Kolch,
            
              T. Santra,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ebc9ddd822239cc4e456b720a62084216223d598" target='_blank'>Cell reprogramming design by transfer learning of functional transcriptional networks.</a></td>
          <td>
            
              T. Wytock,
            
              A. Motter,
            
          </td>
          <td>2024-03-04</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/516c13660aa9eadff6cefbce4ac0292281fdc217" target='_blank'>Reconstructing and comparing signal transduction networks from single cell protein quantification data</a></td>
          <td>
            
              Tim Stohn,
            
              Roderick van Eijl,
            
              Klaas W. Mulder,
            
              L. Wessels,
            
              Evert Bosdriesz,
            
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e91e7987688bdf0c489f65475d14e6e0c48fd69c" target='_blank'>Personalized logical models to investigate cancer response to BRAF treatments in melanomas and colorectal cancers</a></td>
          <td>
            
              Jonas Béal,
            
              L. Pantolini,
            
              Vincent Noël,
            
              E. Barillot,
            
              Laurence Calzone,
            
          </td>
          <td>2020-05-27</td>
          <td>bioRxiv</td>
          <td>29</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf7bdef0a58cda1802097f3b9a57e950a4613b78" target='_blank'>kboolnet: a toolkit for the verification, validation, and visualization of reaction-contingency (rxncon) models</a></td>
          <td>
            
              Willow Carretero Chavez,
            
              Marcus Krantz,
            
              E. Klipp,
            
              I. Kufareva,
            
          </td>
          <td>2023-06-12</td>
          <td>BMC Bioinformatics</td>
          <td>1</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b83bb2aa35405d23f9bea3e6411e6d9a634e4355" target='_blank'>Logic-based modeling and drug repurposing for the prediction of novel therapeutic targets and combination regimens against E2F1-driven melanoma progression</a></td>
          <td>
            
              Nivedita Singh,
            
              Faiz Khan,
            
              Lakshmi Bala,
            
              Julio Vera,
            
              Olaf Wolkenhauer,
            
              B. Pützer,
            
              Stella Logotheti,
            
              Shailendra Gupta,
            
          </td>
          <td>2023-11-22</td>
          <td>BMC Chemistry</td>
          <td>1</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9c97d7ec2f117ff2be64946f4410f4cfc78a9720" target='_blank'>Computational discovery of dynamic cell line specific Boolean networks from multiplex time-course data</a></td>
          <td>
            
              Misbah Razzaq,
            
              Loïc Paulevé,
            
              A. Siegel,
            
              J. Saez-Rodriguez,
            
              Jérémie Bourdon,
            
              Carito Guziolowski,
            
          </td>
          <td>2018-10-01</td>
          <td>PLoS Computational Biology</td>
          <td>22</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3cca65618f8cf2bf9c4135eb9bef0d9f8b6278db" target='_blank'>De novo Prediction of Cell-Drug Sensitivities Using Deep Learning-based Graph Regularized Matrix Factorization</a></td>
          <td>
            
              Shuangxia Ren,
            
              Yifeng Tao,
            
              Ke Yu,
            
              Yifan Xue,
            
              R. Schwartz,
            
              Xinghua Lu,
            
          </td>
          <td>2021-09-30</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>4</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/437cdce6f393d0b2c68769382f9accba3b03668e" target='_blank'>Combinatorial Optimization for Predicting Optimal Cell-state Conversion Paths</a></td>
          <td>
            
              Shaurya Agrawal,
            
              M. Kellis,
            
              Martin Wohlwend,
            
          </td>
          <td>2023-10-28</td>
          <td>2023 16th International Congress on Image and Signal Processing, BioMedical Engineering and Informatics (CISP-BMEI)</td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e7b9460f53b1393fa834eeb46b9c94177968a96a" target='_blank'>Abstract 1568: Predicting stochastic proliferation and death in response to drugs with mechanistic models tailored to genomic, transcriptomic, and proteomic data</a></td>
          <td>
            
              M. Bouhaddou,
            
              A. Barrette,
            
              R. J. Koch,
            
              M. Birtwistle,
            
          </td>
          <td>2017-07-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/918f7f7a0a07ef22413c8d3e174a52a8cf64bfd0" target='_blank'>TumFlow: An AI Model for Predicting New Anticancer Molecules</a></td>
          <td>
            
              Davide Rigoni,
            
              Sachithra Yaddehige,
            
              Nicoletta Bianchi,
            
              A. Sperduti,
            
              Stefano Moro,
            
              Cristian Taccioli,
            
          </td>
          <td>2024-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/488815adde83ad267b507082359af3dfc5928d8c" target='_blank'>CausalBench: A Large-scale Benchmark for Network Inference from Single-cell Perturbation Data</a></td>
          <td>
            
              Mathieu Chevalley,
            
              Yusuf H. Roohani,
            
              Arash Mehrjou,
            
              J. Leskovec,
            
              P. Schwab,
            
          </td>
          <td>2022-10-31</td>
          <td>ArXiv</td>
          <td>13</td>
          <td>132</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/739a16a818bb82059328645b53e7ba825c3f3812" target='_blank'>Perturbation-based gene regulatory network inference to unravel oncogenic mechanisms</a></td>
          <td>
            
              D. Morgan,
            
              Matthew Studham,
            
              Andreas Tjärnberg,
            
              Holger Weishaupt,
            
              F. Swartling,
            
              Torbjörn E. M. Nordling,
            
              E. Sonnhammer,
            
          </td>
          <td>2019-08-15</td>
          <td>Scientific Reports</td>
          <td>5</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b1736febf869c5e02668b650231ddf148ed92a49" target='_blank'>Molecular mechanism matters: Benefits of mechanistic computational models for drug development.</a></td>
          <td>
            
              Lindsay E. Clegg,
            
              F. Mac Gabhann,
            
          </td>
          <td>2015-09-01</td>
          <td>Pharmacological research</td>
          <td>28</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/67a2a552ea17043d93abe625434e942f88995cbb" target='_blank'>Personalized signaling models for personalized treatments</a></td>
          <td>
            
              J. Saez-Rodriguez,
            
              N. Blüthgen,
            
          </td>
          <td>2020-01-01</td>
          <td>Molecular Systems Biology</td>
          <td>29</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e75a189762eb72580797b7115fc0982618113005" target='_blank'>Data-driven learning how oncogenic gene expression locally alters heterocellular networks</a></td>
          <td>
            
              D. Klinke,
            
              Audry Fernández,
            
              Wentao Deng,
            
              A. Razazan,
            
              Habibolla Latifizadeh,
            
              Anika C Pirkey,
            
          </td>
          <td>2020-05-05</td>
          <td>Nature Communications</td>
          <td>3</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/db9af2166c40be0766ea4320affe6391de190b86" target='_blank'>A Community Challenge for Pancancer Drug Mechanism of Action Inference from Perturbational Profile Data</a></td>
          <td>
            
              Eugene Douglass,
            
              Robert J. Allaway,
            
              B. Szalai,
            
              Wenyu Wang,
            
              Tingzhong Tian,
            
              Adrià Fernández-Torras,
            
              Ronald B. Realubit,
            
              Charles Karan,
            
              Shuyu Zheng,
            
              A. Pessia,
            
              Ziaurrehman Tanoli,
            
              Mohieddin Jafari,
            
              Fangping Wan,
            
              Shuya Li,
            
              Yuanpeng Xiong,
            
              Miquel Duran-Frigola,
            
              Martino Bertoni,
            
              Pau Badia-i-Mompel,
            
              Lidia Mateo,
            
              Oriol Guitart-Pla,
            
              Verena Chung,
            
              Jing Tang,
            
              Jianyang Zeng,
            
              P. Aloy,
            
              J. Saez-Rodriguez,
            
              J. Guinney,
            
              D. Gerhard,
            
              A. Califano,
            
          </td>
          <td>2020-12-22</td>
          <td>bioRxiv</td>
          <td>35</td>
          <td>81</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/760b038e5750152cdae63e55a88943bd1c787b5f" target='_blank'>Logic-Based Modeling and Virtual Drug Screening for the Prediction of Novel Therapeutic Targets and Combination Regimens Against E2F1-driven Melanoma Progression</a></td>
          <td>
            
              Nivedita Singh,
            
              Faiz M. Khan,
            
              L. Bala,
            
              J. Vera,
            
              O. Wolkenhauer,
            
              B. Pützer,
            
              Stella Logotheti,
            
              S. Gupta,
            
          </td>
          <td>2021-08-16</td>
          <td></td>
          <td>0</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c72d7f0c3bd80f9e2370e68f263e06e376f29967" target='_blank'>Combining Mechanistic Models and Machine Learning for Personalized Chemotherapy and Surgery Sequencing in Breast Cancer</a></td>
          <td>
            
              Cristian Axenie,
            
              Daria Kurz,
            
          </td>
          <td>2020-06-09</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c72d7f0c3bd80f9e2370e68f263e06e376f29967" target='_blank'>Combining Mechanistic Models and Machine Learning for Personalized Chemotherapy and Surgery Sequencing in Breast Cancer</a></td>
          <td>
            
              Cristian Axenie,
            
              Daria Kurz,
            
          </td>
          <td>2020-06-09</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9d91f8da0e451908c7494241a6b3d27454d3ee67" target='_blank'>D-SPIN constructs gene regulatory network models from multiplexed scRNA-seq data revealing organizing principles of cellular perturbation response</a></td>
          <td>
            
              Jialong Jiang,
            
              Sisi Chen,
            
              Tiffany Tsou,
            
              Christopher S. McGinnis,
            
              Tahmineh Khazaei,
            
              Qin Zhu,
            
              J. H. Park,
            
              Inna-Marie Strazhnik,
            
              John Hanna,
            
              E. Chow,
            
              David A. Sivak,
            
              Zev J. Gartner,
            
              M. Thomson,
            
          </td>
          <td>2023-04-26</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1e8a450388993e75a7a881c6d461fe24b665f6d5" target='_blank'>Diverse Cell Stimulation Kinetics Identify Predictive Signal Transduction Models</a></td>
          <td>
            
              Hossein Jashnsaz,
            
              Z. Fox,
            
              Jason J. Hughes,
            
              Guoliang Li,
            
              B. Munsky,
            
              G. Neuert,
            
          </td>
          <td>2020-01-29</td>
          <td>iScience</td>
          <td>6</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7a326e6a30e331954c6e3a9e3a6038fb400ce5b4" target='_blank'>Identifying commonalities between cell lines and tumors at the single cell level using Sobolev Alignment of deep generative models</a></td>
          <td>
            
              S. Mourragui,
            
              Joseph C. Siefert,
            
              M. Reinders,
            
              M. Loog,
            
              L. Wessels,
            
          </td>
          <td>2022-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cbe976be11c8973fb9bcd5921423f2e10c5afe43" target='_blank'>Prediction of Whole-Cell Transcriptional Response with Machine Learning</a></td>
          <td>
            
              Mohammed Eslami,
            
              Amin Espah Borujeni,
            
              Hamid Doosthosseini,
            
              M. Vaughn,
            
              Hamed Eramian,
            
              Katie J. Clowers,
            
              D. B. Gordon,
            
              N. Gaffney,
            
              Mark Weston,
            
              Diveena Becker,
            
              Yuval Dorfan,
            
              John M. Fonner,
            
              Joshua Urrutia,
            
              Carolyn Corbet,
            
              G. Zheng,
            
              Joe Stubbs,
            
              Alex Cristofaro,
            
              Paul Maschhoff,
            
              Jed Singer,
            
              Christopher A. Voigt,
            
              Enoch Yeung,
            
          </td>
          <td>2021-04-30</td>
          <td>bioRxiv</td>
          <td>9</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/499ad7d88c7bee176b658d70e6cf7b947863eda4" target='_blank'>Machine learning for perturbational single-cell omics.</a></td>
          <td>
            
              Yuge Ji,
            
              M. Lotfollahi,
            
              F. A. Wolf,
            
              Fabian J Theis,
            
          </td>
          <td>2021-06-16</td>
          <td>Cell systems</td>
          <td>58</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/901ecbc48c1208da81d971b259a10214099763b6" target='_blank'>Fine tuning a logical model of cancer cells to predict drug synergies: combining manual curation and automated parameterization</a></td>
          <td>
            
              Åsmund Flobak,
            
              John Zobolas,
            
              Miguel Vazquez,
            
              T. S. Steigedal,
            
              L. Thommesen,
            
              Asle Grislingås,
            
              B. Niederdorfer,
            
              Evelina Folkesson,
            
              Martin Kuiper,
            
          </td>
          <td>2023-11-20</td>
          <td>Frontiers in Systems Biology</td>
          <td>1</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ebced1f95674e41423a3ad8b11091ad5bb874d7" target='_blank'>A scalable, open-source implementation of a large-scale mechanistic model for single cell proliferation and death signaling</a></td>
          <td>
            
              Cemal Erdem,
            
              Arnab Mutsuddy,
            
              Ethan M. Bensman,
            
              William B. Dodd,
            
              Michael M. Saint-Antoine,
            
              M. Bouhaddou,
            
              R. Blake,
            
              Sean M. Gross,
            
              Laura M. Heiser,
            
              F. Feltus,
            
              M. Birtwistle,
            
          </td>
          <td>2022-06-21</td>
          <td>Nature Communications</td>
          <td>9</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/be687744152afab7c2ddb01709a001ccb60917bb" target='_blank'>Causal identification of single-cell experimental perturbation effects with CINEMA-OT</a></td>
          <td>
            
              Mingze Dong,
            
              Bao Wang,
            
              Jessica Wei,
            
              Antonio H. de O. Fonseca,
            
              Curt Perry,
            
              Alexander Frey,
            
              Feriel Ouerghi,
            
              E. Foxman,
            
              J. Ishizuka,
            
              R. M. Dhodapkar,
            
              David van Dijk,
            
          </td>
          <td>2023-04-03</td>
          <td>Nature Methods</td>
          <td>8</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/36f0071697a0bf4d8ef408e9428ec7b87f3f155a" target='_blank'>Deriving executable models of biochemical network dynamics from qualitative data</a></td>
          <td>
            
              L. Nakhleh,
            
              Derek A. Ruths,
            
          </td>
          <td>None</td>
          <td></td>
          <td>2</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d80e004f3217d435786b3d2091a2b00e9d7a5b59" target='_blank'>Inference of tumor inhibition pathways from drug perturbation data</a></td>
          <td>
            
              Saad Haider,
            
              R. Pal,
            
          </td>
          <td>2013-12-01</td>
          <td>2013 IEEE Global Conference on Signal and Information Processing</td>
          <td>5</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf606a3f90e22221c43516e8c4dee7391d95b64e" target='_blank'>Capturing cell heterogeneity in representations of cell populations for image-based profiling using contrastive learning</a></td>
          <td>
            
              Robert van Dijk,
            
              John Arevalo,
            
              M. Babadi,
            
              A. Carpenter,
            
              Shantanu Singh,
            
          </td>
          <td>2023-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/675d66951c0e0f08ceac004dd79db0e81f8dc389" target='_blank'>Stabilized Reconstruction of Signaling Networks from Single-Cell Cue-Response Data</a></td>
          <td>
            
              Sunil Kumar,
            
              Xiao-Kang Lun,
            
              B. Bodenmiller,
            
              M. Rodríguez Martínez,
            
              H. Koeppl,
            
          </td>
          <td>2020-01-27</td>
          <td>Scientific Reports</td>
          <td>3</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b2551be88e4d5d23e724cdcba34eb11c0e1887d1" target='_blank'>Inferring Intracellular Signal Transduction Circuitry from Molecular Perturbation Experiments</a></td>
          <td>
            
              Michelle L. Wynn,
            
              M. Egbert,
            
              N. Consul,
            
              Jungsoo Chang,
            
              Zhiqian Wu,
            
              Sofia D. Meravjer,
            
              S. Schnell,
            
          </td>
          <td>2017-02-10</td>
          <td>Bulletin of Mathematical Biology</td>
          <td>3</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b2551be88e4d5d23e724cdcba34eb11c0e1887d1" target='_blank'>Inferring Intracellular Signal Transduction Circuitry from Molecular Perturbation Experiments</a></td>
          <td>
            
              Michelle L. Wynn,
            
              M. Egbert,
            
              N. Consul,
            
              Jungsoo Chang,
            
              Zhiqian Wu,
            
              Sofia D. Meravjer,
            
              S. Schnell,
            
          </td>
          <td>2017-02-10</td>
          <td>Bulletin of Mathematical Biology</td>
          <td>3</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3a2f7daacad5c864b44c1e765ac131c30ae4318c" target='_blank'>Inferring tumor-specific cancer dependencies through integrating ex vivo drug response assays and drug-protein profiling</a></td>
          <td>
            
              Alina Batzilla,
            
              Junyan Lu,
            
              Jarno Kivioja,
            
              Kerstin Putzker,
            
              Joe D. Lewis,
            
              T. Zenz,
            
              W. Huber,
            
          </td>
          <td>2022-01-12</td>
          <td>PLoS Computational Biology</td>
          <td>2</td>
          <td>82</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7c7c22f81b27dfc622adbe84aee66b666a8f8154" target='_blank'>CellFateScout – a bioinformatics tool for elucidating small molecule signaling pathways that drive cells in a specific direction</a></td>
          <td>
            
              Marcin Siatkowski,
            
              V. Liebscher,
            
              G. Fuellen,
            
          </td>
          <td>2013-11-08</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>3</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6c26c1c68782bb2b4821f9f2343431d3785b3b59" target='_blank'>Learning representations for image-based profiling of perturbations</a></td>
          <td>
            
              Nikita Moshkov,
            
              Michael Bornholdt,
            
              Santiago Benoit,
            
              Matthew Smith,
            
              C. McQuin,
            
              A. Goodman,
            
              Rebecca A. Senft,
            
              Y. Han,
            
              M. Babadi,
            
              P. Horváth,
            
              B. Cimini,
            
              Anne E Carpenter,
            
              Shantanu Singh,
            
              Juan C. Caicedo,
            
          </td>
          <td>2022-08-15</td>
          <td>Nature Communications</td>
          <td>43</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cf6574c6ab1b8f89c7d9ffc8f025aaf2517b1f79" target='_blank'>Identifying Biological Network Structure, Predicting Network Behavior, and Classifying Network State With High Dimensional Model Representation (HDMR)</a></td>
          <td>
            
              Miles A. Miller,
            
              Xiao-Jiang Feng,
            
              Genyuan Li,
            
              H. Rabitz,
            
          </td>
          <td>2012-06-18</td>
          <td>PLoS ONE</td>
          <td>24</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca90d5ff200d1a8e3ecb19f3b2e70cb9be7545c5" target='_blank'>Identification of Significant Gene Expression Changes in Multiple Perturbation Experiments using Knockoffs</a></td>
          <td>
            
              Ting Zhao,
            
              Guangyu Zhu,
            
              Patrick Flaherty,
            
          </td>
          <td>2021-10-19</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4e43c617b070d8f8962d3421414badabdcb73e0d" target='_blank'>Inference of drug off-target effects on cellular signaling using interactome-based deep learning</a></td>
          <td>
            
              N. Meimetis,
            
              D. Lauffenburger,
            
              Avlant Nilsson,
            
          </td>
          <td>2023-10-15</td>
          <td>iScience</td>
          <td>0</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8beba6845c290bd2e449a0f30380648cf0db685f" target='_blank'>Inferring Functional Pathways from Multi-Perturbation Data</a></td>
          <td>
            
              N. Yosef,
            
              Alon Kaufman,
            
              E. Ruppin,
            
          </td>
          <td>2006-07-10</td>
          <td>Bioinformatics</td>
          <td>10</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dc50ce8821edbdb0670dda99ba9945b02c055939" target='_blank'>A Simple and Efficient Pipeline for Construction, Merging, Expansion, and Simulation of Large-Scale, Single-Cell Mechanistic Models</a></td>
          <td>
            
              Cemal Erdem,
            
              Ethan M. Bensman,
            
              Arnab Mutsuddy,
            
              Michael M. Saint-Antoine,
            
              M. Bouhaddou,
            
              R. Blake,
            
              William B. Dodd,
            
              Sean M. Gross,
            
              Laura M. Heiser,
            
              F. Feltus,
            
              M. Birtwistle,
            
          </td>
          <td>2020-11-10</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0038cc391a39ac6f8a7c39b6e5411aef245c08aa" target='_blank'>Systematic reconstruction of molecular pathway signatures using scalable single-cell perturbation screens</a></td>
          <td>
            
              Longda Jiang,
            
              Carol Dalgarno,
            
              Efthymia Papalexi,
            
              Isabella Mascio,
            
              H. Wessels,
            
              Huiyoung Yun,
            
              Nika Iremadze,
            
              G. Lithwick-Yanai,
            
              D. Lipson,
            
              R. Satija,
            
          </td>
          <td>2024-01-30</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1d55eefe94d5ca2fbd2f98a7657720af07a325e4" target='_blank'>Extraction of biological signals by factorization enables the reliable analysis of single-cell transcriptomics</a></td>
          <td>
            
              Feng Zeng,
            
              X. Kong,
            
              Fan Yang,
            
              Ting Chen,
            
              Jiahuai Han,
            
          </td>
          <td>2023-03-04</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fed989fd87c7141bf188cf54581345f04a17c5b1" target='_blank'>Predictive Modeling of Drug Treatment in the Area of Personalized Medicine</a></td>
          <td>
            
              L. Ogilvie,
            
              C. Wierling,
            
              T. Kessler,
            
              H. Lehrach,
            
              B. Lange,
            
          </td>
          <td>2015-12-06</td>
          <td>Cancer Informatics</td>
          <td>21</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/29fcc83b0a23e513c3d2feb42fe69518c5979063" target='_blank'>Fatecode: Cell fate regulator prediction using classification autoencoder perturbation</a></td>
          <td>
            
              Mehrshad Sadria,
            
              A. Layton,
            
              Sidharta Goyal,
            
              Gary D. Bader,
            
          </td>
          <td>2022-12-19</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bff33206f1782c889c393aecb87bbd7939af61f0" target='_blank'>High-Dimensional Gene Expression and Morphology Profiles of Cells across 28,000 Genetic and Chemical Perturbations</a></td>
          <td>
            
              Marzieh Haghighi,
            
              Shantanu Singh,
            
              J. Caicedo,
            
              Anne E Carpenter,
            
          </td>
          <td>2021-09-08</td>
          <td>bioRxiv</td>
          <td>48</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c4b38500c9cc0a09c3265e34d8ca652017d201df" target='_blank'>Relational Dose-Response Modeling for Cancer Drug Studies.</a></td>
          <td>
            
              Wesley Tansey,
            
              Christopher Tosh,
            
              D. Blei,
            
          </td>
          <td>2019-06-10</td>
          <td>arXiv: Machine Learning</td>
          <td>2</td>
          <td>96</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/10ad33cfd2737f28f296070cb3cb2fc7e4156e90" target='_blank'>PHOCOS: inferring multi-feature phenotypic crosstalk networks</a></td>
          <td>
            
              Yue Deng,
            
              S. Altschuler,
            
              Lani F. Wu,
            
          </td>
          <td>2016-06-11</td>
          <td>Bioinformatics</td>
          <td>1</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6f4b1caaec3342214fc92c5e6161daa75ffd9ea8" target='_blank'>Modelling Cellular Perturbations with the Sparse Additive Mechanism Shift Variational Autoencoder</a></td>
          <td>
            
              Michael D. Bereket,
            
              Theofanis Karaletsos,
            
          </td>
          <td>2023-11-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d9376a5aba08c5400b327c29af9c6107626e055f" target='_blank'>Predicting cell morphological responses to perturbations using generative modeling</a></td>
          <td>
            
              A. Palma,
            
              F. Theis,
            
              M. Lotfollahi,
            
          </td>
          <td>2023-07-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3a9a0b2e48180081316331ab76704fc68f0f04fb" target='_blank'>ChemPert: mapping between chemical perturbation and transcriptional response for non-cancer cells</a></td>
          <td>
            
              Menglin Zheng,
            
              S. Okawa,
            
              M. Bravo,
            
              Fei Chen,
            
              M. Martínez‐Chantar,
            
              A. del Sol,
            
          </td>
          <td>2022-05-01</td>
          <td>Nucleic Acids Research</td>
          <td>6</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/199da934ab811e693733e7fd43331f520646f774" target='_blank'>NeuMF: Predicting anti-cancer drug response through a neural matrix factorization model</a></td>
          <td>
            
              Hui Liu,
            
              Jian Yu,
            
              Xiangzhi Chen,
            
              Lin Zhang,
            
          </td>
          <td>2022-06-09</td>
          <td>Current Bioinformatics</td>
          <td>3</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/af786ba590cb9db62ff31318c785685ade68bfd9" target='_blank'>Genome-scale mechanistic modeling of signaling pathways made easy: A bioconductor/cytoscape/web server framework for the analysis of omic data</a></td>
          <td>
            
              Kinza Rian,
            
              Marta R. Hidalgo,
            
              C. Çubuk,
            
              M. M. Falco,
            
              C. Loucera,
            
              M. Esteban-Medina,
            
              Inmaculada Álamo-Álvarez,
            
              M. Peña-Chilet,
            
              J. Dopazo,
            
          </td>
          <td>2021-05-15</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>8</td>
          <td>77</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/810f39fee2ea4ba27ab948178ab11876565d03fe" target='_blank'>Sparse Regression Based Structure Learning of Stochastic Reaction Networks from Single Cell Snapshot Time Series</a></td>
          <td>
            
              A. Klimovskaia,
            
              Stefan Ganscha,
            
              M. Claassen,
            
          </td>
          <td>2016-12-01</td>
          <td>PLoS Computational Biology</td>
          <td>19</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/20d0e30a85cd97e0d2c4e938a40b388b3e088fa2" target='_blank'>Modeling transcriptional profiles of gene perturbation with deep neural network</a></td>
          <td>
            
              Wenke Liu,
            
              Xuya Wang,
            
              D. Mani,
            
              D. Fenyö,
            
          </td>
          <td>2021-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2e023e852082a713445ec3354b84096a4e2f50ea" target='_blank'>Modeling drug combination effects via latent tensor reconstruction</a></td>
          <td>
            
              Tianduanyi Wang,
            
              S. Szedmák,
            
              Haishan Wang,
            
              T. Aittokallio,
            
              T. Pahikkala,
            
              A. Cichońska,
            
              Juho Rousu,
            
          </td>
          <td>2021-04-19</td>
          <td>Bioinformatics</td>
          <td>12</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4dfb2fb4a9f004b39a375d0f2d55f5e4ab225d10" target='_blank'>The DIONESUS algorithm provides scalable and accurate reconstruction of dynamic phosphoproteomic networks to reveal new drug targets.</a></td>
          <td>
            
              Mark F. Ciaccio,
            
              Vincent C. Chen,
            
              R. Jones,
            
              N. Bagheri,
            
          </td>
          <td>2015-07-06</td>
          <td>Integrative biology : quantitative biosciences from nano to macro</td>
          <td>13</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2582151d5accc2521974c0f4afcbfe50b06338f0" target='_blank'>Deep Hidden Physics Modeling of Cell Signaling Networks</a></td>
          <td>
            
              M. Seeger,
            
              J. Longden,
            
              E. Klipp,
            
              R. Linding,
            
          </td>
          <td>2021-06-14</td>
          <td>Current Genomics</td>
          <td>1</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/73856a7b883350f38c5c824eb1a8a2e8d0bec432" target='_blank'>CellDrift: inferring perturbation responses in temporally sampled single-cell data</a></td>
          <td>
            
              Kang Jin,
            
              Daniel J. Schnell,
            
              Guangyuan Li,
            
              N. Salomonis,
            
              V. B. Surya Prasath,
            
              R. Szczesniak,
            
              B. Aronow,
            
          </td>
          <td>2022-04-14</td>
          <td>Briefings in Bioinformatics</td>
          <td>5</td>
          <td>91</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2d6dbb19e42b7a66ce2daaa8d8f0f50ee5999758" target='_blank'>How Artificial Intelligence Unravels the Complex Web of Cancer Drug Response.</a></td>
          <td>
            
              Olivier Elemento,
            
          </td>
          <td>2024-04-08</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/184d07b3a1b80feebef29a192bd27f319319180b" target='_blank'>Biologically-informed neural networks guide mechanistic modeling from sparse experimental data</a></td>
          <td>
            
              John H. Lagergren,
            
              John T. Nardini,
            
              R. Baker,
            
              M. Simpson,
            
              K. Flores,
            
          </td>
          <td>2020-05-26</td>
          <td>PLoS Computational Biology</td>
          <td>71</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0238db2a09463e1111b8a7452d56292ea4679196" target='_blank'>Reverse engineering biological networks: computational approaches for modeling biological systems from perturbation data</a></td>
          <td>
            
              Yungil Kim,
            
          </td>
          <td>2013-09-01</td>
          <td></td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fe5339d9a9ea4e1b61c19559f4c3ea528d718664" target='_blank'>Predicting Cellular Drug Sensitivity using Conditional Modulation of Gene Expression</a></td>
          <td>
            
              Will Connell,
            
              Michael J. Keiser,
            
          </td>
          <td>2021-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f0e44e1e5909c43c25cd90a4513d9270601ee2c6" target='_blank'>Diffusion mapping of drug targets on disease signaling network elements reveals drug combination strategies</a></td>
          <td>
            
              Jielin Xu,
            
              Kelly Regan-Fendt,
            
              Siyuan Deng,
            
              W. Carson,
            
              Philip R. O. Payne,
            
              Fuhai Li,
            
          </td>
          <td>None</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>11</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/888ce556d85d3908b3febbdc50a1911a0f8028ef" target='_blank'>Cell-Vision Fusion: A Swin Transformer-based Approach to Predicting Kinase Inhibitor Mechanism of Action from Cell Painting Data</a></td>
          <td>
            
              William Dee,
            
              Ines Sequeira,
            
              Anna Lobley,
            
              Gregory Slabaugh,
            
          </td>
          <td>2023-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/97625813d38c836a226ea23952f4a4600effd99c" target='_blank'>Logical modeling: Combining manual curation and automated parameterization to predict drug synergies</a></td>
          <td>
            
              Åsmund Flobak,
            
              John Zobolas,
            
              M. Vázquez,
            
              T. S. Steigedal,
            
              L. Thommesen,
            
              Asle Grislingås,
            
              B. Niederdorfer,
            
              Evelina Folkesson,
            
              Martin Kuiper,
            
          </td>
          <td>2021-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f489770b414403bd0cf8aa558b15c2836308c23f" target='_blank'>Sorad: a systems biology approach to predict and modulate dynamic signaling pathway response from phosphoproteome time-course measurements</a></td>
          <td>
            
              Tarmo Äijö,
            
              Kirsi J. Granberg,
            
              H. Lähdesmäki,
            
          </td>
          <td>2013-05-15</td>
          <td>Bioinformatics</td>
          <td>12</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c8f0f88181ef0ee35beff85e1e98d1d4217b0f91" target='_blank'>Characterizing Signal Transduction Networks and Biological Responses Using Computer Simulations and Machine Learning</a></td>
          <td>
            
              A. Prescott,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6d80064d79b9dc89d73a9d1eed8dd4b2c991a94d" target='_blank'>A Perturbation Approach for Refining Boolean Models of Cell Cycle Regulation</a></td>
          <td>
            
              Anand Banerjee,
            
              Asif Iqbal Rahaman,
            
              Alok Mehandale,
            
              P. Kraikivski,
            
          </td>
          <td>2023-11-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a8fe764ee996495b47e9ad27e1376a171892626e" target='_blank'>Causal network inference using biochemical kinetics</a></td>
          <td>
            
              Chris J. Oates,
            
              F. Dondelinger,
            
              N. Bayani,
            
              J. Korkola,
            
              J. Gray,
            
              S. Mukherjee,
            
          </td>
          <td>2014-05-31</td>
          <td>Bioinformatics</td>
          <td>53</td>
          <td>104</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0503969a52a41ce25bc7d79bb7dfa6a958779cb0" target='_blank'>Drug response prediction by inferring pathway-response associations with kernelized Bayesian matrix factorization</a></td>
          <td>
            
              Muhammad Ammad-ud-din,
            
              Suleiman A. Khan,
            
              D. Malani,
            
              A. Murumägi,
            
              O. Kallioniemi,
            
              T. Aittokallio,
            
              Samuel Kaski,
            
          </td>
          <td>2016-06-11</td>
          <td>Bioinformatics</td>
          <td>82</td>
          <td>107</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7ad7516eff9fc696646a72f93252f46f481996f6" target='_blank'>Comparative Network Reconstruction using mixed integer programming</a></td>
          <td>
            
              Evert Bosdriesz,
            
              A. Prahallad,
            
              B. Klinger,
            
              A. Sieber,
            
              A. Bosma,
            
              R. Bernards,
            
              N. Blüthgen,
            
              L. Wessels,
            
          </td>
          <td>2018-01-05</td>
          <td>Bioinformatics</td>
          <td>8</td>
          <td>85</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e2a92780976449d08dbc2406577d5c4baa9e540d" target='_blank'>Abstract 2447: Accurate prediction of the signaling pathway activity elicited by variants of uncertain significance (VUS) and their drug sensitivity using deep convolutional neural networks on florescent microscopy images of cells</a></td>
          <td>
            
              Lior Zimmerman,
            
              Arie Aizenman,
            
              Y. Daitsh,
            
              Z. Barbash,
            
              G. Tarcic,
            
              M. Vidne,
            
          </td>
          <td>2019-07-01</td>
          <td>Bioinformatics, Convergence Science, and Systems Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dc80f608750f8ca33b1d0928a49559b562c964b6" target='_blank'>Nonlinear mixed effects dose response modeling in high throughput drug screens: application to melanoma cell line analysis</a></td>
          <td>
            
              Kuan-Fu Ding,
            
              E. Petricoin,
            
              D. Finlay,
            
              H. Yin,
            
              William P. D. Hendricks,
            
              C. Sereduk,
            
              J. Kiefer,
            
              A. Sekulic,
            
              P. LoRusso,
            
              K. Vuori,
            
              J. Trent,
            
              N. Schork,
            
          </td>
          <td>2017-12-15</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>125</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/79a94a8237b7c28336c2f183f4a4fb3339e77641" target='_blank'>Static network structure can be used to model the phenotypic effects of perturbations in regulatory networks</a></td>
          <td>
            
              Ariel Feiglin,
            
              Adar Hacohen,
            
              Avital Sarusi-Portuguez,
            
              J. Fisher,
            
              R. Unger,
            
              Yanay Ofran,
            
          </td>
          <td>2012-11-01</td>
          <td>Bioinformatics</td>
          <td>21</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/377f7126d7b8b2631ac0ebbe3b2d1f87e8179879" target='_blank'>Network dynamics-based cancer panel stratification for systemic prediction of anticancer drug response</a></td>
          <td>
            
              Minsoo Choi,
            
              Jue Shi,
            
              Yan-ting Zhu,
            
              Ruizhen Yang,
            
              Kwang-Hyun Cho,
            
          </td>
          <td>2017-12-01</td>
          <td>Nature Communications</td>
          <td>38</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0336864918bdaa591b391d6aa3a29830977a06a5" target='_blank'>Inferring reaction network structure from single-cell, multiplex data, using toric systems theory</a></td>
          <td>
            
              Shu Wang,
            
              Jia-Ren Lin,
            
              Eduardo Sontag,
            
              P. Sorger,
            
          </td>
          <td>2019-08-09</td>
          <td>PLoS Computational Biology</td>
          <td>14</td>
          <td>104</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b4ff565cf112d3f9df1bc7f9ca01bb48a8c8f399" target='_blank'>Generating counterfactual explanations of tumor spatial proteomes to discover therapeutic strategies for enhancing immune infiltration</a></td>
          <td>
            
              Z. Wang,
            
              Alexander M. Xu,
            
              Aman Bhargava,
            
              Matt W. Thomson,
            
          </td>
          <td>2023-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ab7876236250c23376a91855071a7847b11714ef" target='_blank'>Causal machine learning for single-cell genomics</a></td>
          <td>
            
              Alejandro Tejada-Lapuerta,
            
              Paul Bertin,
            
              Stefan Bauer,
            
              H. Aliee,
            
              Y. Bengio,
            
              Fabian J. Theis,
            
          </td>
          <td>2023-10-23</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/514b3b15114767c3969c9b5144d73f4676fa5314" target='_blank'>Interpretable Drug Response Prediction using a Knowledge-based Neural Network</a></td>
          <td>
            
              Oliver Snow,
            
              Hossein Sharifi-Noghabi,
            
              Jialin Lu,
            
              O. Zolotareva,
            
              Mark Lee,
            
              M. Ester,
            
          </td>
          <td>2021-08-14</td>
          <td>Proceedings of the 27th ACM SIGKDD Conference on Knowledge Discovery & Data Mining</td>
          <td>5</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/385ecf17e4b8589c0b54575cc27c3b02323fabbb" target='_blank'>Biologically Interpretable VAE with Supervision for Transcriptomics Data Under Ordinal Perturbations</a></td>
          <td>
            
              Seyednami Niyakan,
            
              Byung-Jun Yoon,
            
              Xiaoning Qian,
            
              Xihaier Luo,
            
          </td>
          <td>2024-03-29</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/00955d57b98b9aef7bfc95031b22dbf1e804cf16" target='_blank'>BioNetGMMFit: a Parameter Estimation Tool for BioNetGen using Single-Cell Snapshot Data from Cell Populations Evolving over Time</a></td>
          <td>
            
              John Wu,
            
              William Stewart,
            
              C. Jayaprakash,
            
              J. Das,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a752b4aaddbbd93b174c693ca797329dfa915b90" target='_blank'>Network inference from perturbation time course data</a></td>
          <td>
            
              Deepraj Sarmah,
            
              Gregory R. Smith,
            
              M. Bouhaddou,
            
              Alan D. Stern,
            
              James Erskine,
            
              M. Birtwistle,
            
          </td>
          <td>2018-06-07</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>7</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/194dc65d583a9464023f82846ef3c6501bfdaaa0" target='_blank'>lpNet: a linear programming approach to reconstruct signal transduction networks</a></td>
          <td>
            
              Marta R. A. Matos,
            
              Bettina Knapp,
            
              L. Kaderali,
            
          </td>
          <td>2015-10-01</td>
          <td>Bioinformatics</td>
          <td>3</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ef0933787aee0990325e4b4d80906ec4c1713785" target='_blank'>Extensions of ℓ1 regularization increase detection specificity for cell-type specific parameters in dynamic models</a></td>
          <td>
            
              Pascal Dolejsch,
            
              Helge Hass,
            
              J. Timmer,
            
          </td>
          <td>2019-07-16</td>
          <td>BMC Bioinformatics</td>
          <td>9</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7fb2bbe2d1bae2a2de8eb8030724557293a80534" target='_blank'>Analysis of cancer signaling networks by systems biology to develop therapies.</a></td>
          <td>
            
              Rexxi D. Prasasya,
            
              D. Tian,
            
              P. Kreeger,
            
          </td>
          <td>2011-06-01</td>
          <td>Seminars in cancer biology</td>
          <td>28</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c5b3349cc79f09ecee3d3da88bea9fb8a0d5ebf1" target='_blank'>Generative Modeling of Single Cell Gene Expression for Dose-Dependent Chemical Perturbations</a></td>
          <td>
            
              Omar Kana,
            
              R. Nault,
            
              D. Filipovic,
            
              Daniel Marri,
            
              T. Zacharewski,
            
              S. Bhattacharya,
            
          </td>
          <td>2022-10-07</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c139f34f7295b22c0664df9451801b504d5e5e84" target='_blank'>Perturbation-Theory Machine Learning (PTML) Multilabel Model of the ChEMBL Dataset of Preclinical Assays for Antisarcoma Compounds</a></td>
          <td>
            
              A. Cabrera-Andrade,
            
              A. López-Cortés,
            
              C. Munteanu,
            
              A. Pazos,
            
              Yunierkis Pérez-Castillo,
            
              E. Tejera,
            
              S. Arrasate,
            
              H. González-Díaz,
            
          </td>
          <td>2020-10-15</td>
          <td>ACS Omega</td>
          <td>5</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/48092d6982a36fb3377367ef9c51e539910b5bee" target='_blank'>Inference of dynamic biological networks based on perturbation data</a></td>
          <td>
            
              R. Pal,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f024c47c6740974b859e71e8892149adf48fd40" target='_blank'>Single-cell-specific drug activities are revealed by a tensor imputation algorithm</a></td>
          <td>
            
              Michio Iwata,
            
              Yoshihiro Yamanishi,
            
          </td>
          <td>2022-11-01</td>
          <td>Nature Computational Science</td>
          <td>0</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/003204844f08ca50f3464b3be1c5d9ed07384c31" target='_blank'>Efficient Bayesian inference for mechanistic modelling with high-throughput data</a></td>
          <td>
            
              Simon F. Martina-Perez,
            
              H. Sailem,
            
              R. Baker,
            
          </td>
          <td>2022-02-15</td>
          <td>PLoS Computational Biology</td>
          <td>3</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b49bc5b101b13b27f16b4faff42eeb4610f9d976" target='_blank'>Design of perturbations is the key to inference of tumour specific gene regulation</a></td>
          <td>
            
              Torbjörn E. M. Nordling,
            
              E. W. Jacobsen,
            
          </td>
          <td>None</td>
          <td></td>
          <td>1</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0eb3e57ce69d1ae3d6814a3d72ea43785e97889b" target='_blank'>Learning to encode cellular responses to systematic perturbations with deep generative models</a></td>
          <td>
            
              Yifan Xue,
            
              Michael Q. Ding,
            
              Xinghua Lu,
            
          </td>
          <td>2020-01-14</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>6</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d1a9962c0d8dad044099e4cc4844653bb366f9c6" target='_blank'>Quantifying differences in cell line population dynamics using CellPD</a></td>
          <td>
            
              Edwin F. Juarez,
            
              R. Lau,
            
              Samuel H. Friedman,
            
              Ahmadreza Ghaffarizadeh,
            
              E. Jonckheere,
            
              D. Agus,
            
              S. Mumenthaler,
            
              P. Macklin,
            
          </td>
          <td>2016-09-21</td>
          <td>BMC Systems Biology</td>
          <td>21</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f26ca7716b8e4c809901dfa4fa65e822ede08b3e" target='_blank'>Disentanglement of single-cell data with biolord.</a></td>
          <td>
            
              Z. Piran,
            
              Niv Cohen,
            
              Yedid Hoshen,
            
              M. Nitzan,
            
          </td>
          <td>2024-01-15</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/46117124daca3ecb291aabe3600c96baee745993" target='_blank'>Abstract 636: Prediction of therapeutic response via data-driven maps of tumor cell architecture</a></td>
          <td>
            
              Erica Silva,
            
              Akshat Singhal,
            
              Sungjoon Park,
            
              J. Kreisberg,
            
              T. Ideker,
            
          </td>
          <td>2022-06-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>80</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2521c3d76bc439c961b7003080f4a7a661949547" target='_blank'>Causal Protein-Signaling Networks Derived from Multiparameter Single-Cell Data</a></td>
          <td>
            
              K. Sachs,
            
              O. Perez,
            
              D. Pe’er,
            
              D. Lauffenburger,
            
              G. Nolan,
            
          </td>
          <td>2005-04-22</td>
          <td>Science</td>
          <td>1768</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f349de5cb33a81e8b751673d8f31b24632e6cf85" target='_blank'>Partial differential equation-based inference of migration and proliferation mechanisms in cancer cell populations</a></td>
          <td>
            
              Patrick C. Kinnunen,
            
              Siddharth Srivastava,
            
              Zhenlin Wang,
            
              Kenneth K. Y. Ho,
            
              Brock A. Humphries,
            
              Siyi Chen,
            
              J. Linderman,
            
              G. Luker,
            
              K. Luker,
            
              K. Garikipati,
            
          </td>
          <td>2023-02-19</td>
          <td>None</td>
          <td>0</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ee43f8dc76a2590e1ac367a7fa51fc7ee49b7e92" target='_blank'>Application of modular response analysis to medium- to large-size biological systems</a></td>
          <td>
            
              Meriem Mekedem,
            
              P. Ravel,
            
              J. Colinge,
            
          </td>
          <td>2021-07-27</td>
          <td>PLoS Computational Biology</td>
          <td>2</td>
          <td>47</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e0b92168fd136f2e5f3d3697ce9f73c94b04cd43" target='_blank'>Comparison of Chemical Structure and Cell Morphology Information for Multitask Bioactivity Predictions</a></td>
          <td>
            
              Maria-Anna Trapotsi,
            
              Lewis H. Mervin,
            
              Avid M. Afzal,
            
              Noé Sturm,
            
              O. Engkvist,
            
              I. Barrett,
            
              A. Bender,
            
          </td>
          <td>2021-03-04</td>
          <td>Journal of chemical information and modeling</td>
          <td>23</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c5731dc2144693fe24fe5b5a783b38bc48b495ce" target='_blank'>Predictive modeling based on functional and genomic characterizations</a></td>
          <td>
            
              R. Pal,
            
          </td>
          <td>None</td>
          <td></td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bf54125d5afea1e14f5d83f6dd89a302faed02a3" target='_blank'>PHENSIM: Phenotype Simulator</a></td>
          <td>
            
              S. Alaimo,
            
              Rosaria Valentina Rapicavoli,
            
              G. P. Marceca,
            
              Alessandro La Ferlita,
            
              O. Serebrennikova,
            
              P. Tsichlis,
            
              B. Mishra,
            
              A. Pulvirenti,
            
              A. Ferro,
            
          </td>
          <td>2020-01-21</td>
          <td>PLoS Computational Biology</td>
          <td>7</td>
          <td>80</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/89976b8b9f7ae3c5b19c9eff05f7dbcf183c5171" target='_blank'>Trends and Potential of Machine Learning and Deep Learning in Drug Study at Single-Cell Level</a></td>
          <td>
            
              Ren Qi,
            
              Quan Zou,
            
          </td>
          <td>2023-01-10</td>
          <td>Research</td>
          <td>13</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/39b85b8aebc0242791c8944fd3435e2529943c50" target='_blank'>A Computational Framework for Prediction and Analysis of Cancer Signaling Dynamics from RNA Sequencing Data—Application to the ErbB Receptor Signaling Pathway</a></td>
          <td>
            
              Hiroaki Imoto,
            
              Suxiang Zhang,
            
              Mariko Okada,
            
          </td>
          <td>2020-10-01</td>
          <td>Cancers</td>
          <td>18</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/87d9a8ecdbe88827648236d1407034238e4b6f3b" target='_blank'>Learning signaling networks from combinatorial perturbations by exploiting siRNA off-target effects</a></td>
          <td>
            
              J. Tiuryn,
            
              E. Szczurek,
            
          </td>
          <td>2019-07-01</td>
          <td>Bioinformatics</td>
          <td>5</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6efc655d7b31eb4734e450e33d5ba2b277ab7f47" target='_blank'>Gene expression based inference of cancer drug sensitivity</a></td>
          <td>
            
              Smriti Chawla,
            
              A. Rockstroh,
            
              M. Lehman,
            
              E. Ratther,
            
              Atishay Jain,
            
              Anuneet Anand,
            
              Apoorva Gupta,
            
              Namrata Bhattacharya,
            
              Sarita Poonia,
            
              Priyadarshini Rai,
            
              Nirjhar Das,
            
              A. Majumdar,
            
              Jayadeva,
            
              Gaurav Ahuja,
            
              B. Hollier,
            
              Colleen C. Nelson,
            
              Debarka Sengupta,
            
          </td>
          <td>2022-09-27</td>
          <td>Nature Communications</td>
          <td>38</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ceb07f72804966ed84b70ffbf44c1e4f5859660" target='_blank'>CellFrame: A Data Structure for Abstraction of Cell Biology Experiments and Construction of Perturbation Networks</a></td>
          <td>
            
              Y. Gong,
            
              Zhaolei Zhang,
            
          </td>
          <td>2007-12-01</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>4</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8ef2787ed860af376ae583b1492007cd37876690" target='_blank'>Network Pharmacology Strategies Toward Multi-Target Anticancer Therapies: From Computational Models to Experimental Design Principles</a></td>
          <td>
            
              Jing Tang,
            
              T. Aittokallio,
            
          </td>
          <td>2014-01-01</td>
          <td>Current Pharmaceutical Design</td>
          <td>120</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58ec472ddfdd86e34f573d35350e65391bd867c9" target='_blank'>Toward subtask decomposition-based learning and benchmarking for genetic perturbation outcome prediction and beyond</a></td>
          <td>
            
              Yicheng Gao,
            
              Zhiting Wei,
            
              Kejing Dong,
            
              Jingya Yang,
            
              Guohui Chuai,
            
              Qi Liu,
            
          </td>
          <td>2024-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3bcc6b6b7ad2a5fd0b693d18fd0a0a6c6bebd007" target='_blank'>Predictive Analytics of Phosphoproteins in Breast Cancer Cells</a></td>
          <td>
            
              S. Opiyo,
            
          </td>
          <td>2016-11-14</td>
          <td></td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fdad88ff3273f539824d1052e74013df304b717e" target='_blank'>Deep Representation Learning Determines Drug Mechanism of Action from Cell Painting Images</a></td>
          <td>
            
              Daniel R. Wong,
            
              David J. Logan,
            
              S. Hariharan,
            
              Robert Stanton,
            
              A. Kiruluta,
            
          </td>
          <td>2022-11-17</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4fd0e8babd8a56764bca724c64ef2f68a3149983" target='_blank'>Construction of cell type-specific logic models of signaling networks using CellNOpt.</a></td>
          <td>
            
              Melody K. Morris,
            
              Ioannis N. Melas,
            
              J. Saez-Rodriguez,
            
          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>6</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9944ac68965dc5c11917e08c5b48887d5bf40eb4" target='_blank'>Combinatorial prediction of therapeutic perturbations using causally-inspired neural networks</a></td>
          <td>
            
              Guadalupe Gonzalez,
            
              Isuru Herath,
            
              Kirill Veselkov,
            
              Michael Bronstein,
            
              M. Zitnik,
            
          </td>
          <td>2024-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6241769c55cbe48d2e27def8eb7f5803add638d1" target='_blank'>Predictive Modeling of Multiplex Chemical Phenomics for Novel Cells and Patients: Applied to Personalized Alzheimer’s Disease Drug Repurposing</a></td>
          <td>
            
              Qiao Liu,
            
              Yue Qiu,
            
              Lei Xie,
            
          </td>
          <td>2021-08-10</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/faece74722f29827ea979c8878669ccabe045ffc" target='_blank'>Generative modeling and latent space arithmetics predict single-cell perturbation response across cell types, studies and species</a></td>
          <td>
            
              M. Lotfollahi,
            
              F. A. Wolf,
            
              Fabian J Theis,
            
          </td>
          <td>2018-11-29</td>
          <td>bioRxiv</td>
          <td>21</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4588ed2b6771693f49090af31eedc1e60a68d633" target='_blank'>Predicting Cellular Responses with Variational Causal Inference and Refined Relational Information</a></td>
          <td>
            
              Yulun Wu,
            
              Robert A. Barton,
            
              Zichen Wang,
            
              V. Ioannidis,
            
              Carlo De Donno,
            
              Layne Price,
            
              L. Voloch,
            
              G. Karypis,
            
          </td>
          <td>2022-09-30</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>95</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/470ddf945ed118f04b78fe56683288202ff7d56f" target='_blank'>Learning time-varying information flow from single-cell epithelial to mesenchymal transition data</a></td>
          <td>
            
              Smita Krishnaswamy,
            
              Nevena Zivanovic,
            
              Roshan Sharma,
            
              D. Pe’er,
            
              B. Bodenmiller,
            
          </td>
          <td>2018-10-29</td>
          <td>PLoS ONE</td>
          <td>17</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94e0c5b57a795f63319fb45e2716e458007292e9" target='_blank'>Integrating pharmacogenomics and cheminformatics with diverse disease phenotypes for cell type-guided drug discovery</a></td>
          <td>
            
              A. Halu,
            
              J. Decano,
            
              J. Matamalas,
            
              Mary Whelan,
            
              Takaharu Asano,
            
              Namitra Kalicharran,
            
              Sasha A. Singh,
            
              J. Loscalzo,
            
              M. Aikawa,
            
          </td>
          <td>2022-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>137</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5233a066fa24cb9ac285768e198c3e42ec0525b5" target='_blank'>TranSynergy: Mechanism-driven interpretable deep neural network for the synergistic prediction and pathway deconvolution of drug combinations</a></td>
          <td>
            
              Qiao Liu,
            
              Lei Xie,
            
          </td>
          <td>2020-07-10</td>
          <td>PLoS Computational Biology</td>
          <td>82</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/826ecb9ee14f06b6b3b341db488a16cd2170890c" target='_blank'>Experimental design schemes for learning Boolean network models</a></td>
          <td>
            
              N. Atias,
            
              M. Gershenzon,
            
              Katia Labazin,
            
              R. Sharan,
            
          </td>
          <td>2014-08-22</td>
          <td>Bioinformatics</td>
          <td>13</td>
          <td>68</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ee320fb74e14f0baa7f492bbf9fa2da3c8106a62" target='_blank'>Interpretable deep learning architectures for improving drug response prediction performance: myth or reality?</a></td>
          <td>
            
              Yihui Li,
            
              D. Hostallero,
            
              A. Emad,
            
          </td>
          <td>2023-02-26</td>
          <td>Bioinformatics</td>
          <td>2</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6d0d7ca8fdbc726fad17ba68be1dc6875b02c3d1" target='_blank'>BioXNet: a biologically inspired neural network for deciphering anti-cancer drug response in precision medicine</a></td>
          <td>
            
              Jiannan Yang,
            
              William Ka Kei Wu,
            
              Rina Yee Man Hui,
            
              Ian Chi Kei Wong,
            
              Qingpeng Zhang,
            
          </td>
          <td>2024-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f4542df1391d52171e6193c5d4b5e313b6d1f8fc" target='_blank'>Data-Driven Identiﬁcation of Cancer Cell Population Dynamics Leveraging the Effect of Pre-Treatment for Drug Schedule Design</a></td>
          <td>
            
              Marius Wiggert,
            
              Z. Cohen,
            
              Megan Turnidge,
            
              Ellen M. Langer,
            
              R. Sears,
            
              Margaret P. Chapman,
            
              C. Tomlin,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>78</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/746f799529ecf910a57d2d3cd80048ddb0fb5b76" target='_blank'>Computational methods for characterizing and learning from heterogeneous cell signaling data.</a></td>
          <td>
            
              Patrick C. Kinnunen,
            
              K. Luker,
            
              G. Luker,
            
              J. Linderman,
            
          </td>
          <td>2021-05-04</td>
          <td>Current opinion in systems biology</td>
          <td>7</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6a11afe8dfaf748cdf52c0b2bcf3fd1d7258fa26" target='_blank'>Integrating literature-constrained and data-driven inference of signalling networks</a></td>
          <td>
            
              Federica Eduati,
            
              J De Las Rivas,
            
              B. Camillo,
            
              G. Toffolo,
            
              J. Sáez-Rodríguez,
            
          </td>
          <td>2012-06-25</td>
          <td>Bioinformatics</td>
          <td>42</td>
          <td>56</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c5b0559263bb32e1dd574c073a66e2cf01dc1f9e" target='_blank'>Alternative Pathway Approach for Automating Analysis and Validation of Cell Perturbation Networks and Design of Perturbation Experiments</a></td>
          <td>
            
              Y. Gong,
            
              Zhaolei Zhang,
            
          </td>
          <td>2007-12-01</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>7</td>
          <td>46</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fafc26bc2c7b508a16a6e3cb0c2608871743c606" target='_blank'>CancerInSilico: An R/Bioconductor package for combining mathematical and statistical modeling to simulate time course bulk and single cell gene expression data in cancer</a></td>
          <td>
            
              Thomas Sherman,
            
              L. Kagohara,
            
              Raymon Cao,
            
              Raymond Cheng,
            
              M. Satriano,
            
              M. Considine,
            
              G. Krigsfeld,
            
              Ruchira S. Ranaweera,
            
              Yong Tang,
            
              S. Jablonski,
            
              G. Stein-O’Brien,
            
              Daria A. Gaykalova,
            
              L. Weiner,
            
              C. Chung,
            
              E. Fertig,
            
          </td>
          <td>2018-06-28</td>
          <td>PLoS Computational Biology</td>
          <td>3</td>
          <td>70</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/593d758a97cdca686117bc33080e1561ea0705f2" target='_blank'>MultiCPA: Multimodal Compositional Perturbation Autoencoder</a></td>
          <td>
            
              Kemal Inecik,
            
              Andreas Uhlmann,
            
              M. Lotfollahi,
            
              F. Theis,
            
          </td>
          <td>2022-07-10</td>
          <td>bioRxiv</td>
          <td>6</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/733a5c37291732e0d4d3c42e02327e428a07439b" target='_blank'>Causal Interactions from Proteomic Profiles: Molecular Data Meets Pathway Knowledge</a></td>
          <td>
            
              Özgün Babur,
            
              Augustin Luna,
            
              Anil Korkut,
            
              Funda Durupinar,
            
              M. Siper,
            
              U. Dogrusoz,
            
              Alvaro Sebastian Vaca Jacome,
            
              Ryan Peckner,
            
              K. E. Christianson,
            
              Jacob D. Jaffe,
            
              Paul T. Spellman,
            
              Joseph E. Aslan,
            
              C. Sander,
            
              E. Demir,
            
          </td>
          <td>2020-10-23</td>
          <td>SSRN Electronic Journal</td>
          <td>0</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d9ec7de08382fed1b89485a4a641773f5f079680" target='_blank'>An argument for mechanism-based statistical inference in cancer</a></td>
          <td>
            
              D. Geman,
            
              M. Ochs,
            
              N. Price,
            
              C. Tomasetti,
            
              L. Younes,
            
          </td>
          <td>2014-11-09</td>
          <td>Human Genetics</td>
          <td>10</td>
          <td>67</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2521c3d76bc439c961b7003080f4a7a661949547" target='_blank'>Causal Protein-Signaling Networks Derived from Multiparameter Single-Cell Data</a></td>
          <td>
            
              K. Sachs,
            
              O. Perez,
            
              D. Pe’er,
            
              D. Lauffenburger,
            
              G. Nolan,
            
          </td>
          <td>2005-04-22</td>
          <td>Science</td>
          <td>1768</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7411b1effb6d89c638ee8e1aa6142d3e48609297" target='_blank'>Using computational modeling and experimental synthetic perturbations to probe biological circuits.</a></td>
          <td>
            
              Joshua R. Porter,
            
              Eric Batchelor,
            
          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>4</td>
          <td>21</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/93da388399e3aad95c8bb229f8f792d9ea241d80" target='_blank'>Robust network inference using response logic</a></td>
          <td>
            
              Torsten Gross,
            
              M. Wongchenko,
            
              Yibing Yan,
            
              Nils Blüthgen,
            
          </td>
          <td>2019-02-12</td>
          <td>Bioinformatics</td>
          <td>7</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d14407c282b9df499c35323b905fde042c651558" target='_blank'>Decoding Heterogenous Single-cell Perturbation Responses</a></td>
          <td>
            
              Bicna Song,
            
              Dingyu Liu,
            
              Weiwei Dai,
            
              Natalie F. McMyn,
            
              Qingyang Wang,
            
              Dapeng Yang,
            
              Adam Krejci,
            
              A. Vasilyev,
            
              Nicole Untermoser,
            
              Anke Loregger,
            
              Dongyuan Song,
            
              Breanna Williams,
            
              Bess P. Rosen,
            
              Xiaolong Cheng,
            
              Lumen Chao,
            
              Hanuman Kale,
            
              Hao Zhang,
            
              Yarui Diao,
            
              T. Bürckstümmer,
            
              Jenet M. Siliciano,
            
              J. J. Li,
            
              R. Siliciano,
            
              Danwei Huangfu,
            
              Wei Li,
            
          </td>
          <td>2023-11-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>110</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0cff4da1b6d07f709aa4bccc536dee490961087e" target='_blank'>Integrating Transcriptomic Data with Mechanistic Systems Pharmacology Models for Virtual Drug Combination Trials.</a></td>
          <td>
            
              A. Barrette,
            
              M. Bouhaddou,
            
              M. Birtwistle,
            
          </td>
          <td>2017-10-06</td>
          <td>ACS chemical neuroscience</td>
          <td>18</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/18580ec0a5e4dac5a687043648dd6ad729c5bc30" target='_blank'>Dissecting heterogeneous cell-populations across signaling and disease conditions with PopAlign</a></td>
          <td>
            
              Sisi Chen,
            
              J. H. Park,
            
              E. Charles,
            
              P. Rivaud,
            
              John R. Haliburton,
            
              F. Pichiorri,
            
              M. Thomson,
            
          </td>
          <td>2018-09-21</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c2c68bf290ad61d28e97651c393992776c0c26bb" target='_blank'>Representing high throughput expression profiles via perturbation barcodes reveals compound targets</a></td>
          <td>
            
              Tracey M. Filzen,
            
              Peter S. Kutchukian,
            
              J. Hermes,
            
              Jing Li,
            
              M. Tudor,
            
          </td>
          <td>2016-04-08</td>
          <td>PLoS Computational Biology</td>
          <td>24</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/335745be9347aab6067811287f1f1ede86b7f9da" target='_blank'>DeepNoise: Disentanglement of Experimental Noise from Real Biological Signals based on Fluorescent Microscopy Image Classiﬁcation via Deep Learning</a></td>
          <td>
            
              Sen Yang,
            
              Tao Shen,
            
              Yu-Qiang Fang,
            
              Xiyue Wang,
            
              Jun Zhang,
            
              Wei Yang,
            
              Junzhou Huang,
            
              Xiao Han,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>74</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/72a114324e88575e903582d9d06a07300ea72658" target='_blank'>Stimulus Design for Model Selection and Validation in Cell Signaling</a></td>
          <td>
            
              J. Apgar,
            
              Jared E. Toettcher,
            
              D. Endy,
            
              F. White,
            
              B. Tidor,
            
          </td>
          <td>2008-02-01</td>
          <td>PLoS Computational Biology</td>
          <td>91</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e6de630ffb762090f2eafd68f1e4dc3b97cf5240" target='_blank'>Dynamic probabilistic threshold networks to infer signaling pathways from time-course perturbation data</a></td>
          <td>
            
              N. Kiani,
            
              L. Kaderali,
            
          </td>
          <td>2014-07-22</td>
          <td>BMC Bioinformatics</td>
          <td>14</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e6de630ffb762090f2eafd68f1e4dc3b97cf5240" target='_blank'>Dynamic probabilistic threshold networks to infer signaling pathways from time-course perturbation data</a></td>
          <td>
            
              N. Kiani,
            
              L. Kaderali,
            
          </td>
          <td>2014-07-22</td>
          <td>BMC Bioinformatics</td>
          <td>14</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/eedda48d7313daae94dcc2aba144c5642f1e4613" target='_blank'>Community-Reviewed Biological Network Models for Toxicology and Drug Discovery Applications</a></td>
          <td>
            
              A. Namasivayam,
            
              Alejandro Ferreiro Morales,
            
              Ángela María Fajardo Lacave,
            
              Aravind Tallam,
            
              B. Simovic,
            
              David Garrido Alfaro,
            
              D. Bobbili,
            
              F. Martin,
            
              G. Androsova,
            
              I. Shvydchenko,
            
              Jennifer Park,
            
              J. Calvo,
            
              J. Hoeng,
            
              M. Peitsch,
            
              Manuel González Vélez Racero,
            
              M. Biryukov,
            
              M. Talikka,
            
              M. B. Pérez,
            
              Neha Rohatgi,
            
              N. díaz-díaz,
            
              Rajesh Mandarapu,
            
              Rubén amián Ruiz,
            
              S. Davidyan,
            
              S. Narayanasamy,
            
              S. Boué,
            
              S. Guryanova,
            
              Susana Martínez Arbas,
            
              S. Menon,
            
              Yang Xiang,
            
          </td>
          <td>2016-07-12</td>
          <td>Gene Regulation and Systems Biology</td>
          <td>14</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94635adf1310128039a73d1b311b9ae88e4f0f1e" target='_blank'>Computational models of melanoma</a></td>
          <td>
            
              M. Albrecht,
            
              P. Lucarelli,
            
              D. Kulms,
            
              T. Sauter,
            
          </td>
          <td>2020-05-14</td>
          <td>Theoretical Biology & Medical Modelling</td>
          <td>9</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/79d4464f1b0244e0527f10c290767152067ea5ca" target='_blank'>A Bayesian model of dose-response for cancer drug studies</a></td>
          <td>
            
              Wesley Tansey,
            
              Christopher Tosh,
            
              D. Blei,
            
          </td>
          <td>2022-06-01</td>
          <td>The Annals of Applied Statistics</td>
          <td>4</td>
          <td>96</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0937219158373bdfc74db0e4bcf254133e08411f" target='_blank'>Learning a nonlinear dynamical system model of gene regulation: A perturbed steady-state approach</a></td>
          <td>
            
              Arwen V. Bradley,
            
              Ye Henry Li,
            
              Bokyung Choi,
            
              W. Wong,
            
          </td>
          <td>2012-07-13</td>
          <td>arXiv: Molecular Networks</td>
          <td>19</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2efc878148dcec184b46a5eef061d772272652d3" target='_blank'>Deterministic Effects Propagation Networks for reconstructing protein signaling networks from multiple interventions</a></td>
          <td>
            
              H. Fröhlich,
            
              Ö. Sahin,
            
              D. Arlt,
            
              C. Bender,
            
              T. Beissbarth,
            
          </td>
          <td>2009-10-08</td>
          <td>BMC Bioinformatics</td>
          <td>30</td>
          <td>50</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f95f5827a8af08ab6f2d79b5afa1d707af9950de" target='_blank'>Prediction in the face of uncertainty: a Monte Carlo-based approach for systems biology of cancer treatment.</a></td>
          <td>
            
              C. Wierling,
            
              Alexander Kühn,
            
              Hendrik Hache,
            
              A. Daskalaki,
            
              Elisabeth Maschke-Dutz,
            
              S. Peycheva,
            
              Jian Li,
            
              R. Herwig,
            
              H. Lehrach,
            
          </td>
          <td>2012-08-15</td>
          <td>Mutation research</td>
          <td>30</td>
          <td>123</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1bd61a55ff09cce9029c90dd6900d4b45278f0b4" target='_blank'>In silico modeling predicts drug sensitivity of patient-derived cancer cells</a></td>
          <td>
            
              S. Pingle,
            
              Zeba Sultana,
            
              S. Pastorino,
            
              Pengfei Jiang,
            
              Rajesh Mukthavaram,
            
              Ying S. Chao,
            
              I. Bharati,
            
              Natsuko Nomura,
            
              M. Makale,
            
              T. Abbasi,
            
              S. Kapoor,
            
              Ansu Kumar,
            
              Shahabuddin Usmani,
            
              A. Agrawal,
            
              S. Vali,
            
              S. Kesari,
            
          </td>
          <td>2014-05-21</td>
          <td>Journal of Translational Medicine</td>
          <td>28</td>
          <td>70</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c99a2d7c37b229d96d35cbfeb55f4a4f8a76db51" target='_blank'>Revealing the impact of genomic alterations on cancer cell signaling with a partially transparent deep learning model</a></td>
          <td>
            
              Jonathan D. Young,
            
              Xinghua Lu,
            
          </td>
          <td>2020-05-30</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9f95550e17d2863ce352419b4c2f62ecca35a535" target='_blank'>Abstract B2-41: Inference of differential dependency networks between tumor and stroma by integrated UNDO-KDDN software tool</a></td>
          <td>
            
              Lulu Chen,
            
              Niya Wang,
            
              R. Clarke,
            
              Zhen Zhang,
            
              Yue Wang,
            
          </td>
          <td>2015-11-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aed89b87e3cc585d11f5420aea7683c047a83cb6" target='_blank'>Improvement of experimental testing and network training conditions with genome-wide microarrays for more accurate predictions of drug gene targets</a></td>
          <td>
            
              Lisa M. Christadore,
            
              Lisa Pham,
            
              E. Kolaczyk,
            
              S. Schaus,
            
          </td>
          <td>2014-01-20</td>
          <td>BMC Systems Biology</td>
          <td>2</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ae0bd1fc031e6b91c1595c77d9451d01f440284b" target='_blank'>Quantifying the effect of experimental perturbations at single-cell resolution</a></td>
          <td>
            
              Daniel B. Burkhardt,
            
              Jay S. Stanley,
            
              Alexander Tong,
            
              A. Perdigoto,
            
              Scott A. Gigante,
            
              K. Herold,
            
              Guy Wolf,
            
              Antonio J. Giráldez,
            
              David van Dijk,
            
              Smita Krishnaswamy,
            
          </td>
          <td>2021-02-08</td>
          <td>Nature Biotechnology</td>
          <td>65</td>
          <td>79</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0cb4fe7edfd36dbba9d6b0c41f39c8a736396a10" target='_blank'>Equation discovery for systems biology: finding the structure and dynamics of biological networks from time course data.</a></td>
          <td>
            
              S. Džeroski,
            
              L. Todorovski,
            
          </td>
          <td>2008-08-01</td>
          <td>Current opinion in biotechnology</td>
          <td>36</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8338a8f30d2ddb8b3b492f4dae4cf6e0958837e0" target='_blank'>Identifiability and experimental design in perturbation studies</a></td>
          <td>
            
              Torsten Gross,
            
              N. Blüthgen,
            
          </td>
          <td>2020-02-03</td>
          <td>Bioinformatics</td>
          <td>8</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78db02f069ce3199468cf9a7c6e2f84e337583cc" target='_blank'>PDE-constrained optimization for estimating population dynamics over cell cycle from static single cell measurements</a></td>
          <td>
            
              Karsten Kuritz,
            
              Alain R. Bonny,
            
              J. P. Fonseca,
            
              F. Allgöwer,
            
          </td>
          <td>2020-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6271a8d1a9451e165feb2a8b3c46db75b899f55f" target='_blank'>Distangling Biological Noise in Cellular Images with a focus on Explainability</a></td>
          <td>
            
              Manik Sharma,
            
              Ganapathy Krishnamurthi,
            
          </td>
          <td>2020-07-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1ea5af003d4cfdb49de74af10115d114b397967c" target='_blank'>Predicting activatory and inhibitory drug–target interactions based on structural compound representations and genetically perturbed transcriptomes</a></td>
          <td>
            
              Won-Yung Lee,
            
              Choong-Yeol Lee,
            
              C. Kim,
            
          </td>
          <td>2023-04-12</td>
          <td>PLOS ONE</td>
          <td>1</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/73ebd9f0f3eabc0113b4add3631b55ec5cbb561b" target='_blank'>Using Regularization to Infer Cell Line Specificity in Logical Network Models of Signaling Pathways</a></td>
          <td>
            
              Sébastien De Landtsheer,
            
              P. Lucarelli,
            
              T. Sauter,
            
          </td>
          <td>2018-05-22</td>
          <td>Frontiers in Physiology</td>
          <td>3</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b6eb03c66d700dbbea0473e7de6144bf037edc0" target='_blank'>Computationally predicting clinical drug combination efficacy with cancer cell line screens and independent drug action</a></td>
          <td>
            
              Alexander L. Ling,
            
              R. Huang,
            
          </td>
          <td>2020-11-17</td>
          <td>Nature Communications</td>
          <td>33</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e19082c5e115a6c5dadc6055444e9505afb315e1" target='_blank'>Classification of Single-Cell Gene Expression Trajectories from Incomplete and Noisy Data</a></td>
          <td>
            
              Alireza Karbalayghareh,
            
              U. Braga-Neto,
            
              E. Dougherty,
            
          </td>
          <td>2019-01-01</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>10</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9b4488d13f45fd490582e6d6ad558b49a1162460" target='_blank'>Data-Derived Modeling Characterizes Plasticity of MAPK Signaling in Melanoma</a></td>
          <td>
            
              Martí Bernardo-Faura,
            
              S. Massen,
            
              C. Falk,
            
              N. Brady,
            
              R. Eils,
            
          </td>
          <td>2014-09-01</td>
          <td>PLoS Computational Biology</td>
          <td>20</td>
          <td>95</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/990ca78406f1c4f5e529d537054eea2d99859240" target='_blank'>A signaling visualization toolkit to support rational design of combination therapies and biomarker discovery: SiViT</a></td>
          <td>
            
              J. Bown,
            
              Mark M. Shovman,
            
              P. Robertson,
            
              A. Boiko,
            
              A. Goltsov,
            
              P. Mullen,
            
              D. Harrison,
            
          </td>
          <td>2016-05-18</td>
          <td>Oncotarget</td>
          <td>3</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fafd6f308ebf0f9f1ee3efbef6e4c1b3c9653bf3" target='_blank'>DELVE: feature selection for preserving biological trajectories in single-cell data</a></td>
          <td>
            
              Jolene S Ranek,
            
              Wayne Stallaert,
            
              J. Milner,
            
              Margaret Redick,
            
              Samuel C Wolff,
            
              Adriana S Beltran,
            
              Natalie Stanley,
            
              J. Purvis,
            
          </td>
          <td>2024-03-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ab7248d46ef069e3633f0114b71e9a4e8722b09b" target='_blank'>Success in the DREAM3 Signaling Response Challenge Using Simple Weighted-Average Imputation: Lessons for Community-Wide Experiments in Systems Biology</a></td>
          <td>
            
              N. Clarke,
            
              G. Bourque,
            
          </td>
          <td>2010-01-26</td>
          <td>PLoS ONE</td>
          <td>3</td>
          <td>68</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4a4c4caba7eb86c63e0c07ee1a475ef8ee660247" target='_blank'>Predicting drug polypharmacology from cell morphology readouts using variational autoencoder latent space arithmetic</a></td>
          <td>
            
              Yuen Ler Chow,
            
              Shantanu Singh,
            
              Anne E Carpenter,
            
              G. Way,
            
          </td>
          <td>2021-09-03</td>
          <td>PLoS Computational Biology</td>
          <td>18</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b500090d881e7b0775cfb27435a5da1ca8f7a17c" target='_blank'>Logic-based mechanistic machine learning on high-content images reveals how drugs differentially regulate cardiac fibroblasts</a></td>
          <td>
            
              Anders R. Nelson,
            
              Steven L. M. Christiansen,
            
              Kristen M. Naegle,
            
              J. Saucerman,
            
          </td>
          <td>2023-03-02</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b500090d881e7b0775cfb27435a5da1ca8f7a17c" target='_blank'>Logic-based mechanistic machine learning on high-content images reveals how drugs differentially regulate cardiac fibroblasts</a></td>
          <td>
            
              Anders R. Nelson,
            
              Steven L. M. Christiansen,
            
              Kristen M. Naegle,
            
              J. Saucerman,
            
          </td>
          <td>2023-03-02</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d3598e809d661783e745dd086da8ae1f4d009c05" target='_blank'>A Comprehensive Statistical Model for Cell Signaling</a></td>
          <td>
            
              E. Yörük,
            
              M. Ochs,
            
              D. Geman,
            
              L. Younes,
            
          </td>
          <td>2011-05-01</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>10</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a195afa11b49103054c41ab700a3ae097fccc647" target='_blank'>DeepNoise: Signal and Noise Disentanglement Based on Classifying Fluorescent Microscopy Images via Deep Learning</a></td>
          <td>
            
              Sen Yang,
            
              Tao Shen,
            
              Yu-Qiang Fang,
            
              Xiyue Wang,
            
              Jun Zhang,
            
              Wei Yang,
            
              Junzhou Huang,
            
              Xiao Han,
            
          </td>
          <td>2022-09-13</td>
          <td>Genomics, Proteomics & Bioinformatics</td>
          <td>3</td>
          <td>74</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/771e6ee84353828e640c230c07c64013c8803ab5" target='_blank'>Leveraging multi-omics data to empower quantitative systems pharmacology in immuno-oncology</a></td>
          <td>
            
              Theinmozhi Arulraj,
            
              Hanwen Wang,
            
              Alberto Ippolito,
            
              Shuming Zhang,
            
              E. Fertig,
            
              Aleksander S. Popel,
            
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e1f7ddfdbb771bb0f803ae5b5f1e342fd2f17ef2" target='_blank'>Inferring cellular and molecular processes in single-cell data with non-negative matrix factorization using Python, R, and GenePattern Notebook implementations of CoGAPS</a></td>
          <td>
            
              J. Johnson,
            
              Ashley Tsang,
            
              Jacob T. Mitchell,
            
              Emily F Davis-Marcisak,
            
              Thomas Sherman,
            
              T. Liefeld,
            
              Melanie Loth,
            
              L. Goff,
            
              Jacquelyn W. Zimmerman,
            
              B. Kinny-Köster,
            
              E. Jaffee,
            
              P. Tamayo,
            
              J. Mesirov,
            
              Michael Reich,
            
              E. Fertig,
            
              G. Stein-O’Brien,
            
          </td>
          <td>2022-07-11</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>90</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b84478133d9d79d052399951ba8c79234c20037" target='_blank'>BMC Systems Biology BioMed Central Methodology article</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>460</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a2cdecf7649d9ce754181cfb5d5567c4682db98e" target='_blank'>Using probabilistic models of signaling pathways to predict in vivo drug activity</a></td>
          <td>
            
              P. Sebastián-León,
            
              J. Dopazo,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>77</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8e4a481062fdd70b557da7bcc8719b3f776c26c3" target='_blank'>The misleading certainty of uncertain data in biological network processes</a></td>
          <td>
            
              Michael W. Irvin,
            
              Arvind Ramanathan,
            
              Carlos F. Lopez,
            
          </td>
          <td>2021-05-20</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c11e11ef48b7a694b15aa88af2b752dafa0c85e6" target='_blank'>Learning a nonlinear gene regulation model from perturbed steady-state data</a></td>
          <td>
            
              Arwen Bradley,
            
              Ye,
            
              Li,
            
              Bokyung Choi,
            
              W. Wong,
            
          </td>
          <td>2012-07-13</td>
          <td>None</td>
          <td>1</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2e5c86fad7e22ef184ef596f865b63073f02dec9" target='_blank'>A Multistaged Hyperparallel Optimization of the Fuzzy-Logic Mechanistic Model of Molecular Regulation</a></td>
          <td>
            
              P. Aiyetan,
            
          </td>
          <td>2020-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/575a20013d5441cbe07adefa9c799f500eeb2544" target='_blank'>Bayesian Inference of Signaling Network Topology in a Cancer Cell Line</a></td>
          <td>
            
              S. Hill,
            
              Yiling Lu,
            
              Jennifer Molina,
            
              Laura M. Heiser,
            
              P. Spellman,
            
              T. Speed,
            
              J. Gray,
            
              G. Mills,
            
              S. Mukherjee,
            
          </td>
          <td>2012-11-01</td>
          <td>Bioinformatics</td>
          <td>103</td>
          <td>184</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/881b3732b72fc42968e8ccda0c256f3f5d3a9540" target='_blank'>Analysis and modeling of cancer drug responses using cell cycle phase-specific rate effects</a></td>
          <td>
            
              Sean M. Gross,
            
              Farnaz Mohammadi,
            
              Crystal Sanchez-Aguila,
            
              Paulina J. Zhan,
            
              T. Liby,
            
              Mark A. Dane,
            
              Aaron S. Meyer,
            
              Laura M. Heiser,
            
          </td>
          <td>2020-07-25</td>
          <td>Nature Communications</td>
          <td>7</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/34b8b10bd706b465a4b35bd6573fb290df7f8632" target='_blank'>Pre-Training on In Vitro and Fine-Tuning on Patient-Derived Data Improves Deep Neural Networks for Anti-Cancer Drug-Sensitivity Prediction</a></td>
          <td>
            
              Paul Prasse,
            
              Pascal Iversen,
            
              M. Lienhard,
            
              Kristina Thedinga,
            
              R. Herwig,
            
              T. Scheffer,
            
          </td>
          <td>2022-08-01</td>
          <td>Cancers</td>
          <td>3</td>
          <td>49</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7f9521b9f710d1b17990852bfe6edcda24c67d57" target='_blank'>Analyzing pathway design from drug perturbation experiments</a></td>
          <td>
            
              Noah E. Berlow,
            
              R. Pal,
            
              L. Davis,
            
              C. Keller,
            
          </td>
          <td>2012-10-04</td>
          <td>2012 IEEE Statistical Signal Processing Workshop (SSP)</td>
          <td>4</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ab353723dbf21ca46b048821a572c5a18cc80468" target='_blank'>Better Accuracy in Predicting Efficacy of Drug Combinations Using Novel Algorithm</a></td>
          <td>
            
              Santeri Mentu,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bd3f273590a34e121ddf5c2669c00750c8792e2b" target='_blank'>Characterizing heterogeneous cellular responses to perturbations</a></td>
          <td>
            
              M. Slack,
            
              Elisabeth D. Martinez,
            
              Lani F. Wu,
            
              S. Altschuler,
            
          </td>
          <td>2008-12-09</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>164</td>
          <td>38</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f945e25ec7be816d9c187081af84c5ef6dd91193" target='_blank'>DIVERSE: Bayesian Data IntegratiVE Learning for Precise Drug ResponSE Prediction</a></td>
          <td>
            
              Betul Guvencpaltun,
            
              Samuel Kaski,
            
              Hiroshi Mamitsuka,
            
          </td>
          <td>2021-03-11</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>7</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5917e98fde69eab5466607e591165a98856a0b7f" target='_blank'>From Years to Hours: Accelerating Model Refinement</a></td>
          <td>
            
              Kyu Hyong Park,
            
              Jordan C. Rozum,
            
              Réka Albert,
            
          </td>
          <td>2023-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ce356827ea9b65952d417b6ad241721f65fe24e5" target='_blank'>Computational analysis of biochemical networks for drug target identification and therapeutic intervention design</a></td>
          <td>
            
              N. Paudel,
            
          </td>
          <td>None</td>
          <td></td>
          <td>1</td>
          <td>6</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d8b53c25d331df61bf601a33636784d04ce05d68" target='_blank'>Learning interpretable cellular embedding for inferring biological mechanisms underlying single-cell transcriptomics</a></td>
          <td>
            
              k.-l. Hsieh,
            
              Y. Chu,
            
              P. G. Pilie,
            
              K. Zhang,
            
          </td>
          <td>2024-04-01</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/87cc3107c99c1db2a6a8beed00d4eac32d467c35" target='_blank'>A single-cell gene regulatory network inference method for identifying complex regulatory dynamics across cell phenotypes</a></td>
          <td>
            
              Jianhao Peng,
            
              Guillermo Serrano,
            
              Ian M. Traniello,
            
              M. E. Calleja-Cervantes,
            
              U. V. Chembazhi,
            
              Sushant Bangru,
            
              Teresa Ezponda,
            
              J. R. Rodríguez-Madoz,
            
              A. Kalsotra,
            
              Felipe Prósper,
            
              Idoia Ochoa,
            
              M. Hernaez,
            
          </td>
          <td>2020-04-04</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/87cc3107c99c1db2a6a8beed00d4eac32d467c35" target='_blank'>A single-cell gene regulatory network inference method for identifying complex regulatory dynamics across cell phenotypes</a></td>
          <td>
            
              Jianhao Peng,
            
              Guillermo Serrano,
            
              Ian M. Traniello,
            
              M. E. Calleja-Cervantes,
            
              U. V. Chembazhi,
            
              Sushant Bangru,
            
              Teresa Ezponda,
            
              J. R. Rodríguez-Madoz,
            
              A. Kalsotra,
            
              Felipe Prósper,
            
              Idoia Ochoa,
            
              M. Hernaez,
            
          </td>
          <td>2020-04-04</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f9b4ba528526bb4328fc81d6fd71855cf3d45b87" target='_blank'>WENDY: Gene Regulatory Network Inference with Covariance Dynamics</a></td>
          <td>
            
              Yue Wang,
            
              Peng Zheng,
            
              Yu-Chen Cheng,
            
              Zikun Wang,
            
              Aleksandr Y Aravkin,
            
          </td>
          <td>2024-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6d2587e756e59a2edf3bc23628241e2ea308d755" target='_blank'>Sloppy models, parameter uncertainty, and the role of experimental design.</a></td>
          <td>
            
              J. Apgar,
            
              David K Witmer,
            
              F. White,
            
              B. Tidor,
            
          </td>
          <td>2010-06-01</td>
          <td>Molecular bioSystems</td>
          <td>128</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d70d519d7a3b77e8a130c72da7e66ef5c0d3360d" target='_blank'>Learning the cellular activity representation based on gene regulatory networks for prediction of tumor response to drugs.</a></td>
          <td>
            
              Xinping Xie,
            
              Fengting Wang,
            
              Guanfu Wang,
            
              Weiwei Zhu,
            
              Xiaodong Du,
            
              Hongqiang Wang,
            
          </td>
          <td>2024-04-01</td>
          <td>Artificial intelligence in medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/498f2bc0606e297f071fdba36db552b3bf11792a" target='_blank'>A systems biology method to predict oncogenes and perturbation targets on a cellular level Technology</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8a6a90f34561eae70a352ff887d0c5483a723ebc" target='_blank'>Deep learning based CETSA feature prediction cross multiple cell lines with latent space representation</a></td>
          <td>
            
              Shenghao Zhao,
            
              Xulei Yang,
            
              Zeng Zeng,
            
              Peisheng Qian,
            
              Ziyuan Zhao,
            
              Lingyun Dai,
            
              N. Prabhu,
            
              Pär Nordlund,
            
              Wai Leong Tam,
            
          </td>
          <td>2024-01-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3244f86e0ba72e134a61f4497c72a5ff8277b226" target='_blank'>A virtual B cell lymphoma model to predict effective combination therapy</a></td>
          <td>
            
              W. Du,
            
              R. Goldstein,
            
              Yanwen Jiang,
            
              Omar Aly,
            
              L. Cerchietti,
            
              A. Melnick,
            
              O. Elemento,
            
          </td>
          <td>2014-12-06</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>106</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/00d54fdde2e7f199813e86ab03a7ab6805f23394" target='_blank'>Inferring Signaling Pathway Topologies from Multiple Perturbation Measurements of Specific Biochemical Species</a></td>
          <td>
            
              Tian-Rui Xu,
            
              Vladislav Vyshemirsky,
            
              Amelie Gormand,
            
              A. von Kriegsheim,
            
              M. Girolami,
            
              G. Baillie,
            
              Dominic Ketley,
            
              Allan J. Dunlop,
            
              Graeme Milligan,
            
              M. Houslay,
            
              Walter Kolch,
            
          </td>
          <td>2010-03-16</td>
          <td>Science Signaling</td>
          <td>133</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2bd89a32034fd6d35c445f014cecc7addf92f16c" target='_blank'>Integration of transcriptomics data into agent-based models of solid tumor metastasis</a></td>
          <td>
            
              Jimmy Retzlaff,
            
              Xin Lai,
            
              C. Berking,
            
              J. Vera,
            
          </td>
          <td>2023-01-10</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>3</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c67f967eeb7e3243351500f71f8e9a42870942bd" target='_blank'>Multidimensional single-cell modeling of cellular signaling</a></td>
          <td>
            
              J. D. Wade,
            
              Xiao-Kang Lun,
            
              B. Bodenmiller,
            
              E. Voit,
            
          </td>
          <td>2020-11-16</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/95d039d13b016696484b2fc193d1d20de3e9df88" target='_blank'>Modeling of signal-response cascades using decision tree analysis</a></td>
          <td>
            
              S. Hautaniemi,
            
              Sourabh Kharait,
            
              Akihiro Iwabu,
            
              A. Wells,
            
              D. Lauffenburger,
            
          </td>
          <td>2005-05-01</td>
          <td>Bioinformatics</td>
          <td>70</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c00b3fb240007ac9db87c89bbfb3dec6bbfd7f53" target='_blank'>Many are better than one - next generation multivariate biomarkers for precision oncology</a></td>
          <td>
            
              Jinyan Du,
            
              D. Kirouac,
            
              B. Schoeberl,
            
          </td>
          <td>2017-01-09</td>
          <td>Cancer cell & microenvironment</td>
          <td>0</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8eb67e99e76235ceeed881994c5518e18f76a75c" target='_blank'>caspo: a toolbox for automated reasoning on the response of logical signaling networks families</a></td>
          <td>
            
              S. Videla,
            
              J. Sáez-Rodríguez,
            
              Carito Guziolowski,
            
              A. Siegel,
            
          </td>
          <td>2016-11-23</td>
          <td>Bioinformatics</td>
          <td>19</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/488815adde83ad267b507082359af3dfc5928d8c" target='_blank'>CausalBench: A Large-scale Benchmark for Network Inference from Single-cell Perturbation Data</a></td>
          <td>
            
              Mathieu Chevalley,
            
              Yusuf H. Roohani,
            
              Arash Mehrjou,
            
              J. Leskovec,
            
              P. Schwab,
            
          </td>
          <td>2022-10-31</td>
          <td>ArXiv</td>
          <td>13</td>
          <td>132</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6344032d7e05e3b37e1ccbd27844e226fe017bcb" target='_blank'>The Library of Integrated Network-Based Cellular Signatures NIH Program: System-Level Cataloging of Human Cells Response to Perturbations.</a></td>
          <td>
            
              A. Keenan,
            
              S. L. Jenkins,
            
              Kathleen M. Jagodnik,
            
              Simon Koplev,
            
              Edward He,
            
              Denis Torre,
            
              Zichen Wang,
            
              Anders B. Dohlman,
            
              Moshe C. Silverstein,
            
              Alexander Lachmann,
            
              Maxim V. Kuleshov,
            
              Avi Ma’ayan,
            
              Vasileios Stathias,
            
              Raymond Terryn,
            
              Daniel J. Cooper,
            
              M. Forlin,
            
              Amar Koleti,
            
              D. Vidović,
            
              Caty Chung,
            
              S. Schürer,
            
              Jouzas Vasiliauskas,
            
              M. Pilarczyk,
            
              Behrouz Shamsaei,
            
              Mehdi Fazel,
            
              Yan Ren,
            
              W. Niu,
            
              Nicholas A. Clark,
            
              Shana White,
            
              N. Mahi,
            
              Lixia Zhang,
            
              Michal Kouril,
            
              J. Reichard,
            
              S. Sivaganesan,
            
              M. Medvedovic,
            
              J. Meller,
            
              R. J. Koch,
            
              M. Birtwistle,
            
              R. Iyengar,
            
              E. Sobie,
            
              E. Azeloglu,
            
              J. Kaye,
            
              Jeannette M. Osterloh,
            
              K. Haston,
            
              Jaslin Kalra,
            
              Steve Finkbiener,
            
              Jonathan Li,
            
              P. Milani,
            
              Miriam Adam,
            
              Renan Escalante-Chong,
            
              K. Sachs,
            
              A. LeNail,
            
              Divya Ramamoorthy,
            
              E. Fraenkel,
            
              G. Daigle,
            
              Uzma Hussain,
            
              A. Coye,
            
              J. Rothstein,
            
              D. Sareen,
            
              L. Ornelas,
            
              M. Bañuelos,
            
              B. Mandefro,
            
              Ritchie Ho,
            
              C. Svendsen,
            
              Ryan G. Lim,
            
              Jennifer T. Stocksdale,
            
              M. Casale,
            
              T. Thompson,
            
              Jie Wu,
            
              L. Thompson,
            
              V. Dardov,
            
              Vidya Venkatraman,
            
              Andrea D Matlock,
            
              J. V. Van Eyk,
            
              Jacob D. Jaffe,
            
              Malvina Papanastasiou,
            
              A. Subramanian,
            
              T. Golub,
            
              S. Erickson,
            
              M. Fallahi-Sichani,
            
              M. Hafner,
            
              N. Gray,
            
              Jia-Ren Lin,
            
              Caitlin E. Mills,
            
              J. Muhlich,
            
              M. Niepel,
            
              C. Shamu,
            
              Elizabeth H. Williams,
            
              D. Wrobel,
            
              P. Sorger,
            
              Laura M. Heiser,
            
              J. Gray,
            
              J. Korkola,
            
              G. Mills,
            
              Mark A. LaBarge,
            
              H. Feiler,
            
              Mark A. Dane,
            
              Elmar Bucher,
            
              M. Nederlof,
            
              D. Sudar,
            
              Sean M. Gross,
            
              David Kilburn,
            
              Rebecca Smith,
            
              Kaylyn L. Devlin,
            
              R. Margolis,
            
              L. Derr,
            
              Albert Lee,
            
              Ajay Pillai,
            
          </td>
          <td>2017-11-29</td>
          <td>Cell systems</td>
          <td>289</td>
          <td>184</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/60a57678d8c2ca100491abb88d9a332d1b9e22a7" target='_blank'>Profiling Cell Signaling Networks at Single-cell Resolution</a></td>
          <td>
            
              Xiao-Kang Lun,
            
              B. Bodenmiller,
            
          </td>
          <td>2020-03-04</td>
          <td>Molecular & Cellular Proteomics : MCP</td>
          <td>24</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bba1d4de76b8330bea5f73ddeb99da4e01c02e16" target='_blank'>B IO D ISCOVERY A GENT : A N AI A GENT FOR D ESIGN - ING G ENETIC P ERTURBATION E XPERIMENTS</a></td>
          <td>
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b9733bae80b05e065eb7ef9698ef331b1d9f378c" target='_blank'>Detecting and Removing Inconsistencies between Experimental Data and Signaling Network Topologies Using Integer Linear Programming on Interaction Graphs</a></td>
          <td>
            
              Ioannis N. Melas,
            
              Regina Samaga,
            
              L. Alexopoulos,
            
              Steffen Klamt,
            
          </td>
          <td>2013-09-01</td>
          <td>PLoS Computational Biology</td>
          <td>41</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7c3bd69bf9dadc888bfd85bb4ca183949e7bfc6a" target='_blank'>Assessment of network perturbation amplitudes by applying high-throughput data to causal biological networks</a></td>
          <td>
            
              F. Martin,
            
              T. Thomson,
            
              A. Sewer,
            
              David A. Drubin,
            
              C. Mathis,
            
              D. Weisensee,
            
              Dexter Pratt,
            
              J. Hoeng,
            
              M. Peitsch,
            
          </td>
          <td>2012-05-31</td>
          <td>BMC Systems Biology</td>
          <td>101</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f34f3f17f559c88b561928a179247e8c428d6bf3" target='_blank'>Discrete Logic Modelling Optimization to Contextualize Prior Knowledge Networks Using PRUNET</a></td>
          <td>
            
              Ana Rodriguez,
            
              I. Crespo,
            
              Anna Fournier,
            
              A. del Sol,
            
          </td>
          <td>2015-06-09</td>
          <td>PLOS ONE</td>
          <td>18</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f61827b05e3b39d1a6c6640e56addecb0b990c6a" target='_blank'>Systems Perturbation Analysis of a Large-Scale Signal Transduction Model Reveals Potentially Influential Candidates for Cancer Therapeutics</a></td>
          <td>
            
              B. L. Puniya,
            
              Laura Allen,
            
              C. Hochfelder,
            
              M. Majumder,
            
              T. Helikar,
            
          </td>
          <td>2015-11-10</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>18</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fbf7f988fa139f63575bf182bc2f39a8a5ad7dc0" target='_blank'>Drepmel—A Multi-Omics Melanoma Drug Repurposing Resource for Prioritizing Drug Combinations and Understanding Tumor Microenvironment</a></td>
          <td>
            
              Z. Thompson,
            
              J. Teer,
            
              Jiannong Li,
            
              Zhihua Chen,
            
              E. Welsh,
            
              Yonghong Zhang,
            
              N. Ayoubi,
            
              Z. Eroglu,
            
              A. Tan,
            
              K. Smalley,
            
              Y. Chen,
            
          </td>
          <td>2022-09-01</td>
          <td>Cells</td>
          <td>2</td>
          <td>58</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7315a031786d41325855e8fdd3e3e19d12f61053" target='_blank'>Tensor clustering with algebraic constraints gives interpretable groups of crosstalk mechanisms in breast cancer</a></td>
          <td>
            
              A. Seigal,
            
              Mariano Beguerisse-Díaz,
            
              B. Schoeberl,
            
              M. Niepel,
            
              H. Harrington,
            
          </td>
          <td>2016-12-24</td>
          <td>Journal of the Royal Society Interface</td>
          <td>4</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b05e3b22fc0df7ba3a0d570535bf8818efd2ac18" target='_blank'>Cell-specific gene association network construction from single-cell RNA sequence</a></td>
          <td>
            
              Riasat Azim,
            
              Shulin Wang,
            
          </td>
          <td>2021-09-16</td>
          <td>Cell Cycle</td>
          <td>3</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9c65b074b4707a2b04ae8651aa734fab0612772a" target='_blank'>SystemMatch: optimizing preclinical drug models to human clinical outcomes via generative latent-space matching</a></td>
          <td>
            
              Scott A. Gigante,
            
              Varsha Raghavan,
            
              A. M. Robinson,
            
              R. Barton,
            
              Adeeb H. Rahman,
            
              Drausin Wulsin,
            
              J. Banchereau,
            
              Noam Solomon,
            
              L. Voloch,
            
              F. Theis,
            
          </td>
          <td>2022-05-14</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>145</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b83e489fa272f3250c3d817cfe2d4f5e89c7326b" target='_blank'>Learning Edge Rewiring in EMT From Single Cell Data</a></td>
          <td>
            
              Smita Krishnaswamy,
            
              Nevena Zivanovic,
            
              Roshan Sharma,
            
              D. Pe’er,
            
              B. Bodenmiller,
            
          </td>
          <td>2017-06-25</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78ff3585f4dc208ce372acd896a86df979e84b04" target='_blank'>Integrating knowledge and omics to decipher mechanisms via large‐scale models of signaling networks</a></td>
          <td>
            
              Martín Garrido-Rodriguez,
            
              Katharina Zirngibl,
            
              Olga Ivanova,
            
              Sebastian Lobentanzer,
            
              J. Saez-Rodriguez,
            
          </td>
          <td>2022-07-01</td>
          <td>Molecular Systems Biology</td>
          <td>21</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d4a4667f81d620bfa94f4a43a19a2b29226a2e20" target='_blank'>McSNAC: A software to approximate first-order signaling networks from mass cytometry data</a></td>
          <td>
            
              Darren Wethington,
            
              Sayak Mukherjee,
            
              J. Das,
            
          </td>
          <td>2021-12-03</td>
          <td>Quantitative biology (Beijing, China)</td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0365f88a00f43b81ddbbb5fbe6bbf1fdb97a4de6" target='_blank'>Isolating salient variations of interest in single-cell data with contrastiveVI</a></td>
          <td>
            
              Ethan Weinberger,
            
              Chris Lin,
            
              Su-In Lee,
            
          </td>
          <td>2021-12-23</td>
          <td>Nature Methods</td>
          <td>12</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ce2f254c7a49e1050d4b89c0370089c6605682af" target='_blank'>DTF: Deep Tensor Factorization for predicting anticancer drug synergy</a></td>
          <td>
            
              Zexuan Sun,
            
              Shujun Huang,
            
              Peiran Jiang,
            
              P. Hu,
            
              Zhiyong Lu,
            
          </td>
          <td>2019-11-23</td>
          <td>Bioinformatics</td>
          <td>33</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6657d442f660cdc2a368869097b286ce92592c24" target='_blank'>Abstract 901: BATCHIE: An active learning platform for scalable combination drug screens</a></td>
          <td>
            
              Christopher Tosh,
            
              Mauricio Tec,
            
              Jessica White,
            
              Jeffrey F. Quinn,
            
              G. I. Sanchez,
            
              Paul Calder,
            
              Andrew L Kung,
            
              F. D. Dela Cruz,
            
              Wesley Tansey,
            
          </td>
          <td>2024-03-22</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/556057faef4505a6e365005ee93b54fd7278f8d7" target='_blank'>Analyzing synergistic and non-synergistic interactions in signalling pathways using Boolean Nested Effect Models</a></td>
          <td>
            
              M. Pirkl,
            
              Elisabeth Hand,
            
              D. Kube,
            
              R. Spang,
            
          </td>
          <td>2015-11-17</td>
          <td>Bioinformatics</td>
          <td>20</td>
          <td>49</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3be276b845f398a0b0b4fdf04e21694fbc94efee" target='_blank'>Experimental guidance for discovering genetic networks through hypothesis reduction on time series</a></td>
          <td>
            
              Breschine Cummins,
            
              Francis C. Motta,
            
              R. Moseley,
            
              Anastasia Deckard,
            
              Sophia Campione,
            
              Marcio Gameiro,
            
              Tomáš Gedeon,
            
              K. Mischaikow,
            
              S. Haase,
            
          </td>
          <td>2022-10-01</td>
          <td>PLoS Computational Biology</td>
          <td>0</td>
          <td>47</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/59c81b009db40c80e73020b1c80518f8b9289375" target='_blank'>Sparse dictionary learning recovers pleiotropy from human cell fitness screens</a></td>
          <td>
            
              Joshua Pan,
            
              Jason J. Kwon,
            
              Jessica A. Talamas,
            
              Ashir A. Borah,
            
              F. Vazquez,
            
              J. Boehm,
            
              Aviad Tsherniak,
            
              M. Zitnik,
            
              James M. McFarland,
            
              W. Hahn,
            
          </td>
          <td>2021-11-11</td>
          <td>Cell systems</td>
          <td>16</td>
          <td>135</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7d8107f718d356a3c7b8818e271840763e78bdac" target='_blank'>Discovery of Drug Synergies in Gastric Cancer Cells Predicted by Logical Modeling</a></td>
          <td>
            
              Åsmund Flobak,
            
              A. Baudot,
            
              É. Remy,
            
              L. Thommesen,
            
              D. Thieffry,
            
              Martin Kuiper,
            
              A. Lægreid,
            
          </td>
          <td>2015-08-01</td>
          <td>PLoS Computational Biology</td>
          <td>111</td>
          <td>55</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/360fb8ff8dd2318e5f1a96e5150459e24a8e5f06" target='_blank'>Dr.VAE: improving drug response prediction via modeling of drug perturbation effects</a></td>
          <td>
            
              Ladislav Rampášek,
            
              Daniel Hidru,
            
              P. Smirnov,
            
              B. Haibe-Kains,
            
              A. Goldenberg,
            
          </td>
          <td>2019-03-08</td>
          <td>Bioinformatics</td>
          <td>108</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6b87b9af14033e28c6cf715b0c208f557fc365a8" target='_blank'>TRIOMPHE: Transcriptome-Based Inference and Generation of Molecules with Desired Phenotypes by Machine Learning</a></td>
          <td>
            
              Kazuma Kaitoh,
            
              Yoshihiro Yamanishi,
            
          </td>
          <td>2021-09-16</td>
          <td>Journal of chemical information and modeling</td>
          <td>10</td>
          <td>34</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bc07dfb613ccafbc973aca41dde40873fb19fcc2" target='_blank'>Systematic Analysis of Network-driven Adaptive Resistance to CDK4/6 and Estrogen Receptor Inhibition using Meta-Dynamic Network Modelling</a></td>
          <td>
            
              Anthony Hart,
            
              Sung-young Shin,
            
              Leann Nguyen,
            
          </td>
          <td>2023-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/95d039d13b016696484b2fc193d1d20de3e9df88" target='_blank'>Modeling of signal-response cascades using decision tree analysis</a></td>
          <td>
            
              S. Hautaniemi,
            
              Sourabh Kharait,
            
              Akihiro Iwabu,
            
              A. Wells,
            
              D. Lauffenburger,
            
          </td>
          <td>2005-05-01</td>
          <td>Bioinformatics</td>
          <td>70</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c15025b2adb84a86f475c1b56dd3fd524013d17b" target='_blank'>Boosting an operator-valued kernel-based model for gene regulatory network inference</a></td>
          <td>
            
              Néhémy Lim,
            
              Y. Şenbabaoğlu,
            
              G. Michailidis,
            
              Florence d'Alché-Buc,
            
          </td>
          <td>2013-06-25</td>
          <td></td>
          <td>0</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/99b84ce390b4d74e38658380a51fcba4ab303453" target='_blank'>UvA-DARE ( Digital Academic Repository ) Cyclic causal discovery from continuous equilibrium data</a></td>
          <td>
            
              Mooij,
            
              A. Nicholson,
            
              Padhraic Smyth,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>0</td>
          <td>74</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b27fa6d7136b04aaa1d827afdf6206bf9808eb80" target='_blank'>Identifying cancer pathway dysregulations using differential causal effects</a></td>
          <td>
            
              K. P. Jablonski,
            
              M. Pirkl,
            
              Domagoj Cevid,
            
              P. Bühlmann,
            
              N. Beerenwinkel,
            
          </td>
          <td>2021-05-21</td>
          <td>Bioinformatics</td>
          <td>5</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f53abb1efc651a84264fb1d7da5b8dc49b9d84ae" target='_blank'>Literature and data-driven based inference of signalling interactions using time-course data</a></td>
          <td>
            
              Enio Gjerga,
            
              Enio Gjerga,
            
              P. Trairatphisan,
            
              A. Gábor,
            
              A. Gábor,
            
              J. Saez-Rodriguez,
            
              J. Sáez-Rodríguez,
            
          </td>
          <td>None</td>
          <td>IFAC-PapersOnLine</td>
          <td>3</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/76e33d0ae43f2975586e25f3a91b591e8f0f75c5" target='_blank'>Inferring the Impact of Regulatory Mechanisms that Underpin CD8+ T Cell Control of B16 Tumor Growth In vivo Using Mechanistic Models and Simulation</a></td>
          <td>
            
              D. Klinke,
            
              Qing Wang,
            
          </td>
          <td>2017-01-04</td>
          <td>Frontiers in Pharmacology</td>
          <td>4</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6fc30204f7f20fb3ac4d3ad6f5d04c9cd1102c53" target='_blank'>Signaling network prediction by the Ontology Fingerprint enhanced Bayesian network</a></td>
          <td>
            
              T. Qin,
            
              L. Tsoi,
            
              Kellie J. Sims,
            
              Xinghua Lu,
            
              Jim Zheng,
            
          </td>
          <td>2012-12-17</td>
          <td>BMC Systems Biology</td>
          <td>23</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0babdc5493acddb6091a2c52b73fd07c66b6c1d6" target='_blank'>Community assessment to advance computational prediction of cancer drug combinations in a pharmacogenomic screen</a></td>
          <td>
            
              M. Menden,
            
              Dennis Wang,
            
              M. Mason,
            
              B. Szalai,
            
              K. Bulusu,
            
              Y. Guan,
            
              Thomas V Yu,
            
              Jaewoo Kang,
            
              Minji Jeon,
            
              R. Wolfinger,
            
              Tin Nguyen,
            
              M. Zaslavskiy,
            
              I. S. Jang,
            
              Z. Ghazoui,
            
              M. Ahsen,
            
              Robert M. Vogel,
            
              E. C. Neto,
            
              Thea C. Norman,
            
              E. Tang,
            
              M. Garnett,
            
              G. Y. Veroli,
            
              S. Fawell,
            
              G. Stolovitzky,
            
              J. Guinney,
            
              J. Dry,
            
              J. Saez-Rodriguez,
            
          </td>
          <td>2019-06-17</td>
          <td>Nature Communications</td>
          <td>220</td>
          <td>60</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3f6eafa12f5411411737a9b30517c9616aa7f31a" target='_blank'>Unsupervised Methods to Identify Cellular Signaling Networks from Perturbation Data</a></td>
          <td>
            
              Madhusudan Natarajan,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/84ce858182e9aaf67d340b7ba0e45b93ad39db98" target='_blank'>scPreGAN, a deep generative model for predicting the response of single-cell expression to perturbation</a></td>
          <td>
            
              Xiajie Wei,
            
              Jiayi Dong,
            
              Fei Wang,
            
          </td>
          <td>2022-05-26</td>
          <td>Bioinformatics</td>
          <td>9</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/200f728eee7b2cff84fcd8d2d9b47da3a5a2ab82" target='_blank'>MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data</a></td>
          <td>
            
              Greg Finak,
            
              A. McDavid,
            
              Masanao Yajima,
            
              Jingyuan Deng,
            
              V. Gersuk,
            
              A. Shalek,
            
              Chloe K. Slichter,
            
              Hannah W. Miller,
            
              M. McElrath,
            
              Martin Prlic,
            
              P. Linsley,
            
              R. Gottardo,
            
          </td>
          <td>2015-12-01</td>
          <td>Genome Biology</td>
          <td>1907</td>
          <td>100</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b14c2cd9d82834bc4882f54621d663826d673cf" target='_blank'>Experimental Analysis and Computational Modeling of Network States and Drug Responses in the PI3K/Akt/mTOR Network</a></td>
          <td>
            
              J. Albeck,
            
          </td>
          <td>2010-09-01</td>
          <td></td>
          <td>0</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ec98d57ac46b038ac48020a6e499582a397c5217" target='_blank'>Reconstruction of Protein Networks Using Reverse-Phase Protein Array Data.</a></td>
          <td>
            
              S. von der Heyde,
            
              Johanna Sonntag,
            
              F. Kramer,
            
              C. Bender,
            
              U. Korf,
            
              T. Beissbarth,
            
          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>3</td>
          <td>50</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/54b1f1c5ae47f3c852ff7476d009961baf1c50f3" target='_blank'>Experimental Design of Time Series Data for Learning from Dynamic Bayesian Networks</a></td>
          <td>
            
              David Page,
            
              I. Ong,
            
          </td>
          <td>2005-12-01</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>7</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/61938257dcd466f868c5c81ac5b2eebec0fdd0ee" target='_blank'>Deep Learning Applications for Predicting Pharmacological Properties of Drugs and Drug Repurposing Using Transcriptomic Data.</a></td>
          <td>
            
              A. Aliper,
            
              S. Plis,
            
              Artem V. Artemov,
            
              Alvaro E. Ulloa,
            
              Polina Mamoshina,
            
              A. Zhavoronkov,
            
          </td>
          <td>2016-06-08</td>
          <td>Molecular pharmaceutics</td>
          <td>386</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/584dcc03dd96b6eec4512a2ad7e7c9fc800293fc" target='_blank'>Abstract LB253: Crafted experiments to evaluate feature selection methods for single cell RNA-seq data</a></td>
          <td>
            
              Siyao Liu,
            
              David Corcoran,
            
              Susana Garcia-Recio,
            
              C. Perou,
            
              J. S. Marron,
            
          </td>
          <td>2024-04-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>159</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a3b92781aae07cd6b35e85d7266cf509b642663c" target='_blank'>SCENERY: a web application for (causal) network reconstruction from cytometry data</a></td>
          <td>
            
              Giorgos Papoutsoglou,
            
              Giorgos Athineou,
            
              V. Lagani,
            
              Iordanis Xanthopoulos,
            
              A. Schmidt,
            
              Szabolcs Éliás,
            
              J. Tegnér,
            
              I. Tsamardinos,
            
          </td>
          <td>2017-05-19</td>
          <td>Nucleic Acids Research</td>
          <td>8</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7929604c7e4f58db52122f7895e19a4a46d7d0ec" target='_blank'>Identification of optimal drug combinations targeting cellular networks: integrating phospho-proteomics and computational network analysis.</a></td>
          <td>
            
              S. Iadevaia,
            
              Yiling Lu,
            
              F. C. Morales,
            
              G. Mills,
            
              Prahlad T. Ram,
            
          </td>
          <td>2010-09-01</td>
          <td>Cancer research</td>
          <td>223</td>
          <td>184</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d9722e905fb152f3bb768aaa963bf28e987d71b1" target='_blank'>Hybrid analysis of gene dynamics predicts context-specific expression and offers regulatory insights</a></td>
          <td>
            
              Justin D. Finkle,
            
              N. Bagheri,
            
          </td>
          <td>2019-04-17</td>
          <td>Bioinformatics</td>
          <td>1</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d77cc87500842f15984fa04644f5b52f8b1e4367" target='_blank'>Network-Based Systems Analysis Explains Sequence-Dependent Synergism of Bortezomib and Vorinostat in Multiple Myeloma</a></td>
          <td>
            
              Charvi Nanavati,
            
              D. Mager,
            
          </td>
          <td>2021-08-17</td>
          <td>The AAPS Journal</td>
          <td>2</td>
          <td>42</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8c1b2642b7deee2de5ec2d1277f01e61a675e22c" target='_blank'>Modeling and Simulation of Cell Signaling Networks for Subsequent Analytics Processes Using Big Data and Machine Learning</a></td>
          <td>
            
              Máximo Sánchez-Gutiérrez,
            
              P. P. González-Pérez,
            
          </td>
          <td>2022-04-01</td>
          <td>Bioinformatics and Biology Insights</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/873fccf9e6cd078a3443bc485c0869ff9b3e92d4" target='_blank'>Homologous control of protein signaling networks.</a></td>
          <td>
            
              D. Napoletani,
            
              Michele Signore,
            
              T. Sauer,
            
              Lance Liotta,
            
              E. Petricoin,
            
          </td>
          <td>2010-12-26</td>
          <td>Journal of theoretical biology</td>
          <td>2</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0ac3523e475895f932788b6f1139e60668a32aff" target='_blank'>Discovering novel phenotypes with automatically inferred dynamic models: a partial melanocyte conversion in Xenopus</a></td>
          <td>
            
              Daniel Lobo,
            
              Maria Lobikin,
            
              M. Levin,
            
          </td>
          <td>2017-01-27</td>
          <td>Scientific Reports</td>
          <td>26</td>
          <td>79</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1eb3255208e44de14ac7db3ce29a3f38432c9734" target='_blank'>Infinity Flow: comprehensive single-cell protein profiling via massively parallel flow cytometry and machine learning</a></td>
          <td>
            
              E. Becht,
            
              M. Headley,
            
              E. Newell,
            
              R. Gottardo,
            
          </td>
          <td>2020-05-01</td>
          <td>The Journal of Immunology</td>
          <td>1</td>
          <td>58</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/22d6444a5532ffce7bf19a916b998c31a9a395a0" target='_blank'>Mining of Single-Cell Signaling Time-Series for Dynamic Phenotypes with Clustering.</a></td>
          <td>
            
              M. Dobrzyński,
            
              Marc-Antoine Jacques,
            
              O. Pertz,
            
          </td>
          <td>None</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a595193f78e429286f6816e227a16b9753d835b3" target='_blank'>Towards a Rigorous Assessment of Systems Biology Models: The DREAM3 Challenges</a></td>
          <td>
            
              R. Prill,
            
              D. Marbach,
            
              J. Sáez-Rodríguez,
            
              P. Sorger,
            
              L. Alexopoulos,
            
              Xiaowei Xue,
            
              N. Clarke,
            
              G. Altan-Bonnet,
            
              G. Stolovitzky,
            
          </td>
          <td>2010-02-23</td>
          <td>PLoS ONE</td>
          <td>436</td>
          <td>104</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fa68093ce64240e73b94545459c36af8255c511e" target='_blank'>Inferring Gene Networks: Dream or Nightmare?</a></td>
          <td>
            
              Angela Baralla,
            
              Wieslawa I. Mentzen,
            
              A. G. de la Fuente,
            
          </td>
          <td>2009-03-01</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>34</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f22bb4508e66f9ed4c214cdae0c6c462e3c50e96" target='_blank'>Predicting Causal Relationships from Biological Data: Applying Automated Causal Discovery on Mass Cytometry Data of Human Immune Cells</a></td>
          <td>
            
              Sofia Triantafillou,
            
              V. Lagani,
            
              C. Heinze-Deml,
            
              A. Schmidt,
            
              J. Tegnér,
            
              I. Tsamardinos,
            
          </td>
          <td>2017-03-30</td>
          <td>Scientific Reports</td>
          <td>36</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1fa3c7dc93d61be2fd9307be8ff1cf4c49a38efa" target='_blank'>GeneTrail: A Framework for the Analysis of High-Throughput Profiles</a></td>
          <td>
            
              Nico Gerstner,
            
              Tim Kehl,
            
              Kerstin Lenhof,
            
              Lea Eckhart,
            
              L. Schneider,
            
              D. Stöckel,
            
              C. Backes,
            
              E. Meese,
            
              A. Keller,
            
              H. Lenhof,
            
          </td>
          <td>2021-09-16</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>1</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/46e6788d46be3158a86699997dfcfd78c4b83989" target='_blank'>BaTFLED: Bayesian Tensor Factorization Linked to External Data</a></td>
          <td>
            
              N. Lazar,
            
              M. Gönen,
            
              Kemal Sönmez,
            
          </td>
          <td>2016-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/070cf68141875951ae062963dd8c63c84c096900" target='_blank'>Time varying causal network reconstruction of a mouse cell cycle</a></td>
          <td>
            
              Maryam Masnadi-Shirazi,
            
              M. Maurya,
            
              G. Pao,
            
              Eugene Ke,
            
              I. Verma,
            
              S. Subramaniam,
            
          </td>
          <td>2019-05-29</td>
          <td>BMC Bioinformatics</td>
          <td>2</td>
          <td>118</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c09fa4f63cf7efb2eace65b7fa78f3ec62deee08" target='_blank'>Cyclic Causal Discovery from Continuous Equilibrium Data</a></td>
          <td>
            
              J. Mooij,
            
              T. Heskes,
            
          </td>
          <td>2013-08-11</td>
          <td>ArXiv</td>
          <td>72</td>
          <td>52</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ca6410a37106ae5d68d99c17f729e603b5c1642a" target='_blank'>Efficient parameterization of large-scale dynamic models based on relative measurements</a></td>
          <td>
            
              Leonard Schmiester,
            
              Yannik Schälte,
            
              Fabian Froehlich,
            
              J. Hasenauer,
            
              Daniel Weindl,
            
          </td>
          <td>2019-03-16</td>
          <td>Bioinformatics</td>
          <td>30</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f0063f4c59becc261188fc5a820fb6bdb55f691d" target='_blank'>Uncovering expression signatures of synergistic drug responses via ensembles of explainable machine-learning models</a></td>
          <td>
            
              Joseph D. Janizek,
            
              A. Dincer,
            
              Safiye Celik,
            
              Hugh Chen,
            
              William Chen,
            
              K. Naxerova,
            
              Su-In Lee,
            
          </td>
          <td>2023-05-01</td>
          <td>Nature Biomedical Engineering</td>
          <td>9</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/92e3a7754d090a24e6d1a9fb5a486e177656f4b3" target='_blank'>Causal interactions from proteomic profiles: Molecular data meet pathway knowledge</a></td>
          <td>
            
              O. Babur,
            
              Augustin Luna,
            
              Anil Korkut,
            
              Funda Durupinar,
            
              M. Siper,
            
              U. Dogrusoz,
            
              Alvaro Sebastian Vaca Jacome,
            
              Ryan Peckner,
            
              K. E. Christianson,
            
              Jacob D. Jaffe,
            
              Paul T. Spellman,
            
              J. Aslan,
            
              C. Sander,
            
              E. Demir,
            
          </td>
          <td>2021-05-12</td>
          <td>Patterns</td>
          <td>58</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b7b7d84f0bac0d2ab35ae5eef3c57f582bc8593d" target='_blank'>Theoretical and practical advances in genome halving</a></td>
          <td>
            
              F. Collyn,
            
              L. Guy,
            
              M. Marceau,
            
              M. Simonet,
            
              Claude-Alain H. Roten,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>28362</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b7b7d84f0bac0d2ab35ae5eef3c57f582bc8593d" target='_blank'>Theoretical and practical advances in genome halving</a></td>
          <td>
            
              F. Collyn,
            
              L. Guy,
            
              M. Marceau,
            
              M. Simonet,
            
              Claude-Alain H. Roten,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>28362</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b7b7d84f0bac0d2ab35ae5eef3c57f582bc8593d" target='_blank'>Theoretical and practical advances in genome halving</a></td>
          <td>
            
              F. Collyn,
            
              L. Guy,
            
              M. Marceau,
            
              M. Simonet,
            
              Claude-Alain H. Roten,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>28362</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b7b7d84f0bac0d2ab35ae5eef3c57f582bc8593d" target='_blank'>Theoretical and practical advances in genome halving</a></td>
          <td>
            
              F. Collyn,
            
              L. Guy,
            
              M. Marceau,
            
              M. Simonet,
            
              Claude-Alain H. Roten,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>28362</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b7b7d84f0bac0d2ab35ae5eef3c57f582bc8593d" target='_blank'>Theoretical and practical advances in genome halving</a></td>
          <td>
            
              F. Collyn,
            
              L. Guy,
            
              M. Marceau,
            
              M. Simonet,
            
              Claude-Alain H. Roten,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>28362</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b7b7d84f0bac0d2ab35ae5eef3c57f582bc8593d" target='_blank'>Theoretical and practical advances in genome halving</a></td>
          <td>
            
              F. Collyn,
            
              L. Guy,
            
              M. Marceau,
            
              M. Simonet,
            
              Claude-Alain H. Roten,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>28362</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b7b7d84f0bac0d2ab35ae5eef3c57f582bc8593d" target='_blank'>Theoretical and practical advances in genome halving</a></td>
          <td>
            
              F. Collyn,
            
              L. Guy,
            
              M. Marceau,
            
              M. Simonet,
            
              Claude-Alain H. Roten,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>28362</td>
          <td>14</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/18680502d62bbf7c41a60a9e5befd4de25cadb95" target='_blank'>Structure Learning in Nested Effects Models</a></td>
          <td>
            
              A. Tresch,
            
              F. Markowetz,
            
          </td>
          <td>2007-10-24</td>
          <td>Statistical Applications in Genetics and Molecular Biology</td>
          <td>56</td>
          <td>47</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f746febb4c253dbf9fa2bcb37e3ebbf95804141b" target='_blank'>Estimating information in time-varying signals</a></td>
          <td>
            
              S. A. Cepeda-Humerez,
            
              J. Ruess,
            
              G. Tkačik,
            
          </td>
          <td>2018-12-31</td>
          <td>PLoS Computational Biology</td>
          <td>18</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7c6b01e192c73385280afcbac9f76b4519b59a8c" target='_blank'>Complexities in Quantitative Systems Analysis of Signaling Networks</a></td>
          <td>
            
              C. Kiel,
            
              L. Serrano,
            
          </td>
          <td>None</td>
          <td></td>
          <td>1</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8e133d29b87cf2a314abfa9308ee05434a58d6cf" target='_blank'>Computational modeling of drug response with applications to neuroscience</a></td>
          <td>
            
              R. Herwig,
            
          </td>
          <td>2014-12-01</td>
          <td>Dialogues in Clinical Neuroscience</td>
          <td>3</td>
          <td>49</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1f150157eda1816af4e3294859d5d19e9c642222" target='_blank'>Benchmarking Machine Learning Methods for Synthetic Lethality Prediction in Cancer</a></td>
          <td>
            
              Yimiao Feng,
            
              Yahui Long,
            
              He Wang,
            
              Ouyang Yang,
            
              Quan Li,
            
              Min Wu,
            
              Jie Zheng,
            
          </td>
          <td>2023-11-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c43760f612df15deed3d5233d66e69b6d93084e5" target='_blank'>Reconstruction of Gene Regulatory Networks based on Repairing Sparse Low-rank Matrices</a></td>
          <td>
            
              Y. Chang,
            
              Roel Dobbe,
            
              Palak Bhushan,
            
              J. Gray,
            
              C. Tomlin,
            
          </td>
          <td>2014-12-11</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>104</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/19610013a2fa4d705b1acbf4e81c2c436c7e2899" target='_blank'>Boosting an operator-valued kernel model and application to network inference</a></td>
          <td>
            
              Néhémy Lim,
            
              Y. Şenbabaoğlu,
            
              G. Michailidis,
            
              Florence d'Alché-Buc,
            
          </td>
          <td>2013-06-20</td>
          <td></td>
          <td>0</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/76a177f4f518b1334f51305937286796552b8cea" target='_blank'>A Web Application for Predicting Drug Combination Efficacy Using Monotherapy Data and IDACombo.</a></td>
          <td>
            
              Yunong Xia,
            
              Alexander L. Ling,
            
              Weijie Zhang,
            
              Adam Lee,
            
              Mei-Chi Su,
            
              Robert F. Gruener,
            
              Sampreeti Jena,
            
              Yingbo Huang,
            
              Siddhika Pareek,
            
              Yuting Shan,
            
              R. S. Huang,
            
          </td>
          <td>None</td>
          <td>Journal of cancer science and clinical therapeutics</td>
          <td>2</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/75f581981a222f5b0749f3a969807d0aa573b574" target='_blank'>Creating and analyzing pathway and protein interaction compendia for modelling signal transduction networks</a></td>
          <td>
            
              D. Kirouac,
            
              J. Sáez-Rodríguez,
            
              J. Swantek,
            
              J. Burke,
            
              D. Lauffenburger,
            
              P. Sorger,
            
          </td>
          <td>2012-05-01</td>
          <td>BMC Systems Biology</td>
          <td>77</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f465afbeddc991f3b9faa0121952835168ae6891" target='_blank'>Mini-batch optimization enables training of ODE models on large-scale datasets</a></td>
          <td>
            
              P. Stapor,
            
              Leonard Schmiester,
            
              C. Wierling,
            
              Simon Merkt,
            
              Dilan Pathirana,
            
              B. Lange,
            
              Daniel Weindl,
            
              J. Hasenauer,
            
          </td>
          <td>2019-11-30</td>
          <td>Nature Communications</td>
          <td>6</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f465afbeddc991f3b9faa0121952835168ae6891" target='_blank'>Mini-batch optimization enables training of ODE models on large-scale datasets</a></td>
          <td>
            
              P. Stapor,
            
              Leonard Schmiester,
            
              C. Wierling,
            
              Simon Merkt,
            
              Dilan Pathirana,
            
              B. Lange,
            
              Daniel Weindl,
            
              J. Hasenauer,
            
          </td>
          <td>2019-11-30</td>
          <td>Nature Communications</td>
          <td>6</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/814563494f0b39231d02f0fb8a20587e7da38438" target='_blank'>Inferring Broad Regulatory Biology from Time Course Data: Have We Reached an Upper Bound under Constraints Typical of In Vivo Studies?</a></td>
          <td>
            
              Saurabh Vashishtha,
            
              G. Broderick,
            
              T. Craddock,
            
              M. Fletcher,
            
              N. Klimas,
            
          </td>
          <td>2015-05-18</td>
          <td>PLoS ONE</td>
          <td>8</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/33a02af358fbd16bec8ac57e1ec33dde16830598" target='_blank'>Systematic identification of feature combinations for predicting drug response with Bayesian multi-view multi-task linear regression</a></td>
          <td>
            
              Muhammad Ammad-ud-din,
            
              Suleiman A. Khan,
            
              K. Wennerberg,
            
              T. Aittokallio,
            
          </td>
          <td>2017-07-12</td>
          <td>Bioinformatics</td>
          <td>57</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f686a0c084f941a72e614e40389bd938f0ca42bf" target='_blank'>Computational Systems Biochemistry: Beyond the Static Interactome.</a></td>
          <td>
            
              S. Sarabipour,
            
              F. Mac Gabhann,
            
          </td>
          <td>2018-01-09</td>
          <td>Biochemistry</td>
          <td>1</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bfb478b4825e2c9c60699cecd04fc7020569aa66" target='_blank'>High-Resolution Modeling of Cellular Signaling Networks</a></td>
          <td>
            
              M. Baym,
            
              C. Bakal,
            
              N. Perrimon,
            
              B. Berger,
            
          </td>
          <td>2008-03-30</td>
          <td>None</td>
          <td>8</td>
          <td>143</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7929604c7e4f58db52122f7895e19a4a46d7d0ec" target='_blank'>Identification of optimal drug combinations targeting cellular networks: integrating phospho-proteomics and computational network analysis.</a></td>
          <td>
            
              S. Iadevaia,
            
              Yiling Lu,
            
              F. C. Morales,
            
              G. Mills,
            
              Prahlad T. Ram,
            
          </td>
          <td>2010-09-01</td>
          <td>Cancer research</td>
          <td>223</td>
          <td>184</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/324d3e186193186f6ea885f56c6fc0e4f26a2c90" target='_blank'>Causal network perturbations for instance-specific analysis of single cell and disease samples</a></td>
          <td>
            
              K. Buschur,
            
              M. Chikina,
            
              P. Benos,
            
          </td>
          <td>2019-05-16</td>
          <td>Bioinformatics</td>
          <td>17</td>
          <td>39</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b4b022abcdecc7a1c117c6321a23be450219007e" target='_blank'>OKVAR-Boost: a novel boosting algorithm to infer nonlinear dynamics and interactions in gene regulatory networks</a></td>
          <td>
            
              Néhémy Lim,
            
              Y. Şenbabaoğlu,
            
              G. Michailidis,
            
              Florence d'Alché-Buc,
            
          </td>
          <td>2013-04-10</td>
          <td>Bioinformatics</td>
          <td>32</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8dc4a9931fdd5bf87408a0a55efaa720e7a84e0f" target='_blank'>Robust Prediction of Patient-Specific Clinical Response to Unseen Drugs From in vitro Screens Using Context-aware Deconfounding Autoencoder</a></td>
          <td>
            
              Di He,
            
              Qiao Liu,
            
              Lei Xie,
            
          </td>
          <td>2021-05-21</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bceaf703938d9a36435d575866a495d261076d88" target='_blank'>Generating executable models from signaling network connectivity and semi-quantitative proteomic measurements</a></td>
          <td>
            
              Derek A. Ruths,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>1</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d911b3d5723d950df1e1b5d5f6172e5de972af66" target='_blank'>FLOW: maximum flow framework for the identification of factors mediating the signaling convergence of multiple receptors</a></td>
          <td>
            
              Ron Sheinin,
            
              Shai Dulberg,
            
              A. Kaminitz,
            
              R. Sharan,
            
              A. Madi,
            
          </td>
          <td>2023-01-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>68</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ff0eaf506285f84c57e1fbd24e7d0bdb8796fc6d" target='_blank'>Inferring single-cell protein levels and cell cycle behavior in heterogeneous cell populations</a></td>
          <td>
            
              B. Thijssen,
            
              H. A. Segeren,
            
              Qingwu Liu,
            
              L. Wessels,
            
              B. Westendorp,
            
          </td>
          <td>2023-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/65df4d7a93a21e32eeed146ccb6444899e31a1bb" target='_blank'>Systematic review and network meta-analysis of drug combinations suggested by machine learning on genes and proteins, with the aim of improving the effectiveness of Ipilimumab in treating Melanoma.</a></td>
          <td>
            
              D. Safaei,
            
              A. Kiaei,
            
              M. Boush,
            
              S. Abadijou,
            
              N. Salari,
            
              M. Mohammadi,
            
              Elham Parichehreh,
            
          </td>
          <td>2023-05-14</td>
          <td>None</td>
          <td>3</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dba10c467fdeb0f890e92ef38be03606d8a4d760" target='_blank'>Comparative analysis of cell–cell communication at single-cell resolution</a></td>
          <td>
            
              Aaron J. Wilk,
            
              A. Shalek,
            
              S. Holmes,
            
              C. Blish,
            
          </td>
          <td>2022-02-07</td>
          <td>Nature Biotechnology</td>
          <td>21</td>
          <td>65</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/289c62cd9e88356e3e48a4c6e9ef62209dbd344a" target='_blank'>Data Portal for the Library of Integrated Network-based Cellular Signatures (LINCS) program: integrated access to diverse large-scale cellular perturbation response data</a></td>
          <td>
            
              Amar Koleti,
            
              Raymond Terryn,
            
              Vasileios Stathias,
            
              Caty Chung,
            
              Daniel J. Cooper,
            
              J. Turner,
            
              D. Vidović,
            
              M. Forlin,
            
              Tanya T. Kelley,
            
              A. D’Urso,
            
              Bryce K. Allen,
            
              Denis Torre,
            
              Kathleen M. Jagodnik,
            
              L. Wang,
            
              S. L. Jenkins,
            
              Christopher Mader,
            
              W. Niu,
            
              Mehdi Fazel,
            
              N. Mahi,
            
              M. Pilarczyk,
            
              Nicholas A. Clark,
            
              Behrouz Shamsaei,
            
              J. Meller,
            
              Juozas Vasiliauskas,
            
              J. Reichard,
            
              M. Medvedovic,
            
              Avi Ma’ayan,
            
              Ajay Pillai,
            
              S. Schürer,
            
          </td>
          <td>2017-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>130</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/00466942a9b305f8ffab519c094f24b991194dc8" target='_blank'>Dissecting differential signals in high-throughput data from complex tissues</a></td>
          <td>
            
              Ziyi Li,
            
              Zhijin Wu,
            
              P. Jin,
            
              Hao Wu,
            
          </td>
          <td>2018-08-29</td>
          <td>bioRxiv</td>
          <td>31</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/340b7e143a4e74046e859ec33ff648c4addb5df7" target='_blank'>A simple and flexible computational framework for inferring sources of heterogeneity from single-cell dynamics</a></td>
          <td>
            
              L. Dharmarajan,
            
              Hans-Michael Kaltenbach,
            
              F. Rudolf,
            
              J. Stelling,
            
          </td>
          <td>2018-06-08</td>
          <td>bioRxiv</td>
          <td>14</td>
          <td>18</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1cee87199420b2caff3caa79eb7df4f5d3b9a783" target='_blank'>Seminars in Cell & Developmental Biology</a></td>
          <td>
            
              Denise M Davis,
            
              J. Purvis,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>17</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7442eb37810048fb64b23b53aca660c7b381242d" target='_blank'>Dose–response prediction for in-vitro drug combination datasets: a probabilistic approach</a></td>
          <td>
            
              Leiv Rønneberg,
            
              Paul D. W. Kirk,
            
              M. Zucknick,
            
          </td>
          <td>2023-04-21</td>
          <td>BMC Bioinformatics</td>
          <td>2</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/777fe6e367aa7a01a9e4731974ddcb6fa8a5421b" target='_blank'>SynergyX: a multi-modality mutual attention network for interpretable drug synergy prediction</a></td>
          <td>
            
              Yue Guo,
            
              Haitao Hu,
            
              Wenbo Chen,
            
              Hao Yin,
            
              Jian Wu,
            
              Chang-Yu Hsieh,
            
              Qiaojun He,
            
              Ji Cao,
            
          </td>
          <td>2024-01-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/102fb0eaadbe867ed76354bf3df19d532af97f99" target='_blank'>Knowledge-fused differential dependency network models for detecting significant rewiring in biological networks</a></td>
          <td>
            
              Ye Tian,
            
              Bai Zhang,
            
              E. Hoffman,
            
              R. Clarke,
            
              Zhen Zhang,
            
              I. Shih,
            
              J. Xuan,
            
              D. Herrington,
            
              Y. Wang,
            
          </td>
          <td>2013-10-28</td>
          <td>BMC Systems Biology</td>
          <td>29</td>
          <td>103</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0d95828b6c9ce80677678b1e3cae670233bd2ba3" target='_blank'>A functional module states framework reveals transcriptional states for drug and target prediction.</a></td>
          <td>
            
              Guangrong Qin,
            
              T. Knijnenburg,
            
              David L. Gibbs,
            
              R. Moser,
            
              R. Monnat,
            
              C. Kemp,
            
              I. Shmulevich,
            
          </td>
          <td>2022-01-01</td>
          <td>Cell reports</td>
          <td>1</td>
          <td>81</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/99d13534f4a428159022376d33a35366a86b8b95" target='_blank'>Reverse causal reasoning: applying qualitative causal knowledge to the interpretation of high-throughput data</a></td>
          <td>
            
              Natalie L. Catlett,
            
              Anthony J. Bargnesi,
            
              Stephen Ungerer,
            
              Toby Seagaran,
            
              W. Ladd,
            
              Keith Owen Elliston,
            
              Dexter Pratt,
            
          </td>
          <td>2013-11-23</td>
          <td>BMC Bioinformatics</td>
          <td>95</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/99d13534f4a428159022376d33a35366a86b8b95" target='_blank'>Reverse causal reasoning: applying qualitative causal knowledge to the interpretation of high-throughput data</a></td>
          <td>
            
              Natalie L. Catlett,
            
              Anthony J. Bargnesi,
            
              Stephen Ungerer,
            
              Toby Seagaran,
            
              W. Ladd,
            
              Keith Owen Elliston,
            
              Dexter Pratt,
            
          </td>
          <td>2013-11-23</td>
          <td>BMC Bioinformatics</td>
          <td>95</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d391f3ea4ba237cda17d6ff0c1cfdfeab8b7d363" target='_blank'>A functional module states framework reveals cell states for drug and target prediction</a></td>
          <td>
            
              Guangrong Qin,
            
              T. Knijnenburg,
            
              David L. Gibbs,
            
              R. Moser,
            
              R. Monnat,
            
              C. Kemp,
            
              I. Shmulevich,
            
          </td>
          <td>2020-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>81</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f1929056317f2e4b398b4187f5332e7b2505dc74" target='_blank'>Petri Nets with Fuzzy Logic (PNFL): Reverse Engineering and Parametrization</a></td>
          <td>
            
              R. Küffner,
            
              Tobias Petri,
            
              Lukas Windhager,
            
              R. Zimmer,
            
          </td>
          <td>2010-09-20</td>
          <td>PLoS ONE</td>
          <td>44</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3fb7af4d9db286271d611cd759af819a0c15015c" target='_blank'>Learning Cyclic Signaling Pathway Structures While Minimizing Data Requirements</a></td>
          <td>
            
              K. Sachs,
            
              S. Itani,
            
              Jonathan Basil Fitzgerald,
            
              Lucia Wille,
            
              B. Schoeberl,
            
              M. Dahleh,
            
              G. Nolan,
            
          </td>
          <td>2008-11-01</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>15</td>
          <td>101</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/22d509f45712a799b31d9f381f463d741c57182a" target='_blank'>scapGNN: A graph neural network–based framework for active pathway and gene module inference from single-cell multi-omics data</a></td>
          <td>
            
              Xudong Han,
            
              Bing Wang,
            
              Chenghao Situ,
            
              Yaling Qi,
            
              Hui Zhu,
            
              Yan Li,
            
              Xuejiang Guo,
            
          </td>
          <td>2023-11-01</td>
          <td>PLOS Biology</td>
          <td>2</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/02108c118b34acb56cb5e41d224cf0b1f8d2fbec" target='_blank'>Compound activity prediction with dose-dependent transcriptomic profiles and deep learning</a></td>
          <td>
            
              William J. Godinez,
            
              Vladimir Trifonov,
            
              Bin Fang,
            
              Guray Kuzu,
            
              Luying Pei,
            
              W. A. Guiguemde,
            
              E. Martin,
            
              Frederick J. King,
            
              J. Jenkins,
            
              Peter Skewes-Cox,
            
          </td>
          <td>2023-08-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a34fac43d8af822019eb922d4e4169f99e04823c" target='_blank'>Pseudo-temporal ordering of individual cells reveals dynamics and regulators of cell fate decisions</a></td>
          <td>
            
              C. Trapnell,
            
              D. Cacchiarelli,
            
              J. Grimsby,
            
              Prapti Pokharel,
            
              Shuqiang Li,
            
              Michael Morse,
            
              N. Lennon,
            
              K. Livak,
            
              T. Mikkelsen,
            
              J. Rinn,
            
          </td>
          <td>2014-03-23</td>
          <td>Nature biotechnology</td>
          <td>3828</td>
          <td>91</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78c3cdbb40fdd490515e7d92b83645e889a05e77" target='_blank'>PTML: Perturbation-Theory Machine Learning notes</a></td>
          <td>
            
              H. González-Díaz,
            
          </td>
          <td>2018-07-29</td>
          <td>Proceedings of MOL2NET 2018, International Conference on Multidisciplinary Sciences, 4th edition</td>
          <td>4</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6066680b4266566576bc91fc28a95f18b898f502" target='_blank'>Accelerating the Development of Personalized Cancer Immunotherapy by Integrating Molecular Patients' Profiles with Dynamic Mathematical Models.</a></td>
          <td>
            
              Z. Agur,
            
              M. Elishmereni,
            
              U. Foryś,
            
              Y. Kogan,
            
          </td>
          <td>2020-06-14</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>11</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ec29a85fa243cc233fa19a11be6113e5b3c1ad13" target='_blank'>GexMolGen: Cross-modal Generation of Hit-like Molecules via Large Language Model Encoding of Gene Expression Signatures</a></td>
          <td>
            
              Jiabei Cheng,
            
              Xiaoyong Pan,
            
              Kaiyuan Yang,
            
              Shenghao Cao,
            
              Bin Liu,
            
              Qingran Yan,
            
              Ye Yuan,
            
          </td>
          <td>2024-02-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5c3c4278005e08109ced6029bce3e30144603bc5" target='_blank'>Toward Modeling Context-Specific EMT Regulatory Networks Using Temporal Single Cell RNA-Seq Data</a></td>
          <td>
            
              Daniel Ramirez,
            
              V. Kohar,
            
              Mingyang Lu,
            
          </td>
          <td>2020-04-23</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>22</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7e4407cdea481018cf79932ee44555257b0ac2ed" target='_blank'>Intrinsically Bayesian Robust Classifier for Single-Cell Gene Expression Time Series in Gene Regulatory Networks</a></td>
          <td>
            
              Alireza Karbalayghareh,
            
              U. Braga-Neto,
            
              E. Dougherty,
            
          </td>
          <td>2017-08-20</td>
          <td>Proceedings of the 8th ACM International Conference on Bioinformatics, Computational Biology,and Health Informatics</td>
          <td>2</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b1b8834d34b6fcc076ca62afd924d20b3d555141" target='_blank'>Abstract 3106: Integrated in vitro and machine learning approaches for targeted drug screening in gliomas with concurrent NF1 and ATRX loss</a></td>
          <td>
            
              Swati Dubey,
            
              S. Mohapatra,
            
              M. Yuan,
            
              C. Eberhart,
            
              Fausto J. Rodriguez,
            
          </td>
          <td>2024-03-22</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a403a54265a819b8d0e4f45505431644ecbfc02d" target='_blank'>Dynamics-based data science in biology</a></td>
          <td>
            
              Jifan Shi,
            
              K. Aihara,
            
              Luonan Chen,
            
          </td>
          <td>2021-02-12</td>
          <td>National Science Review</td>
          <td>14</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aed5d73322b026e81679d90a5b93550793ba4b19" target='_blank'>Robust Prediction of Personalized In Vivo Response to Unseen Drugs From In Vitro Screens Using a Novel Context-Aware Deconfounding Autoencoder</a></td>
          <td>
            
              Di He,
            
              Qiao Liu,
            
              Lei Xie,
            
          </td>
          <td>2021-11-12</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4fb7c778fb854581a5980268e837a1ed6b48309e" target='_blank'>Fuzzy modeling and global optimization to predict novel therapeutic targets in cancer cells</a></td>
          <td>
            
              Marco S. Nobile,
            
              G. Votta,
            
              R. Palorini,
            
              S. Spolaor,
            
              Humberto de vitto,
            
              P. Cazzaniga,
            
              F. Ricciardiello,
            
              G. Mauri,
            
              L. Alberghina,
            
              F. Chiaradonna,
            
              D. Besozzi,
            
          </td>
          <td>2019-11-21</td>
          <td>Bioinformatics</td>
          <td>11</td>
          <td>51</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/13f167f52fdd62be8d8dfbc2e92bab9b7e90f920" target='_blank'>Prediction of drug efficacy from transcriptional profiles with deep learning</a></td>
          <td>
            
              Jie Zhu,
            
              Jingxiang Wang,
            
              Xin Wang,
            
              Mingjing Gao,
            
              Bingbing Guo,
            
              Miaomiao Gao,
            
              Jiarui Liu,
            
              Yanqiu Yu,
            
              Liang Wang,
            
              Weikaixin Kong,
            
              Yongpan An,
            
              Zurui Liu,
            
              Xinpei Sun,
            
              Zhuo Huang,
            
              Hong Zhou,
            
              Ning Zhang,
            
              Ruimao Zheng,
            
              Zhengwei Xie,
            
          </td>
          <td>2021-06-17</td>
          <td>Nature Biotechnology</td>
          <td>65</td>
          <td>27</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/729e7b79d8240a360cec4aae90c56ec35de1541c" target='_blank'>Uncovering expression signatures of synergistic drug response using an ensemble of explainable AI models</a></td>
          <td>
            
              Joseph D. Janizek,
            
              A. Dincer,
            
              Safiye Celik,
            
              Hugh Chen,
            
              William Chen,
            
              K. Naxerova,
            
              Su-In Lee,
            
          </td>
          <td>2021-10-07</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fc583004d4583e52c3849627848ea2d290c65e66" target='_blank'>Intrinsically Bayesian robust classifier for single-cell gene expression trajectories in gene regulatory networks</a></td>
          <td>
            
              Alireza Karbalayghareh,
            
              U. Braga-Neto,
            
              E. Dougherty,
            
          </td>
          <td>2018-03-21</td>
          <td>BMC Systems Biology</td>
          <td>10</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a9488eec35424227f278c505eef1285f2fcb3bb3" target='_blank'>Dual-view jointly learning improves personalized drug synergy prediction</a></td>
          <td>
            
              Xueliang Li,
            
              Bihan Shen,
            
              Fangyoumin Feng,
            
              Kun Li,
            
              Hong Li,
            
          </td>
          <td>2024-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7402548044cfd2eea8e12903af5a1d7c8ddd61a1" target='_blank'>Stochastic simulation of Boolean rxncon models: towards quantitative analysis of large signaling networks</a></td>
          <td>
            
              Tomoya Mori,
            
              Max Flöttmann,
            
              Marcus Krantz,
            
              T. Akutsu,
            
              E. Klipp,
            
          </td>
          <td>2015-08-11</td>
          <td>BMC Systems Biology</td>
          <td>20</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4fa5f172b476e7f0de6bac14ca3fd21980ddb5e6" target='_blank'>Computational Model of Heterogeneity in Melanoma: Designing Therapies and Predicting Outcomes</a></td>
          <td>
            
              A. Hodgkinson,
            
              D. Trucu,
            
              M. Lacroix,
            
              L. Le Cam,
            
              O. Radulescu,
            
          </td>
          <td>2022-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>6</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cfb747db00c6e98936d0749d962f04b796ea7036" target='_blank'>Systems analysis of apoptosis protein expression allows the case-specific prediction of cell death responsiveness of melanoma cells</a></td>
          <td>
            
              Egle Passante,
            
              Maximilian L. Würstle,
            
              C. Hellwig,
            
              Martin Leverkus,
            
              M. Rehm,
            
          </td>
          <td>2013-08-09</td>
          <td>Cell Death and Differentiation</td>
          <td>35</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/51c089fef5e76e99ca2fcdf036d63de7ca809cbb" target='_blank'>A reverse-engineering approach to dissect post-translational modulators of transcription factor’s activity from transcriptional data</a></td>
          <td>
            
              G. Gambardella,
            
              I. Peluso,
            
              S. Montefusco,
            
              M. Bansal,
            
              D. Medina,
            
              Neil D. Lawrence,
            
              D. Bernardo,
            
          </td>
          <td>2015-09-03</td>
          <td>BMC Bioinformatics</td>
          <td>7</td>
          <td>68</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7cd32e8383dc06d79ebf6494e88ef8eaf5c1e23b" target='_blank'>TranSiGen: Deep representation learning of chemical-induced transcriptional profile</a></td>
          <td>
            
              X. Tong,
            
              Ning Qu,
            
              Xiangtai Kong,
            
              Shengkun Ni,
            
              Kun Wang,
            
              Lehan Zhang,
            
              Yiming Wen,
            
              Sulin Zhang,
            
              Xutong Li,
            
              Mingyue Zheng,
            
          </td>
          <td>2023-11-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/316a011bf461d3a96965fb9f69398888da19bd9f" target='_blank'>SynerGPT: In-Context Learning for Personalized Drug Synergy Prediction and Drug Design</a></td>
          <td>
            
              Carl N. Edwards,
            
              Aakanksha Naik,
            
              Tushar Khot,
            
              Martin Burke,
            
              Heng Ji,
            
              Tom Hope,
            
          </td>
          <td>2023-06-19</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f87189c3da0abe61e47ccc6636e1effaa403de02" target='_blank'>DREAM4: Combining Genetic and Dynamic Information to Identify Biological Networks and Dynamical Models</a></td>
          <td>
            
              Alex Greenfield,
            
              Aviv Madar,
            
              H. Ostrer,
            
              Richard Bonneau,
            
          </td>
          <td>2010-10-25</td>
          <td>PLoS ONE</td>
          <td>240</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/10f3e77018212b3c146ee1a170b8e8d76d91b035" target='_blank'>Maximum Entropy Reconstructions of Dynamic Signaling Networks from Quantitative Proteomics Data</a></td>
          <td>
            
              J. Locasale,
            
              Alejandro Wolf-Yadlin,
            
          </td>
          <td>2009-08-26</td>
          <td>PLoS ONE</td>
          <td>41</td>
          <td>82</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/96645e644150fae6be9b711f54a840afd7aee085" target='_blank'>A Non-Negative Matrix Tri-Factorization based Method for Predicting Antitumor Drug Sensitivity</a></td>
          <td>
            
              C. Testa,
            
              S. Pidó,
            
              Pietro Pinoli,
            
          </td>
          <td>2021-12-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>19</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8d6497b1c356f45ca04b87838b5d6abc132c7ae1" target='_blank'>Preclinical validation of therapeutic targets predicted by tensor factorization on heterogeneous graphs</a></td>
          <td>
            
              Saee Paliwal,
            
              A. de Giorgio,
            
              Daniel Neil,
            
              Jean-Baptiste Michel,
            
              A. Lacoste,
            
          </td>
          <td>2020-10-26</td>
          <td>Scientific Reports</td>
          <td>33</td>
          <td>22</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f68bf910c14842cad3e3c6d050b984bba0d0d1fb" target='_blank'>Highly Scalable Tensor Factorization for Prediction of Drug-Protein Interaction Type</a></td>
          <td>
            
              Adam Arany,
            
              J. Simm,
            
              Pooya Zakeri,
            
              Tom Haber,
            
              J. Wegner,
            
              V. Chupakhin,
            
              H. Ceulemans,
            
              Y. Moreau,
            
          </td>
          <td>2015-12-01</td>
          <td>arXiv: Machine Learning</td>
          <td>6</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e17ad225cc4b23bf041bc34d3d97d392a71584d7" target='_blank'>Predicting which genes will respond to transcription factor perturbations</a></td>
          <td>
            
              Yiming Kang,
            
              Wooseok Jung,
            
              M. Brent,
            
          </td>
          <td>2022-06-06</td>
          <td>G3: Genes|Genomes|Genetics</td>
          <td>1</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2f9169a9604184a634feb6d610ab87d2a22ab5bf" target='_blank'>Training set expansion: an approach to improving the reconstruction of biological networks from limited and uneven reliable interactions</a></td>
          <td>
            
              Kevin Y. Yip,
            
              M. Gerstein,
            
          </td>
          <td>2008-11-17</td>
          <td>Bioinformatics</td>
          <td>34</td>
          <td>165</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/292253cc2a62f478cbfcc9c9ac6014bca828ca7a" target='_blank'>DeLHCA: Deep transfer learning for high-content analysis of the effects of drugs on immune cells</a></td>
          <td>
            
              Shaista Hussain,
            
              A. Das,
            
              Binh P. Nguyen,
            
              M. Marzuki,
            
              Shuping Lin,
            
              Arun Kumar,
            
              Graham Wright,
            
              Amit Singhal,
            
          </td>
          <td>2019-10-01</td>
          <td>TENCON 2019 - 2019 IEEE Region 10 Conference (TENCON)</td>
          <td>3</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2efbc9a6bb312a00216c760c520359dd6b7773b7" target='_blank'>A Bayesian approach to estimating hidden variables as well as missing and wrong molecular interactions in ordinary differential equation-based mathematical models</a></td>
          <td>
            
              B. Engelhardt,
            
              Maik Kschischo,
            
              H. Fröhlich,
            
          </td>
          <td>2017-06-01</td>
          <td>Journal of the Royal Society Interface</td>
          <td>16</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0d28b2c4ab559ba9d75a767d948e96abdc7643e1" target='_blank'>Signaling pathway models as biomarkers: Patient-specific simulations of JNK activity predict the survival of neuroblastoma patients</a></td>
          <td>
            
              D. Fey,
            
              M. Halasz,
            
              D. Dreidax,
            
              S. Kennedy,
            
              Jordan F Hastings,
            
              Nora Rauch,
            
              Amaya Garcia Munoz,
            
              R. Pilkington,
            
              M. Fischer,
            
              F. Westermann,
            
              W. Kolch,
            
              B. Kholodenko,
            
              David R. Croucher,
            
          </td>
          <td>2015-12-22</td>
          <td>Science Signaling</td>
          <td>136</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7b0a6f80943c7ba8bfe4a2397b0723fcba0569ed" target='_blank'>Linking Proteins to Signaling Pathways for Experiment Design and Evaluation</a></td>
          <td>
            
              I. Farkas,
            
              Adam Szanto-Varnagy,
            
              T. Korcsmáros,
            
          </td>
          <td>2012-04-27</td>
          <td>PLoS ONE</td>
          <td>28</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/db6c3adbb159002b81871b91aab816c09010f3c3" target='_blank'>Maximal entropy inference of oncogenicity from phosphorylation signaling</a></td>
          <td>
            
              T. Graeber,
            
              James R. Heath,
            
              B. Skaggs,
            
              M. Phelps,
            
              F. Remacle,
            
              R. Levine,
            
          </td>
          <td>2010-03-11</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>16</td>
          <td>70</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/23421c6b6b1ab55e6b5369e83167832050578a40" target='_blank'>Building predictive signaling models by perturbing yeast cells with time-varying stimulations resulting in distinct signaling responses</a></td>
          <td>
            
              Hossein Jashnsaz,
            
              Z. Fox,
            
              B. Munsky,
            
              G. Neuert,
            
          </td>
          <td>2021-07-07</td>
          <td>STAR Protocols</td>
          <td>2</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e67af7add8f12d3f49a425a63da0662117fd02f0" target='_blank'>Benchmarking causal reasoning algorithms for gene expression-based compound mechanism of action analysis</a></td>
          <td>
            
              Layla Hosseini-Gerami,
            
              I. Higgins,
            
              D. A. Collier,
            
              Emma Laing,
            
              David Evans,
            
              Howard Broughton,
            
              A. Bender,
            
          </td>
          <td>2022-01-19</td>
          <td>BMC Bioinformatics</td>
          <td>7</td>
          <td>59</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/03c120e239a6f65d88045fcdde4e6fcfe9d4bdaf" target='_blank'>GeneSPIDER - gene regulatory network inference benchmarking with controlled network and data properties.</a></td>
          <td>
            
              Andreas Tjärnberg,
            
              D. Morgan,
            
              Matthew Studham,
            
              Torbjörn E. M. Nordling,
            
              E. Sonnhammer,
            
          </td>
          <td>2017-06-27</td>
          <td>Molecular bioSystems</td>
          <td>14</td>
          <td>63</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/26baf2bf994309f86ecf02b323b16062447b5437" target='_blank'>Gene regulatory network inference and validation using relative change ratio analysis and time-delayed dynamic Bayesian network</a></td>
          <td>
            
              Pengcheng Li,
            
              P. Gong,
            
              Haoni Li,
            
              E. Perkins,
            
              Nan Wang,
            
              Chaoyang Zhang,
            
          </td>
          <td>2014-07-16</td>
          <td>EURASIP Journal on Bioinformatics and Systems Biology</td>
          <td>14</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/98aa7b3a5a4ce5bcf140437c86670cdfee8dec2b" target='_blank'>A novel method of using Deep Belief Networks and genetic perturbation data to search for yeast signaling pathways</a></td>
          <td>
            
              Songjian Lu,
            
              Xiaonan Fan,
            
              Lujia Chen,
            
              Xinghua Lu,
            
          </td>
          <td>2018-09-12</td>
          <td>PLoS ONE</td>
          <td>4</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8d6d144f935235fb1c886ec43fbefcf71bbfda38" target='_blank'>PathwayMap: Molecular Pathway Association with Self-Normalizing Neural Networks</a></td>
          <td>
            
              J. Jiménez,
            
              Davide Sabbadin,
            
              Alberto Cuzzolin,
            
              Gerard Martínez-Rosell,
            
              Jacob Gora,
            
              J. Manchester,
            
              J. Duca,
            
              G. D. Fabritiis,
            
          </td>
          <td>2018-12-26</td>
          <td>Journal of chemical information and modeling</td>
          <td>21</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/07c2d1398fb6f66830cd9eea7223f5cc903715f0" target='_blank'>Learning Signaling Network Structures with Sparsely Distributed Data</a></td>
          <td>
            
              K. Sachs,
            
              S. Itani,
            
              Jennifer N. Carlisle,
            
              G. Nolan,
            
              D. Pe’er,
            
              D. Lauffenburger,
            
          </td>
          <td>2009-02-01</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>31</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58e533bc46cb27eb3860281a30f7c22f912426aa" target='_blank'>Genetic function approximation in the molecular pharmacology of cancer</a></td>
          <td>
            
              Leming Shi,
            
              Yi Fan,
            
              T. Myers,
            
              J. Weinstein,
            
          </td>
          <td>1997-06-09</td>
          <td>Proceedings of International Conference on Neural Networks (ICNN'97)</td>
          <td>3</td>
          <td>122</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9293a7e4b49fa3537016e6e305dd9adf3a971530" target='_blank'>secDrug: a pipeline to discover novel drug combinations to kill drug-resistant multiple myeloma cells using a greedy set cover algorithm and single-cell multi-omics</a></td>
          <td>
            
              H. Kumar,
            
              Suman Mazumder,
            
              Sayak Chakravarti,
            
              Neeraj Sharma,
            
              U. Mukherjee,
            
              Shaji K. Kumar,
            
              L. Baughn,
            
              B. V. Van Ness,
            
              A. Mitra,
            
          </td>
          <td>2022-03-01</td>
          <td>Blood Cancer Journal</td>
          <td>5</td>
          <td>117</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/95f75baf3f1cb48108d2823a2289c2c328109163" target='_blank'>DOSE-L1000: unveiling the intricate landscape of compound-induced transcriptional changes</a></td>
          <td>
            
              Junmin Wang,
            
              Steven Novick,
            
          </td>
          <td>2023-11-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/82729dc0ce591f4c6b7f084bab932f14b419f2a2" target='_blank'>Individualized Prediction of Drug Response and Rational Combination Therapy in NSCLC Using Artificial Intelligence–Enabled Studies of Acute Phosphoproteomic Changes</a></td>
          <td>
            
              E. Coker,
            
              A. Stewart,
            
              Bugra Ozer,
            
              A. Minchom,
            
              L. Pickard,
            
              R. Ruddle,
            
              S. Carreira,
            
              S. Popat,
            
              M. O'Brien,
            
              F. Raynaud,
            
              J. D. de Bono,
            
              B. Al-Lazikani,
            
              U. Banerji,
            
          </td>
          <td>2022-04-03</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>4</td>
          <td>116</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2a2a44bcb591be61a291dd0078ded7e59697a008" target='_blank'>Bioinformatics and Biology Insights</a></td>
          <td>
            
              V. Belcastro,
            
              C. Poussin,
            
              S. Gebel,
            
              C. Mathis,
            
              W. Schlage,
            
              R. Lichtner,
            
              Sibille Quadt-Humme,
            
              Sandra Wagner,
            
              J. Hoeng,
            
              M. Peitsch,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>12</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/aa76568358955b754566591bd497ed51a8b52daa" target='_blank'>Reconstructing gene-regulatory networks from time series, knock-out data, and prior knowledge</a></td>
          <td>
            
              Florian Geier,
            
              Jens Timmer,
            
              Christian Fleck,
            
          </td>
          <td>2007-02-02</td>
          <td>BMC Systems Biology</td>
          <td>1</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/608fce1fa3ea3695fec9b3c5ae77d667820f1371" target='_blank'>Reverse-engineering biological networks from large data sets</a></td>
          <td>
            
              Joseph L. Natale,
            
              David Hofmann,
            
              Dami'an G. Hern'andez,
            
              I. Nemenman,
            
          </td>
          <td>2017-05-17</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>30</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7235c84c21cc7ef502f013a4fa1e9116d47083e8" target='_blank'>Learning from natural variation across the proteomes of single cells</a></td>
          <td>
            
              N. Slavov,
            
          </td>
          <td>2022-01-01</td>
          <td>PLoS Biology</td>
          <td>20</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/4ac8c553765495313df0c13a9c57e5f618166194" target='_blank'>Predicting clinically promising therapeutic hypotheses using tensor factorization</a></td>
          <td>
            
              Jin Yao,
            
              M. Hurle,
            
              M. Nelson,
            
              Pankaj Agarwal,
            
          </td>
          <td>2018-02-27</td>
          <td>BMC Bioinformatics</td>
          <td>10</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0d07275ac874372ec404b4e187eb769690239595" target='_blank'>A thermodynamic-based approach for the resolution and prediction of protein network structures</a></td>
          <td>
            
              E. Flashner-Abramson,
            
              Jonathan Abramson,
            
              F. White,
            
              N. Kravchenko‐Balasha,
            
          </td>
          <td>2018-10-01</td>
          <td>Chemical Physics</td>
          <td>9</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c65cc3ce601427b7fa9002f4e84d8dd437052412" target='_blank'>Crosstalk between Plk1, p53, cell cycle, and G2/M DNA damage checkpoint regulation in cancer: computational modeling and analysis</a></td>
          <td>
            
              Yongwoon Jung,
            
              P. Kraikivski,
            
              Sajad Shafiekhani,
            
              S. Terhune,
            
              R. Dash,
            
          </td>
          <td>2021-12-01</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>22</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c65cc3ce601427b7fa9002f4e84d8dd437052412" target='_blank'>Crosstalk between Plk1, p53, cell cycle, and G2/M DNA damage checkpoint regulation in cancer: computational modeling and analysis</a></td>
          <td>
            
              Yongwoon Jung,
            
              P. Kraikivski,
            
              Sajad Shafiekhani,
            
              S. Terhune,
            
              R. Dash,
            
          </td>
          <td>2021-12-01</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>22</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ec3701819cb1bb58b5228689d3de395a884da64d" target='_blank'>Protocol for stratification of triple-negative breast cancer patients using in silico signaling dynamics</a></td>
          <td>
            
              Hiroaki Imoto,
            
              Sawa Yamashiro,
            
              Ken Murakami,
            
              M. Okada,
            
          </td>
          <td>2022-08-11</td>
          <td>STAR Protocols</td>
          <td>0</td>
          <td>5</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c1e984475d72d8cb2ab92a476b5b453a79cb3b4e" target='_blank'>The Unreasonable Effectiveness of Inverse Reinforcement Learning in Advancing Cancer Research</a></td>
          <td>
            
              John Kalantari,
            
              Heidi Nelson,
            
              N. Chia,
            
          </td>
          <td>2020-04-03</td>
          <td>Proceedings of the ... AAAI Conference on Artificial Intelligence. AAAI Conference on Artificial Intelligence</td>
          <td>9</td>
          <td>43</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/58472b88e77316e307943e685836dca035a39667" target='_blank'>Abstract 43: Using matrix factorization to discover clinically-relevant molecular signatures across cancers</a></td>
          <td>
            
              C. Szeto,
            
              Jingchun Zhu,
            
              James Durbin,
            
              S. Benz,
            
              Charles J. Vaske,
            
              Joshua M. Stuart,
            
              D. Haussler,
            
          </td>
          <td>2011-04-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>166</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9a334f793ddcbd085f65e433da982a821a1463a4" target='_blank'>Attractor estimation and model refinement for stochastic regulatory network models</a></td>
          <td>
            
              J. Knight,
            
              E. Dougherty,
            
          </td>
          <td>2011-12-01</td>
          <td>2011 IEEE International Workshop on Genomic Signal Processing and Statistics (GENSIPS)</td>
          <td>2</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/94aef05cf8ef71ddfd9b8af79c809ba929f83728" target='_blank'>Predictive power of non-identifiable models</a></td>
          <td>
            
              F. Grabowski,
            
              Paweł Nałęcz-Jawecki,
            
              T. Lipniacki,
            
          </td>
          <td>2023-04-08</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>31</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fa6264da924a38d9ff4915eb1f83a484e130ba9e" target='_blank'>Information theoretic approach to complex biological network reconstruction: application to cytokine release in RAW 264.7 macrophages</a></td>
          <td>
            
              Farzaneh Farhangmehr,
            
              M. Maurya,
            
              D. Tartakovsky,
            
              S. Subramaniam,
            
          </td>
          <td>2014-06-25</td>
          <td>BMC Systems Biology</td>
          <td>10</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6f4982921378abd2a762437540520df844791e0e" target='_blank'>Improving dynamic predictions with ensembles of observable models</a></td>
          <td>
            
              Gemma Massonis,
            
              A. F. Villaverde,
            
              J. Banga,
            
          </td>
          <td>2022-11-23</td>
          <td>Bioinformatics</td>
          <td>2</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8f016111dce3382a42e40b4219c4a3fb1be6df6a" target='_blank'>A statistical framework for differential pseudotime analysis with multiple single-cell RNA-seq samples</a></td>
          <td>
            
              Wenpin Hou,
            
              Zhicheng Ji,
            
              Zeyu Chen,
            
              E. Wherry,
            
              S. Hicks,
            
              Hongkai Ji,
            
          </td>
          <td>2021-07-12</td>
          <td>bioRxiv</td>
          <td>13</td>
          <td>106</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/05e4382614c79b87689f82e88e59e9b432d75dd6" target='_blank'>A context-aware deconfounding autoencoder for robust prediction of personalized clinical drug response from cell-line compound screening</a></td>
          <td>
            
              Di He,
            
              Qiao Liu,
            
              You Wu,
            
              Linfu Xie,
            
          </td>
          <td>2022-10-01</td>
          <td>Nature Machine Intelligence</td>
          <td>24</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5cea1d1cc69421bf77d63ea8722c808f0a20b2af" target='_blank'>Estimating drivers of cell state transitions using gene regulatory network models</a></td>
          <td>
            
              D. Schlauch,
            
              K. Glass,
            
              C. Hersh,
            
              E. Silverman,
            
              John Quackenbush,
            
          </td>
          <td>2016-11-21</td>
          <td>BMC Systems Biology</td>
          <td>17</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1e5e2217281dfc3a4ad10a124188ade9cfd1dccb" target='_blank'>BGRMI: A method for inferring gene regulatory networks from time-course gene expression data and its application in breast cancer research</a></td>
          <td>
            
              Luis F. Iglesias-Martinez,
            
              W. Kolch,
            
              T. Santra,
            
          </td>
          <td>2016-11-23</td>
          <td>Scientific Reports</td>
          <td>33</td>
          <td>86</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2987734b087361094a90c596c3b745c066bf057a" target='_blank'>Learning to Rank Drug Combinations</a></td>
          <td>
            
              Fernando Lozano,
            
          </td>
          <td>2013-12-04</td>
          <td>2013 12th International Conference on Machine Learning and Applications</td>
          <td>0</td>
          <td>7</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/72f0e88f83171cd1673ec0a9e3f3d25dd207eff8" target='_blank'>From Cell-Lines to Cancer Patients: Personalized Drug Synergy Prediction</a></td>
          <td>
            
              Halil Ibrahim Kuru,
            
              A. E. Cicek,
            
              Oznur Tastan,
            
          </td>
          <td>2023-02-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5054a527a10b38865ad0c66e3cfafe4c1b5e102e" target='_blank'>Modeling the influence of disease processes on gene expression</a></td>
          <td>
            
              K. Shedden,
            
          </td>
          <td>None</td>
          <td>Statistica Sinica</td>
          <td>1</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b61fec2f03e1ab50ff0bdea63575271f41b4598b" target='_blank'>COMBImage2: a parallel computational framework for higher-order drug combination analysis that includes automated plate design, matched filter based object counting and temporal data mining</a></td>
          <td>
            
              E. Chantzi,
            
              M. Jarvius,
            
              Mia Niklasson,
            
              A. Segerman,
            
              M. Gustafsson,
            
          </td>
          <td>2019-06-04</td>
          <td>BMC Bioinformatics</td>
          <td>2</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a38f3abbe9ec3848fbc68d6044409ab65cb30cc8" target='_blank'>PLATYPUS: A Multiple–View Learning Predictive Framework for Cancer Drug Sensitivity Prediction</a></td>
          <td>
            
              Kiley Graim,
            
              V. Friedl,
            
              Kathleen E. Houlahan,
            
              Joshua M. Stuart,
            
          </td>
          <td>2018-11-01</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>12</td>
          <td>73</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/dec3238935adec4d68c1f9570f1180d2b453847c" target='_blank'>Petri Net Siphon Analysis and Graph Theoretic Measures for Identifying Combination Therapies in Cancer</a></td>
          <td>
            
              Behnam Behinaein,
            
              K. Rudie,
            
              W. Sangrar,
            
          </td>
          <td>None</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>10</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/129430c4fc1d5ededdc77aa30b3b58c596af37e9" target='_blank'>sbv IMPROVER Diagnostic Signature Challenge</a></td>
          <td>
            
              J. Hoeng,
            
              G. Stolovitzky,
            
              M. Peitsch,
            
          </td>
          <td>2013-09-12</td>
          <td>Systems Biomedicine</td>
          <td>1</td>
          <td>61</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1b3be66ef0e461c8915807a6ce667b1c9afc2fc8" target='_blank'>DrugCom: Synergistic Discovery of Drug Combinations Using Tensor Decomposition</a></td>
          <td>
            
              Huiyuan Chen,
            
              Jing Li,
            
          </td>
          <td>2018-11-01</td>
          <td>2018 IEEE International Conference on Data Mining (ICDM)</td>
          <td>24</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/10f45b05c91991583f981ef2a2c70081b88f6ac5" target='_blank'>Correction: Low-Rank Regularization for Learning Gene Expression Programs</a></td>
          <td>
            
              Gui-Bo Ye,
            
              Mengfan Tang,
            
              Jian-Feng Cai,
            
              Q. Nie,
            
              Xiaohui S. Xie,
            
          </td>
          <td>2014-01-13</td>
          <td>PLoS ONE</td>
          <td>2</td>
          <td>48</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ad08323c81d6964da14c88c71ebe99a8da215cd4" target='_blank'>Modeling and interpretation of single-cell proteogenomic data</a></td>
          <td>
            
              Andrew Leduc,
            
              H. Harens,
            
              N. Slavov,
            
          </td>
          <td>2023-08-14</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/ee807ca47062c79f73f6fe79dd21932be70ec75a" target='_blank'>Explaining Black Box Drug Target Prediction Through Model Agnostic Counterfactual Samples</a></td>
          <td>
            
              T. Nguyen,
            
              Thomas P. Quinn,
            
              Thin Nguyen,
            
              T. Tran,
            
          </td>
          <td>2022-07-12</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>4</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d4848673c95383ddc7398b913e8ebc7a19e6b701" target='_blank'>Adversarial Controls for Scientific Machine Learning.</a></td>
          <td>
            
              Kangway V Chuang,
            
              Michael J. Keiser,
            
          </td>
          <td>2018-10-19</td>
          <td>ACS chemical biology</td>
          <td>44</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2c76dbda3fdb9c10c523ff05f788d49d77855aff" target='_blank'>Predicting Mechanisms of Ibrutinib-Resistance in Chronic Lymphocytic Leukemia (CLL) Using Computational-Based Modeling for Personalized Therapy with Clinical Validation</a></td>
          <td>
            
              L. Drusbosky,
            
              A. Paulus,
            
              A. Mary,
            
              L. Ysebaert,
            
              Ansu Kumar,
            
              N. Singh,
            
              Y. Ullal,
            
              P. Bhargav,
            
              H. Azam,
            
              T. Abbasi,
            
              S. Vali,
            
              A. Chanan-Khan,
            
              C. Cogle,
            
          </td>
          <td>2017-12-07</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/5567d65ba9d6430c90454c3526fd06da07ef854d" target='_blank'>Self-Supervised Representation Learning for High-Content Screening</a></td>
          <td>
            
              Daniel Siegismund,
            
              Mario Wieser,
            
              S. Heyse,
            
              Stephan Steigele,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>5</td>
          <td>9</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b806b69552b8e55be99d4f58d0f45d134b4f1078" target='_blank'>Abstract 4911: PAIRWISE: Deep learning-powered personalized medicine approach identifies actionable drug combinations in B cell lymphoma</a></td>
          <td>
            
              Chengqi Xu,
            
              Jake Cohen-Setton,
            
              Zhouyang Shen,
            
              Marta Milo,
            
              Ilkay Us,
            
              Ben Sidders,
            
              Jude Fitzgibbon,
            
              Heng Pan,
            
              Olivier Elemento,
            
              Krishna Bulusu,
            
          </td>
          <td>2024-03-22</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e3065ce462567c5178ed19eec15aee577faf4645" target='_blank'>Optimizing drug combinations against multiple myeloma using a quadratic phenotypic optimization platform (QPOP)</a></td>
          <td>
            
              M. Rashid,
            
              T. Toh,
            
              L. Hooi,
            
              Aleidy Silva,
            
              Yanzhou Zhang,
            
              Pei Fang Tan,
            
              A. Teh,
            
              N. Karnani,
            
              S. Jha,
            
              Chih-Ming Ho,
            
              W. Chng,
            
              D. Ho,
            
              E. Chow,
            
          </td>
          <td>2018-08-08</td>
          <td>Science Translational Medicine</td>
          <td>73</td>
          <td>71</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a07d5c391636b8179b61e9fcdf65779b964bc31e" target='_blank'>Predicting Drug Synergism by Means of Non-Negative Matrix Tri-Factorization</a></td>
          <td>
            
              Pietro Pinoli,
            
              G. Ceddia,
            
              S. Ceri,
            
              M. Masseroli,
            
          </td>
          <td>2021-06-24</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>5</td>
          <td>69</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cb81d359a8f4a00ca2593d46d29500e186f29312" target='_blank'>Abstract 4921: Dynamic and quantitative modeling of drug effect</a></td>
          <td>
            
              Xiangfang Li,
            
              Lijun Qian,
            
              M. Bittner,
            
              E. Dougherty,
            
          </td>
          <td>2011-04-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e81e43513dc07a2d462d1f7c37a5de8d926302aa" target='_blank'>A Functional Network Model of the Metastasis Suppressor PEBP1/RKIP and Its Regulators in Breast Cancer Cells</a></td>
          <td>
            
              Mahmoud Ahmed,
            
              T. Lai,
            
              W. Kim,
            
              D. Kim,
            
          </td>
          <td>2021-12-01</td>
          <td>Cancers</td>
          <td>7</td>
          <td>26</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/872a12398a4a49e27d3e86c1b0860be0f29bf8ef" target='_blank'>Elucidation of Signaling Pathways from Large-Scale Phosphoproteomic Data Using Protein Interaction Networks.</a></td>
          <td>
            
              J. Rudolph,
            
              Marjo de Graauw,
            
              B. van de Water,
            
              T. Geiger,
            
              R. Sharan,
            
          </td>
          <td>2016-12-21</td>
          <td>Cell systems</td>
          <td>50</td>
          <td>68</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/bca2bc97f27a97dbf1673a7c1e43fdd16cdebe28" target='_blank'>Inter-species pathway perturbation prediction via data-driven detection of functional homology</a></td>
          <td>
            
              Christoph Hafemeister,
            
              R. Romero,
            
              E. Bilal,
            
              Pablo Meyer,
            
              R. Norel,
            
              K. Rhrissorrakrai,
            
              Richard Bonneau,
            
              A. Tarca,
            
          </td>
          <td>2015-02-15</td>
          <td>Bioinformatics</td>
          <td>7</td>
          <td>132</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/95fa687c12d553623a68384a6b4f9eaa1486eed5" target='_blank'>Real-Time Gene Expression: Statistical Challenges in Design and Inference</a></td>
          <td>
            
              David L. Gold,
            
              B. Mallick,
            
              K. Coombes,
            
          </td>
          <td>2008-07-16</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>3</td>
          <td>66</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3d7dfa3edccebddf8c1d7ac000eabbd23ad5159a" target='_blank'>Inferring Gene Networks: Dream or Nightmare?</a></td>
          <td>
            
              A. Scheinine,
            
              Wieslawa I. Mentzen,
            
              G. Fotia,
            
              E. Pieroni,
            
              F. Maggio,
            
              G. Mancosu,
            
              A. G. de la Fuente,
            
          </td>
          <td>2009-03-01</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>14</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3da0b4158b95958be1508bba5a04232a4474d61c" target='_blank'>Simple Matrix Factorization Collaborative Filtering for Drug Repositioning on Cell Lines</a></td>
          <td>
            
              Iván Carrera,
            
              E. Tejera,
            
              I. Dutra,
            
          </td>
          <td>None</td>
          <td>None</td>
          <td>3</td>
          <td>20</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/47fa48dbecb71cf2078e7c6593bfe1be8b779f54" target='_blank'>Doubly Penalized LASSO for Reconstruction of Biological Networks</a></td>
          <td>
            
              Behrang Asadi,
            
              D. Tartakovsky,
            
              M. Maurya,
            
              S. Subramaniam,
            
          </td>
          <td>2012-09-27</td>
          <td>Proceedings of the IEEE</td>
          <td>0</td>
          <td>64</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d6856c45e2a53a3e100af6398c1d6cc7a06fe41f" target='_blank'>Identifying Determinants of EGFR-Targeted Therapeutic Biochemical Efficacy Using Computational Modeling</a></td>
          <td>
            
              C S Monast,
            
              M J Lazzara,
            
          </td>
          <td>2014-10-01</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>2</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c3cbcb74e316f58c735a4f2d7bf778ff980a09b7" target='_blank'>Temporal Dynamic Methods for Bulk RNA-Seq Time Series Data</a></td>
          <td>
            
              Vera-Khlara S. Oh,
            
              Robert W. Li,
            
          </td>
          <td>2021-02-27</td>
          <td>Genes</td>
          <td>12</td>
          <td>37</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/046916175e52079504e9003aaa828e629ea62fc1" target='_blank'>Generalized analysis of experimental data for interrelated biological measurements</a></td>
          <td>
            
              Z. Kam,
            
          </td>
          <td>None</td>
          <td>Bulletin of Mathematical Biology</td>
          <td>5</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f79400a684849dd52288b2c8b908c411b74d9279" target='_blank'>Dynamic modeling of the cellular senescence gene regulatory network</a></td>
          <td>
            
              José Américo Nabuco Leva Ferreira Freitas,
            
              O. Bischof,
            
          </td>
          <td>2023-02-01</td>
          <td>Heliyon</td>
          <td>0</td>
          <td>24</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d9b9460fbf943826cad599c25ba40395a39e91d7" target='_blank'>Reverse engineering of logic-based differential equation models using a mixed-integer dynamic optimization approach</a></td>
          <td>
            
              David Henriques,
            
              Miguel Rocha,
            
              J. Sáez-Rodríguez,
            
              J. Banga,
            
          </td>
          <td>2015-05-21</td>
          <td>Bioinformatics</td>
          <td>25</td>
          <td>53</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e75a57803f2ae154eb02c2ff494ab2bb030d7052" target='_blank'>Generative Deep Learning of the Single Cell Dose Response of 2,3,7,8 Tetrachlorodibenzo‐p‐dioxin in Mouse Liver</a></td>
          <td>
            
              Omar Kana,
            
              R. Nault,
            
              T. Zacharewski,
            
              S. Bhattacharya,
            
          </td>
          <td>2022-05-01</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>57</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d911eba195fe26e44f418c5dc8b6f64546679ccd" target='_blank'>Inference of Genetic Networks from Time Course Expression Data Using Functional Regression with Lasso Penalty</a></td>
          <td>
            
              Zhaoping Hong,
            
              H. Lian,
            
          </td>
          <td>2008-04-04</td>
          <td>Communications in Statistics - Theory and Methods</td>
          <td>16</td>
          <td>28</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/9109f271d65db8a2cad0227dd1b26c95fa31d0bd" target='_blank'>Analysis and design of single-cell experiments to harvest fluctuation information while rejecting measurement noise</a></td>
          <td>
            
              Huy D. Vo,
            
              Linda Forero,
            
              Luis U. Aguilera,
            
              B. Munsky,
            
          </td>
          <td>2021-05-12</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>2</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/f145d87d9d6f8905892d8a538b0914aad1364c9c" target='_blank'>Dynamical structure function identifiability conditions enabling signal structure reconstruction</a></td>
          <td>
            
              J. Adebayo,
            
              T. Southwick,
            
              V. Chetty,
            
              Enoch Yeung,
            
              Ye Yuan,
            
              J. Gonçalves,
            
              J. Grose,
            
              J. Prince,
            
              G. Stan,
            
              S. Warnick,
            
          </td>
          <td>2012-12-01</td>
          <td>2012 IEEE 51st IEEE Conference on Decision and Control (CDC)</td>
          <td>54</td>
          <td>35</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/8cef4bd7e0883de1974515ff8fff9f19037fb862" target='_blank'>Network Reconstruction From High-Dimensional Ordinary Differential Equations</a></td>
          <td>
            
              Shizhe Chen,
            
              A. Shojaie,
            
              D. Witten,
            
          </td>
          <td>2016-10-11</td>
          <td>Journal of the American Statistical Association</td>
          <td>60</td>
          <td>44</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/12a69807b3a2aefe973338898cfcccbe31582d35" target='_blank'>Analysing double-strand breaks in cultured cells for drug screening applications by causal inference</a></td>
          <td>
            
              Joseph Boyd,
            
              Alice Pinhiero,
            
              E. D. Nery,
            
              F. Reyal,
            
              Thomas Walter,
            
          </td>
          <td>2018-04-01</td>
          <td>2018 IEEE 15th International Symposium on Biomedical Imaging (ISBI 2018)</td>
          <td>2</td>
          <td>47</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/3846028fab6315eeddb76438dc67b0ab80a88837" target='_blank'>Impulse model-based differential expression analysis of time course sequencing data</a></td>
          <td>
            
              David S. Fischer,
            
              Fabian J Theis,
            
              N. Yosef,
            
          </td>
          <td>2017-03-05</td>
          <td>Nucleic Acids Research</td>
          <td>100</td>
          <td>94</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/49604acc03fff396a4c03f8b5f9c48237fb0c4bb" target='_blank'>MoCHI: neural networks to fit interpretable models and quantify energies, energetic couplings, epistasis and allostery from deep mutational scanning data</a></td>
          <td>
            
              André J. Faure,
            
              Ben Lehner,
            
          </td>
          <td>2024-01-28</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>13</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c7547062acb05cd8616b5479e7a08725ed55eead" target='_blank'>Semi-supervised network inference with time course data and dynamic regression simulation</a></td>
          <td>
            
              P. Nguyen,
            
              R. Braun,
            
          </td>
          <td>2017-06-03</td>
          <td></td>
          <td>0</td>
          <td>12</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/da4c745d005bfaeff422b267c1fbf6226f153ae4" target='_blank'>[Prediction of drug-induced cell viability by SAE-XGBoost algorithm based on LINCS-L1000 perturbation signal].</a></td>
          <td>
            
              Jiaxing Lu,
            
              Ming Chen,
            
              Yufang Qin,
            
              Xiaoqing Yu,
            
          </td>
          <td>2021-04-25</td>
          <td>Sheng wu gong cheng xue bao = Chinese journal of biotechnology</td>
          <td>2</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/0dd77ad0509f3eb7ee1d9295221205798596e8de" target='_blank'>Gene Regulatory Networks: Improving Inferences with Transfer Learning.</a></td>
          <td>
            
              Marcelo Tigre Moura,
            
          </td>
          <td>2023-12-01</td>
          <td>Cellular reprogramming</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/404c801399a8106acc14b1d13526fe4be3446050" target='_blank'>Design principles to assemble drug combinations for effective tuberculosis therapy using interpretable pairwise drug response measurements</a></td>
          <td>
            
              Jonah Larkins-Ford,
            
              Yonatan N. Degefu,
            
              Nhi Van,
            
              Artem Sokolov,
            
              B. Aldridge,
            
          </td>
          <td>2021-12-06</td>
          <td>Cell Reports Medicine</td>
          <td>20</td>
          <td>25</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/78ede1ad9948226d04b3e88663fd237c0439e10b" target='_blank'>DRUG-seq Provides Unbiased Biological Activity Readouts for Neuroscience Drug Discovery</a></td>
          <td>
            
              Jingyao Li,
            
              Daniel J. Ho,
            
              Martin Hénault,
            
              Chian Yang,
            
              M. Neri,
            
              Robin Ge,
            
              S. Renner,
            
              Leandra Mansur,
            
              Alicia Lindeman,
            
              Brian Kelly,
            
              Tayfun Tumkaya,
            
              Xiaoling Ke,
            
              Gilberto J. Soler-Llavina,
            
              Gopi Shanker,
            
              Carsten Russ,
            
              M. Hild,
            
              Caroline Gubser Keller,
            
              J. Jenkins,
            
              Kathleen A. Worringer,
            
              Frederic D. Sigoillot,
            
              Robert J Ihry,
            
          </td>
          <td>2022-05-04</td>
          <td>ACS Chemical Biology</td>
          <td>11</td>
          <td>40</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2713e63d1c3b6691244ac038ecfa69ed82f81626" target='_blank'>Multi-condition and multi-modal temporal profile inference during mouse embryonic development</a></td>
          <td>
            
              Ran Zhang,
            
              Chengxiang Qiu,
            
              Gala Filippova,
            
              Gang Li,
            
              J. Shendure,
            
              Jean-Philippe Vert,
            
              Xinxian Deng,
            
              Christine M. Disteche,
            
              W. S. Noble,
            
          </td>
          <td>2024-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e8485a000603bb15d2382067114ca3f16414da74" target='_blank'>Cell system perturbation for time-resolved quantification of tyrosine phosphorylation in complex samples</a></td>
          <td>
            
              F. White,
            
          </td>
          <td>None</td>
          <td>2006 International Conference of the IEEE Engineering in Medicine and Biology Society</td>
          <td>0</td>
          <td>54</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c2dac0daf8c042db43565e8856a8ea72bd37b364" target='_blank'>Drug Selection via Joint Push and Learning to Rank</a></td>
          <td>
            
              Yicheng He,
            
              Junfeng Liu,
            
              Xia Ning,
            
          </td>
          <td>2018-01-23</td>
          <td>IEEE/ACM Transactions on Computational Biology and Bioinformatics</td>
          <td>15</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/a6c9109c073d99356a02722233aab66d91a64926" target='_blank'>Precision combination therapies based on recurrent oncogenic co-alterations</a></td>
          <td>
            
              Xubin Li,
            
              Elisabeth K. Dowling,
            
              Gonghong Yan,
            
              Z. Dereli,
            
              B. Bozorgui,
            
              Parisa Imanirad,
            
              J. Elnaggar,
            
              Augustin Luna,
            
              D. Menter,
            
              P. Pilié,
            
              T. Yap,
            
              S. Kopetz,
            
              C. Sander,
            
              Anil Korkut,
            
          </td>
          <td>2020-06-04</td>
          <td>bioRxiv</td>
          <td>14</td>
          <td>152</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/03636bf3b4794a837f0241f093fb577d4377a939" target='_blank'>Signaling Transduction Networks in Choroidal Melanoma: A Symbolic Model Approach</a></td>
          <td>
            
              Beatriz Santos-Buitrago,
            
              E. H. Galilea,
            
          </td>
          <td>2019-06-26</td>
          <td>None</td>
          <td>2</td>
          <td>4</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6988f0d7eb53c7b45147f7fa3eeedb42b2cd336d" target='_blank'>Methods for causal inference from gene perturbation experiments and validation</a></td>
          <td>
            
              N. Meinshausen,
            
              Alain Hauser,
            
              Joris M. Mooij,
            
              Jonas Peters,
            
              Philip Versteeg,
            
              Peter Bühlmann,
            
          </td>
          <td>2016-07-05</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>85</td>
          <td>41</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c6353798ff0125fb1817272aaceb8b40446573bf" target='_blank'>Network inference from temporal phosphoproteomics informed by protein-protein interactions</a></td>
          <td>
            
              M. Plank,
            
          </td>
          <td>2023-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/fa7e022fa51f00f63398311d74e46c64f13d62b3" target='_blank'>DeepGEEP: Data-Driven Prediction of Bacterial Biofilm Gene Expression Profiles</a></td>
          <td>
            
              Hamidreza Arjmandi,
            
              C. Corre,
            
              Hamidreza Jahangir,
            
              Adam Noel,
            
          </td>
          <td>2023-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/d8f3759b9ef99adcc9a6eda2acc518d287386ff8" target='_blank'>The regulatory network analysis of single-cell RNA-seq and single-cell ATAC-seq data using differential equation models with perturbations</a></td>
          <td>
            
              Ming Cao,
            
              Hang Mi,
            
              Qinke Peng,
            
          </td>
          <td>2022-12-06</td>
          <td>2022 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>0</td>
          <td>15</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/7e7e0356c816c41a054ac85e13333603dcef024d" target='_blank'>Computational prediction of protein interactions on single cells by proximity sequencing</a></td>
          <td>
            
              J. Xia,
            
              Hoang Van Phan,
            
              Luke Vistain,
            
              Mengjie Chen,
            
              Aly A. Khan,
            
              S. Tay,
            
          </td>
          <td>2023-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/233cd25b3e3a526cb462edbd76456311a9c24bc5" target='_blank'>Reconstruction and signal propagation analysis of the Syk signaling network in breast cancer cells</a></td>
          <td>
            
              A. Naldi,
            
              Romain M. Larive,
            
              Urszula Czerwińska,
            
              S. Urbach,
            
              P. Montcourrier,
            
              C. Roy,
            
              J. Solassol,
            
              G. Freiss,
            
              P. Coopman,
            
              O. Radulescu,
            
          </td>
          <td>2017-03-01</td>
          <td>PLoS Computational Biology</td>
          <td>13</td>
          <td>32</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/487dfb889e545b3d69ff67dfe1d0ebf9adbcf809" target='_blank'>Predicting position along a looping immune response trajectory</a></td>
          <td>
            
              Poonam Rath,
            
              Jessica A Allen,
            
              David S. Schneider,
            
          </td>
          <td>2018-10-08</td>
          <td>PLoS ONE</td>
          <td>2</td>
          <td>36</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/e44b4afef2f7b39c7a46a2176ce1348938f7d477" target='_blank'>Synergistic drug combinations and machine learning for drug repurposing in chordoma</a></td>
          <td>
            
              E. Anderson,
            
              Tammy M. Havener,
            
              Kimberley M. Zorn,
            
              Daniel H. Foil,
            
              T. Lane,
            
              Stephen J. Capuzzi,
            
              Dave Morris,
            
              A. Hickey,
            
              D. Drewry,
            
              S. Ekins,
            
          </td>
          <td>2020-07-31</td>
          <td>Scientific Reports</td>
          <td>28</td>
          <td>73</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/140fa4213c3215aac9179f6cd9f96b25ac850156" target='_blank'>Learning the structure of the mTOR protein signaling pathway from protein phosphorylation data</a></td>
          <td>
            
              A. Salam,
            
              M. Grzegorczyk,
            
          </td>
          <td>2023-01-16</td>
          <td>Journal of Applied Statistics</td>
          <td>0</td>
          <td>17</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/2605915ffc5ef03d847f49828ebaf38ee29df4ce" target='_blank'>Selection of time points for costly experiments: a comparison between human intuition and computer-aided experimental design</a></td>
          <td>
            
              Daphne Ezer,
            
              Joseph Keir,
            
          </td>
          <td>2018-04-16</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>11</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/cd13538560d6fe398d3b75ce8e50470f7667da49" target='_blank'>Clustering High-dimensional Toxicogenomics Data with Rare Signals</a></td>
          <td>
            
              Guojing Cong,
            
              Scott Auerbach,
            
          </td>
          <td>2023-12-04</td>
          <td>2023 IEEE International Conference on Data Mining Workshops (ICDMW)</td>
          <td>1</td>
          <td>1</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/6f0b35b60830cb7c38325311a4e4d477c347c958" target='_blank'>Abstract PR13: Systems and structural biology approaches to elucidate new effectors in KRAS mutant tumors</a></td>
          <td>
            
              Joshua Broyde,
            
              David Simpson,
            
              D. A. Wah,
            
              F. Giorgi,
            
              Donald Petrey,
            
              Mariano J. Alvarez,
            
              A. Silkov,
            
              Alexander Lachmann,
            
              D. Hill,
            
              M. Vidal,
            
              P. Jackson,
            
              B. Honig,
            
              A. Sweet-Cordero,
            
              A. Califano,
            
          </td>
          <td>2015-11-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>115</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/07468cfe9e60fe6a071dc6d29794c8c8b6193e64" target='_blank'>TSMiner: a novel framework for generating time-specific gene regulatory networks from time-series expression profiles</a></td>
          <td>
            
              Mingfei Han,
            
              Xian Liu,
            
              Wen Zhang,
            
              Mengnan Wang,
            
              Wenjing Bu,
            
              Cheng Chang,
            
              Miao Yu,
            
              Yingxing Li,
            
              Chunyan Tian,
            
              Xiaoming Yang,
            
              Yun-ping Zhu,
            
              F. He,
            
          </td>
          <td>2021-07-27</td>
          <td>Nucleic Acids Research</td>
          <td>2</td>
          <td>66</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/b822e494292e7a44173abfef978a7e61987d4a1c" target='_blank'>Deciphering the History of ERK Activity from Fixed-Cell Immunofluorescence Measurements</a></td>
          <td>
            
              A. Ram,
            
              M. Pargett,
            
              Yongin Choi,
            
              D. Murphy,
            
              Markhus Cabel,
            
              N. Kosaisawe,
            
              G. Quon,
            
              John Albeck,
            
          </td>
          <td>2024-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/33bdedf99ff51fac3e2862b18aab126b9338ea05" target='_blank'>Powerful and accurate detection of temporal gene expression patterns from multi-sample multi-stage single-cell transcriptomics data with TDEseq</a></td>
          <td>
            
              Yue Fan,
            
              Lei Li,
            
              Shiquan Sun,
            
          </td>
          <td>2024-03-31</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/1540b4289465b1f957b3ab41dee1eed31b23e277" target='_blank'>Cue-Signal-Response Analysis of TNF-Induced Apoptosis by Partial Least Squares Regression of Dynamic Multivariate Data</a></td>
          <td>
            
              K. Janes,
            
              J. Kelly,
            
              S. Gaudet,
            
              J. Albeck,
            
              P. Sorger,
            
              D. Lauffenburger,
            
          </td>
          <td>None</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>117</td>
          <td>119</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c567fda43e20c10b65e74cf9103d7989b54ea3d5" target='_blank'>Phenotype Prediction Using Regularized Regression on Genetic Data in the DREAM5 Systems Genetics B Challenge</a></td>
          <td>
            
              Po-Ru Loh,
            
              George Tucker,
            
              B. Berger,
            
          </td>
          <td>2011-12-28</td>
          <td>PLoS ONE</td>
          <td>13</td>
          <td>67</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/c516bed13979341cf231d39e4be643ccfbf06105" target='_blank'>Exploring the Contextual Sensitivity of Factors that Determine Cell-to-Cell Variability in Receptor-Mediated Apoptosis</a></td>
          <td>
            
              S. Gaudet,
            
              S. Spencer,
            
              William W. Chen,
            
              P. Sorger,
            
          </td>
          <td>2012-04-01</td>
          <td>PLoS Computational Biology</td>
          <td>86</td>
          <td>104</td>
        </tr>
    
        <tr>
          <td><a href="https://www.semanticscholar.org/paper/de0ff8a5542aafa2ac0581269708d5ed4aec5292" target='_blank'>Regularization Techniques for Inverse Problems in Molecular Biology</a></td>
          <td>
            
              S. Provencher,
            
              R. H. Vogel,
            
          </td>
          <td>None</td>
          <td></td>
          <td>64</td>
          <td>22</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    // columnDefs: [
    //     {"className": "dt-center", "targets": "_all"},
    //     // set width for both columns 0 and 1 as 25%
    //     { width: '40%', targets: 4 }

    //   ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
</script>
</html>